Reviewer 1

Rando et al., provide a thoughtful and comprehensive review of the scientific literature relevant to the COVID-19 pandemic. This review will provide a useful resource for the scientific community but is missing some crucial information and would be improved by a more clear description of how this virus is detected and the limitations of these methods.

**
We are very grateful for your supportive words about this project and for your guidance in making our review of these topics more complete.
**

Major:
1.1 Heparin sulfate dependency: The 'Pathogenic mechanisms' section is lacking any reference to the dependence of SARS-Cov-2 binding to heparin sulfate which has been cited over 100 times since its publication in November and has been supported by work from numerous labs:
https://www.sciencedirect.com/science/article/abs/pii/S0092867420312307

**
This oversight has been corrected.
A discussion of heparan sulfate has now been added to the section "Pathogenic Mechanisms of Coronaviruses."
**

1.2 Furin discussion: The description of furin in this manuscript is confusing; it is highlighted in the conclusion as a mutation that has allowed for increased virulence, but is not mentioned in the opening paragraphs of the paper where host proteases are described (e.g. lines 229-235, line 279), or the section describing the hinge-like movement of the spike protein which is induced by furin (lines 591-603) and only mentioned fleetingly in the middle of a the next paragraph. A more thorough discussion of the furin-site insertion in the SARS-CoV-2 genome (and what information is still missing) earlier or more prominently in the paper is warranted.

**
Thank you for catching this oversight.
Several additions have been made to the text to address the role of furin.
The paragraph that introduces proteolytic priming (previously lines 229-235) has been expanded to discuss the furin injection site insertion and the fact that _in vitro_ studies suggest it increases pathogenesis.
The following line was added to the section on Host Cell Susceptibility (previously beginning at 279): 
"Additionally, the addition of the furin site insertion at the S1/S2 boundary means that SARS-CoV-2 does not require TMPRSS-2 when furin, an ubiquitously expressed endoprotease [@doi:10.1038/nrm934], is present, enabling cell-cell fusion independent of TMPRSS-2 availability [@doi:10.1016/j.molcel.2020.04.022]."
The section about the hinge-like movement of the spike protein (previously 591-603) has been expanded to include: 
"Spike proteins cleaved at the furin-like binding site are substantially more likely to take an open conformation (66%) than those that are uncleaved (17%) [@doi:10.1038/s41594-020-0468-7]."
In the following paragraph, the discussion of the acquisition of the furin insertion site has been moved to the beginning of the paragraph and modified to read: 
"The furin recognition site at the S1/S2 junction is likely to increase pathogenicity via destabilization of the spike protein during fusion to ACE2 and the facilitation of cell-cell adhesion [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507; @doi:10.1038/s41594-020-0468-7; @doi:10/bvgh5b; @doi:10.1006/viro.1999.9716; @doi:10.1016/j.isci.2020.101212]."
**

1.3 Detected versus infectious: The distinction between RT-qPCR detection and viral infection assays in cell culture models should be more clear. Although this point is acknowledged (e.g. lines 713-714) it is not clear whether viral RNA or infectious virus is being described at certain points in the review (i.e. line 663) and a brief description of the advantages and limitations of these methods would be useful for the reader.

**
This is a very important point, so to emphasize it we have added a section entitled "Quantifying Viral Presence" that outlines the different information conveyed by RT-PCR and _in vitro_ cultivation.
We have also expanded the review of aerosol transmission to more adequately contextualize the study mentioned in the above comment.
**

Minor:
1.4 Typos

    Lines 205; The legend of Figure 1 incorrectly states that RNA polymerase is a protease

**
The word "protease" has been changed to "enzyme."
**

    Line 709; typo -> 'either typical or atypical'

**
This change has been made.
**

    Line 755; 'content level' should be 'continental level'?

**
This change has been made.
**

    Line 788; what does adding the 'exposed' variable in this model mean and why is that important for SARS-CoV-2

**
This sentence has been modified to read:
"To capture the dynamics of SARS-CoV-2 accurately, the addition of a fourth compartment, i.e. a susceptible-exposed-infectious-recovered model, may be appropriate because such models account for the relative lengths of incubation and infectious periods [@doi:10.1038/s41598-020-76563-8]."
**

    Line 852; typo, duplicate 'has coordinated'

**
This typo has been corrected.
**

    Line 874; typo, "as in the current pandemic"

**
This change has been made.
**

1.5 This work is important not only for the current moment, but for contextualizing this virus for response to future outbreaks; in my opinion it would be nice to have a sentence in the discussion to this effect (this review represents a ton of work - the effort is worthwhile!)

**
Thank you for pointing this out.
We have added two sentences to the first paragraph of the conclusion that read: 
"With the emergence of three devestating HCoV over the past twenty years, emergent viruses are likely to represent an ongoing threat.
Contextualizing SARS-CoV-2 alongside other viruses serves not only to provide insights that can be immediately useful for combatting this v
irus itself, but may also prove valuable in the face of future viral threats."
Additionally, the last sentence of the manuscript is now: "In the future, interdisciplinary work on SARS-CoV-2 and COVID-19 may guide a response to a new viral threat."
The potential for SARS-CoV-2 research to contribute to the handling of future emerging HCoV was also added to the "Importance" section of the abstract.
**

Reviewer 2

Overall, this was a very comprehensive, thorough review of the current state of knowledge of SARS-CoV2. The manuscript is well written and extensively researched, with great attention to the details of the outbreak with regards to data from various regions of the world, providing an extremely useful resource for the scientific community. The manuscript is well researched and well written for a general scientific audience. I only had a handful of minor concerns, mostly for word choice/organization for clarity but I would leave these changes at the discretion of the authors as they do not detract from the overall narrative of the manuscript.

**
Thank you very much for your encouraging feedback.
**

Minor Points

2.1 Line 196 - insert "the" prior to replication.

**
This change has been made.
**

2.2 Lines 213-214 - wording was a bit odd, might suggest adding "a process conserved among coronaviruses" to the previous sentence.

**
This change has been made.
**

2.3 Line 250-252 - might suggest excluding skin and just keeping mucus, the way this reads suggests the respiratory mucosa provides no protection at all- is this true?

**
The question you raised is interesting -- the Liu et al. (2016) reference, which investigated viral dissemination of SARS-CoV-1 in rhesus macaques, states, "Many viruses, including SARS-CoV, breach this mucosal barrier by infecting the epithelium directly."
This conclusion cites two articles, one of which (Richt et al., 2012 in PLoS One) examined swine influenza A virus (H2N3) in macaques and found higher replication of swine H2N3 than human H2N2 in the nasal mucosa and lung tissue.
The other (Nicholls et al., 2006 in PLoS Medicine) reported the cellular localization of the SARS-CoV-1 virus in deceased SARS patients using a monoclonal antibody, concluding that the virus's "chief target" is the pulmonary alveolar epithelium.
Based on these citations, it doesn't seem appropriate to conclude that the respiratory mucosa does not provide protection at all, but rather that viruses vary in their ability to bypass its protection.
Based on this, the sentence has been rephrased to read: "Infecting the epithelium can help viruses such as SARS-CoV-1 bypass the physical barriers, such as mucus, that comprise the immune system's first line of defense."
**

2.4 Line 275 - suggest changing "can facilitate" to "may facilitate"

**
This change has been made.
**

2.5 Line 311 - reference was not highlighted in blue- not sure if this was a formatting issue but may want to check to make sure this reference is correct.

**
Thank you for catching this error.
The correct reference is a June 2020 letter to the editor in the New England Journal of Medicine (doi:10.1056/NEJMc2010419).
This line has been corrected to refer to the correct article.
**

2.6 Might suggest moving up the Pediatric Presentation (Lines 428-454) to immediately before cytokine release syndrome as it follows the clinical presentation and directly leads into the cytokine release syndrome.

**
This change has been made.
**

2.7 The intro paragraph for the cytokine release syndrome is quite long and could be readily shortened (or even eliminated) without sacrificing the relevant information pertaining to COVID-19 (which is primarily the second 2 paragraphs).

**
This section has been shortened and reorganized to remove extraneous information.
**

2.8 Might suggest moving the cytokine section to be under the Systems-Level Effects rather than Clinical Presentation section, but it could fit in either.

**
The modified section has been left under the Clinical Presentation section, but the Systems-Level Effects section has been renamed to "Insights from Systems Biology" to indicate that it focuses on research in omics rather than systems-level analyses more broadly.
**

2.9 Line 505 - statement regarding other respiratory viruses encoding antagonists to IFN response should reference the relevant studies.

**
This line has been amended to include references to some broad discussions of this feature of viruses, as well as specific instances within betacoronaviruses.
It now reads: "Other respiratory viruses have been found to encode antagonists to the IFN response [@doi:10.1016/j.mib.2010.05.009; @doi:10.1371/journal.pone.0112014], including SARS-CoV-1 [@doi:10.1128/JVI.01782-06] and MERS-CoV [@doi:10.1128/JVI.01845-13]."
**

2.10 Line 513 - word choice- replace extent with magnitude.

**
This change has been made.
**

2.11 Lines 557-559 - this was a really intriguing statement- can the authors provide an additional sentence or two to provide additional context to why this might be the case?

**
The authors of the study described in this passage state the following: "We compared our SARS-CoV-2 interaction map with those of ten other pathogens (Fig. 2e) and found that West Nile virus and _Mycobacterium tuberculosis_ had the most similar host-protein interaction partners. The association with _M. tuberculosis_ is of particular interest as it also infects lung tissue."

On a proximal level, the apparent similarity between the protein-protein interaction networks of SARS-CoV-2 and _M. tuberculosis_ could be an artifact of the pathogens selected for comparison: as the authors themselves point out, both pathogens are known to affect the lungs, while the other pathogens used for comparison are not known for respiratory symptoms (hepatitis C, human immunodeficiency virus, human papillomavirus, Kaposi's sarcoma-associated herpesvirus, and West Nile virus).
Therefore, the original study seems to suggest that altered regulation in the lungs might be driving this similarity.
The sentence here has been rephrased to make this more clear, and now reads: 
"The fact that the host-pathogen interactome of the bacterium *Mycobacterium tuberculosis* was found to be similar to that of SARS-CoV-2 suggests that changes related to lung pathology might comprise a significant contributor to these expression profiles."
**

2.12 I understand that this field moves ridiculously fast, but in the transmission section it might be worth including the most recent variants that are predicted to have increased transmission.

**
Thank you for this suggestion.
A discussion of variants of concern has been added to the section "Molecular Signatures and Transmission," which has been renamed to "Molecular Signatures, Transmission, and Variants of Concern" and moved earlier in the text.
We have also made additional updates to reflect changes in the field since the paper was first reviewed (e.g., pediatric hospitalizations, reproduction number estimates).
**

Reviewer 3

Overall, the manuscript by Rando et al seeks to review and condense the current corpus of knowledge for SARS-CoV-2. The authors do a great job of making the findings of various studies easy to understand. Overall, this reviewer believes that it will be an excellent resource to the scientific community. This reviewer recommends that the author incorporates a handful of suggested changes (below) to improve the already impressive state of this manuscript.

3.1 Structure of Coronaviruses, paragraph 1: Please modify the definition of “non-segmented”. It refers more to the fact that it is a single molecule of ssRNA, than to whether it is contained in a capsid.

**
This error has been correct.
The text now reads:
"They are non-segmented, which means the viral genome is a single continuous strand of DNA, and are enveloped, which means that the genome and capsid are encased by a lipid bilayer."
**

3.2 Cytokine release syndrome, paragraph 1: Please modify the wording from “dysregulated systemic inflammation can cause sepsis…” to something similar to “dysregulated systemic inflammation can contribute to pathogenesis associated with sepsis”. The reviewer is unsure about the relevance of this sentence since sepsis is associated with bacterial infection rather than viral infection.

**
The section "Cytokine release syndrome" has been rewritten in line with comments from reviewer 2 above (comment 2.7).
While most sepsis cases are attributed to bacteria, concerns have been raised that viral sepsis may be underidentified broadly [@doi:10.1183/16000617.0038-2020] and that it may be particularly relevant to SARS-CoV-2 [@doi:10/ggsps7].
The relationship between viruses and sepsis is an oversight in the current manuscript, so the paragraph indicated here has been modified to include background on viral sepsis.
It now reads: "A dysregulated immune response can cause significant damage to the host [@isbn:9780815332183; @isbn:9780071283663; @doi:10.1016/j.chom.2020.05.009] including pathogenesis associated with sepsis. 
Sepsis, which can lead to multi-organ failure and death [@doi:10.1128/MMBR.05015-11; @pmid:22142805], is traditionally associated with bacterial infections, but sepsis associated with viral infections may be underidentified [@doi:10.1183/16000617.0038-2020], and sepsis has emerged as a major concern associated with SARS-CoV-2 infection [@doi:10/ggsps7]."
**

3.3 Cytokine release syndrome, paragraph 2: The relevance of sepsis to this paragraph is also unclear, unless the authors are comparing SARS-CoV-2 infection to other respiratory diseases caused by bacteria. Specifying the relevance of this comparison would be helpful.

**
As with 3.2 above, this paragraphs has been restructured.
We hope the changes serve to emphasize the concerns about sepsis in the context of SARS-CoV-2.
**

3.4 Molecular signatures and transmission: It may be helpful to the readers if the authors included a brief synopsis of viral comparative genomics analyses that have been performed including:

    https://www.sciencedirect.com/science/article/abs/pii/S0140673620317578
    https://www.sciencedirect.com/science/article/pii/S0092867421000805
    https://jvi.asm.org/content/94/17/e00790-20.abstract
    https://www.sciencedirect.com/science/article/pii/S0092867420304864

**
This section of the paper has been expanded based on this suggestion and based on Reviewer 2's comment 2.12.
The four references suggested have been incorporated into this paper and into the separate Evolution manuscript associated with this project.
Additionally, we have included analyses of variants of concern. 
**

3.5 Conclusions, paragraph 3: The same reference (222) is included twice for the same statement.

**
This duplication has been removed.
**
