## Pathogenesis

### Mechanism of Host Infection by SARS-CoV-2

This section would also be great for the introduction of zoonotic diseases which has been shown to be the origin of SARS-CoV2.

#### Primary Transmission and Viral Entry

ACE2 receptors have been identified as the binding site for SARS-CoV-2 in animals and humans [@doi:10.1038/nature02145; @doi:10.1007/s00134-020-05985-9; @doi:10.1128/JVI.00127-20]. 
The virus invades host cells by using ACE2 for entry and the serine protease TMPRSS2 for S protein priming. 
It then fuses with the membrane and enters the cell via endosomes [@doi:10.1016/j.cell.2020.02.052]. 
The C-terminal domain of the SARS-CoV-2 S protein was identified as the key region of the virus that interacts with the human ACE2 (hACE2) receptor. 
The crystal structure of the C-terminal domain of SARS-CoV-2 S protein in complex with human ACE2 reveals stronger interaction and higher affinity for receptor binding than SARS-CoV [doi:10.1016/j.cell.2020.03.045]. 

#### Viral Replication and Spreading

-Basic introduction into replication cycle
-What are the routes of transmission

#### Reproduction Number and Dynamics of Transmission

Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
The basic reproduction number, R~0~, is the expected number of new infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~, for example as some fraction of the population becomes infected, or as interventions are put into place.
R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using mathematical modeling.
Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@doi:10.1126/science.272.5258.74]. 

R~0~ for COVID-19 is estimated to lie in the range R~0~=1.4-6.5 [@doi:10/ggjvr7; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221]; estimates vary considerably depending on the data and the methods used.
Data-derived estimates (i.e. those that do not incorporate SIR-type models into their analysis) typically predict lower values of R~0~.
For data-derived estimates, in one study of international cases, the predicted value is R~0~=1.7 [@doi:10.1101/2020.03.21.20040329], in China (both Hubei province and nationwide), the value is predicted to lie in the range R~0~=2.0-3.6 [@doi:10.2139/ssrn.3524675; @doi:10.1101/2020.01.25.919787; @doi:10/ggjvr7], and on a cruise ship where an outbreak occurred, predicted R~0~=2.28 [@doi:10.1016/j.ijid.2020.02.033].
SIR model-derived estimates of R~0~ range from 2.0 - 6.5 in China [@doi:10.3390/jcm9020462; @doi:10.1101/2020.01.27.20018952; @doi:10.1101/2020.01.23.916726; @doi:10.1101/2020.01.23.20018549] to R~0~=4.8 in France [@doi:10.1101/2020.03.22.20040915]. 
Using the same model as for the French population, this study estimated R~0~=2.6 in South Korea [@doi:10.1101/2020.03.22.20040915], which is consistent with other studies [@tag:Park2020_distancing].
From a meta-analysis of studies estimating R~0~, [@doi:10.1093/jtm/taaa021] predict the median as R~0~=2.79. 

Inference of the effective reproduction number can provide insight into how populations respond to an infection, and the effectiveness of interventions.
In China, R~t~ was predicted to lie in the range 1.6-2.6 in Jan 2020, before travel restrictions [@doi:10/ggptcf].
R~t~ decreased from 2.35 one week before travel restrictions were imposed (Jan 23, 2020), to 1.05 one week after.
Using their model, the authors also estimate the probability of new outbreaks occurring: the probability of a single individual exporting virus causing a large outbreak is 17-25% assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%.
An independent study came to similar conclusions: in a two-week period before Jan 23 finding R~t~=2.38, and decreasing to R~t~ = 1.34 (using data from Jan 24 to Feb 3) or R~t~=0.98 (using data from Jan 24 to Feb 8) [@doi:10.1126/science.abb3221].
In South Korea, R~t~ was inferred for Feb-Mar 2020 in two cities: Daegu (the center of the outbreak), and Seoul [@tag:Park2020_distancing].
Metro data was also analyzed to estimate the effects of social distancing measures.
R~t~ decreased in Daegu from around 3 to <1 over the period that social distancing measures were introduced.
In Seoul, R~t~ decreased slightly, but remained close to 1 (and larger than R~t~ in Daegu).
This highlights that social distancing measures appeared to work to contain the infection in Daegu, but that in Seoul, R~t~ remains above 1, thus secondary outbreaks are possible.
It also shows the importance of region-specific analysis: the large decline in case load nationwide is mainly due to the Daegu region, and could hide persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do.
In Iran, estimates of R~t~ declined from 4.86 in the first week to 2.1 by the fourth week after the first cases were reported [@doi:10.1101/2020.03.20.20038422].
In Europe, analysis of 11 countries inferred the dynamics of R~t~ over a time range from the beginning of the outbreak until March 28, 2020, by which point most countries had implemented major interventions (such as school closures, public gathering bans, and stay-at-home orders) [@doi:10.25561/77731].
Across all countries, the mean R~t~ before interventions began was estimated as 3.87; R~t~ varied considerably, from below 3 in Norway to above 4.5 in Spain.
After interventions, R~t~ decreased by an average of 64% across all countries, with mean R~t~=1.43.
The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden (note that this is in part because Sweden did not implement social distancing measures on the same scale as other countries).
The study concludes that while large changes in R~t~ are observed, it is too early to tell whether the interventions put into place are sufficient to decrease R~t~ below 1.

### Immune Response to SARS-CoV-2

-Cellular responses to SARS-CoV-2 infection
-What is causing neutropenia and lymphopenia observed in COVID-19 patients
-Antibody production against SARS-CoV-2 by patient who recovered vs patient who did not recover
-Cytokines and other soluble factors contribution to immune response

### Systems-level approaches for understanding SARS-CoV-2 pathogenesis

Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].

The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet.
A significant breakthrough in SARS-CoV-2 research was achieved through the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].
Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequencing of clinical samples collected from bronchoalveolar lavage fluid (BALF) [@doi:10.1038/s41586-020-2008-3; @doi:10.1038/s41586-020-2012-7] or from BALF, throat swabs, or isolates of the virus cultured from BALF [@doi:10/ggjr43].
Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) [@doi:10/ggjr43].
However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), indicating a possible origin in bats [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7].

The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection.
Availability of the genome sequence of SARS-CoV-2 enhances the potential for subsequent proteome-level studies to provide further mechanistic insights into the virus' complex pathogenesis.
Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, was reported very recently [@doi:10.1126/science.abb2507].
Even though no comprehensive proteomic analysis of the pathogen or of patients suffering from its infection has yet been reported, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.21203/rs.3.rs-17218/v1].
The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.

There is a high level of sequence homology between SARS-CoV-2 and SARS-CoV, and sera from convalescent SARS-CoV patients might show some efficacy to cross-neutralize SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052].
However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, prohibiting effective cross-neutralization across different SARS strains [@doi:10.1101/2020.03.21.990770].
Consequently, earlier proteome-level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074].
Considering the paucity of omics-level big data sets for SARS-CoV-2 up until now, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.
