## Evolutionary and Genomic Analysis of SARS-CoV-2

### Abstract

### Introduction

### Initial Characterization of SARS-CoV-2

The first genome sequence of the virus was released on January 3, 2020. 
It revealed that the cluster of pneumonia cases seen in Wuhan were caused by a novel coronavirus [@doi:10.46234/ccdcw2020.017].
Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequences developed from clinical samples collected from the lower respiratory tract, namely bronchoalveolar lavage fluid (BALF), and the upper respiratory tract, in the form of throat and nasopharyngeal swabs [@doi:10.1038/s41586-020-2008-3; @doi:10.1038/s41586-020-2012-7; @doi:10/ggjr43].
Analysis of the SARS-CoV-2 genome revealed significant sequence homology with two coronaviruses known to infect humans, with about 79% identity to SARS-CoV-1 and 50% to MERS-CoV [@doi:10/ggjr43].
However, in this analysis, the highest degree of similarity was observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) [@doi:10/ggjr43].
Other analyses have reported even greater similarity between SARS-CoV-2 and the bat coronavirus BatCoV-RaTG13, with identity as high as 96.2% [@doi:10.1038/s41586-020-2012-7; @doi:10.1093/nsr/nwaa036], and the closely related pangolin coronavirus [@doi:10.1101/2020.02.19.950253].
This evidence therefore suggests the SARS-CoV-2 virus is the result of zoonotic transfer of a virus from bats to humans.
Nevertheless, some fragments between SARS-CoV-2 and RATG13 differ by up to 17%, suggesting a complex natural selection process during zoonotic transfer.
While the _S_ region is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253], there is no consensus about the origin of _S_ in SARS-CoV-2, as it could potentially be the result either of recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153].
Though the intermediate host serving as the source for the zoonotic introduction of SARS-CoV-2 to human populations has not yet been identified, the SARS-CoV-2 virus has been placed within the coronavirus phylogeny through comparative genomic analyses.
Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
Indeed, the World Health Organization (WHO) have published their intentions to thoroughly investigate the origins of SARS-CoV-2 [@url:https://www.who.int/docs/default-source/coronaviruse/20200802-tors-chn-and-who-agreed-final-version.pdf?sfvrsn=5323728d_2&download=true].
While the position of the SARS-CoV-2 virus within the coronavirus phylogeny has been largely resolved, the functional consequences of molecular variation between this virus and other viruses, such as its bat and pangolin sister taxa or SARS-CoV-1, remain unknown [@doi:10.1038/s41591-020-0820-9, @doi:10.1038/s41586-020-2012-7].
Fortunately, the basic genome structure of coronaviruses is highly conserved, and insight into the mechanisms the virus uses to enter cells, replicate, and spread is available from prior research on coronaviruses, which has been instrumental in the mobilization of global research to understand the biology of SARS-CoV-2. 

Additionally, worldwide sequencing of viral samples has provided some preliminary insights into possible mechanisms of adaptation in the virus and the detection of novel variants, and omics-based analysis of patient samples has elucidated some of the biological changes the virus induces in its human hosts.

### Coronaviruses and Animal Hosts

Coronaviruses have long been known to infect animals and have been the subject of veterinary medical investigations and vaccine development efforts due to their effect on the health of companion and agricultural animals [@doi:10/ckfn8b].
<!-- TO DO: Expand on the relationship between coronaviruses and animals -- which taxa are most often affected? how severe are veterinary infections? how has vaccine development worked for these diseases?--> 

<!-- TO DO: Be care to check overlap between this section and 20.pharmaceuticals.md. Trim to address just the important information about human-coronavirus interactions that feeds into an evolutionary perspective of how these host/virus interactions evolve across taxa and over time -->
Most coronaviruses show little to no transmission in humans.
However, it is thought that approximately one-third of common cold infections are caused by four seasonal human coronaviruses (HCoV): _Human coronavirus 229E_ (HCoV-229E), _Human coronavirus NL63_ (HCoV-NL63), _Human coronavirus OC43_ (HCoV-OC43), and _Human coronavirus HKU1_ (HCoV-HKU1) [@doi:10.1038/s41579-018-0118-9; @doi:10.3390/diseases4030026; @doi:10.1016/j.jcv.2018.01.019].
The first HCoV were identified in the 1960s: HCoV-229E in 1965 [@doi:10.3181/00379727-121-30734] and HCoV-OC43 in 1967 [@doi:10.1073/pnas.57.4.933].
Both of these viruses typically cause cold-like symptoms, including upper and lower respiratory infections [@doi:10.1186/1743-422X-1-7; @pmid:20700397; @doi:10.1186/s40779-020-00279-z], but they have also been associated with gastrointestinal symptoms [@doi:10.1111/j.1440-1754.2007.01246.x].
Two additional HCoV were subsequently identified [@doi:10.1038/nm1024; @doi:10.1128/JVI.79.2.884-895.2005].
In 2003, HCoV-NL63 [@doi:10.1038/nm1024] was first identified in a 7-month-old infant and then in clinical specimens collected from seven additional patients, five of whom were infants younger than 1 year old and the remainder of whom were adults.
CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and then found in samples collected from a second adult patient [@doi:10.1128/JVI.79.2.884-895.2005].
These viruses are associated with respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612], and also have gastrointestinal involvement in some cases [@doi:10.1111/j.1440-1754.2007.01246.x].
In addition to these relatively mild HCoV, however, highly pathogenic human coronaviruses have been identified, including _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV or SARS-CoV-1) and _Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].

At the time that SARS-CoV-1 emerged in the early 2000s, no HCoV had been identified in almost 40 years [@doi:10.1038/nrmicro.2016.81].
The first case of SARS was reported in November 2002 in the Guangdong Province of China, and over the following month, the disease spread more widely within China and then into several countries across multiple continents [@doi:10.1093/ajcp/aqaa029; @doi:10.1038/nrmicro.2016.81].
Unlike previously identified HCoV, SARS was much more severe, with an estimated death rate of 9.5% [@doi:10.1093/ajcp/aqaa029].
It was also highly contagious via droplet transmission, with a basic reproduction number (R~0~) of 4 (i.e., each person infected was estimated to infect four other people) [@doi:10.1093/ajcp/aqaa029].
However, the identity of the virus behind the infection remained unknown until April of 2003, when the SARS-CoV-1 virus was identified through a worldwide scientific effort spearheaded by the WHO [@doi:10.1038/nrmicro.2016.81].
SARS-CoV-1 belonged to a distinct lineage from the two other HCoV known at the time [@doi:10.1093/ajcp/aqaa029].
By July 2003, the SARS outbreak was officially determined to be under control, with the success credited to infection management practices [@doi:10.1038/nrmicro.2016.81].
A decade later, a second outbreak of severe respiratory illness associated with a coronavirus emerged, this time in the Arabian Peninsula.
This disease, known as Middle East respiratory syndrome (MERS), was linked to another novel coronavirus, MERS-CoV.
The fatality rate associated with MERS is much higher than that of SARS, at almost 35%, but the disease is much less easily transmitted, with an R~0~ of 1 [@doi:10.1093/ajcp/aqaa029].
Although MERS is still circulating, its low reproduction number has allowed for its spread to be contained [@doi:10.1093/ajcp/aqaa029].
The COVID-19 pandemic is thus associated with the seventh HCoV to be identified and the fifth since the turn of the millennium, though additional HCoVs may be in circulation but remain undetected.

SARS-CoV-1 and MERS-CoV were ultimately managed largely through infection management practices (e.g., mask wearing) and properties of the virus itself (i.e., low rate of transmission), respectively [@doi:10.1038/nrmicro.2016.81; @doi:10.1093/ajcp/aqaa029].
Vaccines were not used to control either outbreak, although vaccine development programs were established for SARS-CoV-1 [@doi:10.3390/v11010059].
In general, care for SARS and MERS patients focuses on supportive care and symptom management [@doi:10.1093/ajcp/aqaa029].
Clinical treatments for SARS and MERS developed during the outbreaks generally do not have strong evidence supporting their use.
Common treatments included Ribavirin, an antiviral, often in combination with corticosteroids or sometimes interferon (IFN) medications, which would both be expected to have immunomodulatory effects [@doi:10.1038/nrmicro.2016.81].
However, retrospective and _in vitro_ analyses have reported inconclusive results of these treatments on SARS and the SARS-CoV-1 virus, respectively [@doi:10.1038/nrmicro.2016.81].
IFNs and Ribavirin have shown promise in _in vitro_ analyses of MERS, but their clinical effectiveness remains unknown [@doi:10.1038/nrmicro.2016.81].
Therefore, only limited strategies can be adopted for the pharmaceutical management of COVID-19 from previous severe HCoV infections.
Research in response to prior outbreaks of HCoV-borne infections, such as SARS and MERS, have, however, provided a strong foundation for hypotheses about the pathogenesis of SARS-CoV-2 as well as potential diagnostic and therapeutic approaches.

### Zoonotic Transfer of Coronaviruses

<!-- TO DO: How does zoonotic transfer happen? What role has this played in previous HCoV outbreaks? How do viruses evolve during host species changes?-->

### Evolution of the SARS-CoV-2 Virus

#### Emergence of SARS-CoV-2

<!-- TO DO: Which zoonotic viruses have been identified as likely relatives of SARS-CoV-2? How do we know this viral species evolved naturally (as opposed to the conspiracy theories? What differs between SARS-CoV-2 and its close relatives?-->

#### Evolution of SARS-CoV-2 Variants

Evolution in SARS-CoV-2 has also been observed over a short timescale.
After zoonotic transfer, SARS-CoV-2 continued evolving in the human population [@doi:10.1186/s12967-020-02344-6].
The SARS-CoV-2 mutation rate is moderate compared to other RNA viruses [@doi:10.1016/j.meegid.2020.104351], which likely restricts the pace of evolution in SARS-CoV-2.
Nevertheless, genomic analyses have yielded statistical evidence of ongoing evolution.
Initially, two known variants of the spike protein emerged that differed by a single amino acid at position 614 (G614 and D614), and there is evidence that G614 had become more prevalent than D614 by June 2020 [@doi:10.1016/j.cell.2020.06.043].
While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, this hypothesis has not yet been tested due to a lack of data [@doi:10.1016/j.cell.2020.06.040].
Another study [@doi:10.1016/j.meegid.2020.104351] identified 198 recurrent mutations in a dataset of 7,666 curated sequences.
This pattern of convergent evolution at some sites could indicate that certain mutations confer an adaptive advantage.
While it is evident that SARS-CoV-2 exhibits moderate potential for ongoing and future evolution, the relationship between mutations and pathogenicity is not yet known.
Additional data is needed in order to understand patterns of evolutionary change and whether they are likely to affect virulence.

Several factors could promote the evolution of SARS-CoV-2, including host immunodeficiency and transient exposure to antibodies directed against SARS-CoV-2 proteins. 
A single case study of SARS-CoV-2 infection in an immunocompromised female with chronic lymphocytic leukemia and hypogammaglobulinemia [@doi:10.1016/j.cell.2020.10.049] suggested that an accelerated evolution of the virus could occur in conditions of immunodeficiency. 
A first administration of convalescent plasma did not clear the virus, and an ensuing increase in the genomic diversity in the samples was observed, suggesting an accelerated evolution due to selection pressure. 
A second administration of convalescent plasma cleared the virus from the host 105 days after the initial diagnosis. 
However, throughout the duration of infection, the patient was asymptomatic but contagious.
A second single case study in a 45-year old male with antiphospholipid syndrome [@doi:10.1056/NEJMc2031364] confirmed the earlier results, providing evidence of persistent COVID-19 symptoms in an immunocompromised patient for 154 days following diagnosis, ultimately leading to the death of patient. 
The treatments administered included remdesivir and the Regeneron anti-spike protein antibody cocktail. 
Genomic analyses of the patient's nasopharyngeal swabs confirmed an accelerated evolution of the virus through mutations in the spike gene and the receptor-binding domain. 
In summary, these two case studies suggested an accelerated evolution and persistent shedding of the virus in conditions of immunodeficiency. 
In particular, the first case highlighted the role of convalescent plasma in creating escape variants. 
In fact, one study [@doi:10.1101/2020.12.28.424451] exposed the SARS-CoV-2 virus to convalescent plasma _in vitro_ repeatedly to see how much plasma was required to neutralize the virus.
The results of the first six exposures were similar, but they reported that after the seventh exposure (on day 45), the amount of plasma required began to increase.
In analyzing the viral variants present, they found that this viral escape was promoted by the sudden accumulation of mutations, especially in the receptor-binding domain (RBD) and N-terminal domain (NTD), that quickly rose in frequency.
By the thirteenth exposure (day 85), the virus had evolved three mutations and could not longer be neutralized by the plasma used.
Taken together, these observations suggest that evolutionary analyses of SARS-CoV-2 can provide crucial information about the conditions that promote resistance in SARS-CoV-2 and the kinetics of how resistance develops, information which will be important for understanding the implications of how vaccine regimens are designed and whether/when next-generation vaccines will be needed.

When variants occur, they can rise in frequency by chance or through an adaptive process that confers a competitive advantage to the virus.
Variants that had the D614G mutation in the spike glycoprotein seemed to spread faster. <!-- TO DO: add more history of this variant?-->
However, it has been suggested that the mutation rose in frequency due to early chance events rather than by adaptive events [@doi:10.1001/jama.2020.27124].
Another mutation, Y453F, that occurred in the receptor binding domain of _S_, was first detected in mink; however, the transmission to humans has been established.
In mink, this mutation conferred an advantage by increasing the affinity towards ACE2 [@doi:10.1126/science.abe5901].
Similarly, N501Y mutation induces an increased affinity towards human ACE2 and has been involved in the dominance of B.1.1.7 by outcompeting other variants [@doi:10.1101/2020.12.24.20248822].
Therefore, genomic surveillance is essential to prevent the emergence of super-spreaders [@doi:10.1126/science.abf2946].

Emerging methods are being applied to this problem in an effort to understand which mutations are most likely to be of significant concern.
Novel machine learning methods were developed to predict the mutations in the sequence that promote viral escape.
While they preserve the pathogenicity of the virus, escape mutations change the virus's sequence to evade detection by the immune system. 
By using tools from natural language processing (NLP), viral escape was modeled as an NLP problem [@doi:10.1126/science.abd7331] where a modification makes a sentence grammatically correct but semantically different.
Therefore, language models of viruses could predict mutations that change the presentation of the virus to the immune system but preserve its infectivity.

#### Variants of concern and variants under surveillance

Viral replication naturally leads to the occurrence of mutations, and thus to genetic variation [@doi:10.1038/s41564-020-0690-4].
However, due to an intrinsic RNA proof-reading process in the SARS-CoV-2 virus, the pace of evolution of SARS-CoV-2 is moderate in comparison to other viruses [@doi:10.1186/1471-2148-4-21].
The declaration of the first SARS-CoV-2 variant of concern (VOC) B.1.1.7 in December 2020 has attracted significant media attention.
While the B.1.1.7 lineage garnered attention in November 2020, two genomes of the lineage were detected as early as September 20th, 2020 from routine genomic data sampled in Kent (U.K.) by the COVID-19 Genomics UK Consortium (COG-UK). 
The following day, a second B.1.1.7 genome was reported in greater London [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563; @doi:10.1101/2020.12.30.20249034; @doi:10.1001/jama.2020.27124]
Since then, B.1.1.7 has spread across the UK and internationally, and it has now been detected in at least 62 countries [@url:https://cov-lineages.org/global_report_B.1.1.7.html], despite several countries imposing travel restrictions on travelers from the UK. 
Of the twenty-three mutations that define B.1.1.7 from the original strain isolated in Wuhan (lineage A), fourteen are lineage-specific and three appear to be biologically consequential mutations associated with the spike protein, namely N501Y, P681H, and 69-70del [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf;  @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
The latter is a 6-bp deletion that leads to the loss of two amino acids and has consequences for immune recognition; it may, in conjunction with N501Y, be responsible for the increased transmissibility of the B.1.1.7 VOC due to changes in the RBD that increase binding affinity with ACE2 [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; @doi:10.1101/2020.12.14.422555]. 
B.1.1.7 has increased transmissibility by up to 56%, leading to an R~0~ of approximately 1.4.
While there is also the possibility that this VOC may be associated with increased disease severity, there is currently insufficient evidence to draw any conclusions [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/955239/NERVTAG_paper_on_variant_of_concern__VOC__B.1.1.7.pdf; @doi:10.1101/2020.12.24.20248822; @doi:10.1101/2020.12.30.20249034; @url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/952490/Variant_of_Concern_VOC_202012_01_Technical_Briefing_4_England.pdf].
Other variants also express the 69-70del mutation [@url:https://virological.org/t/identification-of-a-novel-sars-cov-2-spike-69-70-deletion-lineage-circulating-in-the-united-states/577; @doi:10.1101/2020.12.24.20248814], and public health officials in the United States and the UK have been able to use RT-PCR-based assays (ThermoFisher TaqPath COVID-19 assay) to identify sequences with this deletion because it occurs where the qPCR probe binds [@doi:10.1101/2020.12.30.20249034]. 
In the UK, B.1.1.7 is present in more than 97% of diagnostic tests that return negative for S-gene targets and positive for the other targets; thus, the frequency of S-gene target failure can be used as a proxy for the detection of B.1.1.7 [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/950823/Variant_of_Concern_VOC_202012_01_Technical_Briefing_3_-_England.pdf]._
The FDA has highlighted that the performance of three diagnostic tests may be affected by the B.1.1.7 lineage because it could cause false negative tests [@url:https://www.fda.gov/medical-devices/letters-health-care-providers/genetic-variants-sars-cov-2-may-lead-false-negative-results-molecular-tests-detection-sars-cov-2].

While B.1.1.7 is currently the main VOC, other genetic variants not designated VOCs have been detected, including B.1.351 and P.1, both of which emerged independently [@url:https://www.cdc.gov/coronavirus/2019-ncov/more/science-and-research/scientific-brief-emerging-variants.html].
B.1.351 was first detected in October 2020 in South Africa, was later detected in the EU on December 28th, 2020 and has now spread to at least 26 countries [@url:https://www.gov.za/speeches/minister-zweli-mkhize-confirms-8-725-more-cases-coronavirus-covid-19-18-dec-2020-0000; @url:https://cov-lineages.org/global_report_B.1.351.html; @url:https://virological.org/t/tracking-the-international-spread-of-sars-cov-2-lineages-b-1-1-7-and-b-1-351-501y-v2/592]. 
B.1.351 contains several mutations at the RBD including K417N, E484K, and N501Y.
While the biological significance of these mutations are still under investigation, it does appear that this lineage may be associated with increased transmissibility [@doi:10.1101/2020.12.21.20248640] due to the N501Y mutation [@doi:10.1101/2020.12.14.422555; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
The P.1 variant is a sublineage of the B.1.1.28 lineage that was first detected in Japan in samples obtained from four travelers from Brazil during a screening at a Tokyo airport on January 10, 2021 [@url:https://www.ecdc.europa.eu/sites/default/files/documents/COVID-19-risk-related-to-spread-of-new-SARS-CoV-2-variants-EU-EEA-first-update.pdf].
Shortly thereafter, it was established that there was a concentration of cases of the P.1 variant in Manaus, in the Amazonas State, Brazil.
In a small number of samples (n=31) sequenced in Manaus, 42% were identified as the P.1 variant as early as mid-December, but the variant seemed to be absent in genome surveillance testing prior to December [@url:https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586].
To date, at least eight countries have detected the P.1 lineage [@url:https://cov-lineages.org/global_report_P.1.html].
While the majority of P.1 cases detected internationally have been linked to travel originating from Brazil, the UK has also reported evidence of community transmission detected via routine community sequencing [@url:https://cov-lineages.org/global_report_P.1.html; @url:https://www.reuters.com/article/us-health-coronavirus-britain-variants/uk-detects-77-cases-of-south-african-covid-variant-nine-of-brazilian-idUSKBN29T07P].  
P.1 has eight lineage-specific mutations along with three concerning spike protein mutations in the RBD, including K417T, E484K, and N501Y [@doi:10.1101/2020.12.21.20248640].
 
There have been multiple different SARS-CoV-2 lineages detected that have mostly been of no more clinical concern than the original devastating lineage originating in Wuhan [@url:https://cov-lineages.org/lineages.html].
However, the spotlight has been cast on other variants of unknown clinical relevance due to the increase of cases observed that have been associated with B.1.1.7 in particular.  
Although early in its ascendency, B.1.427/429 is SARS-CoV-2 variants that was detected in California, USA and also known as CAL.20C [@doi:10.1101/2021.01.18.21249786].
It was first detected in July 2020 but was not detected again until October 2020. 
In December 2020, B.1.427/429 accounted for ~24% of the total cases in Southern California and ~36% of total cases in the Los Angeles area. 
B.1.427/429 has now been detected in New York and Washington, D.C., and some cases have been reported outside the USA in Oceania [@doi:10.1101/2021.01.18.21249786].
This variant is characterized by five key lineage-specific mutations (ORF1a: I4205V, ORF1b:D1183Y, S: S13I;W152C;L452R).
The latter spike mutation, L452R, is found in an area of the RBD known to resist monoclonal antibodies to the spike protein [@doi:10.1016/j.cell.2020.07.012], and it is hypothesized that this mutation may resist polyclonal sera in convalescent patients or in individuals post-vaccination [@doi:10.1101/2020.12.31.425021; @doi:10.1101/2021.01.18.21249786].
B.1.427/429 is not a designated VOC; however, further research is required to determine the implications of the mutations encoded in this genetic variant.  
Another notable variant has recently been discovered in 35 patients in a Bavarian hospital in Germany; however, the sequencing data has not been published to date and it remains to be determined whether this variant is of any further concern [@url:https://www.br.de/nachrichten/bayern/neue-coronavirus-mutation-35-faelle-in-garmisch-partenkirchen,SMQ1V6u].

There are several shared mutations and deletions between the three lineages, P.1, B.1.1.7, and B.1.315 and indeed other variants of SARS-CoV-2 that are under investigation [@url:https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586].
For example, N501Y, which appears to have occurred independently in each of the three lineages.  
E484K is present in both B.1.351 and P.1 [@url:https://cov-lineages.org/global_report.html].
The mutations N501Y and E484K are found in the RBD within the receptor-binding motif responsible for forming an interface with the ACE2 receptor, which seems to be consequential for ACE2 binding affinity [@doi:10.1016/j.cell.2020.08.012].
Indeed, N501Y is associated with increased virulence and infectivity in mouse models [@doi:10.1126/science.abc4730].
E484K has also been associated with evasion from neutralizing antibodies [@doi:10.1016/j.chom.2020.11.007; @doi:10.1101/2020.12.28.424451; @doi:10.1101/2020.12.31.425021].
The del69-70 (del:11288:9) is also shared between P.1 and B.1.1.7 and happens to be a common deletion found in the N terminal mutation of the spike protein. 
This deletion has also been associated with several RBD mutations [@doi:10.1101/2020.11.19.389916; @doi:10.1101/2020.12.14.422555; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
There is concern that mutations in the spike protein of variants may lead to clinical consequences for transmissibility, disease severity, re-infection, therapeutics, and vaccinations [@doi:10.1126/science.abg4312; @url:https://virological.org/t/sars-cov-2-reinfection-by-the-new-variant-of-concern-voc-p-1-in-amazonas-brazil/596; @doi:10.1101/2020.12.28.424451; @doi:10.1101/2020.12.31.425021; @doi:10.1038/d41586-021-00121-z; @doi:10.1126/science.abf9302; @doi:10.1126/science.371.6527.329]. 

Vaccine producers are working to determine whether the vaccines are still effective against the novel genetic variants.
Moderna recently published data for their mRNA-1273 vaccine that showed no significant impact of neutralization against the B.1.1.7 variant upon vaccination in humans and non-human primates. 
On the other hand, Moderna reported a reduced but significant neutralization against the B.1.351 variant upon vaccination [@doi:10.1101/2021.01.25.427948].
Indeed, Pfizer–BioNTech reported that sera from twenty participants vaccinated with the BNT162b COVID-19 vaccine in previous clinical trials [@doi:10.1056/NEJMoa2034577; @doi:10.1056/NEJMoa2027906] elicited equivalent neutralizing titers against isogenic Y501 SARS-CoV-2 on an N501Y genetic background _in vitro_ [@doi:10.1101/2021.01.07.425740].
Another study has reported that the plasma neutralizing activity against SARS-CoV-2 variants encoding the combination of K417N:E484K:N501Y or E484K or N501Y was variably and significantly reduced in the sera of twenty participants who received either the Pfizer–BioNTech BNT162b (n = 6) vaccine or the Moderna’s mRNA-1273 vaccine (n =14) [@doi:10.1101/2021.01.15.426911]. 
For now, the consensus appears to be that the FDA approved vaccines still seem to be effective against the genetic variants of SARS-CoV-2 and their accompanying mutations, albeit with a slightly lower neutralizing capacity [@doi:10.1073/pnas.2008281117; @doi:10.1101/2021.01.15.426911; @doi:10.1101/2021.01.07.425740; @doi:10.1101/2021.01.25.427948]. 
Further research is required to discern the clinical, prophylactic, and therapeutic consequences of these genetic SARS-CoV-2 variants as the pandemic evolves. 

### Conclusions

As of October 2020 the SARS-CoV-2 virus remains a serious worldwide threat.
The scientific community has responded by rapidly collecting and disseminating information about the SARS-CoV-2 virus and the associated illness, COVID-19.
The rapid identification of the genomic sequence of the virus allowed for early contextualization of SARS-CoV-2 among other known respiratory viruses.
The pathogen is a coronavirus that is closely related to SARS-CoV-1, which caused the SARS pandemics of the early 2000s.
Knowing the phylogenetic context and genomic sequence of the virus then allowed for rapid insights into its structure and pathogenesis.
