## Molecular and Serologic Diagnostic Technologies for SARS-CoV-2

### Abstract

The COVID-19 pandemic has presented many challenges that have spurred biotechnological research to address specific problems.
Diagnostics is one area where biotechnology has been critical.
Diagnostic tests play a vital role in managing a viral threat by facilitating the detection of infected and/or recovered individuals.
From the perspective of what information is provided, these tests fall into two major categories, molecular and serological.
Molecular diagnostic techniques assay whether a virus is present in a biological sample, thus making it possible to identify individuals who are currently infected.
Additionally, when the immune system is exposed to a virus, it responds by producing antibodies specific to the virus.
Serological tests make it possible to identify individuals who have mounted an immune response to a virus of interest and therefore facilitate the identification of individuals who have previously encountered the virus.
These two categories of tests provide different perspectives valuable to understanding the spread of SARS-CoV-2.
Within these categories, different biotechnological approaches offer specific advantages and disadvantages.
Here we review the categories of tests developed for the detection of the SARS-CoV-2 virus or antibodies against SARS-CoV-2 and discuss the role of diagnostics in the COVID-19 pandemic.

### Importance

Testing is critical to pandemic management.
Among molecular tests, messaging about testing strategies has varied widely between countries, with the United States in particular emphasizing the higher sensitivity of polymerase chain reaction tests above immunoassays.
However, these tests offer different advantages, and a holistic view of the testing landscape is needed to identify the information provided by each test and its relevance to addressing different questions.
Another important consideration is the ease of use and ability to scale for each test, which determines how widely they can be deployed.
Here we describe the different diagnostic technologies available as well as the information they provide about SARS-CoV-2 and COVID-19.

### Introduction

Since the emergence of _Severe acute respiratory syndrome-like coronavirus 2_ (SARS-CoV-2) in late 2019, significant international efforts have focused on managing the spread of the virus.
Identifying individuals who have contracted coronavirus disease 2019 (COVID-19) and may be contagious is crucial to reducing the spread of the virus.
Given the high transmissibility of SARS-CoV-2 and the potential for asymptomatic or presymptomatic individuals to be contagious [@individual-pathogenesis], the development of rapid, reliable, and affordable methods to detect SARS-CoV-2 infection is and was vitally important for understanding and controlling spread.
For instance, test-trace-isolate procedures were an early cornerstone of many nations' efforts to control the outbreak [@doi:10.1503/cmaj.202120; @doi:10/ftzx; @doi:10.1016/j.lanwpc.2020.100044].
Such efforts depend on diagnostic testing.

The genetic sequence of the SARS-CoV-2 virus was first released by Chinese officials on January 10, 2020 [@url:https://www.who.int/emergencies/disease-outbreak-news/item/2020-DON233], and the first test to detect the virus was released about 13 days later [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
The genomic information was critical to the development of diagnostic approaches.
There are two main classes of diagnostic tests: molecular tests, which can diagnose an active infection by identifying the presence of SARS-CoV-2, and serological tests, which can assess whether an individual was infected in the past via the presence or absence of antibodies against SARS-CoV-2.
Over the course of the COVID-19 pandemic, a variety of tests have emerged within these two categories.

Molecular tests detect either viral RNA or protein in a patient sample.
They are essential to identifying infected individuals, which can be important for determining courses of action related to treatment, quarantine, and contact tracing.
Tests for viral RNA are done by reverse transcription (RT) of viral RNA to DNA followed by DNA amplification, usually with polymerase chain reaction (PCR) [@DOI:10.1038/jid.2013.1].
Tests for viral proteins typically use an antibody pair for detection as implemented in techniques such as lateral flow tests (LFTs) and enzyme-linked immunosorbent assays (ELISAs) [@doi:10.1007/s00216-010-3661-4; @pubmed:32310382].
Molecular tests require the viral genome sequence in order to develop DNA primers for viral RNA detection or to express a viral protein for use as an antigen in antibody production.

Serological tests, on the other hand, detect the presence of antibodies in blood plasma samples or other biological samples, providing insight into whether an individual has acquired immunity against SARS-CoV-2.
Assays that can detect the presence of antibodies in blood plasma samples include ELISA, lateral flow immunoassay, and chemiluminescence immunoassay (CLIA) [@DOI:10.3389/fmolb.2021.682405].
To distinguish past infection from vaccination, serological tests detect antibodies that bind the nucleocapsid protein of the SARS-CoV-2 virus [@doi:10.1093/infdis/jiaa273].
They are useful for collecting population-level information for epidemiological analysis, as they can be used to estimate the extent of the infection in a given area.
Thus, serological tests may be useful to address population-level questions, such as the percent of cases that manifest as severe versus mild and for guiding public health and economic decisions regarding resource allocation and counter-disease measures.

Molecular and serological tests therefore offer distinct, complementary perspectives on COVID-19 infections.
Some of the same technologies are useful to both strategies, and different technologies have been employed to varying extents throughout the world since the start of the COVID-19 pandemic.
Two of the primary metrics used to evaluate these tests are sensitivity and specificity.
Sensitivity refers to a test's ability to correctly identify a true positive; for example, a test with 50% sensitivity would identify SARS-CoV-2 in only one of every two positive samples.
On the other hand, specificity refers to how well a test is able to identify a negative sample as negative.
This metric can be relevant both in terms of understanding the risk of false positives and in discussing whether a test is susceptible to identifying other coronaviruses.
Here, we review the different types of tests within each category that have been developed and provide perspective on their applications.

### Molecular Tests to Identify SARS-CoV-2

Molecular tests are used to identify distinct genomic subsequences of a viral molecule in a sample or the presence of viral protein, and they thus can be used to diagnose an active viral infection.
An important first step is identifying which biospecimens are likely to contain the virus in infected individuals and then acquiring these samples from the patient(s) to be tested.
Common sampling sources for molecular tests include nasopharyngeal cavity samples, such as throat washes, throat swabs, and saliva [@doi:10/ggp4qx], and stool samples [@doi:10.1002/jmv.25742].
Once a sample is acquired from a patient, molecular tests detect SARS-CoV-2 based on the presence of either viral nucleic acids or viral proteins.

#### PCR-Based Tests

When testing for RNA from viruses like SARS-CoV-2, the first step involves pre-processing in order to create complementary DNA (cDNA) from the RNA sample using RT.
The second step involves the amplification of a region of interest in the cDNA using successive cycles of heating and cooling.
Depending on the application, this amplification is achieved using variations of PCR.
Reverse transcription polymerase chain reaction (RT-PCR) tests determine whether a target is present by amplifying a region of interest of cDNA [@doi:10.1042/BIO20200034].
Some tests use the results of the PCR itself (e.g., a band on a gel) to determine whether the pathogen is present.
However, this approach has not been employed widely in diagnostic testing, and instead most PCR-based tests are quantitative.

##### Quantitative Real-Time PCR

In contrast to RT-PCR, quantitative, real-time PCR uses fluorescent dyes that bind to the amplified DNA, thereby allowing a real time assessment of the amplification procedure [@doi:10.1042/BIO20200034] (in this manuscript we refer to quantitative real-time PCR as qPCR, following the Minimum Information for Publication of Quantitative Real-Time PCR Experiments guidelines [@doi:10.1373/clinchem.2008.112797], and when combined with reverse transcriptase steps, as is required for the evaluation of RNA, it is known as RT-qPCR.)
The time resolution provided by qPCR and RT-qPCR is useful because the amount of fluorescence emitted by the sample is proportional to the amount of DNA amplified, and therefore the amount of virus present can be indirectly measured using the cycle threshold (C~t~) determined by qPCR.

The first test developed and validated for the detection of SARS-CoV-2 used RT-qPCR to detect several regions of the viral genome: the _ORF1b_ of the RNA-dependent RNA polymerase (RdRP), the envelope protein gene (_E_), and the nucleocapsid protein gene (_N_) [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
The publication reporting this test was released on January 23, 2020, less than two weeks after the sequence of the virus was first reported [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents.
Specifically, this test uses two probes against RdRP, one of which is specific to SARS-CoV-2 [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
Importantly, this assay was not found to return false positive results.

In January 2020, Chinese researchers developed a test that used RT-qPCR to identify two gene regions of the viral genome, _ORF1b_ and _N_ [@doi:10.1093/clinchem/hvaa029].
This assay was tested on samples from two COVID-19 patients and a panel of positive and negative controls consisting of RNA extracted from several cultured viruses.
The assay uses the _N_ gene to screen patients, while the _ORF1b_ gene region is used to confirm the infection [@doi:10.1093/clinchem/hvaa029].
The test was designed to detect sequences conserved across sarbecoviruses, or viruses within the same subgenus as SARS-CoV-2.
Considering that _Severe acute respiratory syndrome-related coronavirus 1_ (SARS-CoV-1) and SARS-CoV-2 are the only sarbecoviruses currently known to infect humans, a positive test can be assumed to indicate that the patient is infected with SARS-CoV-2, although this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.
The fact that the targets are so conserved offers the advantage of reduced concern about sensitivity in light of the evolution of SARS-CoV-2.

qPCR tests have played an important role in diagnostics during the COVID-19 pandemic.
For SARS-CoV-2, studies have typically considered a patient to be infectious if the C~t~ is below 33 or sometimes 35 [@doi:10.1007/s10096-020-03913-9; @doi:10.1038/s41467-020-20568-4; @individual-pathogenesis].
A lower C~t~ corresponds to fewer qPCR cycles needed to reach a detectable level, indicating that higher amounts of virus were present in the initial reaction.
Interpretations of the C~t~ values obtained from these tests have raised some interesting questions related to viral load and contagiousness.
Lower C~t~ values correspond to a higher probability of a positive viral culture, but no threshold could discriminate all positive from all negative cultures [@doi:10.2807/1560-7917.ES.2020.25.32.2001483].
Additionally, because of the variability introduced by sample collection and clinical components of testing, C~t~ is not a proxy for viral load [@doi:10.1016/j.jinf.2020.10.017].
Positive PCR results have also been reported for extended periods of time from symptom onset and/or the first positive PCR test [@doi:10.1038/s41586-020-2196-x], meaning that in some cases, a positive PCR may not indicate that someone is contagious [@individual-pathogenesis].

In addition to the nuance required to interpret PCR results, there are also factors that influence their accuracy.
The specificity of these tests is very high [@doi:10.2807/1560-7917.ES.2020.25.50.2000568], meaning that a positive RT-PCR result is very likely to indicate SARS-CoV-2 infection.
All the same, many factors can influence the sensitivity and specificity, with sample collection being a critically important factor in the reliability of RT-PCR results.
In fact, though the analytical specificity of the assay is extremely high, the challenges of implementing testing can introduce variability that results in a lower clinical specificity [@doi:10.1101/2020.04.26.20080911].
The most reliable results were found to come from nasopharyngeal swabs and from pooled nasal and throat swabs, with lower accuracies produced by saliva or by throat or nasal swabs alone [@doi:10/gjrfns; @doi:10.2807/1560-7917.ES.2020.25.50.2000568].
Differences in experimental parameters such as the use of primers more specific to SARS-CoV-2 has been found to improve sensitivity in these specimens [@doi:10.1128/JCM.00310-20].

Additionally, the impact of viral evolution on RT-PCR sensitivity is a concern [@doi:10.1038/s41576-021-00360-w; @doi:10.2807/1560-7917.ES.2020.25.39.2001650].
Using a panel that includes multiple targets can mitigate these effects [@doi:10.1016/j.ijid.2020.06.027].
As an example, public health officials in the United States and the UK used PCR-based assays (ThermoFisher TaqPath COVID-19 assay) to identify sequences with a mutation (69-70del) associated with a variant of concern because the mutation interrupted binding of the qPCR probe [@doi:10.1101/2020.12.30.20249034].
In the UK, one study found that this VOC (B.1.1.7) was present in more than 97% of diagnostic tests that returned negative for S-gene targets and positive for the other targets; thus, the frequency of S-gene target failure could be used as a proxy for the detection of B.1.1.7 [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; @url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/950823/Variant_of_Concern_VOC_202012_01_Technical_Briefing_3_-_England.pdf].
The FDA has highlighted that the performance of several diagnostic tests may be affected by the B.1.1.7 lineage because it could cause false negative tests [@url:https://www.fda.gov/medical-devices/letters-health-care-providers/genetic-variants-sars-cov-2-may-lead-false-negative-results-molecular-tests-detection-sars-cov-2].
Additionally, a test designed to incorporate genomic differences with SARS-CoV-1 was found to offer improved sensitivity and specificity [@doi:10.1128/JCM.00310-20].
Thus, while various factors can influence the exact parameters of testing accuracy, RT-PCR is known to have very high specificity and lower, but still high, sensitivity.
All the same, the weight given to these tests as an indicator of SARS-CoV-2 infection regardless of other clinical considerations is not typical [@doi:10.1101/2020.04.26.20080911].

##### Digital PCR

Digital PCR (dPCR) is a new generation of PCR technologies offering an alternative to traditional qPCR [@doi:10.1038/s41598-017-02217-x].
In dPCR, a sample is partitioned into thousands of compartments, such as nanodroplets (droplet dPCR or ddPCR) or nanowells, and a PCR reaction takes place in each compartment.
This design allows for a digital read-out where each partition is either positive or negative for the nucleic acid sequence being tested for, allowing for absolute target quantification through Poisson statistics.
While dPCR equipment is not yet as common as that for qPCR, dPCR for DNA targets generally achieves higher sensitivity than other PCR technologies while maintaining high specificity, though sensitivity is slightly lower for RNA targets [@doi:10.3390/s18041271].

High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
In one study, Suo et al. [@doi:10.1080/22221751.2020.1772678] performed a double-blind evaluation of ddPCR for SARS-CoV-2 detection.
They evaluated on 63 samples collected from suspected positive outpatients and 14 from supposed convalescent patients.
Of the 63 outpatients, only 21 (33%) were identified as positive for SARS-CoV-2 with qPCR.
However, ddPCR identified 49 (78%) as positive, 10 (16%) as negative, and 4 (6%) as suspected/borderline for SARS-CoV-2 infection.
While both qPCR and ddPCR were found to have very high specificity (100%), this analysis reported that the sensitivity was 40% with qPCR compared to 94% with ddPCR.
Analysis of serial dilutions of a linear DNA standard suggested that ddPCR was approximately 500 times more sensitive than qPCR [@doi:10.1080/22221751.2020.1772678].
Thus, this study suggests that ddPCR provides an extremely sensitive molecular test that is able to detect SARS-CoV-2 even at very low viral loads.

A second study [@doi:10.1016/j.talanta.2020.121726] confirmed that RT-ddPCR is able to detect SARS-CoV-2 at a lower threshold for viral load relative to RT-PCR.
This study analyzed 196 samples, including 103 samples from suspected patients, 77 from contacts and close contacts, and 16 from suspected convalescents, using both RT-qPCR and RT-ddPCR.
First, the authors evaluated samples from the 103 suspected cases.
Using RT-qPCR, 29 (28%) were identified as positive, 25 (24%) as negative, and 49 (48%) as borderline, i.e., the C~t~ value was higher than the positive threshold of 35 but lower than the negative threshold of 40.
When the 61 negative and borderline samples were reanalyzed with ddPCR, 19 (31%) of the negative and 42 (69%) of the borderline samples were identified as positive.
All of the suspected cases were later confirmed to be COVID-19 through a combination of symptom development and RT-qPCR resampling, indicating that ddPCR improved the overall detection rate compared to RT-qPCR from 28.2% to 87.4%.

They repeated this analysis in patient samples from contacts and close contacts.
Patients who tested negative with both methods (n = 48) were observed to remain healthy over a period of 14 days.
Among the remaining 29 samples from contacts, RT-qPCR identified 12 as positive, 1 as negative, and 16 as borderline.
All of the samples that tested positive using RT-qPCR also tested positive using ddPCR.
In contrast, the negative result and all but one of the borderline results were identified as positive by RT-ddPCR, and these patients were later determined to be SARS-CoV-2 positive based on clinical evaluation and repeated molecular sampling.
Similarly, in the final group, 16 convalescent patients, RT-qPCR identified 12 as positive, three as suspect, and one as negative, but RT-dPCR identified all as positive.
The evidence from this study therefore supports a lower limit of detection with ddPCR.
Overall, these studies suggest that ddPCR is a promising tool for overcoming the problem of false negatives in SARS-CoV-2 RNA testing, but this method is unlikely to affect the current pandemic due to its lack of availability.

##### Sequencing

In some cases, the DNA amplified with PCR is sequenced.
Sequencing requires an additional sample pre-processing step called library preparation.
Library preparation is the process of preparing the sample for sequencing, typically by fragmenting the sequences and adding adapters [@doi:10.1016/j.biotechadv.2020.107537].
In some cases, library preparation can involve other modifications of the sample, such as adding barcodes to identify a particular sample within the sequence data.
Barcoding can therefore be used to pool samples from multiple sources.
There are different reagents used for library preparation that are specific to identifying one or more target sections with PCR [@doi:10.1021/acsnano.0c02624].
Sequential pattern matching is then used to identify unique subsequences of the virus, and if sufficient subsequences are found, the test is considered positive.
Therefore, tests that use sequencing require a number of additional molecular and analytical steps relative to tests that use PCR alone.

Sequencing has been an important strategy for discovery of SARS-CoV-2 variants (e.g., see [@DOI:10.1101/2021.01.18.21249786]).
Sequencing elucidates any genetic variants located between the PCR primers.
For this reason, it is critical to genomic surveillance efforts.
Genomic surveillance is an important complement to epidemiological surveillance efforts [@doi:10/hkwr], as described below.
Through genomic surveillance, it has become possible to monitor the emergence of variants of interest and variants of concern (VOC) that may pose additional threats due to increased contagiousness, virulence, or immune escape [@doi:10/hkwr; @url:https://cov-lineages.org].
Sequencing also allows for analysis of the dominant strains in an area at a given time.
Worldwide, the extent of genomic surveillance varies widely, with higher-income countries typically able to sequence a higher percentage of cases [@doi:10.1101/2021.08.21.21262393].
Sequencing efforts are important for identifying variants containing mutations that might affect the reliability of molecular diagnostic tests, as well as mitigation measures such as therapeutics and prophylactics [@doi:10.1038/s41576-021-00360-w; @doi:10.2807/1560-7917.ES.2020.25.39.2001650].
Therefore, sequencing is an important component of diagnostics: while it is not necessary for diagnosing an individual case, it is critical to monitoring trends in the variants affecting a population and to staying aware of emerging variants that may pose additional challenges.

##### Pooled and Automated PCR Testing

Due to limited supplies and the need for more tests, several labs have found ways to pool or otherwise strategically design tests to increase throughput.
The first such result came from Yelin et al. [@doi:10.1093/cid/ciaa531], who reported that they could pool up to 32 samples in a single qPCR run.
This was followed by larger-scale pooling with slightly different methods [@doi:10.1101/2020.04.02.022186].
Although these approaches are also PCR based, they allow for more rapid scaling and higher efficiency for testing than the initial PCR-based methods developed.
Conceptually, pooling could also be employed in analysis with RT-qPCR [@doi:10.1109/ICASSP39728.2021.9413685], and this strategy has been evaluated in settings such as schools [@doi:10.1016/j.eclinm.2021.101082] and hospitals [@doi:10.1038/s41598-020-76043-z].

#### RT-LAMP

RT-PCR remains the gold standard for detection of SARS-CoV-2 RNA from infected patients, but the traditional method requires special equipment and reagents, including a thermocycler.
Loop-mediated isothermal amplification (LAMP) is an alternative to PCR that does not require specialized equipment [@DOI:10.1093/nar/28.12.e63].
In this method, nucleic acids are amplified in a 25 &mu;L reaction that is incubated and chilled on ice [@doi:10.1093/nar/28.12.e63].
It uses primers designed to facilitate auto-cycling strand displacement DNA synthesis [@doi:10.1093/nar/28.12.e63].
LAMP can be combined with reverse transcription (RT-LAMP) to enable the detection of RNA.

One study showed that RT-LAMP is effective for detection of SARS-CoV-2 with excellent specificity and sensitivity and that this method can be applied to unprocessed saliva samples [@doi:10.1038/s41598-021-95799-6].
This method was benchmarked against RT-PCR using 177 human nasopharyngeal RNA samples, of which 126 were COVID positive.
The authors break down the sensitivity of their test according to the C~t~ value from RT-PCR of the same samples; RT-LAMP performs at 100% sensitivity for samples with a C~t~ from RT-PCR of 32 or less.
The performance is worse when considering all RT-PCR positive samples (including those with C~t~ values between 32-40).
However, there is some evidence suggesting that samples obtained from individuals that achieve C~t~ values >30 measured using RT-PCR tend to be less infective that those that record a C~t~ value <30 [@doi:10.1016/j.jiph.2021.08.013; @doi:10.1017/ice.2021.132; @doi:10.1093/cid/ciaa619], so RT-LAMP is still a useful diagnostic tool.
Various combinations of reagents are available, but one example is the WarmStart Colorimetric LAMP 2X Master Mix with a set of six primers developed previously by Zhang et al. [@doi:10.1101/2020.02.26.20028373].
To determine assay sensitivity, serial tenfold dilutions of _in vitro_ transcribed _N_-gene RNA standard were tested using quantities from 10^5^ copies down to 10 copies.
The assay readout is the color of the dye changing from pink to yellow due to binding to the DNA product over 30 minutes.
The RT-LAMP assay was then applied to clinical nasopharyngeal samples.
For viral loads above 100 copies of genomic RNA, the RT-LAMP assay had a sensitivity of 100% and a specificity of 96.1% from purified RNA.
The sensitivity of the direct assay of saliva by RT-LAMP was 85%.
Sensitivity and specificity metrics were obtained by comparison with results from RT-PCR.
RT-LAMP pilot studies for detection of SARS-CoV-2 were reviewed in a meta-analysis [@PMID:34086539].
In the meta-analysis of all 2,112 samples, the cumulative sensitivity of RT-LAMP was calculated at 95.5%, and the cumulative specificity was 99.5%.

This test aims to bring the sensitivity of nucleic acid detection to the point of care or home testing setting.
It could be applied for screening, diagnostics, or as a definitive test for people who are positive based on LFTs (see below).
The estimated cost per test is about 2 euros when RNA extraction is included.
The main strength of this test over RT-PCR is that it can be done isothermally, but the main drawback is that it is about 10-fold less sensitive than RT-PCR.
The low cost, excellent sensitivity/specificity, and quick readout of RT-LAMP makes this an attractive alternative to RT-PCR.
Alternative strategies like RT-LAMP are needed to bring widespread testing away from the lab and into under-resourced areas.

#### CRISPR-based Detection

Technology based on CRISPR (clustered regularly interspaced short palindromic repeats) [@doi:10.1038/s41467-018-04252-2] has also been instrumental in scaling up testing protocols.
Two CRISPR-associated nucleases, Cas12 and Cas13, have been used for nucleic acid detection.
Multiple assays exploiting these nucleases have emerged as potential diagnostic tools for the rapid detection of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The SHERLOCK method (Specific High-sensitivity Enzymatic Reporter unLOCKing) from Sherlock Biosciences relies on Cas13a to discriminate between inputs that differ by a single nucleotide at very low concentrations [@doi:10.1126/science.aam9321].
The target RNA is amplified by real-time recombinase polymerase amplification (RT-RPA) and T7 transcription, and the amplified product activates Cas13a.
The nuclease then cleaves a reporter RNA, which liberates a fluorescent dye from a quencher.
Several groups have used the SHERLOCK method to detect SARS-CoV-2 viral RNA.
An early study reported that the method could detect 7.5 copies of viral RNA in all 10 replicates, 2.5 copies in 6 out of 10, and 1.25 copies in 2 out of 10 runs [@doi:10.1371/journal.ppat.1008705].
It also reported 100% specificity and sensitivity on 114 RNA samples from clinical respiratory samples (61 suspected cases, among which 52 were confirmed and nine were ruled out by metagenomic next-generation sequencing, 17 SARS-CoV-2-negative but human coronavirus (HCoV)-positive cases, and 36 samples from healthy subjects) and a reaction turnaround time of 40 minutes.
A separate study screened four designs of SHERLOCK and extensively tested the best-performing assay.
They determined the limit of detection to be 10 copies/&mu;l using both fluorescent and lateral flow detection [@doi:10.1101/2020.02.26.967026].

LFT strips are simple to use and read, but there are limitations in terms of availability and cost per test.
Another group therefore proposed the CREST (Cas13-based, Rugged, Equitable, Scalable Testing) protocol, which uses a P51 cardboard fluorescence visualizer, powered by a 9-volt battery, for the detection of Cas13 activity instead of immunochromatography [@doi:10.1128/JCM.02402-20].
CREST can be run, from RNA sample to result, with no need for AC power or a dedicated facility, with minimal handling in approximately 2 hours.
Testing was performed on 14 nasopharyngeal swabs.
CREST picked up the same positives as the CDC-recommended TaqMan assay with the exception of one borderline sample that displayed low-quality RNA.
This approach may therefore represent a rapid, accurate, and affordable procedure for detecting SARS-CoV-2.

The DETECTR (DNA Endonuclease-Targeted CRISPR Trans Reporter) method from Mammoth Biosciences involves purification of RNA extracted from patient specimens, amplification of extracted RNAs by loop-mediated amplification, and application of their Cas12-based technology.
In this assay, guide RNAs (gRNAs) were designed to recognize portions of sequences corresponding to the SARS-CoV-2 genome, specifically the N2 and E regions [@doi:10.1038/s41587-020-0513-4].
In the presence of SARS-CoV-2 genetic material, sequence recognition by the gRNAs results in double-stranded DNA cleavage by Cas12, as well as cleavage of a single-stranded DNA molecular beacon.
The cleavage of this molecular beacon acts as a colorimetric reporter that is subsequently read out in a lateral flow assay and indicates the presence of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The 40-minute assay is considered positive if there is detection of both the _E_ and _N_ genes or presumptive positive if there is detection of either of them.
The assay had 95% positive predictive agreement and 100% negative predictive agreement with the US Centers for Disease Control and Prevention SARS-CoV-2 RT-qPCR assay.
The estimated limit of detection was 10 copies per &mu;l reaction, versus 1 copy per &mu;l reaction for the CDC assay.

These results have been confirmed by other DETECTR approaches.
Using RT-RPA for amplification, another group detected 10 copies of synthetic SARS-CoV-2 RNA per &mu;l of input within 60 minutes of RNA sample preparation in a proof-of-principle evaluation [@doi:10.1101/2020.02.29.971127].
Through a similar approach, another group reported detection at 1 copy per &mu;l [@doi:10.1371/journal.pbio.3000978].
The DETECTR protocol was improved by combining RT-RPA and CRISPR-based detection in a one-pot reaction that incubates at a single temperature and by using dual CRISPR RNAs, which increases sensitivity.
This new assay, known as All-In-One Dual CRISPR-Cas12a, detected 4.6 copies of SARS-CoV-2 RNA per &mu;l of input in 40 minutes [@doi:10.1038/s41467-020-18575-6].
Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/&mu;l in 40-60 minutes [@doi:10.1038/s41421-020-0174-y].

It was also reported that electric field gradients can be used to control and accelerate CRISPR assays by co-focusing Cas12-gRNA, reporters, and target [@doi:10.1073/pnas.2010254117].
The authors generated an appropriate electric field gradient using a selective ionic focusing technique known as isotachophoresis (ITP) implemented on a microfluidic chip.
They also used ITP for automated purification of target RNA from raw nasopharyngeal swab samples.
Combining this ITP purification with loop-mediated isothermal amplification, their ITP-enhanced assay achieved detection of SARS-CoV-2 RNA (from raw sample to result) in 30 minutes.

All these methods require upstream nucleic acid amplification prior to CRISPR-based detection.
They rely on type V (Cas12-based) and type IV (Cas13-based) CRISPR systems.
In contrast, type III CRISPR systems have the unique property of initiating a signaling cascade, which could boost the sensitivity of direct RNA detection.
In type III CRISPR systems, guide CRISPR RNAs (crRNAs) are bound by several Cas proteins [@doi:10.1007/978-1-4939-2687-9_4] and can target both DNA and RNA molecules [@doi:10.1016/j.mib.2017.08.003; @doi:10.1016/j.tim.2016.09.012].
A study tested this hypothesis using the type III-A crRNA-guided surveillance complex from _Thermus thermophilus_ [@doi:10.1016/j.xcrm.2021.100319].
The authors showed that activation of the Cas10 polymerase generates three products (cyclic nucleotides, protons, and pyrophosphates) that can all be used to detect SARS-CoV-2 RNA.
Detection of viral RNA in patient samples still required an initial nucleic acid amplification step, but improvements may in the future remove that requirement.

This goal of amplification-free detection was later achieved for a Cas13a-based system [@doi:10.1016/j.cell.2020.12.001].
This approach combined multiple CRISPR RNAs to increase Cas13a activation, which is detected by a fluorescent reporter.
Importantly, because the viral RNA is detected directly, the test yields a quantitative measurement rather than a binary result.
The study also shows that fluorescence can be measured in a custom-made dark box with a mobile phone camera and a low-cost laser illumination and collection optics.
This approach is a truly portable assay for point-of-care diagnostics.
The authors achieved detection of 100 copies/&mu;l of pre-isolated RNA in 30 minutes, and correctly identified all SARS-CoV-2-positive patient RNA samples tested in 5 minutes (n = 20).

There is an increasing body of evidence that CRISPR-based assays offer a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.
In the largest study to date, DETECTR was compared to RT-qPCR on 378 patient samples [@doi:10.1093/infdis/jiaa641].
The authors reported 95% reproducibility.
Both techniques were equally sensitive in detecting SARS-CoV-2.
Lateral flow strips showed 100% correlation to the high-throughput DETECTR assay.
Importantly, DETECTR was 100% specific for SARS-CoV-2 and did not detect other human coronaviruses.
A method based on a Cas9 ortholog from _Francisella novicida_ known as FnCas9 achieved 100% sensitivity and 97% specificity in clinical samples, and the diagnostic kit is reported to have completed regulatory validation in India [@doi:10.1101/2020.09.13.20193581].

#### Immunoassays for the Detection of Antigens

Immunoassays can detect molecular indicators of SARS-CoV-2 infection, such as the proteins that act as antigens from the SARS-CoV-2 virus.
They offer the advantage of generally being faster and requiring less specialized equipment than other molecular tests, especially those involving PCR.
As a result, immunoassays hold particular interest for implementation at home and in situations where resources for PCR testing are limited.
The trade-off is that these tests typically have a lower sensitivity, and sometimes a lower specificity, than other molecular tests.
However, these tests tend to return a positive result five to 12 days after symptom onset, which may therefore correlate more closely with the timeframe during which viral replication occurs [@doi:10.1056/NEJMcp2117115].
Immunoassays for the detection of the SARS-CoV-2 antigen can include LFTs and ELISA, as discussed here, as well as CLIA and chromatographic immunoassays [@doi:10.3389/fmed.2021.797109], as described in the serological testing section below.

##### Lateral Flow Tests

LFTs provide distinct value relative to PCR tests.
They can return results within 30 minutes and can be performed without specialized equipment and at low cost.
They also do not require training to operate and are cheap to produce.
Thus, they can be distributed widely to affected populations making them an important public health measure to curb pandemic spread.
LFTs rely on the detection of viral protein with an antibody.
Often this is done with an antibody sandwich format, where one antibody conjugated to a dye binds at one site on the antigen, and an immobilized antibody on the strip binds at another site [@doi:10.1007/s00216-010-3661-4].
This design allows the dye to accumulate to form a characteristic positive test line on the strip [@doi:10.1007/s00216-010-3661-4].
Outside of COVID-19 diagnostics, the applications of LFTs are broad; they are routinely used for home pregnancy tests, disease detection, and even drugs of abuse detection in urine [@DOI:10.1007/s00216-008-2287-2].

A recent review surveyed the performance of LFTs for detection of current SARS-CoV-2 infection [@PMID:34407759].
This review covered 24 studies that included more than 26,000 total LFTs.
They reported significant heterogeneity in test sensitivities, with estimates ranging from 37.7% to 99.2%.
The estimated specificities of these tests were more homogeneous, spanning 92.4% to 100.0%.

Despite having lower sensitivity than PCR tests, LFTs occupy an important niche in the management of SARS-CoV-2.
Current infection detection by LFTs enables the scale and speed of testing that is beneficial to managing viral spread.
LFTs were available freely to citizens in the United Kingdom until April 1, 2022 [@url:https://www.gov.uk/government/news/government-sets-out-next-steps-for-living-with-covid] and to citizens of the United States in early 2022 [@url:https://www.whitehouse.gov/briefing-room/statements-releases/2022/01/14/fact-sheet-the-biden-administration-to-begin-distributing-at-home-rapid-covid-19-tests-to-americans-for-free].
These tests are particularly useful for ruling out SARS-CoV-2 infection in cases where the likelihood of infection is low (e.g., asymptomatic individuals) and positives (including false positives) can be validated with testing by alternate means [@doi:10/hk34].

##### Enzyme-Linked Immunosorbent Assay

ELISA is a very sensitive immunoassay that can be considered a gold standard for the detection of biological targets, including antibodies and antigenic proteins [@pubmed:32310382].
It can be used to generate either quantitative or qualitative results that can be returned within a few hours [@doi:10.1007/s00253-020-11061-5].
ELISA builds on the idea that antibodies and antigens bind together to form complexes [@pubmed:32310382] and utilizes an enzyme covalently linked to an antibody against the antigen to produce assay signal, usually a color change [@doi:10.1373/clinchem.2005.051532].
The main advantage of ELISA is that it enables signal amplification through the enzyme's activity, which increases sensitivity.
With sandwich ELISA, antibodies are immobilized on a surface such as a plate, and viral protein antigens in the sample bind and are retained [@doi:10.1080/01971528008055786].
A second antibody is added that binds to another site on the antigen is then added, and that second antibody is covalently linked to an enzyme.
A substrate for that enzyme is then added to produce signal, usually light or a color change
The exact strategy for tagging with a reporter enzyme varies among different types of ELISA [@doi:10.1080/01971528008055786; @pubmed:32310382].
For COVID-19 diagnostics, ELISAs have been designed to detect the antigenic Spike protein [@doi:10.12688/wellcomeopenres.15927.1].

One of these assays uses two monoclonal antibodies specific to the nucleocapsid of SARS-CoV-2 to evaluate the relationship between the effect of (estimated) viral load on the ability of the assay to detect the SARS-CoV-2 antigen [@doi:10.1080/14787210.2021.1976144].
This study analyzed 339 naso-oropharyngeal samples that were also analyzed with RT-qPCR as a gold standard.
RT-qPCR identified 147 samples as positive and 192 as negative.
The authors estimated the overall sensitivity and specificity to be 61.9% and 99.0%, respectively.
Sensitivity increased with higher C~t~.
This study also assessed the performance of the ELISA test under different conditions in order to evaluate how robust it would be to the challenges of testing in real-world settings globally.
Higher sensitivity was achieved for samples that were stored under ideal conditions (immediate placement in -80° C).
Therefore, while immediate access to laboratory equipment is an advantage, it is not strictly necessary for ELISA to detect the antigen.

#### Limitations of Molecular Tests

Tests that identify SARS-CoV-2 using molecular technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection.
Among molecular tests, different technologies have different sensitivities and specificities.
In general, specificity is high, and even then, the public health repercussions of a false positive can generally be mitigated with follow-up testing.
On the other hand, a test's sensitivity, which indicates the risk of a false-negative response, can pose significant challenge to large-scale testing.
False negatives are a significant concern for several reasons.
Importantly, clinical reports indicate that it is imperative to exercise caution when interpreting the results of molecular tests for SARS-CoV-2 because negative results do not necessarily mean a patient is virus-free [@doi:10.1128/JCM.00297-20].
To reduce occurrence of false negatives, correct execution of the analysis is crucial [@doi:10.1038/d41587-020-00002-2].
Additionally, PCR-based tests can remain positive for a much longer time than the virus is likely to be actively replicating [@doi:10.1056/NEJMcp2117115], raising concerns about their informativeness after the acute phase of the disease.
Hence, the CDC has advised individuals who suspect they have been re-infected with SARS-CoV-2 to avoid using diagnostic tests within 90 days of receiving a previous positive test [@url:https://www.cdc.gov/coronavirus/2019-ncov/your-health/quarantine-isolation.html].

Additionally, the emerging nature of the COVID-19 pandemic has introduced some challenges related to uncertainty surrounding interactions between SARS-CoV-2 and its human hosts.
For example, viral shedding kinetics are still not well understood but are expected to introduce a significant effect of timing of sample collection on test results [@doi:10.1038/d41587-020-00002-2].
Similarly, the type of specimen could also influence outcomes, as success in viral detection varies among clinical sample types [@doi:10.1038/d41587-020-00002-2; @doi:10/gjrfns; @doi:10.2807/1560-7917.ES.2020.25.50.2000568].
With CRISPR-based testing strategies, the gRNA can recognize off-target interspersed sequences in the viral genome [@doi:10.3389/fcimb.2020.576875], potentially resulting in false positives and a loss of specificity.

There are also significant practical and logistical concerns related to the widespread deployment of molecular tests.
Much of the technology used for molecular tests is expensive, and while it might be available in major hospitals and/or diagnostic centers, it is often not available to smaller facilities [@doi:10.1126/science.abb8400].
At times during the pandemic, the availability of supplies for testing, including swabs and testing media, has also been limited [@doi:10.1128/JCM.00512-20].
Similarly, processing times can be long, and tests might take up to 4 days to return results [@doi:10.1126/science.abb8400], especially during times of high demand, such as spikes in case numbers [@url:https://www.nytimes.com/2021/12/31/travel/covid-test-chaos.html].
Countries have employed various and differing molecular testing strategies as a tool to reduce viral transmission, even among high-income countries [@url:https://www.bloomberg.com/news/newsletters/2022-01-07/a-stark-contrast-between-the-u-s-and-europe-on-tests].
The rapid development of molecular tests has provided a valuable, albeit imperfect, tool to identify active SARS-CoV-2 infections.

### Serological Tests to Identify Recovered Individuals

Although several molecular diagnostic tests to detect viral genetic material have high specificity and sensitivity, they provide information only about active infection, and therefore offer just a snapshot-in-time perspective on the spread of a disease.
Most importantly, they would not work on a patient who has fully recovered from the virus at the time of sample collection.
In such contexts, serological tests are informative.

Serological tests use many of the same technologies as the immunoassays used to detect the presence of an antigen but are instead used to evaluate the presence of antibodies against SARS-CoV-2 in a serum sample.
These tests are particularly useful for insight into population-level dynamics and can also offer a glimpse into the development of antibodies by individual patients during the course of a disease.
Immunoassays can detect antibodies produced by the adaptive immune system in response to viral threat.
Understanding the acquisition and retention of antibodies is important both to the diagnosis of prior (inactive) infections and to the development of vaccines.
The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies [@doi:10.3389/fimmu.2014.00520; @url:https://www.ncbi.nlm.nih.gov/books/NBK27162].
Serological tests detect these antibodies, offering a mechanism through which prior infection can be identified.
However, the complexity of the human immune response means that there are many facets to such analyses.

In general, SARS-CoV-2 infection will induce the immune system to produce antibodies fairly quickly.
Prior research is available about the development of antibodies to SARS-CoV-1 during the course of the associated disease, severe acute respiratory syndrome (SARS).
IgM and IgG antibodies were detected in the second week following SARS-CoV-1 infection.
IgM titers peaked by the first month post-infection, and then declined to undetectable levels after day 180.
IgG titers peaked by day 60 and persisted in all donors through the two-year duration of study [@doi:10.1111/j.1440-1843.2006.00783.x].
Such tests can also illuminate the progression of viral disease, as IgM are the first antibodies produced by the body and indicate that the infection is active.
Once the body has responded to the infection, IgG are produced and gradually replace IgM, indicating that the body has developed immunogenic memory [@doi:10.1002/jmv.25820].
Therefore, it was hoped that the development of assays to detect the presence of IgM and IgG antibodies against SARS-CoV-2 would allow the identification of cases from early in the infection course (via IgM) and for months or years afterwards (via IgG).
Several technologies have been used to develop serological tests for COVID-19, including ELISA, lateral flow immunoassay, chemiluminescence immunoassay, and neutralizing antibody assays [@doi:10.1093/ajcp/aqaa112].

#### ELISA

The application of ELISA to serological testing is complementary to its use in molecular diagnostics (see above).
Instead of using an enzyme-labeled antibody as a probe that binds to the target antigen, the probe is an antigen and the target is an antibody.
The enzyme used for detection and signal amplification is on a secondary antibody raised generally against human IgG or IgM.
In March 2020, the Krammer lab proposed an ELISA test that detects IgG and IgM that react against the receptor-binding domain (RBD) of the spike proteins (S) of the virus [@doi:10.1038/s41591-020-0913-5].
A subsequent ELISA test developed to detect SARS-CoV-2 IgG based on the RBD reported a specificity of over 99% and a sensitivity of up to 88.24%, which was observed in samples collected 21 to 27 days after the onset of infection (approximated with symptom onset or positive PCR test) [@doi:10.3389/fmicb.2020.618097].
Earlier in the disease course, sensitivity was lower: 53.33% between days 0 and 13 and 80.47% between days 14 and 20.
This study reported that their laboratory ELISA outperformed two commercial kits that also used an ELISA design [@doi:10.3389/fmicb.2020.618097].
Therefore, while analysis with ELISA requires laboratory support and equipment, these results do suggest that ELISA achieves relatively high sensitivity, especially in the weeks following infection.
Efforts have been made to develop low-cost strategies for conducting these tests that will make them more accessible worldwide [@doi:10.1016/j.jim.2021.113182].

#### Chemiluminescence Immunoassay

Another early approach investigated for detection of antibodies against SARS-CoV-2 was CLIA.
Like ELISA, CLIA is a type of enzyme immunoassay (EIA) [@doi:10.2147/IJN.S200556].
While the technique varies somewhat, in one approach, a bead is coated with the antigen and then washed with the sample [@doi:10.1007/s13317-017-0097-2].
If the antibody is present in the sample, it will bind to the bead.
Then the bead is exposed to a label, a luminescent molecule that will bind to the antigen/antibody complex and can therefore be used as an indicator [@doi:10.1007/s13317-017-0097-2].
One CLIA approach to identify COVID-19 used a synthetic peptide derived from the amino acid sequence of the SARS-CoV-2 S protein [@doi:10.1093/infdis/jiaa243].
It was highly specific to SARS-CoV-2 and detected IgM in 57.2% and IgG in 71.4% of serum samples from 276 COVID-19 cases confirmed with RT-qPCR.
IgG could be detected within two days of the onset of fever, but IgM could not be detected any earlier [@doi:10.1093/infdis/jiaa243], which has been supported by other analyses as well [@doi:10.1016/j.jcv.2020.104690].
This pattern was consistent with observations in Middle East respiratory syndrome, which is also caused by an HCoV.
In comparisons of different commercial immunoassays, accuracy of CLIA tests were often roughly comparable to other EIAs [@doi:10.1016/j.jcv.2020.104511], although one CLIA did not perform as well as several other EIAs [@doi:10.1016/j.jcv.2020.104690; @doi:10.1128/JCM.01361-20].
The sensitivity and specificities reported vary among CLIA tests and for the detection of IgM versus IgG, but sensitivities and specificities as high as 100% have been reported among various high-throughput tests [@doi:10.1002/jmv.25932; @doi:10.1128/JCM.01361-20; @doi:10.1016/j.jcv.2021.104914].
CLIA has previously been used to develop tests that can be used at point of care (e.g., [@doi:10.2147/IJN.S200556]) which may allow for this technique to become more widely accessible in the future.

#### Lateral Flow Immunoassay

The first serological test approved for emergency use in the United States was developed by Cellex [@url:https://www.fda.gov/media/136625/download].
The Cellex qSARS-CoV-2 IgG/IgM Rapid Test is a chromatographic immunoassay, also known as a lateral flow immunoassay, designed to qualitatively detect IgM and IgG antibodies against SARS-CoV-2 in the plasma of patients suspected to have developed a SARS-CoV-2 infection [@url:https://www.fda.gov/media/136625/download].
The Cellex test cassette contains a pad of SARS-CoV-2 antigens and a nitrocellulose strip with lines for each of IgG and IgM, as well as a control (goat IgG) [@url:https://www.fda.gov/media/136625/download].
In a specimen that contains antibodies against the SARS-CoV-2 antigen, the antibodies will bind to the strip and be captured by the IgM and/or IgG line(s), resulting in a change of color [@url:https://www.fda.gov/media/136625/download].
With this particular assay, results can be read within 15 to 20 minutes [@url:https://www.fda.gov/media/136625/download].
Lateral flow immunoassays are often available at point of care but can have very low sensitivity [@doi:10.1128/JCM.01361-20].

#### Neutralizing Antibody Assays

Neutralizing antibody assays play a functional role in understanding immunity that distinguishes them from other serological tests.
The tests described above are all binding antibody tests.
On the other hand, rather than simply binding an antibody to facilitate detection, neutralizing antibody assays determine whether an antibody response is present that would prevent infection [@doi:10.1128/Spectrum.01202-21; @doi:10.1038/s41467-020-17892-0].
Therefore, these tests serve the purpose of evaluating the extent to which a sample donor has acquired immunity that will reduce susceptibility to SARS-CoV-2.
As a result, neutralizing antibody assays have been used widely to characterize the duration of immunity following infection, to assess vaccine candidates, and to establish correlates of protection against infection and disease [@doi:10.1038/s41591-021-01377-8; @doi:10.1128/JCM.02107-20; @doi:10/gnx76z].
These tests are typically performed in a laboratory [@doi:10.1128/Spectrum.01202-21], and in SARS-CoV-2, the results of neutralizing antibody assays are often correlated with the results of binding antibody tests [@doi:10.1128/Spectrum.01202-21].

The gold standard for assessing the presence of neutralizing antibodies is the plaque reduction neutralization test (PRNT), but this approach does not scale well [@doi:10.1038/s41467-020-17892-0].
An early high-throughput neutralizing antibody assay designed against SARS-CoV-2 used a fluorescently labeled reporter virus that was incubated with different dilutions of patient serum [@doi:10.1038/s41467-020-17892-0].
The cells used for incubation would turn green if antibodies were not present.
Essentially, this assay evaluates whether the virus is able to infect the cell in the presence of the serum.
The specificity of this assay was 100%, and the correlation between the results of this assay and of PRNT was 0.85 with the results suggesting that the sensitivity of the high-throughput approach was higher than that of PRNT [@doi:10.1038/s41467-020-17892-0].
While this approach was performed on a plate and using cells, other methods have been developed using methods such as bead arrays [@doi:10.1016/j.nbt.2021.10.002].

#### Duration of Immune Indicators

While the adaptive immune system produces antibodies in response to SARS-CoV-2 viral challenge, these indicators of seroconversion are unlikely to remain in circulation permanently.
Previously, a two-year longitudinal study following convalesced SARS patients with a mean age of 29 found that IgG antibodies were detectable in all 56 patients surveyed for at least 16 months and remained detectable in all but 4 patients (11.8%) through the full two-year study period [@doi:10.1086/500469].
These results suggest that immunity to SARS-CoV-1 is sustained for at least a year.
Circulating antibody titers to other coronaviruses have been reported to decline significantly after one year [@doi:10.1017/s0950268800048019].
Evidence to date suggests that sustained immunity to the SARS-CoV-2 virus remains for a shorter period of time but at least 6 to 8 months after infection [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @pmid:33443036; @doi:10.1016/j.medj.2021.02.001].
However, this does not mean that all serological evidence of infection dissipates, but rather that the immune response becomes insufficient to neutralize the virus.

In order to study the persistence of SARS-CoV-2 antibodies, one study assessed sustained immunity using 254 blood samples from 188 COVID-19 positive patients [@doi:10.1126/science.abf4063].
The samples were collected at various time points between 6 and 240 days post-symptom onset; some patients were assessed longitudinally.
Of the samples, 43 were collected at least 6 months after symptom onset.
After one month, 98% of patients were seropositive for IgG to S.
Moreover, S IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, with 90% of subjects seropositive at 6 months.
Similarly, at 6 to 8 months 88% of patients were seropositive for RBD IgG, and 90% were seropositive for SARS-CoV-2 neutralizing antibodies.
Another study examined 119 samples from 88 donors who had recovered from mild to severe cases of COVID-19 [@doi:10.1016/j.medj.2021.02.001].
A relatively stable level of IgG and plasma neutralizing antibodies was identified up to 6 months post diagnosis.
Significantly lower but considerable levels of anti-SARS-CoV-2 IgG antibodies were still present in 80% of samples obtained 6 to 8 months post-symptom onset.

Titers of IgM and IgG antibodies against the RBD were found to decrease from 1.3 to 6.2 months post infection in a study of 87 individuals [@doi:10.1038/s41586-021-03207-w].
However, the decline of IgA activity (15%) was less pronounced than that of IgM (53%) or IgG (32%).
It was noted that higher levels of anti-RBD IgG and anti-N total antibodies were detected in individuals that reported persistent post-acute symptoms at both study visits.
Moreover, plasma neutralizing activity decreased five-fold between 1.3 and 6.2 months in an assay of HIV-1 virus pseudotyped with SARS-CoV-2 S protein, and this neutralizing activity was directly correlated with IgG anti-RBD titers [@doi:10.1038/s41586-021-03207-w].
These findings are in accordance with other studies that show that the majority of seroconverters have detectable, albeit decreasing, levels of neutralizing antibodies at least 3 to 6 months post infection [@doi:10.1126/science.abd7728; @doi:10.1038/s41564-020-00813-8; @doi:10.1038/s41467-021-21444-5].

Determining the potency of anti-RBD antibodies early in the course of an infection may be important moving forward, as their neutralizing potency may be prognostic for disease severity and survival [@doi:10.1016/j.cell.2020.12.015].
The duration of immunity might also vary with age [@doi:10.1038/s41590-021-01089-8] or ABO blood type [@doi:10.1080/21505594.2021.2019959].
Autopsies of lymph nodes and spleens from severe acute COVID-19 patients showed a loss of T follicular helper cells and germinal centers that may explain some of the impaired development of antibody responses [@doi:10.1016/j.cell.2020.08.025].
Therefore, serological testing may be time-limited in its ability to detect prior infection.

Other immune indicators of prior infection have also been evaluated to see how they persist over time.
SARS-CoV-2 memory CD8^+^ T cells were slightly decreased (50%) 6 months post-symptom onset.
In this same subset of COVID-19 patients, 93% of subjects had detectable levels of SARS-CoV-2 memory CD4^+^ T cells, of which 42% had more than 1% SARS-CoV-2-specific CD4^+^ T cells.
At 6 months, 92% of patients were positive for SARS-CoV-2 memory CD4^+^ T cells.
Indeed, the abundance of S-specific memory CD4^+^ T cells over time was similar to that of SARS-CoV-2-specific CD4^+^ T cells overall [@doi:10.1126/science.abf4063].
T cell immunity to SARS-CoV-2 at 6 to 8 months following symptom onset has also been confirmed by other studies [@doi:10.1101/2020.11.01.362319; @doi:10.1101/2020.12.08.416636; @doi:10.1016/j.medj.2021.02.001].
In another study, T cell reactivity to SARS-CoV-2 epitopes was also detected in some individuals never been exposed to SARS-CoV-2.
This finding suggests the potential for cross-reactive T cell recognition between SARS-CoV-2 and pre-existing circulating HCoV that are responsible for the “common cold” [@doi:10.1016/j.cell.2020.05.015], but further research is required.
Therefore, whether T cells will provide a more stable measure through which to assess prior infection remains unknown.
Notably, commercial entities have tried to develop tests specifically for T cells, some of which have been authorized by the United States Food and Drug Administration [@doi:10.1038/s41587-021-00920-9; @url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-adaptive-biotechnologies-t-detect-covid-test] to identify people with adaptive T cell immune responses to SARS-CoV-2, either from a previous or ongoing infection.

#### Applications of Serological Tests

In addition to the limitations posed by the fact that antibodies are not permanent indicators of prior infection, serological immunoassays carry a number of limitations that influence their utility in different situations.
Importantly, false positives can occur due to cross-reactivity with other antibodies according to the clinical condition of the patient [@url:https://www.fda.gov/media/136625/download].
Due to the long incubation times and delayed immune responses of infected patients, serological immunoassays are insufficiently sensitive for a diagnosis in the early stages of an infection.
Therefore, such tests must be used in combination with RNA detection tests if intended for diagnostic purposes [@doi:10.1056/NEJMp2025631].
False positives are particularly harmful if they are erroneously interpreted to mean that a population is more likely to have acquired immunity to a disease [@doi:10/gg559s].
Similarly, while serological tests may be of interest to individuals who wish to confirm they were infected with SARS-CoV-2 in the past, their potential for false positives means that they are not currently recommended for this use.
However, in the wake of vaccines becoming widely available, accurate serological tests that could be administered at point of care were investigated in the hope that they could help to prioritize vaccine recipients [@doi:10.1371/journal.pone.0248729].
Another concern with serological testing is the potential for viral evolution to reduce the sensitivity of assays, especially for neutralizing antibody assays.
Chen et al. performed a systematic re-analysis of published data examining the neutralizing effect of serum from vaccinated or recovered individuals on four VOC [@doi:10.1093/cid/ciab646].
They found reduced neutralizing titers against these variants relative to the lineages used for reference.
These findings suggest that such techniques will need to be modified over time as SARS-CoV-2 evolves.

These limitations make serological tests far less useful for diagnostics and for test-and-trace strategies; however, serological testing is valuable for public health monitoring at the population level.
Serosurveys provide a high-level perspective of the prevalence of a disease and can provide insight into the susceptibility of a population as well as variation in severity, e.g., between geographic regions [@doi:10/gg559s].
From a public health perspective, they can also provide insight into the effectiveness of mitigation efforts and to gain insight into risk factors influencing susceptibility [@doi:10/gh7598].
EIA methods are high-throughput [@doi:10.3390/diagnostics10070453; @doi:10.5694/mja2.50766], and, as with molecular tests, additional efforts have been made to scale up the throughput of serological tests [@doi:10.1039/D0LC01068E].
Therefore, serological tests can be useful to developing strategies for the management of viral spread.

Early in the course of the pandemic, it was also hoped that serological tests would provide information relevant to advancing economic recovery.
Some infectious agents can be controlled through "herd immunity", which is when a critical mass within the population acquires immunity through vaccination and/or infection, preventing an infectious agent from spreading widely.
It was hoped that people who had recovered and developed antibodies might be able to return to work [@doi:10.1101/2020.04.14.20065771; @doi:10.1001/jamainternmed.2020.7986].
This strategy would have relied on recovered individuals acquiring long-term immunity, which has not been borne out [@doi:10.1101/2021.12.04.21267114].
Additionally, it was hoped that identifying seroconverters and specifically those who had mounted a strong immune response would reveal strong candidates for convalescent plasma donation [@doi:10.1038/s41591-020-0913-5]; however, convalescent plasma has not been found to offer therapeutic benefit (reviewed in [@individual-pharmaceuticals]).
While these hopes have not been realized, serological tests have been useful for gaining a better understanding of the pandemic [@doi:10/gh7598].

### Possible Alternatives to Current Diagnostic Practices

One possible alternative or complement to molecular and serological testing would be diagnosing COVID-19 cases based on symptomatology.
COVID-19 can present with symptoms similar to other types of pneumonia, and symptoms can vary widely among COVID-19 patients; therefore, clinical presentation is often insufficient as a sole diagnostic criterion.
In addition, identifying and isolating mild or asymptomatic cases is critical to efforts to manage outbreaks.
Even among mildly symptomatic patients, a predictive model based on clinical symptoms had a sensitivity of only 56% and a specificity of 91% [@doi:10.2807/1560-7917.ES.2020.25.16.2000508].
More problematic is that clinical symptom-based tests are only able to identify already symptomatic cases, not presymptomatic or asymptomatic cases.
They may still be important for clinical practice and for reducing tests needed for patients deemed unlikely to have COVID-19.

In some cases, clinical signs may also provide information that can inform diagnosis.
Using computed tomography of the chest in addition to RT-qPCR testing was found to provide a higher sensitivity than either measure alone [@doi:10.1101/2020.02.25.20027755].
X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies [@doi:10.1148/radiol.2020200642].
Other studies have shown that specificity varies between radiologists [@doi:10.1148/radiol.2020200823], though the sensitivity reported here was lower than that published in the previous paper.
While preliminary machine-learning results suggested that chest X-rays might provide high sensitivity and specificity and potentially facilitate the detection of asymptomatic and presymptomatic infections (e.g., [@doi:10.1007/s13246-020-00865-4]), further investigation suggested that such approaches are prone to bias and are unlikely to be clinically useful [@doi:10.1038/s42256-021-00307-0].
Given the above, the widespread use of X-ray tests on otherwise healthy adults is likely inadvisable.

Finally, in addition to genomic and serological surveillance, other types of monitoring have proven useful in managing the pandemic [@doi:10.1016/j.jiph.2020.07.019].
One that has received significant attention is wastewater surveillance.
This approach can use several of the technologies described for molecular testing, such as qPCR and dPCR, as well as _in vitro_ culturing [@doi:10.1016/j.coesh.2020.06.001] and can provide insight into trends in the prevalence of SARS-CoV-2 regionally.

### Strategies and Considerations for Testing

Deciding whom to test, when to test, and which test to use have proven challenging as the COVID-19 pandemic has unfolded.
Early in the COVID-19 pandemic, testing was typically limited to individuals considered high risk for developing serious illness [@url:https://www.cdc.gov/coronavirus/2019-nCoV/hcp/clinical-criteria.html].
This approach often limited testing to people with severe symptoms and people showing mild symptoms that had been in contact with a person who had tested positive.
Individuals who were asymptomatic (i.e., potential spreaders) and individuals who were able to recover at home were thus often unaware of their status.
Therefore, this method of testing administration misses a high proportion of infections and does not allow for test-and-trace methods to be used.
For instance, a study from Imperial College estimates that in Italy, the true number of infections was around 5.9 million in a total population of ~60 million, compared to the 70,000 detected as of March 28, 2020 [@doi:10.25561/77731].
Another analysis, which examined New York state, indicated that as of May 2020, approximately 300,000 cases had been reported in a total population of approximately 20 million [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
This corresponded to ~1.5% of the population, but ~12% of individuals sampled statewide were estimated as positive through antibody tests (along with indications of spatial heterogeneity at higher resolution) [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
Technological advancements that facilitate widespread, rapid testing would therefore be valuable for accurately assessing the rate of infection and aid in controlling the virus' spread.
Additionally, the trade off of accessibility, sensitivity, and time to results has raised some complex questions around which tests are best suited to certain situations.
Immunoassays, including serological tests, have much higher limits of detection than PCR tests do [@doi:10.2142/biophysico.bppb-v18.004].

Changes in public attitudes and the lifting of COVID-19 restrictions due to the multifactorial desire to stimulate economic activities has required a shift of testing paradigms in 2022, despite warnings from public health officials against a hard exit from public health restrictions [@doi:10.1136/bmj.o383; @doi:10.1186/s12889-022-12845-2].
An important strategy for testing moving forward is to determine when someone becomes infectious or is no longer infectious following a positive test for COVID-19.
Generally, patient specimens tend to not contain culturable virus past day 5 of symptom onset [@doi:10.1093/cid/ciaa1764; @{doi:10.1016/S2666-5247(20)30172-5}].
However, due to their sensitivity to post-infectious viral RNA in specimens, PCR-based methods may mislead individuals to believe that they are still infectious several days after symptom onset [@doi:10.1056/NEJMp2025631].
Furthermore, detection of viral RNA can occur days and weeks after an active infection due to the sensitivity of PCR-based methods [@doi:10.1038/s41420-020-00375-y; @doi:10.1038/s41467-020-20568-4; @doi:10.1001/jamanetworkopen.2021.42796].

In contrast, LFTs were thought to have poor sensitivity and their value for identifying infections and managing the pandemic was questioned [@doi:10.1136/bmj.m4941; @doi:10.1136/bmj.n81].
However, LFTs do reliably detect SARS-CoV-2 proteins when there is a high viral load, which appears to correlate with a person’s infectiousness [@doi:10.1128/jcm.02479-21; @doi:10.1056/NEJMcp2117115].
Therefore, LFTs are an important diagnostic tool to determine infectiousness with fast turnaround times, ease of use, and accessibility by the general public [@doi:10/gnmbjd; @doi:10.1056/NEJMp2025631].
One study has suggested that the test sensitivity of LFTs appears to be less important than accessibility to LFTs, frequent testing, and fast reporting times for reducing the impact of viral spread [@doi:10.1126/sciadv.abd5393].
While PCR-based methods are important for COVID-19 surveillance, their use is labor intensive and time consuming, and laboratories are often slow to report results, rendering such methods limited in their surveillance capacity [@doi:10.1056/NEJMp2025631].

These limitations are demonstrated by the estimated 10-fold under-reporting of cases in the United States in 2020 due to shortages in testing and slow rollout of testing and slow reporting of results [@url:https://www.washingtonpost.com/health/2020/06/25/coronavirus-cases-10-times-larger].
However, one strategy that may balance the strengths and weaknesses of both types of tests is to corroborate a positive LFT result using a PCR-based method.
Indeed, in December 2021 sufficient surveillance and reduction of COVID-19 spread using this joint LFT-PCR strategy was demonstrated in Liverpool, U.K., where there was an estimated 21% reduction of cases [@url:https://www.gov.uk/government/publications/liverpool-coronavirus-covid-19-community-testing-pilot-full-evaluation-report-summary/liverpool-coronavirus-covid-19-community-testing-pilot-full-evaluation-report-summary; @doi:10/gnmbjd].

### What Lies Ahead

Diagnostic tools have played an important role during the COVID-19 pandemic.
Different tests offer different advantages (Figure @fig:diag).
Specifically, the results of SARS-CoV-2 diagnostic tests (typically qPCR or LFT-based tests) have been used to estimate the number of infections in the general population, thus informing public health strategies around the globe [@doi:10.1038/s41576-021-00360-w].
During the surges caused by the different SARS-CoV-2 variants between 2020 and 2021, government-sponsored efforts to conduct mass testing and to provide free diagnostic tests to the population were a common occurrence in many parts of the world [@doi:10.1136/bmj.n902; @doi:10.1177/0020852321997552; @doi:10.1016/j.worlddev.2020.105208].
However, recent reports indicate that such public health policies are starting to change during 2022.
For example, it is known that the UK plans to dismantle its COVID-19 testing program and scale back its daily reporting requirements [@url:https://www.independent.co.uk/news/health/who-covid-testing-anil-soni-b2032884.html; @doi:10.1136/bmj.o562].
A similar approach can be seen in the US as well, where multiple state-run testing facilities are closing, despite some groups advocating to keep them open [@url:https://www.9news.com/article/news/health/coronavirus/local-groups-continue-push-for-covid-testing-and-vaccinations/73-bbcd8384-d96a-425e-aaeb-16ac9f36e581; @url:https://www.sltrib.com/news/2022/02/18/utah-will-stop-daily].
These ongoing changes in testing policy are likely to have a direct effect on how the pandemic is managed moving forward.
SARS-CoV-2 diagnostic tests can be used effectively to slow the spread of the disease only when 1) they are used to share testing results in a timely manner so that they can reasonably be used to approximate the number of infections in the population and 2) those tests are easily accessible by the general public.

![
**Summary of Diagnostic Technologies used in COVID-19 Testing.**
The immune response to SARS-CoV-2 means that different diagnostic approaches offer different views of COVID-19.
Early in the infection course, viral load is high.
This means that PCR-based testing and EIA testing for antigens are likely to return positives (as indicated by the green bars at the bottom).
As viral load decreases, EIA antigen tests become negative, but PCR-based tests can still detect even very low viral loads.
From a serological perspective, IgM peaks in the first few weeks following infection and then decreases, while IgG peaks much later in the infection course.
Therefore, serological tests are likely to return positives in first few months following the acute infection course.
Additional detail is available above and in several analyses and reviews [@doi:10.1056/NEJMp2025631; @doi:10.1038/s41591-022-01780-9; @doi:10.1001/jamanetworkopen.2021.42796; @individual-pathogenesis].
](images/diagnostics.png "Summary of COVID-19 diagnostic technologies"){#fig:diag secno=1}

Children are one segment of the population where the importance of the two aforementioned conditions can be exemplified.
This group is particularly vulnerable as there are ongoing challenges with testing in schools, increased COVID-19 mortality rates, and COVID-19-associated orphanhood.
In this regard, although there is evidence of the efficacy of routine diagnostic testing to reduce the probability of having infectious students [@doi:10.3389/fpubh.2021.751451; @doi:10.2807/1560-7917.ES.2021.26.34.2100184], as of March of 2022 there is an increasing number of schools that have stopped or plan to stop contact tracing efforts [@url:https://www.opb.org/article/2022/03/02/oregon-schools-guidance-mask-mandate-end; @url:https://www.wptv.com/news/education/palm-beach-county-public-schools-to-stop-covid-19-contact-tracing], in line with an announcement made by the CDC where it no longer recommended contact tracing as a strategy to contain the virus [@url:https://www.nytimes.com/live/2022/03/02/world/covid-19-tests-cases-vaccine].
An estimated 197 children have died in the US from COVID-19 during the first three months of 2022 [@url:https://www.aap.org/en/pages/2019-novel-coronavirus-covid-19-infections/children-and-covid-19-state-level-data-report], compared to 735 deaths in the preceding 20 months of the pandemic [@url:https://www.theguardian.com/world/2022/mar/11/us-child-covid-deaths-omicron-surge], and millions of children have been orphaned as a consequence of parent or caregiver death due to COVID-19 [@{doi:10.1016/S2352-4642(22)00005-0}].
It is likely that reducing or eliminating testing capacity in schools will directly exacerbate these negative outcomes for the remainder of 2022.

The SARS-CoV-2 diagnostic tools presented in this paper are far less useful if they are difficult to obtain, or if their limited use results in biased data that would lead to ill-informed public health strategies.
Under conditions of limited supply, different strategies for testing are needed [@url:https://openknowledge.worldbank.org/handle/10986/35150].
The pandemic is still an ongoing public health threat and it is worrying that active testing and tracing efforts are a low priority for public health authorities in many countries.
If this trend continues, the lack of testing could result in increased morbidity and mortality and an overall failure to manage the pandemic.
