## Identification and Development of Therapeutics for COVID-19

### Abstract

After emerging in China in late 2019, the novel coronavirus SARS-CoV-2 spread worldwide and as of mid-2021 remains a significant threat globally.
Only a few coronaviruses are known to infect humans, and only two cause infections similar in severity to SARS-CoV-2: _Severe acute respiratory syndrome-related coronavirus_, a closely related species of SARS-CoV-2 that emerged in 2002, and _Middle East respiratory syndrome-related coronavirus_, which emerged in 2012.
Unlike the current pandemic, previous epidemics were controlled rapidly through public health measures, but the body of research investigating severe acute respiratory syndrome and Middle East respiratory syndrome has proven valuable for identifying approaches to treating and preventing novel coronavirus disease 2019 (COVID-19).
Building on this research, the medical and scientific communities have responded rapidly to the COVID-19 crisis to identify many candidate therapeutics.
The approaches used to identify candidates fall into four main categories: adaptation of clinical approaches to diseases with related pathologies, adaptation based on virological properties, adaptation based on host response, and data-driven identification of candidates based on physical properties or on pharmacological compendia.
To date, a small number of therapeutics have already been authorized by regulatory agencies such as the Food and Drug Administration (FDA), while most remain under investigation.
The scale of the COVID-19 crisis offers a rare opportunity to collect data on the effects of candidate therapeutics.
This information provides insight not only into the management of coronavirus diseases, but also into the relative success of different approaches to identifying candidate therapeutics against an emerging disease.

### Importance

The COVID-19 pandemic is a rapidly evolving crisis.
With the worldwide scientific community shifting focus onto the SARS-CoV-2 virus and COVID-19, a large number of possible pharmaceutical approaches for treatment and prevention have been proposed.
What was known about each of these potential interventions evolved rapidly throughout 2020 and 2021.
This fast-paced area of research provides important insight into how the ongoing pandemic can be managed and also demonstrates the power of interdisciplinary collaboration to rapidly understand a virus and match its characteristics with existing or novel pharmaceuticals.
As illustrated by the continued threat of viral epidemics during the current millennium, a rapid and strategic response to emerging viral threats can save lives.
In this review, we explore how different modes of identifying candidate therapeutics have borne out during COVID-19.

### Introduction

The novel coronavirus _Severe acute respiratory syndrome-related coronavirus 2_ (SARS-CoV-2) emerged in late 2019 and quickly precipitated the worldwide spread of novel coronavirus disease 2019 (COVID-19).
COVID-19 is associated with symptoms ranging from mild or even asymptomatic to severe, and up to 2% of patients diagnosed with COVID-19 die from COVID-19-related complications such as acute respiratory disease syndrome (ARDS) [@individual-pathogenesis].
As a result, public health efforts have been critical to mitigating the spread of the virus.
However, as of mid-2021, COVID-19 remains a significant worldwide concern (Figure @fig:csse-deaths), with 2021 cases in some regions surging far above the numbers reported during the initial outbreak in early 2020.
While a number of vaccines have been developed and approved in different countries starting in late 2020 [@individual-vaccines], vaccination efforts have not proceeded at the same pace throughout the world and are not yet close to ending the pandemic.

Due to the continued threat of the virus and the severity of the disease, the identification and development of therapeutic interventions have emerged as significant international priorities.
Prior developments during other recent outbreaks of emerging diseases, especially those caused by human coronaviruses (HCoV), have guided biomedical research into the behavior and treatment of this novel coronavirus infection.
However, previous emerging HCoV-related disease threats were controlled much more quickly than SARS-CoV-2 through public health efforts (Figure @fig:csse-deaths).
The scale of the COVID-19 pandemic has made the repurposing and development of pharmaceuticals more urgent than in previous coronavirus epidemics.

#### Lessons from Prior HCoV Outbreaks

![
**Cumulative global incidence of COVID-19 and SARS.**
As of {{csse_date_pretty}}, {{csse_cases}} COVID-19 cases and {{csse_deaths}} COVID-19 deaths had been reported worldwide since January 22, 2020.
A total of 8,432 cases and 813 deaths were reported for SARS from March 17 to July 11, 2003.
SARS-CoV-1 was officially contained on July 5, 2003, within 9 months of its appearance [@doi:10.3390/ijerph17113973].
In contrast, SARS-CoV-2 remains a significant global threat nearly two years after its emergence.
COVID-19 data are from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University [@https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series; @doi:10/ggnsjk].
SARS data are from the WHO [@url:https://www.who.int/csr/sars/country/en] and were obtained from a dataset on GitHub [@url:https://github.com/imdevskp/sars-2003-outbreak-data-webscraping-code].
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.
]({{csse_deaths_figure}} "Global incidence of COVID-19 and SARS"){#fig:csse-deaths secno=1}

At first, SARS-CoV-2's rapid shift from an unknown virus to a significant worldwide threat closely paralleled the emergence of _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV-1), which was responsible for the 2002-03 SARS epidemic.
The first documented case of COVID-19 was reported in Wuhan, China in November 2019, and the disease quickly spread worldwide in the early months of 2020.
In comparison, the first case of SARS was reported in November 2002 in the Guangdong Province of China, and it spread within China and then into several countries across continents during the first half of 2003 [@doi:10.1093/ajcp/aqaa029; @doi:10.1038/nrmicro.2016.81; @doi:10.3390/ijerph17113973].
In fact, genome sequencing quickly revealed the virus causing COVID-19 to be a novel betacoronavirus closely related to SARS-CoV-1 [@doi:10.46234/ccdcw2020.017].

While similarities between these two viruses are unsurprising given their close phylogenetic relationship, there are also some differences in how the viruses affect humans.
SARS-CoV-1 infection is severe, with an estimated case fatality rate (CFR) for SARS of 9.5% [@doi:10.1093/ajcp/aqaa029], while estimates of the CFR associated with COVID-19 are much lower, at up to 2% [@individual-pathogenesis].
SARS-CoV-1 is highly contagious and spread primarily by droplet transmission, with a basic reproduction number (R~0~) of 4 (i.e., each person infected was estimated to infect four other people) [@doi:10.1093/ajcp/aqaa029].
There is still some controversy whether SARS-CoV-2 is primarily spread by droplets or is primarily airborne [@doi:10.1001/jama.2020.12458; @doi:10/gg6br7; @doi:10/gjqmvq; @doi:10.1136/bmj.n913]. 
Most estimates of its R~0~ fall between 2.5 and 3 [@individual-pathogenesis].
Therefore, SARS is thought to be a deadlier and more transmissible disease than COVID-19.

With the 17-year difference between these two outbreaks, there were major differences in the tools available to efforts to organize international responses.
At the time that SARS-CoV-1 emerged, no new HCoV had been identified in almost 40 years [@doi:10.1038/nrmicro.2016.81].
The identity of the virus underlying the SARS disease remained unknown until April of 2003, when the SARS-CoV-1 virus was characterized through a worldwide scientific effort spearheaded by the World Health Organization (WHO) [@doi:10.1038/nrmicro.2016.81].
In contrast, the SARS-CoV-2 genomic sequence was released on January 3, 2020 [@doi:10.46234/ccdcw2020.017], only days after the international community became aware of the novel pneumonia-like illness now known as COVID-19.
While SARS-CoV-1 belonged to a distinct lineage from the two other HCoVs known at the time of its discovery [@doi:10.1093/ajcp/aqaa029], SARS-CoV-2 is closely related to SARS-CoV-1 and is a more distant relative of another HCoV characterized in 2012, _Middle East respiratory syndrome-related coronavirus_ [@doi:10/ggjr43; @doi:10.1056/NEJMoa1211721].
Significant efforts had been dedicated towards understanding SARS-CoV-1 and MERS-CoV and how they interact with human hosts.
Therefore, SARS-CoV-2 emerged under very different circumstances than SARS-CoV-1 in terms of scientific knowledge about HCoVs and the tools available to characterize them.

Despite the apparent advantages for responding to SARS-CoV-2 infections, COVID-19 has caused many orders of magnitude more deaths than SARS did (Figure @fig:csse-deaths).
The SARS outbreak was officially determined to be under control in July 2003, with the success credited to infection management practices such as mask wearing [@doi:10.1038/nrmicro.2016.81].
_Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) is still circulating and remains a concern; although the fatality rate is very high at almost 35%, the disease is much less easily transmitted, as its R~0~ has been estimated to be 1 [@doi:10.1093/ajcp/aqaa029].
The low R~0~ in combination with public health practices allowed for its spread to be contained [@doi:10.1093/ajcp/aqaa029].
Neither of these trajectories are comparable to that of SARS-CoV-2, which remains a serious threat worldwide over a year and a half after the first cases of COVID-19 emerged (Figure @fig:csse-deaths).

#### Potential Approaches to the Treatment of COVID-19

Therapeutic interventions can utilize two approaches: they can either mitigate the effects of an infection that harms an infected person, or they can hinder the spread of infection within a host by disrupting the viral life cycle.
The goal of the former strategy is to reduce the severity and risks of an active infection, while for the latter, it is to inhibit the replication of a virus once an individual is infected, potentially freezing disease progression.
Additionally, two major approaches can be used to identify interventions that might be relevant to managing an emerging disease or a novel virus: drug repurposing and drug development.
Drug repurposing involves identifying an existing compound that may provide benefits in the context of interest [@doi:10.1038/nrd.2018.168].
This strategy can focus on either approved or investigational drugs, for which there may be applicable preclinical or safety information [@doi:10.1038/nrd.2018.168].
Drug development, on the other hand, provides an opportunity to identify or develop a compound specifically relevant to a particular need, but it is often a lengthy and expensive process characterized by repeated failure [@doi:10.1016/j.trci.2017.10.005].
Drug repurposing therefore tends to be emphasized in a situation like the COVID-19 pandemic due to the potential for a more rapid response.

Even from the early months of the pandemic, studies began releasing results from analyses of approved and investigational drugs in the context of COVID-19.
The rapid timescale of this response meant that, initially, most evidence came from observational studies, which compare groups of patients who did and did not receive a treatment to determine whether it may have had an effect.
This type of study can be conducted rapidly but is subject to confounding.
In contrast, randomized controlled trials (RCTs) are the gold-standard method for assessing the effects of an intervention.
Here, patients are prospectively and randomly assigned to treatment or control conditions, allowing for much stronger interpretations to be drawn; however, data from these trials take much longer to collect.
Both approaches have proven to be important sources of information in the development of a rapid response to the COVID-19 crisis, but as the pandemic draws on and more results become available from RCTs, more definitive answers are becoming available about proposed therapeutics.
Interventional clinical trials are currently investigating or have investigated a large number of possible therapeutics and combinations of therapeutics for the treatment of COVID-19 (Figure @fig:ebm-trials).

![
**COVID-19 clinical trials.**
Trials data are from the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
As of {{ebm_date_pretty}}, there were {{ebm_all_trials}} COVID-19 clinical trials of which {{ebm_interv_trials}} were interventional.
The study types include only types used in at least five trials.
Only interventional trials are analyzed in the figures depicting status, phase, and intervention.
Of the interventional trials, {{ebm_trials_results}} trials had reported results as of {{ebm_date_pretty}}.
Recruitment status and trial phase are shown only for interventional trials in which the status or phase is recorded.
Common interventions refers to interventions used in at least ten trials.
Combinations of interventions, such as hydroxychloroquine with azithromycin, are tallied separately from the individual interventions.
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.
]({{ebm_trials_figure}} "COVID-19 clinical trials"){#fig:ebm-trials secno=1}

The purpose of this review is to provide an evolving resource tracking the status of efforts to repurpose and develop drugs for the treatment of COVID-19.
We highlight four strategies that provide different paradigms for the identification of potential pharmaceutical treatments.
The WHO guidelines [@doi:10.1136/bmj.m3379] and a systematic review [@doi:10.1136/bmj.m2980] are complementary living documents that summarize COVID-19 therapeutics.


### Repurposing Drugs for Symptom Management

A variety of symptom profiles with a range of severity are associated with COVID-19 [@individual-pathogenesis].
In many cases, COVID-19 is not life threatening.
A study of COVID-19 patients in a hospital in Berlin, Germany reported that the highest risk of death was associated with infection-related symptoms, such as sepsis, respiratory symptoms such as ARDS, and cardiovascular failure or pulmonary embolism [@doi:10.1101/2020.06.15.20131540].
Similarly, an analysis in Wuhan, China reported that respiratory failure (associated with ARDS) and sepsis/multi-organ failure accounted for 69.5% and 28.0% of deaths, respectively, among 82 deceased patients [@doi:10.1371/journal.pone.0235458].
COVID-19 is characterized by two phases.
The first is the acute response, where an adaptive immune response to the virus is established and in many cases can mitigate viral damage to organs [@doi:10.1038/s41418-020-0530-3].
The second phase characterizes more severe cases of COVID-19.
Here, patients experience a cytokine storm, whereby excessive production of cytokines floods into circulation, leading to systemic inflammation, immune dysregulation, and multiorgan dysfunction that can cause multiorgan failure and death if untreated [@doi:10.1056/NEJMra2026131].
ARDS-associated respiratory failure can occur during this phase.
Cytokine dysregulation was also identified in patients with SARS [@doi:10/c8mmbg; @doi:10.1186/cc2452]. 

In the early days of the COVID-19 pandemic, physicians sought to identify potential treatments that could benefit patients, and in some cases shared their experiences and advice with the medical community on social media sites such as Twitter [@doi:10.7759/cureus.13594].
These on-the-ground treatment strategies could later be analyzed retrospectively in observational studies or investigated in an interventional paradigm through RCTs.
Several notable cases involved the use of small-molecule drugs, which are synthesized compounds of low molecular weight, typically less than 1 kilodalton (kDa) [@url:https://www.nuventra.com/resources/blog/small-molecules-versus-biologics].
Small-molecule pharmaceutical agents have been a backbone of drug development since the discovery of penicillin in the early twentieth century [@doi:10.1126/science.287.5460.1960].
It and other antibiotics have long been among the best known applications of small molecules to therapeutics, but biotechnological developments such as the prediction of protein-protein interactions (PPIs) have facilitated advances in precise targeting of specific structures using small molecules [@doi:10.1126/science.287.5460.1960].
Small molecule drugs today encompass a wide range of therapeutics beyond antibiotics, including antivirals, protein inhibitors, and many broad-spectrum pharmaceuticals.

Many treatments considered for COVID-19 have relied on a broad-spectrum approach.
These treatments do not specifically target a virus or particular host receptor, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
This approach relies on the fact that when a virus enters a host, the host becomes the virus's environment.
Therefore, the state of the host can also influence the virus's ability to replicate and spread.
The administration and assessment of broad-spectrum small-molecule drugs on a rapid time course was feasible because they are often either available in hospitals, or in some cases may also be prescribed to a large number of out-patients.
One of the other advantages is that these well-established compounds, if found to be beneficial, are often widely available, in contrast to boutique experimental drugs.

In some cases, prior data was available from experiments examining the response of other HCoVs or HCoV infections to a candidate drug.
In addition to non-pharmaceutical interventions such as encouraging non-intubated patients to adopt a prone position [@doi:10.1001/jamainternmed.2020.3030], knowledge about interactions between HCoVs and the human body, many of which emerged from SARS and MERS research over the past two decades, led to the suggestion that a number of common drugs might benefit COVID-19 patients.
However, the short duration and low case numbers of prior outbreaks were less well-suited to the large-scale study of clinical applications than the COVID-19 pandemic is.
As a result, COVID-19 has presented the first opportunity to robustly evaluate treatments that were common during prior HCoV outbreaks to determine their clinical efficacy.
The first year of the COVID-19 pandemic demonstrated that there are several different trajectories that these clinically suggested, widely available candidates can follow when assessed against a widespread, novel viral threat.

One approach to identifying candidate small molecule drugs was to look at the approaches used to treat SARS and MERS.
Treatment of SARS and MERS patients prioritized supportive care and symptom management [@doi:10.1093/ajcp/aqaa029].
Among the clinical treatments for SARS and MERS that were explored, there was generally a lack of evidence indicating whether they were effective.
Most of the supportive treatments for SARS were found inconclusive in meta-analysis [@doi:10.1371/journal.pmed.0030343], and a 2004 review reported that not enough evidence was available to make conclusions about most treatments [@doi:10.1007/s10156-003-0296-9].
However, one strategy adopted from prior HCoV outbreaks is currently the best-known treatment for severe cases of COVID-19.
Corticosteroids represent broad-spectrum treatments and are a well-known, widely available treatment for pneumonia [@doi:10.1002/14651858.CD007720.pub3; @doi:10.1002/14651858.CD007720.pub2; @doi:10.1183/09031936.00154107; @doi:10.1007/s00408-007-9020-3; @doi:10.1371/journal.pone.0047926; @doi:10.1186/cc10103] that have also been debated as a possible treatment for ARDS [@doi:10/ggpxzc; @doi:10/gcv2; @doi:10.1378/chest.06-1743; @doi:10.1378/chest.07-0714; @doi:10.1056/NEJMoa051693; @doi:10.1136/bmj.39537.939039.BE].
Corticosteroids were also used and subsequently evaluated as possible supportive care for SARS and MERS.
In general, studies and meta-analyses did not identify support for corticosteroids to prevent mortality in these HCoV infections [@doi:10.1136/thx.2003.017665; @doi:10.1016/j.jinf.2006.01.005; @doi:10.1038/s41375-020-0848-3]; however, one found that the effects might be masked by variability in treatment protocols, such as dosage and timing [@doi:10.1007/s10156-003-0296-9].
While the corticosteroids most often used to treat SARS were methylprednisolone and hydrocortisone, availability issues for these drugs at the time led to dexamethasone also being used in North America [@doi:10.1056/NEJMp030072].

Dexamethasone (9&alpha;-fluoro-16&alpha;-methylprednisolone) is a synthetic corticosteroid that binds to glucocorticoid receptors [@doi:10.1021/cr068203e; @url:https://www.cebm.net/covid-19/dexamethasone].
It functions as an anti-inflammatory agent by binding to glucocorticoid receptors with higher affinity than endogenous cortisol [@doi:10.1016/B978-0-323-48110-6.00034-X].
Dexamethasone and other steroids are widely available and affordable, and they are often used to treat community-acquired pneumonia [@doi:10.1038/s41392-020-0127-9] as well as chronic inflammatory conditions such as asthma, allergies, and rheumatoid arthritis [@doi:10.1021/ja01545a061; @doi:10.1001/jama.1960.03030070009002; @dailymed-dexamethasone].
Immunosuppressive drugs such as steroids are typically contraindicated in the setting of infection [@doi:10.1177/2040622313485275], but because COVID-19 results in hyperinflammation that appears to contribute to mortality via lung damage, immunosuppression may be a helpful approach to treatment [@doi:10/ggnzmc].
A clinical trial that began in 2012 recently reported that dexamethasone may improve outcomes for patients with ARDS [@doi:10/ggpxzc], but a meta-analysis of a small amount of available data about dexamethasone as a treatment for SARS suggested that it may, in fact, be associated with patient harm [@doi:10/ggks86].
However, the findings in SARS may have been biased by the fact that all of the studies examined were observational and a large number of inconclusive studies were not included [@doi:10/ggq356].
The questions of whether and when to counter hyperinflammation with immunosuppression in the setting of COVID-19 (as in SARS [@doi:10.1186/cc2452]) was an area of intense debate, as the risks of inhibiting antiviral immunity needed to be weighed against the beneficial anti-inflammatory effects [@doi:10/ggq8hs].
As a result, guidelines early in the pandemic typically recommended avoiding treating COVID-19 patients with corticosteroids such as dexamethasone [@doi:10/ggks86].

Despite this initial concern, dexamethasone was evaluated as a potential treatment for COVID-19 (Appendix 1).
Dexamethasone treatment comprised one arm of the multi-site Randomized Evaluation of COVID-19 Therapy (RECOVERY) trial in the United Kingdom [@doi:10.1101/2020.06.22.20137273].
This study found that the 28-day mortality rate was lower in patients receiving dexamethasone than in those receiving standard of care (SOC).
However, this finding was driven by differences in mortality among patients who were receiving mechanical ventilation or supplementary oxygen at the start of the study.
The report indicated that dexamethasone reduced 28-day mortality relative to SOC in patients who were ventilated (29.3% versus 41.4%) and among those who were receiving oxygen supplementation (23.3% versus 26.2%) at randomization, but not in patients who were breathing independently (17.8% versus 14.0%).
These findings also suggested that dexamethasone may have reduced progression to mechanical ventilation, especially among patients who were receiving oxygen support at randomization.
Other analyses have supported the importance of disease course in determining the efficacy of dexamethasone: additional results suggest greater potential for patients who have experienced symptoms for at least seven days and patients who were not breathing independently [@doi:10.1056/NEJMoa2021436].
A meta-analysis that evaluated the results of the RECOVERY trial alongside trials of other corticosteroids, such as hydrocortisone, similarly concluded that corticosteroids may be beneficial to patients with severe COVID-19 who are receiving oxygen supplementation [@doi:10.1053/j.jvca.2020.11.057].
Thus, it seems likely that dexamethasone is useful for treating inflammation associated with immunopathy or cytokine release syndrome (CRS), which is a condition caused by detrimental overactivation of the immune system [@individual-pathogenesis].
In fact, corticosteroids such as dexamethasone are sometimes used to treat CRS [@doi:10.1182/blood-2014-05-552729].
Guidelines were quickly updated to encourage the use of dexamethasone in severe cases [@doi:10.1001/jama.2020.16747], and this affordable and widely available treatment rapidly became a valuable tool against COVID-19 [@doi:10.1016/j.ejphar.2021.173854], with demand surging within days of the preprint's release [@doi:10.1136/bmj.m2512].

### Approaches Targeting the Virus

Therapeutics that directly target the virus itself hold the potential to prevent people infected with SARS-CoV-2 from developing potentially damaging symptoms (Figure @fig:therapeutics).
Such drugs typically fall into the broad category of antivirals.
Antiviral therapies hinder the spread of a virus within the host, rather than destroying existing copies of the virus, and these drugs can vary in their specificity to a narrow or broad range of viral targets.
This process requires inhibiting the replication cycle of a virus by disrupting one of six fundamental steps [@isbn:978-1405136457].
In the first of these steps, the virus attaches to and enters the host cell through endocytosis.
Then the virus undergoes uncoating, which is classically defined as the release of viral contents into the host cell.
Next, the viral genetic material enters the nucleus where it gets replicated during the biosynthesis stage.
During the assembly stage, viral proteins are translated, allowing new viral particles to be assembled.
In the final step new viruses are released into the extracellular environment.
Although antivirals are designed to target a virus, they can also impact other processes in the host and may have unintended effects.
Therefore, these therapeutics must be evaluated for both efficacy and safety.
As the technology to respond to emerging viral threats has also evolved over the past two decades, a number of candidate treatments have been identified for prior viruses that may be relevant to the treatment of COVID-19.

![
**Mechanisms of Action for Potential Therapeutics**
Potential therapeutics currently being studied can target the SARS-CoV-2 virus or modify the host environment through many different mechanisms.
Here, the relationships between the virus, host cells, and several therapeutics are visualized.
Drug names are color-coded according to the grade assigned to them by the Center for Cytokine Storm Treatment & Laboratory's CORONA Project [@url:https://cdcn.org/corona-data-viewer] (Green = A, Lime = B, Orange = C, and Red = D).
](images/N001-LifeCyclePlusDrugs.png "SARS-CoV-2 life cycle and therapeutic mechanisms"){#fig:therapeutics secno=1}

Many antiviral drugs are designed to inhibit the replication of viral genetic material during the biosynthesis step.
Unlike DNA viruses, which can use the host enzymes to propagate themselves, RNA viruses like SARS-CoV-2 depend on their own polymerase, the RNA-dependent RNA polymerase (RdRP), for replication [@doi:10.1007/978-1-4939-2438-7; @doi:10.1002/jmv.25761].
RdRP is therefore a potential target for antivirals against RNA viruses. 
Disruption of RdRP is the proposed mechanism underlying the treatment of SARS and MERS with ribavirin [@doi:10.1080/17460441.2019.1581171].
Ribavirin is an antiviral drug effective against other viral infections that was often used in combination with corticosteroids and sometimes interferon (IFN) medications to treat SARS and MERS [@doi:10.1038/nrmicro.2016.81].
However, analyses of its effects in retrospective and _in vitro_ analyses of SARS and the SARS-CoV-1 virus, respectively, have been inconclusive [@doi:10.1038/nrmicro.2016.81].
While IFNs and ribavirin have shown promise in _in vitro_ analyses of MERS, their clinical effectiveness remains unknown [@doi:10.1038/nrmicro.2016.81].
The current COVID-19 pandemic has provided an opportunity to assess the clinical effects of these treatments.
As one example, ribivarin was also used in the early days of COVID-19, but a retrospective cohort study comparing patients who did and did not receive ribivarin revealed no effect on the mortality rate [@doi:10.1016/j.ijantimicag.2020.106114].

Since nucleotides and nucleosides are the natural building blocks for RNA synthesis, an alternative approach has been to explore nucleoside and nucleotide analogs for their potential to inhibit viral replication.
Analogs containing modifications to nucleotides or nucleosides can disrupt key processes including replication [@doi:10.1016/j.antiviral.2018.04.004].
A single incorporation does not influence RNA transcription; however, multiple events of incorporation lead to the arrest of RNA synthesis [@doi:10.1371/journal.pone.0068347].
One candidate antiviral considered for the treatment of COVID-19 is favipiravir (Avigan), also known as T-705, which was discovered by Toyama Chemical Co., Ltd. [@url:https://www.drugbank.ca/drugs/DB12466].
It was previously found to be effective at blocking viral amplification in several influenza subtypes as well as other RNA viruses, such as _Flaviviridae_ and _Picornaviridae_, through a reduction in plaque formation [@doi:10.1128/AAC.46.4.977-981.2002] and viral replication in Madin-Darby canine kidney cells [@doi:10.1128/AAC.01051-06].
Favipiravir (6-fluoro-3-hydroxy-2-pyrazinecarboxamide) acts as a purine and purine nucleoside analogue that inhibits viral RNA polymerase in a dose-dependent manner across a range of RNA viruses, including influenza viruses [@doi:10.1128/AAC.49.3.981-986.2005; @doi:10.1128/AAC.01074-08; @doi:10.1128/AAC.00356-07; @doi:10.1016/j.bbrc.2012.07.034; @doi:10.1128/AAC.01219-10].
Biochemical experiments showed that favipiravir was recognized as a purine nucleoside analogue and incorporated into the viral RNA template.
In 2014, the drug was approved in Japan for the treatment of influenza that was resistant to conventional treatments like neuraminidase inhibitors [@doi:10.2183/pjab.93.027].
Though initial analyses of favipiravir in observational studies of its effects on COVID-19 patients were promising, recent results of two small RCTs suggest that it is unlikely to affect COVID-19 outcomes (Appendix 1).

In contrast, another nucleoside analog, remdesivir, is one of the few treatments against COVID-19 that has received FDA approval.
Remdesivir (GS-5734) is an intravenous antiviral that was proposed by Gilead Sciences as a possible treatment for Ebola virus disease.
It is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair, causing chain termination [@doi:10.1074/jbc.AC120.013056; @doi:10.1128/mBio.00221-18; @doi:10.1038/s41422-020-0282-0].
Gilead received an emergency use authorization (EUA) for remdesivir from the FDA early in the pandemic (May 2020) and was later found to reduce mortality and recovery time in a double-blind, placebo-controlled, phase III clinical trial performed at 60 trial sites, 45 of which were in the United States [@doi:10.1056/NEJMoa2007764; @clinicaltrials:NCT04280705; @clinicaltrials:NCT04292899; @doi:10.1056/NEJMoa2007016].
Subsequently, the WHO Solidarity trial, a large-scale, open-label trial enrolling 11,330 adult in-patients at 405 hospitals in 30 countries around the world, reported no effect of remdesivir on in-hospital mortality, duration of hospitalization, or progression to mechanical ventilation [@doi:10.1056/NEJMoa2023184].
Therefore, additional clinical trials of remdesivir in different patient pools and in combination with other therapies may be needed to refine its use in the clinic and determine the forces driving these differing results.
Remdesivir offers proof of principle that SARS-CoV-2 can be targeted at the level of viral replication, since remdesivir targets the viral RNA polymerase at high potency.
Identification of such candidates depends on knowledge about the virological properties of a novel threat.
However, the success and relative lack of success, respectively, of remdesivir and favipiravir underscore the fact that drugs with similar mechanisms will not always produce similar results in clinical trials.

### Disrupting Host-Virus Interactions

#### Interrupting Viral Colonization of Cells

Some of the most widely publicized examples of efforts to repurpose drugs for COVID-19 are broad-spectrum, small-molecule drugs where the mechanism of action made it seem that the drug might disrupt interactions between SARS-CoV-2 and human host cells (Figure @fig:therapeutics).
However, the exact outcomes of such treatments are difficult to predict _a priori_, and there are several examples where early enthusiasm was not borne out in subsequent trials.
One of the most famous examples of an analysis of whether a well-known medication could provide benefits to COVID-19 patients came from the assessment of chloroquine (CQ) and hydroxychloroquine (HCQ), which are used for the treatment and prophylaxis of malaria as well as the treatment of lupus erythematosus and rheumatoid arthritis in adults [@dailymed-hcq].
These drugs are lysosomotropic agents, meaning they are weak bases that can pass through the plasma membrane.
It was thought that they might provide benefits against SARS-CoV-2 by interfering with the digestion of antigens within the lysosome and inhibiting CD4 T-cell stimulation while promoting the stimulation of CD8 T-cells [@doi:10.1111/j.1529-8019.2007.00131.x].
These compounds also have anti-inflammatory properties [@doi:10.1111/j.1529-8019.2007.00131.x] and can decrease the production of certain key cytokines involved in the immune response, including interleukin-6 (IL-6) and inhibit the stimulation of Toll-like receptors (TLR) and TLR signaling [@doi:10.1111/j.1529-8019.2007.00131.x].

_In vitro_ analyses reported that CQ inhibited cell entry of SARS-CoV-1 [@doi:10.1186/1743-422X-2-69] and that both CQ and HCQ inhibited viral replication within cultured cells [@doi:10.1093/cid/ciaa237], leading to early hope that it might provide similar therapeutic or protective effects in patients.
However, while the first publication on the clinical application of these compounds to the inpatient treatment of COVID-19 was very positive [@doi:10.1016/j.ijantimicag.2020.105949], it was quickly discredited [@url:https://www.isac.world/news-and-publications/official-isac-statement].
Over the following months, extensive evidence emerged demonstrating that CQ and HCQ offered no benefits for COVID-19 patients and, in fact, carried the risk of dangerous side effects (Appendix 1).
The nail in the coffin came when findings from the large-scale RECOVERY trial were released on October 8, 2020.
This study enrolled 11,197 hospitalized patients whose physicians believed it would not harm them to participate and used a randomized, open-label design to study the effects of HCQ compared to standard of care (SOC) at 176 hospitals in the United Kingdom [@doi:10.1056/NEJMoa2022926].
Rates of COVID-19-related mortality did not differ between the control and HCQ arms, but patients receiving HCQ were slightly more likely to die due to cardiac events.
Patients who received HCQ also had a longer duration of hospitalization than patients receiving usual care and were more likely to progress to mechanical ventilation or death (as a combined outcome).
As a result, enrollment in the HCQ arm of the RECOVERY trial was terminated early [@url:https://www.recoverytrial.net/news/statement-from-the-chief-investigators-of-the-randomised-evaluation-of-covid-19-therapy-recovery-trial-on-hydroxychloroquine-5-june-2020-no-clinical-benefit-from-use-of-hydroxychloroquine-in-hospitalised-patients-with-covid-19].
The story of CQ/HCQ therefore illustrates how initial promising _in vitro_ analyses can fail to translate to clinical usefulness.

A similar story has arisen with the broad-spectrum, small-molecule anthelmintic ivermectin, which is a synthetic analog of avermectin, a bioactive compound produced by a microorganism known as _Streptomyces avermectinius_ and _Streptomyces avermitilis_ [@doi:10.1038/nrmicro1048; @doi:10.1128/AAC.15.3.361].
Avermectin disrupts the ability of parasites to avoid the host immune response by blocking glutamate-gated chloride ion channels in the peripheral nervous system from closing, leading to hyperpolarization of neuronal membranes, disruption of neural transmission, and paralysis [@doi:10.2183/pjab.87.13; @doi:10.1038/nrmicro1048; @doi:10.1038/ja.2017.11].
Ivermectin has been used since the early 1980s to treat endo- and ecto-parasitic infections by helminths, insects, and arachnids in veterinary contexts [@doi:10.1038/nrmicro1048; @doi:10.1128/AAC.15.3.372] and since the late 1980s to treat human parasitic infections as well [@doi:10.1038/nrmicro1048; @doi:10.2183/pjab.87.13].
More recent research has indicated that ivermectin might function as a broad-spectrum antiviral by disrupting the trafficking of viral proteins by both RNA and DNA viruses [@doi:10.1038/s41429-020-0336-z; @doi:10.1038/ja.2017.11; @doi:10.1186/s12941-020-00368-w], although most of these studies have demonstrated this effect _in vitro_ [@doi:10.1186/s12941-020-00368-w].
The potential for antiviral effects on SARS-CoV-2 were investigated _in vitro_, and ivermectin was found to inhibit viral replication in a cell line derived from Vero cells (Vero-hSLAM) [@doi:10.1016/j.antiviral.2020.104787]. 
However, inhibition of viral replication was achieved at concentrations that were much higher than that explored by existing dosage guidelines [@doi:10.4269/ajtmh.20-0271; @doi:10.1002/cpt.1889], which are likely to be associated with significant side effects due to the increased potential that the compound could cross the mammalian blood-brain barrier [@doi:10.1080/13102818.2020.1775118; @doi:10.1371/journal.pntd.0001883].

Retrospective studies and small RCTs began investigating the effects of standard doses of this low-cost, widely available drug.
One retrospective study reported that ivermectin reduced all-cause mortality [@doi:10.1016/j.chest.2020.10.009] while another reported no difference in clinical outcomes or viral clearance [@doi:10.1371/journal.pone.0242184].
Small RCTs enrolling less than 50 patients per arm have also reported a wide array of positive [@doi:10.4103/1995-7645.318304; @doi:10.1093/qjmed/hcab035; @doi:10.1016/j.toxrep.2021.03.003; @doi:10.1016/j.ijid.2020.11.191; @doi:10.1016/j.clinthera.2021.04.007] and negative results [@doi:10.1016/j.eclinm.2020.100720; @doi:10.21203/rs.3.rs-191648/v1].
A slightly larger RCT enrolling 115 patients in two arms reported inconclusive results [@doi:10.18433/jpps32105].
Hope for the potential of ivermectin peaked with the release of a preprint reporting results of a multicenter, double-blind RCT where a four-day course of ivermectin was associated with clinical improvement and earlier viral clearance in 400 symptomatic patients and 200 close contacts [@doi:10.21203/rs.3.rs-100956/v3]; however, concerns were raised about both the integrity of the data and the paper itself [@url:https://grftr.news/why-was-a-major-study-on-ivermectin-for-covid-19-just-retracted; @url:https://steamtraen.blogspot.com/2021/07/Some-problems-with-the-data-from-a-Covid-study.html], and this study was removed by the preprint server Research Square [@doi:10.21203/rs.3.rs-100956/v4].
A similarly sized RCT suggested no effect on the duration of symptoms among 400 patients split evenly across the intervention and control arms [@doi:10.1001/jama.2021.3071], and although meta-analyses have reported both null [@doi:10.1101/2021.05.21.21257595; @doi:10.1101/2021.01.26.21250420] and beneficial [@doi:10.1093/ofid/ofab394; @doi:10.1016/j.dsx.2021.102186; @doi:10.1097/MJT.0000000000001402; @doi:10.1002/rmv.2265; @doi:10.31219/osf.io/k37ft; @doi:10.1097/MJT.0000000000001377; @doi:10.1007/s43440-021-00245-z; @doi:10.1101/2021.04.30.21256415] effects of ivermectin on COVID-19 outcomes, the certainty is likely to be low [@doi:10.1016/j.dsx.2021.102186].
These findings are potentially biased by a small number of low-quality studies, including the preprint that has been taken down [@url:https://gidmk.medium.com/does-ivermectin-work-for-covid-19-1166126c364a], and the authors of one [@doi:10.1093/ofid/ofab358] have issued a notice [@doi:10.1093/ofid/ofab394] that they will revise their study with the withdrawn study removed.
Thus, much like HCQ/CQ, enthusiasm for research that either has not or should not have passed peer review has led to large numbers of patients worldwide receiving treatments that might not have any effect or could even be harmful.
Additionally, comments on the now-removed preprint include inquiries into how best to self-administer veterinary ivermectin as a prophylactic [@doi:10.21203/rs.3.rs-100956/v4], and the FDA has posted information explaining why veterinary ivermectin should not be taken by humans concerned about COVID-19 [@url:https://www.fda.gov/animal-veterinary/product-safety-information/faq-covid-19-and-ivermectin-intended-animals].
Ivermectin is now one of several candidate therapeutics being investigated in the large-scale TOGETHER [@clinicaltrials:NCT04727424] and PRINCIPLE [@url:https://www.principletrial.org/news/ivermectin-to-be-investigated-as-a-possible-treatment-for-covid-19-in-oxford2019s-principle-trial] clinical trials.
The TOGETHER trial, which previously demonstrated no effect of HCQ and lopinavir-ritonavir [@doi:10.1001/jamanetworkopen.2021.6468], released preliminary results in early August 2021 suggesting that ivermectin also has no effect on COVID-19 outcomes [@url:https://rethinkingclinicaltrials.org/news/august-6-2021-early-treatment-of-covid-19-with-repurposed-therapies-the-together-adaptive-platform-trial-edward-mills-phd-frcp].

While CQ/HCQ and ivermectin are well-known medications that have long been prescribed in certain contexts, investigation of another well-established type of pharmaceutical was facilitated by the fact that it was already being taken by a large number of COVID-19 patients.
Angiotensin-converting enzyme inhibitors (ACEIs) and angiotensin II receptor blockers (ARBs) are among today's most commonly prescribed medications, often being used to control blood pressure [@url:https://clincalc.com/DrugStats/Drugs/Lisinopril; @doi:10.1056/NEJMp1901657].
In the United States, for example, they are prescribed well over 100,000,000 times annually [@doi:10.18553/jmcp.2010.16.9.671].
Prior to the COVID-19 pandemic, the relationship between ACE2, ACEIs, and SARS had been considered as possible evidence that ACE2 could serve as a therapeutic target [@doi:10.1016/j.tips.2004.04.001], and the connection had been explored through _in vitro_ and molecular docking analysis [@doi:10.1161/01.HYP.0000146120.29648.36] but ultimately was not pursued clinically [@doi:10/d85vmw].
Data from some animal models suggest that several, but not all, ACEIs and several ARBs increase ACE2 expression in the cells of some organs [@doi:10.1093/cvr/cvaa097], but clinical studies have not established whether plasma ACE2 expression is increased in humans treated with these medications [@doi:10.1093/europace/euw246].
In this case, rather than introducing ARBs/ACEIs, a number of analyses have investigated whether discontinuing use affects COVID-19 outcomes.
An initial observational study of the association of exposure to ACEIs or ARBs with outcomes in COVID-19 was retracted from the _New England Journal of Medicine_ [@doi:10.1056/NEJMoa2007621] due to concerns related to data availability [@doi:10.1056/nejmc2021225].
As RCTs have become available, they have demonstrated no effect of continuing versus discontinuing ARBs/ACEIs on patient outcomes [@doi:10/fvgt; @doi:10.1001/jama.2020.25864] (Appendix 1).
Thus, once again, despite a potential mechanistic association with the pathology of SARS-CoV-2 infection, these medications were not found to influence the trajectory of COVID-19 illness.

For medications that are widely known and common, clinical research into their efficacy against a novel threat can be developed very quickly.
This feasibility can present a double-edged sword.
For example, HCQ and CQ were incorporated into SOC in many countries early in the pandemic and had to be discontinued once their potential to harm COVID-19 patients became apparent [@url:https://www.fda.gov/media/138946/download; @url:https://www.ema.europa.eu/en/news/covid-19-chloroquine-hydroxychloroquine-only-be-used-clinical-trials-emergency-use-programmes].
Dexamethasone remains the major success story from this category of repurposed drugs and is likely to have saved a large number of lives since summer 2020 [@doi:10.1016/j.ejphar.2021.173854].

#### Manipulating the Host Immune Response

Treatments based on understanding a virus and/or how a virus interacts with the human immune system can fall into two categories: they can interact with the innate immune response, which is likely to be a similar response across viruses, or they can be specifically designed to imitate the adaptive immune response to a particular virus. 
In the latter case, conservation of structure or behavior across viruses enables exploring whether drugs developed for one virus can treat another.
During the COVID-19 pandemic, a number of candidate therapeutics have been explored in these categories, with varied success.

Knowledge gained from trying to understand SARS-CoV-1 and MERS-CoV from a fundamental biological perspective and characterize how they interact with the human immune system provides a theoretical basis for identifying candidate therapies.
Biologics are a particularly important class of drugs for efforts to address HCoV through this paradigm.
They are produced from components of living organisms or viruses, historically primarily from animal tissues, but have become increasingly feasible to produce as recombinant technologies have advanced  [@doi:10.1016/j.copbio.2009.10.006].
Biologics have become increasingly feasible to produce as recombinant DNA technologies have advanced [@doi:10.1016/j.copbio.2009.10.006].

There are many differences on the development side between biologics and synthesized pharmaceuticals, such as small molecule drugs.
Typically, biologics are orders of magnitude larger than small molecule drugs and are catabolized by the body to their amino acid components [@doi:10.2174/138920006774832604].
They are often heat sensitive, and their toxicity can vary, as it is not directly associated with the primary effects of the drug; in general, their physiochemical properties are much less understood compared to small molecules [@doi:10.2174/138920006774832604].
Biologics include significant medical breakthroughs such as insulin for the management of diabetes and vaccines, as well monoclonal antibodies (mAbs) and interferons (IFNs), which can be used to target the host immune response after infection.

mAbs have revolutionized the way we treat human diseases and have become some of the best-selling drugs in the pharmaceutical market in recent years [@doi:10.1186/s12929-019-0592-z].
There are currently 79 FDA approved mAbs on the market, including antibodies for viral infections (e.g. Ibalizumab for _Human immunodeficiency virus_ and Palivizumab for _Respiratory syncytial virus_) [@doi:10.1186/s12929-019-0592-z; @doi:10.1146/annurev-immunol-032712-095916].
Virus-specific neutralizing antibodies commonly target viral surface glycoproteins or host structures, thereby inhibiting viral entry through receptor binding interference [@doi:10.3389/fmicb.2017.02323; @doi:10.1080/21645515.2017.1337614].
This interference is predicted to reduce the viral load, mitigate disease, and reduce overall hospitalization.
mAbs can be designed for a particular virus, and significant advances have been made in the speed at which new mAbs can be identified and produced.
At the time of the SARS and MERS epidemics, interest in mAbs to reduce infection was never realized [@doi:10.1093/infdis/jiw080; @doi:10.1016/j.antiviral.2017.03.025], but this allowed for mAbs to quickly be considered among the top candidates against COVID-19.

##### Biologics and the Innate Immune Response

Deaths from COVID-19 often occur when inflammation becomes dysregulated following an immune response to the SARS-CoV-2 virus.
Therefore, one potential approach to reducing COVID-19 mortality rates is to manage the inflammatory response in severely ill patients.
One candidate therapeutic identified that uses this mechanism is tocilizumab (TCZ).
TCZ is a mAb that was developed to manage chronic inflammation caused by the continuous synthesis of the cytokine IL-6 [@doi:10.1101/cshperspect.a016295].
IL-6 is a pro-inflammatory cytokine belonging to the interleukin family, which is comprised by immune system regulators that are primarily responsible for immune cell differentiation.
Often used to treat chronic inflammatory conditions such as rheumatoid arthritis [@doi:10.1101/cshperspect.a016295], TCZ has become a pharmaceutical of interest for the treatment of COVID-19 because of the role IL-6 plays in this disease.
It has also been approved to treat CRS caused by CAR-T treatments [@dailymed-tocilizumab].
While the secretion of IL-6 can be associated with chronic conditions, IL-6 is a key player in the innate immune response and is secreted by macrophages in response to the detection of pathogen-associated molecular patterns and damage-associated molecular patterns [@doi:10.1101/cshperspect.a016295].
An analysis of 191 in-patients at two Wuhan hospitals revealed that blood concentrations of IL-6 differed between patients who did and did not recover from COVID-19.
Patients who ultimately died had higher IL-6 levels at admission than those who recovered [@doi:10/ggnxb3].
Additionally, IL-6 levels remained higher throughout the course of hospitalization in the patients who ultimately died [@doi:10/ggnxb3].

Currently, TCZ is being administered either as a monotherapy or in combination with other treatments in {{ebm_tocilizumab_ct}} interventional COVID-19 clinical trials (Figure @fig:ebm-trials).
A number of retrospective studies have been conducted in several countries [@doi:10/d2pk; @doi:10.1016/j.chest.2020.06.006; @doi:10.1016/j.ejim.2020.05.009; @doi:10.1016/j.medmal.2020.05.001; @doi:10.1093/qjmed/hcaa206; @doi:10.1073/pnas.2005615117].
In general, these studies have reported a positive effect of TCZ on reducing mortality in COVID-19 patients, although due to their retrospective designs, significant limitations are present in all of them (Appendix 1).
It was not until February 11, 2021 that a preprint describing preliminary results of the first RCT of TCZ was released as part of the RECOVERY trial [@doi:10.1101/2021.02.11.21249258].
TCZ was found to reduce 28-day mortality from 33% in patients receiving SOC alone to 29% in those receiving TCZ.
Combined analysis of the RECOVERY trial data with data from smaller RCTs suggested a 13% reduction in 28-day mortality [@doi:10.1101/2021.02.11.21249258].
While this initial report did not include the full results expected from the RECOVERY trial, this large-scale, RCT provides strong evidence that TCZ may offer benefits for COVID-19 patients.
The RECOVERY trial along with results from several other RCTs [@doi:10.1056/NEJMoa2028700; @doi:10.1056/NEJMoa2030340; @clinicaltrials:NCT04409262; @clinicaltrials:NCT04320615; @clinicaltrials:NCT04372186] were cited as support for the EUA issued for TCZ in June 2021 [@url:https://www.fda.gov/media/150319/download].
However, the fact that TCZ suppresses the immune response means that it does carry risks for patients, especially a potential risk of secondary infection (Appendix 1).

TCZ is just one example of a candidate drug targeting the host immune response and specifically excessive inflammation.
For example, interferons (IFNs) have also been investigated; these are a family of cytokines critical to activating the innate immune response against viral infections.
Synairgen has been investigating a candidate drug, SNG001, which is an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation [@clinicaltrials:NCT04385095] that they reported reduced progression to ventilation in a double-blind, placebo-controlled, multi-center study of 101 patients with an average age in the late 50s [@synairgen-SNG001-July-2020; @doi:10/ghjzm4].
However, these findings were not supported by the large-scale WHO Solidarity trial, which reported no significant effect of IFN-&beta;-1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184], although differences in the designs of the two studies, and specifically the severity of illness among enrolled patients, may have influenced their divergent outcomes (Appendix 1).
Other biologics influencing inflammation are also being explored (Appendix 1).
It is also important that studies focused on inflammation as a possible therapeutic target consider the potential differences in baseline inflammation among patients from different backgrounds, which may be caused by differing life experiences (see [@individual-inequality]).

##### Biologics and the Adaptive Immune Response

While TCZ is an example of an mAb focused on managing the innate immune response, other treatments are more specific, targeting the adaptive immune response after an infection.
In some cases, treatments can utilize biologics obtained directly from recovered individuals.
From the very early days of the COVID-19 pandemic, polyclonal antibodies from convalescent plasma were investigated as a potential treatment for COVID-19 [@doi:10/ggqr7s; @doi:10.1001/jama.2020.4940].
Convalescent plasma was used in prior epidemics including SARS, Ebola Virus Disease, and even the 1918 Spanish Influenza [@doi:10/ggqr7s; @doi:10.1002/jmv.25961].
Use of convalescent plasma transfusion (CPT) over more than a century has aimed to reduce symptoms and improve mortality in infected people [@doi:10.1002/jmv.25961], possibly by accelerating viral clearance [@doi:10/ggqr7s].
However, it seems unlikely that this classic treatment confers any benefit for COVID-19 patients.
Several systematic reviews have investigated whether CPT reduced mortality in COVID-19 patients, and although findings from early in the pandemic (up to April 19, 2020) did support use of CPT [@doi:10.1002/jmv.25961], the tide has shifted as the body of available literature has grown [@doi:10.1001/jama.2021.2747].
While titer levels were suggested as a possible determining factor in the success of CPT against COVID-19 [@doi:10.1056/NEJMoa2031893], the large-scale RECOVERY trial evaluated the effect of administering high-titer plasma specifically and found no effect on mortality or hospital discharge over a 28-day period [@doi:10.1101/2021.03.09.21252736].
These results thus suggest that, despite initial optimism and an EUA from the FDA, CPT is unlikely to be an effective therapeutic for COVID-19.

A different narrative is shaping up around the use of mAbs specifically targeting SARS-CoV-2. 
During the first SARS epidemic in 2002, neutralizing antibodies (nAbs) were found in SARS-CoV-1-infected patients [@doi:10.1111/j.1469-0691.2004.01009.x; @doi:10.1086/423286].
Several studies following up on these findings identified various S-glycoprotein epitopes as the major targets of nAbs against SARS-CoV-1 [@doi:10.1517/14712590902763755].
Coronaviruses use trimeric spike (S) glycoproteins on their surface to bind to the host cell, allowing for cell entry [@doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.02.058].
Each S glycoprotein protomer is comprised of an S1 domain, also called the receptor binding domain (RBD), and an S2 domain.
The S1 domain binds to the host cell while the S2 domain facilitates the fusion between the viral envelope and host cell membranes [@doi:10.1517/14712590902763755].
The genomic identity between the RBD of SARS-CoV-1 and SARS-CoV-2 is around 74% [@doi:10.1101/2020.02.16.951723].
Due to this high degree of similarity, preexisting antibodies against SARS-CoV-1 were initially considered candidates for neutralizing activity against SARS-CoV-2.
While some antibodies developed against the SARS-CoV-1 spike protein showed cross-neutralization activity with SARS-CoV-2 [@doi:10.1016/j.tips.2020.07.004; @doi:10.1038/s41586-020-2349-y], others failed to bind to SARS-CoV-2 spike protein at relevant concentrations [@doi:10.1126/science.abb2507].
Cross-neutralizing activities were dependent on whether the epitope recognized by the antibodies were conserved between SARS-CoV-1 and SARS-CoV-2 [@doi:10.1016/j.tips.2020.07.004].

Technological advances in antibody drug design as well as in structural biology massively accelerated the discovery of novel antibody candidates and the mechanisms by which they interact with the target structure.
Within just a year of the structure of the SARS-CoV-2 spike protein being published, an impressive pipeline of monoclonal antibodies targeting SARS-CoV-2 entered clinical trials, with hundreds more candidates in preclinical stages.
The first human monoclonal neutralizing antibody specifically against the SARS-CoV-2 S glycoprotein was developed using hybridoma technology [@doi:10.1101/2020.03.11.987958], where antibody-producing B-cells developed by mice are inserted into myeloma cells to produce a hybrid cell line (the hybridoma) that is grown in culture.
The 47D11 antibody clone was able to cross-neutralize SARS-CoV-1 and SARS-CoV-2.
This antibody (now ABVV-47D11) has recently entered clinical trials in collaboration with AbbVie.
Additionally, an extensive monoclonal neutralizing antibody pipeline has been developed to combat the ongoing pandemic, with over 50 different antibodies in clinical trials [@doi:10.1016/j.bsheal.2021.02.001].
Thus far, the monotherapy sotrovimab and two antibody cocktails (bamlanivimab/estesevimab and casirivimab/imdevimab) have been granted EUAs by the FDA.

One of the studied antibody cocktails consists of bamlanivimab and estesevimab.
Bamlanivimab (Ly-CoV555) is a human mAb that was derived from convalescent plasma donated by a recovered COVID-19 patient, evaluated in research by the National Institute of Allergy and Infectious Diseases (NIAID), and subsequently developed by AbCellera and Eli Lilly.
The neutralizing activity of bamlanivimab was initially demonstrated _in vivo_ using a nonhuman primate model [@doi:10.1101/2020.09.30.318972].
Based on these positive preclinical data, Eli Lilly initiated the first human clinical trial for a monoclonal antibody against SARS-CoV-2.
The phase 1 trial, which was conducted in hospitalized COVID-19 patients, was completed in August 2020 [@clinicaltrials:NCT04411628].
Estesevimab (LY-CoV016 or JS-016) is also a monoclonal neutralizing antibody against the spike protein of SARS-CoV-2.
It was initially developed by Junshi Biosciences and later licensed and developed through Eli Lilly.
A phase 1 clinical trial to assess the safety of etesevimab was completed in October 2020 [@clinicaltrials:NCT04441931].
Etesevimab was shown to bind a different epitope on the spike protein than bamlanivimab, suggesting that the two antibodies used as a combination therapy would further enhance their clinical use compared to a monotherapy [@doi:10.1001/jama.2021.0202].
To assess the efficacy and safety of bamlanivimab alone or in combination with etesevimab for the treatment of COVID-19, a phase 2/3 trial (BLAZE-1) [@clinicaltrials:NCT04427501] was initiated.
The interim analysis of the phase 2 portion suggested that bamlanivimab alone was able to accelerate the reduction in viral load [@doi:10.1056/NEJMoa2029849].
However, more recent data suggests that only the bamlanivimab/etesevimab combination therapy is able to reduce viral load in COVID-19 patients [@doi:10.1001/jama.2021.0202].
Based on this data, the combination therapy received an EUA for COVID-19 from the FDA in February 2021 [@url:https://www.fda.gov/media/145801/download].

A second therapy is comprised of casirivimab and imdevimab (REGN-COV2).
Casirivimab (REGN10933) and imdevimab (REGN10987) are two monoclonal antibodies against the SARS-CoV-2 spike protein.
They were both developed by Regeneron in a parallel high-throughput screening (HTS) to identify neutralizing antibodies from either humanized mice or patient-derived convalescent plasma [@doi:10.1126/science.abd0827].
In these efforts, multiple antibodies were characterized for their ability to bind and neutralize the SARS-CoV-2 spike protein.
The investigators hypothesized that an antibody cocktail, rather than each individual antibody, could increase the therapeutic efficacy while minimizing the risk for virus escape.
Therefore, the authors tested pairs of individual antibodies for their ability to simultaneously bind the RBD of the spike protein.
Based on this data, casirivimab and imdevimab were identified as the lead antibody pair, resulting in the initiation of two clinical trials [@clinicaltrials:NCT04426695; @clinicaltrials:NCT04425629].
Data from this phase 1-3 trial published in the _New England Journal of Medicine_ shows that the REGN-COV2 antibody cocktail reduced viral load, particularly in patients with high viral load or whose endogenous immune response had not yet been initiated [@doi:10.1056/NEJMoa2035002].
However, in patients who already initiated an immune response, exogenous addition of REGN-COV2 did not improve the endogenous immune response.
Both doses were well tolerated with no serious events related to the antibody cocktail.
Based on this data, the FDA granted an EUA for REGN-COV2 in patients with mild to moderate COVID-19 who are at risk of developing severe disease [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-monoclonal-antibodies-treatment-covid-19].
Ongoing efforts are trying to evaluate the efficacy of REGN-COV2 to improve clinical outcomes in hospitalized patients [@clinicaltrials:NCT04426695].

Sotrovimab is the most recent mAb to receive an EUA.
It was identified in the memory B cells of a 2003 survivor of SARS [@doi:10.1038/nm1080] and was found to be cross-reactive with SARS-CoV-2 [@doi:10.1038/s41586-020-2349-y].
This cross-reactivity is likely attributable to conservation within the epitope, with 17 out of 22 residues conserved between the two viruses, four conservatively substituted, and one semi-conservatively substituted [@doi:10.1038/s41586-020-2349-y].
In fact, these residues are highly conserved among sarbecoviruses, a clade that includes SARS-CoV-1 and SARS-CoV-2 [@doi:10.1038/s41586-020-2349-y].
This versatility has led to it being characterized as a "super-antibody" [@doi:10.1038/s41587-021-00980-x], a potent, broadly neutralizing antibody [@doi:10.1038/nri.2017.148].
Interim analysis of data from a clinical trial [@clinicaltrials:NCT04545060] reported high safety and efficacy of this mAb in 583 COVID-19 patients [@doi:10.1101/2021.05.27.21257096].
Compared to placebo, sotrovimab was found to be 85% more effective in reducing progression to the primary endpoint, which was the proportion of patients who, within 29 days, were either hospitalized for more than 24 hours or died.
Additionally, rates of adverse events were comparable, and in some cases lower, among patients receiving sotrovimab compared to patients receiving a placebo.
Sotrovimab therefore represents a mAb therapeutic that is effective against SARS-CoV-2 and may also be effective against other sarbecoviruses.

Several potential limitations remain in the application of mAbs to the treatment of COVID-19.
One of the biggest challenges is identifying antibodies that not only bind to their target, but also prove to be beneficial for disease management.
Currently, use of mAbs is limited to people with mild to moderate disease that are not hospitalized, and it has yet to be determined whether they can be used as a successful treatment option for severe COVID-19 patients.
While preventing people from developing severe illness provides significant benefits, patients with severe illness are at the greatest risk of death, and therefore therapeutics that provide benefits against severe illness are particularly desirable.
It remains to be seen whether mAbs confer any benefits for patients in this category.

Another concern about therapeutics designed to amplify the response to a specific viral target is that they may need to be modified as the virus evolves.
With the ongoing global spread of new SARS-CoV-2 variants, there is a growing concern that mutations in SARS-CoV-2 spike protein could escape antibody neutralization, thereby reducing the efficacy of monoclonal antibody therapeutics and vaccines.
A comprehensive mutagenesis screen recently identified several amino acid substitutions in the SARS-CoV-2 spike protein that can prevent antibody neutralization [@doi:10.1016/j.chom.2021.01.014].
While some mutations result in resistance to only one antibody, others confer broad resistance to multiple mAbs as well as polyclonal human sera, suggesting that some amino acids are "hotspots" for antibody resistance.
However, it was not investigated whether the resistance mutations identified result in a fitness advantage.
Accordingly, an impact on neutralizing efficiency has been reported for the B.1.1.7 (Alpha) variant first identified in the UK and the B.1.351 (Beta) variant first identified in in South Africa [@doi:10.21203/rs.3.rs-228079/v1; @doi:10.1101/2021.02.03.429355; @doi:10.1101/2021.01.25.428137].
As of June 25, 2021, the CDC recommended a pause in the use of bamlanivimab and etesevimab due to decreased efficacy against the P.1 (Gamma) and B.1.351 (Beta) variants of SARS-CoV-2 [@url:https://www.phe.gov/emergency/events/COVID19/investigation-MCM/Bamlanivimab-etesevimab/Pages/bamlanivimab-etesevimab-distribution-pause.aspx].
While the reported impact on antibody neutralization needs to be confirmed _in vivo_, it suggests that some adjustments to therapeutic antibody treatments may be necessary to maintain the efficacy that was reported in previous clinical trials.

Several strategies have been employed to try to mitigate the risk of diminished antibody neutralization.
Antibody cocktails such as those already holding an EUA may help overcome the risk for attenuation of the neutralizing activity of a single monoclonal antibody.
These cocktails consist of antibodies that recognize different epitopes on the spike protein, decreasing the likelihood that a single amino acid change can cause resistance to all antibodies in the cocktail.
However, neutralizing resistance can emerge even against an antibody cocktail if the individual antibodies target subdominant epitopes [@doi:10.1101/2021.02.03.429355].
Another strategy is to develop broadly neutralizing antibodies that target structures that are highly conserved, as these are less likely to mutate [@doi:10.1016/j.immuni.2007.11.018; @doi:10.1126/science.1171491] or to target epitopes that are insensitive to mutations [@doi:10.1016/j.vaccine.2006.04.054].
Sotrovimab, one such "super-antibody", is thought to be somewhat robust to neutralization escape [@doi:10.1086/651022] and has been found to be effective against all variants assessed as of August 12, 2021 [@url:https://covdb.stanford.edu/page/susceptibility-data].
Another antibody (ADG-2) targets a highly conserved epitope that overlaps the hACE2 binding site of all clade 1 sarbecoviruses [@doi:10.1126/science.abf4830].
Prophylactic administration of ADG-2 in an immunocompetent mouse model of COVID-19 resulted in protection against viral replication in the lungs and respiratory burden.
Since the epitope targeted by ADG-2 represents an Achilles’ heel for clade 1 sarbecoviruses, this antibody, like sotrovimab, might be a promising candidate against all circulating variants as well as emerging SARS-related coronaviruses.
To date, it has fared well against the Alpha, Beta, Gamma, and Delta variants [@url:https://covdb.stanford.edu/page/susceptibility-data].

The development of mAbs against SARS-CoV-2 has made it clear that this technology is rapidly adaptable and offers great potential for the response to emerging viral threats.
However, additional investigation may be needed to adapt mAb treatments to SARS-CoV-2 as it evolves and potentially to pursue designs that confer benefits for patients at the greatest risk of death.
While polyclonal antibodies from convalescent plasma have been evaluated as a treatment for COVID-19, these studies have suggested fewer potential benefits against SARS-CoV-2 than mAbs; convalescent plasma therapy has been thoroughly reviewed elsewhere [@doi:10/ggqr7s; @doi:10.1001/jama.2020.4940].
Thus, advances in biologics for COVID-19 illustrate that an understanding of how the host and virus interact can guide therapeutic approaches.
The FDA authorization of two combination mAb therapies, in particular, underscores the potential for this strategy to allow for a rapid response to a novel pathogen.
Additionally, while TCZ is not yet as established, this therapy suggests that the strategy of using biologics to counteract the cytokine storm response may provide therapies for the highest-risk patients.

### High-Throughput Screening for Drug Repurposing

The drug development process is slow and costly, and developing compounds specifically targeted to an emerging viral threat is not a practical short-term solution.
Screening existing drug compounds for alternative indications is a popular alternative [@doi:10.1016/j.drudis.2019.06.014; @doi:10.1111/bph.13895; @doi:10.1039/D0CS01065K; @doi:10.1016/j.coi.2013.08.004].
HTS has been a goal of pharmaceutical development since at least the mid-1980s [@doi:10.1038/sj.bjp.0707373].
Traditionally, phenotypic screens were used to test which compounds would induce a desired change in _in vitro_ or _in vivo_ models, focusing on empirical, function-oriented exploration naïve to molecular mechanism [@doi:10.1126/scitranslmed.aab1201; @doi:10.1038/clpt.2012.236; @doi:10.1038/nrd4366].
In many cases, these screens utilize large libraries that encompass a diverse set of agents varying in many pharmacologically relevant properties (e.g., [@url:https://www.chembridge.com/screening_libraries/diversity_libraries]).
The compounds inducing a desired effect could then be followed up on.
Around the turn of the millennium, advances in molecular biology allowed for HTS to shift towards screening for compounds interacting with a specific molecular target under the hypothesis that modulating that target would have a desired effect.
These approaches both offer pros and cons, and today a popular view is that they are most effective in combination [@doi:10.1038/nrd4366; @doi:10.1016/j.chembiol.2015.11.008; @doi:10.1126/scitranslmed.aab1201].

Today, some efforts to screen compounds for potential repurposing opportunities are experimental, but others use computational HTS approaches [@doi:10.1016/j.drudis.2019.06.014; @doi:10.1016/j.tim.2018.04.004].
Computational drug repurposing screens can take advantage of big data in biology [@doi:10.1038/nrd.2018.168] and as a result are much more feasible today than during the height of the SARS and MERS outbreaks in the early 2000s and early 2010s, respectively.
Advancements in robotics also facilitate the experimental component of HTS [@doi:10.1039/D0CS01065K].
For viral diseases, the goal of drug repurposing is typically to identify existing drugs that have an antiviral effect likely to impede the virus of interest.
While both small molecules and biologics can be candidates for repurposing, the significantly lower price of many small molecule drugs means that they are typically more appealing candidates [@doi:10.1089/adt.2019.950].

Depending on the study design, screens vary in how closely they are tied to a hypothesis.
As with the candidate therapeutics described above, high-throughput experimental or computational screens can proceed based on a hypothesis.
Just as remdesivir was selected as a candidate antiviral because it is a nucleoside analog [@doi:10.1021/acs.jcim.0c00861], so too can high-throughput screens select libraries of compounds based on a molecular hypothesis.
Likewise, when the library of drugs is selected without basis in a potential mechanism, a screen can be considered hypothesis free [@doi:10.1021/acs.jcim.0c00861].
Today, both types of analyses are common both experimentally and computationally.
Both strategies have been applied to identifying candidate therapeutics against SARS-CoV-2.

#### Hypothesis-Driven Screening

Hypothesis-driven screens often select drugs likely to interact with specific viral or host targets or drugs with desired clinical effects, such as immunosuppressants.
There are several properties that might identify a compound as a candidate for an emerging viral disease.
Drugs that interact with a target that is shared between pathogens (i.e., a viral protease or a polymerase) or between a viral pathogen and another illness (i.e., a cancer drug with antiviral potential) are potential candidates, as are drugs that are thought to interact with additional molecular targets beyond those they were developed for [@doi:10.1016/j.tim.2018.04.004].
Such research can be driven by _in vitro_ or _in silico_ experimentation.
Computational analyses depend on identifying compounds that modulate pre-selected proteins in the virus or host.
As a result, they build on experimental research characterizing the molecular features of the virus, host, and candidate compounds [@doi:10.1016/j.coi.2013.08.004].

One example of the application of this approach to COVID-19 research comes from work on protease inhibitors.
Studies have shown that viral proteases play an important role in the life cycle of viruses, including coronaviruses, by modulating the cleavage of viral polyprotein precursors [@doi:10.2174/138161207780162971].
Several FDA-approved drugs target proteases, such as lopinavir and ritonavir for HIV infection and simeprevir for hepatitis C virus infection.
Serine protease inhibitors were previously suggested as possible treatments for SARS and MERS [@doi:10.1016/j.antiviral.2015.01.011].
One early study [@doi:10.1016/j.cell.2020.02.052] suggested that camostat mesylate, a protease inhibitor, could block the entry of SARS-CoV-2 into lung cells _in vitro_.
Two polyproteins encoded by the SARS-CoV-2 replicase gene, pp1a and pp1ab, are critical for viral replication and transcription [@doi:10.1038/s41586-020-2223-y].
These polyproteins must undergo proteolytic processing, which is usually conducted by main protease (M^Pro^), a 33.8-kDa SARS-CoV-2 protease that is therefore fundamental to viral replication and transcription.
Therefore, it was hypothesized that compounds targeting M^Pro^ could be used to prevent or slow the replication of the SARS-CoV-2 virus.

Both computational and experimental approaches facilitated the identification of compounds that might inhibit SARS-CoV-2 M^Pro^.
In 2005, computer-aided design facilitated the development of a Michael acceptor inhibitor, now known as N3, to target M^Pro^ of SARS-like coronaviruses [@doi:10.1371/journal.pbio.0030324].
N3 binds in the substrate binding pocket of M^Pro^ in several HCoV [@doi:10.1007/s13238-013-2841-3; @doi:10.1371/journal.pbio.0030324; @doi:10.1038/srep22677; @doi:10.1128/JVI.02114-07].
The structure of N3-bound SARS-CoV-2 M^Pro^ has been solved, confirming the computational prediction that N3 would similarly bind in the substrate binding pocket of SARS-CoV-2 [@doi:10.1038/s41586-020-2223-y].
N3 was tested _in vitro_ on SARS-CoV-2-infected Vero cells, which belong to a line of cells established from the kidney epithelial cells of an African green monkey, and was found to inhibit SARS-CoV-2 [@doi:10.1038/s41586-020-2223-y].
A library of approximately 10,000 compounds was screened in a fluorescence resonance energy transfer assay constructed using SARS-CoV-2 M^Pro^ expressed in _Escherichia coli_ [@doi:10.1038/s41586-020-2223-y].

Six leads were identified in this hypothesis-driven screen.
_In vitro_ analysis revealed that ebselen had the strongest potency in reducing the viral load in SARS-CoV-2-infected Vero cells [@doi:10.1038/s41586-020-2223-y].
Ebselen is an organoselenium compound with anti-inflammatory and antioxidant properties [@doi:10.1007/s11033-014-3417-x].
Molecular dynamics analysis further demonstrated the potential for ebselen to bind to M^Pro^ and disrupt the protease's enzymatic functions [@doi:10.1126/sciadv.abd0345]. 
However, ebselen is likely to be a promiscuous binder, which could diminish its therapeutic potential [@doi:10.1038/s41586-020-2223-y; @doi:10.1039/c8cc04258f], and compounds with higher specificity may be needed to translate this mechanism effectively to clinical trials.
In July 2020, phase II clinical trials commenced to assess the effects of SPI-1005, an investigational drug from Sound Pharmaceuticals that contains ebselen [@url:https://www.contractpharma.com/contents/view_breaking-news/2020-08-31/fda-clears-spis-ebselen-for-phase-ii-covid-19-trials], on 60 adults presenting with each of moderate [@clinicaltrials:NCT04484025] and severe [@clinicaltrials:NCT04483973] COVID-19.
Other M^Pro^ inhibitors are also being evaluated in clinical trials [@clinicaltrials:NCT04535167; @clinicaltrials:NCT04960202; @clinicaltrials:NCT04960202].
Pending the results of clinical trials, N3 remains a computationally interesting compound based on both computational and experimental data, but whether these potential effects will translate to the clinic remains unknown.

#### Hypothesis-Free Screening

Hypothesis-free screens use a discovery-driven approach, where screens are not targeted to specific viral proteins, host proteins, or desired clinical modulation.
Hypothesis-free drug screening began twenty years ago with the testing of libraries of drugs experimentally.
Today, like many other areas of biology, _in silico_ analyses have become increasingly popular and feasible through advances in biological big data [@doi:10.1021/acs.jcim.0c00861; @doi:10.1080/23808993.2019.1617632].
Many efforts have collected data about interactions between drugs and SARS-CoV-2 and about the host genomic response to SARS-CoV-2 exposure, allowing for hypothesis-free computational screens that seek to identify new candidate therapeutics.
Thus, they utilize a systems biology paradigm to extrapolate the effect of a drug against a virus based on the host interactions with both the virus and the drug [@doi:10.1016/j.coi.2013.08.004].

Resources such as the COVID-19 Drug and Gene Set Library, which at the time of its publication contained 1,620 drugs sourced from 173 experimental and computational drug sets and 18,676 human genes sourced from 444 gene sets [@doi:10.1016/j.patter.2020.100090], facilitate such discovery-driven approaches.
Analysis of these databases indicated that some drugs had been identified as candidates across multiple independent analyses, including high-profile candidates such as CQ/HCQ and remdesivir [@doi:10.1016/j.patter.2020.100090].
Computational screening efforts can then mine databases and other resources to identify potential PPIs among the host, the virus, and established and/or experimental drugs [@doi:10.1038/s41467-020-17189-2].
Subject matter expertise from human users may be integrated to varying extents depending on the platform (e.g,. [@doi:10.1038/s41467-020-17189-2; @doi:10/grs9]).
These resources have allowed studies to identify potential therapeutics for COVID-19 without an _a priori_ reason for selecting them.

One example of a hypothesis-free screen for COVID-19 drugs comes from a PPI-network-based analysis that was published early in the pandemic [@doi:10.1038/s41586-020-2286-9].
Here, researchers cloned the proteins expressed by SARS-CoV-2 _in vitro_ and quantified 332 viral-host PPI using affinity purification mass spectrometry [@doi:10.1038/s41586-020-2286-9].
They identified two SARS-CoV-2 proteins (Nsp6 and Orf9c) that interacted with host Sigma-1 and Sigma-2 receptors.
Sigma receptors are located in the endoplasmic reticulum of many cell types, and type 1 and 2 Sigma receptors have overlapping but distinct affinities for a variety of ligands [@doi:10.1016/j.pharmthera.2009.07.001].
Molecules interacting with the Sigma receptors were then analyzed and found to have an effect on viral infectivity _in vitro_ [@doi:10.1038/s41586-020-2286-9].
A follow-up study evaluated the effect of perturbing these 332 proteins in two cells lines, A549 and Caco-2, using knockdown and knockout methods, respectively, and found that the replication of SARS-CoV-2 in cells from both lines was dependent on the expression of _SIGMAR1_, which is the gene that encodes the Sigma-1 receptor [@pubmed:33060197].
Following these results, drugs interacting with Sigma receptors were suggested as candidates for repurposing for COVID-19 (e.g., [@doi:10.3389/fphar.2020.582310]).
Because many well-known and affordable drugs interact with the Sigma receptors [@doi:10.1038/s41586-020-2286-9; @doi:10.2174/157015905774322516], they became a major focus of drug repurposing efforts.
Some of the drugs suggested by the apparent success of Sigma receptor-targeting drugs were already being investigated at the time.
HCQ, for example, forms ligands with both Sigma-1 and Sigma-2 receptors and was already being explored as a candidate therapeutic for COVID-19 [@doi:10.1038/s41586-020-2286-9].
Thus, this computational approach yielded interest in drugs whose antiviral activity was supported by initial _in vitro_ analyses.

Follow-up research, however, called into question whether the emphasis on drugs interacting with Sigma receptors might be based on a spurious association [@doi:10.1126/science.abi4708].
This study built on the prior work by examining whether antiviral activity among compounds correlated with their affinity for the Sigma receptors and found that it did not.
The study further demonstrated that cationic amphiphilicity was a shared property among many of the candidate drugs identified through both computational and phenotypic screens and that it was likely to be the source of many compounds' proposed antiviral activity [@doi:10.1126/science.abi4708].
Cationic amphiphilicity is associated with the induction of phospholipidosis, which is when phospholipids accumulate in the lysosome [@doi:10.1515/hsz-2019-0270]. 
Phospholipidosis can disrupt viral replication by inhibiting lipid processing [@doi:10.1080/01913123.2020.1850966] (see discussion of HCQ in Appendix 1).
However, phospholipidosis is known to translate poorly from _in vitro_ models to _in vivo_ models or clinical applications.
Thus, this finding suggested that these screens were identifying compounds that shared a physiochemical property rather than a specific target [@doi:10.1126/science.abi4708].
The authors further demonstrated that antiviral activity against SARS-CoV-2 _in vitro_ was correlated with the induction of phospholipidosis for drugs both with and without cationic amphiphilicity [@doi:10.1126/science.abi4708].
This finding supports the idea that the property of cationic amphicility was being detected as a proxy for the shared effect of phospholipidosis [@doi:10.1126/science.abi4708].
They demonstrated that phospholipidosis-inducing drugs were not effective at preventing viral propagation _in vivo_ in a murine model of COVID-19 [@doi:10.1126/science.abi4708].
Therefore, removing hits that induce phospholipidosis from computational and _in vitro_ experimental repurposing screens (e.g., [@doi:10.1016/j.celrep.2021.108959]) may help emphasize those that are more likely to provide clinical benefits.
This work illustrates the importance of considering confounding variables in computational screens, a principle that has been incorporated into more traditional approaches to drug development [@doi:10.1126/science.abj9488].

One drug that acts on Sigma receptors does, however, remain a candidate for the treatment of COVID-19.
Several psychotropic drugs target Sigma receptors in the central nervous system and thus attracted interest as potential COVID-19 therapeutics following the findings of two host-virus PPI studies [@doi:10.1007/s00406-020-01231-x]. 
For several of these drugs, the _in vitro_ antiviral activity [@pubmed:33060197] was not correlated with their affinity for the Sigma-1 receptor [@doi:10.1007/s00406-020-01231-x; @doi:10.1126/science.abi4708] but was correlated with phospholipidosis [@doi:10.1126/science.abi4708].
However, fluvoxamine, a selective serotonin reuptake inhibitor that is a particularly potent Sigma-1 receptor agonist [@doi:10.1007/s00406-020-01231-x], has shown promise as a preventative of severe COVID-19 in a preliminary analysis of data from the large-scale TOGETHER trial [@url:https://rethinkingclinicaltrials.org/news/august-6-2021-early-treatment-of-covid-19-with-repurposed-therapies-the-together-adaptive-platform-trial-edward-mills-phd-frcp].
As of August 6, 2021, this trial had collected data from over 1,400 patients in the fluvoxamine arm of their study, half of whom received a placebo [@url:https://rethinkingclinicaltrials.org/news/august-6-2021-early-treatment-of-covid-19-with-repurposed-therapies-the-together-adaptive-platform-trial-edward-mills-phd-frcp].
Only 74 patients in the fluvoxamine group had progressed to hospitalization for COVID-19 compared to 107 in the placebo group, corresponding to a relative risk of 0.69; additionally, the relative risk of mortality between the two groups was calculated at 0.71.
These findings support the results of small clinical trials that have found fluvoxamine to reduce clinical deterioration relative to a placebo [@doi:10.1001/jama.2020.22760; @doi:10.1093/ofid/ofab050].
However, the ongoing therapeutic potential of fluvoxamine does not contradict the finding that hypothesis-free screening hits can be driven by confounding factors.
The authors point out that its relevance would not just be antiviral as it has a potential immunomodulatory mechanism [@doi:10.1001/jama.2020.22760].
It has been found to be protective against septic shock in an _in vivo_ mouse model [@doi:10.1126/scitranslmed.aau5266].
It is possible that fluvoxamine also exerts an antiviral effect [@doi:10.3389/fphar.2021.652688].
Thus, Sigma-1 receptor activity may contribute to fluvoxamine's potential effects in treating COVID-19, but is not the only mechanism by which this drug can interfere with disease progression.

#### Potential and Limitations of High-Throughput Analyses

Computational screening allows for a large number of compounds to be evaluated to identify those most likely to display a desired behavior or function.
This approach can be guided by a hypothesis or can aim to discover underlying characteristics that produce new hypotheses about the relationship between a host, a virus, and candidate pharmaceuticals.
The examples outlined above illustrate that HTS-based evaluations of drug repurposing can potentially provide valuable insights. 
Computational techniques were used to design compounds targeting M^Pro^ based on an understanding of how this protease aids viral replication, and M^Pro^ inhibitors remain promising candidates [@doi:10.1039/D0CS01065K], although the clinical trial data is not yet available.
Similarly, computational analysis correctly identified the Sigma-1 receptor as a protein of interest.
Although the process of identifying which drugs might modulate the interaction led to an emphasis on candidates that ultimately have not been supported, fluvoxamine remains an appealing candidate.
The difference between the preliminary evidence for fluvoxamine compared to other drugs that interact with Sigma receptors underscores a major critique of hypothesis-free HTS in particular: while these approaches allow for brute force comparison of a large number of compounds against a virus of interest, they lose the element of expertise that is associated with most successes in drug repurposing [@doi:10.1021/acs.jcim.0c00861].

There are also practical limitations to these methods.
One concern is that computational analyses inherently depend on the quality of the data being evaluated.
The urgency of the COVID-19 pandemic led many research groups to pivot towards computational HTS research without familiarity with best practices in this area [@doi:10.1039/D0CS01065K].
As a result, there is an excessive amount of information available from computational studies [@url:https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers], but not all of it is high-quality.
Additionally, the literature used to identify and validate targets can be difficult to reproduce [@doi:10.1038/nrd3439-c1], which may pose challenges to target-based experimental screening and to _in silico_ screens. 
Some efforts to repurpose antivirals have focused on host, rather than viral, proteins [@doi:10.1016/j.coi.2013.08.004], which might be expected to translate poorly _in vivo_ if the targeted proteins serve essential functions in the host.
Concerns about the practicality of hypothesis-free screens to gain novel insights are underscored by the fact that very few or possibly no success stories have emerged from hypothesis-free screens over the past twenty years [@doi:10.1021/acs.jcim.0c00861].
These findings suggest that data-driven research can be an important component of the drug repurposing ecosystem, but that drug repurposing efforts that proceed without a hypothesis, an emphasis on biological mechanisms, or an understanding of confounding effects may not produce viable candidates.

### Considerations in Balancing Different Approaches

The approaches described here offer a variety of advantages and limitations in responding to a novel viral threat and building on existing bodies of knowledge in different ways.
Medicine, pharmacology, basic science (especially virology and immunology), and biological data science can all provide different insights and perspectives for addressing the challenging question of which existing drugs might provide benefits against an emerging viral threat.
A symptom management-driven approach allows clinicians to apply experience with related diseases or related symptoms to organize a rapid response aimed at saving the lives of patients already infected with a new disease.
Oftentimes, the pharmaceutical agents that are applied are small-molecule, broad-spectrum pharmaceuticals that are widely available and affordable to produce, and they may already be available for other purposes, allowing clinicians to administer them to patients quickly either with an EUA or off-label.
In this vein, dexamethasone has emerged as the strongest treatment against severe COVID-19 (Table @tbl:summary-table).

Alternatively, many efforts to repurpose drugs for COVID-19 have built on information gained through basic scientific research of HCoV.
Understanding how related viruses function has allowed researchers to identify possible pharmacological strategies to disrupt pathogenesis (Figure @fig:therapeutics).
Some of the compounds identified through these methods include small-molecule antivirals, which can be boutique and experimental medications like remdesivir (Table @tbl:summary-table).
Other candidate drugs that intercept host-pathogen interactions include biologics, which imitate the function of endogenous host compounds.
Most notably, several mAbs that have been developed (casirivimab, imdevimab, bamlanivimab and etesevimab) or repurposed (sotrovimab, tocilizumab) have now been granted EUAs (Table @tbl:summary-table).
Although not discussed here, several vaccine development programs have also met huge success using a range of strategies [@individual-vaccines].

| Treatment | Grade | Category | FDA Status | Evidence Available | Suggested Effectiveness |
|-----------|-------|----------|-----------------|------------------|-------------------------|
| Dexamethasone | A | Small molecule, broad spectrum | Used off-label | RCT | Supported: RCT shows improved outcomes over SOC, especially in severe cases such as CRS |
|Remdesivir | A | Small molecule, antiviral, adenosine analog | Approved for COVID-19 (and EUA for combination with baricitinib) | RCT | Mixed: Conflicting evidence from large WHO-led Solidarity trial vs US-focused RCT and other studies |
| Tocilizumab |A | Biologic, monoclonal antibody | EUA | RCT | Mixed: It appears that TCZ may work well in combination with dexamethasone in severe cases, but not as monotherapy |
| Sotrovimab | N/A | Biological, monoclonal antibody | EUA | RCT | Supported: Phase 2/3 clinical trial showed reduced hospitalization/death
| Bamlanivimab and etesevimab | B & N/A | Biologic, monoclonal antibodies | EUA | RCT | Supported: Phase 2 clinical trial showed reduction in viral load, but FDA pause recommended because may be less effective against Delta variant|
| Casirivimab and imdevimab | N/A | Biologic, monoclonal antibodies | EUA | RCT | Supported: Reduced viral load at interim analysis |
| Fluvoxamine | B | Small-molecule, Sigma-1 receptor agonist | N/A | RCT | Supported: Support from two small RCTs and preliminary support from interim analysis of TOGETHER
| SNG001 | B | Biologic, interferon | None | RCT | Mixed: Support from initial RCT but no effect found in WHO's Solidarity trial |
| M^Pro^ Protease Inhibitors | N/A | Small molecule, protease inhibitor | None | Computational prediction, _in vitro_ studies | Unknown |
| ARBs & ACEIs | C | Small molecule, broad spectrum | None | Observational studies and some RCTs | Not supported: Observational study retracted, RCTs suggest no association |
| Favipiravir | D | Small molecule, antiviral, nucleoside analog | None | RCT | Not supported: RCTs do not show significant improvements for individuals taking this treatment, good safety profile |
| HCQ/CQ | D | Small molecule, broad spectrum | None | RCT | Not supported, possibly harmful: Non-blinded RCTs showed no improvement over SOC, safety profile may be problematic |
| Convalescent plasma transfusion | D | Biologic, polyclonal antibodies | EUA | RCT | Mixed: Supported in small trials but not in large-scale RECOVERY trial
| Ivermectin | D | Small molecule, broad spectrum | None | RCT | Mixed: Mixed results from small RCTs, major supporting RCT now withdrawn, preliminary results of large RCT (TOGETHER) suggest no effect on emergency room visits or hospitalization for COVID-19

Table: Summary table of candidate therapeutics examined in this manuscript.
"Grade" is the rating given to each treatment by the Systematic Tracker of Off-label/Repurposed Medicines Grades (STORM) maintained by the Center for Cytokine Storm Treatment & Laboratory (CSTL) at the University of Pennsylvania [@url:https://cdcn.org/corona-data-viewer].
A grade of A indicates that a treatment is considered effective, B that all or most RCTs have shown positive results, C that RCT data are not yet available, and D that multiple RCTs have produced negative results.
Treatments not in the STORM database are indicated as N/A.
FDA status is also provided where available.
The evidence available is based on the progression of the therapeutic through the pharmaceutical development pipeline, with RCTs as the most informative source of evidence.
The effectiveness is summarized based on the current available evidence; large trials such as RECOVERY and Solidarity are weighted heavily in this summary.
This table was last updated on August 20, 2021.
{#tbl:summary-table} 

All of the small-molecule drugs evaluated and most of the biologics are repurposed, and thus hinge on a theoretical understanding of how the virus interacts with a human host and how pharmaceuticals can be used to modify those interactions rather than being designed specifically against SARS-CoV-2 or COVID-19.
As a result, significant attention has been paid to computational approaches that automate the identification of potentially desirable interactions.
However, work in COVID-19 has made it clear that relevant compounds can also be masked by confounds, and spurious associations can drive investment in candidate therapeutics that are unlikely to translate to the clinic.
Such spurious hits are especially likely to impact hypothesis-free screens.
However, hypothesis-free screens may still be able to contribute to the drug discovery or repurposing ecosystem, assuming the computational arm of HTS follows the same trends seen in its experimental arm.
In 2011, a landmark study in drug discovery demonstrated that although more new drugs were discovered using target-based rather than phenotypic approaches, the majority of drugs with a novel molecular mechanism of action (MMOA) were identified in phenotypic screens [@doi:10.1038/nrd3480].
This pattern applied only to first-in-class drugs, with most follower drugs produced by target-based screening [@doi:10.1038/clpt.2012.236].
These findings suggest that target-based drug discovery is more successful when building on a known MMOA, and that modulating a target is most valuable when the target is part of a valuable MMOA [@doi:10.1038/nrd4366].
Building on this, many within the field suggested that mechanism-informed phenotypic investigations may be the most useful approach to drug discovery [@doi:10.1038/nrd4366; @doi:10.1016/j.chembiol.2015.11.008; @doi:10.1126/scitranslmed.aab1201].
As it stands, data-driven efforts to identify patterns in the results of computational screens allowed researchers to notice the shared property of cationic amphicility among many of the hits from computational screening analyses [@doi:10.1126/science.abi4708].
While easier said than done, efforts to fill in the black box underlying computational HTS and recognize patterns among the identified compounds aid in moving data-oriented drug repurposing efforts in this direction.

The unpredictable nature of success and failure in drug repurposing for COVID-19 thus highlights one of the tenets of phenotypic screening: there are a lot of "unknown unknowns", and a promising mechanism at the level of an MMOA will not necessarily propagate up to the pathway, cellular, or organismal level [@doi:10.1126/scitranslmed.aab1201].
Despite the fact that apparently mechanistically relevant drugs may exist, identifying effective treatments for a new viral disease is extremely challenging.
Targets of repurposed drugs are often non-specific, meaning that the MMOA can appear to be relevant to COVID-19 without a therapeutic or prophylactic effect being observed in clinical trials.
The difference in the current status of remdesivir and favipiravir as treatments for COVID-19 (Table @tbl:summary-table) underscores how difficult to predict whether a specific compound will produce a desired effect, even when the mechanisms are similar.
Furthermore, the fact that many candidate COVID-19 therapeutics were ultimately identified because of their shared propensity to induce phospholipidosis underscores how challenging it can be to identify a mechanism _in silico_ or _in vitro_ that will translate to a successful treatment.
While significant progress has been made thus far in the pandemic, the therapeutic landscape is likely to continue to evolve as more results become available from clinical trials and as efforts to develop novel therapeutics for COVID-19 progress.

### Towards the Next HCoV Threat

Only very limited testing of candidate therapies was feasible during the SARS and MERS epidemics, and as a result, few treatments were available at the outset of the COVID-19 pandemic.
Even corticosteroids, which were used to treat SARS patients, were a controversial therapeutic prior to the release of the results of the large RECOVERY trial. 
The scale and duration of the COVID-19 pandemic has made it possible to conduct large, rigorous RCTs such as RECOVERY, Solidarity, TOGETHER, and others.
As results from these trials have continued to emerge, it has become clear that small clinical trials often produce spurious results.
In the case of HCQ/CQ, the therapeutic had already attracted so much attention based on small, preliminary (and in some cases, methodologically concerning) studies that it took the results of multiple large studies before attention began to be redirected to more promising candidates [@doi:10.1126/science.368.6496.1166].
In fact, most COVID-19 clinical trials lack the statistical power to reliably test their hypotheses [@doi:10.1038/d41573-021-00037-3; @doi:10.2196/26718].
In the face of an urgent crisis like COVID-19, the desire to act quickly is understandable, but it is imperative that studies maintain strict standards of scientific rigor [@doi:10.1126/science.abj9488; @doi:10.1039/D0CS01065K], especially given the potential dangers of politicization, as illustrated by HCQ/CQ [@doi:10.4269/ajtmh.20-1320].
Potential innovations in clinical trial structure, such as adaptable clinical trials with master protocols [@doi:10.1016/j.cct.2021.106438] or the sharing of data among small clinical trials [@doi:10.2196/26718] may help to address future crises and to bolster the results from smaller studies, respectively. 

In the long-term, new drugs specific for treatment of COVID-19 may also enter development.
Development of novel drugs is likely to be guided by what is known about the pathogenesis and molecular structure of SARS-CoV-2.
For example, understanding the various structural components of SARS-CoV-2 may allow for the development of small molecule inhibitors of those components.
Crystal structures of the SARS-CoV-2 main protease have been resolved [@doi:10.1038/s41586-020-2223-y; @url:https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem.html].
Much work remains to be done to determine further crystal structures of other viral components, understand the relative utility of targeting different viral components, perform additional small molecule inhibitor screens, and determine the safety and efficacy of the potential inhibitors.
While still nascent, work in this area is promising.
Over the longer term, this approach and others may lead to the development of novel therapeutics specifically for COVID-19 and SARS-CoV-2.
Such efforts are likely to prove valuable in managing future emergent HCoV, just as research from the SARS and MERS pandemic has provided a basis for the COVID-19 response.

