<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Halie M. Rando" />
  <meta name="author" content="Casey S. Greene" />
  <meta name="author" content="Michael P. Robson" />
  <meta name="author" content="Simina M. Boca" />
  <meta name="author" content="Nils Wellhausen" />
  <meta name="author" content="Ronan Lordan" />
  <meta name="author" content="Christian Brueffer" />
  <meta name="author" content="Sandipan Ray" />
  <meta name="author" content="Lucy D'Agostino McGowan" />
  <meta name="author" content="Anthony Gitter" />
  <meta name="author" content="Anna Ada Dattoli" />
  <meta name="author" content="Ryan Velazquez" />
  <meta name="author" content="John P. Barton" />
  <meta name="author" content="Jeffrey M. Field" />
  <meta name="author" content="Bharath Ramsundar" />
  <meta name="author" content="Adam L. MacLean" />
  <meta name="author" content="Alexandra J. Lee" />
  <meta name="author" content="Immunology Institute of the Icahn School of Medicine" />
  <meta name="author" content="Fengling Hu" />
  <meta name="author" content="Nafisa M. Jadavji" />
  <meta name="author" content="Elizabeth Sell" />
  <meta name="author" content="Vincent Rubinetti" />
  <meta name="author" content="Jinhui Wang" />
  <meta name="author" content="Diane N. Rafizadeh" />
  <meta name="author" content="Ashwin N. Skelly" />
  <meta name="author" content="Marouen Ben Guebila" />
  <meta name="author" content="Likhitha Kolla" />
  <meta name="author" content="David Manheim" />
  <meta name="author" content="Soumita Ghosh" />
  <meta name="author" content="James Brian Byrd" />
  <meta name="author" content="YoSon Park" />
  <meta name="author" content="Vikas Bansal" />
  <meta name="author" content="Stephen Capone" />
  <meta name="author" content="John J. Dziak" />
  <meta name="author" content="Yuchen Sun" />
  <meta name="author" content="Yanjun Qi" />
  <meta name="author" content="Lamonica Shinholster" />
  <meta name="author" content="Temitayo Lukan" />
  <meta name="author" content="Sergey Knyazev" />
  <meta name="author" content="Dimitri Perrin" />
  <meta name="author" content="Serghei Mangul" />
  <meta name="author" content="Shikta Das" />
  <meta name="author" content="Gregory L Szeto" />
  <meta name="author" content="Tiago Lubiana" />
  <meta name="author" content="David Mai" />
  <meta name="author" content="COVID-19 Review Consortium" />
  <meta name="author" content="Rishi Raj Goel" />
  <meta name="author" content="Joel D Boerckel" />
  <meta name="author" content="Amruta Naik" />
  <meta name="author" content="Yusha Sun" />
  <meta name="author" content="Daniel S. Himmelstein" />
  <meta name="author" content="Jeremy P. Kamil" />
  <meta name="author" content="Jesse G. Meyer" />
  <meta name="author" content="Ariel I. Mundo" />
  <meta name="dcterms.date" content="2022-12-20" />
  <meta name="keywords" content="covid-19, coronavirus, pandemic, viral infection" />
  <title>SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
      margin-bottom: 1em;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta property="og:type" content="article" />
  <meta name="dc.title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta name="citation_title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta property="og:title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta property="twitter:title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta name="dc.date" content="2022-12-20" />
  <meta name="citation_publication_date" content="2022-12-20" />
  <meta property="article:published_time" content="2022-12-20" />
  <meta name="dc.modified" content="2022-12-20T19:46:07+00:00" />
  <meta property="article:modified_time" content="2022-12-20T19:46:07+00:00" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Halie M. Rando" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz School of Medicine, Aurora, Colorado, United States of America" />
  <meta name="citation_author_institution" content="Center for Health AI, University of Colorado Anschutz School of Medicine, Aurora, Colorado, United States of America" />
  <meta name="citation_author_institution" content="Department of Biomedical Informatics, University of Colorado Anschutz School of Medicine, Aurora, Colorado, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7688-1770" />
  <meta name="twitter:creator" content="@tamefoxtime" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz School of Medicine, Aurora, Colorado, United States of America" />
  <meta name="citation_author_institution" content="Center for Health AI, University of Colorado Anschutz School of Medicine, Aurora, Colorado, United States of America" />
  <meta name="citation_author_institution" content="Department of Biomedical Informatics, University of Colorado Anschutz School of Medicine, Aurora, Colorado, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@GreeneScientist" />
  <meta name="citation_author" content="Michael P. Robson" />
  <meta name="citation_author_institution" content="Department of Computing Sciences, Villanova University, Villanova, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-4859-0033" />
  <meta name="citation_author" content="Simina M. Boca" />
  <meta name="citation_author_institution" content="Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1400-3398" />
  <meta name="citation_author" content="Nils Wellhausen" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-8955-7582" />
  <meta name="citation_author" content="Ronan Lordan" />
  <meta name="citation_author_institution" content="Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA" />
  <meta name="citation_author_institution" content="Department of Medicine, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania; Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0001-9668-3368" />
  <meta name="twitter:creator" content="@el_ronan" />
  <meta name="citation_author" content="Christian Brueffer" />
  <meta name="citation_author_institution" content="Department of Clinical Sciences, Lund University, Lund, Sweden" />
  <meta name="citation_author_orcid" content="0000-0002-3826-0989" />
  <meta name="twitter:creator" content="@cbrueffer" />
  <meta name="citation_author" content="Sandipan Ray" />
  <meta name="citation_author_institution" content="Department of Biotechnology, Indian Institute of Technology Hyderabad, Kandi, Sangareddy 502285, Telangana, India" />
  <meta name="citation_author_orcid" content="0000-0002-9960-5768" />
  <meta name="citation_author" content="Lucy D\&#39;Agostino McGowan" />
  <meta name="citation_author_institution" content="Department of Mathematics and Statistics, Wake Forest University, Winston-Salem, North Carolina, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7297-9359" />
  <meta name="twitter:creator" content="@LucyStats" />
  <meta name="citation_author" content="Anthony Gitter" />
  <meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America" />
  <meta name="citation_author_institution" content="Morgridge Institute for Research, Madison, Wisconsin, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5324-9833" />
  <meta name="twitter:creator" content="@anthonygitter" />
  <meta name="citation_author" content="Anna Ada Dattoli" />
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, The Children’s Hospital of Philadelphia, Philadelphia, PA, USA" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA" />
  <meta name="citation_author_orcid" content="0000-0003-1462-831X" />
  <meta name="twitter:creator" content="@aadattoli" />
  <meta name="citation_author" content="Ryan Velazquez" />
  <meta name="citation_author_institution" content="Azimuth1, McLean, Virginia, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-3655-3403" />
  <meta name="citation_author" content="John P. Barton" />
  <meta name="citation_author_institution" content="Department of Physics and Astronomy, University of California-Riverside, Riverside, California, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-1467-421X" />
  <meta name="twitter:creator" content="@_jpbarton" />
  <meta name="citation_author" content="Jeffrey M. Field" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0001-7161-7284" />
  <meta name="citation_author" content="Bharath Ramsundar" />
  <meta name="citation_author_institution" content="The DeepChem Project, https://deepchem.io/" />
  <meta name="citation_author_orcid" content="0000-0001-8450-4262" />
  <meta name="twitter:creator" content="@rbhar90" />
  <meta name="citation_author" content="Adam L. MacLean" />
  <meta name="citation_author_institution" content="Department of Quantitative and Computational Biology, University of Southern California, Los Angeles, California, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0689-7907" />
  <meta name="twitter:creator" content="@adamlmaclean" />
  <meta name="citation_author" content="Alexandra J. Lee" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-0208-3730" />
  <meta name="citation_author" content="Immunology Institute of the Icahn School of Medicine" />
  <meta name="citation_author_institution" content="Immunology Institute of the Icahn School of Medicine" />
  <meta name="citation_author" content="Fengling Hu" />
  <meta name="citation_author_institution" content="Department of Biostatistics, Epidemiology and Informatics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-1081-5038" />
  <meta name="twitter:creator" content="@hufengling" />
  <meta name="citation_author" content="Nafisa M. Jadavji" />
  <meta name="citation_author_institution" content="Biomedical Science, Midwestern University, Glendale, AZ, United States of America" />
  <meta name="citation_author_institution" content="Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada" />
  <meta name="citation_author_orcid" content="0000-0002-3557-7307" />
  <meta name="twitter:creator" content="@nafisajadavji" />
  <meta name="citation_author" content="Elizabeth Sell" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-9658-1107" />
  <meta name="citation_author" content="Vincent Rubinetti" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Center for Health AI, University of Colorado School of Medicine, Aurora, Colorado, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-4655-3773" />
  <meta name="citation_author" content="Jinhui Wang" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5796-8130" />
  <meta name="citation_author" content="Diane N. Rafizadeh" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Department of Chemistry, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-2838-067X" />
  <meta name="citation_author" content="Ashwin N. Skelly" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Institute for Immunology, University of Pennsylvania Perelman School of Medicine, Philadelphia, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1565-3376" />
  <meta name="citation_author" content="Marouen Ben Guebila" />
  <meta name="citation_author_institution" content="Department of Biostatistics, Harvard School of Public Health, Boston, Massachusetts, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-5934-966X" />
  <meta name="twitter:creator" content="@marouenbg" />
  <meta name="citation_author" content="Likhitha Kolla" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1169-906X" />
  <meta name="twitter:creator" content="@lkolla2018" />
  <meta name="citation_author" content="David Manheim" />
  <meta name="citation_author_institution" content="1DaySooner, Delaware, United States of America" />
  <meta name="citation_author_institution" content="Risk and Health Communication Research Center, School of Public Health, University of Haifa, Haifa, Israel" />
  <meta name="citation_author_institution" content="Technion, Israel Institute of Technology, Haifa, Israel" />
  <meta name="citation_author_orcid" content="0000-0001-8599-8380" />
  <meta name="twitter:creator" content="@davidmanheim" />
  <meta name="citation_author" content="Soumita Ghosh" />
  <meta name="citation_author_institution" content="Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-2783-2750" />
  <meta name="citation_author" content="James Brian Byrd" />
  <meta name="citation_author_institution" content="University of Michigan School of Medicine, Ann Arbor, Michigan, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-0509-3520" />
  <meta name="twitter:creator" content="@thebyrdlab" />
  <meta name="citation_author" content="YoSon Park" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-0465-4744" />
  <meta name="twitter:creator" content="@__yoson__" />
  <meta name="citation_author" content="Vikas Bansal" />
  <meta name="citation_author_institution" content="Biomedical Data Science and Machine Learning Group, German Center for Neurodegenerative Diseases, Tübingen 72076, Germany" />
  <meta name="citation_author_orcid" content="0000-0002-0944-7226" />
  <meta name="twitter:creator" content="@VikasBansal1989" />
  <meta name="citation_author" content="Stephen Capone" />
  <meta name="citation_author_institution" content="St. George&#39;s University School of Medicine, St. George&#39;s, Grenada" />
  <meta name="citation_author_orcid" content="0000-0001-7231-1535" />
  <meta name="citation_author" content="John J. Dziak" />
  <meta name="citation_author_institution" content="Edna Bennett Pierce Prevention Research Center, The Pennsylvania State University, University Park, PA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0762-5495" />
  <meta name="citation_author" content="Yuchen Sun" />
  <meta name="citation_author_institution" content="Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America" />
  <meta name="citation_author" content="Yanjun Qi" />
  <meta name="citation_author_institution" content="Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5796-7453" />
  <meta name="citation_author" content="Lamonica Shinholster" />
  <meta name="citation_author_institution" content="Mercer University, Macon, GA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-6285-005X" />
  <meta name="citation_author" content="Temitayo Lukan" />
  <meta name="citation_author_institution" content="University of Pennsylvania, Philadelphia, PA, United States of America" />
  <meta name="citation_author" content="Sergey Knyazev" />
  <meta name="citation_author_institution" content="Georgia State University, Atlanta, GA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0385-1831" />
  <meta name="twitter:creator" content="@SeKnyaz" />
  <meta name="citation_author" content="Dimitri Perrin" />
  <meta name="citation_author_institution" content="School of Computer Science, Queensland University of Technology, Brisbane, Australia" />
  <meta name="citation_author_institution" content="Centre for Data Science, Queensland University of Technology, Brisbane, Australia" />
  <meta name="citation_author_orcid" content="0000-0002-4007-5256" />
  <meta name="twitter:creator" content="@dperrin" />
  <meta name="citation_author" content="Serghei Mangul" />
  <meta name="citation_author_institution" content="Department of Clinical Pharmacy, School of Pharmacy, University of Southern California, Los Angeles, CA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-4770-3443" />
  <meta name="twitter:creator" content="@serghei_mangul" />
  <meta name="citation_author" content="Shikta Das" />
  <meta name="citation_author_institution" content="C4X Discovery, London, United Kingdom" />
  <meta name="citation_author_institution" content="Medical Research Council LHA, Institute of Cardiovascular Studies, University College London, London, United Kingdom" />
  <meta name="citation_author_orcid" content="0000-0002-8291-2788" />
  <meta name="twitter:creator" content="@shikta_das" />
  <meta name="citation_author" content="Gregory L Szeto" />
  <meta name="citation_author_institution" content="Allen Institute for Immunology, Seattle, WA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7604-1333" />
  <meta name="twitter:creator" content="@greg_szeto" />
  <meta name="citation_author" content="Tiago Lubiana" />
  <meta name="citation_author_institution" content="Department of Clinical and Toxicological Analyses, School of Pharmaceutical Sciences, University of São Paulo, São Paulo, Brazil" />
  <meta name="citation_author_orcid" content="0000-0003-2473-2313" />
  <meta name="twitter:creator" content="@lubianat" />
  <meta name="citation_author" content="David Mai" />
  <meta name="citation_author_institution" content="Department of Bioengineering, University of Pennsylvania, Philadelphia, PA, USA" />
  <meta name="citation_author_institution" content="Center for Cellular Immunotherapies, Perelman School of Medicine, and Parker Institute for Cancer Immunotherapy at University of Pennsylvania, Philadelphia, PA, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9238-0164" />
  <meta name="twitter:creator" content="@lococyte" />
  <meta name="citation_author" content="COVID-19 Review Consortium" />
  <meta name="citation_author" content="Rishi Raj Goel" />
  <meta name="citation_author_institution" content="Institute for Immunology, University of Pennsylvania, Philadelphia, PA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-1715-5191" />
  <meta name="twitter:creator" content="@rishirajgoel" />
  <meta name="citation_author" content="Joel D Boerckel" />
  <meta name="citation_author_institution" content="Department of Orthopaedic Surgery, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, United States of America" />
  <meta name="citation_author_institution" content="Department of Bioengineering, University of Pennsylvania, Philadelphia, PA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-3126-3025" />
  <meta name="twitter:creator" content="@jboerckel" />
  <meta name="citation_author" content="Amruta Naik" />
  <meta name="citation_author_institution" content="Children&#39;s Hospital of Philadelphia, Philadelphia, PA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0673-2643" />
  <meta name="citation_author" content="Yusha Sun" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-4835-3000" />
  <meta name="citation_author" content="Daniel S. Himmelstein" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Related Sciences" />
  <meta name="citation_author_orcid" content="0000-0002-3012-7446" />
  <meta name="twitter:creator" content="@dhimmel" />
  <meta name="citation_author" content="Jeremy P. Kamil" />
  <meta name="citation_author_institution" content="Department of Microbiology and Immunology, Louisiana State University Health Sciences Center Shreveport, Shreveport, Louisiana, USA" />
  <meta name="citation_author_orcid" content="0000-0001-8422-7656" />
  <meta name="citation_author" content="Jesse G. Meyer" />
  <meta name="citation_author_institution" content="Department of Biochemistry, Medical College of Wisconsin, Milwaukee, Wisconsin, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-2753-3926" />
  <meta name="citation_author" content="Ariel I. Mundo" />
  <meta name="citation_author_institution" content="Department of Biomedical Engineering, University of Arkansas, Fayetteville, Arkansas, USA" />
  <meta name="citation_author_orcid" content="0000-0002-6014-4538" />
  <link rel="canonical" href="https://greenelab.github.io/covid19-review/" />
  <meta property="og:url" content="https://greenelab.github.io/covid19-review/" />
  <meta property="twitter:url" content="https://greenelab.github.io/covid19-review/" />
  <meta name="citation_fulltext_html_url" content="https://greenelab.github.io/covid19-review/" />
  <meta name="citation_pdf_url" content="https://greenelab.github.io/covid19-review/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://greenelab.github.io/covid19-review/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://greenelab.github.io/covid19-review/v/bc50d123828787c6491434d7edd1bca56859918d/" />
  <meta name="manubot_html_url_versioned" content="https://greenelab.github.io/covid19-review/v/bc50d123828787c6491434d7edd1bca56859918d/" />
  <meta name="manubot_pdf_url_versioned" content="https://greenelab.github.io/covid19-review/v/bc50d123828787c6491434d7edd1bca56859918d/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <meta property="og:image" content="https://github.com/greenelab/covid19-review/raw/bc50d123828787c6491434d7edd1bca56859918d/content/images/thumbnail.png" />
  <meta property="twitter:image" content="https://github.com/greenelab/covid19-review/raw/bc50d123828787c6491434d7edd1bca56859918d/content/images/thumbnail.png" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://greenelab.github.io/covid19-review/v/bc50d123828787c6491434d7edd1bca56859918d/">permalink</a>)
was automatically generated
from <a href="https://github.com/greenelab/covid19-review/tree/bc50d123828787c6491434d7edd1bca56859918d">greenelab/covid19-review@bc50d12</a>
on 2022-12-20.
It is also available as a <a href="https://greenelab.github.io/covid19-review/v/bc50d123828787c6491434d7edd1bca56859918d/manuscript.pdf">PDF</a>.
Snapshots of individual sections have been published <span class="citation" data-cites="GdZc4Yyd wgAGKcBj njpLhBui EM9YkiOF">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>–<a href="#ref-EM9YkiOF" role="doc-biblioref">4</a>)</span> or posted as preprints <span class="citation" data-cites="hIxvVJ7F 1HWmcSB20 rjHK19Dr">(<a href="#ref-hIxvVJ7F" role="doc-biblioref">5</a>–<a href="#ref-rjHK19Dr" role="doc-biblioref">7</a>)</span>.
</em></small></p>
<!-- include the Font Awesome library, per: https://fontawesome.com/start -->
<p><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css">
<span class="banner lightred"><span class="fas fa-info-circle fa-lg"></span> <strong>This in progress manuscript is not intended for the general public.</strong><br>
This is a review paper that is authored by scientists for an audience of scientists to discuss research that is in progress.
If you are interested in guidelines on testing, therapies, or other issues related to your health, you should not use this document.
Instead, you should collect information from your local health department, the <a href="https://www.cdc.gov/coronavirus/2019-ncov/index.html">CDC’s guidance</a>, or your own government.</span></p>
<h1 class="unnumbered" id="authors">Authors</h1>
<ul>
<li><p><strong>Halie M. Rando</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7688-1770">0000-0001-7688-1770</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/rando2">rando2</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/tamefoxtime">tamefoxtime</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz School of Medicine, Aurora, Colorado, United States of America; Center for Health AI, University of Colorado Anschutz School of Medicine, Aurora, Colorado, United States of America; Department of Biomedical Informatics, University of Colorado Anschutz School of Medicine, Aurora, Colorado, United States of America
· Funded by the Gordon and Betty Moore Foundation (GBMF 4552); the National Human Genome Research Institute (R01 HG010067)
</small></p></li>
<li><p><strong>Casey S. Greene</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8713-9213">0000-0001-8713-9213</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/cgreene">cgreene</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/GreeneScientist">GreeneScientist</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Philadelphia, Pennsylvania, United States of America; Department of Biochemistry and Molecular Genetics, University of Colorado Anschutz School of Medicine, Aurora, Colorado, United States of America; Center for Health AI, University of Colorado Anschutz School of Medicine, Aurora, Colorado, United States of America; Department of Biomedical Informatics, University of Colorado Anschutz School of Medicine, Aurora, Colorado, United States of America
· Funded by the Gordon and Betty Moore Foundation (GBMF 4552); the National Human Genome Research Institute (R01 HG010067)
</small></p></li>
<li><p><strong>Michael P. Robson</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-4859-0033">0000-0002-4859-0033</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/mprobson">mprobson</a><br>
<small>
Department of Computing Sciences, Villanova University, Villanova, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Simina M. Boca</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1400-3398">0000-0002-1400-3398</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/SiminaB">SiminaB</a><br>
<small>
Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America
</small></p></li>
<li><p><strong>Nils Wellhausen</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8955-7582">0000-0001-8955-7582</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/nilswellhausen">nilswellhausen</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Ronan Lordan</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-9668-3368">0000-0001-9668-3368</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/RLordan">RLordan</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/el_ronan">el_ronan</a><br>
<small>
Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA; Department of Medicine, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA; Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania; Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Christian Brueffer</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3826-0989">0000-0002-3826-0989</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/cbrueffer">cbrueffer</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/cbrueffer">cbrueffer</a><br>
<small>
Department of Clinical Sciences, Lund University, Lund, Sweden
</small></p></li>
<li><p><strong>Sandipan Ray</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9960-5768">0000-0002-9960-5768</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/rays1987">rays1987</a><br>
<small>
Department of Biotechnology, Indian Institute of Technology Hyderabad, Kandi, Sangareddy 502285, Telangana, India
</small></p></li>
<li><p><strong>Lucy D'Agostino McGowan</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7297-9359">0000-0001-7297-9359</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/LucyMcGowan">LucyMcGowan</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/LucyStats">LucyStats</a><br>
<small>
Department of Mathematics and Statistics, Wake Forest University, Winston-Salem, North Carolina, United States of America
</small></p></li>
<li><p><strong>Anthony Gitter</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5324-9833">0000-0002-5324-9833</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/agitter">agitter</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/anthonygitter">anthonygitter</a><br>
<small>
Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America; Morgridge Institute for Research, Madison, Wisconsin, United States of America
· Funded by John W. and Jeanne M. Rowe Center for Research in Virology
</small></p></li>
<li><p><strong>Anna Ada Dattoli</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1462-831X">0000-0003-1462-831X</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/aadattoli">aadattoli</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/aadattoli">aadattoli</a><br>
<small>
Department of Pathology and Laboratory Medicine, The Children’s Hospital of Philadelphia, Philadelphia, PA, USA; Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA
</small></p></li>
<li><p><strong>Ryan Velazquez</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3655-3403">0000-0002-3655-3403</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/rdvelazquez">rdvelazquez</a><br>
<small>
Azimuth1, McLean, Virginia, United States of America
</small></p></li>
<li><p><strong>John P. Barton</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1467-421X">0000-0003-1467-421X</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/johnbarton">johnbarton</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/_jpbarton">_jpbarton</a><br>
<small>
Department of Physics and Astronomy, University of California-Riverside, Riverside, California, United States of America
</small></p></li>
<li><p><strong>Jeffrey M. Field</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7161-7284">0000-0001-7161-7284</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/Jeff-Field">Jeff-Field</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Bharath Ramsundar</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8450-4262">0000-0001-8450-4262</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/rbharath">rbharath</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/rbhar90">rbhar90</a><br>
<small>
The DeepChem Project, https://deepchem.io/
</small></p></li>
<li><p><strong>Adam L. MacLean</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0689-7907">0000-0003-0689-7907</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/adamlmaclean">adamlmaclean</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/adamlmaclean">adamlmaclean</a><br>
<small>
Department of Quantitative and Computational Biology, University of Southern California, Los Angeles, California, United States of America
</small></p></li>
<li><p><strong>Alexandra J. Lee</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0208-3730">0000-0002-0208-3730</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/ajlee21">ajlee21</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by the Gordon and Betty Moore Foundation (GBMF 4552)
</small></p></li>
<li><p><strong>Immunology Institute of the Icahn School of Medicine</strong><br> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/ismms-himc">ismms-himc</a><br>
<small>
Immunology Institute of the Icahn School of Medicine
</small></p></li>
<li><p><strong>Fengling Hu</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1081-5038">0000-0003-1081-5038</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/hufengling">hufengling</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/hufengling">hufengling</a><br>
<small>
Department of Biostatistics, Epidemiology and Informatics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Nafisa M. Jadavji</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3557-7307">0000-0002-3557-7307</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/nafisajadavji">nafisajadavji</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/nafisajadavji">nafisajadavji</a><br>
<small>
Biomedical Science, Midwestern University, Glendale, AZ, United States of America; Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada
· Funded by the American Heart Association (20AIREA35050015)
</small></p></li>
<li><p><strong>Elizabeth Sell</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9658-1107">0000-0002-9658-1107</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/esell17">esell17</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Vincent Rubinetti</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-4655-3773">0000-0002-4655-3773</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/vincerubinetti">vincerubinetti</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Center for Health AI, University of Colorado School of Medicine, Aurora, Colorado, United States of America
</small></p></li>
<li><p><strong>Jinhui Wang</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5796-8130">0000-0002-5796-8130</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/jinhui2">jinhui2</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Diane N. Rafizadeh</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2838-067X">0000-0002-2838-067X</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/dianerafi">dianerafi</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Department of Chemistry, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by NIH Medical Scientist Training Program T32 GM07170
</small></p></li>
<li><p><strong>Ashwin N. Skelly</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1565-3376">0000-0002-1565-3376</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/anskelly">anskelly</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Institute for Immunology, University of Pennsylvania Perelman School of Medicine, Philadelphia, United States of America
· Funded by NIH Medical Scientist Training Program T32 GM07170
</small></p></li>
<li><p><strong>Marouen Ben Guebila</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-5934-966X">0000-0001-5934-966X</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/marouenbg">marouenbg</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/marouenbg">marouenbg</a><br>
<small>
Department of Biostatistics, Harvard School of Public Health, Boston, Massachusetts, United States of America
</small></p></li>
<li><p><strong>Likhitha Kolla</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1169-906X">0000-0002-1169-906X</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/likhithakolla">likhithakolla</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/lkolla2018">lkolla2018</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by NIH Medical Scientist Training Program T32 GM07170
</small></p></li>
<li><p><strong>David Manheim</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8599-8380">0000-0001-8599-8380</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/davidmanheim">davidmanheim</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/davidmanheim">davidmanheim</a><br>
<small>
1DaySooner, Delaware, United States of America; Risk and Health Communication Research Center, School of Public Health, University of Haifa, Haifa, Israel; Technion, Israel Institute of Technology, Haifa, Israel
· Funded by Center for Effective Altruism, Long Term Future Fund
</small></p></li>
<li><p><strong>Soumita Ghosh</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2783-2750">0000-0002-2783-2750</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/soumitagh">soumitagh</a><br>
<small>
Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>James Brian Byrd</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0509-3520">0000-0002-0509-3520</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/byrdjb">byrdjb</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/thebyrdlab">thebyrdlab</a><br>
<small>
University of Michigan School of Medicine, Ann Arbor, Michigan, United States of America
· Funded by NIH K23HL128909; FastGrants
</small></p></li>
<li><p><strong>YoSon Park</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0465-4744">0000-0002-0465-4744</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/ypar">ypar</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/__yoson__"><strong>yoson</strong></a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
· Funded by NHGRI R01 HG10067
</small></p></li>
<li><p><strong>Vikas Bansal</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0944-7226">0000-0002-0944-7226</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/bansalvi">bansalvi</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/VikasBansal1989">VikasBansal1989</a><br>
<small>
Biomedical Data Science and Machine Learning Group, German Center for Neurodegenerative Diseases, Tübingen 72076, Germany
</small></p></li>
<li><p><strong>Stephen Capone</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7231-1535">0000-0001-7231-1535</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/scapone01">scapone01</a><br>
<small>
St. George’s University School of Medicine, St. George’s, Grenada
</small></p></li>
<li><p><strong>John J. Dziak</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0762-5495">0000-0003-0762-5495</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/dziakj1">dziakj1</a><br>
<small>
Edna Bennett Pierce Prevention Research Center, The Pennsylvania State University, University Park, PA, United States of America
</small></p></li>
<li><p><strong>Yuchen Sun</strong><br> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/kevinsunofficial">kevinsunofficial</a><br>
<small>
Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America
</small></p></li>
<li><p><strong>Yanjun Qi</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5796-7453">0000-0002-5796-7453</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/qiyanjun">qiyanjun</a><br>
<small>
Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America
</small></p></li>
<li><p><strong>Lamonica Shinholster</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6285-005X">0000-0001-6285-005X</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/LSH2126">LSH2126</a><br>
<small>
Mercer University, Macon, GA, United States of America
· Funded by the Center for Global Genomics and Health Equity at the University of Pennsylvania
</small></p></li>
<li><p><strong>Temitayo Lukan</strong><br> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/tlukan">tlukan</a><br>
<small>
University of Pennsylvania, Philadelphia, PA, United States of America
</small></p></li>
<li><p><strong>Sergey Knyazev</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0385-1831">0000-0003-0385-1831</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/Sergey-Knyazev">Sergey-Knyazev</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/SeKnyaz">SeKnyaz</a><br>
<small>
Georgia State University, Atlanta, GA, United States of America
</small></p></li>
<li><p><strong>Dimitri Perrin</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-4007-5256">0000-0002-4007-5256</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/SystemsResearch">SystemsResearch</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/dperrin">dperrin</a><br>
<small>
School of Computer Science, Queensland University of Technology, Brisbane, Australia; Centre for Data Science, Queensland University of Technology, Brisbane, Australia
</small></p></li>
<li><p><strong>Serghei Mangul</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-4770-3443">0000-0003-4770-3443</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/smangul1">smangul1</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/serghei_mangul">serghei_mangul</a><br>
<small>
Department of Clinical Pharmacy, School of Pharmacy, University of Southern California, Los Angeles, CA, United States of America
</small></p></li>
<li><p><strong>Shikta Das</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-8291-2788">0000-0002-8291-2788</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/shiktadas">shiktadas</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/shikta_das">shikta_das</a><br>
<small>
C4X Discovery, London, United Kingdom; Medical Research Council LHA, Institute of Cardiovascular Studies, University College London, London, United Kingdom
</small></p></li>
<li><p><strong>Gregory L Szeto</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7604-1333">0000-0001-7604-1333</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/gregszetoAI">gregszetoAI</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/greg_szeto">greg_szeto</a><br>
<small>
Allen Institute for Immunology, Seattle, WA, United States of America
</small></p></li>
<li><p><strong>Tiago Lubiana</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2473-2313">0000-0003-2473-2313</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/lubianat">lubianat</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/lubianat">lubianat</a><br>
<small>
Department of Clinical and Toxicological Analyses, School of Pharmaceutical Sciences, University of São Paulo, São Paulo, Brazil
</small></p></li>
<li><p><strong>David Mai</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9238-0164">0000-0002-9238-0164</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/davemai">davemai</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/lococyte">lococyte</a><br>
<small>
Department of Bioengineering, University of Pennsylvania, Philadelphia, PA, USA; Center for Cellular Immunotherapies, Perelman School of Medicine, and Parker Institute for Cancer Immunotherapy at University of Pennsylvania, Philadelphia, PA, USA
</small></p></li>
<li><p><strong>COVID-19 Review Consortium</strong><br><br>
<small>
</small></p></li>
<li><p><strong>Rishi Raj Goel</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1715-5191">0000-0003-1715-5191</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/rishirajgoel">rishirajgoel</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/rishirajgoel">rishirajgoel</a><br>
<small>
Institute for Immunology, University of Pennsylvania, Philadelphia, PA, United States of America
</small></p></li>
<li><p><strong>Joel D Boerckel</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-3126-3025">0000-0003-3126-3025</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/jboerckel">jboerckel</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/jboerckel">jboerckel</a><br>
<small>
Department of Orthopaedic Surgery, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, United States of America; Department of Bioengineering, University of Pennsylvania, Philadelphia, PA, United States of America
</small></p></li>
<li><p><strong>Amruta Naik</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0673-2643">0000-0003-0673-2643</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/NAIKA86">NAIKA86</a><br>
<small>
Children’s Hospital of Philadelphia, Philadelphia, PA, United States of America
</small></p></li>
<li><p><strong>Yusha Sun</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-4835-3000">0000-0003-4835-3000</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/yusha-sun">yusha-sun</a><br>
<small>
Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Daniel S. Himmelstein</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3012-7446">0000-0002-3012-7446</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/dhimmel">dhimmel</a> · <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/dhimmel">dhimmel</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Related Sciences
· Funded by GBMF4552
</small></p></li>
<li><p><strong>Jeremy P. Kamil</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8422-7656">0000-0001-8422-7656</a><br>
<small>
Department of Microbiology and Immunology, Louisiana State University Health Sciences Center Shreveport, Shreveport, Louisiana, USA
</small></p></li>
<li><p><strong>Jesse G. Meyer</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2753-3926">0000-0003-2753-3926</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/jessegmeyerlab">jessegmeyerlab</a><br>
<small>
Department of Biochemistry, Medical College of Wisconsin, Milwaukee, Wisconsin, United States of America
· Funded by National Institute of General Medical Sciences (R35 GM142502)
</small></p></li>
<li><p><strong>Ariel I. Mundo</strong><br> <img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-6014-4538">0000-0002-6014-4538</a> · <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/aimundo">aimundo</a><br>
<small>
Department of Biomedical Engineering, University of Arkansas, Fayetteville, Arkansas, USA
</small></p></li>
</ul>
<p><strong>COVID-19 Review Consortium:</strong>
Vikas Bansal, John P. Barton, Simina M. Boca, Joel D Boerckel, Christian Brueffer, James Brian Byrd, Stephen Capone, Shikta Das, Anna Ada Dattoli, John J. Dziak, Jeffrey M. Field, Soumita Ghosh, Anthony Gitter, Rishi Raj Goel, Casey S. Greene, Marouen Ben Guebila, Daniel S. Himmelstein, Fengling Hu, Nafisa M. Jadavji, Jeremy P. Kamil, Sergey Knyazev, Likhitha Kolla, Alexandra J. Lee, Ronan Lordan, Tiago Lubiana, Temitayo Lukan, Adam L. MacLean, David Mai, Serghei Mangul, David Manheim, Lucy D'Agostino McGowan, Jesse G. Meyer, Ariel I. Mundo, Amruta Naik, YoSon Park, Dimitri Perrin, Yanjun Qi, Diane N. Rafizadeh, Bharath Ramsundar, Halie M. Rando, Sandipan Ray, Michael P. Robson, Vincent Rubinetti, Elizabeth Sell, Lamonica Shinholster, Ashwin N. Skelly, Yuchen Sun, Yusha Sun, Gregory L Szeto, Ryan Velazquez, Jinhui Wang, Nils Wellhausen</p>
<p>Authors are ordered arbitrarily.</p>
<h1 data-number="1" id="pathogenesis-symptomatology-and-transmission-of-sars-cov-2-through-analysis-of-viral-genomics-and-structure"><span class="header-section-number">1</span> Pathogenesis, Symptomatology, and Transmission of SARS-CoV-2 through Analysis of Viral Genomics and Structure</h1>
<h2 data-number="1.1" id="abstract"><span class="header-section-number">1.1</span> Abstract</h2>
<p>The novel coronavirus SARS-CoV-2, which emerged in late 2019, has since spread around the world and infected hundreds of millions of people with coronavirus disease 2019 (COVID-19).
While this viral species was unknown prior to January 2020, its similarity to other coronaviruses that infect humans has allowed for rapid insight into the mechanisms that it uses to infect human hosts, as well as the ways in which the human immune system can respond.
Here, we contextualize SARS-CoV-2 among other coronaviruses and identify what is known and what can be inferred about its behavior once inside a human host.
Because the genomic content of coronaviruses, which specifies the virus’s structure, is highly conserved, early genomic analysis provided a significant head start in predicting viral pathogenesis and in understanding potential differences among variants.
The pathogenesis of the virus offers insights into symptomatology, transmission, and individual susceptibility.
Additionally, prior research into interactions between the human immune system and coronaviruses has identified how these viruses can evade the immune system’s protective mechanisms.
We also explore systems-level research into the regulatory and proteomic effects of SARS-CoV-2 infection and the immune response.
Understanding the structure and behavior of the virus serves to contextualize the many facets of the COVID-19 pandemic and can influence efforts to control the virus and treat the disease.</p>
<h2 data-number="1.2" id="importance"><span class="header-section-number">1.2</span> Importance</h2>
<p>COVID-19 involves a number of organ systems and can present with a wide range of symptoms.
From how the virus infects cells to how it spreads between people, the available research suggests that these patterns are very similar to those seen in the closely related viruses SARS-CoV-1 and possibly MERS-CoV.
Understanding the pathogenesis of the SARS-CoV-2 virus also contextualizes how the different biological systems affected by COVID-19 connect.
Exploring the structure, phylogeny, and pathogenesis of the virus therefore helps to guide interpretation of the broader impacts of the virus on the human body and on human populations.
For this reason, an in-depth exploration of viral mechanisms is critical to a robust understanding of SARS-CoV-2 and, potentially, future emergent HCoV.</p>
<h2 data-number="1.3" id="introduction"><span class="header-section-number">1.3</span> Introduction</h2>
<p>The current coronavirus disease 2019 (COVID-19) pandemic, caused by the <em>Severe acute respiratory syndrome-related coronavirus 2</em> (SARS-CoV-2) virus, represents an acute global health crisis.
Symptoms of the disease can range from mild to severe or fatal <span class="citation" data-cites="gHWlMufv">(<a href="#ref-gHWlMufv" role="doc-biblioref">8</a>)</span> and can affect a variety of organs and systems <span class="citation" data-cites="DkrkT4Mb">(<a href="#ref-DkrkT4Mb" role="doc-biblioref">9</a>)</span>.
Outcomes of infection can include acute respiratory distress (ARDS) and acute lung injury, as well as damage to other organ systems <span class="citation" data-cites="DkrkT4Mb 1H0HgI6iy">(<a href="#ref-DkrkT4Mb" role="doc-biblioref">9</a>, <a href="#ref-1H0HgI6iy" role="doc-biblioref">10</a>)</span>.
Understanding the progression of the disease, including these diverse symptoms, depends on understanding how the virus interacts with the host.
Additionally, the fundamental biology of the virus can provide insights into how it is transmitted among people, which can, in turn, inform efforts to control its spread.
As a result, a thorough understanding of the pathogenesis of SARS-CoV-2 is a critical foundation on which to build an understanding of COVID-19 and the pandemic as a whole.</p>
<p>The rapid identification and release of the genomic sequence of the virus in January 2020 <span class="citation" data-cites="Bp847Lfa">(<a href="#ref-Bp847Lfa" role="doc-biblioref">11</a>)</span> provided early insight into the virus in a comparative genomic context.
The viral genomic sequence clusters with known coronaviruses (order <em>Nidovirales</em>, family <em>Coronaviridae</em>, subfamily <em>Orthocoronavirinae</em>).
Phylogenetic analysis of the coronaviruses reveals four major subclades, each corresponding to a genus: the alpha, beta, gamma, and delta coronaviruses.
Among them, alpha and beta coronaviruses infect mammalian species, gamma coronaviruses infect avian species, and delta coronaviruses infect both mammalian and avian species <span class="citation" data-cites="17DSmRo9H">(<a href="#ref-17DSmRo9H" role="doc-biblioref">12</a>)</span>.
The novel virus now known as SARS-CoV-2 was identified as a beta coronavirus belonging to the B lineage based on phylogenetic analysis of a polymerase chain reaction (PCR) amplicon fragment from five patients along with the full genomic sequence <span class="citation" data-cites="IFLf8rsY">(<a href="#ref-IFLf8rsY" role="doc-biblioref">13</a>)</span>.
This lineage also includes the <em>Severe acute respiratory syndrome-related coronavirus</em> (SARS-CoV-1) that caused the 2002-2003 outbreak of Severe Acute Respiratory Syndrome (SARS) in humans <span class="citation" data-cites="IFLf8rsY">(<a href="#ref-IFLf8rsY" role="doc-biblioref">13</a>)</span>.
(Note that these subclades are not to be confused with variants of concern within SARS-CoV-2 labeled with Greek letters; i.e., the Delta variant of SARS-CoV-2 is still a beta coronavirus.)</p>
<p>Because viral structure and mechanisms of pathogenicity are highly conserved within the order, this phylogenetic analysis provided a basis for forming hypotheses about how the virus interacts with hosts, including which tissues, organs, and systems would be most susceptible to SARS-CoV-2 infection.
Coronaviruses that infect humans (HCoV) are not common, but prior research into other HCoV such as SARS-CoV-1 and <em>Middle East respiratory syndrome-related coronavirus</em> (MERS-CoV), as well as other viruses infecting humans such as a variety of influenza species, established a strong foundation that accelerated the pace of SARS-CoV-2 research.</p>
<p>Coronaviruses are large viruses that can be identified by their distinctive “crown-like” shape (Figure <a href="#fig:genome-structure">1</a>).
Their spherical virions are made from lipid envelopes ranging from 100 to 160 nanometers in which peplomers (protruding structures) of two to three spike (S) glycoproteins are anchored, creating the crown <span class="citation" data-cites="RGAUx68v DlBAegL1">(<a href="#ref-RGAUx68v" role="doc-biblioref">14</a>, <a href="#ref-DlBAegL1" role="doc-biblioref">15</a>)</span>.
These spikes, which are critical to both viral pathogenesis and to the response by the host immune response, have been visualized using cryo-electron microscopy <span class="citation" data-cites="CqQVkaqj">(<a href="#ref-CqQVkaqj" role="doc-biblioref">16</a>)</span>.
Because they induce the human immune response, they are also the target of many proposed therapeutic agents <span class="citation" data-cites="njpLhBui wgAGKcBj">(<a href="#ref-wgAGKcBj" role="doc-biblioref">2</a>, <a href="#ref-njpLhBui" role="doc-biblioref">3</a>)</span>.
Viral pathogenesis is typically broken down into three major components: entry, replication, and spread <span class="citation" data-cites="jsMmdH4D">(<a href="#ref-jsMmdH4D" role="doc-biblioref">17</a>)</span>.
However, in order to draw a more complete picture of pathogenesis, it is also necessary to examine how infection manifests clinically, identify systems-level interactions between the virus and the human body, and consider the possible effects of variation or evolutionary change on pathogenesis and virulence.
Thus, clinical medicine and traditional biology are both important pieces of the puzzle of SARS-CoV-2 presentation and pathogenesis.</p>
<h2 data-number="1.4" id="coronavirus-structure-and-pathogenesis"><span class="header-section-number">1.4</span> Coronavirus Structure and Pathogenesis</h2>
<h3 data-number="1.4.1" id="structure-of-coronaviruses"><span class="header-section-number">1.4.1</span> Structure of Coronaviruses</h3>
<p>Genome structure is highly conserved among coronaviruses, meaning that the relationship between the SARS-CoV-2 genome and its pathogenesis can be inferred from prior research in related viral species.
The genomes of viruses in the <em>Nidovirales</em> order share several fundamental characteristics.
They are non-segmented, which means the viral genome is a single continuous strand of RNA, and are enveloped, which means that the genome and capsid are encased by a lipid bilayer.
Coronaviruses have large positive-sense RNA (ssRNA+) genomes ranging from 27 to 32 kilobases in length <span class="citation" data-cites="47hnjvDb 1Fqilxaum">(<a href="#ref-47hnjvDb" role="doc-biblioref">18</a>, <a href="#ref-1Fqilxaum" role="doc-biblioref">19</a>)</span>.
The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp <span class="citation" data-cites="1Fqilxaum">(<a href="#ref-1Fqilxaum" role="doc-biblioref">19</a>)</span>.
Genome organization is highly conserved within the order <span class="citation" data-cites="47hnjvDb">(<a href="#ref-47hnjvDb" role="doc-biblioref">18</a>)</span>.
There are three major genomic regions: one containing the replicase gene, one containing the genes encoding structural proteins, and interspersed accessory genes <span class="citation" data-cites="47hnjvDb">(<a href="#ref-47hnjvDb" role="doc-biblioref">18</a>)</span> (Figure <a href="#fig:genome-structure">1</a>).
The replicase gene comprises about two-thirds of the genome and consists of two open reading frames that are translated with ribosomal frameshifting <span class="citation" data-cites="47hnjvDb">(<a href="#ref-47hnjvDb" role="doc-biblioref">18</a>)</span>.
This polypeptide is then translated into 16 non-structural proteins (nsp), except in gammacoronaviruses where nsp1 is absent, that form the replication machinery used to synthesize viral RNA <span class="citation" data-cites="18GflyMj">(<a href="#ref-18GflyMj" role="doc-biblioref">20</a>)</span>.
The remaining third of the genome encodes structural proteins, including the spike (S), membrane, envelope, and nucleocapsid proteins.
Additional accessory genes are sometimes present between these two regions, depending on the species or strain.
Much attention has been focused on the S protein, which is a critical structure involved in cell entry.</p>
<div id="fig:genome-structure" class="fignos">
<figure>
<img src="images/genome-structure.png" alt="Figure 1: Structure of SARS-CoV-2 capsid and genome. A) The genomic structure of coronaviruses is highly conserved and includes three main regions. Open reading frames (ORF) 1a and 1b contain two polyproteins that encode the non-structural proteins (nsp). The nsp include enzymes such as RNA-dependent RNA Polymerase (RdRp). The last third of the genome encodes structural proteins, including the spike (S), envelope (E), membrane (M) and nucleocapsid (N) proteins. Accessory genes can also be interspersed throughout the genome (18). B) The physical structure of the coronavirus virion, including the components determined by the conserved structural proteins S, E, M and N. This figure was adapted from “Human Coronavirus Structure”, by BioRender.com (2020), retrieved from https://app.biorender.com/biorender-templates." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Structure of SARS-CoV-2 capsid and genome.</strong>
A) The genomic structure of coronaviruses is highly conserved and includes three main regions.
Open reading frames (ORF) 1a and 1b contain two polyproteins that encode the non-structural proteins (nsp).
The nsp include enzymes such as RNA-dependent RNA Polymerase (RdRp).
The last third of the genome encodes structural proteins, including the spike (S), envelope (E), membrane (M) and nucleocapsid (N) proteins.
Accessory genes can also be interspersed throughout the genome <span class="citation" data-cites="47hnjvDb">(<a href="#ref-47hnjvDb" role="doc-biblioref">18</a>)</span>.
B) The physical structure of the coronavirus virion, including the components determined by the conserved structural proteins S, E, M and N.
This figure was adapted from “Human Coronavirus Structure”, by BioRender.com (2020), retrieved from https://app.biorender.com/biorender-templates.</figcaption>
</figure>
</div>
<h3 data-number="1.4.2" id="pathogenic-mechanisms-of-coronaviruses"><span class="header-section-number">1.4.2</span> Pathogenic Mechanisms of Coronaviruses</h3>
<p>While it is possible that SARS-CoV-1 and SARS-CoV-2, like most viruses, enter cells through endocytosis, a process conserved among coronaviruses enables them to target cells for entry through fusion with the plasma membrane <span class="citation" data-cites="FFwktXgx FNetQkRI">(<a href="#ref-FFwktXgx" role="doc-biblioref">21</a>, <a href="#ref-FNetQkRI" role="doc-biblioref">22</a>)</span>.
Cell entry proceeds in three steps: binding, cleavage, and fusion.
First, the viral spike protein binds to a host cell via a recognized receptor or entry point.
Coronaviruses can bind to a range of host receptors <span class="citation" data-cites="13wWdgODZ OVsxrEuX">(<a href="#ref-13wWdgODZ" role="doc-biblioref">23</a>, <a href="#ref-OVsxrEuX" role="doc-biblioref">24</a>)</span>, with binding conserved only at the genus level <span class="citation" data-cites="17DSmRo9H">(<a href="#ref-17DSmRo9H" role="doc-biblioref">12</a>)</span>.
Viruses in the beta coronavirus genus, to which SARS-CoV-2 belongs, are known to bind to the CEACAM1 protein, 5-N-acetyl-9-O-acetyl neuraminic acid, and to angiotensin-converting enzyme 2 (ACE2) <span class="citation" data-cites="13wWdgODZ">(<a href="#ref-13wWdgODZ" role="doc-biblioref">23</a>)</span>.
This recognition is driven by domains in the S1 subunit <span class="citation" data-cites="qcVbT0w4">(<a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>)</span>.
SARS-CoV-2 has a high affinity for human ACE2, which is expressed in the vascular epithelium, other epithelial cells, and cardiovascular and renal tissues <span class="citation" data-cites="So6ceUwB NodtSApl">(<a href="#ref-So6ceUwB" role="doc-biblioref">26</a>, <a href="#ref-NodtSApl" role="doc-biblioref">27</a>)</span>, as well as many others <span class="citation" data-cites="N1UXcG4j">(<a href="#ref-N1UXcG4j" role="doc-biblioref">28</a>)</span>.
The binding process is guided by the molecular structure of the spike protein, which is structured in three segments: an ectodomain, a transmembrane anchor, and an intracellular tail <span class="citation" data-cites="skHKTFMe">(<a href="#ref-skHKTFMe" role="doc-biblioref">29</a>)</span>.
The ectodomain forms the crown-like structures on the viral membrane and contains two subdomains known as the S1 and S2 subunits <span class="citation" data-cites="ueMPuUMe">(<a href="#ref-ueMPuUMe" role="doc-biblioref">30</a>)</span>.
The S1 (N-terminal) domain forms the head of the crown and contains the receptor binding motif, and the S2 (C-terminal) domain forms the stalk that supports the head <span class="citation" data-cites="ueMPuUMe">(<a href="#ref-ueMPuUMe" role="doc-biblioref">30</a>)</span>.
The S1 subunit guides the binding of the virus to the host cell, and the S2 subunit guides the fusion process <span class="citation" data-cites="skHKTFMe">(<a href="#ref-skHKTFMe" role="doc-biblioref">29</a>)</span>.</p>
<p>After the binding of the S1 subunit to an entry point, the spike protein of coronaviruses is often cleaved at the S1/S2 boundary into the S1 and S2 subunits by a host protease <span class="citation" data-cites="qcVbT0w4 kCVy8vjj YeKaMzdJ">(<a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>, <a href="#ref-kCVy8vjj" role="doc-biblioref">31</a>, <a href="#ref-YeKaMzdJ" role="doc-biblioref">32</a>)</span>.
This proteolytic priming is important because it prepares the S protein for fusion <span class="citation" data-cites="kCVy8vjj YeKaMzdJ">(<a href="#ref-kCVy8vjj" role="doc-biblioref">31</a>, <a href="#ref-YeKaMzdJ" role="doc-biblioref">32</a>)</span>.
The two subunits remain bound by van der Waals forces, with the S1 subunit stabilizing the S2 subunit throughout the membrane fusion process <span class="citation" data-cites="qcVbT0w4">(<a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>)</span>.
Cleavage at a second site within S2 (S2’) activates <em>S</em> for fusion by inducing conformational changes <span class="citation" data-cites="qcVbT0w4">(<a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>)</span>.
Similar to SARS-CoV-1, SARS-CoV-2 exhibits redundancy in which host proteases can cleave the S protein <span class="citation" data-cites="15l3di3Wj">(<a href="#ref-15l3di3Wj" role="doc-biblioref">33</a>)</span>.
Both transmembrane protease serine protease-2 (TMPRSS-2) and cathepsins B/L have been shown to mediate SARS-CoV-2 S protein proteolytic priming, and small molecule inhibition of these enzymes fully inhibited viral entry <em>in vitro</em> <span class="citation" data-cites="15l3di3Wj JOJ2n3gC">(<a href="#ref-15l3di3Wj" role="doc-biblioref">33</a>, <a href="#ref-JOJ2n3gC" role="doc-biblioref">34</a>)</span>.
Other proteases known to cleave the S1/S2 boundary in coronaviruses include TMPRSS-4, trypsin, furin, cathepsins, and human airway trypsin-like protease (HAT) <span class="citation" data-cites="JOJ2n3gC">(<a href="#ref-JOJ2n3gC" role="doc-biblioref">34</a>)</span>.</p>
<p>Unlike in SARS-CoV-1, a second cleavage site featuring a furin-like binding motif is also present near the S1/S2 boundary in SARS-CoV-2 <span class="citation" data-cites="7bVyHTLM">(<a href="#ref-7bVyHTLM" role="doc-biblioref">35</a>)</span>.
This site is found in HCoV belonging to the A and C lineages of beta coronavirus, including MERS-CoV, but not in the other known members of the B lineage of beta coronavirus that contains SARS-CoV-1 and SARS-CoV-2 <span class="citation" data-cites="7bVyHTLM">(<a href="#ref-7bVyHTLM" role="doc-biblioref">35</a>)</span>.
It is associated with increased virulence in other viral species <span class="citation" data-cites="7bVyHTLM">(<a href="#ref-7bVyHTLM" role="doc-biblioref">35</a>)</span> and may facilitate membrane fusion of SARS-CoV-2 in the absence of other proteases that prime the S1/S2 site <span class="citation" data-cites="gZRMibZh">(<a href="#ref-gZRMibZh" role="doc-biblioref">36</a>)</span>.
However, given that proteases such as HAT are likely to be present in targets like the human airway, the extent to which this site has had a real-world effect on the spread of SARS-CoV-2 was initially unclear <span class="citation" data-cites="gZRMibZh">(<a href="#ref-gZRMibZh" role="doc-biblioref">36</a>)</span>.
Subsequent research has supported this site as an important contributor to pathogenesis: <em>in vitro</em> analyses have reported that it bolsters pathogenicity specifically in cell lines derived from human airway cells (Calu3 cell line) <span class="citation" data-cites="f27ylIZM AP9eAy68 AfjDQPDh">(<a href="#ref-f27ylIZM" role="doc-biblioref">37</a>–<a href="#ref-AfjDQPDh" role="doc-biblioref">39</a>)</span> and that furin inhibitors reduced pathogenic effects in VeroE6 cells <span class="citation" data-cites="FcRgnv0z">(<a href="#ref-FcRgnv0z" role="doc-biblioref">40</a>)</span>.</p>
<p>Electron microscopy suggests that in some coronaviruses, including SARS-CoV-1 and MERS-CoV, a six-helix bundle separates the two subunits in the postfusion conformation, and the unusual length of this bundle facilitates membrane fusion through the release of additional energy <span class="citation" data-cites="17DSmRo9H">(<a href="#ref-17DSmRo9H" role="doc-biblioref">12</a>)</span>.
The viral membrane can then fuse with the endosomal membrane to release the viral genome into the host cytoplasm.
Once the virus enters a host cell, the replicase gene is translated and assembled into the viral replicase complex.
This complex then synthesizes the double-stranded RNA (dsRNA) genome from the genomic ssRNA(+).
The dsRNA genome is transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes <span class="citation" data-cites="47hnjvDb 1F46J8hWx">(<a href="#ref-47hnjvDb" role="doc-biblioref">18</a>, <a href="#ref-1F46J8hWx" role="doc-biblioref">41</a>)</span>.
From there, the virus can spread into other cells.
In SARS-CoV-2, the insertion of the furin-like binding site near the S1/S2 boundary is also thought to increase cell-cell adhesion, making it possible for the viral genome to spread directly from cell to cell rather than needing to propagate the virion itself <span class="citation" data-cites="5TQeBfZv">(<a href="#ref-5TQeBfZv" role="doc-biblioref">42</a>)</span>.
In this way, the genome of SARS-CoV-2 provides insight into the pathogenic behavior of the virus.</p>
<p>Evidence also suggests that SARS-CoV-2 may take advantage of the specific structure of endothelial cells to enter the circulatory system.
Endothelial cells are specialized epithelial cells <span class="citation" data-cites="17EgPSnfR">(<a href="#ref-17EgPSnfR" role="doc-biblioref">43</a>)</span> that form a barrier between the bloodstream and surrounding tissues.
The endothelium facilitates nutrient, oxygen, and cellular exchange between the blood and vascularized tissues <span class="citation" data-cites="zvVjhfmD">(<a href="#ref-zvVjhfmD" role="doc-biblioref">44</a>)</span>.
The luminal (interior) surface of the endothelium is lined with glycocalyx, a network of both membrane-bound and soluble proteins and carbohydrates, primarily proteoglycans and glycoproteins <span class="citation" data-cites="14dUo0d2a 11mrQ55Mt">(<a href="#ref-14dUo0d2a" role="doc-biblioref">45</a>, <a href="#ref-11mrQ55Mt" role="doc-biblioref">46</a>)</span>.
The glycocalyx varies in thickness from 0.5 microns in the capillaries to 4.5 microns in the carotid arteries and forms a meshwork that localizes both endothelial- and plasma-derived signals to the inner vessel wall <span class="citation" data-cites="14dUo0d2a">(<a href="#ref-14dUo0d2a" role="doc-biblioref">45</a>)</span>.
Heparan sulfate is the dominant proteoglycan in the glycocalyx, representing 50-90% of glycocalyx proteoglycan content <span class="citation" data-cites="muOuyY5D">(<a href="#ref-muOuyY5D" role="doc-biblioref">47</a>)</span>.
The SARS-CoV-2 spike protein can bind directly to heparan sulfate, which serves in part as a scaffolding molecule to facilitate ACE2 binding and entry into endothelial cells <span class="citation" data-cites="11mrQ55Mt">(<a href="#ref-11mrQ55Mt" role="doc-biblioref">46</a>)</span>.
A heparan sulfate binding site has also been identified near the ACE2 binding site on the viral receptor binding domain (RBD), and modeling has suggested that heparan sulfate binding yields an open conformation that facilitates binding to ACE2 on the cell surface <span class="citation" data-cites="11mrQ55Mt">(<a href="#ref-11mrQ55Mt" role="doc-biblioref">46</a>)</span>.
Degrading or removing heparan sulfate was associated with decreased binding <span class="citation" data-cites="11mrQ55Mt">(<a href="#ref-11mrQ55Mt" role="doc-biblioref">46</a>)</span>.
Heparan sulfate may also interact with the S1/S2 proteolytic cleavage site and other binding sites to promote binding affinity <span class="citation" data-cites="eC61y3gI">(<a href="#ref-eC61y3gI" role="doc-biblioref">48</a>)</span>.
Notably, treatment with soluble heparan sulfate or even heparin (a commonly used anti-coagulant and vasodilator that is similar in structure to heparan sulfate <span class="citation" data-cites="JEtIJaPO">(<a href="#ref-JEtIJaPO" role="doc-biblioref">49</a>)</span>) potently blocked spike protein binding and viral infection <span class="citation" data-cites="11mrQ55Mt">(<a href="#ref-11mrQ55Mt" role="doc-biblioref">46</a>)</span>.
This finding is particularly interesting because degradation of heparan sulfate in the glycocalyx has previously been identified as an important contributor to ARDS and sepsis <span class="citation" data-cites="ZVyaYX2h">(<a href="#ref-ZVyaYX2h" role="doc-biblioref">50</a>)</span>, two common and severe outcomes of COVID-19, and suggests that heparan sulfate could be a target for pharmaceutical inhibition of cell entry by SARS-CoV-2 <span class="citation" data-cites="1FNCN2CC4 1C7D3ypZY iucR6FeL 1FKr0hF0S qvv0s5BF">(<a href="#ref-1FNCN2CC4" role="doc-biblioref">51</a>–<a href="#ref-qvv0s5BF" role="doc-biblioref">55</a>)</span>.
Together, this evidence suggests that heparan sulfate can serve as an important adhesion molecule for SARS-CoV-2 cell entry.
It may represent a therapeutic target but has not been pursued as much as other candidate targets <span class="citation" data-cites="njpLhBui">(<a href="#ref-njpLhBui" role="doc-biblioref">3</a>)</span>.</p>
<h3 data-number="1.4.3" id="immune-evasion-strategies"><span class="header-section-number">1.4.3</span> Immune Evasion Strategies</h3>
<p>Research in other HCoV provides some indication of how SARS-CoV-2 infection can proceed despite human immune defenses.
Infecting the epithelium can help viruses such as SARS-CoV-1 bypass the physical barriers, such as mucus, that comprise the immune system’s first line of defense <span class="citation" data-cites="1XbF4GLn">(<a href="#ref-1XbF4GLn" role="doc-biblioref">56</a>)</span>.
Once the virus infiltrates host cells, it is adept at evading detection.
CD163+ and CD68+ macrophage cells are especially crucial for the establishment of SARS-CoV-1 in the body <span class="citation" data-cites="1XbF4GLn">(<a href="#ref-1XbF4GLn" role="doc-biblioref">56</a>)</span>.
These cells most likely serve as viral reservoirs that help shield SARS-CoV-1 from the innate immune response.
According to a study on the viral dissemination of SARS-CoV-1 in Chinese macaques, viral RNA could be detected in some monocytes throughout the process of differentiation into dendritic cells <span class="citation" data-cites="1XbF4GLn">(<a href="#ref-1XbF4GLn" role="doc-biblioref">56</a>)</span>.
This lack of active viral replication allows SARS-CoV-1 to escape the innate immune response because reduced levels of detectable viral RNA allow the virus to avoid both natural killer cells and Toll-like receptors <span class="citation" data-cites="1XbF4GLn">(<a href="#ref-1XbF4GLn" role="doc-biblioref">56</a>)</span>.
Even during replication, SARS-CoV-1 is able to mask its dsRNA genome from detection by the immune system.
Although dsRNA is a pathogen-associated molecular pattern that would typically initiate a response from the innate immune system <span class="citation" data-cites="159GOVLV1">(<a href="#ref-159GOVLV1" role="doc-biblioref">57</a>)</span>, <em>in vitro</em> analysis of nidoviruses including SARS-CoV-1 suggests that these viruses can induce the development of double-membrane vesicles that protect the dsRNA signature from being detected by the host immune system <span class="citation" data-cites="yQO4HFTZ">(<a href="#ref-yQO4HFTZ" role="doc-biblioref">58</a>)</span>.
This protective envelope can therefore insulate these coronaviruses from the innate immune system’s detection mechanism <span class="citation" data-cites="OlEbK4fc">(<a href="#ref-OlEbK4fc" role="doc-biblioref">59</a>)</span>.</p>
<p>HCoVs are also known to interfere with the host immune response, rather than just evade it.
For example, the virulence of SARS-CoV-2 is increased by nsp1, which can suppress host gene expression by stalling mRNA translation and inducing endonucleolytic cleavage and mRNA degradation <span class="citation" data-cites="Q5Q9k6Nc">(<a href="#ref-Q5Q9k6Nc" role="doc-biblioref">60</a>)</span>.
SARS-CoV-1 also evades the immune response by interfering with type I IFN induction signaling, which is a mechanism that leads to cellular resistance to viral infections.
SARS-CoV-1 employs methods such as ubiquitination and degradation of RNA sensor adaptor molecules MAVS and TRAF3/6 <span class="citation" data-cites="G6A0wWdn">(<a href="#ref-G6A0wWdn" role="doc-biblioref">61</a>)</span>.
Also, MERS-CoV downregulates antigen presentation via MHC class I and MHC class II, which leads to a reduction in T cell activation <span class="citation" data-cites="G6A0wWdn">(<a href="#ref-G6A0wWdn" role="doc-biblioref">61</a>)</span>.
These evasion mechanisms, in turn, may facilitate systemic infection.
Coronaviruses such as SARS-CoV-1 are also able to evade the humoral immune response through other mechanisms, such as inhibiting certain cytokine pathways or down-regulating antigen presentation by the cells <span class="citation" data-cites="yQO4HFTZ">(<a href="#ref-yQO4HFTZ" role="doc-biblioref">58</a>)</span>.</p>
<h3 data-number="1.4.4" id="host-cell-susceptibility"><span class="header-section-number">1.4.4</span> Host Cell Susceptibility</h3>
<p>ACE2 and TMPRSS-2 have been identified as the primary entry portal and as a critical protease, respectively, in facilitating the entry of SARS-CoV-1 and SARS-CoV-2 into a target cell <span class="citation" data-cites="2QTH37Xi CqQVkaqj MXWaV7SO 15l3di3Wj JjrO6YbD">(<a href="#ref-CqQVkaqj" role="doc-biblioref">16</a>, <a href="#ref-15l3di3Wj" role="doc-biblioref">33</a>, <a href="#ref-2QTH37Xi" role="doc-biblioref">62</a>–<a href="#ref-JjrO6YbD" role="doc-biblioref">64</a>)</span>.
This finding has led to a hypothesized role for the expression of these molecules in determining which cells, tissues, and organs are most susceptible to SARS-CoV-2 infection.
ACE2 is expressed in numerous organs, such as the heart, kidney, and intestine, but it is most prominently expressed in alveolar epithelial cells; this pattern of expression is expected to contribute to the virus’ association with lung pathology <span class="citation" data-cites="9qsndPCu So6ceUwB 75IAJRbI">(<a href="#ref-So6ceUwB" role="doc-biblioref">26</a>, <a href="#ref-9qsndPCu" role="doc-biblioref">65</a>, <a href="#ref-75IAJRbI" role="doc-biblioref">66</a>)</span> as well as that of SARS <span class="citation" data-cites="1FE1YGsaV">(<a href="#ref-1FE1YGsaV" role="doc-biblioref">67</a>)</span>.
A retrospective observational study reported indirect evidence that certain antineoplastic therapies, such as the chemotherapy drug gemcitabine, may reduce risk of SARS-CoV-2 infection in patients with cancer, possibly via decreased ACE2 expression <span class="citation" data-cites="4zAgRRxf">(<a href="#ref-4zAgRRxf" role="doc-biblioref">68</a>)</span>.
Additionally, the addition of the furin site insertion at the S1/S2 boundary means that SARS-CoV-2 does not require TMPRSS-2 when furin, an ubiquitously expressed endoprotease <span class="citation" data-cites="1C7ggDgVA">(<a href="#ref-1C7ggDgVA" role="doc-biblioref">69</a>)</span>, is present, enabling cell-cell fusion independent of TMPRSS-2 availability <span class="citation" data-cites="bkTglQpW">(<a href="#ref-bkTglQpW" role="doc-biblioref">70</a>)</span>.</p>
<p>Clinical investigations of COVID-19 patients have detected SARS-CoV-2 transcripts in bronchoalveolar lavage fluid (BALF) (93% of specimens), sputum (72%), nasal swabs (63%), fibrobronchoscopy brush biopsies (46%), pharyngeal swabs (32%), feces (29%), and blood (1%) <span class="citation" data-cites="vr7AH83b">(<a href="#ref-vr7AH83b" role="doc-biblioref">71</a>)</span>.
Two studies reported that SARS-CoV-2 could not be detected in urine specimens <span class="citation" data-cites="vr7AH83b azgJqujy">(<a href="#ref-vr7AH83b" role="doc-biblioref">71</a>, <a href="#ref-azgJqujy" role="doc-biblioref">72</a>)</span>; however, a third study identified four urine samples (out of 58) that were positive for SARS-CoV-2 nucleic acids <span class="citation" data-cites="F7nSMvZk">(<a href="#ref-F7nSMvZk" role="doc-biblioref">73</a>)</span>.
Although respiratory failure remains the leading cause of death for COVID-19 patients <span class="citation" data-cites="1AJFJxzmJ">(<a href="#ref-1AJFJxzmJ" role="doc-biblioref">74</a>)</span>, SARS-CoV-2 infection can damage many other organ systems including the heart <span class="citation" data-cites="5gnCuPzp">(<a href="#ref-5gnCuPzp" role="doc-biblioref">75</a>)</span>, kidneys <span class="citation" data-cites="5ET1D3cK w5B6qRKv">(<a href="#ref-5ET1D3cK" role="doc-biblioref">76</a>, <a href="#ref-w5B6qRKv" role="doc-biblioref">77</a>)</span>, liver <span class="citation" data-cites="TTSFlLVC">(<a href="#ref-TTSFlLVC" role="doc-biblioref">78</a>)</span>, and gastrointestinal tract <span class="citation" data-cites="PmE9xedP 3Ak4Mata">(<a href="#ref-PmE9xedP" role="doc-biblioref">79</a>, <a href="#ref-3Ak4Mata" role="doc-biblioref">80</a>)</span>.
As it becomes clear that SARS-CoV-2 infection can damage multiple organs, the scientific community is pursuing multiple avenues of investigation in order to build a consensus about how the virus affects the human body.</p>
<h2 data-number="1.5" id="clinical-presentation-of-covid-19"><span class="header-section-number">1.5</span> Clinical Presentation of COVID-19</h2>
<p>SARS-CoV-2 pathogenesis is closely linked with the clinical presentation of the COVID-19 disease.
Reports have described diverse symptom profiles associated with COVID-19, with a great deal of variability both within and between institutions and regions.
Definitions for non-severe, severe, and critical COVID-19, along with treatment recommendations, are available from the World Health Organization living guidelines <span class="citation" data-cites="17uZcQTT">(<a href="#ref-17uZcQTT" role="doc-biblioref">81</a>)</span>.
A large study from Wuhan, China conducted early in the pandemic identified fever and cough as the two most common symptoms that patients reported at hospital admission <span class="citation" data-cites="15IDA5kX6">(<a href="#ref-15IDA5kX6" role="doc-biblioref">82</a>)</span>, while a retrospective study in China described the clinical presentations of patients infected with SARS-CoV-2 as including lower respiratory tract infection with fever, dry cough, and dyspnea (shortness of breath) <span class="citation" data-cites="10THxyeCg">(<a href="#ref-10THxyeCg" role="doc-biblioref">83</a>)</span>.
This study <span class="citation" data-cites="10THxyeCg">(<a href="#ref-10THxyeCg" role="doc-biblioref">83</a>)</span> noted that upper respiratory tract symptoms were less common, suggesting that the virus preferentially targets cells located in the lower respiratory tract.
However, data from the New York City region <span class="citation" data-cites="19ytsiSpq 1H8gSsSkn">(<a href="#ref-19ytsiSpq" role="doc-biblioref">84</a>, <a href="#ref-1H8gSsSkn" role="doc-biblioref">85</a>)</span> showed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across individuals.
For example, even within New York City, one study <span class="citation" data-cites="19ytsiSpq">(<a href="#ref-19ytsiSpq" role="doc-biblioref">84</a>)</span> identified low oxygen saturation (&lt;90% without the use of supplemental oxygen or ventilation support) in 20.4% of patients upon presentation, with fever being present in 30.7%, while another study <span class="citation" data-cites="1H8gSsSkn">(<a href="#ref-1H8gSsSkn" role="doc-biblioref">85</a>)</span> reported cough (79.4%), fever (77.1%), and dyspnea (56.5%) as the most common presenting symptoms; both of these studies considered only hospitalized patients.
A later study reported radiographic findings such as ground-glass opacity and bilateral patchy shadowing in the lungs of many hospitalized patients, with most COVID-19 patients having lymphocytopenia, or low levels of lymphocytes (a type of white blood cell) <span class="citation" data-cites="15IDA5kX6">(<a href="#ref-15IDA5kX6" role="doc-biblioref">82</a>)</span>.
Patients may also experience loss of smell, myalgias (muscle aches), fatigue, or headache.
Gastrointestinal symptoms can also present <span class="citation" data-cites="18tT1tLJI">(<a href="#ref-18tT1tLJI" role="doc-biblioref">86</a>)</span>, and the CDC includes nausea and vomiting, as well as congestion and runny nose, on its list of symptoms consistent with COVID-19 <span class="citation" data-cites="gHWlMufv">(<a href="#ref-gHWlMufv" role="doc-biblioref">8</a>)</span>.
An analysis of an app-based survey of 500,000 individuals in the U.S. found that among those tested for SARS-CoV-2, a loss of taste or smell, fever, and a cough were significant predictors of a positive test result <span class="citation" data-cites="5tU6buUE">(<a href="#ref-5tU6buUE" role="doc-biblioref">87</a>)</span>.
It is important to note that in this study, the predictive value of symptoms may be underestimated if they are not specific to COVID-19.
This underestimation could occur because the outcome measured was a positive, as opposed to a negative, COVID-19 test result, meaning an association would be more easily identified for symptoms that were primarily or exclusively found with COVID-19.
At the time the surveys were conducted, due to limits in U.S. testing infrastructure, respondents typically needed to have some symptoms known to be specific to COVID-19 in order to qualify for testing.
Widespread testing of asymptomatic individuals may therefore provide additional insight into the range of symptoms associated with COVID-19.</p>
<p>Consistent with the wide range of symptoms observed and the pathogenic mechanisms described above, COVID-19 can affect a variety of systems within the body in addition to causing respiratory problems <span class="citation" data-cites="wK2afyL8">(<a href="#ref-wK2afyL8" role="doc-biblioref">88</a>)</span>.
For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions <span class="citation" data-cites="17hdoiBo9">(<a href="#ref-17hdoiBo9" role="doc-biblioref">89</a>)</span>.
Some patients are at risk for collapsing glomerulopathy <span class="citation" data-cites="9K9lIU2m">(<a href="#ref-9K9lIU2m" role="doc-biblioref">90</a>)</span>.</p>
<p>COVID-19 can also cause neurological complications <span class="citation" data-cites="1CqoZGSKK 1EF9EpbZ5 1F7Vk1wO9">(<a href="#ref-1CqoZGSKK" role="doc-biblioref">91</a>–<a href="#ref-1F7Vk1wO9" role="doc-biblioref">93</a>)</span>, potentially including stroke, seizures or meningitis <span class="citation" data-cites="nBGGUV9s LLm9d62d">(<a href="#ref-nBGGUV9s" role="doc-biblioref">94</a>, <a href="#ref-LLm9d62d" role="doc-biblioref">95</a>)</span>.
One study on autopsy samples suggested that SARS-CoV-2 may be able to enter the central nervous system via the neural–mucosal interface <span class="citation" data-cites="17nNkDj8p">(<a href="#ref-17nNkDj8p" role="doc-biblioref">96</a>)</span>.
However, a study of 41 autopsied brains <span class="citation" data-cites="R1CIWZPb">(<a href="#ref-R1CIWZPb" role="doc-biblioref">97</a>)</span> found no evidence that the virus can actually infect the central nervous system.
Although there was viral RNA in some brain samples, it was only found in very small amounts, and no viral protein was found.
The RNA may have been in the blood vessels or blood components and not in the brain tissue itself.
Instead, the neuropathological effects of COVID-19 are more likely to be caused indirectly by hypoxia, coagulopathy, or inflammatory processes rather than by infection in the brain <span class="citation" data-cites="R1CIWZPb">(<a href="#ref-R1CIWZPb" role="doc-biblioref">97</a>)</span>.
COVID-19 has been associated with an increased incidence of large vessel stroke, particularly in patients under the age of 40 <span class="citation" data-cites="aRNqjkMg">(<a href="#ref-aRNqjkMg" role="doc-biblioref">98</a>)</span>, and other thrombotic events including pulmonary embolism and deep vein thrombosis <span class="citation" data-cites="3UBbDrG2">(<a href="#ref-3UBbDrG2" role="doc-biblioref">99</a>)</span>.
The mechanism behind these complications has been suggested to be related to coagulopathy, with reports indicating the presence of antiphospholipid antibodies <span class="citation" data-cites="8OnbWuhF">(<a href="#ref-8OnbWuhF" role="doc-biblioref">100</a>)</span> and elevated levels of d-dimer and fibrinogen degradation products in deceased patients <span class="citation" data-cites="dUfws1q0">(<a href="#ref-dUfws1q0" role="doc-biblioref">101</a>)</span>.
Other viral infections have been associated with coagulation defects and changes to the coagulation cascade; notably, SARS was also found to lead to disseminated intravascular coagulation and was associated with both pulmonary embolism and deep vein thrombosis <span class="citation" data-cites="p5EhN9Qd">(<a href="#ref-p5EhN9Qd" role="doc-biblioref">102</a>)</span>.
The mechanism behind these insults has been suggested to be related to inflammation-induced increases in the von Willebrand factor clotting protein, leading to a pro-coagulative state <span class="citation" data-cites="p5EhN9Qd">(<a href="#ref-p5EhN9Qd" role="doc-biblioref">102</a>)</span>.
Abnormal clotting (thromboinflammation or coagulopathy) has been increasingly discussed recently as a possible key mechanism in many cases of severe COVID-19, and may be associated with the high d-dimer levels often observed in severe cases <span class="citation" data-cites="v1EIzwfx 18AiyvhO8 PyKMLraw">(<a href="#ref-v1EIzwfx" role="doc-biblioref">103</a>–<a href="#ref-PyKMLraw" role="doc-biblioref">105</a>)</span>.
This excessive clotting in lung capillaries has been suggested to be related to a dysregulated activation of the complement system, part of the innate immune system <span class="citation" data-cites="14dyYQY7s 17y6YeJ6R">(<a href="#ref-14dyYQY7s" role="doc-biblioref">106</a>, <a href="#ref-17y6YeJ6R" role="doc-biblioref">107</a>)</span>.</p>
<p>Finally, concerns have been raised about long-term sequelae of COVID-19.
Some COVID-19 patients have reported that various somatic symptoms (such as shortness of breath, fatigue, chest pain) and psychological (depression, anxiety or mild cognitive impairment) symptoms can last for months after infection <span class="citation" data-cites="2uD7bbjv">(<a href="#ref-2uD7bbjv" role="doc-biblioref">108</a>)</span>.
Such long-term affects occur both in adults <span class="citation" data-cites="Exnc0zrA">(<a href="#ref-Exnc0zrA" role="doc-biblioref">109</a>)</span> and children <span class="citation" data-cites="Tn5MVYGS">(<a href="#ref-Tn5MVYGS" role="doc-biblioref">110</a>)</span>.
Sustained symptoms affecting a variety of biological systems have been reported across many studies (e.g., <span class="citation" data-cites="2uD7bbjv X0iTHBZ7 xgn7ugdb">(<a href="#ref-2uD7bbjv" role="doc-biblioref">108</a>, <a href="#ref-X0iTHBZ7" role="doc-biblioref">111</a>, <a href="#ref-xgn7ugdb" role="doc-biblioref">112</a>)</span>).
The phenomenon of “long COVID” is not fully understood although various possible explanations have been proposed, including damage caused by immune response to infection as well as by the infection itself, in addition to negative consequences of the experience of lengthy illness and hospitalization.
However, a lack of consistency among definitions used in different studies makes it difficult to develop precise definitions or identify specific symptoms associated with long-term effects of COVID-19 <span class="citation" data-cites="Rk8iz9Or 13fa6xusy">(<a href="#ref-Rk8iz9Or" role="doc-biblioref">113</a>, <a href="#ref-13fa6xusy" role="doc-biblioref">114</a>)</span>.
Patient and family support groups for “long haulers” have been formed online, and patient-driven efforts to collect data about post-acute COVID-19 provide valuable sources of information (e.g., <span class="citation" data-cites="X0iTHBZ7">(<a href="#ref-X0iTHBZ7" role="doc-biblioref">111</a>)</span>).
The specific relationship between viral pathogenesis and these reported sequelae remains to be uncovered, however.</p>
<h3 data-number="1.5.1" id="pediatric-presentation"><span class="header-section-number">1.5.1</span> Pediatric Presentation</h3>
<p>The presentation of COVID-19 infection can vary greatly among pediatric patients and, in some cases, manifests in distinct ways from COVID-19 in adults.
Evidence suggests that children and adolescents tend to have mostly asymptomatic infections and that those who are symptomatic typically exhibit mild illness <span class="citation" data-cites="g435TGYc SHtceaPy 8sVs0pB8 L81Cb2ZZ">(<a href="#ref-g435TGYc" role="doc-biblioref">115</a>–<a href="#ref-L81Cb2ZZ" role="doc-biblioref">118</a>)</span>.
One review examined symptoms reported in 17 studies of children infected with COVID-19 during the early months of the COVID-19 epidemic in China and one study from Singapore <span class="citation" data-cites="1BhN2uq3f">(<a href="#ref-1BhN2uq3f" role="doc-biblioref">119</a>)</span>.
In the more than a thousand cases described, the most common reports were for mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose, while three children were reported to be asymptomatic.
Severe lower respiratory infection was described in only one of the pediatric cases reviewed.
Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
Radiologic findings were not always reported in the case studies reviewed, but when they were mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions <span class="citation" data-cites="1BhN2uq3f">(<a href="#ref-1BhN2uq3f" role="doc-biblioref">119</a>)</span>.
Neurological symptoms have also been reported <span class="citation" data-cites="12lVfu2Qe">(<a href="#ref-12lVfu2Qe" role="doc-biblioref">120</a>)</span>.</p>
<p>These analyses indicate that most pediatric cases of COVID-19 are not severe.
Indeed, it is estimated that less than 1% of pediatric cases result in critical illness <span class="citation" data-cites="ShMJKFvG 8sVs0pB8">(<a href="#ref-8sVs0pB8" role="doc-biblioref">117</a>, <a href="#ref-ShMJKFvG" role="doc-biblioref">121</a>)</span>, although reporting suggests that pediatric hospitalizations may be greater with the emergence of the Delta variant of concern (VOC) <span class="citation" data-cites="1Bv67ENp2 NECDdfMZ t4bNlJL9">(<a href="#ref-1Bv67ENp2" role="doc-biblioref">122</a>–<a href="#ref-t4bNlJL9" role="doc-biblioref">124</a>)</span>.
Serious complications and, in relatively rare cases, deaths have occurred <span class="citation" data-cites="MCVe25tf">(<a href="#ref-MCVe25tf" role="doc-biblioref">125</a>)</span>.
Of particular interest, children have occasionally experienced a serious inflammatory syndrome, multisystem inflammatory syndrome in children (MIS-C), following COVID-19 infection <span class="citation" data-cites="jX2iYihE">(<a href="#ref-jX2iYihE" role="doc-biblioref">126</a>)</span>.
This syndrome is similar in some respects to Kawasaki disease, including Kawasaki disease shock syndrome <span class="citation" data-cites="fZwrv8M AmvwCWm3 68JorBeU">(<a href="#ref-fZwrv8M" role="doc-biblioref">127</a>–<a href="#ref-68JorBeU" role="doc-biblioref">129</a>)</span>, and is thought to be a distinct clinical manifestation of SARS-CoV-2 due to its distinct cytokine profile and the presence of burr cells in peripheral blood smears <span class="citation" data-cites="10gSXfOKm 10QhaDu6N">(<a href="#ref-10gSXfOKm" role="doc-biblioref">130</a>, <a href="#ref-10QhaDu6N" role="doc-biblioref">131</a>)</span>.
MIS-C has been associated with heart failure in some cases <span class="citation" data-cites="FYgN6gUL">(<a href="#ref-FYgN6gUL" role="doc-biblioref">132</a>)</span>.
A small number of case studies have identified presentations similar to MIS-C in adults associated with SARS-CoV-2 <span class="citation" data-cites="71To4laE xQGxETbC UBDAxWWv xGHzszJs">(<a href="#ref-71To4laE" role="doc-biblioref">133</a>–<a href="#ref-xGHzszJs" role="doc-biblioref">136</a>)</span>.
However, not all cases of severe COVID-19 in children are characterizable as MIS-C.
A recent study <span class="citation" data-cites="YMYsvZSF">(<a href="#ref-YMYsvZSF" role="doc-biblioref">137</a>)</span> described demographic and clinical variables associated with MIS-C in comparison with non-MIS-C severe acute COVID-19 in young people in the United States.
Efforts to characterize long-term sequelae of SARS-CoV-2 infection in children face the same challenges as in adults, but long-term effects remain a concern in pediatric patients <span class="citation" data-cites="QaIzV00i Tn5MVYGS vqRGmUeQ">(<a href="#ref-Tn5MVYGS" role="doc-biblioref">110</a>, <a href="#ref-QaIzV00i" role="doc-biblioref">138</a>, <a href="#ref-vqRGmUeQ" role="doc-biblioref">139</a>)</span>, although some early studies have suggested that they may be less of a concern than in adults <span class="citation" data-cites="jHAr2gFf 14Z8mBGyq 1HtytUH2X">(<a href="#ref-jHAr2gFf" role="doc-biblioref">140</a>–<a href="#ref-1HtytUH2X" role="doc-biblioref">142</a>)</span>.
Research is ongoing into the differences between the pediatric and adult immune responses to SARS-CoV-2, and future research may shed light on the factors that lead to MIS-C; it is also unknown whether the relative advantages of children against severe COVID-19 will remain in the face of current and future variants <span class="citation" data-cites="7X38RBUl">(<a href="#ref-7X38RBUl" role="doc-biblioref">143</a>)</span>.</p>
<h3 data-number="1.5.2" id="cytokine-release-syndrome"><span class="header-section-number">1.5.2</span> Cytokine Release Syndrome</h3>
<p>The inflammatory response was identified early on as a potential driver of COVID-19 outcomes due to existing research in SARS and emerging research in COVID-19.
While too low of an inflammatory response is a concern because it will fail to eliminate the immune threat <span class="citation" data-cites="16CYY7vzG">(<a href="#ref-16CYY7vzG" role="doc-biblioref">144</a>)</span>, excessive pro-inflammatory cytokine activity can cascade <span class="citation" data-cites="xuMYmc7W">(<a href="#ref-xuMYmc7W" role="doc-biblioref">145</a>)</span> and cause cell damage, among other problems <span class="citation" data-cites="sXusUkLI">(<a href="#ref-sXusUkLI" role="doc-biblioref">146</a>)</span>.
A dysregulated immune response can cause significant damage to the host <span class="citation" data-cites="o6BQnEt7 b1QVdemU 1GnFL9zeN">(<a href="#ref-o6BQnEt7" role="doc-biblioref">147</a>–<a href="#ref-1GnFL9zeN" role="doc-biblioref">149</a>)</span> including pathogenesis associated with sepsis.
Sepsis, which can lead to multi-organ failure and death <span class="citation" data-cites="bZMKqj6e 6eWgCrWj">(<a href="#ref-bZMKqj6e" role="doc-biblioref">150</a>, <a href="#ref-6eWgCrWj" role="doc-biblioref">151</a>)</span>, is traditionally associated with bacterial infections.
However, sepsis associated with viral infections may be underidentified <span class="citation" data-cites="jqm9b6lZ">(<a href="#ref-jqm9b6lZ" role="doc-biblioref">152</a>)</span>, and sepsis has emerged as a major concern associated with SARS-CoV-2 infection <span class="citation" data-cites="sLCBjlMH">(<a href="#ref-sLCBjlMH" role="doc-biblioref">153</a>)</span>.
Hyperactivity of the pro-inflammatory response due to lung infection is commonly associated with acute lung injury and more rarely with the more severe manifestation, ARDS, which can arise from pneumonia, SARS, and COVID-19 <span class="citation" data-cites="xuMYmc7W bZMKqj6e">(<a href="#ref-xuMYmc7W" role="doc-biblioref">145</a>, <a href="#ref-bZMKqj6e" role="doc-biblioref">150</a>)</span>.
Damage to the capillary endothelium can cause leaks that disrupt the balance between pro-inflammatory cytokines and their regulators <span class="citation" data-cites="f61jsRKY">(<a href="#ref-f61jsRKY" role="doc-biblioref">154</a>)</span>, and heightened inflammation in the lungs can also serve as a source for systemic inflammation, or sepsis, and potentially multi-organ failure <span class="citation" data-cites="bZMKqj6e">(<a href="#ref-bZMKqj6e" role="doc-biblioref">150</a>)</span>.
The shift from local to systemic inflammation is a phenomenon often referred to broadly as a cytokine storm <span class="citation" data-cites="bZMKqj6e">(<a href="#ref-bZMKqj6e" role="doc-biblioref">150</a>)</span> or, more precisely, as cytokine release syndrome <span class="citation" data-cites="NITOJ0Ka">(<a href="#ref-NITOJ0Ka" role="doc-biblioref">155</a>)</span>.</p>
<p>Cytokine dysregulation is therefore a significant concern in the context of COVID-19.
In addition to the known role of cytokines in ARDS and lung infection more broadly, immunohistological analysis at autopsy of deceased SARS patients revealed that ACE2-expressing cells that were infected by SARS-CoV-1 showed elevated expression of the cytokines IL-6, IL-1β, and TNF-α <span class="citation" data-cites="NNFGje9g">(<a href="#ref-NNFGje9g" role="doc-biblioref">156</a>)</span>.
Similarly, the introduction of the S protein from SARS-CoV-1 to mouse macrophages was found to increase production of IL-6 and TNF-α <span class="citation" data-cites="QkTGQUcj">(<a href="#ref-QkTGQUcj" role="doc-biblioref">157</a>)</span>.
For SARS-CoV-2 infection leading to COVID-19, early reports described a cytokine storm syndrome-like response in patients with particularly severe infections <span class="citation" data-cites="9qsndPCu 3HdlV9Vf cyUG1Zi2">(<a href="#ref-9qsndPCu" role="doc-biblioref">65</a>, <a href="#ref-3HdlV9Vf" role="doc-biblioref">158</a>, <a href="#ref-cyUG1Zi2" role="doc-biblioref">159</a>)</span>.
Sepsis has been identified as a major contributor to COVID-19-related death.
Among patients hospitalized with COVID-19 in Wuhan, China, 112 out of 191 (59%) developed sepsis, including all 54 of the non-survivors <span class="citation" data-cites="10THxyeCg">(<a href="#ref-10THxyeCg" role="doc-biblioref">83</a>)</span>.</p>
<p>While IL-6 is sometimes used as a biomarker for cytokine storm activity in sepsis <span class="citation" data-cites="bZMKqj6e">(<a href="#ref-bZMKqj6e" role="doc-biblioref">150</a>)</span>, the relationship between cytokine profiles and the risks associated with sepsis may be more complex.
One study of patients with and at risk for ARDS, specifically those who were intubated for medical ventilation, found that shortly after the onset of ARDS, anti-inflammatory cytokine concentration in BALF increased relative to the concentration of pro-inflammatory cytokines <span class="citation" data-cites="f61jsRKY">(<a href="#ref-f61jsRKY" role="doc-biblioref">154</a>)</span>.
The results suggest that an increase in pro-inflammatory cytokines such as IL-6 may signal the onset of ARDS, but recovery depends on an increased anti-inflammatory response <span class="citation" data-cites="f61jsRKY">(<a href="#ref-f61jsRKY" role="doc-biblioref">154</a>)</span>.
However, patients with severe ARDS were excluded from this study.
Another analysis of over 1,400 pneumonia patients in the United States reported that IL-6, tumor necrosis factor (TNF), and IL-10 were elevated at intake in patients who developed severe sepsis and/or ultimately died <span class="citation" data-cites="oSOG8N0f">(<a href="#ref-oSOG8N0f" role="doc-biblioref">160</a>)</span>.
However, unlike the study analyzing pro- and anti-inflammatory cytokines in ARDS patients <span class="citation" data-cites="f61jsRKY">(<a href="#ref-f61jsRKY" role="doc-biblioref">154</a>)</span>, this study reported that unbalanced pro-/anti-inflammatory cytokine profiles were rare.
This discrepancy could be related to the fact that the sepsis study measured only three cytokines.
Although IL-6 has traditionally been considered pro-inflammatory, its pleiotropic effects via both classical and trans-signaling allow it to play an integral role in both the inflammatory and anti-inflammatory responses <span class="citation" data-cites="14VESs80B">(<a href="#ref-14VESs80B" role="doc-biblioref">161</a>)</span>, leading it to be associated with both healthy and pathological responses to viral threat <span class="citation" data-cites="dBAe8aYi">(<a href="#ref-dBAe8aYi" role="doc-biblioref">162</a>)</span>.
While the cytokine levels observed in COVID-19 patients fall outside of the normal range, they are not as high as typically found in patients with ARDS <span class="citation" data-cites="Xcrxlxnp">(<a href="#ref-Xcrxlxnp" role="doc-biblioref">163</a>)</span>.
Regardless of variation in the anti-inflammatory response, prior work has therefore made it clear that pulmonary infection and injury are associated with systemic inflammation and with sepsis.
Inflammation has received significant interest both in regards to the pathology of COVID-19 as well as potential avenues for treatment, as the relationship between the cytokine storm and the pathophysiology of COVID-19 has led to the suggestion that a number of immunomodulatory pharmaceutical interventions could hold therapeutic value for the treatment of COVID-19 <span class="citation" data-cites="RIMWgsWt njpLhBui">(<a href="#ref-njpLhBui" role="doc-biblioref">3</a>, <a href="#ref-RIMWgsWt" role="doc-biblioref">164</a>)</span>.</p>
<h2 data-number="1.6" id="insights-from-systems-biology"><span class="header-section-number">1.6</span> Insights from Systems Biology</h2>
<p>Systems biology provides a cross-disciplinary analytical paradigm through which the host response to an infection can be analyzed.
This field integrates the “omics” fields (genomics, transcriptomics, proteomics, metabolomics, etc.) using bioinformatics and other computational approaches.
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases <span class="citation" data-cites="8esIIWt9">(<a href="#ref-8esIIWt9" role="doc-biblioref">165</a>)</span>.
Omics-based studies have also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections <span class="citation" data-cites="8vUHj9cm">(<a href="#ref-8vUHj9cm" role="doc-biblioref">166</a>)</span>.
Though the complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not yet fully understood, omics technologies offer the opportunity for discovery-driven analysis of biological changes associated with SARS-CoV-2 infection.</p>
<h3 data-number="1.6.1" id="transcriptomics"><span class="header-section-number">1.6.1</span> Transcriptomics</h3>
<p>Through transcriptomic analysis, the effect of a viral infection on gene expression can be assessed.
Transcriptomic analyses, whether <em>in vivo</em> or <em>in situ</em>, can potentially reveal insights into viral pathogenesis by elucidating the host response to the virus.
For example, infection by some viruses, including by the coronaviruses SARS-CoV-2, SARS-CoV-1, and MERS-CoV, is associated with the upregulation of ACE2 in human embryonic kidney cells and human airway epithelial cells <span class="citation" data-cites="9qsndPCu">(<a href="#ref-9qsndPCu" role="doc-biblioref">65</a>)</span>.
This finding suggests that SARS-CoV-2 facilitates the positive regulation of its own transmission between host cells <span class="citation" data-cites="9qsndPCu">(<a href="#ref-9qsndPCu" role="doc-biblioref">65</a>)</span>.
The host immune response also likely plays a key role in mediating infection-associated pathologies.
Therefore, transcriptomics is one critical tool for characterizing the host response in order to gain insight into viral pathogenesis.
For this reason, the application of omics technologies to the process of characterizing the host response is expected to provide novel insights into how hosts respond to SARS-CoV-2 infection and how these changes might influence COVID-19 outcomes.</p>
<p>Several studies have examined the cellular response to SARS-CoV-2 <em>in vitro</em> in comparison to other viruses.
One study <span class="citation" data-cites="pHrstqMQ">(<a href="#ref-pHrstqMQ" role="doc-biblioref">167</a>)</span> compared the transcriptional responses of three human cell lines to SARS-CoV-2 and to other respiratory viruses, including MERS-CoV, SARS-CoV-1, <em>Human parainfluenza virus 3</em>, <em>Respiratory syncytial virus</em>, and <em>Influenza A virus</em>.
The transcriptional response differed between the SARS-CoV-1 infected cells and the cells infected by other viruses, with changes in differential expression specific to each infection type.
Where SARS-CoV-2 was able to replicate efficiently, differential expression analysis revealed that the transcriptional response was significantly different from the response to all of the other viruses tested.
A unique pro-inflammatory cytokine signature associated with SARS-CoV-2 was present in cells exposed to both high and low doses of the virus, with the cytokines IL-6 and IL1RA uniquely elevated in response to SARS-CoV-2 relative to other viruses.
However, one cell line showed significant IFN-I or IFN-III expression when exposed to high, but not low, doses of SARS-CoV-2, suggesting that IFN induction is dependent on the extent of exposure.
These results suggest that SARS-CoV-2 induces a limited antiviral state with low IFN-I or IFN-III expression and a moderate IFN-stimulated gene response, in contrast to other viruses.
Other respiratory viruses have been found to encode antagonists to the IFN response <span class="citation" data-cites="16NxbPnxZ 20zcMj6d">(<a href="#ref-16NxbPnxZ" role="doc-biblioref">168</a>, <a href="#ref-20zcMj6d" role="doc-biblioref">169</a>)</span>, including SARS-CoV-1 <span class="citation" data-cites="DuPmqcag">(<a href="#ref-DuPmqcag" role="doc-biblioref">170</a>)</span> and MERS-CoV <span class="citation" data-cites="6ovCW0LD">(<a href="#ref-6ovCW0LD" role="doc-biblioref">171</a>)</span>.</p>
<p>The analysis of SARS-CoV-2 suggested that this transcriptional state was specific to cells expressing ACE2, as it was not observed in cells lacking expression of this protein except with ACE2 supplementation and at very high (10-fold increase) level of SARS-CoV-2 exposure <span class="citation" data-cites="pHrstqMQ">(<a href="#ref-pHrstqMQ" role="doc-biblioref">167</a>)</span>.
In another study, direct stimulation with inflammatory cytokines such as type I interferons (e.g., IFNβ) was also associated with the upregulation of ACE2 in human bronchial epithelial cells, with treated groups showing four-fold higher ACE2 expression than control groups at 18 hours post-treatment <span class="citation" data-cites="UwVweB2M">(<a href="#ref-UwVweB2M" role="doc-biblioref">172</a>)</span>.
This hypothesis was further supported by studies showing that several nsp in SARS-CoV-2 suppress interferon activity <span class="citation" data-cites="UqnfNPs7">(<a href="#ref-UqnfNPs7" role="doc-biblioref">173</a>)</span> and that the SARS-CoV-2 <em>ORF3b</em> gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV-1 <em>ORF3b</em> gene <span class="citation" data-cites="19YhL3ZEM">(<a href="#ref-19YhL3ZEM" role="doc-biblioref">174</a>)</span>.
Taken together, these findings suggest that a unique cytokine profile is associated with the response to the SARS-CoV-2 virus, and that this response differs depending on the magnitude of exposure.</p>
<p>Susceptibility and IFN induction may also vary by cell type.
Using poly(A) bulk RNA-seq to analyzed dynamic transcriptional responses to SARS-CoV-2 and SARS-CoV-1 revealed negligible susceptibility of cells from the H1299 line (&lt; 0.08 viral read percentage of total reads) compared to those from the Caco-2 and Calu-3 lines (&gt;10% of viral reads) <span class="citation" data-cites="Gj8vlc0W">(<a href="#ref-Gj8vlc0W" role="doc-biblioref">175</a>)</span>.
This finding suggests that the risk of infection varies among cell types, and that cell type could influence which hosts are more or less susceptible.
Based on visual inspection of microscopy images alongside transcriptional profiling, the authors also showed distinct responses among the host cell lines evaluated <span class="citation" data-cites="Gj8vlc0W">(<a href="#ref-Gj8vlc0W" role="doc-biblioref">175</a>)</span>.
In contrast to Caco-2, Calu-3 cells infected with SARS-CoV-2 showed signs of impaired growth and cell death at 24 hours post infection, as well as moderate IFN induction with a strong up-regulation of IFN-stimulated genes.
Interestingly, the results were similar to those reported in Calu-3 cells exposed to much higher levels of SARS-CoV-2 <span class="citation" data-cites="pHrstqMQ">(<a href="#ref-pHrstqMQ" role="doc-biblioref">167</a>)</span>, as described above.
This finding suggests that IFN induction in Calu-3 cells is not dependent on the level of exposure, in contrast to A549-ACE2 cells.
The discrepancy could be explained by the observations that Calu-3 cells are highly susceptible to SARS-CoV-2 and show rapid viral replication <span class="citation" data-cites="JOJ2n3gC">(<a href="#ref-JOJ2n3gC" role="doc-biblioref">34</a>)</span>, whereas A549 cells are incompatible with SARS-CoV-2 infection <span class="citation" data-cites="16EFyBURq">(<a href="#ref-16EFyBURq" role="doc-biblioref">176</a>)</span>.
This discrepancy raises the concern that <em>in vitro</em> models may vary in their similarity to the human response, underscoring the importance of follow-up studies in additional models.</p>
<p>As a result, transcriptional analysis of patient tissue is an important application of omics technology to understanding COVID-19.
Several studies have collected blood samples from COVID-19 patients and analyzed them using RNA-Seq <span class="citation" data-cites="13z84exS5 15LlkGLbh 34hQP4Fi KNGUuuNx WOUILlou 196k8asXZ">(<a href="#ref-13z84exS5" role="doc-biblioref">177</a>–<a href="#ref-196k8asXZ" role="doc-biblioref">182</a>)</span>.
Analyzing gene expression in the blood is valuable to understanding host-pathogen interactions because of the potential to identify alterations associated with the immune response and to gain insights into inflammation, among other potential insights <span class="citation" data-cites="13z84exS5">(<a href="#ref-13z84exS5" role="doc-biblioref">177</a>)</span>.
One study compared gene expression in 39 COVID-19 inpatients admitted with community-acquired pneumonia to that of control donors using whole blood cell transcriptomes <span class="citation" data-cites="13z84exS5">(<a href="#ref-13z84exS5" role="doc-biblioref">177</a>)</span>.
They also evaluated the effect of mild versus severe disease.
A greater number of differentially expressed genes were found in severe patients compared to controls than in mild patients compared to controls.
They also identified that the transcriptional profiles clustered into five groups and that the groups could not be explained by disease severity.
Most severe cases fell into two clusters associated with increased inflammation and granulocyte and neutrophil activation.
The presence of these clusters suggests the possibility that personalized medicine could be useful in the treatment of COVID-19 <span class="citation" data-cites="13z84exS5">(<a href="#ref-13z84exS5" role="doc-biblioref">177</a>)</span>.
Longitudinal analysis of granulocytes from patients with mild versus severe COVID-19 revealed that granulocyte activation-associated factors differentiated the disease states, with greater numbers of differentially expressed genes early in disease course <span class="citation" data-cites="13z84exS5">(<a href="#ref-13z84exS5" role="doc-biblioref">177</a>)</span>.
This study therefore revealed distinct patterns associated with COVID-19 and identified genes and pathways associated with each cluster.</p>
<p>Many other studies have also identified transcriptomic signatures associated with the immune response and inflammation.
Other studies have profiled the transcriptome of BALF <span class="citation" data-cites="34hQP4Fi">(<a href="#ref-34hQP4Fi" role="doc-biblioref">179</a>)</span> and the nasopharynx <span class="citation" data-cites="W6xJeuhR">(<a href="#ref-W6xJeuhR" role="doc-biblioref">183</a>)</span>.
One study used single-cell transcriptomics techniques to investigate cell types including brain and choroid plexus cells compared to healthy controls and controls with influenza; among other signals of neuroinflammation, this study reported cortical T cells only in COVID-19 patients <span class="citation" data-cites="Uakl0VBe">(<a href="#ref-Uakl0VBe" role="doc-biblioref">184</a>)</span>.
Transcriptomic analysis can thus provide insight into the pathogenesis of SARS-CoV-2 and may also be useful in identifying candidate therapeutics <span class="citation" data-cites="13z84exS5">(<a href="#ref-13z84exS5" role="doc-biblioref">177</a>)</span>.</p>
<h3 data-number="1.6.2" id="proteomics"><span class="header-section-number">1.6.2</span> Proteomics</h3>
<p>Proteomics analysis offers an opportunity to characterize the response to a pathogen at a level above transcriptomics.
Especially early on, this primarily involved evaluating the effect of the virus on cell lines.
One early proteomics study investigated changes associated with <em>in vitro</em> SARS-CoV-2 infection using Caco-2 cells <span class="citation" data-cites="11xZWeHN3">(<a href="#ref-11xZWeHN3" role="doc-biblioref">185</a>)</span>.
This study reported that SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.
Another area of interest is whether SARS-CoV-2 is likely to induce similar changes to other HCoV.
For example, because of the high level of sequence homology between SARS-CoV-2 and SARS-CoV-1, it has been hypothesized that sera from convalescent SARS-CoV-1 patients might show some efficacy in cross-neutralizing SARS-CoV-2-S-driven entry <span class="citation" data-cites="15l3di3Wj">(<a href="#ref-15l3di3Wj" role="doc-biblioref">33</a>)</span>.
However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, which would be likely to prohibit effective cross-neutralization across different SARS species <span class="citation" data-cites="GhJYjnft">(<a href="#ref-GhJYjnft" role="doc-biblioref">186</a>)</span>.
Consequently, proteomic analyses of SARS-CoV-1 might also provide some essential information regarding the new pathogen <span class="citation" data-cites="GpnngtWK JLTf2Fwb">(<a href="#ref-GpnngtWK" role="doc-biblioref">187</a>, <a href="#ref-JLTf2Fwb" role="doc-biblioref">188</a>)</span>.</p>
<p>Proteomics research has been able to get ahead of the timeline for development of omics-level big data sets specific to SARS-CoV-2 by adopting a comparative bioinformatics approach.
Data hubs such as UniProt <span class="citation" data-cites="1ZE22clL">(<a href="#ref-1ZE22clL" role="doc-biblioref">189</a>)</span>, NCBI Genome Database <span class="citation" data-cites="RrqhkfKV">(<a href="#ref-RrqhkfKV" role="doc-biblioref">190</a>)</span>, The Immune Epitope Database and Analysis Resource <span class="citation" data-cites="FE7YVar3">(<a href="#ref-FE7YVar3" role="doc-biblioref">191</a>)</span>, and The Virus Pathogen Resource <span class="citation" data-cites="BmUn7iIt">(<a href="#ref-BmUn7iIt" role="doc-biblioref">192</a>)</span> contain a wealth of data from studies in other viruses and even HCoV.
Such databases facilitate the systems-level reconstruction of protein-protein interaction networks, providing opportunities to generate hypotheses about the mechanism of action of SARS-CoV-2 and identify potential drug targets.
In an initial study <span class="citation" data-cites="ZvaFbLRX">(<a href="#ref-ZvaFbLRX" role="doc-biblioref">193</a>)</span>, 26 of the 29 SARS-CoV-2 proteins were cloned and expressed in HEK293T kidney cells, allowing for the identification of 332 high-confidence human proteins interacting with them.
Notably, this study suggested that SARS-CoV-2 interacts with innate immunity pathways.
Ranking pathogens by the similarity between their interactomes and that of SARS-CoV-2 suggested <em>West Nile virus</em>, <em>Mycobacterium tuberculosis</em>, and <em>human papillomavirus</em> infections as the top three hits.
The fact that the host-pathogen interactome of the bacterium <em>Mycobacterium tuberculosis</em> was found to be similar to that of SARS-CoV-2 suggests that changes related to lung pathology might comprise a significant contributor to these expression profiles.
Additionally, it was suggested that the envelope protein, E, could disrupt host bromodomain-containing proteins, i.e., BRD2 and BRD4, that bind to histones, and the spike protein could likely intervene in viral fusion by modulating the GOLGA7-ZDHHC5 acyl-transferase complex to increase palmitoylation, which is a post-translational modification that affects how proteins interact with membranes <span class="citation" data-cites="mXUCjmCh">(<a href="#ref-mXUCjmCh" role="doc-biblioref">194</a>)</span>.</p>
<p>An example of an application of this <em>in silico</em> approach comes from another study <span class="citation" data-cites="NLoN4aYj">(<a href="#ref-NLoN4aYj" role="doc-biblioref">195</a>)</span>, which used patient-derived peripheral blood mononuclear cells to identify 251 host proteins targeted by SARS-CoV-2.
This study also reported that more than 200 host proteins were disrupted following infection.
In particular, a network analysis showed that nsp9 and nsp10 interacted with NF-Kappa-B-Repressing Factor, which encodes a transcriptional repressor that mediates repression of genes responsive to Nuclear Factor kappa-light-chain-enhancer of activated B-cells.
These genes are important to pro-, and potentially also anti-, inflammatory signaling <span class="citation" data-cites="KAqOiTeZ">(<a href="#ref-KAqOiTeZ" role="doc-biblioref">196</a>)</span>.
This finding could explain the exacerbation of the immune response that shapes the pathology and the high cytokine levels characteristic of COVID-19, possibly due to the chemotaxis of neutrophils mediated by IL-8 and IL-6.
Finally, it was suggested <span class="citation" data-cites="1ccnm0N9">(<a href="#ref-1ccnm0N9" role="doc-biblioref">197</a>)</span> that the E protein of both SARS-CoV-1 and SARS-CoV-2 has a conserved Bcl-2 Homology 3-like motif, which could inhibit anti-apoptosis proteins, e.g., BCL2, and trigger the apoptosis of T cells.
Several compounds are known to disrupt the host-pathogen protein interactome, largely through the inhibition of host proteins.
Therefore, this research identifies candidate targets for intervention and suggests that drugs modulating protein-level interactions between virus and host could be relevant to treating COVID-19.</p>
<p>As with other approaches, analyzing the patterns found in infected versus healthy human subjects is also important.
COVID-19 infection has been associated with quantitative changes in transcripts, proteins, metabolites, and lipids in patient blood samples <span class="citation" data-cites="3DOhQZCg">(<a href="#ref-3DOhQZCg" role="doc-biblioref">198</a>)</span>.
One longitudinal study <span class="citation" data-cites="1EAPLqW23">(<a href="#ref-1EAPLqW23" role="doc-biblioref">199</a>)</span> compared COVID-19 patients to symptomatic controls who were PCR-negative for SARS-CoV-2.
The longitudinal nature of this study allowed it to account for differences in the scale of inter- versus intraindividual changes.
At the time of first sampling, common functions of proteins upregulated in COVID-19 patients relative to controls were related to immune system mediation, coagulation, lipid homeostasis, and protease inhibition.
They compared these data to the patient-specific timepoints associated with the highest levels of SARS-CoV-2 antibodies and found that the actin-binding protein gelsolin, which is involved in recovery from disease, showed the steepest decline between those two time points.
Immunoglobulins comprised the only proteins that were significantly different between the COVID-19 and control patients at both of these timepoints.
The most significantly downregulated proteins between these time points were related to inflammation, while the most significantly upregulated proteins were immunoglobulins.
Proteins related to coagulation also increased between the two timepoints.
The selection of a symptomatic control cohort rather than healthy comparisons also suggests that the results are more likely to highlight the response to SARS-CoV-2 and COVID-19 specifically, rather than to disease more broadly.
This study also compared the disease course in patients who ultimately survived to those who died and found that ITIH4, a protein associated with the inflammatory response to trauma, may be a biomarker useful to identifying patients at risk of death.
Thus, these results indicate the value of studying patients in a longitudinal manner over the disease course.
By revealing which genes are perturbed during SARS-CoV-2 infection, proteomics-based analyses can thus provide novel insights into host-virus interaction and serve to generate new avenues of investigation for therapeutics.</p>
<h2 data-number="1.7" id="viral-virulence"><span class="header-section-number">1.7</span> Viral Virulence</h2>
<p>Like that of SARS-CoV-1, the entry of SARS-CoV-2 into host cells is mediated by interactions between the viral spike glycoprotein, S, and human ACE2 (hACE2) <span class="citation" data-cites="15EIBRhef qcVbT0w4 12TZ7hPMA wCbhn23d VX8OWaGj 15l3di3Wj 15Q2XgkK7 t1e4CW9A">(<a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>, <a href="#ref-15l3di3Wj" role="doc-biblioref">33</a>, <a href="#ref-15EIBRhef" role="doc-biblioref">200</a>–<a href="#ref-t1e4CW9A" role="doc-biblioref">205</a>)</span>.
Differences in how the S proteins of the two viruses interact with hACE2 could partially account for the increased transmissibility of SARS-CoV-2.
Studies have reported conflicting binding constants for the S-hACE2 interaction, though they have agreed that the SARS-CoV-2 S protein binds with equal, if not greater, affinity than the SARS-CoV-1 S protein does <span class="citation" data-cites="qcVbT0w4 VX8OWaGj CqQVkaqj">(<a href="#ref-CqQVkaqj" role="doc-biblioref">16</a>, <a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>, <a href="#ref-VX8OWaGj" role="doc-biblioref">203</a>)</span>.
The C-terminal domain of the SARS-CoV-2 S protein in particular was identified as the key region of the virus that interacts with hACE2, and the crystal structure of the C-terminal domain of the SARS-CoV-2 S protein in complex with hACE2 reveals stronger interaction and a higher affinity for receptor binding than that of SARS-CoV-1 <span class="citation" data-cites="15Q2XgkK7">(<a href="#ref-15Q2XgkK7" role="doc-biblioref">204</a>)</span>.
Among the 14 key binding residues identified in the SARS-CoV-1 S protein, eight are conserved in SARS-CoV-2, and the remaining six are semi-conservatively substituted, potentially explaining variation in binding affinity <span class="citation" data-cites="qcVbT0w4 VX8OWaGj">(<a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>, <a href="#ref-VX8OWaGj" role="doc-biblioref">203</a>)</span>.
Studies of crystal structure have shown that the RBD of the SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one <span class="citation" data-cites="qcVbT0w4 CqQVkaqj">(<a href="#ref-CqQVkaqj" role="doc-biblioref">16</a>, <a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>)</span>.
Spike proteins cleaved at the furin-like binding site are substantially more likely to take an open conformation (66%) than those that are uncleaved (17%) <span class="citation" data-cites="GbQPEvZG">(<a href="#ref-GbQPEvZG" role="doc-biblioref">206</a>)</span>.
Because the RBD plays such a critical role in viral entry, blocking its interaction with ACE2 could represent a promising therapeutic approach.
Nevertheless, despite the high structural homology between the SARS-CoV-2 RBD and that of SARS-CoV-1, monoclonal antibodies targeting SARS-CoV-1 RBD failed to bind to SARS-CoV-2-RBD <span class="citation" data-cites="CqQVkaqj">(<a href="#ref-CqQVkaqj" role="doc-biblioref">16</a>)</span>.
However, in early research, sera from convalescent SARS patients were found to inhibit SARS-CoV-2 viral entry <em>in vitro</em>, albeit with lower efficiency than it inhibited SARS-CoV-1 <span class="citation" data-cites="15l3di3Wj">(<a href="#ref-15l3di3Wj" role="doc-biblioref">33</a>)</span>.</p>
<p>Comparative genomic analysis reveals that several regions of the coronavirus genome are likely critical to virulence.
The S1 domain of the spike protein, which contains the receptor binding motif, evolves more rapidly than the S2 domain <span class="citation" data-cites="13wWdgODZ OVsxrEuX">(<a href="#ref-13wWdgODZ" role="doc-biblioref">23</a>, <a href="#ref-OVsxrEuX" role="doc-biblioref">24</a>)</span>.
However, even within the S1 domain, some regions are more conserved than others, with the receptors in S1’s N-terminal domain (S1-NTD) evolving more rapidly than those in its C-terminal domain (S1-CTD) <span class="citation" data-cites="OVsxrEuX">(<a href="#ref-OVsxrEuX" role="doc-biblioref">24</a>)</span>.
Both S1-NTD and S1-CTD are involved in receptor binding and can function as RBDs to bind proteins and sugars <span class="citation" data-cites="13wWdgODZ">(<a href="#ref-13wWdgODZ" role="doc-biblioref">23</a>)</span>, but RBDs in the S1-NTD typically bind to sugars, while those in the S1-CTD recognize protein receptors <span class="citation" data-cites="17DSmRo9H">(<a href="#ref-17DSmRo9H" role="doc-biblioref">12</a>)</span>.
Viral receptors show higher affinity with protein receptors than sugar receptors <span class="citation" data-cites="17DSmRo9H">(<a href="#ref-17DSmRo9H" role="doc-biblioref">12</a>)</span>, which suggests that positive selection on or relaxed conservation of the S1-NTD might reduce the risk of a deleterious mutation that would prevent binding.
The SARS-CoV-2 S protein also contains an RRAR furin recognition site at the S1/S2 junction <span class="citation" data-cites="qcVbT0w4 CqQVkaqj">(<a href="#ref-CqQVkaqj" role="doc-biblioref">16</a>, <a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>)</span>, setting it apart from both bat coronavirus RaTG13, with which it shares 96% genome sequence identity, and SARS-CoV-1 <span class="citation" data-cites="VSkK7CeP">(<a href="#ref-VSkK7CeP" role="doc-biblioref">207</a>)</span>.
Such furin cleavage sites are commonly found in highly virulent influenza viruses <span class="citation" data-cites="NsORsLig vhHB3yyS">(<a href="#ref-NsORsLig" role="doc-biblioref">208</a>, <a href="#ref-vhHB3yyS" role="doc-biblioref">209</a>)</span>.
The furin recognition site at the S1/S2 junction is likely to increase pathogenicity via destabilization of the spike protein during fusion to ACE2 and the facilitation of cell-cell adhesion <span class="citation" data-cites="qcVbT0w4 CqQVkaqj GbQPEvZG NsORsLig vhHB3yyS 5TQeBfZv">(<a href="#ref-CqQVkaqj" role="doc-biblioref">16</a>, <a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>, <a href="#ref-5TQeBfZv" role="doc-biblioref">42</a>, <a href="#ref-GbQPEvZG" role="doc-biblioref">206</a>, <a href="#ref-NsORsLig" role="doc-biblioref">208</a>, <a href="#ref-vhHB3yyS" role="doc-biblioref">209</a>)</span>.
These factors may influence the virulence of SARS-CoV-2 relative to other beta coronaviruses.
Additionally, a major concern has been the emergence of SARS-CoV-2 variants with increased virulence.
The extent to which evolution within SARS-CoV-2 may affect pathogenesis is reviewed below.</p>
<h2 data-number="1.8" id="molecular-signatures-transmission-and-variants-of-concern"><span class="header-section-number">1.8</span> Molecular Signatures, Transmission, and Variants of Concern</h2>
<p>Genetic variation in SARS-CoV-2 has been used to elucidate patterns over time and space.
Many mutations are neutral in their effect and can be used to trace transmission patterns.
Such signatures within SARS-CoV-2 have provided insights during outbreak investigations <span class="citation" data-cites="p9LUiyCN UUAeVUaR 2w7lNKxQ">(<a href="#ref-p9LUiyCN" role="doc-biblioref">210</a>–<a href="#ref-2w7lNKxQ" role="doc-biblioref">212</a>)</span>.
Similar mutations observed in several patients may indicate that the patients belong to the same transmission group.
The tracking of SARS-CoV-2 mutations is recognized as an essential tool for controlling future outbreaks and tracing the path of the spread of SARS-CoV-2.
In the first months of the pandemic in early 2020, early genomic surveillance efforts in Guangdong, China revealed that local transmission rates were low and that most cases arising in the province were imported <span class="citation" data-cites="SezJVKA6">(<a href="#ref-SezJVKA6" role="doc-biblioref">213</a>)</span>.
Since then, efforts have varied widely among countries: for example, the U.K. has coordinated a national database of viral genomes <span class="citation" data-cites="jVbH3kJR">(<a href="#ref-jVbH3kJR" role="doc-biblioref">214</a>)</span>, but efforts to collect this type of data in the United States have been more limited <span class="citation" data-cites="11nsMZ5u3">(<a href="#ref-11nsMZ5u3" role="doc-biblioref">215</a>)</span>.
Studies have applied phylogenetic analyses of viral genomes to determine the source of local COVID-19 outbreaks in Connecticut (USA), <span class="citation" data-cites="W5Z7Ztlg">(<a href="#ref-W5Z7Ztlg" role="doc-biblioref">216</a>)</span>, the New York City area (USA) <span class="citation" data-cites="FVmbE2oW">(<a href="#ref-FVmbE2oW" role="doc-biblioref">217</a>)</span>, and Iceland <span class="citation" data-cites="WJrnUyZf">(<a href="#ref-WJrnUyZf" role="doc-biblioref">218</a>)</span>.
There has been an ongoing effort to collect SARS-CoV-2 genomes throughout the COVID-19 outbreak, and as of summer 2021, millions of genome sequences have been collected from patients.
The sequencing data can be found at GISAID <span class="citation" data-cites="qIXPib3m">(<a href="#ref-qIXPib3m" role="doc-biblioref">219</a>)</span>, NCBI <span class="citation" data-cites="cyzrC7qd">(<a href="#ref-cyzrC7qd" role="doc-biblioref">220</a>)</span>, and COVID-19 data portal <span class="citation" data-cites="BjPoOYSA">(<a href="#ref-BjPoOYSA" role="doc-biblioref">221</a>)</span>.</p>
<p>Ongoing evolution can be observed in genomic data collected through molecular surveillance efforts.
In some cases, mutations can produce functional changes that can impact pathogenesis.
One early example is the spike protein mutation D614G, which appeared in March 2020 and became dominant worldwide by the end of May 2020 <span class="citation" data-cites="RIpPhJ1g MI5vSutb">(<a href="#ref-RIpPhJ1g" role="doc-biblioref">222</a>, <a href="#ref-MI5vSutb" role="doc-biblioref">223</a>)</span>.
This variant was associated with increased infectivity and increased viral load, but not with more severe disease outcomes <span class="citation" data-cites="RIpPhJ1g 8LyA5uLY">(<a href="#ref-RIpPhJ1g" role="doc-biblioref">222</a>, <a href="#ref-8LyA5uLY" role="doc-biblioref">224</a>)</span>.
This increased virulence is likely achieved by altering the conformation of the S1 domain to facilitate binding to ACE2 <span class="citation" data-cites="8LyA5uLY">(<a href="#ref-8LyA5uLY" role="doc-biblioref">224</a>)</span>.
Similarly, the N439K mutation within the RBD of the spike protein is likely associated with increased transmissibility and enhanced binding affinity for hACE2, although it is also not thought to affect disease outcomes <span class="citation" data-cites="owB3rKUG">(<a href="#ref-owB3rKUG" role="doc-biblioref">225</a>)</span>.
In contrast, a mutation in ORF8 that was identified in Singapore in the early months of 2020 was associated with cases of COVID-19 that were less likely to require treatment with supplemental oxygen <span class="citation" data-cites="10igTgCIt">(<a href="#ref-10igTgCIt" role="doc-biblioref">226</a>)</span>, and a deletion surrounding the furin site insertion at the S1/S2 boundary has been identified only rarely in clinical settings <span class="citation" data-cites="wRtP6ZNU">(<a href="#ref-wRtP6ZNU" role="doc-biblioref">227</a>)</span>, suggesting that these mutations may disadvantage viral pathogenesis in human hosts.
Thus, mutations have been associated with both virological and clinical differences in pathogenesis.</p>
<p>Several VOCs have also been identified and designated through molecular surveillance efforts <span class="citation" data-cites="982hTnLI">(<a href="#ref-982hTnLI" role="doc-biblioref">228</a>)</span>.
The Alpha variant (lineage B.1.1.7) was first observed in the U.K. in October 2020 before it quickly spread around the world <span class="citation" data-cites="cmMwbH6K">(<a href="#ref-cmMwbH6K" role="doc-biblioref">229</a>)</span>.
Other variants meriting further investigation have also been identified, including the Beta variant (B.1.351 lineage) first identified in South Africa and the Gamma variant (P.1 lineage) initially associated with outbreaks in Brazil.
These lineages share independently acquired mutations that may affect pathogenicity <span class="citation" data-cites="RhHGzsfD sqhvCTIL ZAXw65fn x2Pm6BuB tzOnKNrn">(<a href="#ref-RhHGzsfD" role="doc-biblioref">230</a>–<a href="#ref-tzOnKNrn" role="doc-biblioref">234</a>)</span>.
For example, they are all associated with a greater binding affinity for hACE2 than that of the wildtype variant <span class="citation" data-cites="ZAXw65fn BEhidjDY Ic7tOIax">(<a href="#ref-ZAXw65fn" role="doc-biblioref">232</a>, <a href="#ref-BEhidjDY" role="doc-biblioref">235</a>, <a href="#ref-Ic7tOIax" role="doc-biblioref">236</a>)</span>, but they were not found to have more efficient cell entry than the wildtype virus <span class="citation" data-cites="V61zrZZ7">(<a href="#ref-V61zrZZ7" role="doc-biblioref">237</a>)</span>.
A fourth VOC, the Delta variant (B.1.617.2 and AY.1, AY.2, and AY.3 lineages), was identified in India in late 2020 <span class="citation" data-cites="pcm6tmYA">(<a href="#ref-pcm6tmYA" role="doc-biblioref">238</a>)</span>.
Some of the mutations associated with this lineage may alter fusogenicity and enhance furin cleavage, among other effects associated with increased pathogenicity <span class="citation" data-cites="LNApbgCL">(<a href="#ref-LNApbgCL" role="doc-biblioref">239</a>)</span>.
The changes in these VOC demonstrate how ongoing evolution in SARS-CoV-2 can drive changes in how the virus interacts with host cells.</p>
<h2 data-number="1.9" id="quantifying-viral-presence"><span class="header-section-number">1.9</span> Quantifying Viral Presence</h2>
<p>Assessing whether a virus is present in a sample is a more complex task than it initially seems.
Many diagnostic tests rely on real-time polymerase chain reaction (RT-PCR) to test for the presence versus absence of a virus <span class="citation" data-cites="hIxvVJ7F">(<a href="#ref-hIxvVJ7F" role="doc-biblioref">5</a>)</span>.
They may report the cycle threshold (C<sub>t</sub>) indicating the number of doubling cycles required for the target (in this case, SARS-CoV-2) to become detectable.
A lower C<sub>t</sub> therefore corresponds to a higher viral load.
The C<sub>t</sub> that corresponds to a positive can vary widely, but is often around 35.
This information is sufficient to answer many questions, since an amplicon must be present in order to be duplicated in RT-PCR.
For example, if a patient is presenting with COVID-19 symptoms, a positive RT-PCR test can confirm the diagnosis.</p>
<p>However, RT-PCR analysis alone cannot provide the information needed to determine whether a virus is present at sufficient levels to be infectious <span class="citation" data-cites="1HaaPNEXD">(<a href="#ref-1HaaPNEXD" role="doc-biblioref">240</a>)</span>.
Some studies have therefore taken the additional step of cultivating samples <em>in vitro</em> in order to observe whether cells become infected with SARS-CoV-2.
One study collected upper respiratory tract samples from COVID-19 patients, analyzed them with RT-PCR to determine the cycle threshold, and then attempted to cultivate the SARS-CoV-2 virus in VeroE6 cells <span class="citation" data-cites="1HaaPNEXD">(<a href="#ref-1HaaPNEXD" role="doc-biblioref">240</a>)</span>.
This study found that out of 246 samples, less than half (103) produced a positive culture.
Moreover, at a C<sub>t</sub> of 35, only 5 out of 60 samples grew <em>in vitro</em>.
Therefore, the RT-PCR-confirmed presence of SARS-CoV-2 in a sample does not necessarily indicate that the virus is present at a high enough concentration to grow and/or spread.</p>
<h2 data-number="1.10" id="mechanisms-of-transmission"><span class="header-section-number">1.10</span> Mechanisms of Transmission</h2>
<p>When a human host is infected with a virus and is contagious, person-to-person viral transmission can occur through several possible mechanisms.
When a contagious individual sneezes, coughs, or exhales, they produce respiratory droplets that can contain a large number of viral particles <span class="citation" data-cites="KMb6knfE">(<a href="#ref-KMb6knfE" role="doc-biblioref">241</a>)</span>.
Viral particles can enter the body of a new host when they then come in contact with the oral, nasal, eye, or other mucus membranes <span class="citation" data-cites="KMb6knfE">(<a href="#ref-KMb6knfE" role="doc-biblioref">241</a>)</span>.
The primary terms typically used to discuss the transmission of viruses via respiratory droplets are droplet, aerosol, and contact transmission <span class="citation" data-cites="CJbohYmK">(<a href="#ref-CJbohYmK" role="doc-biblioref">242</a>)</span>.
The distinction between droplet and aerosol transmission is typically anchored on whether a particle containing the virus is larger or smaller than 5 micrometers (μm) <span class="citation" data-cites="MZlr12J3 CvYwVG8O">(<a href="#ref-MZlr12J3" role="doc-biblioref">243</a>, <a href="#ref-CvYwVG8O" role="doc-biblioref">244</a>)</span>.
Droplet transmission typically refers to contact with large droplets that fall quickly to the ground at close range, such as breathing in droplets produced by a sneeze <span class="citation" data-cites="KMb6knfE MZlr12J3">(<a href="#ref-KMb6knfE" role="doc-biblioref">241</a>, <a href="#ref-MZlr12J3" role="doc-biblioref">243</a>)</span>.
Aerosol transmission typically refers to much smaller particles (less than 5 μm) produced by sneezing, coughing, or exhaling <span class="citation" data-cites="CJbohYmK KMb6knfE">(<a href="#ref-KMb6knfE" role="doc-biblioref">241</a>, <a href="#ref-CJbohYmK" role="doc-biblioref">242</a>)</span> that can remain suspended over a longer period of time and potentially to be moved by air currents <span class="citation" data-cites="KMb6knfE">(<a href="#ref-KMb6knfE" role="doc-biblioref">241</a>)</span>.
It is also possible that viral particles deposited on surfaces via large respiratory droplets could later be aerosolized <span class="citation" data-cites="KMb6knfE">(<a href="#ref-KMb6knfE" role="doc-biblioref">241</a>)</span>.
The transmission of viral particles that have settled on a surface is typically referred to as contact or fomite transmission <span class="citation" data-cites="1FLLN4PSl KMb6knfE">(<a href="#ref-KMb6knfE" role="doc-biblioref">241</a>, <a href="#ref-1FLLN4PSl" role="doc-biblioref">245</a>)</span>.
Any respiratory droplets that settle on a surface could contribute to fomite transmission <span class="citation" data-cites="KMb6knfE">(<a href="#ref-KMb6knfE" role="doc-biblioref">241</a>)</span>.
Droplet and contact transmission are both well-accepted modes of transmission for many viruses associated with common human illnesses, including influenza and rhinovirus <span class="citation" data-cites="KMb6knfE">(<a href="#ref-KMb6knfE" role="doc-biblioref">241</a>)</span>.</p>
<p>The extent to which aerosol transmission contributes to the spread of respiratory viruses is more widely debated.
In influenza A, for example, viral particles can be detected in aerosols produced by infected individuals, but it is not clear to what extent these particles drive the spread of influenza A infection <span class="citation" data-cites="KMb6knfE CJbohYmK AC4okoVf Hct9jRcb i6tTpqwA">(<a href="#ref-KMb6knfE" role="doc-biblioref">241</a>, <a href="#ref-CJbohYmK" role="doc-biblioref">242</a>, <a href="#ref-AC4okoVf" role="doc-biblioref">246</a>–<a href="#ref-i6tTpqwA" role="doc-biblioref">248</a>)</span>.
Regardless of its role in the spread of influenza A, however, aerosol transmission likely played a role in outbreaks such as the 1918 Spanish Influenza (H1N1) and 2009 “swine flu” (pH1N1) <span class="citation" data-cites="i6tTpqwA">(<a href="#ref-i6tTpqwA" role="doc-biblioref">248</a>)</span>.
All three of these mechanisms have been identified as possible contributors to the transmission of HCoVs <span class="citation" data-cites="KMb6knfE">(<a href="#ref-KMb6knfE" role="doc-biblioref">241</a>)</span>, including the highly pathogenic coronaviruses SARS-CoV-1 and MERS-CoV <span class="citation" data-cites="rjVw7V94 G5NJrE75">(<a href="#ref-rjVw7V94" role="doc-biblioref">249</a>, <a href="#ref-G5NJrE75" role="doc-biblioref">250</a>)</span>.
Transmission of SARS-CoV-1 is thought to proceed primarily through droplet transmission, but aerosol transmission is also considered possible <span class="citation" data-cites="KMb6knfE 1D0sSd6ZT zmcRFgYd">(<a href="#ref-KMb6knfE" role="doc-biblioref">241</a>, <a href="#ref-1D0sSd6ZT" role="doc-biblioref">251</a>, <a href="#ref-zmcRFgYd" role="doc-biblioref">252</a>)</span>, and fomite transmission may have also played an important role in some outbreaks <span class="citation" data-cites="yR57NFIB">(<a href="#ref-yR57NFIB" role="doc-biblioref">253</a>)</span>.
Similarly, the primary mechanism of MERS transmission is thought to be droplets because inter-individual transmission appears to be associated with close interpersonal contact (e.g., household or healthcare settings), but aerosolized particles of the MERS virus have been reported to persist much more robustly than influenza A under a range of environmental conditions <span class="citation" data-cites="s3oVNbGV 1FjDLPCye">(<a href="#ref-s3oVNbGV" role="doc-biblioref">254</a>, <a href="#ref-1FjDLPCye" role="doc-biblioref">255</a>)</span>.
However, few of these analyses have sought to grow positive samples in culture and thus to confirm their potential to infect new hosts.</p>
<p>Contact, droplet, and aerosol transmission are therefore all worth evaluating when considering possible modes of transmission for a respiratory virus like SARS-CoV-2.
The stability of the SARS-CoV-2 virus both in aerosols and on a variety of surfaces was found to be similar to that of SARS-CoV-1 <span class="citation" data-cites="yXKcviw8">(<a href="#ref-yXKcviw8" role="doc-biblioref">256</a>)</span>.
Droplet-based and contact transmission were initially put forward as the greatest concern for the spread of SARS-CoV-2 <span class="citation" data-cites="vP6yTZ0y">(<a href="#ref-vP6yTZ0y" role="doc-biblioref">257</a>)</span>, with droplet transmission considered the dominant mechanism driving the spread of the virus <span class="citation" data-cites="82XnTbtX">(<a href="#ref-82XnTbtX" role="doc-biblioref">258</a>)</span> because the risk of fomite transmission under real-world conditions is likely to be substantially lower than the conditions used for experimental analyses <span class="citation" data-cites="cw5j7x80">(<a href="#ref-cw5j7x80" role="doc-biblioref">259</a>)</span>.
The COVID-19 pandemic has, however, exposed significant discrepancies in how terms pertaining to airborne viral particles are interpreted in different contexts <span class="citation" data-cites="MZlr12J3">(<a href="#ref-MZlr12J3" role="doc-biblioref">243</a>)</span>.
The 5-μm distinction between “droplets” and “aerosols” is typical in the biological literature but is likely an artifact of historical science rather than a meaningful boundary in biology or physics <span class="citation" data-cites="CvYwVG8O">(<a href="#ref-CvYwVG8O" role="doc-biblioref">244</a>)</span>.
Additionally, various ambient conditions such as air flow can influence how particles of different sizes fall or spread <span class="citation" data-cites="MZlr12J3">(<a href="#ref-MZlr12J3" role="doc-biblioref">243</a>)</span>.
Despite initial skepticism about airborne transmission of SARS-CoV-2 through small particles <span class="citation" data-cites="CvYwVG8O">(<a href="#ref-CvYwVG8O" role="doc-biblioref">244</a>)</span>, evidence now suggests that small particles can contribute to SARS-CoV-2 transmission <span class="citation" data-cites="16SDeiudC yXKcviw8 9ouDX5IN sUAAY1me">(<a href="#ref-yXKcviw8" role="doc-biblioref">256</a>, <a href="#ref-16SDeiudC" role="doc-biblioref">260</a>–<a href="#ref-sUAAY1me" role="doc-biblioref">262</a>)</span>.
For example, one early study detected SARS-CoV-2 viral particles in air samples taken from hospitals treating COVID-19 patients, although the infectivity of these samples was not assessed <span class="citation" data-cites="bQLMgMGC">(<a href="#ref-bQLMgMGC" role="doc-biblioref">263</a>)</span>.
Subsequently, other studies have been successful in growing SARS-CoV-2 in culture with samples taken from the air <span class="citation" data-cites="Uzfdlcgn lPbg0LYo">(<a href="#ref-Uzfdlcgn" role="doc-biblioref">264</a>, <a href="#ref-lPbg0LYo" role="doc-biblioref">265</a>)</span> while others have not <span class="citation" data-cites="1BWoTPei7 X1nA82rS">(<a href="#ref-1BWoTPei7" role="doc-biblioref">266</a>, <a href="#ref-X1nA82rS" role="doc-biblioref">267</a>)</span> (see <span class="citation" data-cites="184VaR2ib">(<a href="#ref-184VaR2ib" role="doc-biblioref">268</a>)</span> for a systematic review of available findings as of July 2020).
The fact that viable SARS-CoV-2 may exist in aerosolized particles calls into question whether some axioms of COVID-19 prevention, such as 2-meter social distancing, are sufficient <span class="citation" data-cites="1Gqr7yJGi CvYwVG8O Uzfdlcgn">(<a href="#ref-CvYwVG8O" role="doc-biblioref">244</a>, <a href="#ref-Uzfdlcgn" role="doc-biblioref">264</a>, <a href="#ref-1Gqr7yJGi" role="doc-biblioref">269</a>)</span>.</p>
<h3 data-number="1.10.1" id="symptoms-and-viral-spread"><span class="header-section-number">1.10.1</span> Symptoms and Viral Spread</h3>
<p>Other aspects of pathogenesis are also important to understanding how the virus spreads, especially the relationship between symptoms, viral shedding, and contagiousness.
Symptoms associated with reported cases of COVID-19 range from mild to severe <span class="citation" data-cites="gHWlMufv">(<a href="#ref-gHWlMufv" role="doc-biblioref">8</a>)</span>, but some individuals who contract COVID-19 remain asymptomatic throughout the duration of the illness <span class="citation" data-cites="wxVni9Hz">(<a href="#ref-wxVni9Hz" role="doc-biblioref">270</a>)</span>.
The incubation period, or the time period between exposure and the onset of symptoms, has been estimated at five to eight days, with means of 4.91 (95% confidence interval (CI) 4.35-5.69) and 7.54 (95% CI 6.76-8.56) reported in two different Asian cities and a median of 5 (IQR 1 to 6) reported in a small number of patients in a Beijing hospital <span class="citation" data-cites="NIxttl2v QFVSrboR">(<a href="#ref-NIxttl2v" role="doc-biblioref">271</a>, <a href="#ref-QFVSrboR" role="doc-biblioref">272</a>)</span>.</p>
<p>However, the exact relationship between contagiousness and viral shedding remains unclear.
Estimates suggest that viral shedding can, in some cases, begin as early as 12.3 days (95% CI 5.9-17.0) before the onset of symptoms, although this was found to be very rare, with less than 0.1% of transmission events occurring 7 or more days before symptom onset <span class="citation" data-cites="1654TaAK4">(<a href="#ref-1654TaAK4" role="doc-biblioref">273</a>)</span>.
Transmissibility appeared to peak around the onset of symptoms (95% CI -0.9 - 0.9 days), and only 44% (95% CI 30–57%) of transmission events were estimated to occur from presymptomatic contacts <span class="citation" data-cites="1654TaAK4">(<a href="#ref-1654TaAK4" role="doc-biblioref">273</a>)</span>.
A peak in viral load corresponding to the onset of symptoms was also confirmed by another study <span class="citation" data-cites="1HaaPNEXD">(<a href="#ref-1HaaPNEXD" role="doc-biblioref">240</a>)</span>.
As these trends became apparent, concerns arose due to the potential for individuals who did not yet show symptoms to transmit the virus <span class="citation" data-cites="Oyy1OUTT">(<a href="#ref-Oyy1OUTT" role="doc-biblioref">274</a>)</span>.
Recovered individuals may also be able to transmit the virus after their symptoms cease.
A study of the communicable period based on twenty-four individuals who tested positive for SARS-CoV-2 prior to or without developing symptoms estimated that individuals may be contagious for one to twenty-one days, but they note that this estimate may be low <span class="citation" data-cites="wxVni9Hz">(<a href="#ref-wxVni9Hz" role="doc-biblioref">270</a>)</span>.
In an early study, viral nucleic acids were reported to remain at observable levels in the respiratory specimens of recovering hospitalized COVID-19 patients for a median of 20 days and with a maximum observed duration through 37 days, when data collection for the study ceased <span class="citation" data-cites="10THxyeCg">(<a href="#ref-10THxyeCg" role="doc-biblioref">83</a>)</span>.</p>
<p>As more estimates of the duration of viral shedding were released, they converged around approximately three weeks from first positive PCR test and/or onset of symptoms (which, if present, are usually identified within three days of the initial PCR test).
For example, in some studies, viral shedding was reported for up to 28 days following symptom onset <span class="citation" data-cites="Xp5HT2SF">(<a href="#ref-Xp5HT2SF" role="doc-biblioref">275</a>)</span> and for one to 24 days from first positive PCR test, with a median of 12 days <span class="citation" data-cites="azgJqujy">(<a href="#ref-azgJqujy" role="doc-biblioref">72</a>)</span>.
On the other hand, almost 70% of patients were reported to still have symptoms at the time that viral shedding ceased, although all symptoms reduced in prevalence between onset and cessation of viral shedding <span class="citation" data-cites="14fWuiUKS">(<a href="#ref-14fWuiUKS" role="doc-biblioref">276</a>)</span>.
The median time that elapsed between the onset of symptoms and cessation of viral RNA shedding was 23 days and between first positive PCR test and cessation of viral shedding was 17 days <span class="citation" data-cites="14fWuiUKS">(<a href="#ref-14fWuiUKS" role="doc-biblioref">276</a>)</span>.
The fact that this study reported symptom onset to predate the first positive PCR test by an average of three days, however, suggests that there may be some methodological differences between it and related studies.
Furthermore, an analysis of residents of a nursing home with a known SARS-CoV-2 case measured similar viral load in residents who were asymptomatic regardless of whether they later developed symptoms, and the load in the asymptomatic residents was comparable to that of residents who displayed either typical or atypical symptoms <span class="citation" data-cites="ac2kt3rh">(<a href="#ref-ac2kt3rh" role="doc-biblioref">277</a>)</span>.
Taken together, these results suggest that the presence or absence of symptoms are not reliable predictors of viral shedding or of SARS-CoV-2 status (e.g, <span class="citation" data-cites="lVDiHYp">(<a href="#ref-lVDiHYp" role="doc-biblioref">278</a>)</span>).
However, it should be noted that viral shedding is not necessarily a robust indicator of contagiousness.
The risk of spreading the infection was low after ten days from the onset of symptoms, as viral load in sputum was found to be unlikely to pose a significant risk based on efforts to culture samples <em>in vitro</em> <span class="citation" data-cites="Xp5HT2SF">(<a href="#ref-Xp5HT2SF" role="doc-biblioref">275</a>)</span>.
The relationship between symptoms, detectable levels of the virus, and risk of viral spread is therefore complex.</p>
<p>The extent to which asymptomatic or presymptomatic individuals are able to transmit SARS-CoV-2 has been a question of high scientific and community interest.
Early reports (February and March 2020) described transmission from presymptomatic SARS-CoV-2-positive individuals to close family contacts <span class="citation" data-cites="E6Qv6YQ0 18Orbxz8t">(<a href="#ref-E6Qv6YQ0" role="doc-biblioref">279</a>, <a href="#ref-18Orbxz8t" role="doc-biblioref">280</a>)</span>.
One of these reports <span class="citation" data-cites="18Orbxz8t">(<a href="#ref-18Orbxz8t" role="doc-biblioref">280</a>)</span> also included a description of an individual who tested positive for SARS-CoV-2 but never developed symptoms.
Later analyses also sought to estimate the proportion of infections that could be traced back to a presymptomatic or asymptomatic individual (e.g., <span class="citation" data-cites="4M3GM1sg">(<a href="#ref-4M3GM1sg" role="doc-biblioref">281</a>)</span>).
Estimates of the proportion of individuals with asymptomatic infections have varied widely.
The proportion of asymptomatic individuals on board the Diamond Princess cruise ship, which was the site of an early COVID-19 outbreak, was estimated at 17.9% <span class="citation" data-cites="6K5lsF5i">(<a href="#ref-6K5lsF5i" role="doc-biblioref">282</a>)</span>.
In contrast, a model using the prevalence of antibodies among residents of Wuhan, China estimated a much higher rate of asymptomatic cases, at approximately 7 in 8, or 87.5% <span class="citation" data-cites="sk1NbA7K">(<a href="#ref-sk1NbA7K" role="doc-biblioref">283</a>)</span>.
An analysis of the populations of care homes in London found that, among the residents (median age 85), the rate of asymptomatic infection was 43.8%, and among the caretakers (median age 47), the rate was 49.1% <span class="citation" data-cites="kIumgXPI">(<a href="#ref-kIumgXPI" role="doc-biblioref">284</a>)</span>.
The duration of viral shedding may also be longer in individuals with asymptomatic cases of COVID-19 compared to those who do show symptoms <span class="citation" data-cites="10OGkFiGJ">(<a href="#ref-10OGkFiGJ" role="doc-biblioref">285</a>)</span>.
As a result, the potential for individuals who do not know they have COVID-19 to spread the virus raises significant concerns.
In Singapore and Tianjin, two cities studied to estimate incubation period, an estimated 40-50% and 60-80% of cases, respectively, were considered to be caused by contact with asymptomatic individuals <span class="citation" data-cites="NIxttl2v">(<a href="#ref-NIxttl2v" role="doc-biblioref">271</a>)</span>.
An analysis of viral spread in the Italian town of Vo’, which was the site of an early COVID-19 outbreak, revealed that 42.5% of cases were asymptomatic and that the rate was similar across age groups <span class="citation" data-cites="dMjSbAQV">(<a href="#ref-dMjSbAQV" role="doc-biblioref">286</a>)</span>.
The argument was thus made that the town’s lockdown was imperative for controlling the spread of COVID-19 because it isolated asymptomatic individuals.
While more models are likely to emerge to better explore the effect of asymptomatic individuals on SARS-CoV-2 transmission, these results suggest that strategies for identifying and containing asymptomatic but contagious individuals are important for managing community spread.</p>
<h3 data-number="1.10.2" id="estimating-the-fatality-rate"><span class="header-section-number">1.10.2</span> Estimating the Fatality Rate</h3>
<p>Estimating the occurrence of asymptomatic and mild COVID-19 cases is important to identifying the mortality rate associated with COVID-19.
The mortality rate of greatest interest would be the total number of fatalities as a fraction of the total number of people infected.
One commonly reported metric is the case fatality rate (CFR), which compares the number of COVID-19 related deaths to the number of confirmed or suspected cases.
However, in locations without universal testing protocols, it is impossible to identify all infected individuals because so many asymptomatic or mild cases go undetected.
Therefore, a more informative metric is the infection fatality rate (IFR), which compares the known deaths to the estimated number of cases.
It thus requires the same numerator as CFR, but divides by an approximation of the total number of cases rather than only the observed/suspected cases.
IFR varies regionally, with some locations observed to have IFRs as low as 0.17% while others are as high as 1.7% <span class="citation" data-cites="dHGy34wC">(<a href="#ref-dHGy34wC" role="doc-biblioref">287</a>)</span>.
Estimates of CFR at the national and continental level and IFR at the continent level is maintained by the Centre for Evidence-Based Medicine <span class="citation" data-cites="AavOV1He">(<a href="#ref-AavOV1He" role="doc-biblioref">288</a>)</span>.
Several meta-analyses have also sought to estimate IFR at the global scale.
These estimates have varied; one peer-reviewed study aggregated data from 24 other studies and estimated IFR at 0.68% (95% CI 0.53%–0.82%), but a preprint that aggregated data from 139 countries calculated a global IFR of 1.04% (95% CI 0.77%-1.38%) when false negatives were considered in the model <span class="citation" data-cites="12CiweatZ dHGy34wC">(<a href="#ref-dHGy34wC" role="doc-biblioref">287</a>, <a href="#ref-12CiweatZ" role="doc-biblioref">289</a>)</span>.
A similar prevalence estimate was identified through a repeated cross-sectional serosurvey conducted in New York City that estimated the IFR as 0.97% <span class="citation" data-cites="tIjAygts">(<a href="#ref-tIjAygts" role="doc-biblioref">290</a>)</span>.
Examination of serosurvey-based estimates of IFR identified convergence on a global IFR estimate of 0.60% (95% CI 0.42%–0.77%) <span class="citation" data-cites="dHGy34wC">(<a href="#ref-dHGy34wC" role="doc-biblioref">287</a>)</span>.
All of these studies note that IFR varies widely by location, and it is also expected to vary with demographic and health-related variables such as age, sex, prevalence of comorbidities, and access to healthcare and testing <span class="citation" data-cites="SOdviufj">(<a href="#ref-SOdviufj" role="doc-biblioref">291</a>)</span>.
Estimates of infection rates are becoming more feasible as more data becomes available for modeling and will be bolstered as serological testing becomes more common and more widely available.
However, this research may be complicated due to the emergence of variants over time, as well as the varying availability and acceptance of vaccines in different communities and locations.</p>
<h2 data-number="1.11" id="dynamics-of-transmission"><span class="header-section-number">1.11</span> Dynamics of Transmission</h2>
<p>Disease spread dynamics can be estimated using R<sub>0</sub>, the basic reproduction number, and R<sub>t</sub>, the effective reproduction number.
Accurate estimates of both are crucial to understanding the dynamics of infection and to predicting the effects of different interventions.
R<sub>0</sub> is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population <span class="citation" data-cites="16CD4voW1">(<a href="#ref-16CD4voW1" role="doc-biblioref">292</a>)</span>, and is one of the most important epidemiological parameters <span class="citation" data-cites="14l4fvU71">(<a href="#ref-14l4fvU71" role="doc-biblioref">293</a>)</span>.
A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered compartmental model <span class="citation" data-cites="GMDFEjtK SPInPew0">(<a href="#ref-GMDFEjtK" role="doc-biblioref">294</a>, <a href="#ref-SPInPew0" role="doc-biblioref">295</a>)</span>.
In this model, individuals move through three states: susceptible, infected, and recovered; two parameters, <span class="math inline">\(\gamma\)</span> and <span class="math inline">\(\beta\)</span>, specify the rate at which the infectious recover, and the infection transmission rate, respectively, and R<sub>0</sub> is estimated as the ratio of <span class="math inline">\(\beta\)</span> and <span class="math inline">\(\gamma\)</span> <span class="citation" data-cites="YubbhU7G 14l4fvU71">(<a href="#ref-14l4fvU71" role="doc-biblioref">293</a>, <a href="#ref-YubbhU7G" role="doc-biblioref">296</a>)</span>.
A pathogen can invade a susceptible population only if R<sub>0</sub> &gt; 1 <span class="citation" data-cites="14l4fvU71 JRaVgcNd">(<a href="#ref-14l4fvU71" role="doc-biblioref">293</a>, <a href="#ref-JRaVgcNd" role="doc-biblioref">297</a>)</span>.
The spread of an infectious disease at a particular time t can be quantified by R<sub>t</sub>, the effective reproduction number, which assumes that part of the population has already recovered (and thus gained immunity to reinfection) or that mitigating interventions have been put into place.
For example, if only a fraction S<sub>t</sub> of the population is still susceptible, R<sub>t</sub> = S<sub>t</sub> x R<sub>0</sub>.
When R<sub>t</sub> is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R<sub>t</sub> is less than 1, the proportion of the population that is infectious decreases.
R<sub>0</sub> and R<sub>t</sub> can be estimated directly from epidemiological data or inferred using susceptible-infected-recovered-type models.
To capture the dynamics of SARS-CoV-2 accurately, the addition of a fourth compartment, i.e. a susceptible-exposed-infectious-recovered model, may be appropriate because such models account for the relative lengths of incubation and infectious periods <span class="citation" data-cites="cWv3gKK">(<a href="#ref-cWv3gKK" role="doc-biblioref">298</a>)</span>.</p>
<p>Original estimates of R<sub>0</sub> for COVID-19 lie in the range R<sub>0</sub>=1.4-6.5 <span class="citation" data-cites="1E0r4uZy9 LHtuVmaq mHYmt0mv">(<a href="#ref-1E0r4uZy9" role="doc-biblioref">299</a>–<a href="#ref-mHYmt0mv" role="doc-biblioref">301</a>)</span>.
Variation in R<sub>0</sub> is expected between different populations, and the estimated values of R<sub>0</sub> discussed below are for specific populations in specific environments.
The different estimates of R<sub>0</sub> should not necessarily be interpreted as a range of estimates of the same underlying parameter.
In one study of international cases, the predicted value was R<sub>0</sub>=1.7 <span class="citation" data-cites="vITui6ac">(<a href="#ref-vITui6ac" role="doc-biblioref">302</a>)</span>.
In China (both Hubei province and nationwide), the value was predicted to lie in the range R<sub>0</sub>=2.0-3.6 <span class="citation" data-cites="nxM0rP5R WLc2UMgQ 1E0r4uZy9">(<a href="#ref-1E0r4uZy9" role="doc-biblioref">299</a>, <a href="#ref-nxM0rP5R" role="doc-biblioref">303</a>, <a href="#ref-WLc2UMgQ" role="doc-biblioref">304</a>)</span>.
Another estimate based on a cruise ship where an outbreak occurred predicted R<sub>0</sub>=2.28 <span class="citation" data-cites="10bBqMHH7">(<a href="#ref-10bBqMHH7" role="doc-biblioref">305</a>)</span>.
Susceptible-exposed-infectious-recovered model-derived estimates of R<sub>0</sub> range from 2.0 - 6.5 in China <span class="citation" data-cites="ITh0Anof 1BMU7sKbs Yj8Xh4Wz ZzrrVDoE">(<a href="#ref-ITh0Anof" role="doc-biblioref">306</a>–<a href="#ref-ZzrrVDoE" role="doc-biblioref">309</a>)</span> to R<sub>0</sub>=4.8 in France <span class="citation" data-cites="itj26agd">(<a href="#ref-itj26agd" role="doc-biblioref">310</a>)</span>.
Using the same model as for the French population, a study estimated R<sub>0</sub>=2.6 in South Korea <span class="citation" data-cites="itj26agd">(<a href="#ref-itj26agd" role="doc-biblioref">310</a>)</span>, which is consistent with other studies <span class="citation" data-cites="1H6clx00o">(<a href="#ref-1H6clx00o" role="doc-biblioref">311</a>)</span>.
From a meta-analysis of studies estimating R<sub>0</sub>, <span class="citation" data-cites="LHtuVmaq">(<a href="#ref-LHtuVmaq" role="doc-biblioref">300</a>)</span> the median R<sub>0</sub> was estimated to be 2.79 (IQR 1.16) based on twelve studies published between January 1 and February 7, 2020.</p>
<p>Inference of the effective reproduction number can provide insight into how populations respond to an infection and the effectiveness of interventions.
In China, R<sub>t</sub> was predicted to lie in the range 1.6-2.6 in January 2020, before travel restrictions <span class="citation" data-cites="tiRfUgvs">(<a href="#ref-tiRfUgvs" role="doc-biblioref">312</a>)</span>.
R<sub>t</sub> decreased from 2.35 one week before travel restrictions were imposed (January 23, 2020), to 1.05 one week after.
Using their model, the authors also estimated the probability of new outbreaks occurring.
Assuming individual-level variation in transmission comparable to that of MERS or SARS, the probability of a single individual exporting the virus and causing a large outbreak is 17-25%, and assuming variation like that of SARS and transmission patterns like those observed for COVID-19 in Wuhan, the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%.
An independent study came to similar conclusions, finding R<sub>t</sub>=2.38 in the two-week period before January 23 with a decrease to R<sub>t</sub> = 1.34 (using data from January 24 to February 3) or R<sub>t</sub>=0.98 (using data from January 24 to February 8) <span class="citation" data-cites="mHYmt0mv">(<a href="#ref-mHYmt0mv" role="doc-biblioref">301</a>)</span>.
In South Korea, R<sub>t</sub> was inferred for February through March 2020 in two cities, Daegu (the center of the outbreak) and Seoul <span class="citation" data-cites="1H6clx00o">(<a href="#ref-1H6clx00o" role="doc-biblioref">311</a>)</span>.
Metro data was also analyzed to estimate the effects of social distancing measures.
R<sub>t</sub> decreased in Daegu from around 3 to &lt;1 over the period that social distancing measures were introduced.
In Seoul, R<sub>t</sub> decreased slightly, but remained close to 1 (and larger than R<sub>t</sub> in Daegu).
These findings indicate that social distancing measures appeared to be effective in containing the infection in Daegu, but in Seoul, R<sub>t</sub> remained above 1, meaning secondary outbreaks remained possible.
The study also shows the importance of region-specific analysis: the large decline in case load nationwide was mainly due to the Daegu region and could mask persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do.
In Iran, estimates of R<sub>t</sub> declined from 4.86 in the first week to 2.1 by the fourth week after the first cases were reported <span class="citation" data-cites="TH4ymqvJ">(<a href="#ref-TH4ymqvJ" role="doc-biblioref">313</a>)</span>.
In Europe, analysis of 11 countries inferred the dynamics of R<sub>t</sub> over a time range from the beginning of the outbreak until March 28, 2020, by which point most countries had implemented major interventions (such as stay-at-home orders, public gathering bans, and school closures) <span class="citation" data-cites="Frya4XA4">(<a href="#ref-Frya4XA4" role="doc-biblioref">314</a>)</span>.
Across all countries, the mean R<sub>t</sub> before interventions began was estimated as 3.87; R<sub>t</sub> varied considerably, from below 3 in Norway to above 4.5 in Spain.
After interventions, R<sub>t</sub> decreased by an average of 64% across all countries, with mean R<sub>t</sub>=1.43.
The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden, which could be related to the fact that Sweden did not implement social distancing measures on the same scale as other countries.
The study concludes that while large changes in R<sub>t</sub> are observed, it is too early to tell whether the interventions put into place are sufficient to decrease R<sub>t</sub> below 1.</p>
<p>Evolution within SARS-CoV-2 has also driven changes in the estimated reproduction number for different populations at different times.
As of June 2021, the reproduction number had increased globally relative to 2020, and increased transmissibility over the wildtype variant was observed for the Alpha, Beta, Gamma, and Delta VOC <span class="citation" data-cites="AfaxKtHk">(<a href="#ref-AfaxKtHk" role="doc-biblioref">315</a>)</span>.
In the U.S. between December 2020 and January 2021, B.1.1.7 (Alpha) was estimated to have an increased transmission of 35 to 45% relative to common SARS-CoV-2 variants at the time, with B.1.1.7 the dominant SARS-CoV-2 variant in some places at some timepoints <span class="citation" data-cites="JsfUU71J">(<a href="#ref-JsfUU71J" role="doc-biblioref">316</a>)</span>.
This lineage was estimated to have increased transmissibility of 43 to 90% in the U.K. <span class="citation" data-cites="9L4bQ6n7">(<a href="#ref-9L4bQ6n7" role="doc-biblioref">317</a>)</span>.
An estimate of the reproduction number of B.1.1.7 in the U.K. from September to December 2020 yielded 1.59 overall and between 1.56 and 1.95 in different regions of the country <span class="citation" data-cites="tzOnKNrn">(<a href="#ref-tzOnKNrn" role="doc-biblioref">234</a>)</span>.
The Delta variant is particularly transmissible, and it has been estimated to be twice as transmissible than the wildtype variant of SARS-CoV-2 <span class="citation" data-cites="AfaxKtHk">(<a href="#ref-AfaxKtHk" role="doc-biblioref">315</a>)</span>.
A review of the literature describing the Delta variant identified a mean estimated R<sub>0</sub> of 5.08 <span class="citation" data-cites="1EI5i5dA1">(<a href="#ref-1EI5i5dA1" role="doc-biblioref">318</a>)</span>.
Such differences can affect fitness and therefore influence the relative contributions of different lineages to a given viral gene pool over time <span class="citation" data-cites="r8vdQFkf">(<a href="#ref-r8vdQFkf" role="doc-biblioref">319</a>)</span>.
Therefore, the evolution of the virus can result in shifts in the reproduction rate.</p>
<p>More generally, population-level epidemic dynamics can be both observed and modeled <span class="citation" data-cites="YubbhU7G">(<a href="#ref-YubbhU7G" role="doc-biblioref">296</a>)</span>.
Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to make predictions about possible future dynamics, such as the estimation of capacity needs <span class="citation" data-cites="NCcssjLV">(<a href="#ref-NCcssjLV" role="doc-biblioref">320</a>)</span> or the comparison of predicted outcomes among prevention and control strategies <span class="citation" data-cites="11CWFYFDL J4RCpntH">(<a href="#ref-11CWFYFDL" role="doc-biblioref">321</a>, <a href="#ref-J4RCpntH" role="doc-biblioref">322</a>)</span>.
Many current efforts to model R<sub>t</sub> have also led to tools that assist the visualization of estimates in real time or over recent intervals <span class="citation" data-cites="WA1R5R95 17CQnstjh">(<a href="#ref-WA1R5R95" role="doc-biblioref">323</a>, <a href="#ref-17CQnstjh" role="doc-biblioref">324</a>)</span>.
These are valuable resources, yet it is also important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.</p>
<h2 data-number="1.12" id="effect-of-vaccines-on-pathogenesis-and-community-spread"><span class="header-section-number">1.12</span> Effect of Vaccines on Pathogenesis and Community Spread</h2>
<p>The vaccine clinical trial data demonstrate a significant reduction in the likelihood of contracting symptomatic COVID-19, thereby succeeding in the primary goal of vaccination.
The mRNA vaccines in particular were initially so effective in preventing disease that they were also assumed to have an effect on the likelihood of transmission (e.g., venues requiring proof of vaccination).
However, in light of the reduced efficacy in response to VOC, it is especially important to consider whether this assumption is supported by the available evidence.</p>
<p>This question is made up of several components.
The crux is whether vaccinated individuals with a SARS-CoV-2 infection, regardless of symptom status, are as contagious as unvaccinated, infected individuals.
Additionally, as outlined above, an important qualification is that the variants of SARS-CoV-2 circulating at the time of each study must be considered in light of the effect of evolution on vaccine efficacy.</p>
<p>The phase II/III clinical trials evaluating the mRNA vaccines assessed vaccine efficacy based on COVID-19 diagnosis, thereby detecting only patients who received a diagnosis.
In order to identify patients infected with SARS-CoV-2 who did not receive a diagnosis, for example, potentially those who did not develop symptoms, it would be necessary to conduct routine PCR testing even in the absence of symptoms. <!-- Define PCR? -->
Prior to the development of vaccines, the evidence suggested that asymptomatic individuals could spread SARS-CoV-2.
Investigation of viral dynamics of asymptomatic infection in early 2020 indicated that asymptomatic patients continued to shed the virus for a duration similar to that of symptomatic patients <span class="citation" data-cites="ZU1ZF4SW">(<a href="#ref-ZU1ZF4SW" role="doc-biblioref">325</a>)</span> (although viral shedding should not be conflated with contagiousness without further investigation).
Another study found viral load to be higher in the nasopharyngeal/oropharyngeal samples of asymptomatic patients compared to symptomatic patients hospitalized due to symptoms and/or known exposure <span class="citation" data-cites="whGzxrkn">(<a href="#ref-whGzxrkn" role="doc-biblioref">326</a>)</span>.
However, the sample size in both of these studies was small, and a larger study found higher viral load in symptomatic than asymptomatic cases <span class="citation" data-cites="34wAjHW5">(<a href="#ref-34wAjHW5" role="doc-biblioref">327</a>)</span> along with a systematic review finding a reduced probability of asymptomatic transmission <span class="citation" data-cites="1CA0Sj7dn">(<a href="#ref-1CA0Sj7dn" role="doc-biblioref">328</a>)</span>.
While far from conclusive, these studies suggest that asymptomatic cases still cary a risk of transmitting SARS-CoV-2.</p>
<p>One important consideration is therefore how likely vaccinated individuals are to develop asymptomatic SARS-CoV-2.
Considering asymptomatic cases is necessary to establish a more complete picture of efficacy with respect to spread.<!-- To Do: Possible figure indicating pyramid of risk with and without vaccines?-->
Routine testing of healthcare workers in California who had received an mRNA vaccine revealed slightly higher rates of absolute risk for testing positive than those identified in the phase II/III trials, although the extent to which asymptomatic infection influenced these numbers was not investigated <span class="citation" data-cites="13llzZ2qN">(<a href="#ref-13llzZ2qN" role="doc-biblioref">329</a>)</span>.
Another study analyzed the results of COVID-19 screening tests administered to asymptomatic individuals prior to receiving certain medical services at the Mayo Clinic in several locations across the United States.
This study found patients who had received two doses of an mRNA vaccine to be 73% less likely to have asymptomatic COVID-19 than patients who had received zero doses <span class="citation" data-cites="dLmXTkx0">(<a href="#ref-dLmXTkx0" role="doc-biblioref">330</a>)</span>.
Because this study began on December 17, 2020, a date selected to coincide with the first day vaccines were available at the Mayo Clinic, this number may underestimate the efficacy of the vaccines given that many people eligible for early vaccination were at increased risk for exposure (e.g., healthcare workers and residents of long-term care facilities) <span class="citation" data-cites="dLmXTkx0">(<a href="#ref-dLmXTkx0" role="doc-biblioref">330</a>)</span>.
In Israel, a longitudinal study of nearly 12,000 healthcare workers found that of the 5,372 fully vaccinated people with Pfizer/BioNTech BNT162b2, 8 developed symptomatic COVID-19 (0.15%) and 19 developed asymptomatic COVID-19 (0.35%) <span class="citation" data-cites="zHE6Quu6">(<a href="#ref-zHE6Quu6" role="doc-biblioref">331</a>)</span>.
While the study itself analyzed the efficacy of the vaccine based on person-days, these findings also suggest that many or even the majority of SARS-CoV-2 infections in vaccinated individuals are likely to be asymptomatic.
Therefore, in addition to the symptomatic cases reported by the vaccine clinical trials, these findings suggest that asymptomatic cases can also occur in vaccinated people.
In the absence of symptoms, individuals are less likely to know to self-isolate, and therefore evaluating the effect of the vaccine on viral load is critical to understanding the role vaccinated individuals can play in spreading SARS-CoV-2.</p>
<p>Another question of interest is therefore whether vaccinated individuals positive for SARS-CoV-2 carry a similar viral load to unvaccinated individuals.
Viral load is often approximated by cycle threshold (C<sub>t</sub>), or the cycle at which viral presence is detected during RT-qPCR, with a lower C<sub>t</sub> corresponding to a greater viral load.
A prospective cohort study that evaluated front-line workers in six U.S. states from December 2020 to April 2021 reported a 40% reduction in viral load even with just a single dose of an mRNA vaccine <span class="citation" data-cites="D2ZCK63Y">(<a href="#ref-D2ZCK63Y" role="doc-biblioref">332</a>)</span>.
The vaccine also appeared to influence the time to viral clearance: the risk of having detectable levels of SARS-CoV-2 for more than one week was reduced by 66% in participants who had received at least one dose <span class="citation" data-cites="D2ZCK63Y">(<a href="#ref-D2ZCK63Y" role="doc-biblioref">332</a>)</span>.
However, this study compared the mean viral load across the two groups, meaning that these findings cannot be extrapolated across all points in the disease course.
Similarly, between December 2020 and February 2021, positive RT-qPCR tests were analyzed for almost 5,000 Israeli patients <span class="citation" data-cites="119cExL0k">(<a href="#ref-119cExL0k" role="doc-biblioref">333</a>)</span>.
C<sub>t</sub> was analyzed relative to when each patient received the first dose of the Pfizer mRNA vaccine.
A sharp increase in C<sub>t</sub> (corresponding to reduced viral load) was observed between days 11 and 12, consistent with what is known about the onset of immunity following vaccination.
This pattern therefore suggested a direct effect of vaccination on viral load.</p>
<p>Other studies, however, have not offered support for a reduced viral load in breakthrough cases.
In Singapore, which has strict protocols for screening individuals with potential COVID-19 exposure, a retrospective cohort of patients who tested positive for SARS-CoV-2 between April and June 2021 was analyzed to compare viral kinetics and symptom course between vaccinated and unvaccinated cases.
Vaccinated individuals who tested positive experienced fewer symptoms than unvaccinated, SARS-CoV-2-positive individuals and were more likely to be asymptomatic <span class="citation" data-cites="e2Qnnj6R">(<a href="#ref-e2Qnnj6R" role="doc-biblioref">334</a>)</span>.
Additionally, this study analyzed C<sub>t</sub> over time and found that, though the median values were similar between the two groups at disease onset, viral load appeared to decrease more rapidly in vaccinated cases <span class="citation" data-cites="e2Qnnj6R">(<a href="#ref-e2Qnnj6R" role="doc-biblioref">334</a>)</span>.
This study is likely to have evaluated a more accurate representation of all COVID-19 outcomes than has been feasible in most studies, but one limitation was that the RT-PCR reactions were conducted in many different facilities.
A third study investigated viral load (as approximated by C<sub>t</sub>) using samples processed in a single laboratory during the summer of 2021 <span class="citation" data-cites="N5OXLf7V">(<a href="#ref-N5OXLf7V" role="doc-biblioref">335</a>)</span>.
This study identified no significant differences in C<sub>t</sub> between fully vaccinated and unvaccinated cases, but this study used samples sent for diagnosis and was not longitudinal.
It offered the additional benefit of culturing samples to assess whether their C<sub>t</sub> threshold was likely to represent contagiousness and found that SARS-CoV-2 could be cultured from 51 of 55 samples with C<sub>t</sub> less than 25 (the cut-off used in many studies).
Another study of samples collected at two sites in San Francisco, one of which tested only asymptomatic individuals, reported no difference in C<sub>t</sub> between asymptomatic and symptomatic cases regardless of whether vaccination status was included in the model <span class="citation" data-cites="mgscHeDu">(<a href="#ref-mgscHeDu" role="doc-biblioref">336</a>)</span>.
Though each of these three studies offers distinct strengths and weaknesses, taken together, they suggest that viral load is likely to be similar in vaccinated and unvaccinated individuals, but that vaccinated individuals clear the virus more rapidly, meaning that the average viral load is lower over time.</p>
<p>Given the emergence of VOC, especially the Delta and Omicron variants, for which breakthrough infections are more common, the potential for vaccinated individuals to spread SARS-CoV-2 is not static over time.
In fact, studies reporting reduced viral load in vaccinated individuals collected samples, for the most part, prior to the emergence of the Delta variant’s dominance.
The emergence of this variant may partially account for why more recent studies tend to find no difference between viral load in vaccinated and unvaccinated cases.</p>
<p>Taken together, these findings can provide some insight into how vaccines influence community spread.
While vaccinated individuals may be more likely to experience asymptomatic infection, current evidence about viral load in asymptomatic versus symptomatic cases is ambiguous.
Similarly, no conclusions can be drawn about whether viral load is different in vaccinated versus unvaccinated cases.
Therefore, at present, the evidence suggests that vaccinated individuals who are infected can still contribute to community spread.
The one potential mitigating factor supported at present is that differences in the viral kinetics may result in vaccinated cases infecting fewer individuals over time due to a more rapid decrease in viral load <span class="citation" data-cites="e2Qnnj6R">(<a href="#ref-e2Qnnj6R" role="doc-biblioref">334</a>)</span>, although this study did not examine patterns in secondary transmission.
Thus, the virological evidence suggests that public health measures such as masking and distancing remain important even in areas with high vaccination rates.</p>
<h2 data-number="1.13" id="conclusions"><span class="header-section-number">1.13</span> Conclusions</h2>
<p>The novel coronavirus SARS-CoV-2 is the third HCoV to emerge in the 21st century, and research into previous HCoVs has provided a strong foundation for characterizing the pathogenesis and transmission of SARS-CoV-2.
Critical insights into how the virus interacts with human cells have been gained from previous research into HCoVs and other viral infections.
With the emergence of three devastating HCoV over the past twenty years, emergent viruses are likely to represent an ongoing threat.
Contextualizing SARS-CoV-2 alongside other viruses serves not only to provide insights that can be immediately useful for combating this virus itself but may also prove valuable in the face of future viral threats.</p>
<p>Host-pathogen interactions provide a basis not only for understanding COVID-19, but also for developing a response.
As with other HCoVs, the immune response to SARS-CoV-2 is likely driven by detection of its spike protein, which allows it to enter cells through ACE2.
Epithelial cells have also emerged as the major cellular target of the virus, contextualizing the respiratory and gastrointestinal symptoms that are frequently observed in COVID-19.
Many of the mechanisms that facilitate the pathogenesis of SARS-CoV-2 are currently under consideration as possible targets for the treatment or prevention of COVID-19 <span class="citation" data-cites="njpLhBui wgAGKcBj">(<a href="#ref-wgAGKcBj" role="doc-biblioref">2</a>, <a href="#ref-njpLhBui" role="doc-biblioref">3</a>)</span>.
Research in other viruses also provides a foundation for understanding the transmission of SARS-CoV-2 among people and can therefore inform efforts to control the virus’s spread.
Airborne forms of transmission (droplet and aerosol transmission) have emerged as the primary modes by which the virus spreads to new hosts.
Asymptomatic transmission was also a concern in the SARS outbreak of 2002-03 and, as in the current pandemic, presented challenges for estimating rates of infection <span class="citation" data-cites="QGjpUct1">(<a href="#ref-QGjpUct1" role="doc-biblioref">337</a>)</span>.
These insights are important for developing a public health response, such as the CDC’s shift in its recommendations surrounding masking <span class="citation" data-cites="nOGQjd1a">(<a href="#ref-nOGQjd1a" role="doc-biblioref">338</a>)</span>.</p>
<p>Even with the background obtained from research in SARS and MERS, COVID-19 has revealed itself to be a complex and difficult-to-characterize disease that has many possible presentations that vary with age.
Variability in presentation, including cases with no respiratory symptoms or with no symptoms altogether, were also reported during the SARS epidemic at the beginning of the 21st century <span class="citation" data-cites="QGjpUct1">(<a href="#ref-QGjpUct1" role="doc-biblioref">337</a>)</span>.
The variability of both which symptoms present and their severity have presented challenges for public health agencies seeking to provide clear recommendations regarding which symptoms indicate SARS-CoV-2 infection and should prompt isolation.
Asymptomatic cases add complexity both to efforts to estimate statistics such as R<sub>0</sub> and R<sub>t</sub>, which are critical to understanding the transmission of the virus, and IFR, which is an important component of understanding its impact on a given population.
The development of diagnostic technologies over the course of the pandemic has facilitated more accurate identification, including of asymptomatic cases <span class="citation" data-cites="hIxvVJ7F">(<a href="#ref-hIxvVJ7F" role="doc-biblioref">5</a>)</span>.
As more cases have been diagnosed, the health conditions and patient characteristics associated with more severe infection have also become more clear, although there are likely to be significant sociocultural elements that also influence these outcomes <span class="citation" data-cites="Up1vB19z">(<a href="#ref-Up1vB19z" role="doc-biblioref">339</a>)</span>.
While many efforts have focused on adults, and especially older adults because of the susceptibility of this demographic, additional research is needed to understand the presentation of COVID-19 and MIS-C in pediatric patients.
As more information is uncovered about the pathogenesis of HCoV and SARS-CoV-2 specifically, the diverse symptomatology of COVID-19 has and likely will continue to conform with the ever-broadening understanding of how SARS-CoV-2 functions within a human host.</p>
<p>While the SARS-CoV-2 virus is very similar to other HCoV in several ways, including in its genomic structure and the structure of the virus itself, there are also some differences that may account for differences in the COVID-19 pandemic compared to the SARS and MERS epidemics of the past two decades.
The R<sub>0</sub> of SARS-CoV-2 has been estimated to be similar to SARS-CoV-1 but much higher than that of MERS-CoV <span class="citation" data-cites="sP4wQEiM">(<a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>, although a higher R<sub>0</sub> has been estimated for some VOC.
While the structures of the viruses are very similar, evolution among these species may account for differences in their transmissibility and virulence.
For example, the acquisition of a furin cleavage site the S1/S2 boundary within the SARS-CoV-2 S protein may be associated with increased virulence.
Additionally, concerns have been raised about the accumulation of mutations within the SARS-CoV-2 species itself, and whether these could influence virulence <span class="citation" data-cites="17qiILENK">(<a href="#ref-17qiILENK" role="doc-biblioref">341</a>)</span>.
These novel variants may be resistant to vaccines and antibody treatments such as Bamlanivimab that were designed based on the wildtype spike protein <span class="citation" data-cites="njpLhBui rjHK19Dr">(<a href="#ref-njpLhBui" role="doc-biblioref">3</a>, <a href="#ref-rjHK19Dr" role="doc-biblioref">7</a>)</span>.
As a consequence of reliance on targeting the SARS-CoV-2 spike protein for many therapeutic and prophylactic strategies, increased surveillance is required to rapidly identify and prevent the spread of novel SARS-CoV-2 variants with alterations to the spike protein.
The coming of age of genomic technologies has made these types of analyses feasible, and genomics research characterizing changes in SARS-CoV-2 along with temporal and spatial movement is likely to provide additional insights into whether within-species evolution influences the effect of the virus on the human host.
Additionally, the rapid development of sequencing technologies over the past decade has made it possible to rapidly characterize the host response to the virus.
For example, proteomics analysis of patient-derived cells revealed candidate genes whose regulation is altered by SARS-CoV-2 infection, suggesting possible approaches for pharmaceutical invention and providing insight into which systems are likely to be disrupted in COVID-19 <span class="citation" data-cites="NLoN4aYj">(<a href="#ref-NLoN4aYj" role="doc-biblioref">195</a>)</span>.
As more patient data becomes available, the biotechnological advances of the 2000s are expected to allow for more rapid identification of potential drug targets than was feasible during the SARS, or even MERS, pandemic.</p>
<p>Thus, the COVID-19 crisis continues to evolve, but the insights acquired over the past 20 years of HCoV research have provided a solid foundation for understanding the SARS-CoV-2 virus and the disease it causes.
As the scientific community continues to respond to COVID-19 and to elucidate more of the relationships between pathogenesis, transmission, host regulatory responses, and symptomatology, this understanding will no doubt continue to evolve and to reveal additional connections among virology, pathogenesis, and health.
This review represents a collaboration between scientists from diverse backgrounds to contextualize this virus at the union of many different biological disciplines <span class="citation" data-cites="EM9YkiOF">(<a href="#ref-EM9YkiOF" role="doc-biblioref">4</a>)</span>.
At present, understanding the SARS-CoV-2 virus and its pathogenesis is critical to a holistic understanding of the COVID-19 pandemic.
In the future, interdisciplinary work on SARS-CoV-2 and COVID-19 may guide a response to a new viral threat.</p>
<h1 data-number="2" id="evolutionary-perspectives-on-sars-cov-2"><span class="header-section-number">2</span> Evolutionary Perspectives on SARS-CoV-2</h1>
<h2 data-number="2.1" id="abstract-1"><span class="header-section-number">2.1</span> Abstract</h2>
<h2 data-number="2.2" id="importance-1"><span class="header-section-number">2.2</span> Importance</h2>
<!-- Text to incorporate (from vaccines): -->
<!--Move to evolution ms
Some of the gene mutations carried by Omicron, of which there are between 30-37 in the spike gene (15 in the RBD), have previously been associated with increased transmission, greater affinity for ACE2, and escape from neutralizing antibodies when they have been detected in other VOC [@doi:10.1101/2021.11.11.21266068; @doi:10.1016/j.cell.2021.06.020; @doi:10.1101/2021.12.12.472269; @url:https://www.cdc.gov/coronavirus/2019-ncov/science/science-briefs/scientific-brief-omicron-variant.html; @doi:10.1101/2021.12.14.21267772; @doi:10.1101/2021.12.12.472269].
However, multiple animal study preprints suggested that the Omicron variant may not be as severe on the respiratory system as previous SARS-CoV-2 variants as evidenced by reduced lung infectivity, reduced SARS-CoV-2 RNA detection in the lung, and reduced inflammation and pathogenicity in animals [@doi:10.21203/rs.3.rs-1211792/v1; @doi:10.1101/2021.12.26.474085; @doi:10.1101/2021.12.24.474086; @doi:10.1101/2022.01.02.474743; @doi:10.1101/2021.12.17.473248].
-->
<!--Move to evolution
Vaccine efficacy is not the only pharmacological intervention affected by VOC.
Some existing therapeutics, including monoclonal antibody treatments like Bamlanivimab (AbCellera Biologics/ Eli Lilly), were ineffective against the Omicron variant.
Indeed, only Sotrovimab (Vir Biotechnology/GSK) and Tixagevimab (AstraZeneca) to a much lesser extent could effectively neutralize the Omicron variant out of 7 tested monoclonal antibodies [@doi:10.1101/2021.12.14.21267772], which has been verified by others [@doi:10.1101/2021.12.12.472269].
The antigenic shift of the Omicron variant does raise concerns for future VOC and what effects they may have on future vaccines and therapeutics.
-->
<h2 data-number="2.3" id="introduction-1"><span class="header-section-number">2.3</span> Introduction</h2>
<p>The emergence of what is now known to be the pathogen SARS-CoV-2 has dramatically reshaped modern life for the past two years.
The genomic revolution provided the tools needed to understand the virus in ways that were not feasible during previous pandemics.
For example, the first genome sequence of the pathogen was released on January 3, 2020, providing valuable information about the pathogen within a month and a half of the first known cases.
As the pandemic has unfolded, evolutionary questions and methods of investigation have framed the scientific approach to understanding the virus.
These questions have evolved along with the pandemic.
Thus far, five major evolutionary questions have emerged.
The first was “what is it?”, the second “where did it come from?”, the third and fourth address “whom does it affect?”, the fifth “how is it changing?” and the sixth “what is next?”
Evolutionary biology provides a framework through which these questions can be evaluated and explored.</p>
<h2 data-number="2.4" id="question-1-what-is-it"><span class="header-section-number">2.4</span> Question 1: What Is It?</h2>
<p>What is now known as SARS-CoV-2 emerged in November 2019 as an unknown pathogen causing a cluster of pneumonia cases in Wuhan, China.
The initial genome sequence, which was released in early January 2020, revealed the pathogen to be a novel coronavirus <span class="citation" data-cites="Bp847Lfa">(<a href="#ref-Bp847Lfa" role="doc-biblioref">11</a>)</span>.
Although most coronaviruses show little transmission in humans, several human coronaviruses (HCoV) have been identified since the 1960s.
Therefore, in the early days of the pandemic, many strategies to understand or manage the emergent viral threat focused on contextualizing it amongst better-studied coronaviruses.</p>
<p>Many people have previously been infected by an HCoV.
Approximately one-third of common cold infections are thought to be caused by four seasonal HCoV: <em>Human coronavirus 229E</em> (HCoV-229E), <em>Human coronavirus NL63</em> (HCoV-NL63), <em>Human coronavirus OC43</em> (HCoV-OC43), and <em>Human coronavirus HKU1</em> (HCoV-HKU1) <span class="citation" data-cites="UBTUyMGV YAhVzI0A SnIgqxRO">(<a href="#ref-UBTUyMGV" role="doc-biblioref">342</a>–<a href="#ref-SnIgqxRO" role="doc-biblioref">344</a>)</span>.
The first HCoV were identified in the 1960s: HCoV-229E in 1965 <span class="citation" data-cites="gaAv88da">(<a href="#ref-gaAv88da" role="doc-biblioref">345</a>)</span> and HCoV-OC43 in 1967 <span class="citation" data-cites="q6rFk3Kl">(<a href="#ref-q6rFk3Kl" role="doc-biblioref">346</a>)</span>.
Both of these viruses typically cause cold-like symptoms, including upper and lower respiratory infections <span class="citation" data-cites="cEFaGSlu Jnb5pce4 nRPNH9B2">(<a href="#ref-cEFaGSlu" role="doc-biblioref">347</a>–<a href="#ref-nRPNH9B2" role="doc-biblioref">349</a>)</span>, but they have also been associated with gastrointestinal symptoms <span class="citation" data-cites="KEsllU9p">(<a href="#ref-KEsllU9p" role="doc-biblioref">350</a>)</span>.
Two additional HCoV were subsequently identified <span class="citation" data-cites="URhDY7s9 VgGISeiE">(<a href="#ref-URhDY7s9" role="doc-biblioref">351</a>, <a href="#ref-VgGISeiE" role="doc-biblioref">352</a>)</span>.
In 2003, HCoV-NL63 <span class="citation" data-cites="URhDY7s9">(<a href="#ref-URhDY7s9" role="doc-biblioref">351</a>)</span> was first identified in a 7-month-old infant and then in clinical specimens collected from seven additional patients, five of whom were infants younger than 1 year old and the remainder of whom were adults.
CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and then found in samples collected from a second adult patient <span class="citation" data-cites="VgGISeiE">(<a href="#ref-VgGISeiE" role="doc-biblioref">352</a>)</span>.
These viruses are associated with respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals <span class="citation" data-cites="rDtyU6cv">(<a href="#ref-rDtyU6cv" role="doc-biblioref">353</a>)</span>, and also have gastrointestinal involvement in some cases <span class="citation" data-cites="KEsllU9p">(<a href="#ref-KEsllU9p" role="doc-biblioref">350</a>)</span>.</p>
<p>In addition to these relatively mild HCoV, however, highly pathogenic human coronaviruses have been identified, including <em>Severe acute respiratory syndrome-related coronavirus</em> (SARS-CoV or SARS-CoV-1) and <em>Middle East respiratory syndrome-related coronavirus</em> (MERS-CoV) <span class="citation" data-cites="G5NJrE75 UBTUyMGV uXEsLlzb">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>, <a href="#ref-UBTUyMGV" role="doc-biblioref">342</a>, <a href="#ref-uXEsLlzb" role="doc-biblioref">354</a>)</span>.
At the time that SARS-CoV-1 emerged in the early 2000s, no HCoV had been identified in almost 40 years <span class="citation" data-cites="G5NJrE75">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>)</span>.
The first case of SARS was reported in November 2002 in the Guangdong Province of China, and over the following month, the disease spread more widely within China and then into several countries across multiple continents <span class="citation" data-cites="sP4wQEiM G5NJrE75">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>, <a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>.
Unlike previously identified HCoV, SARS was much more severe, with an estimated death rate of 9.5% <span class="citation" data-cites="sP4wQEiM">(<a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>.
It was also highly contagious via droplet transmission, with a basic reproduction number (R<sub>0</sub>) of 4 (i.e., each person infected was estimated to infect four other people) <span class="citation" data-cites="sP4wQEiM">(<a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>.</p>
<p>However, the identity of the virus behind the infection remained unknown until April of 2003, when the SARS-CoV-1 virus was identified through a worldwide scientific effort spearheaded by the WHO <span class="citation" data-cites="G5NJrE75">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>)</span>.
SARS-CoV-1 belonged to a distinct lineage from the two other HCoV known at the time <span class="citation" data-cites="sP4wQEiM">(<a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>.
By July 2003, the SARS outbreak was officially determined to be under control, with the success credited to infection management practices <span class="citation" data-cites="G5NJrE75">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>)</span>.
A decade later, a second outbreak of severe respiratory illness associated with a coronavirus emerged, this time in the Arabian Peninsula.
This disease, known as Middle East respiratory syndrome (MERS), was linked to another novel coronavirus, MERS-CoV.
The fatality rate associated with MERS is much higher than that of SARS, at almost 35%, but the disease is much less easily transmitted, with an R<sub>0</sub> of 1 <span class="citation" data-cites="sP4wQEiM">(<a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>.
Although MERS is still circulating, its low reproduction number has allowed for its spread to be contained <span class="citation" data-cites="sP4wQEiM">(<a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>.
The COVID-19 pandemic is thus associated with the seventh HCoV to be identified and the fifth since the turn of the millennium, though additional HCoVs may be in circulation but remain undetected (e.g., <span class="citation" data-cites="iOksGYVn">(<a href="#ref-iOksGYVn" role="doc-biblioref">355</a>)</span>).</p>
<p>Following the release of the SARS-CoV-2 genome sequence, multiple research groups sequenced the genomes of SARS-CoV-2 specimens identified in clinical samples.
These samples were primarily collected from patients’ lower respiratory tract, namely bronchoalveolar lavage fluid (BALF), and the upper respiratory tract, in the form of throat and nasopharyngeal swabs <span class="citation" data-cites="vHq0hOWQ VSkK7CeP 1Fqilxaum">(<a href="#ref-1Fqilxaum" role="doc-biblioref">19</a>, <a href="#ref-VSkK7CeP" role="doc-biblioref">207</a>, <a href="#ref-vHq0hOWQ" role="doc-biblioref">356</a>)</span>.
Integration of these sequences allowed for a more complete picture of the viral genome.
Analysis of the viral genome revealed significant sequence homology with two known HCoV: the novel coronavirus shared about 79% sequence identity with SARS-CoV-1 and 50% with MERS-CoV <span class="citation" data-cites="1Fqilxaum">(<a href="#ref-1Fqilxaum" role="doc-biblioref">19</a>)</span>.
Therefore, this early phylogenetic analysis of the novel coronavirus allow its similarity to other, known viruses to be established.
SARS-CoV-1 and MERS-CoV were ultimately managed largely through infection management practices (e.g., mask wearing) and properties of the virus itself (i.e., low rate of transmission), respectively <span class="citation" data-cites="G5NJrE75 sP4wQEiM">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>, <a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>.
Research in response to prior outbreaks of HCoV-borne infections, such as SARS and MERS, provided a strong foundation for hypotheses about the pathogenesis of SARS-CoV-2 as well as potential diagnostic and therapeutic approaches, as we review elsewhere <span class="citation" data-cites="GdZc4Yyd njpLhBui 1HWmcSB20 rjHK19Dr">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>, <a href="#ref-njpLhBui" role="doc-biblioref">3</a>, <a href="#ref-1HWmcSB20" role="doc-biblioref">6</a>, <a href="#ref-rjHK19Dr" role="doc-biblioref">7</a>)</span>.
Therefore, this phylogenetic information was valuable for gaining an understanding of the pathogen and identifying strategies to manage it.</p>
<h2 data-number="2.5" id="question-2-where-did-it-come-from"><span class="header-section-number">2.5</span> Question 2: Where Did It Come From?</h2>
<p>Despite the high degree of similarity to SARS-CoV-1, even greater sequence identity was observed between SARS-CoV-2 and zoonotic coronaviruses.
A 2001 literature review estimated that 61% of human pathogens have a zoonotic origin <span class="citation" data-cites="AwAyxtZs">(<a href="#ref-AwAyxtZs" role="doc-biblioref">357</a>)</span>.
A zoonotic disease, or zoonosis, arises when a pathogen can both a) infect and b) cause a disease in humans <span class="citation" data-cites="1EdMJE4DK">(<a href="#ref-1EdMJE4DK" role="doc-biblioref">358</a>)</span>.
As a result, the risk of zoonotic disease increases when there is substantial interaction between humans and wildlife <span class="citation" data-cites="1EdMJE4DK">(<a href="#ref-1EdMJE4DK" role="doc-biblioref">358</a>)</span>.
Many factors can influence this human/wildlife interface and therefore the risk of zoonotic transmission events <span class="citation" data-cites="1EdMJE4DK scqqnokl">(<a href="#ref-1EdMJE4DK" role="doc-biblioref">358</a>, <a href="#ref-scqqnokl" role="doc-biblioref">359</a>)</span>.</p>
<p>In the SARS epidemic, SARS-CoV-1 was also thought to have emerged in a live animal market.
A survey of a market in Shenzhen, China revealed that individuals from two carnivore species, namely several masked palm civets (<em>Paguma larvata</em>) and one raccoon dog (<em>Nyctereutes procyonoides</em>), were likely carriers of SARS-CoV-1, despite presenting as healthy <span class="citation" data-cites="ddJ8UwvM">(<a href="#ref-ddJ8UwvM" role="doc-biblioref">360</a>)</span>.
However, further analysis suggested that these species might be only intermediate hosts who were exposed in the market setting <span class="citation" data-cites="dwmFUaCY">(<a href="#ref-dwmFUaCY" role="doc-biblioref">361</a>)</span>.
A closely related virus was identified in Chinese horseshoe bats (<em>Rhinolophus sinicus</em>), but the sequence identity was only 88% with SARS-CoV-1 <span class="citation" data-cites="TvDJAYpy">(<a href="#ref-TvDJAYpy" role="doc-biblioref">362</a>)</span>.
Therefore, the species of origin for SARS-CoV-1 remains unresolved.</p>
<p>In the case of SARS-CoV-2, early interest for the emergence of the pathogen turned to live-animal markets in Wuhan <span class="citation" data-cites="hWiB2Isn pekmozuU">(<a href="#ref-hWiB2Isn" role="doc-biblioref">363</a>, <a href="#ref-pekmozuU" role="doc-biblioref">364</a>–add-to-Wuhan-riddle)</span>, where it would later emerge that many animals were sold suffering from poor health and hygiene <span class="citation" data-cites="QACJGDk7">(<a href="#ref-QACJGDk7" role="doc-biblioref">365</a>)</span>.
A large percentage of early patients had visited the Huanan seafood market in Wuhan, and next-generation sequencing of samples collected from nine patients, eight of whom had visited the market, revealed extremely high sequence identity (99.98%), indicative of rapid spread <span class="citation" data-cites="1Fqilxaum">(<a href="#ref-1Fqilxaum" role="doc-biblioref">19</a>)</span>.
The sequence of the viral pathogen collected from these patients was also compared to known zoonotic pathogens.
In particular, genomic research quickly highlighted significant similarity (about 88% sequence identity) between SARS-CoV-2 and bat-derived SARS-like coronaviruses, namely bat-SL-CoVZC45 and bat-SL-CoVZXC21 <span class="citation" data-cites="1Fqilxaum">(<a href="#ref-1Fqilxaum" role="doc-biblioref">19</a>)</span>.
Other analyses have reported even greater similarity between SARS-CoV-2 and the bat coronavirus BatCoV-RaTG13, with shared sequence identity as high as 96.2% <span class="citation" data-cites="VSkK7CeP UUAeVUaR">(<a href="#ref-VSkK7CeP" role="doc-biblioref">207</a>, <a href="#ref-UUAeVUaR" role="doc-biblioref">211</a>)</span>.
Bats are well-established as a disease reservoir, including for RNA viruses <span class="citation" data-cites="BUHjrVM8 12G9bKpJg FA1fM4ii">(<a href="#ref-BUHjrVM8" role="doc-biblioref">366</a>–<a href="#ref-FA1fM4ii" role="doc-biblioref">368</a>)</span>.
This evidence therefore suggested that the virus may have emerged as a result of zoonotic transfer of a virus from bats to humans, with the wildlife trade considered a potential source of exposure.</p>
<p>Nevertheless, some fragments of the genome differ between SARS-CoV-2 and RATG13 by up to 17%, suggesting a complex natural selection process.
Additionally, SARS-CoV-2 is closely related (91.02%) to a novel coronaviruses identified in Malayan pangolins (<em>Manis javanica</em>) infected with a respiratory disease in October 2019 <span class="citation" data-cites="ZmBmnPMY">(<a href="#ref-ZmBmnPMY" role="doc-biblioref">369</a>)</span>.
Although the genome-wide sequence identity was lower between SARS-CoV-2 and this pangolin virus than BatCoV-RaTG13, its particularly high similarity in the receptor binding domain (RBD) of the spike (<em>S</em>) gene with SARS-CoV-2 drew further attention <span class="citation" data-cites="Lze8f0UK ZmBmnPMY">(<a href="#ref-ZmBmnPMY" role="doc-biblioref">369</a>, <a href="#ref-Lze8f0UK" role="doc-biblioref">370</a>)</span>.
The SARS-CoV-2 RBD differs from the pangolin coronavirus RBD by only one amino acid change <span class="citation" data-cites="ZmBmnPMY">(<a href="#ref-ZmBmnPMY" role="doc-biblioref">369</a>)</span>, and the sequence identity between the regions is 97.4% <span class="citation" data-cites="Lze8f0UK">(<a href="#ref-Lze8f0UK" role="doc-biblioref">370</a>)</span>.
Pangolins were therefore identified as a potential intermediate host of SARS-CoV-2 between bats and humans.</p>
<p>However, data collected from May 2017 to November 2019 by a research team interested in tick-borne illnesses identified no bats or pangolins sold at these markets leading up to the emergence of COVID-19 <span class="citation" data-cites="QACJGDk7">(<a href="#ref-QACJGDk7" role="doc-biblioref">365</a>)</span>.
Additionally, endemic bat species are typically in hibernation at the time that SARS-CoV-2 emerged <span class="citation" data-cites="1Fqilxaum">(<a href="#ref-1Fqilxaum" role="doc-biblioref">19</a>)</span>.
Therefore, it is possible that animals associated with these markets were infected by bats, but it is not clear whether the disease emerged in a different location and/or whether it is associated with a different species.
There were 38 species observed at the market in the 2.5 years leading up to the emergence of SARS-CoV-2, indicating significant diversity in the animals with which humans were interacting <span class="citation" data-cites="QACJGDk7">(<a href="#ref-QACJGDk7" role="doc-biblioref">365</a>)</span>.
As with SARS-CoV-1, the species of origin for SARS-CoV-2 therefore remains unresolved.</p>
<p>Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory – either purposely engineered and released, or escaped – and instead evolved naturally in an animal host <span class="citation" data-cites="NEIoHDjq">(<a href="#ref-NEIoHDjq" role="doc-biblioref">371</a>)</span>.
However, potentially due to public misunderstanding about recombination and complex evolutionary processes like coevolution, the similarity to pangolin <em>S</em> has resulted in popular conspiracy theories that the virus did not arise naturally.
The similarity of <em>S</em> to that of pangolin viruses could arise from either recombination or coevolution <span class="citation" data-cites="UUAeVUaR wz7kPyn2">(<a href="#ref-UUAeVUaR" role="doc-biblioref">211</a>, <a href="#ref-wz7kPyn2" role="doc-biblioref">372</a>)</span>, rather than requiring human intervention.
Such suspicions may also have been fueled, in part, by the lack of well-characterized bat coronaviruses which means that SARS-CoV-2 is still relatively derived from documented coronaviruses surveyed in bats <span class="citation" data-cites="37lIze3F">(<a href="#ref-37lIze3F" role="doc-biblioref">373</a>)</span>.
While it has been suggested that more thorough investigation of the origins of COVID-19 may have some value <span class="citation" data-cites="13xHe6Stn">(<a href="#ref-13xHe6Stn" role="doc-biblioref">374</a>)</span>, in many cases, support for the “lab-leak” theory is politically motivated <span class="citation" data-cites="CS3vp1Bh">(<a href="#ref-CS3vp1Bh" role="doc-biblioref">375</a>)</span>.
A more robust panel of zoonotic viruses against which to compare SARS-CoV-2 would allow for conclusive dismissal of these politicized claims, underscoring another potential benefit of more thorough monitoring of zoonotic diseases.
More importantly, it would allow researchers to have a better understanding of and to community concerns about potential emerging viral threats.</p>
<h2 data-number="2.6" id="question-3-which-species-are-susceptible"><span class="header-section-number">2.6</span> Question 3: Which Species Are Susceptible?</h2>
<p>Given the strong evidence for a zoonotic origin of SARS-CoV-2, another evolutionary question that received significant attention, especially early on, was whether humans could infect other species with SARS-CoV-2.
In the modern age, opportunities for human-to-animal transmission events could arise in interactions with companion animals, zoo animals, house pests, hunting, urbanized wildlife, and livestock.
Outbreaks of zoonotic diseases have been known to originate in environments such as zoos, farms, and petting zoos <span class="citation" data-cites="XrPq1Dvp">(<a href="#ref-XrPq1Dvp" role="doc-biblioref">376</a>)</span>, indicating that disease transmission is likely to be possible in these contexts.
Additionally, many coronaviruses infect animals and have been the subject of veterinary medical investigations and vaccine development efforts due to their effect on the health of companion and agricultural animals <span class="citation" data-cites="10oGRZVIE">(<a href="#ref-10oGRZVIE" role="doc-biblioref">377</a>)</span>.
Concerns about anthroponotic (human-to-animal) transmission focused on a few issues.
First, if animal species were susceptible to COVID-19-like infection, in addition to concerns about animal health, infections in livestock could have significant effects on food supply chains.
Additionally, even if pathology in these species was limited, if they could serve as viral reservoirs, then they would pose additional risk to humans.
The breadth of species susceptible to infection by a pathogen is known as the pathogen’s host range <span class="citation" data-cites="gPVseu0D">(<a href="#ref-gPVseu0D" role="doc-biblioref">378</a>)</span>.
Understanding the host-pathogen relationship throughout SARS-CoV-2’s host range can therefore offer valuable information for managing the spread of SARS-CoV-2.</p>
<p>The phylogeny of the species implicated in the origination of COVID-19 suggested that the host range of SARS-CoV-2 could encompass many species with a high level of interaction with humans.
Humans last shared an ancestor with bats and pangolins almost 100 million years ago <span class="citation" data-cites="zoq1a1or">(<a href="#ref-zoq1a1or" role="doc-biblioref">379</a>)</span>.
Bats belong to the order Chiroptera and pangolins to Pholidota, which both belong to the clade <em>Pegasoferae</em> <span class="citation" data-cites="Z0OkYpgf 89XVYM09 1EDurLKKP">(<a href="#ref-Z0OkYpgf" role="doc-biblioref">380</a>–<a href="#ref-1EDurLKKP" role="doc-biblioref">382</a>)</span>.
They are closely related to many other species that have close relationships with humans, namely odd-toed ungulates (Euungulata) and carnivores (Carnivora) <span class="citation" data-cites="Z0OkYpgf 89XVYM09 1EDurLKKP hmVYftwt">(<a href="#ref-Z0OkYpgf" role="doc-biblioref">380</a>–<a href="#ref-hmVYftwt" role="doc-biblioref">383</a>)</span>.
The part of the evolutionary tree that includes both humans and the Pegasoferae encompasses many species of significant social and economic importance.
Therefore, concerns were raised that the species with which humans have close interactions, many of which are much more closely related to bats and pangolins than humans are, could also be infected.
It seemed plausible that the host range could include both livestock, many of which are odd-toed ungulates, and companion animals, many of which are carnivores.
Infection of these animals was identified as a major concern <span class="citation" data-cites="j2EJdPne">(<a href="#ref-j2EJdPne" role="doc-biblioref">384</a>)</span>.</p>
<p>Genomic analyses seeking to identify which species were likely to be susceptible focused largely on the comparative genetics of angiotensin-converting enzyme 2 (ACE2).
ACE2 is the primary protein used by SARS-CoV-2 to enter the cell (see <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span>).
Recognition of this protein is largely determined by domains in the S1 subunit of the RBD <span class="citation" data-cites="qcVbT0w4">(<a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>)</span>.
Alignment of the <em>ACE2</em> sequence from 19 species revealed high conservation among mammals <span class="citation" data-cites="19JK8Xa1b">(<a href="#ref-19JK8Xa1b" role="doc-biblioref">385</a>)</span>.
This analysis suggested that non-human primates (three monkey and two ape species), companion animals (dogs and cats), and livestock (both odd- and even-toed ungulates) may all be susceptible to SARS-CoV-2 <span class="citation" data-cites="19JK8Xa1b">(<a href="#ref-19JK8Xa1b" role="doc-biblioref">385</a>)</span>.
Similarly, another study conducted an <em>in silico</em> analysis of ACE2 protein structures and their predicted binding to SARS-CoV-2 for 410 vertebrate species <span class="citation" data-cites="1FxiM6ws1">(<a href="#ref-1FxiM6ws1" role="doc-biblioref">386</a>)</span>.
The species identified as having the highest predicted binding affinities were all primates, including humans.
Other taxa with high predicted affinities included other primates, rodents, even-toed ungulates (namely, several species of cetaceans and deer), and anteaters.
Reindeer were the only domesticated species predicted to belong to either of these groups, but many common zoo animal species with threatened or worse IUCN risk status were identified as at risk.</p>
<p>Considering the evidence generated by <em>in silico</em> studies, it may not be surprising that many cases of reverse zoonotic, or anthroponotic, SARS-CoV-2 transmission have been reported.
Ferrets (<em>Mustela furo</em>) as well as cats and dogs were reported to be susceptible to SARS-CoV-2 in an experimental infection study <span class="citation" data-cites="CZiN878H">(<a href="#ref-CZiN878H" role="doc-biblioref">387</a>)</span>.
The earliest reported anthroponotic transmission events were observed in house pets, primarily cats (<em>Felis catus</em>) <span class="citation" data-cites="ELDaiz1W 17761toGa Ih3JjS7V">(<a href="#ref-ELDaiz1W" role="doc-biblioref">388</a>–<a href="#ref-Ih3JjS7V" role="doc-biblioref">390</a>)</span>.
Similarly, cases of SARS-CoV-2 infection have been reported in dogs (<em>Canis familiaris</em>): two of fifteen dogs monitored for SARS-CoV-2 by the Hong Kong Agriculture, Fisheries, and Conservation Department during the owners’ quarantine in March 2020 were found to be positive for SARS-CoV-2 <span class="citation" data-cites="17761toGa">(<a href="#ref-17761toGa" role="doc-biblioref">389</a>)</span>.
Comparing estimates in studies where cats (<em>Felis catus</em>) living with SARS-CoV-2-positive humans were tested for SARS-CoV-2 suggest that 6 to 15% of house cats may become infected <span class="citation" data-cites="ELDaiz1W">(<a href="#ref-ELDaiz1W" role="doc-biblioref">388</a>)</span>, and a large-scale study of pet dogs and cats in Italy suggested that 4.5% of cats and 12.8% of dogs from known COVID-19-positive households had developed antibodies to the virus <span class="citation" data-cites="KsMjmcLV">(<a href="#ref-KsMjmcLV" role="doc-biblioref">391</a>)</span>.
Some of these SARS-CoV-2-positive domestic carnivores have also shown clinical symptoms <span class="citation" data-cites="14IpxqUxX">(<a href="#ref-14IpxqUxX" role="doc-biblioref">392</a>)</span>, and a pilot study of seven cats and three dogs found that cats, but not dogs, shed SARS-CoV-2 virus for several days after viral challenge, although none of the animals were symptomatic <span class="citation" data-cites="1HOJDUg7F">(<a href="#ref-1HOJDUg7F" role="doc-biblioref">393</a>)</span>.
A few dogs and cats have reportedly died after becoming infected with SARS-CoV-2, although in most cases whether the virus is causally related to the death is unclear <span class="citation" data-cites="18ycM4YxO 1D5YVPEpJ yvRoFaEb 18JLC1Wre vTnD9MOk irb2yWQa">(<a href="#ref-18ycM4YxO" role="doc-biblioref">394</a>–<a href="#ref-yvRoFaEb" role="doc-biblioref">396</a>, <a href="#ref-18JLC1Wre" role="doc-biblioref">397</a>/?sh=4b653381275e, <a href="#ref-vTnD9MOk" role="doc-biblioref">398</a>, <a href="#ref-irb2yWQa" role="doc-biblioref">399</a>)</span>.</p>
<p>Domestic pests, on the other hand, seem to be less susceptible to SARS-CoV-2.
In the comparative genomic analysis of ACE2, the two rodent species analyzed, despite being the most phylogenetically similar to humans aside from the other primates, showed the most sequence divergence in <em>ACE2</em> <span class="citation" data-cites="19JK8Xa1b">(<a href="#ref-19JK8Xa1b" role="doc-biblioref">385</a>)</span>.
This finding was supported by experimental evidence that SARS-CoV-2 cannot use mouse (<em>Mus musculus</em>) ACE2 for cell entry <span class="citation" data-cites="VSkK7CeP">(<a href="#ref-VSkK7CeP" role="doc-biblioref">207</a>)</span>.
In fact, research using murine models to study SARS and COVID-19 therefore uses transgenic mice designed to be sensitive to the virus (as summarized in <span class="citation" data-cites="IumrstNg">(<a href="#ref-IumrstNg" role="doc-biblioref">400</a>)</span>).</p>
<p>Similarly, SARS-CoV-2 in livestock also raised concern because of the potential effect on food supply.
However, studies using <em>in vivo</em> viral challenge reported that livestock species in general do not develop clinical manifestations of SARS-CoV-2 and do not shed infectious virus <span class="citation" data-cites="EVS2FryQ CZiN878H">(<a href="#ref-CZiN878H" role="doc-biblioref">387</a>, <a href="#ref-EVS2FryQ" role="doc-biblioref">401</a>)</span>.
<em>In vitro</em> exposure to SARS-CoV-2 suggested that sheep (<em>Ovis aries</em>), but not cattle (<em>Bos taurus</em>), might be susceptible to infection, but <em>in vivo</em> viral challenge suggested that sheep did not show notable susceptibility to infection <span class="citation" data-cites="17bpCXS4N">(<a href="#ref-17bpCXS4N" role="doc-biblioref">402</a>)</span>.
Similarly, analyses of antibody response <span class="citation" data-cites="BS9gjNG4">(<a href="#ref-BS9gjNG4" role="doc-biblioref">403</a>)</span> suggested that sheep exposed to a high level of human interaction did not appear to have developed infections.
Following viral challenge of several species, including cattle, sheep, and horses (<em>Equus ferus caballus</em>), none were found to shed culturable levels of virus <span class="citation" data-cites="EVS2FryQ">(<a href="#ref-EVS2FryQ" role="doc-biblioref">401</a>)</span>.
As a note, despite the low risk posed by livestock themselves, the working conditions of the meat industry itself were associated with a very high risk of SARS-CoV-2 infection for workers that did cause disruptions to food supply chains <span class="citation" data-cites="hJNwsIkr Aj0mjB0J">(<a href="#ref-hJNwsIkr" role="doc-biblioref">404</a>, <a href="#ref-Aj0mjB0J" role="doc-biblioref">405</a>)</span>.</p>
<p>However, one species of domesticated agricultural animal severely affected by SARS-CoV-2 was the mink (<em>Neovison vison</em>).
While fur farming has declined significantly since the twentieth century, mink farming is still common in China and some European countries, and mink farms continue to exist in the United States.
Mink belong to the Mustelidae family within Carnivora.
SARS-CoV-2 was first reported on mink farms in the Netherlands and Denmark in 2020 <span class="citation" data-cites="1BS2uKJCI EWObKVab">(<a href="#ref-1BS2uKJCI" role="doc-biblioref">406</a>, <a href="#ref-EWObKVab" role="doc-biblioref">407</a>)</span>.
Mink were observed to show symptoms of respiratory infection, with varied severity among individuals <span class="citation" data-cites="1BS2uKJCI">(<a href="#ref-1BS2uKJCI" role="doc-biblioref">406</a>)</span>.
Dissection revealed lung pathology consistent with interstitial pneumonia <span class="citation" data-cites="1BS2uKJCI">(<a href="#ref-1BS2uKJCI" role="doc-biblioref">406</a>)</span>.
An analysis of five farms in the United States reported mortality rates between 35 and 55% of adult minks <span class="citation" data-cites="eZlF3Ang">(<a href="#ref-eZlF3Ang" role="doc-biblioref">408</a>)</span>.
Subsequently, mink farms worldwide reported outbreaks of SARS-CoV-2.
Concerns were amplified when novel variants of SARS-CoV-2 were identified as having emerged on Danish mink farms and spread into the human population <span class="citation" data-cites="tb42t88r EWObKVab TKJuIOB1 EWObKVab cu0cA7e7">(<a href="#ref-EWObKVab" role="doc-biblioref">407</a>, <a href="#ref-EWObKVab" role="doc-biblioref">407</a>, <a href="#ref-tb42t88r" role="doc-biblioref">409</a>–<a href="#ref-cu0cA7e7" role="doc-biblioref">411</a>)</span>.
The fact that these variants appeared in mink populations before being observed in humans suggests that mink can indeed serve as a viral reservoir <span class="citation" data-cites="EWObKVab">(<a href="#ref-EWObKVab" role="doc-biblioref">407</a>)</span>.
Concerns about mink-to-human transmission led to the mass destruction of domesticated mink populations in Europe <span class="citation" data-cites="RlXCplKh DcwRbCdD">(<a href="#ref-RlXCplKh" role="doc-biblioref">412</a>, <a href="#ref-DcwRbCdD" role="doc-biblioref">413</a>)</span>.
Introgression from fur farms into wild populations (i.e., feralization) may have also resulted in the spread of SARS-CoV-2 into wild mink populations <span class="citation" data-cites="sMtrNzmx 1CPlZeGO2">(<a href="#ref-sMtrNzmx" role="doc-biblioref">414</a>, <a href="#ref-1CPlZeGO2" role="doc-biblioref">415</a>)</span>.
Therefore, while the specific zoonotic origin of SARS-CoV-2 may still not be clear, the potential for the virus to take hold in species other than humans has been clearly demonstrated by the mink outbreak.</p>
<p>Finally, some species of zoo animals were also monitored to determine whether they were at risk.
Several species closely related to humans (i.e., the Great Apes) are threatened with extinction and had been identified through <em>in silico</em> studies as likely to be susceptible to SARS-CoV-2 <span class="citation" data-cites="1FxiM6ws1">(<a href="#ref-1FxiM6ws1" role="doc-biblioref">386</a>)</span>, and therefore the potential for a virus to target these close relatives presented a major concern.
In early 2021, three gorillas (<em>Gorilla beringei beringei</em>) at the San Diego Zoo Safari Park developed respiratory symptoms that were confirmed to be associated with SARS-CoV-2 <span class="citation" data-cites="gbCOOK7q">(<a href="#ref-gbCOOK7q" role="doc-biblioref">416</a>)</span>.
Gorillas at other zoos have also been infected <span class="citation" data-cites="95bYQKt8 RCnX1l94 dabOG8Ms">(<a href="#ref-95bYQKt8" role="doc-biblioref">417</a>–<a href="#ref-dabOG8Ms" role="doc-biblioref">419</a>)</span>.
Additionally, given the susceptibility of house cats, it is not so surprising that other felids are also susceptible to SARS-CoV-2.
Infections of several “big cats” including Malayan tigers (<em>Panthera tigris jacksoni</em>), Amur tigers (<em>Panthera tigris altaica</em>), and African lions (<em>Panthera leo krugeri</em>) were reported at New York City’s Bronx Zoo in March 2020 <span class="citation" data-cites="lHwLc22O">(<a href="#ref-lHwLc22O" role="doc-biblioref">420</a>)</span>.
In late 2020, four lions (<em>Panthera leo bleyenberghi</em>) at the Barcelona Zoo also developed respiratory symptoms that were found to be caused by SARS-CoV-2 <span class="citation" data-cites="cu0cA7e7">(<a href="#ref-cu0cA7e7" role="doc-biblioref">411</a>)</span>.
Several captive snow leopards (<em>Panthera uncia</em>) in the United States have died from COVID-19 <span class="citation" data-cites="IJr533OF yQVSsUGy">(<a href="#ref-IJr533OF" role="doc-biblioref">421</a>, <a href="#ref-yQVSsUGy" role="doc-biblioref">422</a>)</span>.</p>
<p>While discussions of zoonoses often focus on the risk that animal diseases carry for human populations, the COVID-19 pandemic has also underscored the risks that human diseases pose for animals.
COVID-19 precautions may have reduced the spread of other respiratory illnesses to wild mountain gorilla (<em>Gorilla beringei beringei</em>) populations <span class="citation" data-cites="54pXgduy">(<a href="#ref-54pXgduy" role="doc-biblioref">423</a>)</span>, reducing one of the most significant threats to this endangered species <span class="citation" data-cites="KOsmKRWs">(<a href="#ref-KOsmKRWs" role="doc-biblioref">424</a>)</span>.
In the case of gorillas, the potential for cross-species application of pharmaceutical advances has also become clear: captive gorillas with COVID-19 received monoclonal antibodies <span class="citation" data-cites="bOYylTUp">(<a href="#ref-bOYylTUp" role="doc-biblioref">425</a>)</span>.
Additionally, several companies are developing veterinary vaccines against SARS-CoV-2.
The most visible has been Zoetis, a veterinary pharmaceutical company, that has developed vaccines that have been administered to several species, including felids in zoos, minks, and gorillas <span class="citation" data-cites="QQlhSDpY KEpTBsAB 118xjDmDG H4B703HM">(<a href="#ref-QQlhSDpY" role="doc-biblioref">426</a>–<a href="#ref-H4B703HM" role="doc-biblioref">429</a>)</span>.
Russian researchers have also developed a COVID-19 vaccine for carnivores <span class="citation" data-cites="dzBcxbby">(<a href="#ref-dzBcxbby" role="doc-biblioref">430</a>)</span>.</p>
<p>Therefore, the host range of SARS-CoV-2 is broad, including primates, bats, and carnivores.
The most severe infections have been observed in humans, felids, and mustelids <span class="citation" data-cites="QQlhSDpY">(<a href="#ref-QQlhSDpY" role="doc-biblioref">426</a>)</span>.
In the United States, as of late 2021, dogs and cats made up the majority of non-human SARS-CoV-2 infections <span class="citation" data-cites="Le00U1aY">(<a href="#ref-Le00U1aY" role="doc-biblioref">431</a>)</span>, but the most severe infections were observed in felids and mustelids (in addition to humans) <span class="citation" data-cites="QQlhSDpY">(<a href="#ref-QQlhSDpY" role="doc-biblioref">426</a>)</span>.
Interestingly, comparing ACE2 binding activity across species <span class="citation" data-cites="vzPlWSZd 13sevtn6P">(<a href="#ref-vzPlWSZd" role="doc-biblioref">432</a>, <a href="#ref-13sevtn6P" role="doc-biblioref">433</a>)</span> revealed that it did not always align with which species known to be susceptible to SARS-CoV-2 infection, suggesting other binding sites might also be important.
While the specific zoonotic origins of SARS-CoV-2 remain unknown, pharmaceutical developments in the treatment of COVID-19 have included non-human species.
The complex relationship between animals, humans, and disease highlights the importance of a broad perspective on health that extends beyond a single species.</p>
<h2 data-number="2.7" id="question-4-do-genes-influence-who-is-affected"><span class="header-section-number">2.7</span> Question 4: Do Genes Influence Who is Affected?</h2>
<p>Throughout the pandemic, many hypotheses have been raised about factors that might influence individuals’ susceptibility to COVID-19 or to severe disease .
Many risk factors, such as underlying health conditions, are related to the body’s inflammatory response, as we review elsewhere <span class="citation" data-cites="Up1vB19z">(<a href="#ref-Up1vB19z" role="doc-biblioref">339</a>)</span>.
Here, we focus narrowly on genetic bases of differences in susceptibility or outcomes.
Historically, the identification of genetic risk factors for a disease typically utilized a candidate gene approach, where a gene of interest was evaluated to identify variants that showed an association with the outcome of interest.
While economical in terms of sequencing, this approach is prone to spurious results when applied to complex traits <span class="citation" data-cites="1GJ6Mge9A">(<a href="#ref-1GJ6Mge9A" role="doc-biblioref">434</a>)</span>.
Today, in the age of next-generation sequencing (NGS), alternative approaches have emerged.
NGS makes it possible to conduct genome-wide scans where a large number of single-nucleotide polymorphisms (SNPs) or variants are evaluated to identify regions of the genome associated with variation in a phenotype.
Genome-wide association studies (GWAS) in particular are a popular approach that employs this strategy.
During COVID-19, both of these paradigms have been applied to the problem of identifying genetic correlates of disease severity.</p>
<h3 data-number="2.7.1" id="candidate-gene-approaches"><span class="header-section-number">2.7.1</span> Candidate-Gene Approaches</h3>
<p>Many candidate genes have been investigated throughout the pandemic.
Here, we review three examples of candidate gene studies in COVID-19.
First, an early study (published in April 2020) investigated a known variant in interferon-induced transmembrane protein 3 (<em>IFITM3</em>) among hospitalized patients in Beijing <span class="citation" data-cites="FbeY1L1s">(<a href="#ref-FbeY1L1s" role="doc-biblioref">435</a>)</span>.
This gene and variant were selected because of a prior candidate gene study by some of the same authors that found an association with influenza severity among Chinese patients during the 2009 influenza A H1N1/09 pandemic <span class="citation" data-cites="EntTrBUW">(<a href="#ref-EntTrBUW" role="doc-biblioref">436</a>)</span>.
Here, they evaluated a small number (n=80) hospitalized COVID-19 patients to determine whether homozygosity for the previously identified risk allele was associated with mild versus severe disease <span class="citation" data-cites="FbeY1L1s">(<a href="#ref-FbeY1L1s" role="doc-biblioref">435</a>)</span>.
They stated that they found an association between homozygosity for the SNP of interest and the severity of COVID-19.<!--Note: in-line stats differ from those in the associated table-->
A follow-up study demonstrated worldwide variation in the frequency of these SNPs <span class="citation" data-cites="96K1aPWA">(<a href="#ref-96K1aPWA" role="doc-biblioref">437</a>)</span>, and subsequent studies claimed to support this result by comparing the frequency of the SNP in different groups to the COVID-19 case fatality rate in those groups; they examined SNPs in several candidate genes and identified an association with another SNP in <em>IFITM3</em> <span class="citation" data-cites="LOaWooow">(<a href="#ref-LOaWooow" role="doc-biblioref">438</a>)</span>.
However, in the original study, the population-level frequency of the risk allele was consistent with its frequency in the mild population <span class="citation" data-cites="EntTrBUW">(<a href="#ref-EntTrBUW" role="doc-biblioref">436</a>)</span>.
A similar analysis examined both SNPs in Britons of different ancestral backgrounds and also reported a correlation <span class="citation" data-cites="HpvliVXv">(<a href="#ref-HpvliVXv" role="doc-biblioref">439</a>)</span>.
While this gene has been investigated for functions potentially relevant to COVID-19 pathogenesis by other groups as well (e.g., <span class="citation" data-cites="QVZpcDYW DQgMzjnS">(<a href="#ref-QVZpcDYW" role="doc-biblioref">440</a>, <a href="#ref-DQgMzjnS" role="doc-biblioref">441</a>)</span>, a follow-up analysis in Germany evaluated the effect of in 239 cases and 252 controls and reported non-significant effects <span class="citation" data-cites="HpvliVXv">(<a href="#ref-HpvliVXv" role="doc-biblioref">439</a>)</span>.
The narrative surrounding <em>IFITM3</em> therefore reflects a broad methodological critique about candidate gene studies, where results often fail to replicate <span class="citation" data-cites="YPiPALQD">(<a href="#ref-YPiPALQD" role="doc-biblioref">442</a>)</span>.
The region associated with this gene was not identified in the large-scale GWAS conducted by the COVID-19 Host Genetics Initiative (COVID-19 HGI) <span class="citation" data-cites="1GS6s8mgC">(<a href="#ref-1GS6s8mgC" role="doc-biblioref">443</a>)</span>, which is described in more detail below.</p>
<p>A second source of genetic variability that was hypothesized to have an effect on COVID-19 outcomes were human leukocyte antigens (HLA), or the major histocompatibility complex (MHC).
Both MHC classes I and II play a critical role in both the innate and adaptive immune system because they are a pivotal component of antigen presentation.
HLA classes I and II are also the most polymorphic loci in the human genome <span class="citation" data-cites="tBgc6ryL">(<a href="#ref-tBgc6ryL" role="doc-biblioref">444</a>)</span>.
Additionally, because HLA polymorphisms are associated with geographic ancestry, study location and participant background offers important context <span class="citation" data-cites="17eP6PTvz">(<a href="#ref-17eP6PTvz" role="doc-biblioref">445</a>)</span>.
Given the important role of the HLA complex in the immune response and the standing variation in the human population, HLA variation has been investigated for potential associations with COVID-19 outcomes.</p>
<p>Several approaches have been taken to evaluate a potential role of HLA in COVID-19.
<em>In silico</em> analysis suggested one particular HLA locus that could affect binding of SARS-CoV-2 peptides to MHC class II <span class="citation" data-cites="1CQ8X6yi2">(<a href="#ref-1CQ8X6yi2" role="doc-biblioref">446</a>)</span>.
Other studies evaluated outcomes using retrospective cohort analyses.
An analysis of 95 South Asian COVID-19 patients found that HLA genotype was not significant in differentiating case severity when the necessary statistical corrections were applied <span class="citation" data-cites="lxEvWysS">(<a href="#ref-lxEvWysS" role="doc-biblioref">447</a>)</span>.<!--risk:HLA-B*51, protect: HLA-B*35-->
Another study in a European population (n =147) did identify HLA alleles associated with severity <span class="citation" data-cites="PyvspFUi">(<a href="#ref-PyvspFUi" role="doc-biblioref">448</a>)</span>.
In St. Louis, MO (USA), another study enrolled 234 COVID-19 cases, who were genotyped for HLA alleles and compared to a control population of 20,000 individuals from the National Marrow Donor Program <span class="citation" data-cites="1DebTO3uY">(<a href="#ref-1DebTO3uY" role="doc-biblioref">449</a>)</span>.
They compared cases and controls on the basis of four “race/ethnic” populations and reported alleles showing a statistical association within each group <span class="citation" data-cites="1DebTO3uY">(<a href="#ref-1DebTO3uY" role="doc-biblioref">449</a>)</span>.
However, because of this stratification, two of the demographic categories had less than ten cases.
Across all of these studies, there was minimal overlap in the risk alleles identified, and the small sample sizes raise concerns about the possibility for spurious hits.
The hypervariability of this region means that statistical power will necessarily be reduced, with much higher recruitment needed than for studies of biallelic loci.
A much larger analysis of 72,912 Israelis, 8.8% of whom tested positive for COVID-19, found no association between HLA genotype and infection or hospitalization <span class="citation" data-cites="1H6m2HVG7">(<a href="#ref-1H6m2HVG7" role="doc-biblioref">450</a>)</span>.
Therefore, while MHC is functionally important to the immune response to COVID-19, it is not clear whether HLA genotypes are predictive of COVID-19 severity, and certainly such studies face exacerbated versions of the typical challenges of candidate gene studies.
Because of the challenges associated with analyzing such a variable region, it was excluded from the large-scale COVID-19 HGI GWAS analysis <span class="citation" data-cites="1GS6s8mgC">(<a href="#ref-1GS6s8mgC" role="doc-biblioref">443</a>)</span>.</p>
<p>Finally, significant attention has been paid to the question of whether ABO blood type is associated with COVID-19 outcomes.<!--This paper is interesting but I'm not totally sure how to incorporate it 10.1371/journal.pone.0251535-->
ABO blood type has been found to modulate susceptibility to other pathogens <span class="citation" data-cites="6pfIhCGV">(<a href="#ref-6pfIhCGV" role="doc-biblioref">451</a>)</span>.
While ABO blood type is a genetic trait, it is more easily evaluated than the genetic regions discussed above because of the simple relationship between genetic variants and phenotype.
The possibility for an association between blood type and COVID-19 infection was raised early in the pandemic in a preprint that reported associations in 2,173 patients in Wuhan and Shenzhen, China <span class="citation" data-cites="1DbV444Pj">(<a href="#ref-1DbV444Pj" role="doc-biblioref">452</a>)</span>.
The protective effect of O and increased risk associated with A blood types that they reported was subsequently investigated by many studies that returned varied results (e.g., <span class="citation" data-cites="1BQ7FxsTt QCVLQvsf 1Hm5TzRvT 19DBCLqWZ">(<a href="#ref-1BQ7FxsTt" role="doc-biblioref">453</a>–<a href="#ref-19DBCLqWZ" role="doc-biblioref">456</a>)</span>; see <span class="citation" data-cites="P5ITbYUZ">(<a href="#ref-P5ITbYUZ" role="doc-biblioref">457</a>)</span> for a literature review).
Observations of higher and lower risk, respectively, of SARS-CoV-2 infection with A and O blood types was supported by a meta-analysis <span class="citation" data-cites="16ZxZ0D9J">(<a href="#ref-16ZxZ0D9J" role="doc-biblioref">458</a>)</span>.
While the support for the association was independent of a mechanism, a possible relationship between ACE activity and blood type has been proposed <span class="citation" data-cites="11G8Z08cp">(<a href="#ref-11G8Z08cp" role="doc-biblioref">459</a>)</span> as has an effect on carbohydrate-carbohydrate interactions relevant to ACE2 binding <span class="citation" data-cites="c2EzrLpZ">(<a href="#ref-c2EzrLpZ" role="doc-biblioref">460</a>)</span>.
This is the only candidate gene described that has received additional support from GWAS, as is discussed below.</p>
<p>The COVID-19 literature related to candidate gene investigations demonstrates relatively low inter-study consistency in findings.
In particular, sample size is a major challenge in designing these studies.
However, for many traits, the relationships between genes and phenotypes are complex, and selecting which variants to sequence is not always straightforward.
As a result, in the age of next-generation sequencing, discovery-driven studies have emerged as an alternative approach.</p>
<h3 data-number="2.7.2" id="genome-wide-association-studies"><span class="header-section-number">2.7.2</span> Genome-Wide Association Studies</h3>
<p>Genome-wide association studies (GWAS) offers a discovery-driven approach that provides a different perspective than candidate gene studies.
Instead of selecting a gene or variant <em>a priori</em>, in GWAS, a large number of SNPs (usually several million) are evaluated at once to identify those most likely to vary in correlation with a trait of interest.
Because of the large number of statistical tests, statistical power and multiple hypothesis testing are both very important considerations in executing GWAS, which have also struggled with issues related to replicability <span class="citation" data-cites="X1avT5DW">(<a href="#ref-X1avT5DW" role="doc-biblioref">461</a>)</span>.
In cases such as COVID-19 where outcomes can differ among ancestry groups (likely for non-genetic reasons, as reviewed in <span class="citation" data-cites="Up1vB19z">(<a href="#ref-Up1vB19z" role="doc-biblioref">339</a>)</span>), it is especially important that GWAS samples be selected with attention paid to ancestry, as incorrect or misleading associations can otherwise be identified with neutral markers indicative of ancestry itself <span class="citation" data-cites="T3GG8iJN">(<a href="#ref-T3GG8iJN" role="doc-biblioref">462</a>)</span>.</p>
<p>Over the past two years, many GWAS have been undertaken with the aim of identifying variants associated with COVID-19 outcomes.
In some cases, the results have been consistent with hypothesized genetic correlates of susceptibility to COVID-19.
One study conducted a GWAS on a total of 435 COVID-19 patients from four countries and identified another HLA allele to be associated with an increased risk of intubation <span class="citation" data-cites="Xq4jYNyv">(<a href="#ref-Xq4jYNyv" role="doc-biblioref">463</a>)</span>.<!--HLA-C*04:01-->
Other GWAS have identified an association with the ABO blood group locus.
One conducted a case/control GWAS in two populations, Italians and Spaniards, with 1980 cases and 2205 controls.
They reported two loci that met the genome-wide significance threshold, one on chromosome 3 and one on chromosome 9 <span class="citation" data-cites="owevcpwb">(<a href="#ref-owevcpwb" role="doc-biblioref">464</a>)</span>.
The hit on chromosome 9 fell on the ABO locus and the alleles identified suggested a protective association with blood group O and a risk association with blood group A <span class="citation" data-cites="owevcpwb">(<a href="#ref-owevcpwb" role="doc-biblioref">464</a>)</span>.</p>
<p>As the pandemic has progressed, large-scale efforts have been assembled to conduct GWAS on massive scales.
In March 2020, COVID-19 HGI was established as a world-wide consortium that combines data to conduct meta-analyses <span class="citation" data-cites="HrYm0pnz">(<a href="#ref-HrYm0pnz" role="doc-biblioref">465</a>)</span>.
One year later, COVID-19 HGI released a meta-analysis of data from 46 studies, comprising 49,562 cases and 1,770,206 controls <span class="citation" data-cites="1GS6s8mgC">(<a href="#ref-1GS6s8mgC" role="doc-biblioref">443</a>)</span>.
They identified 13 loci, seven of which were significant at the genome-wide level when considering all data available, that were associated with one or more phenotypes related to COVID-19 infection or severity.
Notably, strong signals were identified for both of the loci suggested by previous medium-scale GWAS in association with COVID-19 infection <span class="citation" data-cites="owevcpwb">(<a href="#ref-owevcpwb" role="doc-biblioref">464</a>)</span>.
Additionally, several other loci could be mapped onto hypotheses about genetic contributors to immune function, lung function and disease.
This world-wide GWAS study made an effort towards strategic incorporation of genetic information from different ancestral groups.
Interestingly, the risk variant on chromosome 3 is likely to be inherited from Neanderthal introgression, meaning it is likely to be more prevalent in certain populations, especially non-African populations <span class="citation" data-cites="14Vd7Guv9 J1pI7YF1">(<a href="#ref-14Vd7Guv9" role="doc-biblioref">466</a>, <a href="#ref-J1pI7YF1" role="doc-biblioref">467</a>)</span>.
The potential functional relationship between this region of the genome and COVID-19 is unknown, but phenome-wide association study has suggested blood cell traits as a potential trait regulated by this region <span class="citation" data-cites="2J6m4djb">(<a href="#ref-2J6m4djb" role="doc-biblioref">468</a>)</span>.</p>
<p>Identifying genetic variants associated with a complex disease is always complicated.
In COVID-19 studies, the results of candidate gene analyses have in general been difficult to replicate.
However, large-scale collaboration on GWAS has made it possible to detect at least two loci that do appear to replicate across studies and potentially even across ancestral backgrounds.</p>
<h2 data-number="2.8" id="question-5-how-is-it-changing"><span class="header-section-number">2.8</span> Question 5: How is it Changing?</h2>
<p>Evolution in SARS-CoV-2 has also been observed over a short timescale.
After zoonotic transfer, SARS-CoV-2 continued evolving in the human population <span class="citation" data-cites="p9LUiyCN">(<a href="#ref-p9LUiyCN" role="doc-biblioref">210</a>)</span>.
The SARS-CoV-2 mutation rate is moderate compared to other RNA viruses <span class="citation" data-cites="2w7lNKxQ">(<a href="#ref-2w7lNKxQ" role="doc-biblioref">212</a>)</span>, which likely restricts the pace of evolution in SARS-CoV-2.
Nevertheless, genomic analyses have yielded statistical evidence of ongoing evolution.
Initially, two known variants of the spike protein emerged that differed by a single amino acid at position 614 (G614 and D614), and there is evidence that G614 had become more prevalent than D614 by June 2020 <span class="citation" data-cites="RIpPhJ1g">(<a href="#ref-RIpPhJ1g" role="doc-biblioref">222</a>)</span>.
While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, this hypothesis has not yet been tested due to a lack of data <span class="citation" data-cites="DyG1eyK4">(<a href="#ref-DyG1eyK4" role="doc-biblioref">469</a>)</span>.
Another study <span class="citation" data-cites="2w7lNKxQ">(<a href="#ref-2w7lNKxQ" role="doc-biblioref">212</a>)</span> identified 198 recurrent mutations in a dataset of 7,666 curated sequences, all of which defined non-synonymous protein-level modifications.
This pattern of convergent evolution at some sites could indicate that certain mutations confer an adaptive advantage.
While it is evident that SARS-CoV-2 exhibits moderate potential for ongoing and future evolution, the relationship between mutations and pathogenicity is not yet known.
Additional data is needed in order to understand patterns of evolutionary change and the mechanisms by which they might affect virulence.</p>
<p>Several factors could promote the evolution of SARS-CoV-2, including host immunodeficiency and transient exposure to antibodies directed against SARS-CoV-2 proteins.
A single case study of SARS-CoV-2 infection in an immunocompromised female with chronic lymphocytic leukemia and hypogammaglobulinemia <span class="citation" data-cites="HFUP2slc">(<a href="#ref-HFUP2slc" role="doc-biblioref">470</a>)</span> suggested that an accelerated evolution of the virus could occur in conditions of immunodeficiency.
A first administration of convalescent plasma did not clear the virus, and an ensuing increase in the genomic diversity in the samples was observed, suggesting an accelerated evolution due to selection pressure.
A second administration of convalescent plasma cleared the virus from the host 105 days after the initial diagnosis.
However, throughout the duration of infection, the patient was asymptomatic but contagious.
A second single case study in a 45-year old male with antiphospholipid syndrome <span class="citation" data-cites="13x1Imo3Y">(<a href="#ref-13x1Imo3Y" role="doc-biblioref">471</a>)</span> confirmed the earlier results, providing evidence of persistent COVID-19 symptoms in an immunocompromised patient for 154 days following diagnosis, ultimately leading to the death of patient.
The treatments administered included remdesivir and the Regeneron anti-spike protein antibody cocktail.
Genomic analyses of the patient’s nasopharyngeal swabs confirmed an accelerated evolution of the virus through mutations in the spike gene and the receptor-binding domain.
In summary, these two case studies suggested an accelerated evolution and persistent shedding of the virus in conditions of immunodeficiency.
In particular, the first case highlighted the role of convalescent plasma in creating escape variants.
In fact, one study <span class="citation" data-cites="3jfKqJRA">(<a href="#ref-3jfKqJRA" role="doc-biblioref">472</a>)</span> exposed the SARS-CoV-2 virus to convalescent plasma <em>in vitro</em> repeatedly to see how much plasma was required to neutralize the virus.
The results of the first six exposures were similar, but they reported that after the seventh exposure (on day 45), the amount of plasma required began to increase.
In analyzing the viral variants present, they found that this viral escape was promoted by the sudden accumulation of mutations, especially in the receptor-binding domain (RBD) and N-terminal domain (NTD), that quickly rose in frequency.
By the thirteenth exposure (day 85), the virus had evolved three mutations and could no longer be neutralized by the plasma used, even though the plasma was comprised of polyclonal serum that targeted a variety of epitopes.
Taken together, these observations suggest that evolutionary analyses of SARS-CoV-2 can provide crucial information about the conditions that promote resistance in SARS-CoV-2 and the kinetics of how resistance develops, information which will be important for understanding the implications of how vaccine regimens are designed and whether/when next-generation vaccines will be needed.</p>
<p>When variants occur, they can rise in frequency by chance or through an adaptive process that confers a competitive advantage to the virus.
Variants that had the D614G mutation in the spike glycoprotein seemed to spread faster. <!-- TO DO: add more history of this variant?-->
However, it has been suggested that the mutation rose in frequency due to early chance events rather than by adaptive events <span class="citation" data-cites="xFOQs3Qb">(<a href="#ref-xFOQs3Qb" role="doc-biblioref">473</a>)</span>.
Another mutation, Y453F, that occurred in the receptor binding domain of <em>S</em>, was first detected in mink; however, the transmission to humans has been established.
In mink, this mutation conferred an advantage by increasing the affinity towards ACE2 <span class="citation" data-cites="4IJOYEr2">(<a href="#ref-4IJOYEr2" role="doc-biblioref">474</a>)</span>.
Similarly, N501Y mutation induces an increased affinity towards human ACE2 and has been involved in the dominance of B.1.1.7 by outcompeting other variants <span class="citation" data-cites="INygVT9y">(<a href="#ref-INygVT9y" role="doc-biblioref">475</a>)</span>.
Therefore, genomic surveillance is essential to prevent the emergence of super-spreaders <span class="citation" data-cites="17bdYIjnm">(<a href="#ref-17bdYIjnm" role="doc-biblioref">476</a>)</span>.</p>
<p>Emerging methods are being applied to this problem in an effort to understand which mutations are most likely to be of significant concern.
Novel machine learning methods were developed to predict the mutations in the sequence that promote viral escape.
While they preserve the pathogenicity of the virus, escape mutations change the virus’s sequence to evade detection by the immune system.
By using tools from natural language processing (NLP), viral escape was modeled as an NLP problem <span class="citation" data-cites="1FGTHhvPb">(<a href="#ref-1FGTHhvPb" role="doc-biblioref">477</a>)</span> where a modification makes a sentence grammatically correct but semantically different.
Therefore, language models of viruses could predict mutations that change the presentation of the virus to the immune system but preserve its infectivity.</p>
<h3 data-number="2.8.1" id="variants-of-concern-and-variants-under-surveillance"><span class="header-section-number">2.8.1</span> Variants of Concern and Variants under Surveillance</h3>
<p>Viral replication naturally leads to the occurrence of mutations, and thus to genetic variation <span class="citation" data-cites="esiDpXpq">(<a href="#ref-esiDpXpq" role="doc-biblioref">478</a>)</span>.
However, due to an intrinsic RNA proof-reading process in the SARS-CoV-2 virus, the pace of evolution of SARS-CoV-2 is moderate in comparison to other viruses <span class="citation" data-cites="6G6nZMtT">(<a href="#ref-6G6nZMtT" role="doc-biblioref">479</a>)</span>.
The declaration of the first SARS-CoV-2 variant of concern (VOC) B.1.1.7 in December 2020 has attracted significant media attention.
While the B.1.1.7 lineage garnered attention in November 2020, two genomes of the lineage were detected as early as September 20th, 2020 from routine genomic data sampled in Kent (U.K.) by the COVID-19 Genomics UK Consortium (COG-UK).
The following day, a second B.1.1.7 genome was reported in greater London <span class="citation" data-cites="1kRqNje swAa8Nio tzOnKNrn xFOQs3Qb">(<a href="#ref-tzOnKNrn" role="doc-biblioref">234</a>, <a href="#ref-xFOQs3Qb" role="doc-biblioref">473</a>, <a href="#ref-1kRqNje" role="doc-biblioref">480</a>, <a href="#ref-swAa8Nio" role="doc-biblioref">481</a>)</span>
Since then, B.1.1.7 has spread across the UK and internationally, and it has now been detected in at least 62 countries <span class="citation" data-cites="m9qtrWft">(<a href="#ref-m9qtrWft" role="doc-biblioref">482</a>)</span>, despite several countries imposing travel restrictions on travelers from the UK.
Of the twenty-three mutations that define B.1.1.7 from the original strain isolated in Wuhan (lineage A), fourteen are lineage-specific and three appear to be biologically consequential mutations associated with the spike protein, namely N501Y, P681H, and 69-70del <span class="citation" data-cites="1kRqNje swAa8Nio">(<a href="#ref-1kRqNje" role="doc-biblioref">480</a>, <a href="#ref-swAa8Nio" role="doc-biblioref">481</a>)</span>.
The latter is a 6-bp deletion that leads to the loss of two amino acids and has consequences for immune recognition; it may, in conjunction with N501Y, be responsible for the increased transmissibility of the B.1.1.7 VOC due to changes in the RBD that increase binding affinity with ACE2 <span class="citation" data-cites="1kRqNje RhHGzsfD">(<a href="#ref-RhHGzsfD" role="doc-biblioref">230</a>, <a href="#ref-1kRqNje" role="doc-biblioref">480</a>)</span>.
B.1.1.7 has increased transmissibility by up to 56%, leading to an R<sub>0</sub> of approximately 1.4.
Additionally, this VOC has been shown to be associated with increased disease severity and increased mortality <span class="citation" data-cites="lIzRCwLq">(<a href="#ref-lIzRCwLq" role="doc-biblioref">483</a>)</span>.
Other variants also express the 69-70del mutation <span class="citation" data-cites="VDhaLzV3 10Qn2iFj7">(<a href="#ref-VDhaLzV3" role="doc-biblioref">484</a>, <a href="#ref-10Qn2iFj7" role="doc-biblioref">485</a>)</span>, and public health officials in the United States and the UK have been able to use RT-PCR-based assays (ThermoFisher TaqPath COVID-19 assay) to identify sequences with this deletion because it occurs where the qPCR probe binds <span class="citation" data-cites="tzOnKNrn">(<a href="#ref-tzOnKNrn" role="doc-biblioref">234</a>)</span>.
In the UK, B.1.1.7 is present in more than 97% of diagnostic tests that return negative for S-gene targets and positive for the other targets; thus, the frequency of S-gene target failure can be used as a proxy for the detection of B.1.1.7 <span class="citation" data-cites="1kRqNje 11g9KlaY4">(<a href="#ref-1kRqNje" role="doc-biblioref">480</a>, <a href="#ref-11g9KlaY4" role="doc-biblioref">486</a>)</span>.
The FDA has highlighted that the performance of three diagnostic tests may be affected by the B.1.1.7 lineage because it could cause false negative tests <span class="citation" data-cites="F4Le4e1M">(<a href="#ref-F4Le4e1M" role="doc-biblioref">487</a>)</span>.</p>
<p>While B.1.1.7 is currently the main VOC, other genetic variants also currently designated as VOCs have been detected, including B.1.351 and P.1, both of which emerged independently <span class="citation" data-cites="1455MbSH0 1EWBspR2q">(<a href="#ref-1455MbSH0" role="doc-biblioref">488</a>, <a href="#ref-1EWBspR2q" role="doc-biblioref">489</a>)</span>.
B.1.351 was first detected in October 2020 in South Africa, was later detected in the EU on December 28th, 2020 and has now spread to at least 26 countries <span class="citation" data-cites="aXPtLhNl sqhvCTIL 11ZfOrGxJ">(<a href="#ref-sqhvCTIL" role="doc-biblioref">231</a>, <a href="#ref-aXPtLhNl" role="doc-biblioref">490</a>, <a href="#ref-11ZfOrGxJ" role="doc-biblioref">491</a>)</span>.
B.1.351 contains several mutations at the RBD including K417N, E484K, and N501Y.
While the biological significance of these mutations are still under investigation, it does appear that this lineage may be associated with increased transmissibility <span class="citation" data-cites="WgiJaD2d">(<a href="#ref-WgiJaD2d" role="doc-biblioref">492</a>)</span> due to the N501Y mutation <span class="citation" data-cites="RhHGzsfD swAa8Nio">(<a href="#ref-RhHGzsfD" role="doc-biblioref">230</a>, <a href="#ref-swAa8Nio" role="doc-biblioref">481</a>)</span>.
Additionally, an analysis of a pseudovirus expressing the 501Y.V2 spike protein (B.1.351) showed that this variant demonstrates increased resistance to neutralization by convalescent plasma, even though total binding activity remained mostly intact <span class="citation" data-cites="OPGkrmXZ">(<a href="#ref-OPGkrmXZ" role="doc-biblioref">493</a>)</span>.
Further, using a live virus neutralization assay (LVNA), it was shown that 501Y.V2 (B.1.351) is poorly neutralized by convalescent plasma obtained from individuals who responded to non-501Y.V2 variants <span class="citation" data-cites="UzTTUShg">(<a href="#ref-UzTTUShg" role="doc-biblioref">494</a>)</span>.
However, 501Y.V2 infection-elicited plasma was able to cross-neutralize earlier non-501Y.V2 variants, suggesting that vaccines targeting VOCs may be effective against other mutant lineages <span class="citation" data-cites="UzTTUShg">(<a href="#ref-UzTTUShg" role="doc-biblioref">494</a>)</span>.</p>
<p>The P.1 variant is a sublineage of the B.1.1.28 lineage that was first detected in Japan in samples obtained from four travelers from Brazil during a screening at a Tokyo airport on January 10, 2021 <span class="citation" data-cites="FuVm03yl">(<a href="#ref-FuVm03yl" role="doc-biblioref">495</a>)</span>.
Shortly thereafter, it was established that there was a concentration of cases of the P.1 variant in Manaus, Brazil.
In a small number of samples (n=31) sequenced in Manaus, 42% were identified as the P.1 variant as early as mid-December, but the variant seemed to be absent in genome surveillance testing prior to December <span class="citation" data-cites="7hGWIt0g">(<a href="#ref-7hGWIt0g" role="doc-biblioref">496</a>)</span>.
To date, at least eight countries have detected the P.1 lineage <span class="citation" data-cites="IIEqiDeW">(<a href="#ref-IIEqiDeW" role="doc-biblioref">497</a>)</span>.
While the majority of P.1 cases detected internationally have been linked to travel originating from Brazil, the UK has also reported evidence of community transmission detected via routine community sequencing <span class="citation" data-cites="IIEqiDeW 17p7RjLmX">(<a href="#ref-IIEqiDeW" role="doc-biblioref">497</a>, <a href="#ref-17p7RjLmX" role="doc-biblioref">498</a>)</span>.<br />
P.1 has eight lineage-specific mutations along with three concerning spike protein mutations in the RBD, including K417T, E484K, and N501Y <span class="citation" data-cites="WgiJaD2d">(<a href="#ref-WgiJaD2d" role="doc-biblioref">492</a>)</span>.</p>
<p>There have been multiple different SARS-CoV-2 lineages detected that have mostly been of no more clinical concern than the original devastating lineage originating in Wuhan <span class="citation" data-cites="TzywDS3t">(<a href="#ref-TzywDS3t" role="doc-biblioref">499</a>)</span>.
However, the spotlight has been cast on other variants of unknown clinical relevance due to the increase of cases observed that have been associated with B.1.1.7 in particular.<br />
Although early in its ascendency, B.1.427/429 are SARS-CoV-2 variants that was detected in California, USA and also known as CAL.20C <span class="citation" data-cites="1Ha4IVlGr">(<a href="#ref-1Ha4IVlGr" role="doc-biblioref">500</a>)</span>.
It was first detected in July 2020 but was not detected again until October 2020.
In December 2020, B.1.427/429 accounted for ~24% of the total cases in Southern California and ~36% of total cases in the Los Angeles area.
B.1.427/429 have now been detected in several U.S. states and at least 38 countries worldwide <span class="citation" data-cites="BoGqjBAU 1Ha4IVlGr">(<a href="#ref-1Ha4IVlGr" role="doc-biblioref">500</a>, <a href="#ref-BoGqjBAU" role="doc-biblioref">501</a>)</span>.
This variant is characterized by five key lineage-specific mutations (ORF1a: I4205V, ORF1b:D1183Y, S: S13I;W152C;L452R).
The latter spike mutation, L452R, is found in an area of the RBD known to resist monoclonal antibodies to the spike protein <span class="citation" data-cites="xV1qJhIq">(<a href="#ref-xV1qJhIq" role="doc-biblioref">502</a>)</span>, and it is hypothesized that this mutation may resist polyclonal sera in convalescent patients or in individuals post-vaccination <span class="citation" data-cites="1FhpGQXK9 1Ha4IVlGr">(<a href="#ref-1Ha4IVlGr" role="doc-biblioref">500</a>, <a href="#ref-1FhpGQXK9" role="doc-biblioref">503</a>)</span>.
B.1.427/429 are now designated VOCs <span class="citation" data-cites="1EWBspR2q">(<a href="#ref-1EWBspR2q" role="doc-biblioref">489</a>)</span>; however, further research is still required to determine the implications of the mutations encoded in this genetic variant.<br />
Another notable variant has recently been discovered in 35 patients in a Bavarian hospital in Germany; however, the sequencing data has not been published to date and it remains to be determined whether this variant is of any further concern <span class="citation" data-cites="2rGCvk9t">(<a href="#ref-2rGCvk9t" role="doc-biblioref">504</a>)</span>.</p>
<p>There are several shared mutations and deletions between the three lineages, P.1, B.1.1.7, and B.1.315 and indeed other variants of SARS-CoV-2 that are under investigation <span class="citation" data-cites="7hGWIt0g">(<a href="#ref-7hGWIt0g" role="doc-biblioref">496</a>)</span>.
For example, N501Y, which appears to have occurred independently in each of the three lineages.<br />
E484K is present in both B.1.351 and P.1 <span class="citation" data-cites="ma0aswzQ">(<a href="#ref-ma0aswzQ" role="doc-biblioref">505</a>)</span>.
The mutations N501Y and E484K are found in the RBD within the receptor-binding motif responsible for forming an interface with the ACE2 receptor, which seems to be consequential for ACE2 binding affinity <span class="citation" data-cites="RsiMKsNV">(<a href="#ref-RsiMKsNV" role="doc-biblioref">506</a>)</span>.
Indeed, N501Y is associated with increased virulence and infectivity in mouse models <span class="citation" data-cites="rBRU1aGx">(<a href="#ref-rBRU1aGx" role="doc-biblioref">507</a>)</span>.
E484K has also been associated with evasion from neutralizing antibodies <span class="citation" data-cites="156Htv51A 3jfKqJRA 1FhpGQXK9">(<a href="#ref-3jfKqJRA" role="doc-biblioref">472</a>, <a href="#ref-1FhpGQXK9" role="doc-biblioref">503</a>, <a href="#ref-156Htv51A" role="doc-biblioref">508</a>)</span>.
The del69-70 (del:11288:9) is also shared between P.1 and B.1.1.7 and happens to be a common deletion found in the N terminal mutation of the spike protein.
This deletion has also been associated with several RBD mutations <span class="citation" data-cites="15U0IONBy RhHGzsfD swAa8Nio">(<a href="#ref-RhHGzsfD" role="doc-biblioref">230</a>, <a href="#ref-swAa8Nio" role="doc-biblioref">481</a>, <a href="#ref-15U0IONBy" role="doc-biblioref">509</a>)</span>.
There is concern that mutations in the spike protein of variants may lead to clinical consequences for transmissibility, disease severity, re-infection, therapeutics, and vaccinations <span class="citation" data-cites="skeuNfN3 15GoX5q2G 3jfKqJRA 1FhpGQXK9 w1JG0rDF Svu4IEpC 11FTHvRwx">(<a href="#ref-3jfKqJRA" role="doc-biblioref">472</a>, <a href="#ref-1FhpGQXK9" role="doc-biblioref">503</a>, <a href="#ref-skeuNfN3" role="doc-biblioref">510</a>–<a href="#ref-11FTHvRwx" role="doc-biblioref">514</a>)</span>.</p>
<p>Vaccine producers are working to determine whether the vaccines are still effective against the novel genetic variants.
Moderna recently published data for their mRNA-1273 vaccine that showed no significant impact of neutralization against the B.1.1.7 variant upon vaccination in humans and non-human primates.
On the other hand, Moderna reported a reduced but significant neutralization against the B.1.351 variant upon vaccination <span class="citation" data-cites="izey3Z40">(<a href="#ref-izey3Z40" role="doc-biblioref">515</a>)</span>.
Indeed, Pfizer–BioNTech reported that sera from twenty participants vaccinated with the BNT162b COVID-19 vaccine in previous clinical trials <span class="citation" data-cites="CWlYjjIV MD2K7MYB">(<a href="#ref-CWlYjjIV" role="doc-biblioref">516</a>, <a href="#ref-MD2K7MYB" role="doc-biblioref">517</a>)</span> elicited equivalent neutralizing titers against isogenic Y501 SARS-CoV-2 on an N501Y genetic background <em>in vitro</em> <span class="citation" data-cites="xYHivkXH">(<a href="#ref-xYHivkXH" role="doc-biblioref">518</a>)</span>.
Another study has reported that the plasma neutralizing activity against SARS-CoV-2 variants encoding the combination of K417N:E484K:N501Y or E484K or N501Y was variably and significantly reduced in the sera of twenty participants who received either the Pfizer–BioNTech BNT162b (n = 6) vaccine or the Moderna’s mRNA-1273 vaccine (n =14) <span class="citation" data-cites="kcOVBUnj">(<a href="#ref-kcOVBUnj" role="doc-biblioref">519</a>)</span>.
In a study focusing on serum samples from a combination of convalescent individuals, those who obtained the mRNA-1273 vaccine, and those who obtained Novavax, in comparison to the D614G variant, the B.1.419 variant was 2-3 times less sensitive to neutralization while the B.1.351 variant was 9-14 times less sensitive <span class="citation" data-cites="zncU219l">(<a href="#ref-zncU219l" role="doc-biblioref">520</a>)</span>.
Indeed, the E484K substitution seen in the P.1 and B.1.315 variants of the B.1.1.7 lineage are broadly reported to substantially reduce the efficacy of mRNA-based vaccines <span class="citation" data-cites="7MzilPo6 19CcC5AGf zncU219l">(<a href="#ref-zncU219l" role="doc-biblioref">520</a>–<a href="#ref-19CcC5AGf" role="doc-biblioref">522</a>)</span>.
For now, the consensus appears to be that the FDA-approved vaccines still seem to be generally effective against the genetic variants of SARS-CoV-2 and their accompanying mutations, albeit with a lower neutralizing capacity <span class="citation" data-cites="VxRhUVyl kcOVBUnj xYHivkXH izey3Z40">(<a href="#ref-izey3Z40" role="doc-biblioref">515</a>, <a href="#ref-xYHivkXH" role="doc-biblioref">518</a>, <a href="#ref-kcOVBUnj" role="doc-biblioref">519</a>, <a href="#ref-VxRhUVyl" role="doc-biblioref">523</a>)</span>, though select VOCs may present challenges.
Further research is required to discern the clinical, prophylactic, and therapeutic consequences of these genetic SARS-CoV-2 variants as the pandemic evolves.</p>
<h2 data-number="2.9" id="sars-cov-2-evolution-and-vaccine-efficacy"><span class="header-section-number">2.9</span> SARS-CoV-2 Evolution and Vaccine Efficacy</h2>
<p>With these vaccines in place, one concern is how the virus’s continued evolution will affect their efficacy.
Since the start of this pandemic, we have already seen multiple variants emerge: B.1.1.7, which emerged in the UK, B.1.351, which emerged in South Africa, and P.1, which emerged in Brazil.</p>
<p>Viruses evolve or mutate at different rates.
Mutation rate is measured as the number of substitutions per nucleotide per cell infected (μ<sub>s/n/c</sub>) <span class="citation" data-cites="4sZmtyNk">(<a href="#ref-4sZmtyNk" role="doc-biblioref">524</a>)</span>.
RNA viruses tend to have mutation rates between 10<sup>-6</sup> to 10<sup>-4</sup> <span class="citation" data-cites="4sZmtyNk">(<a href="#ref-4sZmtyNk" role="doc-biblioref">524</a>)</span>.
As a reference, influenza A virus has a mutation rate of 10<sup>-5</sup>, whereas the mutation rate of SARS-CoV-2 is lower, with the mutation rate estimated at 10<sup>-6</sup> <span class="citation" data-cites="vESqa6V0">(<a href="#ref-vESqa6V0" role="doc-biblioref">525</a>)</span>.
The accumulation of mutations allows the virus to escape recognition by the immune system <span class="citation" data-cites="2pzbGZvL">(<a href="#ref-2pzbGZvL" role="doc-biblioref">526</a>)</span>.</p>
<p>The efficacy of vaccines depends on their ability to train the immune system to recognize the virus.
Therefore, viruses can develop resistance to vaccines through the accumulation of mutations that affect recognition.
The lower mutation rate of SARS-CoV-2 suggests the possibility of SARS-CoV-2 vaccines having a more long-lasting effect compared to vaccines targeting the influenza A virus.</p>
<h3 data-number="2.9.1" id="alpha-and-beta-variants"><span class="header-section-number">2.9.1</span> Alpha and Beta Variants</h3>
<p>The current SARS-CoV-2 vaccines in distribution have been reported to provide similar efficacy against the B.1.1.7 variant compared to the variants common at the time they were developed but reduced efficacy against the B.1.351 variant <span class="citation" data-cites="LxJvckNs">(<a href="#ref-LxJvckNs" role="doc-biblioref">527</a>)</span>.
Pfizer and Moderna announced that they are working on developing a booster shot to improve efficacy against the B.1.351 variant <span class="citation" data-cites="ZxfNX9xk">(<a href="#ref-ZxfNX9xk" role="doc-biblioref">528</a>)</span>.
The WHO continues to monitor the emergence of variants and their impact on vaccine efficacy <span class="citation" data-cites="lY0XUlUp">(<a href="#ref-lY0XUlUp" role="doc-biblioref">529</a>)</span>.
Previous research in the computational prediction of the efficacy of vaccines targeting the influenza A virus might complement efforts to monitor these types of viral outbreaks <span class="citation" data-cites="YlAWEwlx">(<a href="#ref-YlAWEwlx" role="doc-biblioref">530</a>)</span>.
To adapt, future vaccines may need to account for multiple variants and strains of SARS-CoV-2, and booster shots may be required <span class="citation" data-cites="180UFKjJ2">(<a href="#ref-180UFKjJ2" role="doc-biblioref">531</a>)</span>.</p>
<h3 data-number="2.9.2" id="delta-variant-and-ct"><span class="header-section-number">2.9.2</span> Delta Variant and C<sub>t</sub></h3>
<p>One preprint <span class="citation" data-cites="e2Qnnj6R">(<a href="#ref-e2Qnnj6R" role="doc-biblioref">334</a>)</span> analyzed a retrospective cohort of patients in Singapore who contracted COVID-19 from April to June of 2021.
This study focused on those who were confirmed or inferred to have been infected by the Delta variant of concern, and its aim was to analyze virological kinetics.
They identified 218 cases, 71 (33%) of whom were fully vaccinated with either the Pfizer/BioNTech or Moderna mRNA vaccines, 13 (6%) of whom had received only one dose or had received the second dose less than two weeks prior to infection, and four (2%) of whom had received a vaccine developed with another technology.
Unvaccinated patients were more likely to be symptomatic or to progress to severe COVID-19 and showed more symptoms than vaccinated patients, despite the higher age of the vaccinated cohort.
C<sub>t</sub> was assessed over disease course, although the specific procedures for when additional RT-PCR was conducted is not clear; however, it is stated that the data was smoothed based on day of illness.
There was no significant difference in median C<sub>t</sub> in the initial samples taken from fully vaccinated and unvaccinated patients, but C<sub>t</sub> increased (signifying reduced viral load) more rapidly in fully vaccinated patients.
Like most analyses analyzing C<sub>t</sub> <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span>, this study does not provide the data to make conclusions about contagiousness, as the samples were not cultured.
All the same, these findings do suggest that vaccinated individuals may be able to clear the infection more quickly.</p>
<p>A second analysis was based in Dane County in Wisconsin, USA during summer 2021, when the Delta variant was known to be the dominant variant in the region <span class="citation" data-cites="N5OXLf7V">(<a href="#ref-N5OXLf7V" role="doc-biblioref">335</a>)</span>.
According to Our World in Data <span class="citation" data-cites="3cPwqjhj">(<a href="#ref-3cPwqjhj" role="doc-biblioref">532</a>)</span>, at the beginning of the study, 49.3% of residents of Dane County were fully vaccinated, with this number rising to 51.4% by the end of the study, although an earlier version of the preprint reported the vaccination rate in Dane County as 67.4%.
The authors identified no significant differences in C<sub>t</sub> among fully vaccinated and unvaccinated cases.
The C<sub>t</sub> thresholds reported were consistent with contagiousness as evaluated in other studies, and in the present study, SARS-CoV-2 could be cultured from 51 of 55 samples with C<sub>t</sub> less than 25.
This study was not longitudinal, but the timing of testing relative to symptom onset between symptomatic vaccinated and unvaccinated patients.
The findings of this study are therefore consistent with the idea that vaccinated people are less likely to contract symptomatic or severe COVID-19, but in cases of breakthrough infection, are still likely to be able to transmit SARS-CoV-2 to others.
<!--Among 16 Dane County samples sequenced, 14 (88%) were identified as the Delta variant of concern, and 42 of 50 samples (84%) from WI as a whole were consistent with the Delta VOC.--></p>
<h2 data-number="2.10" id="question-6-what-is-next"><span class="header-section-number">2.10</span> Question 6: What is Next?</h2>
<p>The SARS-CoV-2 pandemic has presented many unprecedented scientific opportunities.
The rapid identification of the genomic sequence of the virus allowed for early contextualization of SARS-CoV-2 among other known respiratory viruses, and the scientific community has continued to collect, analyze, and disseminate information about the SARS-CoV-2 virus and the associated illness, COVID-19 at previously unimaginable rates <span class="citation" data-cites="EM9YkiOF">(<a href="#ref-EM9YkiOF" role="doc-biblioref">4</a>)</span>.
The accessibility of genome sequencing technology has allowed for deep sequencing of the virus to establish a level of viral surveillance that had never before been achieved <span class="citation" data-cites="jVbH3kJR m99E85qV 8PK1sfVT">(<a href="#ref-jVbH3kJR" role="doc-biblioref">214</a>, <a href="#ref-m99E85qV" role="doc-biblioref">533</a>, <a href="#ref-8PK1sfVT" role="doc-biblioref">534</a>)</span>.
The information obtained from genetic, bioinformatics, and evolutionary analysis has played a significant role in shaping the global pandemic response <span class="citation" data-cites="m99E85qV ro76PcuD 42TFgI1p">(<a href="#ref-m99E85qV" role="doc-biblioref">533</a>, <a href="#ref-ro76PcuD" role="doc-biblioref">535</a>, <a href="#ref-42TFgI1p" role="doc-biblioref">536</a>)</span>.
For example, wastewater surveillance has emerged as a potential epidemiological tool to monitor SARS-CoV-2 spread over large regions, complementing clinical surveillance <span class="citation" data-cites="1DqxOlpHl vUXYeKry MWDWrPGu">(<a href="#ref-1DqxOlpHl" role="doc-biblioref">537</a>–<a href="#ref-MWDWrPGu" role="doc-biblioref">539</a>)</span>.
Humans shed SARS-CoV-2 viral RNA in feces <span class="citation" data-cites="TPWaxApy">(<a href="#ref-TPWaxApy" role="doc-biblioref">540</a>)</span> that can be detected in wastewater.
Protocols have been developed to safely and reproducibly isolate and quantify SARS-CoV-2 in samples obtained from wastewater processing plants <span class="citation" data-cites="Gx2TMx2B MWDWrPGu">(<a href="#ref-MWDWrPGu" role="doc-biblioref">539</a>, <a href="#ref-Gx2TMx2B" role="doc-biblioref">541</a>)</span>.
To date, studies show that wastewater surveillance is an effective tool to monitor SARS-CoV-2 spread over large sewersheds <span class="citation" data-cites="1DqxOlpHl vUXYeKry MWDWrPGu 1EuKt1ppV">(<a href="#ref-1DqxOlpHl" role="doc-biblioref">537</a>–<a href="#ref-MWDWrPGu" role="doc-biblioref">539</a>, <a href="#ref-1EuKt1ppV" role="doc-biblioref">542</a>)</span>.
Indeed, data from a study in New York City indicated that wastewater SARS-CoV-2 detection correlated with clinical detection of infection <span class="citation" data-cites="1EuKt1ppV">(<a href="#ref-1EuKt1ppV" role="doc-biblioref">542</a>)</span>.
Similar studies have been conducted in Nevada <span class="citation" data-cites="JwtDsxLy">(<a href="#ref-JwtDsxLy" role="doc-biblioref">543</a>)</span> and Boston <span class="citation" data-cites="8A0XYWq8">(<a href="#ref-8A0XYWq8" role="doc-biblioref">544</a>)</span>.
To date, studies have shown that factors such as temperature, the travel time of wastewater, and diurnal variability may affect detection of SARS-CoV-2 <span class="citation" data-cites="1DqxOlpHl JwtDsxLy">(<a href="#ref-1DqxOlpHl" role="doc-biblioref">537</a>, <a href="#ref-JwtDsxLy" role="doc-biblioref">543</a>)</span>.
Additionally, wastewater surveillance provides a tool to monitor fluctuations in the viral strains present in a community <span class="citation" data-cites="tYFkf9ez 1783YNtiU">(<a href="#ref-tYFkf9ez" role="doc-biblioref">545</a>, <a href="#ref-1783YNtiU" role="doc-biblioref">546</a>)</span>.
Due to its demonstrated utility so far, the United States CDC established the National Wastewater Surveillance System (NWSS), which has emerged as an important surveillance tool for SARS-CoV-2 spread <span class="citation" data-cites="8IiPEZuE">(<a href="#ref-8IiPEZuE" role="doc-biblioref">547</a>)</span>.</p>
<!-- Implications for preventing future pandemics? Vaccine response time?-->
<p>Knowledge of the evolution of SARS-CoV-2 is imperative to managing it moving forward <span class="citation" data-cites="m99E85qV MicGQlWa">(<a href="#ref-m99E85qV" role="doc-biblioref">533</a>, <a href="#ref-MicGQlWa" role="doc-biblioref">548</a>)</span>.</p>
<p>The evolutionary questions highlighted here all point back to the fact that efforts to prevent future epidemics and pandemics will benefit greatly from long-term, sustainable efforts to monitor disease.
Beyond understanding the status and evolution of known pathogens via genomic surveillance, greater preparedness for novel viral threats would also result from monitoring zoonotic disease.
If not addressed, economic and environmental stressors are likely to cause future zoonotic transfer of diseases in the future <span class="citation" data-cites="tvGu9sHq">(<a href="#ref-tvGu9sHq" role="doc-biblioref">549</a>)</span>.
The COVID-19 pandemic has highlighted both the incredible insights available with modern evolutionary and genomic methodologies, but has also revealed the reluctance of political actors to commit resources to these efforts outside of periods of acute need.
The One Health framework has emerged from collaborations by many prominent non-governmental organizations such as the World Heath Organization to promote scientific goals supportive of pandemic preparedness <span class="citation" data-cites="8PK1sfVT">(<a href="#ref-8PK1sfVT" role="doc-biblioref">534</a>)</span>.
Genomic surveillance of human pathogens and of pathogens at the human-wildlife interface is an important component needed to meet the goals of One Health <span class="citation" data-cites="8PK1sfVT">(<a href="#ref-8PK1sfVT" role="doc-biblioref">534</a>)</span>.
These efforts are especially important as anthropogenic alterations to the landscape such as climate change and urbanization increase the risk of zoonotic disease transmission <span class="citation" data-cites="k89g6p8V RBJyozoI">(<a href="#ref-k89g6p8V" role="doc-biblioref">550</a>, <a href="#ref-RBJyozoI" role="doc-biblioref">551</a>)</span>.
With the COVID-19 pandemic serving as a clear illustration of why this surveillance is imperative and of its feasibility, wider awareness and adoption of the One Health paradigm is the last piece needed to develop practices that will prevent the next pandemic.</p>
<h1 data-number="3" id="molecular-and-serologic-diagnostic-technologies-for-sars-cov-2"><span class="header-section-number">3</span> Molecular and Serologic Diagnostic Technologies for SARS-CoV-2</h1>
<h2 data-number="3.1" id="abstract-2"><span class="header-section-number">3.1</span> Abstract</h2>
<p>The COVID-19 pandemic has presented many challenges that have spurred biotechnological research to address specific problems.
Diagnostics is one area where biotechnology has been critical.
Diagnostic tests play a vital role in managing a viral threat by facilitating the detection of infected and/or recovered individuals.
From the perspective of what information is provided, these tests fall into two major categories, molecular and serological.
Molecular diagnostic techniques assay whether a virus is present in a biological sample, thus making it possible to identify individuals who are currently infected.
Additionally, when the immune system is exposed to a virus, it responds by producing antibodies specific to the virus.
Serological tests make it possible to identify individuals who have mounted an immune response to a virus of interest and therefore facilitate the identification of individuals who have previously encountered the virus.
These two categories of tests provide different perspectives valuable to understanding the spread of SARS-CoV-2.
Within these categories, different biotechnological approaches offer specific advantages and disadvantages.
Here we review the categories of tests developed for the detection of the SARS-CoV-2 virus or antibodies against SARS-CoV-2 and discuss the role of diagnostics in the COVID-19 pandemic.</p>
<h2 data-number="3.2" id="importance-2"><span class="header-section-number">3.2</span> Importance</h2>
<p>Testing is critical to pandemic management.
Among molecular tests, messaging about testing strategies has varied widely between countries, with the United States in particular emphasizing the higher sensitivity of polymerase chain reaction tests above immunoassays.
However, these tests offer different advantages, and a holistic view of the testing landscape is needed to identify the information provided by each test and its relevance to addressing different questions.
Another important consideration is the ease of use and ability to scale for each test, which determines how widely they can be deployed.
Here we describe the different diagnostic technologies available as well as the information they provide about SARS-CoV-2 and COVID-19.</p>
<h2 data-number="3.3" id="introduction-2"><span class="header-section-number">3.3</span> Introduction</h2>
<p>Since the emergence of <em>Severe acute respiratory syndrome-like coronavirus 2</em> (SARS-CoV-2) in late 2019, significant international efforts have focused on managing the spread of the virus.
Identifying individuals who have contracted coronavirus disease 2019 (COVID-19) and may be contagious is crucial to reducing the spread of the virus.
Given the high transmissibility of SARS-CoV-2 and the potential for asymptomatic or presymptomatic individuals to be contagious <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span>, the development of rapid, reliable, and affordable methods to detect SARS-CoV-2 infection is and was vitally important for understanding and controlling spread.
For instance, test-trace-isolate procedures were an early cornerstone of many nations’ efforts to control the outbreak <span class="citation" data-cites="164cqH5RU X5zWe6U8 OeNRM6OO">(<a href="#ref-164cqH5RU" role="doc-biblioref">552</a>–<a href="#ref-OeNRM6OO" role="doc-biblioref">554</a>)</span>.
Such efforts depend on diagnostic testing.</p>
<p>The genetic sequence of the SARS-CoV-2 virus was first released by Chinese officials on January 10, 2020 <span class="citation" data-cites="vlGP3RAU">(<a href="#ref-vlGP3RAU" role="doc-biblioref">555</a>)</span>, and the first test to detect the virus was released about 13 days later <span class="citation" data-cites="1AHtg71LX">(<a href="#ref-1AHtg71LX" role="doc-biblioref">556</a>)</span>.
The genomic information was critical to the development of diagnostic approaches.
There are two main classes of diagnostic tests: molecular tests, which can diagnose an active infection by identifying the presence of SARS-CoV-2, and serological tests, which can assess whether an individual was infected in the past via the presence or absence of antibodies against SARS-CoV-2.
Over the course of the COVID-19 pandemic, a variety of tests have emerged within these two categories.</p>
<p>Molecular tests detect either viral RNA or protein in a patient sample.
They are essential to identifying infected individuals, which can be important for determining courses of action related to treatment, quarantine, and contact tracing.
Tests for viral RNA are done by reverse transcription (RT) of viral RNA to DNA followed by DNA amplification, usually with polymerase chain reaction (PCR) <span class="citation" data-cites="4aV8UnGs">(<a href="#ref-4aV8UnGs" role="doc-biblioref">557</a>)</span>.
Tests for viral proteins typically use an antibody pair for detection as implemented in techniques such as lateral flow tests (LFTs) and enzyme-linked immunosorbent assays (ELISAs) <span class="citation" data-cites="PCd1Mvel w3RMVma8">(<a href="#ref-PCd1Mvel" role="doc-biblioref">558</a>, <a href="#ref-w3RMVma8" role="doc-biblioref">559</a>)</span>.
Molecular tests require the viral genome sequence in order to develop DNA primers for viral RNA detection or to express a viral protein for use as an antigen in antibody production.</p>
<p>Serological tests, on the other hand, detect the presence of antibodies in blood plasma samples or other biological samples, providing insight into whether an individual has acquired immunity against SARS-CoV-2.
Assays that can detect the presence of antibodies in blood plasma samples include ELISA, lateral flow immunoassay, and chemiluminescence immunoassay (CLIA) <span class="citation" data-cites="ntuOKrKt">(<a href="#ref-ntuOKrKt" role="doc-biblioref">560</a>)</span>.
To distinguish past infection from vaccination, serological tests detect antibodies that bind the nucleocapsid protein of the SARS-CoV-2 virus <span class="citation" data-cites="sTAHOyh1">(<a href="#ref-sTAHOyh1" role="doc-biblioref">561</a>)</span>.
They are useful for collecting population-level information for epidemiological analysis, as they can be used to estimate the extent of the infection in a given area.
Thus, serological tests may be useful to address population-level questions, such as the percent of cases that manifest as severe versus mild and for guiding public health and economic decisions regarding resource allocation and counter-disease measures.</p>
<p>Molecular and serological tests therefore offer distinct, complementary perspectives on COVID-19 infections.
Some of the same technologies are useful to both strategies, and different technologies have been employed to varying extents throughout the world since the start of the COVID-19 pandemic.
Two of the primary metrics used to evaluate these tests are sensitivity and specificity.
Sensitivity refers to a test’s ability to correctly identify a true positive; for example, a test with 50% sensitivity would identify SARS-CoV-2 in only one of every two positive samples.
On the other hand, specificity refers to how well a test is able to identify a negative sample as negative.
This metric can be relevant both in terms of understanding the risk of false positives and in discussing whether a test is susceptible to identifying other coronaviruses.
Here, we review the different types of tests within each category that have been developed and provide perspective on their applications.</p>
<h2 data-number="3.4" id="molecular-tests-to-identify-sars-cov-2"><span class="header-section-number">3.4</span> Molecular Tests to Identify SARS-CoV-2</h2>
<p>Molecular tests are used to identify distinct genomic subsequences of a viral molecule in a sample or the presence of viral protein, and they thus can be used to diagnose an active viral infection.
An important first step is identifying which biospecimens are likely to contain the virus in infected individuals and then acquiring these samples from the patient(s) to be tested.
Common sampling sources for molecular tests include nasopharyngeal cavity samples, such as throat washes, throat swabs, and saliva <span class="citation" data-cites="fCZXxoas">(<a href="#ref-fCZXxoas" role="doc-biblioref">562</a>)</span>, and stool samples <span class="citation" data-cites="oNFnjRBI">(<a href="#ref-oNFnjRBI" role="doc-biblioref">563</a>)</span>.
Once a sample is acquired from a patient, molecular tests detect SARS-CoV-2 based on the presence of either viral nucleic acids or viral proteins.</p>
<h3 data-number="3.4.1" id="pcr-based-tests"><span class="header-section-number">3.4.1</span> PCR-Based Tests</h3>
<p>When testing for RNA from viruses like SARS-CoV-2, the first step involves pre-processing in order to create complementary DNA (cDNA) from the RNA sample using RT.
The second step involves the amplification of a region of interest in the cDNA using successive cycles of heating and cooling.
Depending on the application, this amplification is achieved using variations of PCR.
Reverse transcription polymerase chain reaction (RT-PCR) tests determine whether a target is present by amplifying a region of interest of cDNA <span class="citation" data-cites="1FvL1dUZ6">(<a href="#ref-1FvL1dUZ6" role="doc-biblioref">564</a>)</span>.
Some tests use the results of the PCR itself (e.g., a band on a gel) to determine whether the pathogen is present.
However, this approach has not been employed widely in diagnostic testing, and instead most PCR-based tests are quantitative.</p>
<h4 data-number="3.4.1.1" id="quantitative-real-time-pcr"><span class="header-section-number">3.4.1.1</span> Quantitative Real-Time PCR</h4>
<p>In contrast to RT-PCR, quantitative, real-time PCR uses fluorescent dyes that bind to the amplified DNA, thereby allowing a real time assessment of the amplification procedure <span class="citation" data-cites="1FvL1dUZ6">(<a href="#ref-1FvL1dUZ6" role="doc-biblioref">564</a>)</span> (in this manuscript we refer to quantitative real-time PCR as qPCR, following the Minimum Information for Publication of Quantitative Real-Time PCR Experiments guidelines <span class="citation" data-cites="YEYTESKL">(<a href="#ref-YEYTESKL" role="doc-biblioref">565</a>)</span>, and when combined with reverse transcriptase steps, as is required for the evaluation of RNA, it is known as RT-qPCR.)
The time resolution provided by qPCR and RT-qPCR is useful because the amount of fluorescence emitted by the sample is proportional to the amount of DNA amplified, and therefore the amount of virus present can be indirectly measured using the cycle threshold (C<sub>t</sub>) determined by qPCR.</p>
<p>The first test developed and validated for the detection of SARS-CoV-2 used RT-qPCR to detect several regions of the viral genome: the <em>ORF1b</em> of the RNA-dependent RNA polymerase (RdRP), the envelope protein gene (<em>E</em>), and the nucleocapsid protein gene (<em>N</em>) <span class="citation" data-cites="1AHtg71LX">(<a href="#ref-1AHtg71LX" role="doc-biblioref">556</a>)</span>.
The publication reporting this test was released on January 23, 2020, less than two weeks after the sequence of the virus was first reported <span class="citation" data-cites="1AHtg71LX">(<a href="#ref-1AHtg71LX" role="doc-biblioref">556</a>)</span>.
In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents.
Specifically, this test uses two probes against RdRP, one of which is specific to SARS-CoV-2 <span class="citation" data-cites="1AHtg71LX">(<a href="#ref-1AHtg71LX" role="doc-biblioref">556</a>)</span>.
Importantly, this assay was not found to return false positive results.</p>
<p>In January 2020, Chinese researchers developed a test that used RT-qPCR to identify two gene regions of the viral genome, <em>ORF1b</em> and <em>N</em> <span class="citation" data-cites="COEfjKo5">(<a href="#ref-COEfjKo5" role="doc-biblioref">566</a>)</span>.
This assay was tested on samples from two COVID-19 patients and a panel of positive and negative controls consisting of RNA extracted from several cultured viruses.
The assay uses the <em>N</em> gene to screen patients, while the <em>ORF1b</em> gene region is used to confirm the infection <span class="citation" data-cites="COEfjKo5">(<a href="#ref-COEfjKo5" role="doc-biblioref">566</a>)</span>.
The test was designed to detect sequences conserved across sarbecoviruses, or viruses within the same subgenus as SARS-CoV-2.
Considering that <em>Severe acute respiratory syndrome-related coronavirus 1</em> (SARS-CoV-1) and SARS-CoV-2 are the only sarbecoviruses currently known to infect humans, a positive test can be assumed to indicate that the patient is infected with SARS-CoV-2, although this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.
The fact that the targets are so conserved offers the advantage of reduced concern about sensitivity in light of the evolution of SARS-CoV-2.</p>
<p>qPCR tests have played an important role in diagnostics during the COVID-19 pandemic.
For SARS-CoV-2, studies have typically considered a patient to be infectious if the C<sub>t</sub> is below 33 or sometimes 35 <span class="citation" data-cites="IzbYsyNs jWdI1NWR GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>, <a href="#ref-IzbYsyNs" role="doc-biblioref">567</a>, <a href="#ref-jWdI1NWR" role="doc-biblioref">568</a>)</span>.
A lower C<sub>t</sub> corresponds to fewer qPCR cycles needed to reach a detectable level, indicating that higher amounts of virus were present in the initial reaction.
Interpretations of the C<sub>t</sub> values obtained from these tests have raised some interesting questions related to viral load and contagiousness.
Lower C<sub>t</sub> values correspond to a higher probability of a positive viral culture, but no threshold could discriminate all positive from all negative cultures <span class="citation" data-cites="1HaaPNEXD">(<a href="#ref-1HaaPNEXD" role="doc-biblioref">240</a>)</span>.
Additionally, because of the variability introduced by sample collection and clinical components of testing, C<sub>t</sub> is not a proxy for viral load <span class="citation" data-cites="MgfhyyrD">(<a href="#ref-MgfhyyrD" role="doc-biblioref">569</a>)</span>.
Positive PCR results have also been reported for extended periods of time from symptom onset and/or the first positive PCR test <span class="citation" data-cites="Xp5HT2SF">(<a href="#ref-Xp5HT2SF" role="doc-biblioref">275</a>)</span>, meaning that in some cases, a positive PCR may not indicate that someone is contagious <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span>.</p>
<p>In addition to the nuance required to interpret PCR results, there are also factors that influence their accuracy.
The specificity of these tests is very high <span class="citation" data-cites="WuM4npVP">(<a href="#ref-WuM4npVP" role="doc-biblioref">570</a>)</span>, meaning that a positive RT-PCR result is very likely to indicate SARS-CoV-2 infection.
The weight given to these tests as an indicator of SARS-CoV-2 infection regardless of other clinical considerations is not typical <span class="citation" data-cites="cASBrIxl">(<a href="#ref-cASBrIxl" role="doc-biblioref">571</a>)</span>.
In fact, while the analytical specificity of the assay is extremely high, the challenges of implementing testing can introduce variability that results in a lower clinical specificity <span class="citation" data-cites="cASBrIxl">(<a href="#ref-cASBrIxl" role="doc-biblioref">571</a>)</span>.
Several factors may influence the sensitivity and specificity, with sample collection being a critically important factor in the reliability of RT-PCR results.
The most reliable results were found to come from nasopharyngeal swabs and from pooled nasal and throat swabs, with lower accuracies produced by saliva or by throat or nasal swabs alone <span class="citation" data-cites="KcasNvUc WuM4npVP">(<a href="#ref-WuM4npVP" role="doc-biblioref">570</a>, <a href="#ref-KcasNvUc" role="doc-biblioref">572</a>)</span>.
Differences in experimental parameters such as the use of primers more specific to SARS-CoV-2 has been found to improve sensitivity in these specimens <span class="citation" data-cites="n7y3AZZ1">(<a href="#ref-n7y3AZZ1" role="doc-biblioref">573</a>)</span>.
Additionally, the impact of viral evolution on RT-PCR sensitivity is a concern <span class="citation" data-cites="SeNutdHD LbnvnCrn">(<a href="#ref-SeNutdHD" role="doc-biblioref">574</a>, <a href="#ref-LbnvnCrn" role="doc-biblioref">575</a>)</span>.
Using a panel that includes multiple targets can mitigate these effects <span class="citation" data-cites="XsXhZSA8">(<a href="#ref-XsXhZSA8" role="doc-biblioref">576</a>)</span>.
Additionally, a test designed to incorporate genomic differences with SARS-CoV-1 was found to offer improved sensitivity and specificity <span class="citation" data-cites="n7y3AZZ1">(<a href="#ref-n7y3AZZ1" role="doc-biblioref">573</a>)</span>.
Thus, while various factors can influence the exact parameters of testing accuracy, RT-PCR is known to have very high specificity and lower, but still high, sensitivity.</p>
<h4 data-number="3.4.1.2" id="digital-pcr"><span class="header-section-number">3.4.1.2</span> Digital PCR</h4>
<p>Digital PCR (dPCR) is a new generation of PCR technologies offering an alternative to traditional qPCR <span class="citation" data-cites="dEBOtTUz">(<a href="#ref-dEBOtTUz" role="doc-biblioref">577</a>)</span>.
In dPCR, a sample is partitioned into thousands of compartments, such as nanodroplets (droplet dPCR or ddPCR) or nanowells, and a PCR reaction takes place in each compartment.
This design allows for a digital read-out where each partition is either positive or negative for the nucleic acid sequence being tested for, allowing for absolute target quantification through Poisson statistics.
While dPCR equipment is not yet as common as that for qPCR, dPCR for DNA targets generally achieves higher sensitivity than other PCR technologies while maintaining high specificity, though sensitivity is slightly lower for RNA targets <span class="citation" data-cites="Hl2pASeg">(<a href="#ref-Hl2pASeg" role="doc-biblioref">578</a>)</span>.</p>
<p>High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
In one study, Suo et al. <span class="citation" data-cites="2WgqQLF8">(<a href="#ref-2WgqQLF8" role="doc-biblioref">579</a>)</span> performed a double-blind evaluation of ddPCR for SARS-CoV-2 detection.
They evaluated on 63 samples collected from suspected positive outpatients and 14 from supposed convalescent patients.
Of the 63 outpatients, only 21 (33%) were identified as positive for SARS-CoV-2 with qPCR.
However, ddPCR identified 49 (78%) as positive, 10 (16%) as negative, and 4 (6%) as suspected/borderline for SARS-CoV-2 infection.
While both qPCR and ddPCR were found to have very high specificity (100%), this analysis reported that the sensitivity was 40% with qPCR compared to 94% with ddPCR.
Analysis of serial dilutions of a linear DNA standard suggested that ddPCR was approximately 500 times more sensitive than qPCR <span class="citation" data-cites="2WgqQLF8">(<a href="#ref-2WgqQLF8" role="doc-biblioref">579</a>)</span>.
Thus, this study suggests that ddPCR provides an extremely sensitive molecular test that is able to detect SARS-CoV-2 even at very low viral loads.</p>
<p>A second study <span class="citation" data-cites="duB3pL6y">(<a href="#ref-duB3pL6y" role="doc-biblioref">580</a>)</span> confirmed that RT-ddPCR is able to detect SARS-CoV-2 at a lower threshold for viral load relative to RT-PCR.
This study analyzed 196 samples, including 103 samples from suspected patients, 77 from contacts and close contacts, and 16 from suspected convalescents, using both RT-qPCR and RT-ddPCR.
First, the authors evaluated samples from the 103 suspected cases.
Using RT-qPCR, 29 (28%) were identified as positive, 25 (24%) as negative, and 49 (48%) as borderline, i.e., the C<sub>t</sub> value was higher than the positive threshold of 35 but lower than the negative threshold of 40.
When the 61 negative and borderline samples were reanalyzed with ddPCR, 19 (31%) of the negative and 42 (69%) of the borderline samples were identified as positive.
All of the suspected cases were later confirmed to be COVID-19 through a combination of symptom development and RT-qPCR resampling, indicating that ddPCR improved the overall detection rate compared to RT-qPCR from 28.2% to 87.4%.</p>
<p>They repeated this analysis in patient samples from contacts and close contacts.
Patients who tested negative with both methods (n = 48) were observed to remain healthy over a period of 14 days.
Among the remaining 29 samples from contacts, RT-qPCR identified 12 as positive, 1 as negative, and 16 as borderline.
All of the samples that tested positive using RT-qPCR also tested positive using ddPCR.
In contrast, the negative result and all but one of the borderline results were identified as positive by RT-ddPCR, and these patients were later determined to be SARS-CoV-2 positive based on clinical evaluation and repeated molecular sampling.
Similarly, in the final group, 16 convalescent patients, RT-qPCR identified 12 as positive, three as suspect, and one as negative, but RT-dPCR identified all as positive.
The evidence from this study therefore supports a lower limit of detection with ddPCR.
Overall, these studies suggest that ddPCR is a promising tool for overcoming the problem of false negatives in SARS-CoV-2 RNA testing, but this method is unlikely to affect the current pandemic due to its lack of availability.</p>
<h4 data-number="3.4.1.3" id="sequencing"><span class="header-section-number">3.4.1.3</span> Sequencing</h4>
<p>In some cases, the DNA amplified with PCR is sequenced.
Sequencing requires an additional sample pre-processing step called library preparation.
Library preparation is the process of preparing the sample for sequencing, typically by fragmenting the sequences and adding adapters <span class="citation" data-cites="jr3z6jGU">(<a href="#ref-jr3z6jGU" role="doc-biblioref">581</a>)</span>.
In some cases, library preparation can involve other modifications of the sample, such as adding barcodes to identify a particular sample within the sequence data.
Barcoding can therefore be used to pool samples from multiple sources.
There are different reagents used for library preparation that are specific to identifying one or more target sections with PCR <span class="citation" data-cites="43modeVu">(<a href="#ref-43modeVu" role="doc-biblioref">582</a>)</span>.
Sequential pattern matching is then used to identify unique subsequences of the virus, and if sufficient subsequences are found, the test is considered positive.
Therefore, tests that use sequencing require a number of additional molecular and analytical steps relative to tests that use PCR alone.</p>
<p>Sequencing has been an important strategy for discovery of SARS-CoV-2 variants (e.g., see <span class="citation" data-cites="1Ha4IVlGr">(<a href="#ref-1Ha4IVlGr" role="doc-biblioref">500</a>)</span>).
Sequencing elucidates any genetic variants located between the PCR primers.
For this reason, it is critical to genomic surveillance efforts.
Genomic surveillance is an important complement to epidemiological surveillance efforts <span class="citation" data-cites="6NDTv0Dw">(<a href="#ref-6NDTv0Dw" role="doc-biblioref">583</a>)</span>, as described below.
Through genomic surveillance, it has become possible to monitor the emergence of variants of interest and variants of concern (VOC) that may pose additional threats due to increased contagiousness, virulence, or immune escape <span class="citation" data-cites="6NDTv0Dw O7L5UTpW">(<a href="#ref-6NDTv0Dw" role="doc-biblioref">583</a>, <a href="#ref-O7L5UTpW" role="doc-biblioref">584</a>)</span>.
Sequencing also allows for analysis of the dominant strains in an area at a given time.
Worldwide, the extent of genomic surveillance varies widely, with higher-income countries typically able to sequence a higher percentage of cases <span class="citation" data-cites="14QdeINFG">(<a href="#ref-14QdeINFG" role="doc-biblioref">585</a>)</span>.
Sequencing efforts are important for identifying variants containing mutations that might affect the reliability of molecular diagnostic tests, as well as mitigation measures such as therapeutics and prophylactics <span class="citation" data-cites="SeNutdHD LbnvnCrn">(<a href="#ref-SeNutdHD" role="doc-biblioref">574</a>, <a href="#ref-LbnvnCrn" role="doc-biblioref">575</a>)</span>.
Therefore, sequencing is an important component of diagnostics: while it is not necessary for diagnosing an individual case, it is critical to monitoring trends in the variants affecting a population and to staying aware of emerging variants that may pose additional challenges.</p>
<h4 data-number="3.4.1.4" id="pooled-and-automated-pcr-testing"><span class="header-section-number">3.4.1.4</span> Pooled and Automated PCR Testing</h4>
<p>Due to limited supplies and the need for more tests, several labs have found ways to pool or otherwise strategically design tests to increase throughput.
The first such result came from Yelin et al. <span class="citation" data-cites="nIW5kgMT">(<a href="#ref-nIW5kgMT" role="doc-biblioref">586</a>)</span>, who reported that they could pool up to 32 samples in a single qPCR run.
This was followed by larger-scale pooling with slightly different methods <span class="citation" data-cites="IQ5kUQEO">(<a href="#ref-IQ5kUQEO" role="doc-biblioref">587</a>)</span>.
Although these approaches are also PCR based, they allow for more rapid scaling and higher efficiency for testing than the initial PCR-based methods developed.
Conceptually, pooling could also be employed in analysis with RT-qPCR <span class="citation" data-cites="KjeNmVZ">(<a href="#ref-KjeNmVZ" role="doc-biblioref">588</a>)</span>, and this strategy has been evaluated in settings such as schools <span class="citation" data-cites="161uMPkFJ">(<a href="#ref-161uMPkFJ" role="doc-biblioref">589</a>)</span> and hospitals <span class="citation" data-cites="9KChw8BC">(<a href="#ref-9KChw8BC" role="doc-biblioref">590</a>)</span>.</p>
<h3 data-number="3.4.2" id="rt-lamp"><span class="header-section-number">3.4.2</span> RT-LAMP</h3>
<p>RT-PCR remains the gold standard for detection of SARS-CoV-2 RNA from infected patients, but the traditional method requires special equipment and reagents, including a thermocycler.
Loop-mediated isothermal amplification (LAMP) is an alternative to PCR that does not require specialized equipment <span class="citation" data-cites="mk0qreRK">(<a href="#ref-mk0qreRK" role="doc-biblioref">591</a>)</span>.
In this method, nucleic acids are amplified in a 25 μL reaction that is incubated and chilled on ice <span class="citation" data-cites="mk0qreRK">(<a href="#ref-mk0qreRK" role="doc-biblioref">591</a>)</span>.
It uses primers designed to facilitate auto-cycling strand displacement DNA synthesis <span class="citation" data-cites="mk0qreRK">(<a href="#ref-mk0qreRK" role="doc-biblioref">591</a>)</span>.
LAMP can be combined with reverse transcription (RT-LAMP) to enable the detection of RNA.</p>
<p>One study showed that RT-LAMP is effective for detection of SARS-CoV-2 with excellent specificity and sensitivity and that this method can be applied to unprocessed saliva samples <span class="citation" data-cites="NzLjxhcg">(<a href="#ref-NzLjxhcg" role="doc-biblioref">592</a>)</span>.
This method was benchmarked against RT-PCR using 177 human nasopharyngeal RNA samples, of which 126 were COVID positive.
The authors break down the sensitivity of their test according to the C<sub>t</sub> value from RT-PCR of the same samples; RT-LAMP performs at 100% sensitivity for samples with a C<sub>t</sub> from RT-PCR of 32 or less.
The performance is worse when considering all RT-PCR positive samples (including those with C<sub>t</sub> values between 32-40).
However, there is some evidence suggesting that samples obtained from individuals that achieve C<sub>t</sub> values &gt;30 measured using RT-PCR tend to be less infective that those that record a C<sub>t</sub> value &lt;30 <span class="citation" data-cites="qLIqQl48 8T6lbq9V 1H1RimUDg">(<a href="#ref-qLIqQl48" role="doc-biblioref">593</a>–<a href="#ref-1H1RimUDg" role="doc-biblioref">595</a>)</span>, so RT-LAMP is still a useful diagnostic tool.
Various combinations of reagents are available, but one example is the WarmStart Colorimetric LAMP 2X Master Mix with a set of six primers developed previously by Zhang et al. <span class="citation" data-cites="HlmXwIYD">(<a href="#ref-HlmXwIYD" role="doc-biblioref">596</a>)</span>.
To determine assay sensitivity, serial tenfold dilutions of <em>in vitro</em> transcribed <em>N</em>-gene RNA standard were tested using quantities from 10<sup>5</sup> copies down to 10 copies.
The assay readout is the color of the dye changing from pink to yellow due to binding to the DNA product over 30 minutes.
The RT-LAMP assay was then applied to clinical nasopharyngeal samples.
For viral loads above 100 copies of genomic RNA, the RT-LAMP assay had a sensitivity of 100% and a specificity of 96.1% from purified RNA.
The sensitivity of the direct assay of saliva by RT-LAMP was 85%.
Sensitivity and specificity metrics were obtained by comparison with results from RT-PCR.
RT-LAMP pilot studies for detection of SARS-CoV-2 were reviewed in a meta-analysis <span class="citation" data-cites="1HXZ5qohE">(<a href="#ref-1HXZ5qohE" role="doc-biblioref">597</a>)</span>.
In the meta-analysis of all 2,112 samples, the cumulative sensitivity of RT-LAMP was calculated at 95.5%, and the cumulative specificity was 99.5%.</p>
<p>This test aims to bring the sensitivity of nucleic acid detection to the point of care or home testing setting.
It could be applied for screening, diagnostics, or as a definitive test for people who are positive based on LFTs (see below).
The estimated cost per test is about 2 euros when RNA extraction is included.
The main strength of this test over RT-PCR is that it can be done isothermally, but the main drawback is that it is about 10-fold less sensitive than RT-PCR.
The low cost, excellent sensitivity/specificity, and quick readout of RT-LAMP makes this an attractive alternative to RT-PCR.
Alternative strategies like RT-LAMP are needed to bring widespread testing away from the lab and into under-resourced areas.</p>
<h3 data-number="3.4.3" id="crispr-based-detection"><span class="header-section-number">3.4.3</span> CRISPR-based Detection</h3>
<p>Technology based on CRISPR (clustered regularly interspaced short palindromic repeats) <span class="citation" data-cites="LDEN4bPG">(<a href="#ref-LDEN4bPG" role="doc-biblioref">598</a>)</span> has also been instrumental in scaling up testing protocols.
Two CRISPR-associated nucleases, Cas12 and Cas13, have been used for nucleic acid detection.
Multiple assays exploiting these nucleases have emerged as potential diagnostic tools for the rapid detection of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The SHERLOCK method (Specific High-sensitivity Enzymatic Reporter unLOCKing) from Sherlock Biosciences relies on Cas13a to discriminate between inputs that differ by a single nucleotide at very low concentrations <span class="citation" data-cites="vVBrQqWB">(<a href="#ref-vVBrQqWB" role="doc-biblioref">599</a>)</span>.
The target RNA is amplified by real-time recombinase polymerase amplification (RT-RPA) and T7 transcription, and the amplified product activates Cas13a.
The nuclease then cleaves a reporter RNA, which liberates a fluorescent dye from a quencher.
Several groups have used the SHERLOCK method to detect SARS-CoV-2 viral RNA.
An early study reported that the method could detect 7.5 copies of viral RNA in all 10 replicates, 2.5 copies in 6 out of 10, and 1.25 copies in 2 out of 10 runs <span class="citation" data-cites="fqgjuNaJ">(<a href="#ref-fqgjuNaJ" role="doc-biblioref">600</a>)</span>.
It also reported 100% specificity and sensitivity on 114 RNA samples from clinical respiratory samples (61 suspected cases, among which 52 were confirmed and nine were ruled out by metagenomic next-generation sequencing, 17 SARS-CoV-2-negative but human coronavirus (HCoV)-positive cases, and 36 samples from healthy subjects) and a reaction turnaround time of 40 minutes.
A separate study screened four designs of SHERLOCK and extensively tested the best-performing assay.
They determined the limit of detection to be 10 copies/μl using both fluorescent and lateral flow detection <span class="citation" data-cites="zSefi0ye">(<a href="#ref-zSefi0ye" role="doc-biblioref">601</a>)</span>.</p>
<p>LFT strips are simple to use and read, but there are limitations in terms of availability and cost per test.
Another group therefore proposed the CREST (Cas13-based, Rugged, Equitable, Scalable Testing) protocol, which uses a P51 cardboard fluorescence visualizer, powered by a 9-volt battery, for the detection of Cas13 activity instead of immunochromatography <span class="citation" data-cites="160dYAlIO">(<a href="#ref-160dYAlIO" role="doc-biblioref">602</a>)</span>.
CREST can be run, from RNA sample to result, with no need for AC power or a dedicated facility, with minimal handling in approximately 2 hours.
Testing was performed on 14 nasopharyngeal swabs.
CREST picked up the same positives as the CDC-recommended TaqMan assay with the exception of one borderline sample that displayed low-quality RNA.
This approach may therefore represent a rapid, accurate, and affordable procedure for detecting SARS-CoV-2.</p>
<p>The DETECTR (DNA Endonuclease-Targeted CRISPR Trans Reporter) method from Mammoth Biosciences involves purification of RNA extracted from patient specimens, amplification of extracted RNAs by loop-mediated amplification, and application of their Cas12-based technology.
In this assay, guide RNAs (gRNAs) were designed to recognize portions of sequences corresponding to the SARS-CoV-2 genome, specifically the N2 and E regions <span class="citation" data-cites="XWCinLld">(<a href="#ref-XWCinLld" role="doc-biblioref">603</a>)</span>.
In the presence of SARS-CoV-2 genetic material, sequence recognition by the gRNAs results in double-stranded DNA cleavage by Cas12, as well as cleavage of a single-stranded DNA molecular beacon.
The cleavage of this molecular beacon acts as a colorimetric reporter that is subsequently read out in a lateral flow assay and indicates the presence of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The 40-minute assay is considered positive if there is detection of both the <em>E</em> and <em>N</em> genes or presumptive positive if there is detection of either of them.
The assay had 95% positive predictive agreement and 100% negative predictive agreement with the US Centers for Disease Control and Prevention SARS-CoV-2 RT-qPCR assay.
The estimated limit of detection was 10 copies per μl reaction, versus 1 copy per μl reaction for the CDC assay.</p>
<p>These results have been confirmed by other DETECTR approaches.
Using RT-RPA for amplification, another group detected 10 copies of synthetic SARS-CoV-2 RNA per μl of input within 60 minutes of RNA sample preparation in a proof-of-principle evaluation <span class="citation" data-cites="4T3aId7a">(<a href="#ref-4T3aId7a" role="doc-biblioref">604</a>)</span>.
Through a similar approach, another group reported detection at 1 copy per μl <span class="citation" data-cites="qWJSwveX">(<a href="#ref-qWJSwveX" role="doc-biblioref">605</a>)</span>.
The DETECTR protocol was improved by combining RT-RPA and CRISPR-based detection in a one-pot reaction that incubates at a single temperature and by using dual CRISPR RNAs, which increases sensitivity.
This new assay, known as All-In-One Dual CRISPR-Cas12a, detected 4.6 copies of SARS-CoV-2 RNA per μl of input in 40 minutes <span class="citation" data-cites="wZKGHcO2">(<a href="#ref-wZKGHcO2" role="doc-biblioref">606</a>)</span>.
Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/μl in 40-60 minutes <span class="citation" data-cites="A8dUPgQd">(<a href="#ref-A8dUPgQd" role="doc-biblioref">607</a>)</span>.</p>
<p>It was also reported that electric field gradients can be used to control and accelerate CRISPR assays by co-focusing Cas12-gRNA, reporters, and target <span class="citation" data-cites="jPdxTaCe">(<a href="#ref-jPdxTaCe" role="doc-biblioref">608</a>)</span>.
The authors generated an appropriate electric field gradient using a selective ionic focusing technique known as isotachophoresis (ITP) implemented on a microfluidic chip.
They also used ITP for automated purification of target RNA from raw nasopharyngeal swab samples.
Combining this ITP purification with loop-mediated isothermal amplification, their ITP-enhanced assay achieved detection of SARS-CoV-2 RNA (from raw sample to result) in 30 minutes.</p>
<p>All these methods require upstream nucleic acid amplification prior to CRISPR-based detection.
They rely on type V (Cas12-based) and type IV (Cas13-based) CRISPR systems.
In contrast, type III CRISPR systems have the unique property of initiating a signaling cascade, which could boost the sensitivity of direct RNA detection.
In type III CRISPR systems, guide CRISPR RNAs (crRNAs) are bound by several Cas proteins <span class="citation" data-cites="16ItkGAe8">(<a href="#ref-16ItkGAe8" role="doc-biblioref">609</a>)</span> and can target both DNA and RNA molecules <span class="citation" data-cites="RjmcDhmt 1HTJW23NU">(<a href="#ref-RjmcDhmt" role="doc-biblioref">610</a>, <a href="#ref-1HTJW23NU" role="doc-biblioref">611</a>)</span>.
A study tested this hypothesis using the type III-A crRNA-guided surveillance complex from <em>Thermus thermophilus</em> <span class="citation" data-cites="QPNcH8m1">(<a href="#ref-QPNcH8m1" role="doc-biblioref">612</a>)</span>.
The authors showed that activation of the Cas10 polymerase generates three products (cyclic nucleotides, protons, and pyrophosphates) that can all be used to detect SARS-CoV-2 RNA.
Detection of viral RNA in patient samples still required an initial nucleic acid amplification step, but improvements may in the future remove that requirement.</p>
<p>This goal of amplification-free detection was later achieved for a Cas13a-based system <span class="citation" data-cites="ZwqOFOo8">(<a href="#ref-ZwqOFOo8" role="doc-biblioref">613</a>)</span>.
This approach combined multiple CRISPR RNAs to increase Cas13a activation, which is detected by a fluorescent reporter.
Importantly, because the viral RNA is detected directly, the test yields a quantitative measurement rather than a binary result.
The study also shows that fluorescence can be measured in a custom-made dark box with a mobile phone camera and a low-cost laser illumination and collection optics.
This approach is a truly portable assay for point-of-care diagnostics.
The authors achieved detection of 100 copies/μl of pre-isolated RNA in 30 minutes, and correctly identified all SARS-CoV-2-positive patient RNA samples tested in 5 minutes (n = 20).</p>
<p>There is an increasing body of evidence that CRISPR-based assays offer a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.
In the largest study to date, DETECTR was compared to RT-qPCR on 378 patient samples <span class="citation" data-cites="KfuBLemx">(<a href="#ref-KfuBLemx" role="doc-biblioref">614</a>)</span>.
The authors reported 95% reproducibility.
Both techniques were equally sensitive in detecting SARS-CoV-2.
Lateral flow strips showed 100% correlation to the high-throughput DETECTR assay.
Importantly, DETECTR was 100% specific for SARS-CoV-2 and did not detect other human coronaviruses.
A method based on a Cas9 ortholog from <em>Francisella novicida</em> known as FnCas9 achieved 100% sensitivity and 97% specificity in clinical samples, and the diagnostic kit is reported to have completed regulatory validation in India <span class="citation" data-cites="15pFvXhZ">(<a href="#ref-15pFvXhZ" role="doc-biblioref">615</a>)</span>.</p>
<h3 data-number="3.4.4" id="immunoassays-for-the-detection-of-antigens"><span class="header-section-number">3.4.4</span> Immunoassays for the Detection of Antigens</h3>
<p>Immunoassays can detect molecular indicators of SARS-CoV-2 infection, such as the proteins that act as antigens from the SARS-CoV-2 virus.
They offer the advantage of generally being faster and requiring less specialized equipment than other molecular tests, especially those involving PCR.
As a result, immunoassays hold particular interest for implementation at home and in situations where resources for PCR testing are limited.
The trade-off is that these tests typically have a lower sensitivity, and sometimes a lower specificity, than other molecular tests.
However, these tests tend to return a positive result five to 12 days after symptom onset, which may therefore correlate more closely with the timeframe during which viral replication occurs <span class="citation" data-cites="Ki0q4jhw">(<a href="#ref-Ki0q4jhw" role="doc-biblioref">616</a>)</span>.
Immunoassays for the detection of the SARS-CoV-2 antigen can include LFTs and ELISA, as discussed here, as well as CLIA and chromatographic immunoassays <span class="citation" data-cites="XriI70m9">(<a href="#ref-XriI70m9" role="doc-biblioref">617</a>)</span>, as described in the serological testing section below.</p>
<h4 data-number="3.4.4.1" id="lateral-flow-tests"><span class="header-section-number">3.4.4.1</span> Lateral Flow Tests</h4>
<p>LFTs provide distinct value relative to PCR tests.
They can return results within 30 minutes and can be performed without specialized equipment and at low cost.
They also do not require training to operate and are cheap to produce.
Thus, they can be distributed widely to affected populations making them an important public health measure to curb pandemic spread.
LFTs rely on the detection of viral protein with an antibody.
Often this is done with an antibody sandwich format, where one antibody conjugated to a dye binds at one site on the antigen, and an immobilized antibody on the strip binds at another site <span class="citation" data-cites="PCd1Mvel">(<a href="#ref-PCd1Mvel" role="doc-biblioref">558</a>)</span>.
This design allows the dye to accumulate to form a characteristic positive test line on the strip <span class="citation" data-cites="PCd1Mvel">(<a href="#ref-PCd1Mvel" role="doc-biblioref">558</a>)</span>.
Outside of COVID-19 diagnostics, the applications of LFTs are broad; they are routinely used for home pregnancy tests, disease detection, and even drugs of abuse detection in urine <span class="citation" data-cites="nD6CEooX">(<a href="#ref-nD6CEooX" role="doc-biblioref">618</a>)</span>.</p>
<p>A recent review surveyed the performance of LFTs for detection of current SARS-CoV-2 infection <span class="citation" data-cites="tvdJOfV9">(<a href="#ref-tvdJOfV9" role="doc-biblioref">619</a>)</span>.
This review covered 24 studies that included more than 26,000 total LFTs.
They reported significant heterogeneity in test sensitivities, with estimates ranging from 37.7% to 99.2%.
The estimated specificities of these tests were more homogeneous, spanning 92.4% to 100.0%.</p>
<p>Despite having lower sensitivity than PCR tests, LFTs occupy an important niche in the management of SARS-CoV-2.
Current infection detection by LFTs enables the scale and speed of testing that is beneficial to managing viral spread.
LFTs were available freely to citizens in the United Kingdom until April 1, 2022 <span class="citation" data-cites="cLQbewzh">(<a href="#ref-cLQbewzh" role="doc-biblioref">620</a>)</span> and to citizens of the United States in early 2022 <span class="citation" data-cites="hFI37GHb">(<a href="#ref-hFI37GHb" role="doc-biblioref">621</a>)</span>.
These tests are particularly useful for ruling out SARS-CoV-2 infection in cases where the likelihood of infection is low (e.g., asymptomatic individuals) and positives (including false positives) can be validated with testing by alternate means <span class="citation" data-cites="1Bm6qAnYY">(<a href="#ref-1Bm6qAnYY" role="doc-biblioref">622</a>)</span>.</p>
<h4 data-number="3.4.4.2" id="enzyme-linked-immunosorbent-assay"><span class="header-section-number">3.4.4.2</span> Enzyme-Linked Immunosorbent Assay</h4>
<p>ELISA is a very sensitive immunoassay that can be considered a gold standard for the detection of biological targets, including antibodies and antigenic proteins <span class="citation" data-cites="w3RMVma8">(<a href="#ref-w3RMVma8" role="doc-biblioref">559</a>)</span>.
It can be used to generate either quantitative or qualitative results that can be returned within a few hours <span class="citation" data-cites="ayaV1tKm">(<a href="#ref-ayaV1tKm" role="doc-biblioref">623</a>)</span>.
ELISA builds on the idea that antibodies and antigens bind together to form complexes <span class="citation" data-cites="w3RMVma8">(<a href="#ref-w3RMVma8" role="doc-biblioref">559</a>)</span> and utilizes an enzyme covalently linked to an antibody against the antigen to produce assay signal, usually a color change <span class="citation" data-cites="15HaJFv2B">(<a href="#ref-15HaJFv2B" role="doc-biblioref">624</a>)</span>.
The main advantage of ELISA is that it enables signal amplification through the enzyme’s activity, which increases sensitivity.
With sandwich ELISA, antibodies are immobilized on a surface such as a plate, and viral protein antigens in the sample bind and are retained <span class="citation" data-cites="XVdGUsUr">(<a href="#ref-XVdGUsUr" role="doc-biblioref">625</a>)</span>.
A second antibody is added that binds to another site on the antigen is then added, and that second antibody is covalently linked to an enzyme.
A substrate for that enzyme is then added to produce signal, usually light or a color change
The exact strategy for tagging with a reporter enzyme varies among different types of ELISA <span class="citation" data-cites="XVdGUsUr w3RMVma8">(<a href="#ref-w3RMVma8" role="doc-biblioref">559</a>, <a href="#ref-XVdGUsUr" role="doc-biblioref">625</a>)</span>.
For COVID-19 diagnostics, ELISAs have been designed to detect the antigenic Spike protein <span class="citation" data-cites="lrHedci6">(<a href="#ref-lrHedci6" role="doc-biblioref">626</a>)</span>.</p>
<p>One of these assays uses two monoclonal antibodies specific to the nucleocapsid of SARS-CoV-2 to evaluate the relationship between the effect of (estimated) viral load on the ability of the assay to detect the SARS-CoV-2 antigen <span class="citation" data-cites="11QmOzidP">(<a href="#ref-11QmOzidP" role="doc-biblioref">627</a>)</span>.
This study analyzed 339 naso-oropharyngeal samples that were also analyzed with RT-qPCR as a gold standard.
RT-qPCR identified 147 samples as positive and 192 as negative.
The authors estimated the overall sensitivity and specificity to be 61.9% and 99.0%, respectively.
Sensitivity increased with higher C<sub>t</sub>.
This study also assessed the performance of the ELISA test under different conditions in order to evaluate how robust it would be to the challenges of testing in real-world settings globally.
Higher sensitivity was achieved for samples that were stored under ideal conditions (immediate placement in -80° C).
Therefore, while immediate access to laboratory equipment is an advantage, it is not strictly necessary for ELISA to detect the antigen.</p>
<h3 data-number="3.4.5" id="limitations-of-molecular-tests"><span class="header-section-number">3.4.5</span> Limitations of Molecular Tests</h3>
<p>Tests that identify SARS-CoV-2 using molecular technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection.
Among molecular tests, different technologies have different sensitivities and specificities.
In general, specificity is high, and even then, the public health repercussions of a false positive can generally be mitigated with follow-up testing.
On the other hand, a test’s sensitivity, which indicates the risk of a false-negative response, can pose significant challenge to large-scale testing.
False negatives are a significant concern for several reasons.
Importantly, clinical reports indicate that it is imperative to exercise caution when interpreting the results of molecular tests for SARS-CoV-2 because negative results do not necessarily mean a patient is virus-free <span class="citation" data-cites="11vIuqmS2">(<a href="#ref-11vIuqmS2" role="doc-biblioref">628</a>)</span>.
To reduce occurrence of false negatives, correct execution of the analysis is crucial <span class="citation" data-cites="Fdc7seLs">(<a href="#ref-Fdc7seLs" role="doc-biblioref">629</a>)</span>.
Additionally, PCR-based tests can remain positive for a much longer time than the virus is likely to be actively replicating <span class="citation" data-cites="Ki0q4jhw">(<a href="#ref-Ki0q4jhw" role="doc-biblioref">616</a>)</span>, raising concerns about their informativeness after the acute phase of the disease.
Hence, the CDC has advised individuals who suspect they have been re-infected with SARS-CoV-2 to avoid using diagnostic tests within 90 days of receiving a previous positive test <span class="citation" data-cites="1HtWSk427">(<a href="#ref-1HtWSk427" role="doc-biblioref">630</a>)</span>.</p>
<p>Additionally, the emerging nature of the COVID-19 pandemic has introduced some challenges related to uncertainty surrounding interactions between SARS-CoV-2 and its human hosts.
For example, viral shedding kinetics are still not well understood but are expected to introduce a significant effect of timing of sample collection on test results <span class="citation" data-cites="Fdc7seLs">(<a href="#ref-Fdc7seLs" role="doc-biblioref">629</a>)</span>.
Similarly, the type of specimen could also influence outcomes, as success in viral detection varies among clinical sample types <span class="citation" data-cites="Fdc7seLs KcasNvUc WuM4npVP">(<a href="#ref-WuM4npVP" role="doc-biblioref">570</a>, <a href="#ref-KcasNvUc" role="doc-biblioref">572</a>, <a href="#ref-Fdc7seLs" role="doc-biblioref">629</a>)</span>.
With CRISPR-based testing strategies, the gRNA can recognize off-target interspersed sequences in the viral genome <span class="citation" data-cites="MNxZTL29">(<a href="#ref-MNxZTL29" role="doc-biblioref">631</a>)</span>, potentially resulting in false positives and a loss of specificity.</p>
<p>There are also significant practical and logistical concerns related to the widespread deployment of molecular tests.
Much of the technology used for molecular tests is expensive, and while it might be available in major hospitals and/or diagnostic centers, it is often not available to smaller facilities <span class="citation" data-cites="NT5mrVGY">(<a href="#ref-NT5mrVGY" role="doc-biblioref">632</a>)</span>.
At times during the pandemic, the availability of supplies for testing, including swabs and testing media, has also been limited <span class="citation" data-cites="AvQI3GSm">(<a href="#ref-AvQI3GSm" role="doc-biblioref">633</a>)</span>.
Similarly, processing times can be long, and tests might take up to 4 days to return results <span class="citation" data-cites="NT5mrVGY">(<a href="#ref-NT5mrVGY" role="doc-biblioref">632</a>)</span>, especially during times of high demand, such as spikes in case numbers <span class="citation" data-cites="U8dFUyCj">(<a href="#ref-U8dFUyCj" role="doc-biblioref">634</a>)</span>.
Countries have employed various and differing molecular testing strategies as a tool to reduce viral transmission, even among high-income countries <span class="citation" data-cites="15MEHt7vJ">(<a href="#ref-15MEHt7vJ" role="doc-biblioref">635</a>)</span>.
The rapid development of molecular tests has provided a valuable, albeit imperfect, tool to identify active SARS-CoV-2 infections.</p>
<h2 data-number="3.5" id="serological-tests-to-identify-recovered-individuals"><span class="header-section-number">3.5</span> Serological Tests to Identify Recovered Individuals</h2>
<p>Although several molecular diagnostic tests to detect viral genetic material have high specificity and sensitivity, they provide information only about active infection, and therefore offer just a snapshot-in-time perspective on the spread of a disease.
Most importantly, they would not work on a patient who has fully recovered from the virus at the time of sample collection.
In such contexts, serological tests are informative.</p>
<p>Serological tests use many of the same technologies as the immunoassays used to detect the presence of an antigen but are instead used to evaluate the presence of antibodies against SARS-CoV-2 in a serum sample.
These tests are particularly useful for insight into population-level dynamics and can also offer a glimpse into the development of antibodies by individual patients during the course of a disease.
Immunoassays can detect antibodies produced by the adaptive immune system in response to viral threat.
Understanding the acquisition and retention of antibodies is important both to the diagnosis of prior (inactive) infections and to the development of vaccines.
The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies <span class="citation" data-cites="fB995nF4 5ejmbqQZ">(<a href="#ref-fB995nF4" role="doc-biblioref">636</a>, <a href="#ref-5ejmbqQZ" role="doc-biblioref">637</a>)</span>.
Serological tests detect these antibodies, offering a mechanism through which prior infection can be identified.
However, the complexity of the human immune response means that there are many facets to such analyses.</p>
<p>In general, SARS-CoV-2 infection will induce the immune system to produce antibodies fairly quickly.
Prior research is available about the development of antibodies to SARS-CoV-1 during the course of the associated disease, severe acute respiratory syndrome (SARS).
IgM and IgG antibodies were detected in the second week following SARS-CoV-1 infection.
IgM titers peaked by the first month post-infection, and then declined to undetectable levels after day 180.
IgG titers peaked by day 60 and persisted in all donors through the two-year duration of study <span class="citation" data-cites="pGXsPCCz">(<a href="#ref-pGXsPCCz" role="doc-biblioref">638</a>)</span>.
Such tests can also illuminate the progression of viral disease, as IgM are the first antibodies produced by the body and indicate that the infection is active.
Once the body has responded to the infection, IgG are produced and gradually replace IgM, indicating that the body has developed immunogenic memory <span class="citation" data-cites="FFMfMDRN">(<a href="#ref-FFMfMDRN" role="doc-biblioref">639</a>)</span>.
Therefore, it was hoped that the development of assays to detect the presence of IgM and IgG antibodies against SARS-CoV-2 would allow the identification of cases from early in the infection course (via IgM) and for months or years afterwards (via IgG).
Several technologies have been used to develop serological tests for COVID-19, including ELISA, lateral flow immunoassay, chemiluminescence immunoassay, and neutralizing antibody assays <span class="citation" data-cites="KxpufU9U">(<a href="#ref-KxpufU9U" role="doc-biblioref">640</a>)</span>.</p>
<h3 data-number="3.5.1" id="elisa"><span class="header-section-number">3.5.1</span> ELISA</h3>
<p>The application of ELISA to serological testing is complementary to its use in molecular diagnostics (see above).
Instead of using an enzyme-labeled antibody as a probe that binds to the target antigen, the probe is an antigen and the target is an antibody.
The enzyme used for detection and signal amplification is on a secondary antibody raised generally against human IgG or IgM.
In March 2020, the Krammer lab proposed an ELISA test that detects IgG and IgM that react against the receptor-binding domain (RBD) of the spike proteins (S) of the virus <span class="citation" data-cites="rvAb0ZJy">(<a href="#ref-rvAb0ZJy" role="doc-biblioref">641</a>)</span>.
A subsequent ELISA test developed to detect SARS-CoV-2 IgG based on the RBD reported a specificity of over 99% and a sensitivity of up to 88.24%, which was observed in samples collected 21 to 27 days after the onset of infection (approximated with symptom onset or positive PCR test) <span class="citation" data-cites="pnHr92FQ">(<a href="#ref-pnHr92FQ" role="doc-biblioref">642</a>)</span>.
Earlier in the disease course, sensitivity was lower: 53.33% between days 0 and 13 and 80.47% between days 14 and 20.
This study reported that their laboratory ELISA outperformed two commercial kits that also used an ELISA design <span class="citation" data-cites="pnHr92FQ">(<a href="#ref-pnHr92FQ" role="doc-biblioref">642</a>)</span>.
Therefore, while analysis with ELISA requires laboratory support and equipment, these results do suggest that ELISA achieves relatively high sensitivity, especially in the weeks following infection.
Efforts have been made to develop low-cost strategies for conducting these tests that will make them more accessible worldwide <span class="citation" data-cites="JEaoCUco">(<a href="#ref-JEaoCUco" role="doc-biblioref">643</a>)</span>.</p>
<h3 data-number="3.5.2" id="chemiluminescence-immunoassay"><span class="header-section-number">3.5.2</span> Chemiluminescence Immunoassay</h3>
<p>Another early approach investigated for detection of antibodies against SARS-CoV-2 was CLIA.
Like ELISA, CLIA is a type of enzyme immunoassay (EIA) <span class="citation" data-cites="IAZeyZZW">(<a href="#ref-IAZeyZZW" role="doc-biblioref">644</a>)</span>.
While the technique varies somewhat, in one approach, a bead is coated with the antigen and then washed with the sample <span class="citation" data-cites="13kfmvz1C">(<a href="#ref-13kfmvz1C" role="doc-biblioref">645</a>)</span>.
If the antibody is present in the sample, it will bind to the bead.
Then the bead is exposed to a label, a luminescent molecule that will bind to the antigen/antibody complex and can therefore be used as an indicator <span class="citation" data-cites="13kfmvz1C">(<a href="#ref-13kfmvz1C" role="doc-biblioref">645</a>)</span>.
One CLIA approach to identify COVID-19 used a synthetic peptide derived from the amino acid sequence of the SARS-CoV-2 S protein <span class="citation" data-cites="ciLSdA3x">(<a href="#ref-ciLSdA3x" role="doc-biblioref">646</a>)</span>.
It was highly specific to SARS-CoV-2 and detected IgM in 57.2% and IgG in 71.4% of serum samples from 276 COVID-19 cases confirmed with RT-qPCR.
IgG could be detected within two days of the onset of fever, but IgM could not be detected any earlier <span class="citation" data-cites="ciLSdA3x">(<a href="#ref-ciLSdA3x" role="doc-biblioref">646</a>)</span>, which has been supported by other analyses as well <span class="citation" data-cites="sGrolLjn">(<a href="#ref-sGrolLjn" role="doc-biblioref">647</a>)</span>.
This pattern was consistent with observations in Middle East respiratory syndrome, which is also caused by an HCoV.
In comparisons of different commercial immunoassays, accuracy of CLIA tests were often roughly comparable to other EIAs <span class="citation" data-cites="Lu9hVHtO">(<a href="#ref-Lu9hVHtO" role="doc-biblioref">648</a>)</span>, although one CLIA did not perform as well as several other EIAs <span class="citation" data-cites="sGrolLjn gwsGoZAk">(<a href="#ref-sGrolLjn" role="doc-biblioref">647</a>, <a href="#ref-gwsGoZAk" role="doc-biblioref">649</a>)</span>.
The sensitivity and specificities reported vary among CLIA tests and for the detection of IgM versus IgG, but sensitivities and specificities as high as 100% have been reported among various high-throughput tests <span class="citation" data-cites="OT7f5HVt gwsGoZAk YZJK3c9X">(<a href="#ref-gwsGoZAk" role="doc-biblioref">649</a>–<a href="#ref-YZJK3c9X" role="doc-biblioref">651</a>)</span>.
CLIA has previously been used to develop tests that can be used at point of care (e.g., <span class="citation" data-cites="IAZeyZZW">(<a href="#ref-IAZeyZZW" role="doc-biblioref">644</a>)</span>) which may allow for this technique to become more widely accessible in the future.</p>
<h3 data-number="3.5.3" id="lateral-flow-immunoassay"><span class="header-section-number">3.5.3</span> Lateral Flow Immunoassay</h3>
<p>The first serological test approved for emergency use in the United States was developed by Cellex <span class="citation" data-cites="n6sKSpIk">(<a href="#ref-n6sKSpIk" role="doc-biblioref">652</a>)</span>.
The Cellex qSARS-CoV-2 IgG/IgM Rapid Test is a chromatographic immunoassay, also known as a lateral flow immunoassay, designed to qualitatively detect IgM and IgG antibodies against SARS-CoV-2 in the plasma of patients suspected to have developed a SARS-CoV-2 infection <span class="citation" data-cites="n6sKSpIk">(<a href="#ref-n6sKSpIk" role="doc-biblioref">652</a>)</span>.
The Cellex test cassette contains a pad of SARS-CoV-2 antigens and a nitrocellulose strip with lines for each of IgG and IgM, as well as a control (goat IgG) <span class="citation" data-cites="n6sKSpIk">(<a href="#ref-n6sKSpIk" role="doc-biblioref">652</a>)</span>.
In a specimen that contains antibodies against the SARS-CoV-2 antigen, the antibodies will bind to the strip and be captured by the IgM and/or IgG line(s), resulting in a change of color <span class="citation" data-cites="n6sKSpIk">(<a href="#ref-n6sKSpIk" role="doc-biblioref">652</a>)</span>.
With this particular assay, results can be read within 15 to 20 minutes <span class="citation" data-cites="n6sKSpIk">(<a href="#ref-n6sKSpIk" role="doc-biblioref">652</a>)</span>.
Lateral flow immunoassays are often available at point of care but can have very low sensitivity <span class="citation" data-cites="gwsGoZAk">(<a href="#ref-gwsGoZAk" role="doc-biblioref">649</a>)</span>.</p>
<h3 data-number="3.5.4" id="neutralizing-antibody-assays"><span class="header-section-number">3.5.4</span> Neutralizing Antibody Assays</h3>
<p>Neutralizing antibody assays play a functional role in understanding immunity that distinguishes them from other serological tests.
The tests described above are all binding antibody tests.
On the other hand, rather than simply binding an antibody to facilitate detection, neutralizing antibody assays determine whether an antibody response is present that would prevent infection <span class="citation" data-cites="vbwRvJPH 1AN4GuRw1">(<a href="#ref-vbwRvJPH" role="doc-biblioref">653</a>, <a href="#ref-1AN4GuRw1" role="doc-biblioref">654</a>)</span>.
Therefore, these tests serve the purpose of evaluating the extent to which a sample donor has acquired immunity that will reduce susceptibility to SARS-CoV-2.
As a result, neutralizing antibody assays have been used widely to characterize the duration of immunity following infection, to assess vaccine candidates, and to establish correlates of protection against infection and disease <span class="citation" data-cites="X4iMjHsT NeoPSRp5 H82x3B90">(<a href="#ref-X4iMjHsT" role="doc-biblioref">655</a>–<a href="#ref-H82x3B90" role="doc-biblioref">657</a>)</span>.
These tests are typically performed in a laboratory <span class="citation" data-cites="vbwRvJPH">(<a href="#ref-vbwRvJPH" role="doc-biblioref">653</a>)</span>, and in SARS-CoV-2, the results of neutralizing antibody assays are often correlated with the results of binding antibody tests <span class="citation" data-cites="vbwRvJPH">(<a href="#ref-vbwRvJPH" role="doc-biblioref">653</a>)</span>.</p>
<p>The gold standard for assessing the presence of neutralizing antibodies is the plaque reduction neutralization test (PRNT), but this approach does not scale well <span class="citation" data-cites="1AN4GuRw1">(<a href="#ref-1AN4GuRw1" role="doc-biblioref">654</a>)</span>.
An early high-throughput neutralizing antibody assay designed against SARS-CoV-2 used a fluorescently labeled reporter virus that was incubated with different dilutions of patient serum <span class="citation" data-cites="1AN4GuRw1">(<a href="#ref-1AN4GuRw1" role="doc-biblioref">654</a>)</span>.
The cells used for incubation would turn green if antibodies were not present.
Essentially, this assay evaluates whether the virus is able to infect the cell in the presence of the serum.
The specificity of this assay was 100%, and the correlation between the results of this assay and of PRNT was 0.85 with the results suggesting that the sensitivity of the high-throughput approach was higher than that of PRNT <span class="citation" data-cites="1AN4GuRw1">(<a href="#ref-1AN4GuRw1" role="doc-biblioref">654</a>)</span>.
While this approach was performed on a plate and using cells, other methods have been developed using methods such as bead arrays <span class="citation" data-cites="Xu7hrQyb">(<a href="#ref-Xu7hrQyb" role="doc-biblioref">658</a>)</span>.</p>
<h3 data-number="3.5.5" id="duration-of-immune-indicators"><span class="header-section-number">3.5.5</span> Duration of Immune Indicators</h3>
<p>While the adaptive immune system produces antibodies in response to SARS-CoV-2 viral challenge, these indicators of seroconversion are unlikely to remain in circulation permanently.
Previously, a two-year longitudinal study following convalesced SARS patients with a mean age of 29 found that IgG antibodies were detectable in all 56 patients surveyed for at least 16 months and remained detectable in all but 4 patients (11.8%) through the full two-year study period <span class="citation" data-cites="11UpvND0K">(<a href="#ref-11UpvND0K" role="doc-biblioref">659</a>)</span>.
These results suggest that immunity to SARS-CoV-1 is sustained for at least a year.
Circulating antibody titers to other coronaviruses have been reported to decline significantly after one year <span class="citation" data-cites="FIOysZLl">(<a href="#ref-FIOysZLl" role="doc-biblioref">660</a>)</span>.
Evidence to date suggests that sustained immunity to the SARS-CoV-2 virus remains for a shorter period of time but at least 6 to 8 months after infection <span class="citation" data-cites="4si096MH L5l2MrPj 1HNwTvE3S H7Hdjjfx">(<a href="#ref-4si096MH" role="doc-biblioref">661</a>–<a href="#ref-H7Hdjjfx" role="doc-biblioref">664</a>)</span>.
However, this does not mean that all serological evidence of infection dissipates, but rather that the immune response becomes insufficient to neutralize the virus.</p>
<p>In order to study the persistence of SARS-CoV-2 antibodies, one study assessed sustained immunity using 254 blood samples from 188 COVID-19 positive patients <span class="citation" data-cites="L5l2MrPj">(<a href="#ref-L5l2MrPj" role="doc-biblioref">662</a>)</span>.
The samples were collected at various time points between 6 and 240 days post-symptom onset; some patients were assessed longitudinally.
Of the samples, 43 were collected at least 6 months after symptom onset.
After one month, 98% of patients were seropositive for IgG to S.
Moreover, S IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, with 90% of subjects seropositive at 6 months.
Similarly, at 6 to 8 months 88% of patients were seropositive for RBD IgG, and 90% were seropositive for SARS-CoV-2 neutralizing antibodies.
Another study examined 119 samples from 88 donors who had recovered from mild to severe cases of COVID-19 <span class="citation" data-cites="H7Hdjjfx">(<a href="#ref-H7Hdjjfx" role="doc-biblioref">664</a>)</span>.
A relatively stable level of IgG and plasma neutralizing antibodies was identified up to 6 months post diagnosis.
Significantly lower but considerable levels of anti-SARS-CoV-2 IgG antibodies were still present in 80% of samples obtained 6 to 8 months post-symptom onset.</p>
<p>Titers of IgM and IgG antibodies against the RBD were found to decrease from 1.3 to 6.2 months post infection in a study of 87 individuals <span class="citation" data-cites="iP9vKRvv">(<a href="#ref-iP9vKRvv" role="doc-biblioref">665</a>)</span>.
However, the decline of IgA activity (15%) was less pronounced than that of IgM (53%) or IgG (32%).
It was noted that higher levels of anti-RBD IgG and anti-N total antibodies were detected in individuals that reported persistent post-acute symptoms at both study visits.
Moreover, plasma neutralizing activity decreased five-fold between 1.3 and 6.2 months in an assay of HIV-1 virus pseudotyped with SARS-CoV-2 S protein, and this neutralizing activity was directly correlated with IgG anti-RBD titers <span class="citation" data-cites="iP9vKRvv">(<a href="#ref-iP9vKRvv" role="doc-biblioref">665</a>)</span>.
These findings are in accordance with other studies that show that the majority of seroconverters have detectable, albeit decreasing, levels of neutralizing antibodies at least 3 to 6 months post infection <span class="citation" data-cites="19MYPpSCx m1wjdQSV NVgdaNGd">(<a href="#ref-19MYPpSCx" role="doc-biblioref">666</a>–<a href="#ref-NVgdaNGd" role="doc-biblioref">668</a>)</span>.</p>
<p>Determining the potency of anti-RBD antibodies early in the course of an infection may be important moving forward, as their neutralizing potency may be prognostic for disease severity and survival <span class="citation" data-cites="1CZlIiD0p">(<a href="#ref-1CZlIiD0p" role="doc-biblioref">669</a>)</span>.
The duration of immunity might also vary with age <span class="citation" data-cites="3LFrwvth">(<a href="#ref-3LFrwvth" role="doc-biblioref">670</a>)</span> or ABO blood type <span class="citation" data-cites="1AAMeFyRr">(<a href="#ref-1AAMeFyRr" role="doc-biblioref">671</a>)</span>.
Autopsies of lymph nodes and spleens from severe acute COVID-19 patients showed a loss of T follicular helper cells and germinal centers that may explain some of the impaired development of antibody responses <span class="citation" data-cites="JcpiBiQt">(<a href="#ref-JcpiBiQt" role="doc-biblioref">672</a>)</span>.
Therefore, serological testing may be time-limited in its ability to detect prior infection.</p>
<p>Other immune indicators of prior infection have also been evaluated to see how they persist over time.
SARS-CoV-2 memory CD8<sup>+</sup> T cells were slightly decreased (50%) 6 months post-symptom onset.
In this same subset of COVID-19 patients, 93% of subjects had detectable levels of SARS-CoV-2 memory CD4<sup>+</sup> T cells, of which 42% had more than 1% SARS-CoV-2-specific CD4<sup>+</sup> T cells.
At 6 months, 92% of patients were positive for SARS-CoV-2 memory CD4<sup>+</sup> T cells.
Indeed, the abundance of S-specific memory CD4<sup>+</sup> T cells over time was similar to that of SARS-CoV-2-specific CD4<sup>+</sup> T cells overall <span class="citation" data-cites="L5l2MrPj">(<a href="#ref-L5l2MrPj" role="doc-biblioref">662</a>)</span>.
T cell immunity to SARS-CoV-2 at 6 to 8 months following symptom onset has also been confirmed by other studies <span class="citation" data-cites="AKY95Yy3 19p3NXKI5 H7Hdjjfx">(<a href="#ref-H7Hdjjfx" role="doc-biblioref">664</a>, <a href="#ref-AKY95Yy3" role="doc-biblioref">673</a>, <a href="#ref-19p3NXKI5" role="doc-biblioref">674</a>)</span>.
In another study, T cell reactivity to SARS-CoV-2 epitopes was also detected in some individuals never been exposed to SARS-CoV-2.
This finding suggests the potential for cross-reactive T cell recognition between SARS-CoV-2 and pre-existing circulating HCoV that are responsible for the “common cold” <span class="citation" data-cites="CHRiQNCx">(<a href="#ref-CHRiQNCx" role="doc-biblioref">675</a>)</span>, but further research is required.
Therefore, whether T cells will provide a more stable measure through which to assess prior infection remains unknown.
Notably, commercial entities have tried to develop tests specifically for T cells, some of which have been authorized by the United States Food and Drug Administration <span class="citation" data-cites="10ICz5Jou uKR3FVp8">(<a href="#ref-10ICz5Jou" role="doc-biblioref">676</a>, <a href="#ref-uKR3FVp8" role="doc-biblioref">677</a>)</span> to identify people with adaptive T cell immune responses to SARS-CoV-2, either from a previous or ongoing infection.</p>
<h3 data-number="3.5.6" id="applications-of-serological-tests"><span class="header-section-number">3.5.6</span> Applications of Serological Tests</h3>
<p>In addition to the limitations posed by the fact that antibodies are not permanent indicators of prior infection, serological immunoassays carry a number of limitations that influence their utility in different situations.
Importantly, false positives can occur due to cross-reactivity with other antibodies according to the clinical condition of the patient <span class="citation" data-cites="n6sKSpIk">(<a href="#ref-n6sKSpIk" role="doc-biblioref">652</a>)</span>.
Due to the long incubation times and delayed immune responses of infected patients, serological immunoassays are insufficiently sensitive for a diagnosis in the early stages of an infection.
Therefore, such tests must be used in combination with RNA detection tests if intended for diagnostic purposes <span class="citation" data-cites="NXf7l0z3">(<a href="#ref-NXf7l0z3" role="doc-biblioref">678</a>)</span>.
False positives are particularly harmful if they are erroneously interpreted to mean that a population is more likely to have acquired immunity to a disease <span class="citation" data-cites="Y35qBYLJ">(<a href="#ref-Y35qBYLJ" role="doc-biblioref">679</a>)</span>.
Similarly, while serological tests may be of interest to individuals who wish to confirm they were infected with SARS-CoV-2 in the past, their potential for false positives means that they are not currently recommended for this use.
However, in the wake of vaccines becoming widely available, accurate serological tests that could be administered at point of care were investigated in the hope that they could help to prioritize vaccine recipients <span class="citation" data-cites="7zusLohQ">(<a href="#ref-7zusLohQ" role="doc-biblioref">680</a>)</span>.
Another concern with serological testing is the potential for viral evolution to reduce the sensitivity of assays, especially for neutralizing antibody assays.
Chen et al. performed a systematic re-analysis of published data examining the neutralizing effect of serum from vaccinated or recovered individuals on four VOC <span class="citation" data-cites="7bzRqtiu">(<a href="#ref-7bzRqtiu" role="doc-biblioref">681</a>)</span>.
They found reduced neutralizing titers against these variants relative to the lineages used for reference.
These findings suggest that such techniques will need to be modified over time as SARS-CoV-2 evolves.</p>
<p>These limitations make serological tests far less useful for diagnostics and for test-and-trace strategies; however, serological testing is valuable for public health monitoring at the population level.
Serosurveys provide a high-level perspective of the prevalence of a disease and can provide insight into the susceptibility of a population as well as variation in severity, e.g., between geographic regions <span class="citation" data-cites="Y35qBYLJ">(<a href="#ref-Y35qBYLJ" role="doc-biblioref">679</a>)</span>.
From a public health perspective, they can also provide insight into the effectiveness of mitigation efforts and to gain insight into risk factors influencing susceptibility <span class="citation" data-cites="16uL34FEP">(<a href="#ref-16uL34FEP" role="doc-biblioref">682</a>)</span>.
EIA methods are high-throughput <span class="citation" data-cites="A3QBAHLv 1Gw1giZBH">(<a href="#ref-A3QBAHLv" role="doc-biblioref">683</a>, <a href="#ref-1Gw1giZBH" role="doc-biblioref">684</a>)</span>, and, as with molecular tests, additional efforts have been made to scale up the throughput of serological tests <span class="citation" data-cites="WgmwXO4P">(<a href="#ref-WgmwXO4P" role="doc-biblioref">685</a>)</span>.
Therefore, serological tests can be useful to developing strategies for the management of viral spread.</p>
<p>Early in the course of the pandemic, it was also hoped that serological tests would provide information relevant to advancing economic recovery.
Some infectious agents can be controlled through “herd immunity”, which is when a critical mass within the population acquires immunity through vaccination and/or infection, preventing an infectious agent from spreading widely.
It was hoped that people who had recovered and developed antibodies might be able to return to work <span class="citation" data-cites="EUrJt06r P9k7RTw6">(<a href="#ref-EUrJt06r" role="doc-biblioref">686</a>, <a href="#ref-P9k7RTw6" role="doc-biblioref">687</a>)</span>.
This strategy would have relied on recovered individuals acquiring long-term immunity, which has not been borne out <span class="citation" data-cites="TDWfc9A9">(<a href="#ref-TDWfc9A9" role="doc-biblioref">688</a>)</span>.
Additionally, it was hoped that identifying seroconverters and specifically those who had mounted a strong immune response would reveal strong candidates for convalescent plasma donation <span class="citation" data-cites="rvAb0ZJy">(<a href="#ref-rvAb0ZJy" role="doc-biblioref">641</a>)</span>; however, convalescent plasma has not been found to offer therapeutic benefit (reviewed in <span class="citation" data-cites="njpLhBui">(<a href="#ref-njpLhBui" role="doc-biblioref">3</a>)</span>).
While these hopes have not been realized, serological tests have been useful for gaining a better understanding of the pandemic <span class="citation" data-cites="16uL34FEP">(<a href="#ref-16uL34FEP" role="doc-biblioref">682</a>)</span>.</p>
<h2 data-number="3.6" id="possible-alternatives-to-current-diagnostic-practices"><span class="header-section-number">3.6</span> Possible Alternatives to Current Diagnostic Practices</h2>
<p>One possible alternative or complement to molecular and serological testing would be diagnosing COVID-19 cases based on symptomatology.
COVID-19 can present with symptoms similar to other types of pneumonia, and symptoms can vary widely among COVID-19 patients; therefore, clinical presentation is often insufficient as a sole diagnostic criterion.
In addition, identifying and isolating mild or asymptomatic cases is critical to efforts to manage outbreaks.
Even among mildly symptomatic patients, a predictive model based on clinical symptoms had a sensitivity of only 56% and a specificity of 91% <span class="citation" data-cites="91dto3z3">(<a href="#ref-91dto3z3" role="doc-biblioref">689</a>)</span>.
More problematic is that clinical symptom-based tests are only able to identify already symptomatic cases, not presymptomatic or asymptomatic cases.
They may still be important for clinical practice and for reducing tests needed for patients deemed unlikely to have COVID-19.</p>
<p>In some cases, clinical signs may also provide information that can inform diagnosis.
Using computed tomography of the chest in addition to RT-qPCR testing was found to provide a higher sensitivity than either measure alone <span class="citation" data-cites="xn686bWT">(<a href="#ref-xn686bWT" role="doc-biblioref">690</a>)</span>.
X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies <span class="citation" data-cites="itCGVb5b">(<a href="#ref-itCGVb5b" role="doc-biblioref">691</a>)</span>.
Other studies have shown that specificity varies between radiologists <span class="citation" data-cites="tkuUFNIX">(<a href="#ref-tkuUFNIX" role="doc-biblioref">692</a>)</span>, though the sensitivity reported here was lower than that published in the previous paper.
While preliminary machine-learning results suggested that chest X-rays might provide high sensitivity and specificity and potentially facilitate the detection of asymptomatic and presymptomatic infections (e.g., <span class="citation" data-cites="nN9zfzPe">(<a href="#ref-nN9zfzPe" role="doc-biblioref">693</a>)</span>), further investigation suggested that such approaches are prone to bias and are unlikely to be clinically useful <span class="citation" data-cites="6kbMsiPz">(<a href="#ref-6kbMsiPz" role="doc-biblioref">694</a>)</span>.
Given the above, the widespread use of X-ray tests on otherwise healthy adults is likely inadvisable.</p>
<p>Finally, in addition to genomic and serological surveillance, other types of monitoring have proven useful in managing the pandemic <span class="citation" data-cites="jsXHJtkB">(<a href="#ref-jsXHJtkB" role="doc-biblioref">695</a>)</span>.
One that has received significant attention is wastewater surveillance.
This approach can use several of the technologies described for molecular testing, such as qPCR and dPCR, as well as <em>in vitro</em> culturing <span class="citation" data-cites="EH5exCzT">(<a href="#ref-EH5exCzT" role="doc-biblioref">696</a>)</span> and can provide insight into trends in the prevalence of SARS-CoV-2 regionally.</p>
<h2 data-number="3.7" id="strategies-and-considerations-for-testing"><span class="header-section-number">3.7</span> Strategies and Considerations for Testing</h2>
<p>Deciding whom to test, when to test, and which test to use have proven challenging as the COVID-19 pandemic has unfolded.
Early in the COVID-19 pandemic, testing was typically limited to individuals considered high risk for developing serious illness <span class="citation" data-cites="OB8BJiwj">(<a href="#ref-OB8BJiwj" role="doc-biblioref">697</a>)</span>.
This approach often limited testing to people with severe symptoms and people showing mild symptoms that had been in contact with a person who had tested positive.
Individuals who were asymptomatic (i.e., potential spreaders) and individuals who were able to recover at home were thus often unaware of their status.
Therefore, this method of testing administration misses a high proportion of infections and does not allow for test-and-trace methods to be used.
For instance, a study from Imperial College estimates that in Italy, the true number of infections was around 5.9 million in a total population of ~60 million, compared to the 70,000 detected as of March 28, 2020 <span class="citation" data-cites="Frya4XA4">(<a href="#ref-Frya4XA4" role="doc-biblioref">314</a>)</span>.
Another analysis, which examined New York state, indicated that as of May 2020, approximately 300,000 cases had been reported in a total population of approximately 20 million <span class="citation" data-cites="IldxlHwA">(<a href="#ref-IldxlHwA" role="doc-biblioref">698</a>)</span>.
This corresponded to ~1.5% of the population, but ~12% of individuals sampled statewide were estimated as positive through antibody tests (along with indications of spatial heterogeneity at higher resolution) <span class="citation" data-cites="IldxlHwA">(<a href="#ref-IldxlHwA" role="doc-biblioref">698</a>)</span>.
Technological advancements that facilitate widespread, rapid testing would therefore be valuable for accurately assessing the rate of infection and aid in controlling the virus’ spread.
Additionally, the trade off of accessibility, sensitivity, and time to results has raised some complex questions around which tests are best suited to certain situations.
Immunoassays, including serological tests, have much higher limits of detection than PCR tests do <span class="citation" data-cites="kpLjx1Kr">(<a href="#ref-kpLjx1Kr" role="doc-biblioref">699</a>)</span>.</p>
<p>Changes in public attitudes and the lifting of COVID-19 restrictions due to the multifactorial desire to stimulate economic activities has required a shift of testing paradigms in 2022, despite warnings from public health officials against a hard exit from public health restrictions <span class="citation" data-cites="s2XX4fx4 W79HEsxE">(<a href="#ref-s2XX4fx4" role="doc-biblioref">700</a>, <a href="#ref-W79HEsxE" role="doc-biblioref">701</a>)</span>.
An important strategy for testing moving forward is to determine when someone becomes infectious or is no longer infectious following a positive test for COVID-19.
Generally, patient specimens tend to not contain culturable virus past day 5 of symptom onset <span class="citation" data-cites="qZT0UMJi VgiCmD2R">(<a href="#ref-qZT0UMJi" role="doc-biblioref">702</a>, <a href="#ref-VgiCmD2R" role="doc-biblioref">703</a>)</span>.
However, due to their sensitivity to post-infectious viral RNA in specimens, PCR-based methods may mislead individuals to believe that they are still infectious several days after symptom onset <span class="citation" data-cites="NXf7l0z3">(<a href="#ref-NXf7l0z3" role="doc-biblioref">678</a>)</span>.
Furthermore, detection of viral RNA can occur days and weeks after an active infection due to the sensitivity of PCR-based methods <span class="citation" data-cites="8oW626i5 jWdI1NWR FMUORrT">(<a href="#ref-jWdI1NWR" role="doc-biblioref">568</a>, <a href="#ref-8oW626i5" role="doc-biblioref">704</a>, <a href="#ref-FMUORrT" role="doc-biblioref">705</a>)</span>.</p>
<p>In contrast, LFTs were thought to have poor sensitivity and their value for identifying infections and managing the pandemic was questioned <span class="citation" data-cites="Jwhgcpu8 5KUr4fYD">(<a href="#ref-Jwhgcpu8" role="doc-biblioref">706</a>, <a href="#ref-5KUr4fYD" role="doc-biblioref">707</a>)</span>.
However, LFTs do reliably detect SARS-CoV-2 proteins when there is a high viral load, which appears to correlate with a person’s infectiousness <span class="citation" data-cites="DBDX1KLX Ki0q4jhw">(<a href="#ref-Ki0q4jhw" role="doc-biblioref">616</a>, <a href="#ref-DBDX1KLX" role="doc-biblioref">708</a>)</span>.
Therefore, LFTs are an important diagnostic tool to determine infectiousness with fast turnaround times, ease of use, and accessibility by the general public <span class="citation" data-cites="UwLT34Vv NXf7l0z3">(<a href="#ref-NXf7l0z3" role="doc-biblioref">678</a>, <a href="#ref-UwLT34Vv" role="doc-biblioref">709</a>)</span>.
One study has suggested that the test sensitivity of LFTs appears to be less important than accessibility to LFTs, frequent testing, and fast reporting times for reducing the impact of viral spread <span class="citation" data-cites="Pk9AqhFn">(<a href="#ref-Pk9AqhFn" role="doc-biblioref">710</a>)</span>.
While PCR-based methods are important for COVID-19 surveillance, their use is labor intensive and time consuming, and laboratories are often slow to report results, rendering such methods limited in their surveillance capacity <span class="citation" data-cites="NXf7l0z3">(<a href="#ref-NXf7l0z3" role="doc-biblioref">678</a>)</span>.</p>
<p>These limitations are demonstrated by the estimated 10-fold under-reporting of cases in the United States in 2020 due to shortages in testing and slow rollout of testing and slow reporting of results <span class="citation" data-cites="u1jzzcCB">(<a href="#ref-u1jzzcCB" role="doc-biblioref">711</a>)</span>.
However, one strategy that may balance the strengths and weaknesses of both types of tests is to corroborate a positive LFT result using a PCR-based method.
Indeed, in December 2021 sufficient surveillance and reduction of COVID-19 spread using this joint LFT-PCR strategy was demonstrated in Liverpool, U.K., where there was an estimated 21% reduction of cases <span class="citation" data-cites="HoWB48LM UwLT34Vv">(<a href="#ref-UwLT34Vv" role="doc-biblioref">709</a>, <a href="#ref-HoWB48LM" role="doc-biblioref">712</a>)</span>.</p>
<h2 data-number="3.8" id="what-lies-ahead"><span class="header-section-number">3.8</span> What Lies Ahead</h2>
<p>Diagnostic tools have played an important role during the COVID-19 pandemic.
Different tests offer different advantages (Figure <a href="#fig:diag">2</a>).
Specifically, the results of SARS-CoV-2 diagnostic tests (typically qPCR or LFT-based tests) have been used to estimate the number of infections in the general population, thus informing public health strategies around the globe <span class="citation" data-cites="SeNutdHD">(<a href="#ref-SeNutdHD" role="doc-biblioref">574</a>)</span>.
During the surges caused by the different SARS-CoV-2 variants between 2020 and 2021, government-sponsored efforts to conduct mass testing and to provide free diagnostic tests to the population were a common occurrence in many parts of the world <span class="citation" data-cites="6cEtIAuH 12KT1j4NG 14fDqEnne">(<a href="#ref-6cEtIAuH" role="doc-biblioref">713</a>–<a href="#ref-14fDqEnne" role="doc-biblioref">715</a>)</span>.
However, recent reports indicate that such public health policies are starting to change during 2022.
For example, it is known that the UK plans to dismantle its COVID-19 testing program and scale back its daily reporting requirements <span class="citation" data-cites="yfpylewZ 5ag9qde4">(<a href="#ref-yfpylewZ" role="doc-biblioref">716</a>, <a href="#ref-5ag9qde4" role="doc-biblioref">717</a>)</span>.
A similar approach can be seen in the US as well, where multiple state-run testing facilities are closing, despite some groups advocating to keep them open <span class="citation" data-cites="NIpc4LRf p0e1mRYk">(<a href="#ref-NIpc4LRf" role="doc-biblioref">718</a>, <a href="#ref-p0e1mRYk" role="doc-biblioref">719</a>)</span>.
These ongoing changes in testing policy are likely to have a direct effect on how the pandemic is managed moving forward.
SARS-CoV-2 diagnostic tests can be used effectively to slow the spread of the disease only when 1) they are used to share testing results in a timely manner so that they can reasonably be used to approximate the number of infections in the population and 2) those tests are easily accessible by the general public.</p>
<div id="fig:diag" class="fignos">
<figure>
<img src="images/diagnostics.png" alt="Figure 2: Summary of Diagnostic Technologies used in COVID-19 Testing. The immune response to SARS-CoV-2 means that different diagnostic approaches offer different views of COVID-19. Early in the infection course, viral load is high. This means that PCR-based testing and EIA testing for antigens are likely to return positives (as indicated by the green bars at the bottom). As viral load decreases, EIA antigen tests become negative, but PCR-based tests can still detect even very low viral loads. From a serological perspective, IgM peaks in the first few weeks following infection and then decreases, while IgG peaks much later in the infection course. Therefore, serological tests are likely to return positives in first few months following the acute infection course. Additional detail is available above and in several analyses and reviews (1, 678, 705, 720)." />
<figcaption aria-hidden="true"><span>Figure 2:</span> <strong>Summary of Diagnostic Technologies used in COVID-19 Testing.</strong>
The immune response to SARS-CoV-2 means that different diagnostic approaches offer different views of COVID-19.
Early in the infection course, viral load is high.
This means that PCR-based testing and EIA testing for antigens are likely to return positives (as indicated by the green bars at the bottom).
As viral load decreases, EIA antigen tests become negative, but PCR-based tests can still detect even very low viral loads.
From a serological perspective, IgM peaks in the first few weeks following infection and then decreases, while IgG peaks much later in the infection course.
Therefore, serological tests are likely to return positives in first few months following the acute infection course.
Additional detail is available above and in several analyses and reviews <span class="citation" data-cites="NXf7l0z3 1G2DPuXl7 FMUORrT GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>, <a href="#ref-NXf7l0z3" role="doc-biblioref">678</a>, <a href="#ref-FMUORrT" role="doc-biblioref">705</a>, <a href="#ref-1G2DPuXl7" role="doc-biblioref">720</a>)</span>.</figcaption>
</figure>
</div>
<p>Children are one segment of the population where the importance of the two aforementioned conditions can be exemplified.
This group is particularly vulnerable as there are ongoing challenges with testing in schools, increased COVID-19 mortality rates, and COVID-19-associated orphanhood.
In this regard, although there is evidence of the efficacy of routine diagnostic testing to reduce the probability of having infectious students <span class="citation" data-cites="1BuS1KbQB yoOScnNH">(<a href="#ref-1BuS1KbQB" role="doc-biblioref">721</a>, <a href="#ref-yoOScnNH" role="doc-biblioref">722</a>)</span>, as of March of 2022 there is an increasing number of schools that have stopped or plan to stop contact tracing efforts <span class="citation" data-cites="r6gl31QZ GN68N2pq">(<a href="#ref-r6gl31QZ" role="doc-biblioref">723</a>, <a href="#ref-GN68N2pq" role="doc-biblioref">724</a>)</span>, in line with an announcement made by the CDC where it no longer recommended contact tracing as a strategy to contain the virus <span class="citation" data-cites="1CGvdSkU0">(<a href="#ref-1CGvdSkU0" role="doc-biblioref">725</a>)</span>.
An estimated 197 children have died in the US from COVID-19 during the first three months of 2022 <span class="citation" data-cites="1G4q8eaJV">(<a href="#ref-1G4q8eaJV" role="doc-biblioref">726</a>)</span>, compared to 735 deaths in the preceding 20 months of the pandemic <span class="citation" data-cites="TSx8mpKC">(<a href="#ref-TSx8mpKC" role="doc-biblioref">727</a>)</span>, and millions of children have been orphaned as a consequence of parent or caregiver death due to COVID-19 <span class="citation" data-cites="WbYvi82">(<a href="#ref-WbYvi82" role="doc-biblioref">728</a>)</span>.
It is likely that reducing or eliminating testing capacity in schools will directly exacerbate these negative outcomes for the remainder of 2022.</p>
<p>The SARS-CoV-2 diagnostic tools presented in this paper are far less useful if they are difficult to obtain, or if their limited use results in biased data that would lead to ill-informed public health strategies.
Under conditions of limited supply, different strategies for testing are needed <span class="citation" data-cites="NrqbGGrB">(<a href="#ref-NrqbGGrB" role="doc-biblioref">729</a>)</span>.
The pandemic is still an ongoing public health threat and it is worrying that active testing and tracing efforts are a low priority for public health authorities in many countries.
If this trend continues, the lack of testing could result in increased morbidity and mortality and an overall failure to manage the pandemic.</p>
<h1 data-number="4" id="identification-and-development-of-therapeutics-for-covid-19"><span class="header-section-number">4</span> Identification and Development of Therapeutics for COVID-19</h1>
<h2 data-number="4.1" id="abstract-3"><span class="header-section-number">4.1</span> Abstract</h2>
<p>After emerging in China in late 2019, the novel coronavirus SARS-CoV-2 spread worldwide and as of mid-2021 remains a significant threat globally.
Only a few coronaviruses are known to infect humans, and only two cause infections similar in severity to SARS-CoV-2: <em>Severe acute respiratory syndrome-related coronavirus</em>, a closely related species of SARS-CoV-2 that emerged in 2002, and <em>Middle East respiratory syndrome-related coronavirus</em>, which emerged in 2012.
Unlike the current pandemic, previous epidemics were controlled rapidly through public health measures, but the body of research investigating severe acute respiratory syndrome and Middle East respiratory syndrome has proven valuable for identifying approaches to treating and preventing novel coronavirus disease 2019 (COVID-19).
Building on this research, the medical and scientific communities have responded rapidly to the COVID-19 crisis to identify many candidate therapeutics.
The approaches used to identify candidates fall into four main categories: adaptation of clinical approaches to diseases with related pathologies, adaptation based on virological properties, adaptation based on host response, and data-driven identification of candidates based on physical properties or on pharmacological compendia.
To date, a small number of therapeutics have already been authorized by regulatory agencies such as the Food and Drug Administration (FDA), while most remain under investigation.
The scale of the COVID-19 crisis offers a rare opportunity to collect data on the effects of candidate therapeutics.
This information provides insight not only into the management of coronavirus diseases, but also into the relative success of different approaches to identifying candidate therapeutics against an emerging disease.</p>
<h2 data-number="4.2" id="importance-3"><span class="header-section-number">4.2</span> Importance</h2>
<p>The COVID-19 pandemic is a rapidly evolving crisis.
With the worldwide scientific community shifting focus onto the SARS-CoV-2 virus and COVID-19, a large number of possible pharmaceutical approaches for treatment and prevention have been proposed.
What was known about each of these potential interventions evolved rapidly throughout 2020 and 2021.
This fast-paced area of research provides important insight into how the ongoing pandemic can be managed and also demonstrates the power of interdisciplinary collaboration to rapidly understand a virus and match its characteristics with existing or novel pharmaceuticals.
As illustrated by the continued threat of viral epidemics during the current millennium, a rapid and strategic response to emerging viral threats can save lives.
In this review, we explore how different modes of identifying candidate therapeutics have borne out during COVID-19.</p>
<h2 data-number="4.3" id="introduction-3"><span class="header-section-number">4.3</span> Introduction</h2>
<p>The novel coronavirus <em>Severe acute respiratory syndrome-related coronavirus 2</em> (SARS-CoV-2) emerged in late 2019 and quickly precipitated the worldwide spread of novel coronavirus disease 2019 (COVID-19).
COVID-19 is associated with symptoms ranging from mild or even asymptomatic to severe, and up to 2% of patients diagnosed with COVID-19 die from COVID-19-related complications such as acute respiratory disease syndrome (ARDS) <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span>.
As a result, public health efforts have been critical to mitigating the spread of the virus.
However, as of mid-2021, COVID-19 remains a significant worldwide concern (Figure <a href="#fig:csse-deaths">3</a>), with 2021 cases in some regions surging far above the numbers reported during the initial outbreak in early 2020.
While a number of vaccines have been developed and approved in different countries starting in late 2020 <span class="citation" data-cites="1HWmcSB20">(<a href="#ref-1HWmcSB20" role="doc-biblioref">6</a>)</span>, vaccination efforts have not proceeded at the same pace throughout the world and are not yet close to ending the pandemic.</p>
<p>Due to the continued threat of the virus and the severity of the disease, the identification and development of therapeutic interventions have emerged as significant international priorities.
Prior developments during other recent outbreaks of emerging diseases, especially those caused by human coronaviruses (HCoV), have guided biomedical research into the behavior and treatment of this novel coronavirus infection.
However, previous emerging HCoV-related disease threats were controlled much more quickly than SARS-CoV-2 through public health efforts (Figure <a href="#fig:csse-deaths">3</a>).
The scale of the COVID-19 pandemic has made the repurposing and development of pharmaceuticals more urgent than in previous coronavirus epidemics.</p>
<h3 data-number="4.3.1" id="lessons-from-prior-hcov-outbreaks"><span class="header-section-number">4.3.1</span> Lessons from Prior HCoV Outbreaks</h3>
<div id="fig:csse-deaths" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/dfc6ff85488d57cbb52f4b2324531be6afaa73f4/csse/csse-deaths.png" alt="Figure 3: Cumulative global incidence of COVID-19 and SARS. As of December 18, 2022, 653,207,634 COVID-19 cases and 6,665,906 COVID-19 deaths had been reported worldwide since January 22, 2020. A total of 8,432 cases and 813 deaths were reported for SARS from March 17 to July 11, 2003. SARS-CoV-1 was officially contained on July 5, 2003, within 9 months of its appearance (730). In contrast, SARS-CoV-2 remains a significant global threat nearly two years after its emergence. COVID-19 data are from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University (731, 732). SARS data are from the WHO (733) and were obtained from a dataset on GitHub (734). See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily." />
<figcaption aria-hidden="true"><span>Figure 3:</span> <strong>Cumulative global incidence of COVID-19 and SARS.</strong>
As of December 18, 2022, 653,207,634 COVID-19 cases and 6,665,906 COVID-19 deaths had been reported worldwide since January 22, 2020.
A total of 8,432 cases and 813 deaths were reported for SARS from March 17 to July 11, 2003.
SARS-CoV-1 was officially contained on July 5, 2003, within 9 months of its appearance <span class="citation" data-cites="bOegE5mX">(<a href="#ref-bOegE5mX" role="doc-biblioref">730</a>)</span>.
In contrast, SARS-CoV-2 remains a significant global threat nearly two years after its emergence.
COVID-19 data are from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University <span class="citation" data-cites="MrwDDw9R U1nCE851">(<a href="#ref-MrwDDw9R" role="doc-biblioref">731</a>, <a href="#ref-U1nCE851" role="doc-biblioref">732</a>)</span>.
SARS data are from the WHO <span class="citation" data-cites="8gZjmHY2">(<a href="#ref-8gZjmHY2" role="doc-biblioref">733</a>)</span> and were obtained from a dataset on GitHub <span class="citation" data-cites="1FrCknYoc">(<a href="#ref-1FrCknYoc" role="doc-biblioref">734</a>)</span>.
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.</figcaption>
</figure>
</div>
<p>At first, SARS-CoV-2’s rapid shift from an unknown virus to a significant worldwide threat closely paralleled the emergence of <em>Severe acute respiratory syndrome-related coronavirus</em> (SARS-CoV-1), which was responsible for the 2002-03 SARS epidemic.
The first documented case of COVID-19 was reported in Wuhan, China in November 2019, and the disease quickly spread worldwide in the early months of 2020.
In comparison, the first case of SARS was reported in November 2002 in the Guangdong Province of China, and it spread within China and then into several countries across continents during the first half of 2003 <span class="citation" data-cites="sP4wQEiM G5NJrE75 bOegE5mX">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>, <a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>, <a href="#ref-bOegE5mX" role="doc-biblioref">730</a>)</span>.
In fact, genome sequencing quickly revealed the virus causing COVID-19 to be a novel betacoronavirus closely related to SARS-CoV-1 <span class="citation" data-cites="Bp847Lfa">(<a href="#ref-Bp847Lfa" role="doc-biblioref">11</a>)</span>.</p>
<p>While similarities between these two viruses are unsurprising given their close phylogenetic relationship, there are also some differences in how the viruses affect humans.
SARS-CoV-1 infection is severe, with an estimated case fatality rate (CFR) for SARS of 9.5% <span class="citation" data-cites="sP4wQEiM">(<a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>, while estimates of the CFR associated with COVID-19 are much lower, at up to 2% <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span>.
SARS-CoV-1 is highly contagious and spread primarily by droplet transmission, with a basic reproduction number (R<sub>0</sub>) of 4 (i.e., each person infected was estimated to infect four other people) <span class="citation" data-cites="sP4wQEiM">(<a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>.
There is still some controversy whether SARS-CoV-2 is primarily spread by droplets or is primarily airborne <span class="citation" data-cites="82XnTbtX cw5j7x80 sUAAY1me ncMFeSeO">(<a href="#ref-82XnTbtX" role="doc-biblioref">258</a>, <a href="#ref-cw5j7x80" role="doc-biblioref">259</a>, <a href="#ref-sUAAY1me" role="doc-biblioref">262</a>, <a href="#ref-ncMFeSeO" role="doc-biblioref">735</a>)</span>.
Most estimates of its R<sub>0</sub> fall between 2.5 and 3 <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span>.
Therefore, SARS is thought to be a deadlier and more transmissible disease than COVID-19.</p>
<p>With the 17-year difference between these two outbreaks, there were major differences in the tools available to efforts to organize international responses.
At the time that SARS-CoV-1 emerged, no new HCoV had been identified in almost 40 years <span class="citation" data-cites="G5NJrE75">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>)</span>.
The identity of the virus underlying the SARS disease remained unknown until April of 2003, when the SARS-CoV-1 virus was characterized through a worldwide scientific effort spearheaded by the World Health Organization (WHO) <span class="citation" data-cites="G5NJrE75">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>)</span>.
In contrast, the SARS-CoV-2 genomic sequence was released on January 3, 2020 <span class="citation" data-cites="Bp847Lfa">(<a href="#ref-Bp847Lfa" role="doc-biblioref">11</a>)</span>, only days after the international community became aware of the novel pneumonia-like illness now known as COVID-19.
While SARS-CoV-1 belonged to a distinct lineage from the two other HCoVs known at the time of its discovery <span class="citation" data-cites="sP4wQEiM">(<a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>, SARS-CoV-2 is closely related to SARS-CoV-1 and is a more distant relative of another HCoV characterized in 2012, <em>Middle East respiratory syndrome-related coronavirus</em> <span class="citation" data-cites="1Fqilxaum 7Ee6Sz9l">(<a href="#ref-1Fqilxaum" role="doc-biblioref">19</a>, <a href="#ref-7Ee6Sz9l" role="doc-biblioref">736</a>)</span>.
Significant efforts had been dedicated towards understanding SARS-CoV-1 and MERS-CoV and how they interact with human hosts.
Therefore, SARS-CoV-2 emerged under very different circumstances than SARS-CoV-1 in terms of scientific knowledge about HCoVs and the tools available to characterize them.</p>
<p>Despite the apparent advantages for responding to SARS-CoV-2 infections, COVID-19 has caused many orders of magnitude more deaths than SARS did (Figure <a href="#fig:csse-deaths">3</a>).
The SARS outbreak was officially determined to be under control in July 2003, with the success credited to infection management practices such as mask wearing <span class="citation" data-cites="G5NJrE75">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>)</span>.
<em>Middle East respiratory syndrome-related coronavirus</em> (MERS-CoV) is still circulating and remains a concern; although the fatality rate is very high at almost 35%, the disease is much less easily transmitted, as its R<sub>0</sub> has been estimated to be 1 <span class="citation" data-cites="sP4wQEiM">(<a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>.
The low R<sub>0</sub> in combination with public health practices allowed for its spread to be contained <span class="citation" data-cites="sP4wQEiM">(<a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>.
Neither of these trajectories are comparable to that of SARS-CoV-2, which remains a serious threat worldwide over a year and a half after the first cases of COVID-19 emerged (Figure <a href="#fig:csse-deaths">3</a>).</p>
<h3 data-number="4.3.2" id="potential-approaches-to-the-treatment-of-covid-19"><span class="header-section-number">4.3.2</span> Potential Approaches to the Treatment of COVID-19</h3>
<p>Therapeutic interventions can utilize two approaches: they can either mitigate the effects of an infection that harms an infected person, or they can hinder the spread of infection within a host by disrupting the viral life cycle.
The goal of the former strategy is to reduce the severity and risks of an active infection, while for the latter, it is to inhibit the replication of a virus once an individual is infected, potentially freezing disease progression.
Additionally, two major approaches can be used to identify interventions that might be relevant to managing an emerging disease or a novel virus: drug repurposing and drug development.
Drug repurposing involves identifying an existing compound that may provide benefits in the context of interest <span class="citation" data-cites="6IJxPIQd">(<a href="#ref-6IJxPIQd" role="doc-biblioref">737</a>)</span>.
This strategy can focus on either approved or investigational drugs, for which there may be applicable preclinical or safety information <span class="citation" data-cites="6IJxPIQd">(<a href="#ref-6IJxPIQd" role="doc-biblioref">737</a>)</span>.
Drug development, on the other hand, provides an opportunity to identify or develop a compound specifically relevant to a particular need, but it is often a lengthy and expensive process characterized by repeated failure <span class="citation" data-cites="1BBiGYn1K">(<a href="#ref-1BBiGYn1K" role="doc-biblioref">738</a>)</span>.
Drug repurposing therefore tends to be emphasized in a situation like the COVID-19 pandemic due to the potential for a more rapid response.</p>
<p>Even from the early months of the pandemic, studies began releasing results from analyses of approved and investigational drugs in the context of COVID-19.
The rapid timescale of this response meant that, initially, most evidence came from observational studies, which compare groups of patients who did and did not receive a treatment to determine whether it may have had an effect.
This type of study can be conducted rapidly but is subject to confounding.
In contrast, randomized controlled trials (RCTs) are the gold-standard method for assessing the effects of an intervention.
Here, patients are prospectively and randomly assigned to treatment or control conditions, allowing for much stronger interpretations to be drawn; however, data from these trials take much longer to collect.
Both approaches have proven to be important sources of information in the development of a rapid response to the COVID-19 crisis, but as the pandemic draws on and more results become available from RCTs, more definitive answers are becoming available about proposed therapeutics.
Interventional clinical trials are currently investigating or have investigated a large number of possible therapeutics and combinations of therapeutics for the treatment of COVID-19 (Figure <a href="#fig:ebm-trials">4</a>).</p>
<div id="fig:ebm-trials" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/dfc6ff85488d57cbb52f4b2324531be6afaa73f4/ebmdatalab/ebmdatalab-trials.png" alt="Figure 4: COVID-19 clinical trials. Trials data are from the University of Oxford Evidence-Based Medicine Data Lab’s COVID-19 TrialsTracker (739). As of December 31, 2020, there were 6,987 COVID-19 clinical trials of which 3,962 were interventional. The study types include only types used in at least five trials. Only interventional trials are analyzed in the figures depicting status, phase, and intervention. Of the interventional trials, 98 trials had reported results as of December 31, 2020. Recruitment status and trial phase are shown only for interventional trials in which the status or phase is recorded. Common interventions refers to interventions used in at least ten trials. Combinations of interventions, such as hydroxychloroquine with azithromycin, are tallied separately from the individual interventions. See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily." />
<figcaption aria-hidden="true"><span>Figure 4:</span> <strong>COVID-19 clinical trials.</strong>
Trials data are from the University of Oxford Evidence-Based Medicine Data Lab’s COVID-19 TrialsTracker <span class="citation" data-cites="SSbnPnzT">(<a href="#ref-SSbnPnzT" role="doc-biblioref">739</a>)</span>.
As of December 31, 2020, there were 6,987 COVID-19 clinical trials of which 3,962 were interventional.
The study types include only types used in at least five trials.
Only interventional trials are analyzed in the figures depicting status, phase, and intervention.
Of the interventional trials, 98 trials had reported results as of December 31, 2020.
Recruitment status and trial phase are shown only for interventional trials in which the status or phase is recorded.
Common interventions refers to interventions used in at least ten trials.
Combinations of interventions, such as hydroxychloroquine with azithromycin, are tallied separately from the individual interventions.
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.</figcaption>
</figure>
</div>
<p>The purpose of this review is to provide an evolving resource tracking the status of efforts to repurpose and develop drugs for the treatment of COVID-19.
We highlight four strategies that provide different paradigms for the identification of potential pharmaceutical treatments.
The WHO guidelines <span class="citation" data-cites="17uZcQTT">(<a href="#ref-17uZcQTT" role="doc-biblioref">81</a>)</span> and a systematic review <span class="citation" data-cites="3bpuMk1r">(<a href="#ref-3bpuMk1r" role="doc-biblioref">740</a>)</span> are complementary living documents that summarize COVID-19 therapeutics.</p>
<h2 data-number="4.4" id="repurposing-drugs-for-symptom-management"><span class="header-section-number">4.4</span> Repurposing Drugs for Symptom Management</h2>
<p>A variety of symptom profiles with a range of severity are associated with COVID-19 <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span>.
In many cases, COVID-19 is not life threatening.
A study of COVID-19 patients in a hospital in Berlin, Germany reported that the highest risk of death was associated with infection-related symptoms, such as sepsis, respiratory symptoms such as ARDS, and cardiovascular failure or pulmonary embolism <span class="citation" data-cites="xnPm1mx8">(<a href="#ref-xnPm1mx8" role="doc-biblioref">741</a>)</span>.
Similarly, an analysis in Wuhan, China reported that respiratory failure (associated with ARDS) and sepsis/multi-organ failure accounted for 69.5% and 28.0% of deaths, respectively, among 82 deceased patients <span class="citation" data-cites="16mSlRgM3">(<a href="#ref-16mSlRgM3" role="doc-biblioref">742</a>)</span>.
COVID-19 is characterized by two phases.
The first is the acute response, where an adaptive immune response to the virus is established and in many cases can mitigate viral damage to organs <span class="citation" data-cites="u87rwxD8">(<a href="#ref-u87rwxD8" role="doc-biblioref">743</a>)</span>.
The second phase characterizes more severe cases of COVID-19.
Here, patients experience a cytokine storm, whereby excessive production of cytokines floods into circulation, leading to systemic inflammation, immune dysregulation, and multiorgan dysfunction that can cause multiorgan failure and death if untreated <span class="citation" data-cites="nCVMwr28">(<a href="#ref-nCVMwr28" role="doc-biblioref">744</a>)</span>.
ARDS-associated respiratory failure can occur during this phase.
Cytokine dysregulation was also identified in patients with SARS <span class="citation" data-cites="gmWrMKlb 7pMLoPq9">(<a href="#ref-gmWrMKlb" role="doc-biblioref">745</a>, <a href="#ref-7pMLoPq9" role="doc-biblioref">746</a>)</span>.</p>
<p>In the early days of the COVID-19 pandemic, physicians sought to identify potential treatments that could benefit patients, and in some cases shared their experiences and advice with the medical community on social media sites such as Twitter <span class="citation" data-cites="RR3eH1DH">(<a href="#ref-RR3eH1DH" role="doc-biblioref">747</a>)</span>.
These on-the-ground treatment strategies could later be analyzed retrospectively in observational studies or investigated in an interventional paradigm through RCTs.
Several notable cases involved the use of small-molecule drugs, which are synthesized compounds of low molecular weight, typically less than 1 kilodalton (kDa) <span class="citation" data-cites="CcEjS89m">(<a href="#ref-CcEjS89m" role="doc-biblioref">748</a>)</span>.
Small-molecule pharmaceutical agents have been a backbone of drug development since the discovery of penicillin in the early twentieth century <span class="citation" data-cites="ZtfLQJhh">(<a href="#ref-ZtfLQJhh" role="doc-biblioref">749</a>)</span>.
It and other antibiotics have long been among the best known applications of small molecules to therapeutics, but biotechnological developments such as the prediction of protein-protein interactions (PPIs) have facilitated advances in precise targeting of specific structures using small molecules <span class="citation" data-cites="ZtfLQJhh">(<a href="#ref-ZtfLQJhh" role="doc-biblioref">749</a>)</span>.
Small molecule drugs today encompass a wide range of therapeutics beyond antibiotics, including antivirals, protein inhibitors, and many broad-spectrum pharmaceuticals.</p>
<p>Many treatments considered for COVID-19 have relied on a broad-spectrum approach.
These treatments do not specifically target a virus or particular host receptor, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
This approach relies on the fact that when a virus enters a host, the host becomes the virus’s environment.
Therefore, the state of the host can also influence the virus’s ability to replicate and spread.
The administration and assessment of broad-spectrum small-molecule drugs on a rapid time course was feasible because they are often either available in hospitals, or in some cases may also be prescribed to a large number of out-patients.
One of the other advantages is that these well-established compounds, if found to be beneficial, are often widely available, in contrast to boutique experimental drugs.</p>
<p>In some cases, prior data was available from experiments examining the response of other HCoVs or HCoV infections to a candidate drug.
In addition to non-pharmaceutical interventions such as encouraging non-intubated patients to adopt a prone position <span class="citation" data-cites="1EZ5IFkua">(<a href="#ref-1EZ5IFkua" role="doc-biblioref">750</a>)</span>, knowledge about interactions between HCoVs and the human body, many of which emerged from SARS and MERS research over the past two decades, led to the suggestion that a number of common drugs might benefit COVID-19 patients.
However, the short duration and low case numbers of prior outbreaks were less well-suited to the large-scale study of clinical applications than the COVID-19 pandemic is.
As a result, COVID-19 has presented the first opportunity to robustly evaluate treatments that were common during prior HCoV outbreaks to determine their clinical efficacy.
The first year of the COVID-19 pandemic demonstrated that there are several different trajectories that these clinically suggested, widely available candidates can follow when assessed against a widespread, novel viral threat.</p>
<p>One approach to identifying candidate small molecule drugs was to look at the approaches used to treat SARS and MERS.
Treatment of SARS and MERS patients prioritized supportive care and symptom management <span class="citation" data-cites="sP4wQEiM">(<a href="#ref-sP4wQEiM" role="doc-biblioref">340</a>)</span>.
Among the clinical treatments for SARS and MERS that were explored, there was generally a lack of evidence indicating whether they were effective.
Most of the supportive treatments for SARS were found inconclusive in meta-analysis <span class="citation" data-cites="jJCnee9O">(<a href="#ref-jJCnee9O" role="doc-biblioref">751</a>)</span>, and a 2004 review reported that not enough evidence was available to make conclusions about most treatments <span class="citation" data-cites="M1ZtcbRp">(<a href="#ref-M1ZtcbRp" role="doc-biblioref">752</a>)</span>.
However, one strategy adopted from prior HCoV outbreaks is currently the best-known treatment for severe cases of COVID-19.
Corticosteroids represent broad-spectrum treatments and are a well-known, widely available treatment for pneumonia <span class="citation" data-cites="Xh7pYr41 ueLoJJY7 11fgsEIDI 17fxIsLBV yl2THbEA KGBLpge1">(<a href="#ref-Xh7pYr41" role="doc-biblioref">753</a>–<a href="#ref-KGBLpge1" role="doc-biblioref">758</a>)</span> that have also been debated as a possible treatment for ARDS <span class="citation" data-cites="1DUXbHUEW WhuiEHlo DWCwruB0 vx02q3mZ V6tUK1F0 4f82Cr8A">(<a href="#ref-1DUXbHUEW" role="doc-biblioref">759</a>–<a href="#ref-4f82Cr8A" role="doc-biblioref">764</a>)</span>.
Corticosteroids were also used and subsequently evaluated as possible supportive care for SARS and MERS.
In general, studies and meta-analyses did not identify support for corticosteroids to prevent mortality in these HCoV infections <span class="citation" data-cites="6qEFbCHC 15K1BCAlx Rmu8ZB9w">(<a href="#ref-6qEFbCHC" role="doc-biblioref">765</a>–<a href="#ref-Rmu8ZB9w" role="doc-biblioref">767</a>)</span>; however, one found that the effects might be masked by variability in treatment protocols, such as dosage and timing <span class="citation" data-cites="M1ZtcbRp">(<a href="#ref-M1ZtcbRp" role="doc-biblioref">752</a>)</span>.
While the corticosteroids most often used to treat SARS were methylprednisolone and hydrocortisone, availability issues for these drugs at the time led to dexamethasone also being used in North America <span class="citation" data-cites="coPH1YK8">(<a href="#ref-coPH1YK8" role="doc-biblioref">768</a>)</span>.</p>
<p>Dexamethasone (9α-fluoro-16α-methylprednisolone) is a synthetic corticosteroid that binds to glucocorticoid receptors <span class="citation" data-cites="q0TqfpEr 2M8wLShq">(<a href="#ref-q0TqfpEr" role="doc-biblioref">769</a>, <a href="#ref-2M8wLShq" role="doc-biblioref">770</a>)</span>.
It functions as an anti-inflammatory agent by binding to glucocorticoid receptors with higher affinity than endogenous cortisol <span class="citation" data-cites="1ETGa7JyL">(<a href="#ref-1ETGa7JyL" role="doc-biblioref">771</a>)</span>.
Dexamethasone and other steroids are widely available and affordable, and they are often used to treat community-acquired pneumonia <span class="citation" data-cites="19dEKfvUE">(<a href="#ref-19dEKfvUE" role="doc-biblioref">772</a>)</span> as well as chronic inflammatory conditions such as asthma, allergies, and rheumatoid arthritis <span class="citation" data-cites="KQ7uhAaa bSNVCQbh 17Dq3dNwL">(<a href="#ref-KQ7uhAaa" role="doc-biblioref">773</a>–<a href="#ref-17Dq3dNwL" role="doc-biblioref">775</a>)</span>.
Immunosuppressive drugs such as steroids are typically contraindicated in the setting of infection <span class="citation" data-cites="nbiwprc">(<a href="#ref-nbiwprc" role="doc-biblioref">776</a>)</span>, but because COVID-19 results in hyperinflammation that appears to contribute to mortality via lung damage, immunosuppression may be a helpful approach to treatment <span class="citation" data-cites="3HdlV9Vf">(<a href="#ref-3HdlV9Vf" role="doc-biblioref">158</a>)</span>.
A clinical trial that began in 2012 recently reported that dexamethasone may improve outcomes for patients with ARDS <span class="citation" data-cites="1DUXbHUEW">(<a href="#ref-1DUXbHUEW" role="doc-biblioref">759</a>)</span>, but a meta-analysis of a small amount of available data about dexamethasone as a treatment for SARS suggested that it may, in fact, be associated with patient harm <span class="citation" data-cites="Ji2fchyS">(<a href="#ref-Ji2fchyS" role="doc-biblioref">777</a>)</span>.
However, the findings in SARS may have been biased by the fact that all of the studies examined were observational and a large number of inconclusive studies were not included <span class="citation" data-cites="xndHds7d">(<a href="#ref-xndHds7d" role="doc-biblioref">778</a>)</span>.
The questions of whether and when to counter hyperinflammation with immunosuppression in the setting of COVID-19 (as in SARS <span class="citation" data-cites="7pMLoPq9">(<a href="#ref-7pMLoPq9" role="doc-biblioref">746</a>)</span>) was an area of intense debate, as the risks of inhibiting antiviral immunity needed to be weighed against the beneficial anti-inflammatory effects <span class="citation" data-cites="suHq6Qnm">(<a href="#ref-suHq6Qnm" role="doc-biblioref">779</a>)</span>.
As a result, guidelines early in the pandemic typically recommended avoiding treating COVID-19 patients with corticosteroids such as dexamethasone <span class="citation" data-cites="Ji2fchyS">(<a href="#ref-Ji2fchyS" role="doc-biblioref">777</a>)</span>.</p>
<p>Despite this initial concern, dexamethasone was evaluated as a potential treatment for COVID-19 (Appendix 1).
Dexamethasone treatment comprised one arm of the multi-site Randomized Evaluation of COVID-19 Therapy (RECOVERY) trial in the United Kingdom <span class="citation" data-cites="pzQoUwz3">(<a href="#ref-pzQoUwz3" role="doc-biblioref">780</a>)</span>.
This study found that the 28-day mortality rate was lower in patients receiving dexamethasone than in those receiving standard of care (SOC).
However, this finding was driven by differences in mortality among patients who were receiving mechanical ventilation or supplementary oxygen at the start of the study.
The report indicated that dexamethasone reduced 28-day mortality relative to SOC in patients who were ventilated (29.3% versus 41.4%) and among those who were receiving oxygen supplementation (23.3% versus 26.2%) at randomization, but not in patients who were breathing independently (17.8% versus 14.0%).
These findings also suggested that dexamethasone may have reduced progression to mechanical ventilation, especially among patients who were receiving oxygen support at randomization.
Other analyses have supported the importance of disease course in determining the efficacy of dexamethasone: additional results suggest greater potential for patients who have experienced symptoms for at least seven days and patients who were not breathing independently <span class="citation" data-cites="H3C2PalE">(<a href="#ref-H3C2PalE" role="doc-biblioref">781</a>)</span>.
A meta-analysis that evaluated the results of the RECOVERY trial alongside trials of other corticosteroids, such as hydrocortisone, similarly concluded that corticosteroids may be beneficial to patients with severe COVID-19 who are receiving oxygen supplementation <span class="citation" data-cites="wNXTNffA">(<a href="#ref-wNXTNffA" role="doc-biblioref">782</a>)</span>.
Thus, it seems likely that dexamethasone is useful for treating inflammation associated with immunopathy or cytokine release syndrome (CRS), which is a condition caused by detrimental overactivation of the immune system <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span>.
In fact, corticosteroids such as dexamethasone are sometimes used to treat CRS <span class="citation" data-cites="H70g2GF9">(<a href="#ref-H70g2GF9" role="doc-biblioref">783</a>)</span>.
Guidelines were quickly updated to encourage the use of dexamethasone in severe cases <span class="citation" data-cites="QgKZfGgK">(<a href="#ref-QgKZfGgK" role="doc-biblioref">784</a>)</span>, and this affordable and widely available treatment rapidly became a valuable tool against COVID-19 <span class="citation" data-cites="1EjFnlgU8">(<a href="#ref-1EjFnlgU8" role="doc-biblioref">785</a>)</span>, with demand surging within days of the preprint’s release <span class="citation" data-cites="e3ZiDOSQ">(<a href="#ref-e3ZiDOSQ" role="doc-biblioref">786</a>)</span>.</p>
<h2 data-number="4.5" id="approaches-targeting-the-virus"><span class="header-section-number">4.5</span> Approaches Targeting the Virus</h2>
<p>Therapeutics that directly target the virus itself hold the potential to prevent people infected with SARS-CoV-2 from developing potentially damaging symptoms (Figure <a href="#fig:therapeutics">5</a>).
Such drugs typically fall into the broad category of antivirals.
Antiviral therapies hinder the spread of a virus within the host, rather than destroying existing copies of the virus, and these drugs can vary in their specificity to a narrow or broad range of viral targets.
This process requires inhibiting the replication cycle of a virus by disrupting one of six fundamental steps <span class="citation" data-cites="14gthSuhm">(<a href="#ref-14gthSuhm" role="doc-biblioref">787</a>)</span>.
In the first of these steps, the virus attaches to and enters the host cell through endocytosis.
Then the virus undergoes uncoating, which is classically defined as the release of viral contents into the host cell.
Next, the viral genetic material enters the nucleus where it gets replicated during the biosynthesis stage.
During the assembly stage, viral proteins are translated, allowing new viral particles to be assembled.
In the final step new viruses are released into the extracellular environment.
Although antivirals are designed to target a virus, they can also impact other processes in the host and may have unintended effects.
Therefore, these therapeutics must be evaluated for both efficacy and safety.
As the technology to respond to emerging viral threats has also evolved over the past two decades, a number of candidate treatments have been identified for prior viruses that may be relevant to the treatment of COVID-19.</p>
<div id="fig:therapeutics" class="fignos">
<figure>
<img src="images/N001-LifeCyclePlusDrugs.png" alt="Figure 5: Mechanisms of Action for Potential Therapeutics Potential therapeutics currently being studied can target the SARS-CoV-2 virus or modify the host environment through many different mechanisms. Here, the relationships between the virus, host cells, and several therapeutics are visualized. Drug names are color-coded according to the grade assigned to them by the Center for Cytokine Storm Treatment &amp; Laboratory’s CORONA Project (788) (Green = A, Lime = B, Orange = C, and Red = D)." />
<figcaption aria-hidden="true"><span>Figure 5:</span> <strong>Mechanisms of Action for Potential Therapeutics</strong>
Potential therapeutics currently being studied can target the SARS-CoV-2 virus or modify the host environment through many different mechanisms.
Here, the relationships between the virus, host cells, and several therapeutics are visualized.
Drug names are color-coded according to the grade assigned to them by the Center for Cytokine Storm Treatment &amp; Laboratory’s CORONA Project <span class="citation" data-cites="zeTnEn7e">(<a href="#ref-zeTnEn7e" role="doc-biblioref">788</a>)</span> (Green = A, Lime = B, Orange = C, and Red = D).</figcaption>
</figure>
</div>
<p>Many antiviral drugs are designed to inhibit the replication of viral genetic material during the biosynthesis step.
Unlike DNA viruses, which can use the host enzymes to propagate themselves, RNA viruses like SARS-CoV-2 depend on their own polymerase, the RNA-dependent RNA polymerase (RdRP), for replication <span class="citation" data-cites="tn6j6IZ0 Sh1FwWps">(<a href="#ref-tn6j6IZ0" role="doc-biblioref">789</a>, <a href="#ref-Sh1FwWps" role="doc-biblioref">790</a>)</span>.
RdRP is therefore a potential target for antivirals against RNA viruses.
Disruption of RdRP is the proposed mechanism underlying the treatment of SARS and MERS with ribavirin <span class="citation" data-cites="6Qlsbuqr">(<a href="#ref-6Qlsbuqr" role="doc-biblioref">791</a>)</span>.
Ribavirin is an antiviral drug effective against other viral infections that was often used in combination with corticosteroids and sometimes interferon (IFN) medications to treat SARS and MERS <span class="citation" data-cites="G5NJrE75">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>)</span>.
However, analyses of its effects in retrospective and <em>in vitro</em> analyses of SARS and the SARS-CoV-1 virus, respectively, have been inconclusive <span class="citation" data-cites="G5NJrE75">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>)</span>.
While IFNs and ribavirin have shown promise in <em>in vitro</em> analyses of MERS, their clinical effectiveness remains unknown <span class="citation" data-cites="G5NJrE75">(<a href="#ref-G5NJrE75" role="doc-biblioref">250</a>)</span>.
The current COVID-19 pandemic has provided an opportunity to assess the clinical effects of these treatments.
As one example, ribivarin was also used in the early days of COVID-19, but a retrospective cohort study comparing patients who did and did not receive ribivarin revealed no effect on the mortality rate <span class="citation" data-cites="kO7WRULi">(<a href="#ref-kO7WRULi" role="doc-biblioref">792</a>)</span>.</p>
<p>Since nucleotides and nucleosides are the natural building blocks for RNA synthesis, an alternative approach has been to explore nucleoside and nucleotide analogs for their potential to inhibit viral replication.
Analogs containing modifications to nucleotides or nucleosides can disrupt key processes including replication <span class="citation" data-cites="MPJdSF1p">(<a href="#ref-MPJdSF1p" role="doc-biblioref">793</a>)</span>.
A single incorporation does not influence RNA transcription; however, multiple events of incorporation lead to the arrest of RNA synthesis <span class="citation" data-cites="ukZRKllk">(<a href="#ref-ukZRKllk" role="doc-biblioref">794</a>)</span>.
One candidate antiviral considered for the treatment of COVID-19 is favipiravir (Avigan), also known as T-705, which was discovered by Toyama Chemical Co., Ltd. <span class="citation" data-cites="IkAU62Pd">(<a href="#ref-IkAU62Pd" role="doc-biblioref">795</a>)</span>.
It was previously found to be effective at blocking viral amplification in several influenza subtypes as well as other RNA viruses, such as <em>Flaviviridae</em> and <em>Picornaviridae</em>, through a reduction in plaque formation <span class="citation" data-cites="EgoJKtW4">(<a href="#ref-EgoJKtW4" role="doc-biblioref">796</a>)</span> and viral replication in Madin-Darby canine kidney cells <span class="citation" data-cites="rG5RCRdJ">(<a href="#ref-rG5RCRdJ" role="doc-biblioref">797</a>)</span>.
Favipiravir (6-fluoro-3-hydroxy-2-pyrazinecarboxamide) acts as a purine and purine nucleoside analogue that inhibits viral RNA polymerase in a dose-dependent manner across a range of RNA viruses, including influenza viruses <span class="citation" data-cites="DpdKJbki 9j7NVPJS IYEoCGHy 7d1TnpE3 15kQ4QYbx">(<a href="#ref-DpdKJbki" role="doc-biblioref">798</a>–<a href="#ref-15kQ4QYbx" role="doc-biblioref">802</a>)</span>.
Biochemical experiments showed that favipiravir was recognized as a purine nucleoside analogue and incorporated into the viral RNA template.
In 2014, the drug was approved in Japan for the treatment of influenza that was resistant to conventional treatments like neuraminidase inhibitors <span class="citation" data-cites="14p3btkRH">(<a href="#ref-14p3btkRH" role="doc-biblioref">803</a>)</span>.
Though initial analyses of favipiravir in observational studies of its effects on COVID-19 patients were promising, recent results of two small RCTs suggest that it is unlikely to affect COVID-19 outcomes (Appendix 1).</p>
<p>In contrast, another nucleoside analog, remdesivir, is one of the few treatments against COVID-19 that has received FDA approval.
Remdesivir (GS-5734) is an intravenous antiviral that was proposed by Gilead Sciences as a possible treatment for Ebola virus disease.
It is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair, causing chain termination <span class="citation" data-cites="YaxKcqq8 lIko9Imd IhdcvbzC">(<a href="#ref-YaxKcqq8" role="doc-biblioref">804</a>–<a href="#ref-IhdcvbzC" role="doc-biblioref">806</a>)</span>.
Gilead received an emergency use authorization (EUA) for remdesivir from the FDA early in the pandemic (May 2020) and was later found to reduce mortality and recovery time in a double-blind, placebo-controlled, phase III clinical trial performed at 60 trial sites, 45 of which were in the United States <span class="citation" data-cites="DtpdBrgt 1ATTPbCCO yTCAmOyt 16zLDIOyk">(<a href="#ref-DtpdBrgt" role="doc-biblioref">807</a>–<a href="#ref-16zLDIOyk" role="doc-biblioref">810</a>)</span>.
Subsequently, the WHO Solidarity trial, a large-scale, open-label trial enrolling 11,330 adult in-patients at 405 hospitals in 30 countries around the world, reported no effect of remdesivir on in-hospital mortality, duration of hospitalization, or progression to mechanical ventilation <span class="citation" data-cites="jShm9uhy">(<a href="#ref-jShm9uhy" role="doc-biblioref">811</a>)</span>.
Therefore, additional clinical trials of remdesivir in different patient pools and in combination with other therapies may be needed to refine its use in the clinic and determine the forces driving these differing results.
Remdesivir offers proof of principle that SARS-CoV-2 can be targeted at the level of viral replication, since remdesivir targets the viral RNA polymerase at high potency.
Identification of such candidates depends on knowledge about the virological properties of a novel threat.
However, the success and relative lack of success, respectively, of remdesivir and favipiravir underscore the fact that drugs with similar mechanisms will not always produce similar results in clinical trials.</p>
<h2 data-number="4.6" id="disrupting-host-virus-interactions"><span class="header-section-number">4.6</span> Disrupting Host-Virus Interactions</h2>
<h3 data-number="4.6.1" id="interrupting-viral-colonization-of-cells"><span class="header-section-number">4.6.1</span> Interrupting Viral Colonization of Cells</h3>
<p>Some of the most widely publicized examples of efforts to repurpose drugs for COVID-19 are broad-spectrum, small-molecule drugs where the mechanism of action made it seem that the drug might disrupt interactions between SARS-CoV-2 and human host cells (Figure <a href="#fig:therapeutics">5</a>).
However, the exact outcomes of such treatments are difficult to predict <em>a priori</em>, and there are several examples where early enthusiasm was not borne out in subsequent trials.
One of the most famous examples of an analysis of whether a well-known medication could provide benefits to COVID-19 patients came from the assessment of chloroquine (CQ) and hydroxychloroquine (HCQ), which are used for the treatment and prophylaxis of malaria as well as the treatment of lupus erythematosus and rheumatoid arthritis in adults <span class="citation" data-cites="1CEGhPPDd">(<a href="#ref-1CEGhPPDd" role="doc-biblioref">812</a>)</span>.
These drugs are lysosomotropic agents, meaning they are weak bases that can pass through the plasma membrane.
It was thought that they might provide benefits against SARS-CoV-2 by interfering with the digestion of antigens within the lysosome and inhibiting CD4 T-cell stimulation while promoting the stimulation of CD8 T-cells <span class="citation" data-cites="16IRfGFKb">(<a href="#ref-16IRfGFKb" role="doc-biblioref">813</a>)</span>.
These compounds also have anti-inflammatory properties <span class="citation" data-cites="16IRfGFKb">(<a href="#ref-16IRfGFKb" role="doc-biblioref">813</a>)</span> and can decrease the production of certain key cytokines involved in the immune response, including interleukin-6 (IL-6) and inhibit the stimulation of Toll-like receptors (TLR) and TLR signaling <span class="citation" data-cites="16IRfGFKb">(<a href="#ref-16IRfGFKb" role="doc-biblioref">813</a>)</span>.</p>
<p><em>In vitro</em> analyses reported that CQ inhibited cell entry of SARS-CoV-1 <span class="citation" data-cites="x6nInyCf">(<a href="#ref-x6nInyCf" role="doc-biblioref">814</a>)</span> and that both CQ and HCQ inhibited viral replication within cultured cells <span class="citation" data-cites="1GBsnkeLK">(<a href="#ref-1GBsnkeLK" role="doc-biblioref">815</a>)</span>, leading to early hope that it might provide similar therapeutic or protective effects in patients.
However, while the first publication on the clinical application of these compounds to the inpatient treatment of COVID-19 was very positive <span class="citation" data-cites="ovHLMvCi">(<a href="#ref-ovHLMvCi" role="doc-biblioref">816</a>)</span>, it was quickly discredited <span class="citation" data-cites="10J3hBiMo">(<a href="#ref-10J3hBiMo" role="doc-biblioref">817</a>)</span>.
Over the following months, extensive evidence emerged demonstrating that CQ and HCQ offered no benefits for COVID-19 patients and, in fact, carried the risk of dangerous side effects (Appendix 1).
The nail in the coffin came when findings from the large-scale RECOVERY trial were released on October 8, 2020.
This study enrolled 11,197 hospitalized patients whose physicians believed it would not harm them to participate and used a randomized, open-label design to study the effects of HCQ compared to standard of care (SOC) at 176 hospitals in the United Kingdom <span class="citation" data-cites="KwFZyfFx">(<a href="#ref-KwFZyfFx" role="doc-biblioref">818</a>)</span>.
Rates of COVID-19-related mortality did not differ between the control and HCQ arms, but patients receiving HCQ were slightly more likely to die due to cardiac events.
Patients who received HCQ also had a longer duration of hospitalization than patients receiving usual care and were more likely to progress to mechanical ventilation or death (as a combined outcome).
As a result, enrollment in the HCQ arm of the RECOVERY trial was terminated early <span class="citation" data-cites="ToVSTZlB">(<a href="#ref-ToVSTZlB" role="doc-biblioref">819</a>)</span>.
The story of CQ/HCQ therefore illustrates how initial promising <em>in vitro</em> analyses can fail to translate to clinical usefulness.</p>
<p>A similar story has arisen with the broad-spectrum, small-molecule anthelmintic ivermectin, which is a synthetic analog of avermectin, a bioactive compound produced by a microorganism known as <em>Streptomyces avermectinius</em> and <em>Streptomyces avermitilis</em> <span class="citation" data-cites="iUFEavbp Z3mIi5lw">(<a href="#ref-iUFEavbp" role="doc-biblioref">820</a>, <a href="#ref-Z3mIi5lw" role="doc-biblioref">821</a>)</span>.
Avermectin disrupts the ability of parasites to avoid the host immune response by blocking glutamate-gated chloride ion channels in the peripheral nervous system from closing, leading to hyperpolarization of neuronal membranes, disruption of neural transmission, and paralysis <span class="citation" data-cites="JqaUFO6M iUFEavbp h0d0XvmB">(<a href="#ref-iUFEavbp" role="doc-biblioref">820</a>, <a href="#ref-JqaUFO6M" role="doc-biblioref">822</a>, <a href="#ref-h0d0XvmB" role="doc-biblioref">823</a>)</span>.
Ivermectin has been used since the early 1980s to treat endo- and ecto-parasitic infections by helminths, insects, and arachnids in veterinary contexts <span class="citation" data-cites="iUFEavbp LTczTlEH">(<a href="#ref-iUFEavbp" role="doc-biblioref">820</a>, <a href="#ref-LTczTlEH" role="doc-biblioref">824</a>)</span> and since the late 1980s to treat human parasitic infections as well <span class="citation" data-cites="iUFEavbp JqaUFO6M">(<a href="#ref-iUFEavbp" role="doc-biblioref">820</a>, <a href="#ref-JqaUFO6M" role="doc-biblioref">822</a>)</span>.
More recent research has indicated that ivermectin might function as a broad-spectrum antiviral by disrupting the trafficking of viral proteins by both RNA and DNA viruses <span class="citation" data-cites="tVRUg0uE h0d0XvmB 1DkK0T25n">(<a href="#ref-h0d0XvmB" role="doc-biblioref">823</a>, <a href="#ref-tVRUg0uE" role="doc-biblioref">825</a>, <a href="#ref-1DkK0T25n" role="doc-biblioref">826</a>)</span>, although most of these studies have demonstrated this effect <em>in vitro</em> <span class="citation" data-cites="1DkK0T25n">(<a href="#ref-1DkK0T25n" role="doc-biblioref">826</a>)</span>.
The potential for antiviral effects on SARS-CoV-2 were investigated <em>in vitro</em>, and ivermectin was found to inhibit viral replication in a cell line derived from Vero cells (Vero-hSLAM) <span class="citation" data-cites="8if8wVXj">(<a href="#ref-8if8wVXj" role="doc-biblioref">827</a>)</span>.
However, inhibition of viral replication was achieved at concentrations that were much higher than that explored by existing dosage guidelines <span class="citation" data-cites="1Bfy4jI64 1HpjZVMRq">(<a href="#ref-1Bfy4jI64" role="doc-biblioref">828</a>, <a href="#ref-1HpjZVMRq" role="doc-biblioref">829</a>)</span>, which are likely to be associated with significant side effects due to the increased potential that the compound could cross the mammalian blood-brain barrier <span class="citation" data-cites="CqfRlfo5 L4pkrMXn">(<a href="#ref-CqfRlfo5" role="doc-biblioref">830</a>, <a href="#ref-L4pkrMXn" role="doc-biblioref">831</a>)</span>.</p>
<p>Retrospective studies and small RCTs began investigating the effects of standard doses of this low-cost, widely available drug.
One retrospective study reported that ivermectin reduced all-cause mortality <span class="citation" data-cites="2vqz8GpA">(<a href="#ref-2vqz8GpA" role="doc-biblioref">832</a>)</span> while another reported no difference in clinical outcomes or viral clearance <span class="citation" data-cites="17pJ7jpyv">(<a href="#ref-17pJ7jpyv" role="doc-biblioref">833</a>)</span>.
Small RCTs enrolling less than 50 patients per arm have also reported a wide array of positive <span class="citation" data-cites="uqtvrCJU OxEDSXRt EOk46HI2 1BWas3u6Y 2BejjOhs">(<a href="#ref-uqtvrCJU" role="doc-biblioref">834</a>–<a href="#ref-2BejjOhs" role="doc-biblioref">838</a>)</span> and negative results <span class="citation" data-cites="TSzFDROU Ve3f25OQ">(<a href="#ref-TSzFDROU" role="doc-biblioref">839</a>, <a href="#ref-Ve3f25OQ" role="doc-biblioref">840</a>)</span>.
A slightly larger RCT enrolling 115 patients in two arms reported inconclusive results <span class="citation" data-cites="17ZhndzAi">(<a href="#ref-17ZhndzAi" role="doc-biblioref">841</a>)</span>.
Hope for the potential of ivermectin peaked with the release of a preprint reporting results of a multicenter, double-blind RCT where a four-day course of ivermectin was associated with clinical improvement and earlier viral clearance in 400 symptomatic patients and 200 close contacts <span class="citation" data-cites="1Bo9wyMhs">(<a href="#ref-1Bo9wyMhs" role="doc-biblioref">842</a>)</span>; however, concerns were raised about both the integrity of the data and the paper itself <span class="citation" data-cites="tG2qcJCU NZZPnKnn">(<a href="#ref-tG2qcJCU" role="doc-biblioref">843</a>, <a href="#ref-NZZPnKnn" role="doc-biblioref">844</a>)</span>, and this study was removed by the preprint server Research Square <span class="citation" data-cites="MbzcgLLh">(<a href="#ref-MbzcgLLh" role="doc-biblioref">845</a>)</span>.
A similarly sized RCT suggested no effect on the duration of symptoms among 400 patients split evenly across the intervention and control arms <span class="citation" data-cites="PcA4ZwSb">(<a href="#ref-PcA4ZwSb" role="doc-biblioref">846</a>)</span>, and although meta-analyses have reported both null <span class="citation" data-cites="b8Hd08ep 17Cooa2LC">(<a href="#ref-b8Hd08ep" role="doc-biblioref">847</a>, <a href="#ref-17Cooa2LC" role="doc-biblioref">848</a>)</span> and beneficial <span class="citation" data-cites="pLcPOKgu xhdEdLXT 1FR7dFWA6 77jd6q1O w1um3tWE wgyQ7l6C BGQbf0zT 1BnKCCFKX">(<a href="#ref-pLcPOKgu" role="doc-biblioref">849</a>–<a href="#ref-1BnKCCFKX" role="doc-biblioref">856</a>)</span> effects of ivermectin on COVID-19 outcomes, the certainty is likely to be low <span class="citation" data-cites="xhdEdLXT">(<a href="#ref-xhdEdLXT" role="doc-biblioref">850</a>)</span>.
These findings are potentially biased by a small number of low-quality studies, including the preprint that has been taken down <span class="citation" data-cites="WBeU4g4R">(<a href="#ref-WBeU4g4R" role="doc-biblioref">857</a>)</span>, and the authors of one <span class="citation" data-cites="3hlhg35F">(<a href="#ref-3hlhg35F" role="doc-biblioref">858</a>)</span> have issued a notice <span class="citation" data-cites="pLcPOKgu">(<a href="#ref-pLcPOKgu" role="doc-biblioref">849</a>)</span> that they will revise their study with the withdrawn study removed.
Thus, much like HCQ/CQ, enthusiasm for research that either has not or should not have passed peer review has led to large numbers of patients worldwide receiving treatments that might not have any effect or could even be harmful.
Additionally, comments on the now-removed preprint include inquiries into how best to self-administer veterinary ivermectin as a prophylactic <span class="citation" data-cites="MbzcgLLh">(<a href="#ref-MbzcgLLh" role="doc-biblioref">845</a>)</span>, and the FDA has posted information explaining why veterinary ivermectin should not be taken by humans concerned about COVID-19 <span class="citation" data-cites="ZUL8cUnf">(<a href="#ref-ZUL8cUnf" role="doc-biblioref">859</a>)</span>.
Ivermectin is now one of several candidate therapeutics being investigated in the large-scale TOGETHER <span class="citation" data-cites="N2gVvs0r">(<a href="#ref-N2gVvs0r" role="doc-biblioref">860</a>)</span> and PRINCIPLE <span class="citation" data-cites="3N6m5mBF">(<a href="#ref-3N6m5mBF" role="doc-biblioref">861</a>)</span> clinical trials.
The TOGETHER trial, which previously demonstrated no effect of HCQ and lopinavir-ritonavir <span class="citation" data-cites="ultwDbEG">(<a href="#ref-ultwDbEG" role="doc-biblioref">862</a>)</span>, released preliminary results in early August 2021 suggesting that ivermectin also has no effect on COVID-19 outcomes <span class="citation" data-cites="v5LPdCYe">(<a href="#ref-v5LPdCYe" role="doc-biblioref">863</a>)</span>.</p>
<p>While CQ/HCQ and ivermectin are well-known medications that have long been prescribed in certain contexts, investigation of another well-established type of pharmaceutical was facilitated by the fact that it was already being taken by a large number of COVID-19 patients.
Angiotensin-converting enzyme inhibitors (ACEIs) and angiotensin II receptor blockers (ARBs) are among today’s most commonly prescribed medications, often being used to control blood pressure <span class="citation" data-cites="CIeWPFHG pbfxOvnK">(<a href="#ref-CIeWPFHG" role="doc-biblioref">864</a>, <a href="#ref-pbfxOvnK" role="doc-biblioref">865</a>)</span>.
In the United States, for example, they are prescribed well over 100,000,000 times annually <span class="citation" data-cites="1BVslkmfk">(<a href="#ref-1BVslkmfk" role="doc-biblioref">866</a>)</span>.
Prior to the COVID-19 pandemic, the relationship between ACE2, ACEIs, and SARS had been considered as possible evidence that ACE2 could serve as a therapeutic target <span class="citation" data-cites="161Tg5PUW">(<a href="#ref-161Tg5PUW" role="doc-biblioref">867</a>)</span>, and the connection had been explored through <em>in vitro</em> and molecular docking analysis <span class="citation" data-cites="ZMzupz3m">(<a href="#ref-ZMzupz3m" role="doc-biblioref">868</a>)</span> but ultimately was not pursued clinically <span class="citation" data-cites="9rA6s6Qp">(<a href="#ref-9rA6s6Qp" role="doc-biblioref">869</a>)</span>.
Data from some animal models suggest that several, but not all, ACEIs and several ARBs increase ACE2 expression in the cells of some organs <span class="citation" data-cites="17xTGqKp2">(<a href="#ref-17xTGqKp2" role="doc-biblioref">870</a>)</span>, but clinical studies have not established whether plasma ACE2 expression is increased in humans treated with these medications <span class="citation" data-cites="ZhhV4ztO">(<a href="#ref-ZhhV4ztO" role="doc-biblioref">871</a>)</span>.
In this case, rather than introducing ARBs/ACEIs, a number of analyses have investigated whether discontinuing use affects COVID-19 outcomes.
An initial observational study of the association of exposure to ACEIs or ARBs with outcomes in COVID-19 was retracted from the <em>New England Journal of Medicine</em> <span class="citation" data-cites="10q7vcaWw">(<a href="#ref-10q7vcaWw" role="doc-biblioref">872</a>)</span> due to concerns related to data availability <span class="citation" data-cites="3GIXDhwz">(<a href="#ref-3GIXDhwz" role="doc-biblioref">873</a>)</span>.
As RCTs have become available, they have demonstrated no effect of continuing versus discontinuing ARBs/ACEIs on patient outcomes <span class="citation" data-cites="IRuDYZRB 1DqZPaCtf">(<a href="#ref-IRuDYZRB" role="doc-biblioref">874</a>, <a href="#ref-1DqZPaCtf" role="doc-biblioref">875</a>)</span> (Appendix 1).
Thus, once again, despite a potential mechanistic association with the pathology of SARS-CoV-2 infection, these medications were not found to influence the trajectory of COVID-19 illness.</p>
<p>For medications that are widely known and common, clinical research into their efficacy against a novel threat can be developed very quickly.
This feasibility can present a double-edged sword.
For example, HCQ and CQ were incorporated into SOC in many countries early in the pandemic and had to be discontinued once their potential to harm COVID-19 patients became apparent <span class="citation" data-cites="l9fFL5hb hTnpKZb6">(<a href="#ref-l9fFL5hb" role="doc-biblioref">876</a>, <a href="#ref-hTnpKZb6" role="doc-biblioref">877</a>)</span>.
Dexamethasone remains the major success story from this category of repurposed drugs and is likely to have saved a large number of lives since summer 2020 <span class="citation" data-cites="1EjFnlgU8">(<a href="#ref-1EjFnlgU8" role="doc-biblioref">785</a>)</span>.</p>
<h3 data-number="4.6.2" id="manipulating-the-host-immune-response"><span class="header-section-number">4.6.2</span> Manipulating the Host Immune Response</h3>
<p>Treatments based on understanding a virus and/or how a virus interacts with the human immune system can fall into two categories: they can interact with the innate immune response, which is likely to be a similar response across viruses, or they can be specifically designed to imitate the adaptive immune response to a particular virus.
In the latter case, conservation of structure or behavior across viruses enables exploring whether drugs developed for one virus can treat another.
During the COVID-19 pandemic, a number of candidate therapeutics have been explored in these categories, with varied success.</p>
<p>Knowledge gained from trying to understand SARS-CoV-1 and MERS-CoV from a fundamental biological perspective and characterize how they interact with the human immune system provides a theoretical basis for identifying candidate therapies.
Biologics are a particularly important class of drugs for efforts to address HCoV through this paradigm.
They are produced from components of living organisms or viruses, historically primarily from animal tissues, but have become increasingly feasible to produce as recombinant technologies have advanced <span class="citation" data-cites="1ALDN7rfe">(<a href="#ref-1ALDN7rfe" role="doc-biblioref">878</a>)</span>.</p>
<p>There are many differences on the development side between biologics and synthesized pharmaceuticals, such as small molecule drugs.
Typically, biologics are orders of magnitude larger than small molecule drugs and are catabolized by the body to their amino acid components <span class="citation" data-cites="OgYDNJt3">(<a href="#ref-OgYDNJt3" role="doc-biblioref">879</a>)</span>.
They are often heat sensitive, and their toxicity can vary, as it is not directly associated with the primary effects of the drug; in general, their physiochemical properties are much less understood compared to small molecules <span class="citation" data-cites="OgYDNJt3">(<a href="#ref-OgYDNJt3" role="doc-biblioref">879</a>)</span>.
Biologics include significant medical breakthroughs such as insulin for the management of diabetes and vaccines, as well monoclonal antibodies (mAbs) and interferons (IFNs), which can be used to target the host immune response after infection.</p>
<p>mAbs have revolutionized the way we treat human diseases and have become some of the best-selling drugs in the pharmaceutical market in recent years <span class="citation" data-cites="1AKi0FYUB">(<a href="#ref-1AKi0FYUB" role="doc-biblioref">880</a>)</span>.
There are currently 79 FDA approved mAbs on the market, including antibodies for viral infections (e.g. Ibalizumab for <em>Human immunodeficiency virus</em> and Palivizumab for <em>Respiratory syncytial virus</em>) <span class="citation" data-cites="1AKi0FYUB yumBaJ6U">(<a href="#ref-1AKi0FYUB" role="doc-biblioref">880</a>, <a href="#ref-yumBaJ6U" role="doc-biblioref">881</a>)</span>.
Virus-specific neutralizing antibodies commonly target viral surface glycoproteins or host structures, thereby inhibiting viral entry through receptor binding interference <span class="citation" data-cites="dUnB3gD6 cwbHiM1o">(<a href="#ref-dUnB3gD6" role="doc-biblioref">882</a>, <a href="#ref-cwbHiM1o" role="doc-biblioref">883</a>)</span>.
This interference is predicted to reduce the viral load, mitigate disease, and reduce overall hospitalization.
mAbs can be designed for a particular virus, and significant advances have been made in the speed at which new mAbs can be identified and produced.
At the time of the SARS and MERS epidemics, interest in mAbs to reduce infection was never realized <span class="citation" data-cites="f50jNbA8 KVTUgJqr">(<a href="#ref-f50jNbA8" role="doc-biblioref">884</a>, <a href="#ref-KVTUgJqr" role="doc-biblioref">885</a>)</span>, but this allowed for mAbs to quickly be considered among the top candidates against COVID-19.</p>
<h4 data-number="4.6.2.1" id="biologics-and-the-innate-immune-response"><span class="header-section-number">4.6.2.1</span> Biologics and the Innate Immune Response</h4>
<p>Deaths from COVID-19 often occur when inflammation becomes dysregulated following an immune response to the SARS-CoV-2 virus.
Therefore, one potential approach to reducing COVID-19 mortality rates is to manage the inflammatory response in severely ill patients.
One candidate therapeutic identified that uses this mechanism is tocilizumab (TCZ).
TCZ is a mAb that was developed to manage chronic inflammation caused by the continuous synthesis of the cytokine IL-6 <span class="citation" data-cites="NDRyNd61">(<a href="#ref-NDRyNd61" role="doc-biblioref">886</a>)</span>.
IL-6 is a pro-inflammatory cytokine belonging to the interleukin family, which is comprised by immune system regulators that are primarily responsible for immune cell differentiation.
Often used to treat chronic inflammatory conditions such as rheumatoid arthritis <span class="citation" data-cites="NDRyNd61">(<a href="#ref-NDRyNd61" role="doc-biblioref">886</a>)</span>, TCZ has become a pharmaceutical of interest for the treatment of COVID-19 because of the role IL-6 plays in this disease.
It has also been approved to treat CRS caused by CAR-T treatments <span class="citation" data-cites="1636MNwvb">(<a href="#ref-1636MNwvb" role="doc-biblioref">887</a>)</span>.
While the secretion of IL-6 can be associated with chronic conditions, IL-6 is a key player in the innate immune response and is secreted by macrophages in response to the detection of pathogen-associated molecular patterns and damage-associated molecular patterns <span class="citation" data-cites="NDRyNd61">(<a href="#ref-NDRyNd61" role="doc-biblioref">886</a>)</span>.
An analysis of 191 in-patients at two Wuhan hospitals revealed that blood concentrations of IL-6 differed between patients who did and did not recover from COVID-19.
Patients who ultimately died had higher IL-6 levels at admission than those who recovered <span class="citation" data-cites="10THxyeCg">(<a href="#ref-10THxyeCg" role="doc-biblioref">83</a>)</span>.
Additionally, IL-6 levels remained higher throughout the course of hospitalization in the patients who ultimately died <span class="citation" data-cites="10THxyeCg">(<a href="#ref-10THxyeCg" role="doc-biblioref">83</a>)</span>.</p>
<p>Currently, TCZ is being administered either as a monotherapy or in combination with other treatments in 73 interventional COVID-19 clinical trials (Figure <a href="#fig:ebm-trials">4</a>).
A number of retrospective studies have been conducted in several countries <span class="citation" data-cites="ayc7zYqi Wt9vgKLY luPHOS0m 17hgtBpcV WEqfULN1 bj2feMy4">(<a href="#ref-ayc7zYqi" role="doc-biblioref">888</a>–<a href="#ref-bj2feMy4" role="doc-biblioref">893</a>)</span>.
In general, these studies have reported a positive effect of TCZ on reducing mortality in COVID-19 patients, although due to their retrospective designs, significant limitations are present in all of them (Appendix 1).
It was not until February 11, 2021 that a preprint describing preliminary results of the first RCT of TCZ was released as part of the RECOVERY trial <span class="citation" data-cites="9wnr4sOS">(<a href="#ref-9wnr4sOS" role="doc-biblioref">894</a>)</span>.
TCZ was found to reduce 28-day mortality from 33% in patients receiving SOC alone to 29% in those receiving TCZ.
Combined analysis of the RECOVERY trial data with data from smaller RCTs suggested a 13% reduction in 28-day mortality <span class="citation" data-cites="9wnr4sOS">(<a href="#ref-9wnr4sOS" role="doc-biblioref">894</a>)</span>.
While this initial report did not include the full results expected from the RECOVERY trial, this large-scale, RCT provides strong evidence that TCZ may offer benefits for COVID-19 patients.
The RECOVERY trial along with results from several other RCTs <span class="citation" data-cites="3qsYQAdF 11XXNLZ90 TBjYbLjT SytvPCdR eUSZ6Jb5">(<a href="#ref-3qsYQAdF" role="doc-biblioref">895</a>–<a href="#ref-eUSZ6Jb5" role="doc-biblioref">899</a>)</span> were cited as support for the EUA issued for TCZ in June 2021 <span class="citation" data-cites="sW1V7yT">(<a href="#ref-sW1V7yT" role="doc-biblioref">900</a>)</span>.
However, the fact that TCZ suppresses the immune response means that it does carry risks for patients, especially a potential risk of secondary infection (Appendix 1).</p>
<p>TCZ is just one example of a candidate drug targeting the host immune response and specifically excessive inflammation.
For example, interferons (IFNs) have also been investigated; these are a family of cytokines critical to activating the innate immune response against viral infections.
Synairgen has been investigating a candidate drug, SNG001, which is an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation <span class="citation" data-cites="5GxCGzrT">(<a href="#ref-5GxCGzrT" role="doc-biblioref">901</a>)</span> that they reported reduced progression to ventilation in a double-blind, placebo-controlled, multi-center study of 101 patients with an average age in the late 50s <span class="citation" data-cites="bD1bgfSb qFDbLzcP">(<a href="#ref-bD1bgfSb" role="doc-biblioref">902</a>, <a href="#ref-qFDbLzcP" role="doc-biblioref">903</a>)</span>.
However, these findings were not supported by the large-scale WHO Solidarity trial, which reported no significant effect of IFN-β-1a on patient survival during hospitalization <span class="citation" data-cites="jShm9uhy">(<a href="#ref-jShm9uhy" role="doc-biblioref">811</a>)</span>, although differences in the designs of the two studies, and specifically the severity of illness among enrolled patients, may have influenced their divergent outcomes (Appendix 1).
Other biologics influencing inflammation are also being explored (Appendix 1).
It is also important that studies focused on inflammation as a possible therapeutic target consider the potential differences in baseline inflammation among patients from different backgrounds, which may be caused by differing life experiences (see <span class="citation" data-cites="Up1vB19z">(<a href="#ref-Up1vB19z" role="doc-biblioref">339</a>)</span>).</p>
<h4 data-number="4.6.2.2" id="biologics-and-the-adaptive-immune-response"><span class="header-section-number">4.6.2.2</span> Biologics and the Adaptive Immune Response</h4>
<p>While TCZ is an example of an mAb focused on managing the innate immune response, other treatments are more specific, targeting the adaptive immune response after an infection.
In some cases, treatments can utilize biologics obtained directly from recovered individuals.
From the very early days of the COVID-19 pandemic, polyclonal antibodies from convalescent plasma were investigated as a potential treatment for COVID-19 <span class="citation" data-cites="f52YwpY3 JPUZjTnq">(<a href="#ref-f52YwpY3" role="doc-biblioref">904</a>, <a href="#ref-JPUZjTnq" role="doc-biblioref">905</a>)</span>.
Convalescent plasma was used in prior epidemics including SARS, Ebola Virus Disease, and even the 1918 Spanish Influenza <span class="citation" data-cites="f52YwpY3 kAKKkMAW">(<a href="#ref-f52YwpY3" role="doc-biblioref">904</a>, <a href="#ref-kAKKkMAW" role="doc-biblioref">906</a>)</span>.
Use of convalescent plasma transfusion (CPT) over more than a century has aimed to reduce symptoms and improve mortality in infected people <span class="citation" data-cites="kAKKkMAW">(<a href="#ref-kAKKkMAW" role="doc-biblioref">906</a>)</span>, possibly by accelerating viral clearance <span class="citation" data-cites="f52YwpY3">(<a href="#ref-f52YwpY3" role="doc-biblioref">904</a>)</span>.
However, it seems unlikely that this classic treatment confers any benefit for COVID-19 patients.
Several systematic reviews have investigated whether CPT reduced mortality in COVID-19 patients, and although findings from early in the pandemic (up to April 19, 2020) did support use of CPT <span class="citation" data-cites="kAKKkMAW">(<a href="#ref-kAKKkMAW" role="doc-biblioref">906</a>)</span>, the tide has shifted as the body of available literature has grown <span class="citation" data-cites="EuYDqR7q">(<a href="#ref-EuYDqR7q" role="doc-biblioref">907</a>)</span>.
While titer levels were suggested as a possible determining factor in the success of CPT against COVID-19 <span class="citation" data-cites="11J6lIInT">(<a href="#ref-11J6lIInT" role="doc-biblioref">908</a>)</span>, the large-scale RECOVERY trial evaluated the effect of administering high-titer plasma specifically and found no effect on mortality or hospital discharge over a 28-day period <span class="citation" data-cites="124Fu0rXD">(<a href="#ref-124Fu0rXD" role="doc-biblioref">909</a>)</span>.
These results thus suggest that, despite initial optimism and an EUA from the FDA, CPT is unlikely to be an effective therapeutic for COVID-19.</p>
<p>A different narrative is shaping up around the use of mAbs specifically targeting SARS-CoV-2.
During the first SARS epidemic in 2002, neutralizing antibodies (nAbs) were found in SARS-CoV-1-infected patients <span class="citation" data-cites="3gp7t98X KULxo48U">(<a href="#ref-3gp7t98X" role="doc-biblioref">910</a>, <a href="#ref-KULxo48U" role="doc-biblioref">911</a>)</span>.
Several studies following up on these findings identified various S-glycoprotein epitopes as the major targets of nAbs against SARS-CoV-1 <span class="citation" data-cites="MTcTiS7i">(<a href="#ref-MTcTiS7i" role="doc-biblioref">912</a>)</span>.
Coronaviruses use trimeric spike (S) glycoproteins on their surface to bind to the host cell, allowing for cell entry <span class="citation" data-cites="15l3di3Wj qcVbT0w4">(<a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>, <a href="#ref-15l3di3Wj" role="doc-biblioref">33</a>)</span>.
Each S glycoprotein protomer is comprised of an S1 domain, also called the receptor binding domain (RBD), and an S2 domain.
The S1 domain binds to the host cell while the S2 domain facilitates the fusion between the viral envelope and host cell membranes <span class="citation" data-cites="MTcTiS7i">(<a href="#ref-MTcTiS7i" role="doc-biblioref">912</a>)</span>.
The genomic identity between the RBD of SARS-CoV-1 and SARS-CoV-2 is around 74% <span class="citation" data-cites="41WAtZmB">(<a href="#ref-41WAtZmB" role="doc-biblioref">913</a>)</span>.
Due to this high degree of similarity, preexisting antibodies against SARS-CoV-1 were initially considered candidates for neutralizing activity against SARS-CoV-2.
While some antibodies developed against the SARS-CoV-1 spike protein showed cross-neutralization activity with SARS-CoV-2 <span class="citation" data-cites="19R52ruRw YigWg7xu">(<a href="#ref-19R52ruRw" role="doc-biblioref">914</a>, <a href="#ref-YigWg7xu" role="doc-biblioref">915</a>)</span>, others failed to bind to SARS-CoV-2 spike protein at relevant concentrations <span class="citation" data-cites="CqQVkaqj">(<a href="#ref-CqQVkaqj" role="doc-biblioref">16</a>)</span>.
Cross-neutralizing activities were dependent on whether the epitope recognized by the antibodies were conserved between SARS-CoV-1 and SARS-CoV-2 <span class="citation" data-cites="19R52ruRw">(<a href="#ref-19R52ruRw" role="doc-biblioref">914</a>)</span>.</p>
<p>Technological advances in antibody drug design as well as in structural biology massively accelerated the discovery of novel antibody candidates and the mechanisms by which they interact with the target structure.
Within just a year of the structure of the SARS-CoV-2 spike protein being published, an impressive pipeline of monoclonal antibodies targeting SARS-CoV-2 entered clinical trials, with hundreds more candidates in preclinical stages.
The first human monoclonal neutralizing antibody specifically against the SARS-CoV-2 S glycoprotein was developed using hybridoma technology <span class="citation" data-cites="XcuzhxrJ">(<a href="#ref-XcuzhxrJ" role="doc-biblioref">916</a>)</span>, where antibody-producing B-cells developed by mice are inserted into myeloma cells to produce a hybrid cell line (the hybridoma) that is grown in culture.
The 47D11 antibody clone was able to cross-neutralize SARS-CoV-1 and SARS-CoV-2.
This antibody (now ABVV-47D11) has recently entered clinical trials in collaboration with AbbVie.
Additionally, an extensive monoclonal neutralizing antibody pipeline has been developed to combat the ongoing pandemic, with over 50 different antibodies in clinical trials <span class="citation" data-cites="hkzKw8NQ">(<a href="#ref-hkzKw8NQ" role="doc-biblioref">917</a>)</span>.
Thus far, the monotherapy sotrovimab and two antibody cocktails (bamlanivimab/estesevimab and casirivimab/imdevimab) have been granted EUAs by the FDA.</p>
<p>One of the studied antibody cocktails consists of bamlanivimab and estesevimab.
Bamlanivimab (Ly-CoV555) is a human mAb that was derived from convalescent plasma donated by a recovered COVID-19 patient, evaluated in research by the National Institute of Allergy and Infectious Diseases (NIAID), and subsequently developed by AbCellera and Eli Lilly.
The neutralizing activity of bamlanivimab was initially demonstrated <em>in vivo</em> using a nonhuman primate model <span class="citation" data-cites="f44jpdvR">(<a href="#ref-f44jpdvR" role="doc-biblioref">918</a>)</span>.
Based on these positive preclinical data, Eli Lilly initiated the first human clinical trial for a monoclonal antibody against SARS-CoV-2.
The phase 1 trial, which was conducted in hospitalized COVID-19 patients, was completed in August 2020 <span class="citation" data-cites="4IdGvvyC">(<a href="#ref-4IdGvvyC" role="doc-biblioref">919</a>)</span>.
Estesevimab (LY-CoV016 or JS-016) is also a monoclonal neutralizing antibody against the spike protein of SARS-CoV-2.
It was initially developed by Junshi Biosciences and later licensed and developed through Eli Lilly.
A phase 1 clinical trial to assess the safety of etesevimab was completed in October 2020 <span class="citation" data-cites="coZGelVr">(<a href="#ref-coZGelVr" role="doc-biblioref">920</a>)</span>.
Etesevimab was shown to bind a different epitope on the spike protein than bamlanivimab, suggesting that the two antibodies used as a combination therapy would further enhance their clinical use compared to a monotherapy <span class="citation" data-cites="Gwct8aFG">(<a href="#ref-Gwct8aFG" role="doc-biblioref">921</a>)</span>.
To assess the efficacy and safety of bamlanivimab alone or in combination with etesevimab for the treatment of COVID-19, a phase 2/3 trial (BLAZE-1) <span class="citation" data-cites="f6m2SeLi">(<a href="#ref-f6m2SeLi" role="doc-biblioref">922</a>)</span> was initiated.
The interim analysis of the phase 2 portion suggested that bamlanivimab alone was able to accelerate the reduction in viral load <span class="citation" data-cites="Cblh7jlS">(<a href="#ref-Cblh7jlS" role="doc-biblioref">923</a>)</span>.
However, more recent data suggests that only the bamlanivimab/etesevimab combination therapy is able to reduce viral load in COVID-19 patients <span class="citation" data-cites="Gwct8aFG">(<a href="#ref-Gwct8aFG" role="doc-biblioref">921</a>)</span>.
Based on this data, the combination therapy received an EUA for COVID-19 from the FDA in February 2021 <span class="citation" data-cites="eNQP4PU">(<a href="#ref-eNQP4PU" role="doc-biblioref">924</a>)</span>.</p>
<p>A second therapy is comprised of casirivimab and imdevimab (REGN-COV2).
Casirivimab (REGN10933) and imdevimab (REGN10987) are two monoclonal antibodies against the SARS-CoV-2 spike protein.
They were both developed by Regeneron in a parallel high-throughput screening (HTS) to identify neutralizing antibodies from either humanized mice or patient-derived convalescent plasma <span class="citation" data-cites="fom0jw7s">(<a href="#ref-fom0jw7s" role="doc-biblioref">925</a>)</span>.
In these efforts, multiple antibodies were characterized for their ability to bind and neutralize the SARS-CoV-2 spike protein.
The investigators hypothesized that an antibody cocktail, rather than each individual antibody, could increase the therapeutic efficacy while minimizing the risk for virus escape.
Therefore, the authors tested pairs of individual antibodies for their ability to simultaneously bind the RBD of the spike protein.
Based on this data, casirivimab and imdevimab were identified as the lead antibody pair, resulting in the initiation of two clinical trials <span class="citation" data-cites="Tn0PkRlx 1stdSBxx">(<a href="#ref-Tn0PkRlx" role="doc-biblioref">926</a>, <a href="#ref-1stdSBxx" role="doc-biblioref">927</a>)</span>.
Data from this phase 1-3 trial published in the <em>New England Journal of Medicine</em> shows that the REGN-COV2 antibody cocktail reduced viral load, particularly in patients with high viral load or whose endogenous immune response had not yet been initiated <span class="citation" data-cites="Mc2i06U">(<a href="#ref-Mc2i06U" role="doc-biblioref">928</a>)</span>.
However, in patients who already initiated an immune response, exogenous addition of REGN-COV2 did not improve the endogenous immune response.
Both doses were well tolerated with no serious events related to the antibody cocktail.
Based on this data, the FDA granted an EUA for REGN-COV2 in patients with mild to moderate COVID-19 who are at risk of developing severe disease <span class="citation" data-cites="HGaNhhM7">(<a href="#ref-HGaNhhM7" role="doc-biblioref">929</a>)</span>.
Ongoing efforts are trying to evaluate the efficacy of REGN-COV2 to improve clinical outcomes in hospitalized patients <span class="citation" data-cites="Tn0PkRlx">(<a href="#ref-Tn0PkRlx" role="doc-biblioref">926</a>)</span>.</p>
<p>Sotrovimab is the most recent mAb to receive an EUA.
It was identified in the memory B cells of a 2003 survivor of SARS <span class="citation" data-cites="BlMyODD1">(<a href="#ref-BlMyODD1" role="doc-biblioref">930</a>)</span> and was found to be cross-reactive with SARS-CoV-2 <span class="citation" data-cites="YigWg7xu">(<a href="#ref-YigWg7xu" role="doc-biblioref">915</a>)</span>.
This cross-reactivity is likely attributable to conservation within the epitope, with 17 out of 22 residues conserved between the two viruses, four conservatively substituted, and one semi-conservatively substituted <span class="citation" data-cites="YigWg7xu">(<a href="#ref-YigWg7xu" role="doc-biblioref">915</a>)</span>.
In fact, these residues are highly conserved among sarbecoviruses, a clade that includes SARS-CoV-1 and SARS-CoV-2 <span class="citation" data-cites="YigWg7xu">(<a href="#ref-YigWg7xu" role="doc-biblioref">915</a>)</span>.
This versatility has led to it being characterized as a “super-antibody” <span class="citation" data-cites="Cuypqy2I">(<a href="#ref-Cuypqy2I" role="doc-biblioref">931</a>)</span>, a potent, broadly neutralizing antibody <span class="citation" data-cites="YPcraeO">(<a href="#ref-YPcraeO" role="doc-biblioref">932</a>)</span>.
Interim analysis of data from a clinical trial <span class="citation" data-cites="1DiKhFgDx">(<a href="#ref-1DiKhFgDx" role="doc-biblioref">933</a>)</span> reported high safety and efficacy of this mAb in 583 COVID-19 patients <span class="citation" data-cites="6EgCVUr1">(<a href="#ref-6EgCVUr1" role="doc-biblioref">934</a>)</span>.
Compared to placebo, sotrovimab was found to be 85% more effective in reducing progression to the primary endpoint, which was the proportion of patients who, within 29 days, were either hospitalized for more than 24 hours or died.
Additionally, rates of adverse events were comparable, and in some cases lower, among patients receiving sotrovimab compared to patients receiving a placebo.
Sotrovimab therefore represents a mAb therapeutic that is effective against SARS-CoV-2 and may also be effective against other sarbecoviruses.</p>
<p>Several potential limitations remain in the application of mAbs to the treatment of COVID-19.
One of the biggest challenges is identifying antibodies that not only bind to their target, but also prove to be beneficial for disease management.
Currently, use of mAbs is limited to people with mild to moderate disease that are not hospitalized, and it has yet to be determined whether they can be used as a successful treatment option for severe COVID-19 patients.
While preventing people from developing severe illness provides significant benefits, patients with severe illness are at the greatest risk of death, and therefore therapeutics that provide benefits against severe illness are particularly desirable.
It remains to be seen whether mAbs confer any benefits for patients in this category.</p>
<p>Another concern about therapeutics designed to amplify the response to a specific viral target is that they may need to be modified as the virus evolves.
With the ongoing global spread of new SARS-CoV-2 variants, there is a growing concern that mutations in SARS-CoV-2 spike protein could escape antibody neutralization, thereby reducing the efficacy of monoclonal antibody therapeutics and vaccines.
A comprehensive mutagenesis screen recently identified several amino acid substitutions in the SARS-CoV-2 spike protein that can prevent antibody neutralization <span class="citation" data-cites="1FtcInMeU">(<a href="#ref-1FtcInMeU" role="doc-biblioref">935</a>)</span>.
While some mutations result in resistance to only one antibody, others confer broad resistance to multiple mAbs as well as polyclonal human sera, suggesting that some amino acids are “hotspots” for antibody resistance.
However, it was not investigated whether the resistance mutations identified result in a fitness advantage.
Accordingly, an impact on neutralizing efficiency has been reported for the B.1.1.7 (Alpha) variant first identified in the UK and the B.1.351 (Beta) variant first identified in in South Africa <span class="citation" data-cites="g9k7N6ig ZDM0wbnx K18v3qZ">(<a href="#ref-g9k7N6ig" role="doc-biblioref">936</a>–<a href="#ref-K18v3qZ" role="doc-biblioref">938</a>)</span>.
As of June 25, 2021, the CDC recommended a pause in the use of bamlanivimab and etesevimab due to decreased efficacy against the P.1 (Gamma) and B.1.351 (Beta) variants of SARS-CoV-2 <span class="citation" data-cites="16pnbuotc">(<a href="#ref-16pnbuotc" role="doc-biblioref">939</a>)</span>.
While the reported impact on antibody neutralization needs to be confirmed <em>in vivo</em>, it suggests that some adjustments to therapeutic antibody treatments may be necessary to maintain the efficacy that was reported in previous clinical trials.</p>
<p>Several strategies have been employed to try to mitigate the risk of diminished antibody neutralization.
Antibody cocktails such as those already holding an EUA may help overcome the risk for attenuation of the neutralizing activity of a single monoclonal antibody.
These cocktails consist of antibodies that recognize different epitopes on the spike protein, decreasing the likelihood that a single amino acid change can cause resistance to all antibodies in the cocktail.
However, neutralizing resistance can emerge even against an antibody cocktail if the individual antibodies target subdominant epitopes <span class="citation" data-cites="ZDM0wbnx">(<a href="#ref-ZDM0wbnx" role="doc-biblioref">937</a>)</span>.
Another strategy is to develop broadly neutralizing antibodies that target structures that are highly conserved, as these are less likely to mutate <span class="citation" data-cites="tnF7yB6H 1Dj5HnmXS">(<a href="#ref-tnF7yB6H" role="doc-biblioref">940</a>, <a href="#ref-1Dj5HnmXS" role="doc-biblioref">941</a>)</span> or to target epitopes that are insensitive to mutations <span class="citation" data-cites="NtlqU5Jy">(<a href="#ref-NtlqU5Jy" role="doc-biblioref">942</a>)</span>.
Sotrovimab, one such “super-antibody”, is thought to be somewhat robust to neutralization escape <span class="citation" data-cites="5sCeztks">(<a href="#ref-5sCeztks" role="doc-biblioref">943</a>)</span> and has been found to be effective against all variants assessed as of August 12, 2021 <span class="citation" data-cites="n1L18DDN">(<a href="#ref-n1L18DDN" role="doc-biblioref">944</a>)</span>.
Another antibody (ADG-2) targets a highly conserved epitope that overlaps the hACE2 binding site of all clade 1 sarbecoviruses <span class="citation" data-cites="B1wzHRpt">(<a href="#ref-B1wzHRpt" role="doc-biblioref">945</a>)</span>.
Prophylactic administration of ADG-2 in an immunocompetent mouse model of COVID-19 resulted in protection against viral replication in the lungs and respiratory burden.
Since the epitope targeted by ADG-2 represents an Achilles’ heel for clade 1 sarbecoviruses, this antibody, like sotrovimab, might be a promising candidate against all circulating variants as well as emerging SARS-related coronaviruses.
To date, it has fared well against the Alpha, Beta, Gamma, and Delta variants <span class="citation" data-cites="n1L18DDN">(<a href="#ref-n1L18DDN" role="doc-biblioref">944</a>)</span>.</p>
<p>The development of mAbs against SARS-CoV-2 has made it clear that this technology is rapidly adaptable and offers great potential for the response to emerging viral threats.
However, additional investigation may be needed to adapt mAb treatments to SARS-CoV-2 as it evolves and potentially to pursue designs that confer benefits for patients at the greatest risk of death.
While polyclonal antibodies from convalescent plasma have been evaluated as a treatment for COVID-19, these studies have suggested fewer potential benefits against SARS-CoV-2 than mAbs; convalescent plasma therapy has been thoroughly reviewed elsewhere <span class="citation" data-cites="f52YwpY3 JPUZjTnq">(<a href="#ref-f52YwpY3" role="doc-biblioref">904</a>, <a href="#ref-JPUZjTnq" role="doc-biblioref">905</a>)</span>.
Thus, advances in biologics for COVID-19 illustrate that an understanding of how the host and virus interact can guide therapeutic approaches.
The FDA authorization of two combination mAb therapies, in particular, underscores the potential for this strategy to allow for a rapid response to a novel pathogen.
Additionally, while TCZ is not yet as established, this therapy suggests that the strategy of using biologics to counteract the cytokine storm response may provide therapies for the highest-risk patients.</p>
<h2 data-number="4.7" id="high-throughput-screening-for-drug-repurposing"><span class="header-section-number">4.7</span> High-Throughput Screening for Drug Repurposing</h2>
<p>The drug development process is slow and costly, and developing compounds specifically targeted to an emerging viral threat is not a practical short-term solution.
Screening existing drug compounds for alternative indications is a popular alternative <span class="citation" data-cites="12nqESooA DOd7lUdu SotNtXcx Jc3MrFiI">(<a href="#ref-12nqESooA" role="doc-biblioref">946</a>–<a href="#ref-Jc3MrFiI" role="doc-biblioref">949</a>)</span>.
HTS has been a goal of pharmaceutical development since at least the mid-1980s <span class="citation" data-cites="94R5R9oq">(<a href="#ref-94R5R9oq" role="doc-biblioref">950</a>)</span>.
Traditionally, phenotypic screens were used to test which compounds would induce a desired change in <em>in vitro</em> or <em>in vivo</em> models, focusing on empirical, function-oriented exploration naïve to molecular mechanism <span class="citation" data-cites="140r2Y6Ti WhCrTFgg qSXkcV36">(<a href="#ref-140r2Y6Ti" role="doc-biblioref">951</a>–<a href="#ref-qSXkcV36" role="doc-biblioref">953</a>)</span>.
In many cases, these screens utilize large libraries that encompass a diverse set of agents varying in many pharmacologically relevant properties (e.g., <span class="citation" data-cites="eT5KkS1V">(<a href="#ref-eT5KkS1V" role="doc-biblioref">954</a>)</span>).
The compounds inducing a desired effect could then be followed up on.
Around the turn of the millennium, advances in molecular biology allowed for HTS to shift towards screening for compounds interacting with a specific molecular target under the hypothesis that modulating that target would have a desired effect.
These approaches both offer pros and cons, and today a popular view is that they are most effective in combination <span class="citation" data-cites="qSXkcV36 LSRYjaxD 140r2Y6Ti">(<a href="#ref-140r2Y6Ti" role="doc-biblioref">951</a>, <a href="#ref-qSXkcV36" role="doc-biblioref">953</a>, <a href="#ref-LSRYjaxD" role="doc-biblioref">955</a>)</span>.</p>
<p>Today, some efforts to screen compounds for potential repurposing opportunities are experimental, but others use computational HTS approaches <span class="citation" data-cites="12nqESooA 1BgpXRivf">(<a href="#ref-12nqESooA" role="doc-biblioref">946</a>, <a href="#ref-1BgpXRivf" role="doc-biblioref">956</a>)</span>.
Computational drug repurposing screens can take advantage of big data in biology <span class="citation" data-cites="6IJxPIQd">(<a href="#ref-6IJxPIQd" role="doc-biblioref">737</a>)</span> and as a result are much more feasible today than during the height of the SARS and MERS outbreaks in the early 2000s and early 2010s, respectively.
Advancements in robotics also facilitate the experimental component of HTS <span class="citation" data-cites="SotNtXcx">(<a href="#ref-SotNtXcx" role="doc-biblioref">948</a>)</span>.
For viral diseases, the goal of drug repurposing is typically to identify existing drugs that have an antiviral effect likely to impede the virus of interest.
While both small molecules and biologics can be candidates for repurposing, the significantly lower price of many small molecule drugs means that they are typically more appealing candidates <span class="citation" data-cites="V2NNL2k2">(<a href="#ref-V2NNL2k2" role="doc-biblioref">957</a>)</span>.</p>
<p>Depending on the study design, screens vary in how closely they are tied to a hypothesis.
As with the candidate therapeutics described above, high-throughput experimental or computational screens can proceed based on a hypothesis.
Just as remdesivir was selected as a candidate antiviral because it is a nucleoside analog <span class="citation" data-cites="Ubugg9KB">(<a href="#ref-Ubugg9KB" role="doc-biblioref">958</a>)</span>, so too can high-throughput screens select libraries of compounds based on a molecular hypothesis.
Likewise, when the library of drugs is selected without basis in a potential mechanism, a screen can be considered hypothesis free <span class="citation" data-cites="Ubugg9KB">(<a href="#ref-Ubugg9KB" role="doc-biblioref">958</a>)</span>.
Today, both types of analyses are common both experimentally and computationally.
Both strategies have been applied to identifying candidate therapeutics against SARS-CoV-2.</p>
<h3 data-number="4.7.1" id="hypothesis-driven-screening"><span class="header-section-number">4.7.1</span> Hypothesis-Driven Screening</h3>
<p>Hypothesis-driven screens often select drugs likely to interact with specific viral or host targets or drugs with desired clinical effects, such as immunosuppressants.
There are several properties that might identify a compound as a candidate for an emerging viral disease.
Drugs that interact with a target that is shared between pathogens (i.e., a viral protease or a polymerase) or between a viral pathogen and another illness (i.e., a cancer drug with antiviral potential) are potential candidates, as are drugs that are thought to interact with additional molecular targets beyond those they were developed for <span class="citation" data-cites="1BgpXRivf">(<a href="#ref-1BgpXRivf" role="doc-biblioref">956</a>)</span>.
Such research can be driven by <em>in vitro</em> or <em>in silico</em> experimentation.
Computational analyses depend on identifying compounds that modulate pre-selected proteins in the virus or host.
As a result, they build on experimental research characterizing the molecular features of the virus, host, and candidate compounds <span class="citation" data-cites="Jc3MrFiI">(<a href="#ref-Jc3MrFiI" role="doc-biblioref">949</a>)</span>.</p>
<p>One example of the application of this approach to COVID-19 research comes from work on protease inhibitors.
Studies have shown that viral proteases play an important role in the life cycle of viruses, including coronaviruses, by modulating the cleavage of viral polyprotein precursors <span class="citation" data-cites="26dpptLw">(<a href="#ref-26dpptLw" role="doc-biblioref">959</a>)</span>.
Several FDA-approved drugs target proteases, such as lopinavir and ritonavir for HIV infection and simeprevir for hepatitis C virus infection.
Serine protease inhibitors were previously suggested as possible treatments for SARS and MERS <span class="citation" data-cites="f7csoyEu">(<a href="#ref-f7csoyEu" role="doc-biblioref">960</a>)</span>.
One early study <span class="citation" data-cites="15l3di3Wj">(<a href="#ref-15l3di3Wj" role="doc-biblioref">33</a>)</span> suggested that camostat mesylate, a protease inhibitor, could block the entry of SARS-CoV-2 into lung cells <em>in vitro</em>.
Two polyproteins encoded by the SARS-CoV-2 replicase gene, pp1a and pp1ab, are critical for viral replication and transcription <span class="citation" data-cites="4Sja6dIz">(<a href="#ref-4Sja6dIz" role="doc-biblioref">961</a>)</span>.
These polyproteins must undergo proteolytic processing, which is usually conducted by main protease (M<sup>Pro</sup>), a 33.8-kDa SARS-CoV-2 protease that is therefore fundamental to viral replication and transcription.
Therefore, it was hypothesized that compounds targeting M<sup>Pro</sup> could be used to prevent or slow the replication of the SARS-CoV-2 virus.</p>
<p>Both computational and experimental approaches facilitated the identification of compounds that might inhibit SARS-CoV-2 M<sup>Pro</sup>.
In 2005, computer-aided design facilitated the development of a Michael acceptor inhibitor, now known as N3, to target M<sup>Pro</sup> of SARS-like coronaviruses <span class="citation" data-cites="15JYG0AIU">(<a href="#ref-15JYG0AIU" role="doc-biblioref">962</a>)</span>.
N3 binds in the substrate binding pocket of M<sup>Pro</sup> in several HCoV <span class="citation" data-cites="XfOTPQPJ 15JYG0AIU 16Gt8FHKW T5r2uPRf">(<a href="#ref-15JYG0AIU" role="doc-biblioref">962</a>–<a href="#ref-T5r2uPRf" role="doc-biblioref">965</a>)</span>.
The structure of N3-bound SARS-CoV-2 M<sup>Pro</sup> has been solved, confirming the computational prediction that N3 would similarly bind in the substrate binding pocket of SARS-CoV-2 <span class="citation" data-cites="4Sja6dIz">(<a href="#ref-4Sja6dIz" role="doc-biblioref">961</a>)</span>.
N3 was tested <em>in vitro</em> on SARS-CoV-2-infected Vero cells, which belong to a line of cells established from the kidney epithelial cells of an African green monkey, and was found to inhibit SARS-CoV-2 <span class="citation" data-cites="4Sja6dIz">(<a href="#ref-4Sja6dIz" role="doc-biblioref">961</a>)</span>.
A library of approximately 10,000 compounds was screened in a fluorescence resonance energy transfer assay constructed using SARS-CoV-2 M<sup>Pro</sup> expressed in <em>Escherichia coli</em> <span class="citation" data-cites="4Sja6dIz">(<a href="#ref-4Sja6dIz" role="doc-biblioref">961</a>)</span>.</p>
<p>Six leads were identified in this hypothesis-driven screen.
<em>In vitro</em> analysis revealed that ebselen had the strongest potency in reducing the viral load in SARS-CoV-2-infected Vero cells <span class="citation" data-cites="4Sja6dIz">(<a href="#ref-4Sja6dIz" role="doc-biblioref">961</a>)</span>.
Ebselen is an organoselenium compound with anti-inflammatory and antioxidant properties <span class="citation" data-cites="17EErh01r">(<a href="#ref-17EErh01r" role="doc-biblioref">966</a>)</span>.
Molecular dynamics analysis further demonstrated the potential for ebselen to bind to M<sup>Pro</sup> and disrupt the protease’s enzymatic functions <span class="citation" data-cites="3qlLwpK7">(<a href="#ref-3qlLwpK7" role="doc-biblioref">967</a>)</span>.
However, ebselen is likely to be a promiscuous binder, which could diminish its therapeutic potential <span class="citation" data-cites="4Sja6dIz vXFGEybh">(<a href="#ref-4Sja6dIz" role="doc-biblioref">961</a>, <a href="#ref-vXFGEybh" role="doc-biblioref">968</a>)</span>, and compounds with higher specificity may be needed to translate this mechanism effectively to clinical trials.
In July 2020, phase II clinical trials commenced to assess the effects of SPI-1005, an investigational drug from Sound Pharmaceuticals that contains ebselen <span class="citation" data-cites="G5cKNGMC">(<a href="#ref-G5cKNGMC" role="doc-biblioref">969</a>)</span>, on 60 adults presenting with each of moderate <span class="citation" data-cites="cUTgHTRj">(<a href="#ref-cUTgHTRj" role="doc-biblioref">970</a>)</span> and severe <span class="citation" data-cites="K13XqU1J">(<a href="#ref-K13XqU1J" role="doc-biblioref">971</a>)</span> COVID-19.
Other M<sup>Pro</sup> inhibitors are also being evaluated in clinical trials <span class="citation" data-cites="27QvV2tx 1EzX93BEG 1EzX93BEG">(<a href="#ref-27QvV2tx" role="doc-biblioref">972</a>, <a href="#ref-1EzX93BEG" role="doc-biblioref">973</a>, <a href="#ref-1EzX93BEG" role="doc-biblioref">973</a>)</span>.
Pending the results of clinical trials, N3 remains a computationally interesting compound based on both computational and experimental data, but whether these potential effects will translate to the clinic remains unknown.</p>
<h3 data-number="4.7.2" id="hypothesis-free-screening"><span class="header-section-number">4.7.2</span> Hypothesis-Free Screening</h3>
<p>Hypothesis-free screens use a discovery-driven approach, where screens are not targeted to specific viral proteins, host proteins, or desired clinical modulation.
Hypothesis-free drug screening began twenty years ago with the testing of libraries of drugs experimentally.
Today, like many other areas of biology, <em>in silico</em> analyses have become increasingly popular and feasible through advances in biological big data <span class="citation" data-cites="Ubugg9KB 1CFhvljOs">(<a href="#ref-Ubugg9KB" role="doc-biblioref">958</a>, <a href="#ref-1CFhvljOs" role="doc-biblioref">974</a>)</span>.
Many efforts have collected data about interactions between drugs and SARS-CoV-2 and about the host genomic response to SARS-CoV-2 exposure, allowing for hypothesis-free computational screens that seek to identify new candidate therapeutics.
Thus, they utilize a systems biology paradigm to extrapolate the effect of a drug against a virus based on the host interactions with both the virus and the drug <span class="citation" data-cites="Jc3MrFiI">(<a href="#ref-Jc3MrFiI" role="doc-biblioref">949</a>)</span>.</p>
<p>Resources such as the COVID-19 Drug and Gene Set Library, which at the time of its publication contained 1,620 drugs sourced from 173 experimental and computational drug sets and 18,676 human genes sourced from 444 gene sets <span class="citation" data-cites="fYBqSPGl">(<a href="#ref-fYBqSPGl" role="doc-biblioref">975</a>)</span>, facilitate such discovery-driven approaches.
Analysis of these databases indicated that some drugs had been identified as candidates across multiple independent analyses, including high-profile candidates such as CQ/HCQ and remdesivir <span class="citation" data-cites="fYBqSPGl">(<a href="#ref-fYBqSPGl" role="doc-biblioref">975</a>)</span>.
Computational screening efforts can then mine databases and other resources to identify potential PPIs among the host, the virus, and established and/or experimental drugs <span class="citation" data-cites="YRqgrrXw">(<a href="#ref-YRqgrrXw" role="doc-biblioref">976</a>)</span>.
Subject matter expertise from human users may be integrated to varying extents depending on the platform (e.g,. <span class="citation" data-cites="YRqgrrXw 1HnR6AuMx">(<a href="#ref-YRqgrrXw" role="doc-biblioref">976</a>, <a href="#ref-1HnR6AuMx" role="doc-biblioref">977</a>)</span>).
These resources have allowed studies to identify potential therapeutics for COVID-19 without an <em>a priori</em> reason for selecting them.</p>
<p>One example of a hypothesis-free screen for COVID-19 drugs comes from a PPI-network-based analysis that was published early in the pandemic <span class="citation" data-cites="ZvaFbLRX">(<a href="#ref-ZvaFbLRX" role="doc-biblioref">193</a>)</span>.
Here, researchers cloned the proteins expressed by SARS-CoV-2 <em>in vitro</em> and quantified 332 viral-host PPI using affinity purification mass spectrometry <span class="citation" data-cites="ZvaFbLRX">(<a href="#ref-ZvaFbLRX" role="doc-biblioref">193</a>)</span>.
They identified two SARS-CoV-2 proteins (Nsp6 and Orf9c) that interacted with host Sigma-1 and Sigma-2 receptors.
Sigma receptors are located in the endoplasmic reticulum of many cell types, and type 1 and 2 Sigma receptors have overlapping but distinct affinities for a variety of ligands <span class="citation" data-cites="KQ3m1fh0">(<a href="#ref-KQ3m1fh0" role="doc-biblioref">978</a>)</span>.
Molecules interacting with the Sigma receptors were then analyzed and found to have an effect on viral infectivity <em>in vitro</em> <span class="citation" data-cites="ZvaFbLRX">(<a href="#ref-ZvaFbLRX" role="doc-biblioref">193</a>)</span>.
A follow-up study evaluated the effect of perturbing these 332 proteins in two cells lines, A549 and Caco-2, using knockdown and knockout methods, respectively, and found that the replication of SARS-CoV-2 in cells from both lines was dependent on the expression of <em>SIGMAR1</em>, which is the gene that encodes the Sigma-1 receptor <span class="citation" data-cites="GZ4E3pwn">(<a href="#ref-GZ4E3pwn" role="doc-biblioref">979</a>)</span>.
Following these results, drugs interacting with Sigma receptors were suggested as candidates for repurposing for COVID-19 (e.g., <span class="citation" data-cites="sLjk8RiX">(<a href="#ref-sLjk8RiX" role="doc-biblioref">980</a>)</span>).
Because many well-known and affordable drugs interact with the Sigma receptors <span class="citation" data-cites="ZvaFbLRX 8KoefI79">(<a href="#ref-ZvaFbLRX" role="doc-biblioref">193</a>, <a href="#ref-8KoefI79" role="doc-biblioref">981</a>)</span>, they became a major focus of drug repurposing efforts.
Some of the drugs suggested by the apparent success of Sigma receptor-targeting drugs were already being investigated at the time.
HCQ, for example, forms ligands with both Sigma-1 and Sigma-2 receptors and was already being explored as a candidate therapeutic for COVID-19 <span class="citation" data-cites="ZvaFbLRX">(<a href="#ref-ZvaFbLRX" role="doc-biblioref">193</a>)</span>.
Thus, this computational approach yielded interest in drugs whose antiviral activity was supported by initial <em>in vitro</em> analyses.</p>
<p>Follow-up research, however, called into question whether the emphasis on drugs interacting with Sigma receptors might be based on a spurious association <span class="citation" data-cites="j2enN2JO">(<a href="#ref-j2enN2JO" role="doc-biblioref">982</a>)</span>.
This study built on the prior work by examining whether antiviral activity among compounds correlated with their affinity for the Sigma receptors and found that it did not.
The study further demonstrated that cationic amphiphilicity was a shared property among many of the candidate drugs identified through both computational and phenotypic screens and that it was likely to be the source of many compounds’ proposed antiviral activity <span class="citation" data-cites="j2enN2JO">(<a href="#ref-j2enN2JO" role="doc-biblioref">982</a>)</span>.
Cationic amphiphilicity is associated with the induction of phospholipidosis, which is when phospholipids accumulate in the lysosome <span class="citation" data-cites="M4yLbwsC">(<a href="#ref-M4yLbwsC" role="doc-biblioref">983</a>)</span>.
Phospholipidosis can disrupt viral replication by inhibiting lipid processing <span class="citation" data-cites="1BV8xlc5v">(<a href="#ref-1BV8xlc5v" role="doc-biblioref">984</a>)</span> (see discussion of HCQ in Appendix 1).
However, phospholipidosis is known to translate poorly from <em>in vitro</em> models to <em>in vivo</em> models or clinical applications.
Thus, this finding suggested that these screens were identifying compounds that shared a physiochemical property rather than a specific target <span class="citation" data-cites="j2enN2JO">(<a href="#ref-j2enN2JO" role="doc-biblioref">982</a>)</span>.
The authors further demonstrated that antiviral activity against SARS-CoV-2 <em>in vitro</em> was correlated with the induction of phospholipidosis for drugs both with and without cationic amphiphilicity <span class="citation" data-cites="j2enN2JO">(<a href="#ref-j2enN2JO" role="doc-biblioref">982</a>)</span>.
This finding supports the idea that the property of cationic amphicility was being detected as a proxy for the shared effect of phospholipidosis <span class="citation" data-cites="j2enN2JO">(<a href="#ref-j2enN2JO" role="doc-biblioref">982</a>)</span>.
They demonstrated that phospholipidosis-inducing drugs were not effective at preventing viral propagation <em>in vivo</em> in a murine model of COVID-19 <span class="citation" data-cites="j2enN2JO">(<a href="#ref-j2enN2JO" role="doc-biblioref">982</a>)</span>.
Therefore, removing hits that induce phospholipidosis from computational and <em>in vitro</em> experimental repurposing screens (e.g., <span class="citation" data-cites="15kk2xRrj">(<a href="#ref-15kk2xRrj" role="doc-biblioref">985</a>)</span>) may help emphasize those that are more likely to provide clinical benefits.
This work illustrates the importance of considering confounding variables in computational screens, a principle that has been incorporated into more traditional approaches to drug development <span class="citation" data-cites="ugC5dX0E">(<a href="#ref-ugC5dX0E" role="doc-biblioref">986</a>)</span>.</p>
<p>One drug that acts on Sigma receptors does, however, remain a candidate for the treatment of COVID-19.
Several psychotropic drugs target Sigma receptors in the central nervous system and thus attracted interest as potential COVID-19 therapeutics following the findings of two host-virus PPI studies <span class="citation" data-cites="ZZ2cNql8">(<a href="#ref-ZZ2cNql8" role="doc-biblioref">987</a>)</span>.
For several of these drugs, the <em>in vitro</em> antiviral activity <span class="citation" data-cites="GZ4E3pwn">(<a href="#ref-GZ4E3pwn" role="doc-biblioref">979</a>)</span> was not correlated with their affinity for the Sigma-1 receptor <span class="citation" data-cites="ZZ2cNql8 j2enN2JO">(<a href="#ref-j2enN2JO" role="doc-biblioref">982</a>, <a href="#ref-ZZ2cNql8" role="doc-biblioref">987</a>)</span> but was correlated with phospholipidosis <span class="citation" data-cites="j2enN2JO">(<a href="#ref-j2enN2JO" role="doc-biblioref">982</a>)</span>.
However, fluvoxamine, a selective serotonin reuptake inhibitor that is a particularly potent Sigma-1 receptor agonist <span class="citation" data-cites="ZZ2cNql8">(<a href="#ref-ZZ2cNql8" role="doc-biblioref">987</a>)</span>, has shown promise as a preventative of severe COVID-19 in a preliminary analysis of data from the large-scale TOGETHER trial <span class="citation" data-cites="v5LPdCYe">(<a href="#ref-v5LPdCYe" role="doc-biblioref">863</a>)</span>.
As of August 6, 2021, this trial had collected data from over 1,400 patients in the fluvoxamine arm of their study, half of whom received a placebo <span class="citation" data-cites="v5LPdCYe">(<a href="#ref-v5LPdCYe" role="doc-biblioref">863</a>)</span>.
Only 74 patients in the fluvoxamine group had progressed to hospitalization for COVID-19 compared to 107 in the placebo group, corresponding to a relative risk of 0.69; additionally, the relative risk of mortality between the two groups was calculated at 0.71.
These findings support the results of small clinical trials that have found fluvoxamine to reduce clinical deterioration relative to a placebo <span class="citation" data-cites="jy4vKMhj s4UuQP07">(<a href="#ref-jy4vKMhj" role="doc-biblioref">988</a>, <a href="#ref-s4UuQP07" role="doc-biblioref">989</a>)</span>.
However, the ongoing therapeutic potential of fluvoxamine does not contradict the finding that hypothesis-free screening hits can be driven by confounding factors.
The authors point out that its relevance would not just be antiviral as it has a potential immunomodulatory mechanism <span class="citation" data-cites="jy4vKMhj">(<a href="#ref-jy4vKMhj" role="doc-biblioref">988</a>)</span>.
It has been found to be protective against septic shock in an <em>in vivo</em> mouse model <span class="citation" data-cites="jMLLdjmN">(<a href="#ref-jMLLdjmN" role="doc-biblioref">990</a>)</span>.
It is possible that fluvoxamine also exerts an antiviral effect <span class="citation" data-cites="18xfYamot">(<a href="#ref-18xfYamot" role="doc-biblioref">991</a>)</span>.
Thus, Sigma-1 receptor activity may contribute to fluvoxamine’s potential effects in treating COVID-19, but is not the only mechanism by which this drug can interfere with disease progression.</p>
<h3 data-number="4.7.3" id="potential-and-limitations-of-high-throughput-analyses"><span class="header-section-number">4.7.3</span> Potential and Limitations of High-Throughput Analyses</h3>
<p>Computational screening allows for a large number of compounds to be evaluated to identify those most likely to display a desired behavior or function.
This approach can be guided by a hypothesis or can aim to discover underlying characteristics that produce new hypotheses about the relationship between a host, a virus, and candidate pharmaceuticals.
The examples outlined above illustrate that HTS-based evaluations of drug repurposing can potentially provide valuable insights.
Computational techniques were used to design compounds targeting M<sup>Pro</sup> based on an understanding of how this protease aids viral replication, and M<sup>Pro</sup> inhibitors remain promising candidates <span class="citation" data-cites="SotNtXcx">(<a href="#ref-SotNtXcx" role="doc-biblioref">948</a>)</span>, although the clinical trial data is not yet available.
Similarly, computational analysis correctly identified the Sigma-1 receptor as a protein of interest.
Although the process of identifying which drugs might modulate the interaction led to an emphasis on candidates that ultimately have not been supported, fluvoxamine remains an appealing candidate.
The difference between the preliminary evidence for fluvoxamine compared to other drugs that interact with Sigma receptors underscores a major critique of hypothesis-free HTS in particular: while these approaches allow for brute force comparison of a large number of compounds against a virus of interest, they lose the element of expertise that is associated with most successes in drug repurposing <span class="citation" data-cites="Ubugg9KB">(<a href="#ref-Ubugg9KB" role="doc-biblioref">958</a>)</span>.</p>
<p>There are also practical limitations to these methods.
One concern is that computational analyses inherently depend on the quality of the data being evaluated.
The urgency of the COVID-19 pandemic led many research groups to pivot towards computational HTS research without familiarity with best practices in this area <span class="citation" data-cites="SotNtXcx">(<a href="#ref-SotNtXcx" role="doc-biblioref">948</a>)</span>.
As a result, there is an excessive amount of information available from computational studies <span class="citation" data-cites="dUvvJvv6">(<a href="#ref-dUvvJvv6" role="doc-biblioref">992</a>)</span>, but not all of it is high-quality.
Additionally, the literature used to identify and validate targets can be difficult to reproduce <span class="citation" data-cites="yWY8AHZe">(<a href="#ref-yWY8AHZe" role="doc-biblioref">993</a>)</span>, which may pose challenges to target-based experimental screening and to <em>in silico</em> screens.
Some efforts to repurpose antivirals have focused on host, rather than viral, proteins <span class="citation" data-cites="Jc3MrFiI">(<a href="#ref-Jc3MrFiI" role="doc-biblioref">949</a>)</span>, which might be expected to translate poorly <em>in vivo</em> if the targeted proteins serve essential functions in the host.
Concerns about the practicality of hypothesis-free screens to gain novel insights are underscored by the fact that very few or possibly no success stories have emerged from hypothesis-free screens over the past twenty years <span class="citation" data-cites="Ubugg9KB">(<a href="#ref-Ubugg9KB" role="doc-biblioref">958</a>)</span>.
These findings suggest that data-driven research can be an important component of the drug repurposing ecosystem, but that drug repurposing efforts that proceed without a hypothesis, an emphasis on biological mechanisms, or an understanding of confounding effects may not produce viable candidates.</p>
<h2 data-number="4.8" id="considerations-in-balancing-different-approaches"><span class="header-section-number">4.8</span> Considerations in Balancing Different Approaches</h2>
<p>The approaches described here offer a variety of advantages and limitations in responding to a novel viral threat and building on existing bodies of knowledge in different ways.
Medicine, pharmacology, basic science (especially virology and immunology), and biological data science can all provide different insights and perspectives for addressing the challenging question of which existing drugs might provide benefits against an emerging viral threat.
A symptom management-driven approach allows clinicians to apply experience with related diseases or related symptoms to organize a rapid response aimed at saving the lives of patients already infected with a new disease.
Oftentimes, the pharmaceutical agents that are applied are small-molecule, broad-spectrum pharmaceuticals that are widely available and affordable to produce, and they may already be available for other purposes, allowing clinicians to administer them to patients quickly either with an EUA or off-label.
In this vein, dexamethasone has emerged as the strongest treatment against severe COVID-19 (Table <a href="#tbl:summary-table">1</a>).</p>
<p>Alternatively, many efforts to repurpose drugs for COVID-19 have built on information gained through basic scientific research of HCoV.
Understanding how related viruses function has allowed researchers to identify possible pharmacological strategies to disrupt pathogenesis (Figure <a href="#fig:therapeutics">5</a>).
Some of the compounds identified through these methods include small-molecule antivirals, which can be boutique and experimental medications like remdesivir (Table <a href="#tbl:summary-table">1</a>).
Other candidate drugs that intercept host-pathogen interactions include biologics, which imitate the function of endogenous host compounds.
Most notably, several mAbs that have been developed (casirivimab, imdevimab, bamlanivimab and etesevimab) or repurposed (sotrovimab, tocilizumab) have now been granted EUAs (Table <a href="#tbl:summary-table">1</a>).
Although not discussed here, several vaccine development programs have also met huge success using a range of strategies <span class="citation" data-cites="1HWmcSB20 rjHK19Dr">(<a href="#ref-1HWmcSB20" role="doc-biblioref">6</a>, <a href="#ref-rjHK19Dr" role="doc-biblioref">7</a>)</span>.</p>
<div id="tbl:summary-table" class="tablenos">
<table id="tbl:summary-table">
<caption><span>Table 1:</span> Summary table of candidate therapeutics examined in this manuscript.
“Grade” is the rating given to each treatment by the Systematic Tracker of Off-label/Repurposed Medicines Grades (STORM) maintained by the Center for Cytokine Storm Treatment &amp; Laboratory (CSTL) at the University of Pennsylvania <span class="citation" data-cites="zeTnEn7e">(<a href="#ref-zeTnEn7e" role="doc-biblioref">788</a>)</span>.
A grade of A indicates that a treatment is considered effective, B that all or most RCTs have shown positive results, C that RCT data are not yet available, and D that multiple RCTs have produced negative results.
Treatments not in the STORM database are indicated as N/A.
FDA status is also provided where available.
The evidence available is based on the progression of the therapeutic through the pharmaceutical development pipeline, with RCTs as the most informative source of evidence.
The effectiveness is summarized based on the current available evidence; large trials such as RECOVERY and Solidarity are weighted heavily in this summary.
This table was last updated on August 20, 2021.
</caption>
<colgroup>
<col style="width: 12%" />
<col style="width: 7%" />
<col style="width: 11%" />
<col style="width: 19%" />
<col style="width: 20%" />
<col style="width: 28%" />
</colgroup>
<thead>
<tr class="header">
<th>Treatment</th>
<th>Grade</th>
<th>Category</th>
<th>FDA Status</th>
<th>Evidence Available</th>
<th>Suggested Effectiveness</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Dexamethasone</td>
<td>A</td>
<td>Small molecule, broad spectrum</td>
<td>Used off-label</td>
<td>RCT</td>
<td>Supported: RCT shows improved outcomes over SOC, especially in severe cases such as CRS</td>
</tr>
<tr class="even">
<td>Remdesivir</td>
<td>A</td>
<td>Small molecule, antiviral, adenosine analog</td>
<td>Approved for COVID-19 (and EUA for combination with baricitinib)</td>
<td>RCT</td>
<td>Mixed: Conflicting evidence from large WHO-led Solidarity trial vs US-focused RCT and other studies</td>
</tr>
<tr class="odd">
<td>Tocilizumab</td>
<td>A</td>
<td>Biologic, monoclonal antibody</td>
<td>EUA</td>
<td>RCT</td>
<td>Mixed: It appears that TCZ may work well in combination with dexamethasone in severe cases, but not as monotherapy</td>
</tr>
<tr class="even">
<td>Sotrovimab</td>
<td>N/A</td>
<td>Biologic, monoclonal antibody</td>
<td>EUA</td>
<td>RCT</td>
<td>Supported: Phase 2/3 clinical trial showed reduced hospitalization/death</td>
</tr>
<tr class="odd">
<td>Bamlanivimab and etesevimab</td>
<td>B &amp; N/A</td>
<td>Biologic, monoclonal antibodies</td>
<td>EUA</td>
<td>RCT</td>
<td>Supported: Phase 2 clinical trial showed reduction in viral load, but FDA pause recommended because may be less effective against Delta variant</td>
</tr>
<tr class="even">
<td>Casirivimab and imdevimab</td>
<td>N/A</td>
<td>Biologic, monoclonal antibodies</td>
<td>EUA</td>
<td>RCT</td>
<td>Supported: Reduced viral load at interim analysis</td>
</tr>
<tr class="odd">
<td>Fluvoxamine</td>
<td>B</td>
<td>Small-molecule, Sigma-1 receptor agonist</td>
<td>N/A</td>
<td>RCT</td>
<td>Supported: Support from two small RCTs and preliminary support from interim analysis of TOGETHER</td>
</tr>
<tr class="even">
<td>SNG001</td>
<td>B</td>
<td>Biologic, interferon</td>
<td>None</td>
<td>RCT</td>
<td>Mixed: Support from initial RCT but no effect found in WHO’s Solidarity trial</td>
</tr>
<tr class="odd">
<td>M<sup>Pro</sup> Protease Inhibitors</td>
<td>N/A</td>
<td>Small molecule, protease inhibitor</td>
<td>None</td>
<td>Computational prediction, <em>in vitro</em> studies</td>
<td>Unknown</td>
</tr>
<tr class="even">
<td>ARBs &amp; ACEIs</td>
<td>C</td>
<td>Small molecule, broad spectrum</td>
<td>None</td>
<td>Observational studies and some RCTs</td>
<td>Not supported: Observational study retracted, RCTs suggest no association</td>
</tr>
<tr class="odd">
<td>Favipiravir</td>
<td>D</td>
<td>Small molecule, antiviral, nucleoside analog</td>
<td>None</td>
<td>RCT</td>
<td>Not supported: RCTs do not show significant improvements for individuals taking this treatment, good safety profile</td>
</tr>
<tr class="even">
<td>HCQ/CQ</td>
<td>D</td>
<td>Small molecule, broad spectrum</td>
<td>None</td>
<td>RCT</td>
<td>Not supported, possibly harmful: Non-blinded RCTs showed no improvement over SOC, safety profile may be problematic</td>
</tr>
<tr class="odd">
<td>Convalescent plasma transfusion</td>
<td>D</td>
<td>Biologic, polyclonal antibodies</td>
<td>EUA</td>
<td>RCT</td>
<td>Mixed: Supported in small trials but not in large-scale RECOVERY trial</td>
</tr>
<tr class="even">
<td>Ivermectin</td>
<td>D</td>
<td>Small molecule, broad spectrum</td>
<td>None</td>
<td>RCT</td>
<td>Mixed: Mixed results from small RCTs, major supporting RCT now withdrawn, preliminary results of large RCT (TOGETHER) suggest no effect on emergency room visits or hospitalization for COVID-19</td>
</tr>
</tbody>
</table>
</div>
<p>All of the small-molecule drugs evaluated and most of the biologics are repurposed, and thus hinge on a theoretical understanding of how the virus interacts with a human host and how pharmaceuticals can be used to modify those interactions rather than being designed specifically against SARS-CoV-2 or COVID-19.
As a result, significant attention has been paid to computational approaches that automate the identification of potentially desirable interactions.
However, work in COVID-19 has made it clear that relevant compounds can also be masked by confounds, and spurious associations can drive investment in candidate therapeutics that are unlikely to translate to the clinic.
Such spurious hits are especially likely to impact hypothesis-free screens.
However, hypothesis-free screens may still be able to contribute to the drug discovery or repurposing ecosystem, assuming the computational arm of HTS follows the same trends seen in its experimental arm.
In 2011, a landmark study in drug discovery demonstrated that although more new drugs were discovered using target-based rather than phenotypic approaches, the majority of drugs with a novel molecular mechanism of action (MMOA) were identified in phenotypic screens <span class="citation" data-cites="rlm4VbAG">(<a href="#ref-rlm4VbAG" role="doc-biblioref">994</a>)</span>.
This pattern applied only to first-in-class drugs, with most follower drugs produced by target-based screening <span class="citation" data-cites="WhCrTFgg">(<a href="#ref-WhCrTFgg" role="doc-biblioref">952</a>)</span>.
These findings suggest that target-based drug discovery is more successful when building on a known MMOA, and that modulating a target is most valuable when the target is part of a valuable MMOA <span class="citation" data-cites="qSXkcV36">(<a href="#ref-qSXkcV36" role="doc-biblioref">953</a>)</span>.
Building on this, many within the field suggested that mechanism-informed phenotypic investigations may be the most useful approach to drug discovery <span class="citation" data-cites="qSXkcV36 LSRYjaxD 140r2Y6Ti">(<a href="#ref-140r2Y6Ti" role="doc-biblioref">951</a>, <a href="#ref-qSXkcV36" role="doc-biblioref">953</a>, <a href="#ref-LSRYjaxD" role="doc-biblioref">955</a>)</span>.
As it stands, data-driven efforts to identify patterns in the results of computational screens allowed researchers to notice the shared property of cationic amphicility among many of the hits from computational screening analyses <span class="citation" data-cites="j2enN2JO">(<a href="#ref-j2enN2JO" role="doc-biblioref">982</a>)</span>.
While easier said than done, efforts to fill in the black box underlying computational HTS and recognize patterns among the identified compounds aid in moving data-oriented drug repurposing efforts in this direction.</p>
<p>The unpredictable nature of success and failure in drug repurposing for COVID-19 thus highlights one of the tenets of phenotypic screening: there are a lot of “unknown unknowns”, and a promising mechanism at the level of an MMOA will not necessarily propagate up to the pathway, cellular, or organismal level <span class="citation" data-cites="140r2Y6Ti">(<a href="#ref-140r2Y6Ti" role="doc-biblioref">951</a>)</span>.
Despite the fact that apparently mechanistically relevant drugs may exist, identifying effective treatments for a new viral disease is extremely challenging.
Targets of repurposed drugs are often non-specific, meaning that the MMOA can appear to be relevant to COVID-19 without a therapeutic or prophylactic effect being observed in clinical trials.
The difference in the current status of remdesivir and favipiravir as treatments for COVID-19 (Table <a href="#tbl:summary-table">1</a>) underscores how difficult to predict whether a specific compound will produce a desired effect, even when the mechanisms are similar.
Furthermore, the fact that many candidate COVID-19 therapeutics were ultimately identified because of their shared propensity to induce phospholipidosis underscores how challenging it can be to identify a mechanism <em>in silico</em> or <em>in vitro</em> that will translate to a successful treatment.
While significant progress has been made thus far in the pandemic, the therapeutic landscape is likely to continue to evolve as more results become available from clinical trials and as efforts to develop novel therapeutics for COVID-19 progress.</p>
<h2 data-number="4.9" id="towards-the-next-hcov-threat"><span class="header-section-number">4.9</span> Towards the Next HCoV Threat</h2>
<p>Only very limited testing of candidate therapies was feasible during the SARS and MERS epidemics, and as a result, few treatments were available at the outset of the COVID-19 pandemic.
Even corticosteroids, which were used to treat SARS patients, were a controversial therapeutic prior to the release of the results of the large RECOVERY trial.
The scale and duration of the COVID-19 pandemic has made it possible to conduct large, rigorous RCTs such as RECOVERY, Solidarity, TOGETHER, and others.
As results from these trials have continued to emerge, it has become clear that small clinical trials often produce spurious results.
In the case of HCQ/CQ, the therapeutic had already attracted so much attention based on small, preliminary (and in some cases, methodologically concerning) studies that it took the results of multiple large studies before attention began to be redirected to more promising candidates <span class="citation" data-cites="RfR64JTd">(<a href="#ref-RfR64JTd" role="doc-biblioref">995</a>)</span>.
In fact, most COVID-19 clinical trials lack the statistical power to reliably test their hypotheses <span class="citation" data-cites="WwRJN5u3 yyBcr7R9">(<a href="#ref-WwRJN5u3" role="doc-biblioref">996</a>, <a href="#ref-yyBcr7R9" role="doc-biblioref">997</a>)</span>.
In the face of an urgent crisis like COVID-19, the desire to act quickly is understandable, but it is imperative that studies maintain strict standards of scientific rigor <span class="citation" data-cites="ugC5dX0E SotNtXcx">(<a href="#ref-SotNtXcx" role="doc-biblioref">948</a>, <a href="#ref-ugC5dX0E" role="doc-biblioref">986</a>)</span>, especially given the potential dangers of politicization, as illustrated by HCQ/CQ <span class="citation" data-cites="J9N2GoTR">(<a href="#ref-J9N2GoTR" role="doc-biblioref">998</a>)</span>.
Potential innovations in clinical trial structure, such as adaptable clinical trials with master protocols <span class="citation" data-cites="XB0dP3L6">(<a href="#ref-XB0dP3L6" role="doc-biblioref">999</a>)</span> or the sharing of data among small clinical trials <span class="citation" data-cites="yyBcr7R9">(<a href="#ref-yyBcr7R9" role="doc-biblioref">997</a>)</span> may help to address future crises and to bolster the results from smaller studies, respectively.</p>
<p>In the long-term, new drugs specific for treatment of COVID-19 may also enter development.
Development of novel drugs is likely to be guided by what is known about the pathogenesis and molecular structure of SARS-CoV-2.
For example, understanding the various structural components of SARS-CoV-2 may allow for the development of small molecule inhibitors of those components.
Crystal structures of the SARS-CoV-2 main protease have been resolved <span class="citation" data-cites="4Sja6dIz AJLaaguT">(<a href="#ref-4Sja6dIz" role="doc-biblioref">961</a>, <a href="#ref-AJLaaguT" role="doc-biblioref">1000</a>)</span>.
Much work remains to be done to determine further crystal structures of other viral components, understand the relative utility of targeting different viral components, perform additional small molecule inhibitor screens, and determine the safety and efficacy of the potential inhibitors.
While still nascent, work in this area is promising.
Over the longer term, this approach and others may lead to the development of novel therapeutics specifically for COVID-19 and SARS-CoV-2.
Such efforts are likely to prove valuable in managing future emergent HCoV, just as research from the SARS and MERS pandemic has provided a basis for the COVID-19 response.</p>
<h1 data-number="5" id="appendix-identification-and-development-of-therapeutics-for-covid-19"><span class="header-section-number">5</span> Appendix: Identification and Development of Therapeutics for COVID-19</h1>
<h2 data-number="5.1" id="dexamethasone"><span class="header-section-number">5.1</span> Dexamethasone</h2>
<p>In order to understand how dexamethasone reduces inflammation, it is necessary to consider the stress response broadly.
In response to stress, corticotropin‐releasing hormone stimulates the release of neurotransmitters known as catecholamines, such as epinephrine, and steroid hormones known as glucocorticoids, such as cortisol <span class="citation" data-cites="jMzODpwS Riev3em1">(<a href="#ref-jMzODpwS" role="doc-biblioref">1001</a>, <a href="#ref-Riev3em1" role="doc-biblioref">1002</a>)</span>.
While catecholamines are often associated with the fight-or-flight response, the specific role that glucocorticoids play is less clear, although they are thought to be important to restoring homeostasis <span class="citation" data-cites="JZyVNRCN">(<a href="#ref-JZyVNRCN" role="doc-biblioref">1003</a>)</span>.
Immune challenge is a stressor that is known to interact closely with the stress response.
The immune system can therefore interact with the central nervous system; for example, macrophages can both respond to and produce catecholamines <span class="citation" data-cites="jMzODpwS">(<a href="#ref-jMzODpwS" role="doc-biblioref">1001</a>)</span>.
Additionally, the production of both catecholamines and glucocorticoids is associated with inhibition of proinflammatory cytokines such as IL-6, IL-12, and tumor necrosis factor-α (TNF‐α) and the stimulation of anti-inflammatory cytokines such as IL-10, meaning that the stress response can regulate inflammatory immune activity <span class="citation" data-cites="Riev3em1">(<a href="#ref-Riev3em1" role="doc-biblioref">1002</a>)</span>.
Administration of dexamethasone has been found to correspond to dose-dependent inhibition of IL-12 production, but not to affect IL-10 <span class="citation" data-cites="SiXrQUgV">(<a href="#ref-SiXrQUgV" role="doc-biblioref">1004</a>)</span>; the fact that this relationship could be disrupted by administration of a glucocorticoid-receptor antagonist suggests that it is regulated by the receptor itself <span class="citation" data-cites="SiXrQUgV">(<a href="#ref-SiXrQUgV" role="doc-biblioref">1004</a>)</span>.
Thus, the administration of dexamethasone for COVID-19 is likely to simulate the release of glucocorticoids endogenously during stress, resulting in binding of the synthetic steroid to the glucocorticoid receptor and the associated inhibition of the production of proinflammatory cytokines.
In this model, dexamethasone reduces inflammation by stimulating the biological mechanism that reduces inflammation following a threat such as immune challenge.</p>
<p>Initial support for dexamethasone as a treatment for COVID-19 came from the United Kingdom’s RECOVERY trial <span class="citation" data-cites="pzQoUwz3">(<a href="#ref-pzQoUwz3" role="doc-biblioref">780</a>)</span>, which assigned over 6,000 hospitalized COVID-19 patients to the standard of care (SOC) or treatment (dexamethasone) arms of the trial at a 2:1 ratio.
At the time of randomization, some patients were ventilated (16%), others were on non-invasive oxygen (60%), and others were breathing independently (24%).
Patients in the treatment arm were administered dexamethasone either orally or intravenously at 6 mg per day for up to 10 days.
The primary end-point was the patient’s status at 28-days post-randomization (mortality, discharge, or continued hospitalization), and secondary outcomes analyzed included the progression to invasive mechanical ventilation over the same period.
The 28-day mortality rate was found to be lower in the treatment group than in the SOC group (21.6% vs 24.6%, <em>p</em> &lt; 0.001).
However, the effect was driven by improvements in patients receiving mechanical ventilation or supplementary oxygen.
One possible confounder is that patients receiving mechanical ventilation tended to be younger than patients who were not receiving respiratory support (by 10 years on average) and to have had symptoms for a longer period.
However, adjusting for age did not change the conclusions, although the duration of symptoms was found to be significantly associated with the effect of dexamethasone administration.
Thus, this large, randomized, and multi-site, albeit not placebo-controlled, study suggests that administration of dexamethasone to patients who are unable to breathe independently may significantly improve survival outcomes.
Additionally, dexamethasone is a widely available and affordable medication, raising the hope that it could be made available to COVID-19 patients globally.</p>
<p>It is not surprising that administration of an immunosuppressant would be most beneficial in severe cases where the immune system was dysregulated towards inflammation.
However, it is also unsurprising that care must be taken in administering an immunosuppressant to patients fighting a viral infection.
In particular, the concern has been raised that treatment with dexamethasone might increase patient susceptibility to concurrent (e.g., nosocomial) infections <span class="citation" data-cites="EgvIETTq">(<a href="#ref-EgvIETTq" role="doc-biblioref">1005</a>)</span>.
Additionally, the drug could potentially slow viral clearance and inhibit patients’ ability to develop antibodies to SARS-CoV-2 <span class="citation" data-cites="EgvIETTq Ji2fchyS">(<a href="#ref-Ji2fchyS" role="doc-biblioref">777</a>, <a href="#ref-EgvIETTq" role="doc-biblioref">1005</a>)</span>, with the lack of data about viral clearance being put forward as a major limitation of the RECOVERY trial <span class="citation" data-cites="TytQ90Bl">(<a href="#ref-TytQ90Bl" role="doc-biblioref">1006</a>)</span>.
Furthermore, dexamethasone has been associated with side effects that include psychosis, glucocorticoid-induced diabetes, and avascular necrosis <span class="citation" data-cites="Ji2fchyS">(<a href="#ref-Ji2fchyS" role="doc-biblioref">777</a>)</span>, and the RECOVERY trial did not report outcomes with enough detail to be able to determine whether they observed similar complications.
The effects of dexamethasone have also been found to differ among populations, especially in high-income versus middle- or low-income countries <span class="citation" data-cites="1E4Qx2UqB">(<a href="#ref-1E4Qx2UqB" role="doc-biblioref">1007</a>)</span>.
However, since the RECOVERY trial’s results were released, strategies have been proposed for administering dexamethasone alongside more targeted treatments to minimize the likelihood of negative side effects <span class="citation" data-cites="EgvIETTq">(<a href="#ref-EgvIETTq" role="doc-biblioref">1005</a>)</span>.
Given the available evidence, dexamethasone is currently the most promising treatment for severe COVID-19.</p>
<h2 data-number="5.2" id="favipiravir"><span class="header-section-number">5.2</span> Favipiravir</h2>
<p>The effectiveness of favipiravir for treating patients with COVID-19 is currently under investigation.
Evidence for the drug inhibiting viral RNA polymerase are based on time-of-drug addition studies that found that viral loads were reduced with the addition of favipiravir in early times post-infection <span class="citation" data-cites="DpdKJbki 7d1TnpE3 15kQ4QYbx">(<a href="#ref-DpdKJbki" role="doc-biblioref">798</a>, <a href="#ref-7d1TnpE3" role="doc-biblioref">801</a>, <a href="#ref-15kQ4QYbx" role="doc-biblioref">802</a>)</span>.
An open-label, nonrandomized, before-after controlled study for COVID-19 was recently conducted <span class="citation" data-cites="13Oz7yRPx">(<a href="#ref-13Oz7yRPx" role="doc-biblioref">1008</a>)</span>.
The study included 80 COVID-19 patients (35 treated with favipiravir, 45 control) from the isolation ward of the National Clinical Research Center for Infectious Diseases (The Third People’s Hospital of Shenzhen), Shenzhen, China.
The patients in the control group were treated with other antivirals, such as lopinavir and ritonavir.
It should be noted that although the control patients received antivirals, two subsequent large-scale analyses, the WHO Solidarity trial and the Randomized Evaluation of COVID-19 Therapy (RECOVERY) trial, identified no effect of lopinavir or of a lopinavir-ritonavir combination, respectively, on the metrics of COVID-19-related mortality that each assessed <span class="citation" data-cites="jShm9uhy fTWlYlvp xM0IhxIr">(<a href="#ref-jShm9uhy" role="doc-biblioref">811</a>, <a href="#ref-fTWlYlvp" role="doc-biblioref">1009</a>, <a href="#ref-xM0IhxIr" role="doc-biblioref">1010</a>)</span>.
Treatment was applied on days 2-14; treatment stopped either when viral clearance was confirmed or at day 14.
The efficacy of the treatment was measured by, first, the time until viral clearance using Kaplan-Meier survival curves, and, second, the improvement rate of chest computed tomography (CT) scans on day 14 after treatment.
The study found that favipiravir increased the speed of recovery, measured as viral clearance from the patient by RT-PCR, with patients receiving favipiravir recovering in four days compared to 11 days for patients receiving antivirals such as lopinavir and ritonavir.
Additionally, the lung CT scans of patients treated with favipiravir showed significantly higher improvement rates (91%) on day 14 compared to control patients (62%, <em>p</em> = 0.004).
However, there were adverse side effects in 4 (11%) favipiravir-treated patients and 25 (56%) control patients.
The adverse side effects included diarrhea, vomiting, nausea, rash, and liver and kidney injury.
Despite the study reporting clinical improvement in favipiravir-treated patients, several study design issues are problematic and lower confidence in the overall conclusions.
For example, the study was neither randomized nor blinded.
Moreover, the selection of patients did not take into consideration important factors such as previous clinical conditions or sex, and there was no age categorization.
Additionally, it should be noted that this study was temporarily retracted and then restored without an explanation <span class="citation" data-cites="hfAF6aDr">(<a href="#ref-hfAF6aDr" role="doc-biblioref">1011</a>)</span>.</p>
<p>In late 2020 and early 2021, the first randomized controlled trials of favipiravir for the treatment of COVID-19 released results <span class="citation" data-cites="APU0cUZz 15rGJX7Hl 32n9UEkQ">(<a href="#ref-APU0cUZz" role="doc-biblioref">1012</a>–<a href="#ref-32n9UEkQ" role="doc-biblioref">1014</a>)</span>.
One study <span class="citation" data-cites="15rGJX7Hl">(<a href="#ref-15rGJX7Hl" role="doc-biblioref">1013</a>)</span> was retracted in November 2021 due to concerns about the data.
Of the two remaining, the first <span class="citation" data-cites="APU0cUZz">(<a href="#ref-APU0cUZz" role="doc-biblioref">1012</a>)</span> used a randomized, controlled, open-label design to compare two drugs, favipiravir and baloxavir marboxil, to SOC alone.
Here, SOC included antivirals such as lopinavir/ritonavir and was administered to all patients.
The primary endpoint analyzed was viral clearance at day 14.
The sample size for this study was very small, with 29 total patients enrolled, and no significant effect of the treatments was found for the primary or any of the secondary outcomes analyzed, which included mortality.
The second trial examined 60 patients and reported a significant effect of favipiravir on viral clearance at four days (a secondary endpoint), but not at 10 days (the primary endpoint) <span class="citation" data-cites="32n9UEkQ">(<a href="#ref-32n9UEkQ" role="doc-biblioref">1014</a>)</span>.
This study, as well as a prior study of favipiravir <span class="citation" data-cites="qwlYY6jn">(<a href="#ref-qwlYY6jn" role="doc-biblioref">1015</a>)</span>, also reported that the drug was generally well-tolerated.
Thus, in combination, these small studies suggest that the effects of favipiravir as a treatment for COVID-19 cannot be determined based on the available evidence, but additionally, none raise major concerns about the safety profile of the drug.</p>
<h2 data-number="5.3" id="remdesivir"><span class="header-section-number">5.3</span> Remdesivir</h2>
<p>At the outset of the COVID-19 pandemic, remdesivir did not have any have any FDA-approved use.
A clinical trial in the Democratic Republic of Congo found some evidence of effectiveness against ebola virus disease (EVD), but two antibody preparations were found to be more effective, and remdesivir was not pursued <span class="citation" data-cites="gnfviuGp">(<a href="#ref-gnfviuGp" role="doc-biblioref">1016</a>)</span>.
Remdesivir also inhibits polymerase and replication of the coronaviruses MERS-CoV and SARS-CoV-1 in cell culture assays with submicromolar IC50s <span class="citation" data-cites="1H0NPTMw9">(<a href="#ref-1H0NPTMw9" role="doc-biblioref">1017</a>)</span>.
It has also been found to inhibit SARS-CoV-2, showing synergy with CQ <em>in vitro</em> <span class="citation" data-cites="IhdcvbzC">(<a href="#ref-IhdcvbzC" role="doc-biblioref">806</a>)</span>.</p>
<p>Remdesivir was first used on some COVID-19 patients under compassionate use guidelines <span class="citation" data-cites="D4HwvRMd zoLViMNg">(<a href="#ref-gleq16qf" role="doc-biblioref">1020</a>)</span>.
All were in late stages of COVID-19 infection, and initial reports were inconclusive about the drug’s efficacy.
Gilead Sciences, the maker of remdesivir, led a recent publication that reported outcomes for compassionate use of the drug in 61 patients hospitalized with confirmed COVID-19.
Here, 200 mg of remdesivir was administered intravenously on day 1, followed by a further 100 mg/day for 9 days <span class="citation" data-cites="16zLDIOyk">(<a href="#ref-16zLDIOyk" role="doc-biblioref">810</a>)</span>.
There were significant issues with the study design, or lack thereof.
There was no randomized control group.
The inclusion criteria were variable: some patients only required low doses of oxygen, while others required ventilation.
The study included many sites, potentially with variable inclusion criteria and treatment protocols.
The patients analyzed had mixed demographics.
There was a short follow-up period of investigation.
Eight patients were excluded from the analysis mainly due to missing post-baseline information; thus, their health was unaccounted for.
Therefore, even though the study reported clinical improvement in 68% of the 53 patients ultimately evaluated, due to the significant issues with study design, it could not be determined whether treatment with remdesivir had an effect or whether these patients would have recovered regardless of treatment.
Another study comparing 5- and 10-day treatment regimens reported similar results but was also limited because of the lack of a placebo control <span class="citation" data-cites="w5y1LWUR">(<a href="#ref-w5y1LWUR" role="doc-biblioref">1021</a>)</span>.
These studies did not alter the understanding of the efficacy of remdesivir in treating COVID-19, but the encouraging results provided motivation for placebo-controlled studies.</p>
<p>The double-blind placebo-controlled ACTT-1 trial <span class="citation" data-cites="DtpdBrgt 1ATTPbCCO">(<a href="#ref-DtpdBrgt" role="doc-biblioref">807</a>, <a href="#ref-1ATTPbCCO" role="doc-biblioref">808</a>)</span> recruited 1,062 patients and randomly assigned them to placebo treatment or treatment with remdesivir.
Patients were stratified for randomization based on site and the severity of disease presentation at baseline <span class="citation" data-cites="DtpdBrgt">(<a href="#ref-DtpdBrgt" role="doc-biblioref">807</a>)</span>.
The treatment was 200 mg on day 1, followed by 100 mg on days 2 through 10.
Data was analyzed from a total of 1,059 patients who completed the 29-day course of the trial, with 517 assigned to remdesivir and 508 to placebo <span class="citation" data-cites="DtpdBrgt">(<a href="#ref-DtpdBrgt" role="doc-biblioref">807</a>)</span>.
The two groups were well matched demographically and clinically at baseline.
Those who received remdesivir had a median recovery time of 10 days, as compared with 15 days in those who received placebo (rate ratio for recovery, 1.29; 95% confidence interval (CI), 1.12 to 1.49; <em>p</em> &lt; 0.001).
The Kaplan-Meier estimates of mortality by 14 days were 6.7% with remdesivir and 11.9% with placebo, with a hazard ratio (HR) for death of 0.55 and a 95% CI of 0.36 to 0.83, and at day 29, remdesivir corresponded to 11.4% and the placebo to 15.2% (HR: 0.73; 95% CI, 0.52 to 1.03).
Serious adverse events were reported in 131 of the 532 patients who received remdesivir (24.6%) and in 163 of the 516 patients in the placebo group (31.6%).
This study also reported an association between remdesivir administration and both clinical improvement and a lack of progression to more invasive respiratory intervention in patients receiving non-invasive and invasive ventilation at randomization <span class="citation" data-cites="DtpdBrgt">(<a href="#ref-DtpdBrgt" role="doc-biblioref">807</a>)</span>.
Largely on the results of this trial, the FDA reissued and expanded the EUA for remdesivir for the treatment of hospitalized COVID-19 patients ages twelve and older <span class="citation" data-cites="1BYoe0y2H">(<a href="#ref-1BYoe0y2H" role="doc-biblioref">1022</a>)</span>.
Additional clinical trials <span class="citation" data-cites="IhdcvbzC oso9r21L 1q4PmqTm rUTsIlZs JSoOOX98">(<a href="#ref-IhdcvbzC" role="doc-biblioref">806</a>, <a href="#ref-oso9r21L" role="doc-biblioref">1023</a>–<a href="#ref-JSoOOX98" role="doc-biblioref">1026</a>)</span> are currently underway to evaluate the use of remdesivir to treat COVID-19 patients at both early and late stages of infection and in combination with other drugs (Figure <a href="#fig:ebm-trials">4</a>).
As of October 22, 2020, remdesivir received FDA approval based on three clinical trials <span class="citation" data-cites="1DTFVDWTn">(<a href="#ref-1DTFVDWTn" role="doc-biblioref">1027</a>)</span>.</p>
<p>However, results suggesting no effect of remdesivir on survival were reported by the WHO Solidarity trial <span class="citation" data-cites="jShm9uhy">(<a href="#ref-jShm9uhy" role="doc-biblioref">811</a>)</span>.
Patients were randomized in equal proportions into four experimental conditions and a control condition, corresponding to four candidate treatments for COVID-19 and SOC, respectively; no placebo was administered.
The 2,750 patients in the remdesivir group were administered 200 mg intravenously on the first day and 100 mg on each subsequent day until day 10 and assessed for in-hospital death (primary endpoint), duration of hospitalization, and progression to mechanical ventilation.
There were also 2,708 control patients who would have been eligible and able to receive remdesivir were they not assigned to the control group.
A total of 604 patients among these two cohorts died during initial hospitalization, with 301 in the remdesivir group and 303 in the control group.
The rate ratio of death between these two groups was therefore not significant (0.95, <em>p</em> = 0.50), suggesting that the administration of remdesivir did not affect survival.
The two secondary analyses similarly did not find any effect of remdesivir.
Additionally, the authors compared data from their study with data from three other studies of remdesivir (including <span class="citation" data-cites="DtpdBrgt">(<a href="#ref-DtpdBrgt" role="doc-biblioref">807</a>)</span>) stratified by supplemental oxygen status.
A meta-analysis of the four studies yielded an overall rate ratio for death of 0.91 (<em>p</em> = 0.20).
These results thus do not support the previous findings that remdesivir reduced median recovery time and mortality risk in COVID-19 patients.</p>
<p>In response to the results of the Solidarity trial, Gilead, which manufactures remdesivir, released a statement pointing to the fact that the Solidarity trial was not placebo-controlled or double-blind and at the time of release, the statement had not been peer reviewed <span class="citation" data-cites="zQT9JLP1">(<a href="#ref-zQT9JLP1" role="doc-biblioref">1028</a>)</span>; these sentiments have been echoed elsewhere <span class="citation" data-cites="vJVD0Wle">(<a href="#ref-vJVD0Wle" role="doc-biblioref">1029</a>)</span>.
Other critiques of this study have noted that antivirals are not typically targeted at patients with severe illness, and therefore remdesivir could be more beneficial for patients with mild rather than severe cases <span class="citation" data-cites="pQkqi9ni xM0IhxIr">(<a href="#ref-xM0IhxIr" role="doc-biblioref">1010</a>, <a href="#ref-pQkqi9ni" role="doc-biblioref">1030</a>)</span>.
However, the publication associated with the trial sponsored by Gilead did purport an effect of remdesivir on patients with severe disease, identifying an 11 versus 18 day recovery period (rate ratio for recovery: 1.31, 95% CI 1.12 to 1.52) <span class="citation" data-cites="DtpdBrgt">(<a href="#ref-DtpdBrgt" role="doc-biblioref">807</a>)</span>.
Additionally, a smaller analysis of 598 patients, of whom two-thirds were randomized to receive remdesivir for either 5 or 10 days, reported a small effect of treatment with remdesivir for five days relative to standard of care in patients with moderate COVID-19 <span class="citation" data-cites="6NLZTSS0">(<a href="#ref-6NLZTSS0" role="doc-biblioref">1031</a>)</span>.
These results suggest that remdesivir could improve outcomes for patients with moderate COVID-19, but that additional information would be needed to understand the effects of different durations of treatment.
Therefore, the Solidarity trial may point to limitations in the generalizability of other research on remdesivir, especially since the broad international nature of the Solidarity clinical trial, which included countries with a wide range of economic profiles and a variety of healthcare systems, provides a much-needed global perspective in a pandemic <span class="citation" data-cites="xM0IhxIr">(<a href="#ref-xM0IhxIr" role="doc-biblioref">1010</a>)</span>.
On the other hand, only 62% of patients in the Solidarity trial were randomized on the day of admission or one day afterwards <span class="citation" data-cites="jShm9uhy">(<a href="#ref-jShm9uhy" role="doc-biblioref">811</a>)</span>, and concerns have been raised that differences in disease progression could influence the effectiveness of remdesivir <span class="citation" data-cites="xM0IhxIr">(<a href="#ref-xM0IhxIr" role="doc-biblioref">1010</a>)</span>.
Despite the findings of the Solidarity trial, remdesivir remains available for the treatment of COVID-19 in many places.
Remdesivir has also been investigated in combination with other drugs, such as baricitinib, which is an inhibitor of Janus kinase 1 and 2 <span class="citation" data-cites="pDl6Q4UP">(<a href="#ref-pDl6Q4UP" role="doc-biblioref">1032</a>)</span>; the FDA has issued an EUA for the combination of remdesivir and baricitinib in adult and pediatric patients <span class="citation" data-cites="1HiXNTn3b">(<a href="#ref-1HiXNTn3b" role="doc-biblioref">1033</a>)</span>.
Follow-up studies are needed and, in many cases, are underway to further investigate remdesivir-related outcomes.</p>
<p>Similarly, the extent to which the remdesivir dosing regimen could influence outcomes continues to be under consideration.
A randomized, open-label trial compared the effect of remdesivir on 397 patients with severe COVID-19 over 5 versus 10 days <span class="citation" data-cites="w5y1LWUR yTCAmOyt">(<a href="#ref-yTCAmOyt" role="doc-biblioref">809</a>, <a href="#ref-w5y1LWUR" role="doc-biblioref">1021</a>)</span>, complementing the study that found that a 5-day course of remdesivir improved outcomes for patients with moderate COVID-19 but a 10-day course did not <span class="citation" data-cites="6NLZTSS0">(<a href="#ref-6NLZTSS0" role="doc-biblioref">1031</a>)</span>.
Patients in the two groups were administered 200 mg of remdesivir intravenously on the first day, followed by 100 mg on the subsequent four or nine days, respectively.
The two groups differed significantly in their clinical status, with patients assigned to the 10-day group having more severe illness.
This study also differed from most because it included not only adults, but also pediatric patients as young as 12 years old.
It reported no significant differences across several outcomes for patients receiving a 5-day or 10-day course, when correcting for baseline clinical status.
The data did suggest that the 10-day course might reduce mortality in the most severe patients at day 14, but the representation of this group in the study population was too low to justify any conclusions <span class="citation" data-cites="w5y1LWUR">(<a href="#ref-w5y1LWUR" role="doc-biblioref">1021</a>)</span>.
Thus, additional research is also required to determine whether the dosage and duration of remdesivir administration influences outcomes.</p>
<p>In summary, remdesivir is the first FDA approved anti-viral against SARS-CoV-2 as well as the first FDA approved COVID-19 treatment.
Early investigations of this drug established proof of principle that drugs targeting the virus can benefit COVID-19 patients.
Moreover, one of the most successful strategies for developing therapeutics for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.
Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including human immunodeficiency virus (HIV) and herpes.
Notably, the HIV and herpes polymerases are a reverse transcriptase and a DNA polymerase, respectively, whereas SARS-CoV-2 encodes an RdRP, so most of the commonly used polymerase inhibitors are not likely to be active against SARS-CoV-2.
In clinical use, polymerase inhibitors show short term benefits for HIV patients, but for long term benefits they must be part of combination regimens.
They are typically combined with protease inhibitors, integrase inhibitors, and even other polymerase inhibitors.
Remdesivir provides evidence that a related approach may be beneficial for the treatment of COVID-19.</p>
<h2 data-number="5.4" id="hydroxychloroquine-and-chloroquine"><span class="header-section-number">5.4</span> Hydroxychloroquine and Chloroquine</h2>
<p>CQ and hydroxychloroquine (HCQ) increase cellular pH by accumulating in their protonated form inside lysosomes <span class="citation" data-cites="ggYRK4FU 16IRfGFKb">(<a href="#ref-16IRfGFKb" role="doc-biblioref">813</a>, <a href="#ref-ggYRK4FU" role="doc-biblioref">1034</a>)</span>.
This shift in pH inhibits the breakdown of proteins and peptides by the lysosomes during the process of proteolysis <span class="citation" data-cites="16IRfGFKb">(<a href="#ref-16IRfGFKb" role="doc-biblioref">813</a>)</span>.
Interest in CQ and HCQ for treating COVID-19 was catalyzed by a mechanism observed in <em>in vitro</em> studies of both SARS-CoV-1 and SARS-CoV-2.
In one study, CQ inhibited viral entry of SARS-CoV-1 into Vero E6 cells, a cell line that was derived from Vero cells in 1968, through the elevation of endosomal pH and the terminal glycosylation of ACE2 <span class="citation" data-cites="x6nInyCf">(<a href="#ref-x6nInyCf" role="doc-biblioref">814</a>)</span>.
Increased pH within the cell, as discussed above, inhibits proteolysis, and terminal glycosylation of ACE2 is thought to interfere with virus-receptor binding.
An <em>in vitro</em> study of SARS-CoV-2 infection of Vero cells found both HCQ and CQ to be effective in inhibiting viral replication, with HCQ being more potent <span class="citation" data-cites="1GBsnkeLK">(<a href="#ref-1GBsnkeLK" role="doc-biblioref">815</a>)</span>.
Additionally, an early case study of three COVID-19 patients reported the presence of antiphospholipid antibodies in all three patients <span class="citation" data-cites="8OnbWuhF">(<a href="#ref-8OnbWuhF" role="doc-biblioref">100</a>)</span>.
Antiphospholipid antibodies are central to the diagnosis of the antiphospholipid syndrome, a disorder that HCQ has often been used to treat <span class="citation" data-cites="nEaG534R rL8owM5Y 134vCyU3E">(<a href="#ref-nEaG534R" role="doc-biblioref">1035</a>–<a href="#ref-134vCyU3E" role="doc-biblioref">1037</a>)</span>.
Because the 90% effective concentration (EC<sub>90</sub>) of CQ in Vero E6 cells (6.90 μM) can be achieved in and tolerated by rheumatoid arthritis (RA) patients, it was hypothesized that it might also be possible to achieve the effective concentration in COVID-19 patients <span class="citation" data-cites="WFGcbZhe">(<a href="#ref-WFGcbZhe" role="doc-biblioref">1038</a>)</span>.
Additionally, clinical trials have reported HCQ to be effective in treating HIV <span class="citation" data-cites="hFGjQQN8">(<a href="#ref-hFGjQQN8" role="doc-biblioref">1039</a>)</span> and chronic Hepatitis C <span class="citation" data-cites="rQNzKxRd">(<a href="#ref-rQNzKxRd" role="doc-biblioref">1040</a>)</span>.
Together, these studies triggered initial enthusiasm about the therapeutic potential for HCQ and CQ against COVID-19.
HCQ/CQ has been proposed both as a treatment for COVID-19 and a prophylaxis against SARS-CoV-2 exposure, and trials often investigated these drugs in combination with azithromycin (AZ) and/or zinc supplementation.
However, as more evidence has emerged, it has become clear that HCQ/CQ offer no benefits against SARS-CoV-2 or COVID-19.</p>
<h3 data-number="5.4.1" id="trials-assessing-therapeutic-administration-of-hcqcq"><span class="header-section-number">5.4.1</span> Trials Assessing Therapeutic Administration of HCQ/CQ</h3>
<p>The initial study evaluating HCQ as a treatment for COVID-19 patients was published on March 20, 2020 by Gautret et al. <span class="citation" data-cites="ovHLMvCi">(<a href="#ref-ovHLMvCi" role="doc-biblioref">816</a>)</span>.
This non-randomized, non-blinded, non-placebo clinical trial compared HCQ to SOC in 42 hospitalized patients in southern France.
It reported that patients who received HCQ showed higher rates of virological clearance by nasopharyngeal swab on days 3-6 when compared to SOC.
This study also treated six patients with both HCQ + AZ and found this combination therapy to be more effective than HCQ alone.
However, the design and analyses used showed weaknesses that severely limit interpretability of results, including the small sample size and the lack of: randomization, blinding, placebo (no “placebo pill” given to SOC group), Intention-To-Treat analysis, correction for sequential multiple comparisons, and trial pre-registration.
Furthermore, the trial arms were entirely confounded by the hospital and there were false negative outcome measurements (see <span class="citation" data-cites="36WPxcLu">(<a href="#ref-36WPxcLu" role="doc-biblioref">1041</a>)</span>).
Two of these weaknesses are due to inappropriate data analysis and can therefore be corrected <em>post hoc</em> by recalculating the p-values (lack of Intention-To-Treat analysis and multiple comparisons).
However, all other weaknesses are fundamental design flaws and cannot be corrected for.
Thus, the conclusions cannot be generalized outside of the study.
The International Society of Antimicrobial Chemotherapy, the scientific organization that publishes the journal where the article appeared, subsequently announced that the article did not meet its expected standard for publications <span class="citation" data-cites="10J3hBiMo">(<a href="#ref-10J3hBiMo" role="doc-biblioref">817</a>)</span>, although it has not been officially retracted.</p>
<p>Because of the preliminary data presented in this study, HCQ treatment was subsequently explored by other researchers.
About one week later, a follow-up case study reported that 11 consecutive patients were treated with HCQ + AZ using the same dosing regimen <span class="citation" data-cites="18KmvBOFr">(<a href="#ref-18KmvBOFr" role="doc-biblioref">1042</a>)</span>.
One patient died, two were transferred to the intensive care unit (ICU), and one developed a prolonged QT interval, leading to discontinuation of HCQ + AZ administration.
As in the Gautret et al. study, the outcome assessed was virological clearance at day 6 post-treatment, as measured from nasopharyngeal swabs.
Of the ten living patients on day 6, eight remained positive for SARS-CoV-2 RNA.
Like in the original study, interpretability was severely limited by the lack of a comparison group and the small sample size.
However, these results stand in contrast to the claims by Gautret et al. that all six patients treated with HCQ + AZ tested negative for SARS-CoV-2 RNA by day 6 post-treatment.
This case study illustrated the need for further investigation using robust study design to evaluate the efficacy of HCQ and/or CQ.</p>
<p>On April 10, 2020, a randomized, non-placebo trial of 62 COVID-19 patients at the Renmin Hospital of Wuhan University was released <span class="citation" data-cites="wzHb7Mj5">(<a href="#ref-wzHb7Mj5" role="doc-biblioref">1043</a>)</span>.
This study investigated whether HCQ decreased time to fever break or time to cough relief when compared to SOC <span class="citation" data-cites="wzHb7Mj5">(<a href="#ref-wzHb7Mj5" role="doc-biblioref">1043</a>)</span>.
This trial found HCQ decreased both average time to fever break and average time to cough relief, defined as mild or no cough.
While this study improved on some of the methodological flaws in Gautret et al. by randomizing patients, it also had several flaws in trial design and data analysis that prevent generalization of the results.
These weaknesses include the lack of placebo, lack of correction for multiple primary outcomes, inappropriate choice of outcomes, lack of sufficient detail to understand analysis, drastic disparities between pre-registration <span class="citation" data-cites="65ggO2DM">(<a href="#ref-65ggO2DM" role="doc-biblioref">1044</a>)</span> and published protocol (including differences in the inclusion and exclusion criteria, the number of experimental groups, the number of patients enrolled, and the outcome analyzed), and small sample size.
The choice of outcomes may be inappropriate as both fevers and cough may break periodically without resolution of illness.
Additionally, for these outcomes, the authors reported that 23 of 62 patients did not have a fever and 25 of 62 patients did not have a cough at the start of the study, but the authors failed to describe how these patients were included in a study assessing time to fever break and time to cough relief.
It is important to note here that the authors claimed “neither the research performers nor the patients were aware of the treatment assignments.”
This blinding seems impossible in a non-placebo trial because at the very least, providers would know whether they were administering a medication or not, and this knowledge could lead to systematic differences in the administration of care.
Correction for multiple primary outcomes can be adjusted <em>post hoc</em> by recalculating p-values, but all of the other issues were design and statistical weaknesses that cannot be corrected for.
Additionally, disparities between the pre-registered and published protocols raise concerns about experimental design.
The design limitations mean that the conclusions cannot be generalized outside of the study.</p>
<p>A second randomized trial, conducted by the Shanghai Public Health Clinical Center, analyzed whether HCQ increased rates of virological clearance at day 7 in respiratory pharyngeal swabs compared to SOC <span class="citation" data-cites="o5gwQA7W">(<a href="#ref-o5gwQA7W" role="doc-biblioref">1045</a>)</span>.
This trial was published in Chinese along with an abstract in English, and only the English abstract was read and interpreted for this review.
The trial found comparable outcomes in virological clearance rate, time to virological clearance, and time to body temperature normalization between the treatment and control groups.
The small sample size is one weakness, with only 30 patients enrolled and 15 in each arm.
This problem suggests the study is underpowered to detect potentially useful differences and precludes interpretation of results.
Additionally, because only the abstract could be read, other design and analysis issues could be present.
Thus, though these studies added randomization to their assessment of HCQ, their conclusions should be interpreted very cautiously.
These two studies assessed different outcomes and reached differing conclusions about the efficacy of HCQ for treating COVID-19; the designs of both studies, especially with respect to sample size, meant that no general conclusions can be made about the efficacy of the drug.</p>
<p>Several widely reported studies on HCQ also have issues with data integrity and/or provenance.
A Letter to the Editor published in <em>BioScience Trends</em> on March 16, 2020 claimed that numerous clinical trials have shown that HCQ is superior to control treatment in inhibiting the exacerbation of COVID-19 pneumonia <span class="citation" data-cites="aiItPxSY">(<a href="#ref-aiItPxSY" role="doc-biblioref">1046</a>)</span>.
This letter has been cited by numerous primary literature, review articles, and media alike <span class="citation" data-cites="1BdQKtXnY z8XnLYWM">(<a href="#ref-1BdQKtXnY" role="doc-biblioref">1047</a>, <a href="#ref-z8XnLYWM" role="doc-biblioref">1048</a>)</span>.
However, the letter referred to 15 pre-registration identifiers from the Chinese Clinical Trial Registry.
When these identifiers are followed back to the registry, most trials claim they are not yet recruiting patients or are currently recruiting patients.
For all of these 15 identifiers, no data uploads or links to publications could be located on the pre-registrations.
At the very least, the lack of availability of the primary data means the claim that HCQ is efficacious against COVID-19 pneumonia cannot be verified.
Similarly, a recent multinational registry analysis <span class="citation" data-cites="1BApBNHvd">(<a href="#ref-1BApBNHvd" role="doc-biblioref">1049</a>)</span> analyzed the efficacy of CQ and HCQ with and without a macrolide, which is a class of antibiotics that includes Azithromycin, for the treatment of COVID-19.
The study observed 96,032 patients split into a control and four treatment conditions (CQ with and without a macrolide; HCQ with and without a macrolide).
They concluded that treatment with CQ or HCQ was associated with increased risk of <em>de novo</em> ventricular arrhythmia during hospitalization.
However, this study has since been retracted by <em>The Lancet</em> due to an inability to validate the data used <span class="citation" data-cites="11FsxWalS">(<a href="#ref-11FsxWalS" role="doc-biblioref">1050</a>)</span>.
These studies demonstrate that increased skepticism in evaluation of the HCQ/CQ and COVID-19 literature may be warranted, possibly because of the significant attention HCQ and CQ have received as possible treatments for COVID-19 and the politicization of these drugs.</p>
<p>Despite the fact that the study suggesting that CQ/HCQ increased risk of ventricular arrhythmia in COVID-19 patients has now been retracted, previous studies have identified risks associated with HCQ/CQ.
A patient with systemic lupus erythematosus developed a prolonged QT interval that was likely exacerbated by use of HCQ in combination with renal failure <span class="citation" data-cites="dY9kXLAS">(<a href="#ref-dY9kXLAS" role="doc-biblioref">1051</a>)</span>.
A prolonged QT interval is associated with ventricular arrhythmia <span class="citation" data-cites="wy3S7Cpa">(<a href="#ref-wy3S7Cpa" role="doc-biblioref">1052</a>)</span>.
Furthermore, a separate study <span class="citation" data-cites="OQ0b31BA">(<a href="#ref-OQ0b31BA" role="doc-biblioref">1053</a>)</span> investigated the safety associated with the use of HCQ with and without macrolides between 2000 and 2020.
The study involved 900,000 cases treated with HCQ and 300,000 cases treated with HCQ + AZ.
The results indicated that short-term use of HCQ was not associated with additional risk, but that HCQ + AZ was associated with an enhanced risk of cardiovascular complications (such as a 15% increased risk of chest pain, calibrated HR = 1.15, 95% CI, 1.05 to 1.26) and a two-fold increased 30-day risk of cardiovascular mortality (calibrated HR = 2.19; 95% CI, 1.22 to 3.94).
Therefore, whether studies utilize HCQ alone or HCQ in combination with a macrolide may be an important consideration in assessing risk.
As results from initial investigations of these drug combinations have emerged, concerns about the efficacy and risks of treating COVID-19 with HCQ and CQ have led to the removal of CQ/HCQ from SOC practices in several countries <span class="citation" data-cites="1CgZR55GZ 1EcscrT32">(<a href="#ref-1CgZR55GZ" role="doc-biblioref">1054</a>, <a href="#ref-1EcscrT32" role="doc-biblioref">1055</a>)</span>.
As of May 25, 2020, WHO had suspended administration of HCQ as part of the worldwide Solidarity Trial <span class="citation" data-cites="AMQ5402v">(<a href="#ref-AMQ5402v" role="doc-biblioref">1056</a>)</span>, and later the final results of this large-scale trial that compared 947 patients administered HCQ to 906 controls revealed no effect on the primary outcome, mortality during hospitalization (rate ratio: 1.19; <em>p</em> = 0.23)</p>
<p>Additional research has emerged largely identifying HCQ/CQ to be ineffective against COVID-19 while simultaneously revealing a number of significant side effects.
A randomized, open-label, non-placebo trial of 150 COVID-19 patients was conducted in parallel at 16 government-designated COVID-19 centers in China to assess the safety and efficacy of HCQ <span class="citation" data-cites="RXr9BKKT">(<a href="#ref-RXr9BKKT" role="doc-biblioref">1057</a>)</span>.
The trial compared treatment with HCQ in conjunction with SOC to SOC alone in 150 infected patients who were assigned randomly to the two groups (75 per group).
The primary endpoint of the study was the negative conversion rate of SARS-CoV-2 in 28 days, and the investigators found no difference in this parameter between the groups (estimated difference between SOC plus HCQ and SOC 4.1%; 95% CI, –10.3% to 18.5%).
The secondary endpoints were an amelioration of the symptoms of the disease such as axillary temperature ≤36.6°C, SpO2 &gt;94% on room air, and disappearance of symptoms like shortness of breath, cough, and sore throat.
The median time to symptom alleviation was similar across different conditions (19 days in HCQ + SOC versus 21 days in SOC, <em>p</em> = 0.97).
Additionally, 30% of the patients receiving SOC+HCQ reported adverse outcomes compared to 8.8% of patients receiving only SOC, with the most common adverse outcome in the SOC+HCQ group being diarrhea (10% versus 0% in the SOC group, <em>p</em> = 0.004).
However, there are several factors that limit the interpretability of this study.
Most of the enrolled patients had mild-to-moderate symptoms (98%), and the average age was 46.
SOC in this study included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir, and Interferon alfa), which the authors note could influence the results.
Thus, they note that an ideal SOC would need to exclude the use of antivirals, but that ceasing antiviral treatment raised ethical concerns at the time that the study was conducted.
In this trial, the samples used to test for the presence of the SARS-CoV-2 virus were collected from the upper respiratory tract, and the authors indicated that the use of upper respiratory samples may have introduced false negatives (e.g., <span class="citation" data-cites="vr7AH83b">(<a href="#ref-vr7AH83b" role="doc-biblioref">71</a>)</span>).
Another limitation of the study that the authors acknowledge was that the HCQ treatment began, on average, at a 16-day delay from the symptom onset.
The fact that this study was open-label and lacked a placebo limits interpretation, and additional analysis is required to determine whether HCQ reduces inflammatory response.
Therefore, despite some potential areas of investigation identified in <em>post hoc</em> analysis, this study cannot be interpreted as providing support for HCQ as a therapeutic against COVID-19.
This study provided no support for HCQ against COVID-19, as there was no difference between the two groups in either negative seroconversion at 28 days or symptom alleviation, and in fact, more severe adverse outcomes were reported in the group receiving HCQ.</p>
<p>Additional evidence comes from a retrospective analysis <span class="citation" data-cites="MENbRfOl">(<a href="#ref-MENbRfOl" role="doc-biblioref">1058</a>)</span> that examined data from 368 COVID-19 patients across all United States Veteran Health Administration medical centers.
The study retrospectively investigated the effect of the administration of HCQ (n=97), HCQ + AZ (n=113), and no HCQ (n=158) on 368 patients.
The primary outcomes assessed were death and the need for mechanical ventilation.
Standard supportive care was rendered to all patients.
Due to the low representation of women (N=17) in the available data, the analysis included only men, and the median age was 65 years.
The rate of death was 27.8% in the HCQ-only treatment group, 22.1% in the HCQ + AZ treatment group, and 14.1% in the no-HCQ group.
These data indicated a statistically significant elevation in the risk of death for the HCQ-only group compared to the no-HCQ group (adjusted HR: 2.61, <em>p</em> = 0.03), but not for the HCQ + AZ group compared to the no-HCQ group (adjusted HR: 1.14; <em>p</em> = 0.72).
Further, the risk of ventilation was similar across all three groups (adjusted HR: 1.43, <em>p</em> = 0.48 (HCQ) and 0.43, <em>p</em> = 0.09 (HCQ + AZ) compared to no HCQ).
The study thus showed evidence of an association between increased mortality and HCQ in this cohort of COVID-19 patients but no change in rates of mechanical ventilation among the treatment conditions.
The study had a few limitations: it was not randomized, and the baseline vital signs, laboratory tests, and prescription drug use were significantly different among the three groups.
All of these factors could potentially influence treatment outcome.
Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not part of the study.
The reported result that HCQ + AZ is safer than HCQ contradicts the findings of the previous large-scale analysis of twenty years of records that found HCQ + AZ to be more frequently associated with cardiac arrhythmia than HCQ alone <span class="citation" data-cites="OQ0b31BA">(<a href="#ref-OQ0b31BA" role="doc-biblioref">1053</a>)</span>; whether this discrepancy is caused by the pathology of COVID-19, is influenced by age or sex, or is a statistical artifact is not presently known.</p>
<p>Finally, findings from the RECOVERY trial were released on October 8, 2020.
This study used a randomized, open-label design to study the effects of HCQ compared to SOC in 11,197 patients at 176 hospitals in the United Kingdom <span class="citation" data-cites="KwFZyfFx">(<a href="#ref-KwFZyfFx" role="doc-biblioref">818</a>)</span>.
Patients were randomized into either the control group or one of the treatment arms, with twice as many patients enrolled in the control group as any treatment group.
Of the patients eligible to receive HCQ, 1,561 were randomized into the HCQ arm, and 3,155 were randomized into the control arm.
The demographics of the HCQ and control groups were similar in terms of average age (65 years), proportion female (approximately 38%), ethnic make-up (73% versus 76% white), and prevalence of pre-existing conditions (56% versus 57% overall).
In the HCQ arm of the study, patients received 800 mg at baseline and again after 6 hours, then 400 mg at 12 hours and every subsequent 12 hours.
The primary outcome analyzed was all-cause mortality, and patient vital statistics were reported by physicians upon discharge or death, or else at 28 days following HCQ administration if they remained hospitalized.
The secondary outcome assessed was the combined risk of progression to invasive mechanical ventilation or death within 28 days.
By the advice of an external data monitoring committee, the HCQ arm of the study was reviewed early, leading to it being closed due a lack of support for HCQ as a treatment for COVID-19.
COVID-19-related mortality was not affected by HCQ in the RECOVERY trial (rate ratio, 1.09; 95% CI, 0.97 to 1.23; <em>p</em> = 0.15), but cardiac events were increased in the HCQ arm (0.4 percentage points), as was the duration of hospitalization (rate ratio for discharge alive within 28 days: 0.90; 95% CI, 0.83 to 0.98) and likelihood of progression to mechanical ventilation or death (risk ratio 1.14; 95% CI, 1.03 to 1.27).
This large-scale study thus builds upon studies in the United States and China to suggest that HCQ is not an effective treatment, and in fact may negatively impact COVID-19 patients due to its side effects.
Therefore, though none of the studies have been blinded, examining them together makes it clear that the available evidence points to significant dangers associated with the administration of HCQ to hospitalized COVID-19 patients, without providing any support for its efficacy.</p>
<h3 data-number="5.4.2" id="hcq-for-the-treatment-of-mild-cases"><span class="header-section-number">5.4.2</span> HCQ for the Treatment of Mild Cases</h3>
<p>One additional possible therapeutic application of HCQ considered was the treatment of mild COVID-19 cases in otherwise healthy individuals.
This possibility was assessed in a randomized, open-label, multi-center analysis conducted in Catalonia (Spain) <span class="citation" data-cites="V8fkkVZg">(<a href="#ref-V8fkkVZg" role="doc-biblioref">1059</a>)</span>.
This analysis enrolled adults 18 and older who had been experiencing mild symptoms of COVID-19 for fewer than five days.
Participants were randomized into an HCQ arm (N=136) and a control arm (N=157), and those in the treatment arm were administered 800 mg of HCQ on the first day of treatment followed by 400 mg on each of the subsequent six days.
The primary outcome assessed was viral clearance at days 3 and 7 following the onset of treatment, and secondary outcomes were clinical progression and time to complete resolution of symptoms.
No significant differences between the two groups were found: the difference in viral load between the HCQ and control groups was 0.01 (95% CI, -0.28 to 0.29) at day 3 and -0.07 (95% CI -0.44 to 0.29) at day 7, the relative risk of hospitalization was 0.75 (95% CI, 0.32 to 1.77), and the difference in time to complete resolution of symptoms was -2 days (<em>p</em> = 0.38).
This study thus suggests that HCQ does not improve recovery from COVID-19, even in otherwise healthy adult patients with mild symptoms.</p>
<h3 data-number="5.4.3" id="prophylactic-administration-of-hcq"><span class="header-section-number">5.4.3</span> Prophylactic Administration of HCQ</h3>
<p>An initial study of the possible prophylactic application of HCQ utilized a randomized, double-blind, placebo-controlled design to analyze the administration of HCQ prophylactically <span class="citation" data-cites="NiVdIN7Z">(<a href="#ref-NiVdIN7Z" role="doc-biblioref">1060</a>)</span>.
Asymptomatic adults in the United States and Canada who had been exposed to SARS-CoV-2 within the past four days were enrolled in an online study to evaluate whether administration of HCQ over five days influenced the probability of developing COVID-19 symptoms over a 14-day period.
Of the participants, 414 received HCQ and 407 received a placebo.
No significant difference in the rate of symptomatic illness was observed between the two groups (11.8% HCQ, 14.3% placebo, <em>p</em> = 0.35).
The HCQ condition was associated with side effects, with 40.1% of patients reporting side effects compared to 16.8% in the control group (<em>p</em> &lt; 0.001).
However, likely due to the high enrollment of healthcare workers (66% of participants) and the well-known side effects associated with HCQ, a large number of participants were able to correctly identify whether they were receiving HCQ or a placebo (46.5% and 35.7%, respectively).
Furthermore, due to a lack of availability of diagnostic testing, only 20 of the 107 cases were confirmed with a PCR-based test to be positive for SARS-CoV-2.
The rest were categorized as “probable” or “possible” cases by a panel of four physicians who were blind to the treatment status.
One possible confounder is that a patient presenting one or more symptoms, which included diarrhea, was defined as a “possible” case, but diarrhea is also a common side effect of HCQ.
Additionally, four of the twenty PCR-confirmed cases did not develop symptoms until after the observation period had completed, suggesting that the 14-day trial period may not have been long enough or that some participants also encountered secondary exposure events.
Finally, in addition to the young age of the participants in this study, which ranged from 32 to 51, there were possible impediments to generalization introduced by the selection process, as 2,237 patients who were eligible but had already developed symptoms by day 4 were enrolled in a separate study.
It is therefore likely that asymptomatic cases were over-represented in this sample, which would not have been detected based on the diagnostic criteria used.
Therefore, while this study does represent the first effort to conduct a randomized, double-blind, placebo-controlled investigation of HCQ’s effect on COVID-19 prevention after SARS-CoV-2 exposure in a large sample, the lack of PCR tests and several other design flaws significantly impede interpretation of the results.
However, in line with the results from therapeutic studies, once again no evidence was found suggesting an effect of HCQ against COVID-19.</p>
<p>A second study <span class="citation" data-cites="oaqFxV4W">(<a href="#ref-oaqFxV4W" role="doc-biblioref">1061</a>)</span> examined the effect of administering HCQ to healthcare workers as a pre-exposure prophylactic.
The primary outcome assessed was the conversion from SARS-CoV-2 negative to SARS-CoV-2 positive status over the 8 week study period.
This study was also randomized, double-blind, and placebo-controlled, and it sought to address some of the limitations of the first prophylactic study.
The goal was to enroll 200 healthcare workers, preferentially those working with COVID-19 patients, at two hospitals within the University of Pennsylvania hospital system in Philadelphia, PA.
Participants were randomized 1:1 to receive either 600 mg of HCQ daily or a placebo, and their SARS-CoV-2 infection status and antibody status were assessed using RT-PCR and serological testing, respectively, at baseline, 4 weeks, and 8 weeks following the beginning of the treatment period.
The statistical design of the study accounted for interim analyses at 50 and 100 participants in case efficacy or futility of HCQ for prophylaxis became clear earlier than completion of enrollment.
The 139 individuals enrolled comprised a study population that was fairly young (average age 33) and made of largely of people who were white, women, and without pre-existing conditions.
At the second interim analysis, more individuals in the treatment group than the control group had contracted COVID-19 (4 versus 3), causing the estimated z-score to fall below the pre-established threshold for futility.
As a result, the trial was terminated early, offering additional evidence against the use of HCQ for prophylaxis.</p>
<h3 data-number="5.4.4" id="summary-of-hcqcq-research-findings"><span class="header-section-number">5.4.4</span> Summary of HCQ/CQ Research Findings</h3>
<p>Early <em>in vitro</em> evidence indicated that HCQ could be an effective therapeutic against SARS-CoV-2 and COVID-19, leading to significant media attention and public interest in its potential as both a therapeutic and prophylactic.
Initially it was hypothesized that CQ/HCQ might be effective against SARS-CoV-2 in part because CQ and HCQ have both been found to inhibit the expression of CD154 in T-cells and to reduce TLR signaling that leads to the production of pro-inflammatory cytokines <span class="citation" data-cites="j3rGUH1f">(<a href="#ref-j3rGUH1f" role="doc-biblioref">1062</a>)</span>.
Clinical trials for COVID-19 have more often used HCQ rather than CQ because it offers the advantages of being cheaper and having fewer side effects than CQ.
However, research has not found support for a positive effect of HCQ on COVID-19 patients.
Multiple clinical studies have already been carried out to assess HCQ as a therapeutic agent for COVID-19, and many more are in progress.
To date, none of these studies have used randomized, double-blind, placebo-controlled designs with a large sample size, which would be the gold standard.
Despite the design limitations (which would be more likely to produce false positives than false negatives), initial optimism about HCQ has largely dissipated.
The most methodologically rigorous analysis of HCQ as a prophylactic <span class="citation" data-cites="NiVdIN7Z">(<a href="#ref-NiVdIN7Z" role="doc-biblioref">1060</a>)</span> found no significant differences between the treatment and control groups, and the WHO’s global Solidarity trial similarly reported no effect of HCQ on mortality <span class="citation" data-cites="jShm9uhy">(<a href="#ref-jShm9uhy" role="doc-biblioref">811</a>)</span>.
Thus, HCQ/CQ are not likely to be effective therapeutic or prophylactic agents against COVID-19.
One case study identified drug-induced phospholipidosis as the cause of death for a COVID-19 patient treated with HCQ <span class="citation" data-cites="1BV8xlc5v">(<a href="#ref-1BV8xlc5v" role="doc-biblioref">984</a>)</span>, suggesting that in some cases, the proposed mechanism of action may ultimately be harmful.
Additionally, one study identified an increased risk of mortality in older men receiving HCQ, and administration of HCQ and HCQ + AZ did not decrease the use of mechanical ventilation in these patients <span class="citation" data-cites="MENbRfOl">(<a href="#ref-MENbRfOl" role="doc-biblioref">1058</a>)</span>.
HCQ use for COVID-19 could also lead to shortages for anti-malarial or anti-rheumatic use, where it has documented efficacy.
Despite significant early attention, these drugs appear to be ineffective against COVID-19.
Several countries have now removed CQ/HCQ from their SOC for COVID-19 due to the lack of evidence of efficacy and the frequency of adverse effects.</p>
<h2 data-number="5.5" id="ace-inhibitors-and-angiotensin-ii-receptor-blockers"><span class="header-section-number">5.5</span> ACE Inhibitors and Angiotensin II Receptor Blockers</h2>
<p>Several clinical trials testing the effects of ACEIs or ARBs on COVID-19 outcomes are ongoing <span class="citation" data-cites="H0oEjZR8 kuUcA1G9 3ccJAyrG NCb9Wq2z 5fkIarR7 c7V9rAFR JP1lU6DV">(<a href="#ref-H0oEjZR8" role="doc-biblioref">1063</a>–<a href="#ref-JP1lU6DV" role="doc-biblioref">1069</a>)</span>.
Clinical trials are needed because the findings of the various observational studies bearing on this topic cannot be interpreted as indicating a protective effect of the drug <span class="citation" data-cites="19quOF2kK k5xEN3Ke">(<a href="#ref-19quOF2kK" role="doc-biblioref">1070</a>, <a href="#ref-k5xEN3Ke" role="doc-biblioref">1071</a>)</span>.
Two analyses <span class="citation" data-cites="H0oEjZR8 JP1lU6DV">(<a href="#ref-H0oEjZR8" role="doc-biblioref">1063</a>, <a href="#ref-JP1lU6DV" role="doc-biblioref">1069</a>)</span> have reported no effect of continuing or discontinuing ARBs and ACEIs on patients admitted to the hospital for COVID-19.
The first, known as REPLACE COVID <span class="citation" data-cites="IRuDYZRB">(<a href="#ref-IRuDYZRB" role="doc-biblioref">874</a>)</span>, was a randomized, open-label study that enrolled patients who were admitted to the hospital for COVID-19 and were taking an ACEI at the time of admission.
They enrolled 152 patients at 20 hospitals across seven countries and randomized them into two arms, continuation (n=75) and discontinuation (n=77).
The primary outcome evaluated was a global rank score that integrated several dimensions of illness.
The components of this global rank score, such as time to death and length of mechanical ventilation, were evaluated as secondary endpoints.
This analysis reported no differences between the two groups in the primary or any of the secondary outcomes.</p>
<p>Similarly, a second study <span class="citation" data-cites="1DqZPaCtf">(<a href="#ref-1DqZPaCtf" role="doc-biblioref">875</a>)</span> used a randomized, open-label design to examine the effects of continuing versus discontinuing ARBs and ACEIs on patients hospitalized for mild to moderate COVID-19 at 29 hospitals in Brazil.
This study enrolled 740 patients but had to exclude one trial site from all analyses due to the discovery of violations of Good Clinical Trial practice and data falsification.
After this exclusion, 659 patients remained, with 334 randomized to discontinuation and 325 to continuation.
In this study, the primary endpoint analyzed was the number of days that patients were alive and not hospitalized within 30 days of enrollment.
The secondary outcomes included death (including in-hospital death separately), number of days hospitalized, and specific clinical outcomes such as heart failure or stroke.
Once again, no significant differences were found between the two groups.
Initial studies of randomized interventions therefore suggest that ACEIs and ARBs are unlikely to affect COVID-19 outcomes.
These results are also consistent with findings from observational studies (summarized in <span class="citation" data-cites="IRuDYZRB">(<a href="#ref-IRuDYZRB" role="doc-biblioref">874</a>)</span>).
Additional information about ACE2, observational studies of ACEIs and ARBs in COVID-19, and clinical trials on this topic have been summarized <span class="citation" data-cites="1Hrsy8Hiq">(<a href="#ref-1Hrsy8Hiq" role="doc-biblioref">1072</a>)</span>.
Therefore, despite the promising potential mechanism, initial results have not provided support for ACEIs and ARBs as therapies for COVID-19.</p>
<h2 data-number="5.6" id="tocilizumab"><span class="header-section-number">5.6</span> Tocilizumab</h2>
<p>Human IL-6 is a 26-kDa glycoprotein that consists of 184 amino acids and contains two potential N-glycosylation sites and four cysteine residues.
It binds to a type I cytokine receptor (IL-6Rα or glycoprotein 80) that exists in both membrane-bound (IL-6Rα) and soluble (sIL-6Rα) forms <span class="citation" data-cites="VXDsRwHC">(<a href="#ref-VXDsRwHC" role="doc-biblioref">1073</a>)</span>.
It is not the binding of IL-6 to the receptor that initiates pro- and/or anti-inflammatory signaling, but rather the binding of the complex to another subunit, known as IL-6Rβ or glycoprotein 130 (gp130) <span class="citation" data-cites="14xZS2Gic VXDsRwHC">(<a href="#ref-VXDsRwHC" role="doc-biblioref">1073</a>, <a href="#ref-14xZS2Gic" role="doc-biblioref">1074</a>)</span>.
Unlike membrane-bound IL-6Rα, which is only found on hepatocytes and some types of leukocytes, gp130 is found on most cells <span class="citation" data-cites="LIqxTdN">(<a href="#ref-LIqxTdN" role="doc-biblioref">1075</a>)</span>.
When IL-6 binds to sIL-6Rα, the complex can then bind to a gp130 protein on any cell <span class="citation" data-cites="LIqxTdN">(<a href="#ref-LIqxTdN" role="doc-biblioref">1075</a>)</span>.
The binding of IL-6 to IL-6Rα is termed classical signaling, while its binding to sIL-6Rα is termed trans-signaling <span class="citation" data-cites="1BxkPZYnj 1EEloBjzn LIqxTdN">(<a href="#ref-LIqxTdN" role="doc-biblioref">1075</a>–<a href="#ref-1EEloBjzn" role="doc-biblioref">1077</a>)</span>.
These two signaling processes are thought to play different roles in health and illness.
For example, trans-signaling may play a role in the proliferation of mucosal T-helper TH2 cells associated with asthma, while an earlier step in this proliferation process may be regulated by classical signaling <span class="citation" data-cites="LIqxTdN">(<a href="#ref-LIqxTdN" role="doc-biblioref">1075</a>)</span>.
Similarly, IL-6 is known to play a role in Crohn’s Disease via trans-, but not classical, signaling <span class="citation" data-cites="LIqxTdN">(<a href="#ref-LIqxTdN" role="doc-biblioref">1075</a>)</span>.
Both classical and trans-signaling can occur through three independent pathways: the Janus-activated kinase-STAT3 pathway, the Ras/Mitogen-Activated Protein Kinases pathway and the Phosphoinositol-3 Kinase/Akt pathway <span class="citation" data-cites="VXDsRwHC">(<a href="#ref-VXDsRwHC" role="doc-biblioref">1073</a>)</span>.
These signaling pathways are involved in a variety of different functions, including cell type differentiation, immunoglobulin synthesis, and cellular survival signaling pathways, respectively <span class="citation" data-cites="VXDsRwHC">(<a href="#ref-VXDsRwHC" role="doc-biblioref">1073</a>)</span>.
The ultimate result of the IL-6 cascade is to direct transcriptional activity of various promoters of pro-inflammatory cytokines, such as IL-1, TFN, and even IL-6 itself, through the activity of NF-κB <span class="citation" data-cites="VXDsRwHC">(<a href="#ref-VXDsRwHC" role="doc-biblioref">1073</a>)</span>.
IL-6 synthesis is tightly regulated both transcriptionally and post-transcriptionally, and it has been shown that viral proteins can enhance transcription of the IL-6 gene by strengthening the DNA-binding activity between several transcription factors and IL-6 gene-cis-regulatory elements <span class="citation" data-cites="1H8o8TFtM">(<a href="#ref-1H8o8TFtM" role="doc-biblioref">1078</a>)</span>.
Therefore, drugs inhibiting the binding of IL-6 to IL-6Rα or sIL-6Rα are of interest for combating the hyperactive inflammatory response characteristic of cytokine release syndrome (CRS) and cytokine storm syndrome (CSS).
TCZ is a humanized monoclonal antibody that binds both to the insoluble and soluble receptor of IL-6, providing de facto inhibition of the IL-6 immune cascade.
Interest in TCZ as a possible treatment for COVID-19 was piqued by early evidence indicating that COVID-19 deaths may be induced by the hyperactive immune response, often referred to as CRS or CSS <span class="citation" data-cites="10THxyeCg">(<a href="#ref-10THxyeCg" role="doc-biblioref">83</a>)</span>, as IL-6 plays a key role in this response <span class="citation" data-cites="bZMKqj6e">(<a href="#ref-bZMKqj6e" role="doc-biblioref">150</a>)</span>.
The observation of elevated IL-6 in patients who died relative to those who recovered <span class="citation" data-cites="10THxyeCg">(<a href="#ref-10THxyeCg" role="doc-biblioref">83</a>)</span> could reflect an over-production of proinflammatory interleukins, suggesting that TCZ could potentially palliate some of the most severe symptoms of COVID-19 associated with increased cytokine production.</p>
<p>This early interest in TCZ as a possible treatment for COVID-19 was bolstered by a very small retrospective study in China that examined 20 patients with severe symptoms in early February 2020 and reported rapid improvement in symptoms following treatment with TCZ <span class="citation" data-cites="bj2feMy4">(<a href="#ref-bj2feMy4" role="doc-biblioref">893</a>)</span>.
Subsequently, a number of retrospective studies have been conducted in several countries.
Many studies use a retrospective, observational design, where they compare outcomes for COVID-19 patients who received TCZ to those who did not over a set period of time.
For example, one of the largest retrospective, observational analyses released to date <span class="citation" data-cites="ayc7zYqi">(<a href="#ref-ayc7zYqi" role="doc-biblioref">888</a>)</span>, consisting of 1,351 patients admitted to several care centers in Italy, compared the rates at which patients who received TCZ died or progressed to invasive medical ventilation over a 14-day period compared to patients receiving only SOC.
Under this definition, SOC could include other drugs such as HCQ, azithromycin, lopinavir-ritonavir or darunavir-cobicistat, or heparin.
While this study was not randomized, a subset of patients who were eligible to receive TCZ were unable to obtain it due to shortages; however, these groups were not directly compared in the analysis.
After adjusting for variables such as age, sex, and SOFA (sequential organ failure assessment) score, they found that patients treated with TCZ were less likely to progress to invasive medical ventilation and/or death (adjusted HR = 0.61, CI 0.40-0.92, <em>p</em> = 0.020); analysis of death and ventilation separately suggests that this effect may have been driven by differences in the death rate (20% of control versus 7% of TCZ-treated patients).
The study reported particular benefits for patients whose PaO<sub>2</sub>/FiO<sub>2</sub> ratio, also known as the Horowitz Index for Lung Function, fell below a 150 mm Hg threshold.
They found no differences between groups administered subcutaneous versus intravenous TCZ.</p>
<p>Another retrospective observational analysis of interest examined the charts of patients at a hospital in Connecticut, USA where 64% of all 239 COVID-19 patients in the study period were administered TCZ based on assignment by a standardized algorithm <span class="citation" data-cites="Wt9vgKLY">(<a href="#ref-Wt9vgKLY" role="doc-biblioref">889</a>)</span>.
They found that TCZ administration was associated with more similar rates of survivorship in patients with severe versus nonsevere COVID-19 at intake, defined based on the amount of supplemental oxygen needed.
They therefore proposed that their algorithm was able to identify patients presenting with or likely to develop CRS as good candidates for TCZ.
This study also reported higher survivorship in Black and Hispanic patients compared to white patients when adjusted for age.
The major limitation with interpretation for these studies is that there may be clinical characteristics that influenced medical practitioners decisions to administer TCZ to some patients and not others.
One interesting example therefore comes from an analysis of patients at a single hospital in Brescia, Italy, where TCZ was not available for a period of time <span class="citation" data-cites="luPHOS0m">(<a href="#ref-luPHOS0m" role="doc-biblioref">890</a>)</span>.
This study compared COVID-19 patients admitted to the hospital before and after March 13, 2020, when the hospital received TCZ.
Therefore, patients who would have been eligible for TCZ prior to this arbitrary date did not receive it as treatment, making this retrospective analysis something of a natural experiment.
Despite this design, demographic factors did not appear to be consistent between the two groups, and the average age of the control group was older than the TCZ group.
The control group also had a higher percentage of males and a higher incidence of comorbidities such as diabetes and heart disease.
All the same, the multivariate HR, which adjusted for these clinical and demographic factors, found a significant difference between survival in the two groups (HR=0.035, CI=0.004-0.347, <em>p</em> = 0.004).
The study reported improvement of survival outcomes after the addition of TCZ to the SOC regime, with 11 of 23 patients (47.8%) admitted prior to March 13th dying compared to 2 of 62 (3.2%) admitted afterwards (HR=0.035; 95% CI, 0.004 to 0.347; <em>p</em> = 0.004).
They also reported a reduced progression to mechanical ventilation in the TCZ group.
However, this study also holds a significant limitation: the time delay between the two groups means that knowledge about how to treat the disease likely improved over this timeframe as well.
All the same, the results of these observational retrospective studies provide support for TCZ as a pharmaceutical of interest for follow-up in clinical trials.</p>
<p>Other retrospective analyses have utilized a case-control design to match pairs of patients with similar baseline characteristics, only one of whom received TCZ for COVID-19.
In one such study, TCZ was significantly associated with a reduced risk of progression to intensive care unit (ICU) admission or death <span class="citation" data-cites="17hgtBpcV">(<a href="#ref-17hgtBpcV" role="doc-biblioref">891</a>)</span>.
This study examined only 20 patients treated with TCZ (all but one of the patients treated with TCZ in the hospital during the study period) and compared them to 25 patients receiving SOC.
For the combined primary endpoint of death and/or ICU admission, only 25% of patients receiving TCZ progressed to an endpoint compared to 72% in the SOC group (<em>p</em> = 0.002, presumably based on a chi-square test based on the information provided in the text).
When the two endpoints were examined separately, progression to invasive medical ventilation remained significant (32% SOC compared to 0% TCZ, <em>p</em> = 0.006) but not for mortality (48% SOC compared to 25% TCZ, <em>p</em> = 0.066).
In contrast, a study that compared 96 patients treated with TCZ to 97 patients treated with SOC only in New York City found that differences in mortality did not differ between the two groups, but that this difference did become significant when intubated patients were excluded from the analysis <span class="citation" data-cites="WEqfULN1">(<a href="#ref-WEqfULN1" role="doc-biblioref">892</a>)</span>.
Taken together, these findings suggest that future clinical trials of TCZ may want to include intubation as an endpoint.
However, these studies should be approached with caution, not only because of the small number of patients enrolled and the retrospective design, but also because they performed a large number of statistical tests and did not account for multiple hypothesis testing.
In general, caution must be exercised when interpreting subgroup analyses after a primary combined endpoint analysis.
These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated during CRS/CSS, and preventing the worsening of the clinical picture of the patients by potential new viral infections.
Early meta-analyses and systematic reviews have investigated the available data about TCZ for COVID-19.
One meta-analysis <span class="citation" data-cites="3yV2bL9S">(<a href="#ref-3yV2bL9S" role="doc-biblioref">1079</a>)</span> evaluated 19 studies published or released as preprints prior to July 1, 2020 and found that the overall trends were supportive of the frequent conclusion that TCZ does improve survivorship, with a significant HR of 0.41 (<em>p</em> &lt; 0.001).
This trend improved when they excluded studies that administered a steroid alongside TCZ, with a significant HR of 0.04 (<em>p</em> &lt; 0.001).
They also found some evidence for reduced invasive ventilation or ICU admission, but only when excluding all studies except a small number whose estimates were adjusted for the possible bias introduced by the challenges of stringency during the enrollment process.
A systematic analysis of sixteen case-control studies of TCZ estimated an odds ratio of mortality of 0.453 (95% CI 0.376–0.547, <em>p</em> &lt; 0.001), suggesting possible benefits associated with TCZ treatment <span class="citation" data-cites="WmK4YTdx">(<a href="#ref-WmK4YTdx" role="doc-biblioref">1080</a>)</span>.
Although these estimates are similar, it is important to note that they are drawing from the same literature and are therefore likely to be affected by the same potential biases in publication.
A different systematic review of studies investigating TCZ treatment for COVID-19 analyzed 31 studies that had been published or released as pre-prints and reported that none carried a low risk of bias <span class="citation" data-cites="h4reluu">(<a href="#ref-h4reluu" role="doc-biblioref">1081</a>)</span>.
Therefore, the present evidence is not likely to be sufficient for conclusions about the efficacy of TCZ.</p>
<p>On February 11, 2021, a preprint describing the first randomized control trial of TCZ was released as part of the RECOVERY trial <span class="citation" data-cites="9wnr4sOS">(<a href="#ref-9wnr4sOS" role="doc-biblioref">894</a>)</span>.
Of the 21,550 patients enrolled in the RECOVERY trial at the time, 4,116 adults hospitalized with COVID-19 across the 131 sites in the United Kingdom were assigned to the arm of the trial evaluating the effect of TCZ.
Among them, 2,022 were randomized to receive TCZ and 2,094 were randomized to SOC, with 79% of patients in each group available for analysis at the time that the initial report was released.
The primary outcome measured was 28-day mortality, and TCZ was found to reduce 28-day mortality from 33% of patients receiving SOC alone to 29% of those receiving TCZ, corresponding to a rate ratio of 0.86 (95% CI 0.77-0.96; <em>p</em> = 0.007).
TCZ was also significantly associated with the probability of hospital discharge within 28 days for living patients, which was 47% in the SOC group and 54% in the TCZ group (rate ratio 1.22, 95% CI 1.12-1.34, <em>p</em> &lt; 0.0001).
A potential statistical interaction between TCZ and corticosteroids was observed, with the combination providing greater mortality benefits than TCZ alone, but the authors note that caution is advisable in light of the number of statistical tests conducted.
Combining the RECOVERY trial data with data from seven smaller randomized control trials indicates that TCZ is associated with a 13% reduction in 28-day mortality (rate ratio 0.87, 95% CI 0.79-0.96, <em>p</em> = 0.005) <span class="citation" data-cites="9wnr4sOS">(<a href="#ref-9wnr4sOS" role="doc-biblioref">894</a>)</span>.</p>
<p>There are possible risks associated with the administration of TCZ for COVID-19.
TCZ has been used for over a decade to treat RA <span class="citation" data-cites="CRriJXDk">(<a href="#ref-CRriJXDk" role="doc-biblioref">1082</a>)</span>, and a recent study found the drug to be safe for pregnant and breastfeeding women <span class="citation" data-cites="6xLhs9st">(<a href="#ref-6xLhs9st" role="doc-biblioref">1083</a>)</span>.
However, TCZ may increase the risk of developing infections <span class="citation" data-cites="CRriJXDk">(<a href="#ref-CRriJXDk" role="doc-biblioref">1082</a>)</span>, and RA patients with chronic hepatitis B infections had a high risk of hepatitis B virus reactivation when TCZ was administered in combination with other RA drugs <span class="citation" data-cites="IJzmtJOr">(<a href="#ref-IJzmtJOr" role="doc-biblioref">1084</a>)</span>.
As a result, TCZ is contraindicated in patients with active infections such as tuberculosis <span class="citation" data-cites="SiOKIbl3">(<a href="#ref-SiOKIbl3" role="doc-biblioref">1085</a>)</span>.
Previous studies have investigated, with varying results, a possible increased risk of infection in RA patients administered TCZ <span class="citation" data-cites="6Agll71m zRRSvckT">(<a href="#ref-6Agll71m" role="doc-biblioref">1086</a>, <a href="#ref-zRRSvckT" role="doc-biblioref">1087</a>)</span>, although another study reported that the incidence rate of infections was higher in clinical practice RA patients treated with TCZ than in the rates reported by clinical trials <span class="citation" data-cites="TgV8gb30">(<a href="#ref-TgV8gb30" role="doc-biblioref">1088</a>)</span>.
In the investigation of 544 Italian COVID-19 patients, the group treated with TCZ was found to be more likely to develop secondary infections, with 24% compared to 4% in the control group (<em>p</em> &lt; 0.0001) <span class="citation" data-cites="ayc7zYqi">(<a href="#ref-ayc7zYqi" role="doc-biblioref">888</a>)</span>.
Reactivation of hepatitis B and herpes simplex virus 1 was also reported in a small number of patients in this study, all of whom were receiving TCZ.
A July 2020 case report described negative outcomes of two COVID-19 patients after receiving TCZ, including one death; however, both patients were intubated and had entered septic shock prior to receiving TCZ <span class="citation" data-cites="p0tzgJvA">(<a href="#ref-p0tzgJvA" role="doc-biblioref">1089</a>)</span>, likely indicating a severe level of cytokine production.
Additionally, D-dimer and sIL2R levels were reported by one study to increase in patients treated with TCZ, which raised concerns because of the potential association between elevated D-dimer levels and thrombosis and between sIL2R and diseases where T-cell regulation is compromised <span class="citation" data-cites="Wt9vgKLY">(<a href="#ref-Wt9vgKLY" role="doc-biblioref">889</a>)</span>.
An increased risk of bacterial infection was also identified in a systematic review of the literature, based on the unadjusted estimates reported <span class="citation" data-cites="3yV2bL9S">(<a href="#ref-3yV2bL9S" role="doc-biblioref">1079</a>)</span>.
In the RECOVERY trial, however, only three out of 2,022 participants in the group receiving TCZ developed adverse reactions determined to be associated with the intervention, and no excess deaths were reported <span class="citation" data-cites="9wnr4sOS">(<a href="#ref-9wnr4sOS" role="doc-biblioref">894</a>)</span>.
TCZ administration to COVID-19 patients is not without risks and may introduce additional risk of developing secondary infections; however, while caution may be prudent when treating patients who have latent viral infections, the results of the RECOVERY trial indicate that adverse reactions to TCZ are very rare among COVID-19 patients broadly.</p>
<p>In summary, approximately 33% of hospitalized COVID-19 patients develop ARDS <span class="citation" data-cites="epRQ9yZu">(<a href="#ref-epRQ9yZu" role="doc-biblioref">1090</a>)</span>, which is caused by an excessive early response of the immune system which can be a component of CRS/CSS <span class="citation" data-cites="Wt9vgKLY SiOKIbl3">(<a href="#ref-Wt9vgKLY" role="doc-biblioref">889</a>, <a href="#ref-SiOKIbl3" role="doc-biblioref">1085</a>)</span>.
This overwhelming inflammation is triggered by IL-6.
TCZ is an inhibitor of IL-6 and therefore may neutralize the inflammatory pathway that leads to the cytokine storm.
The mechanism suggests TCZ could be beneficial for the treatment of COVID-19 patients experiencing excessive immune activity, and the RECOVERY trial reported a reduction in 28-day mortality.
Interest in TCZ as a treatment for COVID-19 was also supported by two meta-analyses <span class="citation" data-cites="ZHqWXFcN 3yV2bL9S">(<a href="#ref-3yV2bL9S" role="doc-biblioref">1079</a>, <a href="#ref-ZHqWXFcN" role="doc-biblioref">1091</a>)</span>, but a third meta-analysis found that all of the available literature at that time carried a risk of bias <span class="citation" data-cites="h4reluu">(<a href="#ref-h4reluu" role="doc-biblioref">1081</a>)</span>.
Additionally, different studies used different dosages, number of doses, and methods of administration.
Ongoing research may be needed to optimize administration of TCZ <span class="citation" data-cites="hvOwSJvK">(<a href="#ref-hvOwSJvK" role="doc-biblioref">1092</a>)</span>, although similar results were reported by one study for intravenous and subcutaneous administration <span class="citation" data-cites="ayc7zYqi">(<a href="#ref-ayc7zYqi" role="doc-biblioref">888</a>)</span>.
Clinical trials that are in progress are likely to provide additional insight into the effectiveness of this drug for the treatment of COVID-19 along with how it should be administered.</p>
<h2 data-number="5.7" id="interferons"><span class="header-section-number">5.7</span> Interferons</h2>
<p>IFNs are a family of cytokines critical to activating the innate immune response against viral infections.
Interferons are classified into three categories based on their receptor specificity: types I, II and III <span class="citation" data-cites="bZMKqj6e">(<a href="#ref-bZMKqj6e" role="doc-biblioref">150</a>)</span>.
Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾) induce the expression of antiviral proteins <span class="citation" data-cites="LtFtUo2P">(<a href="#ref-LtFtUo2P" role="doc-biblioref">1093</a>)</span>.
Among these IFNs, IFN-𝛽 has already been found to strongly inhibit the replication of other coronaviruses, such as SARS-CoV-1, in cell culture, while IFN-𝛼 and 𝛾 were shown to be less effective in this context <span class="citation" data-cites="LtFtUo2P">(<a href="#ref-LtFtUo2P" role="doc-biblioref">1093</a>)</span>.
There is evidence that patients with higher susceptibility to ARDS indeed show deficiency in IFN-𝛽.
For instance, infection with other coronaviruses impairs IFN-𝛽 expression and synthesis, allowing the virus to escape the innate immune response <span class="citation" data-cites="Nfu9kiae">(<a href="#ref-Nfu9kiae" role="doc-biblioref">1094</a>)</span>.
On March 18 2020, Synairgen plc received approval to start a phase II trial for SNG001, an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation <span class="citation" data-cites="5GxCGzrT">(<a href="#ref-5GxCGzrT" role="doc-biblioref">901</a>)</span>.
SNG001, which contains recombinant interferon beta-1a, was previously shown to be effective in reducing viral load in an <em>in vivo</em> model of swine flu and <em>in vitro</em> models of other coronavirus infections <span class="citation" data-cites="7k9Wp9or">(<a href="#ref-7k9Wp9or" role="doc-biblioref">1095</a>)</span>.
In July 2020, a press release from Synairgen stated that SNG001 reduced progression to ventilation in a double-blind, placebo-controlled, multi-center study of 101 patients with an average age in the late 50s <span class="citation" data-cites="bD1bgfSb">(<a href="#ref-bD1bgfSb" role="doc-biblioref">902</a>)</span>.
These results were subsequently published in November 2020 <span class="citation" data-cites="qFDbLzcP">(<a href="#ref-qFDbLzcP" role="doc-biblioref">903</a>)</span>.
The study reports that the participants were assigned at a ratio of 1:1 to receive either SNG001 or a placebo that lacked the active compound, by inhalation for up to 14 days.
The primary outcome they assessed was the change in patients’ score on the WHO Ordinal Scale for Clinical Improvement (OSCI) at trial day 15 or 16.
SNG001 was associated with an odds ratio of improvement on the OSCI scale of 2.32 (95% CI 1.07 – 5.04, <em>p</em> = 0.033) in the intention-to-treat analysis and 2.80 (95% CI 1.21 – 6.52, <em>p</em> = 0.017) in the per-protocol analysis, corresponding to significant improvement in the SNG001 group on the OSCI at day 15/16.
Some of the secondary endpoints analyzed also showed differences: at day 28, the OR for clinical improvement on the OSCI was 3.15 (95% CI 1.39 – 7.14, <em>p</em> = 0.006), and the odds of recovery at day 15/16 and at day 28 were also significant between the two groups.
Thus, this study suggested that IFN-𝛽1 administered via SNG001 may improve clinical outcomes.</p>
<p>In contrast, the WHO Solidarity trial reported no significant effect of IFN-β-1a on patient survival during hospitalization <span class="citation" data-cites="jShm9uhy">(<a href="#ref-jShm9uhy" role="doc-biblioref">811</a>)</span>.
Here, the primary outcome analyzed was in-hospital mortality, and the rate ratio for the two groups was 1.16 (95% CI, 0.96 to 1.39; <em>p</em> = 0.11) administering IFN-β-1a to 2050 patients and comparing their response to 2,050 controls.
However, there are a few reasons that the different findings of the two trials might not speak to the underlying efficacy of this treatment strategy.
One important consideration is the stage of COVID-19 infection analyzed in each study.
The Synairgen trial enrolled only patients who were not receiving invasive ventilation, corresponding to a less severe stage of disease than many patients enrolled in the SOLIDARITY trial, as well as a lower overall rate of mortality <span class="citation" data-cites="11hbhD5Rk">(<a href="#ref-11hbhD5Rk" role="doc-biblioref">1096</a>)</span>.
Additionally, the methods of administration differed between the two trials, with the SOLIDARITY trial administering IFN-β-1a subcutaneously <span class="citation" data-cites="11hbhD5Rk">(<a href="#ref-11hbhD5Rk" role="doc-biblioref">1096</a>)</span>.
The differences in findings between the studies suggests that the method of administration might be relevant to outcomes, with nebulized IFN-β-1a more directly targeting receptors in the lungs.
A trial that analyzed the effect of subcutaneously administered IFN-β-1a on patients with ARDS between 2015 and 2017 had also reported no effect on 28-day mortality <span class="citation" data-cites="B2urVDo5">(<a href="#ref-B2urVDo5" role="doc-biblioref">1097</a>)</span>, while a smaller study analyzing the effect of subcutaneous IFN administration did find a significant improvement in 28-day mortality for COVID-19 <span class="citation" data-cites="xMRJgRES">(<a href="#ref-xMRJgRES" role="doc-biblioref">1098</a>)</span>.
At present, several ongoing clinical trials are investigating the potential effects of IFN-β-1a, including in combination with therapeutics such as remdesivir <span class="citation" data-cites="17so9kzfj">(<a href="#ref-17so9kzfj" role="doc-biblioref">1099</a>)</span> and administered via inhalation <span class="citation" data-cites="5GxCGzrT">(<a href="#ref-5GxCGzrT" role="doc-biblioref">901</a>)</span>.
Thus, as additional information becomes available, a more detailed understanding of whether and under which circumstances IFN-β-1a is beneficial to COVID-19 patients should develop.</p>
<h2 data-number="5.8" id="potential-avenues-of-interest-for-therapeutic-development"><span class="header-section-number">5.8</span> Potential Avenues of Interest for Therapeutic Development</h2>
<p>Given what is currently known about these therapeutics for COVID-19, a number of related therapies beyond those explored above may also prove to be of interest.
For example, the demonstrated benefit of dexamethasone and the ongoing potential of tocilizumab for treatment of COVID-19 suggests that other anti-inflammatory agents might also hold value for the treatment of COVID-19.
Current evidence supporting the treatment of severe COVID-19 with dexamethasone suggests that the need to curtail the cytokine storm inflammatory response transcends the risks of immunosuppression, and other anti-inflammatory agents may therefore benefit patients in this phase of the disease.
While dexamethasone is considered widely available and generally affordable, the high costs of biologics such as tocilizumab therapy may present obstacles to wide-scale distribution of this drug if it proves of value.
At the doses used for RA patients, the cost for tocilizumab ranges from $179.20 to $896 per dose for the IV form and $355 for the pre-filled syringe <span class="citation" data-cites="7fZ9eyMk">(<a href="#ref-7fZ9eyMk" role="doc-biblioref">1100</a>)</span>.
Several other anti-inflammatory agents used for the treatment of autoimmune diseases may also be able to counter the effects of the cytokine storm induced by the virus, and some of these, such as cyclosporine, are likely to be more cost-effective and readily available than biologics <span class="citation" data-cites="trWLoLFz">(<a href="#ref-trWLoLFz" role="doc-biblioref">1101</a>)</span>.
While tocilizumab targets IL-6, several other inflammatory markers could be potential targets, including TNF-α.
Inhibition of TNF-α by a compound such as Etanercept was previously suggested for treatment of SARS-CoV-1 <span class="citation" data-cites="LDQlolY3">(<a href="#ref-LDQlolY3" role="doc-biblioref">1102</a>)</span> and may be relevant for SARS-CoV-2 as well.
Another anti-IL-6 antibody, sarilumab, is also being investigated <span class="citation" data-cites="4DAIEbQF amy3rDsj">(<a href="#ref-4DAIEbQF" role="doc-biblioref">1103</a>, <a href="#ref-amy3rDsj" role="doc-biblioref">1104</a>)</span>.
Baricitinib and other small molecule inhibitors of the Janus-activated kinase pathway also curtail the inflammatory response and have been suggested as potential options for SARS-CoV-2 infections <span class="citation" data-cites="WF7ymA4m">(<a href="#ref-WF7ymA4m" role="doc-biblioref">1105</a>)</span>.
Baricitinib, in particular, may be able to reduce the ability of SARS-CoV-2 to infect lung cells <span class="citation" data-cites="uCns3aFw">(<a href="#ref-uCns3aFw" role="doc-biblioref">1106</a>)</span>.
Clinical trials studying baricitinib in COVID-19 have already begun in the US and in Italy <span class="citation" data-cites="jKRGxazA JCMhG5r9">(<a href="#ref-jKRGxazA" role="doc-biblioref">1107</a>, <a href="#ref-JCMhG5r9" role="doc-biblioref">1108</a>)</span>.
Identification and targeting of further inflammatory markers that are relevant in SARS-CoV-2 infection may be of value for curtailing the inflammatory response and lung damage.</p>
<p>In addition to immunosuppressive treatments, which are most beneficial late in disease progression, much research is focused on identifying therapeutics for early-stage patients.
For example, although studies of HCQ have not supported the early theory-driven interest in this antiviral treatment, alternative compounds with related mechanisms may still have potential.
Hydroxyferroquine derivatives of HCQ have been described as a class of bioorganometallic compounds that exert antiviral effects with some selectivity for SARS-CoV-1 <em>in vitro</em> <span class="citation" data-cites="Cd5uMaAr">(<a href="#ref-Cd5uMaAr" role="doc-biblioref">1109</a>)</span>.
Future work could explore whether such compounds exert antiviral effects against SARS-CoV-2 and whether they would be safer for use in COVID-19.</p>
<p>Another potential approach is the development of antivirals, which could be broad-spectrum, specific to coronaviruses, or targeted to SARS-CoV-2.
Development of new antivirals is complicated by the fact that none have yet been approved for human coronaviruses.
Intriguing new options are emerging, however.
Beta-D-N4-hydroxycytidine is an orally bioavailable ribonucleotide analog showing broad-spectrum activity against RNA viruses, which may inhibit SARS-CoV-2 replication <em>in vitro</em> and <em>in vivo</em> in mouse models of HCoVs <span class="citation" data-cites="TZleuwHX">(<a href="#ref-TZleuwHX" role="doc-biblioref">1110</a>)</span>.
A range of other antivirals are also in development.
Development of antivirals will be further facilitated as research reveals more information about the interaction of SARS-CoV-2 with the host cell and host cell genome, mechanisms of viral replication, mechanisms of viral assembly, and mechanisms of viral release to other cells; this can allow researchers to target specific stages and structures of the viral life cycle.
Finally, antibodies against viruses, also known as antiviral monoclonal antibodies, could be an alternative as well and are described in detail in an above section.
The goal of antiviral antibodies is to neutralize viruses through either cell-killing activity or blocking of viral replication <span class="citation" data-cites="1FTnvyZlf">(<a href="#ref-1FTnvyZlf" role="doc-biblioref">1111</a>)</span>.
They may also engage the host immune response, encouraging the immune system to hone in on the virus.
Given the cytokine storm that results from immune system activation in response to the virus, which has been implicated in worsening of the disease, a neutralizing antibody (nAb) may be preferable.
Upcoming work may explore the specificity of nAbs for their target, mechanisms by which the nAbs impede the virus, and improvements to antibody structure that may enhance the ability of the antibody to block viral activity.</p>
<p>Some research is also investigating potential therapeutics and prophylactics that would interact with components of the innate immune response.
For example, TLRs are pattern recognition receptors that recognize pathogen- and damage-associated molecular patterns and contribute to innate immune recognition and, more generally, promotion of both the innate and adaptive immune responses <span class="citation" data-cites="o6BQnEt7">(<a href="#ref-o6BQnEt7" role="doc-biblioref">147</a>)</span>.
In mouse models, poly(I:C) and CpG, which are agonists of Toll-like receptors TLR3 and TLR9, respectively, showed protective effects when administered prior to SARS-CoV-1 infection <span class="citation" data-cites="Ull2rQ5L">(<a href="#ref-Ull2rQ5L" role="doc-biblioref">1112</a>)</span>.
Therefore, TLR agonists hold some potential for broad-spectrum prophylaxis.</p>
<h1 data-number="6" id="application-of-traditional-vaccine-development-strategies-to-sars-cov-2"><span class="header-section-number">6</span> Application of Traditional Vaccine Development Strategies to SARS-CoV-2</h1>
<h2 data-number="6.1" id="abstract-4"><span class="header-section-number">6.1</span> Abstract</h2>
<p>Over the past 150 years, vaccines have revolutionized the relationship between people and disease.
During the COVID-19 pandemic, technologies such as mRNA vaccines have received significant attention due to their novelty and successes.
However, more traditional vaccine development platforms have also been applied against SARS-CoV-2, yielding important tools in the worldwide fight against the virus.</p>
<p>A variety of approaches have been used to develop COVID-19 vaccines that are now authorized for use in countries around the world.
In this review, we highlight strategies that focus on the viral capsid outwards, rather than on the nucleic acids inside.
Such approaches broadly fall into two categories: whole-virus vaccines and subunit vaccines.
Whole-virus vaccine approaches use the virus itself, either in an inactivated or attenuated state.
Subunit vaccines isolate an immunogenic component of the virus using various strategies that is then introduced through vaccination.
We highlight specific vaccine candidates that utilize these approaches in different ways.
In a companion manuscript, we review the more recent and novel development of nucleic-acid based vaccine technologies.</p>
<p>We further consider the role that these COVID-19 vaccine development programs have played in providing immunity to people around the world.
Well-established vaccine technologies have proved especially important because of the significant role they have played in COVID-19 vaccine access at the global scale.
Vaccine development programs that use established platforms have been undertaken in a much wider range of countries than those using nucleic-acid-based technologies, which have been led by wealthy Western countries.
Therefore, these vaccine platforms, while less cutting-edge on the biotechnology side, have proven to be extremely important to the management of SARS-CoV-2.</p>
<h2 data-number="6.2" id="importance-4"><span class="header-section-number">6.2</span> Importance</h2>
<p>As of December 18, 2022, there have been over 653,207,634 SARS-CoV-2 cases, and the virus has taken the lives of at least 6,665,906 people globally <span class="citation" data-cites="U1nCE851">(<a href="#ref-U1nCE851" role="doc-biblioref">732</a>)</span>.
The development, production, and distribution of vaccines is imperative to saving lives, preventing illness, and reducing the economic and social burdens caused by the COVID-19 pandemic.
Much emphasis has been placed on vaccines that use cutting-edge biotechnology.
However, more traditional methods of vaccine development that were refined throughout the twentieth century have been critical in increasing vaccine access worldwide.
Effective deployment is necessary to reducing the susceptibility of the world’s population, which is especially important in light of emerging variants.
In this review, we discuss the safety, efficacy, and distribution of vaccines developed using established technologies.
In a separate review, we describe the vaccines developed using nucleic acid-based vaccine platforms.
From the current literature, it is clear that the well-established vaccine technologies are also highly effective against SARS-CoV-2 and are being used to address the challenges of COVID-19 globally, including in low- and middle-income countries.
This worldwide approach is critical for reducing the devastating impact of SARS-CoV-2.</p>
<h2 data-number="6.3" id="introduction-4"><span class="header-section-number">6.3</span> Introduction</h2>
<p>The development of vaccines is widely considered one of the most important medical advances in human history.
Over the past 150 years, several approaches to vaccination have been developed and refined <span class="citation" data-cites="YY3x3bBV">(<a href="#ref-YY3x3bBV" role="doc-biblioref">1113</a>)</span>.
The COVID-19 pandemic has generated atypical circumstances compared to past health crises, leading to differences in vaccine development against this pathogen.
One way in which the COVID-19 pandemic differs from prior global health crises is that the SARS-CoV-2 viral genome was sequenced, assembled, and released very early on (January 2020).
This genomic information has been important in shaping the biomedical response to this pathogen <span class="citation" data-cites="GdZc4Yyd njpLhBui">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>, <a href="#ref-njpLhBui" role="doc-biblioref">3</a>)</span>.
All the same, vaccines have been developed since long before the concept of a virus or a viral genome was known, and as early as September 2020, there were over 180 vaccine candidates against SARS-CoV-2 in development, many of which used more traditional vaccine technologies <span class="citation" data-cites="dqpEe5Lz">(<a href="#ref-dqpEe5Lz" role="doc-biblioref">1114</a>)</span>.
However, media attention has largely focused on vaccine development platforms that have produced vaccine candidates using new technologies, especially mRNA vaccines due to their rapid deployment.
We review vaccine technologies used for SARS-CoV-2 in two parts: here, the application of traditional vaccine development platforms to SARS-CoV-2, and separately, nucleic acid-based approaches <span class="citation" data-cites="rjHK19Dr">(<a href="#ref-rjHK19Dr" role="doc-biblioref">7</a>)</span>.</p>
<p>Understanding vaccine development programs that are using well-established technologies is important for a global perspective on COVID-19.
As of December 2, 2022, 50 SARS-CoV-2 vaccines have been approved for use in at least one country.
A resource tracking the distribution of 27 vaccines indicates that, as of December 18, 2022, 13.0 billion doses have been administered across 223 countries <span class="citation" data-cites="sRy6js2o">(<a href="#ref-sRy6js2o" role="doc-biblioref">1115</a>)</span>.
Many of these vaccines use platforms that do not require information about the viral genome, with 20 developed using subunit and 11 using whole-virus approaches.
The types of vaccines available varies widely throughout the world, as the process of developing and deploying a vaccine is complex and often requires coordination between government, industry, academia, and philanthropic entities <span class="citation" data-cites="plfPrQP7">(<a href="#ref-plfPrQP7" role="doc-biblioref">1116</a>)</span>.</p>
<p>Another difference between prior global health crises and the COVID-19 pandemic is the way that vaccines are evaluated.
A vaccine’s success is often discussed in terms of vaccine efficacy (VE), which describes the protection conferred during clinical trials <span class="citation" data-cites="DhPGhZPn">(<a href="#ref-DhPGhZPn" role="doc-biblioref">1117</a>)</span>.
The real-world protection offered by a vaccine is referred to as its effectiveness <span class="citation" data-cites="DhPGhZPn">(<a href="#ref-DhPGhZPn" role="doc-biblioref">1117</a>)</span>.
Additionally, protection can mean different things in different contexts.
In general, the goal of a vaccine is to prevent disease, especially severe disease, rather than infection itself.
As a proxy for VE, vaccine developers often test their candidates for serum neutralizing activity, which has been proposed as a biomarker for adaptive immunity in other respiratory illnesses <span class="citation" data-cites="wiGjCZC8">(<a href="#ref-wiGjCZC8" role="doc-biblioref">1118</a>)</span>.
The duration and intensity of the COVID-19 pandemic has made it possible to test multiple vaccines in phase III trials, where the effect of the vaccines on a cohort’s likelihood of contracting SARS-CoV-2 can be evaluated, whereas this has not always been feasible for other infectious diseases.
In some cases (e.g., SARS), the pathogen has been controlled before vaccine candidates were available, while in others (e.g., MERS), the scale of the epidemic has been smaller.
Vaccine development is traditionally a slow process, but the urgency of the COVID-19 pandemic created an atypical vaccine development ecosystem where fast development and production was prioritized.
Estimates of VE have been released for many vaccine candidates across a number of technology types based on phase III trial data.</p>
<p>However, efficacy is not a static value, and real-world efficacy can vary with location and over time.
Shifts in efficacy have been an especially heightened topic of concern since late 2021 given the potential for variants of SARS-CoV-2 to influence VE.
Due to viral evolution, vaccine developers are in an arms race with a pathogen that benefits from mutations that reduce its susceptibility to adaptive immunity.
The evolution of several variants of concern (VOC) presents significant challenges for vaccines developed based on the index strain identified in Wuhan in late 2019.
We discuss these variants in depth elsewhere <span class="citation" data-cites="17qiILENK">(<a href="#ref-17qiILENK" role="doc-biblioref">341</a>)</span>.
To date, the most significant VOC identified are Alpha (2020), Beta (2020), Gamma (2020), Delta (2021), and Omicron (2021), with various subvariants of Omicron being the most recently identified (2022).
The efficacy of vaccines in the context of these variants is discussed where information is available.</p>
<p>While the relationship between a vaccine and a pathogen is not static, the data clearly demonstrates that a variety of efficacious vaccines have been developed against SARS-CoV-2.
Here we discuss a selection of programs that use well-established vaccine biotechnologies.
These programs have been undertaken worldwide, in complement to the more cutting-edge approaches developed and distributed in the United States (U.S.), the European Union (E.U.), the United Kingdom (U.K.), and Russia <span class="citation" data-cites="rjHK19Dr">(<a href="#ref-rjHK19Dr" role="doc-biblioref">7</a>)</span>.
In this review, we discuss vaccine development using two well-established technologies, whole-virus vaccination and subunit vaccination, and the role these technologies have played in the global response to the COVID-19 pandemic.</p>
<h2 data-number="6.4" id="whole-virus-vaccines"><span class="header-section-number">6.4</span> Whole-Virus Vaccines</h2>
<p>Whole-virus vaccines have the longest history among vaccine development approaches.
Variolation, which is widely considered the first vaccination strategy in human history, is one example <span class="citation" data-cites="Q10m9bJ 1Clt2Bek3">(<a href="#ref-Q10m9bJ" role="doc-biblioref">1119</a>, <a href="#ref-1Clt2Bek3" role="doc-biblioref">1120</a>)</span>.
Famously, variolation was employed against smallpox when healthy individuals were exposed to pus from an individual infected with what was believed to be either cowpox or horsepox <span class="citation" data-cites="Q10m9bJ 1Clt2Bek3 ZUHALvLg 1DFnwhtrq">(<a href="#ref-Q10m9bJ" role="doc-biblioref">1119</a>–<a href="#ref-1DFnwhtrq" role="doc-biblioref">1122</a>)</span>.
This approach worked by inducing a mild case of a disease.
Therefore, while whole-virus vaccines confer adaptive immunity, they also raise safety concerns <span class="citation" data-cites="kC2tx3JC K0Ltu31S ZUHALvLg">(<a href="#ref-ZUHALvLg" role="doc-biblioref">1121</a>, <a href="#ref-kC2tx3JC" role="doc-biblioref">1123</a>, <a href="#ref-K0Ltu31S" role="doc-biblioref">1124</a>)</span>.
As of 2005, most vaccines still used whole-virus platforms <span class="citation" data-cites="U9ZIZWkB">(<a href="#ref-U9ZIZWkB" role="doc-biblioref">1125</a>)</span>, and these technologies remain valuable tools in vaccine development today <span class="citation" data-cites="YY3x3bBV">(<a href="#ref-YY3x3bBV" role="doc-biblioref">1113</a>)</span>.
Whole virus vaccine candidates have been developed for COVID-19 using both live attenuated viruses and inactivated whole viruses.</p>
<h3 data-number="6.4.1" id="live-attenuated-virus-vaccines"><span class="header-section-number">6.4.1</span> Live-Attenuated Virus Vaccines</h3>
<p>Live-attenuated virus vaccines (LAV), also known as replication-competent vaccines, use a weakened, living version of a disease-causing virus or a version of a virus that is modified to induce an immune response <span class="citation" data-cites="dqpEe5Lz">(<a href="#ref-dqpEe5Lz" role="doc-biblioref">1114</a>)</span>.
Whether variolation is the first example of a LAV being used to induce immunity is debated <span class="citation" data-cites="kC2tx3JC YY3x3bBV">(<a href="#ref-YY3x3bBV" role="doc-biblioref">1113</a>, <a href="#ref-kC2tx3JC" role="doc-biblioref">1123</a>)</span>.
The first deliberate (albeit pathogen-naïve) attempt to develop an attenuated viral vaccine dates back to Louis Pasteur’s efforts in 1885 to inoculate a child against rabies.
The next intentional LAVs were developed against the yellow fever virus in 1935 and influenza in 1936 <span class="citation" data-cites="iX8wXLPW">(<a href="#ref-iX8wXLPW" role="doc-biblioref">1126</a>)</span>.</p>
<p>Early efforts in LAV development relied on either the identification of a related virus that was less virulent in humans (e.g., cowpox/horsepox or rotavirus vaccines) or the culturing of a virus <em>in vitro</em> <span class="citation" data-cites="ZUHALvLg YY3x3bBV">(<a href="#ref-YY3x3bBV" role="doc-biblioref">1113</a>, <a href="#ref-ZUHALvLg" role="doc-biblioref">1121</a>)</span>.
Today, a virus can be attenuated by passaging it in a foreign host until, due to selection pressure, the virus loses its efficacy in the original host.
Alternatively, selective gene deletion or codon deoptimization can be utilized to attenuate the virus <span class="citation" data-cites="dqpEe5Lz">(<a href="#ref-dqpEe5Lz" role="doc-biblioref">1114</a>)</span>, or foreign antigens can be integrated into an attenuated viral vector <span class="citation" data-cites="4RuaSyLg">(<a href="#ref-4RuaSyLg" role="doc-biblioref">1127</a>)</span>.
LAVs tend to be restricted to viral replication in the tissues around the location of inoculation <span class="citation" data-cites="iX8wXLPW">(<a href="#ref-iX8wXLPW" role="doc-biblioref">1126</a>)</span>, and some can be administered intranasally <span class="citation" data-cites="dqpEe5Lz">(<a href="#ref-dqpEe5Lz" role="doc-biblioref">1114</a>)</span>.</p>
<p>Today, LAVs are used globally to prevent diseases caused by viruses such as measles, mumps, rubella, polio, influenza, varicella zoster, and the yellow fever virus <span class="citation" data-cites="wZ2tXSUH">(<a href="#ref-wZ2tXSUH" role="doc-biblioref">1128</a>)</span>.
There were attempts to develop LAVs against both SARS-CoV-1 and MERS-CoV <span class="citation" data-cites="7RHpaAHu">(<a href="#ref-7RHpaAHu" role="doc-biblioref">1129</a>)</span>, but no vaccines were approved.
It is generally recognized that LAVs induce an immune response similar to natural infection, and they are favored because they induce long-lasting and robust immunity that can protect from disease.
This strong protective effect is induced in part by the immune response to the range of viral antigens available from LAV, which tend to be more immunogenic than those from non-replicating vaccines <span class="citation" data-cites="zLL2yOJK 7RHpaAHu kC2tx3JC">(<a href="#ref-kC2tx3JC" role="doc-biblioref">1123</a>, <a href="#ref-7RHpaAHu" role="doc-biblioref">1129</a>, <a href="#ref-zLL2yOJK" role="doc-biblioref">1130</a>)</span>.</p>
<h3 data-number="6.4.2" id="application-to-covid-19"><span class="header-section-number">6.4.2</span> Application to COVID-19</h3>
<p>To date, LAVs have not been widely deployed against SARS-CoV-2 and COVID-19.
All the same, there are several COVID-19 LAV candidates in the early (preclinical/phase I) stages of investigation.
These candidates utilize different approaches.</p>
<p>One candidate in the preclinical stage is YF-S0, a single-dose LAV developed at Belgium’s Katholieke Universiteit Leuven that uses live-attenuated yellow fever 17D (YF17D) as a vector for a noncleavable prefusion conformation of the spike antigen of SARS-CoV-2 <span class="citation" data-cites="4RuaSyLg">(<a href="#ref-4RuaSyLg" role="doc-biblioref">1127</a>)</span>.
YF-S0 induced a robust immune response in three animal models and prevented SARS-CoV-2 infection in macaques and hamsters <span class="citation" data-cites="4RuaSyLg">(<a href="#ref-4RuaSyLg" role="doc-biblioref">1127</a>)</span>.</p>
<p>Other programs are developing codon deoptimized LAV candidates <span class="citation" data-cites="nwyfEEPl iCUWeMfX xMxJTYge">(<a href="#ref-nwyfEEPl" role="doc-biblioref">1131</a>–<a href="#ref-xMxJTYge" role="doc-biblioref">1133</a>)</span>.
This approach follows the synthetic attenuated virus engineering (SAVE) strategy to select codon substitutions that are suboptimal for the virus <span class="citation" data-cites="Gnk3ge6D xMxJTYge">(<a href="#ref-xMxJTYge" role="doc-biblioref">1133</a>, <a href="#ref-Gnk3ge6D" role="doc-biblioref">1134</a>)</span>.
New York-based Codagenix and the Serum Institute of India reported a successful preclinical investigation <span class="citation" data-cites="xMxJTYge">(<a href="#ref-xMxJTYge" role="doc-biblioref">1133</a>)</span> of an intranasally administered deoptimized SARS-CoV-2 LAV known as COVI-VAC, and COVI-VAC entered phase I human trials and dosed its first participants in January 2021 <span class="citation" data-cites="iCUWeMfX bPMpOwp8">(<a href="#ref-iCUWeMfX" role="doc-biblioref">1132</a>, <a href="#ref-bPMpOwp8" role="doc-biblioref">1135</a>)</span>.
This vaccine is optimized through the removal of the furin cleavage site (see <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span> for a discussion of this site’s importance) and deoptimization of 283 codons <span class="citation" data-cites="2VWtlcwJ">(<a href="#ref-2VWtlcwJ" role="doc-biblioref">1136</a>)</span>.
The results of the COVI-VAC phase I human trials are expected soon.</p>
<p>Another company, Meissa Vaccines in Kansas, U.S., which also develops vaccines for respiratory syncytial virus (RSV), has developed an intranasal live-attenuated chimeric vaccine.
Chimeric vaccines integrate genomic content from multiple viruses to create a more stable LAV <span class="citation" data-cites="14GSJSuHC">(<a href="#ref-14GSJSuHC" role="doc-biblioref">1137</a>)</span>.
Enrollment for phase I human trials began in March 2021 and recruitment is ongoing <span class="citation" data-cites="iCUWeMfX 8ZMg94iW">(<a href="#ref-iCUWeMfX" role="doc-biblioref">1132</a>, <a href="#ref-8ZMg94iW" role="doc-biblioref">1138</a>)</span>.</p>
<p>Finally, Bacillus Calmette-Guerin (BCG) vaccines that use LAVs are being investigated for the prophylaxis of COVID-19 (see online Appendix <span class="citation" data-cites="yTsmmAYC">(<a href="#ref-yTsmmAYC" role="doc-biblioref">1139</a>)</span>).
The purpose of the BCG vaccine is to prevent tuberculosis, but non-specific effects against other respiratory illnesses have suggested a possible benefit against COVID-19 <span class="citation" data-cites="1CJtdlM6d">(<a href="#ref-1CJtdlM6d" role="doc-biblioref">1140</a>)</span>.
However, a multicenter trial that randomly assigned participants 60 years and older to vaccination with BCG (n = 1,008) or placebo (n = 1,006) found that BCG vaccination had no effect on the incidence of SARS-CoV-2 or other respiratory infections over the course of 12 months <span class="citation" data-cites="eHrzWQ6D">(<a href="#ref-eHrzWQ6D" role="doc-biblioref">1141</a>)</span>.
Despite these findings, BCG vaccination was associated with a stronger cytokine (IL-6) response following <em>ex vivo</em> stimulation of peripheral blood mononuclear cells in patients with no known history of COVID-19 <span class="citation" data-cites="eHrzWQ6D">(<a href="#ref-eHrzWQ6D" role="doc-biblioref">1141</a>)</span>.
Additionally, SARS-CoV-2-positive individuals who had received the BCG vaccine one year prior showed increased immunoglobulin (Ig) responses to the SARS-CoV-2 spike protein and receptor binding domain (RBD) relative to individuals who had received a placebo vaccine <span class="citation" data-cites="eHrzWQ6D">(<a href="#ref-eHrzWQ6D" role="doc-biblioref">1141</a>)</span>.
Currently, investigations of BCG vaccines against COVID-19 are being sponsored by institutes in Australia in collaboration with the Bill and Melinda Gates Foundation <span class="citation" data-cites="9m3rP633">(<a href="#ref-9m3rP633" role="doc-biblioref">1142</a>)</span> and by Texas A&amp;M University in collaboration with numerous other U.S. institutions <span class="citation" data-cites="xdqxBruc">(<a href="#ref-xdqxBruc" role="doc-biblioref">1143</a>)</span>.</p>
<p>Safety data is not yet available for human studies of LAV COVID-19 vaccines.
In general, though, safety concerns previously associated with LAV have been largely mitigated in the modern manufacturing process.
Manufacturers use safe and reliable methods to produce large quantities of vaccines once they have undergone rigorous preclinical studies and clinical trials to evaluate their safety and efficacy.
However, one remaining safety concern may be contributing to the relatively slow emergence of LAV candidates against COVID-19: they still may present risk to individuals who are immunocompromised <span class="citation" data-cites="bgKUtUIL">(<a href="#ref-bgKUtUIL" role="doc-biblioref">1144</a>)</span>, which is an even greater concern when dealing with a novel virus and disease.
Additional data are needed to ascertain how this technology performs in the case of SARS-CoV-2.
Despite the long and trusted history of LAV development, this vaccine strategy has not been favored against COVID-19, as other technologies have shown greater expediency and safety compared to the time-consuming nature of developing LAVs for a novel virus.</p>
<h3 data-number="6.4.3" id="response-to-variants-of-concern"><span class="header-section-number">6.4.3</span> Response to Variants of Concern</h3>
<p>While no phase III trial data is available for LAV vaccine candidates, some manufacturers have proactively sought to respond to the emergence of VOC.
For example, a poster reported that Syrian golden hamsters who received COVI-VAC were significantly less likely to lose weight following viral challenge with the Beta VOC <span class="citation" data-cites="2VWtlcwJ">(<a href="#ref-2VWtlcwJ" role="doc-biblioref">1136</a>)</span>.
Additionally, the protective effect of YF-S0 against VOC has been investigated in hamsters <span class="citation" data-cites="z1FVz33g">(<a href="#ref-z1FVz33g" role="doc-biblioref">1145</a>)</span>.
Even for a small number of hamsters that developed breakthrough infections after exposure to the index strain or the Alpha variant, viral loads were very low <span class="citation" data-cites="z1FVz33g">(<a href="#ref-z1FVz33g" role="doc-biblioref">1145</a>)</span>.
However, much higher rates of breakthrough infection and higher viral loads were observed when the hamsters were exposed to the Beta variant <span class="citation" data-cites="z1FVz33g">(<a href="#ref-z1FVz33g" role="doc-biblioref">1145</a>)</span>.
Reduced seroconversion and neutralizing antibody (nAb) titers were also observed against the Beta and Gamma variants <span class="citation" data-cites="z1FVz33g">(<a href="#ref-z1FVz33g" role="doc-biblioref">1145</a>)</span>.
As a result, a modified version of YF-S0, called YF-S0*, was developed to include a modified spike protein intended to increase immunogenicity by including the full spectrum of amino acids found in the Gamma VOC as well as stabilizing the S protein’s conformation <span class="citation" data-cites="z1FVz33g">(<a href="#ref-z1FVz33g" role="doc-biblioref">1145</a>)</span>.
The updated vaccine was again tested in Syrian golden hamsters.
No breakthrough infections were observed following vaccination with YF-S0* and exposure to the index strain and the Alpha, Beta, Gamma, and Delta variants <span class="citation" data-cites="z1FVz33g">(<a href="#ref-z1FVz33g" role="doc-biblioref">1145</a>)</span>.
YF-S0* also reduced the infectious viral load in the lungs of several VOCs (Alpha, Beta, Gamma, and Delta) relative to a sham comparison <span class="citation" data-cites="z1FVz33g">(<a href="#ref-z1FVz33g" role="doc-biblioref">1145</a>)</span>, and the likelihood of the Delta variant spreading to unvaccinated co-housed hamsters was significantly reduced by YF-S0* <span class="citation" data-cites="z1FVz33g">(<a href="#ref-z1FVz33g" role="doc-biblioref">1145</a>)</span>.
The updated vaccine was also associated with the increased production of nAbs against the Omicron variant compared to YF-S0 <span class="citation" data-cites="z1FVz33g">(<a href="#ref-z1FVz33g" role="doc-biblioref">1145</a>)</span>.
Therefore, while data from human studies are not available, preclinical results suggest that LAV vaccines likely confer some protection against VOC even when designed with the index strain.
Modifications to the design may make this protection more robust as SARS-CoV-2 evolves.</p>
<h2 data-number="6.5" id="inactivated-whole-virus-vaccines"><span class="header-section-number">6.5</span> Inactivated Whole-Virus Vaccines</h2>
<p>Inactivated whole-virus (IWV) vaccines are another well-established vaccine platform.
This platform uses full virus particles generally produced via cell culture that have been rendered non-infectious by chemical (i.e., formaldehyde or β-propiolactone <span class="citation" data-cites="PwjPrwXa">(<a href="#ref-PwjPrwXa" role="doc-biblioref">1146</a>)</span>) or physical (i.e., heat or ultraviolet radiation) means.
In general, these vaccines mimic the key properties of the virus that stimulate a robust immune response, but the risk of adverse reactions is reduced because the virus is inactivated and thus unable to replicate.
Though these viral particles are inactivated, they retain the capacity to prime the immune system.
The size of the viral particle makes it ideal for uptake by antigen-presenting cells, which leads to the stimulation of helper T-cells <span class="citation" data-cites="7Knbo28h">(<a href="#ref-7Knbo28h" role="doc-biblioref">1147</a>)</span>.
Additionally, the array of epitopes on the surface of the virus increases antibody binding efficiency <span class="citation" data-cites="7Knbo28h">(<a href="#ref-7Knbo28h" role="doc-biblioref">1147</a>)</span>.
The native conformation of the surface proteins, which is also important for eliciting an immune response, is preserved using these techniques <span class="citation" data-cites="10peSXMZx">(<a href="#ref-10peSXMZx" role="doc-biblioref">1148</a>)</span>.
Membrane proteins, which support B-cell responses to surface proteins, are also induced by this method <span class="citation" data-cites="iAa7uWOm">(<a href="#ref-iAa7uWOm" role="doc-biblioref">1149</a>)</span>.</p>
<p>IWV vaccines have been a valuable tool in efforts to control many viruses.
Some targets of IWV vaccines have included influenza viruses, poliovirus, and hepatitis A virus.
Inactivated vaccines can generally be generated relatively quickly once the pathogenic virus has been isolated and can be passaged in cell culture <span class="citation" data-cites="7RHpaAHu xQSh0vVt">(<a href="#ref-7RHpaAHu" role="doc-biblioref">1129</a>, <a href="#ref-xQSh0vVt" role="doc-biblioref">1150</a>)</span>.
During COVID-19, though the World Health Organization (WHO) has been slower to approve IWV vaccine candidates than those developed with nucleic acid-based technologies, IWV vaccine development was also fast.
In China, the first emergency use authorization (EUA) was granted to an IWV vaccine in July 2020, with full approval following that December <span class="citation" data-cites="ynMHxaPL gwspw48F">(<a href="#ref-ynMHxaPL" role="doc-biblioref">1151</a>, <a href="#ref-gwspw48F" role="doc-biblioref">1152</a>)</span>.<br />
The fact that these vaccines have not received as much public attention (at least in Western media) as nucleic acid vaccines for SARS-CoV-2 may be due at least in part to the novelty of nucleic acid vaccine technologies <span class="citation" data-cites="l28xUYm7">(<a href="#ref-l28xUYm7" role="doc-biblioref">1153</a>)</span>, which are more modular and immunogenic <span class="citation" data-cites="rjHK19Dr">(<a href="#ref-rjHK19Dr" role="doc-biblioref">7</a>)</span>.</p>
<p>Past applications to human coronaviruses (HCoV) have focused predominantly on SARS-CoV-1.
Preclinical studies have demonstrated that IWV SARS-CoV-1 vaccine candidates elicited immune responses <em>in vivo</em>.
These vaccines generated nAb titers at concentrations similar to those evoked by recombinant protein vaccines <span class="citation" data-cites="1DymXCWa0 10peSXMZx">(<a href="#ref-10peSXMZx" role="doc-biblioref">1148</a>, <a href="#ref-1DymXCWa0" role="doc-biblioref">1154</a>)</span>.
Studies in ferrets and non-human primates demonstrated that IWV vaccines can offer protection against infection due to nAb and SARS-CoV-1-specific T cell responses <span class="citation" data-cites="4Hh1wpwV">(<a href="#ref-4Hh1wpwV" role="doc-biblioref">1155</a>)</span>.
However, several attempts to develop IWV vaccines against both SARS-CoV-1 and MERS-CoV were hindered by incidences of vaccine-associated disease enhancement (VADE) in preclinical studies <span class="citation" data-cites="4AwyoMvQ">(<a href="#ref-4AwyoMvQ" role="doc-biblioref">1156</a>)</span>.
In one example of a study in macaques, an inactivated SARS-CoV-1 vaccine induced even more severe lung damage than the virus due to an enhanced immune reaction <span class="citation" data-cites="ZXAfLbxM">(<a href="#ref-ZXAfLbxM" role="doc-biblioref">1157</a>)</span>.
Independent studies in mice also demonstrated evidence of lung immunopathology due to VADE in response to MERS-CoV IWV vaccination <span class="citation" data-cites="ihrfEtMq 8qw9OBKX">(<a href="#ref-ihrfEtMq" role="doc-biblioref">1158</a>, <a href="#ref-8qw9OBKX" role="doc-biblioref">1159</a>)</span>.
The exact mechanisms responsible for VADE remain elusive due to the specificity of the virus-host interactions involved, but VADE is the subject of investigation in preclinical SARS-CoV-2 vaccine studies to ensure the safety of any potential vaccines that may reach phase I trials and beyond <span class="citation" data-cites="4AwyoMvQ">(<a href="#ref-4AwyoMvQ" role="doc-biblioref">1156</a>)</span>.</p>
<h3 data-number="6.5.1" id="application-to-covid-19-1"><span class="header-section-number">6.5.1</span> Application to COVID-19</h3>
<div id="tbl:approved-whole-virus" class="tablenos">
<table id="tbl:approved-whole-virus">
<caption><span>Table 2:</span> Inactivated whole-virus vaccines approved in at least one country <span class="citation" data-cites="jswAyWIs">(<a href="#ref-jswAyWIs" role="doc-biblioref">1160</a>)</span> as of December 2, 2022.
</caption>
<colgroup>
<col style="width: 30%" />
<col style="width: 69%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Vaccine</th>
<th style="text-align: left;">Company</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Covaxin</td>
<td style="text-align: left;">Bharat Biotech</td>
</tr>
<tr class="even">
<td style="text-align: left;">KoviVac</td>
<td style="text-align: left;">Chumakov Center</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Turkovac</td>
<td style="text-align: left;">Health Institutes of Turkey</td>
</tr>
<tr class="even">
<td style="text-align: left;">FAKHRAVAC (MIVAC)</td>
<td style="text-align: left;">Organization of Defensive Innovation and Research</td>
</tr>
<tr class="odd">
<td style="text-align: left;">QazVac</td>
<td style="text-align: left;">Research Institute for Biological Safety Problems (RIBSP)</td>
</tr>
<tr class="even">
<td style="text-align: left;">KCONVAC</td>
<td style="text-align: left;">Shenzhen Kangtai Biological Products Co</td>
</tr>
<tr class="odd">
<td style="text-align: left;">COVIran Barekat</td>
<td style="text-align: left;">Shifa Pharmed Industrial Co</td>
</tr>
<tr class="even">
<td style="text-align: left;">Covilo</td>
<td style="text-align: left;">Sinopharm (Beijing)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Inactivated (Vero Cells)</td>
<td style="text-align: left;">Sinopharm (Wuhan)</td>
</tr>
<tr class="even">
<td style="text-align: left;">CoronaVac</td>
<td style="text-align: left;">Sinovac</td>
</tr>
<tr class="odd">
<td style="text-align: left;">VLA2001</td>
<td style="text-align: left;">Valneva</td>
</tr>
</tbody>
</table>
</div>
<p>Several whole-virus vaccines have been developed against COVID-19 and are available in countries around the world (Table <a href="#tbl:approved-whole-virus">2</a>).
As of December 18, 2022, 10 vaccines developed with IWV technology are being distributed in 118 countries (Figure <a href="#fig:iwv-distrib">6</a>).</p>
<div id="fig:iwv-distrib" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/dfc6ff85488d57cbb52f4b2324531be6afaa73f4/owiddata/maps/inactivated.png" alt="Figure 6: Worldwide availability of vaccines developed using inactivated whole viruses. This figure reflects the number of vaccines based on whole inactivated virus technology that were available in each country as of December 18, 2022. These data are retrieved from Our World in Data (532) and plotted using geopandas (1161). See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily." />
<figcaption aria-hidden="true"><span>Figure 6:</span> <strong>Worldwide availability of vaccines developed using inactivated whole viruses.</strong>
This figure reflects the number of vaccines based on whole inactivated virus technology that were available in each country as of December 18, 2022.
These data are retrieved from Our World in Data <span class="citation" data-cites="3cPwqjhj">(<a href="#ref-3cPwqjhj" role="doc-biblioref">532</a>)</span> and plotted using geopandas <span class="citation" data-cites="iGEyNO42">(<a href="#ref-iGEyNO42" role="doc-biblioref">1161</a>)</span>.
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.</figcaption>
</figure>
</div>
<p>One of those, CoronaVac, was developed by Beijing-based biopharmaceutical company Sinovac.
The developers inactivated a SARS-CoV-2 strain collected in China with β-propiolactone and propagated it using Vero cells <span class="citation" data-cites="7RHpaAHu">(<a href="#ref-7RHpaAHu" role="doc-biblioref">1129</a>)</span>.
The vaccine is coupled with an aluminum adjuvant to increase immunogenicity <span class="citation" data-cites="7RHpaAHu">(<a href="#ref-7RHpaAHu" role="doc-biblioref">1129</a>)</span>.
In phase I and II clinical trials, CoronaVac elicited a strong immunogenic response in animal models and the development of nAbs in human participants <span class="citation" data-cites="Ozya5HP5 14fILrRWg N1txjPtt">(<a href="#ref-Ozya5HP5" role="doc-biblioref">1162</a>–<a href="#ref-N1txjPtt" role="doc-biblioref">1164</a>)</span>.
Administration followed a prime-boost regimen using a 0.5 ml dose containing 3 μg of inactivated SARS-CoV-2 virus per dose <span class="citation" data-cites="1A5wiKQAW">(<a href="#ref-1A5wiKQAW" role="doc-biblioref">1165</a>)</span>.</p>
<p>Results from a two-dose phase III trial following a 14-day prime boost became available in late 2020 <span class="citation" data-cites="1FF7JOwSH">(<a href="#ref-1FF7JOwSH" role="doc-biblioref">1166</a>)</span>, and an interim analysis identified specific IgG nAbs against the S1 RBD and a robust IFN-γ secreting T cell response was induced via immunization with CoronaVac <span class="citation" data-cites="UERG6dAd">(<a href="#ref-UERG6dAd" role="doc-biblioref">1167</a>)</span>.
Safety analysis of the CoronaVac vaccine during the phase II trial revealed that most adverse reactions were either mild (grade 1) or moderate (grade 2) in severity.
In adults aged 18 to 59 years receiving a variety of dosage schedules, site injection pain was consistently the most common symptom reported <span class="citation" data-cites="N1txjPtt">(<a href="#ref-N1txjPtt" role="doc-biblioref">1164</a>)</span>.
In older adults, the most common local and systemic reactions were pain at the injection site (9%) and fever (3%), respectively <span class="citation" data-cites="Ozya5HP5">(<a href="#ref-Ozya5HP5" role="doc-biblioref">1162</a>)</span>.
In phase III trials, minimal side effects were reported <span class="citation" data-cites="1FF7JOwSH">(<a href="#ref-1FF7JOwSH" role="doc-biblioref">1166</a>)</span>.</p>
<p>Estimates of CoronaVac’s VE have varied across trials.
CoronaVac demonstrated an efficacy of a little over 50% in Brazil, which was contested by Turkish officials claiming an efficacy of 91.25%, but ultimately after multiple announcements, the efficacy debate was settled at over 50% <span class="citation" data-cites="18mREgqUz V7MXd4X4">(<a href="#ref-18mREgqUz" role="doc-biblioref">1168</a>, <a href="#ref-V7MXd4X4" role="doc-biblioref">1169</a>)</span>.
Subsequently, an interim analysis of the phase III randomized placebo-controlled trials conducted in Turkey enrolling 10,214 participants (approximately 2:1 vaccine:placebo) indicated efficacy of 83.5%, with minimal side effects reported <span class="citation" data-cites="1FF7JOwSH">(<a href="#ref-1FF7JOwSH" role="doc-biblioref">1166</a>)</span>, and a prospective national cohort study in Chile reported an adjusted estimated effectiveness of 66% for the prevention of COVID-19 with an estimated 90% and 87% prevention of hospitalization and death, respectively <span class="citation" data-cites="q2wQJULu">(<a href="#ref-q2wQJULu" role="doc-biblioref">1170</a>)</span>.</p>
<p>CoronaVac was first approved in China and has now been distributed in 65 countries across Africa, Asia, Europe, North America, and South America, including Brazil, Cambodia, Chile, Colombia, Laos, Malaysia, Mexico, Turkey, Ukraine, and Uruguay <span class="citation" data-cites="wByD9WaX sRy6js2o">(<a href="#ref-sRy6js2o" role="doc-biblioref">1115</a>, <a href="#ref-wByD9WaX" role="doc-biblioref">1171</a>)</span>.
As of August 2021, Sinovac had reportedly produced over a billion doses of CoronaVac <span class="citation" data-cites="wByD9WaX">(<a href="#ref-wByD9WaX" role="doc-biblioref">1171</a>)</span>.</p>
<p>Two additional inactivated vaccine candidates were developed following a similar approach by the state-owned China National Pharmaceutical Group Co., Ltd., more commonly known as Sinopharm CNBG.
One, known as BBIBP-CorV or Covilo, was developed in Beijing using the HB02 variant of SARS-CoV-2.
At Sinopharm CNBG’s Wuhan Institute, a second vaccine was developed using the WIV04 variant of SARS-CoV-2 <span class="citation" data-cites="miMRIMwa">(<a href="#ref-miMRIMwa" role="doc-biblioref">1172</a>)</span>.
The viruses were purified, propagated using Vero cells, isolated, and inactivated using β-propiolactone <span class="citation" data-cites="VlnLw2HV miMRIMwa">(<a href="#ref-miMRIMwa" role="doc-biblioref">1172</a>, <a href="#ref-VlnLw2HV" role="doc-biblioref">1173</a>)</span>.
Both vaccines are adjuvanted with aluminum hydroxide <span class="citation" data-cites="VlnLw2HV miMRIMwa">(<a href="#ref-miMRIMwa" role="doc-biblioref">1172</a>, <a href="#ref-VlnLw2HV" role="doc-biblioref">1173</a>)</span>.</p>
<p>Preclinical studies indicated that Covilo induced sufficient nAb titers in mice, and a prime-boost immunization scheme of 2 μg/dose was sufficient to protect rhesus macaques from disease <span class="citation" data-cites="VlnLw2HV">(<a href="#ref-VlnLw2HV" role="doc-biblioref">1173</a>)</span>.
For the other vaccine, nAbs were detected in all groups 14 days after the final dose in the phase I part of the trial <span class="citation" data-cites="T3MYavsH">(<a href="#ref-T3MYavsH" role="doc-biblioref">1174</a>)</span>, with similar findings reported in interim phase II data <span class="citation" data-cites="T3MYavsH">(<a href="#ref-T3MYavsH" role="doc-biblioref">1174</a>)</span>.
In phase II trials, the BBIBP-CorV vaccine appeared well-tolerated, with 23% of participants in the vaccine condition (482 total participants, 3:1, vaccine:placebo) reporting at least one adverse reaction characterized as mild to moderate <span class="citation" data-cites="fPGgVKYL">(<a href="#ref-fPGgVKYL" role="doc-biblioref">1175</a>)</span>.
In phase III trials, Sinopharm CNBG’s Covilo vaccine made from the WIV04 strain achieved an efficacy of 72.8% and was well tolerated <span class="citation" data-cites="yN5KOfvE">(<a href="#ref-yN5KOfvE" role="doc-biblioref">1176</a>)</span>.
Sinopharm affiliates in the UAE in early December 2020 claimed the vaccine had 86% efficacy, which was later at odds with a Sinopharm Beijing affiliate that stated that Covilo had a 79.34% efficacy later that same month <span class="citation" data-cites="w7gO6yGn">(<a href="#ref-w7gO6yGn" role="doc-biblioref">1177</a>)</span>.</p>
<p>Other vaccine development programs have been led through industry partnerships with governmental organizations.
Bharat Biotech International Ltd., which is the biggest producer of vaccines globally, collaborated with the Indian Council of Medical Research (ICMR) National Institute of Virology (NIV) to develop another IWV, Covaxin, which is also known as BBV152.
Preclinical studies of Covaxin in hamsters <span class="citation" data-cites="bcGxW9fA">(<a href="#ref-bcGxW9fA" role="doc-biblioref">1178</a>)</span> and macaques <span class="citation" data-cites="GgdjKrYi">(<a href="#ref-GgdjKrYi" role="doc-biblioref">1179</a>)</span> indicated that the vaccine induced protective responses deemed sufficient to move forward to human trials.
Phase I (July 2020) and phase I/II (September to October 2020) studies indicated that Covaxin adjuvanted with alum and a Toll-like receptor 7/8 (TLR7/8) agonist was safe and immunogenic <span class="citation" data-cites="CGuGeB7m GxQSMH5l">(<a href="#ref-CGuGeB7m" role="doc-biblioref">1180</a>, <a href="#ref-GxQSMH5l" role="doc-biblioref">1181</a>)</span>.
These two studies demonstrated that the vaccine induced significant humoral and cell-mediated responses, as assessed by measuring binding <span class="citation" data-cites="CGuGeB7m">(<a href="#ref-CGuGeB7m" role="doc-biblioref">1180</a>)</span> and neutralising <span class="citation" data-cites="CGuGeB7m GxQSMH5l">(<a href="#ref-CGuGeB7m" role="doc-biblioref">1180</a>, <a href="#ref-GxQSMH5l" role="doc-biblioref">1181</a>)</span> antibodies, cytokines <span class="citation" data-cites="CGuGeB7m GxQSMH5l">(<a href="#ref-CGuGeB7m" role="doc-biblioref">1180</a>, <a href="#ref-GxQSMH5l" role="doc-biblioref">1181</a>)</span>, CD3^+, CD4^+, and CD8^+ T-cells <span class="citation" data-cites="CGuGeB7m">(<a href="#ref-CGuGeB7m" role="doc-biblioref">1180</a>)</span>, with some formulations also eliciting Th1-skewed memory T-cell responses <span class="citation" data-cites="GxQSMH5l">(<a href="#ref-GxQSMH5l" role="doc-biblioref">1181</a>)</span>.
Only mild to moderate side-effects were reported upon immunization <span class="citation" data-cites="CGuGeB7m GxQSMH5l">(<a href="#ref-CGuGeB7m" role="doc-biblioref">1180</a>, <a href="#ref-GxQSMH5l" role="doc-biblioref">1181</a>)</span>.
In July 2021, Covaxin’s overall vaccine efficacy was estimated at 77.8% for the prevention of COVID-19 based on a final enrollment of 25,798 people (approximately 1:1 vaccine:placebo) <span class="citation" data-cites="n7BupOQ6">(<a href="#ref-n7BupOQ6" role="doc-biblioref">1182</a>)</span>.
Only mild to moderate side-effects reported upon immunization <span class="citation" data-cites="CGuGeB7m GxQSMH5l">(<a href="#ref-CGuGeB7m" role="doc-biblioref">1180</a>, <a href="#ref-GxQSMH5l" role="doc-biblioref">1181</a>)</span>, and in phase II trials, the BBIBP-CorV vaccine appeared well-tolerated, with 23% of participants in the vaccine condition (482 total participants, 3:1, vaccine:placebo) reporting at least one adverse reaction characterized as mild to moderate <span class="citation" data-cites="fPGgVKYL">(<a href="#ref-fPGgVKYL" role="doc-biblioref">1175</a>)</span>.
As of September 2021, Covaxin has been approved for emergency use in 30 countries across Africa, Asia, Europe, and South America, including Guyana, India, Iran, Zimbabwe, Nepal, Mauritius, Mexico, Nepal, Paraguay, and the Philippines <span class="citation" data-cites="19tYVbg8H">(<a href="#ref-19tYVbg8H" role="doc-biblioref">1183</a>)</span>.</p>
<p>Although IWV vaccine candidates were well-tolerated overall, both Sinovac’s CoronaVac and Sinopharm’s inactivated WIV04 vaccine trials were affected by concerns about adverse events.
In CoronaVac’s trial of adults 18-59, 2% (n=7) of participants reported severe adverse events <span class="citation" data-cites="Ozya5HP5">(<a href="#ref-Ozya5HP5" role="doc-biblioref">1162</a>)</span>, causing the trial to be halted for investigation <span class="citation" data-cites="aq22z8M7">(<a href="#ref-aq22z8M7" role="doc-biblioref">1184</a>)</span>.
They were determined to be unrelated to the vaccine <span class="citation" data-cites="Ozya5HP5 aq22z8M7">(<a href="#ref-Ozya5HP5" role="doc-biblioref">1162</a>, <a href="#ref-aq22z8M7" role="doc-biblioref">1184</a>)</span>, which is now widely distributed.
Similarly, a trial of the Sinopharm WIV04 vaccine in Peru was briefly paused due to safety concerns in relation to neurological symptoms <span class="citation" data-cites="d09adg1G">(<a href="#ref-d09adg1G" role="doc-biblioref">1185</a>)</span>, but this was later deemed unrelated to the vaccine, and the trial continued <span class="citation" data-cites="nxEJTLGU">(<a href="#ref-nxEJTLGU" role="doc-biblioref">1186</a>)</span>.</p>
<h3 data-number="6.5.2" id="real-world-safety-and-efficacy"><span class="header-section-number">6.5.2</span> Real-World Safety and Efficacy</h3>
<p>In the past, problems that arose during the manufacturing of IWV vaccines could present safety issues, but oversight of the manufacturing process has helped to improve IWV vaccine safety <span class="citation" data-cites="1Fybz4oKI">(<a href="#ref-1Fybz4oKI" role="doc-biblioref">1187</a>)</span>.
Nevertheless, the departure from norms necessitated by the COVID-19 crisis raised concerns about whether oversight would occur at pre-pandemic standards <span class="citation" data-cites="1Fybz4oKI">(<a href="#ref-1Fybz4oKI" role="doc-biblioref">1187</a>)</span>.
In general, the IWV COVID-19 vaccines have reported very few issues with safety.
Additionally, safety audits have proactively identified concerns.
For example, in April 2022, the WHO suspended procurement of Covaxin from Bharat Pharmaceuticals due to concerns about deviation from good manufacturing practice in their production facilities <span class="citation" data-cites="19hWVUQQ">(<a href="#ref-19hWVUQQ" role="doc-biblioref">1188</a>)</span>.
However, no safety issues had been reported in association with this vaccine.
Rare cases of VADE have been reported in association with CoronaVac <span class="citation" data-cites="14KuNOjDi">(<a href="#ref-14KuNOjDi" role="doc-biblioref">1189</a>)</span>.</p>
<p>More concern has arisen around the issue of efficacy.
A major consideration in vaccine development is that vaccines lose efficacy as mutations accumulate in the epitopes of the circulating virus; IWV vaccines may be particularly affected by viral evolution <span class="citation" data-cites="K0Ltu31S">(<a href="#ref-K0Ltu31S" role="doc-biblioref">1124</a>)</span>.
This loss of specificity over time is likely to be influenced by the evolution of the virus, and specifically by the rate of evolution in the region of the genome that codes for the antigenic spike protein.
The Beta variant appears to be more resistant to nAbs in sera from individuals immunized with Sinovac than the Alpha variant or wildtype virus, indicating that emerging variants may be of concern <span class="citation" data-cites="s2O5iyCV">(<a href="#ref-s2O5iyCV" role="doc-biblioref">1190</a>)</span>.
In agreement with previous studies demonstrating sera from individuals vaccinated with Covaxin efficiently neutralized the Alpha variant (B.1.1.7) and the Delta variant (B.1.617.2) <span class="citation" data-cites="lr7INjf6 UYE3NvU4 IvOEe7bV">(<a href="#ref-lr7INjf6" role="doc-biblioref">1191</a>–<a href="#ref-IvOEe7bV" role="doc-biblioref">1193</a>)</span>, the phase III trial reported a 65.2% efficacy against the Delta variant (B.1.617.2) <span class="citation" data-cites="1DD2pssEJ">(<a href="#ref-1DD2pssEJ" role="doc-biblioref">1194</a>)</span>.
Another study reported that sera from individuals immunized with Covaxin produced effective nAbs against the Delta variant and the so-called Delta plus variant (AY.1) <span class="citation" data-cites="Pir4RpLb">(<a href="#ref-Pir4RpLb" role="doc-biblioref">1195</a>)</span>.
Indeed, sera obtained from Covaxin boosted individuals (n=13) <span class="citation" data-cites="r5HwkOx8">(<a href="#ref-r5HwkOx8" role="doc-biblioref">1196</a>)</span> and those who were vaccinated with Covaxin but recovered from a breakthrough infection (n=31) also neutralized the Omicron variant <span class="citation" data-cites="CqXCzqeS">(<a href="#ref-CqXCzqeS" role="doc-biblioref">1197</a>)</span>.</p>
<p>Notably, a preprint reported that antisera (i.e., the antibody-containing component of the sera) from 12 people immunized with BBIBP-CorV exhibited nAb capacity against the Beta variant (B.1.351), wild type SARS-CoV-2 (NB02), and one of the original variants of SARS-CoV-2 (D614G) <span class="citation" data-cites="w9KwrmQT">(<a href="#ref-w9KwrmQT" role="doc-biblioref">1198</a>)</span>.
Another preprint including sera from 282 participants used a surrogate neutralizing assay, a test that generally correlates with nAbs, to determine that Covilo appears to induce nAbs against the binding of the RBD of wild type SARS-CoV-2 and the Alpha, Beta, and Delta variants to ACE2 <span class="citation" data-cites="GZ5Sf8Yd">(<a href="#ref-GZ5Sf8Yd" role="doc-biblioref">1199</a>)</span>.
Indeed, a study showed that the Alpha variant exhibited very little resistance to neutralization by sera of those immunized with Covilo, but the Beta variant was more resistant to neutralization by almost a factor of 3 <span class="citation" data-cites="s2O5iyCV">(<a href="#ref-s2O5iyCV" role="doc-biblioref">1190</a>)</span>.
The authors noted that no evidence of VADE was detected using this vaccine in phase II data <span class="citation" data-cites="T3MYavsH">(<a href="#ref-T3MYavsH" role="doc-biblioref">1174</a>)</span>.</p>
<p>Concern was raised about the efficacy of CoronaVac following reports that over 350 doctors became ill with COVID-19 in Indonesia despite being immunized with CoronaVac <span class="citation" data-cites="fs7G9HyV">(<a href="#ref-fs7G9HyV" role="doc-biblioref">1200</a>)</span>.
In addition to concerns raised by the evolution of SARS-CoV-2, it is important to consider the duration of immunity over time.
Studies are underway to determine whether a booster immunization is required for several IWV vaccines, including CoronaVac <span class="citation" data-cites="1GcPxd9Bn">(<a href="#ref-1GcPxd9Bn" role="doc-biblioref">1201</a>)</span> and Covaxin <span class="citation" data-cites="s1TGwKbT">(<a href="#ref-s1TGwKbT" role="doc-biblioref">1202</a>)</span>.
A phase I/II clinical trial of CoronaVac in an elderly cohort (adults 60 years and older) in China determined that by 6 to 8 months following the second dose, nAb titers were detected below the seropositive cutoff <span class="citation" data-cites="AcxNwvVQ">(<a href="#ref-AcxNwvVQ" role="doc-biblioref">1203</a>)</span>.
One preprint has reported that 6 months after the second vaccination, a booster dose of CoronaVac markedly increased geometric mean titers of SARS-CoV-2 nAbs <span class="citation" data-cites="1BPnaMPs4">(<a href="#ref-1BPnaMPs4" role="doc-biblioref">1204</a>)</span>.
However, the reduction of nAbs was ameliorated by a booster dose administered 8 months after the second CoronaVac dose.
A preprint study of healthcare workers in China has since indicated that a booster shot of Covilo elevates B cell and T cell responses and increases nAb titers <span class="citation" data-cites="QHtyW0Jz">(<a href="#ref-QHtyW0Jz" role="doc-biblioref">1205</a>)</span>.
In May 2021, the UAE announced it would consider booster shots for all citizens who had been immunized with Covilo, which was shortly followed by a similar announcement in Bahrain, and by August 29, 2021, the UAE mandated booster shots for all residents who had received Covilo <span class="citation" data-cites="wByD9WaX">(<a href="#ref-wByD9WaX" role="doc-biblioref">1171</a>)</span>.</p>
<p>Additionally, heterologous vaccine boosters are also being considered in many cases.
Indeed, Chinese <span class="citation" data-cites="9oJ3sbrk">(<a href="#ref-9oJ3sbrk" role="doc-biblioref">1206</a>)</span> and Chilean <span class="citation" data-cites="uPt61a0E">(<a href="#ref-uPt61a0E" role="doc-biblioref">1207</a>)</span> researchers have opted to investigate options to administer different vaccines (e.g., an mRNA vaccine dose) as a booster dose to individuals who have already received two doses of the IWV vaccine CoronaVac.
Another study determined that using a viral-vectored vaccine (CanSino’s Convidecia) or an mRNA vaccine (Pfizer/BioNTech’s BNT162b2) instead of CoronaVac in a prime-boost vaccination regimen could induce a more robust immune response <span class="citation" data-cites="hcnFsRig 1Cd41ucny">(<a href="#ref-hcnFsRig" role="doc-biblioref">1208</a>, <a href="#ref-1Cd41ucny" role="doc-biblioref">1209</a>)</span>.
Today, the WHO has developed recommendations for booster immunization for several whole-virus vaccines.
In some cases (Valneva-VLA2001 <span class="citation" data-cites="wN3MgkJc">(<a href="#ref-wN3MgkJc" role="doc-biblioref">1210</a>)</span>, Covaxin <span class="citation" data-cites="Sis3GVtq">(<a href="#ref-Sis3GVtq" role="doc-biblioref">1211</a>)</span>, Covilo <span class="citation" data-cites="huSLgEcW">(<a href="#ref-huSLgEcW" role="doc-biblioref">1212</a>)</span>, Sinopharm-WIBP Inactivated (Vero Cell) <span class="citation" data-cites="QTUuPhrc">(<a href="#ref-QTUuPhrc" role="doc-biblioref">1213</a>)</span>), boosters are recommended only for high-risk and/or high-priority groups (e.g., the immunocompromised and medical professionals, respectively), while for Sinovac-CoronaVac <span class="citation" data-cites="nc4lLcIg">(<a href="#ref-nc4lLcIg" role="doc-biblioref">1214</a>)</span>, they are recommended more broadly.
Studies are also investigating the effects of booster doses in other vaccines <span class="citation" data-cites="QWRc7Ob DuPE0SrS keNQQxsq">(<a href="#ref-QWRc7Ob" role="doc-biblioref">1215</a>–<a href="#ref-keNQQxsq" role="doc-biblioref">1217</a>)</span>, though some are being investigated or deployed primarily as heterologous boosters in populations vaccinated with a different primary series <span class="citation" data-cites="20kWCMZS VYzX6ru8">(<a href="#ref-DuPE0SrS" role="doc-biblioref">1216</a>)</span>.
As new vaccines are approved by the WHO, more time elapses since many received the primary series, and new variants emerge, booster recommendations are likely to increase.</p>
<h2 data-number="6.6" id="subunit-vaccines"><span class="header-section-number">6.6</span> Subunit Vaccines</h2>
<p>Efforts to overcome the limitations of live-virus vaccines led to the development of approaches first to inactivate viruses (circa 1900), leading to IWV vaccines, and then to purifying proteins from viruses cultured in eggs (circa 1920) <span class="citation" data-cites="YY3x3bBV dggZoRQD">(<a href="#ref-YY3x3bBV" role="doc-biblioref">1113</a>, <a href="#ref-dggZoRQD" role="doc-biblioref">1220</a>)</span>.
The purification of proteins then set the stage for the development of subunit vaccines based on the principle that the immune system can be stimulated by introducing one or more proteins or peptides isolated from the virus.
Today, such approaches often use antigens isolated from the surface of the viral particle that are key targets of the immune system (protein subunit vaccines).
Advances in biological engineering have also facilitated the development of approaches like viral-like particle (VLP) vaccines using nanotechnology <span class="citation" data-cites="99C1xJ2E">(<a href="#ref-99C1xJ2E" role="doc-biblioref">1221</a>)</span>.
VLPs share the conformation of a virus’s capsid, thereby acting as an antigen, but lack the replication machinery <span class="citation" data-cites="lH2HMMZi">(<a href="#ref-lH2HMMZi" role="doc-biblioref">1222</a>)</span>.
Both protein subunit and VLP vaccines thus mimic the principle of whole virus vaccines but lack the genetic material required for replication, removing the risk of infection <span class="citation" data-cites="1Bxg7Wj6w">(<a href="#ref-1Bxg7Wj6w" role="doc-biblioref">1223</a>)</span>.</p>
<p>Unlike whole-virus vaccines, which introduce the whole virus, subunit vaccines stimulate the immune system by introducing one or more proteins or peptides of the virus that have been isolated.
The main advantage of this platform is that subunit vaccines are considered very safe, as the antigen alone cannot cause an infection <span class="citation" data-cites="1FfwyYaj7">(<a href="#ref-1FfwyYaj7" role="doc-biblioref">1224</a>)</span>.
Both protein subunit and VLP vaccines thus mimic the principle of whole virus vaccines but lack the genetic material required for replication, removing the risk of infection <span class="citation" data-cites="1Bxg7Wj6w">(<a href="#ref-1Bxg7Wj6w" role="doc-biblioref">1223</a>)</span>.
Protein subunit vaccines can stimulate antibodies and CD4<sup>+</sup> T-cell responses <span class="citation" data-cites="12eGVhH5I lH2HMMZi">(<a href="#ref-lH2HMMZi" role="doc-biblioref">1222</a>, <a href="#ref-12eGVhH5I" role="doc-biblioref">1225</a>)</span>.</p>
<p>The subunit approach is also favored for its consistency in production.
The components can be designed for a highly targeted immune response to a specific pathogen using synthetic immunogenic particles, allowing the vaccine to be engineered to avoid allergen and reactogenic sequences <span class="citation" data-cites="124bnGvPp">(<a href="#ref-124bnGvPp" role="doc-biblioref">1226</a>)</span>.
One limitation is that, in the case of protein subunit vaccines, adjuvants are usually required to boost the immune response <span class="citation" data-cites="mv42t1HV">(<a href="#ref-mv42t1HV" role="doc-biblioref">1227</a>)</span> (see online Appendix <span class="citation" data-cites="yTsmmAYC">(<a href="#ref-yTsmmAYC" role="doc-biblioref">1139</a>)</span>).
Adjuvants, which are compounds that elicit an immunogenic effect, include alum (aluminum hydroxide), squalene- or saponin-based adjuvants, and Freund’s incomplete/complete adjuvants, although the latter is avoided in human and veterinary medicine due to high toxicity <span class="citation" data-cites="rioTBsLc 124bnGvPp vrNgFknF">(<a href="#ref-124bnGvPp" role="doc-biblioref">1226</a>, <a href="#ref-rioTBsLc" role="doc-biblioref">1228</a>, <a href="#ref-vrNgFknF" role="doc-biblioref">1229</a>)</span>.</p>
<div id="tbl:approved-subunit" class="tablenos">
<table id="tbl:approved-subunit">
<caption><span>Table 3:</span> Subunit vaccines approved for use in at least one country <span class="citation" data-cites="jswAyWIs">(<a href="#ref-jswAyWIs" role="doc-biblioref">1160</a>)</span> as of December 2, 2022.
</caption>
<colgroup>
<col style="width: 35%" />
<col style="width: 50%" />
<col style="width: 14%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Vaccine</th>
<th style="text-align: left;">Company</th>
<th style="text-align: left;">Platform</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Zifivax</td>
<td style="text-align: left;">Anhui Zhifei Longcom</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="even">
<td style="text-align: left;">Noora vaccine</td>
<td style="text-align: left;">Bagheiat-allah University of Medical Sciences</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Corbevax</td>
<td style="text-align: left;">Biological E Limited</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="even">
<td style="text-align: left;">Abdala</td>
<td style="text-align: left;">Center for Genetic Engineering and Biotechnology (CIGB)</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Soberana 02</td>
<td style="text-align: left;">Instituto Finlay de Vacunas Cuba</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="even">
<td style="text-align: left;">Soberana Plus</td>
<td style="text-align: left;">Instituto Finlay de Vacunas Cuba</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="odd">
<td style="text-align: left;">V-01</td>
<td style="text-align: left;">Livzon Mabpharm Inc</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="even">
<td style="text-align: left;">Covifenz</td>
<td style="text-align: left;">Medicago</td>
<td style="text-align: left;">VLP</td>
</tr>
<tr class="odd">
<td style="text-align: left;">MVC-COV1901</td>
<td style="text-align: left;">Medigen</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="even">
<td style="text-align: left;">Recombinant SARS-CoV-2 Vaccine (CHO Cell)</td>
<td style="text-align: left;">National Vaccine and Serum Institute</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Nuvaxovid</td>
<td style="text-align: left;">Novavax</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="even">
<td style="text-align: left;">IndoVac</td>
<td style="text-align: left;">PT Bio Farma</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Razi Cov Pars</td>
<td style="text-align: left;">Razi Vaccine and Serum Research Institute</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="even">
<td style="text-align: left;">VidPrevtyn Beta</td>
<td style="text-align: left;">Sanofi/GSK</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="odd">
<td style="text-align: left;">COVOVAX (Novavax formulation)</td>
<td style="text-align: left;">Serum Institute of India</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="even">
<td style="text-align: left;">SKYCovione</td>
<td style="text-align: left;">SK Bioscience Co Ltd</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="odd">
<td style="text-align: left;">TAK-019 (Novavax formulation)</td>
<td style="text-align: left;">Takeda</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="even">
<td style="text-align: left;">SpikoGen</td>
<td style="text-align: left;">Vaxine/CinnaGen Co.</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Aurora-CoV</td>
<td style="text-align: left;">Vector State Research Center of Virology and Biotechnology</td>
<td style="text-align: left;">protein subunit</td>
</tr>
<tr class="even">
<td style="text-align: left;">EpiVacCorona</td>
<td style="text-align: left;">Vector State Research Center of Virology and Biotechnology</td>
<td style="text-align: left;">protein subunit</td>
</tr>
</tbody>
</table>
</div>
<p>Protein subunit vaccine development efforts for both SARS-CoV-1 and MERS-CoV usually focused on the immunogenic RBD of the S protein <span class="citation" data-cites="cLAQnckq 1EirBATaN 1AOG59epD">(<a href="#ref-cLAQnckq" role="doc-biblioref">1230</a>–<a href="#ref-1AOG59epD" role="doc-biblioref">1232</a>)</span>.
The search for a potential SARS-CoV-1 vaccine included the development of vaccines targeting the full-length or trimeric S protein <span class="citation" data-cites="Ow2ICHez tzZeWNPV">(<a href="#ref-Ow2ICHez" role="doc-biblioref">1233</a>, <a href="#ref-tzZeWNPV" role="doc-biblioref">1234</a>)</span>, those focused on the RBD protein only <span class="citation" data-cites="1EirBATaN DsfTQFmb 1AOG59epD cLAQnckq">(<a href="#ref-cLAQnckq" role="doc-biblioref">1230</a>–<a href="#ref-1AOG59epD" role="doc-biblioref">1232</a>, <a href="#ref-DsfTQFmb" role="doc-biblioref">1235</a>)</span> or non-RBD S protein fragments <span class="citation" data-cites="tzZeWNPV IYjNaaqv">(<a href="#ref-tzZeWNPV" role="doc-biblioref">1234</a>, <a href="#ref-IYjNaaqv" role="doc-biblioref">1236</a>)</span>, and those targeting the N and M proteins <span class="citation" data-cites="HvO79P9u VUcwpJKL 7hbgOaiE">(<a href="#ref-HvO79P9u" role="doc-biblioref">1237</a>–<a href="#ref-7hbgOaiE" role="doc-biblioref">1239</a>)</span>.
These efforts have been thoroughly reviewed elsewhere <span class="citation" data-cites="9Zv0eLa9">(<a href="#ref-9Zv0eLa9" role="doc-biblioref">1240</a>)</span>.
There have been examples of successful preclinical research including candidate RBD219N-1, a 218-amino-acid residue of the SARS-CoV-1 RBD that, when adjuvanted to aluminum hydroxide, was capable of eliciting a high antibody response of both nAbs and RBD-specific mAbs in both pseudovirus and live virus infections of immunized mice <span class="citation" data-cites="8723Jsa">(<a href="#ref-8723Jsa" role="doc-biblioref">1241</a>)</span>.</p>
<p>Similarly to the SARS-CoV-1 vaccine candidates, the MERS-CoV protein subunit vaccine candidates generally target the RBD <span class="citation" data-cites="9Zv0eLa9 1EirBATaN Mki0DaYb aApaHV1w deUFGhNI sfM5QV3m">(<a href="#ref-1EirBATaN" role="doc-biblioref">1231</a>, <a href="#ref-9Zv0eLa9" role="doc-biblioref">1240</a>, <a href="#ref-Mki0DaYb" role="doc-biblioref">1242</a>–<a href="#ref-sfM5QV3m" role="doc-biblioref">1245</a>)</span>, with some targeting the full length S protein <span class="citation" data-cites="oghHqZDt">(<a href="#ref-oghHqZDt" role="doc-biblioref">1246</a>)</span>, non-RBD protein fragments such as the SP3 peptide <span class="citation" data-cites="11Zz9H0Dl">(<a href="#ref-11Zz9H0Dl" role="doc-biblioref">1247</a>)</span>, and the recombinant N-terminal domain (rNTD) <span class="citation" data-cites="ZXsnsfvb">(<a href="#ref-ZXsnsfvb" role="doc-biblioref">1248</a>)</span>.
Other strategies investigating the potential use of the full length S DNA have also been investigated in mice and rhesus macaques, which elicited immune responses <span class="citation" data-cites="GurQD2dO">(<a href="#ref-GurQD2dO" role="doc-biblioref">1249</a>)</span>, but these responses were not as effective as the combination of S DNA and the S1 subunit protein together <span class="citation" data-cites="GurQD2dO T7W7hnB9">(<a href="#ref-GurQD2dO" role="doc-biblioref">1249</a>, <a href="#ref-T7W7hnB9" role="doc-biblioref">1250</a>)</span>.
No protein subunit vaccine for MERS-CoV has progressed beyond preclinical research to date.
VLPs have been investigated for development of vaccines against MERS and SARS <span class="citation" data-cites="oqty7gXw eHe78HXD">(<a href="#ref-oqty7gXw" role="doc-biblioref">1251</a>, <a href="#ref-eHe78HXD" role="doc-biblioref">1252</a>)</span> including testing in animal models <span class="citation" data-cites="cnYnzav2 G87TcArN">(<a href="#ref-cnYnzav2" role="doc-biblioref">1253</a>, <a href="#ref-G87TcArN" role="doc-biblioref">1254</a>)</span>, but once again, only preclinical data against HCoV has been collected <span class="citation" data-cites="jLJEygoA">(<a href="#ref-jLJEygoA" role="doc-biblioref">1255</a>)</span>.
However, protein subunit vaccines do play a role in public health and have contributed to vaccination against hepatitis B <span class="citation" data-cites="155fGivMy">(<a href="#ref-155fGivMy" role="doc-biblioref">1256</a>)</span> and pertussis <span class="citation" data-cites="CQog2bB7 1CYqHUt3n">(<a href="#ref-CQog2bB7" role="doc-biblioref">1257</a>, <a href="#ref-1CYqHUt3n" role="doc-biblioref">1258</a>)</span> since the 1980s and human papillomavirus since 2006 <span class="citation" data-cites="FVWwwB0d">(<a href="#ref-FVWwwB0d" role="doc-biblioref">1259</a>)</span>.
They are likely to continue to contribute to public health for the foreseeable future due to ongoing research in vaccines against influenza, SARS-CoV-2, Epstein-Barr virus, dengue virus, and human papillomavirus among others <span class="citation" data-cites="1FQlt5Lqz 8dSIiLCt aAYBP21H">(<a href="#ref-1FQlt5Lqz" role="doc-biblioref">1260</a>–<a href="#ref-aAYBP21H" role="doc-biblioref">1262</a>)</span>.</p>
<h3 data-number="6.6.1" id="application-to-covid-19-2"><span class="header-section-number">6.6.1</span> Application to COVID-19</h3>
<p>The development of subunit vaccines against SARS-CoV-2 is a remarkable achievement given the short period of time since the emergence of SARS-CoV-2 in late 2019, particularly considering these types of vaccines have not played a major role in previous pandemics compared to LAV and IWV vaccines.
More than 20 protein subunit vaccines from companies such as Sanofi/GlaxoSmithKline, Nanogen, and the Serum Institute of India have entered clinical trials for COVID-19 since the beginning of the pandemic <span class="citation" data-cites="8dSIiLCt">(<a href="#ref-8dSIiLCt" role="doc-biblioref">1261</a>)</span>, 20 have been approved, and at least 8 are being administered worldwide <span class="citation" data-cites="sRy6js2o NdbXBY1h">(<a href="#ref-sRy6js2o" role="doc-biblioref">1115</a>, <a href="#ref-NdbXBY1h" role="doc-biblioref">1263</a>)</span>.
As of December 18, 2022, protein subunit vaccines are being distributed in at least 42 countries (Figure <a href="#fig:ps-distrib">7</a>).</p>
<div id="fig:ps-distrib" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/dfc6ff85488d57cbb52f4b2324531be6afaa73f4/owiddata/maps/protein_subunit.png" alt="Figure 7: Worldwide availability of vaccines developed using protein subunit. This figure reflects the number of vaccines based on protein subunit technology that were available in each country as of December 18, 2022. These data are retrieved from Our World in Data (532, 1115) and plotted using geopandas (1161). See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily." />
<figcaption aria-hidden="true"><span>Figure 7:</span> <strong>Worldwide availability of vaccines developed using protein subunit.</strong>
This figure reflects the number of vaccines based on protein subunit technology that were available in each country as of December 18, 2022.
These data are retrieved from Our World in Data <span class="citation" data-cites="3cPwqjhj sRy6js2o">(<a href="#ref-3cPwqjhj" role="doc-biblioref">532</a>, <a href="#ref-sRy6js2o" role="doc-biblioref">1115</a>)</span> and plotted using geopandas <span class="citation" data-cites="iGEyNO42">(<a href="#ref-iGEyNO42" role="doc-biblioref">1161</a>)</span>.
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.</figcaption>
</figure>
</div>
<p>VLP vaccines have not progressed as rapidly.
Programs seeking to develop VLP vaccines have used either the full-length S protein or the RBD of the S protein specifically as an antigen, although some use several different SARS-CoV-2 proteins <span class="citation" data-cites="1FfwyYaj7">(<a href="#ref-1FfwyYaj7" role="doc-biblioref">1224</a>)</span>.
As of December 18, 2022, only one VLP was available in one country (Canada) <span class="citation" data-cites="sRy6js2o">(<a href="#ref-sRy6js2o" role="doc-biblioref">1115</a>)</span>.<!--Canada/Covifenz info is hardcoded--></p>
<div id="fig:vlp-distrib" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/dfc6ff85488d57cbb52f4b2324531be6afaa73f4/owiddata/maps/VLP.png" alt="Figure 8: Worldwide availability of vaccines developed with VLPs. This figure reflects the number of vaccines based on VLP technology that were available in each country as of December 18, 2022. These data are retrieved from Our World in Data (532) and plotted using geopandas (1161). See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily." />
<figcaption aria-hidden="true"><span>Figure 8:</span> <strong>Worldwide availability of vaccines developed with VLPs.</strong>
This figure reflects the number of vaccines based on VLP technology that were available in each country as of December 18, 2022.
These data are retrieved from Our World in Data <span class="citation" data-cites="3cPwqjhj">(<a href="#ref-3cPwqjhj" role="doc-biblioref">532</a>)</span> and plotted using geopandas <span class="citation" data-cites="iGEyNO42">(<a href="#ref-iGEyNO42" role="doc-biblioref">1161</a>)</span>.
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.</figcaption>
</figure>
</div>
<h3 data-number="6.6.2" id="nuvaxovid"><span class="header-section-number">6.6.2</span> Nuvaxovid</h3>
<p>One protein subunit vaccine for SARS-CoV-2 is NVX-CoV2373 or Nuvaxovid, produced by U.S. company Novavax and partners.
NVX-CoV2373 is a protein nanoparticle vaccine constructed from a mutated prefusion SARS-CoV-2 spike protein in combination with a specialized saponin-based adjuvant to elicit an immune response against SARS-CoV-2.
The spike protein is recombinantly expressed in Sf9 insect cells <span class="citation" data-cites="Qk33ZrIC">(<a href="#ref-Qk33ZrIC" role="doc-biblioref">1264</a>)</span>, which have previously been used for several other FDA-approved protein therapeutics <span class="citation" data-cites="RQR2sOmx">(<a href="#ref-RQR2sOmx" role="doc-biblioref">1265</a>)</span>, and contains mutations in the furin cleavage site (682-RRAR-685 to 682-QQAQ-685) along with two proline substitutions (K986P and V987P) that stabilize the protein <span class="citation" data-cites="nHjzMY5L">(<a href="#ref-nHjzMY5L" role="doc-biblioref">1266</a>)</span>, including improving thermostability <span class="citation" data-cites="Qk33ZrIC">(<a href="#ref-Qk33ZrIC" role="doc-biblioref">1264</a>)</span>.</p>
<p>In preclinical mouse models, Novavax-CoV2373 elicited high anti-spike IgG titers 21 to 28 days post-vaccination that could neutralize the SARS-CoV-2 virus and protect the animals against viral challenge, with particularly strong effects when administered with the proprietary adjuvant Matrix-M<sup>TM</sup> <span class="citation" data-cites="Qk33ZrIC">(<a href="#ref-Qk33ZrIC" role="doc-biblioref">1264</a>)</span>.
In a phase I/II trial, a two-dose regimen of NVX-CoV2373 was found to induce anti-spike IgG levels and nAb titers exceeding those observed in convalescent plasma donated by symptomatic patients <span class="citation" data-cites="dMLXxGAI">(<a href="#ref-dMLXxGAI" role="doc-biblioref">1267</a>)</span>.
In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response.</p>
<p>In a phase III randomized, observer-blinded, placebo-controlled clinical trial in the U.K., 14,039 participants received two 5-μg doses of NVX-CoV2373 or placebo administered 21 days apart in a 1:1 ratio from late September to late November 2020 <span class="citation" data-cites="1D0f8OrG8">(<a href="#ref-1D0f8OrG8" role="doc-biblioref">1268</a>)</span>.
In the phase III trial, the efficacy of Novavax’s Nuvaxovid was reported to be 89.7%, with a total of 10 patients developing COVID-19 in the vaccine group versus 96 in the placebo group <span class="citation" data-cites="1D0f8OrG8">(<a href="#ref-1D0f8OrG8" role="doc-biblioref">1268</a>)</span>.
No hospitalizations or deaths were reported in the vaccine group.
An additional phase III randomized, observer-blinded, placebo-controlled trial was conducted in the U.S. and Mexico, enrolling 29,949 participants and administering at least 1 vaccine in a 2:1 ratio from late December 2020 to late February 2021 <span class="citation" data-cites="IQOyFnfl">(<a href="#ref-IQOyFnfl" role="doc-biblioref">1269</a>)</span>.
This trial <span class="citation" data-cites="IQOyFnfl">(<a href="#ref-IQOyFnfl" role="doc-biblioref">1269</a>)</span> used the same primary endpoints as the initial phase III trial conducted in the U.K. <span class="citation" data-cites="1D0f8OrG8">(<a href="#ref-1D0f8OrG8" role="doc-biblioref">1268</a>)</span>.
A vaccine efficacy of 90.4% was reported based on 77 cases total, 63 of which occurred in the placebo group.
All moderate to severe cases of COVID-19 occurred in the placebo group.
Hospitalization and death were not evaluated as individual secondary endpoints, but were instead included in the definition of severe COVID-19; all-cause mortality was comparable between the placebo and treatment conditions <span class="citation" data-cites="IQOyFnfl">(<a href="#ref-IQOyFnfl" role="doc-biblioref">1269</a>)</span>.</p>
<p>The conclusions of both trials indicate that the NVX-CoV2373 vaccine is safe and effective against COVID-19.
In both trials, the vaccine was found to be well-tolerated <span class="citation" data-cites="oc5SBo0q 1D0f8OrG8">(<a href="#ref-1D0f8OrG8" role="doc-biblioref">1268</a>, <a href="#ref-oc5SBo0q" role="doc-biblioref">1270</a>)</span>.
The Novavax vaccine has since been authorized for use in several places, including the United Kingdom <span class="citation" data-cites="q3wIcP4d">(<a href="#ref-q3wIcP4d" role="doc-biblioref">1271</a>)</span>, the E.U. <span class="citation" data-cites="rPtsd03">(<a href="#ref-rPtsd03" role="doc-biblioref">1272</a>)</span>, and the U.S. <span class="citation" data-cites="x9xXbzuY">(<a href="#ref-x9xXbzuY" role="doc-biblioref">1273</a>)</span>.</p>
<h3 data-number="6.6.3" id="covifenz"><span class="header-section-number">6.6.3</span> Covifenz</h3>
<p>The leading example of a VLP approach applied to COVID-19 comes from Covifenz, developed by Canadian company Medicago <span class="citation" data-cites="pxEbK2Vl">(<a href="#ref-pxEbK2Vl" role="doc-biblioref">1274</a>)</span>.
This vaccine was developed using plant-based VLP technology <span class="citation" data-cites="18Xmc8WMR">(<a href="#ref-18Xmc8WMR" role="doc-biblioref">1275</a>)</span> that the company had been investigating in order to develop a high-throughput quadrivalent VLP platform to provide protection against influenza <span class="citation" data-cites="WqqE64cZ">(<a href="#ref-WqqE64cZ" role="doc-biblioref">1276</a>)</span>.
The approach utilizes <em>Nicotiana benthamiana</em>, an Australian relative of the tobacco plant, as an upstream bioreactor <span class="citation" data-cites="WqqE64cZ 18mcJS5j4">(<a href="#ref-WqqE64cZ" role="doc-biblioref">1276</a>, <a href="#ref-18mcJS5j4" role="doc-biblioref">1277</a>)</span>.
Specifically, the <em>S</em> gene from SARS-CoV-2 in its prefusion conformation is inserted into a bacterial vector (<em>Agrobacterium tumefaciens</em>) that then infects the plant cells <span class="citation" data-cites="WqqE64cZ 18mcJS5j4">(<a href="#ref-WqqE64cZ" role="doc-biblioref">1276</a>, <a href="#ref-18mcJS5j4" role="doc-biblioref">1277</a>)</span>.
Expression of the S glycoprotein causes the production of VLPs composed of S trimers anchored in a lipid envelope that accumulate between the plasma membrane and the cell wall of the plant cell <span class="citation" data-cites="18mcJS5j4">(<a href="#ref-18mcJS5j4" role="doc-biblioref">1277</a>)</span>.
Because these VLPs do not contain the SARS-CoV-2 genome, they offer similar advantages to whole-virus vaccines while mitigating the risks <span class="citation" data-cites="WqqE64cZ 18mcJS5j4">(<a href="#ref-WqqE64cZ" role="doc-biblioref">1276</a>, <a href="#ref-18mcJS5j4" role="doc-biblioref">1277</a>)</span>.</p>
<p>In the phase I study, 180 Canadian adults ages 18 to 55 years old were administered Covifenz as two doses, 21 days apart, with three different dosages evaluated <span class="citation" data-cites="18mcJS5j4">(<a href="#ref-18mcJS5j4" role="doc-biblioref">1277</a>)</span>.
This study reported that when the VLPs were administered with AS03, an oil-in-water emulsion containing α-tocopherol and squalene <span class="citation" data-cites="KqsZ2xpA">(<a href="#ref-KqsZ2xpA" role="doc-biblioref">1278</a>)</span>, as an adjuvant, the vaccine elicited an nAb response that was significantly (approximately 10 times) higher than that in convalescent sera <span class="citation" data-cites="18mcJS5j4">(<a href="#ref-18mcJS5j4" role="doc-biblioref">1277</a>)</span>.
The phase III trial examined 24,141 adults assigned to the treatment and control conditions at a 1:1 ratio between March and September of 2021 <span class="citation" data-cites="hZr9yVvu">(<a href="#ref-hZr9yVvu" role="doc-biblioref">1279</a>)</span>.</p>
<p>Covifenz was reported to be 71% effective in preventing COVID-19 in the per-protocol analysis <span class="citation" data-cites="hZr9yVvu">(<a href="#ref-hZr9yVvu" role="doc-biblioref">1279</a>)</span>.
Efficacy was only slightly lower in the intention-to-treat group at 69%, with the VE for the prevention of moderate-to-severe disease in this group estimated at 78.8%.
Over 24,000 participants were included in the safety analysis, which reported that 92.3% of vaccine recipients reported local adverse events compared to 45.5% of placebo recipients, with rates for systemic adverse events at 87.3% and 65.0%, respectively.
The adverse effects reported were generally mild to moderate, with the most common adverse effects being injection site pain, headache, myalgia, fatigue, and general discomfort.
Only three patients (two in the vaccine group) reported grade 4 events, all after the second dose.
The vaccine was approved for use in adults ages 18 to 65 by Health Canada in February 2022 <span class="citation" data-cites="1JmpfIPn">(<a href="#ref-1JmpfIPn" role="doc-biblioref">1280</a>)</span>.</p>
<p>Plant-based expression systems such as this are relatively new <span class="citation" data-cites="18mcJS5j4">(<a href="#ref-18mcJS5j4" role="doc-biblioref">1277</a>)</span> but are likely to offer unparalleled feasibility at scale given the speed and low-cost associated with the platform <span class="citation" data-cites="kbvZWBoy">(<a href="#ref-kbvZWBoy" role="doc-biblioref">1281</a>)</span>.
Additionally, the Covifenz vaccine can be stored at 2 to 8°C.
However, the worldwide footprint of Covifenz, and of VLP-based technologies against SARS-CoV-2 broadly, remains small, with only 1 VLP vaccine approved for distribution in 1 countries (Figure <a href="#fig:vlp-distrib">8</a>).
Approval and administration of Covifenz in countries outside of Canada has been limited by concerns at the WHO about ties between Medicago and the tobacco industry <span class="citation" data-cites="pxEbK2Vl aEJzXfl6">(<a href="#ref-pxEbK2Vl" role="doc-biblioref">1274</a>, <a href="#ref-aEJzXfl6" role="doc-biblioref">1282</a>)</span>.
While other species of plants have been explored as the upstream bioreactors for plant-derived VLPs, the specific species of tobacco used increased yield dramatically <span class="citation" data-cites="11WYMWwb">(<a href="#ref-11WYMWwb" role="doc-biblioref">1283</a>)</span>.</p>
<h3 data-number="6.6.4" id="real-world-safety-and-efficacy-1"><span class="header-section-number">6.6.4</span> Real-World Safety and Efficacy</h3>
<p>To date, data about the effect of viral evolution on the efficacy of subunit vaccines has been limited.
<em>Post hoc</em> analysis in the phase III trial determined that the NovaVax vaccine had an efficacy of 86.3% against the Alpha variant (identified based on the presence of the 69–70del polymorphism) and 96.4% against variants lacking the 69-70del polymorphism <span class="citation" data-cites="1D0f8OrG8">(<a href="#ref-1D0f8OrG8" role="doc-biblioref">1268</a>)</span>.
In the second phase III trial <span class="citation" data-cites="oc5SBo0q">(<a href="#ref-oc5SBo0q" role="doc-biblioref">1270</a>)</span>, whole-genome sequencing was obtained from 61 of the 77 cases, and 79% of infections were identified as a VOC or variant of interest (VOI) that had been characterized at the time of the study.
Vaccine efficacy against cases caused by VOC, among which the Alpha variant was predominant (88.6%), was reported to be 92.6% <span class="citation" data-cites="oc5SBo0q">(<a href="#ref-oc5SBo0q" role="doc-biblioref">1270</a>)</span>.
In late 2020, an analysis of efficacy in South African adults revealed an overall efficacy of 60.1% and a slightly lower efficacy of 50.1% against B.1.351 in particular <span class="citation" data-cites="14K1ANV1T">(<a href="#ref-14K1ANV1T" role="doc-biblioref">1284</a>)</span>.
Additionally, an analysis of a booster dose of NVX-CoV2373 administered six months after the primary series revealed a significant increase in neutralizing activity against VOC including Delta and Omicron <span class="citation" data-cites="S28ZsPCI">(<a href="#ref-S28ZsPCI" role="doc-biblioref">1285</a>)</span>.
It has also been reported that Novavax initiated booster trials in the U.K. <span class="citation" data-cites="wByD9WaX">(<a href="#ref-wByD9WaX" role="doc-biblioref">1171</a>)</span>.</p>
<p>Because the Covifenz results became available relatively recently (May 2022), limited data is available since the publication of phase III trial results <span class="citation" data-cites="hZr9yVvu">(<a href="#ref-hZr9yVvu" role="doc-biblioref">1279</a>)</span>.
However, it should be noted that the Covifenz trials were conducted in 2021, at a time during which the B.1.617.2 (Delta) and P.1 (Gamma) variants were predominant <span class="citation" data-cites="hZr9yVvu">(<a href="#ref-hZr9yVvu" role="doc-biblioref">1279</a>)</span>.
Genomic analysis of 122 out of 176 cases (165 in the per-protocol population) revealed that none of the COVID-19 cases reported were caused by the original Wuhan strain.
Instead, 45.9% of cases were identified as the Delta variant, 43.4% as Gamma, 4.9% as Alpha, and 5.8% as VOIs.
Therefore, the efficacy data from this phase III trial may be lower than it would have been if the trial had occurred earlier in the course of SARS-CoV-2’s evolution given that the S glycoprotein expressed in the VLPs was isolated from a 2020 sample of SARS-CoV-2 <span class="citation" data-cites="hZr9yVvu">(<a href="#ref-hZr9yVvu" role="doc-biblioref">1279</a>)</span>.</p>
<h2 data-number="6.7" id="global-vaccine-status-and-distribution"><span class="header-section-number">6.7</span> Global Vaccine Status and Distribution</h2>
<p>The unprecedented deployment of COVID-19 vaccines in under a year from the identification of SARS-CoV-2 led to a new challenge: the formation of rapid global vaccine production and distribution plans.
The development of vaccines is costly and complicated, but vaccine distribution can be just as challenging.
Logistical considerations such as transport, storage, equipment (e.g., syringes), the workforce to administer the vaccines, and a continual supply from the manufacturers to meet global demands all must be accounted for and vary globally due to economic, geographic, and sociopolitical reasons <span class="citation" data-cites="RG0vzlcE 19CWe6pdS d0kUYq5Z">(<a href="#ref-RG0vzlcE" role="doc-biblioref">1286</a>–<a href="#ref-d0kUYq5Z" role="doc-biblioref">1288</a>)</span>.
As of December 12, 2022, at least 13.0 billion vaccine doses had been administered in at least 223 countries worldwide using 27 different vaccines <span class="citation" data-cites="3cPwqjhj">(<a href="#ref-3cPwqjhj" role="doc-biblioref">532</a>)</span>.&gt;
The daily global vaccination rate at this time was 287.0 per million.</p>
<p>However, the distribution of these doses is not uniform around the globe.
Latin America leads world vaccination rates with at least 82% of individuals in this region receiving one vaccine dose followed by the U.S. and Canada (81%), Asia-Pacific (81%), Europe (70%), the Middle East (58%), followed by Africa with only 33% by November 2022 <span class="citation" data-cites="dfl5iCJI">(<a href="#ref-dfl5iCJI" role="doc-biblioref">1289</a>)</span>.
It is estimated that only ~25% of individuals in low- and middle-income countries have received one vaccine dose <span class="citation" data-cites="cWMPXfju sRy6js2o">(<a href="#ref-sRy6js2o" role="doc-biblioref">1115</a>, <a href="#ref-cWMPXfju" role="doc-biblioref">1290</a>)</span>.
Vaccine production and distribution varies from region to region and seems to depend on the availability of the vaccines and potentially a country’s resources and wealth <span class="citation" data-cites="kL8PlRJu">(<a href="#ref-kL8PlRJu" role="doc-biblioref">1291</a>)</span>.</p>
<p>One effort to reduce these disparities is the COVID-19 Vaccines Global Access (COVAX) Facility, a multilateral initiative as part of the Access to COVID-19 Tools (ACT) Accelerator coordinated by the WHO, Gavi, the Vaccine Alliance, and the Coalition for Epidemic Preparedness Innovations (CEPI), the latter two of which are supported by the Bill and Melinda Gates Foundation.
Their intention is to accelerate the development of COVID-19 vaccines, diagnostics, and therapeutics and to ensure the equitable distribution of vaccines to low- and middle-income countries <span class="citation" data-cites="3Gq7ETv7 KzHIbPMY">(<a href="#ref-3Gq7ETv7" role="doc-biblioref">1292</a>, <a href="#ref-KzHIbPMY" role="doc-biblioref">1293</a>)</span>.
COVAX invested in several vaccine programs to ensure they would have access to successful vaccine candidates <span class="citation" data-cites="1H0PiQpLz">(<a href="#ref-1H0PiQpLz" role="doc-biblioref">1294</a>)</span>.
However, the initiative has been less successful than was initially hoped due to less participation from high-income countries than was required for COVAX to meet its goals <span class="citation" data-cites="k2clrWSf">(<a href="#ref-k2clrWSf" role="doc-biblioref">1295</a>)</span>.</p>
<p>Additionally, the vaccine technologies available differ widely around the globe.
As we review elsewhere <span class="citation" data-cites="rjHK19Dr">(<a href="#ref-rjHK19Dr" role="doc-biblioref">7</a>)</span>, wealthier nations have invested heavily in mRNA and DNA vaccines.
In contrast, as we describe above, many countries outside of Europe and North America have developed highly effective vaccines using more traditional approaches.
As a result, there is a clear relationship between a country’s gross domestic product (GDP) and its access to vaccines (Figure <a href="#fig:gdpscatter">9</a>).</p>
<div id="fig:gdpscatter" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/dfc6ff85488d57cbb52f4b2324531be6afaa73f4/OWID_doseTypebyGDP.png" alt="Figure 9: Vaccine Distribution across Platform Types as a Function of GDP. The total number of doses distributed within each country as of December 18, 2022, by platform type, is shown as a function of GDP. These data are retrieved from Our World in Data (532, 1115) and plotted using the Python package plotnine (1296). Lines show a general trend in the data and are drawn using geom_smooth (1297). The list of countries included in the dataset is available from OWID (1298). See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily. Axes are not scaled per capita because both variables are modulated by population size." />
<figcaption aria-hidden="true"><span>Figure 9:</span> <strong>Vaccine Distribution across Platform Types as a Function of GDP.</strong>
The total number of doses distributed within each country as of December 18, 2022, by platform type, is shown as a function of GDP.
These data are retrieved from Our World in Data <span class="citation" data-cites="3cPwqjhj sRy6js2o">(<a href="#ref-3cPwqjhj" role="doc-biblioref">532</a>, <a href="#ref-sRy6js2o" role="doc-biblioref">1115</a>)</span> and plotted using the Python package plotnine <span class="citation" data-cites="L5wrzuPV">(<a href="#ref-L5wrzuPV" role="doc-biblioref">1296</a>)</span>.
Lines show a general trend in the data and are drawn using geom_smooth <span class="citation" data-cites="48MbU9B0">(<a href="#ref-48MbU9B0" role="doc-biblioref">1297</a>)</span>.
The list of countries included in the dataset is available from OWID <span class="citation" data-cites="FCTCkZoZ">(<a href="#ref-FCTCkZoZ" role="doc-biblioref">1298</a>)</span>.
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.
Axes are not scaled per capita because both variables are modulated by population size.</figcaption>
</figure>
</div>
<!-- Figure is not rendering -->
<p>When vaccines first became available, the wealthy nations of North America and Europe secured most of the limited COVID-19 vaccine stocks <span class="citation" data-cites="1AvwH3T5y">(<a href="#ref-1AvwH3T5y" role="doc-biblioref">1299</a>)</span>.
Throughout 2021, low- and middle-income countries faced steep competition with high-income countries for vaccines, and the rates of vaccination reflected this unequal distribution <span class="citation" data-cites="JM2MJEBG">(<a href="#ref-JM2MJEBG" role="doc-biblioref">1300</a>)</span>.
While the wealthiest countries in these regions could compete with each other for vaccines independent of programs such as COVAX <span class="citation" data-cites="JM2MJEBG">(<a href="#ref-JM2MJEBG" role="doc-biblioref">1300</a>)</span>, other countries in these regions have faced challenges in acquiring vaccines developed by the world’s wealthiest nations.
Fortunately, while mRNA and DNA vaccine development programs are not widespread, vaccine development using whole-virus and subunit technologies has been undertaken worldwide.
China and India, in particular, have developed several vaccines that are now widely available in these densely populated countries (see online Appendix <span class="citation" data-cites="yTsmmAYC">(<a href="#ref-yTsmmAYC" role="doc-biblioref">1139</a>)</span>).</p>
<p>Still, many nations, especially in Africa, are reliant on the COVAX Facility, who have promised 600 million doses to the continent <span class="citation" data-cites="1EnpYQzIq">(<a href="#ref-1EnpYQzIq" role="doc-biblioref">1301</a>)</span>.
The COVAX plan seeks to ensure that all participating countries would be allocated vaccines in proportion to their population sizes.
Once each country has received vaccine doses to account for 20% of their population, the country’s risk profile will determine its place in subsequent phases of vaccine distribution.
However, several limitations of this framework exist, including that the COVAX scheme seems to go against the WHO’s own ethical principles of human well-being, equal respect, and global equity and that other frameworks might have been more suitable <span class="citation" data-cites="12QaZb4si">(<a href="#ref-12QaZb4si" role="doc-biblioref">1302</a>)</span>.
Furthermore, COVAX is supposed to allow poorer countries access to affordable vaccines, but the vaccines are driven by publicly traded companies that are required to make a profit <span class="citation" data-cites="kL8PlRJu">(<a href="#ref-kL8PlRJu" role="doc-biblioref">1291</a>)</span>.</p>
<p>In any case, COVAX provides access to COVID-19 vaccines that may otherwise have been difficult for some countries to obtain.
COVAX aimed to distribute 2 billion vaccine doses globally by the end of 2021 <span class="citation" data-cites="7dkwQDUf">(<a href="#ref-7dkwQDUf" role="doc-biblioref">1303</a>)</span>.
According to Gavi, as of January 2022, COVAX had distributed over 1 billion vaccines to 144 participants of the program <span class="citation" data-cites="JNpdOLGo">(<a href="#ref-JNpdOLGo" role="doc-biblioref">1304</a>)</span>, short of its target but still a major global achievement.
It is envisaged that COVAX may also receive additional donations of doses from Western nations who purchased surplus vaccines in the race to vaccinate their populations, which will be a welcome boost to the vaccination programs of low- and middle-income countries <span class="citation" data-cites="sr5oRBgc">(<a href="#ref-sr5oRBgc" role="doc-biblioref">1305</a>)</span>.</p>
<p>In general, deciding on the prioritization and allocation of the COVID-19 vaccines is also a challenging task due to ethical and operational considerations.
Various frameworks, models, and methods have been proposed to tackle these issues with many countries, regions, or U.S. states devising their own distribution and administration plans <span class="citation" data-cites="S8WhufUV dLbKv1xi jbpdQdOw z5c17nGB s2zZd6pb">(<a href="#ref-S8WhufUV" role="doc-biblioref">1306</a>–<a href="#ref-s2zZd6pb" role="doc-biblioref">1310</a>)</span>.
The majority of the distribution plans prioritized offering vaccines to key workers such as health care workers and those who are clinically vulnerable, such as the elderly, the immunocompromised, and individuals with comorbidities, before targeting the rest of the population, who are less likely to experience severe outcomes from COVID-19 <span class="citation" data-cites="sEyIoYCS">(<a href="#ref-sEyIoYCS" role="doc-biblioref">1311</a>)</span>.
The availability of vaccines developed in a variety of countries using a variety of platforms is likely to work in favor of worldwide vaccine access.
The initiative by Texas Children’s Hospital and Baylor College of Medicine to develop Corbevax, a patent-free COVID-19 protein subunit vaccine, is an important step towards vaccine equity because the manufacturing specifications can be shared globally.
Corbevax can be produced at low cost using existing technology and is now licensed to Biological E. Limited (BioE), an Indian company specializing in low-cost vaccine production <span class="citation" data-cites="T5hm2GMj">(<a href="#ref-T5hm2GMj" role="doc-biblioref">1312</a>)</span>.
The vaccine has been approved for distribution in India and Botswana <span class="citation" data-cites="RueOviyL">(<a href="#ref-RueOviyL" role="doc-biblioref">1313</a>)</span>.</p>
<p>Logistical challenges and geographical barriers also dictate the availability of certain vaccines.
Many countries have had poor availability of ultra-low temperature freezers, leading to challenges of distribution for vaccines such as mRNA vaccines that require storage at very low temperatures <span class="citation" data-cites="ZCOfVmLC CfthGlKw QpYrg3Eh">(<a href="#ref-ZCOfVmLC" role="doc-biblioref">1314</a>–<a href="#ref-QpYrg3Eh" role="doc-biblioref">1316</a>)</span>.
Furthermore, ancillary supplies such as vaccine containers, diluents for frozen or lyophilized vaccines, disinfecting wipes, bandages, needles, syringes, sharps and biological waste disposal containers are also required, which may not be readily available in geographically isolated locations and can be bulky and expensive to ship <span class="citation" data-cites="ZCOfVmLC">(<a href="#ref-ZCOfVmLC" role="doc-biblioref">1314</a>)</span>. ƒ√
While some of these challenges in vaccine rollout in low- and middle-income countries are being addressed through COVAX <span class="citation" data-cites="LwA7POre">(<a href="#ref-LwA7POre" role="doc-biblioref">1317</a>)</span>, many issues persist worldwide <span class="citation" data-cites="h68ER2wh sjDDQBRj vxL9u0id">(<a href="#ref-h68ER2wh" role="doc-biblioref">1318</a>–<a href="#ref-vxL9u0id" role="doc-biblioref">1320</a>)</span>.
COVAX also failed to distribute its promised two billion vaccine doses on time due to multiple complications <span class="citation" data-cites="Pjo8Db92">(<a href="#ref-Pjo8Db92" role="doc-biblioref">1321</a>)</span>.</p>
<p>Another major challenge to global vaccine distribution is vaccine hesitancy, which the WHO has designated as a leading global health threat <span class="citation" data-cites="1683Ay0Ma">(<a href="#ref-1683Ay0Ma" role="doc-biblioref">1322</a>)</span>.
Polling in the U.S. in January 2021 suggested that 20% of individuals were reluctant to receive a vaccine at that time, with a further 31% expressing some hesitancy to a lesser extent <span class="citation" data-cites="vgBdaJfl 9WzhBzsN">(<a href="#ref-vgBdaJfl" role="doc-biblioref">1323</a>, <a href="#ref-9WzhBzsN" role="doc-biblioref">1324</a>)</span>.
A survey of 8,243 long-term healthcare workers in November 2020 (Indiana, USA) reported that only 69% of respondents would ever consider receiving an FDA-approved vaccine due to their perceived risk of side effects (70%), health concerns (34%), efficacy (20%), and religious beliefs (12%) <span class="citation" data-cites="1GU6vsFSK">(<a href="#ref-1GU6vsFSK" role="doc-biblioref">1325</a>)</span>.
Notably, almost a third of parents surveyed in the United States in March 2021 expressed concerns about vaccinating their children against COVID-19 <span class="citation" data-cites="3Bv9rDjF">(<a href="#ref-3Bv9rDjF" role="doc-biblioref">1326</a>)</span>.
Indeed, vaccine hesitancy has been reported as a significant barrier to vaccine distribution in countries in North and South America, Europe, Asia, and Africa <span class="citation" data-cites="aMyvkKF6 Qc6ugUZB 13Qw1QyDZ ADXpa112 qLCe8Fn4">(<a href="#ref-aMyvkKF6" role="doc-biblioref">1327</a>–<a href="#ref-qLCe8Fn4" role="doc-biblioref">1331</a>)</span>.
Various factors have been associated with increased vaccine hesitancy including access to compelling misinformation via social media <span class="citation" data-cites="rY8TOiPE IQRxQrGA">(<a href="#ref-rY8TOiPE" role="doc-biblioref">1332</a>, <a href="#ref-IQRxQrGA" role="doc-biblioref">1333</a>)</span>, religious and conservative political beliefs <span class="citation" data-cites="14zu298e4 udYZFdZJ hCKsU5CZ XWoyDldi">(<a href="#ref-14zu298e4" role="doc-biblioref">1334</a>–<a href="#ref-XWoyDldi" role="doc-biblioref">1337</a>)</span>, and safety and efficacy concerns <span class="citation" data-cites="3Bv9rDjF">(<a href="#ref-3Bv9rDjF" role="doc-biblioref">1326</a>)</span>, to highlight a few.
Many of the concerns regarding safety and efficacy have focused on the novel mRNA technologies due to the perceived speed of their development and expedited clinical trial process <span class="citation" data-cites="s6yd91AQ">(<a href="#ref-s6yd91AQ" role="doc-biblioref">1338</a>)</span>; however, general vaccine hesitancy relating to traditional vaccine platforms existed long before the pandemic and the distribution of the novel mRNA vaccines <span class="citation" data-cites="vIrtP9Np qSSc0Ow0">(<a href="#ref-vIrtP9Np" role="doc-biblioref">1339</a>, <a href="#ref-qSSc0Ow0" role="doc-biblioref">1340</a>)</span>.
While in the United States, it was hoped that Novavax’s Nuvaxovid would appeal to the vaccine hesitant <span class="citation" data-cites="1ADXp4JiF 1HJpEx8e7">(<a href="#ref-1ADXp4JiF" role="doc-biblioref">1341</a>, <a href="#ref-1HJpEx8e7" role="doc-biblioref">1342</a>)</span>, but this protein subunit vaccine has not led to the uptake hoped <span class="citation" data-cites="13O5dayXS MMu5x58p">(<a href="#ref-13O5dayXS" role="doc-biblioref">1343</a>, <a href="#ref-MMu5x58p" role="doc-biblioref">1344</a>)</span>.</p>
<h2 data-number="6.8" id="conclusions-1"><span class="header-section-number">6.8</span> Conclusions</h2>
<p>Much attention has focused on the most novel vaccine technologies that have been deployed against SARS-CoV-2, but the established vaccine platforms discussed here have all made a significant impact on human health during the twentieth century and in some cases even earlier.
The COVID-19 pandemic has demonstrated new potential in these established technologies.
In the early 2000s, these technologies were explored for managing SARS-CoV-1 <span class="citation" data-cites="H4USOXie AOGjkjCq">(<a href="#ref-H4USOXie" role="doc-biblioref">1345</a>, <a href="#ref-AOGjkjCq" role="doc-biblioref">1346</a>)</span>, but the epidemic was controlled before those efforts came to fruition <span class="citation" data-cites="HyYY2agc">(<a href="#ref-HyYY2agc" role="doc-biblioref">1347</a>)</span>.
Similarly, these technologies were explored for MERS-CoV, but outbreaks were sporadic and difficult to predict, making vaccine testing and the development of a vaccination strategy difficult <span class="citation" data-cites="138O0v19T">(<a href="#ref-138O0v19T" role="doc-biblioref">1348</a>)</span>.
However, in the COVID-19 pandemic, most of these technologies have been used to accelerate vaccine development programs worldwide.
Therefore, they are also offering the opportunity to respond quickly to an emergent pathogen.</p>
<p>While these tried-and-true technologies don’t always produce vaccines with the highly desirable VE reported in mRNA clinical trials (which exceeded 90%), the efficacies are still very high, and these vaccines are extremely effective at preventing severe illness and death.
As a result, the greater accessibility and stability of these vaccines makes them extremely valuable for the global effort to mitigate the loss of life from SARS-CoV-2.</p>
<h1 data-number="7" id="appendix-additional-information-about-established-vaccine-platforms-for-covid-19"><span class="header-section-number">7</span> Appendix: Additional Information about Established Vaccine Platforms for COVID-19</h1>
<h2 data-number="7.1" id="sinovacs-coronavac"><span class="header-section-number">7.1</span> Sinovac’s CoronaVac</h2>
<p>The CoronaVac vaccine was developed by Sinovac, a Beijing-based biopharmaceutical company.
The vaccine uses an inactivate whole virus with the addition of an aluminum adjuvant <span class="citation" data-cites="RGPoDfHS">(<a href="#ref-RGPoDfHS" role="doc-biblioref">1349</a>)</span>.
Pre-clinical trials were performed using BALB/c mice and rhesus macaques <span class="citation" data-cites="14fILrRWg">(<a href="#ref-14fILrRWg" role="doc-biblioref">1163</a>)</span>.
The SARS-CoV-2 strains used in this trial isolated from 11 hospitalized patients (5 from China, 3 from Italy, 1 from the United Kingdom (U.K.), 1 from Spain, 1 from Switzerland).
A phylogenetic analysis demonstrated that the strains were representative of the variants circulating at the time.
One of the strains from China, CN2, was used as the inactivated and purified virus while the other 10 strains were used to challenge.
CN2 was grown in Vero cells.
The immunogenicity of the vaccine candidate was evaluated with an ELISA assay.
Ten mice were injected with the vaccine on day 0 and day 7 with varying doses (0, 1.5, 3, or 6 μg), and 10 mice were treated with physiological saline as the control.
IgG developed in the serum of all vaccinated mice.</p>
<p>Using the same setup, immunogenicity was also assessed in macaques.
Four macaques were assigned to each of four groups: treatment with 3 μg at day 0, 7, and 14, treatment with a high dose of 6 μg at day 0, 7, and 14, administration of a placebo vaccine, and administration of only the adjuvant.
All vaccinated macaques induced IgGs and neutralizing antibodies.
After challenge with SARS-CoV-2 strain CN1, vaccinated macaques were protected compared to control macaques (placebo or adjuvant only) based on histology and viral loads collected from different regions of the lung.</p>
<p>Phase I/II clinical trials were conducted in adults 18-59 years old <span class="citation" data-cites="N1txjPtt">(<a href="#ref-N1txjPtt" role="doc-biblioref">1164</a>)</span> and adults over 60 years old <span class="citation" data-cites="Ozya5HP5">(<a href="#ref-Ozya5HP5" role="doc-biblioref">1162</a>)</span> in China.
In the case of adults 18-59 years old, a single center, randomized, double-blind, placebo-controlled phase I/II trial was conducted in April 2020.
Patients in this study were recruited from the community in Suining County of Jiangsu province, China.
For the phase I trial, 144 (of 185 screened) participants were enrolled, with 72 enrolled in the 14-day interval cohort (i.e., treated on day 0 and day 14) and 72 in the 28-day interval cohort.
This group of 72 participants was split into 2 blocks for a low-dose (3 μg) and high-dose (6 μg) vaccine.
Within each block, participants were randomly assigned vaccination with CoronaVac or placebo (aluminum diluent without the virus) at a 2:1 ratio.
Both the vaccine and placebo were prepared in a Good Manufacturing Practice-accredited facility of Sinovac Life Sciences (Beijing, China).</p>
<p>The phase II trial followed the same organization of participants, this time using 300 enrolled participants in the 14-day and another 300 enrolled in the 28-day groups.
One change of note was that the vaccine was produced using a highly automated bioreactor (ReadyToProcess WAVE 25, GE, Umeå, Sweden) to increase vaccine production capacity.
This change resulted in a higher intact spike protein content.
The authors of this study were not aware of this antigen-level difference between the vaccine batches for the phase I/II when the ethical approval for the trials occurred.</p>
<p>To assess adverse responses, participants were asked to record any events up to 7 days post-treatment.
The reported adverse events were graded according to the China National Medical Products Administration guidelines.
In the phase I trial, the overall incidence of adverse reactions was 29-38% of patients in the 0 to 14 day group and 13-17% in the 0 to 28 day vaccination group.
The most common symptom was pain at the injection site, which was reported by 17-21% of patients in the 0 to 14 day cohort and 13% in the 0 to 28 day cohort.
Most adverse reactions were mild (grade 1), with patients recovering within 48 hours.
A single case of acute hypersensitivity with manifestation of urticaria 48 hours following the first dose of study drug was reported in the 6 μg group
Both the 14-day and 28-day cohorts had a strong neutralizing antibody (Ab) response.
The neutralizing Ab response was measured using a micro cytopathogenic effect assay, which assesses the minimum dilution of neutralizing Ab to be 50% protective against structural changes in host cells in response to viral infection <span class="citation" data-cites="1GUVvcQjL">(<a href="#ref-1GUVvcQjL" role="doc-biblioref">1350</a>)</span>.
Additionally IgG antibody titers against the receptor binding domain were also measured using ELISA.</p>
<p>Another phase I/II study was performed with older patients (over 60 years of age) <span class="citation" data-cites="Ozya5HP5">(<a href="#ref-Ozya5HP5" role="doc-biblioref">1162</a>)</span>.
The study conducted a single-center, randomized, double-blind, placebo-controlled trial.
The phase I trial looked at dose escalation using 3 dosages: 1.5, 3, and 6 μg.
The mean age of participants was 65.8 years (standard deviation = 4.8).
Of 95 screened participants, 72 were enrolled.
These 72 participants were split into low (3 μg) and high (6 μg) dose groups.
Within each group, 24 participants received the treatment and 12 the placebo.
A neutralizing antibody response against live SARS-CoV-2 was detected compared to baseline using the same micro cytopathogenic effect assay.
This response was similar across the two dose concentrations.
Additionally, they did not observe a difference in response between age groups (60–64 years, 65–69 years, and ≥70 years).</p>
<p>In phase II the mean age was 66.6 years (standard deviation = 4.7).
499 participants were screened and 350 were enrolled.
300 were evenly split into 1.5, 3, and 6 μg dose groups, and the remaining 50 were assigned to the placebo group.
Again, they found a neutralizing antibody response in phase II.
There wasn’t a significant different between the response to 3 μg versus 6 μg, but the response in both these conditions was higher than in the 1.5 μg condition.</p>
<p>Participants were required to record adverse reaction events within the first 7 days after each dose.
The safety results were combined across phase I and II.
All adverse reactions were either mild (grade 1) or moderate (grade 2) in severity.
The most common symptom was pain at the injection site (9%) and fever (3%).
2% (7 participants) reported severe adverse events (4 from the 1.5 μg group, 1 from the 3 μg group, 2 from the 6 μg group), though these were found to be unrelated to the vaccine.</p>
<p>Overall, the results from the pre-clinical and phase I/II clinical trials were promising, especially the fact that the immune response was consistent in older adults (&gt; 60 years).
Additional phase III trials were conducted in Brazil in a randomized, multicenter, endpoint driven, double-blind, placebo-controlled clinical trial enrolling nearly 10,000 healthcare workers <span class="citation" data-cites="KewHbkLZ F1lYwPSY">(<a href="#ref-KewHbkLZ" role="doc-biblioref">1351</a>, <a href="#ref-F1lYwPSY" role="doc-biblioref">1352</a>)</span>.
As of September 2021, CoronaVac trials are now also being held in the Philippines and Hong Kong, bringing the total number of registered phase III trials investigating the safety and efficacy of CoronaVac to 9, with emergency use approval in 40 countries <span class="citation" data-cites="19z4gdhDS">(<a href="#ref-19z4gdhDS" role="doc-biblioref">1353</a>)</span>.
CoronaVac also appears to be suitable for use in immunocompromised patients such as those with autoimmune rheumatic diseases according to phase IV trials <span class="citation" data-cites="8vzglCry">(<a href="#ref-8vzglCry" role="doc-biblioref">1354</a>)</span>, and the vaccine was also well tolerated and induced humoral responses in phase I trials in children aged 3 to 17 years, which will now be examined in phase II and III clinical trials <span class="citation" data-cites="6EYqf6s7">(<a href="#ref-6EYqf6s7" role="doc-biblioref">1355</a>)</span>.</p>
<h2 data-number="7.2" id="covaxin"><span class="header-section-number">7.2</span> Covaxin</h2>
<p>In India, the Covaxin vaccine received emergency authorization on January 3, 2021, despite the lack of phase III data until March 3, 2021, which was communicated via press release <span class="citation" data-cites="Ks3L7qHG">(<a href="#ref-Ks3L7qHG" role="doc-biblioref">1356</a>)</span>.
Following a press release of the phase III data indicating 80.6% efficacy in 25,800 participants <span class="citation" data-cites="Ks3L7qHG DaDKZXdu">(<a href="#ref-Ks3L7qHG" role="doc-biblioref">1356</a>, <a href="#ref-DaDKZXdu" role="doc-biblioref">1357</a>)</span>, Zimbabwe authorized the use of Covaxin <span class="citation" data-cites="13yEnvOyP">(<a href="#ref-13yEnvOyP" role="doc-biblioref">1358</a>)</span>.
This was followed by a detailed preprint of the double-blind, randomized, controlled phase III trial that was made available in July 2021, with a final enrollment of 25,798 people (~1:1 vaccine:placebo) <span class="citation" data-cites="n7BupOQ6">(<a href="#ref-n7BupOQ6" role="doc-biblioref">1182</a>)</span>.
The vaccine was reported as well tolerated, with an overall vaccine efficacy of 77.8% for the prevention of COVID-19.
Efficacy against severe disease and asymptomatic infection was reported as 93.4% and 63.6% respectively.</p>
<p>It is not yet clear what level of protection Covaxin offers beyond 6 to 8 months post the second vaccine; consequently, the potential requirement of a booster immunization is being explored <span class="citation" data-cites="s1TGwKbT">(<a href="#ref-s1TGwKbT" role="doc-biblioref">1202</a>)</span>.
Furthermore, Bharat Biotech is considering other vaccine regimens such as providing one initial immunization with Covaxin followed by two immunizations with its intranasal vaccine (BBV154) <span class="citation" data-cites="UU0pj3ii">(<a href="#ref-UU0pj3ii" role="doc-biblioref">1359</a>)</span>.
U.S.-based Ocugen Inc., a co-development partner of Bharat Biotech, is leading the application for an Emergency Use Authorization (EUA) for Covaxin intended for the U.S. market.
It has been reported that Bharat Biotech will soon release its phase II and III pediatric trial results <span class="citation" data-cites="3Q7kvwO3">(<a href="#ref-3Q7kvwO3" role="doc-biblioref">1360</a>)</span>.
However, the WHO approval of the Covaxin has been delayed <span class="citation" data-cites="AeulXXFm">(<a href="#ref-AeulXXFm" role="doc-biblioref">1361</a>)</span>.</p>
<h2 data-number="7.3" id="sinopharms-clinical-trials-of-two-vaccine-candidates"><span class="header-section-number">7.3</span> Sinopharm’s Clinical Trials of Two Vaccine Candidates</h2>
<p>Sinopharm Wuhan Institute developed their SARS-CoV-2 inactivated vaccine using the WIV04 strain isolated from a patient at the Jinyintan Hospital in Wuhan, China <span class="citation" data-cites="miMRIMwa">(<a href="#ref-miMRIMwa" role="doc-biblioref">1172</a>)</span>.
This vaccine was also passaged in Vero cells, inactivated using β-propiolactone, and was adjuvanted with aluminum hydroxide.
This vaccine is administered intramuscularly using 5 μg of virus per dose.
Preclinical data providing supporting evidence for the use of this vaccine is not available publicly.
Despite the lack of publicly available preclinical results, Sinopharm Wuhan Institute initiated phase I/II trials, which reported on varying dosing and prime-boost regimens.</p>
<p>A combined phase I/II RCT of Sinopharm’s BBIBP-CorV, also known as Covilo, followed <span class="citation" data-cites="fPGgVKYL">(<a href="#ref-fPGgVKYL" role="doc-biblioref">1175</a>)</span>.
In phase I, 192 participants were randomized with varying doses of 2 μg, 4 μg, or 8 μg/dose or a placebo, and they received the same as a second dose 28 days later.
Approximately 29% of participants reported at least 1 adverse event, most commonly fever, and neutralizing antibody titers were reported for all doses.
In the phase II trial, 482 participants were enrolled (3:1, vaccine:placebo).
Participants in the vaccine condition received either a single 8 μg dose or a double immunization of a 4 μg/dose that was administered 14, 21, or 28 days post the prime dose.
Participants in the placebo condition received the placebo on one of the same four schedules.
The vaccine appeared well-tolerated, with 23% reporting at least one adverse reaction characterized as mild to moderate.
It was reported that all participants had a humoral immune response to the vaccines by day 42 but that the double immunization dosing regimen of 4 μg/dose achieved higher neutralizing antibody titers than a single dose of 8 μg and that the highest response was seen in the double-immunization regimen when at least 21 days separated the two doses <span class="citation" data-cites="fPGgVKYL">(<a href="#ref-fPGgVKYL" role="doc-biblioref">1175</a>)</span>.
Similar findings were reported in another phase I/II trial published by the same authors <span class="citation" data-cites="yN5KOfvE">(<a href="#ref-yN5KOfvE" role="doc-biblioref">1176</a>)</span>.</p>
<h2 data-number="7.4" id="novavaxs-nuvaxovid-nvx-cov2373"><span class="header-section-number">7.4</span> Novavax’s Nuvaxovid (NVX-CoV2373)</h2>
<p>Novavax’s Nuvaxovid is a particularly appealing candidate because the improved stability caused by the proline substitutions is particularly critical to facilitating global distribution, particularly to regions where local refrigerator/freezer capacities are limited.
Importantly, these amino acid substitutions did not affect the ability of the spike protein to bind the hACE2 receptor (the target receptor of SARS-CoV-2 spike protein).
The Novavax-CoV2373 vaccine candidate uses a proprietary, saponin-based Matrix-M<sup>TM</sup> adjuvant that contains two different 40nm-sized particles formed by formulating purified saponin with cholesterol and phospholipids <span class="citation" data-cites="1F52Wz7mx">(<a href="#ref-1F52Wz7mx" role="doc-biblioref">1362</a>)</span>.
In preclinical models, the use of the Matrix-M adjuvant potentiated the cellular and humoral immune responses to influenza vaccines <span class="citation" data-cites="scwOT7dw aD5iMC0Q twmXSpc9 1F52Wz7mx">(<a href="#ref-1F52Wz7mx" role="doc-biblioref">1362</a>–<a href="#ref-twmXSpc9" role="doc-biblioref">1365</a>)</span>.
Importantly, Matrix-M adjuvant-containing vaccines have shown acceptable safety profiles in human clinical trials <span class="citation" data-cites="jVCL0201">(<a href="#ref-jVCL0201" role="doc-biblioref">1366</a>)</span>.</p>
<p>Novavax-CoV2373 induced a multifunctional CD4/CD8 T-cell responses and generate high frequencies of follicular helper T-cells and B-cell germinal centers after vaccination.
These findings were subsequently evaluated in a baboon primate model, in which Novavax-CoV2373 also elicited high antibody titers against the SARS-CoV-2 spike protein, as well as an antigen specific T-cell response.
Based on this data Novavax initiated a phase I/II clinical trial to evaluate the safety and immunogenicity of Novavax-CoV2373 with Matrix-M <span class="citation" data-cites="Nq0cimEs dMLXxGAI">(<a href="#ref-dMLXxGAI" role="doc-biblioref">1267</a>, <a href="#ref-Nq0cimEs" role="doc-biblioref">1367</a>)</span>.</p>
<p>The phase I/II trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms <span class="citation" data-cites="dMLXxGAI">(<a href="#ref-dMLXxGAI" role="doc-biblioref">1267</a>)</span>.
Participants that received the recombinant SARS-CoV-2 vaccine with or without the Matrix-M adjuvant got two injections, 21 days apart.
Primary outcomes that were assessed include reactogenicity, lab-values (serum chemistry and hematology), and anti-spike IgG levels.
Secondary outcomes measured included virus neutralization, T-cell responses, and unsolicited adverse events.
The authors reported that no serious treatment-related adverse events occurred in any of the treatment arms.
Reactogenicity was mostly absent and of short duration.
The two-dose vaccine regimen induced anti-spike IgG levels and neutralizing antibody-titers exceeding those in the convalescent plasma of symptomatic patients.
The outcomes of this trial suggest that Novavax-CoV2373 has an acceptable safety profile and is able to induce a strong immune response with high neutralizing antibody titers.</p>
<p>The phase II component of the trial was designed to identify the dose regimen for the next clinical trial stage <span class="citation" data-cites="ynvQwwo9">(<a href="#ref-ynvQwwo9" role="doc-biblioref">1368</a>)</span>.
Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 μg or 25 μg Novavax-CoV2373 or a sodium-chloride placebo in two doses, 21 days apart.
In line with the phase I data, reactogenicity remained mild to moderate, with no more than 1% of participants in any group reporting grade 3 AEs, and of short duration.
Both dose levels were able to induce high anti-spike IgG titers as well as neutralizing antibody responses after the second dose.
Based on both safety and efficacy data, the 5 μg dosing regimen was selected as the optimal dose regimen for the phase III trial.</p>
<p>Novavax announced an efficacy of 89.3% based on their phase III trial in the U.K. and South Africa <span class="citation" data-cites="1D0f8OrG8 1EWB3hKG 16iOtJ1vE">(<a href="#ref-1D0f8OrG8" role="doc-biblioref">1268</a>, <a href="#ref-1EWB3hKG" role="doc-biblioref">1369</a>, <a href="#ref-16iOtJ1vE" role="doc-biblioref">1370</a>)</span>.
This trial included over 15,000 participants in the U.K. and 4,000 participants in South Africa.
The primary endpoint of the trial was the occurrence or absence of PCR-confirmed, symptomatic mild, moderate or severe COVID-19 from 7 days after the second dose onward.
In the first interim analysis (U.K.), 56 cases of COVID-19 were observed in the placebo group compared to 6 cases in the treatment group.
Importantly, the vaccine candidate also shows significant clinical efficacy against the prevalent U.K. and South African variants.
The company has also initiated the development of new constructs to select candidates that can be used as a booster against new strains and plans to initiate clinical trials for these new constructs in the second quarter of 2021.
Side effects were monitored in 2,310 participants and were generally considered mild, with low incidences of headache, muscle pain, and fatigue.
A similar efficacy of 90.4% was subsequently estimated in a phase III trial in the United States in Mexico <span class="citation" data-cites="IQOyFnfl">(<a href="#ref-IQOyFnfl" role="doc-biblioref">1269</a>)</span>.</p>
<p>Given the apparent need for boosters, interest has also emerged in whether vaccines against SARS-CoV-2 can be administered along with annual flu vaccines.
Early data came from the Novavax NVX-CoV2373 protein subunit vaccine.
In a subgroup of approximately 400 patients enrolled from the U.K. phase III trial who received either NVX-CoV2373 or placebo 1:1, a concomitant dose of adjuvanted seasonal influenza vaccines (either a trivalent vaccine or a quadrivalent vaccine) was administered <span class="citation" data-cites="IUekaKY0">(<a href="#ref-IUekaKY0" role="doc-biblioref">1371</a>)</span>.
This study demonstrated that both types of vaccines could be safely administered together.
While no change to the immune response was noted for the influenza vaccine, a notable reduction of the antibody response for the NVX-CoV2373 was reported, but efficacy was still high at 87.5% <span class="citation" data-cites="IUekaKY0">(<a href="#ref-IUekaKY0" role="doc-biblioref">1371</a>)</span>.
Novavax has since started phase I/II trials to investigate the administration of its own influenza vaccine, NanoFlu<sup>™</sup>, concomitantly with NVX-CoV2373 <span class="citation" data-cites="rclKBvtk">(<a href="#ref-rclKBvtk" role="doc-biblioref">1372</a>)</span>, which appeared to be safe and effective in preclinical studies <span class="citation" data-cites="bOwPRh6q">(<a href="#ref-bOwPRh6q" role="doc-biblioref">1373</a>)</span>.</p>
<h2 data-number="7.5" id="protein-subunit-vaccine-development-programs-prior-to-sars-cov-2"><span class="header-section-number">7.5</span> Protein Subunit Vaccine Development Programs Prior to SARS-CoV-2</h2>
<p>Earlier studies examined the immunogenicity of a SARS-CoV-1 RBD fused with IgG1 Fc.
This recombinant fusion protein could induce a robust long-lasting neutralizing antibody and cellular immune response that protected mice from SARS-CoV-1 <span class="citation" data-cites="7RHpaAHu 1EirBATaN cLAQnckq">(<a href="#ref-7RHpaAHu" role="doc-biblioref">1129</a>, <a href="#ref-cLAQnckq" role="doc-biblioref">1230</a>, <a href="#ref-1EirBATaN" role="doc-biblioref">1231</a>)</span>.
While there have been other potential protein subunit vaccines for SARS-CoV-1 investigated <em>in vivo</em> <span class="citation" data-cites="7RHpaAHu 9Zv0eLa9">(<a href="#ref-7RHpaAHu" role="doc-biblioref">1129</a>, <a href="#ref-9Zv0eLa9" role="doc-biblioref">1240</a>)</span>, none of these candidates have successfully completed clinical trials, more than likely due to the fact that the SARS-CoV-1 epidemic mostly ended by May 2004, and there was thus less of a demand or funding for SARS-CoV-1 vaccine research.</p>
<p>Similar vaccine candidates have emerged that target the RBD found in the S1 subunit of the trimeric MERS-CoV S protein, which binds to dipeptidyl-peptidase 4 (DPP4 also known as hCD26), the entry point through which MERS-CoV infects cells <span class="citation" data-cites="yoFGwkUO KZFnwRIN YZcHzPcs">(<a href="#ref-yoFGwkUO" role="doc-biblioref">1374</a>–<a href="#ref-YZcHzPcs" role="doc-biblioref">1376</a>)</span>.
After initially determining that an RBD subunit candidate (S377-588-Fc) could elicit neutralizing antibodies <span class="citation" data-cites="GFWQw4mi">(<a href="#ref-GFWQw4mi" role="doc-biblioref">1377</a>)</span>, a study in mice determined that the administration of three sequential doses of RBD-Fc vaccine coupled with MF59, a squalene immunogenic adjuvant, induced humoral and systemic immunity in mice <span class="citation" data-cites="SF030Zqn">(<a href="#ref-SF030Zqn" role="doc-biblioref">1378</a>)</span>.
Mice that had been transduced with Ad5-hCD26 and subsequently challenged with MERS-CoV five days later did not show evidence of viral infection in the lungs versus control mice at ten days post vaccination <span class="citation" data-cites="SF030Zqn">(<a href="#ref-SF030Zqn" role="doc-biblioref">1378</a>)</span>.
Other variations of this vaccine approach include a stable S trimer vaccine whereby proline-substituted variants of S2 can maintain a stable prefusion conformation of the S2 domain <span class="citation" data-cites="7RHpaAHu">(<a href="#ref-7RHpaAHu" role="doc-biblioref">1129</a>)</span>.
This approach leads to broad and potent neutralizing antibodies <span class="citation" data-cites="7RHpaAHu">(<a href="#ref-7RHpaAHu" role="doc-biblioref">1129</a>)</span></p>
<h2 data-number="7.6" id="complementary-approaches-to-vaccine-development"><span class="header-section-number">7.6</span> Complementary Approaches to Vaccine Development</h2>
<p>A complementary approach to other vaccine development programs that is being investigated explores the potential for vaccines that are not made from the SARS-CoV-2 virus to confer what has been termed trained immunity.
In a recent review <span class="citation" data-cites="103fS7Kz2">(<a href="#ref-103fS7Kz2" role="doc-biblioref">1379</a>)</span>, trained immunity was defined as forms of memory that are temporary (e.g., months or years) and reversible.
It is induced by exposure to whole-microorganism vaccines or other microbial stimuli that generates heterologous protective effects.
Trained immunity can be displayed by innate immune cells or innate immune features of other cells, and it is characterized by alterations to immune responsiveness to future immune challenges due to epigenetic and metabolic mechanisms.
These alterations can take the form of either an increased or decreased response to immune challenge by a pathogen.
Trained immunity elicited by non-SARS-CoV-2 whole-microorganism vaccines could potentially improve SARS-CoV-2 susceptibility or severity <span class="citation" data-cites="Vu1VILWK">(<a href="#ref-Vu1VILWK" role="doc-biblioref">1380</a>)</span>.</p>
<p>One type of stimulus which research indicates can induce trained immunity is bacillus Calmette-Guerin (BCG) vaccination.
BCG is an attenuated form of bacteria <em>Mycobacterium bovis</em>.
The vaccine is most commonly administered for the prevention of tuberculosis in humans.
Clinical trials in non-SARS-CoV-2-infected adults have been designed to assess whether BCG vaccination could have prophylactic effects against SARS-CoV-2 by reducing susceptibility, preventing infection, or reducing disease severity.
A number of trials are now evaluating the effects of the BCG vaccine or the related vaccine VPM1002 <span class="citation" data-cites="Vu1VILWK 9m3rP633 y9IYdfM3 xdqxBruc 962rELVS EuwTWcPi dQtUeruv DjXsPR8O 10OE6y3Pv 86OjIybR ITO15LIz VkZGZxLn 13JVjMfQI nk2MVsld 1E2t9tr8h">(<a href="#ref-9m3rP633" role="doc-biblioref">1142</a>, <a href="#ref-xdqxBruc" role="doc-biblioref">1143</a>, <a href="#ref-Vu1VILWK" role="doc-biblioref">1380</a>–<a href="#ref-1E2t9tr8h" role="doc-biblioref">1392</a>)</span>.</p>
<p>The ongoing trials are using a number of different approaches.
Some trials enroll healthcare workers, other trials hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge, and yet another set of trials older adults (&gt;50 years) under chronic care for conditions like hypertension and diabetes.
One set of trials, for example, uses time until first infection as the primary study endpoint; more generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
Some analyses have suggested a possible correlation at the country level between the frequency of BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 <span class="citation" data-cites="Vu1VILWK">(<a href="#ref-Vu1VILWK" role="doc-biblioref">1380</a>)</span>.
Currently it is unclear whether this correlation has any connection to trained immunity.
Many possible confounding factors are also likely to vary among countries, such as age distribution, detection efficiency, stochastic epidemic dynamic effects, differences in healthcare capacity over time in relation to epidemic dynamics, and these have not been adequately accounted for in current analyses. <!-- TO DO: add in study that debunks this-->
It is unclear whether there is an effect of the timing of BCG vaccination, both during an individual’s life cycle and relative to the COVID-19 pandemic.
Additionally, given that severe SARS-CoV-2 may be associated with a dysregulated immune response, it is unclear what alterations to the immune response would be most likely to be protective versus pathogenic (e.g., <span class="citation" data-cites="i5k18bpX 1GnFL9zeN Vu1VILWK 1228YjPRv">(<a href="#ref-1GnFL9zeN" role="doc-biblioref">149</a>, <a href="#ref-Vu1VILWK" role="doc-biblioref">1380</a>, <a href="#ref-i5k18bpX" role="doc-biblioref">1393</a>, <a href="#ref-1228YjPRv" role="doc-biblioref">1394</a>)</span>).
The article <span class="citation" data-cites="Vu1VILWK">(<a href="#ref-Vu1VILWK" role="doc-biblioref">1380</a>)</span> proposes that trained immunity might lead to an earlier and stronger response, which could in turn reduce viremia and the risk of later, detrimental immunopathology.
While trained immunity is an interesting possible avenue to complement vaccine development efforts through the use of an existing vaccine, additional research is required to assess whether the BCG vaccine is likely to confer trained immunity in the case of SARS-CoV-2.</p>
<h2 data-number="7.7" id="india-and-chinas-roles-in-vaccine-innovation-and-development"><span class="header-section-number">7.7</span> India and China’s Roles in Vaccine Innovation and Development</h2>
<p>The nations of China and India have played a major role as COVID-19 vaccination developers and providers.
Considering India produced approximately 60% of the world’s vaccines prior to the pandemic, it is no surprise that the nation has developed and is developing several COVID-19 vaccine candidates.
In addition to Covaxin, the Bio E subunit vaccine CORBEVAX is being produced by Biological E in collaboration with U.S.-based Dynavax and the Baylor College of Medicine <span class="citation" data-cites="gsNWcXHn">(<a href="#ref-gsNWcXHn" role="doc-biblioref">1395</a>)</span>.
These two home-grown vaccines are now approved for adults and children as young as five (CORBEVAX) and six (Covaxin) <span class="citation" data-cites="I9Ha5JL3">(<a href="#ref-I9Ha5JL3" role="doc-biblioref">1396</a>)</span>.</p>
<p>Other vaccines licensed by India were developed elsewhere but produced in India.
For example, Novavax (developed in the United States) has signed an agreement with the Serum Institute of India allowing them to produce up to 2 billion doses a year <span class="citation" data-cites="e8pnj0O3">(<a href="#ref-e8pnj0O3" role="doc-biblioref">1397</a>)</span>.
Similarly, many people within India have been vaccinated with the AstraZeneca-University of Oxford vaccine, known as Covishield in India, which is also produced by the Serum Institute of India <span class="citation" data-cites="gsNWcXHn">(<a href="#ref-gsNWcXHn" role="doc-biblioref">1395</a>)</span>.
India is also developing vaccines using cutting-edge nucleic-acid-based platforms..
These include ZyCov-D, a DNA vaccine produced by Zydus Cadila, HGCO19 and India’s first mRNA vaccine, produced by Genova and HDT Biotech Corporation (of the U.S.) <span class="citation" data-cites="gsNWcXHn">(<a href="#ref-gsNWcXHn" role="doc-biblioref">1395</a>)</span>.
Additionally, in February 2021, Bharat Biotech received approval from Indian officials to commence a phase I study of an intranasal chimpanzee-adenovirus (ChAd) vectored SARS-CoV-2-S vaccine called BBV154 <span class="citation" data-cites="P9mD7Gc9">(<a href="#ref-P9mD7Gc9" role="doc-biblioref">1398</a>)</span>.</p>
<p>In China, the Sinopharm-Beijing Institute vaccine, the Sinopharm-Wuhan Institute of Biological Products vaccine, the Sinovac Biotech (CoronaVac) vaccine, and CanSino Biologics vaccine are the main vaccines being distributed.
Sinovac and Sinopharm aimed to produce 2 billion doses by the end of 2021, and they have distributed vaccines as aid to the Philippines and Pakistan <span class="citation" data-cites="gdTtuj5e">(<a href="#ref-gdTtuj5e" role="doc-biblioref">1399</a>)</span>.
In contrast, the Sinopharm-Wuhan vaccine, which has been approved for use in China since February 25, 2021, has been distributed almost exclusively within China, with limited supplies distributed to the United Arab Emirates <span class="citation" data-cites="wByD9WaX">(<a href="#ref-wByD9WaX" role="doc-biblioref">1171</a>)</span>.
On the same date, the CanSino vaccine was approved for use in China and has been granted emergency use in several other countries <span class="citation" data-cites="wByD9WaX">(<a href="#ref-wByD9WaX" role="doc-biblioref">1171</a>)</span>.</p>
<p>However, the vaccine approval and distribution processes in China have come under increased scrutiny from other nations.
China was criticized for administering vaccines to thousands of government officials and state-owned businesses in September 2020, prior to the completion of phase III clinical trials <span class="citation" data-cites="1Bgnim0gX">(<a href="#ref-1Bgnim0gX" role="doc-biblioref">1400</a>)</span>.
The behavior of Chinese officials has also come into question due to misinformation campaigns questioning the safety of Western vaccine candidates such as Moderna and Pfizer-BioNTech in a way that is intended to highlight the benefits of their own vaccine candidates <span class="citation" data-cites="gdTtuj5e">(<a href="#ref-gdTtuj5e" role="doc-biblioref">1399</a>)</span>.
China in particular took aim at mRNA technologies, but Chinese companies have since developed their own mRNA vaccines targeting the omicron variant, one of which is due to begin trials soon in the UAE <span class="citation" data-cites="V2Gz2O9G">(<a href="#ref-V2Gz2O9G" role="doc-biblioref">1401</a>)</span>.
Furthermore, delays in vaccine distribution have also caused issues, particularly in Turkey where 10 million doses of Sinovac were due to arrive by December 2020, but instead only 3 million were delivered in early January <span class="citation" data-cites="gdTtuj5e">(<a href="#ref-gdTtuj5e" role="doc-biblioref">1399</a>)</span>.
Similar delays and shortages of doses promised were reported by officials in the Philippines, Egypt, Morocco, and the United Arab Emirates <span class="citation" data-cites="SGWVc4nD 12zVLzkpB">(<a href="#ref-SGWVc4nD" role="doc-biblioref">1402</a>, <a href="#ref-12zVLzkpB" role="doc-biblioref">1403</a>)</span>.
All the same, Sinovac’s vaccine has since been approved for use in countries around the world <span class="citation" data-cites="wByD9WaX">(<a href="#ref-wByD9WaX" role="doc-biblioref">1171</a>)</span>.</p>
<h1 data-number="8" id="the-coming-of-age-of-nucleic-acid-vaccines-during-covid-19"><span class="header-section-number">8</span> The Coming of Age of Nucleic Acid Vaccines during COVID-19</h1>
<h2 data-number="8.1" id="abstract-5"><span class="header-section-number">8.1</span> Abstract</h2>
<p>In the 21<sup>st</sup> century, several emergent viruses have posed a global threat.
Each pathogen has emphasized the value of rapid and scalable vaccine development programs.
The ongoing SARS-CoV-2 pandemic has made the importance of such efforts especially clear.</p>
<p>New biotechnological advances in vaccinology allow for recent advances that provide only the nucleic acid building blocks of an antigen, eliminating many safety concerns.
During the COVID-19 pandemic, these DNA and RNA vaccines have facilitated the development and deployment of vaccines at an unprecedented pace.
This success was attributable at least in part to broader shifts in scientific research relative to prior epidemics; the genome of SARS-CoV-2 was available as early as January 2020, facilitating global efforts in the development of DNA and RNA vaccines within two weeks of the international community becoming aware of the new viral threat.
Additionally, these technologies that were previously only theoretical are not only safe but also highly efficacious.</p>
<p>Although historically a slow process, the rapid development of vaccines during the COVID-19 crisis reveals a major shift in vaccine technologies.
Here, we provide historical context for the emergence of these paradigm-shifting vaccines.
We describe several DNA and RNA vaccines and in terms of their efficacy, safety, and approval status.
We also discuss patterns in worldwide distribution.
The advances made since early 2020 provide an exceptional illustration of how rapidly vaccine development technology has advanced in the last two decades in particular and suggest a new era in vaccines against emerging pathogens.</p>
<h2 data-number="8.2" id="importance-5"><span class="header-section-number">8.2</span> Importance</h2>
<p>The SARS-CoV-2 pandemic has caused untold damage globally, presenting unusual demands on but also unique opportunities for vaccine development.
The development, production, and distribution of vaccines is imperative to saving lives, preventing severe illness, and reducing the economic and social burdens caused by the COVID-19 pandemic.
Although vaccine technologies that provide the DNA or RNA sequence of an antigen had never previously been approved for use in humans, they have played a major role in the management of SARS-CoV-2.
In this review we discuss the history of these vaccines and how they have been applied to SARS-CoV-2.
Additionally, given that the evolution of new SARS-CoV-2 variants continues to present a significant challenge in 2022, these vaccines remain an important and evolving tool in the biomedical response to the pandemic.</p>
<h2 data-number="8.3" id="introduction-5"><span class="header-section-number">8.3</span> Introduction</h2>
<p>The SARS-CoV-2 virus emerged at the end of 2019 and soon spread around the world.
In response, the Coalition for Epidemic Preparedness Innovations quickly began coordinating global health agencies and pharmaceutical companies to develop vaccines, as vaccination is one of the primary approaches available to combat the effects of a virus.
Vaccines can bolster the immune response to a virus at both the individual and population levels, thereby reducing fatalities and severe illness and potentially driving a lower rate of infection even for a highly infectious virus like SARS-CoV-2.
However, vaccines have historically required a lengthy development process due to both the experimental and regulatory demands.</p>
<p>As we review in a companion manuscript <span class="citation" data-cites="1HWmcSB20">(<a href="#ref-1HWmcSB20" role="doc-biblioref">6</a>)</span>, vaccine technologies prior to the COVID-19 pandemic were largely based on triggering an immune response by introducing a virus or one of its components.
Such vaccines are designed to induce an adaptive immune response without causing the associated viral illness.
Each time a virus emerges that poses a significant global threat, as has happened several times over the past 20 years, the value of a rapid vaccine response is underscored.
With progressive biotechnological developments, this objective has become increasingly tangible.</p>
<p>In the current century, significant advances in vaccine development have largely been built on genomics, as is somewhat unsurprising given the impact of the Genomic Revolution across all biology.
This shift towards nucleic acid-based technologies opens a new frontier in vaccinology, where just the sequence encoding an antigen can be introduced to induce an immune response.
While other platforms can carry some risks related to introducing all or part of a virus <span class="citation" data-cites="1HWmcSB20">(<a href="#ref-1HWmcSB20" role="doc-biblioref">6</a>)</span>, nucleic acid-based platforms eliminate these risks entirely.
Additionally, vaccine technologies that could be adjusted for novel viral threats are appealing because this modular approach would mean they could enter trials quickly in response to a new pathogen of concern.</p>
<h2 data-number="8.4" id="honing-a-21st-century-response-to-emergent-viral-threats"><span class="header-section-number">8.4</span> Honing a 21<sup>st</sup> Century Response to Emergent Viral Threats</h2>
<p>Recently, vaccine technologies have been developed and refined in response to several epidemics that did not reach the level of destruction caused by COVID-19.
Emergent viral threats of the 21<sup>st</sup> century include severe acute respiratory syndrome (SARS), the H1N1 influenza strain known as swine flu, Middle East respiratory syndrome (MERS), Ebola virus disease, COVID-19, and, most recently, monkeypox, all of which have underscored the importance of a rapid global response to a new infectious virus.
Because the vaccine development process has historically been slow, the use of vaccines to control most of these epidemics was limited.</p>
<p>One of the more successful recent vaccine development programs was for H1N1 influenza.
This program benefited from the strong existing infrastructure for influenza vaccines along with the fact that regulatory agencies had determined that vaccines produced using egg- and cell-based platforms could be licensed under the regulations used for a strain change <span class="citation" data-cites="HyYY2agc">(<a href="#ref-HyYY2agc" role="doc-biblioref">1347</a>)</span>.
Although a monovalent H1N1 vaccine was not available before the pandemic peaked in the United States of America (U.S.A.) and Europe, it became available soon afterward as a stand-alone vaccine that was eventually incorporated into commercially available seasonal influenza vaccines <span class="citation" data-cites="HyYY2agc">(<a href="#ref-HyYY2agc" role="doc-biblioref">1347</a>)</span>.
Critiques of the production and distribution of the H1N1 vaccine have stressed the need for alternative development-and-manufacturing platforms that can be readily adapted to new pathogens.</p>
<p>Efforts to develop such approaches had been undertaken prior to the COVID-19 pandemic.
DNA vaccine development efforts began for SARS-CoV-1 but did not proceed past animal testing <span class="citation" data-cites="AOGjkjCq">(<a href="#ref-AOGjkjCq" role="doc-biblioref">1346</a>)</span>.
Likewise, the development of viral-vectored Ebola virus vaccines was undertaken, but the pace of vaccine development was behind the spread of the virus from early on <span class="citation" data-cites="vTrIB9zS">(<a href="#ref-vTrIB9zS" role="doc-biblioref">1404</a>)</span>.
Although a candidate Ebola vaccine V920 showed promise in preclinical and clinical testing, it did not receive breakthrough therapy designation until the summer of 2016, by which time the Ebola outbreak was winding down <span class="citation" data-cites="8uuVgxzA">(<a href="#ref-8uuVgxzA" role="doc-biblioref">1405</a>)</span>.
Therefore, the COVID-19 pandemic has been the first case where vaccines have been available early enough to significantly influence outcomes at the global scale.</p>
<p>The pandemic caused by SARS-CoV-2 has highlighted a confluence of circumstances that positioned vaccine development as a key player in efforts to control the virus and mitigate its damage.
This virus did not follow the same trajectory as other emergent viruses of recent note, such as SARS-CoV-1, MERS-CoV, and Ebola virus, none of which presented a global threat for such a sustained duration (see visualization in <span class="citation" data-cites="njpLhBui">(<a href="#ref-njpLhBui" role="doc-biblioref">3</a>)</span>).
Spread of the SARS-CoV-2 virus has remained out of control in many parts of the world into 2022, especially with the emergence of novel variants exhibiting increased rates of transmission <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span>.
While, for a variety of reasons, SARS-CoV-2 was not controlled as rapidly as the viruses underlying prior 21<sup>st</sup> century epidemics, vaccine development technology had also progressed based on these and other prior viral threats to the point that a rapid international vaccine development response was possible.</p>
<h2 data-number="8.5" id="development-of-covid-19-vaccines-using-dna-and-rna-platforms"><span class="header-section-number">8.5</span> Development of COVID-19 Vaccines using DNA and RNA Platforms</h2>
<p>Vaccine development programs for COVID-19 emerged very quickly.
The first administration of a dose of a COVID-19 vaccine to a human trial participant occurred on March 16, 2020 <span class="citation" data-cites="fQvzeptv 1GA95MF2m">(<a href="#ref-fQvzeptv" role="doc-biblioref">1406</a>, <a href="#ref-1GA95MF2m" role="doc-biblioref">1407</a>)</span>, marking an extremely rapid response to the emergence of SARS-CoV-2.
Within a few weeks of this first trial launching, at least 78 vaccine development programs were active <span class="citation" data-cites="1GA95MF2m">(<a href="#ref-1GA95MF2m" role="doc-biblioref">1407</a>)</span>, and by September 2020, there were over 180 vaccine candidates against SARS-CoV-2 in development <span class="citation" data-cites="dqpEe5Lz">(<a href="#ref-dqpEe5Lz" role="doc-biblioref">1114</a>)</span>.
As of December 2, 2022, 50 SARS-CoV-2 vaccines have been approved world wide and 27 are being administered throughout the world, with 13.0 billion doses administered across 223 countries.
The first critical step towards developing a vaccine against SARS-CoV-2 was characterizing the viral target, which happened extremely early in the COVID-19 outbreak with the sequencing and dissemination of the viral genome in early January 2020 <span class="citation" data-cites="vlGP3RAU">(<a href="#ref-vlGP3RAU" role="doc-biblioref">555</a>)</span> (Figure <a href="#fig:virus">10</a>).
This genomic information allowed for an early identification of the sequence of the Spike (S) protein (Figure <a href="#fig:virus">10</a>), which is the antigen and induces an immune response <span class="citation" data-cites="Vnbw9o3T 13wCBLnnu">(<a href="#ref-Vnbw9o3T" role="doc-biblioref">1408</a>, <a href="#ref-13wCBLnnu" role="doc-biblioref">1409</a>)</span>.</p>
<div id="fig:virus" class="fignos">
<figure>
<img src="images/SARS_CoV_2.png" alt="Figure 10: Structure of the SARS-CoV-2 virus. The development of vaccines depends on the immune system recognizing the virus. Here, the structure of SARS-CoV-2 is represented both in the abstract and against a visualization of the virion. The abstracted visualization was made using BioRender (1410) using the template “Human Coronavirus Structure” by BioRender (August 2020) (1411). The microscopy was conducted by the National Institute of Allergy and Infectious Diseases (1412)." />
<figcaption aria-hidden="true"><span>Figure 10:</span> <strong>Structure of the SARS-CoV-2 virus.</strong>
The development of vaccines depends on the immune system recognizing the virus.
Here, the structure of SARS-CoV-2 is represented both in the abstract and against a visualization of the virion.
The abstracted visualization was made using BioRender <span class="citation" data-cites="14FBejgLM">(<a href="#ref-14FBejgLM" role="doc-biblioref">1410</a>)</span> using the template “Human Coronavirus Structure” by BioRender (August 2020) <span class="citation" data-cites="EAzPxBbg">(<a href="#ref-EAzPxBbg" role="doc-biblioref">1411</a>)</span>.
The microscopy was conducted by the National Institute of Allergy and Infectious Diseases <span class="citation" data-cites="Jzj97hJh">(<a href="#ref-Jzj97hJh" role="doc-biblioref">1412</a>)</span>.</figcaption>
</figure>
</div>
<p>During the development process, one measure used to assess whether a vaccine candidate is likely to provide protection is serum neutralizing activity <span class="citation" data-cites="wiGjCZC8">(<a href="#ref-wiGjCZC8" role="doc-biblioref">1118</a>)</span>.
This assay evaluates the presence of antibodies that can neutralize, or prevent infection by, the virus in question.
Often, titration is used to determine the extent of neutralization activity.
However, unlike in efforts to develop vaccines for prior viral threats, the duration of the COVID-19 pandemic has made it possible to also test vaccines in phase III trials where the effect of the vaccines on a cohort’s likelihood of contracting SARS-CoV-2 was evaluated.</p>
<h2 data-number="8.6" id="theory-and-implementation-of-nucleic-acid-vaccines"><span class="header-section-number">8.6</span> Theory and Implementation of Nucleic Acid Vaccines</h2>
<p>Biomedical research in the 21<sup>st</sup> century has been significantly influenced by the genomic revolution.
While traditional methods of vaccine development, such as inactivated whole viruses are still used today <span class="citation" data-cites="1HWmcSB20">(<a href="#ref-1HWmcSB20" role="doc-biblioref">6</a>)</span>, vaccine development is no exception.
The shift towards omics-based approaches to vaccine development began to take hold with the meningococcal type B vaccine, which was developed using reverse vaccinology in the early 2010s <span class="citation" data-cites="MCZBJ5sF fw8IwtHq">(<a href="#ref-MCZBJ5sF" role="doc-biblioref">1413</a>, <a href="#ref-fw8IwtHq" role="doc-biblioref">1414</a>)</span>.
Under this approach, the genome of a pathogen is screened to identify potential vaccine targets <span class="citation" data-cites="fw8IwtHq">(<a href="#ref-fw8IwtHq" role="doc-biblioref">1414</a>)</span>, and pathogens of interest are then expressed <em>in vitro</em> and tested in animal models to determine their immunogenicity <span class="citation" data-cites="fw8IwtHq">(<a href="#ref-fw8IwtHq" role="doc-biblioref">1414</a>)</span>.
In this way, the genomic revolution catalyzed a fundamental shift in the development of vaccines.
Such technologies could revolutionize the role of vaccines given their potential to address one of the major limitations of vaccines today and facilitate the design of therapeutic, rather than just prophylactic, vaccines <span class="citation" data-cites="kqerKJKY">(<a href="#ref-kqerKJKY" role="doc-biblioref">1415</a>)</span>.</p>
<p>Nucleic-acid based approaches share an underlying principle: a vector that delivers the information needed to produce an antigen.
When the host cells manufacture the antigen, it can then trigger an immune response.
The fact that no part of the virus is introduced aside from the genetic code of the antigen means that these vaccines carry no risk of infection.
Such approaches build on subunit vaccination strategies, where a component of a virus (e.g., an antigenic protein) is delivered by the vaccine.
Platforms based on genomic sequencing began to be explored beginning in the 1980s as genetic research became increasingly feasible.
Advances in genetic engineering allowed for gene sequences of specific viral antigens to be grown <em>in vitro</em> <span class="citation" data-cites="YY3x3bBV">(<a href="#ref-YY3x3bBV" role="doc-biblioref">1113</a>)</span>.
Studies also demonstrated that model organisms could be induced to construct antigens that would trigger an immune response <span class="citation" data-cites="U9ZIZWkB pWMIo6pD uPszIvSj">(<a href="#ref-U9ZIZWkB" role="doc-biblioref">1125</a>, <a href="#ref-pWMIo6pD" role="doc-biblioref">1416</a>, <a href="#ref-uPszIvSj" role="doc-biblioref">1417</a>)</span>.
These two developments sparked interest in whether it could be possible to identify any or all of the antigens encoded by a virus’s genome and train the immune response to recognize them.</p>
<p>The delivery and presentation of antigens is fundamental to inducing immunity against a virus.
Vaccines that deliver nucleic acids allow the introduction of foreign substances to the body to induce both humoral and cellular immune responses <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.
Delivering a nucleic acid sequence to host cells allows the host to synthesize an antigen without exposure to a viral threat <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.
Host-synthesized antigens can activate both humoral and cellular immunity <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>, as they can be presented in complex with major histocompatibility complex (MHC) I and II, which can activate either T- or B-cells <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.
In contrast, prior approaches activated only MHC II <span class="citation" data-cites="uPszIvSj">(<a href="#ref-uPszIvSj" role="doc-biblioref">1417</a>)</span>.
Because these vaccines encode specific proteins, providing many of the benefits of a protein subunit vaccine, they do not carry any risk of DNA being live, replicating, or spreading, and their manufacturing process lends itself to scalability <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.
Here, opportunities can be framed in terms of the central dogma of genetics: instead of directly providing the proteins from the infectious agents, vaccines developers are exploring the potential for the delivery of DNA or RNA to induce the cell to produce proteins from the virus that in turn induce a host immune response.</p>
<h2 data-number="8.7" id="cross-platform-considerations-in-vaccine-development"><span class="header-section-number">8.7</span> Cross-Platform Considerations in Vaccine Development</h2>
<p>Certain design decisions are relevant to vaccine development across multiple platforms.
One applies to the platforms that deliver the antigen, which in the case of SARS-CoV-2 vaccines is the S protein.
The prefusion conformation of the S protein, which is the structure before the virus fuses to the host cell membrane, is metastable <span class="citation" data-cites="R7Xdh5nH">(<a href="#ref-R7Xdh5nH" role="doc-biblioref">1419</a>)</span>, and the release of energy during membrane fusion drives this process forward following destabilization <span class="citation" data-cites="17DSmRo9H 3uddYea8">(<a href="#ref-17DSmRo9H" role="doc-biblioref">12</a>, <a href="#ref-3uddYea8" role="doc-biblioref">1420</a>)</span>.
Due to the significant conformational changes that occur during membrane fusion <span class="citation" data-cites="qcVbT0w4 hIc3bKWe zK0rFpz1">(<a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>, <a href="#ref-hIc3bKWe" role="doc-biblioref">1421</a>, <a href="#ref-zK0rFpz1" role="doc-biblioref">1422</a>)</span>, S protein immunogens that are stabilized in the prefusion conformation are of particular interest, especially because a prefusion stabilized <em>Middle East respiratory syndrome-related coronavirus</em> (MERS-CoV) S antigen was found to elicit an improved antibody response <span class="citation" data-cites="oghHqZDt">(<a href="#ref-oghHqZDt" role="doc-biblioref">1246</a>)</span>.
Moreover, the prefusion conformation offers an opportunity to target S2, a region of the S protein that accumulates mutations at a slower rate <span class="citation" data-cites="13wWdgODZ OVsxrEuX oghHqZDt">(<a href="#ref-13wWdgODZ" role="doc-biblioref">23</a>, <a href="#ref-OVsxrEuX" role="doc-biblioref">24</a>, <a href="#ref-oghHqZDt" role="doc-biblioref">1246</a>)</span> (see also <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span>).
Vaccine developers can stabilize the prefusion conformer by selecting versions of the S protein containing mutations that lock the position <span class="citation" data-cites="lvq9hGmj">(<a href="#ref-lvq9hGmj" role="doc-biblioref">1423</a>)</span>.
The immune response to the Spike protein when it is stabilized in this conformation is improved over other S structures <span class="citation" data-cites="10UC562ga">(<a href="#ref-10UC562ga" role="doc-biblioref">1424</a>)</span>.
Thus, vaccines that use this prefusion stabilized conformation are expected to not only offer improved immunogenicity, but also be more resilient to the accumulation of mutations in SARS-CoV-2.</p>
<p>Another cross-platform consideration is the use of adjuvants.
Adjuvants include a variety of molecules or larger microbial-related products that affect the immune system broadly or an immune response of interest.
They can either be comprised of or contain immunostimulants or immunomodulators.
Adjuvants are sometimes included within vaccines in order to enhance the immune response.
Different adjuvants can regulate different types of immune responses, so the type or combination of adjuvants used in a vaccine will depend on both the type of vaccine and concern related to efficacy and safety.
A variety of possible mechanisms for adjuvants have been investigated <span class="citation" data-cites="13bVbfc5h 122h6fIxE uO0uqhxc">(<a href="#ref-13bVbfc5h" role="doc-biblioref">1425</a>–<a href="#ref-uO0uqhxc" role="doc-biblioref">1427</a>)</span>.</p>
<p>Due to viral evolution, vaccine developers are in an arms race with a pathogen that benefits from mutations that reduce its susceptibility to adaptive immunity.
The evolution of several variants of concern (VOC) presents significant challenges for vaccines developed based on the index strain identified in Wuhan in late 2019.
We discuss these variants in depth elsewhere in the COVID-19 Review Consortium project <span class="citation" data-cites="yTsmmAYC">(<a href="#ref-yTsmmAYC" role="doc-biblioref">1139</a>)</span>.
To date, the most significant variants of concern identified are Alpha (2020), Beta (2020), Gamma (2020), Delta (2021), Omicron (2021), and related Omicron subvariants (2022).
The effectiveness or efficacy (i.e., trial or real-world prevention, respectively) of vaccines in the context of these variants is discussed where information is available.</p>
<h2 data-number="8.8" id="dna-vaccine-platforms"><span class="header-section-number">8.8</span> DNA Vaccine Platforms</h2>
<p>DNA vaccine technologies have developed slowly over the past thirty years.
These vaccines introduce a vector containing a DNA sequence that encodes antigen(s) selected to induce a specific immune response <span class="citation" data-cites="uPszIvSj">(<a href="#ref-uPszIvSj" role="doc-biblioref">1417</a>)</span>.
Early attempts revealed issues with low immunogenicity <span class="citation" data-cites="U9ZIZWkB uPszIvSj 12jFcMeQY">(<a href="#ref-U9ZIZWkB" role="doc-biblioref">1125</a>, <a href="#ref-uPszIvSj" role="doc-biblioref">1417</a>, <a href="#ref-12jFcMeQY" role="doc-biblioref">1428</a>)</span>.
Additionally, initial skepticism about the approach suggested that DNA vaccines might bind to the host genome or induce autoimmune disease <span class="citation" data-cites="fwumPoq1 BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>, <a href="#ref-fwumPoq1" role="doc-biblioref">1429</a>)</span>, but pre-clinical and clinical studies have consistently disproved this hypothesis and indicated DNA vaccines to be safe <span class="citation" data-cites="12jFcMeQY">(<a href="#ref-12jFcMeQY" role="doc-biblioref">1428</a>)</span>.
Another concern, antibiotic resistance introduced during the plasmid selection process, did remain a concern during this initial phase of development <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>, but this issue was resolved through strategic vector design <span class="citation" data-cites="Wjtx0VXu 5fcD0JWR">(<a href="#ref-Wjtx0VXu" role="doc-biblioref">1430</a>, <a href="#ref-5fcD0JWR" role="doc-biblioref">1431</a>)</span>.
However, for many years, the immunogenicity of DNA vaccines failed to reach expectations <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.
Several developments during the 2010s led to greater efficacy of DNA vaccines <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.
However, no DNA vaccines had been approved for use in humans prior to the COVID-19 pandemic <span class="citation" data-cites="12jFcMeQY yriARFOF">(<a href="#ref-12jFcMeQY" role="doc-biblioref">1428</a>, <a href="#ref-yriARFOF" role="doc-biblioref">1432</a>)</span>.
As of December 2, 2022, 10 vaccines have been approved worldwide (Table <a href="#tbl:approved-DNA">4</a>).
These vaccines fall into two categories, vaccines that are vectored with a plasmid and those that are vectored with another virus.</p>
<div id="tbl:approved-DNA" class="tablenos">
<table id="tbl:approved-DNA">
<caption><span>Table 4:</span> DNA vaccines approved in at least one country <span class="citation" data-cites="jswAyWIs">(<a href="#ref-jswAyWIs" role="doc-biblioref">1160</a>)</span> as of December 2, 2022.
</caption>
<colgroup>
<col style="width: 43%" />
<col style="width: 27%" />
<col style="width: 28%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Vaccine</th>
<th style="text-align: left;">Company</th>
<th style="text-align: left;">Platform</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">iNCOVACC</td>
<td style="text-align: left;">Bharat Biotech</td>
<td style="text-align: left;">non replicating viral vector</td>
</tr>
<tr class="even">
<td style="text-align: left;">Convidecia</td>
<td style="text-align: left;">CanSino</td>
<td style="text-align: left;">non replicating viral vector</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Convidecia Air</td>
<td style="text-align: left;">CanSino</td>
<td style="text-align: left;">non replicating viral vector</td>
</tr>
<tr class="even">
<td style="text-align: left;">Gam-COVID-Vac</td>
<td style="text-align: left;">Gamaleya</td>
<td style="text-align: left;">non replicating viral vector</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Sputnik Light</td>
<td style="text-align: left;">Gamaleya</td>
<td style="text-align: left;">non replicating viral vector</td>
</tr>
<tr class="even">
<td style="text-align: left;">Sputnik V</td>
<td style="text-align: left;">Gamaleya</td>
<td style="text-align: left;">non replicating viral vector</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Jcovden</td>
<td style="text-align: left;">Janssen (Johnson &amp; Johnson)</td>
<td style="text-align: left;">non replicating viral vector</td>
</tr>
<tr class="even">
<td style="text-align: left;">Vaxzevria</td>
<td style="text-align: left;">Oxford/AstraZeneca</td>
<td style="text-align: left;">non replicating viral vector</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Covishield (Oxford/ AstraZeneca formulation)</td>
<td style="text-align: left;">Serum Institute of India</td>
<td style="text-align: left;">non replicating viral vector</td>
</tr>
<tr class="even">
<td style="text-align: left;">ZyCoV-D</td>
<td style="text-align: left;">Zydus Cadila</td>
<td style="text-align: left;">plasmid vectored</td>
</tr>
</tbody>
</table>
</div>
<h3 data-number="8.8.1" id="plasmid-vectored-dna-vaccines"><span class="header-section-number">8.8.1</span> Plasmid-Vectored DNA Vaccines</h3>
<p>Many DNA vaccines use a plasmid vector-based approach, where the sequence encoding the antigen(s) against which an immune response is sought are cultivated in a plasmid and delivered directly to an appropriate tissue <span class="citation" data-cites="XnrBoKVk">(<a href="#ref-XnrBoKVk" role="doc-biblioref">1433</a>)</span>.
Plasmids can also be designed to act as adjuvants by targeting essential regulators of pathways such as the inflammasome or simply just specific cytokines <span class="citation" data-cites="fwumPoq1 NcVQyEao">(<a href="#ref-fwumPoq1" role="doc-biblioref">1429</a>, <a href="#ref-NcVQyEao" role="doc-biblioref">1434</a>)</span>.
The DNA itself may also stimulate the innate immune response <span class="citation" data-cites="uPszIvSj 5fcD0JWR">(<a href="#ref-uPszIvSj" role="doc-biblioref">1417</a>, <a href="#ref-5fcD0JWR" role="doc-biblioref">1431</a>)</span>.
Once the plasmid brings the DNA sequence to an antigen-presenting cell (APC), the host machinery can be used to construct antigen(s) from the transported genetic material, and the body can then synthesize antibodies in response <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.
The vectors are edited to remove extra sequences <span class="citation" data-cites="5fcD0JWR">(<a href="#ref-5fcD0JWR" role="doc-biblioref">1431</a>)</span>.
These types of manufacturing advances have improved the safety and throughput of this platform <span class="citation" data-cites="5fcD0JWR">(<a href="#ref-5fcD0JWR" role="doc-biblioref">1431</a>)</span>.</p>
<h4 data-number="8.8.1.1" id="prior-applications"><span class="header-section-number">8.8.1.1</span> Prior Applications</h4>
<p>In the 1990s and 2000s, DNA vaccines delivered via plasmids sparked significant scientific interest, leading to a large number of preclinical trials <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.
Early preclinical trials primarily focused on long-standing disease threats, including viral diseases such as rabies and parasitic diseases such as malaria, and promising results led to phase I testing of the application of this technology to human immunodeficiency virus (HIV), influenza, malaria, and other diseases of concern during this period <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.
Although they were well-tolerated, these early attempts to develop vaccines were generally not very successful in inducing immunity to the target pathogen, with either limited T-cell or limited neutralizing antibody responses observed <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.</p>
<p>Early plasmid-vectored DNA vaccine trials targeted HIV and subsequently diseases of worldwide importance such as malaria and hepatitis B <span class="citation" data-cites="3EKs730C">(<a href="#ref-3EKs730C" role="doc-biblioref">1435</a>)</span>.
The concern with these early development projects was immunogenicity, not safety <span class="citation" data-cites="3EKs730C">(<a href="#ref-3EKs730C" role="doc-biblioref">1435</a>)</span>.
Around the turn of the millennium, a hepatitis B vaccine development program demonstrated that these vaccines can induce both antibody and cellular immune response <span class="citation" data-cites="jPpzjaYO">(<a href="#ref-jPpzjaYO" role="doc-biblioref">1436</a>)</span>.
Prior to COVID-19, however, plasmid-vectored DNA vaccines had been approved for commercial use only in veterinary populations <span class="citation" data-cites="11NePN3S6 fgs4epPY Wn7dAdt">(<a href="#ref-11NePN3S6" role="doc-biblioref">1437</a>–<a href="#ref-Wn7dAdt" role="doc-biblioref">1439</a>)</span>.
Between 2005 and 2006, several DNA vaccines were developed for non-human animal populations, including against viruses including a rhabdovirus in fish <span class="citation" data-cites="LBoZ0TI7">(<a href="#ref-LBoZ0TI7" role="doc-biblioref">1440</a>)</span>, porcine reproductive and respiratory syndrome virus <span class="citation" data-cites="23N627SI">(<a href="#ref-23N627SI" role="doc-biblioref">1441</a>)</span>, and West Nile virus in horses <span class="citation" data-cites="du9lwp7h">(<a href="#ref-du9lwp7h" role="doc-biblioref">1442</a>)</span>.
Within the past five years, additional plasmid-vectored vaccines for immunization against viruses were developed against a herpesvirus (in mice) <span class="citation" data-cites="1AKXh1YvF">(<a href="#ref-1AKXh1YvF" role="doc-biblioref">1443</a>)</span> and an alphavirus (in fish) <span class="citation" data-cites="12LJUhdVJ">(<a href="#ref-12LJUhdVJ" role="doc-biblioref">1444</a>)</span>.</p>
<h4 data-number="8.8.1.2" id="applications-to-covid-19"><span class="header-section-number">8.8.1.2</span> Applications to COVID-19</h4>
<p>Several plasmid-vectored DNA vaccines have been developed against COVID-19 (Table <a href="#tbl:approved-DNA">4</a>).
In fact, the ZyCoV-D vaccines developed by India’s Zydus Cadila is the first plasmid-vectored DNA vaccine to receive approval or to be used in human medicine <span class="citation" data-cites="AfMvzFuk eIn1Qf3N 11WPX816v">(<a href="#ref-AfMvzFuk" role="doc-biblioref">1445</a>–<a href="#ref-11WPX816v" role="doc-biblioref">1447</a>)</span>.
Another plasmid-vectored DNA vaccine, INO-4800 <span class="citation" data-cites="xuzLfS0y">(<a href="#ref-xuzLfS0y" role="doc-biblioref">1448</a>)</span>, was developed by Inovio Pharmaceuticals Technology that uses electroporation as an adjuvant.
Electroporation was developed as a solution to the issue of limited immunogenicity by increasing the permeability of cell membranes by delivering electrical pulses <span class="citation" data-cites="1Hsm2J1sc">(<a href="#ref-1Hsm2J1sc" role="doc-biblioref">1449</a>)</span>.
It has been shown that electroporation can enhance vaccine efficacy by up to 100-fold, as measured by increases in antigen-specific antibody titers <span class="citation" data-cites="H6tWVs5R">(<a href="#ref-H6tWVs5R" role="doc-biblioref">1450</a>)</span>.
The temporary formation of pores through electroporation facilitates the successful transportation of macromolecules into cells, allowing cells to robustly take up INO-4800 for the production of an antibody response.
For INO-4800, a plasmid-vectored vaccine is delivered through intradermal injection which is then followed by electroporation with a device known as CELLECTRA® <span class="citation" data-cites="4xraQp8j">(<a href="#ref-4xraQp8j" role="doc-biblioref">1451</a>)</span>.
The safety of the CELLECTRA® device has been studied for over seven years, and these studies support the further development of electroporation as a safe vaccine delivery method <span class="citation" data-cites="1Hsm2J1sc">(<a href="#ref-1Hsm2J1sc" role="doc-biblioref">1449</a>)</span>.</p>
<p>These vaccines therefore represent implementations of a new platform technology.
In particular, they offer the advantage of a temperature-stable vaccine, facilitating worldwide administration <span class="citation" data-cites="OYnqjMlC">(<a href="#ref-OYnqjMlC" role="doc-biblioref">1452</a>)</span>.
Although an exciting development in DNA vaccines, the cost of vaccine manufacturing and electroporation may make scaling the use of this technology for prophylactic use for the general public difficult.</p>
<h4 data-number="8.8.1.3" id="trial-safety-and-immunogenicity"><span class="header-section-number">8.8.1.3</span> Trial Safety and Immunogenicity</h4>
<p>The INO-4800 trials began with a phase I trial evaluating two different doses administered as a two-dose series <span class="citation" data-cites="4xraQp8j">(<a href="#ref-4xraQp8j" role="doc-biblioref">1451</a>)</span>.
This trial found the vaccine to be safe, with only six adverse events (AEs) reported by 39 participants, all grade 1, and effective, with all but three participants of 38 developing serum IgG binding titers to the SARS-CoV-2 S protein <span class="citation" data-cites="4xraQp8j">(<a href="#ref-4xraQp8j" role="doc-biblioref">1451</a>)</span>.
In the phase II trial of 401 adults at high risk of exposure to SARS-CoV-2 similarly supported the safety and efficacy of INO-4800.
Only one treatment-related AE was observed and the vaccine was found to be associated with a significant increase in neutralizing activity <span class="citation" data-cites="OYnqjMlC">(<a href="#ref-OYnqjMlC" role="doc-biblioref">1452</a>)</span>.
Results of phase III trials are not yet available <span class="citation" data-cites="pxEE3VEQ R79Wr1hU yraL6YQa Wk6spoae">(<a href="#ref-pxEE3VEQ" role="doc-biblioref">1453</a>–<a href="#ref-Wk6spoae" role="doc-biblioref">1456</a>)</span>.</p>
<p>Trials of ZyCoV-D have progressed further.
This vaccine uses a plasmid to deliver the expression-competent Spike protein and IgE signal peptides to the vaccinee <span class="citation" data-cites="1CCmltPec">(<a href="#ref-1CCmltPec" role="doc-biblioref">1457</a>)</span>.
During the phase I trial, vaccination with a needle versus a needle-free injection system was evaluated, and the vaccine can now be administered without a needle <span class="citation" data-cites="AfMvzFuk eIn1Qf3N">(<a href="#ref-AfMvzFuk" role="doc-biblioref">1445</a>, <a href="#ref-eIn1Qf3N" role="doc-biblioref">1446</a>)</span>.
A phase III trial enrolling over 27,000 patients found no difference in AEs between the placebo and treatment groups and estimated the efficacy of ZyCoV-D to be 66.6% <span class="citation" data-cites="3xlXzOoW">(<a href="#ref-3xlXzOoW" role="doc-biblioref">1458</a>)</span>.
It was authorized for people ages 12 and older <span class="citation" data-cites="11WPX816v">(<a href="#ref-11WPX816v" role="doc-biblioref">1447</a>)</span>
The highly portable design offers advantages over traditional vaccines <span class="citation" data-cites="1CCmltPec">(<a href="#ref-1CCmltPec" role="doc-biblioref">1457</a>)</span>, especially as the emergence of variants continues to challenge the effectiveness of vaccines.
As of August 2022, ZyCoV-D has only been approved in India <span class="citation" data-cites="15gaai8Xh">(<a href="#ref-15gaai8Xh" role="doc-biblioref">1459</a>)</span> and is not tracked by Our World in Data <span class="citation" data-cites="sRy6js2o">(<a href="#ref-sRy6js2o" role="doc-biblioref">1115</a>)</span>.</p>
<h4 data-number="8.8.1.4" id="real-world-safety-and-effectiveness"><span class="header-section-number">8.8.1.4</span> Real-World Safety and Effectiveness</h4>
<p>In terms of the ability of plasmid-vectored vaccines to neutralize VOC, varying information is available.
The situation for ZyCoV-D is somewhat different, as their phase III trial occurred during the Delta wave in India <span class="citation" data-cites="3xlXzOoW">(<a href="#ref-3xlXzOoW" role="doc-biblioref">1458</a>)</span>.
At present, no major press releases have addressed the vaccine’s ability to neutralize Omicron and related VOC, but reporting suggests that the manufacturers were optimistic about the vaccine in light of the Omicron variant as of late 2021 <span class="citation" data-cites="BTB3S3sT">(<a href="#ref-BTB3S3sT" role="doc-biblioref">1460</a>)</span>.</p>
<p>As for INO-4800, studies have examined whether the induced immune response can neutralize existing VOC.
They assessed neutralization of several VOC relative to the index strain and found no difference in neutralization between the index strain and the Gamma VOC (P.1) <span class="citation" data-cites="mbBuH8XY">(<a href="#ref-mbBuH8XY" role="doc-biblioref">1461</a>)</span>.
However, neutralization of the Alpha and Beta VOC was significantly lower (approximately two and seven times, respectively) <span class="citation" data-cites="mbBuH8XY">(<a href="#ref-mbBuH8XY" role="doc-biblioref">1461</a>)</span>.
These findings are in line with the shifts in effectiveness reported for other vaccines <span class="citation" data-cites="1HWmcSB20">(<a href="#ref-1HWmcSB20" role="doc-biblioref">6</a>)</span>.
In addition to loss of neutralizing activity due to viral evolution, studies have also evaluated the decline in neutralizing antibodies (nAbs) induced by INO-4800 over time.
Levels of nAbs remained statistically significant relative to the pre-vaccination baseline for six months <span class="citation" data-cites="CSlbNoGU">(<a href="#ref-CSlbNoGU" role="doc-biblioref">1462</a>)</span>.
Administration of a booster dose induced a significant increase of titers relative to their pre-booster levels <span class="citation" data-cites="CSlbNoGU">(<a href="#ref-CSlbNoGU" role="doc-biblioref">1462</a>)</span>.
Given the timing of this trial (enrollment between April and July 2020), it is unlikely that participants were exposed to VOC associated with decreased efficacy.</p>
<p>In light of the emergence of VOC against which many vaccines show lower effectiveness, Inovio Pharmaceuticals began to develop a new vaccine with the goal of improving robustness against known and future VOC <span class="citation" data-cites="ysgD4Dcf">(<a href="#ref-ysgD4Dcf" role="doc-biblioref">1463</a>)</span>.
Known as INO-4802, this vaccine was designed to express a pan-Spike immunogen <span class="citation" data-cites="Aynz3sBj">(<a href="#ref-Aynz3sBj" role="doc-biblioref">1464</a>)</span>.
Booster studies in rodents <span class="citation" data-cites="12zreC1Tk">(<a href="#ref-12zreC1Tk" role="doc-biblioref">1465</a>)</span> and non-human primates <span class="citation" data-cites="Aynz3sBj">(<a href="#ref-Aynz3sBj" role="doc-biblioref">1464</a>)</span> suggest that it may be more effective than INO-4800 in providing immunity to VOC such as Delta and Omicron when administered as part of a heterologous boost regimen, although boosting with INO-4800 was also very effective in increasing immunity in rhesus macaques <span class="citation" data-cites="Aynz3sBj">(<a href="#ref-Aynz3sBj" role="doc-biblioref">1464</a>)</span>.
Therefore, boosting is likely to be an important strategy for this vaccine, especially as the virus continues to evolve.</p>
<h3 data-number="8.8.2" id="viral-vectored-dna-vaccines"><span class="header-section-number">8.8.2</span> Viral-Vectored DNA Vaccines</h3>
<p>Plasmids are not the only vector that can be used to deliver sequences associated with viral antigens.
Genetic material from the target virus can also be delivered using a second virus as a vector.
Viral vectors have emerged as a safe and efficient method to furnish the nucleotide sequences of an antigen to the immune system <span class="citation" data-cites="1Ff2BDzkT">(<a href="#ref-1Ff2BDzkT" role="doc-biblioref">1466</a>)</span>.
The genetic content of the vector virus is often altered to prevent it from replicating, but replication-competent viruses can also be used under certain circumstances <span class="citation" data-cites="1FpZkxdl4">(<a href="#ref-1FpZkxdl4" role="doc-biblioref">1467</a>)</span>.
Once the plasmid or viral vector brings the DNA sequence to an APC, the host machinery can be used to construct antigen(s) from the transported genetic material, and the host can then synthesize antibodies in response <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.</p>
<p>One of the early viral vectors explored was adenovirus, with serotype 5 (Ad5) being particularly effective <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.
This technology rose in popularity during the 2000s due to its being more immunogenic in humans and non-human primates than plasmid-vectored DNA vaccines <span class="citation" data-cites="BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>)</span>.
In the 2000s, interest also arose in utilizing simian adenoviruses as vectors because of the reduced risk that human vaccine recipients would have prior exposure resulting in adaptive immunity <span class="citation" data-cites="BsrTDzJ2 XRmk1S6R">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>, <a href="#ref-XRmk1S6R" role="doc-biblioref">1468</a>)</span>, and chimpanzee adenoviruses were explored as a potential vector in the development of a vaccine against MERS-CoV <span class="citation" data-cites="Jkm7jfS8">(<a href="#ref-Jkm7jfS8" role="doc-biblioref">1469</a>)</span>.</p>
<p>Today, various viral-vector platforms including poxviruses <span class="citation" data-cites="8bpbvIro 1AZfAQ5py">(<a href="#ref-8bpbvIro" role="doc-biblioref">1470</a>, <a href="#ref-1AZfAQ5py" role="doc-biblioref">1471</a>)</span>, adenoviruses <span class="citation" data-cites="zX5UKhti">(<a href="#ref-zX5UKhti" role="doc-biblioref">1472</a>)</span>, and vesicular stomatitis viruses <span class="citation" data-cites="SNwg8Qkf lvi4DH2g">(<a href="#ref-SNwg8Qkf" role="doc-biblioref">1473</a>, <a href="#ref-lvi4DH2g" role="doc-biblioref">1474</a>)</span> are being developed,
Viral-vector vaccines are able to induce both an antibody and cellular response; however, the response is limited due to the immunogenicity of the viral vector used <span class="citation" data-cites="YRgRziXN zX5UKhti">(<a href="#ref-zX5UKhti" role="doc-biblioref">1472</a>, <a href="#ref-YRgRziXN" role="doc-biblioref">1475</a>)</span>.
An important consideration in identifying potential vectors is the immune response to the vector.
Both the innate and adaptive immune responses can potentially respond to the vector, limiting the ability of the vaccine to transfer information to the immune system <span class="citation" data-cites="tbs2wD7F">(<a href="#ref-tbs2wD7F" role="doc-biblioref">1476</a>)</span>.
Different vectors are associated with different levels of reactogenicity; for example, adenoviruses elicit a much stronger innate immune response than replication deficient adeno-associated viruses derived from parvoviruses <span class="citation" data-cites="tbs2wD7F">(<a href="#ref-tbs2wD7F" role="doc-biblioref">1476</a>)</span>.
Additionally, using a virus circulating widely in human populations as a vector presents additional challenges because vaccine recipients may already have developed an immune response to the vector <span class="citation" data-cites="IUplTKEg">(<a href="#ref-IUplTKEg" role="doc-biblioref">1477</a>)</span>.
Furthermore, repeated exposure to adenoviruses via viral-vectored DNA vaccines may increase reactivity to these vectors over time, presenting a challenge that will need to be considered in long-term development of these vaccines <span class="citation" data-cites="BJRcCUIU gDDhHhRd">(<a href="#ref-BJRcCUIU" role="doc-biblioref">1478</a>, <a href="#ref-gDDhHhRd" role="doc-biblioref">1479</a>)</span>.</p>
<h4 data-number="8.8.2.1" id="prior-applications-1"><span class="header-section-number">8.8.2.1</span> Prior Applications</h4>
<p>There are several viral vector vaccines that are available for veterinary use <span class="citation" data-cites="MvKb0qJC BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>, <a href="#ref-MvKb0qJC" role="doc-biblioref">1480</a>)</span>, but prior to the COVID-19 pandemic, only one viral vector vaccine was approved by the United States’ Food and Drug Administration (FDA) for use in humans.
This vaccine is vectored with a recombinant vesicular stomatitis virus and targeted against the Ebola virus <span class="citation" data-cites="9g5tmszW">(<a href="#ref-9g5tmszW" role="doc-biblioref">1481</a>)</span>.
Additionally, several phase I and phase II clinical trials for other vaccines are ongoing <span class="citation" data-cites="1Ff2BDzkT">(<a href="#ref-1Ff2BDzkT" role="doc-biblioref">1466</a>)</span>, and the technology is currently being explored for its potential against numerous infectious diseases including malaria <span class="citation" data-cites="OZJWUaDW 3tkGuMXx">(<a href="#ref-OZJWUaDW" role="doc-biblioref">1482</a>, <a href="#ref-3tkGuMXx" role="doc-biblioref">1483</a>)</span>, Ebola <span class="citation" data-cites="AgZwwt5u 9BEMTYn8 PbGQOOI">(<a href="#ref-AgZwwt5u" role="doc-biblioref">1484</a>–<a href="#ref-PbGQOOI" role="doc-biblioref">1486</a>)</span>, and HIV <span class="citation" data-cites="1C8hgfvDF SAIfGNkZ">(<a href="#ref-1C8hgfvDF" role="doc-biblioref">1487</a>, <a href="#ref-SAIfGNkZ" role="doc-biblioref">1488</a>)</span>.</p>
<p>The threat of MERS and SARS initiated interest in the application of viral vector vaccines to human coronaviruses <span class="citation" data-cites="Jkm7jfS8">(<a href="#ref-Jkm7jfS8" role="doc-biblioref">1469</a>)</span>, but efforts to apply this technology to these pathogens had not yet led to a successful vaccine candidate.
In the mid-to-late 2000s, adenoviral vectored vaccines against SARS were found to induce SARS-CoV-specific IgA in the lungs of mice <span class="citation" data-cites="umEOWDY5">(<a href="#ref-umEOWDY5" role="doc-biblioref">1489</a>)</span> but were later found to offer incomplete protection in ferret models <span class="citation" data-cites="DGTFML2b">(<a href="#ref-DGTFML2b" role="doc-biblioref">1490</a>)</span>.
Gamaleya National Center of Epidemiology and Microbiology in Moscow sought to use an adenovirus platform for the development of vaccines for MERS-CoV and Ebola virus, although neither of the previous vaccines were internationally licensed <span class="citation" data-cites="UCI0TCHy">(<a href="#ref-UCI0TCHy" role="doc-biblioref">1491</a>)</span>.</p>
<p>In 2017, results were published from an initial investigation of two vaccine candidates against MERS-CoV containing the MERS-CoV <em>S</em> gene vectored with chimpanzee adenovirus, Oxford University #1 (ChAdOx1), a replication-deficient chimpanzee adenovirus <span class="citation" data-cites="P94sxWp4">(<a href="#ref-P94sxWp4" role="doc-biblioref">1492</a>)</span>.
This study reported that a candidate containing the complete S protein sequence induced a stronger neutralizing antibody response in mice than candidates vectored with modified vaccinia virus Ankara.</p>
<p>The candidate was pursued in additional research, and in the summer of 2020 results of two studies were published.
The first reported that a single dose of ChAdOx1 MERS induced an immune response and inhibited viral replication in macaques <span class="citation" data-cites="3NtMBDMM">(<a href="#ref-3NtMBDMM" role="doc-biblioref">1493</a>)</span>.
The second reported promising results from a phase I trial that administered the vaccine to adults and measured safety, tolerability, and immune response <span class="citation" data-cites="ERfSJf5B">(<a href="#ref-ERfSJf5B" role="doc-biblioref">1494</a>)</span>.</p>
<h4 data-number="8.8.2.2" id="application-to-covid-19-3"><span class="header-section-number">8.8.2.2</span> Application to COVID-19</h4>
<div id="fig:nrvv-distrib" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/dfc6ff85488d57cbb52f4b2324531be6afaa73f4/owiddata/maps/non_replicating_viral_vector.png" alt="Figure 11: Worldwide availability of vaccines developed using non-replicating viral vectors. This figure reflects the number of vaccines using non-replicating viral vectors that were available in each country as of December 18, 2022. These data are retrieved from Our World in Data (1115) and plotted using geopandas (1161). See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily. Note that this figure draws from a different data source than Table 4 and does not necessarily include data for every vaccine developed within this category." />
<figcaption aria-hidden="true"><span>Figure 11:</span> <strong>Worldwide availability of vaccines developed using non-replicating viral vectors.</strong><!--As a general comment on the choropleths, is there any way to increase resolution. Hard to see what’s happening in Europe and south east Asia.--><!--Updated from 300 to 600 after this comment in #1169, let's see if it helped-->
This figure reflects the number of vaccines using non-replicating viral vectors that were available in each country as of December 18, 2022.
These data are retrieved from Our World in Data <span class="citation" data-cites="sRy6js2o">(<a href="#ref-sRy6js2o" role="doc-biblioref">1115</a>)</span> and plotted using geopandas <span class="citation" data-cites="iGEyNO42">(<a href="#ref-iGEyNO42" role="doc-biblioref">1161</a>)</span>.
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.
Note that this figure draws from a different data source than Table <a href="#tbl:approved-DNA">4</a> and does not necessarily include data for every vaccine developed within this category.</figcaption>
</figure>
</div>
<p>While not all of the above results were available at the time that vaccine development programs against SARS-CoV-2 began, at least three viral vector vaccines have also been developed against SARS-CoV-2 (Figure <a href="#fig:nrvv-distrib">11</a>).
First, a collaboration between AstraZeneca and researchers at the University of Oxford successfully applied a viral vector approach to the development of a vaccine against SARS-CoV-2 using the replication-deficient ChAdOx1 vector modified to encode the S protein of SARS-CoV-2 <span class="citation" data-cites="1037p4Gvs">(<a href="#ref-1037p4Gvs" role="doc-biblioref">1495</a>)</span>.
In a phase I trial, the immunogenic potential of vaccine candidate ChAdOx1 nCoV-19 was demonstrated through the immune challenge of two animal models, mice and rhesus macaques <span class="citation" data-cites="1037p4Gvs">(<a href="#ref-1037p4Gvs" role="doc-biblioref">1495</a>)</span>.
In a phase I/II trial, patients receiving the ChAdOx1 nCoV-19 vaccine developed antibodies to the SARS-CoV-2 Spike protein that peaked by day 28, with these levels remaining stable until a second observation at day 56 <span class="citation" data-cites="2bBVSpM">(<a href="#ref-2bBVSpM" role="doc-biblioref">1496</a>)</span>.</p>
<p>Second, a viral vector approach was applied by Russia’s Gamaleya Research Institute of Epidemiology and Microbiology to develop Sputnik V, a replication-deficient recombinant adenovirus (rAd) vaccine that combines two adenovirus vectors, rAd26-S and rAd5-S, that express the full-length SARS-CoV-2 Spike glycoprotein.
These vectors are intramuscularly administered individually using two separate vaccines in a prime-boost regimen.
The rAd26-S is administered first, followed by rAd5-S 21 days later.
Both vaccines deliver 10<sup>11</sup> viral particles per dose.
This approach is designed to overcome any potential pre-existing immunity to adenovirus in the population <span class="citation" data-cites="sRAZYY9C">(<a href="#ref-sRAZYY9C" role="doc-biblioref">1497</a>)</span>, as some individuals may possess immunity to Ad5 <span class="citation" data-cites="8jwp261S">(<a href="#ref-8jwp261S" role="doc-biblioref">1498</a>)</span>.
Sputnik V is the only recombinant adenovirus vaccine to utilize two vectors.</p>
<p>Third, Janssen Pharmaceuticals, Inc., a subsidiary of Johnson &amp; Johnson, developed a viral vector vaccine in collaboration with and funded by the United States’ “Operation Warp Speed” <span class="citation" data-cites="D3Px25HN 57BTbcko">(<a href="#ref-D3Px25HN" role="doc-biblioref">1499</a>, <a href="#ref-57BTbcko" role="doc-biblioref">1500</a>)</span>.
The vaccine candidate JNJ-78436735, formerly known as Ad26.COV2-S, is a monovalent vaccine that is composed of a replication-deficient adenovirus serotype 26 (Ad26) vector expressing the stabilized prefusion S protein of SARS-CoV-2 <span class="citation" data-cites="pWf2T8J8 10UC562ga">(<a href="#ref-10UC562ga" role="doc-biblioref">1424</a>, <a href="#ref-pWf2T8J8" role="doc-biblioref">1501</a>)</span>.
Unlike the other two viral vector vaccines available to date, JNJ-78436735 requires only a single dose, a characteristic that was expected to aid in global deployment <span class="citation" data-cites="gOOBv1MD">(<a href="#ref-gOOBv1MD" role="doc-biblioref">1502</a>)</span>.
JNJ-78436735 was selected from among a number of initial candidate designs <span class="citation" data-cites="10UC562ga">(<a href="#ref-10UC562ga" role="doc-biblioref">1424</a>)</span> and tested <em>in vivo</em> in Syrian golden hamsters and Rhesus macaques to assess safety and immunogenicity <span class="citation" data-cites="10UC562ga gOOBv1MD HmMIiIv2 EpOXYGt4">(<a href="#ref-10UC562ga" role="doc-biblioref">1424</a>, <a href="#ref-gOOBv1MD" role="doc-biblioref">1502</a>–<a href="#ref-EpOXYGt4" role="doc-biblioref">1504</a>)</span>.
The JNJ-78436735 candidate was selected for its favorable immunogenicity profile and ease of manufacturability <span class="citation" data-cites="10UC562ga gOOBv1MD HmMIiIv2 EpOXYGt4">(<a href="#ref-10UC562ga" role="doc-biblioref">1424</a>, <a href="#ref-gOOBv1MD" role="doc-biblioref">1502</a>–<a href="#ref-EpOXYGt4" role="doc-biblioref">1504</a>)</span> and was found to confer protection against SARS-CoV-2 in macaques even after six months <span class="citation" data-cites="HGVDPMLm">(<a href="#ref-HGVDPMLm" role="doc-biblioref">1505</a>)</span>.
The one- versus two-dose regimen was then tested in volunteers through a phase I/IIa trial <span class="citation" data-cites="pWf2T8J8 69GoEX0X">(<a href="#ref-pWf2T8J8" role="doc-biblioref">1501</a>, <a href="#ref-69GoEX0X" role="doc-biblioref">1506</a>)</span>.
A major difference between this vaccine and the other two in this category is that the S protein immunogen is stabilized in its prefusion conformation, while in the Sputnik V and AstraZeneca vaccines it is not.</p>
<p>As of December 18, 2022, data describing the distribution of 5 viral-vectored vaccines in 202 countries are available (Figure <a href="#fig:nrvv-distrib">11</a>).
ChAdOx1 nCoV-19 was first approved for emergency use on December 30, 2020 in the U.K. <span class="citation" data-cites="1A7PjhDDR">(<a href="#ref-1A7PjhDDR" role="doc-biblioref">1507</a>)</span>.
Sputnik V was available soon after, and early as January 2021, Sputnik V had been administered to 1.5 million Russians <span class="citation" data-cites="X5LkVfY6">(<a href="#ref-X5LkVfY6" role="doc-biblioref">1508</a>)</span> and began distributing doses to other countries within Europe such as Belarus, Bosnia-Herzegovina, Hungary, San Marino, Serbia, and Slovakia <span class="citation" data-cites="16LczMwFO Z0V7NK7Y 16GYKbrOq">(<a href="#ref-16LczMwFO" role="doc-biblioref">1509</a>–<a href="#ref-16GYKbrOq" role="doc-biblioref">1511</a>)</span>.</p>
<h4 data-number="8.8.2.3" id="trial-estimates-of-safety-and-efficacy"><span class="header-section-number">8.8.2.3</span> Trial Estimates of Safety and Efficacy</h4>
<p>The first DNA viral-vectored vaccine for which efficacy estimates became available was AstraZeneca’s ChAdOx1 nCoV-19.
In December 2020, preliminary results of the phase III trial were released detailing randomized control trials conducted in the United Kingdom (U.K.), Brazil, and South Africa between April and November 2020 <span class="citation" data-cites="Vnbw9o3T">(<a href="#ref-Vnbw9o3T" role="doc-biblioref">1408</a>)</span>.
These trials compared ChAdOx1 nCoV-19 to a control, but the design of each study varied; pooling data across studies indicated an overall efficacy of 70.4%.
For Sputnik V, the phase III trial indicated an overall vaccine efficacy of 91.6% for symptomatic COVID-19 <span class="citation" data-cites="gLAIyAHm">(<a href="#ref-gLAIyAHm" role="doc-biblioref">1512</a>)</span>.
As for Janssen, the vaccine was well-tolerated, and across all regions studied, it was found to be 66.9% effective after 28 days for the prevention of moderate to severe COVID-19 and to be 81.7% effective for the prevention of laboratory-confirmed severe COVID-19 <span class="citation" data-cites="GOZYHZz0">(<a href="#ref-GOZYHZz0" role="doc-biblioref">1513</a>)</span>.
There were no COVID-19-associated deaths in the vaccine group.
However, the emergence of the Beta variant in the South African trial population was associated with a slightly reduced efficacy (64% two weeks after receipt), and all of the COVID-19-associated deaths in the trial occurred in the South African placebo cohort <span class="citation" data-cites="GOZYHZz0">(<a href="#ref-GOZYHZz0" role="doc-biblioref">1513</a>)</span>.
In February 2021, the FDA issued an EUA for the Janssen vaccine based on interim results from the phase III trial <span class="citation" data-cites="iWMHpTBJ 1FcpboRMm">(<a href="#ref-iWMHpTBJ" role="doc-biblioref">1514</a>, <a href="#ref-1FcpboRMm" role="doc-biblioref">1515</a>)</span>.</p>
<p>Two of the three vaccines have faced a number of criticisms surrounding the implementation of their clinical trials.
In the race to develop vaccines against SARS-CoV-2, President Vladimir Putin of Russia announced the approval of the Sputnik V vaccines on August 11, 2020 in the absence of clinical evidence <span class="citation" data-cites="3KMxmQhV">(<a href="#ref-3KMxmQhV" role="doc-biblioref">1516</a>)</span>.
A press release on November 11, 2020 indicated positive results from an interim analysis of the phase III Sputnik V trials, which reported 92% efficacy in 16,000 volunteers <span class="citation" data-cites="JSzDvnk6">(<a href="#ref-JSzDvnk6" role="doc-biblioref">1517</a>)</span>.
However, this release came only two days after both Pfizer and BioNTech reported that their vaccines had an efficacy over 90%, which led to significant skepticism of the Russian findings for a myriad of reasons including the lack of a published protocol and the “reckless” approval of the vaccine in Russia months prior to the publication of the interim results of the phase III trial <span class="citation" data-cites="JSzDvnk6 Yzz3rwqk">(<a href="#ref-JSzDvnk6" role="doc-biblioref">1517</a>, <a href="#ref-Yzz3rwqk" role="doc-biblioref">1518</a>)</span>.
Consequently, many international scientific agencies and public health bodies expressed concern that due diligence to the clinical trial process was subverted for the sake of expediency, leading many to question the safety and efficacy of Sputnik V <span class="citation" data-cites="15DiM98Ae x4aIj5Fr 3KMxmQhV">(<a href="#ref-3KMxmQhV" role="doc-biblioref">1516</a>, <a href="#ref-15DiM98Ae" role="doc-biblioref">1519</a>, <a href="#ref-x4aIj5Fr" role="doc-biblioref">1520</a>)</span>.
Despite regulatory, safety, and efficacy concerns, pre-orders for 1 billion doses of the Sputnik V were reported within days of the vaccine’s approval in Russia <span class="citation" data-cites="3KMxmQhV">(<a href="#ref-3KMxmQhV" role="doc-biblioref">1516</a>)</span>.
Almost a month later, the phase I/II trial data was published <span class="citation" data-cites="PNZEiId1">(<a href="#ref-PNZEiId1" role="doc-biblioref">1521</a>)</span>
It wasn’t until February 2021, six months after its approval in Russia, that interim results of the phase III trial were released <span class="citation" data-cites="gLAIyAHm">(<a href="#ref-gLAIyAHm" role="doc-biblioref">1512</a>)</span>.
This publication reported a VE of 91% and a low rate of serious AEs, although there were several serious AEs that were determined not to be associated with the vaccine by an independent data monitoring committee about which little other information was released <span class="citation" data-cites="hSWPgHOv">(<a href="#ref-hSWPgHOv" role="doc-biblioref">1522</a>)</span>.</p>
<p>AstraZeneca’s clinical trial also faced criticism.
The trial was paused in September 2020 following a severe adverse event in one participant <span class="citation" data-cites="vwhmuwto">(<a href="#ref-vwhmuwto" role="doc-biblioref">1523</a>)</span>.
It was restarted soon after <span class="citation" data-cites="Fz6kXAHy">(<a href="#ref-Fz6kXAHy" role="doc-biblioref">1524</a>)</span>, but it seems that the recent pause was not mentioned to the FDA during a call the morning before the story broke <span class="citation" data-cites="Kt4zFpsF">(<a href="#ref-Kt4zFpsF" role="doc-biblioref">1525</a>)</span>.
Additionally, individual sites within the trial employed somewhat different designs but were combined for analysis.
For example, in South Africa, the trial was double-blinded, whereas in the U.K. and Brazil it was single-blinded, and one of the two trials carried out in the U.K. evaluated two dosing regimens (low dose or standard dose, both followed by standard dose).
Some of the trials used a meningococcal conjugate vaccine (MenACWY) as a control, while others used saline.
Data was pooled across countries for analysis, a design decision that was approved by regulators but raised some questions when higher efficacy was reported in a subgroup of patients who received a low-dose followed by a standard dose.
This group came about because some participants in the U.K. were erroneously primed with a much lower dose, which turned out to have higher efficacy than the intended dose <span class="citation" data-cites="4mDUvRId">(<a href="#ref-4mDUvRId" role="doc-biblioref">1526</a>)</span>.
Combining the data then led to confusion surrounding the VE, as VE varied widely among conditions (e.g., 62% VE in the standard dose group vs 90% in the group that received a low prime dose <span class="citation" data-cites="Vnbw9o3T">(<a href="#ref-Vnbw9o3T" role="doc-biblioref">1408</a>)</span>).
Subsequent research, however, suggests that reducing the prime dose may, in fact, elicit a superior immune response in the long-term despite a lower initial response <span class="citation" data-cites="SUGCFKUo">(<a href="#ref-SUGCFKUo" role="doc-biblioref">1527</a>)</span>.
Therefore, this error may serendipitously improve efficacy of vaccine-vectored vaccines broadly.</p>
<h4 data-number="8.8.2.4" id="real-world-safety-and-efficacy-2"><span class="header-section-number">8.8.2.4</span> Real-World Safety and Efficacy</h4>
<p>Following the trials, additional concerns have been raised about some of these vaccines.
Within a few days to a few weeks following their first dose of the AstraZeneca vaccine, three women developed extensive venous sinus thrombosis <span class="citation" data-cites="5gqQwOi6">(<a href="#ref-5gqQwOi6" role="doc-biblioref">1528</a>)</span>.
In March 2021, administration of the vaccine was paused in several European countries while a possible link to thrombotic events was investigated <span class="citation" data-cites="Q4qW6ARY">(<a href="#ref-Q4qW6ARY" role="doc-biblioref">1529</a>)</span>, as these adverse events had not been observed in clinical trials, but the European Medicine Agency (EMA) soon determined that 25 events were not related to the vaccine <span class="citation" data-cites="xSBGseR">(<a href="#ref-xSBGseR" role="doc-biblioref">1530</a>)</span>.
The following month, the United States paused administration of the Janssen vaccine for ten days due to 15 similar AEs <span class="citation" data-cites="TtPVfMOL 1F1ma0vMT">(<a href="#ref-TtPVfMOL" role="doc-biblioref">1531</a>, <a href="#ref-1F1ma0vMT" role="doc-biblioref">1532</a>)</span>, but the EMA, U.S. Centers for Disease Control, and the FDA’s Advisory Committee on Immunization Practices again identified the events as being very rare and the benefits of the vaccine as likely to outweigh its risks <span class="citation" data-cites="iRvYqPq6 MnBdD0Mr 2uAR3HBq EqBIFx5T">(<a href="#ref-iRvYqPq6" role="doc-biblioref">1533</a>–<a href="#ref-EqBIFx5T" role="doc-biblioref">1536</a>)</span>.
In Denmark and Norway, population-based estimates suggested AstraZeneca’s vaccine increased incidence of venous thromboembolic events by 11 cases over baseline per 100,000 doses <span class="citation" data-cites="tTpW7jvQ">(<a href="#ref-tTpW7jvQ" role="doc-biblioref">1537</a>)</span>.
Estimates of the incidence in other western countries have also been low <span class="citation" data-cites="rWaWz2Yg">(<a href="#ref-rWaWz2Yg" role="doc-biblioref">1538</a>)</span>.
In the US, thromboembolic events following the Janssen vaccine have also been very rare <span class="citation" data-cites="MnBdD0Mr">(<a href="#ref-MnBdD0Mr" role="doc-biblioref">1534</a>)</span>.
Subsequently, a potential mechanism was identified: the adenovirus vector binding to platelet factor 4 <span class="citation" data-cites="z5vyJuGl XwlVcLVP">(<a href="#ref-z5vyJuGl" role="doc-biblioref">1539</a>, <a href="#ref-XwlVcLVP" role="doc-biblioref">1540</a>)</span>.
Because this adverse event is so rare, the risk is likely still outweighed by the risks associated with contracting COVID-19 <span class="citation" data-cites="KUnUe1UK">(<a href="#ref-KUnUe1UK" role="doc-biblioref">1541</a>)</span>, which is also associated with thrombotic events) <span class="citation" data-cites="fcm6ElC0 1F1ma0vMT">(<a href="#ref-1F1ma0vMT" role="doc-biblioref">1532</a>, <a href="#ref-fcm6ElC0" role="doc-biblioref">1542</a>)</span>.
Similarly, concerns about Guillain-Barré syndrome arose in connection to the Janssen vaccine, but these events have similarly been determined to be very rare and the benefits to outweigh the risks <span class="citation" data-cites="EqBIFx5T">(<a href="#ref-EqBIFx5T" role="doc-biblioref">1536</a>)</span>.</p>
<p>Given that vaccines from multiple platforms are now widely available, people at increased risk of a specific severe AE may have options to pursue vaccination with a platform that does not carry such risks.
For example, a woman in the U.S. with a history of thromboembolic concerns might feel more comfortable with an mRNA vaccine (described below), where such AEs have not been identified in association with COVID-19 vaccination.
However, within the U.S.A., no clear framework has been established for advising patients on whether a specific vaccine may be preferable for their individual concerns now that vaccines based on three different technologies are widely available (see <span class="citation" data-cites="1HWmcSB20">(<a href="#ref-1HWmcSB20" role="doc-biblioref">6</a>)</span> for information about Novavax, which is a protein subunit vaccine).</p>
<h2 data-number="8.9" id="mrna-vaccines"><span class="header-section-number">8.9</span> mRNA Vaccines</h2>
<p>Building on DNA vaccine technology, RNA vaccines are an even more recent advancement for vaccine development.
Interest in messenger RNA (mRNA) vaccines emerged around 1990 following <em>in vitro</em> and animal model studies that demonstrated that mRNA could be transferred into cells <span class="citation" data-cites="D7ou3S22 2YZ70C2y">(<a href="#ref-D7ou3S22" role="doc-biblioref">1543</a>, <a href="#ref-2YZ70C2y" role="doc-biblioref">1544</a>)</span>.
mRNA contains the minimum information needed to create a protein <span class="citation" data-cites="2YZ70C2y">(<a href="#ref-2YZ70C2y" role="doc-biblioref">1544</a>)</span>.
RNA vaccines are therefore nucleic-acid based modalities that code for viral antigens against which the human body elicits a humoral and cellular immune response.</p>
<p>The strategy behind mRNA vaccines operates one level above the DNA: instead of directly furnishing the gene sequence associated with an antigen to the host, it provides the mRNA transcribed from the DNA sequence.
The mRNA is transcribed <em>in vitro</em> and delivered to cells via lipid nanoparticles (LNP) <span class="citation" data-cites="HCImhzy8">(<a href="#ref-HCImhzy8" role="doc-biblioref">1545</a>)</span>.
It is recognized by ribosomes <em>in vivo</em> and then translated and modified into functional proteins <span class="citation" data-cites="K0Ltu31S">(<a href="#ref-K0Ltu31S" role="doc-biblioref">1124</a>)</span>.
The resulting intracellular viral proteins are displayed on surface MHC proteins, provoking a strong CD8+ T cell response as well as a CD4<sup>+</sup> T cell and B cell-associated antibody responses <span class="citation" data-cites="K0Ltu31S">(<a href="#ref-K0Ltu31S" role="doc-biblioref">1124</a>)</span>.
mRNA is naturally not very stable and can degrade quickly in the extracellular environment or the cytoplasm.
The LNP covering protects the mRNA from enzymatic degradation outside of the cell <span class="citation" data-cites="zNKWlCwE">(<a href="#ref-zNKWlCwE" role="doc-biblioref">1546</a>)</span>.
Codon optimization to prevent secondary structure formation and modifications of the poly-A tail as well as the 5’ untranslated region to promote ribosomal complex binding can increase mRNA expression in cells.
Furthermore, purifying out double-stranded RNA and immature RNA with fast performance liquid chromatography and high performance liquid chromatography technology will improve translation of the mRNA in the cell <span class="citation" data-cites="K0Ltu31S pRoqjur8">(<a href="#ref-K0Ltu31S" role="doc-biblioref">1124</a>, <a href="#ref-pRoqjur8" role="doc-biblioref">1547</a>)</span>.</p>
<p>There are three types of RNA vaccines: non-replicating, <em>in vivo</em> self-replicating, and <em>in vitro</em> dendritic cell non-replicating <span class="citation" data-cites="1EM5nGaYd">(<a href="#ref-1EM5nGaYd" role="doc-biblioref">1548</a>)</span>.
Non-replicating mRNA vaccines consist of a simple open reading frame for the viral antigen flanked by the 5’ UTR and 3’ poly-A tail.
<em>In vivo</em> self-replicating vaccines encode a modified viral genome derived from single-stranded, positive sense RNA alphaviruses <span class="citation" data-cites="K0Ltu31S pRoqjur8">(<a href="#ref-K0Ltu31S" role="doc-biblioref">1124</a>, <a href="#ref-pRoqjur8" role="doc-biblioref">1547</a>)</span>.
The RNA genome encodes the viral antigen along with proteins of the genome replication machinery, including an RNA polymerase.
Structural proteins required for viral assembly are not included in the engineered genome <span class="citation" data-cites="K0Ltu31S">(<a href="#ref-K0Ltu31S" role="doc-biblioref">1124</a>)</span>.
Self-replicating vaccines produce more viral antigens over a longer period of time, thereby evoking a more robust immune response <span class="citation" data-cites="1EM5nGaYd">(<a href="#ref-1EM5nGaYd" role="doc-biblioref">1548</a>)</span>.
Finally, <em>in vitro</em> dendritic cell non-replicating RNA vaccines limit transfection to dendritic cells.
Dendritic cells are potent antigen-presenting immune cells that easily take up mRNA and present fragments of the translated peptide on their MHC proteins, which can then interact with T cell receptors.
Ultimately, primed T follicular helper cells can stimulate germinal center B cells that also present the viral antigen to produce antibodies against the virus <span class="citation" data-cites="3LMMW7F0">(<a href="#ref-3LMMW7F0" role="doc-biblioref">1549</a>)</span>.
These cells are isolated from the patient, then grown and transfected <em>ex vivo</em> <span class="citation" data-cites="ENBWnhAh">(<a href="#ref-ENBWnhAh" role="doc-biblioref">1550</a>)</span>.
They can then be reintroduced to the patient <span class="citation" data-cites="ENBWnhAh">(<a href="#ref-ENBWnhAh" role="doc-biblioref">1550</a>)</span>.</p>
<p>In addition to the benefits of nucleic acid vaccines broadly, mRNA confers specific advantages compared to DNA vaccines and other platforms <span class="citation" data-cites="vh2AH9sg">(<a href="#ref-vh2AH9sg" role="doc-biblioref">1551</a>)</span>.
Some of these advantages fall within the domain of safety.
Unlike DNA vaccines, mRNA technologies are naturally degradable and non-integrating, and they do not need to cross the nuclear membrane in addition to the plasma membrane for their effects to be seen <span class="citation" data-cites="K0Ltu31S">(<a href="#ref-K0Ltu31S" role="doc-biblioref">1124</a>)</span>.
Additionally, the half life can be regulated by the contents of the 5’ and 3’ untranslated regions <span class="citation" data-cites="xUUwXDNv">(<a href="#ref-xUUwXDNv" role="doc-biblioref">1552</a>)</span>.
In comparison to vaccines that use live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, thereby reducing the risk of a detrimental immune response in the host <span class="citation" data-cites="wYZ6qJMu">(<a href="#ref-wYZ6qJMu" role="doc-biblioref">1553</a>)</span>.
Furthermore, mRNA vaccines are easily, affordably, and rapidly scalable, despite the fact that it took time to reach the scale needed to manufacture vaccines at a scale sufficient for the global population <span class="citation" data-cites="vh2AH9sg">(<a href="#ref-vh2AH9sg" role="doc-biblioref">1551</a>)</span>.</p>
<h4 data-number="8.9.0.1" id="prior-applications-2"><span class="header-section-number">8.9.0.1</span> Prior Applications</h4>
<p>Although mRNA vaccines have been developed for therapeutic and prophylactic purposes, none have previously been licensed or commercialized.
Challenges were caused by the instability of mRNA molecules, the design requirements of an efficient delivery system, and the potential for mRNA to elicit either a very strong immune response or to stimulate the immune system in secondary ways <span class="citation" data-cites="17lluDFcc kqerKJKY">(<a href="#ref-kqerKJKY" role="doc-biblioref">1415</a>, <a href="#ref-17lluDFcc" role="doc-biblioref">1554</a>)</span>.
As of the 2010s, mRNA was still considered a promising technology for future advances in vaccine development <span class="citation" data-cites="2YZ70C2y">(<a href="#ref-2YZ70C2y" role="doc-biblioref">1544</a>)</span>, but prior to 2020, no mRNA vaccines had been approved for use in humans, despite significant advances in the development of this technology <span class="citation" data-cites="ENBWnhAh">(<a href="#ref-ENBWnhAh" role="doc-biblioref">1550</a>)</span>.
This approach showed promise in animal models and preliminary clinical trials for several indications, including rabies, coronavirus, influenza, and cytomegalovirus <span class="citation" data-cites="3EUiWZdN">(<a href="#ref-3EUiWZdN" role="doc-biblioref">1555</a>)</span>.
Preclinical data previously identified effective antibody generation against full-length purified influenza hemagglutinin stalk-encoding mRNA in mice, rabbits, and ferrets <span class="citation" data-cites="6wZy2mn8">(<a href="#ref-6wZy2mn8" role="doc-biblioref">1556</a>)</span>.
Similar immunological responses for mRNA vaccines were observed in humans in phase I and II clinical trials operated by the pharmaceutical-development companies Curevac and Moderna for rabies, flu, and zika <span class="citation" data-cites="pRoqjur8">(<a href="#ref-pRoqjur8" role="doc-biblioref">1547</a>)</span>.
Positively charged bilayer LNPs carrying the mRNA attract negatively charged cell membranes, endocytose into the cytoplasm <span class="citation" data-cites="zNKWlCwE">(<a href="#ref-zNKWlCwE" role="doc-biblioref">1546</a>)</span>, and facilitate endosomal escape.
LNPs can be coated with modalities recognized and engulfed by specific cell types, and LNPs that are 150 nm or less effectively enter into lymphatic vessels <span class="citation" data-cites="zNKWlCwE Djz8x39x">(<a href="#ref-zNKWlCwE" role="doc-biblioref">1546</a>, <a href="#ref-Djz8x39x" role="doc-biblioref">1557</a>)</span>.
Therefore, while these technologies elegantly capitalize on decades of research in vaccine development as well as the tools of the genomic revolution, it was largely unknown prior to the SARS-CoV-2 pandemic whether this potential could be realized in a real-world vaccination effort.</p>
<h4 data-number="8.9.0.2" id="application-to-covid-19-4"><span class="header-section-number">8.9.0.2</span> Application to COVID-19</h4>
<div id="tbl:approved-RNA" class="tablenos">
<table id="tbl:approved-RNA">
<caption><span>Table 5:</span> mRNA vaccines approved in at least one country <span class="citation" data-cites="jswAyWIs">(<a href="#ref-jswAyWIs" role="doc-biblioref">1160</a>)</span> as of December 2, 2022.
As a note, this table includes licensing of existing mRNA technology, i.e., TAK-919 is used to describe Takeda’s manufacturing of Moderna’s formulation.
</caption>
<colgroup>
<col style="width: 56%" />
<col style="width: 43%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Vaccine</th>
<th style="text-align: left;">Company</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">GEMCOVAC-19</td>
<td style="text-align: left;">Gennova Biopharmaceuticals Limited</td>
</tr>
<tr class="even">
<td style="text-align: left;">Spikevax</td>
<td style="text-align: left;">Moderna</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Spikevax Bivalent Original/Omicron BA.1</td>
<td style="text-align: left;">Moderna</td>
</tr>
<tr class="even">
<td style="text-align: left;">Spikevax Bivalent Original/Omicron BA.4/BA.5</td>
<td style="text-align: left;">Moderna</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Comirnaty</td>
<td style="text-align: left;">Pfizer/BioNTech</td>
</tr>
<tr class="even">
<td style="text-align: left;">Comirnaty Bivalent Original/Omicron BA.1</td>
<td style="text-align: left;">Pfizer/BioNTech</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Comirnaty Bivalent Original/Omicron BA.4/BA.5</td>
<td style="text-align: left;">Pfizer/BioNTech</td>
</tr>
<tr class="even">
<td style="text-align: left;">TAK-919 (Moderna formulation)</td>
<td style="text-align: left;">Takeda</td>
</tr>
<tr class="odd">
<td style="text-align: left;">AWcorna</td>
<td style="text-align: left;">Walvax</td>
</tr>
</tbody>
</table>
</div>
<p>Given the potential for mRNA technology to be quickly adapted for a new pathogen, it was favored as a potential vaccine against COVID-19, and fortunately, the prior work in mRNA vaccine development paid off, with 9 mRNA vaccines available in at least one country as of December 2, 2022 (Table <a href="#tbl:approved-RNA">5</a>).
In the vaccines developed under this approach, the mRNA coding for a stabilized prefusion Spike protein, which is immunogenic <span class="citation" data-cites="5x25saIz">(<a href="#ref-5x25saIz" role="doc-biblioref">1558</a>)</span>, is furnished to the immune system in order to train its response.</p>
<p>Two vaccine candidates in this category emerged with promising phase III results at the end of 2020.
Both require two doses approximately one month apart.
The first was Pfizer/BioNTech’s BNT162b2, which contains the full prefusion stabilized, membrane-anchored SARS-CoV-2 Spike protein in a vaccine formulation based on modified mRNA (modRNA) technology <span class="citation" data-cites="1CsCQi9wT 10VyxCgQU">(<a href="#ref-1CsCQi9wT" role="doc-biblioref">1559</a>, <a href="#ref-10VyxCgQU" role="doc-biblioref">1560</a>)</span>.
The second mRNA vaccine, mRNA-1273 developed by ModernaTX, is comprised by a conventional LNP-encapsulated RNA encoding a full-length prefusion stabilized S protein for SARS-CoV-2 <span class="citation" data-cites="Biu1CQeQ">(<a href="#ref-Biu1CQeQ" role="doc-biblioref">1561</a>)</span>.
The vaccine candidates developed against SARS-CoV-2 using mRNA vectors utilize similar principles and technologies, although there are slight differences in implementation among candidates such as the formulation of the platform and the specific components of the Spike protein encapsulated (e.g., the full Spike protein vs. the RBD alone) <span class="citation" data-cites="suRY1e0N">(<a href="#ref-suRY1e0N" role="doc-biblioref">1562</a>)</span>.
As of December 18, 2022, 2 mRNA vaccines are available in 169 countries (Figure <a href="#fig:mRNA-distrib">12</a>).</p>
<div id="fig:mRNA-distrib" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/dfc6ff85488d57cbb52f4b2324531be6afaa73f4/owiddata/maps/RNA.png" alt="Figure 12: Worldwide availability of vaccines developed using mRNA. This figure reflects the number of vaccines based on mRNA technology that were available in each country as of December 18, 2022. These data are retrieved from Our World in Data (1115) and plotted using geopandas (1161). See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily. Note that this figure draws from a different data source than Table 5 and does not necessarily include data for every vaccine developed within this category." />
<figcaption aria-hidden="true"><span>Figure 12:</span> <strong>Worldwide availability of vaccines developed using mRNA.</strong>
This figure reflects the number of vaccines based on mRNA technology that were available in each country as of December 18, 2022.
These data are retrieved from Our World in Data <span class="citation" data-cites="sRy6js2o">(<a href="#ref-sRy6js2o" role="doc-biblioref">1115</a>)</span> and plotted using geopandas <span class="citation" data-cites="iGEyNO42">(<a href="#ref-iGEyNO42" role="doc-biblioref">1161</a>)</span>.
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.
Note that this figure draws from a different data source than Table <a href="#tbl:approved-RNA">5</a> and does not necessarily include data for every vaccine developed within this category.</figcaption>
</figure>
</div>
<p>The rapid and simultaneous development of these vaccines was met with some controversy related to intellectual property (IP).
First, the National Institutes of Health (NIH) and Moderna became involved in a patent dispute, after researchers at the NIH argued they were unfairly excluded from some patents filed based on their IP after they generated the stabilized modRNA sequence used in the vaccine <span class="citation" data-cites="1F8hc8XkK">(<a href="#ref-1F8hc8XkK" role="doc-biblioref">1563</a>)</span>.
Ultimately, in late 2021, Moderna backed down on the patent application <span class="citation" data-cites="BrC2lh5x">(<a href="#ref-BrC2lh5x" role="doc-biblioref">1564</a>)</span>.
However, in August 2022, the company filed their own suit against Pfizer/BioNTech over IP related to the modRNA used in the latter’s COVID-19 vaccine <span class="citation" data-cites="BrC2lh5x ZdLFrJY9">(<a href="#ref-BrC2lh5x" role="doc-biblioref">1564</a>, <a href="#ref-ZdLFrJY9" role="doc-biblioref">1565</a>)</span>.
The outcome of this suit remains to be seen.</p>
<h4 data-number="8.9.0.3" id="trial-safety-and-immunogenicity-1"><span class="header-section-number">8.9.0.3</span> Trial Safety and Immunogenicity</h4>
<p>The VEs revealed by the Pfizer/BioNTech and Moderna clinical trials exceeded expectations.
In a phase II/III multinational trial, the Pfizer/BioNTech’s BNT162b2 vaccine was associated with a 95% efficacy against laboratory-confirmed COVID-19 and with mild-to-moderate local and systemic effects but a low risk of serious AEs when the prime-boost doses were administered 21 days apart <span class="citation" data-cites="CWlYjjIV">(<a href="#ref-CWlYjjIV" role="doc-biblioref">516</a>)</span>.
The ModernaTX mRNA-1273 vaccine was the second mRNA vaccine to release phase III results, despite being the first mRNA vaccine to enter phase I clinical trials and publish interim results of their phase III trial a few months later.
Their study reported a 94.5% vaccine efficacy in preventing symptomatic COVID-19 in adults who received the vaccine at 99 sites around the United States <span class="citation" data-cites="ZYxoabEm">(<a href="#ref-ZYxoabEm" role="doc-biblioref">1566</a>)</span>.
Similar to BNT162b2, the mRNA-1273 vaccine was associated with mild-to-moderate AEs but with a low risk of serious AEs <span class="citation" data-cites="ZYxoabEm">(<a href="#ref-ZYxoabEm" role="doc-biblioref">1566</a>)</span>.
In late 2020, both vaccines received approval from the FDA under an emergency use authorization <span class="citation" data-cites="cAaN4Te0 13Ou1UUAd">(<a href="#ref-cAaN4Te0" role="doc-biblioref">1567</a>, <a href="#ref-13Ou1UUAd" role="doc-biblioref">1568</a>)</span>, and these vaccines have been widely distributed, primarily in North America and the European Union <span class="citation" data-cites="wByD9WaX">(<a href="#ref-wByD9WaX" role="doc-biblioref">1171</a>)</span>.
As the first mRNA vaccines to make it to market, these two highly efficacious vaccines demonstrate the power of this emerging technology, which has previously attracted scientific interest because of its potential to be used to treat non-infectious as well as infectious diseases.</p>
<h4 data-number="8.9.0.4" id="real-world-safety-and-effectiveness-1"><span class="header-section-number">8.9.0.4</span> Real-World Safety and Effectiveness</h4>
<p>As vaccines were rolled out, one study sought to monitor their effectiveness in a real-world setting.
Between December 2020 and April 2021, this prospective cohort study obtained weekly nasal swabs from 3,975 individuals at high risk of SARS-CoV-2 exposure (health care workers, frontline workers, etc.) within the United States <span class="citation" data-cites="D2ZCK63Y">(<a href="#ref-D2ZCK63Y" role="doc-biblioref">332</a>)</span>.
Among these participants, 3,179 (80%) had received at least one dose of an mRNA vaccine, and of those, 2,686 (84%) were fully vaccinated, corresponding to 68% of trial participants overall.
For each vaccinated participant (defined here as having received at least dose 1 more than 7 days ago) whose sample tested positive for SARS-CoV-2, they categorized the viral lineage(s) present in the sample as well as in samples from 3-4 unvaccinated individuals matched by site and testing date.
Overall efficacy of mRNA vaccines was estimated at 91% with full vaccination, similar to the reports from the clinical trials.
The occurrence of fevers was also lower in individuals who were partially or fully vaccinated, and the duration of symptoms was approximately 6 days shorter.
Among the five cases in fully vaccinated and 11 cases in partially vaccinated participants, the rate of infection by VOC was much higher than in the unvaccinated population (30% versus 10%), suggesting that the vaccine was less effective against the VOC than the index strain.</p>
<p>The WHO continues to monitor the emergence of variants and their impact on vaccine efficacy <span class="citation" data-cites="lY0XUlUp">(<a href="#ref-lY0XUlUp" role="doc-biblioref">529</a>)</span>.
In general, mRNA vaccines remain highly effective against severe illness and death, but the effectiveness against infection generally has declined.
A study monitoring infections in a Minnesota cohort from January to July 2021 estimated that the effectiveness of the Moderna vaccine fell to 86% and Pfizer to 76%, although protection against hospitalization remained at 91% and 85%, respectively <span class="citation" data-cites="3Uhu8LzO">(<a href="#ref-3Uhu8LzO" role="doc-biblioref">1569</a>)</span>.
In July of that year, as the Delta variant became dominant in the U.S.A., these estimates all fell, to an effectiveness of 76% for Moderna and 42% for Pfizer and effectiveness against hospitalization of 81% and 75%, respectively <span class="citation" data-cites="3Uhu8LzO">(<a href="#ref-3Uhu8LzO" role="doc-biblioref">1569</a>)</span>.</p>
<p>With the emergence of the Omicron VOC, vaccine effectiveness has likely further declined.
A study in a diverse cohort in Southern California, U.S.A. found the effectiveness of the Moderna vaccine in participants who had received only the primary course to be 44% <span class="citation" data-cites="JszheMvx">(<a href="#ref-JszheMvx" role="doc-biblioref">1570</a>)</span>.
A study in South Africa compared case and hospitalization records from a 4-week period where Omicron was dominant to a 2-month period where Delta was dominant and found that the effectiveness against hospitalization during the Omicron wave was approximately 70% compared to 93% during the Delta wave <span class="citation" data-cites="S6RHdOTJ">(<a href="#ref-S6RHdOTJ" role="doc-biblioref">1571</a>)</span>.
Similarly, a large study in England of 2.5 million individuals suggested that not only the variants circulating, but also the time since vaccination, played a large role in vaccine effectiveness <span class="citation" data-cites="JEucQ5jm">(<a href="#ref-JEucQ5jm" role="doc-biblioref">1572</a>)</span>.
Shortly after the BNT162b2 primary course, effectiveness against the Omicron VOC was as high as 65.5%, but this declined to below 10% by six months after the second dose.
For mRNA-1273, the decline was from 75.1% to 14.9%.
Therefore, it is unsurprising that in spite of vaccination programs, infection rates and hospitalization rates climbed in early 2022 in many Western countries including the United States <span class="citation" data-cites="19qv58Mv3 TkFSco2t">(<a href="#ref-19qv58Mv3" role="doc-biblioref">1573</a>, <a href="#ref-TkFSco2t" role="doc-biblioref">1574</a>)</span>, especially given that many places simultaneously began to loosen public health restrictions designed to reduce viral spread.</p>
<p>On the side of safety, the only major concern that has been raised is a possible link between mRNA vaccination and myocarditis, especially in young men <span class="citation" data-cites="EqBIFx5T">(<a href="#ref-EqBIFx5T" role="doc-biblioref">1536</a>)</span>.
This concern began with case reports of a small number of cases of myocarditis following vaccination in several countries <span class="citation" data-cites="16SmBKzGd SN0p27mm">(<a href="#ref-16SmBKzGd" role="doc-biblioref">1575</a>, <a href="#ref-SN0p27mm" role="doc-biblioref">1576</a>)</span>.
Following these reports, the Israeli Ministry of Health began surveillance to monitor the occurrence of myocarditis <span class="citation" data-cites="hEVGb0Ne">(<a href="#ref-hEVGb0Ne" role="doc-biblioref">1577</a>)</span>.
They identified 283 cases, almost exactly half of which occurred following vaccination with Pfizer’s BNT162b2.
Close analysis of these cases determined that the vaccine did have a significant effect on the incidence of myocarditis; however, the rate of myocarditis remained low overall <span class="citation" data-cites="hEVGb0Ne">(<a href="#ref-hEVGb0Ne" role="doc-biblioref">1577</a>)</span>.
The identification of young men as a population at particular risk of this AE was supported, and the risk was found to be greater after the second dose than the first.
Both this study and a study evaluating data collected from US population-based surveillance identified an increased risk with additional doses <span class="citation" data-cites="13El9IHNw">(<a href="#ref-13El9IHNw" role="doc-biblioref">1578</a>)</span>.
However, most findings suggest that this AE does not have long-term negative effects; a 2021 meta-analysis identified 69 cases, all of which resulted in full recovery <span class="citation" data-cites="bC5oAO24">(<a href="#ref-bC5oAO24" role="doc-biblioref">1579</a>)</span>.
Although these events are very rare, as with the possible thromboembolic AEs associated with viral-vectored DNA vaccines, these findings suggest that it may be prudent to offer a framework for decision making for patients particularly concerned about specific AEs in settings where multiple vaccines are available.</p>
<h2 data-number="8.10" id="booster-doses"><span class="header-section-number">8.10</span> Booster Doses</h2>
<p>Due to waning effectiveness of vaccines over time, especially in light of viral evolution, boosters have emerged as an important strategy in retaining the benefits of vaccination over time.
Booster shots are now recommended in many places, and boosters that account for multiple variants and strains of SARS-CoV-2 are now available in some places <span class="citation" data-cites="180UFKjJ2">(<a href="#ref-180UFKjJ2" role="doc-biblioref">531</a>)</span>.
For example, in the U.S.A., the FDA recently recommended bivalent booster doses designed to account for the Omicron VOC <span class="citation" data-cites="TOyAW98u wQgkg2M4 1FMDHQCgs">(<a href="#ref-TOyAW98u" role="doc-biblioref">1580</a>–<a href="#ref-1FMDHQCgs" role="doc-biblioref">1582</a>)</span>.
In this case, bivalent refers to the fact that doses deliver both the original formulation and an updated vaccine designed for the Omicron subvariants circulating in summer 2022.
The fact that the FDA did not require additional clinical trials from manufacturers for Omicron subvariants BA.4 and BA.5 specifically suggests that the rapid authorization of strain changes in response to emerging VOC may be increasingly attainable <span class="citation" data-cites="Slgwtax5">(<a href="#ref-Slgwtax5" role="doc-biblioref">1583</a>)</span>.
Results suggest that this fourth dose offered at least a short-term increase in VE against Omicron subvariants and also provided additional protection against hospitalization <span class="citation" data-cites="1F0F5pWsm">(<a href="#ref-1F0F5pWsm" role="doc-biblioref">1584</a>)</span>.</p>
<p>Homologous booster doses have been investigated for most vaccines.
For example, over 14,000 adults were administered a booster (second) dose of the Janssen Ad26.COV2.S vaccine <span class="citation" data-cites="Cs2RaaCI">(<a href="#ref-Cs2RaaCI" role="doc-biblioref">1585</a>)</span>.
The booster dose was highly efficacious, with severe COVID-19 and hospitalization prevented almost completely in the vaccinated group.
A booster dose was also found to improve immune response for Sputnik V vaccinees <span class="citation" data-cites="WtUhPusE">(<a href="#ref-WtUhPusE" role="doc-biblioref">1586</a>)</span>.
For the AstraZeneca vaccine, a different approach was taken.
In the interest of distributing first doses as widely as possible, in some places the time between the first and second doses was extended.
One study assessed the immunogenicity and reactogenicity associated with delaying the second dose in the prime-boost series until up to 45 weeks after the first, reporting that an extended inter-dose period was associated with increased antibody titers 28 days after the second dose <span class="citation" data-cites="17s40tqgW">(<a href="#ref-17s40tqgW" role="doc-biblioref">1587</a>)</span>.
This analysis also revealed that a third dose provided an additional boost in neutralizing activity <span class="citation" data-cites="17s40tqgW">(<a href="#ref-17s40tqgW" role="doc-biblioref">1587</a>)</span>.</p>
<p>Third and fourth doses have been introduced for at least some populations in many places in response to the Omicron variant.
An early study in Israeli healthcare workers showed that the additional immunization was safe and immunogenic with antibody titers restored to peak-third dose titers.
No severe illness was reported in the cohort studied (274 versus 426 age-matched controls), and vaccine efficacy against infection was reported at 30% for BNT162b2 and 11% for mRNA-1273 <span class="citation" data-cites="Jv71MaZb">(<a href="#ref-Jv71MaZb" role="doc-biblioref">1588</a>)</span>.
Other studies reported that a third dose of BNT162b2 raised vaccine effectiveness to 67.2% for approximately the first month but that the effectiveness dropped to 45.7% <span class="citation" data-cites="JEucQ5jm">(<a href="#ref-JEucQ5jm" role="doc-biblioref">1572</a>)</span>.
Reduced and even low efficacy against infection does not undermine the value of vaccination, considering the vaccines are intended to prevent severe disease, hospitalization, and death rather than infection generally.
However, these findings do suggest that boosters will likely be needed as the virus continues to evolve.</p>
<p>Many trials have also investigated heterologous boosting approaches.
In particular, the mRNA vaccines are a popular choice for booster doses regardless of primary series.
In general, such approaches have been found to confer favorable immunogenicity relative to homologous boosters (e.g., <span class="citation" data-cites="L0dD93f8 ybtjun8H H6vRjWQg 15rMr9nh7 cNs4cL36 s1xmA2K6 exu6jkTJ">(<a href="#ref-L0dD93f8" role="doc-biblioref">1589</a>–<a href="#ref-exu6jkTJ" role="doc-biblioref">1595</a>)</span> and many other studies).
Due to remaining concerns about rare thromboembolic events, vaccinees who received AstraZeneca for their primary course are advised in some countries to seek a heterologous booster <span class="citation" data-cites="EzS3LTYM">(<a href="#ref-EzS3LTYM" role="doc-biblioref">1596</a>)</span>, although such guidances are not supported by the evidence, which indicates that the first dose of AstraZeneca is most likely to be linked to these rare events <span class="citation" data-cites="kFl0x9VR">(<a href="#ref-kFl0x9VR" role="doc-biblioref">1597</a>)</span>.
In general, heterologous boosting with mRNA vaccines elicits a strong immune response.
For patients who received BNT162b2 as a heterologous booster following a ChAdOx1 primary series, the vaccine effectiveness was estimated to be 62.4% initially, dropping to 39.6% after 10 weeks <span class="citation" data-cites="JEucQ5jm">(<a href="#ref-JEucQ5jm" role="doc-biblioref">1572</a>)</span>.
For a heterologous mRNA-1273 booster, the effectiveness was estimated to be slightly higher (70.1% and 60.9% following ChAdOx1 and 73.9% to 64.4% following BNT162b2) <span class="citation" data-cites="JEucQ5jm">(<a href="#ref-JEucQ5jm" role="doc-biblioref">1572</a>)</span>.
Therefore, subsequent booster doses may remain an ongoing component of strategies to combat SARS-CoV-2.</p>
<p>Although the vaccines developed based on the index strain remain highly effective at preventing severe illness and death, they serve much less utility at preventing illness broadly than they did early in the pandemic.
Therefore, many manufacturers are exploring potential reformulations based on VOC that have emerged since the beginning of the pandemic.
In June 2022, Moderna released data describing the effect of their bivalent mRNA booster, mRNA-1273.214, designed to protect against the Omicron variant <span class="citation" data-cites="OWsXXUTT">(<a href="#ref-OWsXXUTT" role="doc-biblioref">1598</a>)</span>.
A 50 μg dose of mRNA-1273.214 was administered to 437 participants.
One month later, the neutralizing geometric mean titer ratio was assessed against several variants of SARS-CoV-2, including Omicron.
The immune response was higher against all variants assessed, including Omicron, than for boosting with the original formulation (mRNA-1273).
Another formulation, mRNA-1273.211, developed based on the Beta variant, has been associated with durable protection as long as six months after dosing.
The associated publications suggest that this novel formulation offers significant protection against Omicron and other VOC <span class="citation" data-cites="aPBb6pOb 198OSKmOq">(<a href="#ref-aPBb6pOb" role="doc-biblioref">1599</a>, <a href="#ref-198OSKmOq" role="doc-biblioref">1600</a>)</span>.
In August 2022, Pfizer also announced successful development of a new formulation effective against Omicron <span class="citation" data-cites="1D9Fxz7bM">(<a href="#ref-1D9Fxz7bM" role="doc-biblioref">1601</a>)</span>.</p>
<p>Modularity has been proposed as one of the advantages to developing DNA and mRNA vaccines.
This design would allow for faster adaptation to viral evolution.
However, in the arms race against SARS-CoV-2, the vaccines are still lagging behind the virus.
This disadvantage may change as regulators become more familiar with these vaccines and as a critical mass of data is accumulated.
Given the apparent need for boosters, interest has also emerged in whether updated formulations of SARS-CoV-2 vaccines can be administered along with annual flu vaccines to improve immunity to novel variants.</p>
<h2 data-number="8.11" id="conclusions-2"><span class="header-section-number">8.11</span> Conclusions</h2>
<p>COVID-19 has seen the coming-of-age of vaccine technologies that have been in development since the late 20<sup>th</sup> century but had never before been authorized for use.
Vaccines that employ DNA and RNA eliminate all concerns about potential infection due to the vaccine components.
The vaccines described above demonstrate the potential for these technologies to facilitate a quick response to an emerging pathogen.
Additionally, their efficacy in trials far exceeded expectations, especially in the case of RNA vaccines.
These technologies hold significant potential to drive improvements in human health over the coming years.</p>
<p>Traditional vaccine technologies were built on the principle of using either a weakened version of the virus or a fragment of the virus.
COVID-19 has highlighted the fact that in recent years, the field has undergone a paradigm shift towards reverse vaccinology.
Reverse vaccinology emphasizes a discovery-driven approach to vaccine development based on knowledge of the viral genome <span class="citation" data-cites="jU9YFYvB">(<a href="#ref-jU9YFYvB" role="doc-biblioref">1602</a>)</span>.
This strategy was explored during development of a DNA vaccine against the Zika virus <span class="citation" data-cites="u0dESADU">(<a href="#ref-u0dESADU" role="doc-biblioref">1603</a>)</span>.
Though the disease was controlled before the vaccine became available <span class="citation" data-cites="HyYY2agc">(<a href="#ref-HyYY2agc" role="doc-biblioref">1347</a>)</span>, the response demonstrated the potential for modular technologies to facilitate a response to emerging viral threats <span class="citation" data-cites="u0dESADU">(<a href="#ref-u0dESADU" role="doc-biblioref">1603</a>)</span>.
The potential for such vaccines to benefit the field of oncology has encouraged vaccine developers to invest in next-generation approaches, which has spurred the diversification of vaccine development programs <span class="citation" data-cites="wPl93ATP BsrTDzJ2">(<a href="#ref-BsrTDzJ2" role="doc-biblioref">1418</a>, <a href="#ref-wPl93ATP" role="doc-biblioref">1604</a>)</span>.
As a result, during the COVID-19 pandemic, these modular technologies have taken center stage in controlling a viral threat for the first time.</p>
<p>The safety and efficacy of vaccines that use these new technologies has exceeded expectations.
While there were rare reports of severe AEs such as myocarditis (mRNA platforms) and thromboembolic events (viral-vectored DNA platforms), widespread availability of both types of vaccines would allow individuals to choose (particularly relevant in this case because myocarditis has primarily been reported in men and thromboembolic events primarily in women).
Estimates of efficacy have varied widely, but in all cases are high.
Estimates of the efficacy of DNA vaccine platforms have typically fallen either in the range of approximately 67% (ZyCoV-D and Janssen) or 90% (Sputnik V).
AstraZeneca’s trial produced estimates in both ranges, with the standard dosage producing an efficacy of 62% and the lower prime dose producing a VE of 90%.
The mRNA vaccine trials were somewhat higher, with VE estimated at approximately 95% for both the Moderna and Pfizer/BioNTech clinical trials.
However, in all cases, the efficacy against severe illness and death were very high.
Therefore, all of these vaccines are useful tools for combating COVID-19.</p>
<p>Furthermore, the fact that vaccine efficacy is not a static value has become particularly salient, as real-world effectiveness has changed with location and over time.
COVID-19 vaccines have been challenged by the emergence of VOC.
These VOC generally carry genetic mutations that code for an altered Spike protein (i.e., the antigen), so the antibodies resulting from immunization with vaccines developed from the index strain neutralize them less effectively <span class="citation" data-cites="1B4h40dm5 yqFoGUHl">(<a href="#ref-1B4h40dm5" role="doc-biblioref">1605</a>, <a href="#ref-yqFoGUHl" role="doc-biblioref">1606</a>)</span>.
Despite some reports of varying and reduced effectiveness or efficacy of the mRNA vaccines against the Alpha (B.1.1.7), Beta (B.1.351), and Delta (B.1.617.2) variants versus the original SARS-CoV-2 strain or the D614G variant <span class="citation" data-cites="x5yLFKk8 19dwMfMGe 63wnlBQD">(<a href="#ref-x5yLFKk8" role="doc-biblioref">1607</a>–<a href="#ref-63wnlBQD" role="doc-biblioref">1609</a>)</span>, the greatest concern to date has been the Omicron variant (B.1.1.529), which was first identified in November 2021 <span class="citation" data-cites="k7L0WGEM yqFoGUHl">(<a href="#ref-yqFoGUHl" role="doc-biblioref">1606</a>, <a href="#ref-k7L0WGEM" role="doc-biblioref">1610</a>)</span>.
As of March 2022, the Omicron variant accounted for 95% of all infections sequenced in the United States <span class="citation" data-cites="1Bv67ENp2">(<a href="#ref-1Bv67ENp2" role="doc-biblioref">122</a>)</span> and was linked to an increased risk of SARS-CoV-2 reinfection <span class="citation" data-cites="1B4h40dm5">(<a href="#ref-1B4h40dm5" role="doc-biblioref">1605</a>)</span> and further infection of those who have been vaccinated with the mRNA vaccines <span class="citation" data-cites="lexoTbIa">(<a href="#ref-lexoTbIa" role="doc-biblioref">1611</a>)</span>.</p>
<p>One of the downsides of this leap in vaccine technologies, however, is that they have largely been developed by wealthy countries, including countries in the European Union, the United States, the U.K., and Russia.
As a result, they are also largely available to residents of wealthy countries, primarily in Europe and North America.<!--Add scatter plot with GDP and RNA doses/population, same for DNA-->
Although the VE of DNA vaccines tends to be lower than that of mRNA vaccines <span class="citation" data-cites="YTceJugW">(<a href="#ref-YTceJugW" role="doc-biblioref">1612</a>)</span>, they still provide excellent protection against severe illness and are much easier to distribute due to less complex demands for storage.
Efforts such as COVAX that aim to expand access to vaccines developed by wealthy countries have not been as successful as hoped <span class="citation" data-cites="17V4Lh5uy">(<a href="#ref-17V4Lh5uy" role="doc-biblioref">1613</a>)</span>.
Fortunately, vaccine development programs using more established technologies have been undertaken in many middle-income countries, and those vaccines have been more accessible globally <span class="citation" data-cites="1HWmcSB20">(<a href="#ref-1HWmcSB20" role="doc-biblioref">6</a>)</span>.
Additionally, efforts to develop new formulations of DNA vaccines in lower- and middle-income countries are increasingly being undertaken <span class="citation" data-cites="4eOROyon">(<a href="#ref-4eOROyon" role="doc-biblioref">1614</a>)</span>.</p>
<p>The modular nature of nucleic acid-based vaccine platforms has opened a new frontier in responding to emerging viral illnesses.
The RNA vaccines received an EUA in only a few months more than it took to identify the pathogen causing SARS in 2002.
Given the variety of options available for preventing severe illness and death, it is possible that certain vaccines may be preferable for certain demographics (e.g., young women might choose an mRNA vaccine to entirely mitigate the very low risk of blood clots <span class="citation" data-cites="EqBIFx5T">(<a href="#ref-EqBIFx5T" role="doc-biblioref">1536</a>)</span>).
However, this option is likely only available to people in high-income countries.
In lower-income countries, access to vaccines broadly is a more critical issue.
Different vaccines may confer advantages in different countries, and vaccine development in a variety of cultural contexts is therefore important <span class="citation" data-cites="hOkTKQ6z">(<a href="#ref-hOkTKQ6z" role="doc-biblioref">1615</a>)</span>.
Without widespread access to vaccines on the global scale, SARS-CoV-2 will continue evolving, presenting a threat to all nations.</p>
<h1 data-number="9" id="appendix-additional-information-about-novel-vaccine-platforms-for-covid-19"><span class="header-section-number">9</span> Appendix: Additional Information about Novel Vaccine Platforms for COVID-19</h1>
<h2 data-number="9.1" id="plasmid-vectored-dna-vaccines-1"><span class="header-section-number">9.1</span> Plasmid-Vectored DNA Vaccines</h2>
<h3 data-number="9.1.1" id="ino-4800"><span class="header-section-number">9.1.1</span> INO-4800</h3>
<p>The phase I trial for INO-4800 began enrolling participants in April 2020 in Philadelphia, PA at the Perelman School of Medicine and at the Center for Pharmaceutical Research in Kansas City, MO.
This trial examined two different doses administered in a two-dose regimen <span class="citation" data-cites="4xraQp8j">(<a href="#ref-4xraQp8j" role="doc-biblioref">1451</a>)</span>.
Among the 39 participants, only six AEs were reported and all were grade 1 <span class="citation" data-cites="4xraQp8j">(<a href="#ref-4xraQp8j" role="doc-biblioref">1451</a>)</span>.
Efficacy was evaluated based on blood samples collected pre- and post-vaccination, and all but three participants of 38 included in the analysis were found to have serum IgG binding titers to the spike protein after vaccination <span class="citation" data-cites="4xraQp8j">(<a href="#ref-4xraQp8j" role="doc-biblioref">1451</a>)</span>.</p>
<p>Results from the phase II trial were released as a preprint in May 2021 and reported findings based on administering INO-4800 to 401 adult volunteers at high risk of exposure to SARS-CoV-2 <span class="citation" data-cites="OYnqjMlC">(<a href="#ref-OYnqjMlC" role="doc-biblioref">1452</a>)</span>.
The phase II results supported that the vaccine was safe, with 1,446 treatment-related AEs observed across 281 participants, all but one of which were grade 1 or grade 2.
The single grade 3 event was joint stiffness <span class="citation" data-cites="OYnqjMlC">(<a href="#ref-OYnqjMlC" role="doc-biblioref">1452</a>)</span>.
The rates of AEs in the placebo group were not reported.
To assess the immunogenicity of INO-4800, pre- and post-vaccination blood samples were collected and evaluated for a humoral immune response to the spike protein, and the treatment group was identified to show significantly greater neutralizing activity than the placebo group <span class="citation" data-cites="OYnqjMlC">(<a href="#ref-OYnqjMlC" role="doc-biblioref">1452</a>)</span>.</p>
<p>The phase II/III trials are ongoing in several countries, including the United States, Mexico, India, and Colombia <span class="citation" data-cites="pxEE3VEQ R79Wr1hU yraL6YQa Wk6spoae">(<a href="#ref-pxEE3VEQ" role="doc-biblioref">1453</a>–<a href="#ref-Wk6spoae" role="doc-biblioref">1456</a>)</span>.
Therefore, vaccine efficacy data from a large study population is not yet available.</p>
<h2 data-number="9.2" id="viral-vectored-dna-vaccines-1"><span class="header-section-number">9.2</span> Viral-Vectored DNA Vaccines</h2>
<h3 data-number="9.2.1" id="chadox1-ncov-19-astrazeneca"><span class="header-section-number">9.2.1</span> ChAdOx1 nCoV-19 (AstraZeneca)</h3>
<p>Prior analyses of viral vector vaccines against human coronaviruses (HCoVs) had indicated that this approach showed potential for inducing an immune response, but little information was available about the effect on real-world immunity.
In the first phase of development, a candidate ChAdOx1 nCoV-19 was evaluated through the immune challenge of two animal models, mice and rhesus macaques <span class="citation" data-cites="1037p4Gvs">(<a href="#ref-1037p4Gvs" role="doc-biblioref">1495</a>)</span>.
Animals in the treatment condition were observed to develop neutralizing antibodies specific to SARS-CoV-2 (both macaques and mice) and to show reduced clinical scores when exposed to SARS-CoV-2 (macaques) <span class="citation" data-cites="1037p4Gvs">(<a href="#ref-1037p4Gvs" role="doc-biblioref">1495</a>)</span>.</p>
<p>Next, a phase I/II trial was undertaken using a single-blind, randomized controlled design <span class="citation" data-cites="2bBVSpM">(<a href="#ref-2bBVSpM" role="doc-biblioref">1496</a>)</span>.
ChAdOx1 nCoV-19 and a control, the meningococcal conjugate vaccine MenACWY, were administered intramuscularly to adults ages 18 to 55 at five sites within the United Kingdom (U.K.) at a 1:1 ratio (n=543 and n=534, respectively).
All but ten participants received a single dose; this small group received a booster 28 days after their first dose of ChAdOx1 nCoV-19.
Commonly reported local adverse reactions included mild-to-moderate pain and tenderness at the injection site over the course of seven days, while the most common systemic adverse reactions were fatigue and headache; some patients reported severe adverse systemic effects.
The study also reported that many common reactions could be reduced through the administration of paracetamol (acetaminophen), and paracetamol was not found to reduce immunogenicity.
Patients receiving the ChAdOx1 nCoV-19 vaccine developed antibodies to the SARS-CoV-2 spike protein that peaked by day 28, with these levels remaining stable until a second observation at day 56 except in the ten patients who received a booster dose at day 28, in whom they increased by day 56.
Analysis of serum indicated that participants developed antibodies to both S and the RBD, and that 100% of them achieved neutralizing titers by day 28.
By day 35, the neutralization titers of vaccinated patients was comparable to that observed with plasma from convalescents.
This initial study therefore suggested that the vaccine was likely to confer protection against SARS-CoV-2, although analysis of its efficacy in preventing COVID-19 was not reported.</p>
<p>The primary outcome assessed was symptomatic, laboratory-confirmed COVID-19.
There were 131 cases observed among the 11,636 participants eligible for the primary efficacy analysis, corresponding to an overall efficacy of 70.4% (30 out of 5,807 in the vaccine arm and 101 out of 5,829 in the control arm); the 95.8% CI was reported as 54.8 to 80.6.
However, a higher efficacy was reported in the subgroup of patients who received a low-dose followed by a standard dose (90.0%, 95% CI 67.4 to 97·0).
A total of ten cases of severe COVID-19 resulting in hospitalization were observed among trial participants, and all of these occurred in patients in the control arm of the study.
In line with the previously reported safety profiling for this vaccine, serious adverse events were reported to be comparable across the two arms of the study, with only three events identified as potentially associated with the vaccine itself.</p>
<p>Additional data about the efficacy of this vaccine became available in a preprint released on March 2, 2021 <span class="citation" data-cites="eV6UplSu">(<a href="#ref-eV6UplSu" role="doc-biblioref">1616</a>)</span>.
This report provided data describing the efficacy of ChAdOx1 nCoV-19, along with Pfizer/BioNTech’s BNT162b2, in the U.K. between December 8, 2020 and February 19, 2021 and specifically sought to evaluate the efficacy of the vaccine in the presence of a potentially more contagious variant of concern, B.1.1.7.
All participants in this study were age 70 or older and the efficacy was estimated to increase from 60% at 28 days after vaccination to 73% at 35 days after vaccination, although the standard error also increased over this time.
Therefore, preliminary results suggest that in a number of samples, this vaccine confers a high level of protection against SARS-CoV-2.</p>
<h3 data-number="9.2.2" id="sputnik-v-gam-covid-vac-and-gam-covid-vac-lyo"><span class="header-section-number">9.2.2</span> Sputnik-V (Gam-COVID-Vac and Gam-COVID-Vac-Lyo)</h3>
<p>The vaccine Gam-COVID-Vac, nicknamed Sputnik V in reference to the space race and “V for vaccine”, was developed by the Gamaleya National Center of Epidemiology and Microbiology in Moscow.
The development of Sputnik V was financed by the Russian Direct Investment Fund (RDIF) <span class="citation" data-cites="3KMxmQhV SxiGicKs">(<a href="#ref-3KMxmQhV" role="doc-biblioref">1516</a>, <a href="#ref-SxiGicKs" role="doc-biblioref">1617</a>)</span>.
The Sputnik V vaccines are available in both a lyophilized (Gam-COVID-Vac-Lyo) and frozen form (Gam-COVID-Vac), which are stored at 2-8°C and -18°C respectively <span class="citation" data-cites="PNZEiId1">(<a href="#ref-PNZEiId1" role="doc-biblioref">1521</a>)</span>.
The lyophilized vaccine is convenient for distribution and storage, particularly to remote or disadvantaged areas <span class="citation" data-cites="3KBugyZN">(<a href="#ref-3KBugyZN" role="doc-biblioref">1618</a>)</span>.</p>
<p>In the phase I/II trial study conducted between late June and early August 2020, 76 participants (18-60 years old) were split into two groups of 38 participants, which were non-randomized in two hospitals in Russia.
In phase I, 9 patients received rAd26 and 9 patients received rAd5-S to assess safety over 28 days.
In phase II, at least 5 days after the completion of phase I, 20 patients received a prime-boost vaccination of rAd26-S on day 0 and rAd5-S on day 2, which was administered intramuscularly.
The phase I/II trial reported that both vaccines were deemed safe and well tolerated.
The most common adverse events reported were mild, such as pain at the injection site (58%), hypothermia (50%), headaches (42%), fatigue (28%), and joint and muscle pain (24%).
Seroconversion was observed in all participants three weeks post the second vaccination (day 42), and all participants produced antibodies to the SARS-CoV-2 glycoprotein.
RBD-specific IgG levels were high in both the frozen and lyophilized versions of the vaccine (14,703 and 11,143 respectively), indicating a sufficient immune response to both.
Three weeks post the second vaccination, the virus-neutralizing geometric mean antibody titers were 49.25 and 45.95 from the frozen and lyophilized vaccines, respectively.
At 28 days, median cell proliferation of 1.3% CD4<sup>+</sup> and 1.1% CD8<sup>+</sup> were reported for the lyophilized vaccine and 2.5% CD4<sup>+</sup> and 1.3% CD8<sup>+</sup> for the vaccine stored frozen.
These results indicated that both forms of Sputnik V appeared to be safe and induced a humoral and cellular response in human subjects <span class="citation" data-cites="PNZEiId1">(<a href="#ref-PNZEiId1" role="doc-biblioref">1521</a>)</span>, which may be robust enough to persist and not wane rapidly <span class="citation" data-cites="sRAZYY9C">(<a href="#ref-sRAZYY9C" role="doc-biblioref">1497</a>)</span>.</p>
<p>In February 2021, the interim results of the phase III randomized, double-blind, placebo-controlled trial were published in <em>The Lancet</em> <span class="citation" data-cites="gLAIyAHm">(<a href="#ref-gLAIyAHm" role="doc-biblioref">1512</a>)</span>.
The participants were randomly assigned to receive either a 0.5 mL/dose of vaccine or placebo, which was comprised of the vaccine buffer composition, that was delivered intramuscularly using the same prime-boost regimen as in the phase I/II trials.
From September 7 to November 24, 19,866 participants completed the trial.
Of the 14,964 participants who received the vaccine, 16 (0.1%) were confirmed to have COVID-19, whereas 62 of the 4,902 participants (1.3%) in the placebo group were confirmed to have COVID-19.
Of these participants, no moderate or severe cases of COVID-19 were reported in the vaccine group, juxtaposed with 20 in the placebo group.
However, only symptomatic individuals were confirmed for SARS-CoV-2 infection in this trial.
Therefore, asymptomatic infections were not detected, thus potentially inflating the efficacy estimate.
Overall, a vaccine efficacy of 91.6% (95% CI 85.6-95.2) was reported, where an efficacy of 91.8% was reported for those over 60 years old and 92.7% for those who were 51-60 years old.
Indeed, 14 days after the first dose, 87.6% efficacy was achieved and the immunity required to prevent disease occurred within 18 days of vaccination.</p>
<p>Based on these results, scientists are investigating the potential for a single dose regimen of the rAd26-S sputnik V vaccine <span class="citation" data-cites="wCTVieA3">(<a href="#ref-wCTVieA3" role="doc-biblioref">1619</a>)</span>.
By the end of the trial, 7,485 participants reported adverse events, of which 94% were grade I.
Of the 68 participants who experienced serious adverse events during the trial, 45 from the vaccine group and 23 from the placebo groups, none were reported to be associated with the vaccination.
Likewise, 4 deaths occurred during the trial period that were not related to the vaccine <span class="citation" data-cites="gLAIyAHm">(<a href="#ref-gLAIyAHm" role="doc-biblioref">1512</a>)</span>.
The interim findings of the phase III trial indicate that the Sputnik V vaccine regimen appears to be safe with 91.6% efficacy.
Gamaleya had intended to reach a total of 40,000 participants for the completion of their phase III trial.
However, the trial has stopped enrolling participants and the numbers have been cut to 31,000 as many individuals in the placebo group dropped out of the study to obtain the vaccine <span class="citation" data-cites="XC4joIv9">(<a href="#ref-XC4joIv9" role="doc-biblioref">1620</a>)</span>.
Other trials involving Sputnik V are currently underway in Belarus, India, the United Arab Emirates, and Venezuela <span class="citation" data-cites="vrzaW9Gb">(<a href="#ref-vrzaW9Gb" role="doc-biblioref">1621</a>)</span>.</p>
<p>Preliminary results of a trial of Argentinian healthcare workers in Buenos Aires who were vaccinated with the Sputnik V rAd26-R vector-based vaccine seems to support the short term safety of the first vaccination <span class="citation" data-cites="uOMlbWFc">(<a href="#ref-uOMlbWFc" role="doc-biblioref">1622</a>)</span>.
Of the 707 vaccinated healthcare workers, 71.3% of the 96.6% of respondents reported at least one adverse event attributed to the vaccine.
Of these individuals, 68% experienced joint and muscle pain, 54% had injection site pain, 11% reported redness and swelling, 40% had a fever, and 5% reported diarrhea.
Only 5% of the vaccinated participants experienced serious adverse events that required medical attention, of which one was monitored as an inpatient.</p>
<p>Additionally, an independent assessment of Sputnik V in a phase II clinical trial in India found the vaccine to be effective, but the data is not yet publicly available <span class="citation" data-cites="jv875POj">(<a href="#ref-jv875POj" role="doc-biblioref">1623</a>)</span>.
On December 21, 2020, Gamaleya, AstraZeneca, R-Pharm, and the Russian Direct Investment Fund agreed to assess the safety and immunogenicity of the combined use of components of the AstraZeneca and University of Oxford AZD1222 (ChAdOx1) vaccine and the rAd26-S component of the Sputnik V vaccine in clinical trials <span class="citation" data-cites="QzHvKB8C">(<a href="#ref-QzHvKB8C" role="doc-biblioref">1624</a>)</span>.
This agreement hopes to establish scientific and business relations between the entities with an aim to co-develop a vaccine providing long-term immunization.
The trial, which will begin enrollment soon, will include 100 participants in a phase II open-label study and is hoped to be complete within 6 months.
Participants will first receive an intramuscular dose of AZD1222 on day 1, followed by a dose of rAd26 on day 29 and be monitored from day 1 for 180 days in total.
The primary outcomes measured will include incidence of serious adverse events post first dose until the end of the study.
Secondary outcome measures will include incidence of local and systemic adverse events 7 days post each dose, a time course of antibody responses for the Spike protein and the presence of anti-SARS-CoV-2 neutralizing antibodies <span class="citation" data-cites="1G2ROkAsZ">(<a href="#ref-1G2ROkAsZ" role="doc-biblioref">1625</a>)</span>.</p>
<p>Overall, there is hesitancy surrounding the management of the Sputnik V vaccine approval process and concerns over whether the efficacy data may be inflated due to a lack of asymptomatic testing within the trial.
However, the interim results of the phase III study were promising and further trials are underway, which will likely shed light on the overall efficacy and safety of the Sputnik V vaccine regimen.
There may be some advantage to the Sputnik V approach including the favorable storage conditions afforded by choice between a frozen and lyophilized vaccine.
Furthermore, the producers of Gam-COVID-Vac state that they can produce the vaccine at a cost of less than $10 per dose or less than $20 per patient <span class="citation" data-cites="AfkC38Sh">(<a href="#ref-AfkC38Sh" role="doc-biblioref">1626</a>)</span>.</p>
<h2 data-number="9.3" id="janssens-jnj-78436735"><span class="header-section-number">9.3</span> Janssen’s JNJ-78436735</h2>
<p>The Johnson and Johnson (J&amp;J) vaccine developed by Janssen Pharmaceuticals, Inc., a subsidiary of J&amp;J, was conducted in collaboration with and funded by “Operation Warp Speed” <span class="citation" data-cites="D3Px25HN 57BTbcko pWf2T8J8 10UC562ga">(<a href="#ref-10UC562ga" role="doc-biblioref">1424</a>, <a href="#ref-D3Px25HN" role="doc-biblioref">1499</a>–<a href="#ref-pWf2T8J8" role="doc-biblioref">1501</a>)</span>.
The vaccine was developed using Janssen’s AdVac<sup>®</sup> and PER.C6 platforms that were previously utilized to develop the European Commission-approved Ebola vaccine (Ad26 ZEBOV and MVN-BN-Filo) and their Zika, respiratory syncytial virus, and human immunodeficiency virus investigational vaccine candidates <span class="citation" data-cites="10uLoe1rR">(<a href="#ref-10uLoe1rR" role="doc-biblioref">1627</a>)</span>.</p>
<p>The development of a single-dose vaccine was desired by J&amp;J from the outset, with global deployment being a key priority <span class="citation" data-cites="gOOBv1MD">(<a href="#ref-gOOBv1MD" role="doc-biblioref">1502</a>)</span>.
Using their AdVac<sup>®</sup> technology, the vaccine can remain stable for up to two years between -15 and -25℃ and at least three months at 2 to 8℃ <span class="citation" data-cites="10uLoe1rR">(<a href="#ref-10uLoe1rR" role="doc-biblioref">1627</a>)</span>.
This allows the vaccine to be distributed easily without the requirement for very low temperature storage, unlike many of the other COVID-19 vaccine candidates.
J&amp;J screened numerous potential vaccine candidates <em>in vitro</em> and in animal models using varying different designs of the S protein, heterologous signal peptides, and prefusion-stabilizing substitutions <span class="citation" data-cites="10UC562ga">(<a href="#ref-10UC562ga" role="doc-biblioref">1424</a>)</span>.
A select few candidates were further investigated as a single dose regimen in Syrian golden hamsters, a single dose regimen in rhesus macaques, and a single- and two-dose regimen in both adult and aged rhesus macaques <span class="citation" data-cites="10UC562ga gOOBv1MD HmMIiIv2 EpOXYGt4">(<a href="#ref-10UC562ga" role="doc-biblioref">1424</a>, <a href="#ref-gOOBv1MD" role="doc-biblioref">1502</a>–<a href="#ref-EpOXYGt4" role="doc-biblioref">1504</a>)</span>.
A SARS-CoV-2 challenge study in rhesus macaques showed that vaccine doses as low as 2 x 10<sup>9</sup> viral particles/mL was sufficient to induce strong protection in bronchoalveolar lavage but that doses higher than 1.125 x 10<sup>10</sup> were required to close achieve close to complete protection in nasal swabs <span class="citation" data-cites="lVoienSE">(<a href="#ref-lVoienSE" role="doc-biblioref">1628</a>)</span>.
Indeed, six months post-immunization, levels of S-binding and neutralizing antibodies in rhesus macaques indicated that the JNJ-78436735 vaccine conferred durable protection against SARS-CoV-2 <span class="citation" data-cites="HGVDPMLm">(<a href="#ref-HGVDPMLm" role="doc-biblioref">1505</a>)</span>.</p>
<p>Following selection of the JNJ-78436735 vaccine, J&amp;J began phase I/IIa trials.
The interim phase I/IIa data was placed on the <em>medRxiv</em> preprint server on September 25th, 2020 <span class="citation" data-cites="14g52GtO3">(<a href="#ref-14g52GtO3" role="doc-biblioref">1629</a>)</span> and was later published in the <em>New England Journal of Medicine</em> on January 13th, 2021 <span class="citation" data-cites="pWf2T8J8">(<a href="#ref-pWf2T8J8" role="doc-biblioref">1501</a>)</span>.
The phase I/IIa multi-center, randomized, placebo-controlled trial enrolled 402 healthy participants between 18-55 years old and a further 403 healthy older participants ≥ 65 years old <span class="citation" data-cites="pWf2T8J8">(<a href="#ref-pWf2T8J8" role="doc-biblioref">1501</a>)</span>.
Patients were administered either a placebo, a low dose (5 x 10<sup>10</sup> viral particles per mL), or a high dose (1 X 10<sup>11</sup> viral particles per mL) intramuscularly as part of either a single- or two-dose regimen.
All patients received injections 56 days apart, but participants in the single-dose condition received the placebo at the second appointment.
Those who received only one dose of either vaccine received a placebo dose at their second vaccination visit.
The primary endpoints of both the trial were safety and reactogenicity of each dose.
Fatigue, headache, myalgia, and pain at the injection site were the most frequent solicited adverse events reported by participants.
Although less common, particularly for those in the elderly cohort and those on the low dose regimen, the most frequent systemic adverse effect was fever.
Overall, immunization was well tolerated, particularly at the lower dose concentration.
In terms of reactogenicity, over 90% of those who received either the low or high dose demonstrated seroconversion in a neutralization assay using wild-type SARS-CoV-2, 29 days after immunization <span class="citation" data-cites="pWf2T8J8">(<a href="#ref-pWf2T8J8" role="doc-biblioref">1501</a>)</span>.
Neutralizing geometric mean ratio of antibody titers (GMT) between 224-354 were detected regardless of age.
By day 57, 100% of the 18-55 year old participants had neutralizing GMT (288-488), which remained stable until day 71.
In the ≥ 65 years old cohort, the incidence of seroconversion for the low- and high-dose was 96% and 88% respectively by day 29.</p>
<p>GMTs for the low and high doses were slightly lower for participants ≥ 65 years old (196 and 127 respectively), potentially indicating slightly lower immunogenicity.
Seroconversion of the S antibodies was detected in 99% of individuals between 18-55 years old for the low and high doses (GMTs 528 and 695 respectively), with similar findings reported for the ≥ 65 years old.
Indeed, both dose concentrations also induced robust Th1 cytokine-producing S-specific CD4<sup>+</sup> T cells and CD8<sup>+</sup> T cell responses in both age groups.
The findings of the phase I/IIa study supported further investigation of a single immunization using the low dose vaccine.
Therefore, 25 patients were enrolled for a second randomized double-blind, placebo-controlled phase I clinical trial currently being conducted in Boston, Massachusetts for 2 years <span class="citation" data-cites="1CBMCD5I2">(<a href="#ref-1CBMCD5I2" role="doc-biblioref">1630</a>)</span>.
Participants received either a single dose followed by a placebo, or a double dose of either a low dose (5 x 10<sup>10</sup> viral particles/mL) or a high dose (1 x 10<sup>11</sup> viral particles/mL) vaccine administered intramuscularly on day 1 or day 57.
Placebo-only recipients received a placebo dose on day 1 and 57.
Interim analyses conducted on day 71 indicated that binding and neutralizing antibodies developed 8 days after administration in 90% and 25% of vaccine recipients, respectively.
Binding and neutralizing antibodies were detected in 100% of vaccine recipients by day 57 after a single dose immunization.
Spike-specific antibodies were highly prevalent (GMT 2432 to 5729) as were neutralizing antibodies (GMT 242 to 449) in the vaccinated groups.
Indeed, CD4<sup>+</sup> and CD8<sup>+</sup> T-cell responses were also induced, which may provide additional protection, particularly if antibodies wane or poorly respond to infection <span class="citation" data-cites="1GRYnvF01">(<a href="#ref-1GRYnvF01" role="doc-biblioref">1631</a>)</span>.</p>
<p>On September 23rd, 2020, J&amp;J launched its phase III trial ENSEMBLE and released the study protocol to the public <span class="citation" data-cites="10uLoe1rR 7n6WEkK8">(<a href="#ref-10uLoe1rR" role="doc-biblioref">1627</a>, <a href="#ref-7n6WEkK8" role="doc-biblioref">1632</a>)</span>.
The trial intended to enroll 60,000 volunteers to assess the safety and efficacy of the single vaccine dose versus placebo with primary endpoints of 14 and 28 days post-immunization <span class="citation" data-cites="10uLoe1rR">(<a href="#ref-10uLoe1rR" role="doc-biblioref">1627</a>)</span>.
The trial was conducted in Argentina, Brazil, Chile, Colombia, Mexico, Peru, South Africa, and the U.S.
The trial was paused briefly in October 2020 to investigate a “serious medical event”, but resumed shortly after <span class="citation" data-cites="5BTmfe4Y">(<a href="#ref-5BTmfe4Y" role="doc-biblioref">1633</a>)</span>.</p>
<p>An interim analysis was reported via press release on January 29th, 2021 <span class="citation" data-cites="iWMHpTBJ 1FcpboRMm">(<a href="#ref-iWMHpTBJ" role="doc-biblioref">1514</a>, <a href="#ref-1FcpboRMm" role="doc-biblioref">1515</a>)</span>.
The interim data included 43,783 participants who accrued 468 symptomatic cases of COVID-19.
It was reported that the JNJ-78436735 vaccine was 66% effective across all regions studied for the prevention of moderate to severe COVID-19 28 days post-vaccination in those aged 18 years and older.
Notably, JNJ-78436735 was 85% effective for the prevention of laboratory-confirmed severe COVID-19 and 100% protection against COVID-19-related hospitalization and death 28 days post-vaccination across all study sites.
Efficacy of the vaccine against severe COVID-19 increased over time, and there were no cases of COVID-19 reported in immunized participants after day 49.
The trial also determined that the vaccine candidate has a favorable safety profile as determined by an independent Data and Safety Monitoring Board.
The vaccine was well tolerated, consistent with previous vaccines produced using the AdVac<sup>®</sup> platform.
Fever occurred in 9% of vaccine recipients, with grade 3 fever occurring in only 0.2% of recipients.
Serious adverse events were reportedly higher in the placebo group than the vaccine group, and no anaphylaxis was reported <span class="citation" data-cites="1FcpboRMm">(<a href="#ref-1FcpboRMm" role="doc-biblioref">1515</a>)</span>.</p>
<p>At the time the phase III trial was being conducted, several concerning variants, including B.1.1.7 <span class="citation" data-cites="m9qtrWft">(<a href="#ref-m9qtrWft" role="doc-biblioref">482</a>)</span> and B.1.351 <span class="citation" data-cites="sqhvCTIL">(<a href="#ref-sqhvCTIL" role="doc-biblioref">231</a>)</span>, were spreading across the globe.
In particular, B.1.351 was first identified in South Africa, which was one of the JNJ-78436735 vaccine trial sites.
Therefore, the J&amp;J investigators also analyzed the efficacy of the JNJ-78436735 vaccine associated with their various trial sites to determine any potential risk of reduced efficacy as a result of the novel variants.
It was determined that JNJ-78436735 was 72% effective in the U.S., 66% effective in Latin America, and 57% effective in South Africa 28 days post-vaccination.
These findings underpin the importance of monitoring for the emergence of novel SARS-CoV-2 variants and determining their effects on vaccine efficacy.</p>
<p>Looking forward, Janssen are also running a phase III randomized, double-blind, placebo-controlled clinical trial, Ensemble 2, which aims to assess the efficacy, safety, and immunogenicity of a two-dose regimen of JNJ-78436735 administered 57 days apart.
This trial will enroll 30,000 participants ≥ 18 years old from Belgium, Colombia, France, Germany, Philippines, South Africa, Spain, U.K., and the U.S. <span class="citation" data-cites="sx7F1ktj">(<a href="#ref-sx7F1ktj" role="doc-biblioref">1634</a>)</span>.
This trial will also include participants with and without comorbidities associated with an increased risk of COVID-19.</p>
<h2 data-number="9.4" id="rna-vaccines"><span class="header-section-number">9.4</span> RNA Vaccines</h2>
<p>RNA vaccines are nucleic-acid based modalities that code for viral antigens against which the human body elicits a humoral and cellular immune response.
The resulting intracellular viral proteins are displayed on surface MHC proteins, provoking a strong CD8+ T cell response as well as a CD4+ T cell and B cell-associated antibody responses <span class="citation" data-cites="K0Ltu31S">(<a href="#ref-K0Ltu31S" role="doc-biblioref">1124</a>)</span>.
Given the potential for this technology to be quickly adapted for a new pathogen, it has held significant interest for the treatment of COVID-19.
The results of the interim analyses of two mRNA vaccine candidates became available at the end of 2020 and provided strong support for this emerging approach to vaccination.
Below we describe in detail the results available as of February 2021 for two such candidates, mRNA-1273 produced by ModernaTX and BNT162b2 produced by Pfizer, Inc. and BioNTech.
As of August 2022, the U.S. FDA has issues approvals or emergency use authorizations of versions of these vaccines for adults and for children 6 months and older <span class="citation" data-cites="Ude2jR1F">(<a href="#ref-Ude2jR1F" role="doc-biblioref">1635</a>)</span>.</p>
<h3 data-number="9.4.1" id="modernatx-mrna-vaccine"><span class="header-section-number">9.4.1</span> ModernaTX mRNA Vaccine</h3>
<p>ModernaTX’s mRNA-1273 vaccine was the first COVID-19 vaccine to enter a phase I clinical trial in the United States.
An initial report described the results of enrolling forty-five participants who were administered intramuscular injections of mRNA-1273 in their deltoid muscle on day 1 and day 29, with the goal of following patients for the next twelve months <span class="citation" data-cites="wiGjCZC8">(<a href="#ref-wiGjCZC8" role="doc-biblioref">1118</a>)</span>.
Healthy males and non-pregnant females aged 18-55 years were recruited for this study and divided into three groups receiving 25, 100, or 250 μg of mRNA-1273.
IgG ELISA assays on patient serology samples were used to examine the immunogenicity of the vaccine <span class="citation" data-cites="Biu1CQeQ">(<a href="#ref-Biu1CQeQ" role="doc-biblioref">1561</a>)</span>.
Binding antibodies were observed at two weeks after the first dose at all concentrations.
At the time point one week after the second dose was administered on day 29, the pseudotyped lentivirus reporter single-round-of-infection neutralization assay, which was used to assess neutralizing activity, reached a median level similar to the median observed in convalescent plasma samples.
Participants reported mild and moderate systemic adverse events after the day 1 injection, and one severe local event was observed in each of the two highest dose levels.
The second injection led to severe systemic adverse events for three of the participants at the highest dose levels, with one participant in the group being evaluated at an urgent care center on the day after the second dose.
The reported localized adverse events from the second dose were similar to those from the first.</p>
<p>Several months later, a press release from ModernaTX described the results of the first interim analysis of the vaccine <span class="citation" data-cites="IMmoqofb">(<a href="#ref-IMmoqofb" role="doc-biblioref">1636</a>)</span>.
On November 16, 2020, a report was released describing the initial results from phase III testing, corresponding to the first 95 cases of COVID-19 in the 30,000 enrolled participants <span class="citation" data-cites="IMmoqofb">(<a href="#ref-IMmoqofb" role="doc-biblioref">1636</a>)</span>, with additional data released to the FDA on December 17, 2020 <span class="citation" data-cites="QEdBZMLe">(<a href="#ref-QEdBZMLe" role="doc-biblioref">1637</a>)</span>.
These results were subsequently published in a peer-reviewed journal (<em>The New England Journal of Medicine</em>) on December 30, 2020 <span class="citation" data-cites="ZYxoabEm">(<a href="#ref-ZYxoabEm" role="doc-biblioref">1566</a>)</span>.
The first group of 30,420 study participants was randomized to receive the vaccine or a placebo at a ratio of 1:1 <span class="citation" data-cites="ZYxoabEm">(<a href="#ref-ZYxoabEm" role="doc-biblioref">1566</a>)</span>.
Administration occurred at 99 sites within the United States in two sessions, spaced 28 days apart <span class="citation" data-cites="16kGLLlT8 ZYxoabEm">(<a href="#ref-ZYxoabEm" role="doc-biblioref">1566</a>, <a href="#ref-16kGLLlT8" role="doc-biblioref">1638</a>)</span>.
Patients reporting COVID-19 symptoms upon follow-up were tested for SARS-CoV-2 using a nasopharyngeal swab that was evaluated with RT-PCR <span class="citation" data-cites="16kGLLlT8">(<a href="#ref-16kGLLlT8" role="doc-biblioref">1638</a>)</span>.
The initial preliminary analysis reported the results of the cases observed up until a cut-off date of November 11, 2020.
Of these first 95 cases reported, 90 occurred in participants receiving the placebo compared to 5 cases in the group receiving the vaccine <span class="citation" data-cites="IMmoqofb">(<a href="#ref-IMmoqofb" role="doc-biblioref">1636</a>)</span>.
These results suggested the vaccine is 94.5% effective in preventing COVID-19.
Additionally, eleven severe cases of COVID-19 were observed, and all eleven occurred in participants receiving the placebo.
The publication reported the results through an extended cut-off date of November 21, 2020, corresponding to 196 cases <span class="citation" data-cites="ZYxoabEm">(<a href="#ref-ZYxoabEm" role="doc-biblioref">1566</a>)</span>.
Of these, 11 occurred in the vaccine group and 185 in the placebo group, corresponding to an efficacy of 94.1%.
Once again, all of the severe cases of COVID-19 observed (n=30) occurred in the placebo group, including one death.
Thus, as more cases are reported, the efficacy of the vaccine has remained above 90%, and no cases of severe COVID-19 have yet been reported in participants receiving the vaccine.</p>
<p>These findings suggest the possibility that the vaccine might bolster immune defenses even for subjects who do still develop a SARS-CoV-2 infection.
The study was designed with an explicit goal of including individuals at high risk for COVID-19, including older adults, people with underlying health conditions, and people of color <span class="citation" data-cites="vkczroFC">(<a href="#ref-vkczroFC" role="doc-biblioref">1639</a>)</span>.
The phase III trial population was comprised by approximately 25.3% adults over age 65 in the initial report and 24.8% in the publication <span class="citation" data-cites="16kGLLlT8">(<a href="#ref-16kGLLlT8" role="doc-biblioref">1638</a>)</span>.
Among the cases reported by both interim analyses, 16-17% occurred in older adults <span class="citation" data-cites="IMmoqofb ZYxoabEm">(<a href="#ref-ZYxoabEm" role="doc-biblioref">1566</a>, <a href="#ref-IMmoqofb" role="doc-biblioref">1636</a>)</span>.
Additionally, approximately 10% of participants identified a Black or African-American background and 20% identified Hispanic or Latino ethnicity <span class="citation" data-cites="16kGLLlT8 ZYxoabEm">(<a href="#ref-ZYxoabEm" role="doc-biblioref">1566</a>, <a href="#ref-16kGLLlT8" role="doc-biblioref">1638</a>)</span>.
Among the first 95 cases, 12.6% occurred in participants identifying a Hispanic or Latino background and 4% in participants reporting a Black or African-American background <span class="citation" data-cites="IMmoqofb">(<a href="#ref-IMmoqofb" role="doc-biblioref">1636</a>)</span>; in the publication, they indicated only that 41 of the cases reported in the placebo group and 1 case in the treatment group occurred in “communities of color”, corresponding to 21.4% of all cases <span class="citation" data-cites="ZYxoabEm">(<a href="#ref-ZYxoabEm" role="doc-biblioref">1566</a>)</span>.
While the sample size in both analyses is small relative to the study population of over 30,000, these results suggest that the vaccine is likely to be effective in people from a variety of backgrounds.</p>
<p>In-depth safety data was released by ModernaTX as part of their application for an EUA from the FDA and summarized in the associated publication <span class="citation" data-cites="ZYxoabEm 16kGLLlT8">(<a href="#ref-ZYxoabEm" role="doc-biblioref">1566</a>, <a href="#ref-16kGLLlT8" role="doc-biblioref">1638</a>)</span>.
Because the detail provided in the report is greater than that provided in the publication, here we emphasize the results observed at the time of the first analysis.
Overall, a large percentage of participants reported adverse effects when solicited, and these reports were higher in the vaccine group than in the placebo group (94.5% versus 59.5%, respectively, at the time of the initial analysis) <span class="citation" data-cites="16kGLLlT8">(<a href="#ref-16kGLLlT8" role="doc-biblioref">1638</a>)</span>.
Some of these events met the criteria for grade 3 (local or systemic) or grade 4 (systemic only) toxicity <span class="citation" data-cites="16kGLLlT8">(<a href="#ref-16kGLLlT8" role="doc-biblioref">1638</a>)</span>, but most were grade 1 or grade 2 and lasted 2-3 days <span class="citation" data-cites="ZYxoabEm">(<a href="#ref-ZYxoabEm" role="doc-biblioref">1566</a>)</span>.
The most common local adverse reaction was pain at the injection site, reported by 83.7% of participants receiving the first dose of the vaccine and 88.4% upon receiving the second dose, compared to 19.8% and 17.0%, respectively, of patients in the placebo condition <span class="citation" data-cites="16kGLLlT8">(<a href="#ref-16kGLLlT8" role="doc-biblioref">1638</a>)</span>.
Fewer than 5% of vaccine recipients reported grade 3 pain at either administration.
Other frequent local reactions included erythema, swelling, and lymphadenopathy <span class="citation" data-cites="16kGLLlT8">(<a href="#ref-16kGLLlT8" role="doc-biblioref">1638</a>)</span>.
For systemic adverse reactions, fatigue was the most common <span class="citation" data-cites="16kGLLlT8">(<a href="#ref-16kGLLlT8" role="doc-biblioref">1638</a>)</span>.
Among participants receiving either dose of the vaccine, 68.5% reported fatigue compared to 36.1% participants receiving the placebo <span class="citation" data-cites="16kGLLlT8">(<a href="#ref-16kGLLlT8" role="doc-biblioref">1638</a>)</span>.
The level of fatigue experienced was usually fairly mild, with only 9.6% and 1.3% of participants in the vaccine and placebo conditions, respectively, reporting grade 3 fatigue <span class="citation" data-cites="16kGLLlT8">(<a href="#ref-16kGLLlT8" role="doc-biblioref">1638</a>)</span>, which corresponds to significant interference with daily activity <span class="citation" data-cites="oJWLuU0h">(<a href="#ref-oJWLuU0h" role="doc-biblioref">1640</a>)</span>.
Based on the results of the report, an EUA was issued on December 18, 2020 to allow distribution of this vaccine in the United States <span class="citation" data-cites="13Ou1UUAd">(<a href="#ref-13Ou1UUAd" role="doc-biblioref">1568</a>)</span>, and it was shortly followed by an Interim Order authorizing distribution of the vaccine in Canada <span class="citation" data-cites="HwoGQ6DD">(<a href="#ref-HwoGQ6DD" role="doc-biblioref">1641</a>)</span> and a conditional marketing authorization by the European Medicines Agency to facilitate distribution in the European Union <span class="citation" data-cites="4hVgIXyi">(<a href="#ref-4hVgIXyi" role="doc-biblioref">1642</a>)</span>.</p>
<h3 data-number="9.4.2" id="pfizerbiontech-bnt162b2"><span class="header-section-number">9.4.2</span> Pfizer/BioNTech BNT162b2</h3>
<p>ModernaTX was, in fact, the second company to release news of a successful interim analysis of an mRNA vaccine and receive an EUA.
The first report came from Pfizer and BioNTech’s mRNA vaccine BNT162b2 on November 9, 2020 <span class="citation" data-cites="16hlR7Xgi">(<a href="#ref-16hlR7Xgi" role="doc-biblioref">1643</a>)</span>, and a preliminary report was published in the <em>New England Journal of Medicine</em> one month later <span class="citation" data-cites="CWlYjjIV">(<a href="#ref-CWlYjjIV" role="doc-biblioref">516</a>)</span>.
This vaccine candidate should not be confused with a similar candidate from Pfizer/BioNTech, BNT162b1, that delivered only the RBD of the spike protein <span class="citation" data-cites="dlqAfQ7t T3BkYtz2">(<a href="#ref-dlqAfQ7t" role="doc-biblioref">1644</a>, <a href="#ref-T3BkYtz2" role="doc-biblioref">1645</a>)</span>, which was not advanced to a phase III trial because of the improved reactogenicity/immunogenicity profile of BNT162b2 <span class="citation" data-cites="MD2K7MYB">(<a href="#ref-MD2K7MYB" role="doc-biblioref">517</a>)</span>.</p>
<p>During the phase III trial of BNT162b2, 43,538 participants were enrolled 1:1 in the placebo and the vaccine candidate and received two 30-μg doses 21 days apart <span class="citation" data-cites="CWlYjjIV">(<a href="#ref-CWlYjjIV" role="doc-biblioref">516</a>)</span>.
Of these enrolled participants, 21,720 received BNT162b2 and 21,728 received a placebo <span class="citation" data-cites="CWlYjjIV">(<a href="#ref-CWlYjjIV" role="doc-biblioref">516</a>)</span>.
Recruitment occurred at 135 sites across six countries: Argentina, Brazil, Germany, South Africa, Turkey, and the United States.
An initial press release described the first 94 cases, which were consistent with 90% efficacy of the vaccine at 7 days following the second dose <span class="citation" data-cites="16hlR7Xgi">(<a href="#ref-16hlR7Xgi" role="doc-biblioref">1643</a>)</span>.
The release of the full trial information covered a longer period and analyzed the first 170 cases occurring at least 7 days after the second dose, 8 of which occurred in patients who had received BNT162b2.
The press release characterized the study population as diverse, reporting that 42% of the participants worldwide came from non-white backgrounds, including 10% Black and 26% Hispanic or Latino <span class="citation" data-cites="EMWkcH5x">(<a href="#ref-EMWkcH5x" role="doc-biblioref">1646</a>)</span>.
Within the United States, 10% and 13% of participants, respectively, identified themselves as having Black or Hispanic/Latino backgrounds <span class="citation" data-cites="EMWkcH5x">(<a href="#ref-EMWkcH5x" role="doc-biblioref">1646</a>)</span>.
Additionally, 41% of participants worldwide were 56 years of age or older <span class="citation" data-cites="EMWkcH5x">(<a href="#ref-EMWkcH5x" role="doc-biblioref">1646</a>)</span>, and they reported that the efficacy of the vaccine in adults over 65 was 94% <span class="citation" data-cites="Ufs4s7hG">(<a href="#ref-Ufs4s7hG" role="doc-biblioref">1647</a>)</span>.
The primary efficacy analysis of the phase III study was concluded on November 18, 2020 <span class="citation" data-cites="Ufs4s7hG">(<a href="#ref-Ufs4s7hG" role="doc-biblioref">1647</a>)</span>, and the final results indicted 94.6% efficacy of the vaccine <span class="citation" data-cites="CWlYjjIV">(<a href="#ref-CWlYjjIV" role="doc-biblioref">516</a>)</span>.</p>
<p>The safety profile of the vaccine was also assessed <span class="citation" data-cites="CWlYjjIV">(<a href="#ref-CWlYjjIV" role="doc-biblioref">516</a>)</span>.
A subset of patients were followed for reactogenicity using electronic diaries, with the data collected from these 8,183 participants comprising the solicited safety events analyzed.
Much like those who received the ModernaTX vaccine candidate, a large proportion of participants reported experiencing site injection pain within 7 days of vaccination.
While percentages are broken down by age group in the publication, these proportions correspond to approximately 78% and 73% of all participants after the first and second doses, respectively, overall.
Only a small percentage of these events (less than 1%) were rated as serious, with the rest being mild or moderate, and none reached grade 4.
Some participants also reported redness or swelling, and the publication indicates that in most cases, such events resolved within 1 to 2 days.
Participants also experienced systemic effects, including fever (in most cases lower than 38.9°C and more common after dose 2), fatigue (25-50% of participants depending on age group and dose), headache (25-50% of participants depending on age group and dose), chills, and muscle or joint pain; more rarely, patients could experience gastrointestinal effects such as vomiting or diarrhea.
As with the local events, these events were almost always grade 1 or 2.
While some events were reported by the placebo groups, these events were much rarer than in the treatment group even though compliance was similar.
Based on the efficacy and safety information released, the vaccine was approved in early December by the United Kingdom’s Medicines and Healthcare Products Regulatory Agency with administration outside of a clinical trial beginning on December 8, 2020 <span class="citation" data-cites="31rsgfCB fhw7l8VO">(<a href="#ref-31rsgfCB" role="doc-biblioref">1648</a>, <a href="#ref-fhw7l8VO" role="doc-biblioref">1649</a>)</span>.
As of December 11, 2020, the United States FDA approved this vaccine under an emergency use authorization <span class="citation" data-cites="cAaN4Te0">(<a href="#ref-cAaN4Te0" role="doc-biblioref">1567</a>)</span>, and in August 2021, it received full approval for ages 16 and older <span class="citation" data-cites="lwxl64fn">(<a href="#ref-lwxl64fn" role="doc-biblioref">1650</a>)</span>.</p>
<h3 data-number="9.4.3" id="neutralizing-of-voc"><span class="header-section-number">9.4.3</span> Neutralizing of VOC</h3>
<p>Prior to studies examining the effectiveness of vaccines in real-world settings (summarized in <span class="citation" data-cites="rjHK19Dr">(<a href="#ref-rjHK19Dr" role="doc-biblioref">7</a>)</span>, several studies reported reduced efficacy of the mRNA vaccines based on the measurement of antibody titers.
Plasma from individuals double-dosed with Pfizer/BioNTech’s BNT162b2 vaccine had up to a 16-fold reduction in neutralizing capacity against the Omicron variant <span class="citation" data-cites="gZ33CJWT">(<a href="#ref-gZ33CJWT" role="doc-biblioref">1651</a>)</span> and a reduced efficacy (70%) <span class="citation" data-cites="S6RHdOTJ">(<a href="#ref-S6RHdOTJ" role="doc-biblioref">1571</a>)</span>.
Estimates for the mRNA vaccines range from a 2-fold to over a 20-fold drop in neutralisation titers <span class="citation" data-cites="j172syOP">(<a href="#ref-j172syOP" role="doc-biblioref">1652</a>)</span>, hence the push for third and fourth doses of mRNA vaccines in many Western countries.
A third mRNA vaccine dose does increase antibody titers, but these levels also wane with time <span class="citation" data-cites="vJlYzFrS">(<a href="#ref-vJlYzFrS" role="doc-biblioref">1653</a>)</span>.
Notably, immunocompromised individuals such as cancer patients seem to elicit a sufficient protective immune response against the Omicron variant when they have been boosted with a third dose of either mRNA vaccine, albeit a blunted response <span class="citation" data-cites="sNDCRMQ5">(<a href="#ref-sNDCRMQ5" role="doc-biblioref">1654</a>)</span>.
While antibody titers do correlate with protection <span class="citation" data-cites="J069om3D 1AtNPSzpd QYbPf88B 1HCZbWd9m 4WhXhBth">(<a href="#ref-J069om3D" role="doc-biblioref">1655</a>–<a href="#ref-4WhXhBth" role="doc-biblioref">1659</a>)</span>, they are not the only mechanisms of immune protection.
For example, T cell and non-neutralizing antibody responses may be unaffected or less affected by the new VOC, and they warrant further investigation.</p>
<h2 data-number="9.5" id="global-vaccine-status-and-distribution-1"><span class="header-section-number">9.5</span> Global Vaccine Status and Distribution</h2>
<p>In North America, the majority of vaccines distributed until March 2021 have been produced by Pfizer-BioNTech and Moderna.
In Canada, the vaccine approval process is conducted by Health Canada, which uses a fast-tracked process whereby vaccine producers can submit data as it becomes available to allow for rapid review.
An approval may be granted following reviews of the available phase III clinical data.
This is followed by a period of pharmacovigilance in the population using their post-market surveillance system, which will monitor the long-term safety and efficacy of any vaccines <span class="citation" data-cites="1Mq7vYgc vZF52A03">(<a href="#ref-1Mq7vYgc" role="doc-biblioref">1660</a>, <a href="#ref-vZF52A03" role="doc-biblioref">1661</a>)</span>.
Health Canada has authorized the use of the Pfizer (December 9th, 2020), Moderna (December 23rd, 2020), Oxford-AstraZeneca (February 26th, 2021), and the Janssen (March 5th, 2021) vaccines, and the Novavax Inc vaccine is also under consideration <span class="citation" data-cites="ZkB13SYG">(<a href="#ref-ZkB13SYG" role="doc-biblioref">1662</a>)</span>.
While Canada initially projected that by the end of September 2021 a vaccine would be available for all Canadian adults, they now predict that it may be possible earlier as more vaccines have been approved and become available <span class="citation" data-cites="fQM1moSe">(<a href="#ref-fQM1moSe" role="doc-biblioref">1663</a>)</span>.</p>
<p>In the U.S., vaccines are required to have demonstrated safety and efficacy in phase III trials before manufacturers apply for an emergency use authorization (EUA) from the FDA.
If an EUA is granted, an additional evaluation of the safety and efficacy of the vaccines is conducted by the CDC’s Advisory Committee on Immunization Practices (ACIP) who also provide guidance on vaccine prioritization.
On December 1st, 2020, ACIP provided an interim phase 1a recommendation that healthcare workers and long-term care facility residents should be the first to be offered any vaccine approved <span class="citation" data-cites="18BMz232x">(<a href="#ref-18BMz232x" role="doc-biblioref">1664</a>)</span>.
This was shortly followed by an EUA on December 11th, 2020 for the use of the Pfizer-BioNTech COVID vaccine <span class="citation" data-cites="17wU8KTSP">(<a href="#ref-17wU8KTSP" role="doc-biblioref">1665</a>)</span>, which was distributed and administered to the first healthcare workers on December 14th, 2020 <span class="citation" data-cites="123cVqUNO">(<a href="#ref-123cVqUNO" role="doc-biblioref">1666</a>)</span>.
Shortly thereafter, an EUA for the Moderna vaccine was issued on December 18th, 2020 <span class="citation" data-cites="7yAHeCqZ">(<a href="#ref-7yAHeCqZ" role="doc-biblioref">1667</a>)</span>.
On December 20th, 2020, ACIP updated their initial recommendations to suggest that vaccinations should be offered to people aged 75 years old and older and to non-healthcare frontline workers in phase 1b <span class="citation" data-cites="Y3jvGtR9">(<a href="#ref-Y3jvGtR9" role="doc-biblioref">1668</a>)</span>.
On the same date, it was recommended that phase 1c should include people aged 65-74 years old, individuals between the ages of 16-74 years old at high-risk due to health conditions, and essential workers ineligible in phase 1b <span class="citation" data-cites="Y3jvGtR9">(<a href="#ref-Y3jvGtR9" role="doc-biblioref">1668</a>)</span>.
On the following day, December 21st, 2020, the first Moderna vaccines used outside of clinical trials were administered to American healthcare workers, which was the same day that President-elect Biden and Dr. Biden received their first doses of the Pfizer-BioNTech vaccine live on television to instill confidence in the approval and vaccination process <span class="citation" data-cites="f5yIh2Xp">(<a href="#ref-f5yIh2Xp" role="doc-biblioref">1669</a>)</span>.</p>
<p>On February 27th, 2020, the FDA issued an EUA for the Janssen COVID-19 Vaccine <span class="citation" data-cites="BG7N9ETs">(<a href="#ref-BG7N9ETs" role="doc-biblioref">1670</a>)</span>.
This was followed by an update on recommendations by ACIP for the use of the Janssen COVID-19 vaccine for those over 18 years old <span class="citation" data-cites="yNaiGtW1">(<a href="#ref-yNaiGtW1" role="doc-biblioref">1671</a>)</span>.
The Janssen vaccine was first distributed to healthcare facilities on March 1st, 2021.
On March 12, 2021, the WHO added the Janssen vaccine to the list of safe and effective emergency tools for COVID-19 <span class="citation" data-cites="Vd1wOy6d">(<a href="#ref-Vd1wOy6d" role="doc-biblioref">1672</a>)</span>.
While the CDC’s ACIP can provide recommendations, it is up to the public health authorities of each state, territory, and tribe to interpret the guidance and determine who will be vaccinated first <span class="citation" data-cites="1CcsUnCiw">(<a href="#ref-1CcsUnCiw" role="doc-biblioref">1673</a>)</span>.
Prior to distribution of the Janssen vaccine, over 103 million doses of the Moderna and Pfizer-BioNTech vaccines were delivered across the U.S., with almost 79 million doses administered.
Of the total population, 15.6% have received at least one dose and 7.9% have received a second dose of either the Moderna (~38.3 million) or the Pfizer-BioNTech (~40.2 million) vaccines by February 28th, 2021 <span class="citation" data-cites="iiTciwn1">(<a href="#ref-iiTciwn1" role="doc-biblioref">1674</a>)</span>.
President Biden’s administration has predicted that by the end of May 2021 there may be enough vaccine supply available for all adults in the U.S. <span class="citation" data-cites="ZkZ6ToLh 13bndHWdk">(<a href="#ref-ZkZ6ToLh" role="doc-biblioref">1675</a>, <a href="#ref-13bndHWdk" role="doc-biblioref">1676</a>)</span>.
However, vaccine production, approval, and distribution was not straightforward in the U.S., as information was initially sparse and the rollout of vaccines was complicated by poor planning and leadership due to political activities prior to the change of administration in January 2021 <span class="citation" data-cites="XOLE6iJT">(<a href="#ref-XOLE6iJT" role="doc-biblioref">1677</a>)</span>.
These political complications highlight the importance of the transparent vaccine approval process conducted by the FDA <span class="citation" data-cites="1Bgnim0gX">(<a href="#ref-1Bgnim0gX" role="doc-biblioref">1400</a>)</span>.</p>
<p>Outside the U.S., the Moderna and Pfizer-BioNTech vaccines have been administered in 29 and 69 other countries, respectively, mainly in Europe and North America <span class="citation" data-cites="dfl5iCJI">(<a href="#ref-dfl5iCJI" role="doc-biblioref">1289</a>)</span>.
The Janssen vaccine has so far only been administered in South Africa and the U.S. <span class="citation" data-cites="dfl5iCJI I0vakLIc">(<a href="#ref-dfl5iCJI" role="doc-biblioref">1289</a>, <a href="#ref-I0vakLIc" role="doc-biblioref">1678</a>)</span>, but it has also been approved in Bahrain, the European Union (E.U.), Iceland, Liechtenstein, and Norway <span class="citation" data-cites="wByD9WaX">(<a href="#ref-wByD9WaX" role="doc-biblioref">1171</a>)</span>.
On March 11th, 2021, Johnson &amp; Johnson received approval from the European Medicines Agency (EMA) for conditional marketing authorization of their vaccine <span class="citation" data-cites="hYospmts">(<a href="#ref-hYospmts" role="doc-biblioref">1679</a>)</span>.
Notably, on March 2nd, 2021, rivals Johnson &amp; Johnson and Merck announced that they entered an agreement to increase production of the Janssen vaccine to meet global demand <span class="citation" data-cites="hHW8U8rE">(<a href="#ref-hHW8U8rE" role="doc-biblioref">1680</a>)</span>.</p>
<p>The U.K. was the first country to approve use of the Pfizer-BioNTech vaccine on December 2nd, 2020 <span class="citation" data-cites="133HGZMEL">(<a href="#ref-133HGZMEL" role="doc-biblioref">1681</a>)</span>, and it was later approved by EMA on December 21st, 2020 <span class="citation" data-cites="ddNKvUqH">(<a href="#ref-ddNKvUqH" role="doc-biblioref">1682</a>)</span>.
The U.K. was also the first to administer the Pfizer-BioNTech vaccine, making it the first COVID-19 vaccine supported by phase III data to be administered outside of clinical trials on December 8th, 2020.
The Oxford-AstraZeneca vaccine, was approved by the Medicines and Healthcare Products Regulatory Agency (MHRA) in the U.K. and by EMA in the E.U. on December 30th (2020) <span class="citation" data-cites="1HIxYDTsj">(<a href="#ref-1HIxYDTsj" role="doc-biblioref">1683</a>)</span> and January 29th (2021) <span class="citation" data-cites="KUnUe1UK">(<a href="#ref-KUnUe1UK" role="doc-biblioref">1541</a>)</span>, respectively.
The Oxford-AstraZeneca vaccine was first administered in the UK on January 4th, 2021 <span class="citation" data-cites="q1Ui5Fm8">(<a href="#ref-q1Ui5Fm8" role="doc-biblioref">1684</a>)</span>, and it is now being used in 53 countries in total, including Brazil, India, Pakistan, Mexico, and spanning most of Europe <span class="citation" data-cites="dfl5iCJI">(<a href="#ref-dfl5iCJI" role="doc-biblioref">1289</a>)</span>.
The Moderna vaccine was authorized for use in the E.U. by EMA on January 6th, 2021 <span class="citation" data-cites="1GGF2oHr5">(<a href="#ref-1GGF2oHr5" role="doc-biblioref">1685</a>)</span> and in the U.K. by MHRA on January 8th, 2021 <span class="citation" data-cites="k9X9pXJe">(<a href="#ref-k9X9pXJe" role="doc-biblioref">1686</a>)</span>.
As of March 5th, 2021, 22 million people in the U.K. had received at least one vaccine dose <span class="citation" data-cites="cWMPXfju">(<a href="#ref-cWMPXfju" role="doc-biblioref">1290</a>)</span>.</p>
<p>While the Pfizer-BioNTech vaccine was the first to be distributed following phase III clinical trials, the first COVID-19 vaccine to be widely administered to people prior to the completion of phase III clinical trials was Sputnik V.
Sputnik V was administered to as many as 1.5 million Russians by early January <span class="citation" data-cites="X5LkVfY6">(<a href="#ref-X5LkVfY6" role="doc-biblioref">1508</a>)</span> due to the establishment of mass vaccination clinics in December 2020, prior to which only approximately 100,000 Russians had already been vaccinated <span class="citation" data-cites="uvQMgFXB Vwv7l7Hd">(<a href="#ref-uvQMgFXB" role="doc-biblioref">1687</a>, <a href="#ref-Vwv7l7Hd" role="doc-biblioref">1688</a>)</span>.
Doses of Sputnik V have also been distributed to other parts of Europe <span class="citation" data-cites="16LczMwFO Z0V7NK7Y 16GYKbrOq">(<a href="#ref-16LczMwFO" role="doc-biblioref">1509</a>–<a href="#ref-16GYKbrOq" role="doc-biblioref">1511</a>)</span>.
Hungary was the first E.U. member country to approve and distribute Sputnik V outside of Russia <span class="citation" data-cites="125VEHWS7">(<a href="#ref-125VEHWS7" role="doc-biblioref">1689</a>)</span>, despite the EMA stating that they had neither approved nor received a request for approval of Sputnik V <span class="citation" data-cites="tsHhIaZS">(<a href="#ref-tsHhIaZS" role="doc-biblioref">1690</a>)</span>.
Hungary is also in talks with China to procure the Sinopharm vaccines, which have been approved by Hungarian health authorities but also have not received approval by EMA in the E.U. <span class="citation" data-cites="125VEHWS7">(<a href="#ref-125VEHWS7" role="doc-biblioref">1689</a>)</span>.
In Latin America, production facilities in both Brazil and Argentina will allow for increased production capacity of Sputnik V and doses have been distributed to Mexico, Argentina, Bolivia, Nicaragua, Paraguay, and Venezuela <span class="citation" data-cites="ID8IywJM">(<a href="#ref-ID8IywJM" role="doc-biblioref">1691</a>)</span>.
Guinea was the first African nation to administer Sputnik V in December 2020, and the Central African Republic, Zimbabwe, and the Ivory Coast have all registered their interest in purchasing doses of the vaccine <span class="citation" data-cites="ID8IywJM">(<a href="#ref-ID8IywJM" role="doc-biblioref">1691</a>)</span>.
In the Middle East, Iran has received its first doses of Sputnik V and the United Arab Emirates is conducting phase III trials <span class="citation" data-cites="ID8IywJM">(<a href="#ref-ID8IywJM" role="doc-biblioref">1691</a>)</span>.
In Asia, while China’s vaccine candidates are favored, the Philippines, Nepal, and Uzbekistan have sought Sputnik V doses <span class="citation" data-cites="37uTx4hs">(<a href="#ref-37uTx4hs" role="doc-biblioref">1692</a>)</span>.
In total, the RDIF claims to have received orders totalling 1.2 billion doses by over 50 countries worldwide <span class="citation" data-cites="37uTx4hs">(<a href="#ref-37uTx4hs" role="doc-biblioref">1692</a>)</span> and at least 18 countries are currently administering Sputnik V around the globe <span class="citation" data-cites="dfl5iCJI">(<a href="#ref-dfl5iCJI" role="doc-biblioref">1289</a>)</span>.
Sputnik V has been an attractive vaccine for many countries due to its relatively low price, high efficacy, and its favorable storage conditions.
For some countries, Russia and China have also been more palatable politically than vaccine suppliers in the West <span class="citation" data-cites="ID8IywJM FAQXPsyc">(<a href="#ref-ID8IywJM" role="doc-biblioref">1691</a>, <a href="#ref-FAQXPsyc" role="doc-biblioref">1693</a>)</span>.
For others, the delays in the distribution of the other, more-favored candidates has been a motivating factor for pursuing the Sputnik V and Chinese alternatives <span class="citation" data-cites="FAQXPsyc Z0V7NK7Y">(<a href="#ref-Z0V7NK7Y" role="doc-biblioref">1510</a>, <a href="#ref-FAQXPsyc" role="doc-biblioref">1693</a>)</span>.
Additionally, Germany has stated that if Sputnik V were approved by EMA, it would be considered by the E.U. <span class="citation" data-cites="zCPl6A82">(<a href="#ref-zCPl6A82" role="doc-biblioref">1694</a>)</span>.
Russia is developing other vaccine candidates and has approved a third vaccine, CoviVac, which is an inactivated vaccine produced by the Chumakov Centre in Moscow, despite the fact the clinical trials have yet to begin <span class="citation" data-cites="hpScvlYg">(<a href="#ref-hpScvlYg" role="doc-biblioref">1695</a>)</span>.</p>
<h1 data-number="10" id="dietary-supplements-and-nutraceuticals-under-investigation-for-covid-19-prevention-and-treatment"><span class="header-section-number">10</span> Dietary Supplements and Nutraceuticals Under Investigation for COVID-19 Prevention and Treatment</h1>
<h2 data-number="10.1" id="abstract-6"><span class="header-section-number">10.1</span> Abstract</h2>
<p>Coronavirus disease 2019 (COVID-19) has caused global disruption and a significant loss of life.
Existing treatments that can be repurposed as prophylactic and therapeutic agents could reduce the pandemic’s devastation.
Emerging evidence of potential applications in other therapeutic contexts has led to the investigation of dietary supplements and nutraceuticals for COVID-19.
Such products include vitamin C, vitamin D, omega 3 polyunsaturated fatty acids, probiotics, and zinc, all of which are currently under clinical investigation.
In this review, we critically appraise the evidence surrounding dietary supplements and nutraceuticals for the prophylaxis and treatment of COVID-19.
Overall, further study is required before evidence-based recommendations can be formulated, but nutritional status plays a significant role in patient outcomes, and these products could help alleviate deficiencies.
For example, evidence indicates that vitamin D deficiency may be associated with greater incidence of infection and severity of COVID-19, suggesting that vitamin D supplementation may hold prophylactic or therapeutic value.
A growing number of scientific organizations are now considering recommending vitamin D supplementation to those at high risk of COVID-19.
Because research in vitamin D and other nutraceuticals and supplements is preliminary, here we evaluate the extent to which these nutraceutical and dietary supplements hold potential in the COVID-19 crisis.</p>
<h2 data-number="10.2" id="importance-6"><span class="header-section-number">10.2</span> Importance</h2>
<p>Sales of dietary supplements and nutraceuticals have increased during the pandemic due to their perceived “immune-boosting” effects.
However, little is known about the efficacy of these dietary supplements and nutraceuticals against the novel coronavirus (SARS-CoV-2) or the disease it causes, COVID-19.
This review provides a critical overview of the potential prophylactic and therapeutic value of various dietary supplements and nutraceuticals from the evidence available to date.
These include vitamin C, vitamin D, and zinc, which are often perceived by the public as treating respiratory infections or supporting immune health.
Consumers need to be aware of misinformation and false promises surrounding some supplements, which may be subject to limited regulation by authorities.
However, considerably more research is required to determine whether dietary supplements and nutraceuticals exhibit prophylactic and therapeutic value against SARS-CoV-2 infection and COVID-19.
This review provides perspective on which nutraceuticals and supplements are involved in biological processes that are relevant to recovery from or prevention of COVID-19.</p>
<h2 data-number="10.3" id="introduction-6"><span class="header-section-number">10.3</span> Introduction</h2>
<p>The year 2020 saw scientists and the medical community scrambling to repurpose or discover novel host-directed therapies against the coronavirus disease 2019 (COVID-19) pandemic caused by the spread of the novel <em>Severe acute respiratory syndrome-related coronavirus 2</em> (SARS-CoV-2).
This rapid effort led to the identification of some promising pharmaceutical therapies for hospitalized patients, such as remdesivir and dexamethasone.
Furthermore, most societies have adopted non-pharmacological preventative measures such as utilizing public health strategies that reduce the transmission of SARS-CoV-2.
However, during this time, many individuals sought additional protections via the consumption of various dietary supplements and nutraceuticals that they believed to confer beneficial effects.
While a patient’s nutritional status does seem to play a role in COVID-19 susceptibility and outcomes <span class="citation" data-cites="Lyd8s4Mk UFrQw1gC Hd5V5Usd 1FAOfFx5A 5tHMedSi">(<a href="#ref-Lyd8s4Mk" role="doc-biblioref">1696</a>–<a href="#ref-5tHMedSi" role="doc-biblioref">1700</a>)</span>, the beginning of the pandemic saw sales of vitamins and other supplements soar despite a lack of any evidence supporting their use against COVID-19.
In the United States, for example, dietary supplement and nutraceutical sales have shown modest annual growth in recent years (approximately 5%, or a $345 million increase in 2019), but during the six-week period preceding April 5, 2020, they increased by 44% ($435 million) relative to the same period in 2019 <span class="citation" data-cites="A225e05z">(<a href="#ref-A225e05z" role="doc-biblioref">1701</a>)</span>.
While growth subsequently leveled off, sales continued to boom, with a further 16% ($151 million) increase during the six weeks preceding May 17, 2020 relative to 2019 <span class="citation" data-cites="A225e05z">(<a href="#ref-A225e05z" role="doc-biblioref">1701</a>)</span>.
In France, New Zealand, India, and China, similar trends in sales were reported <span class="citation" data-cites="LMkJA6Ke puD7Ydak 1FjpTy3Dd 5NJabJny">(<a href="#ref-LMkJA6Ke" role="doc-biblioref">1702</a>–<a href="#ref-5NJabJny" role="doc-biblioref">1705</a>)</span>.
The increase in sales was driven by a consumer perception that dietary supplements and nutraceuticals would protect consumers from infection and/or mitigate the impact of infection due to the various “immune-boosting” claims of these products <span class="citation" data-cites="10JmgRgZG 18NItmm1M">(<a href="#ref-10JmgRgZG" role="doc-biblioref">1706</a>, <a href="#ref-18NItmm1M" role="doc-biblioref">1707</a>)</span>.</p>
<p>Due to the significant interest from the general public in dietary additives, whether and to what extent nutraceuticals or dietary supplements can provide any prophylactic or therapeutic benefit remains a topic of interest for the scientific community.
Nutraceuticals and dietary supplements are related but distinct non-pharmaceutical products.
Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value <span class="citation" data-cites="xLPeszeO 2Us0gNMr">(<a href="#ref-xLPeszeO" role="doc-biblioref">1708</a>, <a href="#ref-2Us0gNMr" role="doc-biblioref">1709</a>)</span>.
The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but also aid in the prophylaxis and/or treatment of a disorder or disease <span class="citation" data-cites="X0jEogv4">(<a href="#ref-X0jEogv4" role="doc-biblioref">1710</a>)</span>.
However, dietary supplements and nutraceuticals, unlike pharmaceuticals, are not subject to the same regulatory protocols that protect consumers of medicines.
Indeed, nutraceuticals do not entirely fall under the responsibility of the Food and Drug Administration (FDA), but they are monitored as dietary supplements according to the Dietary Supplement, Health and Education Act 1994 (DSHEA) <span class="citation" data-cites="10Tnnl6nF">(<a href="#ref-10Tnnl6nF" role="doc-biblioref">1711</a>)</span> and the Food and Drug Administration Modernization Act 1997 (FDAMA) <span class="citation" data-cites="16eEQf0HM">(<a href="#ref-16eEQf0HM" role="doc-biblioref">1712</a>)</span>.
Due to increases in sales of dietary supplements and nutraceuticals, in 1996 the FDA established the Office of Dietary Supplement Programs (ODSP) to increase surveillance.
Novel products or nutraceuticals must now submit a new dietary ingredient notification to the ODSP for review.
There are significant concerns that these legislations do not adequately protect the consumer as they ascribe responsibility to the manufacturers to ensure the safety of the product before manufacturing or marketing <span class="citation" data-cites="5TmtJWVQ">(<a href="#ref-5TmtJWVQ" role="doc-biblioref">1713</a>)</span>.
Manufacturers are not required to register or even seek approval from the FDA to produce or sell food supplements or nutraceuticals.
Health or nutrient content claims for labeling purposes are approved based on an authoritative statement from the Academy of Sciences or relevant federal authorities once the FDA has been notified and on the basis that the information is known to be true and not deceptive <span class="citation" data-cites="5TmtJWVQ">(<a href="#ref-5TmtJWVQ" role="doc-biblioref">1713</a>)</span>.
Therefore, there is often a gap between perceptions by the American public about a nutraceutical or dietary supplement and the actual clinical evidence surrounding its effects.</p>
<p>Despite differences in regulations, similar challenges exist outside of the United States.
In Europe, where the safety of supplements is monitored by the European Union (EU) under Directive 2002/46/EC <span class="citation" data-cites="CWdgoTUe">(<a href="#ref-CWdgoTUe" role="doc-biblioref">1714</a>/?uri=celex%3A32002L0046)</span>.
However, nutraceuticals are not directly mentioned.
Consequently, nutraceuticals can be generally described as either a medicinal product under Directive 2004/27/EC <span class="citation" data-cites="G7TnkK6t">(<a href="#ref-G7TnkK6t" role="doc-biblioref">1715</a>/?uri=CELEX:32004L0027)</span> or as a ‘foodstuff’ under Directive 2002/46/EC of the European council.
In order to synchronize the various existing legislations, Regulation EC 1924/2006 on nutrition and health claims was put into effect to assure customers of safety and efficacy of products and to deliver understandable information to consumers.
However, specific legislation for nutraceuticals is still elusive.
Health claims are permitted on a product label only following compliance and authorization according to the European Food Safety Authority (EFSA) guidelines on nutrition and health claims <span class="citation" data-cites="dzOveIQY">(<a href="#ref-dzOveIQY" role="doc-biblioref">1716</a>)</span>.
EFSA does not currently distinguish between food supplements and nutraceuticals for health claim applications of new products, as claim authorization is dependent on the availability of clinical data in order to substantiate efficacy <span class="citation" data-cites="MhvDTmeu">(<a href="#ref-MhvDTmeu" role="doc-biblioref">1717</a>)</span>.
These guidelines seem to provide more protection to consumers than the FDA regulations but potentially at the cost of innovation in the sector <span class="citation" data-cites="LBf0kxOm">(<a href="#ref-LBf0kxOm" role="doc-biblioref">1718</a>)</span>.
The situation becomes even more complicated when comparing regulations at a global level, as countries such as China and India have existing regulatory frameworks for traditional medicines and phytomedicines not commonly consumed in Western society <span class="citation" data-cites="jovk0AV9">(<a href="#ref-jovk0AV9" role="doc-biblioref">1719</a>)</span>.
Currently, there is debate among scientists and regulatory authorities surrounding the development of a widespread regulatory framework to deal with the challenges of safety and health claim substantiation for nutraceuticals <span class="citation" data-cites="MhvDTmeu 5TmtJWVQ">(<a href="#ref-5TmtJWVQ" role="doc-biblioref">1713</a>, <a href="#ref-MhvDTmeu" role="doc-biblioref">1717</a>)</span>, as these products do not necessarily follow the same rigorous clinical trial frameworks used to approve the use of pharmaceuticals.
Such regulatory disparities have been highlighted by the pandemic, as many individuals and companies have attempted to profit from the vulnerabilities of others by overstating claims in relation to the treatment of COVID-19 using supplements and nutraceuticals.
The FDA has written several letters to prevent companies marketing or selling products based on false hyperbolic promises about preventing SARS-CoV-2 infection or treating COVID-19 <span class="citation" data-cites="l2tEuVtb Qo0C93P6 133nfCM4l">(<a href="#ref-l2tEuVtb" role="doc-biblioref">1720</a>–<a href="#ref-133nfCM4l" role="doc-biblioref">1722</a>)</span>.
These letters came in response to efforts to market nutraceutical prophylactics against COVID-19, some of which charged the consumer as much as $23,000 <span class="citation" data-cites="7B3shLSx">(<a href="#ref-7B3shLSx" role="doc-biblioref">1723</a>)</span>.
There have even been some incidents highlighted in the media because of their potentially life threatening consequences; for example, the use of oleandrin was touted as a potential “cure” by individuals close to the former President of the United States despite its high toxicity <span class="citation" data-cites="Ssmsdskt">(<a href="#ref-Ssmsdskt" role="doc-biblioref">1724</a>)</span>.
Thus, heterogeneous and at times relaxed regulatory standards have permitted high-profile cases of the sale of nutraceuticals and dietary supplements that are purported to provide protection against COVID-19, despite a lack of research into these compounds.</p>
<p>Notwithstanding the issues of poor safety, efficacy, and regulatory oversight, some dietary supplements and nutraceuticals have exhibited therapeutic and prophylactic potential.
Some have been linked with reduced immunopathology, antiviral and anti-inflammatory activities, or even the prevention of acute respiratory distress syndrome (ARDS) <span class="citation" data-cites="GT7yDBjv 10JmgRgZG EEa0SZCF">(<a href="#ref-10JmgRgZG" role="doc-biblioref">1706</a>, <a href="#ref-GT7yDBjv" role="doc-biblioref">1725</a>, <a href="#ref-EEa0SZCF" role="doc-biblioref">1726</a>)</span>.
A host of potential candidates have been highlighted in the literature that target various aspects of the COVID-19 viral pathology, while others are thought to prime the host immune system.
These candidates include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) <span class="citation" data-cites="2hMFPQJR">(<a href="#ref-2hMFPQJR" role="doc-biblioref">1727</a>)</span>.
<em>In vitro</em> and <em>in vivo</em> studies suggest that nutraceuticals containing phycocyanobilin, N-acetylcysteine, glucosamine, selenium or phase 2 inductive nutraceuticals (e.g. ferulic acid, lipoic acid, or sulforaphane) can prevent or modulate RNA virus infections via amplification of the signaling activity of mitochondrial antiviral-signaling protein (MAVS) and activation of Toll-like receptor 7 <span class="citation" data-cites="eyBAUUCl">(<a href="#ref-eyBAUUCl" role="doc-biblioref">1728</a>)</span>.
Phase 2 inductive molecules used in the production of nutraceuticals are known to activate nuclear factor erythroid 2–related factor 2 (Nrf2), which is a protein regulator of antioxidant enzymes that leads to the induction of several antioxidant enzymes, such as gamma-glutamylcysteine synthetase.
While promising, further animal and human studies are required to assess the therapeutic potential of these various nutrients and nutraceuticals against COVID-19.
For the purpose of this review, we have highlighted some of the main dietary supplements and nutraceuticals that are currently under investigation for their potential prophylactic and therapeutic applications.
These include n-3 PUFA, zinc, vitamins C and D, and probiotics.</p>
<h2 data-number="10.4" id="n-3-pufa"><span class="header-section-number">10.4</span> n-3 PUFA</h2>
<p>One category of supplements that has been explored for beneficial effects against various viral infections are the n-3 PUFAs <span class="citation" data-cites="2hMFPQJR">(<a href="#ref-2hMFPQJR" role="doc-biblioref">1727</a>)</span>, commonly referred to as omega-3 fatty acids, which include eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).
EPA and DHA intake can come from a diet high in fish or through dietary supplementation with fish oils or purified oils <span class="citation" data-cites="Ufvsmc3N">(<a href="#ref-Ufvsmc3N" role="doc-biblioref">1729</a>)</span>.
Other, more sustainable sources of EPA and DHA include algae <span class="citation" data-cites="Gzqj7Okm Txn1mlRl">(<a href="#ref-Gzqj7Okm" role="doc-biblioref">1730</a>, <a href="#ref-Txn1mlRl" role="doc-biblioref">1731</a>)</span>, which can also be exploited for their rich abundance of other bioactive compounds such as angiotensin converting enzyme inhibitor peptides and antiviral agents including phycobiliproteins, sulfated polysaccharides, and calcium-spirulan <span class="citation" data-cites="S1A3WazG">(<a href="#ref-S1A3WazG" role="doc-biblioref">1732</a>)</span>.
n-3 PUFAs have been investigated for many years for their therapeutic potential <span class="citation" data-cites="4xymCvKH">(<a href="#ref-4xymCvKH" role="doc-biblioref">1733</a>)</span>.
Supplementation with fish oils is generally well tolerated <span class="citation" data-cites="4xymCvKH">(<a href="#ref-4xymCvKH" role="doc-biblioref">1733</a>)</span>, and intake of n-3 PUFAs through dietary sources or supplementation is specifically encouraged for vulnerable groups such as pregnant and lactating women <span class="citation" data-cites="17CHOdNbe g4dU6DMd">(<a href="#ref-17CHOdNbe" role="doc-biblioref">1734</a>, <a href="#ref-g4dU6DMd" role="doc-biblioref">1735</a>)</span>.
As a result, these well-established compounds have drawn significant interest for their potential immune effects and therapeutic potential.</p>
<p>Particular interest has arisen in n-3 PUFAs as potential therapeutics against diseases associated with inflammation.
n-3 PUFAs have been found to modulate inflammation by influencing processes such as leukocyte chemotaxis, adhesion molecule expression, and the production of eicosanoids <span class="citation" data-cites="JNyhLAfe cvX4fl0q">(<a href="#ref-JNyhLAfe" role="doc-biblioref">1736</a>, <a href="#ref-cvX4fl0q" role="doc-biblioref">1737</a>)</span>.
This and other evidence indicates that n-3 PUFAs may have the capacity to modulate the adaptive immune response <span class="citation" data-cites="JNyhLAfe Ufvsmc3N 2Us0gNMr">(<a href="#ref-2Us0gNMr" role="doc-biblioref">1709</a>, <a href="#ref-Ufvsmc3N" role="doc-biblioref">1729</a>, <a href="#ref-JNyhLAfe" role="doc-biblioref">1736</a>)</span>; for example, they have been found to influence antigen presentation and the production of CD4(+) Th1 cells, among other relevant effects <span class="citation" data-cites="165UR0Ipw">(<a href="#ref-165UR0Ipw" role="doc-biblioref">1738</a>)</span>.
Certainly, preliminary evidence from banked blood samples from 100 COVID-19 patients suggests that patients with a higher omega-3 index, a measure of the amount of EPA and DHA in red blood cells, had a lower risk of death due to COVID-19 <span class="citation" data-cites="TTfh7mb9">(<a href="#ref-TTfh7mb9" role="doc-biblioref">1739</a>)</span>.
Interest has also arisen as to whether nutritional status related to n-3 PUFAs can also affect inflammation associated with severe disease, such as ARDS or sepsis <span class="citation" data-cites="FtxL59jJ 12vyTsM9W">(<a href="#ref-FtxL59jJ" role="doc-biblioref">1740</a>, <a href="#ref-12vyTsM9W" role="doc-biblioref">1741</a>)</span>.
ARDS and sepsis hold particular concern in the treatment of severe COVID-19; an analysis of 82 deceased COVID-19 patients in Wuhan during January to February 2020 reported that respiratory failure (associated with ARDS) was the cause of death in 69.5% of cases, and sepsis or multi-organ failure accounted for 28.0% of deaths <span class="citation" data-cites="16mSlRgM3">(<a href="#ref-16mSlRgM3" role="doc-biblioref">742</a>)</span>.
Research in ARDS prior to current pandemic suggests that n-3 PUFAs may hold some therapeutic potential.
One study randomized 16 consecutive ARDS patients to receive either a fish oil-enriched lipid emulsion or a control lipid emulsion (comprised of 100% long-chain triglycerides) under a double-blinded design <span class="citation" data-cites="LmdFoxGc">(<a href="#ref-LmdFoxGc" role="doc-biblioref">1742</a>)</span>.
They reported a statistically significant reduction in leukotriene B4 levels in the group receiving the fish oil-enriched emulsion, suggesting that the fish oil supplementation may have reduced inflammation.
However, they also reported that most of their tests were not statistically significant, and therefore it seems that additional research using larger sample sizes is required.
A recent meta-analysis of 10 randomized controlled trials (RCTs) examining the effects of n-3 PUFAs on ARDS patients did not find evidence of any effect on mortality, although the effect on secondary outcomes could not be determined due to a low quality of evidence <span class="citation" data-cites="q9FzffQx">(<a href="#ref-q9FzffQx" role="doc-biblioref">1743</a>)</span>.
However, another meta-analysis that examined 24 RCTs studying the effects of n-3 fatty acids on sepsis, including ARDS-induced sepsis, did find support for an effect on mortality when n-3 fatty acids were administered via enteral nutrition, although a paucity of high-quality evidence again limited conclusions <span class="citation" data-cites="EwajTsK5">(<a href="#ref-EwajTsK5" role="doc-biblioref">1744</a>)</span>.
Therefore, despite theoretical support for an immunomodulatory effect of n-3 PUFAs in COVID-19, evidence from existing RCTs is insufficient to determine whether supplementation offers an advantage in a clinical setting that would be relevant to COVID-19.</p>
<p>Another potential mechanism that has led to interest in n-3 PUFAs as protective against viral infections including COVID-19 is its potential as a precursor molecule for the biosynthesis of endogenous specialized proresolving mediators (SPM), such as protectins and resolvins, that actively resolve inflammation and infection <span class="citation" data-cites="BIVORG2X">(<a href="#ref-BIVORG2X" role="doc-biblioref">1745</a>)</span>.
SPM have exhibited beneficial effects against a variety of lung infections, including some caused by RNA viruses <span class="citation" data-cites="uPOTvvjR sxgmhJ68">(<a href="#ref-uPOTvvjR" role="doc-biblioref">1746</a>, <a href="#ref-sxgmhJ68" role="doc-biblioref">1747</a>)</span>.
Several mechanisms for SPM have been proposed, including preventing the release of pro-inflammatory cytokines and chemokines or increasing phagocytosis of cellular debris by macrophages <span class="citation" data-cites="OSEnHt6H">(<a href="#ref-OSEnHt6H" role="doc-biblioref">1748</a>)</span>.
In influenza, SPM promote antiviral B lymphocytic activities <span class="citation" data-cites="MK0GQ55y">(<a href="#ref-MK0GQ55y" role="doc-biblioref">1749</a>)</span>, and protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery <span class="citation" data-cites="fIo3ESPM">(<a href="#ref-fIo3ESPM" role="doc-biblioref">1750</a>)</span>.
It has thus been hypothesized that SPM could aid in the resolution of the cytokine storm and pulmonary inflammation associated with COVID-19 <span class="citation" data-cites="14LjKv9ZH PRLlghQS">(<a href="#ref-14LjKv9ZH" role="doc-biblioref">1751</a>, <a href="#ref-PRLlghQS" role="doc-biblioref">1752</a>)</span>.
Another theory is that some comorbidities, such as obesity, could lead to deficiencies of SPM, which could in turn be related to the occurrence of adverse outcomes for COVID-19 <span class="citation" data-cites="oLunKkIG">(<a href="#ref-oLunKkIG" role="doc-biblioref">1753</a>)</span>.
However, not all studies are in agreement that n-3 PUFAs or their resulting SPM are effective against infections <span class="citation" data-cites="B8toe6Mr">(<a href="#ref-B8toe6Mr" role="doc-biblioref">1754</a>)</span>.
At a minimum, the effectiveness of n-3 PUFAs against infections would be dependent on the dosage, timing, and the specific pathogens responsible <span class="citation" data-cites="3Xht24k3">(<a href="#ref-3Xht24k3" role="doc-biblioref">1755</a>)</span>.
On another level, there is still the question of whether fish oils can raise the levels of SPM levels upon ingestion and in response to acute inflammation in humans <span class="citation" data-cites="Ge00xeR3">(<a href="#ref-Ge00xeR3" role="doc-biblioref">1756</a>)</span>.
Currently, Karolinska University Hospital is running a trial that will measure the levels of SPM as a secondary outcome following intravenous supplementation of n-3 PUFAs in hospitalized COVID-19 patients to determine whether n-3 PUFAs provides therapeutic value <span class="citation" data-cites="n93SvQ2p pIfWCVkv">(<a href="#ref-n93SvQ2p" role="doc-biblioref">1757</a>, <a href="#ref-pIfWCVkv" role="doc-biblioref">1758</a>)</span>.
Therefore, while this mechanism provides theoretical support for a role for n-3 PUFAs against COVID-19, experimental support is still needed.</p>
<p>A third possible mechanism by which n-3 PUFAs could benefit COVID-19 patients arises from the fact that some COVID-19 patients, particularly those with comorbidities, are at a significant risk of thrombotic complications including arterial and venous thrombosis <span class="citation" data-cites="17jegDTvF PyKMLraw">(<a href="#ref-PyKMLraw" role="doc-biblioref">105</a>, <a href="#ref-17jegDTvF" role="doc-biblioref">1759</a>)</span>.
Therefore, the use of prophylactic and therapeutic anticoagulants and antithrombotic agents is under consideration <span class="citation" data-cites="SxPCbOB1 rWVL91jq">(<a href="#ref-SxPCbOB1" role="doc-biblioref">1760</a>, <a href="#ref-rWVL91jq" role="doc-biblioref">1761</a>)</span>.
Considering that there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties <span class="citation" data-cites="umzPtQG7 V9MDMfpj Ufvsmc3N">(<a href="#ref-Ufvsmc3N" role="doc-biblioref">1729</a>, <a href="#ref-umzPtQG7" role="doc-biblioref">1762</a>, <a href="#ref-V9MDMfpj" role="doc-biblioref">1763</a>)</span>, they may have therapeutic value against the prothrombotic complications of COVID-19.
In particular, concerns have been raised within the medical community about using investigational therapeutics on COVID-19 patients who are already on antiplatelet therapies due to pre-existing comorbidities because the introduction of such therapeutics could lead to issues with dosing and drug choice and/or negative drug-drug interactions <span class="citation" data-cites="SxPCbOB1">(<a href="#ref-SxPCbOB1" role="doc-biblioref">1760</a>)</span>.
In such cases, dietary sources of n-3 fatty acids or other nutraceuticals with antiplatelet activities could hold particular value for reducing the risk of thrombotic complications in patients already receiving pharmaceutical antiplatelet therapies.
A new clinical trial <span class="citation" data-cites="QKHdIx2w">(<a href="#ref-QKHdIx2w" role="doc-biblioref">1764</a>)</span> is currently recruiting COVID-19 positive patients to investigate the anti-inflammatory activity of a recently developed, highly purified nutraceutical derivative of EPA known as icosapent ethyl (Vascepa<sup>TM</sup>) <span class="citation" data-cites="12Q4bY0vF">(<a href="#ref-12Q4bY0vF" role="doc-biblioref">1765</a>)</span>.
Other randomized controlled trials that are in the preparatory stages intend to investigate the administration of EPA and other bioactive compounds to COVID-19 positive patients in order to observe whether anti-inflammatory effects or disease state improvements occur <span class="citation" data-cites="8lPjdnE7 1AGVAbQuf">(<a href="#ref-8lPjdnE7" role="doc-biblioref">1766</a>, <a href="#ref-1AGVAbQuf" role="doc-biblioref">1767</a>)</span>.
Finally, while there have been studies investigating the therapeutic value of n-3 fatty acids against ARDS in humans, there is still limited evidence of their effectiveness <span class="citation" data-cites="XHtzuR5c">(<a href="#ref-XHtzuR5c" role="doc-biblioref">1768</a>)</span>.
It should be noted that the overall lack of human studies in this area means there is limited evidence as to whether these supplements could affect COVID-19 infection.
Consequently, the clinical trials that are underway and those that have been proposed will provide valuable insight into whether the anti-inflammatory potential of n-3 PUFAs and their derivatives can be beneficial to the treatment of COVID-19.
All the same, while the evidence is not present to draw conclusions about whether n-3 PUFAs will be useful in treating COVID-19, there is likely little harm associated with a diet rich in fish oils, and interest in n-3 PUFA supplementation by the general public is unlikely to have negative effects.</p>
<h2 data-number="10.5" id="zinc"><span class="header-section-number">10.5</span> Zinc</h2>
<p>Zinc is nutrient supplement that may exhibit some benefits against RNA viral infections.
Zinc is a trace metal obtained from dietary sources or supplementation and is important for the maintenance of immune cells involved in adaptive and innate immunity <span class="citation" data-cites="VMI1lOFR">(<a href="#ref-VMI1lOFR" role="doc-biblioref">1769</a>)</span>.
Supplements can be administered orally as a tablet or as a lozenge and are available in many forms, such as zinc picolinate, zinc acetate, and zinc citrate.
Zinc is also available from dietary sources including meat, seafood, nuts, seeds, legumes, and dairy.
The role of zinc in immune function has been extensively reviewed <span class="citation" data-cites="VMI1lOFR">(<a href="#ref-VMI1lOFR" role="doc-biblioref">1769</a>)</span>.
Zinc is an important signaling molecule, and zinc levels can alter host defense systems.
In inflammatory situations such as an infection, zinc can regulate leukocyte immune responses and modulate the nuclear factor kappa-light-chain-enhancer of activated B cells, thus altering cytokine production <span class="citation" data-cites="Qif5U0Ig 1CEzU1G34">(<a href="#ref-Qif5U0Ig" role="doc-biblioref">1770</a>, <a href="#ref-1CEzU1G34" role="doc-biblioref">1771</a>)</span>.
In particular, zinc supplementation can increase natural killer cell levels, which are important cells for host defense against viral infections <span class="citation" data-cites="7ElRN12v VMI1lOFR">(<a href="#ref-VMI1lOFR" role="doc-biblioref">1769</a>, <a href="#ref-7ElRN12v" role="doc-biblioref">1772</a>)</span>.
As a result of these immune-related functions, zinc is also under consideration for possible benefits against COVID-19.</p>
<p>Adequate zinc intake has been associated with reduced incidence of infection <span class="citation" data-cites="kxuSwrCI">(<a href="#ref-kxuSwrCI" role="doc-biblioref">1773</a>)</span> and antiviral immunity <span class="citation" data-cites="IAOnO0Gj">(<a href="#ref-IAOnO0Gj" role="doc-biblioref">1774</a>)</span>.
A randomized, double-blind, placebo-controlled trial that administered zinc supplementation to elderly subjects over the course of a year found that zinc supplementation decreased susceptibility to infection and that zinc deficiency was associated with increased susceptibility to infection <span class="citation" data-cites="kxuSwrCI">(<a href="#ref-kxuSwrCI" role="doc-biblioref">1773</a>)</span>.
Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms <span class="citation" data-cites="sgr0mlJK lPexOhy6">(<a href="#ref-sgr0mlJK" role="doc-biblioref">1775</a>, <a href="#ref-lPexOhy6" role="doc-biblioref">1776</a>)</span>.
An observational study showed that COVID-19 patients had significantly lower zinc levels in comparison to healthy controls and that zinc-deficient COVID-19 patients (those with levels less than 80 μg/dl) tended to have more complications (70.4% vs 30.0%, <em>p</em> = 0.009) and potentially prolonged hospital stays (7.9 vs 5.7 days, <em>p</em> = 0.048) relative to patients who were not zinc deficient <span class="citation" data-cites="1BU6jG2lv">(<a href="#ref-1BU6jG2lv" role="doc-biblioref">1777</a>)</span>.
In coronaviruses specifically, <em>in vitro</em> evidence has demonstrated that the combination of zinc (Zn<sup>2+</sup>) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a fluorescently tagged SARS-CoV-1) and a variety of other RNA viruses <span class="citation" data-cites="6aOUJdHa NB1udFHF">(<a href="#ref-6aOUJdHa" role="doc-biblioref">1778</a>, <a href="#ref-NB1udFHF" role="doc-biblioref">1779</a>)</span>.
Currently, there are over twenty clinical trials registered with the intention to use zinc in a preventative or therapeutic manner for COVID-19.
However, many of these trials proposed the use of zinc in conjunction with hydroxychloroquine and azithromycin <span class="citation" data-cites="XJrj3MiZ 1EqFW3G4X Hc9mjVB2 T0aXNTIp">(<a href="#ref-XJrj3MiZ" role="doc-biblioref">1780</a>–<a href="#ref-T0aXNTIp" role="doc-biblioref">1783</a>)</span>, and it is not known how the lack of evidence supporting the use of hydroxychloroquine will affect investigation of zinc.
One retrospective observational study of New York University Langone hospitals in New York compared outcomes among hospitalized COVID-19 patients administered hydroxychloroquine and azithromycin with zinc sulfate (n = 411) versus hydroxychloroquine and azithromycin alone (n = 521).
Notably, zinc is the only treatment that was used in this trial that is still under consideration as a therapeutic agent due to the lack of efficacy and potential adverse events associated with hydroxychloroquine and azithromycin against COVID-19 <span class="citation" data-cites="SmqHOsyB T94OBz6q 1F4wtzwpV">(<a href="#ref-SmqHOsyB" role="doc-biblioref">1784</a>–<a href="#ref-1F4wtzwpV" role="doc-biblioref">1786</a>)</span>.
While the addition of zinc sulfate did not affect the duration of hospitalization, the length of ICU stays or patient ventilation duration, univariate analyses indicated that zinc did increase the frequency of patients discharged and decreased the requirement for ventilation, referrals to the ICU, and mortality <span class="citation" data-cites="Xi9zDqwZ">(<a href="#ref-Xi9zDqwZ" role="doc-biblioref">1787</a>)</span>.
However, a smaller retrospective study at Hoboken University Medical Center New Jersey failed to find an association between zinc supplementation and survival of hospitalized patients <span class="citation" data-cites="110pLAjkl">(<a href="#ref-110pLAjkl" role="doc-biblioref">1788</a>)</span>.
Therefore, whether zinc contributes to COVID-19 recovery remains unclear.
Other trials are now investigating zinc in conjunction with other supplements such as vitamin C or n-3 PUFA <span class="citation" data-cites="1GGbb9IZt 1AGVAbQuf">(<a href="#ref-1AGVAbQuf" role="doc-biblioref">1767</a>, <a href="#ref-1GGbb9IZt" role="doc-biblioref">1789</a>)</span>.
Though there is, overall, encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19; thus, the clinical trials that are currently underway will provide vital information on the efficacious use of zinc in COVID-19 prevention and/or treatment.
However, given the limited risk and the potential association between zinc deficiency and illness, maintaining a healthy diet to ensure an adequate zinc status may be advisable for individuals seeking to reduce their likelihood of infection.</p>
<h2 data-number="10.6" id="vitamin-c"><span class="header-section-number">10.6</span> Vitamin C</h2>
<p>Vitamins B, C, D, and E have also been suggested as potential nutrient supplement interventions for COVID-19 <span class="citation" data-cites="2hMFPQJR 3RCeQRRc">(<a href="#ref-2hMFPQJR" role="doc-biblioref">1727</a>, <a href="#ref-3RCeQRRc" role="doc-biblioref">1790</a>)</span>.
In particular vitamin C has been proposed as a potential therapeutic agent against COVID-19 due to its long history of use against the common cold and other respiratory infections <span class="citation" data-cites="190FMJu3y EbAGaMxL">(<a href="#ref-190FMJu3y" role="doc-biblioref">1791</a>, <a href="#ref-EbAGaMxL" role="doc-biblioref">1792</a>)</span>.
Vitamin C can be obtained via dietary sources such as fruits and vegetables or via supplementation.
Vitamin C plays a significant role in promoting immune function due to its effects on various immune cells.
It affects inflammation by modulating cytokine production, decreasing histamine levels, enhancing the differentiation and proliferation of T- and B-lymphocytes, increasing antibody levels, and protecting against the negative effects of reactive oxygen species, among other effects related to COVID-19 pathology <span class="citation" data-cites="9czQt5cR RjFxOURM 13M3VAfY9">(<a href="#ref-9czQt5cR" role="doc-biblioref">1793</a>–<a href="#ref-13M3VAfY9" role="doc-biblioref">1795</a>)</span>.
Vitamin C is utilized by the body during viral infections, as evinced by lower concentrations in leukocytes and lower concentrations of urinary vitamin C.
Post-infection, these levels return to baseline ranges <span class="citation" data-cites="uQFCDDIy 10nBT1HbX 1Bj8VMlSk aEEZ8Pq9 gZRPoTM9">(<a href="#ref-uQFCDDIy" role="doc-biblioref">1796</a>–<a href="#ref-gZRPoTM9" role="doc-biblioref">1800</a>)</span>.
It has been shown that as little as 0.1 g/d of vitamin C can maintain normal plasma levels of vitamin C in healthy individuals, but higher doses of at least 1-3 g/d are required for critically ill patients in ICUs <span class="citation" data-cites="DPhRTQw0">(<a href="#ref-DPhRTQw0" role="doc-biblioref">1801</a>)</span>.
Indeed, vitamin C deficiency appears to be common among COVID-19 patients <span class="citation" data-cites="WEWGGYKl Zrm5dyM1">(<a href="#ref-WEWGGYKl" role="doc-biblioref">1802</a>, <a href="#ref-Zrm5dyM1" role="doc-biblioref">1803</a>)</span>.
COVID-19 is also associated with the formation of microthrombi and coagulopathy <span class="citation" data-cites="17y6YeJ6R">(<a href="#ref-17y6YeJ6R" role="doc-biblioref">107</a>)</span> that contribute to its characteristic lung pathology <span class="citation" data-cites="19taZjxIT">(<a href="#ref-19taZjxIT" role="doc-biblioref">1804</a>)</span>, but these symptoms can be ameliorated by early infusions of vitamin C to inhibit endothelial surface P-selectin expression and platelet-endothelial adhesion <span class="citation" data-cites="WKBgtN6W">(<a href="#ref-WKBgtN6W" role="doc-biblioref">1805</a>)</span>.
Intravenous vitamin C also reduced D-dimer levels in a case study of 17 COVID-19 patients <span class="citation" data-cites="d9353XOW">(<a href="#ref-d9353XOW" role="doc-biblioref">1806</a>)</span>.
D-dimer levels are an important indicator of thrombus formation and breakdown and are notably elevated in COVID-19 patients <span class="citation" data-cites="18AiyvhO8 v1EIzwfx">(<a href="#ref-v1EIzwfx" role="doc-biblioref">103</a>, <a href="#ref-18AiyvhO8" role="doc-biblioref">104</a>)</span>.
There is therefore preliminary evidence suggesting that vitamin C status and vitamin C administration may be relevant to COVID-19 outcomes.</p>
<p>Larger-scale studies of vitamin C, however, have provided mixed results.
A recent meta-analysis found consistent support for regular vitamin C supplementation reducing the duration of the common cold, but that supplementation with vitamin C (&gt; 200 mg) failed to reduce the incidence of colds <span class="citation" data-cites="Pb5PCH7y">(<a href="#ref-Pb5PCH7y" role="doc-biblioref">1807</a>)</span>.
Individual studies have found Vitamin C to reduce the susceptibility of patients to lower respiratory tract infections, such as pneumonia <span class="citation" data-cites="JARvpqRw">(<a href="#ref-JARvpqRw" role="doc-biblioref">1808</a>)</span>.
Another meta-analysis demonstrated that in twelve trials, vitamin C supplementation reduced the length of stay of patients in intensive care units (ICUs) by 7.8% (95% CI: 4.2% to 11.2%; <em>p</em> = 0.00003).
Furthermore, high doses (1-3 g/day) significantly reduced the length of an ICU stay by 8.6% in six trials (<em>p</em> = 0.003).
Vitamin C also shortened the duration of mechanical ventilation by 18.2% in three trials in which patients required intervention for over 24 hours (95% CI 7.7% to 27%; p = 0.001) <span class="citation" data-cites="DPhRTQw0">(<a href="#ref-DPhRTQw0" role="doc-biblioref">1801</a>)</span>.
Despite these findings, an RCT of 167 patients known as CITRUS ALI failed to show a benefit of a 96-hour infusion of vitamin C to treat ARDS <span class="citation" data-cites="Rt5Aik4p">(<a href="#ref-Rt5Aik4p" role="doc-biblioref">1809</a>)</span>.
Clinical trials specifically investigating vitamin C in the context of COVID-19 have now begun, as highlighted by Carr et al. <span class="citation" data-cites="EbAGaMxL">(<a href="#ref-EbAGaMxL" role="doc-biblioref">1792</a>)</span>.
These trials intend to investigate the use of intravenous vitamin C in hospitalized COVID-19 patients.
The first trial to report initial results took place in Wuhan, China <span class="citation" data-cites="nwa7WdNM">(<a href="#ref-nwa7WdNM" role="doc-biblioref">1810</a>)</span>.
These initial results indicated that the administration of 12 g/12 hr of intravenous vitamin C for 7 days in 56 critically ill COVID-19 patients resulted in a promising reduction of 28-day mortality (<em>p</em> = 0.06) in univariate survival analysis <span class="citation" data-cites="ZX7LDDp0">(<a href="#ref-ZX7LDDp0" role="doc-biblioref">1811</a>)</span>.
Indeed, the same study reported a significant decrease in IL-6 levels by day 7 of vitamin C infusion (<em>p</em> = 0.04) <span class="citation" data-cites="RM8v7Y7H">(<a href="#ref-RM8v7Y7H" role="doc-biblioref">1812</a>)</span>.
Additional studies that are being conducted in Canada, China, Iran, and the USA will provide additional insight into whether vitamin C supplementation affects COVID-19 outcomes on a larger scale.</p>
<p>Even though evidence supporting the use of vitamin C is beginning to emerge, we will not know how effective vitamin C is as a therapeutic for quite some time.
Currently (as of January 2021) over fifteen trials are registered with clinicaltrials.gov that are either recruiting, active or are currently in preparation.
When completed, these trials will provide crucial evidence on the efficacy of vitamin C as a therapeutic for COVID-19 infection.
However, the majority of supplementation studies investigate the intravenous infusion of vitamin C in severe patients.
Therefore, there is a lack of studies investigating the potential prophylactic administration of vitamin C via oral supplementation for healthy individuals or potentially asymptomatic SARS-CoV-2 positive patients.
Once again, vitamin C intake is part of a healthy diet and the vitamin likely presents minimal risk, but its potential prophylactic or therapeutic effects against COVID-19 are yet to be determined.
To maintain vitamin C status, it would be prudent for individuals to ensure that they consume the recommended dietary allowance of vitamin C to maintain a healthy immune system <span class="citation" data-cites="Lyd8s4Mk">(<a href="#ref-Lyd8s4Mk" role="doc-biblioref">1696</a>)</span>.
The recommended dietary allowance according to the FDA is 75-90 mg/d, whereas EFSA recommends 110 mg/d <span class="citation" data-cites="hhf96MB1">(<a href="#ref-hhf96MB1" role="doc-biblioref">1813</a>)</span>.</p>
<h2 data-number="10.7" id="vitamin-d"><span class="header-section-number">10.7</span> Vitamin D</h2>
<p>Of all of the supplements currently under investigation, vitamin D has become a leading prophylactic and therapeutic candidate against SARS-CoV-2.
Vitamin D can modulate both the adaptive and innate immune system and is associated with various aspects of immune health and antiviral defense <span class="citation" data-cites="3Nyo3kmR SKP7JCfL 15xVUSQRH 15ESVyKi0 npPvtBLw">(<a href="#ref-3Nyo3kmR" role="doc-biblioref">1814</a>–<a href="#ref-npPvtBLw" role="doc-biblioref">1818</a>)</span>.
Vitamin D can be sourced through diet or supplementation, but it is mainly biosynthesized by the body on exposure to ultraviolet light (UVB) from sunlight.
Vitamin D deficiency is associated with an increased susceptibility to infection <span class="citation" data-cites="n3v9Thlz">(<a href="#ref-n3v9Thlz" role="doc-biblioref">1819</a>)</span>.
In particular, vitamin D deficient patients are at risk of developing acute respiratory infections <span class="citation" data-cites="iWLsflrG">(<a href="#ref-iWLsflrG" role="doc-biblioref">1820</a>)</span> and ARDS <span class="citation" data-cites="iWLsflrG">(<a href="#ref-iWLsflrG" role="doc-biblioref">1820</a>)</span>.
1,25-dihydroxyvitamin D3 is the active form of vitamin D that is involved in adaptive and innate responses; however, due to its low concentration and a short half life of a few hours, vitamin D levels are typically measured by the longer lasting and more abundant precursor 25-hydroxyvitamin D.
The vitamin D receptor is expressed in various immune cells, and vitamin D is an immunomodulator of antigen presenting cells, dendritic cells, macrophages, monocytes, and T- and B-lymphocytes <span class="citation" data-cites="n3v9Thlz nwEQBjV4">(<a href="#ref-n3v9Thlz" role="doc-biblioref">1819</a>, <a href="#ref-nwEQBjV4" role="doc-biblioref">1821</a>)</span>.
Due to its potential immunomodulating properties, vitamin D supplementation may be advantageous to maintain a healthy immune system.</p>
<p>Early in the pandemic it was postulated that an individual’s vitamin D status could significantly affect their risk of developing COVID-19 <span class="citation" data-cites="9ac0GYir">(<a href="#ref-9ac0GYir" role="doc-biblioref">1822</a>)</span>.
This hypothesis was derived from the fact that the current pandemic emerged in Wuhan China during winter, when 25-hydroxyvitamin D concentrations are at their lowest due to a lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer and higher 25-hydroxyvitamin D concentrations would be higher, the number of cases was low.
This led researchers to question whether there was a seasonal component to the SARS-CoV-2 pandemic and whether vitamin D levels might play a role <span class="citation" data-cites="9ac0GYir LpqYmd4C 1EEHgnhPC CS8krfan">(<a href="#ref-9ac0GYir" role="doc-biblioref">1822</a>–<a href="#ref-CS8krfan" role="doc-biblioref">1825</a>)</span>.
Though it is assumed that COVID-19 is seasonal, multiple other factors that can affect vitamin D levels should also be considered.
These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others.
Indeed, it has been estimated that each degree of latitude north of 28 degrees corresponded to a 4.4% increase of COVID-19 mortality, indirectly linking a persons vitamin D levels via exposure to UVB light to COVID-19 mortality <span class="citation" data-cites="LpqYmd4C">(<a href="#ref-LpqYmd4C" role="doc-biblioref">1823</a>)</span>.</p>
<p>As the pandemic has evolved, additional research of varying quality has investigated some of the potential links identified early in the pandemic <span class="citation" data-cites="9ac0GYir">(<a href="#ref-9ac0GYir" role="doc-biblioref">1822</a>)</span> between vitamin D and COVID-19.
Indeed, studies are beginning to investigate whether there is any prophylactic and/or therapeutic relationship between vitamin D and COVID-19.
A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than those of SARS-CoV-2 negative patients (24.6 ng/ml; <em>p</em> = 0.004), an association that held when stratifying patients greater than 70 years old <span class="citation" data-cites="1Gy7Svl72">(<a href="#ref-1Gy7Svl72" role="doc-biblioref">1826</a>)</span>.
These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and CT scans of the lungs were obtained upon hospitalization <span class="citation" data-cites="GMAEaWD1">(<a href="#ref-GMAEaWD1" role="doc-biblioref">1827</a>)</span>.
A significant difference in 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched hospitalized controls.
It is not clear from the study which diseases caused the control subjects to be admitted at the time of their 25-hydroxyvitamin D measurement, which makes it difficult to assess the observations reported.
Both female and male patients possessed lower median 25-hydroxyvitamin D concentrations than the control group as a whole (18.6 ng/ml versus 21.5 ng/ml; <em>p</em> = 0.0016) and a higher rate of vitamin D deficiency (58.6% versus 42.5%).
However, when comparisons were stratified by sex, evidence of sexual dimorphism became apparent, as female patients had equivalent levels of 25-hydroxyvitamin D to females in the control group, whereas male patients were deficient in 25-hydroxyvitamin D relative to male controls (67% versus 49%; <em>p</em> = 0.0006).
Notably, vitamin D deficiency was progressively lower in males with advancing radiological disease stages (<em>p</em> = 0.001).
However, these studies are supported by several others that indicate that vitamin D status may be an independent risk factor for the severity of COVID-19 <span class="citation" data-cites="GM9W6HPG kN3eqnZW 5MvUc8IL 18O6t6Jo">(<a href="#ref-GM9W6HPG" role="doc-biblioref">1828</a>–<a href="#ref-18O6t6Jo" role="doc-biblioref">1831</a>)</span> and in COVID-19 patients relative to population-based controls <span class="citation" data-cites="8ZxZKpYS">(<a href="#ref-8ZxZKpYS" role="doc-biblioref">1832</a>)</span>.
Indeed, serum concentrations of 25-hydroxyvitamin D above 30 ng/ml, which indicate vitamin D sufficiency, seems to be associated with a reduction in serum C-reactive protein, an inflammatory marker, along with increased lymphocyte levels, which suggests that vitamin D levels may modulate the immune response by reducing risk for cytokine storm in response to SARS-CoV-2 infection <span class="citation" data-cites="8ZxZKpYS">(<a href="#ref-8ZxZKpYS" role="doc-biblioref">1832</a>)</span>.
A study in India determined that COVID-19 fatality was higher in patients with severe COVID-19 and low serum 25-hydroxyvitamin D (mean level 6.2 ng/ml; 97% vitamin D deficient) levels versus asymptomatic non-severe patients with higher levels of vitamin D (mean level 27.9 ng/ml; 33% vitamin D deficient) <span class="citation" data-cites="QAEcHBzN">(<a href="#ref-QAEcHBzN" role="doc-biblioref">1833</a>)</span>.
In the same study, vitamin D deficiency was associated with higher levels of inflammatory markers including IL-6, ferritin, and tumor necrosis factor α.
Collectively, these studies add to a multitude of observational studies reporting potential associations between low levels of 25-hydroxyvitamin D and COVID-19 incidence and severity <span class="citation" data-cites="1Gy7Svl72 EhakYR7D xv6TOhJg 8ZxZKpYS ZheSCcim pNuUm1NE 5aqg2qxi 18O6t6Jo 10kobTskt Tcc0pGgI">(<a href="#ref-1Gy7Svl72" role="doc-biblioref">1826</a>, <a href="#ref-18O6t6Jo" role="doc-biblioref">1831</a>, <a href="#ref-8ZxZKpYS" role="doc-biblioref">1832</a>, <a href="#ref-EhakYR7D" role="doc-biblioref">1834</a>–<a href="#ref-Tcc0pGgI" role="doc-biblioref">1840</a>)</span>.</p>
<p>Despite the large number of studies establishing a link between vitamin D status and COVID-19 severity, an examination of data from the UK Biobank did not support this thesis <span class="citation" data-cites="FZ8yXmxQ 7goCwNjQ">(<a href="#ref-FZ8yXmxQ" role="doc-biblioref">1841</a>, <a href="#ref-7goCwNjQ" role="doc-biblioref">1842</a>)</span>.
These analyses examined 25-hydroxyvitamin D concentrations alongside SARS-CoV-2 positivity and COVID-19 mortality in over 340,000 UK Biobank participants.
However, these studies have caused considerable debate that will likely be settled following further studies <span class="citation" data-cites="EtCW6lkq ClClnlJb">(<a href="#ref-EtCW6lkq" role="doc-biblioref">1843</a>, <a href="#ref-ClClnlJb" role="doc-biblioref">1844</a>)</span>.
Overall, while the evidence suggests that there is likely an association between low serum 25-hydroxyvitamin D and COVID-19 incidence, these studies must be interpreted with caution, as there is the potential for reverse causality, bias, and other confounding factors including that vitamin D deficiency is also associated with numerous pre-existing conditions and risk factors that can increase the risk for severe COVID-19 <span class="citation" data-cites="Lyd8s4Mk 1F2cyuTqD LpqYmd4C F4hC9qly">(<a href="#ref-Lyd8s4Mk" role="doc-biblioref">1696</a>, <a href="#ref-LpqYmd4C" role="doc-biblioref">1823</a>, <a href="#ref-1F2cyuTqD" role="doc-biblioref">1845</a>, <a href="#ref-F4hC9qly" role="doc-biblioref">1846</a>)</span>.</p>
<p>While these studies inform us of the potential importance of vitamin D sufficiency and the risk of SARS-CoV-2 infection and severe COVID-19, they fail to conclusively determine whether vitamin D supplementation can therapeutically affect the clinical course of COVID-19.
In one study, 40 vitamin D deficient asymptomatic or mildly symptomatic participants patients were either randomized to receive 60,000 IU of cholecalciferol daily for at least 7 days (n = 16) or a placebo (n = 24) with a target serum 25-hydroxyvitamin D level &gt;50 ng/ml.
At day 7, 10 patients achieved &gt;50 ng/ml, followed by another 2 by day 14.
By the end of the study, the treatment group had a greater proportion of vitamin D-deficient participants that tested negative for SARS-CoV-2 RNA, and they had a significantly lower fibrinogen levels, potentially indicating a beneficial effect <span class="citation" data-cites="kurM6Av7">(<a href="#ref-kurM6Av7" role="doc-biblioref">1847</a>)</span>.
A pilot study in Spain determined that early administration of high dose calcifediol (~21,000 IU days 1-2 and ~11,000 IU days 3-7 of hospital admission) with hydroxychloroquine and azithromycin to 50 hospitalized COVID-19 patients significantly reduced ICU admissions and may have reduced disease severity versus hydroxychloroquine and azithromycin alone <span class="citation" data-cites="rYXmRFCk">(<a href="#ref-rYXmRFCk" role="doc-biblioref">1848</a>)</span>.
Although this study received significant criticism from the National Institute for Health and Care Excellence (NICE) in the UK <span class="citation" data-cites="OAzFPABw">(<a href="#ref-OAzFPABw" role="doc-biblioref">1849</a>)</span>, an independent follow-up statistical analysis supported the findings of the study with respect to the results of cholecalciferol treatment <span class="citation" data-cites="1FHsBdDVU">(<a href="#ref-1FHsBdDVU" role="doc-biblioref">1850</a>)</span>.
Another trial of 986 patients hospitalized for COVID-19 in three UK hospitals administered cholecalciferol (≥ 280,000 IU in a time period of 7 weeks) to 151 patients and found an association with a reduced risk of COVID-19 mortality, regardless of baseline 25-hydroxyvitamin D levels <span class="citation" data-cites="UlHOCdfk">(<a href="#ref-UlHOCdfk" role="doc-biblioref">1851</a>)</span>.
However, a double-blind, randomized, placebo-controlled trial of 240 hospitalized COVID-19 patients in São Paulo, Brazil administered a single 200,000 IU oral dose of vitamin D.
At the end of the study, there was a 24 ng/mL difference of 25-hydroxyvitamin D levels in the treatment group versus the placebo group (<em>p</em> = 0.001), and 87% of the treatment group were vitamin D sufficient versus ~11% in the placebo group.
Supplementation was well tolerated.
However, there was no reduction in the length of hospital stay or mortality, and no change to any other relevant secondary outcomes were reported <span class="citation" data-cites="1GjnSzUKx">(<a href="#ref-1GjnSzUKx" role="doc-biblioref">1852</a>)</span>.
These early findings are thus still inconclusive with regards to the therapeutic value of vitamin D supplementation.
However, other trials are underway, including one trial that is investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic COVID-19 patients and whether vitamin D prevents patient deterioration <span class="citation" data-cites="162t7zdGv">(<a href="#ref-162t7zdGv" role="doc-biblioref">1853</a>)</span>.
Other trials are examining various factors including mortality, symptom recovery, severity of disease, rates of ventilation, inflammatory markers such as C-reactive protein and IL-6, blood cell counts, and the prophylactic capacity of vitamin D administration <span class="citation" data-cites="2K0jV5Bn 162t7zdGv rBiojRHk 6kLoX5qT">(<a href="#ref-162t7zdGv" role="doc-biblioref">1853</a>–<a href="#ref-6kLoX5qT" role="doc-biblioref">1856</a>)</span>.
Concomitant administration of vitamin D with pharmaceuticals such as aspirin <span class="citation" data-cites="32C2erIe">(<a href="#ref-32C2erIe" role="doc-biblioref">1857</a>)</span> and bioactive molecules such as resveratrol <span class="citation" data-cites="tLVliODQ">(<a href="#ref-tLVliODQ" role="doc-biblioref">1858</a>)</span> is also under investigation.</p>
<p>The effectiveness of vitamin D supplementation against COVID-19 remains open for debate.
All the same, there is no doubt that vitamin D deficiency is a widespread issue and should be addressed not only because of its potential link to SARS-CoV-2 incidence <span class="citation" data-cites="Btt5H6f1">(<a href="#ref-Btt5H6f1" role="doc-biblioref">1859</a>)</span>, but also due to its importance for overall health.
There is a possibility that safe exposure to sunlight could improve endogenous synthesis of vitamin D, potentially strengthening the immune system.
However, sun exposure is not sufficient on its own, particularly in the winter months.
Indeed, while the possible link between vitamin D status and COVID-19 is further investigated, preemptive supplementation of vitamin D and encouraging people to maintain a healthy diet for optimum vitamin D status is likely to raise serum levels of 25-hydroxyvitamin D while being unlikely to carry major health risks.
These principles seem to be the basis of a number of guidelines issued by some countries and scientific organizations that have advised supplementation of vitamin D during the pandemic.
The Académie Nationale de Médecine in France recommends rapid testing of 25-hydroxyvitamin D for people over 60 years old to identify those most at risk of vitamin D deficiency and advises them to obtain a bolus dose of 50,000 to 100,000 IU vitamin D to limit respiratory complications.
It has also recommended that those under 60 years old should take 800 to 1,000 IU daily if they receive a SARS-CoV-2 positive test <span class="citation" data-cites="llQ4v7QI">(<a href="#ref-llQ4v7QI" role="doc-biblioref">1860</a>/)</span>.
In Slovenia, doctors have been advised to provide nursing home patients with vitamin D <span class="citation" data-cites="M4v2N83p">(<a href="#ref-M4v2N83p" role="doc-biblioref">1861</a>)</span>.
Both Public Health England and Public Health Scotland have advised members of the Black, Asian, and minority ethnic communities to supplement for vitamin D in light of evidence that they may be at higher risk for vitamin D deficiency along with other COVID-19 risk factors, a trend that has also been observed in the United States <span class="citation" data-cites="yoAwg53u StktHVeV">(<a href="#ref-yoAwg53u" role="doc-biblioref">1862</a>, <a href="#ref-StktHVeV" role="doc-biblioref">1863</a>)</span>.
However, other UK scientific bodies including the NICE recommend that individuals supplement for vitamin D as per usual UK government advice but warn that people should not supplement for vitamin D solely to prevent COVID-19.
All the same, the NICE has provided guidelines for research to investigate the supplementation of vitamin D in the context of COVID-19 <span class="citation" data-cites="4DE4ljR9">(<a href="#ref-4DE4ljR9" role="doc-biblioref">1864</a>)</span>.
Despite vitamin D deficiency being a widespread issue in the United States <span class="citation" data-cites="lhYw2CB">(<a href="#ref-lhYw2CB" role="doc-biblioref">1865</a>)</span>, the National Institutes of Health have stated that there is “insufficient data to recommend either for or against the use of vitamin D for the prevention or treatment of COVID-19” <span class="citation" data-cites="KNwYH6Y4">(<a href="#ref-KNwYH6Y4" role="doc-biblioref">1866</a>/)</span>.
These are just some examples of how public health guidance has responded to the emerging evidence regarding vitamin D and COVID-19.
Outside of official recommendations, there is also evidence that individuals may be paying increased attention to their vitamin D levels, as a survey of Polish consumers showed that 56% of respondents used vitamin D during the pandemic <span class="citation" data-cites="19yy9SHf1">(<a href="#ref-19yy9SHf1" role="doc-biblioref">1867</a>)</span>.
However, some companies have used the emerging evidence surrounding vitamin D to sell products that claim to prevent and treat COVID-19, which in one incident required a federal court to intervene and issue an injunction barring the sale of vitamin-D-related products due to the lack of clinical data supporting these claims <span class="citation" data-cites="xkewCiC9">(<a href="#ref-xkewCiC9" role="doc-biblioref">1868</a>)</span>.
It is clear that further studies and clinical trials are required to conclusively determine the prophylactic and therapeutic potential of vitamin D supplementation against COVID-19.
Until such time that sufficient evidence emerges, individuals should follow their national guidelines surrounding vitamin D intake to achieve vitamin D sufficiency.</p>
<h2 data-number="10.8" id="probiotics"><span class="header-section-number">10.8</span> Probiotics</h2>
<p>Probiotics are “live microorganisms that, when administered in adequate amounts, confer a health benefit on the host” <span class="citation" data-cites="1FyHVZ9yx">(<a href="#ref-1FyHVZ9yx" role="doc-biblioref">1869</a>)</span>.
Some studies suggest that probiotics are beneficial against common viral infections, and there is modest evidence to suggest that they can modulate the immune response <span class="citation" data-cites="pvZU55k9 HKeroTXj">(<a href="#ref-pvZU55k9" role="doc-biblioref">1870</a>, <a href="#ref-HKeroTXj" role="doc-biblioref">1871</a>)</span>.
As a result, it has been hypothesized that probiotics may have therapeutic value worthy of investigation against SARS-CoV-2 <span class="citation" data-cites="10e8cHDs8">(<a href="#ref-10e8cHDs8" role="doc-biblioref">1872</a>)</span>.
Probiotics and next-generation probiotics, which are more akin to pharmacological-grade supplements, have been associated with multiple potential beneficial effects for allergies, digestive tract disorders, and even metabolic diseases through their anti-inflammatory and immunomodulatory effects <span class="citation" data-cites="5fU7dexB 1GCrGCp5P">(<a href="#ref-5fU7dexB" role="doc-biblioref">1873</a>, <a href="#ref-1GCrGCp5P" role="doc-biblioref">1874</a>)</span>.
However, the mechanisms by which probiotics affect these various conditions would likely differ among strains, with the ultimate effect of the probiotic depending on the heterogeneous set of bacteria present <span class="citation" data-cites="1GCrGCp5P">(<a href="#ref-1GCrGCp5P" role="doc-biblioref">1874</a>)</span>.
Some of the beneficial effects of probiotics include reducing inflammation by promoting the expression of anti-inflammatory mediators, inhibiting Toll-like receptors 2 and 4, competing directly with pathogens, synthesizing antimicrobial substances or other metabolites, improving intestinal barrier function, and/or favorably altering the gut microbiota and the brain-gut axis <span class="citation" data-cites="B2GeSSQo 1GCrGCp5P nmXF8vkS">(<a href="#ref-1GCrGCp5P" role="doc-biblioref">1874</a>–<a href="#ref-nmXF8vkS" role="doc-biblioref">1876</a>)</span>.
It is also thought that lactobacilli such as <em>Lactobacillus paracasei, Lactobacillus plantarum</em> and <em>Lactobacillus rhamnosus</em> have the capacity to bind to and inactivate some viruses via adsorptive and/or trapping mechanisms <span class="citation" data-cites="sX7GyWpV">(<a href="#ref-sX7GyWpV" role="doc-biblioref">1877</a>)</span>.
Other probiotic lactobacilli and even non-viable bacterium-like particles have been shown to reduce both viral attachment to host cells and viral titers, along with reducing cytokine synthesis, enhancing the antiviral IFN-α response, and inducing various other antiviral mechanisms <span class="citation" data-cites="IkuChxSp 8gQnAy2v zGORHhOM sX7GyWpV g0qfbA7l DNbZomOZ p9eKMfO2 x46F06Fr uU3GXiaf">(<a href="#ref-sX7GyWpV" role="doc-biblioref">1877</a>–<a href="#ref-uU3GXiaf" role="doc-biblioref">1885</a>)</span>.
These antiviral and immunobiotic mechanisms and others have been reviewed in detail elsewhere <span class="citation" data-cites="EEa0SZCF Sp05opqA 10e8cHDs8">(<a href="#ref-EEa0SZCF" role="doc-biblioref">1726</a>, <a href="#ref-10e8cHDs8" role="doc-biblioref">1872</a>, <a href="#ref-Sp05opqA" role="doc-biblioref">1886</a>)</span>.
However, there is also a bi-directional relationship between the lungs and gut microbiota known as the gut-lung axis <span class="citation" data-cites="13AWoYvqB">(<a href="#ref-13AWoYvqB" role="doc-biblioref">1887</a>)</span>, whereby gut microbial metabolites and endotoxins may affect the lungs via the circulatory system and the lung microbiota in return may affect the gut <span class="citation" data-cites="Smqu1EYN">(<a href="#ref-Smqu1EYN" role="doc-biblioref">1888</a>)</span>.
Therefore, the gut-lung axis may play role in our future understanding of COVID-19 pathogenesis and become a target for probiotic treatments <span class="citation" data-cites="d27mQ4kO">(<a href="#ref-d27mQ4kO" role="doc-biblioref">1889</a>)</span>.
Moreover, as microbial dysbiosis of the respiratory tract and gut may play a role in some viral infections, it has been suggested that SARS-CoV-2 may interact with our commensal microbiota [<span class="citation" data-cites="EEa0SZCF">(<a href="#ref-EEa0SZCF" role="doc-biblioref">1726</a>)</span>; <span class="citation" data-cites="HOd4nJPq">(<a href="#ref-HOd4nJPq" role="doc-biblioref">1890</a>)</span>; 10.3389/fmicb.2020.01840] and that the lung microbiome could play a role in developing immunity to viral infections <span class="citation" data-cites="OwqHmSrX">(<a href="#ref-OwqHmSrX" role="doc-biblioref">1891</a>)</span>.
These postulations, if correct, could lead to the development of novel probiotic and prebiotic treatments.
However, significant research is required to confirm these associations and their relevance to patient care, if any.</p>
<p>Probiotic therapies and prophylactics may also confer some advantages for managing symptoms of COVID-19 or risks associated with its treatment.
Probiotics have tentatively been associated with the reduction of risk and duration of viral upper respiratory tract infections <span class="citation" data-cites="1MIGrWIP 58eXvYEV kjavvCzp">(<a href="#ref-1MIGrWIP" role="doc-biblioref">1892</a>–<a href="#ref-kjavvCzp" role="doc-biblioref">1894</a>)</span>.
Some meta-analyses that have assessed the efficacy of probiotics in viral respiratory infections have reported moderate reductions in the incidence and duration of infection <span class="citation" data-cites="58eXvYEV dJIGDt4k">(<a href="#ref-58eXvYEV" role="doc-biblioref">1893</a>, <a href="#ref-dJIGDt4k" role="doc-biblioref">1895</a>)</span>.
Indeed, randomized controlled trials have shown that administering <em>Bacillus subtilis</em> and <em>Enterococcus faecalis</em> <span class="citation" data-cites="AJav6c61">(<a href="#ref-AJav6c61" role="doc-biblioref">1896</a>)</span>, <em>Lactobacillus rhamnosus GG</em> <span class="citation" data-cites="1E2aDJp4T">(<a href="#ref-1E2aDJp4T" role="doc-biblioref">1897</a>)</span>, or <em>Lactobacillus casei</em> and <em>Bifidobacterium breve</em> with galactooligosaccharides <span class="citation" data-cites="zccFgajh">(<a href="#ref-zccFgajh" role="doc-biblioref">1898</a>)</span> via the nasogastric tube to ventilated patients reduced the occurrence of ventilator-associated pneumonia in comparison to the respective control groups in studies of viral infections and sepsis.
These findings were also supported by a recent meta-analysis <span class="citation" data-cites="1DI06gFPc">(<a href="#ref-1DI06gFPc" role="doc-biblioref">1899</a>)</span>.
Additionally, COVID-19 patients carry a significant risk of ventilator-associated bacterial pneumonia <span class="citation" data-cites="164mVBFZx">(<a href="#ref-164mVBFZx" role="doc-biblioref">1900</a>)</span>, but it can be challenging for clinicians to diagnose this infection due to the fact that severe COVID-19 infection presents with the symptoms of pneumonia <span class="citation" data-cites="w2VFo0h1">(<a href="#ref-w2VFo0h1" role="doc-biblioref">1901</a>)</span>.
Therefore, an effective prophylactic therapy for ventilator-associated pneumonia in severe COVID-19 patients would carry significant therapeutic value.
Additionally, in recent years, probiotics have become almost synonymous with the treatment of gastrointestinal issues due to their supposed anti-inflammatory and immunomodulatory effects <span class="citation" data-cites="gcyQ73v7">(<a href="#ref-gcyQ73v7" role="doc-biblioref">1902</a>)</span>.
Notably, gastrointestinal symptoms commonly occur in COVID-19 patients <span class="citation" data-cites="67ihxDiV">(<a href="#ref-67ihxDiV" role="doc-biblioref">1903</a>)</span>, and angiotensin-converting enzyme 2, the portal by which SARS-CoV-2 enters human cells, is highly expressed in enterocytes of the ileum and colon, suggesting that these organs may be a potential route of infection <span class="citation" data-cites="8Hvgbl3n Nhxoefru">(<a href="#ref-8Hvgbl3n" role="doc-biblioref">1904</a>, <a href="#ref-Nhxoefru" role="doc-biblioref">1905</a>)</span>.
Indeed, SARS-CoV-2 viral RNA has been detected in human feces <span class="citation" data-cites="TPWaxApy PmE9xedP">(<a href="#ref-PmE9xedP" role="doc-biblioref">79</a>, <a href="#ref-TPWaxApy" role="doc-biblioref">540</a>)</span>, and fecal-oral transmission of the virus has not yet been ruled out <span class="citation" data-cites="r9HNMqTS">(<a href="#ref-r9HNMqTS" role="doc-biblioref">1906</a>)</span>.
Rectal swabs of some SARS-CoV-2 positive pediatric patients persistently tested positive for several days despite negative nasopharyngeal tests, indicating the potential for fecal viral shedding <span class="citation" data-cites="eOcc1BUx">(<a href="#ref-eOcc1BUx" role="doc-biblioref">1907</a>)</span>.
However, there is conflicting evidence for the therapeutic value of various probiotics against the incidence or severity of gastrointestinal symptoms in viral or bacterial infections such as gastroenteritis <span class="citation" data-cites="VwStn9Ps UtAKolza">(<a href="#ref-VwStn9Ps" role="doc-biblioref">1908</a>, <a href="#ref-UtAKolza" role="doc-biblioref">1909</a>)</span>.
Nevertheless, it has been proposed that the administration of probiotics to COVID-19 patients and healthcare workers may prevent or ameliorate the gastrointestinal symptoms of COVID-19, a hypothesis that several clinical trials are now preparing to investigate <span class="citation" data-cites="1AqF1ESKq 9UYsmETk">(<a href="#ref-1AqF1ESKq" role="doc-biblioref">1910</a>, <a href="#ref-9UYsmETk" role="doc-biblioref">1911</a>)</span>.
Other studies are investigating whether probiotics may affect patient outcomes following SARS-CoV-2 infection <span class="citation" data-cites="15pDlYtom">(<a href="#ref-15pDlYtom" role="doc-biblioref">1912</a>)</span>.</p>
<p>Generally, the efficacy of probiotic use is a controversial topic among scientists.
In Europe, EFSA has banned the term probiotics on products labels, which has elicited either criticism for EFSA or support for probiotics from researchers in the field <span class="citation" data-cites="m3SexO9F 5FTOk7ol 1FyHVZ9yx">(<a href="#ref-1FyHVZ9yx" role="doc-biblioref">1869</a>, <a href="#ref-m3SexO9F" role="doc-biblioref">1913</a>, <a href="#ref-5FTOk7ol" role="doc-biblioref">1914</a>)</span>.
This regulation is due to the hyperbolic claims placed on the labels of various probiotic products, which lack rigorous scientific data to support their efficacy.
Overall, the data supporting probiotics in the treatment or prevention of many different disorders and diseases is not conclusive, as the quality of the evidence is generally considered low <span class="citation" data-cites="1MIGrWIP">(<a href="#ref-1MIGrWIP" role="doc-biblioref">1892</a>)</span>.
However, in the case of probiotics and respiratory infections, the evidence seems to be supportive of their potential therapeutic value.
Consequently, several investigations are underway to investigate the prophylactic and therapeutic potential of probiotics for COVID-19.
The blind use of conventional probiotics for COVID-19 is currently cautioned against until the pathogenesis of SARS-CoV-2 can be further established <span class="citation" data-cites="kv94VRLF">(<a href="#ref-kv94VRLF" role="doc-biblioref">1915</a>)</span>.
Until clinical trials investigating the prophylactic and therapeutic potential of probiotics for COVID-19 are complete, it is not possible to provide an evidence-based recommendation for their use.
Despite these concerns, complementary use of probiotics as an adjuvant therapeutic has been proposed by the Chinese National Health Commission and National Administration of Traditional Chinese Medicine <span class="citation" data-cites="3Ak4Mata">(<a href="#ref-3Ak4Mata" role="doc-biblioref">80</a>)</span>.
While supply issues prevented the probiotics market from showing the same rapid response to the COVID-19 as some other supplements, many suppliers are reporting growth during the pandemic <span class="citation" data-cites="3Hg8dHMg">(<a href="#ref-3Hg8dHMg" role="doc-biblioref">1916</a>)</span>.
Therefore, the public response once again seems to have adopted supplements promoted as bolstering the immune response despite a lack of evidence suggesting they are beneficial for preventing or mitigating COVID-19.</p>
<h2 data-number="10.9" id="discussion"><span class="header-section-number">10.9</span> Discussion</h2>
<p>In this review, we report the findings to date of analyses of several dietary supplements and nutraceuticals.
While existing evidence suggests potential benefits of n-3 PUFA and probiotic supplementation for COVID-19 treatment and prophylaxis, clinical data is still lacking, although trials are underway.
Both zinc and vitamin C supplementation in hospitalized patients seem to be associated with positive outcomes; however, further clinical trials are required.
In any case, vitamin C and zinc intake are part of a healthy diet and likely present minimal risk when supplemented, though their potential prophylactic or therapeutic effects against COVID-19 are yet to be determined.
On the other hand, mounting evidence from observational studies indicates that there is an association between vitamin D deficiency and COVID-19 incidence has also been supported by meta-analysis <span class="citation" data-cites="sWzpGBil">(<a href="#ref-sWzpGBil" role="doc-biblioref">1917</a>)</span>.
Indeed, scientists are working to confirm these findings and to determine whether a patient’s serum 25-hydroxyvitamin D levels are also associated with COVID-19 severity.
Clinical trials are required to determine whether preemptive vitamin D supplementation may mitigate against severe COVID-19.
In terms of the therapeutic potential of vitamin D, initial evidence from clinical trials is conflicting but seems to indicate that vitamin D supplementation may reduce COVID-19 severity <span class="citation" data-cites="rYXmRFCk">(<a href="#ref-rYXmRFCk" role="doc-biblioref">1848</a>)</span>.
The various clinical trials currently underway will be imperative to provide information on the efficacious use of vitamin D supplementation for COVID-19 prevention and/or treatment.</p>
<p>The purported prophylactic and therapeutic benefits of dietary supplements and nutraceuticals for multiple disorders, diseases, and infections has been the subject of significant research and debate for the last few decades.
Inevitably, scientists are also investigating the potential for these various products to treat or prevent COVID-19.
This interest also extends to consumers, which led to a remarkable increase of sales of dietary supplements and nutraceuticals throughout the pandemic due to a desire to obtain additional protections from infection and disease.
The nutraceuticals discussed in this review, namely vitamin C, vitamin D, n-3 PUFA, zinc, and probiotics, were selected because of potential biological mechanisms that could beneficially affect viral and respiratory infections and because they are currently under clinical investigation.
Specifically, these compounds have all been found to influence cellular processes related to inflammation.
Inflammation is particularly relevant to COVID-19 because of the negative outcomes (often death) observed in a large number of patients whose immune response becomes hyperactive in response to SARS-CoV-2, leading to severe outcomes such as ARDS and sepsis <span class="citation" data-cites="nCVMwr28">(<a href="#ref-nCVMwr28" role="doc-biblioref">744</a>)</span>.
Additionally, there is a well-established link between diet and inflammation <span class="citation" data-cites="1HHbyV6d">(<a href="#ref-1HHbyV6d" role="doc-biblioref">1918</a>)</span>, potentially mediated in part by the microbiome <span class="citation" data-cites="ViPBMxtf">(<a href="#ref-ViPBMxtf" role="doc-biblioref">1919</a>)</span>.
Thus, the idea that dietary modifications or supplementation could be used to modify the inflammatory response is tied to a broader view of how diet and the immune system are interconnected.
The supplements and nutraceuticals discussed here therefore lie in sharp contrast to other alleged nutraceutical or dietary supplements that have attracted during the pandemic, such as colloidal silver <span class="citation" data-cites="wtYA1GbG">(<a href="#ref-wtYA1GbG" role="doc-biblioref">1920</a>)</span>, which have no known nutritional function and can be harmful.
Importantly, while little clinical evidence is available about the effects of any supplements against COVID-19, the risks associated with those discussed above are likely to be low, and in some cases, they can be obtained from dietary sources alone.</p>
<p>There are various other products and molecules that have garnered scientific interest and could merit further investigation.
These include polyphenols, lipid extracts, and tomato-based nutraceuticals, all of which have been suggested for the potential prevention of cardiovascular complications of COVID-19 such as thrombosis <span class="citation" data-cites="rWVL91jq EEa0SZCF">(<a href="#ref-EEa0SZCF" role="doc-biblioref">1726</a>, <a href="#ref-rWVL91jq" role="doc-biblioref">1761</a>)</span>.
Melatonin is another supplement that has been identified as a potential antiviral agent against SARS-CoV-2 using computational methods <span class="citation" data-cites="TDa7cEUV">(<a href="#ref-TDa7cEUV" role="doc-biblioref">1921</a>)</span>, and it has also been highlighted as a potential therapeutic agent for COVID-19 due to its documented antioxidant, anti-apoptotic, immunomodulatory, and anti-inflammatory effects <span class="citation" data-cites="rWVL91jq GJAYfMR0 2CXoItPz">(<a href="#ref-rWVL91jq" role="doc-biblioref">1761</a>, <a href="#ref-GJAYfMR0" role="doc-biblioref">1922</a>, <a href="#ref-2CXoItPz" role="doc-biblioref">1923</a>)</span>.
Notably, melatonin, vitamin D and zinc have attracted public attention because they were included in the treatment plan of the former President of the United States upon his hospitalization due to COVID-19 <span class="citation" data-cites="MNWnrcz6">(<a href="#ref-MNWnrcz6" role="doc-biblioref">1924</a>)</span>.
These are just some of the many substances and supplements that are currently under investigation but as of yet lack evidence to support their use for the prevention or treatment of COVID-19.
While there is plenty of skepticism put forward by physicians and scientists surrounding the use of supplements, these statements have not stopped consumers from purchasing these products, with one study reporting that online searches for dietary supplements in Poland began trending with the start of the pandemic <span class="citation" data-cites="19yy9SHf1">(<a href="#ref-19yy9SHf1" role="doc-biblioref">1867</a>)</span>.
Additionally, supplement usage increased between the first and second wave of the pandemic.
Participants reported various reasons for their use of supplements, including to improve immunity (60%), to improve overall health (57%), and to fill nutrient gaps in their diet (53%).
Other efforts to collect large datasets regarding such behavior have also sought to explore a possible association between vitamin or supplement consumption and COVID-19.
An observational analysis of survey responses from 327,720 users of the COVID Symptom Study App found that the consumption of n-3 PUFA supplements, probiotics, multivitamins, and vitamin D was associated with a lower risk of SARS-CoV-2 infection in women but not men after adjusting for potential confounders <span class="citation" data-cites="RJfF7BYs">(<a href="#ref-RJfF7BYs" role="doc-biblioref">1925</a>)</span>.
According to the authors, the sexual dimorphism observed may in part be because supplements may better support females due to known differences between the male and female immune systems, or it could be due to behavioral and health consciousness differences between the sexes <span class="citation" data-cites="RJfF7BYs">(<a href="#ref-RJfF7BYs" role="doc-biblioref">1925</a>)</span>.
Certainly, randomized controlled trials are required to investigate these findings further.</p>
<p>Finally, it is known that a patient’s nutritional status affects health outcomes in various infectious diseases <span class="citation" data-cites="5tHMedSi">(<a href="#ref-5tHMedSi" role="doc-biblioref">1700</a>)</span>, and COVID-19 is no different <span class="citation" data-cites="pQxCF4GH Hd5V5Usd d6c8mjAk">(<a href="#ref-Hd5V5Usd" role="doc-biblioref">1698</a>, <a href="#ref-pQxCF4GH" role="doc-biblioref">1926</a>, <a href="#ref-d6c8mjAk" role="doc-biblioref">1927</a>)</span>.
Some of the main risk factors for severe COVID-19, which also happen to be linked to poor nutritional status, include obesity, hypertension, cardiovascular diseases, type II diabetes mellitus, and indeed age-related malnutrition <span class="citation" data-cites="Hd5V5Usd Lyd8s4Mk 1HL90Mz7X">(<a href="#ref-Lyd8s4Mk" role="doc-biblioref">1696</a>, <a href="#ref-Hd5V5Usd" role="doc-biblioref">1698</a>, <a href="#ref-1HL90Mz7X" role="doc-biblioref">1928</a>)</span>.
Although not the main focus of this review, it is important to consider the nutritional challenges associated with severe COVID-19 patients.
Hospitalized COVID-19 patients tend to report an unusually high loss of appetite preceding admission, some suffer diarrhea and gastrointestinal symptoms that result in significantly lower food intake, and patients with poorer nutritional status were more likely to have worse outcomes and require nutrition therapy <span class="citation" data-cites="179QIwE1l">(<a href="#ref-179QIwE1l" role="doc-biblioref">1929</a>)</span>.
Dysphagia also seems to be a significant problem in pediatric patients that suffered multisystem inflammatory syndrome <span class="citation" data-cites="ARM0HdR3">(<a href="#ref-ARM0HdR3" role="doc-biblioref">1930</a>)</span> and rehabilitating COVID-19 patients, potentially contributing to poor nutritional status <span class="citation" data-cites="1DmkRczqw">(<a href="#ref-1DmkRczqw" role="doc-biblioref">1931</a>)</span>.
Almost two-thirds of discharged COVID-19 ICU patients exhibit significant weight loss, of which 26% had weight loss greater than 10% <span class="citation" data-cites="d6c8mjAk">(<a href="#ref-d6c8mjAk" role="doc-biblioref">1927</a>)</span>.
As investigated in this review, hospitalized patients also tend to exhibit vitamin D deficiency or insufficiency, which may be associated with greater disease severity <span class="citation" data-cites="sWzpGBil">(<a href="#ref-sWzpGBil" role="doc-biblioref">1917</a>)</span>.
Therefore, further research is required to determine how dietary supplements and nutraceuticals may contribute to the treatment of severely ill and rehabilitating patients, who often rely on enteral nutrition.</p>
<h2 data-number="10.10" id="conclusions-3"><span class="header-section-number">10.10</span> Conclusions</h2>
<p>Despite all the potential benefits of nutraceutical and dietary supplement interventions presented, currently there is a paucity of clinical evidence to support their use for the prevention or mitigation of COVID-19 infection.
Nevertheless, optimal nutritional status can prime an individual’s immune system to protect against the effects of acute respiratory viral infections by supporting normal maintenance of the immune system <span class="citation" data-cites="5tHMedSi Lyd8s4Mk">(<a href="#ref-Lyd8s4Mk" role="doc-biblioref">1696</a>, <a href="#ref-5tHMedSi" role="doc-biblioref">1700</a>)</span>.
Nutritional strategies can also play a role in the treatment of hospitalized patients, as malnutrition is a risk to COVID-19 patients <span class="citation" data-cites="1DmkRczqw">(<a href="#ref-1DmkRczqw" role="doc-biblioref">1931</a>)</span>.
Overall, supplementation of vitamin C, vitamin D, and zinc may be an effective method of ensuring their adequate intake to maintain optimal immune function, which may also convey beneficial effects against viral infections due to their immunomodulatory effects.
Individuals should pay attention to their nutritional status, particularly their intake of vitamin D, considering that vitamin D deficiency is widespread.
The prevailing evidence seems to indicate an association between vitamin D deficiency with COVID-19 incidence and, potentially, severity <span class="citation" data-cites="LpqYmd4C">(<a href="#ref-LpqYmd4C" role="doc-biblioref">1823</a>)</span>.
As a result, some international authorities have advised the general public, particularly those at high risk of infection, to consider vitamin D supplementation.
However, further well-controlled clinical trials are required to confirm these observations.</p>
<p>Many supplements and nutraceuticals designed for various ailments that are available in the United States and beyond are not strictly regulated <span class="citation" data-cites="lMhFSIiF">(<a href="#ref-lMhFSIiF" role="doc-biblioref">1932</a>)</span>.
Consequently, there can be safety and efficacy concerns associated with many of these products.
Often, the vulnerable members of society can be exploited in this regard and, unfortunately, the COVID-19 pandemic has proven no different.
As mentioned above, the FDA has issued warnings to several companies for advertising falsified claims in relation to the preventative and therapeutic capabilities of their products against COVID-19 <span class="citation" data-cites="fez8aZAu">(<a href="#ref-fez8aZAu" role="doc-biblioref">1933</a>)</span>.
Further intensive investigation is required to establish the effects of these nutraceuticals, if any, against COVID-19.
Until more effective therapeutics are established, the most effective mitigation strategies consist of encouraging standard public health practices such as regular hand washing with soap, wearing a face mask, and covering a cough with your elbow <span class="citation" data-cites="2Xu90WtD">(<a href="#ref-2Xu90WtD" role="doc-biblioref">1934</a>)</span>, along with following social distancing measures, “stay at home” guidelines, expansive testing, and contact tracing <span class="citation" data-cites="EhbZIMo0 12Met0WvX">(<a href="#ref-EhbZIMo0" role="doc-biblioref">1935</a>, <a href="#ref-12Met0WvX" role="doc-biblioref">1936</a>)</span>.
Indeed, in light of this review, it would also be pertinent to adopt a healthy diet and lifestyle following national guidelines in order to maintain optimal immune health.
Because of the broad public appeal of dietary supplements and nutraceuticals, it is important to evaluate the evidence regarding the use of such products.
We will continue to update this review as more findings become available.</p>
<h1 data-number="11" id="social-factors-influencing-covid-19-exposure-and-outcomes"><span class="header-section-number">11</span> Social Factors Influencing COVID-19 Exposure and Outcomes</h1>
<h2 data-number="11.1" id="social-factors-influencing-covid-19-outcomes"><span class="header-section-number">11.1</span> Social Factors Influencing COVID-19 Outcomes</h2>
<p>In addition to understanding the fundamental biology of the SARS-CoV-2 virus and COVID-19, it is critical to consider how the broader environment can influence both COVID-19 outcomes and efforts to develop and implement treatments for the disease.
The evidence clearly indicates that social environmental factors are critical determinants of individuals’ and communities’ risks related to COVID-19.
There are distinct components to COVID-19 susceptibility, and an individual’s risk can be elevated at one or all stages from exposure to recovery/mortality: an individual may be more likely to be exposed to the virus, more likely to get infected once exposed, more likely to have serious complications once infected, and be less likely to receive adequate care once they are seriously ill.
The fact that differences in survival between Black and white patients were no longer significant after controlling for comorbidities and socioeconomic status (type of insurance, neighborhood deprivation score, and hospital where treatment was received) in addition to sex and age <span class="citation" data-cites="hrOv770z">(<a href="#ref-hrOv770z" role="doc-biblioref">1937</a>)</span> underscores the relevance of social factors to understanding mortality differences between racial and ethnic groups.
Moreover, the Black patients were younger and more likely to be female than white patients, yet still had a higher mortality rate without correction for the other variables <span class="citation" data-cites="hrOv770z">(<a href="#ref-hrOv770z" role="doc-biblioref">1937</a>)</span>.
Here, we outline a few systemic reasons that may exacerbate the COVID-19 pandemic in communities of color.</p>
<h2 data-number="11.2" id="factors-observed-to-be-associated-with-susceptibility"><span class="header-section-number">11.2</span> Factors Observed to be Associated with Susceptibility</h2>
<p>As COVID-19 has spread into communities around the globe, it has become clear that the risks associated with this disease are not equally shared by all individuals or all communities.
Significant disparities in outcomes have led to interest in the demographic, biomedical, and social factors that influence COVID-19 severity.
Untangling the factors influencing COVID-19 susceptibility is a complex undertaking.
Among patients who are admitted to the hospital, outcomes have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy <span class="citation" data-cites="15IDA5kX6 11xfoqpEl qlJ8hCUs">(<a href="#ref-15IDA5kX6" role="doc-biblioref">82</a>, <a href="#ref-11xfoqpEl" role="doc-biblioref">1938</a>, <a href="#ref-qlJ8hCUs" role="doc-biblioref">1939</a>)</span>.
However, hospitalization rates vary by location <span class="citation" data-cites="xZBEFXiG">(<a href="#ref-xZBEFXiG" role="doc-biblioref">1940</a>)</span>.
This variation may be influenced by demographic (e.g., average age in the area), medical (e.g., the prevalence of comorbid conditions such as diabetes), and social (e.g., income or healthcare availability) factors that vary geographically.
Additionally, some of the same factors may influence an individual’s probability of exposure to SARS-CoV-2, their risk of developing a more serious case of COVID-19 that would require hospitalization, and their access to medical support.
As a result, quantifying or comparing susceptibility among individuals, communities, or other groups requires consideration of a number of complex phenomena that intersect across many disciplines of research.
In this section, the term “risk factor” is used to refer to variables that are statistically associated with more severe COVID-19 outcomes.
Some are intrinsic characteristics that have been observed to carry an association with variation in outcomes, whereas others may be more functionally linked to the pathophysiology of COVID-19.</p>
<h3 data-number="11.2.1" id="patient-traits-associated-with-increased-risk"><span class="header-section-number">11.2.1</span> Patient Traits Associated with Increased Risk</h3>
<p>Two traits that have been consistently associated with more severe COVID-19 outcomes are male sex and advanced age (typically defined as 60 or older, with the greatest risk among those 85 and older <span class="citation" data-cites="aZlVFn2R">(<a href="#ref-aZlVFn2R" role="doc-biblioref">1941</a>)</span>).
In the United States, males and older individuals diagnosed with COVID-19 were found to be more likely to require hospitalization <span class="citation" data-cites="1D6lVXWbJ Zk5CfYeW">(<a href="#ref-1D6lVXWbJ" role="doc-biblioref">1942</a>, <a href="#ref-Zk5CfYeW" role="doc-biblioref">1943</a>)</span>.
A retrospective study of hospitalized Chinese patients <span class="citation" data-cites="10THxyeCg">(<a href="#ref-10THxyeCg" role="doc-biblioref">83</a>)</span> found that a higher probability of mortality was associated with older age, and world-wide, population age structure has been found to be an important variable for explaining differences in outbreak severity <span class="citation" data-cites="BQ4pZGxJ">(<a href="#ref-BQ4pZGxJ" role="doc-biblioref">1944</a>)</span>.
The CFR for adults over 80 has been estimated upwards of 14% or even 20% <span class="citation" data-cites="1DnayDKc4">(<a href="#ref-1DnayDKc4" role="doc-biblioref">1945</a>)</span>.
Male sex has also been identified as a risk factor for severe COVID-19 outcomes, including death <span class="citation" data-cites="gObtnWSD DqBeNrR5 ConBOYBB">(<a href="#ref-gObtnWSD" role="doc-biblioref">1946</a>, <a href="#ref-DqBeNrR5" role="doc-biblioref">1947</a>, <a href="#ref-ConBOYBB" role="doc-biblioref">1948</a>/)</span>.
Early reports from China and Europe indicated that even though the case rates were similar across males and females, males were at elevated risk for hospital admission, ICU admission, and death <span class="citation" data-cites="DqBeNrR5">(<a href="#ref-DqBeNrR5" role="doc-biblioref">1947</a>)</span>, although data from some US states indicates more cases among females, potentially due to gender representation in care-taking professions <span class="citation" data-cites="fH3qecwy">(<a href="#ref-fH3qecwy" role="doc-biblioref">1949</a>)</span>.
In older age groups (e.g., age 60 and older), comparable absolute numbers of male and female cases actually suggests a higher rate of occurrence in males, due to increased skew in the sex ratio <span class="citation" data-cites="DqBeNrR5">(<a href="#ref-DqBeNrR5" role="doc-biblioref">1947</a>)</span>.
Current estimates based on worldwide data suggest that, compared to females, males may be 30% more likely to be hospitalized, 80% more likely to be admitted to the ICU, and 40% more likely to die as a result of COVID-19 <span class="citation" data-cites="ConBOYBB">(<a href="#ref-ConBOYBB" role="doc-biblioref">1948</a>/)</span>.
There also may be a compounding effect of advanced age and male sex, with differences time to recovery worst for males over 60 years old relative to female members of their age cohort <span class="citation" data-cites="RcYBFjm1">(<a href="#ref-RcYBFjm1" role="doc-biblioref">1950</a>)</span>.</p>
<p>Both of these risk factors can be approached through the lens of biology.
The biological basis for greater susceptibility with age is likely linked to the prevalence of extenuating health conditions such as heart failure or diabetes <span class="citation" data-cites="1DnayDKc4">(<a href="#ref-1DnayDKc4" role="doc-biblioref">1945</a>)</span>.
Several hypotheses have been proposed to account for differences in severity between males and females.
For example, some evidence suggests that female sex hormones may be protective <span class="citation" data-cites="DqBeNrR5 fH3qecwy">(<a href="#ref-DqBeNrR5" role="doc-biblioref">1947</a>, <a href="#ref-fH3qecwy" role="doc-biblioref">1949</a>)</span>.
ACE2 expression in the kidneys of male mice was observed to be twice as high as that of females, and a regulatory effect of estradiol on ACE2 expression was demonstrated by removing the gonads and then supplementing with estradiol <span class="citation" data-cites="OJvjTsWz fH3qecwy">(<a href="#ref-fH3qecwy" role="doc-biblioref">1949</a>, <a href="#ref-OJvjTsWz" role="doc-biblioref">1951</a>)</span>.
Other work in mice has shown an inverse association between mortality due to SARS-CoV-1 and estradiol, suggesting a protective role for the sex hormone <span class="citation" data-cites="fH3qecwy">(<a href="#ref-fH3qecwy" role="doc-biblioref">1949</a>)</span>.
Similarly, evidence suggests that similar patterns might be found in other tissues.
A preliminary analysis identified higher levels of ACE2 expression in the myocardium of male patients with aortic valve stenosis showed than female patients, although this pattern was not found in controls <span class="citation" data-cites="DqBeNrR5">(<a href="#ref-DqBeNrR5" role="doc-biblioref">1947</a>)</span>.
Additionally, research has indicated that females respond to lower doses than males of heart medications that act on the Renin angiotensin aldosterone system (RAAS) pathway, which is shared with ACE2 <span class="citation" data-cites="DqBeNrR5">(<a href="#ref-DqBeNrR5" role="doc-biblioref">1947</a>)</span>.
Additionally, several components of the immune response, including the inflammatory response, may differ in intensity and timing between males and females <span class="citation" data-cites="fH3qecwy OJvjTsWz">(<a href="#ref-fH3qecwy" role="doc-biblioref">1949</a>, <a href="#ref-OJvjTsWz" role="doc-biblioref">1951</a>)</span>.
This hypothesis is supported by some preliminary evidence showing that female patients who recovered from severe COVID-19 had higher antibody titers than males <span class="citation" data-cites="fH3qecwy">(<a href="#ref-fH3qecwy" role="doc-biblioref">1949</a>)</span>.
Sex steroids can also bind to immune cell receptors to influence cytokine production <span class="citation" data-cites="DqBeNrR5">(<a href="#ref-DqBeNrR5" role="doc-biblioref">1947</a>)</span>.
Additionally, social factors may influence risks related to both age and sex: for example, older adults are more likely to live in care facilities, which have been a source for a large number of outbreaks <span class="citation" data-cites="ujWsdPdH">(<a href="#ref-ujWsdPdH" role="doc-biblioref">1952</a>)</span>, and gender roles may also influence exposure and/or susceptibility due to differences in care-taking and/or risky behaviors (e.g., caring for elder relatives and smoking, respectively) <span class="citation" data-cites="DqBeNrR5">(<a href="#ref-DqBeNrR5" role="doc-biblioref">1947</a>)</span> among men and women (however, it should be noted that both transgender men and women are suspected to be at heightened risk <span class="citation" data-cites="agpMLmxt">(<a href="#ref-agpMLmxt" role="doc-biblioref">1953</a>)</span>.)</p>
<h3 data-number="11.2.2" id="comorbid-health-conditions"><span class="header-section-number">11.2.2</span> Comorbid Health Conditions</h3>
<p>A number of pre-existing or comorbid conditions have repeatedly been identified as risk factors for more severe COVID-19 outcomes.
Several underlying health conditions were identified at high prevalence among hospitalized patients, including obesity, diabetes, hypertension, lung disease, and cardiovascular disease <span class="citation" data-cites="xZBEFXiG">(<a href="#ref-xZBEFXiG" role="doc-biblioref">1940</a>)</span>.
Higher Sequential Organ Failure Assessment (SOFA) scores have been associated with a higher probability of mortality <span class="citation" data-cites="10THxyeCg">(<a href="#ref-10THxyeCg" role="doc-biblioref">83</a>)</span>, and comorbid conditions such as cardiovascular and lung disease as well as obesity were also associated with an increased risk of hospitalization and death, even when correcting for age and sex <span class="citation" data-cites="gObtnWSD">(<a href="#ref-gObtnWSD" role="doc-biblioref">1946</a>)</span>.
Diabetes may increase the risk of lengthy hospitalization <span class="citation" data-cites="9NfXSHjH">(<a href="#ref-9NfXSHjH" role="doc-biblioref">1954</a>)</span> or of death <span class="citation" data-cites="9NfXSHjH 13BpwZaaq">(<a href="#ref-9NfXSHjH" role="doc-biblioref">1954</a>, <a href="#ref-13BpwZaaq" role="doc-biblioref">1955</a>)</span>.
<span class="citation" data-cites="UkCdtLwF">(<a href="#ref-UkCdtLwF" role="doc-biblioref">1956</a>)</span> and <span class="citation" data-cites="OT5GEOej">(<a href="#ref-OT5GEOej" role="doc-biblioref">1957</a>)</span> discuss possible ways in which COVID-19 and diabetes may interact.
Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2 <span class="citation" data-cites="ZLmNMDrD 2Ahpn6R2">(<a href="#ref-ZLmNMDrD" role="doc-biblioref">1958</a>, <a href="#ref-2Ahpn6R2" role="doc-biblioref">1959</a>)</span>.
Obesity is considered an underlying risk factor for other health problems, and the mechanism for its contributions to COVID-19 hospitalization or mortality is not yet clear <span class="citation" data-cites="1AmBg80Un">(<a href="#ref-1AmBg80Un" role="doc-biblioref">1960</a>)</span>.
Dementia and cancer were also associated with the risk of death in an analysis of a large number (more than 20,000) COVID-19 patients in the United Kingdom <span class="citation" data-cites="gObtnWSD">(<a href="#ref-gObtnWSD" role="doc-biblioref">1946</a>)</span>.
It should be noted that comorbid conditions are inextricably tied to age, as conditions tend to be accumulated over time, but that the prevalence of individual comorbidities or of population health overall can vary regionally <span class="citation" data-cites="1GakM8xhV">(<a href="#ref-1GakM8xhV" role="doc-biblioref">1961</a>)</span>.
Several comorbidities that are highly prevalent in older adults, such as COPD, hypertension, cardiovascular disease, and diabetes, have been associated with CFRs upwards of 8% compared to an estimate of 1.4% in people without comorbidities <span class="citation" data-cites="1DnayDKc4 H6dfv6Wv">(<a href="#ref-1DnayDKc4" role="doc-biblioref">1945</a>, <a href="#ref-H6dfv6Wv" role="doc-biblioref">1962</a>)</span>. <!-- will this url work for Manubot?-->
Therefore, both age and health are important considerations when predicting the impact of COVID-19 on a population <span class="citation" data-cites="1GakM8xhV">(<a href="#ref-1GakM8xhV" role="doc-biblioref">1961</a>)</span>.
However, other associations may exist, such as patients with sepsis having higher SOFA scores – in fact, SOFA was developed for the assessment of organ failure in the context of sepsis, and the acronym originally stood for Sepsis-Related Organ Failure Assessment <span class="citation" data-cites="TuzwDCPt a0h975P6">(<a href="#ref-TuzwDCPt" role="doc-biblioref">1963</a>, <a href="#ref-a0h975P6" role="doc-biblioref">1964</a>)</span>.
Additionally, certain conditions are likely to be more prevalent under or exacerbated by social conditions, especially poverty, as is discussed further below.</p>
<h3 data-number="11.2.3" id="ancestry"><span class="header-section-number">11.2.3</span> Ancestry</h3>
<p>A number of studies have suggested associations between individuals’ racial and ethnic backgrounds and their COVID-19 risk.
In particular, Black Americans are consistently identified as carrying a higher burden of COVID-19 than white Americans <span class="citation" data-cites="1D6lVXWbJ Zk5CfYeW">(<a href="#ref-1D6lVXWbJ" role="doc-biblioref">1942</a>, <a href="#ref-Zk5CfYeW" role="doc-biblioref">1943</a>)</span>, with differences in the rates of kidney complications from COVID-19 particularly pronounced <span class="citation" data-cites="17hdoiBo9">(<a href="#ref-17hdoiBo9" role="doc-biblioref">89</a>)</span>.
Statistics from a number of cities indicate significant discrepancies between the proportion of COVID-19 cases and deaths in Black Americans relative to their representation in the general population <span class="citation" data-cites="jKKxa9Yy">(<a href="#ref-jKKxa9Yy" role="doc-biblioref">1965</a>)</span>.
In addition to Black Americans, disproportionate harm and mortality from COVID-19 has also been noted in Latino/Hispanic Americans and in Native American and Alaskan Native communities, including the Navajo nation [<span class="citation" data-cites="yK5bczRb">(<a href="#ref-yK5bczRb" role="doc-biblioref">1966</a>)</span>; <span class="citation" data-cites="u4MMV8aJ">(<a href="#ref-u4MMV8aJ" role="doc-biblioref">1967</a>)</span>; <span class="citation" data-cites="JAo2QxY">(<a href="#ref-JAo2QxY" role="doc-biblioref">1968</a>)</span>; https://www.nytimes.com/2020/04/09/us/coronavirus-navajo-nation.html?searchResultPosition=10; <span class="citation" data-cites="12WZObWRf">(<a href="#ref-12WZObWRf" role="doc-biblioref">1969</a>)</span>; <span class="citation" data-cites="Pp8GWnQm">(<a href="#ref-Pp8GWnQm" role="doc-biblioref">1970</a>)</span>; <span class="citation" data-cites="J9T8eR5H">(<a href="#ref-J9T8eR5H" role="doc-biblioref">1971</a>)</span>].
In Brazil, indigenous communities likewise carry an increased burden of COVID-19 <span class="citation" data-cites="ODm1CaEq">(<a href="#ref-ODm1CaEq" role="doc-biblioref">1972</a>)</span>.
In the United Kingdom, nonwhite ethnicity (principally Black or South Asian) was one of several factors found to be associated with a higher risk of death from COVID-19 <span class="citation" data-cites="17gnxmOnf">(<a href="#ref-17gnxmOnf" role="doc-biblioref">1973</a>)</span>.</p>
<p>From a genetic standpoint, it is highly unlikely that ancestry itself predisposes individuals to contracting COVID-19 or to experiencing severe COVID-19 outcomes.
Examining human genetic diversity indicates variation over a geographic continuum, and that most human genetic variation is associated with the African continent <span class="citation" data-cites="pwbrxYv8">(<a href="#ref-pwbrxYv8" role="doc-biblioref">1974</a>)</span>.
African-Americans are also a more genetically diverse group relative to European-Americans, with a large number of rare alleles and a much smaller fraction of common alleles identified in African-Americans <span class="citation" data-cites="CMAPEmTp">(<a href="#ref-CMAPEmTp" role="doc-biblioref">1975</a>)</span>.
Therefore, the idea that African ancestry (at the continent level) might convey some sort of genetic risk for severe COVID-19 contrasts with what is known about worldwide human genetic diversity <span class="citation" data-cites="q4yhes98">(<a href="#ref-q4yhes98" role="doc-biblioref">1976</a>)</span>.
The possibility for genetic variants that confer some risk or some protection remains possible, but has not been widely explored, especially at a global level.
Research in Beijing of a small number (n=80) hospitalized COVID-19 patients revealed an association between severe COVID-19 outcomes and homozygosity for an allele in the interferon-induced transmembrane protein 3 (IFITM3) gene, which was selected as a candidate because it was previously found to be associated with influenza outcomes in Chinese patients <span class="citation" data-cites="FbeY1L1s">(<a href="#ref-FbeY1L1s" role="doc-biblioref">435</a>)</span>.
Genetic factors may also play a role in the risk of respiratory failure for COVID-19 <span class="citation" data-cites="owevcpwb nn67UTjV I2mIU0wn">(<a href="#ref-owevcpwb" role="doc-biblioref">464</a>, <a href="#ref-nn67UTjV" role="doc-biblioref">1977</a>, <a href="#ref-I2mIU0wn" role="doc-biblioref">1978</a>)</span>.
However, genetic variants associated with outcomes within ancestral groups are far less surprising than genetic variants explaining outcomes between groups.
Alleles in <em>ACE2</em> and <em>TMPRSS2</em> have been identified that vary in frequency among ancestral groups <span class="citation" data-cites="KbDdcSgo">(<a href="#ref-KbDdcSgo" role="doc-biblioref">1979</a>)</span>, but whether these variants are associated with COVID-19 susceptibility has not been explored.</p>
<p>Instead, examining patterns of COVID-19 susceptibility on a global scale that suggest that social factors are of primary importance in predicting mortality.
Reports from several sub-Saharan African countries have indicated that the effects of the COVID-19 pandemic have been less severe than expected based on the outbreaks in China and Italy.
In Kenya, for example, estimates of national prevalence based on testing blood donors for SARS-CoV-2 antibodies were consistent with 5% of Kenyan adults having recovered from COVID-19 <span class="citation" data-cites="4SqaP0Ub">(<a href="#ref-4SqaP0Ub" role="doc-biblioref">1980</a>)</span>.
This high seroprevalence of antibodies lies in sharp contrast to the low number of COVID-19 fatalities in Kenya, which at the time was 71 out of 2093 known cases <span class="citation" data-cites="4SqaP0Ub">(<a href="#ref-4SqaP0Ub" role="doc-biblioref">1980</a>)</span>.
Likewise, a serosurvey of health care workers in Blantyre City, Malawi reported an adjusted antibody prevalence of 12.3%, suggesting that the virus had been circulating more widely than thought and that the death rate was up eight times lower than models had predicted <span class="citation" data-cites="eWIOKa0u">(<a href="#ref-eWIOKa0u" role="doc-biblioref">1981</a>)</span>.
While several possible hypotheses for the apparent reduced impact of COVID-19 on the African continent are being explored, such as young demographics in many places <span class="citation" data-cites="p4AYZss0">(<a href="#ref-p4AYZss0" role="doc-biblioref">1982</a>)</span>, these reports present a stark contrast to the severity of COVID-19 in Americans and Europeans of African descent.
Additionally, ethnic minorities in the United Kingdom also tend to be younger than white British living in the same areas, yet the burden of COVID-19 is still more serious for minorities, especially people of Black Caribbean ancestry, both in absolute numbers and when controlling for age and location <span class="citation" data-cites="xFKmPg6l">(<a href="#ref-xFKmPg6l" role="doc-biblioref">1983</a>/)</span>.
Furthermore, the groups in the United States and United Kingdom that have been identified as carrying elevated COVID-19 burden, namely Black American, indigenous American, and Black and South Asian British, are quite distinct in their position on the human ancestral tree.
What is shared across these groups is instead a history of disenfranchisement under colonialism and ongoing systematic racism.
A large analysis of over 11,000 COVID-19 patients hospitalized in 92 hospitals across U.S. states revealed that Black patients were younger, more often female, more likely to be on Medicaid, more likely to have comorbidities, and came from neighborhoods identified as more economically deprived than white patients <span class="citation" data-cites="hrOv770z">(<a href="#ref-hrOv770z" role="doc-biblioref">1937</a>)</span>.
This study reported that when these factors were accounted for, the differences in mortality between Black and white patients were no longer significant.
Thus, the current evidence suggests that the apparent correlations between ancestry and health outcomes must be examined in the appropriate social context.</p>
<h2 data-number="11.3" id="environmental-influences-on-susceptibility"><span class="header-section-number">11.3</span> Environmental Influences on Susceptibility</h2>
<h3 data-number="11.3.1" id="exposure-to-covid-19"><span class="header-section-number">11.3.1</span> Exposure to COVID-19</h3>
<p>Social distancing has emerged as one of the main social policies used to manage the COVID-19 epidemic in many countries.
Many governments issued stay-at-home orders, especially in the initial months of the crisis.
However, data clearly indicates that these orders impacted different socioeconomic groups differently.
In U.S. counties with and without stay-at-home orders, smartphone tracking indicated a significant decrease in the general population’s mobility in April relative to February through March of 2020 (-52.3% and -60.8%, respectively) <span class="citation" data-cites="bDNDEYNn">(<a href="#ref-bDNDEYNn" role="doc-biblioref">1984</a>)</span>.
A linear relationship was observed between counties’ reduction in mobility and their wealth and health, as measured by access to health care, food security, income, space, and other factors <span class="citation" data-cites="bDNDEYNn">(<a href="#ref-bDNDEYNn" role="doc-biblioref">1984</a>)</span>.
Counties with greater reductions in mobility were also found to have much lower child poverty and household crowding and to be more racially segregated, and to have fewer youth and more elderly residents <span class="citation" data-cites="bDNDEYNn">(<a href="#ref-bDNDEYNn" role="doc-biblioref">1984</a>)</span>.
Similar associations between wealth and decreased mobility were observed in cellphone GPS data from Colombia, Indonesia, and Mexico collected between January and May 2020 <span class="citation" data-cites="RdnghmVU">(<a href="#ref-RdnghmVU" role="doc-biblioref">1985</a>)</span>, as well as in a very large data set from several US cities <span class="citation" data-cites="9OksxLmM">(<a href="#ref-9OksxLmM" role="doc-biblioref">1986</a>)</span>.
These disparities in mobility are likely to be related to the role that essential workers have played during the pandemic.
Essential workers are disproportionately likely to be female, people of color, immigrants, and to have an income below 200% of the poverty line <span class="citation" data-cites="15qfRPnUG">(<a href="#ref-15qfRPnUG" role="doc-biblioref">1987</a>)</span>.
Black Americans in particular are over-represented among front-line workers and in professions where social distancing is infeasible <span class="citation" data-cites="HihYbn6k">(<a href="#ref-HihYbn6k" role="doc-biblioref">1988</a>)</span>.
Health care work in particular presents an increased risk of exposure to SARS-CoV-2 <span class="citation" data-cites="HihYbn6k F4laSOXM hAYohR8m DOyiuaup 13qkVV6eN">(<a href="#ref-HihYbn6k" role="doc-biblioref">1988</a>–<a href="#ref-13qkVV6eN" role="doc-biblioref">1992</a>)</span>.
In the United Kingdom, (South) Asians are more likely than their white counterparts to be medical professionals <span class="citation" data-cites="xFKmPg6l">(<a href="#ref-xFKmPg6l" role="doc-biblioref">1983</a>/)</span>, although BAME medical professionals are still disproportionately represented in the proportion of National Health Service staff deaths <span class="citation" data-cites="4zjepZ8T">(<a href="#ref-4zjepZ8T" role="doc-biblioref">1993</a>)</span>.
Similar trends have been reported for nurses, especially nurses of color, in the United States <span class="citation" data-cites="2gxJPENM">(<a href="#ref-2gxJPENM" role="doc-biblioref">1994</a>/-/files/graphics/0920_Covid19_SinsOfOmission_Data_Report.pdf)</span>.
Furthermore, beyond the risks associated with work itself, use of public transportation may also impact COVID-19 risk <span class="citation" data-cites="Moc3BREQ">(<a href="#ref-Moc3BREQ" role="doc-biblioref">1995</a>)</span>.
The socioeconomic and racial/ethnic gaps in who is working on the front lines of the pandemic make it clear that socioeconomic privilege is likely to decrease the probability of exposure to SARS-CoV-2.</p>
<p>Increased risk of exposure can also arise outside the workplace.
Nursing homes and skilled nursing facilities received attention early on as high-risk locations for COVID-19 outbreaks <span class="citation" data-cites="aM2XPE3J">(<a href="#ref-aM2XPE3J" role="doc-biblioref">1996</a>)</span>.
Prisons and detention centers also confer a high risk of exposure or infection <span class="citation" data-cites="BeedQx3I 2uUcva6Y">(<a href="#ref-BeedQx3I" role="doc-biblioref">1997</a>, <a href="#ref-2uUcva6Y" role="doc-biblioref">1998</a>)</span>.
Populations in care facilities are largely older adults, and in the United States, incarcerated people are more likely to be male and persons of color, especially Black <span class="citation" data-cites="zuipD7ZP">(<a href="#ref-zuipD7ZP" role="doc-biblioref">1999</a>)</span>.
Additionally, multi-generational households are less common among non-Hispanic white Americans than people of other racial and ethnic backgrounds <span class="citation" data-cites="OzhmHOxJ">(<a href="#ref-OzhmHOxJ" role="doc-biblioref">2000</a>)</span>, increasing the risk of exposure for more susceptible family members.
Analysis suggests that household crowding may also be associated with increased risk of COVID-19 exposure <span class="citation" data-cites="bDNDEYNn">(<a href="#ref-bDNDEYNn" role="doc-biblioref">1984</a>)</span>, and household crowding is associated with poverty <span class="citation" data-cites="13wfXDXqr">(<a href="#ref-13wfXDXqr" role="doc-biblioref">2001</a>)</span>.
Forms of economic insecurity like housing insecurity, which is associated with poverty and more pronounced in communities subjected to racism <span class="citation" data-cites="1Axz3a1Hj 64YOO3yt">(<a href="#ref-1Axz3a1Hj" role="doc-biblioref">2002</a>, <a href="#ref-64YOO3yt" role="doc-biblioref">2003</a>)</span>, would be likely to increase household crowding and other possible sources of exposure.
As a result, facets of systemic inequality such as mass incarceration of Black Americans and poverty are likely to increase the risk of exposure outside of the workplace.</p>
<h3 data-number="11.3.2" id="severity-of-covid-19-following-exposure"><span class="header-section-number">11.3.2</span> Severity of COVID-19 Following Exposure</h3>
<p>Following exposure to SARS-CoV-2, the likelihood that an individual develops COVID-19 and the severity of the disease presentation can be influenced by a number of social factors.
As discussed above, a number of patient characteristics are associated with the likelihood of severe COVID-19 symptoms.
In some cases, these trends run counter to those expected given rates of exposure: for example, although women are more likely to be exposed, men are more likely to be diagnosed with, hospitalized from, or die from COVID-19 <span class="citation" data-cites="fH3qecwy">(<a href="#ref-fH3qecwy" role="doc-biblioref">1949</a>)</span>.
In the case of comorbid conditions and racial/ethnic demographics, however, social factors are highly likely to modulate or at least influence the apparent association between these traits and the increased risk from COVID-19.
In particular, the comorbidities and racial/ethnic correlates of severe COVID-19 outcomes suggest that poverty confers additional risk for COVID-19.</p>
<p>In order to explore the relationship between poverty and COVID-19 outcomes, it is necessary to consider how poverty impacts biology.
In particular, we focus on the United States and the United Kingdom.
Comorbidities that increase risk for COVID-19, including obesity, type II diabetes, hypertension, and cardiovascular disease, are known to be intercorrelated <span class="citation" data-cites="sRZCzj6j">(<a href="#ref-sRZCzj6j" role="doc-biblioref">2004</a>)</span>.
Metabolic conditions related to heightened inflammation, like obesity, type II diabetes, and hypertension, are more strongly associated with negative COVID-19 outcomes than other comorbid conditions, such as chronic heart disease <span class="citation" data-cites="JizR1ocj">(<a href="#ref-JizR1ocj" role="doc-biblioref">2005</a>)</span>.
As discussed above, dysregulated inflammation characteristic of cytokine release syndrome is one of the greatest concerns for COVID-19-related death.
Therefore, it is possible that chronic inflammation characteristic of these metabolic conditions predisposes patients to COVID-19-related death <span class="citation" data-cites="JizR1ocj">(<a href="#ref-JizR1ocj" role="doc-biblioref">2005</a>)</span>.
The association between these diseases and severe COVID-19 outcomes is a concern from a health equity perspective because poverty exposes people to “obesogenic” conditions <span class="citation" data-cites="nYLB9EMw">(<a href="#ref-nYLB9EMw" role="doc-biblioref">2006</a>)</span> and is therefore unsurprisingly associated with higher incidence of obesity and associated disorders <span class="citation" data-cites="h1e9QTlK">(<a href="#ref-h1e9QTlK" role="doc-biblioref">2007</a>)</span>.
Furthermore, cell phone GPS data suggests that lower socioeconomic status may also be associated with decreased access to healthy food choices during the COVID-19 pandemic <span class="citation" data-cites="94eOAGnA 5G5UozXd">(<a href="#ref-94eOAGnA" role="doc-biblioref">2008</a>, <a href="#ref-5G5UozXd" role="doc-biblioref">2009</a>)</span>, suggesting that health-related risk factors for COVID-19 may be exacerbated as the pandemic continues <span class="citation" data-cites="17yfPiYAS">(<a href="#ref-17yfPiYAS" role="doc-biblioref">2010</a>)</span>.
Chronic inflammation is a known outcome of chronic stress (e.g., <span class="citation" data-cites="Hu7PyNNp P2wUkjrR 109EtOYzw 11oGPv27i">(<a href="#ref-Hu7PyNNp" role="doc-biblioref">2011</a>–<a href="#ref-11oGPv27i" role="doc-biblioref">2014</a>)</span>).
Therefore, the chronic stress of poverty is likely to influence health broadly (as summarized in <span class="citation" data-cites="KZSJklnl">(<a href="#ref-KZSJklnl" role="doc-biblioref">2015</a>)</span>) and especially during the stress of the ongoing pandemic.</p>
<p>A preprint <span class="citation" data-cites="5BMDcXz3">(<a href="#ref-5BMDcXz3" role="doc-biblioref">2016</a>)</span> provided observational evidence that geographical areas in the United States that suffer from worse air pollution by fine particulate matter have also suffered more COVID-19 deaths per capita, after adjusting for demographic covariates.
Although lack of individual-level exposure data and the impossibility of randomization make it difficult to elucidate the exact causal mechanism, this finding would be consistent with similar findings for all-cause mortality (e.g., <span class="citation" data-cites="vImXUPNS">(<a href="#ref-vImXUPNS" role="doc-biblioref">2017</a>)</span>).
Exposure to air pollution is associated with both poverty (e.g., <span class="citation" data-cites="6ACdDJOV">(<a href="#ref-6ACdDJOV" role="doc-biblioref">2018</a>)</span>) and chronic inflammation <span class="citation" data-cites="11Ift2eIl">(<a href="#ref-11Ift2eIl" role="doc-biblioref">2019</a>)</span>.
Other outcomes of environmental racism, such as the proximity of abandoned uranium mines to Navajo land, can also cause respiratory illnesses and other health issues <span class="citation" data-cites="J9T8eR5H">(<a href="#ref-J9T8eR5H" role="doc-biblioref">1971</a>)</span>.
Similarly, preliminary findings indicate that nutritional status (e.g., vitamin D deficiency <span class="citation" data-cites="8ZxZKpYS">(<a href="#ref-8ZxZKpYS" role="doc-biblioref">1832</a>)</span>) may be associated with COVID-19 outcomes, and reduced access to grocery stores and fresh food often co-occurs with environmental racism <span class="citation" data-cites="J9T8eR5H aHM6WZzd">(<a href="#ref-J9T8eR5H" role="doc-biblioref">1971</a>, <a href="#ref-aHM6WZzd" role="doc-biblioref">2020</a>)</span>.
Taken together, the evidence suggests that low-income workers who face greater exposure to SARS-CoV-2 due to their home or work conditions are also more likely to face environmental and social stressors associated with increased inflammation, and therefore with increased risk from COVID-19.
In particular, structural racism can play an important role on disease severity after SARS-CoV-2 exposure, due to consequences of racism which include an increased likelihood of poverty and its associated food and housing instability.
COVID-19 can thus be considered a “syndemic”, or a synergistic interaction between several epidemics <span class="citation" data-cites="11Ldf3Ch3">(<a href="#ref-11Ldf3Ch3" role="doc-biblioref">2021</a>)</span>.
As a result, it is not surprising that people from minoritized backgrounds and/or with certain pre-existing conditions are more likely to suffer severe effects of COVID-19, but these “risk factors” are likely to be causally linked to poverty <span class="citation" data-cites="XtKjlYFF">(<a href="#ref-XtKjlYFF" role="doc-biblioref">2022</a>)</span>.</p>
<h3 data-number="11.3.3" id="access-to-treatment"><span class="header-section-number">11.3.3</span> Access to Treatment</h3>
<p>Finally, COVID-19 outcomes can be influenced by access to healthcare.
Receiving care for COVID-19 can, but does not always, include receiving a positive test for the SARS-CoV-2 virus.
For example, it is common to see treatment guidelines for suspected cases regardless of whether the presence of SARS-CoV-2 has been confirmed (e.g., <span class="citation" data-cites="cSA2h9hO">(<a href="#ref-cSA2h9hO" role="doc-biblioref">2023</a>)</span>).
Whether and where a patient is diagnosed can depend on their access to testing, which can vary both between and within countries.
In the United States, it is not always clear whether an individual will have access to free testing <span class="citation" data-cites="1Intrlax jc9vc7ws">(<a href="#ref-1Intrlax" role="doc-biblioref">2024</a>, <a href="#ref-jc9vc7ws" role="doc-biblioref">2025</a>)</span>.
The concern has been raised that more economic privilege is likely to correspond to increased access to testing, at least within the United States <span class="citation" data-cites="13MGsHBtw">(<a href="#ref-13MGsHBtw" role="doc-biblioref">2026</a>)</span>.
This is supported by the fact that African Americans seem to be more likely to be diagnosed in the hospital, while individuals from other groups were more likely to have been diagnosed in ambulatory settings in the community <span class="citation" data-cites="1D6lVXWbJ">(<a href="#ref-1D6lVXWbJ" role="doc-biblioref">1942</a>)</span>.
Any delays in treatment are a cause for concern <span class="citation" data-cites="13MGsHBtw">(<a href="#ref-13MGsHBtw" role="doc-biblioref">2026</a>)</span>, which could potentially be increased by an inability to acquire testing because in the United States, insurance coverage for care received can depend on a positive test <span class="citation" data-cites="P8VNkYTE">(<a href="#ref-P8VNkYTE" role="doc-biblioref">2027</a>)</span>.</p>
<p>Another important question is whether patients with moderate to severe cases are able to access hospital facilities and treatments, to the extent that they have been identified.
Early findings from China as of February 2020 suggested the COVID-19 mortality rate to be much lower in the most developed regions of the country <span class="citation" data-cites="tLVYjfMk">(<a href="#ref-tLVYjfMk" role="doc-biblioref">2028</a>)</span>, although reported mortality is generally an estimate of CFR, which is dependent on rates of testing.
Efforts to make treatment accessible for all confirmed and suspected cases of COVID-19 in China are credited with expanding care to people with fewer economic resources <span class="citation" data-cites="cOQK0zNa">(<a href="#ref-cOQK0zNa" role="doc-biblioref">2029</a>)</span>.
In the United States, access to healthcare varies widely, with certain sectors of the workforce less likely to have health insurance; many essential workers in transportation, food service, and other frontline fields are among those likely to be uninsured or underinsured <span class="citation" data-cites="13MGsHBtw">(<a href="#ref-13MGsHBtw" role="doc-biblioref">2026</a>)</span>.
As of 2018, Hispanic Americans of all races were much less likely to have health insurance than people from non-Hispanic backgrounds <span class="citation" data-cites="gOiB5HDd">(<a href="#ref-gOiB5HDd" role="doc-biblioref">2030</a>)</span>.
Therefore, access to diagnostics and care prior to the development of severe COVID-19 is likely to vary depending on socioeconomic and social factors, many of which overlap with the risks of exposure and of developing more severe COVID-19 symptoms.
This discrepancy ties into concerns about broad infrastructural challenges imposed by COVID-19.
A major concern in many countries has been the saturation of healthcare systems due to the volume of COVID-19 hospitalizations (e.g., <span class="citation" data-cites="NCcssjLV">(<a href="#ref-NCcssjLV" role="doc-biblioref">320</a>)</span>).
Similarly, there have been shortages of supplies such as ventilators that are critical to the survival of many COVID-19 patients, leading to extensive ethical discussions about how to allocate limited resources among patients <span class="citation" data-cites="15Yaf3fc8 1As9kWVvA FaGAJ7AI B3awlFvs">(<a href="#ref-15Yaf3fc8" role="doc-biblioref">2031</a>–<a href="#ref-B3awlFvs" role="doc-biblioref">2034</a>)</span>.
Although it is generally considered unethical to consider demographic factors such as age, sex, race, or ethnicity while making such decisions, and ideally this information would not be shared with triage teams tasked with allocating limited resources among patients <span class="citation" data-cites="2i1Y7SYn">(<a href="#ref-2i1Y7SYn" role="doc-biblioref">2035</a>)</span>, there are substantial concerns about implicit and explicit biases against older adults <span class="citation" data-cites="GTL02UT6">(<a href="#ref-GTL02UT6" role="doc-biblioref">2036</a>)</span>, premature infants <span class="citation" data-cites="KQfJ4llw">(<a href="#ref-KQfJ4llw" role="doc-biblioref">2037</a>)</span>, and people with disabilities or comorbidities <span class="citation" data-cites="K6yc6elC 1EaLRFk21 2i1Y7SYn">(<a href="#ref-2i1Y7SYn" role="doc-biblioref">2035</a>, <a href="#ref-K6yc6elC" role="doc-biblioref">2038</a>, <a href="#ref-1EaLRFk21" role="doc-biblioref">2039</a>)</span>.
Because of the greater burden of chronic disease in populations subjected to systemic racism, algorithms intended to be blind to race and ethnicity could, in fact, reinforce systemic inequalities caused by structural racism <span class="citation" data-cites="cPsuWT7U 1D1FX4zQZ LrqhFVIz">(<a href="#ref-cPsuWT7U" role="doc-biblioref">2040</a>–<a href="#ref-LrqhFVIz" role="doc-biblioref">2042</a>)</span>.
Because of this inequality, it has been argued that groups facing health disparities should be prioritized by these algorithms <span class="citation" data-cites="PFHCN5ep">(<a href="#ref-PFHCN5ep" role="doc-biblioref">2043</a>)</span>.
This approach would carry its own ethical concerns, including the fact that many resources that need to be distributed do not have well-established risks and benefits <span class="citation" data-cites="PFHCN5ep">(<a href="#ref-PFHCN5ep" role="doc-biblioref">2043</a>)</span>.</p>
<p>As the pandemic has progressed, it has become clear that ICU beds and ventilators are not the only limited resources that needs to be allocated, and, in fact, the survival rate for patients who receive mechanical ventilation is lower than these discussions would suggest <span class="citation" data-cites="1ExRX1x4C">(<a href="#ref-1ExRX1x4C" role="doc-biblioref">2044</a>)</span>.
Allocation of interventions that may reduce suffering, including palliative care, has become critically important <span class="citation" data-cites="1ExRX1x4C ztTunmqF">(<a href="#ref-1ExRX1x4C" role="doc-biblioref">2044</a>, <a href="#ref-ztTunmqF" role="doc-biblioref">2045</a>)</span>.
The ambiguities surrounding the risks and benefits associated with therapeutics that have been approved under emergency use authorizations also present ethical concerns related to the distribution of resources <span class="citation" data-cites="PFHCN5ep">(<a href="#ref-PFHCN5ep" role="doc-biblioref">2043</a>)</span>.
For example, remdesivir, discussed above, is currently available for the treatment of COVID-19 under compassionate use guidelines and through expanded access programs, and in many cases has been donated to hospitals by Gilead <span class="citation" data-cites="18et5lcBc 11gslRJJU">(<a href="#ref-18et5lcBc" role="doc-biblioref">2046</a>, <a href="#ref-11gslRJJU" role="doc-biblioref">2047</a>)</span>.
Regulations guiding the distribution of drugs in situations like these typically do not address how to determine which patients receive them <span class="citation" data-cites="11gslRJJU">(<a href="#ref-11gslRJJU" role="doc-biblioref">2047</a>)</span>.
Prioritizing marginalized groups for treatment with a drug like remdesivir would also be unethical because it would entail disproportionately exposing these groups to a therapeutic that may or not be beneficial <span class="citation" data-cites="PFHCN5ep">(<a href="#ref-PFHCN5ep" role="doc-biblioref">2043</a>)</span>.
On the other hand, given that the drug is one of the most promising treatments available for many patients, using a framework that tacitly feeds into structural biases would also be unethical.
At present, the report prepared for the Director of the CDC by Ethics Subcommittee of the CDC fails to address the complexity of this ethical question given the state of structural racism in the United States, instead stating that “prioritizing individuals according to their chances for short-term survival also avoids ethically irrelevant considerations, such as race or socioeconomic status” <span class="citation" data-cites="ugEOYgfx">(<a href="#ref-ugEOYgfx" role="doc-biblioref">2048</a>)</span>.
In many cases, experimental therapeutics are made available only through participation in clinical trials <span class="citation" data-cites="1A5E2TkBS">(<a href="#ref-1A5E2TkBS" role="doc-biblioref">2049</a>)</span>.
However, given the history of medical trials abusing minority communities, especially Black Americans, there is a history of unequal representation in clinical trial enrollment <span class="citation" data-cites="1A5E2TkBS">(<a href="#ref-1A5E2TkBS" role="doc-biblioref">2049</a>)</span>.
As a result, the standard practice of requiring enrollment in a clinical trial in order to receive experimental treatment may also reinforce patterns established by systemic racism.
<!--Risk of comorbid health conditions associated with more severe outcomes may be influenced by long-term damage caused by chronic stress related to traumatic social experiences [@doi:10.1007/s12170-013-0338-5], perhaps mediated by cardiovascular risk factors [@doi:10.1101/2020.05.10.20097253], although the effects of chronic stress have not yet been researched in the specific case of COVID-19 disparities.--></p>
<h3 data-number="11.3.4" id="access-to-and-representation-in-clinical-trials"><span class="header-section-number">11.3.4</span> Access to and Representation in Clinical Trials</h3>
<p>Experimental treatments are often made available to patients primarily or even exclusively through clinical trials.
The advantage of this approach is that clinical trials are designed to collect rigorous data about the effects of a treatment on patients.
The disadvantage is that access to clinical trials is not equal among all people who suffer from a disease.
Two important considerations that can impact an individual’s access to clinical trials are geography and social perceptions of clinical trials.
For the first, the geographic distribution of trial recruitment efforts are typically bounded and can vary widely among difference locations, and for the second, the social context of medical interactions can impact strategies for and the success of outreach to different communities.
Differential access to clinical trials raises concerns because it introduces biases that can influence scientific and medical research on therapeutics and prophylactics broadly.
Concerns about bias in clinical trials need to address both trial recruitment and operation.
In the present crisis, such biases are particularly salient because COVID-19 is a disease of global concern.
Treatment is needed by people all over the world, and clinical research that characterizes treatment outcomes in a variety of populations is critically important.</p>
<p>Global representation in clinical trials is important to ensuring that experimental treatments are available equally to COVID-19 patients who may need them.
The advantage to a patient of participation in a clinical trial is that they may receive an experimental treatment they would not have been able to access otherwise.
The potential downsides of participation include that the efficacy and side effects of such treatments are often poorly characterized and that patients who enroll in clinical trials will in some cases run the risk of being assigned to a placebo condition where they do not receive the treatment but miss out on opportunities to receive other treatments.
The benefits and burdens of clinical trials therefore need to be weighed carefully to ensure that they don’t reinforce existing health disparities.
The WHO Director‐General Tedros Adhanom Ghebreyesus stated his condemnation of utilizing low and middle income countries as test subjects for clinical trials, yet having highly developed countries as the majority of clinical trial representation is also not the answer <span class="citation" data-cites="TFKtmAIg">(<a href="#ref-TFKtmAIg" role="doc-biblioref">2050</a>)</span>.
Figure <a href="#fig:ebm-map">13</a> showcases two choropleths detailing COVID-19 clinical trial recruitment by country.
China, the United States, and France are among the countries with the most clinical trial recruiting for trials with single-country enrollment.
Many countries have little to no clinical trial recruiting, with the continents of Africa and South America much less represented than Asia, Europe, and North America.
Trials that recruit across multiple countries do appear to broaden geographic representation, but these trials seem to be heavily dominated by the United States and European Union.</p>
<div id="fig:ebm-map" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/dfc6ff85488d57cbb52f4b2324531be6afaa73f4/ebmdatalab/ebmdatalab-map.png" alt="Figure 13: Geographic distribution of COVID-19 clinical trials. The density of clinical trials is reported at the country level. As of December 31, 2020, there are 6,987 trials in the University of Oxford Evidence-Based Medicine Data Lab’s COVID-19 TrialsTracker (739), of which 3,962 are interventional. The top figure demonstrates the density of interventional trials recruiting only from a singular country, while the bottom shows the distribution of recruitment for interventional trials that involve more than one country." />
<figcaption aria-hidden="true"><span>Figure 13:</span> <strong>Geographic distribution of COVID-19 clinical trials.</strong>
The density of clinical trials is reported at the country level.
As of December 31, 2020, there are 6,987 trials in the University of Oxford Evidence-Based Medicine Data Lab’s COVID-19 TrialsTracker <span class="citation" data-cites="SSbnPnzT">(<a href="#ref-SSbnPnzT" role="doc-biblioref">739</a>)</span>, of which 3,962 are interventional.
The top figure demonstrates the density of interventional trials recruiting only from a singular country, while the bottom shows the distribution of recruitment for interventional trials that involve more than one country.</figcaption>
</figure>
</div>
<p>A few different concerns arise from this skewed geographic representation in clinical trial recruitment.
First, treatments such as remdesivir that are promising but primarily available to clinical trial participants are unlikely to be accessible by people in many countries.
Second, it raises the concern that the findings of clinical trials will be based on participants from many of the wealthiest countries, which may lead to ambiguity in whether the findings can be extrapolated to COVID-19 patients elsewhere.
Especially with the global nature of COVID-19, equitable access to therapeutics and vaccines has been a concern at the forefront of many discussions about policy (e.g., <span class="citation" data-cites="MYW3TeXJ">(<a href="#ref-MYW3TeXJ" role="doc-biblioref">2051</a>)</span>, yet data like that shown in Figure <a href="#fig:ebm-map">13</a> demonstrates that accessibility is likely to be a significant issue.
Another concern with the heterogeneous international distribution of clinical trials is that the governments of countries leading these clinical trials might prioritize their own populations once vaccines are developed, causing unequal health outcomes <span class="citation" data-cites="rh28K8GB">(<a href="#ref-rh28K8GB" role="doc-biblioref">2052</a>)</span>.
Additionally, even within a single state in the United States (Maryland), geography was found to influence the likelihood of being recruited into or enrolled in a clinical trial, with patients in under-served rural areas less likely to enroll <span class="citation" data-cites="RZiEc6kh">(<a href="#ref-RZiEc6kh" role="doc-biblioref">2053</a>)</span>.
Thus, geography both on the global and local levels may influence when treatments and vaccines are available and who is able to access them.
Efforts such as the African Union’s efforts to coordinate and promote vaccine development <span class="citation" data-cites="BLKtWt4w">(<a href="#ref-BLKtWt4w" role="doc-biblioref">2054</a>)</span> are therefore critical to promoting equity in the COVID-19 response.</p>
<p>Even when patients are located within the geographic recruitment area of clinical trials, however, there can still be demographic inequalities in enrollment.
When efforts are made to ensure equal opportunity to participate in clinical trials, there is no significant difference in participation among racial/ethnic groups <span class="citation" data-cites="FasLN7DR">(<a href="#ref-FasLN7DR" role="doc-biblioref">2055</a>)</span>.
However, within the United States, real clinical trial recruitment numbers have indicated for many years that racial minorities, especially African-Americans, tend to be under-represented (e.g., <span class="citation" data-cites="14jpPBgHd GNMGw3Ff ZdYcndGw QNqutuqe">(<a href="#ref-14jpPBgHd" role="doc-biblioref">2056</a>–<a href="#ref-QNqutuqe" role="doc-biblioref">2059</a>)</span>).
This trend is especially concerning given the disproportionate impact of COVID-19 on African-Americans.
Early evidence suggests that the proportion of Black, Latinx, and Native American participants in clinical trials for drugs such as remdesivir is much lower than the representation of these groups among COVID-19 patients <span class="citation" data-cites="2DrMhVAo">(<a href="#ref-2DrMhVAo" role="doc-biblioref">2060</a>)</span>.</p>
<p>One proposed explanation for differences among racial and ethnic groups in clinical trial enrollment refers to different experiences in healthcare settings.
While some plausible reasons for the disparity in communication between physicians and patients could be a lack of awareness and education, mistrust in healthcare professionals, and a lack of health insurance <span class="citation" data-cites="FasLN7DR">(<a href="#ref-FasLN7DR" role="doc-biblioref">2055</a>)</span>, a major concern is that patients from certain racial and ethnic groups are marginalized even while seeking healthcare.
In the United States, many patients experience “othering” from physicians and other medical professionals due to their race or other external characteristics such as gender (e.g., <span class="citation" data-cites="M7sLngTJ">(<a href="#ref-M7sLngTJ" role="doc-biblioref">2061</a>)</span>).
Many studies have sought to characterize implicit biases in healthcare providers and whether they affect their perceptions or treatment of patients.
A systematic review that examined 37 such studies reported that most (31) identified racial and/or ethnic biases in healthcare providers in many different roles, although the evidence about whether these biases translated to different attitudes towards patients was mixed <span class="citation" data-cites="sDygnf7I">(<a href="#ref-sDygnf7I" role="doc-biblioref">2062</a>)</span>, with similar findings reported by a second systematic review <span class="citation" data-cites="upbPnFPI">(<a href="#ref-upbPnFPI" role="doc-biblioref">2063</a>)</span>.
However, data about real-world patient outcomes are very limited, with most studies relying on clinical vignette-based exercises <span class="citation" data-cites="sDygnf7I">(<a href="#ref-sDygnf7I" role="doc-biblioref">2062</a>)</span>, and other analyses suggest that physician implicit bias could impact the patient’s perception of the negativity/positivity of the interaction regardless of the physician’s explicit behavior towards the patient <span class="citation" data-cites="AaFVjNFc">(<a href="#ref-AaFVjNFc" role="doc-biblioref">2064</a>)</span>.
Because racism is a common factor in both, negative patient experiences with medical professionals are likely to compound other issues of systemic inequality, such as a lack of access to adequate care, a lack of insurance, or increased exposure to SARS-CoV-2 <span class="citation" data-cites="smVwFlNn">(<a href="#ref-smVwFlNn" role="doc-biblioref">2065</a>)</span>.
Furthermore, the experience of being othered is not only expected to impact patients’ trust in and comfort with their provider, but also may directly impact whether or not the patient is offered the opportunity to participate in a clinical trial at all.
Some studies suggest communication between physicians and patients impacts whether or not a physician offers a patient participation in a clinical trial.
For example, researchers utilized a linguistic analysis to assess mean word count of phrases related to clinical trial enrollment, such as voluntary participation, clinical trial, etc. <span class="citation" data-cites="FasLN7DR">(<a href="#ref-FasLN7DR" role="doc-biblioref">2055</a>)</span>.
The data indicated that the mean word count of the entire visit was 1.5 times more for white patients in comparison to Black patients.
In addition, the greatest disparity between white and Black patients’ experience was the discussion of risks, with over 2 times as many risk-related words spoken with white patients than Black patients <span class="citation" data-cites="FasLN7DR">(<a href="#ref-FasLN7DR" role="doc-biblioref">2055</a>)</span>.
The trends observed for other clinical trials raise the concern that COVID-19 clinical trial information may not be discussed as thoroughly or as often with Black patients compared to white patients.</p>
<p>These discrepancies are especially concerning given that many COVID-19 treatments are being or are considered being made available to patients prior to FDA approval through Emergency Use Authorizations.
In the past, African-Americans have been over-represented relative to national demographics in use of the FDA’s Exception From Informed Consent (EFIC) pathway <span class="citation" data-cites="Vp7KObiA">(<a href="#ref-Vp7KObiA" role="doc-biblioref">2066</a>)</span>.
Through this pathway, people who are incapacitated can receive an experimental treatment even if they are not able to consent and there is not sufficient time to seek approval from an authorized representative.
This pathway presents concerns, however, when it is considered in the context of a long history of systematic abuses in medical experimentation where informed consent was not obtained from people of color, such as the Tuskegee syphilis experiments <span class="citation" data-cites="JNha7DEl">(<a href="#ref-JNha7DEl" role="doc-biblioref">2067</a>)</span>.
While the goal of EFIC approval is to provide treatment to patients who urgently need it, the combination of the ongoing legacy of racism in medicine renders this trend concerning.
With COVID-19, efforts to prioritize people who suffer from systemic racism are often designed with the goal of righting some of these inequalities (e.g., <span class="citation" data-cites="i53Z5Ye8">(<a href="#ref-i53Z5Ye8" role="doc-biblioref">2068</a>)</span>), but particular attention to informed consent will be imperative in ensuring these trials are ethical given that the benefits and risks of emerging treatments are still poorly characterized.
Making a substantial effort to run inclusive clinical trials is also important because of the possibility that racism could impact how a patient responds to a treatment.
For example, as discussed above, dexamethasone has been identified as a promising treatment for patients experiencing cytokine release syndrome, but the mechanism of action is tied to the stress response.
A study from 2005 reported that Black asthma patients showed reduced responsiveness to dexamethasone in comparison to white patients and suggested Black patients might therefore require higher doses of the drug <span class="citation" data-cites="GP85Y1Jo">(<a href="#ref-GP85Y1Jo" role="doc-biblioref">2069</a>)</span>.
In the context of chronic stress caused by systemic racism, this result is not surprising: chronic stress is associated with dysregulated production of glucocorticoids <span class="citation" data-cites="gkEMgdku">(<a href="#ref-gkEMgdku" role="doc-biblioref">2070</a>)</span> and glucocorticoid receptor resistance <span class="citation" data-cites="BlEZu684">(<a href="#ref-BlEZu684" role="doc-biblioref">2071</a>)</span>.
However, it underscores the critical need for treatment guidelines to take into account differences in life experience, which would be facilitated by the recruitment of patients from a wide range of backgrounds.
Attention to the social aspects of clinical trial enrollment must therefore be an essential component of the medical research community’s response to COVID-19.</p>
<!--Table for future version: ### Indirect Effects of the Pandemic
Besides the direct harms caused by infection, many populations are in indirect risk of serious harm due to the social and economic effects of the pandemic and of the efforts needed to fight it.
These might include individuals with substance use disorder [@doi:10.7326/M20-1141], victims of natural disasters [@doi:10.1038/s41558-020-0804-2], and victims of human trafficking [@doi:10.1016/j.eclinm.2020.100409].
The pandemic could also delay the fight against other major infectious diseases, such as HIV, malaria and tuberculosis, potentially leading to a further increase of mortality [@doi:10.25561/78670].
Although they are beyond the scope of this paper, further research is needed in order to prevent these harms.
[There are already some policy recommendations in literature that we cite - we should probably direct folks there. I think we probably want a stronger synthesis and closing than this] -->
<h2 data-number="11.4" id="conclusions-and-future-directions"><span class="header-section-number">11.4</span> Conclusions and Future Directions</h2>
<p>As the COVID-19 pandemic evolves, the scientific community’s response will be critical for identifying potential pharmacological and biotechnological developments that may aid in combating the virus and the disease it causes.
However, this global crisis highlights the importance of mounting a response based on collaboration among a wide variety of disciplines.
Understanding the basic science of the virus and its pathogenesis is imperative for identifying and envisioning possible diagnostic and therapeutic approaches; understanding how social factors can influence outcomes and shape implementation of a response is critical to disseminating any scientific advancements.
Summarizing such a complex and ever-changing topic presents a number of challenges.
This review represents the effort of over 50 contributors <!-- TO DO: check with final number -->to distill and interpret the available information.
However, this text represents a dynamic and evolving document, and we welcome continued contributions from all researchers who have insights into how these topics intersect.
A multidisciplinary perspective is critical to understanding this evolving crisis, and in this review we seek to use open science tools to coordinate a response among a variety of researchers.
We intend to publish additional updates as the situation evolves.</p>
<!-- Additional info on inflammation removed from pathogenesis manuscript, can use here if useful
Inflammation is one of the most visible components of the immune response, as it is responsible for the hallmarks of injury, such as pain, heat, and swelling [@doi:10.18632/oncotarget.23208].
The inflammatory response has received particular attention for its role in both a healthy response to infection and a pathogenic one.
In response to injury or to signaling by pattern recognition receptors indicating the detection of a molecular pattern associated with a pathogen or foreign body, the immune system stimulates leukocytes that travel to the site of the threat, where they then produce cytokines [@doi:10.18632/oncotarget.23208].
Elevated levels of inflammation over the long-term are associated with many chronic health conditions, including type 2 diabetes, dementia and Alzheimer's, and arthritis [@doi:10.1038/s41591-019-0675-0].
Some notable pro-inflammatory cytokines include the interleukins IL-1&beta; and IL-6 and tumor necrosis factor &alpha; (TNF-&alpha;) [@doi:10.1097/AIA.0b013e318034194e].
A number of interleukins and interferons play anti-inflammatory roles, and receptors or receptor antagonists for inflammatory cytokines are also important for regulating inflammation [@doi:10.1097/AIA.0b013e318034194e].
IL-10 is an anti-inflammatory cytokine of particular note because it regulates the expression of TNF-&alpha;, IL-1, and IL-6 [@doi:10.1097/AIA.0b013e318034194e].

Symptoms of a disease can be caused by a pathogen, but they can also be caused by the immune system's reaction to the pathogen.
Cytokines are a diverse group of small proteins that play an important role in intercellular signaling [@doi:10.1128/MMBR.05015-11].
They can be both pro- and anti-inflammatory, meaning they can either stimulate or inhibit the production of additional cytokines [@doi:10.1128/MMBR.05015-11; @doi:10.1097/AIA.0b013e318034194e], with anti-inflammatory cytokines playing an immunoregulatory role complementary to the cascading effect of pro-inflammatory cytokines [@doi:10.1128/MMBR.05015-11; @doi:10.1097/AIA.0b013e318034194e].
When the pro- and anti-inflammatory responses are both commensurate with the threat posed, the immune system drives a shift back to homeostasis [@doi:10.1111/imr.12671].
-->
<h1 data-number="12" id="an-open-publishing-response-to-the-covid-19-infodemic"><span class="header-section-number">12</span> An Open-Publishing Response to the COVID-19 Infodemic</h1>
<h2 data-number="12.1" id="abstract-7"><span class="header-section-number">12.1</span> ABSTRACT</h2>
<p>The COVID-19 pandemic catalyzed the rapid dissemination of papers and preprints investigating the disease and its associated virus, SARS-CoV-2.
The multifaceted nature of COVID-19 demands a multidisciplinary approach, but the urgency of the crisis combined with the need for social distancing measures present unique challenges to collaborative science.
We applied a massive online open publishing approach to this problem using Manubot.
Through GitHub, collaborators summarized and critiqued COVID-19 literature, creating a review manuscript.
Manubot automatically compiled citation information for referenced preprints, journal publications, websites, and clinical trials.
Continuous integration workflows retrieved up-to-date data from online sources nightly, regenerating some of the manuscript’s figures and statistics.
Manubot rendered the manuscript into PDF, HTML, LaTeX, and DOCX outputs, immediately updating the version available online upon the integration of new content.
Through this effort, we organized over 50 scientists from a range of backgrounds who evaluated over 1,500 sources and developed seven literature reviews.
While many efforts from the computational community have focused on mining COVID-19 literature, our project illustrates the power of open publishing to organize both technical and non-technical scientists to aggregate and disseminate information in response to an evolving crisis.</p>
<h2 class="unnumbered" id="keywords">KEYWORDS</h2>
<p>COVID-19, living document, open publishing, open-source, data integration, manubot</p>
<h2 data-number="12.2" id="introduction-7"><span class="header-section-number">12.2</span> INTRODUCTION</h2>
<p>Coronavirus Disease 2019 (COVID-19) caused a worldwide public health crisis that has reshaped many aspects of society.
The scientific community has, in turn, devoted significant attention and resources towards COVID-19 and the associated virus, SARS-CoV-2, resulting in the release of data and publications at a rate and scale never previously seen for a single topic.
Over 20,000 articles about COVID-19 were released in the first four months of the pandemic <span class="citation" data-cites="7ub6VM4Z">(<a href="#ref-7ub6VM4Z" role="doc-biblioref">2072</a>)</span>, causing an “infodemic” <span class="citation" data-cites="7ub6VM4Z nnfOazAC">(<a href="#ref-7ub6VM4Z" role="doc-biblioref">2072</a>, <a href="#ref-nnfOazAC" role="doc-biblioref">2073</a>)</span>.
The COVID-19 Open Research Dataset (CORD-19) <span class="citation" data-cites="CiOwklc6">(<a href="#ref-CiOwklc6" role="doc-biblioref">2074</a>)</span>, which was developed in part with the goal of training machine learning algorithms on COVID-19-related text, illustrates the growth of related scholarly literature (Figure <a href="#fig:cord19-growth">14</a>).
This resource was developed by querying several sources for terms related to SARS-CoV-2 and COVID-19, as well as the coronaviruses SARS-CoV-1 and MERS-CoV and their associated diseases <span class="citation" data-cites="CiOwklc6">(<a href="#ref-CiOwklc6" role="doc-biblioref">2074</a>)</span>.
CORD-19 contained 1056660 manuscripts as of 2022-06-02.
Additional curation by CoronaCentral <span class="citation" data-cites="Ybg667S0">(<a href="#ref-Ybg667S0" role="doc-biblioref">2075</a>)</span> has produced, at present, a set of over 180,000 publications particularly relevant to COVID-19 and closely related viruses.
Despite many advances in understanding the virus and the disease, there are also downsides to the availability of so much information.
“Excessive publication” has been recognized as a concern for over forty years <span class="citation" data-cites="DfSr1Ohc">(<a href="#ref-DfSr1Ohc" role="doc-biblioref">2076</a>)</span> and has been discussed with respect to the COVID-19 literature <span class="citation" data-cites="dUvvJvv6">(<a href="#ref-dUvvJvv6" role="doc-biblioref">992</a>)</span>.
Any effort to synthesize, summarize, and contextualize COVID-19 research will face a vast corpus of potentially relevant material.</p>
<div id="fig:cord19-growth" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/87225c9935edc816c9c6d8d0e049af7b60b36c95/CORD-19/cord19-growth.png" alt="Figure 14: Growth of the CORD-19 dataset. The number of articles has proliferated, with both traditional and preprint manuscripts in the corpus. The first release (March 16, 2020) contained 28,000 documents (2074). As of 2022-06-02, this had increased to 1056660 articles. Of these, 42343 are preprints from arXiv, medRxiv, and bioRxiv." />
<figcaption aria-hidden="true"><span>Figure 14:</span> <strong>Growth of the CORD-19 dataset.</strong>
The number of articles has proliferated, with both traditional and preprint manuscripts in the corpus.
The first release (March 16, 2020) contained 28,000 documents <span class="citation" data-cites="CiOwklc6">(<a href="#ref-CiOwklc6" role="doc-biblioref">2074</a>)</span>.
As of 2022-06-02, this had increased to 1056660 articles.
Of these, 42343 are preprints from <em>arXiv</em>, <em>medRxiv</em>, and <em>bioRxiv</em>.</figcaption>
</figure>
</div>
<p>Information was released rapidly by both traditional publishers and preprint servers, and many papers faced subsequent scrutiny.
The number of COVID-19 papers retracted may be higher, and potentially much higher, than is typical, although a thorough investigation of this question requires more time to elapse <span class="citation" data-cites="ZUk10707 caxpZEmy">(<a href="#ref-ZUk10707" role="doc-biblioref">2077</a>, <a href="#ref-caxpZEmy" role="doc-biblioref">2078</a>)</span>.
Many papers and preprints are also associated with corrections or expressions of concern<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a> <span class="citation" data-cites="caxpZEmy">(<a href="#ref-caxpZEmy" role="doc-biblioref">2078</a>)</span>.
Preprints are released prior to peer review, but some traditional publishing venues have fast-tracked COVID-19 papers through peer review, leading to questions about whether they are held to typical standards <span class="citation" data-cites="1Dez1ZOc5">(<a href="#ref-1Dez1ZOc5" role="doc-biblioref">2079</a>)</span>.
Therefore, evaluating the COVID-19 literature requires not only digesting available information but also monitoring subsequent changes.</p>
<p>Because of the fast-moving nature of the topic, many efforts to summarize and synthesize the COVID-19 literature have been undertaken.
These efforts include newsletters<a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a> <span class="citation" data-cites="JdWiPJCL">(<a href="#ref-JdWiPJCL" role="doc-biblioref">2080</a>)</span>, web portals<a href="#fn3" class="footnote-ref" id="fnref3" role="doc-noteref"><sup>3</sup></a> <span class="citation" data-cites="1CBWvhTdy">(<a href="#ref-1CBWvhTdy" role="doc-biblioref">2081</a>)</span> or the now-defunct http://covidpreprints.com<a href="#fn4" class="footnote-ref" id="fnref4" role="doc-noteref"><sup>4</sup></a>, comments on preprint servers<a href="#fn5" class="footnote-ref" id="fnref5" role="doc-noteref"><sup>5</sup></a> <span class="citation" data-cites="YZ4cHNuH">(<a href="#ref-YZ4cHNuH" role="doc-biblioref">2082</a>)</span>, and even a journal<a href="#fn6" class="footnote-ref" id="fnref6" role="doc-noteref"><sup>6</sup></a>.
However, the explosive rate of publication presents challenges for such efforts, many of which are no longer active.
Similarly, many literature reviews have been written on the available COVID-19 literature <span class="citation" data-cites="I2EsJmfs evtsR3C5 18eCxyLhx SAE5ME3N xOs5ctsW">(<a href="#ref-I2EsJmfs" role="doc-biblioref">2083</a>–<a href="#ref-xOs5ctsW" role="doc-biblioref">2087</a>)</span>, but static reviews quickly become outdated as new research is released or existing research is retracted or superseded.
One example is a review of topics in COVID-19 research including vaccine development <span class="citation" data-cites="xOs5ctsW">(<a href="#ref-xOs5ctsW" role="doc-biblioref">2087</a>)</span>.
This review was published on July 10, 2020, four days before Moderna released the surprisingly promising results of their phase 1 trial <span class="citation" data-cites="wiGjCZC8">(<a href="#ref-wiGjCZC8" role="doc-biblioref">1118</a>)</span> that changed expectations surrounding vaccines.
Therefore, the COVID-19 publishing climate presented a challenge where curation of the literature by a diverse group of experts in a format that could respond quickly to high-volume, high-velocity information was desirable.</p>
<p>We therefore sought to develop a platform for scientific discussion and collaboration around COVID-19 by adapting open publishing infrastructure to accommodate the scale of COVID-19 publishing.
Recent advances in open publishing have created an infrastructure that facilitates distributed, version-controlled collaboration on manuscripts <span class="citation" data-cites="YuJbg3zO">(<a href="#ref-YuJbg3zO" role="doc-biblioref">2088</a>)</span>.
Manubot <span class="citation" data-cites="YuJbg3zO">(<a href="#ref-YuJbg3zO" role="doc-biblioref">2088</a>)</span> is a collaborative framework developed to adapt open-source software development techniques and version control for manuscript writing.
With Manubot, manuscripts are managed and maintained using GitHub, a popular, online version control interface.
We selected Manubot because it offers several advantages over comparable collaborative writing platforms such as Authorea, Overleaf, Google Docs, Word Online, or wikis <span class="citation" data-cites="YuJbg3zO">(<a href="#ref-YuJbg3zO" role="doc-biblioref">2088</a>)</span>.
Citation-by-identifier ensures consistent reference metadata standards that would be difficult to maintain manually in a manuscript with dozens of authors and over 1,500 citations.
Manubot’s pull request-based contribution model balances the goals of making the project open to everyone and maintaining scientific accuracy.
All contributions are reviewed, discussed, and formally approved on GitHub before text updates appear in the public-facing manuscript<a href="#fn7" class="footnote-ref" id="fnref7" role="doc-noteref"><sup>7</sup></a>.
Continuous integration (CI) seamlessly combines author-produced text and figures with automatically generated and updated statistics and figures derived from external data sources and the manuscript’s own content.
In addition, the authors who initially launched this project included Manubot developers who had prior successes using Manubot for massively open and traditional manuscripts.</p>
<p>Collaboration via massively open online papers has been identified as a strategy for promoting inclusion and interdisciplinary thought <span class="citation" data-cites="PoDz2q0A">(<a href="#ref-PoDz2q0A" role="doc-biblioref">2089</a>)</span>.
However, the Manubot workflow can be intimidating to contributors who are not well-versed in git <span class="citation" data-cites="PoDz2q0A">(<a href="#ref-PoDz2q0A" role="doc-biblioref">2089</a>)</span>.
The synthesis and discussion of the emerging literature by biomedical scientists and clinicians is imperative to a robust interpretation of COVID-19 research.
Such efforts in biology often rely on What You See Is What You Get tools such as Google Docs, despite the significant limitations of these platforms in the face of excessive publication.
We recognized that the problem of synthesizing the COVID-19 literature lent itself well to the Manubot platform, but that the potential technical expertise required to work with Manubot presented a barrier to domain experts.</p>
<p>Here, we describe the adaptation of Manubot to facilitate collaboration in the extreme case of the COVID-19 infodemic, with the objective of developing a centralized platform for summarizing and synthesizing a massive amount of preprints, news stories, journal publications, and data.
Unlike prior collaborations built on Manubot, most contributors to the COVID-19 collaborative literature review came from biology or medicine.
The members of the COVID-19 Review Consortium consolidated information about the virus in the context of related viruses and to synthesize rapidly emerging literature.
Manubot provided the infrastructure to manage contributions from the community and create a living, scholarly document integrating data from multiple sources.
Its back-end allowed biomedical scientists to sort and distill informative content out of the overwhelming flood of information <span class="citation" data-cites="1HZeeO4Cs">(<a href="#ref-1HZeeO4Cs" role="doc-biblioref">2090</a>)</span> in order to provide a resource that would be useful to the broader scientific community.
This case study demonstrates the value of open collaborative writing tools such as Manubot to emerging challenges.
Because it is open source software, we were able to adapt and customize Manubot to flexibly meet the needs of COVID-19 review.
Recording the evolution of information over time and assembling a resource that auto-updated in response to the evolving crisis revealed the particular value that Manubot holds for managing rapid changes in scientific thought.</p>
<h2 data-number="12.3" id="methods"><span class="header-section-number">12.3</span> METHODS</h2>
<h3 data-number="12.3.1" id="contributor-recruitment-and-roles"><span class="header-section-number">12.3.1</span> Contributor Recruitment and Roles</h3>
<p>First, it was necessary to establish Manubot as a platform accessible to researchers with limited experience working version control, given that this is not typically emphasized in biology and medicine <span class="citation" data-cites="1HmO21gZN OO1DuZd 4ny1onB0">(<a href="#ref-1HmO21gZN" role="doc-biblioref">2091</a>–<a href="#ref-4ny1onB0" role="doc-biblioref">2093</a>)</span>.
Contributors were recruited primarily by word of mouth and on Twitter, and we also collaborated with existing efforts to train early-career researchers.
We invited potential collaborators to contribute a short introduction on a GitHub issue in order to collect information about participants and provide an introduction to working with GitHub issues.
Interested participants were encouraged to contribute in several ways.
One option was to catalog articles of interest as issues.
We developed a standardized set of questions for contributors to consider when evaluating an article following a framework often used for assessing medical literature.
This approach emphasizes examining the methods used, assignment (whether the study was observational or randomized), assessment, results, interpretation, and how well the study extrapolates <span class="citation" data-cites="17OQtAY4l">(<a href="#ref-17OQtAY4l" role="doc-biblioref">2094</a>)</span>.
Contributors were also invited to contribute or edit text using GitHub’s pull request system.
These contributions were not strictly defined and could range from minor corrections to punctuation and grammar to large-scale additions of text.
Finally, a small number of contributors (the authors of this paper) contributed technical expertise, either through the development of standardized approaches to the evaluation of papers based on the MAARIE Framework <span class="citation" data-cites="k1GJLUxP">(<a href="#ref-k1GJLUxP" role="doc-biblioref">2095</a>)</span>, the writing of code to generate manuscript figures, or the addition of features to Manubot.
All of these additions were also submitted as pull requests, either to the COVID-19 review repository or to an external repository, as appropriate.</p>
<p>Each pull request was reviewed and approved by at least one other contributor before being merged into the main branch.
We tagged potential reviewers based on the introductions they had contributed in order to encourage participation.
Authorship was determined based on the Contributor Roles Taxonomy<a href="#fn8" class="footnote-ref" id="fnref8" role="doc-noteref"><sup>8</sup></a>.
Due to the permeability of ideas among different sections, contributors to a specific manuscript were recognized with masthead authorship, while all contributors to the project were recognized with consortium authorship on all papers.
Emphasizing the use of issues and pull requests was designed to encourage authors with and without git experience to discuss papers and provide feedback (both formal and informal) on proposed text additions or changes.
We also used the Gitter chat platform<a href="#fn9" class="footnote-ref" id="fnref9" role="doc-noteref"><sup>9</sup></a> to promote informal questions and sharing of information among collaborators.</p>
<h3 data-number="12.3.2" id="utilization-and-expansion-of-manubot"><span class="header-section-number">12.3.2</span> Utilization and Expansion of Manubot</h3>
<p>Applying Manubot’s existing capabilities allowed us to confront several challenges common in large-scale collaborations, such as maintaining a record of contributions that allowed us to allocate credit appropriately or to contact the original author if questions arose.
Additionally, an up-to-date version of the content was available at all times online in HTML<a href="#fn10" class="footnote-ref" id="fnref10" role="doc-noteref"><sup>10</sup></a> or PDF format<a href="#fn11" class="footnote-ref" id="fnref11" role="doc-noteref"><sup>11</sup></a>.
This approach also allowed us to minimize the demand on authors to curate and sync bibliographic resources.
Manubot provides the functionality to create a bibliography using digital object identifiers (DOIs), website URLs, or other identifiers such as PubMed identifiers and arXiv IDs.
The author can insert a citation in-line using a format such as <code>[@doi:10.1371/journal.pcbi.1007128]</code>.
Manubot then obtains reference metadata, exports the citations as Citation Style Language JSON Data Items, and renders the bibliographic information needed to generate the references section <span class="citation" data-cites="YuJbg3zO">(<a href="#ref-YuJbg3zO" role="doc-biblioref">2088</a>)</span>.
This approach allows multiple authors to work on a piece of text without needing to make manual adjustments to the reference lists.</p>
<p>Due to the needs of this project, several new features were implemented in Manubot.
Because of the ever-evolving nature of the COVID-19 crisis, figures and statistics in the text quickly became outdated.
To address this concern, Manubot and GitHub’s CI features were used to create figures that integrated online data sources and to dynamically update information, such as the current number of active COVID-19 clinical trials <span class="citation" data-cites="njpLhBui">(<a href="#ref-njpLhBui" role="doc-biblioref">3</a>)</span>, within the text of the manuscripts (Figure <a href="#fig:manubot-workflow">15</a>).
GitHub Actions runs a nightly workflow to update these external data and regenerate the statistics and figures for the manuscript.
The workflow uses the GitHub API to detect and save the latest commit of the external data sources that are GitHub repositories<a href="#fn12" class="footnote-ref" id="fnref12" role="doc-noteref"><sup>12</sup></a>.
It then downloads versioned data from that snapshot of the external repositories and runs bash and Python scripts to calculate the desired statistics and produce the summary figures using Matplotlib <span class="citation" data-cites="1026Gxdsi">(<a href="#ref-1026Gxdsi" role="doc-biblioref">2096</a>)</span>.
The statistics are stored in JSON files that are accessed by Manubot to populate the values of placeholder template variables dynamically every time the manuscript is built.
For instance, the template variable <code>{{ebm_trials_results}}</code> in the manuscript is replaced by the actual number of clinical trials with results, 98.
The template variables also include versioned URLs to the dynamically updated figures.
The JSON files and figures are stored in the <code>external-resources</code> branch of the GitHub repository, providing versioned storage.
The GitHub Actions workflow automatically adds and commits the new JSON files and figures to the <code>external-resources</code> branch every time it runs, and Manubot uses the latest version of these resources when it builds the manuscript.
The GitHub Actions workflow file is available online<a href="#fn13" class="footnote-ref" id="fnref13" role="doc-noteref"><sup>13</sup></a>, as are the scripts<a href="#fn14" class="footnote-ref" id="fnref14" role="doc-noteref"><sup>14</sup></a>.
The Python package versions are also available<a href="#fn15" class="footnote-ref" id="fnref15" role="doc-noteref"><sup>15</sup></a>.<!-- To Do: These files are archived with [Zenodo?](...). --></p>
<div id="fig:manubot-workflow" class="fignos">
<figure>
<img src="images/covid-19-review-workflow-horizontal.png" alt="Figure 15: COVID-19 review GitHub repository organization and workflows. Manubot uses CI to combine author-contributed content with automatically updated information from outside sources. A nightly workflow updates figures and statistics derived from external resources. Authors write text and add figures to the master branch (starred) via GitHub pull requests. Manubot generates updated manuscript outputs for each new git commit, integrating the static text and figures with the dynamic statistics and figures and automatically-extracted citation information. GitHub Pages hosts the latest HTML and PDF versions of the manuscript along with permanent links to prior versions." />
<figcaption aria-hidden="true"><span>Figure 15:</span> <strong>COVID-19 review GitHub repository organization and workflows.</strong>
Manubot uses CI to combine author-contributed content with automatically updated information from outside sources.
A nightly workflow updates figures and statistics derived from external resources.
Authors write text and add figures to the <code>master</code> branch (starred) via GitHub pull requests.
Manubot generates updated manuscript outputs for each new git commit, integrating the static text and figures with the dynamic statistics and figures and automatically-extracted citation information.
GitHub Pages hosts the latest HTML and PDF versions of the manuscript along with permanent links to prior versions.</figcaption>
</figure>
</div>
<p>Another issue identified was the need for standardized citation to clinical trials.
Other researchers identified the same need<a href="#fn16" class="footnote-ref" id="fnref16" role="doc-noteref"><sup>16</sup></a>.
Trials that are registered with clinicaltrials.gov receive a unique clinical trial identifier, or “NCT ID.”
Because clinical trials are registered long before results are published, referencing clinical trial identifiers was a priority.
Manubot uses the Zotero translation server<a href="#fn17" class="footnote-ref" id="fnref17" role="doc-noteref"><sup>17</sup></a> to extract citation metadata for some types of citations.
However, Zotero did not support clinical trial identifiers and could not extract relevant metadata from their URLs.
In order to pull clinical trial metadata associated into Manubot, we added Zotero support for these identifiers.
To achieve this, we query clinicaltrials.gov to retrieve XML metadata associated with each identifier using JavaScript<a href="#fn18" class="footnote-ref" id="fnref18" role="doc-noteref"><sup>18</sup></a>.
This extension enables citing a trial as <code>@clinicaltrials:NCT04280705</code> instead of the URL.
Then, when Manubot requests clinical trial metadata from the Zotero translation server, the response includes the trial sponsors, responsible investigators, title, and summary.
Manubot now supports directly citing hundreds of registered Compact Uniform Resource Identifiers<a href="#fn19" class="footnote-ref" id="fnref19" role="doc-noteref"><sup>19</sup></a>, beyond just the <code>clinicaltrials</code> identifier.</p>
<p>Because of the large number of citations used in this manuscript and the fast-moving nature of COVID-19 research, keeping track of retractions, corrections, and notices of concern also became a challenge.
We implemented a new Manubot plugin to support “smart citations” in the HTML build of manuscripts.
The plugin uses the scite <span class="citation" data-cites="14UJbLWf4">(<a href="#ref-14UJbLWf4" role="doc-biblioref">2097</a>)</span> service to display a badge below any citation with a DOI.
The badge contains a set of icons and numbers that indicate how many times that source has been mentioned, supported, or disputed and whether there have been any important editorial notices.
We were thus able to identify references that needed to be reevaluated by an expert.
This addition was invaluable given the nature of the project, where we were disseminating rapidly evolving information of great consequence from over 1,500 different sources.
The badges also allow readers to ascertain a rough approximation of the reliability of cited sources at a glance.</p>
<p>Because most collaborators were writing and editing text through the GitHub website rather than in a local text editor, we also needed to add spell-checking functionalities to Manubot.
We integrated an existing Pandoc<a href="#fn20" class="footnote-ref" id="fnref20" role="doc-noteref"><sup>20</sup></a> spell-check extension with AppVeyor CI to automatically post spelling errors as comments in a GitHub pull request.
The comment reported both unique misspelled tokens and all locations where the token was detected.
Project maintainers managed a custom dictionary to allow over 1,500 scientific and technical terms that were not common English words.
Spell-checking also helped standardize the writing style across dozens of authors by detecting features such as British versus American English spellings.
The actual spell-checking was implemented using GNU Aspell<a href="#fn21" class="footnote-ref" id="fnref21" role="doc-noteref"><sup>21</sup></a> and the Pandoc spellcheck filter<a href="#fn22" class="footnote-ref" id="fnref22" role="doc-noteref"><sup>22</sup></a>.
The filter enables checking only the manuscript text, ignoring URLs and formatting.</p>
<p>Manubot can render a manuscript in several formats that serve different purposes.
Prior to this project, Manubot could use Pandoc to convert the markdown-formatted manuscript to HTML, PDF, and DOCX formats.
We expanded this functionality to export individual sections of the manuscript as separate DOCX files while still rendering the complete manuscript in HTML and PDF formats.
This development was necessary because the manuscript grew so large that it needed to be split into seven separate papers for journal submission while still maintaining shared GitHub discussion across topics.
When exporting an individual section, Manubot customizes the manuscript title, authors, and author contributions to pertain to that specific section.
In addition, we expanded the export formats to include partial LaTeX support via Pandoc.
Pandoc converts the markdown content for an individual section to TeX and the Citation Style Language JSON, which contains reference metadata generated by Manubot, to BibTeX.
We customized a LaTeX template and reformatted the Manubot metadata, such as authors and their affiliations, for the LaTeX template.
The exported TeX file requires manual refinement but contains all manuscript content and most of the formatting.
Because LaTeX is required for manuscript submission in many fields, automating most of the process of converting markdown to a submission-friendly format expands Manubot’s potential user base.
Manubot users can write in the simple markdown format, render the manuscript in continuously-updated PDF or interactive HTML formats, and export the manuscript in DOCX or TeX and BibTeX for submission to traditional publishers, taking full advantage of Pandoc’s powerful document conversion capabilities and Manubot’s automation.</p>
<h2 data-number="12.4" id="results"><span class="header-section-number">12.4</span> RESULTS</h2>
<h3 data-number="12.4.1" id="recruitment-and-manuscript-development"><span class="header-section-number">12.4.1</span> Recruitment and Manuscript Development</h3>
<div id="fig:projectstats" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/2c853716f1745468c8180fd6da351de2bd8f1c00/analyze-ms-stats/manuscript_stats.png" alt="Figure 16: Project growth over time. The number of authors, word count, and number of references have all grown dramatically from when the project began on March 20, 2020. As of September 10, 2021, there were 52 authors (including consortia), 1676 references, and 138213 words. The spike in word count during summer 2020 was caused by erroneous duplication and subsequent removal of a large appendix." />
<figcaption aria-hidden="true"><span>Figure 16:</span> <strong>Project growth over time.</strong>
The number of authors, word count, and number of references have all grown dramatically from when the project began on March 20, 2020.
As of September 10, 2021, there were 52 authors (including consortia), 1676 references, and 138213 words.
The spike in word count during summer 2020 was caused by erroneous duplication and subsequent removal of a large appendix.</figcaption>
</figure>
</div>
<!--To Do: make the figure URL dynamic and add commas to large numbers-->
<p>Coverage by <em>Nature Toolbox</em> <span class="citation" data-cites="AE0QcVgJ">(<a href="#ref-AE0QcVgJ" role="doc-biblioref">2098</a>)</span> and an associated tweet<a href="#fn23" class="footnote-ref" id="fnref23" role="doc-noteref"><sup>23</sup></a> about the project on April 1, 2020 attracted the interest of the scientific community (Figure <a href="#fig:projectstats">16</a>).
Because the GitHub issues and comment systems are similar to other common web commenting systems, authors learned these tools quickly.
The Gitter chat also presented a low barrier to entry.
The manuscript continued to grow throughout the first year and a half of the project in both word count and the number of references (Figure <a href="#fig:projectstats">16</a>).
Though only a fraction of potential contributors contributed to the text included in the manuscripts (Figure <a href="#fig:projectstats">16</a>), many contributors remained engaged over the long term (Figure <a href="#fig:projectdots">17</a>).
Additionally, new contributors continued to join even into the second year of the project.</p>
<div id="fig:projectdots" class="fignos">
<figure>
<img src="https://github.com/greenelab/covid19-review/raw/e12aa314707d5a66f20e1b2179d4b29b3efef0c0/contrib-viz/covid19-review-contribution-dot.png" alt="Figure 17: User contributions to the manuscript text over time. The dot size indicates the number of words added or edited each month since March 2020. The figure does not depict other types of author contributions such as literature summaries, pull request review, visualization, or software." />
<figcaption aria-hidden="true"><span>Figure 17:</span> <strong>User contributions to the manuscript text over time.</strong>
The dot size indicates the number of words added or edited each month since March 2020.
The figure does not depict other types of author contributions such as literature summaries, pull request review, visualization, or software.</figcaption>
</figure>
</div>
<!-- To Do: use latest version of the figure after adding it to workflow -->
<p>In order to make the project more accessible, we developed resources explaining how to use GitHub’s web interface to develop and edit text for Manubot assuming no prior experience working with version control.
These tutorials explained how to open an issue, open a pull request, and review a pull request<a href="#fn24" class="footnote-ref" id="fnref24" role="doc-noteref"><sup>24</sup></a>.
Additionally, the framework for evaluating literature was converted into issue templates to simplify the review of new articles.
Articles were classified as <em>diagnostic</em>, <em>therapeutic</em>, or <em>other</em>, with an associated template developed to guide the review of papers and preprints in each category.
A total of 285 new paper issues had been opened as of September 13, 2021.<!--To Do: automate (currently have open and closed issues but not paper issues) --></p>
<p>The manuscripts produced by the consortium (excluding this one) will be submitted to <em>mSystems</em> as part of a special issue that provides support for continuous updates as more information becomes available.
One has been published and two are available as preprints.
This approach allows for a version of record to be maintained alongside the most recent version, which is always available through GitHub.
These manuscripts cover a wide range of topics including the fundamental biology of SARS-CoV-2 (pathogenesis <span class="citation" data-cites="GdZc4Yyd">(<a href="#ref-GdZc4Yyd" role="doc-biblioref">1</a>)</span> and evolution), biomedical advances in responding to the virus and COVID-19 (pharmaceuticals <span class="citation" data-cites="njpLhBui">(<a href="#ref-njpLhBui" role="doc-biblioref">3</a>)</span>, nutraceuticals <span class="citation" data-cites="wgAGKcBj">(<a href="#ref-wgAGKcBj" role="doc-biblioref">2</a>)</span>, vaccines, and diagnostic technologies), and biological and social factors influencing disease transmission and outcomes.
To date, 52 authors are associated with the consortium (Figure <a href="#fig:projectstats">16</a>).</p>
<p>More formal recruitment efforts to integrate with existing projects providing support for undergraduate students during COVID-19 were also successful.
We incorporated summaries written by the students, post-docs, and faculty of the Immunology Institute at the Mount Sinai School of Medicine<a href="#fn25" class="footnote-ref" id="fnref25" role="doc-noteref"><sup>25</sup></a> <span class="citation" data-cites="YZ4cHNuH">(<a href="#ref-YZ4cHNuH" role="doc-biblioref">2082</a>)</span>.
Additionally, two of the consortium authors were undergraduate students recruited through the American Physician Scientist Association’s Virtual Summer Research Program.
Thus, the consortium was successful in providing a venue for researchers across all career stages to continue investigating and publishing at a time when many biomedical researchers were unable to access their laboratory facilities.</p>
<h3 data-number="12.4.2" id="integrating-data"><span class="header-section-number">12.4.2</span> Integrating Data</h3>
<p>We integrated data into the manuscripts from several sources (Figure <a href="#fig:manubot-workflow">15</a>).
Worldwide cases and deaths were tracked by the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University<a href="#fn26" class="footnote-ref" id="fnref26" role="doc-noteref"><sup>26</sup></a>.
The clinical trials statistics and figure were generated based on data from the University of Oxford Evidence-Based Medicine Data Lab’s COVID-19 TrialsTracker <span class="citation" data-cites="SSbnPnzT">(<a href="#ref-SSbnPnzT" role="doc-biblioref">739</a>)</span>.
Information about vaccine distribution was extracted from Our World In Data<a href="#fn27" class="footnote-ref" id="fnref27" role="doc-noteref"><sup>27</sup></a> <span class="citation" data-cites="ZHvhFakW">(<a href="#ref-ZHvhFakW" role="doc-biblioref">2099</a>)</span>.
(Figure <a href="#fig:cord19-growth">14</a>) integrates data from the CORD-19 dataset <span class="citation" data-cites="CiOwklc6">(<a href="#ref-CiOwklc6" role="doc-biblioref">2074</a>)</span>.</p>
<p>Manubot’s bibliographic management capabilities were critical because the amount of relevant literature published far outstripped what we had anticipated at the beginning of the project.
As of September 10, 2021, there were 1676 references (Figure <a href="#fig:projectstats">16</a>).
The scite plugin provided a way to visually inspect the reference list to identify possible references of concern.
This and the other new features required for the COVID-19 project are now included in Manubot’s rootstock, which is the template GitHub repository for creating a new manuscript.
Using CI, Manubot now checks that the manuscript was built correctly, runs spell-checking, and cross-references the manuscripts cited in this review.
In addition, Manubot rootstock now supports citing clinical trial identifiers such as <code>clinicaltrials:NCT04292899</code> <span class="citation" data-cites="yTCAmOyt">(<a href="#ref-yTCAmOyt" role="doc-biblioref">809</a>)</span>.</p>
<h2 data-number="12.5" id="discussion-1"><span class="header-section-number">12.5</span> DISCUSSION</h2>
<p>The current project was based in the GitHub repository <code>greenelab/covid19-review</code> using Manubot <span class="citation" data-cites="YuJbg3zO">(<a href="#ref-YuJbg3zO" role="doc-biblioref">2088</a>)</span> to continuously generate the manuscript.
The Manubot framework facilitated a massive collaborative review on an urgent topic.
We demonstrated the utility of Manubot to a project where many contributors lacked expertise or even experience working with version control.
This effort has produced not only seven literature reviews on topics relevant to the COVID-19 pandemic, but has also generated cyberinfrastructure for training novice users in GitHub.
We also extended the functionalities of Manubot to provide more of the benefits of What You See Is What You Get platforms such as Google Docs (Table <a href="#tbl:manubot-addons">6</a>).
Open publishing thus allowed us to harness the domain expertise of a large group of non-technical users to respond to the flood of COVID-19 publications.</p>
<p>Several existing and new features in Manubot aid in responding to the challenges posed by the infodemic.
Manuscripts are written in markdown and can be rendered in several formats providing different advantages to users.
For example, beyond building just a PDF, Manubot also renders the manuscript in HTML, DOCX, and now, LaTeX (in a more limited capacity).
The interactive HTML manuscript format offers several advantages over a static PDF to harmonize available resources and address specific problems related to COVID-19.
The integration of scite into the HTML build makes references more manageable by visually indicating whether their results are contested or whether they have been corrected or retracted.
Cross-referencing different pieces of the manuscript, such as cited preprints with reviews stored in an appendix, is another interactive option presented by HTML.
The DOCX format was preferred by most non-technical users for reviewing the final version of the manuscript and was useful for creating submissions to a biological journal.
Additionally, because of the heavy emphasis on Word processing in biology, Manubot’s ability to generate DOCX outputs was expanded to allow users to generate DOCX files containing only a section of the manuscript.
In our case, where the full project is nearly 150,000 words, this allows individual pieces to be shared more easily.
Finally, the preliminary addition of LaTeX output is useful for researchers from computational fields who submit papers in TeX format and removes the step of reformatting markdown prior to submission.</p>
<div id="tbl:manubot-addons" class="tablenos">
<table id="tbl:manubot-addons">
<caption><span>Table 6:</span> Manubot extensions for the COVID-19 review.
</caption>
<colgroup>
<col style="width: 26%" />
<col style="width: 73%" />
</colgroup>
<thead>
<tr class="header">
<th>Type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>CI</td>
<td>Regularly download external data sources, generate new figures and statistics, and read them when Manubot builds the latest manuscript</td>
</tr>
<tr class="even">
<td>CI</td>
<td>Post spell-checking reports as pull request comments</td>
</tr>
<tr class="odd">
<td>Citations</td>
<td>Zotero extension to report more relevant clinical trial metadata from <a href="https://clinicaltrials.gov" class="uri">https://clinicaltrials.gov</a></td>
</tr>
<tr class="even">
<td>Citations</td>
<td>Cite any Compact Uniform Resource Identifier, such as <code>clinicaltrials</code> or <code>ncbigene</code></td>
</tr>
<tr class="odd">
<td>Citations</td>
<td>scite badges to track retractions, corrections, and notices of concern</td>
</tr>
<tr class="even">
<td>Outputs</td>
<td>Improved support for Pandoc’s LaTeX output</td>
</tr>
<tr class="odd">
<td>Outputs</td>
<td>Build complete manuscript alongside individual sections as standalone documents</td>
</tr>
</tbody>
</table>
</div>
<p>The COVID-19 Review Consortium provided a platform for researchers to engage in scientific investigation early in the pandemic when many biological scientists were unable to access their research spaces.
In turn, by seeking to adapt Manubot to allow for broader participation, we made a number of improvements that are expected to increase its appeal to researchers from all backgrounds.
Manubot provided a way for contributors from a variety of backgrounds, including early-career researchers, to join a massive collaborative project while demonstrating their individual contributions to the larger work and gaining experience with version control.
The licensing and infrastructure also provide the basis for individuals to adapt from this project to create their own snapshots of the COVID-19 literature that derive from, but are not wholly identical to, the primary versions of these reviews.
This project suggests that massive online open publishing efforts can indeed advance scholarship through inclusion <span class="citation" data-cites="PoDz2q0A">(<a href="#ref-PoDz2q0A" role="doc-biblioref">2089</a>)</span>, including during the extreme challenges presented by the COVID-19 pandemic.</p>
<p>Some challenges did arise in efforts to include an academically diverse set of authors.
The barriers to entry posed by git and GitHub likely still reduced participation from individuals who might have otherwise been interested.
Using pull requests as a tool for writing text is also unfamiliar to many or most scientists, and the review process can be slow, which might cause interested contributors to lose interest.
Additionally, the pull request model may limit people from providing general feedback on the manuscript or a section of the manuscript, unless there is an open pull request.
As a result, some feedback came through email or comments on the DOCX outputs that were then translated into issues or pull requests by the project managers.
Given that our approach hinged on these version control tools, it is likely that our group of contributors was biased towards those who were interested in or experienced with computational tools.
The trajectory of the pandemic itself also likely influenced participation: engagement waned over the course of the pandemic as labs opened back up and researchers were able to return to their work, and we recruited very few senior clinicians to the project, which is unsurprising given the load on medical professionals during this time.
Engagement that waxes and wanes is, however, typical when writing massively open online papers <span class="citation" data-cites="PoDz2q0A">(<a href="#ref-PoDz2q0A" role="doc-biblioref">2089</a>)</span>.
Adding features such as spell-check did improve usability, and additional features such as automatically checking the formatting of citations could further improve the usability of this tool.
In the future, a formal study of participation could allow for quantification of these biases and improved efforts to foster inclusion.</p>
<p>Additional limitations are challenges associated with massively open online papers in general.
With such a large amount of text, it is not possible to keep all sections of the manuscript up to date at all times.
Readers are not able to distinguish when each section was updated.
Even GitHub’s blame functionality does not distinguish minor changes from substantive updates to the text.
While much of the data and statistics update automatically, the text itself required updating by human experts.
This asynchronicity could potentially introduce incompatibility between the figures and the surrounding text.
Similarly, in line with the collaboration-related challenges of the project, some authors returned to update their text, while others did not.
As a result, the lead authors of each paper often spent several weeks prior to journal submission updating the text to reflect new developments in each area.
In the future, it may be possible to streamline this process through integration with a tool such as CoronaCentral <span class="citation" data-cites="Ybg667S0">(<a href="#ref-Ybg667S0" role="doc-biblioref">2075</a>)</span> to automatically identify relevant, high-impact papers that need to be included, although expertise would still be required to incorporate them.
Another challenge involves tracking preprints as they are reviewed or critiqued, revised, and potentially published.
While updating the content of the manuscript would likely fall to human contributors, automatic detection of published versions of preprints <span class="citation" data-cites="pESBLU4c">(<a href="#ref-pESBLU4c" role="doc-biblioref">2100</a>)</span> could be integrated in the future.
These challenges are exacerbated by the scale of the infodemic, but developing solutions would benefit future projects tracking more typical trends in publication.
Similarly, outputting machine readable summaries of key information in the COVID-19 review manuscripts could reduce their contribution to the infodemic.
As it stands, the integration of Compact Uniform Resource Identifier does make a step in this direction.
Formal identifiers could be used to extract relationships among clinical trials, genes, publications, and other entities.
Thus, the experience of using Manubot for a massive project has laid the foundation for future additions to enhance user experience and inclusivity.</p>
<h2 data-number="12.6" id="conclusion"><span class="header-section-number">12.6</span> CONCLUSION</h2>
<p>With the worldwide scientific community uniting during 2020 and 2021 to investigate COVID-19 from a wide range of perspectives, findings from many disciplines are relevant on a rapid timescale to a broad scientific audience.
As many other efforts have described, the publishing rate of formal manuscripts and preprints about COVID-19 has been unprecedented <span class="citation" data-cites="7ub6VM4Z">(<a href="#ref-7ub6VM4Z" role="doc-biblioref">2072</a>)</span>, and efforts to review the body of COVID-19 literature are faced with an ever-expanding corpus to evaluate.
In the case of the seven manuscripts produced by the COVID-19 Review Consortium, Manubot allows for continuous updating of the manuscripts as the pandemic enters its second year and the landscape shifts with the emergence of promising therapeutics and vaccines <span class="citation" data-cites="njpLhBui">(<a href="#ref-njpLhBui" role="doc-biblioref">3</a>)</span>.
These manuscripts pull data from external sources and update information and visualizations daily using CI.
By off-loading some updates to computational pipelines, domain experts can focus on the broader implications of new information as it emerges.
Centralizing, summarizing, and critiquing data and literature broadly relevant to COVID-19 can expedite the interdisciplinary scientific process that is currently happening at an advanced pace.
As of September 13, 2021, 2886 commits have been made to the manuscript across 575 merged pull requests.
The efforts of the COVID-19 Review Consortium illustrate the value of including open source tools, including those focused on open publishing, in these efforts.
By facilitating the versioning of text, such platforms also allow for documentation of the evolution of thought in an evolving area and formal analysis of a collaborative project.
This application of version control holds the potential to improve scientific publishing in a range of disciplines, including those outside of traditional computational fields.
While Manubot is a technologically complex tool, this project demonstrates that it can be applied to a variety of projects.
Future work can address remaining limitations and continue to advance Manubot as an inclusive tool for open publishing projects.</p>
<h1 class="page_break_before" data-number="13" id="additional-items"><span class="header-section-number">13</span> Additional Items</h1>
<h2 data-number="13.1" id="competing-interests"><span class="header-section-number">13.1</span> Competing Interests</h2>
<table>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<thead>
<tr class="header">
<th>Author</th>
<th>Competing Interests</th>
<th>Last Reviewed</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Halie M. Rando</td>
<td>None</td>
<td>2021-01-20</td>
</tr>
<tr class="even">
<td>Casey S. Greene</td>
<td>None</td>
<td>2021-01-20</td>
</tr>
<tr class="odd">
<td>Michael P. Robson</td>
<td>None</td>
<td>2020-11-12</td>
</tr>
<tr class="even">
<td>Simina M. Boca</td>
<td>Currently an employee at AstraZeneca, Gaithersburg, MD, USA, may own stock or stock options.</td>
<td>2021-07-01</td>
</tr>
<tr class="odd">
<td>Nils Wellhausen</td>
<td>None</td>
<td>2020-11-03</td>
</tr>
<tr class="even">
<td>Ronan Lordan</td>
<td>None</td>
<td>2020-11-03</td>
</tr>
<tr class="odd">
<td>Christian Brueffer</td>
<td>Employee and shareholder of SAGA Diagnostics AB.</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Sandipan Ray</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="odd">
<td>Lucy D'Agostino McGowan</td>
<td>Received consulting fees from Acelity and Sanofi in the past five years</td>
<td>2020-11-10</td>
</tr>
<tr class="even">
<td>Anthony Gitter</td>
<td>Filed a patent application with the Wisconsin Alumni Research Foundation related to classifying activated T cells</td>
<td>2020-11-10</td>
</tr>
<tr class="odd">
<td>Anna Ada Dattoli</td>
<td>None</td>
<td>2020-03-26</td>
</tr>
<tr class="even">
<td>Ryan Velazquez</td>
<td>None</td>
<td>2020-11-10</td>
</tr>
<tr class="odd">
<td>John P. Barton</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Jeffrey M. Field</td>
<td>None</td>
<td>2020-11-12</td>
</tr>
<tr class="odd">
<td>Bharath Ramsundar</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Adam L. MacLean</td>
<td>None</td>
<td>2021-02-23</td>
</tr>
<tr class="odd">
<td>Alexandra J. Lee</td>
<td>None</td>
<td>2020-11-09</td>
</tr>
<tr class="even">
<td>Immunology Institute of the Icahn School of Medicine</td>
<td>None</td>
<td>2020-04-07</td>
</tr>
<tr class="odd">
<td>Fengling Hu</td>
<td>None</td>
<td>2020-04-08</td>
</tr>
<tr class="even">
<td>Nafisa M. Jadavji</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="odd">
<td>Elizabeth Sell</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Vincent Rubinetti</td>
<td>None</td>
<td>2021-04-29</td>
</tr>
<tr class="odd">
<td>Jinhui Wang</td>
<td>None</td>
<td>2021-01-21</td>
</tr>
<tr class="even">
<td>Diane N. Rafizadeh</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="odd">
<td>Ashwin N. Skelly</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Marouen Ben Guebila</td>
<td>None</td>
<td>2021-08-02</td>
</tr>
<tr class="odd">
<td>Likhitha Kolla</td>
<td>None</td>
<td>2020-11-16</td>
</tr>
<tr class="even">
<td>David Manheim</td>
<td>None</td>
<td>2022-03-15</td>
</tr>
<tr class="odd">
<td>Soumita Ghosh</td>
<td>None</td>
<td>2020-11-09</td>
</tr>
<tr class="even">
<td>James Brian Byrd</td>
<td>Funded by FastGrants to conduct a COVID-19-related clinical trial</td>
<td>2020-11-12</td>
</tr>
<tr class="odd">
<td>YoSon Park</td>
<td>YoSon Park is affiliated with Pfizer Worldwide Research. The author has no financial interests to declare and contributed as an author prior to joining Pfizer, and the work was not part of a Pfizer collaboration nor was it funded by Pfizer.</td>
<td>2020-01-22</td>
</tr>
<tr class="even">
<td>Vikas Bansal</td>
<td>None</td>
<td>2021-01-25</td>
</tr>
<tr class="odd">
<td>Stephen Capone</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>John J. Dziak</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="odd">
<td>Yuchen Sun</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Yanjun Qi</td>
<td>None</td>
<td>2020-07-09</td>
</tr>
<tr class="odd">
<td>Lamonica Shinholster</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Temitayo Lukan</td>
<td>None</td>
<td>2020-11-10</td>
</tr>
<tr class="odd">
<td>Sergey Knyazev</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Dimitri Perrin</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="odd">
<td>Serghei Mangul</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="even">
<td>Shikta Das</td>
<td>None</td>
<td>2020-08-13</td>
</tr>
<tr class="odd">
<td>Gregory L Szeto</td>
<td>None</td>
<td>2020-11-16</td>
</tr>
<tr class="even">
<td>Tiago Lubiana</td>
<td>None</td>
<td>2020-11-11</td>
</tr>
<tr class="odd">
<td>David Mai</td>
<td>None</td>
<td>2021-01-08</td>
</tr>
<tr class="even">
<td>COVID-19 Review Consortium</td>
<td>None</td>
<td>2021-01-16</td>
</tr>
<tr class="odd">
<td>Rishi Raj Goel</td>
<td>None</td>
<td>2021-01-20</td>
</tr>
<tr class="even">
<td>Joel D Boerckel</td>
<td>None</td>
<td>2021-03-26</td>
</tr>
<tr class="odd">
<td>Amruta Naik</td>
<td>None</td>
<td>2021-04-05</td>
</tr>
<tr class="even">
<td>Yusha Sun</td>
<td>None</td>
<td>2021-04-10</td>
</tr>
<tr class="odd">
<td>Daniel S. Himmelstein</td>
<td>None</td>
<td>2021-04-30</td>
</tr>
<tr class="even">
<td>Jeremy P. Kamil</td>
<td>None</td>
<td>2022-10-11</td>
</tr>
<tr class="odd">
<td>Jesse G. Meyer</td>
<td>None</td>
<td>2022-01-06</td>
</tr>
<tr class="even">
<td>Ariel I. Mundo</td>
<td>None</td>
<td>2021-12-19</td>
</tr>
</tbody>
</table>
<h2 data-number="13.2" id="author-contributions"><span class="header-section-number">13.2</span> Author Contributions</h2>
<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="header">
<th>Author</th>
<th>Contributions</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Halie M. Rando</td>
<td>D, E, Project Administration, Software, Visualization, Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Casey S. Greene</td>
<td>Conceptualization, Project Administration, Software, Supervision, Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Michael P. Robson</td>
<td>Methodology, Software, Supervision</td>
</tr>
<tr class="even">
<td>Simina M. Boca</td>
<td>Methodology, Project Administration, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Nils Wellhausen</td>
<td>Project Administration, Visualization, Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Ronan Lordan</td>
<td>Conceptualization, Project Administration, Visualization, Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Christian Brueffer</td>
<td>Project Administration, Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Sandipan Ray</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Lucy D'Agostino McGowan</td>
<td>Methodology, Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Anthony Gitter</td>
<td>Methodology, Project Administration, Software, Visualization, Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Anna Ada Dattoli</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="even">
<td>Ryan Velazquez</td>
<td>Methodology, Software, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>John P. Barton</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Jeffrey M. Field</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Bharath Ramsundar</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Adam L. MacLean</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Alexandra J. Lee</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Immunology Institute of the Icahn School of Medicine</td>
<td>Data Curation</td>
</tr>
<tr class="odd">
<td>Fengling Hu</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Nafisa M. Jadavji</td>
<td>Supervision, Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Elizabeth Sell</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Vincent Rubinetti</td>
<td>Software, Visualization, Writing - Original Draft</td>
</tr>
<tr class="odd">
<td>Jinhui Wang</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Diane N. Rafizadeh</td>
<td>Project Administration, Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Ashwin N. Skelly</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Marouen Ben Guebila</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Likhitha Kolla</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>David Manheim</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Soumita Ghosh</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="even">
<td>James Brian Byrd</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>YoSon Park</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Vikas Bansal</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Stephen Capone</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>John J. Dziak</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Yuchen Sun</td>
<td>Visualization</td>
</tr>
<tr class="even">
<td>Yanjun Qi</td>
<td>Visualization</td>
</tr>
<tr class="odd">
<td>Lamonica Shinholster</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="even">
<td>Temitayo Lukan</td>
<td>Investigation, Writing - Original Draft</td>
</tr>
<tr class="odd">
<td>Sergey Knyazev</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Dimitri Perrin</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Serghei Mangul</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Shikta Das</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Gregory L Szeto</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Tiago Lubiana</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>David Mai</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>COVID-19 Review Consortium</td>
<td>Project Administration</td>
</tr>
<tr class="odd">
<td>Rishi Raj Goel</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Joel D Boerckel</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Amruta Naik</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Yusha Sun</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Daniel S. Himmelstein</td>
<td>Software</td>
</tr>
<tr class="even">
<td>Jeremy P. Kamil</td>
<td>Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Jesse G. Meyer</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Ariel I. Mundo</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
</tbody>
</table>
<h2 data-number="13.3" id="acknowledgements"><span class="header-section-number">13.3</span> Acknowledgements</h2>
<p>We thank Nick DeVito for assistance with the Evidence-Based Medicine Data Lab COVID-19 TrialsTracker data.
We thank Yael Evelyn Marshall who contributed writing (original draft) as well as reviewing and editing of pieces of the text but who did not formally approve the manuscript, as well as Ronnie Russell, who contributed text to and helped develop the structure of the manuscript early in the writing process and Matthias Fax who helped with writing and editing text related to diagnostics.
We are also very grateful to James Fraser for suggestions about successes and limitations in the area of computational screening for drug repurposing.
We are grateful to the following contributors for reviewing pieces of the text: Nadia Danilova, James Eberwine and Ipsita Krishnan.
<!-- To Do: Add  Patryk Orzechowski to acknowledgements for vaccines--></p>
<h2 class="page_break_before" data-number="13.4" id="references"><span class="header-section-number">13.4</span> References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" data-entry-spacing="1" data-line-spacing="2" role="doc-bibliography">
<div id="ref-GdZc4Yyd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Rando HM, MacLean AL, Lee AJ, Lordan R, Ray S, Bansal V, Skelly AN, Sell E, Dziak JJ, Shinholster L, D'Agostino McGowan L, Ben Guebila M, Wellhausen N, Knyazev S, Boca SM, Capone S, Qi Y, Park Y, Mai D, Sun Y, Boerckel JD, Brueffer C, Byrd JB, Kamil JP, Wang J, Velazquez R, Szeto GL, Barton JP, Goel RR, Mangul S, Lubiana T, COVID-19 Review Consortium Vikas Bansal, John P. Barton, Simina M. Boca, Joel D. Boerckel, Christian Brueffer, James Brian Byrd, Stephen Capone, Shikta Das, Anna Ada Dattoli, John J. Dziak, Jeffrey M. Field, Soumita Ghosh, Anthony Gitter, Rishi Raj Goel, Casey S. Greene, Marouen Ben Guebila, Daniel S. Himmelstein, Fengling Hu, Nafisa M. Jadavji, Jeremy P. Kamil, Sergey Knyazev, Likhitha Kolla, Alexandra J. Lee, Ronan Lordan, Tiago Lubiana, Temitayo Lukan, Adam L. MacLean, David Mai, Serghei Mangul, David Manheim, Lucy D’Agostino McGowan, Amruta Naik, YoSon Park, Dimitri Perrin, Yanjun Qi, Diane N. Rafizadeh, Bharath Ramsundar, Halie M. Rando, Sandipan Ray, Michael P. Robson, Vincent Rubinetti, Elizabeth Sell, Lamonica Shinholster, Ashwin N. Skelly, Yuchen Sun, Yusha Sun, Gregory L. Szeto, Ryan Velazquez, Jinhui Wang, Nils Wellhausen, Gitter A, Greene CS. 2021. <a href="https://doi.org/10.1128/mSystems.00095-21">Pathogenesis, Symptomatology, and Transmission of SARS-CoV-2 through Analysis of Viral Genomics and Structure</a>. mSystems 6:e0009521.</div>
</div>
<div id="ref-wgAGKcBj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Lordan R, Rando HM, COVID-19 Review Consortium, Greene CS. 2021. <a href="https://doi.org/10.1128/mSystems.00122-21">Dietary Supplements and Nutraceuticals under Investigation for COVID-19 Prevention and Treatment</a>. mSystems 6:e00122–21.</div>
</div>
<div id="ref-njpLhBui" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Rando HM, Wellhausen N, Ghosh S, Lee AJ, Dattoli AA, Hu F, Byrd JB, Rafizadeh DN, Lordan R, Qi Y, Sun Y, Brueffer C, Field JM, Ben Guebila M, Jadavji NM, Skelly AN, Ramsundar B, Wang J, Goel RR, Park Y, COVID-19 Review Consortium Vikas Bansal, John P. Barton, Simina M. Boca, Joel D. Boerckel, Christian Brueffer, James Brian Byrd, Stephen Capone, Shikta Das, Anna Ada Dattoli, John J. Dziak, Jeffrey M. Field, Soumita Ghosh, Anthony Gitter, Rishi Raj Goel, Casey S. Greene, Marouen Ben Guebila, Daniel S. Himmelstein, Fengling Hu, Nafisa M. Jadavji, Jeremy P. Kamil, Sergey Knyazev, Likhitha Kolla, Alexandra J. Lee, Ronan Lordan, Tiago Lubiana, Temitayo Lukan, Adam L. MacLean, David Mai, Serghei Mangul, David Manheim, Lucy D’Agostino McGowan, Amruta Naik, YoSon Park, Dimitri Perrin, Yanjun Qi, Diane N. Rafizadeh, Bharath Ramsundar, Halie M. Rando, Sandipan Ray, Michael P. Robson, Vincent Rubinetti, Elizabeth Sell, Lamonica Shinholster, Ashwin N. Skelly, Yuchen Sun, Yusha Sun, Gregory L. Szeto, Ryan Velazquez, Jinhui Wang, Nils Wellhausen, Boca SM, Gitter A, Greene CS. 2021. <a href="https://doi.org/10.1128/mSystems.00233-21">Identification and Development of Therapeutics for COVID-19</a>. mSystems 6:e0023321.</div>
</div>
<div id="ref-EM9YkiOF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Rando HM, Boca SM, D'Agostino McGowan L, Himmelstein DS, Robson MP, Rubinetti V, Velazquez R, Greene CS, Gitter A. 2021. <a href="http://ceur-ws.org/Vol-2976/paper-2.pdf">An Open-Publishing Response to the COVID-19 Infodemic</a>. Proceedings of the Workshop on Digital Infrastructures for Scholarly Content Objects (DISCO 2021) 2976:29–38.</div>
</div>
<div id="ref-hIxvVJ7F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Rando HM, Brueffer C, Lordan R, Dattoli AA, Manheim D, Meyer JG, Mundo AI, Perrin D, Mai D, Wellhausen N, Consortium CR, Gitter A, Greene CS. 2022. <a href="https://arxiv.org/abs/2204.12598">Molecular and Serologic Diagnostic Technologies for SARS-CoV-2</a>. 2204.12598arXiv. arXiv.</div>
</div>
<div id="ref-1HWmcSB20" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Rando HM, Lordan R, Lee AJ, Naik A, Wellhausen N, Sell E, Kolla L, Consortium CR, Gitter A, Greene CS. 2022. <a href="https://arxiv.org/abs/2208.08907">Application of Traditional Vaccine Development Strategies to SARS-CoV-2</a>. 2208.08907arXiv. arXiv.</div>
</div>
<div id="ref-rjHK19Dr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Rando HM, Lordan R, Kolla L, Sell E, Lee AJ, Wellhausen N, Naik A, Kamil JP, Gitter A, Greene CS, Consortium the CR. 2022. <a href="https://arxiv.org/abs/2210.07247">The Coming of Age of Nucleic Acid Vaccines during COVID-19</a>. 2210.07247arXiv. arXiv.</div>
</div>
<div id="ref-gHWlMufv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">CDC. 2020. Coronavirus Disease 2019 (COVID-19) – Symptoms. Centers for Disease Control and Prevention. <a href="https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html">https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-DkrkT4Mb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Cucinotta D, Vanelli M. 2020. <a href="https://doi.org/10.23750/abm.v91i1.9397">WHO Declares COVID-19 a Pandemic</a>. Acta Bio Medica Atenei Parmensis 91:157–160.</div>
</div>
<div id="ref-1H0HgI6iy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Li L, Huang Q, Wang DC, Ingbar DH, Wang X. 2020. <a href="https://doi.org/10.1002/ctm2.16">Acute lung injury in patients with COVID‐19 infection</a>. Clinical and Translational Medicine 10:20–27.</div>
</div>
<div id="ref-Bp847Lfa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Tan W, Zhao X, Ma X, Wang W, Niu P, Xu W, F. Gao G, Wu G, MHC Key Laboratory of Biosafety, National Institute for Viral Disease Control and Prevention, China CDC, Beijing, China, Center for Biosafety Mega-Science, Chinese Academy of Sciences, Beijing, China. 2020. <a href="https://doi.org/10.46234/ccdcw2020.017">A Novel Coronavirus Genome Identified in a Cluster of Pneumonia Cases — Wuhan, China 2019−2020</a>. China CDC Weekly 2:61–62.</div>
</div>
<div id="ref-17DSmRo9H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Li F. 2016. <a href="https://doi.org/10.1146/annurev-virology-110615-042301">Structure, Function, and Evolution of Coronavirus Spike Proteins</a>. Annual Review of Virology 3:237–261.</div>
</div>
<div id="ref-IFLf8rsY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Chan JF-W, Yuan S, Kok K-H, To KK-W, Chu H, Yang J, Xing F, Liu J, Yip CC-Y, Poon RW-S, Tsoi H-W, Lo SK-F, Chan K-H, Poon VK-M, Chan W-M, Ip JD, Cai J-P, Cheng VC-C, Chen H, Hui CK-M, Yuen K-Y. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30154-9">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</a>. The Lancet 395:514–523.</div>
</div>
<div id="ref-RGAUx68v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">2007. Fields virology5th ed. Wolters Kluwer Health/Lippincott Williams &amp; Wilkins, Philadelphia.</div>
</div>
<div id="ref-DlBAegL1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Broer R, Boson B, Spaan W, Cosset F-L, Corver J. 2006. <a href="https://doi.org/10.1128/jvi.80.3.1302-1310.2006">Important Role for the Transmembrane Domain of Severe Acute Respiratory Syndrome Coronavirus Spike Protein during Entry</a>. Journal of Virology 80:1302–1310.</div>
</div>
<div id="ref-CqQVkaqj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Wrapp D, Wang N, Corbett KS, Goldsmith JA, Hsieh C-L, Abiona O, Graham BS, McLellan JS. 2020. <a href="https://doi.org/10.1126/science.abb2507">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</a>. Science 367:1260–1263.</div>
</div>
<div id="ref-jsMmdH4D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">1996. Medical microbiology4th ed. University of Texas Medical Branch at Galveston, Galveston, Tex.</div>
</div>
<div id="ref-47hnjvDb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Fehr AR, Perlman S. 2015. <a href="https://doi.org/10.1007/978-1-4939-2438-7_1">Coronaviruses: An Overview of Their Replication and Pathogenesis</a>Methods in Molecular Biology. Springer Science and Business Media LLC.</div>
</div>
<div id="ref-1Fqilxaum" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Lu R, Zhao X, Li J, Niu P, Yang B, Wu H, Wang W, Song H, Huang B, Zhu N, Bi Y, Ma X, Zhan F, Wang L, Hu T, Zhou H, Hu Z, Zhou W, Zhao L, Chen J, Meng Y, Wang J, Lin Y, Yuan J, Xie Z, Ma J, Liu WJ, Wang D, Xu W, Holmes EC, Gao GF, Wu G, Chen W, Shi W, Tan W. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30251-8">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</a>. The Lancet 395:565–574.</div>
</div>
<div id="ref-18GflyMj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Chen Y, Liu Q, Guo D. 2020. <a href="https://doi.org/10.1002/jmv.25681">Emerging coronaviruses: Genome structure, replication, and pathogenesis</a>. Journal of Medical Virology 92:418–423.</div>
</div>
<div id="ref-FFwktXgx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Wang H, Yang P, Liu K, Guo F, Zhang Y, Zhang G, Jiang C. 2008. <a href="https://doi.org/10.1038/cr.2008.15">SARS coronavirus entry into host cells through a novel clathrin- and caveolae-independent endocytic pathway</a>. Cell Research 18:290–301.</div>
</div>
<div id="ref-FNetQkRI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Mercer J, Schelhaas M, Helenius A. 2010. <a href="https://doi.org/10.1146/annurev-biochem-060208-104626">Virus Entry by Endocytosis</a>. Annual Review of Biochemistry 79:803–833.</div>
</div>
<div id="ref-13wWdgODZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Belouzard S, Millet JK, Licitra BN, Whittaker GR. 2012. <a href="https://doi.org/10.3390/v4061011">Mechanisms of Coronavirus Cell Entry Mediated by the Viral Spike Protein</a>. Viruses 4:1011–1033.</div>
</div>
<div id="ref-OVsxrEuX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Jaimes JA, André NM, Chappie JS, Millet JK, Whittaker GR. 2020. <a href="https://doi.org/10.1016/j.jmb.2020.04.009">Phylogenetic Analysis and Structural Modeling of SARS-CoV-2 Spike Protein Reveals an Evolutionary Distinct and Proteolytically Sensitive Activation Loop</a>. Journal of Molecular Biology 432:3309–3325.</div>
</div>
<div id="ref-qcVbT0w4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Walls AC, Park Y-J, Tortorici MA, Wall A, McGuire AT, Veesler D. 2020. <a href="https://doi.org/10.1016/j.cell.2020.02.058">Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</a>. Cell 181:281–292.e6.</div>
</div>
<div id="ref-So6ceUwB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Zhang H, Penninger JM, Li Y, Zhong N, Slutsky AS. 2020. <a href="https://doi.org/10.1007/s00134-020-05985-9">Angiotensin-converting enzyme 2 (ACE2) as a SARS-CoV-2 receptor: molecular mechanisms and potential therapeutic target</a>. Intensive Care Medicine 46:586–590.</div>
</div>
<div id="ref-NodtSApl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Jia HP, Look DC, Hickey M, Shi L, Pewe L, Netland J, Farzan M, Wohlford-Lenane C, Perlman S, McCray PB. 2006. <a href="https://doi.org/10.1007/978-0-387-33012-9_85">Infection of Human Airway Epithelia by Sars Coronavirus is Associated with ACE2 Expression and Localization</a>Advances in Experimental Medicine and Biology. Springer Science and Business Media LLC.</div>
</div>
<div id="ref-N1UXcG4j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Hikmet F, Méar L, Edvinsson Å, Micke P, Uhlén M, Lindskog C. 2020. <a href="https://doi.org/10.15252/msb.20209610">The protein expression profile of ACE2 in human tissues</a>. Molecular Systems Biology 16.</div>
</div>
<div id="ref-skHKTFMe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Li F. 2015. <a href="https://doi.org/10.1128/jvi.02615-14">Receptor Recognition Mechanisms of Coronaviruses: a Decade of Structural Studies</a>. Journal of Virology 89:1954–1964.</div>
</div>
<div id="ref-ueMPuUMe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">Du L, He Y, Zhou Y, Liu S, Zheng B-J, Jiang S. 2009. <a href="https://doi.org/10.1038/nrmicro2090">The spike protein of SARS-CoV — a target for vaccine and therapeutic development</a>. Nature Reviews Microbiology 7:226–236.</div>
</div>
<div id="ref-kCVy8vjj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">de Haan CAM, Rottier PJM. 2005. <a href="https://doi.org/10.1016/s0065-3527(05)64006-7">Molecular Interactions in the Assembly of Coronaviruses</a>Advances in Virus Research. Elsevier BV.</div>
</div>
<div id="ref-YeKaMzdJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Tang T, Bidon M, Jaimes JA, Whittaker GR, Daniel S. 2020. <a href="https://doi.org/10.1016/j.antiviral.2020.104792">Coronavirus membrane fusion mechanism offers a potential target for antiviral development</a>. Antiviral Research 178:104792.</div>
</div>
<div id="ref-15l3di3Wj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Hoffmann M, Kleine-Weber H, Schroeder S, Krüger N, Herrler T, Erichsen S, Schiergens TS, Herrler G, Wu N-H, Nitsche A, Müller MA, Drosten C, Pöhlmann S. 2020. <a href="https://doi.org/10.1016/j.cell.2020.02.052">SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</a>. Cell 181:271–280.e8.</div>
</div>
<div id="ref-JOJ2n3gC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Ou X, Liu Y, Lei X, Li P, Mi D, Ren L, Guo L, Guo R, Chen T, Hu J, Xiang Z, Mu Z, Chen X, Chen J, Hu K, Jin Q, Wang J, Qian Z. 2020. <a href="https://doi.org/10.1038/s41467-020-15562-9">Characterization of spike glycoprotein of SARS-CoV-2 on virus entry and its immune cross-reactivity with SARS-CoV</a>. Nature Communications 11:1620.</div>
</div>
<div id="ref-7bVyHTLM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline">Coutard B, Valle C, de Lamballerie X, Canard B, Seidah NG, Decroly E. 2020. <a href="https://doi.org/10.1016/j.antiviral.2020.104742">The spike glycoprotein of the new coronavirus 2019-nCoV contains a furin-like cleavage site absent in CoV of the same clade</a>. Antiviral Research 176:104742.</div>
</div>
<div id="ref-gZRMibZh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline">Xia S, Lan Q, Su S, Wang X, Xu W, Liu Z, Zhu Y, Wang Q, Lu L, Jiang S. 2020. <a href="https://doi.org/10.1038/s41392-020-0184-0">The role of furin cleavage site in SARS-CoV-2 spike protein-mediated membrane fusion in the presence or absence of trypsin</a>. Sig Transduct Target Ther 5.</div>
</div>
<div id="ref-f27ylIZM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline">Johnson BA, Xie X, Kalveram B, Lokugamage KG, Muruato A, Zou J, Zhang X, Juelich T, Smith JK, Zhang L, Bopp N, Schindewolf C, Vu M, Vanderheiden A, Swetnam D, Plante JA, Aguilar P, Plante KS, Lee B, Weaver SC, Suthar MS, Routh AL, Ren P, Ku Z, An Z, Debbink K, Shi PY, Freiberg AN, Menachery VD. 2020. <a href="https://doi.org/10.1101/2020.08.26.268854">Furin Cleavage Site Is Key to SARS-CoV-2 Pathogenesis</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-AP9eAy68" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline">Johnson BA, Xie X, Bailey AL, Kalveram B, Lokugamage KG, Muruato A, Zou J, Zhang X, Juelich T, Smith JK, Zhang L, Bopp N, Schindewolf C, Vu M, Vanderheiden A, Winkler ES, Swetnam D, Plante JA, Aguilar P, Plante KS, Popov V, Lee B, Weaver SC, Suthar MS, Routh AL, Ren P, Ku Z, An Z, Debbink K, Diamond MS, Shi P-Y, Freiberg AN, Menachery VD. 2021. <a href="https://doi.org/10.1038/s41586-021-03237-4">Loss of furin cleavage site attenuates SARS-CoV-2 pathogenesis</a>. Nature 591:293–299.</div>
</div>
<div id="ref-AfjDQPDh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Peacock TP, Goldhill DH, Zhou J, Baillon L, Frise R, Swann OC, Kugathasan R, Penn R, Brown JC, Sanchez-David RY, Braga L, Williamson MK, Hassard JA, Staller E, Hanley B, Osborn M, Giacca M, Davidson AD, Matthews DA, Barclay WS. 2021. <a href="https://doi.org/10.1038/s41564-021-00908-w">The furin cleavage site in the SARS-CoV-2 spike protein is required for transmission in ferrets</a>. Nat Microbiol 6:899–909.</div>
</div>
<div id="ref-FcRgnv0z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline">Cheng Y-W, Chao T-L, Li C-L, Chiu M-F, Kao H-C, Wang S-H, Pang Y-H, Lin C-H, Tsai Y-M, Lee W-H, Tao M-H, Ho T-C, Wu P-Y, Jang L-T, Chen P-J, Chang S-Y, Yeh S-H. 2020. <a href="https://doi.org/10.1016/j.celrep.2020.108254">Furin Inhibitors Block SARS-CoV-2 Spike Protein Cleavage to Suppress Virus Production and Cytopathic Effects</a>. Cell Reports 33:108254.</div>
</div>
<div id="ref-1F46J8hWx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline">Coronaviridae ~ ViralZone. <a href="https://viralzone.expasy.org/30">https://viralzone.expasy.org/30</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-5TQeBfZv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline">Jaimes JA, Millet JK, Whittaker GR. 2020. <a href="https://doi.org/10.1016/j.isci.2020.101212">Proteolytic Cleavage of the SARS-CoV-2 Spike Protein and the Role of the Novel S1/S2 Site</a>. iScience 23:101212.</div>
</div>
<div id="ref-17EgPSnfR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline">Dyer L, Patterson C. 2010. <a href="https://doi.org/10.1055/s-0030-1253446">Development of the Endothelium: An Emphasis on Heterogeneity</a>. Semin Thromb Hemost 36:227–235.</div>
</div>
<div id="ref-zvVjhfmD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline">Thorgeirsson G, Robertson AL. 1978. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2018350">The vascular endothelium-pathobiologic significance.</a> Am J Pathol 93:803–48.</div>
</div>
<div id="ref-14dUo0d2a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline">Reitsma S, Slaaf DW, Vink H, van Zandvoort MAMJ, oude Egbrink MGA. 2007. <a href="https://doi.org/10.1007/s00424-007-0212-8">The endothelial glycocalyx: composition, functions, and visualization</a>. Pflugers Arch - Eur J Physiol 454:345–359.</div>
</div>
<div id="ref-11mrQ55Mt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">46. </div><div class="csl-right-inline">Clausen TM, Sandoval DR, Spliid CB, Pihl J, Perrett HR, Painter CD, Narayanan A, Majowicz SA, Kwong EM, McVicar RN, Thacker BE, Glass CA, Yang Z, Torres JL, Golden GJ, Bartels PL, Porell RN, Garretson AF, Laubach L, Feldman J, Yin X, Pu Y, Hauser BM, Caradonna TM, Kellman BP, Martino C, Gordts PLSM, Chanda SK, Schmidt AG, Godula K, Leibel SL, Jose J, Corbett KD, Ward AB, Carlin AF, Esko JD. 2020. <a href="https://doi.org/10.1016/j.cell.2020.09.033">SARS-CoV-2 Infection Depends on Cellular Heparan Sulfate and ACE2</a>. Cell 183:1043–1057.e15.</div>
</div>
<div id="ref-muOuyY5D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">47. </div><div class="csl-right-inline">Ihrcke NS, Wrenshall LE, Lindman BJ, Platt JL. 1993. <a href="https://doi.org/10.1016/0167-5699(93)90265-m">Role of heparan sulfate in immune system-blood vessel interactions</a>. Immunology Today 14:500–505.</div>
</div>
<div id="ref-eC61y3gI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">48. </div><div class="csl-right-inline">Liu L, Chopra P, Li X, Bouwman KM, Tompkins SM, Wolfert MA, de Vries RP, Boons G-J. 2020. <a href="https://doi.org/10.1101/2020.05.10.087288">Heparan sulfate proteoglycans as attachment factor for SARS-CoV-2</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-JEtIJaPO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">49. </div><div class="csl-right-inline">Shriver Z, Capila I, Venkataraman G, Sasisekharan R. 2011. <a href="https://doi.org/10.1007/978-3-642-23056-1_8">Heparin and Heparan Sulfate: Analyzing Structure and Microheterogeneity</a>, p. 159–176. <em>In</em> Heparin - A Century of Progress. Springer Berlin Heidelberg.</div>
</div>
<div id="ref-ZVyaYX2h" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">50. </div><div class="csl-right-inline">LaRivière WB, Schmidt EP. 2018. <a href="https://doi.org/10.1016/bs.ctm.2018.08.005">The Pulmonary Endothelial Glycocalyx in ARDS: A Critical Role for Heparan Sulfate</a>, p. 33–52. <em>In</em> Current Topics in Membranes. Elsevier.</div>
</div>
<div id="ref-1FNCN2CC4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">51. </div><div class="csl-right-inline">Zhang Q, Chen CZ, Swaroop M, Xu M, Wang L, Lee J, Wang AQ, Pradhan M, Hagen N, Chen L, Shen M, Luo Z, Xu X, Xu Y, Huang W, Zheng W, Ye Y. 2020. <a href="https://doi.org/10.1038/s41421-020-00222-5">Heparan sulfate assists SARS-CoV-2 in cell entry and can be targeted by approved drugs in vitro</a>. Cell Discov 6.</div>
</div>
<div id="ref-1C7D3ypZY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">52. </div><div class="csl-right-inline">Buijsers B, Yanginlar C, Maciej-Hulme ML, de Mast Q, van der Vlag J. 2020. <a href="https://doi.org/10.1016/j.ebiom.2020.102969">Beneficial non-anticoagulant mechanisms underlying heparin treatment of COVID-19 patients</a>. eBioMedicine 59:102969.</div>
</div>
<div id="ref-iucR6FeL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">53. </div><div class="csl-right-inline">Salah HM, Naser JA, Calcaterra G, Bassareo PP, Mehta JL. 2020. <a href="https://doi.org/10.1016/j.amjcard.2020.08.005">The Effect of Anticoagulation Use on Mortality in COVID-19 Infection</a>. The American Journal of Cardiology 134:155–157.</div>
</div>
<div id="ref-1FKr0hF0S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">54. </div><div class="csl-right-inline">Salaris C, Scarpa M, Elli M, Bertolini A, Guglielmetti S, Pregliasco F, Blandizzi C, Brun P, Castagliuolo I. 2021. <a href="https://doi.org/10.3390/nu13020328">Protective Effects of Lactoferrin against SARS-CoV-2 Infection In Vitro</a>. Nutrients 13:328.</div>
</div>
<div id="ref-qvv0s5BF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">55. </div><div class="csl-right-inline">Wadowski PP, Jilma B, Kopp CW, Ertl S, Gremmel T, Koppensteiner R. 2021. <a href="https://doi.org/10.3389/fimmu.2021.607306">Glycocalyx as Possible Limiting Factor in COVID-19</a>. Front Immunol 12.</div>
</div>
<div id="ref-1XbF4GLn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">56. </div><div class="csl-right-inline">Liu L, Wei Q, Nishiura K, Peng J, Wang H, Midkiff C, Alvarez X, Qin C, Lackner A, Chen Z. 2015. <a href="https://doi.org/10.1038/mi.2015.127">Spatiotemporal interplay of severe acute respiratory syndrome coronavirus and respiratory mucosal cells drives viral dissemination in rhesus macaques</a>. Mucosal Immunology 9:1089–1101.</div>
</div>
<div id="ref-159GOVLV1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">57. </div><div class="csl-right-inline">Poynter SJ, DeWitte-Orr SJ. 2018. <a href="https://doi.org/10.3389/fimmu.2018.00829">Understanding Viral dsRNA-Mediated Innate Immune Responses at the Cellular Level Using a Rainbow Trout Model</a>. Frontiers in Immunology 9:829.</div>
</div>
<div id="ref-yQO4HFTZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">58. </div><div class="csl-right-inline">Snijder EJ, van der Meer Y, Zevenhoven-Dobbe J, Onderwater JJM, van der Meulen J, Koerten HK, Mommaas AM. 2006. <a href="https://doi.org/10.1128/jvi.02501-05">Ultrastructure and Origin of Membrane Vesicles Associated with the Severe Acute Respiratory Syndrome Coronavirus Replication Complex</a>. Journal of Virology 80:5927–5940.</div>
</div>
<div id="ref-OlEbK4fc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">59. </div><div class="csl-right-inline">Li X, Geng M, Peng Y, Meng L, Lu S. 2020. <a href="https://doi.org/10.1016/j.jpha.2020.03.001">Molecular immune pathogenesis and diagnosis of COVID-19</a>. Journal of Pharmaceutical Analysis 10:102–108.</div>
</div>
<div id="ref-Q5Q9k6Nc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">60. </div><div class="csl-right-inline">Thoms M, Buschauer R, Ameismeier M, Koepke L, Denk T, Hirschenberger M, Kratzat H, Hayn M, Mackens-Kiani T, Cheng J, Straub JH, Stürzel CM, Fröhlich T, Berninghausen O, Becker T, Kirchhoff F, Sparrer KMJ, Beckmann R. 2020. <a href="https://doi.org/10.1126/science.abc8665">Structural basis for translational shutdown and immune evasion by the Nsp1 protein of SARS-CoV-2</a>. Science 369:1249–1255.</div>
</div>
<div id="ref-G6A0wWdn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">61. </div><div class="csl-right-inline">2020. Immune responses in COVID-19 and potential vaccines: Lessons learned from SARS and MERS epidemic. Asian Pacific Journal of Allergy and Immunology https://doi.org/<a href="https://doi.org/10.12932/ap-200220-0772">10.12932/ap-200220-0772</a>.</div>
</div>
<div id="ref-2QTH37Xi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">62. </div><div class="csl-right-inline">Li W, Moore MJ, Vasilieva N, Sui J, Wong SK, Berne MA, Somasundaran M, Sullivan JL, Luzuriaga K, Greenough TC, Choe H, Farzan M. 2003. <a href="https://doi.org/10.1038/nature02145">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</a>. Nature 426:450–454.</div>
</div>
<div id="ref-MXWaV7SO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">63. </div><div class="csl-right-inline">Matsuyama S, Nagata N, Shirato K, Kawase M, Takeda M, Taguchi F. 2010. <a href="https://doi.org/10.1128/jvi.01542-10">Efficient Activation of the Severe Acute Respiratory Syndrome Coronavirus Spike Protein by the Transmembrane Protease TMPRSS2</a>. Journal of Virology 84:12658–12664.</div>
</div>
<div id="ref-JjrO6YbD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">64. </div><div class="csl-right-inline">Glowacka I, Bertram S, Muller MA, Allen P, Soilleux E, Pfefferle S, Steffen I, Tsegaye TS, He Y, Gnirss K, Niemeyer D, Schneider H, Drosten C, Pohlmann S. 2011. <a href="https://doi.org/10.1128/jvi.02232-10">Evidence that TMPRSS2 Activates the Severe Acute Respiratory Syndrome Coronavirus Spike Protein for Membrane Fusion and Reduces Viral Control by the Humoral Immune Response</a>. Journal of Virology 85:4122–4134.</div>
</div>
<div id="ref-9qsndPCu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">65. </div><div class="csl-right-inline">Zhuang M, Cheng Y, Zhang J, Jiang X, Wang L, Deng J, Wang P. 2020. <a href="https://doi.org/10.1002/jmv.26139">Increasing host cellular receptor—angiotensin‐converting enzyme 2 expression by coronavirus may facilitate 2019‐nCoV (or SARS‐CoV‐2) infection</a>. J Med Virol 92:2693–2701.</div>
</div>
<div id="ref-75IAJRbI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">66. </div><div class="csl-right-inline">Li Y, Zhou W, Yang L, You R. 2020. <a href="https://doi.org/10.1016/j.phrs.2020.104833">Physiological and pathological regulation of ACE2, the SARS-CoV-2 receptor</a>. Pharmacological Research 157:104833.</div>
</div>
<div id="ref-1FE1YGsaV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">67. </div><div class="csl-right-inline">Hamming I, Timens W, Bulthuis M, Lely A, Navis G, van Goor H. 2004. <a href="https://doi.org/10.1002/path.1570">Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis</a>. The Journal of Pathology 203:631–637.</div>
</div>
<div id="ref-4zAgRRxf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">68. </div><div class="csl-right-inline">Foote MB, White JR, Jee J, Argilés G, Wan JCM, Rousseau B, Pessin MS, Diaz LA Jr. 2021. <a href="https://doi.org/10.1001/jamaoncol.2021.3585">Association of Antineoplastic Therapy With Decreased SARS-CoV-2 Infection Rates in Patients With Cancer</a>. JAMA Oncol 7:1686.</div>
</div>
<div id="ref-1C7ggDgVA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">69. </div><div class="csl-right-inline">Thomas G. 2002. <a href="https://doi.org/10.1038/nrm934">Furin at the cutting edge: From protein traffic to embryogenesis and disease</a>. Nat Rev Mol Cell Biol 3:753–766.</div>
</div>
<div id="ref-bkTglQpW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">70. </div><div class="csl-right-inline">Hoffmann M, Kleine-Weber H, Pöhlmann S. 2020. <a href="https://doi.org/10.1016/j.molcel.2020.04.022">A Multibasic Cleavage Site in the Spike Protein of SARS-CoV-2 Is Essential for Infection of Human Lung Cells</a>. Molecular Cell 78:779–784.e5.</div>
</div>
<div id="ref-vr7AH83b" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">71. </div><div class="csl-right-inline">Wang W, Xu Y, Gao R, Lu R, Han K, Wu G, Tan W. 2020. Detection of SARS-CoV-2 in Different Types of Clinical Specimens. JAMA https://doi.org/<a href="https://doi.org/10.1001/jama.2020.3786">10.1001/jama.2020.3786</a>.</div>
</div>
<div id="ref-azgJqujy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">72. </div><div class="csl-right-inline">Young BE, Ong SWX, Kalimuddin S, Low JG, Tan SY, Loh J, Ng O-T, Marimuthu K, Ang LW, Mak TM, Lau SK, Anderson DE, Chan KS, Tan TY, Ng TY, Cui L, Said Z, Kurupatham L, Chen MI-C, Chan M, Vasoo S, Wang L-F, Tan BH, Lin RTP, Lee VJM, Leo Y-S, Lye DC, for the Singapore 2019 Novel Coronavirus Outbreak Research Team. 2020. <a href="https://doi.org/10.1001/jama.2020.3204">Epidemiologic Features and Clinical Course of Patients Infected With SARS-CoV-2 in Singapore</a>. JAMA 323:1488.</div>
</div>
<div id="ref-F7nSMvZk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">73. </div><div class="csl-right-inline">Ling Y, Xu S-B, Lin Y-X, Tian D, Zhu Z-Q, Dai F-H, Wu F, Song Z-G, Huang W, Chen J, Hu B-J, Wang S, Mao E-Q, Zhu L, Zhang W-H, Lu H-Z. 2020. <a href="https://doi.org/10.1097/cm9.0000000000000774">Persistence and clearance of viral RNA in 2019 novel coronavirus disease rehabilitation patients</a>. Chinese Medical Journal 133:1039–1043.</div>
</div>
<div id="ref-1AJFJxzmJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">74. </div><div class="csl-right-inline">Menter T, Haslbauer JD, Nienhold R, Savic S, Hopfer H, Deigendesch N, Frank S, Turek D, Willi N, Pargger H, Bassetti S, Leuppi JD, Cathomas G, Tolnay M, Mertz KD, Tzankov A. 2020. <a href="https://doi.org/10.1111/his.14134">Postmortem examination of COVID‐19 patients reveals diffuse alveolar damage with severe capillary congestion and variegated findings in lungs and other organs suggesting vascular dysfunction</a>. Histopathology 77:198–209.</div>
</div>
<div id="ref-5gnCuPzp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">75. </div><div class="csl-right-inline">Shi S, Qin M, Shen B, Cai Y, Liu T, Yang F, Gong W, Liu X, Liang J, Zhao Q, Huang H, Yang B, Huang C. 2020. <a href="https://doi.org/10.1001/jamacardio.2020.0950">Association of Cardiac Injury With Mortality in Hospitalized Patients With COVID-19 in Wuhan, China</a>. JAMA Cardiology 5:802.</div>
</div>
<div id="ref-5ET1D3cK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">76. </div><div class="csl-right-inline">Wang S, Zhou X, Zhang T, Wang Z. 2020. <a href="https://doi.org/10.1038/s41585-020-0319-7">The need for urogenital tract monitoring in COVID-19</a>. Nature Reviews Urology 17:314–315.</div>
</div>
<div id="ref-w5B6qRKv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">77. </div><div class="csl-right-inline">Fanelli V, Fiorentino M, Cantaluppi V, Gesualdo L, Stallone G, Ronco C, Castellano G. 2020. <a href="https://doi.org/10.1186/s13054-020-02872-z">Acute kidney injury in SARS-CoV-2 infected patients</a>. Critical Care 24:155.</div>
</div>
<div id="ref-TTSFlLVC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">78. </div><div class="csl-right-inline">Zhang C, Shi L, Wang F-S. 2020. <a href="https://doi.org/10.1016/s2468-1253(20)30057-1">Liver injury in COVID-19: management and challenges</a>. The Lancet Gastroenterology &amp; Hepatology 5:428–430.</div>
</div>
<div id="ref-PmE9xedP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">79. </div><div class="csl-right-inline">Xiao F, Tang M, Zheng X, Liu Y, Li X, Shan H. 2020. <a href="https://doi.org/10.1053/j.gastro.2020.02.055">Evidence for Gastrointestinal Infection of SARS-CoV-2</a>. Gastroenterology 158:1831–1833.e3.</div>
</div>
<div id="ref-3Ak4Mata" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">80. </div><div class="csl-right-inline">Gao QY, Chen YX, Fang JY. 2020. <a href="https://doi.org/10.1111/1751-2980.12851">2019 Novel coronavirus infection and gastrointestinal tract</a>. Journal of Digestive Diseases 21:125–126.</div>
</div>
<div id="ref-17uZcQTT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">81. </div><div class="csl-right-inline">Agarwal A, Rochwerg B, Lamontagne F, Siemieniuk RA, Agoritsas T, Askie L, Lytvyn L, Leo Y-S, Macdonald H, Zeng L, Amin W, da Silva ARA, Aryal D, Barragan FAJ, Bausch FJ, Burhan E, Calfee CS, Cecconi M, Chacko B, Chanda D, Dat VQ, De Sutter A, Du B, Freedman S, Geduld H, Gee P, Gotte M, Harley N, Hashmi M, Hunt B, Jehan F, Kabra SK, Kanda S, Kim Y-J, Kissoon N, Krishna S, Kuppalli K, Kwizera A, Castro-Rial ML, Lisboa T, Lodha R, Mahaka I, Manai H, Mendelson M, Battista Migliori G, Mino G, Nsutebu E, Preller J, Pshenichnaya N, Qadir N, Relan P, Sabzwari S, Sarin R, Shankar-Hari M, Sharland M, Shen Y, Ranganathan SS, Souza JP, Stegemann M, Swanstrom R, Ugarte S, Uyeki T, Venkatapuram S, Vuyiseka D, Wijewickrama A, Tran L, Zeraatkar D, Bartoszko JJ, Ge L, Brignardello-Petersen R, Owen A, Guyatt G, Diaz J, Kawano-Dourado L, Jacobs M, Vandvik PO. 2020. <a href="https://doi.org/10.1136/bmj.m3379">A living WHO guideline on drugs for covid-19</a>. BMJ m3379.</div>
</div>
<div id="ref-15IDA5kX6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">82. </div><div class="csl-right-inline">Guan W, Ni Z, Hu Y, Liang W, Ou C, He J, Liu L, Shan H, Lei C, Hui DSC, Du B, Li L, Zeng G, Yuen K-Y, Chen R, Tang C, Wang T, Chen P, Xiang J, Li S, Wang J, Liang Z, Peng Y, Wei L, Liu Y, Hu Y, Peng P, Wang J, Liu J, Chen Z, Li G, Zheng Z, Qiu S, Luo J, Ye C, Zhu S, Zhong N. 2020. <a href="https://doi.org/10.1056/nejmoa2002032">Clinical Characteristics of Coronavirus Disease 2019 in China</a>. New England Journal of Medicine 382:1708–1720.</div>
</div>
<div id="ref-10THxyeCg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">83. </div><div class="csl-right-inline">Zhou F, Yu T, Du R, Fan G, Liu Y, Liu Z, Xiang J, Wang Y, Song B, Gu X, Guan L, Wei Y, Li H, Wu X, Xu J, Tu S, Zhang Y, Chen H, Cao B. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30566-3">Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study</a>. The Lancet 395:1054–1062.</div>
</div>
<div id="ref-19ytsiSpq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">84. </div><div class="csl-right-inline">Richardson S, Hirsch JS, Narasimhan M, Crawford JM, McGinn T, Davidson KW, Barnaby DP, Becker LB, Chelico JD, Cohen SL, Cookingham J, Coppa K, Diefenbach MA, Dominello AJ, Duer-Hefele J, Falzon L, Gitlin J, Hajizadeh N, Harvin TG, Hirschwerk DA, Kim EJ, Kozel ZM, Marrast LM, Mogavero JN, Osorio GA, Qiu M, Zanos TP, and the Northwell COVID-19 Research Consortium. 2020. <a href="https://doi.org/10.1001/jama.2020.6775">Presenting Characteristics, Comorbidities, and Outcomes Among 5700 Patients Hospitalized With COVID-19 in the New York City Area</a>. JAMA 323:2052.</div>
</div>
<div id="ref-1H8gSsSkn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">85. </div><div class="csl-right-inline">Goyal P, Choi JJ, Pinheiro LC, Schenck EJ, Chen R, Jabri A, Satlin MJ, Campion TR, Nahid M, Ringel JB, Hoffman KL, Alshak MN, Li HA, Wehmeyer GT, Rajan M, Reshetnyak E, Hupert N, Horn EM, Martinez FJ, Gulick RM, Safford MM. 2020. <a href="https://doi.org/10.1056/nejmc2010419">Clinical Characteristics of Covid-19 in New York City</a>. New England Journal of Medicine 382:2372–2374.</div>
</div>
<div id="ref-18tT1tLJI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">86. </div><div class="csl-right-inline">Burke RM, Killerby ME, Newton S, Ashworth CE, Berns AL, Brennan S, Bressler JM, Bye E, Crawford R, Harduar Morano L, Lewis NM, Markus TM, Read JS, Rissman T, Taylor J, Tate JE, Midgley CM, Balachandran N, Dahl RM, Dott M, Gilani Z, Grober A, Leung J, O’Hegarty M, Person J, Ricaldi JN, Roth NM, Sejvar JJ, Shimabukuro T, Tran CH, Watson JT, Whitham H, Chiou H, Clogher P, Duca LM, Dratch A, Feldpausch A, Fill M-M, Ghinai I, Holshue M, Scott S, Westergaard R, Case Investigation Form Working Group, Case Investigation Form Working Group. 2020. <a href="https://doi.org/10.15585/mmwr.mm6928a2">Symptom Profiles of a Convenience Sample of Patients with COVID-19 — United States, January–April 2020</a>. MMWR Morbidity and Mortality Weekly Report 69:904–908.</div>
</div>
<div id="ref-5tU6buUE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">87. </div><div class="csl-right-inline">Allen WE, Altae-Tran H, Briggs J, Jin X, McGee G, Shi A, Raghavan R, Kamariza M, Nova N, Pereta A, Danford C, Kamel A, Gothe P, Milam E, Aurambault J, Primke T, Li W, Inkenbrandt J, Huynh T, Chen E, Lee C, Croatto M, Bentley H, Lu W, Murray R, Travassos M, Coull BA, Openshaw J, Greene CS, Shalem O, King G, Probasco R, Cheng DR, Silbermann B, Zhang F, Lin X. 2020. <a href="https://doi.org/10.1038/s41562-020-00944-2">Population-scale longitudinal mapping of COVID-19 symptoms, behaviour and testing</a>. Nat Hum Behav 4:972–982.</div>
</div>
<div id="ref-wK2afyL8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">88. </div><div class="csl-right-inline">Gupta A, Madhavan MV, Sehgal K, Nair N, Mahajan S, Sehrawat TS, Bikdeli B, Ahluwalia N, Ausiello JC, Wan EY, Freedberg DE, Kirtane AJ, Parikh SA, Maurer MS, Nordvig AS, Accili D, Bathon JM, Mohan S, Bauer KA, Leon MB, Krumholz HM, Uriel N, Mehra MR, Elkind MSV, Stone GW, Schwartz A, Ho DD, Bilezikian JP, Landry DW. 2020. <a href="https://doi.org/10.1038/s41591-020-0968-3">Extrapulmonary manifestations of COVID-19</a>. Nature Medicine 26:1017–1032.</div>
</div>
<div id="ref-17hdoiBo9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">89. </div><div class="csl-right-inline">Hirsch JS, Ng JH, Ross DW, Sharma P, Shah HH, Barnett RL, Hazzan AD, Fishbane S, Jhaveri KD, Abate M, Andrade HP, Barnett RL, Bellucci A, Bhaskaran MC, Corona AG, Chang BF, Finger M, Fishbane S, Gitman M, Halinski C, Hasan S, Hazzan AD, Hirsch JS, Hong S, Jhaveri KD, Khanin Y, Kuan A, Madireddy V, Malieckal D, Muzib A, Nair G, Nair VV, Ng JH, Parikh R, Ross DW, Sakhiya V, Sachdeva M, Schwarz R, Shah HH, Sharma P, Singhal PC, Uppal NN, Wanchoo R, Bessy Suyin Flores Chang, Ng JHwei. 2020. <a href="https://doi.org/10.1016/j.kint.2020.05.006">Acute kidney injury in patients hospitalized with COVID-19</a>. Kidney International 98:209–218.</div>
</div>
<div id="ref-9K9lIU2m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">90. </div><div class="csl-right-inline">Velez JCQ, Caza T, Larsen CP. 2020. <a href="https://doi.org/10.1038/s41581-020-0332-3">COVAN is the new HIVAN: the re-emergence of collapsing glomerulopathy with COVID-19</a>. Nat Rev Nephrol 16:565–567.</div>
</div>
<div id="ref-1CqoZGSKK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">91. </div><div class="csl-right-inline">Wu Y, Xu X, Chen Z, Duan J, Hashimoto K, Yang L, Liu C, Yang C. 2020. <a href="https://doi.org/10.1016/j.bbi.2020.03.031">Nervous system involvement after infection with COVID-19 and other coronaviruses</a>. Brain, Behavior, and Immunity 87:18–22.</div>
</div>
<div id="ref-1EF9EpbZ5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">92. </div><div class="csl-right-inline">Ellul MA, Benjamin L, Singh B, Lant S, Michael BD, Easton A, Kneen R, Defres S, Sejvar J, Solomon T. 2020. <a href="https://doi.org/10.1016/s1474-4422(20)30221-0">Neurological associations of COVID-19</a>. The Lancet Neurology 19:767–783.</div>
</div>
<div id="ref-1F7Vk1wO9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">93. </div><div class="csl-right-inline">Boldrini M, Canoll PD, Klein RS. 2021. <a href="https://doi.org/10.1001/jamapsychiatry.2021.0500">How COVID-19 Affects the Brain</a>. JAMA Psychiatry 78:682.</div>
</div>
<div id="ref-nBGGUV9s" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">94. </div><div class="csl-right-inline">Finsterer J, Stollberger C. 2020. <a href="https://doi.org/10.1002/jmv.26000">Update on the neurology of COVID‐19</a>. Journal of Medical Virology 92:2316–2318.</div>
</div>
<div id="ref-LLm9d62d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">95. </div><div class="csl-right-inline">Koralnik IJ, Tyler KL. 2020. <a href="https://doi.org/10.1002/ana.25807">COVID-19: A Global Threat to the Nervous System</a>. Annals of Neurology 88:1–11.</div>
</div>
<div id="ref-17nNkDj8p" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">96. </div><div class="csl-right-inline">Meinhardt J, Radke J, Dittmayer C, Franz J, Thomas C, Mothes R, Laue M, Schneider J, Brünink S, Greuel S, Lehmann M, Hassan O, Aschman T, Schumann E, Chua RL, Conrad C, Eils R, Stenzel W, Windgassen M, Rößler L, Goebel H-H, Gelderblom HR, Martin H, Nitsche A, Schulz-Schaeffer WJ, Hakroush S, Winkler MS, Tampe B, Scheibe F, Körtvélyessy P, Reinhold D, Siegmund B, Kühl AA, Elezkurtaj S, Horst D, Oesterhelweg L, Tsokos M, Ingold-Heppner B, Stadelmann C, Drosten C, Corman VM, Radbruch H, Heppner FL. 2020. <a href="https://doi.org/10.1038/s41593-020-00758-5">Olfactory transmucosal SARS-CoV-2 invasion as a port of central nervous system entry in individuals with COVID-19</a>. Nature Neuroscience 24:168–175.</div>
</div>
<div id="ref-R1CIWZPb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">97. </div><div class="csl-right-inline">Thakur KT, Miller EH, Glendinning MD, Al-Dalahmah O, Banu MA, Boehme AK, Boubour AL, Bruce SS, Chong AM, Claassen J, Faust PL, Hargus G, Hickman RA, Jambawalikar S, Khandji AG, Kim CY, Klein RS, Lignelli-Dipple A, Lin C-C, Liu Y, Miller ML, Moonis G, Nordvig AS, Overdevest JB, Prust ML, Przedborski S, Roth WH, Soung A, Tanji K, Teich AF, Agalliu D, Uhlemann A-C, Goldman JE, Canoll P. 2021. <a href="https://doi.org/10.1093/brain/awab148">COVID-19 neuropathology at Columbia University Irving Medical Center/New York Presbyterian Hospital</a>. Brain 144:2696–2708.</div>
</div>
<div id="ref-aRNqjkMg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">98. </div><div class="csl-right-inline">Oxley TJ, Mocco J, Majidi S, Kellner CP, Shoirah H, Singh IP, De Leacy RA, Shigematsu T, Ladner TR, Yaeger KA, Skliut M, Weinberger J, Dangayach NS, Bederson JB, Tuhrim S, Fifi JT. 2020. <a href="https://doi.org/10.1056/nejmc2009787">Large-Vessel Stroke as a Presenting Feature of Covid-19 in the Young</a>. New England Journal of Medicine 382:e60.</div>
</div>
<div id="ref-3UBbDrG2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">99. </div><div class="csl-right-inline">Klok FA, Kruip MJHA, van der Meer NJM, Arbous MS, Gommers DAMPJ, Kant KM, Kaptein FHJ, van Paassen J, Stals MAM, Huisman MV, Endeman H. 2020. <a href="https://doi.org/10.1016/j.thromres.2020.04.013">Incidence of thrombotic complications in critically ill ICU patients with COVID-19</a>. Thrombosis Research 191:145–147.</div>
</div>
<div id="ref-8OnbWuhF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">100. </div><div class="csl-right-inline">Zhang Y, Xiao M, Zhang S, Xia P, Cao W, Jiang W, Chen H, Ding X, Zhao H, Zhang H, Wang C, Zhao J, Sun X, Tian R, Wu W, Wu D, Ma J, Chen Y, Zhang D, Xie J, Yan X, Zhou X, Liu Z, Wang J, Du B, Qin Y, Gao P, Qin X, Xu Y, Zhang W, Li T, Zhang F, Zhao Y, Li Y, Zhang S. 2020. <a href="https://doi.org/10.1056/nejmc2007575">Coagulopathy and Antiphospholipid Antibodies in Patients with Covid-19</a>. New England Journal of Medicine 382:e38.</div>
</div>
<div id="ref-dUfws1q0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">101. </div><div class="csl-right-inline">Tang N, Li D, Wang X, Sun Z. 2020. <a href="https://doi.org/10.1111/jth.14768">Abnormal coagulation parameters are associated with poor prognosis in patients with novel coronavirus pneumonia</a>. Journal of Thrombosis and Haemostasis 18:844–847.</div>
</div>
<div id="ref-p5EhN9Qd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">102. </div><div class="csl-right-inline">Goeijenbier M, van Wissen M, van de Weg C, Jong E, Gerdes VEA, Meijers JCM, Brandjes DPM, van Gorp ECM. 2012. <a href="https://doi.org/10.1002/jmv.23354">Review: Viral infections and mechanisms of thrombosis and bleeding</a>. Journal of Medical Virology 84:1680–1696.</div>
</div>
<div id="ref-v1EIzwfx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">103. </div><div class="csl-right-inline">McGonagle D, O'Donnell JS, Sharif K, Emery P, Bridgewood C. 2020. <a href="https://doi.org/10.1016/s2665-9913(20)30121-1">Immune mechanisms of pulmonary intravascular coagulopathy in COVID-19 pneumonia</a>. The Lancet Rheumatology 2:e437–e445.</div>
</div>
<div id="ref-18AiyvhO8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">104. </div><div class="csl-right-inline">Guglielmetti G, Quaglia M, Sainaghi PP, Castello LM, Vaschetto R, Pirisi M, Corte FD, Avanzi GC, Stratta P, Cantaluppi V. 2020. <a href="https://doi.org/10.1186/s13054-020-03060-9">“War to the knife” against thromboinflammation to protect endothelial function of COVID-19 patients</a>. Critical Care 24:365.</div>
</div>
<div id="ref-PyKMLraw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">105. </div><div class="csl-right-inline">Becker RC. 2020. <a href="https://doi.org/10.1007/s11239-020-02134-3">COVID-19 update: Covid-19-associated coagulopathy</a>. Journal of Thrombosis and Thrombolysis 50:54–67.</div>
</div>
<div id="ref-14dyYQY7s" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">106. </div><div class="csl-right-inline">Java A, Apicelli AJ, Liszewski MK, Coler-Reilly A, Atkinson JP, Kim AHJ, Kulkarni HS. 2020. <a href="https://doi.org/10.1172/jci.insight.140711">The complement system in COVID-19: friend and foe?</a> JCI Insight 5:e140711.</div>
</div>
<div id="ref-17y6YeJ6R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">107. </div><div class="csl-right-inline">Song W-C, FitzGerald GA. 2020. <a href="https://doi.org/10.1172/jci140183">COVID-19, microangiopathy, hemostatic activation, and complement</a>. Journal of Clinical Investigation 10.1172/JCI140183.</div>
</div>
<div id="ref-2uD7bbjv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">108. </div><div class="csl-right-inline">Nalbandian A, Sehgal K, Gupta A, Madhavan MV, McGroder C, Stevens JS, Cook JR, Nordvig AS, Shalev D, Sehrawat TS, Ahluwalia N, Bikdeli B, Dietz D, Der-Nigoghossian C, Liyanage-Don N, Rosner GF, Bernstein EJ, Mohan S, Beckley AA, Seres DS, Choueiri TK, Uriel N, Ausiello JC, Accili D, Freedberg DE, Baldwin M, Schwartz A, Brodie D, Garcia CK, Elkind MSV, Connors JM, Bilezikian JP, Landry DW, Wan EY. 2021. <a href="https://doi.org/10.1038/s41591-021-01283-z">Post-acute COVID-19 syndrome</a>. Nat Med 27:601–615.</div>
</div>
<div id="ref-Exnc0zrA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">109. </div><div class="csl-right-inline">Horwitz LI, Garry K, Prete AM, Sharma S, Mendoza F, Kahan T, Karpel H, Duan E, Hochman KA, Weerahandi H. 2021. <a href="https://doi.org/10.1007/s11606-021-07032-9">Six-Month Outcomes in Patients Hospitalized with Severe COVID-19</a>. J GEN INTERN MED 36:3772–3777.</div>
</div>
<div id="ref-Tn5MVYGS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">110. </div><div class="csl-right-inline">Buonsenso D, Munblit D, De Rose C, Sinatti D, Ricchiuto A, Carfi A, Valentini P. 2021. <a href="https://doi.org/10.1111/apa.15870">Preliminary evidence on long COVID in children</a>. Acta Paediatr 110:2208–2211.</div>
</div>
<div id="ref-X0iTHBZ7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">111. </div><div class="csl-right-inline">Davis HE, Assaf GS, McCorkell L, Wei H, Low RJ, Re'em Y, Redfield S, Austin JP, Akrami A. 2021. <a href="https://doi.org/10.1016/j.eclinm.2021.101019">Characterizing long COVID in an international cohort: 7 months of symptoms and their impact</a>. EClinicalMedicine 38:101019.</div>
</div>
<div id="ref-xgn7ugdb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">112. </div><div class="csl-right-inline">Huang C, Huang L, Wang Y, Li X, Ren L, Gu X, Kang L, Guo L, Liu M, Zhou X, Luo J, Huang Z, Tu S, Zhao Y, Chen L, Xu D, Li Y, Li C, Peng L, Li Y, Xie W, Cui D, Shang L, Fan G, Xu J, Wang G, Wang Y, Zhong J, Wang C, Wang J, Zhang D, Cao B. 2021. <a href="https://doi.org/10.1016/s0140-6736(20)32656-8">6-month consequences of COVID-19 in patients discharged from hospital: a cohort study</a>. The Lancet 397:220–232.</div>
</div>
<div id="ref-Rk8iz9Or" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">113. </div><div class="csl-right-inline">Rando HM, Bennett TD, Byrd JB, Bramante C, Callahan TJ, Chute CG, Davis HE, Deer R, Gagnier J, Koraishy FM, Liu F, McMurry JA, Moffitt RA, Pfaff ER, Reese JT, Relevo R, Robinson PN, Saltz JH, Solomonides A, Sule A, Topaloglu U, Haendel MA. 2021. <a href="https://doi.org/10.1101/2021.03.20.21253896">Challenges in defining Long COVID: Striking differences across literature, Electronic Health Records, and patient-reported information</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-13fa6xusy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">114. </div><div class="csl-right-inline">Deer RR, Rock MA, Vasilevsky N, Carmody L, Rando H, Anzalone AJ, Callahan TJ, Bramante CT, Chute CG, Greene CS, Gagnier J, Chu H, Koraishy FM, Liang C, Liu F, Madlock-Brown CR, Mazzotti DR, McNair DS, Parker AM, Coleman BD, Davis HE, Perry MA, Reese JT, Saltz J, Solomonides AE, Sule AA, Stein GS, Köhler S, Monteith TS, Madhira V, Kimble WD, Kavuluru R, Hillegass WB, Chan LE, Byrd JB, Boudreau EA, Liu H, McMurry JA, Pfaff E, Matentzoglu N, Relevo R, Moffitt RA, Schuff RA, Solway J, Spratt H, Bergquist T, Bennett TD, Basson MD, Topaloglu U, Wang L, Haendel MA, Robinson PN. 2021. <a href="https://doi.org/10.1101/2021.06.23.21259416">Characterizing Long COVID: Deep Phenotype of a Complex Condition</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-g435TGYc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">115. </div><div class="csl-right-inline">Fontanet A, Grant R, Tondeur L, Madec Y, Grzelak L, Cailleau I, Ungeheuer M-N, Renaudat C, Pellerin SF, Kuhmel L, Staropoli I, Anna F, Charneau P, Demeret C, Bruel T, Schwartz O, Hoen B. 2020. SARS-CoV-2 infection in primary schools in northern France: A retrospective cohort study in an area of high transmission. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.06.25.20140178">10.1101/2020.06.25.20140178</a>.</div>
</div>
<div id="ref-SHtceaPy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">116. </div><div class="csl-right-inline">Lu X, Zhang L, Du H, Zhang J, Li YY, Qu J, Zhang W, Wang Y, Bao S, Li Y, Wu C, Liu H, Liu D, Shao J, Peng X, Yang Y, Liu Z, Xiang Y, Zhang F, Silva RM, Pinkerton KE, Shen K, Xiao H, Xu S, Wong GWK. 2020. <a href="https://doi.org/10.1056/nejmc2005073">SARS-CoV-2 Infection in Children</a>. New England Journal of Medicine 382:1663–1665.</div>
</div>
<div id="ref-8sVs0pB8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">117. </div><div class="csl-right-inline">Ludvigsson JF. 2020. <a href="https://doi.org/10.1111/apa.15270">Systematic review of COVID‐19 in children shows milder cases and a better prognosis than adults</a>. Acta Paediatrica 109:1088–1095.</div>
</div>
<div id="ref-L81Cb2ZZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">118. </div><div class="csl-right-inline">Lordan R, FitzGerald GA, Grosser T. 2020. <a href="https://doi.org/10.1126/science.abe5765">Reopening schools during COVID-19</a>. Science 369:1146–1146.</div>
</div>
<div id="ref-1BhN2uq3f" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">119. </div><div class="csl-right-inline">Castagnoli R, Votto M, Licari A, Brambilla I, Bruno R, Perlini S, Rovida F, Baldanti F, Marseglia GL. 2020. <a href="https://doi.org/10.1001/jamapediatrics.2020.1467">Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection in Children and Adolescents</a>. JAMA Pediatrics 174:882.</div>
</div>
<div id="ref-12lVfu2Qe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">120. </div><div class="csl-right-inline">Abdel-Mannan O, Eyre M, Löbel U, Bamford A, Eltze C, Hameed B, Hemingway C, Hacohen Y. 2020. <a href="https://doi.org/10.1001/jamaneurol.2020.2687">Neurologic and Radiographic Findings Associated With COVID-19 Infection in Children</a>. JAMA Neurology 77:1440.</div>
</div>
<div id="ref-ShMJKFvG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">121. </div><div class="csl-right-inline">Parri N, Lenge M, Buonsenso D. 2020. <a href="https://doi.org/10.1056/nejmc2007617">Children with Covid-19 in Pediatric Emergency Departments in Italy</a>. New England Journal of Medicine 383:187–190.</div>
</div>
<div id="ref-1Bv67ENp2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">122. </div><div class="csl-right-inline">CDC. 2020. COVID Data Tracker. Centers for Disease Control and Prevention. <a href="https://covid.cdc.gov/covid-data-tracker">https://covid.cdc.gov/covid-data-tracker</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-NECDdfMZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">123. </div><div class="csl-right-inline">Mahase E. 2021. <a href="https://doi.org/10.1136/bmj.n1513">Delta variant: What is happening with transmission, hospital admissions, and restrictions?</a> BMJ n1513.</div>
</div>
<div id="ref-t4bNlJL9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">124. </div><div class="csl-right-inline">Anthes E. 2021. <a href="https://www.nytimes.com/2021/08/09/health/coronavirus-children-delta.html">The Delta Variant Is Sending More Children to the Hospital. Are They Sicker, Too?</a> The New York Times.</div>
</div>
<div id="ref-MCVe25tf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">125. </div><div class="csl-right-inline">Hoang A, Chorath K, Moreira A, Evans M, Burmeister-Morton F, Burmeister F, Naqvi R, Petershack M, Moreira A. 2020. <a href="https://doi.org/10.1016/j.eclinm.2020.100433">COVID-19 in 7780 pediatric patients: A systematic review</a>. EClinicalMedicine 24:100433.</div>
</div>
<div id="ref-jX2iYihE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">126. </div><div class="csl-right-inline">Koskela U, Helve O, Sarvikivi E, Helminen M, Nieminen T, Peltola V, Renko M, Saxén H, Pasma H, Pokka T, Honkila M, Tapiainen T. 2021. <a href="https://doi.org/10.1111/apa.16051">Multi‐inflammatory syndrome and Kawasaki disease in children during the COVID‐19 pandemic: A nationwide register‐based study and time series analysis</a>. Acta Paediatr 110:3063–3068.</div>
</div>
<div id="ref-fZwrv8M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">127. </div><div class="csl-right-inline">Chiotos K, Bassiri H, Behrens EM, Blatz AM, Chang J, Diorio C, Fitzgerald JC, Topjian A, John ARO. 2020. <a href="https://doi.org/10.1093/jpids/piaa069">Multisystem Inflammatory Syndrome in Children During the Coronavirus 2019 Pandemic: A Case Series</a>. Journal of the Pediatric Infectious Diseases Society 9:393–398.</div>
</div>
<div id="ref-AmvwCWm3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">128. </div><div class="csl-right-inline">Whittaker E, Bamford A, Kenny J, Kaforou M, Jones CE, Shah P, Ramnarayan P, Fraisse A, Miller O, Davies P, Kucera F, Brierley J, McDougall M, Carter M, Tremoulet A, Shimizu C, Herberg J, Burns JC, Lyall H, Levin M, for the PIMS-TS Study Group and EUCLIDS and PERFORM Consortia. 2020. <a href="https://doi.org/10.1001/jama.2020.10369">Clinical Characteristics of 58 Children With a Pediatric Inflammatory Multisystem Syndrome Temporally Associated With SARS-CoV-2</a>. JAMA 324:259.</div>
</div>
<div id="ref-68JorBeU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">129. </div><div class="csl-right-inline">Greene AG, Saleh M, Roseman E, Sinert R. 2020. <a href="https://doi.org/10.1016/j.ajem.2020.05.117">Toxic shock-like syndrome and COVID-19: Multisystem inflammatory syndrome in children (MIS-C)</a>. The American Journal of Emergency Medicine 38:2492.e5–2492.e6.</div>
</div>
<div id="ref-10gSXfOKm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">130. </div><div class="csl-right-inline">Diorio C, Henrickson SE, Vella LA, McNerney KO, Chase J, Burudpakdee C, Lee JH, Jasen C, Balamuth F, Barrett DM, Banwell BL, Bernt KM, Blatz AM, Chiotos K, Fisher BT, Fitzgerald JC, Gerber JS, Gollomp K, Gray C, Grupp SA, Harris RM, Kilbaugh TJ, John ARO, Lambert M, Liebling EJ, Paessler ME, Petrosa W, Phillips C, Reilly AF, Romberg ND, Seif A, Sesok-Pizzini DA, Sullivan KE, Vardaro J, Behrens EM, Teachey DT, Bassiri H. 2020. <a href="https://doi.org/10.1172/jci140970">Multisystem inflammatory syndrome in children and COVID-19 are distinct presentations of SARS–CoV-2</a>. Journal of Clinical Investigation 130:5967–5975.</div>
</div>
<div id="ref-10QhaDu6N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">131. </div><div class="csl-right-inline">Consiglio CR, Cotugno N, Sardh F, Pou C, Amodio D, Rodriguez L, Tan Z, Zicari S, Ruggiero A, Pascucci GR, Santilli V, Campbell T, Bryceson Y, Eriksson D, Wang J, Marchesi A, Lakshmikanth T, Campana A, Villani A, Rossi P, Landegren N, Palma P, Brodin P. 2020. <a href="https://doi.org/10.1016/j.cell.2020.09.016">The Immunology of Multisystem Inflammatory Syndrome in Children with COVID-19</a>. Cell 183:968–981.e7.</div>
</div>
<div id="ref-FYgN6gUL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">132. </div><div class="csl-right-inline">Belhadjer Z, Méot M, Bajolle F, Khraiche D, Legendre A, Abakka S, Auriau J, Grimaud M, Oualha M, Beghetti M, Wacker J, Ovaert C, Hascoet S, Selegny M, Malekzadeh-Milani S, Maltret A, Bosser G, Giroux N, Bonnemains L, Bordet J, Di Filippo S, Mauran P, Falcon-Eicher S, Thambo J-B, Lefort B, Moceri P, Houyel L, Renolleau S, Bonnet D. 2020. <a href="https://doi.org/10.1161/circulationaha.120.048360">Acute Heart Failure in Multisystem Inflammatory Syndrome in Children in the Context of Global SARS-CoV-2 Pandemic</a>. Circulation 142:429–436.</div>
</div>
<div id="ref-71To4laE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">133. </div><div class="csl-right-inline">Shaigany S, Gnirke M, Guttmann A, Chong H, Meehan S, Raabe V, Louie E, Solitar B, Femia A. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)31526-9">An adult with Kawasaki-like multisystem inflammatory syndrome associated with COVID-19</a>. The Lancet 396:e8–e10.</div>
</div>
<div id="ref-xQGxETbC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">134. </div><div class="csl-right-inline">Nune A, Iyengar KP, Goddard C, Ahmed AE. 2021. <a href="https://doi.org/10.1136/bcr-2021-243888">Multisystem inflammatory syndrome in an adult following the SARS-CoV-2 vaccine (MIS-V)</a>. BMJ Case Rep 14:e243888.</div>
</div>
<div id="ref-UBDAxWWv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">135. </div><div class="csl-right-inline">Sokolovsky S, Soni P, Hoffman T, Kahn P, Scheers-Masters J. 2021. <a href="https://doi.org/10.1016/j.ajem.2020.06.053">COVID-19 associated Kawasaki-like multisystem inflammatory disease in an adult</a>. The American Journal of Emergency Medicine 39:253.e1–253.e2.</div>
</div>
<div id="ref-xGHzszJs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">136. </div><div class="csl-right-inline">Boudhabhay I, Rabant M, Roumenina LT, Coupry L-M, Poillerat V, Marchal A, Frémeaux-Bacchi V, El Karoui K, Monchi M, Pourcine F. 2021. <a href="https://doi.org/10.3389/fimmu.2021.680567">Case Report: Adult Post-COVID-19 Multisystem Inflammatory Syndrome and Thrombotic Microangiopathy</a>. Front Immunol 12.</div>
</div>
<div id="ref-YMYsvZSF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">137. </div><div class="csl-right-inline">Feldstein LR, Tenforde MW, Friedman KG, Newhams M, Rose EB, Dapul H, Soma VL, Maddux AB, Mourani PM, Bowens C, Maamari M, Hall MW, Riggs BJ, Giuliano JS Jr, Singh AR, Li S, Kong M, Schuster JE, McLaughlin GE, Schwartz SP, Walker TC, Loftis LL, Hobbs CV, Halasa NB, Doymaz S, Babbitt CJ, Hume JR, Gertz SJ, Irby K, Clouser KN, Cvijanovich NZ, Bradford TT, Smith LS, Heidemann SM, Zackai SP, Wellnitz K, Nofziger RA, Horwitz SM, Carroll RW, Rowan CM, Tarquinio KM, Mack EH, Fitzgerald JC, Coates BM, Jackson AM, Young CC, Son MBF, Patel MM, Newburger JW, Randolph AG. 2021. <a href="https://doi.org/10.1001/jama.2021.2091">Characteristics and Outcomes of US Children and Adolescents With Multisystem Inflammatory Syndrome in Children (MIS-C) Compared With Severe Acute COVID-19</a>. JAMA 325:1074.</div>
</div>
<div id="ref-QaIzV00i" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">138. </div><div class="csl-right-inline">Buonsenso D, Munblit D, De Rose C, Sinatti D, Ricchiuto A, Carfi A, Valentini P. 2021. <a href="https://doi.org/10.1101/2021.01.23.21250375">Preliminary Evidence on Long COVID in children</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-vqRGmUeQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">139. </div><div class="csl-right-inline">Brackel CLH, Lap CR, Buddingh EP, van Houten MA, van der Sande LJTM, Langereis EJ, Bannier MAGE, Pijnenburg MWH, Hashimoto S, Terheggen‐Lagro SWJ. 2021. <a href="https://doi.org/10.1002/ppul.25521">Pediatric long‐COVID: An overlooked phenomenon?</a> Pediatric Pulmonology 56:2495–2502.</div>
</div>
<div id="ref-jHAr2gFf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">140. </div><div class="csl-right-inline">Say D, Crawford N, McNab S, Wurzel D, Steer A, Tosif S. 2021. <a href="https://doi.org/10.1016/s2352-4642(21)00124-3">Post-acute COVID-19 outcomes in children with mild and asymptomatic disease</a>. The Lancet Child &amp;amp; Adolescent Health 5:e22–e23.</div>
</div>
<div id="ref-14Z8mBGyq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">141. </div><div class="csl-right-inline">Radtke T, Ulyte A, Puhan MA, Kriemler S. 2021. <a href="https://doi.org/10.1101/2021.05.16.21257255">Long-term symptoms after SARS-CoV-2 infection in school children: population-based cohort with 6-months follow-up</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-1HtytUH2X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">142. </div><div class="csl-right-inline">Radtke T, Ulyte A, Puhan MA, Kriemler S. 2021. <a href="https://doi.org/10.1001/jama.2021.11880">Long-term Symptoms After SARS-CoV-2 Infection in Children and Adolescents</a>. JAMA 326:869.</div>
</div>
<div id="ref-7X38RBUl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">143. </div><div class="csl-right-inline">Mallapaty S. 2021. <a href="https://doi.org/10.1038/d41586-021-02423-8">Kids and COVID: why young immune systems are still on top</a>. Nature 597:166–168.</div>
</div>
<div id="ref-16CYY7vzG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">144. </div><div class="csl-right-inline">Cicchese JM, Evans S, Hult C, Joslyn LR, Wessler T, Millar JA, Marino S, Cilfone NA, Mattila JT, Linderman JJ, Kirschner DE. 2018. <a href="https://doi.org/10.1111/imr.12671">Dynamic balance of pro- and anti-inflammatory signals controls disease and limits pathology</a>. Immunological Reviews 285:147–167.</div>
</div>
<div id="ref-xuMYmc7W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">145. </div><div class="csl-right-inline">Elenkov IJ, Iezzoni DG, Daly A, Harris AG, Chrousos GP. 2005. <a href="https://doi.org/10.1159/000087104">Cytokine Dysregulation, Inflammation and Well-Being</a>. Neuroimmunomodulation 12:255–269.</div>
</div>
<div id="ref-sXusUkLI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">146. </div><div class="csl-right-inline">Chen L, Deng H, Cui H, Fang J, Zuo Z, Deng J, Li Y, Wang X, Zhao L. 2017. <a href="https://doi.org/10.18632/oncotarget.23208">Inflammatory responses and inflammation-associated diseases in organs</a>. Oncotarget 9:7204–7218.</div>
</div>
<div id="ref-o6BQnEt7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">147. </div><div class="csl-right-inline">2002. Molecular biology of the cell4th ed. Garland Science, New York.</div>
</div>
<div id="ref-b1QVdemU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">148. </div><div class="csl-right-inline">Widmaier EP, Raff H, Strang KT. 2008. Vander's human physiology: the mechanisms of body function. McGraw-Hill Higher Education, Boston.</div>
</div>
<div id="ref-1GnFL9zeN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">149. </div><div class="csl-right-inline">McKechnie JL, Blish CA. 2020. <a href="https://doi.org/10.1016/j.chom.2020.05.009">The Innate Immune System: Fighting on the Front Lines or Fanning the Flames of COVID-19?</a> Cell Host &amp; Microbe 27:863–869.</div>
</div>
<div id="ref-bZMKqj6e" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">150. </div><div class="csl-right-inline">Tisoncik JR, Korth MJ, Simmons CP, Farrar J, Martin TR, Katze MG. 2012. <a href="https://doi.org/10.1128/mmbr.05015-11">Into the Eye of the Cytokine Storm</a>. Microbiology and Molecular Biology Reviews 76:16–32.</div>
</div>
<div id="ref-6eWgCrWj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">151. </div><div class="csl-right-inline">Hall MJ, Williams SN, DeFrances CJ, Golosinskiy A. 2011. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22142805">Inpatient care for septicemia or sepsis: a challenge for patients and hospitals.</a> NCHS Data Brief 1–8.</div>
</div>
<div id="ref-jqm9b6lZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">152. </div><div class="csl-right-inline">Gu X, Zhou F, Wang Y, Fan G, Cao B. 2020. <a href="https://doi.org/10.1183/16000617.0038-2020">Respiratory viral sepsis: epidemiology, pathophysiology, diagnosis and treatment</a>. Eur Respir Rev 29:200038.</div>
</div>
<div id="ref-sLCBjlMH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">153. </div><div class="csl-right-inline">Li H, Liu L, Zhang D, Xu J, Dai H, Tang N, Su X, Cao B. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30920-x">SARS-CoV-2 and viral sepsis: observations and hypotheses</a>. The Lancet 395:1517–1520.</div>
</div>
<div id="ref-f61jsRKY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">154. </div><div class="csl-right-inline">PARK WILLIAM Y, GOODMAN RICHARD B, STEINBERG KENNETH P, RUZINSKI JOHN T, RADELLA F, PARK DAVID R, PUGIN J, SKERRETT SHAWN J, HUDSON LEONARD D, MARTIN THOMAS R. 2001. <a href="https://doi.org/10.1164/ajrccm.164.10.2104013">Cytokine Balance in the Lungs of Patients with Acute Respiratory Distress Syndrome</a>. American Journal of Respiratory and Critical Care Medicine 164:1896–1903.</div>
</div>
<div id="ref-NITOJ0Ka" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">155. </div><div class="csl-right-inline">Shimabukuro-Vornhagen A, Gödel P, Subklewe M, Stemmler HJ, Schlößer HA, Schlaak M, Kochanek M, Böll B, von Bergwelt-Baildon MS. 2018. <a href="https://doi.org/10.1186/s40425-018-0343-9">Cytokine release syndrome</a>. Journal for ImmunoTherapy of Cancer 6:56.</div>
</div>
<div id="ref-NNFGje9g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">156. </div><div class="csl-right-inline">He L, Ding Y, Zhang Q, Che X, He Y, Shen H, Wang H, Li Z, Zhao L, Geng J, Deng Y, Yang L, Li J, Cai J, Qiu L, Wen K, Xu X, Jiang S. 2006. <a href="https://doi.org/10.1002/path.2067">Expression of elevated levels of pro-inflammatory cytokines in SARS-CoV-infected ACE2 &lt;sup&gt;+&lt;/sup&gt; cells in SARS patients: relation to the acute lung injury and pathogenesis of SARS</a>. The Journal of Pathology 210:288–297.</div>
</div>
<div id="ref-QkTGQUcj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">157. </div><div class="csl-right-inline">Wang W, Ye L, Ye L, Li B, Gao B, Zeng Y, Kong L, Fang X, Zheng H, Wu Z, She Y. 2007. <a href="https://doi.org/10.1016/j.virusres.2007.02.007">Up-regulation of IL-6 and TNF-α induced by SARS-coronavirus spike protein in murine macrophages via NF-κB pathway</a>. Virus Research 128:1–8.</div>
</div>
<div id="ref-3HdlV9Vf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">158. </div><div class="csl-right-inline">Mehta P, McAuley DF, Brown M, Sanchez E, Tattersall RS, Manson JJ. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30628-0">COVID-19: consider cytokine storm syndromes and immunosuppression</a>. The Lancet 395:1033–1034.</div>
</div>
<div id="ref-cyUG1Zi2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">159. </div><div class="csl-right-inline">Mangalmurti N, Hunter CA. 2020. <a href="https://doi.org/10.1016/j.immuni.2020.06.017">Cytokine Storms: Understanding COVID-19</a>. Immunity 53:19–25.</div>
</div>
<div id="ref-oSOG8N0f" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">160. </div><div class="csl-right-inline">Kellum JA. 2007. <a href="https://doi.org/10.1001/archinte.167.15.1655">Understanding the Inflammatory Cytokine Response in Pneumonia and Sepsis</a>. Archives of Internal Medicine 167:1655.</div>
</div>
<div id="ref-14VESs80B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">161. </div><div class="csl-right-inline">Scheller J, Chalaris A, Schmidt-Arras D, Rose-John S. 2011. <a href="https://doi.org/10.1016/j.bbamcr.2011.01.034">The pro- and anti-inflammatory properties of the cytokine interleukin-6</a>. Biochimica et Biophysica Acta (BBA) - Molecular Cell Research 1813:878–888.</div>
</div>
<div id="ref-dBAe8aYi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">162. </div><div class="csl-right-inline">Velazquez-Salinas L, Verdugo-Rodriguez A, Rodriguez LL, Borca MV. 2019. <a href="https://doi.org/10.3389/fmicb.2019.01057">The Role of Interleukin 6 During Viral Infections</a>. Frontiers in Microbiology 10:1057.</div>
</div>
<div id="ref-Xcrxlxnp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">163. </div><div class="csl-right-inline">Sinha P, Matthay MA, Calfee CS. 2020. <a href="https://doi.org/10.1001/jamainternmed.2020.3313">Is a “Cytokine Storm” Relevant to COVID-19?</a> JAMA Internal Medicine 180:1152.</div>
</div>
<div id="ref-RIMWgsWt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">164. </div><div class="csl-right-inline">Liu B, Li M, Zhou Z, Guan X, Xiang Y. 2020. <a href="https://doi.org/10.1016/j.jaut.2020.102452">Can we use interleukin-6 (IL-6) blockade for coronavirus disease 2019 (COVID-19)-induced cytokine release syndrome (CRS)?</a> Journal of Autoimmunity 111:102452.</div>
</div>
<div id="ref-8esIIWt9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">165. </div><div class="csl-right-inline">Eckhardt M, Hultquist JF, Kaake RM, Hüttenhain R, Krogan NJ. 2020. <a href="https://doi.org/10.1038/s41576-020-0212-5">A systems approach to infectious disease</a>. Nature Reviews Genetics 21:339–354.</div>
</div>
<div id="ref-8vUHj9cm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">166. </div><div class="csl-right-inline">Ray S, Patel SK, Kumar V, Damahe J, Srivastava S. 2014. <a href="https://doi.org/10.1002/prca.201300074">Differential expression of serum/plasma proteins in various infectious diseases: Specific or nonspecific signatures</a>. PROTEOMICS - Clinical Applications 8:53–72.</div>
</div>
<div id="ref-pHrstqMQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">167. </div><div class="csl-right-inline">Blanco-Melo D, Nilsson-Payant BE, Liu W-C, Uhl S, Hoagland D, Møller R, Jordan TX, Oishi K, Panis M, Sachs D, Wang TT, Schwartz RE, Lim JK, Albrecht RA, tenOever BR. 2020. <a href="https://doi.org/10.1016/j.cell.2020.04.026">Imbalanced Host Response to SARS-CoV-2 Drives Development of COVID-19</a>. Cell 181:1036–1045.e9.</div>
</div>
<div id="ref-16NxbPnxZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">168. </div><div class="csl-right-inline">Versteeg GA, García-Sastre A. 2010. <a href="https://doi.org/10.1016/j.mib.2010.05.009">Viral tricks to grid-lock the type I interferon system</a>. Current Opinion in Microbiology 13:508–516.</div>
</div>
<div id="ref-20zcMj6d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">169. </div><div class="csl-right-inline">Stewart CE, Randall RE, Adamson CS. 2014. <a href="https://doi.org/10.1371/journal.pone.0112014">Inhibitors of the Interferon Response Enhance Virus Replication In Vitro</a>. PLoS ONE 9:e112014.</div>
</div>
<div id="ref-DuPmqcag" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">170. </div><div class="csl-right-inline">Kopecky-Bromberg SA, Martínez-Sobrido L, Frieman M, Baric RA, Palese P. 2007. <a href="https://doi.org/10.1128/jvi.01782-06">Severe Acute Respiratory Syndrome Coronavirus Open Reading Frame (ORF) 3b, ORF 6, and Nucleocapsid Proteins Function as Interferon Antagonists</a>. J Virol 81:548–557.</div>
</div>
<div id="ref-6ovCW0LD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">171. </div><div class="csl-right-inline">Niemeyer D, Zillinger T, Muth D, Zielecki F, Horvath G, Suliman T, Barchet W, Weber F, Drosten C, Müller MA. 2013. <a href="https://doi.org/10.1128/jvi.01845-13">Middle East Respiratory Syndrome Coronavirus Accessory Protein 4a Is a Type I Interferon Antagonist</a>. J Virol 87:12489–12495.</div>
</div>
<div id="ref-UwVweB2M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">172. </div><div class="csl-right-inline">Wang P-H, Cheng Y. 2020. Increasing Host Cellular Receptor—Angiotensin-Converting Enzyme 2 (ACE2) Expression by Coronavirus may Facilitate 2019-nCoV Infection. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.24.963348">10.1101/2020.02.24.963348</a>.</div>
</div>
<div id="ref-UqnfNPs7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">173. </div><div class="csl-right-inline">Yuen C-K, Lam J-Y, Wong W-M, Mak L-F, Wang X, Chu H, Cai J-P, Jin D-Y, To KK-W, Chan JF-W, Yuen K-Y, Kok K-H. 2020. <a href="https://doi.org/10.1080/22221751.2020.1780953">SARS-CoV-2 nsp13, nsp14, nsp15 and orf6 function as potent interferon antagonists</a>. Emerging Microbes &amp;amp; Infections 9:1418–1428.</div>
</div>
<div id="ref-19YhL3ZEM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">174. </div><div class="csl-right-inline">Konno Y, Kimura I, Uriu K, Fukushi M, Irie T, Koyanagi Y, Sauter D, Gifford RJ, Nakagawa S, Sato K. 2020. <a href="https://doi.org/10.1016/j.celrep.2020.108185">SARS-CoV-2 ORF3b Is a Potent Interferon Antagonist Whose Activity Is Increased by a Naturally Occurring Elongation Variant</a>. Cell Reports 32:108185.</div>
</div>
<div id="ref-Gj8vlc0W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">175. </div><div class="csl-right-inline">Emanuel W, Kirstin M, Vedran F, Asija D, Theresa GL, Roberto A, Filippos K, David K, Salah A, Christopher B, Anja R, Ivano L, Andranik I, Tommaso M, Simone DG, Patrick PJ, Alexander MM, Daniela N, Matthias S, Altuna A, Nikolaus R, Christian D, Markus L. 2020. Bulk and single-cell gene expression profiling of SARS-CoV-2 infected human cell lines identifies molecular targets for therapeutic intervention. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.05.05.079194">10.1101/2020.05.05.079194</a>.</div>
</div>
<div id="ref-16EFyBURq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">176. </div><div class="csl-right-inline">Harcourt J, Tamin A, Lu X, Kamili S, Sakthivel SKumar, Murray J, Queen K, Tao Y, Paden CR, Zhang J, Li Y, Uehara A, Wang H, Goldsmith C, Bullock HA, Wang L, Whitaker B, Lynch B, Gautam R, Schindewolf C, Lokugamage KG, Scharton D, Plante JA, Mirchandani D, Widen SG, Narayanan K, Makino S, Ksiazek TG, Plante KS, Weaver SC, Lindstrom S, Tong S, Menachery VD, Thornburg NJ. 2020. Isolation and characterization of SARS-CoV-2 from the first US COVID-19 patient. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.02.972935">10.1101/2020.03.02.972935</a>.</div>
</div>
<div id="ref-13z84exS5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">177. </div><div class="csl-right-inline">Aschenbrenner AC, Mouktaroudi M, Krämer B, Oestreich M, Antonakos N, Nuesch-Germano M, Gkizeli K, Bonaguro L, Reusch N, Baßler K, Saridaki M, Knoll R, Pecht T, Kapellos TS, Doulou S, Kröger C, Herbert M, Holsten L, Horne A, Gemünd ID, Rovina N, Agrawal S, Dahm K, van Uelft M, Drews A, Lenkeit L, Bruse N, Gerretsen J, Gierlich J, Becker M, Händler K, Kraut M, Theis H, Mengiste S, De Domenico E, Schulte-Schrepping J, Seep L, Raabe J, Hoffmeister C, ToVinh M, Keitel V, Rieke G, Talevi V, Skowasch D, Aziz NA, Pickkers P, van de Veerdonk FL, Netea MG, Schultze JL, Kox M, Breteler MMB, Nattermann J, Koutsoukou A, Giamarellos-Bourboulis EJ, Ulas T. 2021. <a href="https://doi.org/10.1186/s13073-020-00823-5">Disease severity-specific neutrophil signatures in blood transcriptomes stratify COVID-19 patients</a>. Genome Med 13.</div>
</div>
<div id="ref-15LlkGLbh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">178. </div><div class="csl-right-inline">Bernardes JP, Mishra N, Tran F, Bahmer T, Best L, Blase JI, Bordoni D, Franzenburg J, Geisen U, Josephs-Spaulding J, Köhler P, Künstner A, Rosati E, Aschenbrenner AC, Bacher P, Baran N, Boysen T, Brandt B, Bruse N, Dörr J, Dräger A, Elke G, Ellinghaus D, Fischer J, Forster M, Franke A, Franzenburg S, Frey N, Friedrichs A, Fuß J, Glück A, Hamm J, Hinrichsen F, Hoeppner MP, Imm S, Junker R, Kaiser S, Kan YH, Knoll R, Lange C, Laue G, Lier C, Lindner M, Marinos G, Markewitz R, Nattermann J, Noth R, Pickkers P, Rabe KF, Renz A, Röcken C, Rupp J, Schaffarzyk A, Scheffold A, Schulte-Schrepping J, Schunk D, Skowasch D, Ulas T, Wandinger K-P, Wittig M, Zimmermann J, Busch H, Hoyer BF, Kaleta C, Heyckendorf J, Kox M, Rybniker J, Schreiber S, Schultze JL, Rosenstiel P, Banovich NE, Desai T, Eickelberg O, Haniffa M, Horvath P, Kropski JA, Lafyatis R, Lundeberg J, Meyer K, Nawijn MC, Nikolic M, Ordovas Montanes J, Pe’er D, Tata PR, Rawlins E, Regev A, Reyfman P, Samakovlis C, Schultze J, Shalek A, Shepherd D, Spence J, Teichmann S, Theis F, Tsankov A, van den Berge M, von Papen M, Whitsett J, Zaragosi LE, Angelov A, Bals R, Bartholomäus A, Becker A, Bezdan D, Bonifacio E, Bork P, Clavel T, Colme-Tatche M, Diefenbach A, Dilthey A, Fischer N, Förstner K, Frick J-S, Gagneur J, Goesmann A, Hain T, Hummel M, Janssen S, Kalinowski J, Kallies R, Kehr B, Keller A, Kim-Hellmuth S, Klein C, Kohlbacher O, Korbel JO, Kurth I, Landthaler M, Li Y, Ludwig K, Makarewicz O, Marz M, McHardy A, Mertes C, Nöthen M, Nürnberg P, Ohler U, Ossowski S, Overmann J, Peter S, Pfeffer K, Poetsch AR, Pühler A, Rajewsky N, Ralser M, Rieß O, Ripke S, Nunes da Rocha U, Rosenstiel P, Saliba A-E, Sander LE, Sawitzki B, Schiffer P, Schulte E-C, Schultze JL, Sczyrba A, Stegle O, Stoye J, Theis F, Vehreschild J, Vogel J, von Kleist M, Walker A, Walter J, Wieczorek D, Ziebuhr J. 2020. <a href="https://doi.org/10.1016/j.immuni.2020.11.017">Longitudinal Multi-omics Analyses Identify Responses of Megakaryocytes, Erythroid Cells, and Plasmablasts as Hallmarks of Severe COVID-19</a>. Immunity 53:1296–1314.e9.</div>
</div>
<div id="ref-34hQP4Fi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">179. </div><div class="csl-right-inline">Xiong Y, Liu Y, Cao L, Wang D, Guo M, Jiang A, Guo D, Hu W, Yang J, Tang Z, Wu H, Lin Y, Zhang M, Zhang Q, Shi M, Liu Y, Zhou Y, Lan K, Chen Y. 2020. <a href="https://doi.org/10.1080/22221751.2020.1747363">Transcriptomic characteristics of bronchoalveolar lavage fluid and peripheral blood mononuclear cells in COVID-19 patients</a>. Emerging Microbes &amp;amp; Infections 9:761–770.</div>
</div>
<div id="ref-KNGUuuNx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">180. </div><div class="csl-right-inline">Liu C, Martins AJ, Lau WW, Rachmaninoff N, Chen J, Imberti L, Mostaghimi D, Fink DL, Burbelo PD, Dobbs K, Delmonte OM, Bansal N, Failla L, Sottini A, Quiros-Roldan E, Han KL, Sellers BA, Cheung F, Sparks R, Chun T-W, Moir S, Lionakis MS, Rossi C, Su HC, Kuhns DB, Cohen JI, Notarangelo LD, Tsang JS, Abers MS, Apps R, Bosticardo M, Milanez-Almeida P, Mulè MP, Shaw E, Zhang Y, Castelli F, Muiesan ML, Tomasoni G, Scolari F, Tucci A. 2021. <a href="https://doi.org/10.1016/j.cell.2021.02.018">Time-resolved systems immunology reveals a late juncture linked to fatal COVID-19</a>. Cell 184:1836–1857.e22.</div>
</div>
<div id="ref-WOUILlou" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">181. </div><div class="csl-right-inline">Prokop JW, Hartog NL, Chesla D, Faber W, Love CP, Karam R, Abualkheir N, Feldmann B, Teng L, McBride T, Leimanis ML, English BK, Holsworth A, Frisch A, Bauss J, Kalpage N, Derbedrossian A, Pinti RM, Hale N, Mills J, Eby A, VanSickle EA, Pageau SC, Shankar R, Chen B, Carcillo JA, Sanfilippo D, Olivero R, Bupp CP, Rajasekaran S. 2021. <a href="https://doi.org/10.3389/fimmu.2021.694243">High-Density Blood Transcriptomics Reveals Precision Immune Signatures of SARS-CoV-2 Infection in Hospitalized Individuals</a>. Front Immunol 12.</div>
</div>
<div id="ref-196k8asXZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">182. </div><div class="csl-right-inline">Arunachalam PS, Wimmers F, Mok CKP, Perera RAPM, Scott M, Hagan T, Sigal N, Feng Y, Bristow L, Tak-Yin Tsang O, Wagh D, Coller J, Pellegrini KL, Kazmin D, Alaaeddine G, Leung WS, Chan JMC, Chik TSH, Choi CYC, Huerta C, Paine McCullough M, Lv H, Anderson E, Edupuganti S, Upadhyay AA, Bosinger SE, Maecker HT, Khatri P, Rouphael N, Peiris M, Pulendran B. 2020. <a href="https://doi.org/10.1126/science.abc6261">Systems biological assessment of immunity to mild versus severe COVID-19 infection in humans</a>. Science 369:1210–1220.</div>
</div>
<div id="ref-W6xJeuhR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">183. </div><div class="csl-right-inline">Cheemarla NR, Watkins TA, Mihaylova VT, Wang B, Zhao D, Wang G, Landry ML, Foxman EF. 2021. <a href="https://doi.org/10.1084/jem.20210583">Dynamic innate immune response determines susceptibility to SARS-CoV-2 infection and early replication kinetics</a>. Journal of Experimental Medicine 218.</div>
</div>
<div id="ref-Uakl0VBe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">184. </div><div class="csl-right-inline">Yang AC, Kern F, Losada PM, Agam MR, Maat CA, Schmartz GP, Fehlmann T, Stein JA, Schaum N, Lee DP, Calcuttawala K, Vest RT, Berdnik D, Lu N, Hahn O, Gate D, McNerney MW, Channappa D, Cobos I, Ludwig N, Schulz-Schaeffer WJ, Keller A, Wyss-Coray T. 2021. <a href="https://doi.org/10.1038/s41586-021-03710-0">Dysregulation of brain and choroid plexus cell types in severe COVID-19</a>. Nature 595:565–571.</div>
</div>
<div id="ref-11xZWeHN3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">185. </div><div class="csl-right-inline">Bojkova D, Klann K, Koch B, Widera M, Krause D, Ciesek S, Cinatl J, Münch C. 2020. <a href="https://doi.org/10.1038/s41586-020-2332-7">Proteomics of SARS-CoV-2-infected host cells reveals therapy targets</a>. Nature 583:469–472.</div>
</div>
<div id="ref-GhJYjnft" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">186. </div><div class="csl-right-inline">Ju B, Zhang Q, Ge X, Wang R, Yu J, Shan S, Zhou B, Song S, Tang X, Yu J, Ge J, Lan J, Yuan J, Wang H, Zhao J, Zhang S, Wang Y, Shi X, Liu L, Wang X, Zhang Z, Zhang L. 2020. Potent human neutralizing antibodies elicited by SARS-CoV-2 infection. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.21.990770">10.1101/2020.03.21.990770</a>.</div>
</div>
<div id="ref-GpnngtWK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">187. </div><div class="csl-right-inline">Chen J-H, Chang Y-W, Yao C-W, Chiueh T-S, Huang S-C, Chien K-Y, Chen A, Chang F-Y, Wong C-H, Chen Y-J. 2004. <a href="https://doi.org/10.1073/pnas.0407992101">Plasma proteome of severe acute respiratory syndrome analyzed by two-dimensional gel electrophoresis and mass spectrometry</a>. Proceedings of the National Academy of Sciences 101:17039–17044.</div>
</div>
<div id="ref-JLTf2Fwb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">188. </div><div class="csl-right-inline">He R, Dobie F, Ballantine M, Leeson A, Li Y, Bastien N, Cutts T, Andonov A, Cao J, Booth TF, Plummer FA, Tyler S, Baker L, Li X. 2004. <a href="https://doi.org/10.1016/j.bbrc.2004.02.074">Analysis of multimerization of the SARS coronavirus nucleocapsid protein</a>. Biochemical and Biophysical Research Communications 316:476–483.</div>
</div>
<div id="ref-1ZE22clL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">189. </div><div class="csl-right-inline">The UniProt Consortium. 2019. <a href="https://doi.org/10.1093/nar/gky1049">UniProt: a worldwide hub of protein knowledge</a>. Nucleic Acids Research 47:D506–D515.</div>
</div>
<div id="ref-RrqhkfKV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">190. </div><div class="csl-right-inline">Home - Genome - NCBI. <a href="https://www.ncbi.nlm.nih.gov/genome">https://www.ncbi.nlm.nih.gov/genome</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-FE7YVar3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">191. </div><div class="csl-right-inline">Vita R, Mahajan S, Overton JA, Dhanda SK, Martini S, Cantrell JR, Wheeler DK, Sette A, Peters B. 2019. <a href="https://doi.org/10.1093/nar/gky1006">The Immune Epitope Database (IEDB): 2018 update</a>. Nucleic Acids Research 47:D339–D343.</div>
</div>
<div id="ref-BmUn7iIt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">192. </div><div class="csl-right-inline">Pickett BE, Sadat EL, Zhang Y, Noronha JM, Squires RB, Hunt V, Liu M, Kumar S, Zaremba S, Gu Z, Zhou L, Larson CN, Dietrich J, Klem EB, Scheuermann RH. 2012. <a href="https://doi.org/10.1093/nar/gkr859">ViPR: an open bioinformatics database and analysis resource for virology research</a>. Nucleic Acids Research 40:D593–D598.</div>
</div>
<div id="ref-ZvaFbLRX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">193. </div><div class="csl-right-inline">Gordon DE, Jang GM, Bouhaddou M, Xu J, Obernier K, White KM, O’Meara MJ, Rezelj VV, Guo JZ, Swaney DL, Tummino TA, Hüttenhain R, Kaake RM, Richards AL, Tutuncuoglu B, Foussard H, Batra J, Haas K, Modak M, Kim M, Haas P, Polacco BJ, Braberg H, Fabius JM, Eckhardt M, Soucheray M, Bennett MJ, Cakir M, McGregor MJ, Li Q, Meyer B, Roesch F, Vallet T, Mac Kain A, Miorin L, Moreno E, Naing ZZC, Zhou Y, Peng S, Shi Y, Zhang Z, Shen W, Kirby IT, Melnyk JE, Chorba JS, Lou K, Dai SA, Barrio-Hernandez I, Memon D, Hernandez-Armenta C, Lyu J, Mathy CJP, Perica T, Pilla KB, Ganesan SJ, Saltzberg DJ, Rakesh R, Liu X, Rosenthal SB, Calviello L, Venkataramanan S, Liboy-Lugo J, Lin Y, Huang X-P, Liu Y, Wankowicz SA, Bohn M, Safari M, Ugur FS, Koh C, Savar NS, Tran QD, Shengjuler D, Fletcher SJ, O’Neal MC, Cai Y, Chang JCJ, Broadhurst DJ, Klippsten S, Sharp PP, Wenzell NA, Kuzuoglu-Ozturk D, Wang H-Y, Trenker R, Young JM, Cavero DA, Hiatt J, Roth TL, Rathore U, Subramanian A, Noack J, Hubert M, Stroud RM, Frankel AD, Rosenberg OS, Verba KA, Agard DA, Ott M, Emerman M, Jura N, von Zastrow M, Verdin E, Ashworth A, Schwartz O, d’Enfert C, Mukherjee S, Jacobson M, Malik HS, Fujimori DG, Ideker T, Craik CS, Floor SN, Fraser JS, Gross JD, Sali A, Roth BL, Ruggero D, Taunton J, Kortemme T, Beltrao P, Vignuzzi M, García-Sastre A, Shokat KM, Shoichet BK, Krogan NJ. 2020. <a href="https://doi.org/10.1038/s41586-020-2286-9">A SARS-CoV-2 protein interaction map reveals targets for drug repurposing</a>. Nature 583:459–468.</div>
</div>
<div id="ref-mXUCjmCh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">194. </div><div class="csl-right-inline">Sobocińska J, Roszczenko-Jasińska P, Ciesielska A, Kwiatkowska K. 2018. <a href="https://doi.org/10.3389/fimmu.2017.02003">Protein Palmitoylation and Its Role in Bacterial and Viral Infections</a>. Frontiers in Immunology 8:2003.</div>
</div>
<div id="ref-NLoN4aYj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">195. </div><div class="csl-right-inline">Li J, Guo M, Tian X, Liu C, Wang X, Yang X, Wu P, Xiao Z, Qu Y, Yin Y, Fu J, Zhu Z, Liu Z, Peng C, Zhu T, Liang Q. 2020. Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.31.019216">10.1101/2020.03.31.019216</a>.</div>
</div>
<div id="ref-KAqOiTeZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">196. </div><div class="csl-right-inline">Lawrence T. 2009. <a href="https://doi.org/10.1101/cshperspect.a001651">The Nuclear Factor NF- B Pathway in Inflammation</a>. Cold Spring Harbor Perspectives in Biology 1:a001651–a001651.</div>
</div>
<div id="ref-1ccnm0N9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">197. </div><div class="csl-right-inline">Navratil V, Lionnard L, Longhi S, Hardwick JM, Combet C, Aouacheria A. 2020. The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) envelope (E) protein harbors a conserved BH3-like sequence. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.04.09.033522">10.1101/2020.04.09.033522</a>.</div>
</div>
<div id="ref-3DOhQZCg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">198. </div><div class="csl-right-inline">Overmyer KA, Shishkova E, Miller IJ, Balnis J, Bernstein MN, Peters-Clarke TM, Meyer JG, Quan Q, Muehlbauer LK, Trujillo EA, He Y, Chopra A, Chieng HC, Tiwari A, Judson MA, Paulson B, Brademan DR, Zhu Y, Serrano LR, Linke V, Drake LA, Adam AP, Schwartz BS, Singer HA, Swanson S, Mosher DF, Stewart R, Coon JJ, Jaitovich A. 2021. <a href="https://doi.org/10.1016/j.cels.2020.10.003">Large-Scale Multi-omic Analysis of COVID-19 Severity</a>. Cell Systems 12:23–40.e7.</div>
</div>
<div id="ref-1EAPLqW23" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">199. </div><div class="csl-right-inline">Geyer PE, Arend FM, Doll S, Louiset M, Virreira Winter S, Müller‐Reif JB, Torun FM, Weigand M, Eichhorn P, Bruegel M, Strauss MT, Holdt LM, Mann M, Teupser D. 2021. <a href="https://doi.org/10.15252/emmm.202114167">High‐resolution serum proteome trajectories in COVID‐19 reveal patient‐specific seroconversion</a>. EMBO Mol Med 13.</div>
</div>
<div id="ref-15EIBRhef" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">200. </div><div class="csl-right-inline">Li F. 2005. <a href="https://doi.org/10.1126/science.1116480">Structure of SARS Coronavirus Spike Receptor-Binding Domain Complexed with Receptor</a>. Science 309:1864–1868.</div>
</div>
<div id="ref-12TZ7hPMA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">201. </div><div class="csl-right-inline">Yan R, Zhang Y, Li Y, Xia L, Guo Y, Zhou Q. 2020. <a href="https://doi.org/10.1126/science.abb2762">Structural basis for the recognition of SARS-CoV-2 by full-length human ACE2</a>. Science 367:1444–1448.</div>
</div>
<div id="ref-wCbhn23d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">202. </div><div class="csl-right-inline">Shang J, Ye G, Shi K, Wan Y, Luo C, Aihara H, Geng Q, Auerbach A, Li F. 2020. <a href="https://doi.org/10.1038/s41586-020-2179-y">Structural basis of receptor recognition by SARS-CoV-2</a>. Nature 581:221–224.</div>
</div>
<div id="ref-VX8OWaGj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">203. </div><div class="csl-right-inline">Lan J, Ge J, Yu J, Shan S, Zhou H, Fan S, Zhang Q, Shi X, Wang Q, Zhang L, Wang X. 2020. Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.19.956235">10.1101/2020.02.19.956235</a>.</div>
</div>
<div id="ref-15Q2XgkK7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">204. </div><div class="csl-right-inline">Wang Q, Zhang Y, Wu L, Niu S, Song C, Zhang Z, Lu G, Qiao C, Hu Y, Yuen K-Y, Wang Q, Zhou H, Yan J, Qi J. 2020. <a href="https://doi.org/10.1016/j.cell.2020.03.045">Structural and Functional Basis of SARS-CoV-2 Entry by Using Human ACE2</a>. Cell 181:894–904.e9.</div>
</div>
<div id="ref-t1e4CW9A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">205. </div><div class="csl-right-inline">Wan Y, Shang J, Graham R, Baric RS, Li F. 2020. <a href="https://doi.org/10.1128/jvi.00127-20">Receptor Recognition by the Novel Coronavirus from Wuhan: an Analysis Based on Decade-Long Structural Studies of SARS Coronavirus</a>. Journal of Virology 94:e00127–20.</div>
</div>
<div id="ref-GbQPEvZG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">206. </div><div class="csl-right-inline">Wrobel AG, Benton DJ, Xu P, Roustan C, Martin SR, Rosenthal PB, Skehel JJ, Gamblin SJ. 2020. <a href="https://doi.org/10.1038/s41594-020-0468-7">SARS-CoV-2 and bat RaTG13 spike glycoprotein structures inform on virus evolution and furin-cleavage effects</a>. Nat Struct Mol Biol 27:763–767.</div>
</div>
<div id="ref-VSkK7CeP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">207. </div><div class="csl-right-inline">Zhou P, Yang X-L, Wang X-G, Hu B, Zhang L, Zhang W, Si H-R, Zhu Y, Li B, Huang C-L, Chen H-D, Chen J, Luo Y, Guo H, Jiang R-D, Liu M-Q, Chen Y, Shen X-R, Wang X, Zheng X-S, Zhao K, Chen Q-J, Deng F, Liu L-L, Yan B, Zhan F-X, Wang Y-Y, Xiao G-F, Shi Z-L. 2020. <a href="https://doi.org/10.1038/s41586-020-2012-7">A pneumonia outbreak associated with a new coronavirus of probable bat origin</a>. Nature 579:270–273.</div>
</div>
<div id="ref-NsORsLig" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">208. </div><div class="csl-right-inline">Chen J, Lee KH, Steinhauer DA, Stevens DJ, Skehel JJ, Wiley DC. 1998. <a href="https://doi.org/10.1016/s0092-8674(00)81771-7">Structure of the Hemagglutinin Precursor Cleavage Site, a Determinant of Influenza Pathogenicity and the Origin of the Labile Conformation</a>. Cell 95:409–417.</div>
</div>
<div id="ref-vhHB3yyS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">209. </div><div class="csl-right-inline">Steinhauer DA. 1999. <a href="https://doi.org/10.1006/viro.1999.9716">Role of Hemagglutinin Cleavage for the Pathogenicity of Influenza Virus</a>. Virology 258:1–20.</div>
</div>
<div id="ref-p9LUiyCN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">210. </div><div class="csl-right-inline">Pachetti M, Marini B, Benedetti F, Giudici F, Mauro E, Storici P, Masciovecchio C, Angeletti S, Ciccozzi M, Gallo RC, Zella D, Ippodrino R. 2020. <a href="https://doi.org/10.1186/s12967-020-02344-6">Emerging SARS-CoV-2 mutation hot spots include a novel RNA-dependent-RNA polymerase variant</a>. Journal of Translational Medicine 18:179.</div>
</div>
<div id="ref-UUAeVUaR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">211. </div><div class="csl-right-inline">Tang X, Wu C, Li X, Song Y, Yao X, Wu X, Duan Y, Zhang H, Wang Y, Qian Z, Cui J, Lu J. 2020. <a href="https://doi.org/10.1093/nsr/nwaa036">On the origin and continuing evolution of SARS-CoV-2</a>. National Science Review 7:1012–1023.</div>
</div>
<div id="ref-2w7lNKxQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">212. </div><div class="csl-right-inline">van Dorp L, Acman M, Richard D, Shaw LP, Ford CE, Ormond L, Owen CJ, Pang J, Tan CCS, Boshier FAT, Ortiz AT, Balloux F. 2020. <a href="https://doi.org/10.1016/j.meegid.2020.104351">Emergence of genomic diversity and recurrent mutations in SARS-CoV-2</a>. Infection, Genetics and Evolution 83:104351.</div>
</div>
<div id="ref-SezJVKA6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">213. </div><div class="csl-right-inline">Lu J, du Plessis L, Liu Z, Hill V, Kang M, Lin H, Sun J, François S, Kraemer MUG, Faria NR, McCrone JT, Peng J, Xiong Q, Yuan R, Zeng L, Zhou P, Liang C, Yi L, Liu J, Xiao J, Hu J, Liu T, Ma W, Li W, Su J, Zheng H, Peng B, Fang S, Su W, Li K, Sun R, Bai R, Tang X, Liang M, Quick J, Song T, Rambaut A, Loman N, Raghwani J, Pybus OG, Ke C. 2020. <a href="https://doi.org/10.1016/j.cell.2020.04.023">Genomic Epidemiology of SARS-CoV-2 in Guangdong Province, China</a>. Cell 181:997–1003.e9.</div>
</div>
<div id="ref-jVbH3kJR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">214. </div><div class="csl-right-inline">2020. <a href="https://doi.org/10.1016/s2666-5247(20)30054-9">An integrated national scale SARS-CoV-2 genomic surveillance network</a>. The Lancet Microbe 1:e99–e100.</div>
</div>
<div id="ref-11nsMZ5u3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">215. </div><div class="csl-right-inline">CDC. 2020. Cases, Data, and Surveillance. Centers for Disease Control and Prevention. <a href="https://www.cdc.gov/coronavirus/2019-ncov/variants/spheres.html">https://www.cdc.gov/coronavirus/2019-ncov/variants/spheres.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-W5Z7Ztlg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">216. </div><div class="csl-right-inline">Fauver JR, Petrone ME, Hodcroft EB, Shioda K, Ehrlich HY, Watts AG, Vogels CBF, Brito AF, Alpert T, Muyombwe A, Razeq J, Downing R, Cheemarla NR, Wyllie AL, Kalinich CC, Ott IM, Quick J, Loman NJ, Neugebauer KM, Greninger AL, Jerome KR, Roychoudhury P, Xie H, Shrestha L, Huang M-L, Pitzer VE, Iwasaki A, Omer SB, Khan K, Bogoch II, Martinello RA, Foxman EF, Landry ML, Neher RA, Ko AI, Grubaugh ND. 2020. <a href="https://doi.org/10.1016/j.cell.2020.04.021">Coast-to-Coast Spread of SARS-CoV-2 during the Early Epidemic in the United States</a>. Cell 181:990–996.e5.</div>
</div>
<div id="ref-FVmbE2oW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">217. </div><div class="csl-right-inline">Gonzalez-Reiche AS, Hernandez MM, Sullivan MJ, Ciferri B, Alshammary H, Obla A, Fabre S, Kleiner G, Polanco J, Khan Z, Alburquerque B, van de Guchte A, Dutta J, Francoeur N, Melo BS, Oussenko I, Deikus G, Soto J, Sridhar SH, Wang Y-C, Twyman K, Kasarskis A, Altman DR, Smith M, Sebra R, Aberg J, Krammer F, García-Sastre A, Luksza M, Patel G, Paniz-Mondolfi A, Gitman M, Sordillo EM, Simon V, van Bakel H. 2020. <a href="https://doi.org/10.1126/science.abc1917">Introductions and early spread of SARS-CoV-2 in the New York City area</a>. Science eabc1917.</div>
</div>
<div id="ref-WJrnUyZf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">218. </div><div class="csl-right-inline">Gudbjartsson DF, Helgason A, Jonsson H, Magnusson OT, Melsted P, Norddahl GL, Saemundsdottir J, Sigurdsson A, Sulem P, Agustsdottir AB, Eiriksdottir B, Fridriksdottir R, Gardarsdottir EE, Georgsson G, Gretarsdottir OS, Gudmundsson KR, Gunnarsdottir TR, Gylfason A, Holm H, Jensson BO, Jonasdottir A, Jonsson F, Josefsdottir KS, Kristjansson T, Magnusdottir DN, le Roux L, Sigmundsdottir G, Sveinbjornsson G, Sveinsdottir KE, Sveinsdottir M, Thorarensen EA, Thorbjornsson B, Löve A, Masson G, Jonsdottir I, Möller AD, Gudnason T, Kristinsson KG, Thorsteinsdottir U, Stefansson K. 2020. <a href="https://doi.org/10.1056/nejmoa2006100">Spread of SARS-CoV-2 in the Icelandic Population</a>. New England Journal of Medicine 382:2302–2315.</div>
</div>
<div id="ref-qIXPib3m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">219. </div><div class="csl-right-inline">GISAID - Initiative. <a href="https://www.gisaid.org/">https://www.gisaid.org/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-cyzrC7qd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">220. </div><div class="csl-right-inline">NCBI SARS-CoV-2 Resources. <a href="https://www.ncbi.nlm.nih.gov/sars-cov-2/">https://www.ncbi.nlm.nih.gov/sars-cov-2/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-BjPoOYSA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">221. </div><div class="csl-right-inline">COVID-19 Data Portal - accelerating scientific research through data. <a href="https://www.covid19dataportal.org/">https://www.covid19dataportal.org/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-RIpPhJ1g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">222. </div><div class="csl-right-inline">Korber B, Fischer WM, Gnanakaran S, Yoon H, Theiler J, Abfalterer W, Hengartner N, Giorgi EE, Bhattacharya T, Foley B, Hastie KM, Parker MD, Partridge DG, Evans CM, Freeman TM, de Silva TI, McDanal C, Perez LG, Tang H, Moon-Walker A, Whelan SP, LaBranche CC, Saphire EO, Montefiori DC, Angyal A, Brown RL, Carrilero L, Green LR, Groves DC, Johnson KJ, Keeley AJ, Lindsey BB, Parsons PJ, Raza M, Rowland-Jones S, Smith N, Tucker RM, Wang D, Wyles MD. 2020. <a href="https://doi.org/10.1016/j.cell.2020.06.043">Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus</a>. Cell 182:812–827.e19.</div>
</div>
<div id="ref-MI5vSutb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">223. </div><div class="csl-right-inline">Yurkovetskiy L, Wang X, Pascal KE, Tomkins-Tinch C, Nyalile TP, Wang Y, Baum A, Diehl WE, Dauphin A, Carbone C, Veinotte K, Egri SB, Schaffner SF, Lemieux JE, Munro JB, Rafique A, Barve A, Sabeti PC, Kyratsous CA, Dudkina NV, Shen K, Luban J. 2020. <a href="https://doi.org/10.1016/j.cell.2020.09.032">Structural and Functional Analysis of the D614G SARS-CoV-2 Spike Protein Variant</a>. Cell 183:739–751.e8.</div>
</div>
<div id="ref-8LyA5uLY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">224. </div><div class="csl-right-inline">Plante JA, Liu Y, Liu J, Xia H, Johnson BA, Lokugamage KG, Zhang X, Muruato AE, Zou J, Fontes-Garfias CR, Mirchandani D, Scharton D, Bilello JP, Ku Z, An Z, Kalveram B, Freiberg AN, Menachery VD, Xie X, Plante KS, Weaver SC, Shi P-Y. 2020. <a href="https://doi.org/10.1038/s41586-020-2895-3">Spike mutation D614G alters SARS-CoV-2 fitness</a>. Nature 592:116–121.</div>
</div>
<div id="ref-owB3rKUG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">225. </div><div class="csl-right-inline">Thomson EC, Rosen LE, Shepherd JG, Spreafico R, da Silva Filipe A, Wojcechowskyj JA, Davis C, Piccoli L, Pascall DJ, Dillen J, Lytras S, Czudnochowski N, Shah R, Meury M, Jesudason N, De Marco A, Li K, Bassi J, O’Toole A, Pinto D, Colquhoun RM, Culap K, Jackson B, Zatta F, Rambaut A, Jaconi S, Sreenu VB, Nix J, Zhang I, Jarrett RF, Glass WG, Beltramello M, Nomikou K, Pizzuto M, Tong L, Cameroni E, Croll TI, Johnson N, Di Iulio J, Wickenhagen A, Ceschi A, Harbison AM, Mair D, Ferrari P, Smollett K, Sallusto F, Carmichael S, Garzoni C, Nichols J, Galli M, Hughes J, Riva A, Ho A, Schiuma M, Semple MG, Openshaw PJM, Fadda E, Baillie JK, Chodera JD, Rihn SJ, Lycett SJ, Virgin HW, Telenti A, Corti D, Robertson DL, Snell G. 2021. <a href="https://doi.org/10.1016/j.cell.2021.01.037">Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity</a>. Cell 184:1171–1187.e20.</div>
</div>
<div id="ref-10igTgCIt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">226. </div><div class="csl-right-inline">Young BE, Fong S-W, Chan Y-H, Mak T-M, Ang LW, Anderson DE, Lee CY-P, Amrun SN, Lee B, Goh YS, Su YCF, Wei WE, Kalimuddin S, Chai LYA, Pada S, Tan SY, Sun L, Parthasarathy P, Chen YYC, Barkham T, Lin RTP, Maurer-Stroh S, Leo Y-S, Wang L-F, Renia L, Lee VJ, Smith GJD, Lye DC, Ng LFP. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)31757-8">Effects of a major deletion in the SARS-CoV-2 genome on the severity of infection and the inflammatory response: an observational cohort study</a>. The Lancet 396:603–611.</div>
</div>
<div id="ref-wRtP6ZNU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">227. </div><div class="csl-right-inline">Liu Z, Zheng H, Lin H, Li M, Yuan R, Peng J, Xiong Q, Sun J, Li B, Wu J, Yi L, Peng X, Zhang H, Zhang W, Hulswit RJG, Loman N, Rambaut A, Ke C, Bowden TA, Pybus OG, Lu J. 2020. <a href="https://doi.org/10.1128/jvi.00790-20">Identification of Common Deletions in the Spike Protein of Severe Acute Respiratory Syndrome Coronavirus 2</a>. J Virol 94.</div>
</div>
<div id="ref-982hTnLI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">228. </div><div class="csl-right-inline">Tao K, Tzou PL, Nouhin J, Gupta RK, de Oliveira T, Kosakovsky Pond SL, Fera D, Shafer RW. 2021. <a href="https://doi.org/10.1038/s41576-021-00408-x">The biological and clinical significance of emerging SARS-CoV-2 variants</a>. Nat Rev Genet 22:757–773.</div>
</div>
<div id="ref-cmMwbH6K" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">229. </div><div class="csl-right-inline">Grabowski F, Preibisch G, Giziński S, Kochańczyk M, Lipniacki T. 2021. <a href="https://doi.org/10.3390/v13030392">SARS-CoV-2 Variant of Concern 202012/01 Has about Twofold Replicative Advantage and Acquires Concerning Mutations</a>. Viruses 13:392.</div>
</div>
<div id="ref-RhHGzsfD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">230. </div><div class="csl-right-inline">Kemp S, Harvey W, Datir R, Collier D, Ferreira I, Meng B, Carabelii A, Robertson DL, Gupta RK, COVID-19 Genomics UK (COG-UK) consortium. 2021. Recurrent emergence and transmission of a SARS-CoV-2 Spike deletion H69/V70. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.12.14.422555">10.1101/2020.12.14.422555</a>.</div>
</div>
<div id="ref-sqhvCTIL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">231. </div><div class="csl-right-inline">B.1.351 report. <a href="https://cov-lineages.org/global_report_B.1.351.html">https://cov-lineages.org/global_report_B.1.351.html</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-ZAXw65fn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">232. </div><div class="csl-right-inline">Ramanathan M, Ferguson ID, Miao W, Khavari PA. 2021. <a href="https://doi.org/10.1016/s1473-3099(21)00262-0">SARS-CoV-2 B.1.1.7 and B.1.351 spike variants bind human ACE2 with increased affinity</a>. The Lancet Infectious Diseases 21:1070.</div>
</div>
<div id="ref-x2Pm6BuB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">233. </div><div class="csl-right-inline">Spratt AN, Kannan SR, Woods LT, Weisman GA, Quinn TP, Lorson CL, Sönnerborg A, Byrareddy SN, Singh K. 2021. <a href="https://doi.org/10.1016/j.csbj.2021.06.037">Evolution, correlation, structural impact and dynamics of emerging SARS-CoV-2 variants</a>. Computational and Structural Biotechnology Journal 19:3799–3809.</div>
</div>
<div id="ref-tzOnKNrn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">234. </div><div class="csl-right-inline">Volz E, Mishra S, Chand M, Barrett JC, Johnson R, Geidelberg L, Hinsley WR, Laydon DJ, Dabrera G, O’Toole Á, Amato R, Ragonnet-Cronin M, Harrison I, Jackson B, Ariani CV, Boyd O, Loman NJ, McCrone JT, Gonçalves S, Jorgensen D, Myers R, Hill V, Jackson DK, Gaythorpe K, Groves N, Sillitoe J, Kwiatkowski DP, Flaxman S, Ratmann O, Bhatt S, Hopkins S, Gandy A, Rambaut A, Ferguson NM, The COVID-19 Genomics UK (COG-UK) consortium. 2021. Transmission of SARS-CoV-2 Lineage B.1.1.7 in England: Insights from linking epidemiological and genetic data. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.12.30.20249034">10.1101/2020.12.30.20249034</a>.</div>
</div>
<div id="ref-BEhidjDY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">235. </div><div class="csl-right-inline">Laffeber C, de Koning K, Kanaar R, Lebbink JHG. 2021. <a href="https://doi.org/10.1016/j.jmb.2021.167058">Experimental Evidence for Enhanced Receptor Binding by Rapidly Spreading SARS-CoV-2 Variants</a>. Journal of Molecular Biology 433:167058.</div>
</div>
<div id="ref-Ic7tOIax" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">236. </div><div class="csl-right-inline">Ali F, Kasry A, Amin M. 2021. <a href="https://doi.org/10.1016/j.medidd.2021.100086">The new SARS-CoV-2 strain shows a stronger binding affinity to ACE2 due to N501Y mutant</a>. Medicine in Drug Discovery 10:100086.</div>
</div>
<div id="ref-V61zrZZ7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">237. </div><div class="csl-right-inline">Hoffmann M, Arora P, Groß R, Seidel A, Hörnich BF, Hahn AS, Krüger N, Graichen L, Hofmann-Winkler H, Kempf A, Winkler MS, Schulz S, Jäck H-M, Jahrsdörfer B, Schrezenmeier H, Müller M, Kleger A, Münch J, Pöhlmann S. 2021. <a href="https://doi.org/10.1016/j.cell.2021.03.036">SARS-CoV-2 variants B.1.351 and P.1 escape from neutralizing antibodies</a>. Cell 184:2384–2393.e12.</div>
</div>
<div id="ref-pcm6tmYA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">238. </div><div class="csl-right-inline">Tracking SARS-CoV-2 variants. <a href="https://www.who.int/activities/tracking-SARS-CoV-2-variants">https://www.who.int/activities/tracking-SARS-CoV-2-variants</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-LNApbgCL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">239. </div><div class="csl-right-inline">Saito A, Nasser H, Uriu K, Kosugi Y, Irie T, Shirakawa K, Sadamasu K, Kimura I, Ito J, Wu J, Ozono S, Tokunaga K, Butlertanaka EP, Tanaka YL, Shimizu R, Shimizu K, Fukuhara T, Kawabata R, Sakaguchi T, Yoshida I, Asakura H, Nagashima M, Yoshimura K, Kazuma Y, Nomura R, Horisawa Y, Takaori-Kondo A, Nakagawa S, Ikeda T, Sato K. 2021. <a href="https://doi.org/10.1101/2021.06.17.448820">SARS-CoV-2 spike P681R mutation enhances and accelerates viral fusion</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-1HaaPNEXD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">240. </div><div class="csl-right-inline">Singanayagam A, Patel M, Charlett A, Lopez Bernal J, Saliba V, Ellis J, Ladhani S, Zambon M, Gopal R. 2020. <a href="https://doi.org/10.2807/1560-7917.es.2020.25.32.2001483">Duration of infectiousness and correlation with RT-PCR cycle threshold values in cases of COVID-19, England, January to May 2020</a>. Eurosurveillance 25.</div>
</div>
<div id="ref-KMb6knfE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">241. </div><div class="csl-right-inline">La Rosa G, Fratini M, Della Libera S, Iaconelli M, Muscillo M. 2013. <a href="https://doi.org/10.4415/ann_13_02_03">Viral infections acquired indoors through airborne, droplet or contact transmission.</a> Ann Ist Super Sanita 49:124–32.</div>
</div>
<div id="ref-CJbohYmK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">242. </div><div class="csl-right-inline">Shiu EYC, Leung NHL, Cowling BJ. 2019. <a href="https://doi.org/10.1097/qco.0000000000000563">Controversy around airborne versus droplet transmission of respiratory viruses</a>. Current Opinion in Infectious Diseases 32:372–379.</div>
</div>
<div id="ref-MZlr12J3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">243. </div><div class="csl-right-inline">Tang JW, Bahnfleth WP, Bluyssen PM, Buonanno G, Jimenez JL, Kurnitski J, Li Y, Miller S, Sekhar C, Morawska L, Marr LC, Melikov AK, Nazaroff WW, Nielsen PV, Tellier R, Wargocki P, Dancer SJ. 2021. <a href="https://doi.org/10.1016/j.jhin.2020.12.022">Dismantling myths on the airborne transmission of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2)</a>. Journal of Hospital Infection 110:89–96.</div>
</div>
<div id="ref-CvYwVG8O" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">244. </div><div class="csl-right-inline">Randall K, Ewing ET, Marr L, Jimenez J, Bourouiba L. 2021. <a href="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3829873">How Did We Get Here: What Are Droplets and Aerosols and How Far Do They Go? A Historical Perspective on the Transmission of Respiratory Infectious Diseases</a>. SSRN.</div>
</div>
<div id="ref-1FLLN4PSl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">245. </div><div class="csl-right-inline">Transmission of SARS-CoV-2: implications for infection prevention precautions. <a href="https://www.who.int/news-room/commentaries/detail/transmission-of-sars-cov-2-implications-for-infection-prevention-precautions">https://www.who.int/news-room/commentaries/detail/transmission-of-sars-cov-2-implications-for-infection-prevention-precautions</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-AC4okoVf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">246. </div><div class="csl-right-inline">Lemieux C, Brankston G, Gitterman L, Hirji Z, Gardam M. 2007. <a href="https://doi.org/10.3201/eid1301.061202">Questioning Aerosol Transmission of Influenza</a>. Emerging Infectious Diseases 13:173–175.</div>
</div>
<div id="ref-Hct9jRcb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">247. </div><div class="csl-right-inline">Smieszek T, Lazzari G, Salathé M. 2019. <a href="https://doi.org/10.1038/s41598-019-38825-y">Assessing the Dynamics and Control of Droplet- and Aerosol-Transmitted Influenza Using an Indoor Positioning System</a>. Scientific Reports 9:2185.</div>
</div>
<div id="ref-i6tTpqwA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">248. </div><div class="csl-right-inline">Richard M, Fouchier RAM. 2016. <a href="https://doi.org/10.1093/femsre/fuv039">Influenza A virus transmission via respiratory aerosols or droplets as it relates to pandemic potential</a>. FEMS Microbiology Reviews 40:68–85.</div>
</div>
<div id="ref-rjVw7V94" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">249. </div><div class="csl-right-inline">Weiss SR, Leibowitz JL. 2011. <a href="https://doi.org/10.1016/b978-0-12-385885-6.00009-2">Coronavirus Pathogenesis</a>Advances in Virus Research. Elsevier BV.</div>
</div>
<div id="ref-G5NJrE75" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">250. </div><div class="csl-right-inline">de Wit E, van Doremalen N, Falzarano D, Munster VJ. 2016. <a href="https://doi.org/10.1038/nrmicro.2016.81">SARS and MERS: recent insights into emerging coronaviruses</a>. Nature Reviews Microbiology 14:523–534.</div>
</div>
<div id="ref-1D0sSd6ZT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">251. </div><div class="csl-right-inline">Li Y, Huang X, Yu ITS, Wong TW, Qian H. 2005. <a href="https://doi.org/10.1111/j.1600-0668.2004.00317.x">Role of air distribution in SARS transmission during the largest nosocomial outbreak in Hong Kong</a>. Indoor Air 15:83–95.</div>
</div>
<div id="ref-zmcRFgYd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">252. </div><div class="csl-right-inline">Booth TF, Kournikakis B, Bastien N, Ho J, Kobasa D, Stadnyk L, Li Y, Spence M, Paton S, Henry B, Mederski B, White D, Low DE, McGeer A, Simor A, Vearncombe M, Downey J, Jamieson FB, Tang P, Plummer F. 2005. <a href="https://doi.org/10.1086/429634">Detection of Airborne Severe Acute Respiratory Syndrome (SARS) Coronavirus and Environmental Contamination in SARS Outbreak Units</a>. The Journal of Infectious Diseases 191:1472–1477.</div>
</div>
<div id="ref-yR57NFIB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">253. </div><div class="csl-right-inline">Xiao S, Li Y, Wong T, Hui DSC. 2017. <a href="https://doi.org/10.1371/journal.pone.0181558">Role of fomites in SARS transmission during the largest hospital outbreak in Hong Kong</a>. PLOS ONE 12:e0181558.</div>
</div>
<div id="ref-s3oVNbGV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">254. </div><div class="csl-right-inline">van Doremalen N, Bushmaker T, Munster VJ. 2013. <a href="https://doi.org/10.2807/1560-7917.es2013.18.38.20590">Stability of Middle East respiratory syndrome coronavirus (MERS-CoV) under different environmental conditions</a>. Eurosurveillance 18.</div>
</div>
<div id="ref-1FjDLPCye" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">255. </div><div class="csl-right-inline">Mackay IM, Arden KE. 2015. <a href="https://doi.org/10.1186/s12985-015-0439-5">MERS coronavirus: diagnostics, epidemiology and transmission</a>. Virology Journal 12:222.</div>
</div>
<div id="ref-yXKcviw8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">256. </div><div class="csl-right-inline">van Doremalen N, Bushmaker T, Morris DH, Holbrook MG, Gamble A, Williamson BN, Tamin A, Harcourt JL, Thornburg NJ, Gerber SI, Lloyd-Smith JO, de Wit E, Munster VJ. 2020. <a href="https://doi.org/10.1056/nejmc2004973">Aerosol and Surface Stability of SARS-CoV-2 as Compared with SARS-CoV-1</a>. New England Journal of Medicine 382:1564–1567.</div>
</div>
<div id="ref-vP6yTZ0y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">257. </div><div class="csl-right-inline">Peng X, Xu X, Li Y, Cheng L, Zhou X, Ren B. 2020. <a href="https://doi.org/10.1038/s41368-020-0075-9">Transmission routes of 2019-nCoV and controls in dental practice</a>. International Journal of Oral Science 12:9.</div>
</div>
<div id="ref-82XnTbtX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">258. </div><div class="csl-right-inline">Klompas M, Baker MA, Rhee C. 2020. <a href="https://doi.org/10.1001/jama.2020.12458">Airborne Transmission of SARS-CoV-2</a>. JAMA 324:441.</div>
</div>
<div id="ref-cw5j7x80" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">259. </div><div class="csl-right-inline">Goldman E. 2020. <a href="https://doi.org/10.1016/s1473-3099(20)30561-2">Exaggerated risk of transmission of COVID-19 by fomites</a>. The Lancet Infectious Diseases 20:892–893.</div>
</div>
<div id="ref-16SDeiudC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">260. </div><div class="csl-right-inline">Prather KA, Wang CC, Schooley RT. 2020. <a href="https://doi.org/10.1126/science.abc6197">Reducing transmission of SARS-CoV-2</a>. Science 368:1422–1424.</div>
</div>
<div id="ref-9ouDX5IN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">261. </div><div class="csl-right-inline">Morawska L, Milton DK. 2020. <a href="https://doi.org/10.1093/cid/ciaa939">It Is Time to Address Airborne Transmission of Coronavirus Disease 2019 (COVID-19)</a>. Clinical Infectious Diseases ciaa939.</div>
</div>
<div id="ref-sUAAY1me" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">262. </div><div class="csl-right-inline">Greenhalgh T, Jimenez JL, Prather KA, Tufekci Z, Fisman D, Schooley R. 2021. <a href="https://doi.org/10.1016/s0140-6736(21)00869-2">Ten scientific reasons in support of airborne transmission of SARS-CoV-2</a>. The Lancet 397:1603–1605.</div>
</div>
<div id="ref-bQLMgMGC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">263. </div><div class="csl-right-inline">Liu Y, Ning Z, Chen Y, Guo M, Liu Y, Gali NK, Sun L, Duan Y, Cai J, Westerdahl D, Liu X, Xu K, Ho K, Kan H, Fu Q, Lan K. 2020. <a href="https://doi.org/10.1038/s41586-020-2271-3">Aerodynamic analysis of SARS-CoV-2 in two Wuhan hospitals</a>. Nature 582:557–560.</div>
</div>
<div id="ref-Uzfdlcgn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">264. </div><div class="csl-right-inline">Lednicky JA, Lauzardo M, Fan ZH, Jutla A, Tilly TB, Gangwar M, Usmani M, Shankar SN, Mohamed K, Eiguren-Fernandez A, Stephenson CJ, Alam MdM, Elbadry MA, Loeb JC, Subramaniam K, Waltzek TB, Cherabuddi K, Morris JG Jr., Wu C-Y. 2020. <a href="https://doi.org/10.1016/j.ijid.2020.09.025">Viable SARS-CoV-2 in the air of a hospital room with COVID-19 patients</a>. International Journal of Infectious Diseases 100:476–482.</div>
</div>
<div id="ref-lPbg0LYo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">265. </div><div class="csl-right-inline">Santarpia JL, Herrera VL, Rivera DN, Ratnesar-Shumate S, Reid StP, Denton PW, Martens JWS, Fang Y, Conoan N, Callahan MV, Lawler JV, Brett-Major DM, Lowe JJ. 2020. <a href="https://doi.org/10.1101/2020.07.13.20041632">The Infectious Nature of Patient-Generated SARS-CoV-2 Aerosol</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-1BWoTPei7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">266. </div><div class="csl-right-inline">Santarpia JL, Rivera DN, Herrera VL, Morwitzer MJ, Creager HM, Santarpia GW, Crown KK, Brett-Major DM, Schnaubelt ER, Broadhurst MJ, Lawler JV, Reid StP, Lowe JJ. 2020. <a href="https://doi.org/10.1038/s41598-020-69286-3">Aerosol and surface contamination of SARS-CoV-2 observed in quarantine and isolation care</a>. Sci Rep 10.</div>
</div>
<div id="ref-X1nA82rS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">267. </div><div class="csl-right-inline">Zhou J, Otter JA, Price JR, Cimpeanu C, Meno Garcia D, Kinross J, Boshier PR, Mason S, Bolt F, Holmes AH, Barclay WS. 2020. <a href="https://doi.org/10.1093/cid/ciaa905">Investigating Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Surface and Air Contamination in an Acute Healthcare Setting During the Peak of the Coronavirus Disease 2019 (COVID-19) Pandemic in London</a>. Clinical Infectious Diseases 73:e1870–e1877.</div>
</div>
<div id="ref-184VaR2ib" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">268. </div><div class="csl-right-inline">Comber L, O Murchu E, Drummond L, Carty PG, Walsh KA, De Gascun CF, Connolly MA, Smith SM, O'Neill M, Ryan M, Harrington P. 2020. <a href="https://doi.org/10.1002/rmv.2184">Airborne transmission of SARS‐CoV‐2 via aerosols</a>. Rev Med Virol 31.</div>
</div>
<div id="ref-1Gqr7yJGi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">269. </div><div class="csl-right-inline">Bourouiba L. 2020. Turbulent Gas Clouds and Respiratory Pathogen Emissions. JAMA https://doi.org/<a href="https://doi.org/10.1001/jama.2020.4756">10.1001/jama.2020.4756</a>.</div>
</div>
<div id="ref-wxVni9Hz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">270. </div><div class="csl-right-inline">Hu Z, Song C, Xu C, Jin G, Chen Y, Xu X, Ma H, Chen W, Lin Y, Zheng Y, Wang J, Hu Z, Yi Y, Shen H. 2020. <a href="https://doi.org/10.1007/s11427-020-1661-4">Clinical characteristics of 24 asymptomatic infections with COVID-19 screened among close contacts in Nanjing, China</a>. Science China Life Sciences 63:706–711.</div>
</div>
<div id="ref-NIxttl2v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">271. </div><div class="csl-right-inline">Tindale LC, Stockdale JE, Coombe M, Garlock ES, Lau WYV, Saraswat M, Zhang L, Chen D, Wallinga J, Colijn C. 2020. <a href="https://doi.org/10.7554/elife.57149">Evidence for transmission of COVID-19 prior to symptom onset</a>. eLife 9:e57149.</div>
</div>
<div id="ref-QFVSrboR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">272. </div><div class="csl-right-inline">Chang D, Mo G, Yuan X, Tao Y, Peng X, Wang F-S, Xie L, Sharma L, Dela Cruz CS, Qin E. 2020. <a href="https://doi.org/10.1164/rccm.202003-0524le">Time Kinetics of Viral Clearance and Resolution of Symptoms in Novel Coronavirus Infection</a>. American Journal of Respiratory and Critical Care Medicine 201:1150–1152.</div>
</div>
<div id="ref-1654TaAK4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">273. </div><div class="csl-right-inline">He X, Lau EHY, Wu P, Deng X, Wang J, Hao X, Lau YC, Wong JY, Guan Y, Tan X, Mo X, Chen Y, Liao B, Chen W, Hu F, Zhang Q, Zhong M, Wu Y, Zhao L, Zhang F, Cowling BJ, Li F, Leung GM. 2020. <a href="https://doi.org/10.1038/s41591-020-0869-5">Temporal dynamics in viral shedding and transmissibility of COVID-19</a>. Nature Medicine 26:672–675.</div>
</div>
<div id="ref-Oyy1OUTT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">274. </div><div class="csl-right-inline">CDC. 2020. COVID-19 and Your Health. Centers for Disease Control and Prevention. <a href="https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html">https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-Xp5HT2SF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">275. </div><div class="csl-right-inline">Wölfel R, Corman VM, Guggemos W, Seilmaier M, Zange S, Müller MA, Niemeyer D, Jones TC, Vollmar P, Rothe C, Hoelscher M, Bleicker T, Brünink S, Schneider J, Ehmann R, Zwirglmaier K, Drosten C, Wendtner C. 2020. <a href="https://doi.org/10.1038/s41586-020-2196-x">Virological assessment of hospitalized patients with COVID-2019</a>. Nature 581:465–469.</div>
</div>
<div id="ref-14fWuiUKS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">276. </div><div class="csl-right-inline">Corsini Campioli C, Cano Cevallos E, Assi M, Patel R, Binnicker MJ, O’Horo JC. 2020. <a href="https://doi.org/10.1016/j.jcv.2020.104577">Clinical predictors and timing of cessation of viral RNA shedding in patients with COVID-19</a>. Journal of Clinical Virology 130:104577.</div>
</div>
<div id="ref-ac2kt3rh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">277. </div><div class="csl-right-inline">Arons MM, Hatfield KM, Reddy SC, Kimball A, James A, Jacobs JR, Taylor J, Spicer K, Bardossy AC, Oakley LP, Tanwar S, Dyal JW, Harney J, Chisty Z, Bell JM, Methner M, Paul P, Carlson CM, McLaughlin HP, Thornburg N, Tong S, Tamin A, Tao Y, Uehara A, Harcourt J, Clark S, Brostrom-Smith C, Page LC, Kay M, Lewis J, Montgomery P, Stone ND, Clark TA, Honein MA, Duchin JS, Jernigan JA. 2020. <a href="https://doi.org/10.1056/nejmoa2008457">Presymptomatic SARS-CoV-2 Infections and Transmission in a Skilled Nursing Facility</a>. New England Journal of Medicine 382:2081–2090.</div>
</div>
<div id="ref-lVDiHYp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">278. </div><div class="csl-right-inline">Baggett TP, Keyes H, Sporn N, Gaeta JM. 2020. <a href="https://doi.org/10.1001/jama.2020.6887">Prevalence of SARS-CoV-2 Infection in Residents of a Large Homeless Shelter in Boston</a>. JAMA 323:2191.</div>
</div>
<div id="ref-E6Qv6YQ0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">279. </div><div class="csl-right-inline">Bai Y, Yao L, Wei T, Tian F, Jin D-Y, Chen L, Wang M. 2020. <a href="https://doi.org/10.1001/jama.2020.2565">Presumed Asymptomatic Carrier Transmission of COVID-19</a>. JAMA 323:1406.</div>
</div>
<div id="ref-18Orbxz8t" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">280. </div><div class="csl-right-inline">Li P, Fu J-B, Li K-F, Liu J-N, Wang H-L, Liu L-J, Chen Y, Zhang Y-L, Liu S-L, Tang A, Tong Z-D, Yan J-B. 2020. <a href="https://doi.org/10.1016/j.ijid.2020.03.027">Transmission of COVID-19 in the terminal stages of the incubation period: A familial cluster</a>. International Journal of Infectious Diseases 96:452–453.</div>
</div>
<div id="ref-4M3GM1sg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">281. </div><div class="csl-right-inline">Zhang P, Tian F, Wan Y, Cai J, Qian Z, Wu R, Zhang Y, Zhang S, Li H, Li M, Trevathan E, Lin H. 2020. A Cohort of SARS-CoV-2 Infected Asymptomatic and Pre-Symptomatic Contacts from COVID-19 Contact Tracing in Hubei Province, China: Short-Term Outcomes. SSRN Electronic Journal https://doi.org/<a href="https://doi.org/10.2139/ssrn.3678556">10.2139/ssrn.3678556</a>.</div>
</div>
<div id="ref-6K5lsF5i" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">282. </div><div class="csl-right-inline">Mizumoto K, Kagaya K, Zarebski A, Chowell G. 2020. <a href="https://doi.org/10.2807/1560-7917.es.2020.25.10.2000180">Estimating the asymptomatic proportion of coronavirus disease 2019 (COVID-19) cases on board the Diamond Princess cruise ship, Yokohama, Japan, 2020</a>. Eurosurveillance 25.</div>
</div>
<div id="ref-sk1NbA7K" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">283. </div><div class="csl-right-inline">Zhang K, Tong W, Wang X, Lau JY-N. 2020. <a href="https://doi.org/10.1093/pcmedi/pbaa032">Estimated prevalence and viral transmissibility in subjects with asymptomatic SARS-CoV-2 infections in Wuhan, China</a>. Precision Clinical Medicine 3:301–305.</div>
</div>
<div id="ref-kIumgXPI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">284. </div><div class="csl-right-inline">Ladhani SN, Chow JY, Janarthanan R, Fok J, Crawley-Boevey E, Vusirikala A, Fernandez E, Perez MS, Tang S, Dun-Campbell K, Evans EW-, Bell A, Patel B, Amin-Chowdhury Z, Aiano F, Paranthaman K, Ma T, Saavedra-Campos M, Myers R, Ellis J, Lackenby A, Gopal R, Patel M, Brown C, Chand M, Brown K, Ramsay ME, Hopkins S, Shetty N, Zambon M. 2020. <a href="https://doi.org/10.1016/j.eclinm.2020.100533">Investigation of SARS-CoV-2 outbreaks in six care homes in London, April 2020</a>. EClinicalMedicine 26:100533.</div>
</div>
<div id="ref-10OGkFiGJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">285. </div><div class="csl-right-inline">Long Q-X, Tang X-J, Shi Q-L, Li Q, Deng H-J, Yuan J, Hu J-L, Xu W, Zhang Y, Lv F-J, Su K, Zhang F, Gong J, Wu B, Liu X-M, Li J-J, Qiu J-F, Chen J, Huang A-L. 2020. <a href="https://doi.org/10.1038/s41591-020-0965-6">Clinical and immunological assessment of asymptomatic SARS-CoV-2 infections</a>. Nature Medicine 26:1200–1204.</div>
</div>
<div id="ref-dMjSbAQV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">286. </div><div class="csl-right-inline">Lavezzo E, Franchin E, Ciavarella C, Cuomo-Dannenburg G, Barzon L, Del Vecchio C, Rossi L, Manganelli R, Loregian A, Navarin N, Abate D, Sciro M, Merigliano S, De Canale E, Vanuzzo MC, Besutti V, Saluzzo F, Onelia F, Pacenti M, Parisi SG, Carretta G, Donato D, Flor L, Cocchio S, Masi G, Sperduti A, Cattarino L, Salvador R, Nicoletti M, Caldart F, Castelli G, Nieddu E, Labella B, Fava L, Drigo M, Gaythorpe KAM, Brazzale AR, Toppo S, Trevisan M, Baldo V, Donnelly CA, Ferguson NM, Dorigatti I, Crisanti A, Imperial College COVID-19 Response Team, Imperial College COVID-19 Response Team. 2020. <a href="https://doi.org/10.1038/s41586-020-2488-1">Suppression of a SARS-CoV-2 outbreak in the Italian municipality of Vo’</a>. Nature 584:425–429.</div>
</div>
<div id="ref-dHGy34wC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">287. </div><div class="csl-right-inline">Meyerowitz-Katz G, Merone L. 2020. <a href="https://doi.org/10.1016/j.ijid.2020.09.1464">A systematic review and meta-analysis of published research data on COVID-19 infection fatality rates</a>. International Journal of Infectious Diseases 101:138–148.</div>
</div>
<div id="ref-AavOV1He" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">288. </div><div class="csl-right-inline">Global Covid-19 Case Fatality Rates. The Centre for Evidence-Based Medicine. <a href="https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/">https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-12CiweatZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">289. </div><div class="csl-right-inline">Grewelle R, De Leo G. 2020. Estimating the Global Infection Fatality Rate of COVID-19. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.05.11.20098780">10.1101/2020.05.11.20098780</a>.</div>
</div>
<div id="ref-tIjAygts" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">290. </div><div class="csl-right-inline">Stadlbauer D, Tan J, Jiang K, Hernandez MM, Fabre S, Amanat F, Teo C, Arunkumar GA, McMahon M, Capuano C, Twyman K, Jhang J, Nowak MD, Simon V, Sordillo EM, van Bakel H, Krammer F. 2020. <a href="https://doi.org/10.1038/s41586-020-2912-6">Repeated cross-sectional sero-monitoring of SARS-CoV-2 in New York City</a>. Nature 590:146–150.</div>
</div>
<div id="ref-SOdviufj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">291. </div><div class="csl-right-inline">What do we know about the risk of dying from COVID-19? Our World in Data. <a href="https://ourworldindata.org/covid-mortality-risk">https://ourworldindata.org/covid-mortality-risk</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-16CD4voW1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">292. </div><div class="csl-right-inline">Heesterbeek JAP, Dietz K. 1996. <a href="https://doi.org/10.1111/j.1467-9574.1996.tb01482.x">The concept of R &lt;sub&gt;o&lt;/sub&gt; in epidemic theory</a>. Statistica Neerlandica 50:89–110.</div>
</div>
<div id="ref-14l4fvU71" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">293. </div><div class="csl-right-inline">Keeling MJ, Rohani P. 2008. Modeling infectious diseases in humans and animals. Princeton University Press, Princeton.</div>
</div>
<div id="ref-GMDFEjtK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">294. </div><div class="csl-right-inline">1997. <a href="https://doi.org/10.1098/rspa.1927.0118">A contribution to the mathematical theory of epidemics</a>. Proceedings of the Royal Society of London Series A, Containing Papers of a Mathematical and Physical Character 115:700–721.</div>
</div>
<div id="ref-SPInPew0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">295. </div><div class="csl-right-inline">Anderson RM, May RM. 1979. <a href="https://doi.org/10.1038/280361a0">Population biology of infectious diseases: Part I</a>. Nature 280:361–367.</div>
</div>
<div id="ref-YubbhU7G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">296. </div><div class="csl-right-inline">Cobey S. 2020. <a href="https://doi.org/10.1126/science.abb5659">Modeling infectious disease dynamics</a>. Science 368:713–714.</div>
</div>
<div id="ref-JRaVgcNd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">297. </div><div class="csl-right-inline">2007. Theoretical ecology: principles and applications. Oxford University Press, Oxford ; New York.</div>
</div>
<div id="ref-cWv3gKK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">298. </div><div class="csl-right-inline">Heng K, Althaus CL. 2020. <a href="https://doi.org/10.1038/s41598-020-76563-8">The approximately universal shapes of epidemic curves in the Susceptible-Exposed-Infectious-Recovered (SEIR) model</a>. Sci Rep 10.</div>
</div>
<div id="ref-1E0r4uZy9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">299. </div><div class="csl-right-inline">Wu JT, Leung K, Leung GM. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30260-9">Nowcasting and forecasting the potential domestic and international spread of the 2019-nCoV outbreak originating in Wuhan, China: a modelling study</a>. The Lancet 395:689–697.</div>
</div>
<div id="ref-LHtuVmaq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">300. </div><div class="csl-right-inline">Liu Y, Gayle AA, Wilder-Smith A, Rocklöv J. 2020. <a href="https://doi.org/10.1093/jtm/taaa021">The reproductive number of COVID-19 is higher compared to SARS coronavirus</a>. Journal of Travel Medicine 27:taaa021.</div>
</div>
<div id="ref-mHYmt0mv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">301. </div><div class="csl-right-inline">Li R, Pei S, Chen B, Song Y, Zhang T, Yang W, Shaman J. 2020. <a href="https://doi.org/10.1126/science.abb3221">Substantial undocumented infection facilitates the rapid dissemination of novel coronavirus (SARS-CoV-2)</a>. Science 368:489–493.</div>
</div>
<div id="ref-vITui6ac" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">302. </div><div class="csl-right-inline">Ma S, Zhang J, Zeng M, Yun Q, Guo W, Zheng Y, Zhao S, Wang MH, Yang Z. 2020. Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.21.20040329">10.1101/2020.03.21.20040329</a>.</div>
</div>
<div id="ref-nxM0rP5R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">303. </div><div class="csl-right-inline">Majumder M, Mandl KD. 2020. Early Transmissibility Assessment of a Novel Coronavirus in Wuhan, China. SSRN Electronic Journal https://doi.org/<a href="https://doi.org/10.2139/ssrn.3524675">10.2139/ssrn.3524675</a>.</div>
</div>
<div id="ref-WLc2UMgQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">304. </div><div class="csl-right-inline">Liu T, Hu J, Xiao J, He G, Kang M, Rong Z, Lin L, Zhong H, Huang Q, Deng A, Zeng W, Tan X, Zeng S, Zhu Z, Li J, Gong D, Wan D, Chen S, Guo L, Li Y, Sun L, Liang W, Song T, He J, Ma W. 2020. Time-varying transmission dynamics of Novel Coronavirus Pneumonia in China. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.01.25.919787">10.1101/2020.01.25.919787</a>.</div>
</div>
<div id="ref-10bBqMHH7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">305. </div><div class="csl-right-inline">Zhang S, Diao M, Yu W, Pei L, Lin Z, Chen D. 2020. <a href="https://doi.org/10.1016/j.ijid.2020.02.033">Estimation of the reproductive number of novel coronavirus (COVID-19) and the probable outbreak size on the Diamond Princess cruise ship: A data-driven analysis</a>. International Journal of Infectious Diseases 93:201–204.</div>
</div>
<div id="ref-ITh0Anof" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">306. </div><div class="csl-right-inline">Tang B, Wang X, Li Q, Bragazzi NL, Tang S, Xiao Y, Wu J. 2020. <a href="https://doi.org/10.3390/jcm9020462">Estimation of the Transmission Risk of the 2019-nCoV and Its Implication for Public Health Interventions</a>. Journal of Clinical Medicine 9:462.</div>
</div>
<div id="ref-1BMU7sKbs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">307. </div><div class="csl-right-inline">Cao Z, Zhang Q, Lu X, Pfeiffer D, Jia Z, Song H, Zeng DD. 2020. <a href="https://doi.org/10.1101/2020.01.27.20018952">Estimating the effective reproduction number of the 2019-nCoV in China</a>. medRxiv 2020.01.27.20018952.</div>
</div>
<div id="ref-Yj8Xh4Wz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">308. </div><div class="csl-right-inline">Shen M, Peng Z, Xiao Y, Zhang L. 2020. Modelling the epidemic trend of the 2019 novel coronavirus outbreak in China. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.01.23.916726">10.1101/2020.01.23.916726</a>.</div>
</div>
<div id="ref-ZzrrVDoE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">309. </div><div class="csl-right-inline">Read JM, Bridgen JRE, Cummings DAT, Ho A, Jewell CP. 2020. Novel coronavirus 2019-nCoV: early estimation of epidemiological parameters and epidemic predictions. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.01.23.20018549">10.1101/2020.01.23.20018549</a>.</div>
</div>
<div id="ref-itj26agd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">310. </div><div class="csl-right-inline">Roques L, Klein E, Papaïx J, Sar A, Soubeyrand S. 2020. Using early data to estimate the actual infection fatality ratio from COVID-19 in France. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.22.20040915">10.1101/2020.03.22.20040915</a>.</div>
</div>
<div id="ref-1H6clx00o" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">311. </div><div class="csl-right-inline">Park SW, Sun K, Viboud C, Grenfell BT, Dushoff J. 2020. <a href="https://doi.org/10.3201/eid2611.201099">Potential Role of Social Distancing in Mitigating Spread of Coronavirus Disease, South Korea</a>. Emerg Infect Dis 26:2697–2700.</div>
</div>
<div id="ref-tiRfUgvs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">312. </div><div class="csl-right-inline">Kucharski AJ, Russell TW, Diamond C, Liu Y, Edmunds J, Funk S, Eggo RM, Sun F, Jit M, Munday JD, Davies N, Gimma A, van Zandvoort K, Gibbs H, Hellewell J, Jarvis CI, Clifford S, Quilty BJ, Bosse NI, Abbott S, Klepac P, Flasche S. 2020. <a href="https://doi.org/10.1016/s1473-3099(20)30144-4">Early dynamics of transmission and control of COVID-19: a mathematical modelling study</a>. The Lancet Infectious Diseases 20:553–558.</div>
</div>
<div id="ref-TH4ymqvJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">313. </div><div class="csl-right-inline">Sahafizadeh E, Sartoli S. 2020. Estimating the reproduction number of COVID-19 in Iran using epidemic modeling. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.20.20038422">10.1101/2020.03.20.20038422</a>.</div>
</div>
<div id="ref-Frya4XA4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">314. </div><div class="csl-right-inline">Flaxman S, Mishra S, Gandy A, Unwin H, Coupland H, Mellan T, Zhu H, Berah T, Eaton J, Perez Guzman P, Schmit N, Cilloni L, Ainslie K, Baguelin M, Blake I, Boonyasiri A, Boyd O, Cattarino L, Ciavarella C, Cooper L, Cucunuba Perez Z, Cuomo-Dannenburg G, Dighe A, Djaafara A, Dorigatti I, Van Elsland S, Fitzjohn R, Fu H, Gaythorpe K, Geidelberg L, Grassly N, Green W, Hallett T, Hamlet A, Hinsley W, Jeffrey B, Jorgensen D, Knock E, Laydon D, Nedjati Gilani G, Nouvellet P, Parag K, Siveroni I, Thompson H, Verity R, Volz E, Walters C, Wang H, Wang Y, Watson O, Winskill P, Xi X, Whittaker C, Walker P, Ghani A, Donnelly C, Riley S, Okell L, Vollmer M, Ferguson N, Bhatt S. 2020. <a href="https://doi.org/10.25561/77731">Report 13: Estimating the number of infections and the impact of non-pharmaceutical interventions on COVID-19 in 11 European countries</a>. Imperial College London.</div>
</div>
<div id="ref-AfaxKtHk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">315. </div><div class="csl-right-inline">Campbell F, Archer B, Laurenson-Schafer H, Jinnai Y, Konings F, Batra N, Pavlin B, Vandemaele K, Van Kerkhove MD, Jombart T, Morgan O, le Polain de Waroux O. 2021. <a href="https://doi.org/10.2807/1560-7917.es.2021.26.24.2100509">Increased transmissibility and global spread of SARS-CoV-2 variants of concern as at June 2021</a>. Eurosurveillance 26.</div>
</div>
<div id="ref-JsfUU71J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">316. </div><div class="csl-right-inline">Washington NL, Gangavarapu K, Zeller M, Bolze A, Cirulli ET, Barrett KMS, Larsen BB, Anderson C, White S, Cassens T, Jacobs S, Levan G, Nguyen J, Ramirez JM III, Rivera-Garcia C, Sandoval E, Wang X, Wong D, Spencer E, Robles-Sikisaka R, Kurzban E, Hughes LD, Deng X, Wang C, Servellita V, Valentine H, De Hoff P, Seaver P, Sathe S, Gietzen K, Sickler B, Antico J, Hoon K, Liu J, Harding A, Bakhtar O, Basler T, Austin B, Isaksson M, Febbo PG, Becker D, Laurent M, McDonald E, Yeo GW, Knight R, Laurent LC, de Feo E, Worobey M, Chiu C, Suchard MA, Lu JT, Lee W, Andersen KG. 2021. <a href="https://doi.org/10.1101/2021.02.06.21251159">Genomic epidemiology identifies emergence and rapid transmission of SARS-CoV-2 B.1.1.7 in the United States</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-9L4bQ6n7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">317. </div><div class="csl-right-inline">Davies NG, Abbott S, Barnard RC, Jarvis CI, Kucharski AJ, Munday JD, Pearson CAB, Russell TW, Tully DC, Washburne AD, Wenseleers T, Gimma A, Waites W, Wong KLM, van Zandvoort K, Silverman JD, Diaz-Ordaz K, Keogh R, Eggo RM, Funk S, Jit M, Atkins KE, Edmunds WJ. 2021. <a href="https://doi.org/10.1126/science.abg3055">Estimated transmissibility and impact of SARS-CoV-2 lineage B.1.1.7 in England</a>. Science 372.</div>
</div>
<div id="ref-1EI5i5dA1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">318. </div><div class="csl-right-inline">Liu Y, Rocklöv J. 2021. <a href="https://doi.org/10.1093/jtm/taab124">The reproductive number of the Delta variant of SARS-CoV-2 is far higher compared to the ancestral SARS-CoV-2 virus</a>. Journal of Travel Medicine 28.</div>
</div>
<div id="ref-r8vdQFkf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">319. </div><div class="csl-right-inline">Ito K, Piantham C, Nishiura H. 2021. <a href="https://doi.org/10.2807/1560-7917.es.2021.26.27.2100570">Predicted dominance of variant Delta of SARS-CoV-2 before Tokyo Olympic Games, Japan, July 2021</a>. Eurosurveillance 26.</div>
</div>
<div id="ref-NCcssjLV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">320. </div><div class="csl-right-inline">Moghadas SM, Shoukat A, Fitzpatrick MC, Wells CR, Sah P, Pandey A, Sachs JD, Wang Z, Meyers LA, Singer BH, Galvani AP. 2020. <a href="https://doi.org/10.1073/pnas.2004064117">Projecting hospital utilization during the COVID-19 outbreaks in the United States</a>. Proceedings of the National Academy of Sciences 117:9122–9126.</div>
</div>
<div id="ref-11CWFYFDL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">321. </div><div class="csl-right-inline">Prem K, Liu Y, Russell TW, Kucharski AJ, Eggo RM, Davies N, Jit M, Klepac P, Flasche S, Clifford S, Pearson CAB, Munday JD, Abbott S, Gibbs H, Rosello A, Quilty BJ, Jombart T, Sun F, Diamond C, Gimma A, van Zandvoort K, Funk S, Jarvis CI, Edmunds WJ, Bosse NI, Hellewell J. 2020. <a href="https://doi.org/10.1016/s2468-2667(20)30073-6">The effect of control strategies to reduce social mixing on outcomes of the COVID-19 epidemic in Wuhan, China: a modelling study</a>. The Lancet Public Health 5:e261–e270.</div>
</div>
<div id="ref-J4RCpntH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">322. </div><div class="csl-right-inline">Gatto M, Bertuzzo E, Mari L, Miccoli S, Carraro L, Casagrandi R, Rinaldo A. 2020. <a href="https://doi.org/10.1073/pnas.2004978117">Spread and dynamics of the COVID-19 epidemic in Italy: Effects of emergency containment measures</a>. Proceedings of the National Academy of Sciences 117:10484–10491.</div>
</div>
<div id="ref-WA1R5R95" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">323. </div><div class="csl-right-inline">EpiForecasts and the CMMID Covid working group. Covid-19: Temporal variation in transmission during the COVID-19 outbreak. <a href="https://epiforecasts.io/covid/">https://epiforecasts.io/covid/</a>. Retrieved 27 July 2020.</div>
</div>
<div id="ref-17CQnstjh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">324. </div><div class="csl-right-inline">Systrom K, Vladeck T, Krieger M. Rt COVID-19. <a href="https://rt.live/">https://rt.live/</a>. Retrieved 27 July 2020.</div>
</div>
<div id="ref-ZU1ZF4SW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">325. </div><div class="csl-right-inline">Zhou R, Li F, Chen F, Liu H, Zheng J, Lei C, Wu X. 2020. <a href="https://doi.org/10.1016/j.ijid.2020.05.030">Viral dynamics in asymptomatic patients with COVID-19</a>. International Journal of Infectious Diseases 96:288–290.</div>
</div>
<div id="ref-whGzxrkn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">326. </div><div class="csl-right-inline">Hasanoglu I, Korukluoglu G, Asilturk D, Cosgun Y, Kalem AK, Altas AB, Kayaaslan B, Eser F, Kuzucu EA, Guner R. 2020. <a href="https://doi.org/10.1007/s15010-020-01548-8">Higher viral loads in asymptomatic COVID-19 patients might be the invisible part of the iceberg</a>. Infection 49:117–126.</div>
</div>
<div id="ref-34wAjHW5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">327. </div><div class="csl-right-inline">Tsukagoshi H, Shinoda D, Saito M, Okayama K, Sada M, Kimura H, Saruki N. 2021. <a href="https://doi.org/10.3390/v13020304">Relationships between Viral Load and the Clinical Course of COVID-19</a>. Viruses 13:304.</div>
</div>
<div id="ref-1CA0Sj7dn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">328. </div><div class="csl-right-inline">Byambasuren O, Cardona M, Bell K, Clark J, McLaws M-L, Glasziou P. 2020. <a href="https://doi.org/10.3138/jammi-2020-0030">Estimating the extent of asymptomatic COVID-19 and its potential for community transmission: Systematic review and meta-analysis</a>. Official Journal of the Association of Medical Microbiology and Infectious Disease Canada 5:223–234.</div>
</div>
<div id="ref-13llzZ2qN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">329. </div><div class="csl-right-inline">Keehner J, Horton LE, Pfeffer MA, Longhurst CA, Schooley RT, Currier JS, Abeles SR, Torriani FJ. 2021. <a href="https://doi.org/10.1056/nejmc2101927">SARS-CoV-2 Infection after Vaccination in Health Care Workers in California</a>. N Engl J Med 384:1774–1775.</div>
</div>
<div id="ref-dLmXTkx0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">330. </div><div class="csl-right-inline">Tande AJ, Pollock BD, Shah ND, Farrugia G, Virk A, Swift M, Breeher L, Binnicker M, Berbari EF. 2021. <a href="https://doi.org/10.1093/cid/ciab229">Impact of the Coronavirus Disease 2019 (COVID-19) Vaccine on Asymptomatic Infection Among Patients Undergoing Preprocedural COVID-19 Molecular Screening</a>. Clinical Infectious Diseases 74:59–65.</div>
</div>
<div id="ref-zHE6Quu6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">331. </div><div class="csl-right-inline">Angel Y, Spitzer A, Henig O, Saiag E, Sprecher E, Padova H, Ben-Ami R. 2021. <a href="https://doi.org/10.1001/jama.2021.7152">Association Between Vaccination With BNT162b2 and Incidence of Symptomatic and Asymptomatic SARS-CoV-2 Infections Among Health Care Workers</a>. JAMA 325:2457.</div>
</div>
<div id="ref-D2ZCK63Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">332. </div><div class="csl-right-inline">Thompson MG, Burgess JL, Naleway AL, Tyner H, Yoon SK, Meece J, Olsho LEW, Caban-Martinez AJ, Fowlkes AL, Lutrick K, Groom HC, Dunnigan K, Odean MJ, Hegmann K, Stefanski E, Edwards LJ, Schaefer-Solle N, Grant L, Ellingson K, Kuntz JL, Zunie T, Thiese MS, Ivacic L, Wesley MG, Mayo Lamberte J, Sun X, Smith ME, Phillips AL, Groover KD, Yoo YM, Gerald J, Brown RT, Herring MK, Joseph G, Beitel S, Morrill TC, Mak J, Rivers P, Poe BP, Lynch B, Zhou Y, Zhang J, Kelleher A, Li Y, Dickerson M, Hanson E, Guenther K, Tong S, Bateman A, Reisdorf E, Barnes J, Azziz-Baumgartner E, Hunt DR, Arvay ML, Kutty P, Fry AM, Gaglani M. 2021. <a href="https://doi.org/10.1056/nejmoa2107058">Prevention and Attenuation of Covid-19 with the BNT162b2 and mRNA-1273 Vaccines</a>. N Engl J Med 385:320–329.</div>
</div>
<div id="ref-119cExL0k" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">333. </div><div class="csl-right-inline">Levine-Tiefenbrun M, Yelin I, Katz R, Herzel E, Golan Z, Schreiber L, Wolf T, Nadler V, Ben-Tov A, Kuint J, Gazit S, Patalon T, Chodick G, Kishony R. 2021. <a href="https://doi.org/10.1038/s41591-021-01316-7">Initial report of decreased SARS-CoV-2 viral load after inoculation with the BNT162b2 vaccine</a>. Nat Med 27:790–792.</div>
</div>
<div id="ref-e2Qnnj6R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">334. </div><div class="csl-right-inline">Chia PY, Xiang Ong SW, Chiew CJ, Ang LW, Chavatte J-M, Mak T-M, Cui L, Kalimuddin S, Chia WN, Tan CW, Ann Chai LY, Tan SY, Zheng S, Pin Lin RT, Wang L, Leo Y-S, Lee VJ, Lye DC, Young BE. 2021. <a href="https://doi.org/10.1101/2021.07.28.21261295">Virological and serological kinetics of SARS-CoV-2 Delta variant vaccine-breakthrough infections: a multi-center cohort study</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-N5OXLf7V" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">335. </div><div class="csl-right-inline">Riemersma KK, Haddock LA III, Wilson NA, Minor N, Eickhoff J, Grogan BE, Kita-Yarbro A, Halfmann PJ, Segaloff HE, Kocharian A, Florek KR, Westergaard R, Bateman A, Jeppson GE, Kawaoka Y, O’Connor DH, Friedrich TC, Grande KM. 2021. <a href="https://doi.org/10.1101/2021.07.31.21261387">Shedding of Infectious SARS-CoV-2 Despite Vaccination</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-mgscHeDu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">336. </div><div class="csl-right-inline">Acharya CB, Schrom J, Mitchell AM, Coil DA, Marquez C, Rojas S, Wang CY, Liu J, Pilarowski G, Solis L, Georgian E, Petersen M, DeRisi J, Michelmore R, Havlir D. 2021. <a href="https://doi.org/10.1101/2021.09.28.21264262">No Significant Difference in Viral Load Between Vaccinated and Unvaccinated, Asymptomatic and Symptomatic Groups When Infected with SARS-CoV-2 Delta Variant</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-QGjpUct1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">337. </div><div class="csl-right-inline">Anderson RM, Fraser C, Ghani AC, Donnelly CA, Riley S, Ferguson NM, Leung GM, Lam TH, Hedley AJ. 2004. <a href="https://doi.org/10.1098/rstb.2004.1490">Epidemiology, transmission dynamics and control of SARS: the 2002–2003 epidemic</a>. Philosophical Transactions of the Royal Society of London Series B: Biological Sciences 359:1091–1105.</div>
</div>
<div id="ref-nOGQjd1a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">338. </div><div class="csl-right-inline">2021. A timeline of the CDC's advice on face masks. Los Angeles Times. <a href="https://www.latimes.com/science/story/2021-07-27/timeline-cdc-mask-guidance-during-covid-19-pandemic">https://www.latimes.com/science/story/2021-07-27/timeline-cdc-mask-guidance-during-covid-19-pandemic</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-Up1vB19z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">339. </div><div class="csl-right-inline">COVID-19 Review Consortium. 2021. <a href="https://greenelab.github.io/covid19-review/v/32afa309f69f0466a91acec5d0df3151fe4d61b5/#social-factors-influencing-covid-19-exposure-and-outcomes">Social Factors Influencing COVID-19 Exposure and Outcomes</a>. Manubot.</div>
</div>
<div id="ref-sP4wQEiM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">340. </div><div class="csl-right-inline">Guarner J. 2020. <a href="https://doi.org/10.1093/ajcp/aqaa029">Three Emerging Coronaviruses in Two Decades</a>. American Journal of Clinical Pathology 153:420–421.</div>
</div>
<div id="ref-17qiILENK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">341. </div><div class="csl-right-inline">COVID-19 Review Consortium. 2021. <a href="https://greenelab.github.io/covid19-review/v/910dd7b7479f5336a1c911c57446829bef015dbe/#evolutionary-and-genomic-analysis-of-sars-cov-2">Evolutionary and Genomic Analysis of SARS-CoV-2</a>. Manubot.</div>
</div>
<div id="ref-UBTUyMGV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">342. </div><div class="csl-right-inline">Cui J, Li F, Shi Z-L. 2018. <a href="https://doi.org/10.1038/s41579-018-0118-9">Origin and evolution of pathogenic coronaviruses</a>. Nature Reviews Microbiology 17:181–192.</div>
</div>
<div id="ref-YAhVzI0A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">343. </div><div class="csl-right-inline">Lim Y, Ng Y, Tam J, Liu D. 2016. <a href="https://doi.org/10.3390/diseases4030026">Human Coronaviruses: A Review of Virus–Host Interactions</a>. Diseases 4:26.</div>
</div>
<div id="ref-SnIgqxRO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">344. </div><div class="csl-right-inline">Killerby ME, Biggs HM, Haynes A, Dahl RM, Mustaquim D, Gerber SI, Watson JT. 2018. <a href="https://doi.org/10.1016/j.jcv.2018.01.019">Human coronavirus circulation in the United States 2014–2017</a>. Journal of Clinical Virology 101:52–56.</div>
</div>
<div id="ref-gaAv88da" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">345. </div><div class="csl-right-inline">Hamre D, Procknow JJ. 1966. <a href="https://doi.org/10.3181/00379727-121-30734">A New Virus Isolated from the Human Respiratory Tract.</a> Experimental Biology and Medicine 121:190–193.</div>
</div>
<div id="ref-q6rFk3Kl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">346. </div><div class="csl-right-inline">McIntosh K, Dees JH, Becker WB, Kapikian AZ, Chanock RM. 1967. <a href="https://doi.org/10.1073/pnas.57.4.933">Recovery in tracheal organ cultures of novel viruses from patients with respiratory disease.</a> Proceedings of the National Academy of Sciences 57:933–940.</div>
</div>
<div id="ref-cEFaGSlu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">347. </div><div class="csl-right-inline">Pyrc K, Jebbink MF, Berkhout B, van der Hoek L. 2004. <a href="https://doi.org/10.1186/1743-422x-1-7">:(unav)</a>. Virology Journal 1:7.</div>
</div>
<div id="ref-Jnb5pce4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">348. </div><div class="csl-right-inline">Abdul-Rasool S, Fielding BC. 2010. <a href="https://doi.org/10.2174/1874357901004010076">Understanding Human Coronavirus HCoV-NL63.</a> Open Virol J 4:76–84.</div>
</div>
<div id="ref-nRPNH9B2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">349. </div><div class="csl-right-inline">Luo X, Zhou G-Z, Zhang Y, Peng L-H, Zou L-P, Yang Y-S. 2020. <a href="https://doi.org/10.1186/s40779-020-00279-z">Coronaviruses and gastrointestinal diseases</a>. Military Medical Research 7:49.</div>
</div>
<div id="ref-KEsllU9p" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">350. </div><div class="csl-right-inline">Vabret A, Dina J, Gouarin S, Petitjean J, Tripey V, Brouard J, Freymuth F. 2008. <a href="https://doi.org/10.1111/j.1440-1754.2007.01246.x">Human (non-severe acute respiratory syndrome) coronavirus infections in hospitalised children in France</a>. Journal of Paediatrics and Child Health 44:176–181.</div>
</div>
<div id="ref-URhDY7s9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">351. </div><div class="csl-right-inline">van der Hoek L, Pyrc K, Jebbink MF, Vermeulen-Oost W, Berkhout RJM, Wolthers KC, Wertheim-van Dillen PME, Kaandorp J, Spaargaren J, Berkhout B. 2004. <a href="https://doi.org/10.1038/nm1024">Identification of a new human coronavirus</a>. Nature Medicine 10:368–373.</div>
</div>
<div id="ref-VgGISeiE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">352. </div><div class="csl-right-inline">Woo PCY, Lau SKP, Chu C, Chan K, Tsoi H, Huang Y, Wong BHL, Poon RWS, Cai JJ, Luk W, Poon LLM, Wong SSY, Guan Y, Peiris JSM, Yuen K. 2005. <a href="https://doi.org/10.1128/jvi.79.2.884-895.2005">Characterization and Complete Genome Sequence of a Novel Coronavirus, Coronavirus HKU1, from Patients with Pneumonia</a>. Journal of Virology 79:884–895.</div>
</div>
<div id="ref-rDtyU6cv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">353. </div><div class="csl-right-inline">Pene F, Merlat A, Vabret A, Rozenberg F, Buzyn A, Dreyfus F, Cariou A, Freymuth F, Lebon P. 2003. <a href="https://doi.org/10.1086/377612">Coronavirus 229E-Related Pneumonia in Immunocompromised Patients</a>. Clinical Infectious Diseases 37:929–932.</div>
</div>
<div id="ref-uXEsLlzb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">354. </div><div class="csl-right-inline">Corman VM, Muth D, Niemeyer D, Drosten C. 2018. <a href="https://doi.org/10.1016/bs.aivir.2018.01.001">Hosts and Sources of Endemic Human Coronaviruses</a>Advances in Virus Research. Elsevier BV.</div>
</div>
<div id="ref-iOksGYVn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">355. </div><div class="csl-right-inline">Vlasova AN, Diaz A, Damtie D, Xiu L, Toh T-H, Lee JS-Y, Saif LJ, Gray GC. 2021. <a href="https://doi.org/10.1093/cid/ciab456">Novel Canine Coronavirus Isolated from a Hospitalized Patient With Pneumonia in East Malaysia</a>. Clinical Infectious Diseases 74:446–454.</div>
</div>
<div id="ref-vHq0hOWQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">356. </div><div class="csl-right-inline">Wu F, Zhao S, Yu B, Chen Y-M, Wang W, Song Z-G, Hu Y, Tao Z-W, Tian J-H, Pei Y-Y, Yuan M-L, Zhang Y-L, Dai F-H, Liu Y, Wang Q-M, Zheng J-J, Xu L, Holmes EC, Zhang Y-Z. 2020. <a href="https://doi.org/10.1038/s41586-020-2008-3">A new coronavirus associated with human respiratory disease in China</a>. Nature 579:265–269.</div>
</div>
<div id="ref-AwAyxtZs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">357. </div><div class="csl-right-inline">Taylor LH, Latham SM, woolhouse MEJ. 2001. <a href="https://doi.org/10.1098/rstb.2001.0888">Risk factors for human disease emergence</a>. Phil Trans R Soc Lond B 356:983–989.</div>
</div>
<div id="ref-1EdMJE4DK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">358. </div><div class="csl-right-inline">Han BA, Kramer AM, Drake JM. 2016. <a href="https://doi.org/10.1016/j.pt.2016.04.007">Global Patterns of Zoonotic Disease in Mammals</a>. Trends in Parasitology 32:565–577.</div>
</div>
<div id="ref-scqqnokl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">359. </div><div class="csl-right-inline">Wolfe ND, Daszak P, Kilpatrick AM, Burke DS. 2005. <a href="https://doi.org/10.3201/eid1112.040789">Bushmeat Hunting, Deforestation, and Prediction of Zoonotic Disease</a>. Emerg Infect Dis 11:1822–1827.</div>
</div>
<div id="ref-ddJ8UwvM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">360. </div><div class="csl-right-inline">Guan Y, Zheng BJ, He YQ, Liu XL, Zhuang ZX, Cheung CL, Luo SW, Li PH, Zhang LJ, Guan YJ, Butt KM, Wong KL, Chan KW, Lim W, Shortridge KF, Yuen KY, Peiris JSM, Poon LLM. 2003. <a href="https://doi.org/10.1126/science.1087139">Isolation and Characterization of Viruses Related to the SARS Coronavirus from Animals in Southern China</a>. Science 302:276–278.</div>
</div>
<div id="ref-dwmFUaCY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">361. </div><div class="csl-right-inline">Ye Z-W, Yuan S, Yuen K-S, Fung S-Y, Chan C-P, Jin D-Y. 2020. <a href="https://doi.org/10.7150/ijbs.45472">Zoonotic origins of human coronaviruses</a>. Int J Biol Sci 16:1686–1697.</div>
</div>
<div id="ref-TvDJAYpy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">362. </div><div class="csl-right-inline">Lau SKP, Woo PCY, Li KSM, Huang Y, Tsoi H-W, Wong BHL, Wong SSY, Leung S-Y, Chan K-H, Yuen K-Y. 2005. <a href="https://doi.org/10.1073/pnas.0506735102">Severe acute respiratory syndrome coronavirus-like virus in Chinese horseshoe bats</a>. Proc Natl Acad Sci USA 102:14040–14045.</div>
</div>
<div id="ref-hWiB2Isn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">363. </div><div class="csl-right-inline">Cyranoski D. 2020. <a href="https://doi.org/10.1038/d41586-020-00548-w">Mystery deepens over animal source of coronavirus</a>. Nature 579:18–19.</div>
</div>
<div id="ref-pekmozuU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">364. </div><div class="csl-right-inline">Ill husband and wife add to Wuhan riddle - The Standard. <a href="https://www.thestandard.com.hk/sections-news-print/215457/Ill-husband-and-wife">https://www.thestandard.com.hk/sections-news-print/215457/Ill-husband-and-wife</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-QACJGDk7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">365. </div><div class="csl-right-inline">Xiao X, Newman C, Buesching CD, Macdonald DW, Zhou Z-M. 2021. <a href="https://doi.org/10.1038/s41598-021-91470-2">Animal sales from Wuhan wet markets immediately prior to the COVID-19 pandemic</a>. Sci Rep 11.</div>
</div>
<div id="ref-BUHjrVM8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">366. </div><div class="csl-right-inline">Brook CE, Dobson AP. 2015. <a href="https://doi.org/10.1016/j.tim.2014.12.004">Bats as ‘special’ reservoirs for emerging zoonotic pathogens</a>. Trends in Microbiology 23:172–180.</div>
</div>
<div id="ref-12G9bKpJg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">367. </div><div class="csl-right-inline">Han H-J, Wen H, Zhou C-M, Chen F-F, Luo L-M, Liu J, Yu X-J. 2015. <a href="https://doi.org/10.1016/j.virusres.2015.05.006">Bats as reservoirs of severe emerging infectious diseases</a>. Virus Research 205:1–6.</div>
</div>
<div id="ref-FA1fM4ii" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">368. </div><div class="csl-right-inline">Calisher CH, Childs JE, Field HE, Holmes KV, Schountz T. 2006. <a href="https://doi.org/10.1128/cmr.00017-06">Bats: Important Reservoir Hosts of Emerging Viruses</a>. Clin Microbiol Rev 19:531–545.</div>
</div>
<div id="ref-ZmBmnPMY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">369. </div><div class="csl-right-inline">Zhang T, Wu Q, Zhang Z. 2020. <a href="https://doi.org/10.1016/j.cub.2020.03.022">Probable Pangolin Origin of SARS-CoV-2 Associated with the COVID-19 Outbreak</a>. Current Biology 30:1346–1351.e2.</div>
</div>
<div id="ref-Lze8f0UK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">370. </div><div class="csl-right-inline">Lam TT-Y, Jia N, Zhang Y-W, Shum MH-H, Jiang J-F, Zhu H-C, Tong Y-G, Shi Y-X, Ni X-B, Liao Y-S, Li W-J, Jiang B-G, Wei W, Yuan T-T, Zheng K, Cui X-M, Li J, Pei G-Q, Qiang X, Cheung WY-M, Li L-F, Sun F-F, Qin S, Huang J-C, Leung GM, Holmes EC, Hu Y-L, Guan Y, Cao W-C. 2020. <a href="https://doi.org/10.1038/s41586-020-2169-0">Identifying SARS-CoV-2-related coronaviruses in Malayan pangolins</a>. Nature 583:282–285.</div>
</div>
<div id="ref-NEIoHDjq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">371. </div><div class="csl-right-inline">Andersen KG, Rambaut A, Lipkin WI, Holmes EC, Garry RF. 2020. <a href="https://doi.org/10.1038/s41591-020-0820-9">The proximal origin of SARS-CoV-2</a>. Nature Medicine 26:450–452.</div>
</div>
<div id="ref-wz7kPyn2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">372. </div><div class="csl-right-inline">Li X, Giorgi EE, Marichannegowda MH, Foley B, Xiao C, Kong X-P, Chen Y, Gnanakaran S, Korber B, Gao F. 2020. <a href="https://doi.org/10.1126/sciadv.abb9153">Emergence of SARS-CoV-2 through recombination and strong purifying selection</a>. Science Advances 6:eabb9153.</div>
</div>
<div id="ref-37lIze3F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">373. </div><div class="csl-right-inline">Banerjee A, Doxey AC, Mossman K, Irving AT. 2021. <a href="https://doi.org/10.1016/j.tree.2020.12.002">Unraveling the Zoonotic Origin and Transmission of SARS-CoV-2</a>. Trends in Ecology &amp;amp; Evolution 36:180–184.</div>
</div>
<div id="ref-13xHe6Stn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">374. </div><div class="csl-right-inline">Bloom JD, Chan YA, Baric RS, Bjorkman PJ, Cobey S, Deverman BE, Fisman DN, Gupta R, Iwasaki A, Lipsitch M, Medzhitov R, Neher RA, Nielsen R, Patterson N, Stearns T, van Nimwegen E, Worobey M, Relman DA. 2021. <a href="https://doi.org/10.1126/science.abj0016">Investigate the origins of COVID-19</a>. Science 372:694–694.</div>
</div>
<div id="ref-CS3vp1Bh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">375. </div><div class="csl-right-inline">Maxmen A. 2021. <a href="https://doi.org/10.1038/d41586-021-01383-3">Divisive COVID ‘lab leak’ debate prompts dire warnings from researchers</a>. Nature 594:15–16.</div>
</div>
<div id="ref-XrPq1Dvp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">376. </div><div class="csl-right-inline">Bender JB, Shulman SA, __. 2004. <a href="https://doi.org/10.2460/javma.2004.224.1105">Reports of zoonotic disease outbreaks associated with animal exhibits and availability of recommendations for preventing zoonotic disease transmission from animals to people in such settings</a>. javma 224:1105–1109.</div>
</div>
<div id="ref-10oGRZVIE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">377. </div><div class="csl-right-inline">Saif LJ. 1993. <a href="https://doi.org/10.1016/0378-1135(93)90030-b">Coronavirus immunogens</a>. Veterinary Microbiology 37:285–297.</div>
</div>
<div id="ref-gPVseu0D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">378. </div><div class="csl-right-inline">Fermin G. 2018. <a href="https://doi.org/10.1016/b978-0-12-811257-1.00005-x">Host Range, Host–Virus Interactions, and Virus Transmission</a>, p. 101–134. <em>In</em> Viruses. Elsevier.</div>
</div>
<div id="ref-zoq1a1or" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">379. </div><div class="csl-right-inline">Kim J, Farré M, Auvil L, Capitanu B, Larkin DM, Ma J, Lewin HA. 2017. <a href="https://doi.org/10.1073/pnas.1702012114">Reconstruction and evolutionary history of eutherian chromosomes</a>. Proc Natl Acad Sci USA 114.</div>
</div>
<div id="ref-Z0OkYpgf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">380. </div><div class="csl-right-inline">Tsagkogeorga G, Parker J, Stupka E, Cotton James A, Rossiter Stephen J. 2013. <a href="https://doi.org/10.1016/j.cub.2013.09.014">Phylogenomic Analyses Elucidate the Evolutionary Relationships of Bats</a>. Current Biology 23:2262–2267.</div>
</div>
<div id="ref-89XVYM09" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">381. </div><div class="csl-right-inline">Nishihara H, Hasegawa M, Okada N. 2006. <a href="https://doi.org/10.1073/pnas.0603797103">Pegasoferae, an unexpected mammalian clade revealed by tracking ancient retroposon insertions</a>. Proc Natl Acad Sci USA 103:9929–9934.</div>
</div>
<div id="ref-1EDurLKKP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">382. </div><div class="csl-right-inline">Murphy WJ, Pringle TH, Crider TA, Springer MS, Miller W. 2007. <a href="https://doi.org/10.1101/gr.5918807">Using genomic data to unravel the root of the placental mammal phylogeny</a>. Genome Res 17:413–421.</div>
</div>
<div id="ref-hmVYftwt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">383. </div><div class="csl-right-inline">Murphy WJ, Eizirik E. 2009. <a href="http://www.timetree.org/public/data/pdf/Murphy2009Chap71.pdf">Placental mammals (Eutheria)</a>, p. 471–474. <em>In</em> The Timetree of Life. Oxford University Press.</div>
</div>
<div id="ref-j2EJdPne" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">384. </div><div class="csl-right-inline">McNamara T, Richt JA, Glickman L. 2020. <a href="https://doi.org/10.1089/vbz.2020.2650">A Critical Needs Assessment for Research in Companion Animals and Livestock Following the Pandemic of COVID-19 in Humans</a>. Vector-Borne and Zoonotic Diseases 20:393–405.</div>
</div>
<div id="ref-19JK8Xa1b" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">385. </div><div class="csl-right-inline">Li R, Qiao S, Zhang G. 2020. <a href="https://doi.org/10.1016/j.jinf.2020.02.013">Analysis of angiotensin-converting enzyme 2 (ACE2) from different species sheds some light on cross-species receptor usage of a novel coronavirus 2019-nCoV</a>. Journal of Infection 80:469–496.</div>
</div>
<div id="ref-1FxiM6ws1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">386. </div><div class="csl-right-inline">Damas J, Hughes GM, Keough KC, Painter CA, Persky NS, Corbo M, Hiller M, Koepfli K-P, Pfenning AR, Zhao H, Genereux DP, Swofford R, Pollard KS, Ryder OA, Nweeia MT, Lindblad-Toh K, Teeling EC, Karlsson EK, Lewin HA. 2020. <a href="https://doi.org/10.1073/pnas.2010146117">Broad host range of SARS-CoV-2 predicted by comparative and structural analysis of ACE2 in vertebrates</a>. Proc Natl Acad Sci USA 117:22311–22322.</div>
</div>
<div id="ref-CZiN878H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">387. </div><div class="csl-right-inline">Opriessnig T, Huang Y. 2020. <a href="https://doi.org/10.1111/xen.12621">Update on possible animal sources for COVID‐19 in humans</a>. Xenotransplantation 27.</div>
</div>
<div id="ref-ELDaiz1W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">388. </div><div class="csl-right-inline">Hosie MJ, Hofmann-Lehmann R, Hartmann K, Egberink H, Truyen U, Addie DD, Belák S, Boucraut-Baralon C, Frymus T, Lloret A, Lutz H, Marsilio F, Pennisi MG, Tasker S, Thiry E, Möstl K. 2021. <a href="https://doi.org/10.3390/v13020185">Anthropogenic Infection of Cats during the 2020 COVID-19 Pandemic</a>. Viruses 13:185.</div>
</div>
<div id="ref-17761toGa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">389. </div><div class="csl-right-inline">Sit THC, Brackman CJ, Ip SM, Tam KWS, Law PYT, To EMW, Yu VYT, Sims LD, Tsang DNC, Chu DKW, Perera RAPM, Poon LLM, Peiris M. 2020. <a href="https://doi.org/10.1038/s41586-020-2334-5">Infection of dogs with SARS-CoV-2</a>. Nature 586:776–778.</div>
</div>
<div id="ref-Ih3JjS7V" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">390. </div><div class="csl-right-inline">Decaro N, Vaccari G, Lorusso A, Lorusso E, De Sabato L, Patterson EI, Di Bartolo I, Hughes GL, Teodori L, Desario C, Colitti B, Ricci D, Buonavoglia D, Rosati S, Martella V, Cammà C, Agrimi U, Elia G. 2021. <a href="https://doi.org/10.3201/eid2707.204959">Possible Human-to-Dog Transmission of SARS-CoV-2, Italy, 2020</a>. Emerg Infect Dis 27:1981–1984.</div>
</div>
<div id="ref-KsMjmcLV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">391. </div><div class="csl-right-inline">Patterson EI, Elia G, Grassi A, Giordano A, Desario C, Medardo M, Smith SL, Anderson ER, Prince T, Patterson GT, Lorusso E, Lucente MS, Lanave G, Lauzi S, Bonfanti U, Stranieri A, Martella V, Solari Basano F, Barrs VR, Radford AD, Agrimi U, Hughes GL, Paltrinieri S, Decaro N. 2020. <a href="https://doi.org/10.1038/s41467-020-20097-0">Evidence of exposure to SARS-CoV-2 in cats and dogs from households in Italy</a>. Nat Commun 11.</div>
</div>
<div id="ref-14IpxqUxX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">392. </div><div class="csl-right-inline">Mohebali M, Hassanpour G, Zainali M, Gouya MM, Khayatzadeh S, Parsaei M, Sarafraz N, Hassanzadeh M, Azarm A, Salehi-Vaziri M, Sasani F, Heidari Z, Jalali T, Pouriayevali MH, Shoja Z, Ahmadi Z, Sadjadi M, Tavakoli M, Azad-Manjiri S, Karami C, Zarei Z. 2022. <a href="https://doi.org/10.1016/j.virusres.2022.198673">SARS-CoV-2 in domestic cats (Felis catus) in the northwest of Iran: Evidence for SARS-CoV-2 circulating between human and cats</a>. Virus Research 310:198673.</div>
</div>
<div id="ref-1HOJDUg7F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">393. </div><div class="csl-right-inline">Bosco-Lauth AM, Hartwig AE, Porter SM, Gordy PW, Nehring M, Byas AD, VandeWoude S, Ragan IK, Maison RM, Bowen RA. 2020. <a href="https://doi.org/10.1073/pnas.2013102117">Experimental infection of domestic dogs and cats with SARS-CoV-2: Pathogenesis, transmission, and response to reexposure in cats</a>. Proc Natl Acad Sci USA 117:26382–26388.</div>
</div>
<div id="ref-18ycM4YxO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">394. </div><div class="csl-right-inline">2020. Exclusive: Buddy, first dog to test positive for COVID-19 in the U.S., has died. Animals. <a href="https://www.nationalgeographic.com/animals/article/first-dog-to-test-positive-for-covid-in-us-dies">https://www.nationalgeographic.com/animals/article/first-dog-to-test-positive-for-covid-in-us-dies</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1D5YVPEpJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">395. </div><div class="csl-right-inline">Connecticut puppy that died unexpectedly tested positive for coronavirus, UConn researchers say. Hartford Courant. <a href="https://www.courant.com/coronavirus/hc-news-coronavirus-uconn-dog-infection-20210413-yrbg6icd6bcirk6ylwnzkxi6di-story.html">https://www.courant.com/coronavirus/hc-news-coronavirus-uconn-dog-infection-20210413-yrbg6icd6bcirk6ylwnzkxi6di-story.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-yvRoFaEb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">396. </div><div class="csl-right-inline">North Carolina dog that died after 'acute' illness tests positive for coronavirus. NBC News. <a href="https://www.nbcnews.com/news/us-news/north-carolina-dog-died-after-acute-illness-tests-positive-coronavirus-n1236477">https://www.nbcnews.com/news/us-news/north-carolina-dog-died-after-acute-illness-tests-positive-coronavirus-n1236477</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-18JLC1Wre" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">397. </div><div class="csl-right-inline">Hart R. Kitten Dies After Catching Covid As Study Uncovers More Evidence Of Human-To-Cat Transmission. Forbes. <a href="https://www.forbes.com/sites/roberthart/2021/04/23/kitten-dies-after-catching-covid-as-study-uncovers-more-evidence-of-human-to-cat-transmission/">https://www.forbes.com/sites/roberthart/2021/04/23/kitten-dies-after-catching-covid-as-study-uncovers-more-evidence-of-human-to-cat-transmission/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-vTnD9MOk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">398. </div><div class="csl-right-inline">Hosie MJ, Epifano I, Herder V, Orton RJ, Stevenson A, Johnson N, MacDonald E, Dunbar D, McDonald M, Howie F, Tennant B, Herrity D, Da Silva Filipe A, Streicker DG, Willett BJ, Murcia PR, Jarrett RF, Robertson DL, Weir W. 2021. <a href="https://doi.org/10.1002/vetr.247">Detection of SARS‐CoV‐2 in respiratory samples from cats in the UK associated with human‐to‐cat transmission</a>. Veterinary Record 188.</div>
</div>
<div id="ref-irb2yWQa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">399. </div><div class="csl-right-inline">Collins A. Investigation into cat’s death, cat had COVID. https://wwwwbrccom. <a href="https://www.wbrc.com/2020/10/09/investigation-into-cats-death-cat-had-covid/">https://www.wbrc.com/2020/10/09/investigation-into-cats-death-cat-had-covid/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-IumrstNg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">400. </div><div class="csl-right-inline">do Vale B, Lopes AP, Fontes M da C, Silvestre M, Cardoso L, Coelho AC. 2021. <a href="https://doi.org/10.1007/s11259-021-09787-2">Bats, pangolins, minks and other animals - villains or victims of SARS-CoV-2?</a> Vet Res Commun 45:1–19.</div>
</div>
<div id="ref-EVS2FryQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">401. </div><div class="csl-right-inline">Bosco-Lauth AM, Walker A, Guilbert L, Porter S, Hartwig A, McVicker E, Bielefeldt-Ohmann H, Bowen RA. 2021. <a href="https://doi.org/10.1080/22221751.2021.2003724">Susceptibility of livestock to SARS-CoV-2 infection</a>. Emerging Microbes &amp;amp; Infections 10:2199–2201.</div>
</div>
<div id="ref-17bpCXS4N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">402. </div><div class="csl-right-inline">Gaudreault NN, Cool K, Trujillo JD, Morozov I, Meekins DA, McDowell C, Bold D, Carossino M, Balaraman V, Mitzel D, Kwon T, Madden DW, Artiaga BL, Pogranichniy RM, Roman-Sosa G, Wilson WC, Balasuriya UBR, García-Sastre A, Richt JA. 2022. <a href="https://doi.org/10.1080/22221751.2022.2037397">Susceptibility of sheep to experimental co-infection with the ancestral lineage of SARS-CoV-2 and its alpha variant</a>. Emerging Microbes &amp;amp; Infections 11:662–675.</div>
</div>
<div id="ref-BS9gjNG4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">403. </div><div class="csl-right-inline">Villanueva-Saz S, Giner J, Fernández A, Lacasta D, Ortín A, Ramos JJ, Ferrer LM, Ruiz de Arcaute M, Tobajas AP, Pérez MD, Verde M, Marteles D, Hurtado-Guerrero R, Pardo J, Santiago L, González-Ramírez AM, Macías-León J, García-García A, Taleb V, Lira-Navarrete E, Paño-Pardo JR, Ruíz H. 2021. <a href="https://doi.org/10.3390/ani11071984">Absence of SARS-CoV-2 Antibodies in Natural Environment Exposure in Sheep in Close Contact with Humans</a>. Animals 11:1984.</div>
</div>
<div id="ref-hJNwsIkr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">404. </div><div class="csl-right-inline">Taylor CA, Boulos C, Almond D. 2020. <a href="https://doi.org/10.1073/pnas.2010115117">Livestock plants and COVID-19 transmission</a>. Proc Natl Acad Sci USA 117:31706–31715.</div>
</div>
<div id="ref-Aj0mjB0J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">405. </div><div class="csl-right-inline">Marchant-Forde JN, Boyle LA. 2020. <a href="https://doi.org/10.3389/fvets.2020.585787">COVID-19 Effects on Livestock Production: A One Welfare Issue</a>. Front Vet Sci 7.</div>
</div>
<div id="ref-1BS2uKJCI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">406. </div><div class="csl-right-inline">Oreshkova N, Molenaar RJ, Vreman S, Harders F, Oude Munnink BB, Hakze-van der Honing RW, Gerhards N, Tolsma P, Bouwstra R, Sikkema RS, Tacken MG, de Rooij MM, Weesendorp E, Engelsma MY, Bruschke CJ, Smit LA, Koopmans M, van der Poel WH, Stegeman A. 2020. <a href="https://doi.org/10.2807/1560-7917.es.2020.25.23.2001005">SARS-CoV-2 infection in farmed minks, the Netherlands, April and May 2020</a>. Eurosurveillance 25.</div>
</div>
<div id="ref-EWObKVab" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">407. </div><div class="csl-right-inline">Hammer AS, Quaade ML, Rasmussen TB, Fonager J, Rasmussen M, Mundbjerg K, Lohse L, Strandbygaard B, Jørgensen CS, Alfaro-Núñez A, Rosenstierne MW, Boklund A, Halasa T, Fomsgaard A, Belsham GJ, Bøtner A. 2021. <a href="https://doi.org/10.3201/eid2702.203794">SARS-CoV-2 Transmission between Mink (&lt;i&gt;Neovison vison&lt;/i&gt;) and Humans, Denmark</a>. Emerg Infect Dis 27:547–551.</div>
</div>
<div id="ref-eZlF3Ang" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">408. </div><div class="csl-right-inline">Eckstrand CD, Baldwin TJ, Rood KA, Clayton MJ, Lott JK, Wolking RM, Bradway DS, Baszler T. 2021. <a href="https://doi.org/10.1371/journal.ppat.1009952">An outbreak of SARS-CoV-2 with high mortality in mink (Neovison vison) on multiple Utah farms</a>. PLoS Pathog 17:e1009952.</div>
</div>
<div id="ref-tb42t88r" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">409. </div><div class="csl-right-inline">Larsen HD, Fonager J, Lomholt FK, Dalby T, Benedetti G, Kristensen B, Urth TR, Rasmussen M, Lassaunière R, Rasmussen TB, Strandbygaard B, Lohse L, Chaine M, Møller KL, Berthelsen A-SN, Nørgaard SK, Sönksen UW, Boklund AE, Hammer AS, Belsham GJ, Krause TG, Mortensen S, Bøtner A, Fomsgaard A, Mølbak K. 2021. <a href="https://doi.org/10.2807/1560-7917.es.2021.26.5.210009">Preliminary report of an outbreak of SARS-CoV-2 in mink and mink farmers associated with community spread, Denmark, June to November 2020</a>. Eurosurveillance 26.</div>
</div>
<div id="ref-TKJuIOB1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">410. </div><div class="csl-right-inline">Hoffmann M, Zhang L, Krüger N, Graichen L, Kleine-Weber H, Hofmann-Winkler H, Kempf A, Nessler S, Riggert J, Winkler MS, Schulz S, Jäck H-M, Pöhlmann S. 2021. <a href="https://doi.org/10.1016/j.celrep.2021.109017">SARS-CoV-2 mutations acquired in mink reduce antibody-mediated neutralization</a>. Cell Reports 35:109017.</div>
</div>
<div id="ref-cu0cA7e7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">411. </div><div class="csl-right-inline">Fernández-Bellon H, Rodon J, Fernández-Bastit L, Almagro V, Padilla-Solé P, Lorca-Oró C, Valle R, Roca N, Grazioli S, Trogu T, Bensaid A, Carrillo J, Izquierdo-Useros N, Blanco J, Parera M, Noguera-Julián M, Clotet B, Moreno A, Segalés J, Vergara-Alert J. 2021. <a href="https://doi.org/10.3390/v13091683">Monitoring Natural SARS-CoV-2 Infection in Lions (Panthera leo) at the Barcelona Zoo: Viral Dynamics and Host Responses</a>. Viruses 13:1683.</div>
</div>
<div id="ref-RlXCplKh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">412. </div><div class="csl-right-inline">Frutos R, Devaux CA. 2020. <a href="https://doi.org/10.1016/j.nmni.2020.100816">Mass culling of minks to protect the COVID-19 vaccines: is it rational?</a> New Microbes and New Infections 38:100816.</div>
</div>
<div id="ref-DcwRbCdD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">413. </div><div class="csl-right-inline">Enserink M. 2020. <a href="https://doi.org/10.1126/science.368.6496.1169">Coronavirus rips through Dutch mink farms, triggering culls</a>. Science 368:1169–1169.</div>
</div>
<div id="ref-sMtrNzmx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">414. </div><div class="csl-right-inline">Harrington LA, Díez‐León M, Gómez A, Harrington A, Macdonald DW, Maran T, Põdra M, Roy S. 2021. <a href="https://doi.org/10.1002/fee.2344">Wild American mink (
&lt;i&gt;Neovison vison&lt;/i&gt;
) may pose a COVID‐19 threat</a>. Front Ecol Environ 19:266–267.</div>
</div>
<div id="ref-1CPlZeGO2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">415. </div><div class="csl-right-inline">Aguiló-Gisbert J, Padilla-Blanco M, Lizana V, Maiques E, Muñoz-Baquero M, Chillida-Martínez E, Cardells J, Rubio-Guerri C. 2021. <a href="https://doi.org/10.3390/ani11051422">First Description of SARS-CoV-2 Infection in Two Feral American Mink (Neovison vison) Caught in the Wild</a>. Animals 11:1422.</div>
</div>
<div id="ref-gbCOOK7q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">416. </div><div class="csl-right-inline">USDA APHIS | Confirmation of COVID-19 in Gorillas at a California Zoo. <a href="https://www.aphis.usda.gov/aphis/newsroom/stakeholder-info/sa_by_date/sa-2021/sa-01/ca-gorillas-sars-cov-2">https://www.aphis.usda.gov/aphis/newsroom/stakeholder-info/sa_by_date/sa-2021/sa-01/ca-gorillas-sars-cov-2</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-95bYQKt8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">417. </div><div class="csl-right-inline">2022. Dallas Zoo says 5 gorillas have tested positive for COVID-19. wfaacom. <a href="https://www.wfaa.com/article/news/local/dallas-zoo-5-gorillas-tested-positive-covid-19/287-901a235f-b120-49ff-9fbe-1a1a6dfbadb5">https://www.wfaa.com/article/news/local/dallas-zoo-5-gorillas-tested-positive-covid-19/287-901a235f-b120-49ff-9fbe-1a1a6dfbadb5</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-RCnX1l94" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">418. </div><div class="csl-right-inline">2021. Nearly all gorillas at Atlanta's zoo have contracted COVID-19. CTVNews. <a href="https://www.ctvnews.ca/health/coronavirus/nearly-all-gorillas-at-atlanta-s-zoo-have-contracted-covid-19-1.5586112">https://www.ctvnews.ca/health/coronavirus/nearly-all-gorillas-at-atlanta-s-zoo-have-contracted-covid-19-1.5586112</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-dabOG8Ms" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">419. </div><div class="csl-right-inline">Staff K9N. 2021. Kansas City Zoo gorillas recovering from COVID-19. KMBC. <a href="https://www.kmbc.com/article/kc-zoo-gorillas-recovering-from-covid/37873956">https://www.kmbc.com/article/kc-zoo-gorillas-recovering-from-covid/37873956</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-lHwLc22O" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">420. </div><div class="csl-right-inline">McAloose D, Laverack M, Wang L, Killian ML, Caserta LC, Yuan F, Mitchell PK, Queen K, Mauldin MR, Cronk BD, Bartlett SL, Sykes JM, Zec S, Stokol T, Ingerman K, Delaney MA, Fredrickson R, Ivančić M, Jenkins-Moore M, Mozingo K, Franzen K, Bergeson NH, Goodman L, Wang H, Fang Y, Olmstead C, McCann C, Thomas P, Goodrich E, Elvinger F, Smith DC, Tong S, Slavinski S, Calle PP, Terio K, Torchetti MK, Diel DG. 2020. <a href="https://doi.org/10.1128/mbio.02220-20">From People to
&lt;i&gt;Panthera&lt;/i&gt;
: Natural SARS-CoV-2 Infection in Tigers and Lions at the Bronx Zoo</a>. mBio 11.</div>
</div>
<div id="ref-IJr533OF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">421. </div><div class="csl-right-inline">2022. A snow leopard at Miller Park Zoo dies from COVID-induced pneumonia. WGLT. <a href="https://www.wglt.org/local-news/2022-01-06/a-snow-leopard-at-miller-park-zoo-is-mclean-countys-latest-death-from-covid-19">https://www.wglt.org/local-news/2022-01-06/a-snow-leopard-at-miller-park-zoo-is-mclean-countys-latest-death-from-covid-19</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-yQVSsUGy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">422. </div><div class="csl-right-inline">Thebault R. 2021. <a href="https://www.washingtonpost.com/science/2021/11/14/snow-leopard-death-covid/">A zoo’s three ‘beloved’ snow leopards die of covid-19</a>. Washington Post.</div>
</div>
<div id="ref-54pXgduy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">423. </div><div class="csl-right-inline">Anthes E. 2022. <a href="https://www.nytimes.com/2022/02/21/health/gorillas-respiratory-illness-colds.html">When People Take Pandemic Precautions, Gorillas Breathe Easier</a>. The New York Times.</div>
</div>
<div id="ref-KOsmKRWs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">424. </div><div class="csl-right-inline">Cranfield MR. 2008. <a href="https://doi.org/10.1002/ajp.20564">Mountain gorilla research: the risk of disease transmission relative to the benefit from the perspective of ecosystem health</a>. Am J Primatol 70:751–754.</div>
</div>
<div id="ref-bOYylTUp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">425. </div><div class="csl-right-inline">Firozi P. 2021. <a href="https://www.washingtonpost.com/nation/2021/09/12/zoo-atlanta-gorillas-coronavirus-vaccine/">A group of gorillas is being treated for covid. The great apes will soon get their shots, too, zoo says.</a> Washington Post.</div>
</div>
<div id="ref-QQlhSDpY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">426. </div><div class="csl-right-inline">Hoyte A, Webster M, Ameiss K, Conlee DA, Hainer N, Hutchinson K, Burakova Y, Dominowski PJ, Baima ET, King VL, Rosey EL, Hardham JM, Millership J, Kumar M. 2022. <a href="https://doi.org/10.1016/j.vetmic.2022.109395">Experimental veterinary SARS-CoV-2 vaccine cross neutralization of the Delta (B.1.617.2) variant virus in cats</a>. Veterinary Microbiology 268:109395.</div>
</div>
<div id="ref-KEpTBsAB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">427. </div><div class="csl-right-inline">COVID-19 Animal Vaccines. Zoetis. <a href="https://www.zoetis.com/news-and-insights/featured-stories/zoetis-emerging-infectious-disease-capabilities-support-covid-19-solutions-for-great-apes-and-minks">https://www.zoetis.com/news-and-insights/featured-stories/zoetis-emerging-infectious-disease-capabilities-support-covid-19-solutions-for-great-apes-and-minks</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-118xjDmDG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">428. </div><div class="csl-right-inline">Sharun K, Tiwari R, Saied AA, Dhama K. 2021. <a href="https://doi.org/10.1016/j.vaccine.2021.10.053">SARS-CoV-2 vaccine for domestic and captive animals: An effort to counter COVID-19 pandemic at the human-animal interface</a>. Vaccine 39:7119–7122.</div>
</div>
<div id="ref-H4B703HM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">429. </div><div class="csl-right-inline">Do we need to have a Covid vaccine for domestic animals? The Irish Times. <a href="https://www.irishtimes.com/life-and-style/do-we-need-to-have-a-covid-vaccine-for-domestic-animals-1.4736360">https://www.irishtimes.com/life-and-style/do-we-need-to-have-a-covid-vaccine-for-domestic-animals-1.4736360</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-dzBcxbby" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">430. </div><div class="csl-right-inline">Chavda VP, Feehan J, Apostolopoulos V. 2021. <a href="https://doi.org/10.3390/vaccines9060631">A Veterinary Vaccine for SARS-CoV-2: The First COVID-19 Vaccine for Animals</a>. Vaccines 9:631.</div>
</div>
<div id="ref-Le00U1aY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">431. </div><div class="csl-right-inline">Nguyen C, Campos R, Horn • • M. Cats and Dogs Top List of COVID-19 Infected Animals in US. NBC Bay Area. <a href="https://www.nbcbayarea.com/investigations/cats-and-dogs-top-list-of-covid-19-infected-animals-in-u-s/2625085/">https://www.nbcbayarea.com/investigations/cats-and-dogs-top-list-of-covid-19-infected-animals-in-u-s/2625085/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-vzPlWSZd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">432. </div><div class="csl-right-inline">Liu Y, Hu G, Wang Y, Ren W, Zhao X, Ji F, Zhu Y, Feng F, Gong M, Ju X, Zhu Y, Cai X, Lan J, Guo J, Xie M, Dong L, Zhu Z, Na J, Wu J, Lan X, Xie Y, Wang X, Yuan Z, Zhang R, Ding Q. 2021. <a href="https://doi.org/10.1073/pnas.2025373118">Functional and genetic analysis of viral receptor ACE2 orthologs reveals a broad potential host range of SARS-CoV-2</a>. Proc Natl Acad Sci USA 118.</div>
</div>
<div id="ref-13sevtn6P" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">433. </div><div class="csl-right-inline">Conceicao C, Thakur N, Human S, Kelly JT, Logan L, Bialy D, Bhat S, Stevenson-Leggett P, Zagrajek AK, Hollinghurst P, Varga M, Tsirigoti C, Tully M, Chiu C, Moffat K, Silesian AP, Hammond JA, Maier HJ, Bickerton E, Shelton H, Dietrich I, Graham SC, Bailey D. 2020. <a href="https://doi.org/10.1371/journal.pbio.3001016">The SARS-CoV-2 Spike protein has a broad tropism for mammalian ACE2 proteins</a>. PLoS Biol 18:e3001016.</div>
</div>
<div id="ref-1GJ6Mge9A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">434. </div><div class="csl-right-inline">Sullivan PF. 2007. <a href="https://doi.org/10.1016/j.biopsych.2006.11.010">Spurious Genetic Associations</a>. Biological Psychiatry 61:1121–1126.</div>
</div>
<div id="ref-FbeY1L1s" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">435. </div><div class="csl-right-inline">Zhang Y, Qin L, Zhao Y, Zhang P, Xu B, Li K, Liang L, Zhang C, Dai Y, Feng Y, Sun J, Hu Z, Xiang H, Knight JC, Dong T, Jin R. 2020. <a href="https://doi.org/10.1093/infdis/jiaa224">Interferon-Induced Transmembrane Protein 3 Genetic Variant rs12252-C Associated With Disease Severity in Coronavirus Disease 2019</a>. The Journal of Infectious Diseases 222:34–37.</div>
</div>
<div id="ref-EntTrBUW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">436. </div><div class="csl-right-inline">Zhang Y-H, Zhao Y, Li N, Peng Y-C, Giannoulatou E, Jin R-H, Yan H-P, Wu H, Liu J-H, Liu N, Wang D-Y, Shu Y-L, Ho L-P, Kellam P, McMichael A, Dong T. 2013. <a href="https://doi.org/10.1038/ncomms2433">Interferon-induced transmembrane protein-3 genetic variant rs12252-C is associated with severe influenza in Chinese individuals</a>. Nat Commun 4.</div>
</div>
<div id="ref-96K1aPWA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">437. </div><div class="csl-right-inline">Kim Y-C, Jeong B-H. 2020. <a href="https://doi.org/10.1007/s00251-020-01188-0">Ethnic variation in risk genotypes based on single nucleotide polymorphisms (SNPs) of the interferon-inducible transmembrane 3 (IFITM3) gene, a susceptibility factor for pandemic 2009 H1N1 influenza A virus</a>. Immunogenetics 72:447–453.</div>
</div>
<div id="ref-LOaWooow" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">438. </div><div class="csl-right-inline">Kim Y-C, Jeong B-H. 2020. <a href="https://doi.org/10.3390/genes12010042">Strong Correlation between the Case Fatality Rate of COVID-19 and the rs6598045 Single Nucleotide Polymorphism (SNP) of the Interferon-Induced Transmembrane Protein 3 (IFITM3) Gene at the Population-Level</a>. Genes 12:42.</div>
</div>
<div id="ref-HpvliVXv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">439. </div><div class="csl-right-inline">Nikoloudis D, Kountouras D, Hiona A. 2020. <a href="https://doi.org/10.7717/peerj.10402">The frequency of combined IFITM3 haplotype involving the reference alleles of both rs12252 and rs34481144 is in line with COVID-19 standardized mortality ratio of ethnic groups in England</a>. PeerJ 8:e10402.</div>
</div>
<div id="ref-QVZpcDYW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">440. </div><div class="csl-right-inline">Shi G, Kenney AD, Kudryashova E, Zani A, Zhang L, Lai KK, Hall‐Stoodley L, Robinson RT, Kudryashov DS, Compton AA, Yount JS. 2020. <a href="https://doi.org/10.15252/embj.2020106501">Opposing activities of IFITM proteins in SARS‐CoV‐2 infection</a>. EMBO J 40.</div>
</div>
<div id="ref-DQgMzjnS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">441. </div><div class="csl-right-inline">Zheng M, Zhao X, Zheng S, Chen D, Du P, Li X, Jiang D, Guo J-T, Zeng H, Lin H. 2020. <a href="https://doi.org/10.1080/22221751.2020.1787797">Bat SARS-Like WIV1 coronavirus uses the ACE2 of multiple animal species as receptor and evades IFITM3 restriction &lt;i&gt;via&lt;/i&gt; TMPRSS2 activation of membrane fusion</a>. Emerging Microbes &amp;amp; Infections 9:1567–1579.</div>
</div>
<div id="ref-YPiPALQD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">442. </div><div class="csl-right-inline">Tabor HK, Risch NJ, Myers RM. 2002. <a href="https://doi.org/10.1038/nrg796">Candidate-gene approaches for studying complex genetic traits: practical considerations</a>. Nat Rev Genet 3:391–397.</div>
</div>
<div id="ref-1GS6s8mgC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">443. </div><div class="csl-right-inline">, Niemi MEK, Karjalainen J, Liao RG, Neale BM, Daly M, Ganna A, Pathak GA, Andrews SJ, Kanai M, Veerapen K, Fernandez-Cadenas I, Schulte EC, Striano P, Marttila M, Minica C, Marouli E, Karim MA, Wendt FR, Savage J, Sloofman L, Butler-Laporte G, Kim H-N, Kanoni S, Okada Y, Byun J, Han Y, Uddin MJ, Smith GD, Willer CJ, Buxbaum JD, Mehtonen J, Finucane H, Cordioli M, Martin AR, Zhou W, Pasaniuc B, Julienne H, Aschard H, Shi H, Yengo L, Polimanti R, Ghoussaini M, Schwartzentruber J, Dunham I, Chwialkowska K, Francescatto M, Trankiem A, Balaconis MK, Davis L, Lee S, Priest J, Renieri A, Sankaran VG, van Heel D, Deelen P, Richards JB, Nakanishi T, Biesecker L, Kerchberger VE, Baillie JK, Mari F, Bernasconi A, Baillie SC, Canakoglu A, Wolford B, Faucon A, Dutta AK, Schurmann C, Harry E, Birney E, Nguyen H, Nasir J, Kaunisto M, Solomonson M, Dueker N, Vadgama N, Limou S, Rahmouni S, Mbarek H, Darwish D, Uddin MM, Albertos R, Pérez-Tur J, Li R, Folkersen L, Moltke I, Koelling N, Teumer A, Kousathanas A, Utrilla A, Verdugo RA, Zárate R, Medina-Gómez C, Gómez-Cabrero D, Carnero-Montoro E, Cadilla CL, Moreno-Estrada A, Garmendia A, Moya L, Sedaghati-Khayat B, Boua PR, Favé M-J, Francioli L, Lemaçon A, Migeotte I, Patel S, Varnai R, Szentpeteri JL, Sipeky C, Colombo F, von Hohenstaufen K, Lio P, Vallerga C, Wang Q, Tanigawa Y, Im H, Han C, Song H, Lim J, Lee Y, Kim S, Im S, Atanasovska B, Ahmad HF, Boer C, Jansen P, Franke L, Kaja E, Pasko D, Kennis-Szilagyi I, Kornilov SA, Prijatelj V, Prokić I, Sivanadhan I, Perumal S, Esmaeeli S, Pearson NM, Auton A, Shelton JF, Shastri AJ, Filshtein-Sonmez T, Coker D, Symons A, Esparza-Gordillo J, Aslibekyan S, O’Connell J, Ye C, Weldon CH, Perera M, O’Leary K, Tuck M, O’Brien T, Meltzer D, O’Donnell P, Nutescu E, Yang G, Alarcon C, Herrmann S, Mazurek S, Banagan J, Hamidi Z, Barbour A, Raffat N, Moreno D, Friedman P, Ferwerda B, van de Beek D, Brouwer MC, Vlaar APJ, Wiersinga WJ, Posthuma D, Tissink E, Koos Zwinderman AH, Uffelmann E, van Agtmael M, Algera AG, van Baarle F, Bax D, Beudel M, Jan Bogaard H, Bomers M, Bonta PI, Bos L, Botta M, de Brabander J, de Bree G, de Bruin S, Bugiani M, Bulle E, Chouchane O, Cloherty A, Dongelmans D, Elbers P, Fleuren L, Geerlings S, Geerts B, Geijtenbeek T, Girbes A, Goorhuis B, Grobusch MP, Hafkamp F, Hagens L, Hamann J, Harris V, Hemke R, Hermans SM, Heunks L, Hollmann M, Horn J, Hovius JW, de Jong MD, Koning R, van Mourik N, Nellen J, Nossent EJ, Paulus F, Peters E, van der Poll T, Preckel B, Prins JM, Raasveld J, Reijnders T, Schinkel M, Schultz MJ, Schuurman A, Sigaloff K, Smit M, Stijnis CS, Stilma W, Teunissen C, Thoral P, Tsonas A, van der Valk M, Veelo D, de Vries H, van Vugt M, Wouters D, Minnaar RP, Kromhout A, van Uffelen KWJ, Wolterman RA, Roberts G, Park D, Ball CA, Coignet M, McCurdy S, Knight S, Partha R, Rhead B, Zhang M, Berkowitz N, Gaddis M, Noto K, Ruiz L, Pavlovic M, Hong EL, Rand K, Girshick A, Guturu H, Baltzell AH, Rahmouni S, Guntz J, Beguin Y, Pigazzini S, Nkambule L, Bouysran Y, Busson A, Peyrassol X, Wilkin F, Pichon B, Smits G, Vandernoot I, Goffard J-C, Georges M, Moutschen M, Misset B, Darcis G, Guiot J, Jadot L, Azarzar S, Dellot P, Gofflot S, Claassen S, Bertrand A, Parzibut G, Clarinval M, Moermans C, Malaise O, El Kandoussi K, Thonon R, Huynen P, Mesdagh A, Melo S, Jacques N, Di Valentin E, Giroule F, Collignon A, Radermecker C, Lebrun M, Perée H, Latour S, Barada O, Sanchez J, Josse C, Boujemla B, Meunier M, Mariavelle E, Anania S, Gazon H, Juszczak D, Fadeur M, Camby S, Meuris C, Thys M, Jacques J, Henket M, Léonard P, Frippiat F, Giot J-B, Sauvage A-S, Von Frenckell C, Mni M, Wéry M, Staderoli A, Belhaj Y, Lambermont B, Morrison DR, Mooser V, Forgetta V, Li R, Ghosh B, Laurent L, Belisle A, Henry D, Abdullah T, Adeleye O, Mamlouk N, Kimchi N, Afrasiabi Z, Rezk N, Vulesevic B, Bouab M, Guzman C, Petitjean L, Tselios C, Xue X, Afilalo J, Afilalo M, Oliveira M, Brenner B, Brassard N, Durand M, Schurr E, Lepage P, Ragoussis J, Auld D, Chassé M, Kaufmann DE, Lathrop GM, Adra D, Davis LK, Cox NJ, Below JE, Sealock JM, Faucon AB, Shuey MM, Polikowsky HG, Petty LE, Shaw DM, Chen H-H, Zhu W, Ludwig KU, Schröder J, Maj C, Rolker S, Nöthen MM, Fazaal J, Keitel V, Jensen B-EO, Feldt T, Kurth I, Marx N, Dreher M, Pink I, Cornberg M, Illig T, Lehmann C, Schommers P, Augustin M, Rybniker J, Knopp L, Eggermann T, Volland S, Altmüller J, Berger MM, Brenner T, Hinney A, Witzke O, Bals R, Herr C, Ludwig N, Walter J, Fuchsberger C, Pattaro C, De Grandi A, Pramstaller P, Emmert D, Melotti R, Foco L, Mascalzoni D, Gögele M, Domingues F, Hicks A, Gignoux CR, Wicks SJ, Crooks K, Barnes KC, Daya M, Shortt J, Rafaels N, Chavan S, Goldstein DB, Kiryluk K, Sengupta S, Chung W, Reilly MP, Khan A, Wang C, Povysil G, Bhardwaj N, Gharavi AG, Ionita-Laza I, Shang N, O’Byrne SM, Nandakumar R, Menon A, So YS, Hod E, Pendrick D, Kim H-N, Park S-K, Kim H-L, Kang CK, Lee H-J, Song K-H, Jae Yoon K, Paik N-J, Seok W, Yoon H, Joo E-J, Chang Y, Ryu S, Park WB, Su Park J, Un Park K, Ham SY, Jung J, Kim ES, Kim HB, Ellinghaus D, Degenhardt F, Cáceres M, Juzenas S, Lenz TL, Albillos A, Julià A, Heidecker B, Garcia F, Kurth F, Tran F, Hanses F, Zoller H, Holter JC, Fernández J, Sander LE, Rosenstiel P, Koehler P, de Cid R, Asselta R, Schreiber S, Hehr U, Prati D, Baselli G, Valenti L, Bujanda L, Banales JM, Duga S, D’Amato M, Romero-Gómez M, Buti M, Invernizzi P, Franke A, Hov JR, Karlsen TH, Folseraas T, Maya-Miles D, Teles A, Azuure C, Wacker EM, Uellendahl-Werth F, ElAbd H, Arora J, Lerga-Jaso J, Wienbrandt L, Rühlemann MC, Wendorff M, Vadla MS, Lenning OB, Özer O, Myhre R, Raychaudhuri S, Tanck A, Gassner C, Hemmrich-Stanisak G, Kässens J, Figuera Basso ME, Schulzky M, Wittig M, Braun N, Wesse T, Albrecht W, Yi X, Ortiz AB, Chercoles AG, Ruiz A, Mantovani A, Holten AR, Mayer A, Cherubini A, Protti A, Aghemo A, Gerussi A, Ramirez A, Braun A, Barreira A, Lleo A, Kildal AB, Glück A, Nolla AC, Latiano A, Dyrhol-Riise AM, Muscatello A, Voza A, Rando-Segura A, Solier A, Karina B, Cortes B, Mateos B, Nafria-Jimenez B, Schaefer B, Bellinghausen C, Ferrando C, Quereda C, Skurk C, Thibeault C, Spinner CD, Lange C, Hu C, Cappadona C, Bianco C, Sancho C, Lihaug Hoff DA, Galimberti D, Jiménez D, Pestaña D, Toapanta D, Azzolini E, Scarpini E, Helbig ET, Urrechaga E, Paraboschi EM, Pontali E, Reverter E, Navas E, Arana E, Sánchez FG, Ceriotti F, Malvestiti F, Mesonero F, Pezzoli G, Lamorte G, Neb H, My I, Hernández I, de Rojas I, Galván-Femenia I, Heyckendorf J, Rybniker J, Badia JR, Schneider J, Goikoetxea J, Kraft J, Müller KE, Gaede KI, Garcia-Etxebarria K, Tonby K, Heggelund L, Izquierdo-Sanchez L, Sumoy L, Lippert LJ, Terranova L, Garbarino L, Téllez L, Roade L, Ostadreza M, Intxausti M, Kogevinas M, Gutiérrez-Stampa MA, Vehreschild MJGT, Marquié M, Castoldi M, Cecconi M, Boada M, Seilmaier MJ, Mazzocco M, Rodríguez-Gandía M, Ayo NI, Blay N, Martínez N, Cornely OA, Palmieri O, Tentorio P, Rodrigues PM, España PP, Hoffmann P, Bacher P, Suwalski P, de Pablo R, Nieto R, Badalamenti S, Ciesek S, Bombace S, Wilfling S, Brunak S, Heilmann-Heimbach S, Ripke S, Bahmer T, Landmesser U, Protzer U, Rimoldi V, Skogen V, Andrade V, Moreno V, Poller W, Farre X, Wang X, Khodamoradi Y, Karadeniz Z, de Salazar A, Palom A, Garcia-Fernandez A-E, Blanco-Grau A, Zanella A, Bandera A, Nebel A, Biondi A, Caballero-Garralda A, Gori A, Lind A, Fracanzani AL, Peschuck A, Pesenti A, de la Horra C, Milani C, Paccapelo C, Angelini C, Cea C, Muñiz-Diaz E, Sandoval E, Calderón EJ, Solligård E, Aziz F, Martinelli-Boneschi F, Peyvandi F, Blasi F, Medrano FJ, Rodriguez-Frias F, Müller F, Grasselli G, Costantino G, Cardamone G, Foti G, Matullo G, Kurihara H, Afset JE, Damås JK, Ampuero J, Martín J, Erdmann J, Bergan J, Goerg S, Ferrusquía-Acosta J, Quero JH, Delgado J, Guerrero JM, Risnes K, Bettini LR, Moreira L, Gustad LT, Santoro L, Scudeller L, Riveiro-Barciela M, Schaefer M, Carrabba M, Valsecchi MG, Hernandez-Tejero M, Acosta-Herrera M, D’Angiò M, Baldini M, Cazzaniga M, Ciccarelli M, Bocciolone M, Miozzo M, Chueca N, Montano N, Faverio P, Preatoni P, Bonfanti P, Omodei P, Castro P, Ferrer R, Gualtierotti R, Gallego-Durán R, Morilla R, Haider S, Marsal S, Aneli S, Pelusi S, Bosari S, Aliberti S, Dudman S, Zheng T, Pumarola T, Cejudo TG, Monzani V, Friaza V, Peter W, Dopazo X, Duga S, May S, Grimsrud MM, Gudbjartsson DF, Stefansson K, Sulem P, Sveinbjornsson G, Melsted P, Norddahl G, Swerford Moore KH, Thorsteinsdottir U, Holm H, Alarcón-Riquelme ME, Bernardo D, Martínez-Bueno M, Rello SR, Magi R, Milani L, Metspalu A, Laisk T, Läll K, Lepamets M, Esko T, Reimann E, Naaber P, Laane E, Pesukova J, Peterson P, Kisand K, Tabri J, Allos R, Hensen K, Starkopf J, Ringmets I, Tamm A, Kallaste A, Alavere H, Metsalu K, Puusepp M, Kristiansson K, Koskelainen S, Perola M, Donner K, Kivinen K, Palotie A, Palotie A, Rivolta C, Bochud P-Y, Bibert S, Boillat N, Nussle SG, Albrich W, Quinodoz M, Kamdar D, Suh N, Neofytos D, Erard V, Voide C, Bochud PY, Rivolta C, Bibert S, Quinodoz M, Kamdar D, Neofytos D, Erard V, Voide C, Friolet R, Vollenweider P, Pagani JL, Oddo M, zu Bentrup FM, Conen A, Clerc O, Marchetti O, Guillet A, Guyat-Jacques C, Foucras S, Rime M, Chassot J, Jaquet M, Viollet RM, Lannepoudenx Y, Portopena L, Desgranges F, Filippidis P, Guéry B, Haefliger D, Kampouri EE, Manuel O, Munting A, Papadimitriou-Olivgeris M, Regina J, Rochat-Stettler L, Suttels V, Tadini E, Tschopp J, Van Singer M, Viala B, Boillat-Blanco N, Brahier T, Hügli O, Meuwly JY, Pantet O, Nussle SG, Bochud M, D’Acremont V, Younes SE, Albrich WC, Suh N, Cerny A, O’Mahony L, von Mering C, Bochud PY, Frischknecht M, Kleger G-R, Filipovic M, Kahlert CR, Wozniak H, Negro TR, Pugin J, Bouras K, Knapp C, Egger T, Perret A, Montillier P, di Bartolomeo C, Barda B, de Cid R, Carreras A, Moreno V, Galván-Femenía I, Blay N, Farré X, Sumoy L, Cortés B, Mercader JM, Guindo-Martinez M, Torrents D, Garcia-Aymerich J, Castaño-Vinyals G, Dobaño C, Gori M, Renieri A, Mondelli MU, Castelli F, Vaghi M, Rusconi S, Montagnani F, Bargagli E, Franchi F, Mazzei MA, Cantarini L, Tacconi D, Feri M, Scala R, Spargi G, Nencioni C, Bandini M, Caldarelli GP, Spagnesi M, Canaccini A, Ognibene A, D’Arminio Monforte A, Girardis M, Antinori A, Francisci D, Schiaroli E, Scotton PG, Panese S, Scaggiante R, Monica MD, Capasso M, Fiorentino G, Castori M, Aucella F, Di Biagio A, Masucci L, Valente S, Mandalà M, Zucchi P, Giannattasio F, Coviello DA, Mussini C, Bosio G, Tavecchia L, Crotti L, Rizzi M, La Rovere MT, Sarzi-Braga S, Bussotti M, Ravaglia S, Artuso R, Perrella A, Romani D, Bergomi P, Catena E, Vincenti A, Ferri C, Grassi D, Pessina G, Tumbarello M, Di Pietro M, Sabrina R, Luchi S, Barbieri C, Acquilini D, Andreucci E, Paciosi F, Segala FV, Tiseo G, Falcone M, Lista M, Poscente M, De Vivo O, Petrocelli P, Guarnaccia A, Baroni S, Perticaroli V, Furini S, Dei S, Benetti E, Picchiotti N, Sanarico M, Ceri S, Pinoli P, Raimondi F, Biscarini F, Stella A, Bergomi M, Zguro K, Capitani K, Tanfoni M, Fallerini C, Daga S, Baldassarri M, Fava F, Frullanti E, Valentino F, Doddato G, Giliberti A, Tita R, Amitrano S, Bruttini M, Croci S, Meloni I, Mencarelli MA, Lo Rizzo C, Pinto AM, Beligni G, Tommasi A, Di Sarno L, Palmieri M, Carriero ML, Alaverdian D, Iuso N, Inchingolo G, Busani S, Bruno R, Vecchia M, Belli MA, Mantovani S, Ludovisi S, Quiros-Roldan E, Antoni MD, Zanella I, Siano M, Emiliozzi A, Fabbiani M, Rossetti B, Zanelli G, Bergantini L, D’Alessandro M, Cameli P, Bennet D, Anedda F, Marcantonio S, Scolletta S, Guerrini S, Conticini E, Frediani B, Spertilli C, Donati A, Guidelli L, Corridi M, Croci L, Piacentini P, Desanctis E, Cappelli S, Verzuri A, Anemoli V, Pancrazi A, Lorubbio M, Merlini E, Miraglia FG, Venturelli S, Cossarizza A, Vergori A, Gabrieli A, Riva A, Paciosi F, Andretta F, Gatti F, Parisi SG, Baratti S, Piscopo C, Russo R, Andolfo I, Iolascon A, Carella M, Merla G, Squeo GM, Raggi P, Marciano C, Perna R, Bassetti M, Sanguinetti M, Giorli A, Salerni L, Parravicini P, Menatti E, Trotta T, Coiro G, Lena F, Martinelli E, Mancarella S, Gabbi C, Maggiolo F, Ripamonti D, Bachetti T, Suardi C, Parati G, Bottà G, Di Domenico P, Rancan I, Bianchi F, Colombo R, van Heel DA, Hunt KA, Trembath RC, Huang QQ, Martin HC, Mason D, Trivedi B, Wright J, Finer S, Griffiths CJ, Akhtar S, Anwar M, Arciero E, Ashraf S, Breen G, Chung R, Curtis CJ, Chowdhury M, Colligan G, Deloukas P, Durham C, Finer S, Griffiths C, Huang QQ, Hurles M, Hunt KA, Hussain S, Islam K, Khan A, Khan A, Lavery C, Lee SH, Lerner R, MacArthur D, MacLaughlin B, Martin H, Mason D, Miah S, Newman B, Safa N, Tahmasebi F, Trembath RC, Trivedi B, van Heel DA, Wright J, Smith AV, Boughton AP, Li KW, LeFaive J, Annis A, Jannes CE, Krieger JE, Pereira AC, Velho M, Marques E, Lima IR, Tada MT, Valino K, McCarthy M, Rosenberger C, Lee JE, Chang D, Hammer C, Hunkapiller J, Mahajan A, Pendergrass S, Sucheston-Campbell L, Yaspan B, Lee HS, Shin E, Jang HY, Kim S, Kym S, Kim Y-S, Jeong H, Kwon KT, Kim S-W, Kim JY, Jang YR, Kim H ah, Lee JY, Lee JE, Lee S, Choe K-W, Kang YM, Jee SH, Jung KJ, Parikh V, Ashley E, Wheeler M, Rivas M, Bustamante C, Pinksy B, Febbo P, Farh K, Schroth GP, deSouza F, Dalton K, Christle J, Deboever C, Szalma S, Tanigawa Y, Rubinacci S, Delaneau O, Gorzynski J, de Jong H, Sutton S, Youlton N, Joshi R, Jimenez-Morales D, Hughes C, Amar D, Ioannidis A, Hershman S, Kirillova A, Seo K, Huang Y, Shoura M, Hammond N, Watson N, Raja A, Huang C, Sahoo M, Wang H, Zhen J, Rakitko A, Ilinsky V, Yermakovich D, Popov I, Chernitsov A, Kovalenko E, Krasnenko A, Plotnikov N, Stetsenko I, Kim A, Cirulli ET, Schiabor Barrett KM, Bolze A, White S, Washington NL, Lu JT, Riffle S, Tanudjaja F, Wang X, Ramirez JM III, Leonetti N, Sandoval E, Neveux I, Dabe S, Grzymski JJ, Esteban Miñano JI, Aguirre LA, López-Collazo E, de la Mata Pazos M, Cerrato L, Folkersen L, Lozano-Rodríguez R, Avendaño-Ortiz J, Arcos VT, Montalbán-Hernández KM, Quiroga JV, Pascual-Iglesias A, Maroun-Eid C, Martín-Quirós A, Namkoong H, Okada Y, Imoto S, Katayama K, Fukunaga K, Kitagawa Y, Sato T, Hasegawa N, Kumanogoh A, Kimura A, Ai M, Tokunaga K, Kanai T, Miyano S, Ogawa S, Edahiro R, Sonehara K, Shirai Y, Kanai M, Ishii M, Kabata H, Masaki K, Kamata H, Ikemura S, Chubachi S, Okamori S, Terai H, Tanaka H, Morita A, Lee H, Asakura T, Sasaki J, Morisaki H, Uwamino Y, Nanki K, Mikami Y, Tomono K, Kato K, Matsuda F, Takahashi M, Hizawa N, Takeda Y, Hirata H, Shiroyama T, Miyawaki S, Suzuki K, Maeda Y, Nii T, Noda Y, Niitsu T, Adachi Y, Enomoto T, Amiya S, Hara R, Takahashi K, Anzai T, Hasegawa T, Ito S, Koike R, Endo A, Uchimura Y, Miyazaki Y, Honda T, Tateishi T, Tohda S, Ichimura N, Sonobe K, Sassa C, Nakajima J, Nannya Y, Omae Y, Takahashi K, Harada N, Hiki M, Takagi H, Nakamura A, Tagaya E, Kawana M, Arimura K, Ishiguro T, Takayanagi N, Isono T, Takaku Y, Takano K, Anan R, Nakajima Y, Nakano Y, Nishio K, Ueda S, Hayashi R, Tateno H, Hase I, Yoshida S, Suzuki S, Mitamura K, Saito F, Ueda T, Azuma M, Nagasaki T, Yasui Y, Hasegawa Y, Mutoh Y, Yoshiyama T, Shoko T, Kojima M, Adachi T, Ishikawa M, Takahashi K, Watanabe K, Manabe T, Ito F, Fukui T, Funatsu Y, Koh H, Hirai Y, Kawashima H, Narita A, Niwa K, Sekikawa Y, Saito F, Yoshiya K, Yoshihara T, Suzuki Y, Nakayama S, Masuzawa K, Nishi K, Nishitsuji M, Tani M, Inoue T, Hirano T, Kobayashi K, Miyazawa N, Kimura Y, Sado R, Ogura T, Kitamura H, Murohashi K, Nakachi I, Baba R, Arai D, Fuke S, Saito H, Kuwahara N, Fujiwara A, Okada T, Baba T, Noda J, Mashimo S, Yagi K, Shiomi T, Hashiguchi M, Odani T, Mochimaru T, Oyamada Y, Mori N, Izumi N, Nagata K, Taki R, Murakami K, Yamada M, Sugiura H, Hayashi K, Shimizu T, Gon Y, Fujitani S, Tsuchida T, Yoshida T, Kagaya T, Kita T, Sakagami S, Kimizuka Y, Kawana A, Nakamura Y, Ishikura H, Takata T, Kikuchi T, Taniyama D, Nakamura M, Kodama N, Kaneyama Y, Maeda S, Nagasaki Y, Okamoto M, Ishihara S, Ito A, Chihara Y, Takeuchi M, Onoi K, Hashimoto N, Wakahara K, Ando A, Masuda M, Wakabayashi A, Watanabe H, Sageshima H, Nakada T-A, Abe R, Shimada T, Kawamura K, Ichikado K, Nishiyama K, Yamasaki M, Hashimoto S, Kusaka Y, Ohba T, Isogai S, Takada M, Kanda H, Komase Y, Sano F, Asano K, Oguma T, Harada M, Takahashi T, Shibusawa T, Abe S, Kono Y, Togashi Y, Izumo T, Inomata M, Awano N, Ogawa S, Ogata T, Ishihara S, Kanehiro A, Ozaki S, Fuchimoto Y, Kitagawa Y, Yoshida S, Ogura S, Nishiyama K, Yoshida K, Beppu S, Fukuyama S, Eriguchi Y, Yonekawa A, Inoue Y, Yamagata K, Chiba S, Narumoto O, Nagai H, Ooshima N, Motegi M, Sagara H, Tanaka A, Ohta S, Shibata Y, Tanino Y, Sato Y, Yamada Y, Hashino T, Shinoki M, Iwagoe H, Imamura T, Umeda A, Shimada H, Endo M, Hayashi S, Takahashi M, Nakano S, Yatomi M, Maeno T, Ishii T, Utsugi M, Ono A, Kanaoka K, Ihara S, Komuta K, Franke L, Boezen M, Claringbould A, Lopera E, Warmerdam R, Vonk JudithM, van Blokland I, Lanting P, Ori APS, Obeidat M, Hernández Cordero AI, Sin DD, Bossé Y, Joubert P, Hao K, Nickle D, Timens W, van den Berge M, Feng Y-CA, Mercader J, Weiss ST, Karlson EW, Smoller JW, Murphy SN, Meigs JB, Woolley AE, Green RC, Perez EF, Wolford B, Zöllner S, Wang J, Beck A, Sloofman LG, Ascolillo S, Sebra RP, Collins BL, Levy T, Sealfon SC, Jordan DM, Thompson RC, Gettler K, Chaudhary K, Belbin GM, Preuss M, Hoggart C, Choi S, Underwood SJ, Salib I, Britvan B, Keller K, Tang L, Peruggia M, Hiester LL, Niblo K, Aksentijevich A, Labkowsky A, Karp A, Zlatopolsky M, Zyndorf M, Charney AW, Beckmann ND, Schadt EE, Abul-Husn NS, Cho JH, Itan Y, Kenny EE, Loos RJF, Nadkarni GN, Do R, O’Reilly P, Huckins LM, Ferreira MAR, Abecasis GR, Leader JB, Cantor MN, Justice AE, Carey DJ, Chittoor G, Josyula NS, Kosmicki JA, Horowitz JE, Baras A, Gass MC, Yadav A, Mirshahi T, Jan Hottenga J, Bartels M, de Geus EJC, Nivard MG, Verma A, Ritchie MD, Rader D, Li B, Verma SS, Lucas A, Bradford Y, Zara F, Salpietro V, Scala M, Iacomino M, Scudieri P, Bocciardi R, Minetti C, Riva A, Vari MS, Rahier J-F, Giorgio E, Carli D, Louis E, Bulik CM, Landén M, Brusco A, Ferrero GB, Madia F, Fundín B, Ismail SI, Saad C, Al-Sarraj Y, Badji RM, Al-Muftah W, Al Thani A, Afifi N, Klovins J, Rovite V, Rescenko R, Peculis R, Ustinova M, Zeberg H, Frithiof R, Hultström M, Lipcsey M, Johnson R, Geschwind DH, Freimer N, Butte MJ, Geschwind DH, Pasaniuc B, Ding Y, Chiu A, Chang TS, Boutros P, Moutsianas L, Caulfield MJ, Scott RH, Walker S, Stuckey A, Odhams CA, Rhodes D, Fowler T, Rendon A, Chan G, Arumugam P, Karczewski KJ, Wilson DJ, Spencer CA, Crook DW, Wyllie DH, O’Connell AM, Atkinson EG, Kanai M, Tsuo K, Baya N, Turley P, Gupta R, Walters RK, Palmer DS, Sarma G, Solomonson M, Cheng N, Lu W, Churchhouse C, Goldstein JI, King D, Seed C, Daly MJ, Neale BM, Bryant S, Satterstrom FK, Band G, Earle SG, Lin S-K, Arning N, Armstrong J, Rudkin JK, Callier S, Bryant S, Cusick C, Soranzo N, Zhao JH, Danesh J, Di Angelantonio E, Butterworth AS, Sun YV, Huffman JE, Cho K, O’Donnell CJ, Tsao P, Gaziano JM, Peloso G, Ho Y-L, Mian M, Scaggiante F, Chang X, Glessner JR, Hakonarson H, McGuigan PJ, Prockter Moore LS, Vizcaychipi MP, Hall K, Campbell A, Nichol A, Ward G, Page VJ, Semple MG, Adeniji K, Agranoff D, Agwuh K, Ail D, Aldera EL, Alegria A, Angus B, Ashish A, Atkinson D, Bari S, Barlow G, Barnass S, Barrett N, Bassford C, Basude S, Baxter D, Beadsworth M, Bernatoniene J, Berridge J, Best N, Bothma P, Chadwick D, Brittain-Long R, Bulteel N, Burden T, Burtenshaw A, Caruth V, Chambler D, Chee N, Child J, Chukkambotla S, Clark T, Collini P, Cosgrove C, Cupitt J, Cutino-Moguel M-T, Dark P, Dawson C, Dervisevic S, Donnison P, Douthwaite S, Drummond A, DuRand I, Dushianthan A, Dyer T, Evans C, Eziefula C, Fegan C, Finn A, Fullerton D, Garg S, Garg A, Gkrania-Klotsas E, Godden J, Goldsmith A, Graham C, Hardy E, Hartshorn S, Harvey D, Havalda P, Hawcutt DB, Hobrok M, Hodgson L, Hormis A, Jacobs M, Jain S, Jennings P, Kaliappan A, Kasipandian V, Kegg S, Kelsey M, Kendall J, Kerrison C, Kerslake I, Koch O, Koduri G, Koshy G, Laha S, Laird S, Larkin S, Leiner T, Lillie P, Limb J, Linnett V, Little J, Lyttle M, MacMahon M, MacNaughton E, Mankregod R, Masson H, Matovu E, McCullough K, McEwen R, Meda M, Mills GH, Minton J, Ward K, Mirfenderesky M, Mohandas K, Mok Q, Moon J, Moore E, Morgan P, Morris C, Mortimore K, Moses S, Mpenge M, Mulla R, Murphy M, Nagel M, Nagarajan T, Nelson M, O’Shea MK, Otahal I, Ostermann M, Pais M, Panchatsharam S, Papakonstantinou D, Paraiso H, Patel B, Pattison N, Pepperell J, Peters M, Phull M, Pintus S, Pooni JS, Post F, Price D, Prout R, Rae N, Reschreiter H, Reynolds T, Richardson N, Roberts M, Roberts D, Rose A, Rousseau G, Ryan B, Saluja T, Shah A, Shanmuga P, Sharma A, Shawcross A, Sizer J, Shankar-Hari M, Smith R, Snelson C, Spittle N, Staines N, Stambach T, Stewart R, Subudhi P, Szakmany T, Tatham K, Thomas J, Thompson C, Thompson R, Tridente A, Tupper-Carey D, Twagira M, Ustianowski A, Vallotton N, Vincent-Smith L, Visuvanathan S, Vuylsteke A, Waddy S, Wake R, Walden A, Welters I, Whitehouse T, Whittaker P, Whittington A, Papineni P, Wijesinghe M, Williams M, Wilson L, Cole S, Winchester S, Wiselka M, Wolverson A, Wooton DG, Workman A, Yates B, Young P, Beale R, Bretherick AD, Clohisey S, Fourman MH, Furniss J, Gountouna E, Grimes G, Haley C, Harrison D, Hayward C, Keating S, Klaric L, Klenerman P, Law A, Meynert AM, Millar J, Pairo-Castineira E, Parkinson N, Ponting CP, Porteous DJ, Rawlik K, Richmond A, Rowan K, Russell CD, Scott RH, Shen X, Shih B, Tenesa A, Vitart V, Wang B, Wilson JF, Wu Y, Yang J, Yang Z, Zechner M, Zhai R, Zheng C, Norman L, Pius R, Drake TM, Fairfield CJ, Knight SR, Mclean KA, Murphy D, Shaw CA, Dalton J, Girvan M, Saviciute E, Roberts S, Harrison J, Marsh L, Connor M, Halpin S, Jackson C, Gamble C, Leeming G, Law A, Wham M, Hendry R, Scott-Brown J, Begg C, Hinds C, Wai Ho AY, Horby PW, Knight J, Ling L, Maslove D, McAuley D, Montgomery H, Nichol A, Openshaw PJM, Semple MG, Shankar-Hari M, Summers C, Walsh T, Armstrong L, Bates H, Dooks E, Farquhar F, Hairsine B, McParland C, Packham S, Alldis Z, Astin-Chamberlain R, Bibi F, Biddle J, Blow S, Bolton M, Borra C, Bowles R, Burton M, Choudhury Y, Collier D, Cox A, Easthope A, Ebano P, Fotiadis S, Gurasashvili J, Halls R, Hartridge P, Kallon D, Kassam J, Lancoma-Malcolm I, Matharu M, May P, Mitchelmore O, Newman T, Patel M, Pheby J, Pinzuti I, Prime Z, Prysyazhna O, Shiel J, Taylor M, Tierney C, Wood S, Zak A, Zongo O, Forsey M, Kaliappan A, Nicholson A, Riches J, Vertue M, Wasson C, Finn S, Green J, Collins E, King B, Grauslyte L, Hussain M, Phull M, Pogreban T, Rosaroso L, Salciute E, Franke G, Wong J, George A, Akeroyd L, Bano S, Bromley M, Gurr L, Lawton T, Morgan J, Sellick K, Warren D, Wilkinson B, McGowan J, Ledgard C, Stacey A, Pye K, Bellwood R, Bentley M, Hobrok M, Loosley R, McGuinness H, Tench H, Wolf-Roberts R, Gibson S, Lyle A, McNeela F, Radhakrishnan J, Hughes A, Ali A, Brady M, Dale S, Dance A, Gledhill L, Greig J, Hanson K, Holdroyd K, Home M, Kelly D, Kitson R, Matapure L, Melia D, Mellor S, Nortcliffe T, Pinnell J, Robinson M, Shaw L, Shaw R, Thomis L, Wilson A, Wood T, Bayo L-A, Merwaha E, Ishaq T, Hanley S, Antcliffe D, Banach D, Brett S, Coghlan P, Fernandez Z, Gordon A, Rojo R, Arias SS, Templeton M, Jha R, Krishnamurthy V, Lim L, Bi R, Scholefield B, Ashton L, Williams A, Cheyne C, Saunderson A, Allan A, Anderson F, Kaye C, Liew J, Medhora J, Scott T, Trumper E, Botello A, Polgarova P, Stroud K, Meaney E, Jones M, Ng A, Agrawal S, Pathan N, White D, Daubney E, Elston K, Parker R, Reddy A, Turner-Bone I, Wilding L, Harding P, Jacob R, Jones C, Denmade C, Croft M, White I, Lim L, Griffin D, Muchenje N, Mupudzi M, Partridge R, Conyngham J-A, Thomas R, Wright M, Corral MA, Bastion V, Clarke D, David B, Kent H, Lorusso R, Lubimbi G, Murdoch S, Penacerrada M, Thomas A, Valentine J, Vochin A, Wulandari R, Djeugam B, Dawson J, Garrioch S, Tolson M, Aldridge J, de Almeida Martins LG, Carungcong J, Beavis S, Dale K, Gascoyne R, Hawes J, Pritchard K, Stevenson L, Whileman A, Cowley A, Highgate J, Crawley R, Crew A, Cunningham M, Daniels A, Harrison L, Hope S, Inweregbu K, Jones S, Lancaster N, Matthews J, Nicholson A, Wray G, Benham L, Bradshaw Z, Brown J, Caswell M, Cupitt J, Melling S, Preston S, Slawson N, Stoddard E, Warden S, Combes E, Joefield T, Monnery S, Beech V, Trotman S, Hopkins B, Scriven J, Thrasyvoulou L, Willis H, Anderson S, Birch J, Collins E, Hammerton K, O’Leary R, Abernathy C, Foster L, Gratrix A, Martinson V, Parkinson P, Stones E, Carbral-Ortega L, Kapoor R, Loader D, Castle K, Brandwood C, Smith L, Clark R, Birchall K, Kolakaluri L, Baines D, Sukumaran A, Mapfunde I, Meredith M, Morris L, Ryan L, Clark A, Sampson J, Peters C, Dent M, Langley M, Ashraf S, Wei S, Andrew A, Chablani M, Kirkby A, Netherton K, Bates M, Dasgin J, Gill J, Nilsson A, Scriven J, Apetri E, Basikolo C, Blackledge B, Catlow L, Charles B, Dark P, Doonan R, Harris J, Harvey A, Horner D, Knowles K, Lee S, Lomas D, Lyons C, Marsden T, McLaughlan D, McMorrow L, Pendlebury J, Perez J, Poulaka M, Proudfoot N, Slaughter M, Slevin K, Taylor M, Thomas V, Walker D, Michael A, Collis M, Clark M, Coulding M, Jude E, McCormick J, Mercer O, Potla D, Rehman H, Savill H, Turner V, Davey M, Golden D, Seaman R, Hunt J, Dearden J, Dobson E, Mulcahy M, Munt S, O’Connor G, Philbin J, Rishton C, Tully R, Winnard S, Cagova L, Fofano A, Garner L, Holcombe H, Mepham S, Mitchell AM, Mwaura L, Praman K, Vuylsteke A, Zamikula J, Bercades G, Brealey D, Hass I, MacCallum N, Martir G, Raith E, Reyes A, Smyth D, Taylor A, Hughes RA, Thomas H, Rees A, Duskova M, Phipps J, Brooks S, Edwards M, Alexander P, Allen S, Bradley-Potts J, Brantwood C, Egan J, Felton T, Padden G, Ward L, Moss S, Glasgow S, Beesley K, Board S, Kubisz-Pudelko A, Lewis A, Perry J, Pippard L, Wood D, Buckley C, Brown A, Gregory J, O’Connell S, Smith T, Belagodu Z, Fuller B, Gherman A, Olufuwa O, Paramsothy R, Stuart C, Oakley N, Kamundi C, Tyl D, Collins K, Silva P, Taylor J, King L, Coates C, Crowley M, Wakefield P, Beadle J, Johnson L, Sargeant J, Anderson M, Jardine C, Williams D, Parris V, Quaid S, Watson E, Melville J, Naisbitt J, Joseph R, Lazo M, Walton O, Neal A, Hill M, Kannan T, Wild L, Allan E, Darlington K, Davies F, Easton J, Kumar S, Lean R, Menzies D, Pugh R, Qiu X, Davies L, Williams H, Scanlon J, Davies G, Mackay C, Lewis J, Rees S, Coetzee S, Gales A, Otahal I, Raj M, Sell C, Langton H, Prout R, Watters M, Novis C, Arbane G, Bociek A, Campos S, Grau N, Jones TO, Lim R, Marotti M, Ostermann M, Shankar-Hari M, Whitton C, Barron A, Collins C, Kaul S, Passmore H, Prendergast C, Reed A, Rogers P, Shokkar R, Woodruff M, Middleton H, Polgar O, Nolan C, Thwaites V, Mahay K, Sri-Chandana C, Scherewode J, Stephenson L, Marsh S, Bancroft H, Bellamy M, Carmody M, Daglish J, Moore F, Rhodes J, Sangombe M, Kadiri S, Scriven J, Ayers A, Harrison W, North J, Cavazza A, Cockrell M, Corcoran E, Depante M, Finney C, Jerome E, McPhail M, Nayak M, Noble H, O’Reilly K, Pappa E, Saha R, Saha S, Smith J, Knighton A, Gill M, Paul P, Ratnam V, Shelton S, Wynter I, Baptista D, Crowe R, Fernandes R, Herdman-Grant R, Joseph A, Loveridge A, McKenley I, Morino E, Naranjo A, Simms R, Sollesta K, Swain A, Venkatesh H, Khera J, Fox J, Barber R, Hewitt C, Hilldrith A, Jackson-Lawrence K, Shepardson S, Wills M, Butler S, Tavares S, Cunningham A, Hindale J, Arif S, George L, Twiss S, Wright D, Holland M, Keenan N, Lyons M, Wassall H, Marsh C, Mahenthran M, Carter E, Kong T, Adanini O, Bhatia N, Msiska M, Mew L, Mwaura E, Stewart R, Williams F, Wren L, Sutherland S-B, Battle C, Brinkworth E, Harford R, Murphy C, Newey L, Rees T, Williams M, Arnold S, Brealey D, Hardy J, Houlden H, Moncur E, Raith E, Tariq A, Tucci A, Convery K, Fottrell-Gould D, Hudig L, Keshet-Price J, Randell G, Stammers K, Abdelrazik M, Bakthavatsalam D, Elhassan M, Ganesan A, Haldeos A, Moreno-Cuesta J, Purohit D, Vincent R, Xavier K, Rohit K, Alasdair F, Saleem M, David C, Jenkins S, Lamond Z, Wall A, Yates B, Reynolds J, Campbell H, Thompsom M, Dodds S, Duffy S, Butcher D, O’Sullivan S, Butterworth-Cowin N, Deacon B, Hibbert M, Pothecary C, Tetla D, Woodford C, Durga L, Kennard-Holden G, de Gordoa LO-R, Peasgood E, Phillips C, Skinner D, Gaylard J, Mullan D, Newman J, Davies E, Roche L, Sathe S, Brimfield L, Daly Z, Pogson D, Rose S, Collins A, Khaliq W, Gude ET, Allen L, Beranova E, Crisp N, Deery J, Hazelton T, Knight A, Price C, Tilbey S, Turki S, Turney S, Giles J, Booth S, Bell G, English K, Katary A, Wilcox L, Campbell R, Clarke N, Whiteside J, Mascarenhas M, Donaldson A, Matheson J, Barrett F, O’Hara M, O’Keefe L, Bradley C, Collier D, Hormis A, Walker R, Maynard V, Patel T, Smith M, Chukkambotla S, Kazi A, Hartley J, Dykes J, Hijazi M, Keith S, Khan M, Ryan-Smith J, Springle P, Thomas J, Truman N, Saad S, Coleman D, Fine C, Matt R, Gay B, Dalziel J, Ali S, Goodchild D, Harling R, Bhatterjee R, Goddard W, Davison C, Duberly S, Hargreaves J, Bolton R, Laha S, Verlander M, Williams A, Blackman H, Creagh-Brown B, Donlon S, Michalak-Glinska N, Mtuwa S, Pristopan V, Salberg A, Smith E, Stone S, Piercy C, Verula J, Burda D, Montaser R, Harden L, Mayangao I, Marriott C, Bradley P, Harris C, Cooper J, Finch C, Liderth S, Quinn A, Waddington N, Fidler K, Tagliavini E, Donnelly K, Abel L, Brett M, Digby B, Gemmell L, Hornsby J, MacGoey P, O’Neil P, Price R, Rodden N, Rooney K, Sundaram R, Thomson N, Flanagan R, Hughes G, Latham S, McKenna E, Anderson J, Hull R, Rhead K, Branney D, Frankham J, Pitts S, White N, Cristiano D, Dormand N, Farzad Z, Gummadi M, Liyanage K, Patel BV, Salmi S, Sloane G, Thwaites V, Varghese M, Zborowski AC, Bean S, Burt K, Spivey M, Eastgate-Jackson C, Filipe H, Martin D, Maharajh A, Garcia SM, De Neef M, Deacon B, Lynch C, Pothecary C, Roche L, Howe GS, Singh J, Turner K, Ellis H, Stroud N, Cherian S, Cutler S, Heron AE, Roynon-Reed A, Szakmany T, Williams G, Richards O, Cheema Y, Ahmad N, Barker J, Bauchmuller K, Bird S, Cawthron K, Harrington K, Jackson Y, Kibutu F, Lenagh B, Masuko S, Mills GH, Raithatha A, Wiles M, Willson J, Newell H, Lye A, Nwafor L, Jarman C, Rowland-Jones S, Foote D, Cole J, Thompson R, Watson J, Hesseldon L, Macharia I, Chetam L, Smith J, Ford A, Anderson S, Birchall K, Housley K, Walker S, Milner L, Hanratty H, Trower H, Phillips P, Oxspring S, Donne B, Bevan E, Martin J, Trodd D, Watson G, Brown CW, Bunni L, Jennings C, Latif M, Marshall R, Subramanian G, Bandla N, Gellamucho M, Davies M, Thompson C, Trim F, Eapen B, Ahmed C, Baines B, Clamp S, Colley J, Haq R, Hayes A, Hulme J, Hussain S, Joseph S, Kumar R, Maqsood Z, Purewal M, Chandler B, Elliott K, Mallinson J, Turnbull A, Dent K, Horsley E, Akhtar MN, Pearson S, Potoczna D, Spencer S, Blakemore H, Borislavova B, Faulkner B, Gendall E, Goff E, Hayes K, Thomas M, Worner R, Smith K, Stephens D, Delgado CC, Dawson D, Ding L, Durrant G, Ezeobu O, Farnell-Ward S, Harrison A, Kanu R, Leaver S, Maccacari E, Manna S, Saluzzio RP, Queiroz J, Samakomva T, Sicat C, Texeira J, Da Gloria EF, Lisboa A, Rawlins J, Mathew J, Kinch A, Hurt WJ, Shah N, Clark V, Thanasi M, Yun N, Patel K, Crickmore V, Debreceni G, Wilkins J, Nicol L, Burn I, Hambrook G, Manso K, Penn R, Shanmugasundaram P, Tebbutt J, Thornton D, Rostron A, Roy A, Woods L, Cornell S, Wakinshaw F, Rogerson K, Jarmain J, Anderson P, Archer K, Austin K, Davis C, Durie A, Kelsall O, Thrush J, Vigurs C, Wild L, Wood H-L, Tranter H, Harrison A, Cowley N, McAlindon M, Burtenshaw A, Digby S, Low E, Morgan A, Cother N, Rankin T, Clayton S, McCurdy A, Allibone S, Mary-Genetu R, Kasipandian V, Patel A, Mac A, Murphy A, Mahjoob P, Nazari R, Worsley L, Fagan A, Mohamed Ali IA, Beaumont K, Blunt M, Coton Z, Curgenven H, Elsaadany M, Fernandes K, Ally SM, Rangarajan H, Sarathy V, Selvanayagam S, Vedage D, White M, Fernandez-Roman J, Hamilton DO, Johnson E, Johnston B, Martinez ML, Mulla S, Shaw D, Waite AAC, Waugh V, Welters ID, Williams K, Bemand T, Black E, Rosa AD, Howle R, Jhanji S, Baikady RR, Tatham KC, Thomas B, Halkes M, Mercer P, Thornton L, West J, Baird T, Ruddy J, Reece-Anthony R, Birt M, Cowton A, Kay A, Kent M, Potts K, Wilkinson A, Naylor S, Brown E, Clark M, Purvis S, Cole J, Davies M, Davies R, Duffin D, Hill H, Player B, Thomas E, Williams A, Beith CM, Black K, Clements S, Morrison A, Strachan D, Taylor M, Clarkson M, D’Sylva S, Norman K, Coventry T, Fowler S, MacMahon M, McGregor A, Brady A, Chan R, Little J, McIvor S, Prady H, Whittle H, Mathew B, Clapham M, Harper R, Poultney U, Rice P, Smith T, Mutch R, Baird Y, Butler A, Chadbourn I, Folkes L, Fox H, Gardner A, Gomez R, Hobden G, Hodgson L, King K, Margarson M, Martindale T, Meadows E, Raynard D, Thirlwall Y, Helm D, Margalef J, Greer S, Shuker K, Tridente A, Smuts S, Duffield J, Smith O, Mallon L, Claire W, Birkinshaw I, Carter J, Howard K, Ingham J, Joy R, Pearson H, Roche S, Scott Z, Knights E, Price A, Thomas A, Thorpe C, Abraheem A, Bamford P, Cawley K, Dunmore C, Faulkner M, Girach R, Jeffrey H, Jones R, London E, Nagra I, Nasir F, Sainsbury H, Smedley C, Khade R, Sundar A, Tsinaslanidis G, Behan T, Burnett C, Hatton J, Heeney E, Mitra A, Newton M, Pollard R, Stead R, Birch J, Bough L, Goodsell J, Tutton R, Williams P, Williams S, Winter-Goodwin B, Auld F, Donnachie J, Edmond I, Prentice L, Runciman N, Salutous D, Symon L, Todd A, Turner P, Short A, Sweeney L, Murdoch E, Senaratne D, Burns K, Higham A, Anderson T, Hawcutt D, O’Malley L, Rad L, Rogers N, Saunderson P, Allison KS, Afolabi D, Whitbread J, Jones D, Dore R, Lankester L, Nikitas N, Wells C, Stowe B, Spencer K, Cathcart S, Duffy K, Puxty A, Puxty K, Turner L, Ireland J, Semple G, Barry P, Hilltout P, Evitts J, Tyler A, Waldron J, Irvine V, Shelley B, Akinkugbe O, Bamford A, Beech E, Belfield H, Bell M, Davies C, Jones GAL, McHugh T, Meghari H, O’Neill L, Peters MJ, Ray S, Tomas AL, Easthope A, Gorman C, Gupta A, Timlick E, Brady R, Bonner S, Hugill K, Jones J, Liggett S, Bashyal A, Davidson N, Hutton P, McKechnie S, Wilson J, Flint N, Rekha P, Hales D, Cruz C, Pattison N, Gopal S, Harris N, Lake V, Metherell S, Radford E, Clement I, Patel B, Gulati A, Hays C, Webster K, Hudson A, Webster A, Stephenson E, McCormack L, Slater V, Nixon R, Hanson H, Fearby M, Kelly S, Bridgett V, Robinson P, Almaden-Boyle C, Austin P, Cabrelli L, Cole S, Casey M, Chapman S, Whyte C, Brayne A, Fisher E, Hunt J, Jackson P, Kaye D, Love N, Parkin J, Tuckey V, van Koutrik L, Carter S, Andrew B, Findlay L, Adams K, Bruce M, Connolly K, Duncan T, T.-Michael H, Lindergard G, Hey S, Fox C, Alfonso J, Durrans LJ, Guerin J, Blackledge B, Harris J, Hruska M, Eltayeb A, Lamb T, Hodgkiss T, Cooper L, Rothwell J, Dennis C, McGregor A, Parris V, Srikaran S, Sukha A, Davies K, O’Brien L, Omar Z, Otahal I, Perkins E, Lewis T, Sutherland I, Brooke H, Buckley S, Suarez JC, Charlesworth R, Hansson K, Norris J, Poole A, Rose A, Sandhu R, Sloan B, Smithson E, Thirumaran M, Wagstaff V, Metcalfe A, Camsooksai J, Humphrey C, Jenkins S, Reschreiter H, Wadams B, DeAth Y, Adams C, Agasou A, Arden T, Bowes A, Boyle P, Beekes M, Button H, Capps N, Carnahan M, Carter A, Childs D, Donaldson D, Hard K, Hurford F, Hussain Y, Javaid A, Jones J, Jose S, Leigh M, Martin T, Millward H, Motherwell N, Rikunenko R, Stickley J, Summers J, Ting L, Tivenan H, Tonks L, Wilcox R, Bokhari M, Linnett V, Lucas R, McCormick W, Ritzema J, Sanderson A, Wild H, Baxter N, Henderson S, Kennedy-Hay S, McParland C, Rooney L, Sim M, McCreath G, Brunton M, Caterson J, Coles H, Frise M, Rai SG, Jacques N, Keating L, Tilney E, Bartley S, Bhuie P, Downes C, Holding K, Riches K, Hilton M, Hayman M, Subramanian D, Daniel P, Zitter L, Benyon S, Marriott S, Park L, Keenan S, Gordon E, Quinn H, Baines K, Andrew G, Baillie JK, Barclay L, Callaghan M, Campbell R, Clark S, Hope D, Marshall L, McCulloch C, Briton K, Singleton J, Birch S, Higham A, Simpson K, Craig J, Demetriou C, Eckbad C, Hierons S, Howie L, Mitchard S, Ramos L, Serrano-Ruiz A, White K, Kelly F, Amin V, Anastasescu E, Anumakonda V, Karthik K, Kausar R, Reid K, Smith J, Imeson-Wood J, Bellini A, Bryant J, Mayer A, Pickard A, Roe N, Sowter J, Howlett A, Criste K, Cusack R, Golder K, Golding H, Jones O, Leggett S, Male M, Marani M, Prager K, Williams T, Roberts B, Salmon K, Gondo P, Hadebe B, Kayani A, Masunda B, Ahmed A, Morris A, Jakkula S, Long K, Whiteley S, Wilby E, Ogg B, Moultrie S, Odam M, Bewley J, Garland Z, Grimmer L, Gumbrill B, Johnson R, Sweet K, Webster D, Efford G, Bennett S, Goodwin E, Jackson M, Kent A, Tibke C, Woodyatt W, Zaki A, Daniel A, Finn J, Saha R, Staines N, Easthope A, Bremmer P, Allan J, Geary T, Houston G, Meikle A, O’Brien P, Bell D, Boyle R, Douglas K, Glass L, Lee E, Lennon L, Rattray A, Charnock R, McFarland D, Cosgrove D, Attwood B, Parsons P, Carmody S, Oblak M, Popescu M, Thankachen M, Baruah R, Morris S, Ferguson S, Shepherd A, Altabaibeh A, Alvaro A, Gilbert K, Ma L, Mostoles L, Parmar C, Simpson K, Jetha C, Booker L, Pratley A, Cosier T, Millen G, Richardson N, Schumacher N, Weston H, Rand J, Alex B, Bach B, Barclay WS, Bogaert D, Chand M, Cooke GS, Docherty AB, Dunning J, da Silva Filipe A, Fletcher T, Green CA, Harrison EM, Hiscox JA, Ijaz S, Khoo S, Klenerman P, Lim WS, Mentzer AJ, Merson L, Noursadeghi M, Moore SC, Palmarini M, Paxton WA, Pollakis G, Price N, Rambaut A, Robertson DL, Russell CD, Sancho-Shimizu V, Scott JT, de Silva T, Sigfrid L, Solomon T, Sriskandan S, Stuart D, Tedder RS, Thomson EC, Roger Thompson AA, Thwaites RS, Turtle LCW, Gupta RK, Palmieri C, Swann OV, Zambon M, Dumas M-E, Griffin JL, Takats Z, Chechi K, Andrikopoulos P, Osagie A, Olanipekun M, Liggi S, Lewis MR, Correia G dos S, Sands CJ, Takis P, Maslen L, Greenhalf W, Shaw V, McDonald SE, Keating S, Ahmed KA, Armstrong JA, Ashworth M, Asiimwe IG, Bakshi S, Barlow SL, Booth L, Brennan B, Bullock K, Catterall BWA, Clark JJ, Clarke EA, Cooper L, Cox H, Davis C, Dincarslan O, Dunn C, Dyer P, Elliott A, Evans A, Finch L, Fisher LWS, Foster T, Garcia-Dorival I, Greenhalf W, Gunning P, Hartley C, Jensen RL, Jones CB, Jones TR, Khandaker S, King K, Kiy RT, Koukorava C, Lake A, Lant S, Latawiec D, Lavelle-Langham L, Lefteri D, Lett L, Livoti LA, Mancini M, McDonald S, McEvoy L, McLauchlan J, Metelmann S, Miah NS, Middleton J, Mitchell J, Moore SC, Murphy EG, Penrice-Randal R, Pilgrim J, Prince T, Reynolds W, Ridley PM, Sales D, Shaw VE, Shears RK, Small B, Subramaniam KS, Szemiel A, Taggart A, Tanianis-Hughes J, Thomas J, Trochu E, van Tonder L, Wilcock E, Zhang JE, Flaherty L, Maziere N, Cass E, Carracedo AD, Carlucci N, Holmes A, Massey H, Murphy L, Wrobel N, McCafferty S, Morrice K, MacLean A, Armstrong R, Boz C, Brown A, Clark R, Coutts A, Cullum L, Day N, Donnelly L, Duncan E, Fawkes A, Finernan P, Gilchrist T, Golightly A, Hafezi K, Law D, Law R, Law S, Macgillivray L, Maclean A, Mal H, McCafferty S, Mcmaster E, Meikle J, Moore SC, Morrice K, Murphy L, Oosthuyzen W, Paterson T, Stenhouse A, Swets M, Szoor-McElhinney H, Taneski F, Wackett T, Ward M, Weaver J, Wrobel N, Coyle J, Gallagher B, Lidstone-Scott R, Hamilton D, Schon K, Furlong A, Biggs H, Griffiths F, Andrews E, Brickell K, Smyth M, Murphy L, Carson G, Hardwick H, Donohue C. 2021. <a href="https://doi.org/10.1038/s41586-021-03767-x">Mapping the human genetic architecture of COVID-19</a>. Nature 600:472–477.</div>
</div>
<div id="ref-tBgc6ryL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">444. </div><div class="csl-right-inline">Erlich H. 2012. <a href="https://doi.org/10.1111/j.1399-0039.2012.01881.x">HLA DNA typing: past, present, and future</a>. Tissue Antigens 80:1–11.</div>
</div>
<div id="ref-17eP6PTvz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">445. </div><div class="csl-right-inline">Gourraud P-A, Khankhanian P, Cereb N, Yang SY, Feolo M, Maiers M, D. Rioux J, Hauser S, Oksenberg J. 2014. <a href="https://doi.org/10.1371/journal.pone.0097282">HLA Diversity in the 1000 Genomes Dataset</a>. PLoS ONE 9:e97282.</div>
</div>
<div id="ref-1CQ8X6yi2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">446. </div><div class="csl-right-inline">de Sousa E, Ligeiro D, Lérias JR, Zhang C, Agrati C, Osman M, El-Kafrawy SA, Azhar EI, Ippolito G, Wang F-S, Zumla A, Maeurer M. 2020. <a href="https://doi.org/10.1016/j.ijid.2020.07.016">Mortality in COVID-19 disease patients: Correlating the association of major histocompatibility complex (MHC) with severe acute respiratory syndrome 2 (SARS-CoV-2) variants</a>. International Journal of Infectious Diseases 98:454–459.</div>
</div>
<div id="ref-lxEvWysS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">447. </div><div class="csl-right-inline">Naemi FMA, Al‐adwani S, Al‐khatabi H, Al‐nazawi A. 2021. <a href="https://doi.org/10.1002/jmv.27003">Association between the HLA genotype and the severity of COVID‐19 infection among South Asians</a>. Journal of Medical Virology 93:4430–4437.</div>
</div>
<div id="ref-PyvspFUi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">448. </div><div class="csl-right-inline">Langton DJ, Bourke SC, Lie BA, Reiff G, Natu S, Darlay R, Burn J, Echevarria C. 2021. <a href="https://doi.org/10.1111/tan.14284">The influence of HLA genotype on the severity of COVID‐19 infection</a>. HLA 98:14–22.</div>
</div>
<div id="ref-1DebTO3uY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">449. </div><div class="csl-right-inline">Schindler E, Dribus M, Duffy BF, Hock K, Farnsworth CW, Gragert L, Liu C. 2021. <a href="https://doi.org/10.1111/tan.14387">&lt;scp&gt;HLA&lt;/scp&gt;
genetic polymorphism in patients with Coronavirus Disease 2019 in Midwestern United States</a>. HLA 98:370–379.</div>
</div>
<div id="ref-1H6m2HVG7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">450. </div><div class="csl-right-inline">Ben Shachar S, Barda N, Manor S, Israeli S, Dagan N, Carmi S, Balicer R, Zisser B, Louzoun Y. 2021. <a href="https://doi.org/10.1007/s10875-021-01071-x">MHC Haplotyping of SARS-CoV-2 Patients: HLA Subtypes Are Not Associated with the Presence and Severity of COVID-19 in the Israeli Population</a>. J Clin Immunol 41:1154–1161.</div>
</div>
<div id="ref-6pfIhCGV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">451. </div><div class="csl-right-inline">Rubin R. 2020. <a href="https://doi.org/10.1001/jama.2020.16516">Investigating Whether Blood Type Is Linked to COVID-19 Risk</a>. JAMA 324:1273.</div>
</div>
<div id="ref-1DbV444Pj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">452. </div><div class="csl-right-inline">Zhao J, Yang Y, Huang H, Li D, Gu D, Lu X, Zhang Z, Liu L, Liu T, Liu Y, He Y, Sun B, Wei M, Yang G, Wang X, Zhang L, Zhou X, Xing M, Wang PG. 2020. Relationship between the ABO Blood Group and the COVID-19 Susceptibility. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.11.20031096">10.1101/2020.03.11.20031096</a>.</div>
</div>
<div id="ref-1BQ7FxsTt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">453. </div><div class="csl-right-inline">Kerbage A, Haddad SF, Nasr L, Riachy A, Mekhael E, Nassim N, Hoyek K, Sleilaty G, Nasr F, Riachy M. 2021. <a href="https://doi.org/10.1002/jmv.27444">Impact of ABO and Rhesus blood groups on COVID‐19 susceptibility and severity: A case‐control study</a>. Journal of Medical Virology 94:1162–1166.</div>
</div>
<div id="ref-QCVLQvsf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">454. </div><div class="csl-right-inline">Fan Q, Zhang W, Li B, Li D-J, Zhang J, Zhao F. 2020. <a href="https://doi.org/10.3389/fcimb.2020.00404">Association Between ABO Blood Group System and COVID-19 Susceptibility in Wuhan</a>. Front Cell Infect Microbiol 10.</div>
</div>
<div id="ref-1Hm5TzRvT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">455. </div><div class="csl-right-inline">Hoiland RL, Fergusson NA, Mitra AR, Griesdale DEG, Devine DV, Stukas S, Cooper J, Thiara S, Foster D, Chen LYC, Lee AYY, Conway EM, Wellington CL, Sekhon MS. 2020. <a href="https://doi.org/10.1182/bloodadvances.2020002623">The association of ABO blood group with indices of disease severity and multiorgan dysfunction in COVID-19</a>. Blood Advances 4:4981–4989.</div>
</div>
<div id="ref-19DBCLqWZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">456. </div><div class="csl-right-inline">Anderson JL, May HT, Knight S, Bair TL, Muhlestein JB, Knowlton KU, Horne BD. 2021. <a href="https://doi.org/10.1001/jamanetworkopen.2021.7429">Association of Sociodemographic Factors and Blood Group Type With Risk of COVID-19 in a US Population</a>. JAMA Netw Open 4:e217429.</div>
</div>
<div id="ref-P5ITbYUZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">457. </div><div class="csl-right-inline">Zhang Y, Garner R, Salehi S, La Rocca M, Duncan D. 2021. <a href="https://doi.org/10.1007/s00277-021-04489-w">Association between ABO blood types and coronavirus disease 2019 (COVID-19), genetic associations, and underlying molecular mechanisms: a literature review of 23 studies</a>. Ann Hematol 100:1123–1132.</div>
</div>
<div id="ref-16ZxZ0D9J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">458. </div><div class="csl-right-inline">Wu B-B, Gu D-Z, Yu J-N, Yang J, Shen W-Q. 2020. <a href="https://doi.org/10.1016/j.meegid.2020.104485">Association between ABO blood groups and COVID-19 infection, severity and demise: A systematic review and meta-analysis</a>. Infection, Genetics and Evolution 84:104485.</div>
</div>
<div id="ref-11G8Z08cp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">459. </div><div class="csl-right-inline">Dai X. 2020. <a href="https://doi.org/10.1177/2047487320922370">ABO blood group predisposes to COVID-19 severity and cardiovascular diseases</a>. Eur J Prev Cardiolog 27:1436–1437.</div>
</div>
<div id="ref-c2EzrLpZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">460. </div><div class="csl-right-inline">Silva-Filho JC, Melo CGF de, Oliveira JL de. 2020. <a href="https://doi.org/10.1016/j.mehy.2020.110155">The influence of ABO blood groups on COVID-19 susceptibility and severity: A molecular hypothesis based on carbohydrate-carbohydrate interactions</a>. Medical Hypotheses 144:110155.</div>
</div>
<div id="ref-X1avT5DW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">461. </div><div class="csl-right-inline">Marigorta UM, Rodríguez JA, Gibson G, Navarro A. 2018. <a href="https://doi.org/10.1016/j.tig.2018.03.005">Replicability and Prediction: Lessons and Challenges from GWAS</a>. Trends in Genetics 34:504–517.</div>
</div>
<div id="ref-T3GG8iJN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">462. </div><div class="csl-right-inline">Price AL, Zaitlen NA, Reich D, Patterson N. 2010. <a href="https://doi.org/10.1038/nrg2813">New approaches to population stratification in genome-wide association studies</a>. Nat Rev Genet 11:459–463.</div>
</div>
<div id="ref-Xq4jYNyv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">463. </div><div class="csl-right-inline">Weiner J 3rd, Suwalski P, Holtgrewe M, Rakitko A, Thibeault C, Müller M, Patriki D, Quedenau C, Krüger U, Ilinsky V, Popov I, Balnis J, Jaitovich A, Helbig ET, Lippert LJ, Stubbemann P, Real LM, Macías J, Pineda JA, Fernandez-Fuertes M, Wang X, Karadeniz Z, Saccomanno J, Doehn J-M, Hübner R-H, Hinzmann B, Salvo M, Blueher A, Siemann S, Jurisic S, Beer JH, Rutishauser J, Wiggli B, Schmid H, Danninger K, Binder R, Corman VM, Mühlemann B, Arjun Arkal R, Fragiadakis GK, Mick E, COMET C, Calfee CS, Erle DJ, Hendrickson CM, Kangelaris KN, Krummel MF, Woodruff PG, Langelier CR, Venkataramani U, García F, Zyla J, Drosten C, Alice B, Jones TC, Suttorp N, Witzenrath M, Hippenstiel S, Zemojtel T, Skurk C, Poller W, Borodina T, Pa-COVID SG, Ripke S, Sander LE, Beule D, Landmesser U, Guettouche T, Kurth F, Heidecker B. 2021. <a href="https://doi.org/10.1016/j.eclinm.2021.101099">Increased risk of severe clinical course of COVID-19 in carriers of HLA-C*04:01</a>. EClinicalMedicine 40:101099.</div>
</div>
<div id="ref-owevcpwb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">464. </div><div class="csl-right-inline">The Severe Covid-19 GWAS Group. 2020. <a href="https://doi.org/10.1056/nejmoa2020283">Genomewide Association Study of Severe Covid-19 with Respiratory Failure</a>. New England Journal of Medicine 383:1522–1534.</div>
</div>
<div id="ref-HrYm0pnz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">465. </div><div class="csl-right-inline">2020. <a href="https://doi.org/10.1038/s41431-020-0636-6">The COVID-19 Host Genetics Initiative, a global initiative to elucidate the role of host genetic factors in susceptibility and severity of the SARS-CoV-2 virus pandemic</a>. Eur J Hum Genet 28:715–718.</div>
</div>
<div id="ref-14Vd7Guv9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">466. </div><div class="csl-right-inline">Zeberg H, Pääbo S. 2020. <a href="https://doi.org/10.1038/s41586-020-2818-3">The major genetic risk factor for severe COVID-19 is inherited from Neanderthals</a>. Nature 587:610–612.</div>
</div>
<div id="ref-J1pI7YF1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">467. </div><div class="csl-right-inline">Ding Q, Hu Y, Xu S, Wang J, Jin L. 2013. <a href="https://doi.org/10.1093/molbev/mst260">Neanderthal Introgression at Chromosome 3p21.31 Was Under Positive Natural Selection in East Asians</a>. Molecular Biology and Evolution 31:683–695.</div>
</div>
<div id="ref-2J6m4djb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">468. </div><div class="csl-right-inline">Zhou J, Sun Y, Huang W, Ye K. 2021. <a href="https://doi.org/10.1093/gerona/glab035">Altered Blood Cell Traits Underlie a Major Genetic Locus of Severe COVID-19</a>. The Journals of Gerontology: Series A 76:e147–e154.</div>
</div>
<div id="ref-DyG1eyK4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">469. </div><div class="csl-right-inline">Grubaugh ND, Hanage WP, Rasmussen AL. 2020. <a href="https://doi.org/10.1016/j.cell.2020.06.040">Making Sense of Mutation: What D614G Means for the COVID-19 Pandemic Remains Unclear</a>. Cell 182:794–795.</div>
</div>
<div id="ref-HFUP2slc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">470. </div><div class="csl-right-inline">Avanzato VA, Matson MJ, Seifert SN, Pryce R, Williamson BN, Anzick SL, Barbian K, Judson SD, Fischer ER, Martens C, Bowden TA, de Wit E, Riedo FX, Munster VJ. 2020. <a href="https://doi.org/10.1016/j.cell.2020.10.049">Case Study: Prolonged Infectious SARS-CoV-2 Shedding from an Asymptomatic Immunocompromised Individual with Cancer</a>. Cell 183:1901–1912.e9.</div>
</div>
<div id="ref-13x1Imo3Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">471. </div><div class="csl-right-inline">Choi B, Choudhary MC, Regan J, Sparks JA, Padera RF, Qiu X, Solomon IH, Kuo H-H, Boucau J, Bowman K, Adhikari UD, Winkler ML, Mueller AA, Hsu TY-T, Desjardins M, Baden LR, Chan BT, Walker BD, Lichterfeld M, Brigl M, Kwon DS, Kanjilal S, Richardson ET, Jonsson AH, Alter G, Barczak AK, Hanage WP, Yu XG, Gaiha GD, Seaman MS, Cernadas M, Li JZ. 2020. <a href="https://doi.org/10.1056/nejmc2031364">Persistence and Evolution of SARS-CoV-2 in an Immunocompromised Host</a>. New England Journal of Medicine 383:2291–2293.</div>
</div>
<div id="ref-3jfKqJRA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">472. </div><div class="csl-right-inline">Andreano E, Piccini G, Licastro D, Casalino L, Johnson NV, Paciello I, Monego SD, Pantano E, Manganaro N, Manenti A, Manna R, Casa E, Hyseni I, Benincasa L, Montomoli E, Amaro RE, McLellan JS, Rappuoli R. 2020. SARS-CoV-2 escape &lt;i&gt;in vitro&lt;/i&gt; from a highly neutralizing COVID-19 convalescent plasma. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.12.28.424451">10.1101/2020.12.28.424451</a>.</div>
</div>
<div id="ref-xFOQs3Qb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">473. </div><div class="csl-right-inline">Lauring AS, Hodcroft EB. 2021. Genetic Variants of SARS-CoV-2—What Do They Mean? JAMA https://doi.org/<a href="https://doi.org/10.1001/jama.2020.27124">10.1001/jama.2020.27124</a>.</div>
</div>
<div id="ref-4IJOYEr2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">474. </div><div class="csl-right-inline">Oude Munnink BB, Sikkema RS, Nieuwenhuijse DF, Molenaar RJ, Munger E, Molenkamp R, van der Spek A, Tolsma P, Rietveld A, Brouwer M, Bouwmeester-Vincken N, Harders F, Hakze-van der Honing R, Wegdam-Blans MCA, Bouwstra RJ, GeurtsvanKessel C, van der Eijk AA, Velkers FC, Smit LAM, Stegeman A, van der Poel WHM, Koopmans MPG. 2021. <a href="https://doi.org/10.1126/science.abe5901">Transmission of SARS-CoV-2 on mink farms between humans and mink and back to humans</a>. Science 371:172–177.</div>
</div>
<div id="ref-INygVT9y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">475. </div><div class="csl-right-inline">Davies NG, Barnard RC, Jarvis CI, Kucharski AJ, Munday J, Pearson CAB, Russell TW, Tully DC, Abbott S, Gimma A, Waites W, Wong KL, van Zandvoort K, Eggo RM, Funk S, Jit M, Atkins KE, Edmunds WJ, CMMID COVID-19 Working Group. 2020. Estimated transmissibility and severity of novel SARS-CoV-2 Variant of Concern 202012/01 in England. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.12.24.20248822">10.1101/2020.12.24.20248822</a>.</div>
</div>
<div id="ref-17bdYIjnm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">476. </div><div class="csl-right-inline">du Plessis L, McCrone JT, Zarebski AE, Hill V, Ruis C, Gutierrez B, Raghwani J, Ashworth J, Colquhoun R, Connor TR, Faria NR, Jackson B, Loman NJ, O’Toole Á, Nicholls SM, Parag KV, Scher E, Vasylyeva TI, Volz EM, Watts A, Bogoch II, Khan K, Aanensen DM, Kraemer MUG, Rambaut A, Pybus OG, COVID-19 Genomics UK (COG-UK) Consortium†. 2021. <a href="https://doi.org/10.1126/science.abf2946">Establishment and lineage dynamics of the SARS-CoV-2 epidemic in the UK</a>. Science eabf2946.</div>
</div>
<div id="ref-1FGTHhvPb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">477. </div><div class="csl-right-inline">Hie B, Zhong ED, Berger B, Bryson B. 2021. <a href="https://doi.org/10.1126/science.abd7331">Learning the language of viral evolution and escape</a>. Science 371:284–288.</div>
</div>
<div id="ref-esiDpXpq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">478. </div><div class="csl-right-inline">Grubaugh ND, Petrone ME, Holmes EC. 2020. <a href="https://doi.org/10.1038/s41564-020-0690-4">We shouldn’t worry when a virus mutates during disease outbreaks</a>. Nature Microbiology 5:529–530.</div>
</div>
<div id="ref-6G6nZMtT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin"><a href="https://doi.org/10.1186/1471-2148-4-21">479. Zhao Z, Li H, Wu X, Zhong Y, Zhang K, Zhang Y-P, Boerwinkle E, Fu Y-X. 2004.. BMC Evol Biol 4:21</a></div>.
</div>
<div id="ref-1kRqNje" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">480. </div><div class="csl-right-inline">Collington E. 2020. PHE document. <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf</a>.</div>
</div>
<div id="ref-swAa8Nio" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">481. </div><div class="csl-right-inline">2020. Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations. Virological. <a href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563">https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-m9qtrWft" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">482. </div><div class="csl-right-inline">B.1.1.7 report. <a href="https://cov-lineages.org/global_report_B.1.1.7.html">https://cov-lineages.org/global_report_B.1.1.7.html</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-lIzRCwLq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">483. </div><div class="csl-right-inline">Davies NG, Jarvis CI, Edmunds WJ, Jewell NP, Diaz-Ordaz K, Keogh RH. 2021. <a href="https://doi.org/10.1038/s41586-021-03426-1">Increased mortality in community-tested cases of SARS-CoV-2 lineage B.1.1.7</a>. Nature 593:270–274.</div>
</div>
<div id="ref-VDhaLzV3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">484. </div><div class="csl-right-inline">2020. Identification of a novel SARS-CoV-2 Spike 69-70 deletion lineage circulating in the United States. Virological. <a href="https://virological.org/t/identification-of-a-novel-sars-cov-2-spike-69-70-deletion-lineage-circulating-in-the-united-states/577">https://virological.org/t/identification-of-a-novel-sars-cov-2-spike-69-70-deletion-lineage-circulating-in-the-united-states/577</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-10Qn2iFj7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">485. </div><div class="csl-right-inline">Washington NL, White S, Barrett KMS, Cirulli ET, Bolze A, Lu JT. 2020. S gene dropout patterns in SARS-CoV-2 tests suggest spread of the H69del/V70del mutation in the US. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.12.24.20248814">10.1101/2020.12.24.20248814</a>.</div>
</div>
<div id="ref-11g9KlaY4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">486. </div><div class="csl-right-inline">2021. Investigation of novel SARS-CoV-2 variant: Variant of Concern 202012/01. Public Health England. <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/959360/Variant_of_Concern_VOC_202012_01_Technical_Briefing_3.pdf">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/959360/Variant_of_Concern_VOC_202012_01_Technical_Briefing_3.pdf</a>.</div>
</div>
<div id="ref-F4Le4e1M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">487. </div><div class="csl-right-inline">Health C for D and R. 2021. <a href="https://www.fda.gov/medical-devices/letters-health-care-providers/genetic-variants-sars-cov-2-may-lead-false-negative-results-molecular-tests-detection-sars-cov-2">Genetic Variants of SARS-CoV-2 May Lead to False Negative Results with Molecular Tests for Detection of SARS-CoV-2 - Letter to Clinical Laboratory Staff and Health Care Providers</a>. FDA.</div>
</div>
<div id="ref-1455MbSH0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">488. </div><div class="csl-right-inline">CDC. 2020. Coronavirus Disease 2019 (COVID-19). Centers for Disease Control and Prevention. <a href="https://www.cdc.gov/coronavirus/2019-ncov/more/science-and-research/scientific-brief-emerging-variants.html">https://www.cdc.gov/coronavirus/2019-ncov/more/science-and-research/scientific-brief-emerging-variants.html</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-1EWBspR2q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">489. </div><div class="csl-right-inline">CDC. 2020. Coronavirus Disease 2019 (COVID-19). Centers for Disease Control and Prevention. <a href="https://www.cdc.gov/coronavirus/2019-ncov/variants/variant-surveillance.html">https://www.cdc.gov/coronavirus/2019-ncov/variants/variant-surveillance.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-aXPtLhNl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">490. </div><div class="csl-right-inline">Minister Zweli Mkhize confirms 8 725 more cases of Coronavirus COVID-19 | South African Government. <a href="https://www.gov.za/speeches/minister-zweli-mkhize-confirms-8-725-more-cases-coronavirus-covid-19-18-dec-2020-0000">https://www.gov.za/speeches/minister-zweli-mkhize-confirms-8-725-more-cases-coronavirus-covid-19-18-dec-2020-0000</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-11ZfOrGxJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">491. </div><div class="csl-right-inline">2021. Tracking the international spread of SARS-CoV-2 lineages B.1.1.7 and B.1.351/501Y-V2. Virological. <a href="https://virological.org/t/tracking-the-international-spread-of-sars-cov-2-lineages-b-1-1-7-and-b-1-351-501y-v2/592">https://virological.org/t/tracking-the-international-spread-of-sars-cov-2-lineages-b-1-1-7-and-b-1-351-501y-v2/592</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-WgiJaD2d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">492. </div><div class="csl-right-inline">Tegally H, Wilkinson E, Giovanetti M, Iranzadeh A, Fonseca V, Giandhari J, Doolabh D, Pillay S, San EJ, Msomi N, Mlisana K, von Gottberg A, Walaza S, Allam M, Ismail A, Mohale T, Glass AJ, Engelbrecht S, Van Zyl G, Preiser W, Petruccione F, Sigal A, Hardie D, Marais G, Hsiao M, Korsman S, Davies M-A, Tyers L, Mudau I, York D, Maslo C, Goedhals D, Abrahams S, Laguda-Akingba O, Alisoltani-Dehkordi A, Godzik A, Wibmer CK, Sewell BT, Lourenço J, Alcantara LCJ, Pond SLK, Weaver S, Martin D, Lessells RJ, Bhiman JN, Williamson C, de Oliveira T. 2020. Emergence and rapid spread of a new severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) lineage with multiple spike mutations in South Africa. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.12.21.20248640">10.1101/2020.12.21.20248640</a>.</div>
</div>
<div id="ref-OPGkrmXZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">493. </div><div class="csl-right-inline">Wibmer CK, Ayres F, Hermanus T, Madzivhandila M, Kgagudi P, Oosthuysen B, Lambson BE, de Oliveira T, Vermeulen M, van der Berg K, Rossouw T, Boswell M, Ueckermann V, Meiring S, von Gottberg A, Cohen C, Morris L, Bhiman JN, Moore PL. 2021. <a href="https://doi.org/10.1038/s41591-021-01285-x">SARS-CoV-2 501Y.V2 escapes neutralization by South African COVID-19 donor plasma</a>. Nat Med 27:622–625.</div>
</div>
<div id="ref-UzTTUShg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">494. </div><div class="csl-right-inline">Cele S, Gazy I, Jackson L, Hwa S-H, Tegally H, Lustig G, Giandhari J, Pillay S, Wilkinson E, Naidoo Y, Karim F, Ganga Y, Khan K, Bernstein M, Balazs AB, Gosnell BI, Hanekom W, Moosa M-YS, Lessells RJ, de Oliveira T, Sigal A. 2021. <a href="https://doi.org/10.1038/s41586-021-03471-w">Escape of SARS-CoV-2 501Y.V2 from neutralization by convalescent plasma</a>. Nature 593:142–146.</div>
</div>
<div id="ref-FuVm03yl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">495. </div><div class="csl-right-inline">ECDC. 2021.
Risk of spread of new SARS-CoV-2 variants of concern in the EU/EEA - first update
. <a href="https://www.ecdc.europa.eu/sites/default/files/documents/COVID-19-risk-related-to-spread-of-new-SARS-CoV-2-variants-EU-EEA-first-update.pdf">https://www.ecdc.europa.eu/sites/default/files/documents/COVID-19-risk-related-to-spread-of-new-SARS-CoV-2-variants-EU-EEA-first-update.pdf</a>.</div>
</div>
<div id="ref-7hGWIt0g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">496. </div><div class="csl-right-inline">2021. Genomic characterisation of an emergent SARS-CoV-2 lineage in Manaus: preliminary findings. Virological. <a href="https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586">https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-IIEqiDeW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">497. </div><div class="csl-right-inline">P.1 report. <a href="https://cov-lineages.org/global_report_P.1.html">https://cov-lineages.org/global_report_P.1.html</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-17p7RjLmX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">498. </div><div class="csl-right-inline">Staff R. 2021. <a href="https://www.reuters.com/article/uk-health-coronavirus-britain-variants-idUSKBN29T07E">UK detects 77 cases of South African COVID variant, nine of Brazilian</a>. Reuters.</div>
</div>
<div id="ref-TzywDS3t" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">499. </div><div class="csl-right-inline">PANGO lineages. <a href="https://cov-lineages.org/lineages.html">https://cov-lineages.org/lineages.html</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-1Ha4IVlGr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">500. </div><div class="csl-right-inline">Zhang W, Davis BD, Chen SS, Martinez JMS, Plummer JT, Vail E. 2021. Emergence of a novel SARS-CoV-2 strain in Southern California, USA. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2021.01.18.21249786">10.1101/2021.01.18.21249786</a>.</div>
</div>
<div id="ref-BoGqjBAU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">501. </div><div class="csl-right-inline">GISAID - hCov19 Variants. <a href="https://gisaid.org/hcov19-variants/">https://gisaid.org/hcov19-variants/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-xV1qJhIq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">502. </div><div class="csl-right-inline">Li Q, Wu J, Nie J, Zhang L, Hao H, Liu S, Zhao C, Zhang Q, Liu H, Nie L, Qin H, Wang M, Lu Q, Li X, Sun Q, Liu J, Zhang L, Li X, Huang W, Wang Y. 2020. <a href="https://doi.org/10.1016/j.cell.2020.07.012">The Impact of Mutations in SARS-CoV-2 Spike on Viral Infectivity and Antigenicity</a>. Cell 182:1284–1294.e9.</div>
</div>
<div id="ref-1FhpGQXK9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">503. </div><div class="csl-right-inline">Greaney AJ, Loes AN, Crawford KHD, Starr TN, Malone KD, Chu HY, Bloom JD. 2021. Comprehensive mapping of mutations to the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human serum antibodies. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.12.31.425021">10.1101/2020.12.31.425021</a>.</div>
</div>
<div id="ref-2rGCvk9t" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">504. </div><div class="csl-right-inline">2021. Neue Corona-Variante: 35 Fälle in Garmisch-Partenkirchen. BR24. <a href="https://www.br.de/nachrichten/bayern/neue-coronavirus-mutation-35-faelle-in-garmisch-partenkirchen,SMQ1V6u">https://www.br.de/nachrichten/bayern/neue-coronavirus-mutation-35-faelle-in-garmisch-partenkirchen,SMQ1V6u</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-ma0aswzQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">505. </div><div class="csl-right-inline">PANGO lineages. <a href="https://cov-lineages.org/global_report.html">https://cov-lineages.org/global_report.html</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-RsiMKsNV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">506. </div><div class="csl-right-inline">Starr TN, Greaney AJ, Hilton SK, Ellis D, Crawford KHD, Dingens AS, Navarro MJ, Bowen JE, Tortorici MA, Walls AC, King NP, Veesler D, Bloom JD. 2020. <a href="https://doi.org/10.1016/j.cell.2020.08.012">Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding</a>. Cell 182:1295–1310.e20.</div>
</div>
<div id="ref-rBRU1aGx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">507. </div><div class="csl-right-inline">Gu H, Chen Q, Yang G, He L, Fan H, Deng Y-Q, Wang Y, Teng Y, Zhao Z, Cui Y, Li Y, Li X-F, Li J, Zhang N-N, Yang X, Chen S, Guo Y, Zhao G, Wang X, Luo D-Y, Wang H, Yang X, Li Y, Han G, He Y, Zhou X, Geng S, Sheng X, Jiang S, Sun S, Qin C-F, Zhou Y. 2020. <a href="https://doi.org/10.1126/science.abc4730">Adaptation of SARS-CoV-2 in BALB/c mice for testing vaccine efficacy</a>. Science eabc4730.</div>
</div>
<div id="ref-156Htv51A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">508. </div><div class="csl-right-inline">Greaney AJ, Starr TN, Gilchuk P, Zost SJ, Binshtein E, Loes AN, Hilton SK, Huddleston J, Eguia R, Crawford KHD, Dingens AS, Nargi RS, Sutton RE, Suryadevara N, Rothlauf PW, Liu Z, Whelan SPJ, Carnahan RH, Crowe JE, Bloom JD. 2021. <a href="https://doi.org/10.1016/j.chom.2020.11.007">Complete Mapping of Mutations to the SARS-CoV-2 Spike Receptor-Binding Domain that Escape Antibody Recognition</a>. Cell Host &amp; Microbe 29:44–57.e9.</div>
</div>
<div id="ref-15U0IONBy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">509. </div><div class="csl-right-inline">McCarthy KR, Rennick LJ, Nambulli S, Robinson-McCarthy LR, Bain WG, Haidar G, Duprex WP. 2021. Recurrent deletions in the SARS-CoV-2 spike glycoprotein drive antibody escape. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.11.19.389916">10.1101/2020.11.19.389916</a>.</div>
</div>
<div id="ref-skeuNfN3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">510. </div><div class="csl-right-inline">Kupferschmidt K. 2021. Viral mutations may cause another ‘very, very bad’ COVID-19 wave, scientists warn. Science https://doi.org/<a href="https://doi.org/10.1126/science.abg4312">10.1126/science.abg4312</a>.</div>
</div>
<div id="ref-15GoX5q2G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">511. </div><div class="csl-right-inline">2021. SARS-CoV-2 reinfection by the new Variant of Concern (VOC) P.1 in Amazonas, Brazil. Virological. <a href="https://virological.org/t/sars-cov-2-reinfection-by-the-new-variant-of-concern-voc-p-1-in-amazonas-brazil/596">https://virological.org/t/sars-cov-2-reinfection-by-the-new-variant-of-concern-voc-p-1-in-amazonas-brazil/596</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-w1JG0rDF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">512. </div><div class="csl-right-inline">Callaway E. 2021. <a href="https://doi.org/10.1038/d41586-021-00121-z">Fast-spreading COVID variant can elude immune responses</a>. Nature 589:500–501.</div>
</div>
<div id="ref-Svu4IEpC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">513. </div><div class="csl-right-inline">Starr TN, Greaney AJ, Addetia A, Hannon WW, Choudhary MC, Dingens AS, Li JZ, Bloom JD. 2021. <a href="https://doi.org/10.1126/science.abf9302">Prospective mapping of viral mutations that escape antibodies used to treat COVID-19</a>. Science eabf9302.</div>
</div>
<div id="ref-11FTHvRwx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">514. </div><div class="csl-right-inline">Kupferschmidt K. 2021. <a href="https://doi.org/10.1126/science.371.6527.329">New mutations raise specter of ‘immune escape’</a>. Science 371:329–330.</div>
</div>
<div id="ref-izey3Z40" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">515. </div><div class="csl-right-inline">Wu K, Werner AP, Moliva JI, Koch M, Choi A, Stewart-Jones GBE, Bennett H, Boyoglu-Barnum S, Shi W, Graham BS, Carfi A, Corbett KS, Seder RA, Edwards DK. 2021. mRNA-1273 vaccine induces neutralizing antibodies against spike mutants from global SARS-CoV-2 variants. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2021.01.25.427948">10.1101/2021.01.25.427948</a>.</div>
</div>
<div id="ref-CWlYjjIV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">516. </div><div class="csl-right-inline">Polack FP, Thomas SJ, Kitchin N, Absalon J, Gurtman A, Lockhart S, Perez JL, Pérez Marc G, Moreira ED, Zerbini C, Bailey R, Swanson KA, Roychoudhury S, Koury K, Li P, Kalina WV, Cooper D, Frenck RW, Hammitt LL, Türeci Ö, Nell H, Schaefer A, Ünal S, Tresnan DB, Mather S, Dormitzer PR, Şahin U, Jansen KU, Gruber WC. 2020. <a href="https://doi.org/10.1056/nejmoa2034577">Safety and Efficacy of the BNT162b2 mRNA Covid-19 Vaccine</a>. New England Journal of Medicine 383:2603–2615.</div>
</div>
<div id="ref-MD2K7MYB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">517. </div><div class="csl-right-inline">Walsh EE, Frenck RW, Falsey AR, Kitchin N, Absalon J, Gurtman A, Lockhart S, Neuzil K, Mulligan MJ, Bailey R, Swanson KA, Li P, Koury K, Kalina W, Cooper D, Fontes-Garfias C, Shi P-Y, Türeci Ö, Tompkins KR, Lyke KE, Raabe V, Dormitzer PR, Jansen KU, Şahin U, Gruber WC. 2020. <a href="https://doi.org/10.1056/nejmoa2027906">Safety and Immunogenicity of Two RNA-Based Covid-19 Vaccine Candidates</a>. New England Journal of Medicine 383:2439–2450.</div>
</div>
<div id="ref-xYHivkXH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">518. </div><div class="csl-right-inline">Xie X, Zou J, Fontes-Garfias CR, Xia H, Swanson KA, Cutler M, Cooper D, Menachery VD, Weaver S, Dormitzer PR, Shi P-Y. 2021. Neutralization of N501Y mutant SARS-CoV-2 by BNT162b2 vaccine-elicited sera. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2021.01.07.425740">10.1101/2021.01.07.425740</a>.</div>
</div>
<div id="ref-kcOVBUnj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">519. </div><div class="csl-right-inline">Wang Z, Schmidt F, Weisblum Y, Muecksch F, Barnes CO, Finkin S, Schaefer-Babajew D, Cipolla M, Gaebler C, Lieberman JA, Oliveira TY, Yang Z, Abernathy ME, Huey-Tubman KE, Hurley A, Turroja M, West KA, Gordon K, Millard KG, Ramos V, Da Silva J, Xu J, Colbert RA, Patel R, Dizon J, Unson-O’Brien C, Shimeliovich I, Gazumyan A, Caskey M, Bjorkman PJ, Casellas R, Hatziioannou T, Bieniasz PD, Nussenzweig MC. 2021. mRNA vaccine-elicited antibodies to SARS-CoV-2 and circulating variants. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2021.01.15.426911">10.1101/2021.01.15.426911</a>.</div>
</div>
<div id="ref-zncU219l" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">520. </div><div class="csl-right-inline">Shen X, Tang H, Pajon R, Smith G, Glenn GM, Shi W, Korber B, Montefiori DC. 2021. <a href="https://doi.org/10.1056/nejmc2103740">Neutralization of SARS-CoV-2 Variants B.1.429 and B.1.351</a>. N Engl J Med 384:2352–2354.</div>
</div>
<div id="ref-7MzilPo6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">521. </div><div class="csl-right-inline">Jangra S, Ye C, Rathnasinghe R, Stadlbauer D, Krammer F, Simon V, Martinez-Sobrido L, García-Sastre A, Schotsaert M, Alshammary H, Amoako AA, Awawda MH, Beach KF, Bermúdez-González MC, Chernet RL, Eaker LQ, Ferreri ED, Floda DL, Gleason CR, Kleiner G, Jurczyszak D, Matthews JC, Mendez WA, Mulder LCF, Russo KT, Salimbangon A-BT, Saksena M, Shin AS, Sominsky LA, Srivastava K. 2021. SARS-CoV-2 spike E484K mutation reduces antibody neutralisation. The Lancet Microbe https://doi.org/<a href="https://doi.org/10.1016/s2666-5247(21)00068-9">10.1016/s2666-5247(21)00068-9</a>.</div>
</div>
<div id="ref-19CcC5AGf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">522. </div><div class="csl-right-inline">Collier DA, De Marco A, Ferreira IATM, Meng B, Datir RP, Walls AC, Kemp SA, Bassi J, Pinto D, Silacci-Fregni C, Bianchi S, Tortorici MA, Bowen J, Culap K, Jaconi S, Cameroni E, Snell G, Pizzuto MS, Pellanda AF, Garzoni C, Riva A, Baker S, Dougan G, Hess C, Kingston N, Lehner PJ, Lyons PA, Matheson NJ, Owehand WH, Saunders C, Summers C, Thaventhiran JED, Toshner M, Weekes MP, Bucke A, Calder J, Canna L, Domingo J, Elmer A, Fuller S, Harris J, Hewitt S, Kennet J, Jose S, Kourampa J, Meadows A, O’Brien C, Price J, Publico C, Rastall R, Ribeiro C, Rowlands J, Ruffolo V, Tordesillas H, Bullman B, Dunmore BJ, Fawke S, Gräf S, Hodgson J, Huang C, Hunter K, Jones E, Legchenko E, Matara C, Martin J, Mescia F, O’Donnell C, Pointon L, Pond N, Shih J, Sutcliffe R, Tilly T, Treacy C, Tong Z, Wood J, Wylot M, Bergamaschi L, Betancourt A, Bower G, Cossetti C, De Sa A, Epping M, Grenfell R, Hinch A, Huhn O, Jackson S, Jarvis I, Lewis D, Marsden J, Nice F, Okecha G, Omarjee O, Perera M, Richoz N, Romashova V, Yarkoni NS, Sharma R, Stefanucci L, Stephens J, Strezlecki M, Turner L, De Bie EMDD, Bunclark K, Josipovic M, Mackay M, Rossi S, Selvan M, Spencer S, Yong C, Ansaripour A, Michael A, Mwaura L, Patterson C, Polwarth G, Polgarova P, di Stefano G, Fahey C, Michel R, Bong S-H, Coudert JD, Holmes E, Allison J, Butcher H, Caputo D, Clapham-Riley D, Dewhurst E, Furlong A, Graves B, Gray J, Ivers T, Kasanicki M, Le Gresley E, Linger R, Meloy S, Muldoon F, Ovington N, Papadia S, Phelan I, Stark H, Stirrups KE, Townsend P, Walker N, Webster J, Elmer A, Kingston N, Graves B, McCoy LE, Smith KGC, Bradley JR, Temperton N, Ceron-Gutierrez L, Barcenas-Morales G, Robson SC, Loman NJ, Connor TR, Golubchik T, Martinez Nunez RT, Ludden C, Corden S, Johnston I, Bonsall D, Smith CP, Awan AR, Bucca G, Torok ME, Saeed K, Prieto JA, Jackson DK, Hamilton WL, Snell LB, Moore C, Harrison EM, Goncalves S, Fairley DJ, Loose MW, Watkins J, Livett R, Moses S, Amato R, Nicholls S, Bull M, Smith DL, Barrett J, Aanensen DM, Curran MD, Parmar S, Aggarwal D, Shepherd JG, Parker MD, Glaysher S, Bashton M, Underwood AP, Pacchiarini N, Loveson KF, Carabelli AM, Templeton KE, Langford CF, Sillitoe J, de Silva TI, Wang D, Kwiatkowski D, Rambaut A, O’Grady J, Cottrell S, Holden MTG, Thomson EC, Osman H, Andersson M, Chauhan AJ, Hassan-Ibrahim MO, Lawniczak M, Alderton A, Chand M, Constantinidou C, Unnikrishnan M, Darby AC, Hiscox JA, Paterson S, Martincorena I, Robertson DL, Volz EM, Page AJ, Pybus OG, Bassett AR, Ariani CV, Spencer Chapman MH, Li KK, Shah RN, Jesudason NG, Taha Y, McHugh MP, Dewar R, Jahun AS, McMurray C, Pandey S, McKenna JP, Nelson A, Young GR, McCann CM, Elliott S, Lowe H, Temperton B, Roy S, Price A, Rey S, Wyles M, Rooke S, Shaaban S, de Cesare M, Letchford L, Silveira S, Pelosi E, Wilson-Davies E, Hosmillo M, O’Toole Á, Hesketh AR, Stark R, du Plessis L, Ruis C, Adams H, Bourgeois Y, Michell SL, Grammatopoulos D, Edgeworth J, Breuer J, Todd JA, Fraser C, Buck D, John M, Kay GL, Palmer S, Peacock SJ, Heyburn D, Weldon D, Robinson E, McNally A, Muir P, Vipond IB, Boyes J, Sivaprakasam V, Salluja T, Dervisevic S, Meader EJ, Park NR, Oliver K, Jeffries AR, Ott S, da Silva Filipe A, Simpson DA, Williams C, Masoli JAH, Knight BA, Jones CR, Koshy C, Ash A, Casey A, Bosworth A, Ratcliffe L, Xu-McCrae L, Pymont HM, Hutchings S, Berry L, Jones K, Halstead F, Davis T, Holmes C, Iturriza-Gomara M, Lucaci AO, Randell PA, Cox A, Madona P, Harris KA, Brown JR, Mahungu TW, Irish-Tavares D, Haque T, Hart J, Witele E, Fenton ML, Liggett S, Graham C, Swindells E, Collins J, Eltringham G, Campbell S, McClure PC, Clark G, Sloan TJ, Jones C, Lynch J, Warne B, Leonard S, Durham J, Williams T, Haldenby ST, Storey N, Alikhan N-F, Holmes N, Moore C, Carlile M, Perry M, Craine N, Lyons RA, Beckett AH, Goudarzi S, Fearn C, Cook K, Dent H, Paul H, Davies R, Blane B, Girgis ST, Beale MA, Bellis KL, Dorman MJ, Drury E, Kane L, Kay S, McGuigan S, Nelson R, Prestwood L, Rajatileka S, Batra R, Williams RJ, Kristiansen M, Green A, Justice A, Mahanama AIK, Samaraweera B, Hadjirin NF, Quick J, Poplawski R, Kermack LM, Reynolds N, Hall G, Chaudhry Y, Pinckert ML, Georgana I, Moll RJ, Thornton A, Myers R, Stockton J, Williams CA, Yew WC, Trotter AJ, Trebes A, MacIntyre-Cockett G, Birchley A, Adams A, Plimmer A, Gatica-Wilcox B, McKerr C, Hilvers E, Jones H, Asad H, Coombes J, Evans JM, Fina L, Gilbert L, Graham L, Cronin M, Kumziene-Summerhayes S, Taylor S, Jones S, Groves DC, Zhang P, Gallis M, Louka SF, Starinskij I, Jackson C, Gourtovaia M, Tonkin-Hill G, Lewis K, Tovar-Corona JM, James K, Baxter L, Alam MT, Orton RJ, Hughes J, Vattipally S, Ragonnet-Cronin M, Nascimento FF, Jorgensen D, Boyd O, Geidelberg L, Zarebski AE, Raghwani J, Kraemer MUG, Southgate J, Lindsey BB, Freeman TM, Keatley J-P, Singer JB, de Oliveira Martins L, Yeats CA, Abudahab K, Taylor BEW, Menegazzo M, Danesh J, Hogsden W, Eldirdiri S, Kenyon A, Mason J, Robinson TI, Holmes A, Price J, Hartley JA, Curran T, Mather AE, Shankar G, Jones R, Howe R, Morgan S, Wastenge E, Chapman MR, Mookerjee S, Stanley R, Smith W, Peto T, Eyre D, Crook D, Vernet G, Kitchen C, Gulliver H, Merrick I, Guest M, Munn R, Bradley DT, Wyatt T, Beaver C, Foulser L, Palmer S, Churcher CM, Brooks E, Smith KS, Galai K, McManus GM, Bolt F, Coll F, Meadows L, Attwood SW, Davies A, De Lacy E, Downing F, Edwards S, Scarlett GP, Jeremiah S, Smith N, Leek D, Sridhar S, Forrest S, Cormie C, Gill HK, Dias J, Higginson EE, Maes M, Young J, Wantoch M, Jamrozy D, Lo S, Patel M, Hill V, Bewshea CM, Ellard S, Auckland C, Harrison I, Bishop C, Chalker V, Richter A, Beggs A, Best A, Percival B, Mirza J, Megram O, Mayhew M, Crawford L, Ashcroft F, Moles-Garcia E, Cumley N, Hopes R, Asamaphan P, Niebel MO, Gunson RN, Bradley A, Maclean A, Mollett G, Blacow R, Bird P, Helmer T, Fallon K, Tang J, Hale AD, Macfarlane-Smith LR, Harper KL, Carden H, Machin NW, Jackson KA, Ahmad SSY, George RP, Turtle L, O’Toole E, Watts J, Breen C, Cowell A, Alcolea-Medina A, Charalampous T, Patel A, Levett LJ, Heaney J, Rowan A, Taylor GP, Shah D, Atkinson L, Lee JCD, Westhorpe AP, Jannoo R, Lowe HL, Karamani A, Ensell L, Chatterton W, Pusok M, Dadrah A, Symmonds A, Sluga G, Molnar Z, Baker P, Bonner S, Essex S, Barton E, Padgett D, Scott G, Greenaway J, Payne BAI, Burton-Fanning S, Waugh S, Raviprakash V, Sheriff N, Blakey V, Williams L-A, Moore J, Stonehouse S, Smith L, Davidson RK, Bedford L, Coupland L, Wright V, Chappell JG, Tsoleridis T, Ball J, Khakh M, Fleming VM, Lister MM, Howson-Wells HC, Berry L, Boswell T, Joseph A, Willingham I, Duckworth N, Walsh S, Wise E, Moore N, Mori M, Cortes N, Kidd S, Williams R, Gifford L, Bicknell K, Wyllie S, Lloyd A, Impey R, Malone CS, Cogger BJ, Levene N, Monaghan L, Keeley AJ, Partridge DG, Raza M, Evans C, Johnson K, Betteridge E, Farr BW, Goodwin S, Quail MA, Scott C, Shirley L, Thurston SAJ, Rajan D, Bronner IF, Aigrain L, Redshaw NM, Lensing SV, McCarthy S, Makunin A, Balcazar CE, Gallagher MD, Williamson KA, Stanton TD, Michelsen ML, Warwick-Dugdale J, Manley R, Farbos A, Harrison JW, Sambles CM, Studholme DJ, Lackenby A, Mbisa T, Platt S, Miah S, Bibby D, Manso C, Hubb J, Dabrera G, Ramsay M, Bradshaw D, Schaefer U, Groves N, Gallagher E, Lee D, Williams D, Ellaby N, Hartman H, Manesis N, Patel V, Ledesma J, Twohig KA, Allara E, Pearson C, Cheng JKJ, Bridgewater HE, Frost LR, Taylor-Joyce G, Brown PE, Tong L, Broos A, Mair D, Nichols J, Carmichael SN, Smollett KL, Nomikou K, Aranday-Cortes E, Johnson N, Nickbakhsh S, Vamos EE, Hughes M, Rainbow L, Eccles R, Nelson C, Whitehead M, Gregory R, Gemmell M, Wierzbicki C, Webster HJ, Fisher CL, Signell AW, Betancor G, Wilson HD, Nebbia G, Flaviani F, Cerda AC, Merrill TV, Wilson RE, Cotic M, Bayzid N, Thompson T, Acheson E, Rushton S, O’Brien S, Baker DJ, Rudder S, Aydin A, Sang F, Debebe J, Francois S, Vasylyeva TI, Zamudio ME, Gutierrez B, Marchbank A, Maksimovic J, Spellman K, McCluggage K, Morgan M, Beer R, Afifi S, Workman T, Fuller W, Bresner C, Angyal A, Green LR, Parsons PJ, Tucker RM, Brown R, Whiteley M, Bonfield J, Puethe C, Whitwham A, Liddle J, Rowe W, Siveroni I, Le-Viet T, Gaskin A, Johnson R, Abnizova I, Aigrain L, Alderton A, Ali M, Allen L, Amato R, Anderson R, Ariani C, Austin-Guest S, Bala S, Barrett J, Bassett A, Battleday K, Beal J, Beale M, Beaver C, Bellany S, Bellerby T, Bellis K, Berger D, Berriman M, Betteridge E, Bevan P, Binley S, Bishop J, Blackburn K, Bonfield J, Boughton N, Bowker S, Brendler-Spaeth T, Bronner I, Brooklyn T, Buddenborg SK, Bush R, Caetano C, Cagan A, Carter N, Cartwright J, Monteiro TC, Chapman L, Chillingworth T-J, Clapham P, Clark R, Clarke A, Clarke C, Cole D, Cook E, Coppola M, Cornell L, Cornwell C, Corton C, Crackett A, Cranage A, Craven H, Craw S, Crawford M, Cutts T, Dabrowska M, Davies M, Davies R, Dawson J, Day C, Densem A, Dibling T, Dockree C, Dodd D, Dogga S, Dorman M, Dougan G, Dougherty M, Dove A, Drummond L, Drury E, Dudek M, Durham J, Durrant L, Easthope E, Eckert S, Ellis P, Farr B, Fenton M, Ferrero M, Flack N, Fordham H, Forsythe G, Foulser L, Francis M, Fraser A, Freeman A, Galvin A, Garcia-Casado M, Gedny A, Girgis S, Glover J, Goncalves S, Goodwin S, Gould O, Gourtovaia M, Gray A, Gray E, Griffiths C, Gu Y, Guerin F, Hamilton W, Hanks H, Harrison E, Harrott A, Harry E, Harvison J, Heath P, Hernandez-Koutoucheva A, Hobbs R, Holland D, Holmes S, Hornett G, Hough N, Huckle L, Hughes-Hallet L, Hunter A, Inglis S, Iqbal S, Jackson A, Jackson D, James K, Jamrozy D, Verdejo CJ, Johnston I, Jones M, Kallepally K, Kane L, Kay K, Kay S, Keatley J, Keith A, King A, Kitchin L, Kleanthous M, Klimekova M, Korlevic P, Krasheninnkova K, Kwiatkowski D, Lane G, Langford C, Laverack A, Law K, Lawniczak M, Lensing S, Leonard S, Letchford L, Lewis K, Lewis-Wade A, Liddle J, Lin Q, Lindsay S, Linsdell S, Livett R, Lo S, Long R, Lovell J, Lovell J, Ludden C, Mack J, Maddison M, Makunin A, Mamun I, Mansfield J, Marriott N, Martin M, Martincorena I, Mayho M, McCarthy S, McClintock J, McGuigan S, McHugh S, McMinn L, Meadows C, Mobley E, Moll R, Morra M, Morrow L, Murie K, Nash S, Nathwani C, Naydenova P, Neaverson A, Nelson R, Nerou E, Nicholson J, Nimz T, Noell GG, O’Meara S, Ohan V, Oliver K, Olney C, Ormond D, Oszlanczi A, Palmer S, Pang YF, Pardubska B, Park N, Parmar A, Patel G, Patel M, Payne M, Peacock S, Petersen A, Plowman D, Preston T, Prestwood L, Puethe C, Quail M, Rajan D, Rajatileka S, Rance R, Rawlings S, Redshaw N, Reynolds J, Reynolds M, Rice S, Richardson M, Roberts C, Robinson K, Robinson M, Robinson D, Rogers H, Rojo EM, Roopra D, Rose M, Rudd L, Sadri R, Salmon N, Saul D, Schwach F, Scott C, Seekings P, Shirley L, Sillitoe J, Simms A, Sinnott M, Sivadasan S, Siwek B, Sizer D, Skeldon K, Skelton J, Slater-Tunstill J, Sloper L, Smerdon N, Smith C, Smith C, Smith J, Smith K, Smith M, Smith S, Smith T, Sneade L, Soria CD, Sousa C, Souster E, Sparkes A, Spencer-Chapman M, Squares J, Stanley R, Steed C, Stickland T, Still I, Stratton M, Strickland M, Swann A, Swiatkowska A, Sycamore N, Swift E, Symons E, Szluha S, Taluy E, Tao N, Taylor K, Taylor S, Thompson S, Thompson M, Thomson M, Thomson N, Thurston S, Tonkin-Hill G, Toombs D, Topping B, Tovar-Corona J, Ungureanu D, Uphill J, Urbanova J, Van PJ, Vancollie V, Voak P, Walker D, Walker M, Waller M, Ward G, Weatherhogg C, Webb N, Weldon D, Wells A, Wells E, Westwood L, Whipp T, Whiteley T, Whitton G, Whitwham A, Widaa S, Williams M, Wilson M, Wright S, Harvey W, Virgin HW, Lanzavecchia A, Piccoli L, Doffinger R, Wills M, Veesler D, Corti D, Gupta RK. 2021. <a href="https://doi.org/10.1038/s41586-021-03412-7">Sensitivity of SARS-CoV-2 B.1.1.7 to mRNA vaccine-elicited antibodies</a>. Nature 593:136–141.</div>
</div>
<div id="ref-VxRhUVyl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">523. </div><div class="csl-right-inline">Dearlove B, Lewitus E, Bai H, Li Y, Reeves DB, Joyce MG, Scott PT, Amare MF, Vasan S, Michael NL, Modjarrad K, Rolland M. 2020. <a href="https://doi.org/10.1073/pnas.2008281117">A SARS-CoV-2 vaccine candidate would likely match all currently circulating variants</a>. Proceedings of the National Academy of Sciences 117:23652–23662.</div>
</div>
<div id="ref-4sZmtyNk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">524. </div><div class="csl-right-inline">Sanjuán R, Nebot MR, Chirico N, Mansky LM, Belshaw R. 2010. <a href="https://doi.org/10.1128/jvi.00694-10">Viral Mutation Rates</a>. J Virol 84:9733–9748.</div>
</div>
<div id="ref-vESqa6V0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">525. </div><div class="csl-right-inline">Manzanares-Meza LD, Medina-Contreras O. 2020. <a href="https://doi.org/10.24875/bmhim.20000183">SARS-CoV-2 and influenza: a comparative overview and treatment implications</a>. BMHIM 77.</div>
</div>
<div id="ref-2pzbGZvL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">526. </div><div class="csl-right-inline">Li X, Deem MW. 2016. <a href="https://doi.org/10.1093/protein/gzw017">Influenza evolution and H3N2 vaccine effectiveness, with application to the 2014/2015 season</a>. Protein Engineering, Design and Selection 29:309–315.</div>
</div>
<div id="ref-LxJvckNs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">527. </div><div class="csl-right-inline">Liu Y, Liu J, Xia H, Zhang X, Fontes-Garfias CR, Swanson KA, Cai H, Sarkar R, Chen W, Cutler M, Cooper D, Weaver SC, Muik A, Sahin U, Jansen KU, Xie X, Dormitzer PR, Shi P-Y. 2021. <a href="https://doi.org/10.1056/nejmc2102017">Neutralizing Activity of BNT162b2-Elicited Serum</a>. N Engl J Med 384:1466–1468.</div>
</div>
<div id="ref-ZxfNX9xk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">528. </div><div class="csl-right-inline">2021. Covid-19 vaccine effectiveness affected by variants. Pharmaceutical Technology. <a href="https://www.pharmaceutical-technology.com/comment/covid-19-vaccine-effectiveness-affected-by-variants/">https://www.pharmaceutical-technology.com/comment/covid-19-vaccine-effectiveness-affected-by-variants/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-lY0XUlUp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">529. </div><div class="csl-right-inline">The effects of virus variants on COVID-19 vaccines. <a href="https://www.who.int/news-room/feature-stories/detail/the-effects-of-virus-variants-on-covid-19-vaccines">https://www.who.int/news-room/feature-stories/detail/the-effects-of-virus-variants-on-covid-19-vaccines</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-YlAWEwlx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">530. </div><div class="csl-right-inline">Bonomo ME, Deem MW. 2018. <a href="https://doi.org/10.1093/cid/ciy323">Predicting Influenza H3N2 Vaccine Efficacy From Evolution of the Dominant Epitope</a>. Clinical Infectious Diseases 67:1129–1131.</div>
</div>
<div id="ref-180UFKjJ2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">531. </div><div class="csl-right-inline">Kim JH, Marks F, Clemens JD. 2021. <a href="https://doi.org/10.1038/s41591-021-01230-y">Looking beyond COVID-19 vaccine phase 3 trials</a>. Nat Med 27:205–211.</div>
</div>
<div id="ref-3cPwqjhj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">532. </div><div class="csl-right-inline">Mathieu E, Ritchie H, Rodés-Guirao L, Appel C, Giattino C, Hasell J, Macdonald B, Dattani S, Beltekian D, Ortiz-Ospina E, Roser M. 2020. <a href="https://ourworldindata.org/coronavirus">Coronavirus Pandemic (COVID-19)</a>. Our World in Data.</div>
</div>
<div id="ref-m99E85qV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">533. </div><div class="csl-right-inline">Knyazev S, Chhugani K, Sarwal V, Ayyala R, Singh H, Karthikeyan S, Deshpande D, Baykal PI, Comarova Z, Lu A, Porozov Y, Vasylyeva TI, Wertheim JO, Tierney BT, Chiu CY, Sun R, Wu A, Abedalthagafi MS, Pak VM, Nagaraj SH, Smith AL, Skums P, Pasaniuc B, Komissarov A, Mason CE, Bortz E, Lemey P, Kondrashov F, Beerenwinkel N, Lam TT-Y, Wu NC, Zelikovsky A, Knight R, Crandall KA, Mangul S. 2022. <a href="https://doi.org/10.1038/s41592-022-01444-z">Unlocking capacities of genomics for the COVID-19 response and future pandemics</a>. Nat Methods 19:374–380.</div>
</div>
<div id="ref-8PK1sfVT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">534. </div><div class="csl-right-inline">Aarestrup FM, Bonten M, Koopmans M. 2021. <a href="https://doi.org/10.1016/j.lanepe.2021.100210">Pandemics– One Health preparedness for the next</a>. The Lancet Regional Health - Europe 9:100210.</div>
</div>
<div id="ref-ro76PcuD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">535. </div><div class="csl-right-inline">Li J, Lai S, Gao GF, Shi W. 2021. <a href="https://doi.org/10.1038/s41586-021-04188-6">The emergence, genomic diversity and global spread of SARS-CoV-2</a>. Nature 600:408–418.</div>
</div>
<div id="ref-42TFgI1p" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">536. </div><div class="csl-right-inline">Tessema SK, Inzaule SC, Christoffels A, Kebede Y, de Oliveira T, Ouma AEO, Happi CT, Nkengasong JN. 2020. <a href="https://doi.org/10.1016/s2666-5247(20)30117-8">Accelerating genomics-based surveillance for COVID-19 response in Africa</a>. The Lancet Microbe 1:e227–e228.</div>
</div>
<div id="ref-1DqxOlpHl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">537. </div><div class="csl-right-inline">Hart OE, Halden RU. 2020. <a href="https://doi.org/10.1016/j.scitotenv.2020.138875">Computational analysis of SARS-CoV-2/COVID-19 surveillance by wastewater-based epidemiology locally and globally: Feasibility, economy, opportunities and challenges</a>. Science of The Total Environment 730:138875.</div>
</div>
<div id="ref-vUXYeKry" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">538. </div><div class="csl-right-inline">Medema G, Been F, Heijnen L, Petterson S. 2020. <a href="https://doi.org/10.1016/j.coesh.2020.09.006">Implementation of environmental surveillance for SARS-CoV-2 virus to support public health decisions: Opportunities and challenges</a>. Current Opinion in Environmental Science &amp;amp; Health 17:49–71.</div>
</div>
<div id="ref-MWDWrPGu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">539. </div><div class="csl-right-inline">Ahmed W, Bivins A, Bertsch PM, Bibby K, Choi PM, Farkas K, Gyawali P, Hamilton KA, Haramoto E, Kitajima M, Simpson SL, Tandukar S, Thomas KV, Mueller JF. 2020. <a href="https://doi.org/10.1016/j.coesh.2020.09.003">Surveillance of SARS-CoV-2 RNA in wastewater: Methods optimization and quality control are crucial for generating reliable public health information</a>. Current Opinion in Environmental Science &amp;amp; Health 17:82–93.</div>
</div>
<div id="ref-TPWaxApy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">540. </div><div class="csl-right-inline">Wu Y, Guo C, Tang L, Hong Z, Zhou J, Dong X, Yin H, Xiao Q, Tang Y, Qu X, Kuang L, Fang X, Mishra N, Lu J, Shan H, Jiang G, Huang X. 2020. <a href="https://doi.org/10.1016/s2468-1253(20)30083-2">Prolonged presence of SARS-CoV-2 viral RNA in faecal samples</a>. The Lancet Gastroenterology &amp; Hepatology 5:434–435.</div>
</div>
<div id="ref-Gx2TMx2B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">541. </div><div class="csl-right-inline">Trujillo M, Cheung K, Gao A, Hoxie I, Kannoly S, Kubota N, San KM, Smyth DS, Dennehy JJ. 2021. <a href="https://doi.org/10.1371/journal.pone.0257454">Protocol for safe, affordable, and reproducible isolation and quantitation of SARS-CoV-2 RNA from wastewater</a>. PLoS ONE 16:e0257454.</div>
</div>
<div id="ref-1EuKt1ppV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">542. </div><div class="csl-right-inline">Hoar C, Chauvin F, Clare A, McGibbon H, Castro E, Patinella S, Katehis D, Dennehy JJ, Trujillo M, Smyth DS, Silverman AI. 2022. <a href="https://doi.org/10.1039/d1ew00747e">Monitoring SARS-CoV-2 in wastewater during New York City's second wave of COVID-19: sewershed-level trends and relationships to publicly available clinical testing data</a>. Environ Sci: Water Res Technol 8:1021–1035.</div>
</div>
<div id="ref-JwtDsxLy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">543. </div><div class="csl-right-inline">Gerrity D, Papp K, Stoker M, Sims A, Frehner W. 2021. <a href="https://doi.org/10.1016/j.wroa.2020.100086">Early-pandemic wastewater surveillance of SARS-CoV-2 in Southern Nevada: Methodology, occurrence, and incidence/prevalence considerations</a>. Water Research X 10:100086.</div>
</div>
<div id="ref-8A0XYWq8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">544. </div><div class="csl-right-inline">Wu F, Zhang J, Xiao A, Gu X, Lee WL, Armas F, Kauffman K, Hanage W, Matus M, Ghaeli N, Endo N, Duvallet C, Poyet M, Moniz K, Washburne AD, Erickson TB, Chai PR, Thompson J, Alm EJ. 2020. <a href="https://doi.org/10.1128/msystems.00614-20">SARS-CoV-2 Titers in Wastewater Are Higher than Expected from Clinically Confirmed Cases</a>. mSystems 5.</div>
</div>
<div id="ref-tYFkf9ez" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">545. </div><div class="csl-right-inline">Schussman MK, Roguet A, Schmoldt A, Dinan B, McLellan SL. 2022. <a href="https://doi.org/10.1101/2022.02.18.22271188">Wastewater surveillance using ddPCR reveals highly accurate tracking of Omicron variant due to altered N1 probe binding efficiency</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-1783YNtiU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">546. </div><div class="csl-right-inline">Lee WL, Gu X, Armas F, Wu F, Chandra F, Chen H, Xiao A, Leifels M, Chua FJD, Kwok GW, Tay JY, Lim CY, Thompson J, Alm EJ. 2021. <a href="https://doi.org/10.1101/2021.12.21.21268077">Quantitative detection of SARS-CoV-2 Omicron BA.1 and BA.2 variants in wastewater through allele-specific RT-qPCR</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-8IiPEZuE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">547. </div><div class="csl-right-inline">CDC. 2022. National Wastewater Surveillance System. Centers for Disease Control and Prevention. <a href="https://www.cdc.gov/healthywater/surveillance/wastewater-surveillance/wastewater-surveillance.html">https://www.cdc.gov/healthywater/surveillance/wastewater-surveillance/wastewater-surveillance.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-MicGQlWa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">548. </div><div class="csl-right-inline">Vavrek D, Speroni L, Curnow KJ, Oberholzer M, Moeder V, Febbo PG. 2021. <a href="https://doi.org/10.1101/2021.01.12.21249613">Genomic surveillance at scale is required to detect newly emerging strains at an early timepoint</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-tvGu9sHq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">549. </div><div class="csl-right-inline">Dobson AP, Pimm SL, Hannah L, Kaufman L, Ahumada JA, Ando AW, Bernstein A, Busch J, Daszak P, Engelmann J, Kinnaird MF, Li BV, Loch-Temzelides T, Lovejoy T, Nowak K, Roehrdanz PR, Vale MM. 2020. <a href="https://doi.org/10.1126/science.abc3189">Ecology and economics for pandemic prevention</a>. Science 369:379–381.</div>
</div>
<div id="ref-k89g6p8V" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">550. </div><div class="csl-right-inline">Carlson CJ, Albery GF, Merow C, Trisos CH, Zipfel CM, Eskew EA, Olival KJ, Ross N, Bansal S. 2022. <a href="https://doi.org/10.1038/s41586-022-04788-w">Climate change increases cross-species viral transmission risk</a>. Nature 607:555–562.</div>
</div>
<div id="ref-RBJyozoI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">551. </div><div class="csl-right-inline">Neiderud C-J. 2015. <a href="https://doi.org/10.3402/iee.v5.27060">How urbanization affects the epidemiology of emerging infectious diseases</a>. Infection Ecology &amp;amp; Epidemiology 5:27060.</div>
</div>
<div id="ref-164cqH5RU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">552. </div><div class="csl-right-inline">Matukas LM, Dhalla IA, Laupacis A. 2020. <a href="https://doi.org/10.1503/cmaj.202120">Aggressively find, test, trace and isolate to beat COVID-19</a>. CMAJ 192:E1164–E1165.</div>
</div>
<div id="ref-X5zWe6U8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">553. </div><div class="csl-right-inline">Jefferies S, French N, Gilkison C, Graham G, Hope V, Marshall J, McElnay C, McNeill A, Muellner P, Paine S, Prasad N, Scott J, Sherwood J, Yang L, Priest P. 2020. <a href="https://doi.org/10.1016/s2468-2667(20)30225-5">COVID-19 in New Zealand and the impact of the national response: a descriptive epidemiological study</a>. The Lancet Public Health 5:e612–e623.</div>
</div>
<div id="ref-OeNRM6OO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">554. </div><div class="csl-right-inline">Summers J, Cheng H-Y, Lin H-H, Barnard LT, Kvalsvig A, Wilson N, Baker MG. 2020. <a href="https://doi.org/10.1016/j.lanwpc.2020.100044">Potential lessons from the Taiwan and New Zealand health responses to the COVID-19 pandemic</a>. The Lancet Regional Health - Western Pacific 4:100044.</div>
</div>
<div id="ref-vlGP3RAU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">555. </div><div class="csl-right-inline">Novel Coronavirus – China. <a href="https://www.who.int/emergencies/disease-outbreak-news/item/2020-DON233">https://www.who.int/emergencies/disease-outbreak-news/item/2020-DON233</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1AHtg71LX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">556. </div><div class="csl-right-inline">Corman VM, Landt O, Kaiser M, Molenkamp R, Meijer A, Chu DK, Bleicker T, Brünink S, Schneider J, Schmidt ML, Mulders DG, Haagmans BL, van der Veer B, van den Brink S, Wijsman L, Goderski G, Romette J-L, Ellis J, Zambon M, Peiris M, Goossens H, Reusken C, Koopmans MP, Drosten C. 2020. <a href="https://doi.org/10.2807/1560-7917.es.2020.25.3.2000045">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</a>. Eurosurveillance 25.</div>
</div>
<div id="ref-4aV8UnGs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">557. </div><div class="csl-right-inline">Garibyan L, Avashia N. 2013. <a href="https://doi.org/10.1038/jid.2013.1">Polymerase Chain Reaction</a>. Journal of Investigative Dermatology 133:1–4.</div>
</div>
<div id="ref-PCd1Mvel" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">558. </div><div class="csl-right-inline">Ngom B, Guo Y, Wang X, Bi D. 2010. <a href="https://doi.org/10.1007/s00216-010-3661-4">Development and application of lateral flow test strip technology for detection of infectious agents and chemical contaminants: a review</a>. Anal Bioanal Chem 397:1113–1135.</div>
</div>
<div id="ref-w3RMVma8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">559. </div><div class="csl-right-inline">Alhajj M, Farhana A. 2022. Enzyme Linked Immunosorbent AssayStatPearls. StatPearls. <a href="https://pubmed.ncbi.nlm.nih.gov/32310382">https://pubmed.ncbi.nlm.nih.gov/32310382</a>.</div>
</div>
<div id="ref-ntuOKrKt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">560. </div><div class="csl-right-inline">Gong F, Wei H, Li Q, Liu L, Li B. 2021. <a href="https://doi.org/10.3389/fmolb.2021.682405">Evaluation and Comparison of Serological Methods for COVID-19 Diagnosis</a>. Front Mol Biosci 8.</div>
</div>
<div id="ref-sTAHOyh1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">561. </div><div class="csl-right-inline">Burbelo PD, Riedo FX, Morishima C, Rawlings S, Smith D, Das S, Strich JR, Chertow DS, Davey RT Jr, Cohen JI. 2020. <a href="https://doi.org/10.1093/infdis/jiaa273">Sensitivity in Detection of Antibodies to Nucleocapsid and Spike Proteins of Severe Acute Respiratory Syndrome Coronavirus 2 in Patients With Coronavirus Disease 2019</a>. The Journal of Infectious Diseases 222:206–213.</div>
</div>
<div id="ref-fCZXxoas" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">562. </div><div class="csl-right-inline">To KK-W, Tsang OT-Y, Leung W-S, Tam AR, Wu T-C, Lung DC, Yip CC-Y, Cai J-P, Chan JM-C, Chik TS-H, Lau DP-L, Choi CY-C, Chen L-L, Chan W-M, Chan K-H, Ip JD, Ng AC-K, Poon RW-S, Luo C-T, Cheng VC-C, Chan JF-W, Hung IF-N, Chen Z, Chen H, Yuen K-Y. 2020. <a href="https://doi.org/10.1016/s1473-3099(20)30196-1">Temporal profiles of viral load in posterior oropharyngeal saliva samples and serum antibody responses during infection by SARS-CoV-2: an observational cohort study</a>. The Lancet Infectious Diseases 20:565–574.</div>
</div>
<div id="ref-oNFnjRBI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">563. </div><div class="csl-right-inline">Zhang J, Wang S, Xue Y. 2020. <a href="https://doi.org/10.1002/jmv.25742">Fecal specimen diagnosis 2019 novel coronavirus–infected pneumonia</a>. Journal of Medical Virology 92:680–682.</div>
</div>
<div id="ref-1FvL1dUZ6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">564. </div><div class="csl-right-inline">Adams G. 2020. <a href="https://doi.org/10.1042/bio20200034">A beginner’s guide to RT-PCR, qPCR and RT-qPCR</a>. The Biochemist 42:48–53.</div>
</div>
<div id="ref-YEYTESKL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">565. </div><div class="csl-right-inline">Bustin SA, Benes V, Garson JA, Hellemans J, Huggett J, Kubista M, Mueller R, Nolan T, Pfaffl MW, Shipley GL, Vandesompele J, Wittwer CT. 2009. <a href="https://doi.org/10.1373/clinchem.2008.112797">The MIQE Guidelines: Minimum Information for Publication of Quantitative Real-Time PCR Experiments</a>. Clinical Chemistry 55:611–622.</div>
</div>
<div id="ref-COEfjKo5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">566. </div><div class="csl-right-inline">Chu DKW, Pan Y, Cheng SMS, Hui KPY, Krishnan P, Liu Y, Ng DYM, Wan CKC, Yang P, Wang Q, Peiris M, Poon LLM. 2020. <a href="https://doi.org/10.1093/clinchem/hvaa029">Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia</a>. Clinical Chemistry 66:549–555.</div>
</div>
<div id="ref-IzbYsyNs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">567. </div><div class="csl-right-inline">La Scola B, Le Bideau M, Andreani J, Hoang VT, Grimaldier C, Colson P, Gautret P, Raoult D. 2020. <a href="https://doi.org/10.1007/s10096-020-03913-9">Viral RNA load as determined by cell culture as a management tool for discharge of SARS-CoV-2 patients from infectious disease wards</a>. Eur J Clin Microbiol Infect Dis 39:1059–1061.</div>
</div>
<div id="ref-jWdI1NWR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">568. </div><div class="csl-right-inline">van Kampen JJA, van de Vijver DAMC, Fraaij PLA, Haagmans BL, Lamers MM, Okba N, van den Akker JPC, Endeman H, Gommers DAMPJ, Cornelissen JJ, Hoek RAS, van der Eerden MM, Hesselink DA, Metselaar HJ, Verbon A, de Steenwinkel JEM, Aron GI, van Gorp ECM, van Boheemen S, Voermans JC, Boucher CAB, Molenkamp R, Koopmans MPG, Geurtsvankessel C, van der Eijk AA. 2021. <a href="https://doi.org/10.1038/s41467-020-20568-4">Duration and key determinants of infectious virus shedding in hospitalized patients with coronavirus disease-2019 (COVID-19)</a>. Nat Commun 12.</div>
</div>
<div id="ref-MgfhyyrD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">569. </div><div class="csl-right-inline">Dahdouh E, Lázaro-Perona F, Romero-Gómez MP, Mingorance J, García-Rodriguez J. 2021. <a href="https://doi.org/10.1016/j.jinf.2020.10.017">Ct values from SARS-CoV-2 diagnostic PCR assays should not be used as direct estimates of viral load</a>. Journal of Infection 82:414–451.</div>
</div>
<div id="ref-WuM4npVP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">570. </div><div class="csl-right-inline">Wikramaratna PS, Paton RS, Ghafari M, Lourenço J. 2020. <a href="https://doi.org/10.2807/1560-7917.es.2020.25.50.2000568">Estimating the false-negative test probability of SARS-CoV-2 by RT-PCR</a>. Eurosurveillance 25.</div>
</div>
<div id="ref-cASBrIxl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">571. </div><div class="csl-right-inline">Cohen AN, Kessel B, Milgroom MG. 2020. <a href="https://doi.org/10.1101/2020.04.26.20080911">Diagnosing SARS-CoV-2 infection: the danger of over-reliance on positive test results</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-KcasNvUc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">572. </div><div class="csl-right-inline">Tsang NNY, So HC, Ng KY, Cowling BJ, Leung GM, Ip DKM. 2021. <a href="https://doi.org/10.1016/s1473-3099(21)00146-8">Diagnostic performance of different sampling approaches for SARS-CoV-2 RT-PCR testing: a systematic review and meta-analysis</a>. The Lancet Infectious Diseases 21:1233–1245.</div>
</div>
<div id="ref-n7y3AZZ1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">573. </div><div class="csl-right-inline">Chan JF-W, Yip CC-Y, To KK-W, Tang TH-C, Wong SC-Y, Leung K-H, Fung AY-F, Ng AC-K, Zou Z, Tsoi H-W, Choi GK-Y, Tam AR, Cheng VC-C, Chan K-H, Tsang OT-Y, Yuen K-Y. 2020. <a href="https://doi.org/10.1128/jcm.00310-20">Improved Molecular Diagnosis of COVID-19 by the Novel, Highly Sensitive and Specific COVID-19-RdRp/Hel Real-Time Reverse Transcription-PCR Assay Validated In Vitro and with Clinical Specimens</a>. J Clin Microbiol 58.</div>
</div>
<div id="ref-SeNutdHD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">574. </div><div class="csl-right-inline">Mercer TR, Salit M. 2021. <a href="https://doi.org/10.1038/s41576-021-00360-w">Testing at scale during the COVID-19 pandemic</a>. Nat Rev Genet 22:415–426.</div>
</div>
<div id="ref-LbnvnCrn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">575. </div><div class="csl-right-inline">Ziegler K, Steininger P, Ziegler R, Steinmann J, Korn K, Ensser A. 2020. <a href="https://doi.org/10.2807/1560-7917.es.2020.25.39.2001650">SARS-CoV-2 samples may escape detection because of a single point mutation in the N gene</a>. Eurosurveillance 25.</div>
</div>
<div id="ref-XsXhZSA8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">576. </div><div class="csl-right-inline">Peñarrubia L, Ruiz M, Porco R, Rao SN, Juanola-Falgarona M, Manissero D, López-Fontanals M, Pareja J. 2020. <a href="https://doi.org/10.1016/j.ijid.2020.06.027">Multiple assays in a real-time RT-PCR SARS-CoV-2 panel can mitigate the risk of loss of sensitivity by new genomic variants during the COVID-19 outbreak</a>. International Journal of Infectious Diseases 97:225–229.</div>
</div>
<div id="ref-dEBOtTUz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">577. </div><div class="csl-right-inline">Taylor SC, Laperriere G, Germain H. 2017. <a href="https://doi.org/10.1038/s41598-017-02217-x">Droplet Digital PCR versus qPCR for gene expression analysis with low abundant targets: from variable nonsense to publication quality data</a>. Sci Rep 7.</div>
</div>
<div id="ref-Hl2pASeg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">578. </div><div class="csl-right-inline">Quan P-L, Sauzade M, Brouzes E. 2018. <a href="https://doi.org/10.3390/s18041271">dPCR: A Technology Review</a>. Sensors 18:1271.</div>
</div>
<div id="ref-2WgqQLF8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">579. </div><div class="csl-right-inline">Suo T, Liu X, Feng J, Guo M, Hu W, Guo D, Ullah H, Yang Y, Zhang Q, Wang X, Sajid M, Huang Z, Deng L, Chen T, Liu F, Xu K, Liu Y, Zhang Q, Liu Y, Xiong Y, Chen G, Lan K, Chen Y. 2020. <a href="https://doi.org/10.1080/22221751.2020.1772678">ddPCR: a more accurate tool for SARS-CoV-2 detection in low viral load specimens</a>. Emerging Microbes &amp; Infections 9:1259–1268.</div>
</div>
<div id="ref-duB3pL6y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">580. </div><div class="csl-right-inline">Dong L, Zhou J, Niu C, Wang Q, Pan Y, Sheng S, Wang X, Zhang Y, Yang J, Liu M, Zhao Y, Zhang X, Zhu T, Peng T, Xie J, Gao Y, Wang D, Dai X, Fang X. 2021. <a href="https://doi.org/10.1016/j.talanta.2020.121726">Highly accurate and sensitive diagnostic detection of SARS-CoV-2 by digital PCR</a>. Talanta 224:121726.</div>
</div>
<div id="ref-jr3z6jGU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">581. </div><div class="csl-right-inline">Hess JF, Kohl TA, Kotrová M, Rönsch K, Paprotka T, Mohr V, Hutzenlaub T, Brüggemann M, Zengerle R, Niemann S, Paust N. 2020. <a href="https://doi.org/10.1016/j.biotechadv.2020.107537">Library preparation for next generation sequencing: A review of automation strategies</a>. Biotechnology Advances 41:107537.</div>
</div>
<div id="ref-43modeVu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">582. </div><div class="csl-right-inline">Udugama B, Kadhiresan P, Kozlowski HN, Malekjahani A, Osborne M, Li VYC, Chen H, Mubareka S, Gubbay JB, Chan WCW. 2020. <a href="https://doi.org/10.1021/acsnano.0c02624">Diagnosing COVID-19: The Disease and Tools for Detection</a>. ACS Nano 14:3822–3835.</div>
</div>
<div id="ref-6NDTv0Dw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">583. </div><div class="csl-right-inline">Robishaw JD, Alter SM, Solano JJ, Shih RD, DeMets DL, Maki DG, Hennekens CH. 2021. <a href="https://doi.org/10.1016/s2666-5247(21)00121-x">Genomic surveillance to combat COVID-19: challenges and opportunities</a>. The Lancet Microbe 2:e481–e484.</div>
</div>
<div id="ref-O7L5UTpW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">584. </div><div class="csl-right-inline">Cov-Lineages. <a href="https://cov-lineages.org/">https://cov-lineages.org/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-14QdeINFG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">585. </div><div class="csl-right-inline">Brito AF, Semenova E, Dudas G, Hassler GW, Kalinich CC, Kraemer MUG, Ho J, Tegally H, Githinji G, Agoti CN, Matkin LE, Whittaker C, Howden BP, Sintchenko V, Zuckerman NS, Mor O, Blankenship HM, Oliveira T de, Lin RTP, Siqueira MM, Resende PC, Vasconcelos ATR, Spilki FR, Aguiar RS, Alexiev I, Ivanov IN, Philipova I, Carrington CVF, Sahadeo NSD, Gurry C, Maurer-Stroh S, Naidoo D, von Eije KJ, Perkins MD, Kerkhove M van, Hill SC, Sabino EC, Pybus OG, Dye C, Bhatt S, Flaxman S, Suchard MA, Grubaugh ND, Baele G, Faria NR. 2021. <a href="https://doi.org/10.1101/2021.08.21.21262393">Global disparities in SARS-CoV-2 genomic surveillance</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-nIW5kgMT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">586. </div><div class="csl-right-inline">Yelin I, Aharony N, Tamar ES, Argoetti A, Messer E, Berenbaum D, Shafran E, Kuzli A, Gandali N, Shkedi O, Hashimshony T, Mandel-Gutfreund Y, Halberthal M, Geffen Y, Szwarcwort-Cohen M, Kishony R. 2020. <a href="https://doi.org/10.1093/cid/ciaa531">Evaluation of COVID-19 RT-qPCR Test in Multi sample Pools</a>. Clinical Infectious Diseases 71:2073–2078.</div>
</div>
<div id="ref-IQ5kUQEO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">587. </div><div class="csl-right-inline">Nelson AC, Auch B, Schomaker M, Gohl DM, Grady P, Johnson D, Kincaid R, Karnuth KE, Daniel J, Fiege JK, Fay EJ, Bold T, Langlois RA, Beckman KB, Yohe S. 2020. Analytical Validation of a COVID-19 qRT-PCR Detection Assay Using a 384-well Format and Three Extraction Methods. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.04.02.022186">10.1101/2020.04.02.022186</a>.</div>
</div>
<div id="ref-KjeNmVZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">588. </div><div class="csl-right-inline">Heidarzadeh A, Narayanan K. 2021. <a href="https://doi.org/10.1109/icassp39728.2021.9413685">Two-Stage Adaptive Pooling with RT-QPCR for Covid-19 Screening</a>ICASSP 2021 - 2021 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP). IEEE.</div>
</div>
<div id="ref-161uMPkFJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">589. </div><div class="csl-right-inline">Joachim A, Dewald F, Suárez I, Zemlin M, Lang I, Stutz R, Marthaler A, Bosse HM, Lübke N, Münch J, Bernard M-A, Jeltsch K, Tönshoff B, Weidner N, Kräusslich H-G, Birzele L, Hübner J, Schmied P, Meyer-Bühn M, Horemheb-Rubio G, Cornely OA, Haverkamp H, Wiesmüller G, Fätkenheuer G, Hero B, Kaiser R, Dötsch J, Rybniker J, Cosgun ZC, Hünseler C, Schönenkorb J, Wurm J, Klein F, Heger E, Knops E, Sierra-Aragón S, Kretschmer AC, Sprute R, Kossow A, Hellmich M, Shah-Hosseini K, Weiss M, Goedicke-Fritz S, Kaiser E, Meyer S, Seiwert N, Smola S, Pfuhl T, Lohse S, Schupp A-K, Timm J, Gröne N, Lesmann H, Bredahl R, Schneble L, Turinsky M, Patry C, Hoffmann GF, Müller B, Börner K, Schnitzler P, Heuser A-M, Welker A, von Both U, Kern A. 2021. <a href="https://doi.org/10.1016/j.eclinm.2021.101082">Pooled RT-qPCR testing for SARS-CoV-2 surveillance in schools - a cluster randomised trial</a>. EClinicalMedicine 39:101082.</div>
</div>
<div id="ref-9KChw8BC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">590. </div><div class="csl-right-inline">Hirotsu Y, Maejima M, Shibusawa M, Nagakubo Y, Hosaka K, Amemiya K, Sueki H, Hayakawa M, Mochizuki H, Tsutsui T, Kakizaki Y, Miyashita Y, Omata M. 2020. <a href="https://doi.org/10.1038/s41598-020-76043-z">Pooling RT-qPCR testing for SARS-CoV-2 in 1000 individuals of healthy and infection-suspected patients</a>. Sci Rep 10.</div>
</div>
<div id="ref-mk0qreRK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">591. </div><div class="csl-right-inline">Notomi T. 2000. <a href="https://doi.org/10.1093/nar/28.12.e63">Loop-mediated isothermal amplification of DNA</a>. Nucleic Acids Research 28:63e–63.</div>
</div>
<div id="ref-NzLjxhcg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">592. </div><div class="csl-right-inline">Amaral C, Antunes W, Moe E, Duarte AG, Lima LMP, Santos C, Gomes IL, Afonso GS, Vieira R, Teles HSS, Reis MS, da Silva MAR, Henriques AM, Fevereiro M, Ventura MR, Serrano M, Pimentel C. 2021. <a href="https://doi.org/10.1038/s41598-021-95799-6">A molecular test based on RT-LAMP for rapid, sensitive and inexpensive colorimetric detection of SARS-CoV-2 in clinical samples</a>. Sci Rep 11.</div>
</div>
<div id="ref-qLIqQl48" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">593. </div><div class="csl-right-inline">Al Bayat S, Mundodan J, Hasnain S, Sallam M, Khogali H, Ali D, Alateeg S, Osama M, Elberdiny A, Al-Romaihi H, Al-Thani MHJ. 2021. <a href="https://doi.org/10.1016/j.jiph.2021.08.013">Can the cycle threshold (Ct) value of RT-PCR test for SARS CoV2 predict infectivity among close contacts?</a> Journal of Infection and Public Health 14:1201–1205.</div>
</div>
<div id="ref-8T6lbq9V" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">594. </div><div class="csl-right-inline">Mowrer CT, Creager H, Cawcutt K, Birge J, Lyden E, Van Schooneveld TC, Rupp ME, Hewlett A. 2021. <a href="https://doi.org/10.1017/ice.2021.132">Evaluation of cycle threshold values at deisolation</a>. Infect Control Hosp Epidemiol 43:794–796.</div>
</div>
<div id="ref-1H1RimUDg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">595. </div><div class="csl-right-inline">Tom MR, Mina MJ. 2020. <a href="https://doi.org/10.1093/cid/ciaa619">To Interpret the SARS-CoV-2 Test, Consider the Cycle Threshold Value</a>. Clinical Infectious Diseases 71:2252–2254.</div>
</div>
<div id="ref-HlmXwIYD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">596. </div><div class="csl-right-inline">Zhang Y, Odiwuor N, Xiong J, Sun L, Nyaruaba RO, Wei H, Tanner NA. 2020. <a href="https://doi.org/10.1101/2020.02.26.20028373">Rapid Molecular Detection of SARS-CoV-2 (COVID-19) Virus RNA Using Colorimetric LAMP</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-1HXZ5qohE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">597. </div><div class="csl-right-inline">Subali AD, Wiyono L. 2021. <a href="https://doi.org/10.1080/20477724.2021.1933335">Reverse Transcriptase Loop Mediated Isothermal Amplification (RT-LAMP) for COVID-19 diagnosis: a systematic review and meta-analysis.</a> Pathog Glob Health 115:281–291.</div>
</div>
<div id="ref-LDEN4bPG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">598. </div><div class="csl-right-inline">Adli M. 2018. <a href="https://doi.org/10.1038/s41467-018-04252-2">The CRISPR tool kit for genome editing and beyond</a>. Nat Commun 9.</div>
</div>
<div id="ref-vVBrQqWB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">599. </div><div class="csl-right-inline">Gootenberg JS, Abudayyeh OO, Lee JW, Essletzbichler P, Dy AJ, Joung J, Verdine V, Donghia N, Daringer NM, Freije CA, Myhrvold C, Bhattacharyya RP, Livny J, Regev A, Koonin EV, Hung DT, Sabeti PC, Collins JJ, Zhang F. 2017. <a href="https://doi.org/10.1126/science.aam9321">Nucleic acid detection with CRISPR-Cas13a/C2c2</a>. Science 356:438–442.</div>
</div>
<div id="ref-fqgjuNaJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">600. </div><div class="csl-right-inline">Hou T, Zeng W, Yang M, Chen W, Ren L, Ai J, Wu J, Liao Y, Gou X, Li Y, Wang X, Su H, Gu B, Wang J, Xu T. 2020. <a href="https://doi.org/10.1371/journal.ppat.1008705">Development and evaluation of a rapid CRISPR-based diagnostic for COVID-19</a>. PLoS Pathog 16:e1008705.</div>
</div>
<div id="ref-zSefi0ye" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">601. </div><div class="csl-right-inline">Metsky HC, Freije CA, Kosoko-Thoroddsen T-SF, Sabeti PC, Myhrvold C. 2020. CRISPR-based surveillance for COVID-19 using genomically-comprehensive machine learning design. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.26.967026">10.1101/2020.02.26.967026</a>.</div>
</div>
<div id="ref-160dYAlIO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">602. </div><div class="csl-right-inline">Rauch JN, Valois E, Solley SC, Braig F, Lach RS, Audouard M, Ponce-Rojas JC, Costello MS, Baxter NJ, Kosik KS, Arias C, Acosta-Alvear D, Wilson MZ. 2021. <a href="https://doi.org/10.1128/jcm.02402-20">A Scalable, Easy-to-Deploy Protocol for Cas13-Based Detection of SARS-CoV-2 Genetic Material</a>. J Clin Microbiol 59.</div>
</div>
<div id="ref-XWCinLld" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">603. </div><div class="csl-right-inline">Broughton JP, Deng X, Yu G, Fasching CL, Servellita V, Singh J, Miao X, Streithorst JA, Granados A, Sotomayor-Gonzalez A, Zorn K, Gopez A, Hsu E, Gu W, Miller S, Pan C-Y, Guevara H, Wadford DA, Chen JS, Chiu CY. 2020. <a href="https://doi.org/10.1038/s41587-020-0513-4">CRISPR–Cas12-based detection of SARS-CoV-2</a>. Nature Biotechnology 38:870–874.</div>
</div>
<div id="ref-4T3aId7a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">604. </div><div class="csl-right-inline">Lucia C, Federico P-B, Alejandra GC. 2020. An ultrasensitive, rapid, and portable coronavirus SARS-CoV-2 sequence detection method based on CRISPR-Cas12. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.29.971127">10.1101/2020.02.29.971127</a>.</div>
</div>
<div id="ref-qWJSwveX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">605. </div><div class="csl-right-inline">Xiong D, Dai W, Gong J, Li G, Liu N, Wu W, Pan J, Chen C, Jiao Y, Deng H, Ye J, Zhang X, Huang H, Li Q, Xue L, Zhang X, Tang G. 2020. <a href="https://doi.org/10.1371/journal.pbio.3000978">Rapid detection of SARS-CoV-2 with CRISPR-Cas12a</a>. PLoS Biol 18:e3000978.</div>
</div>
<div id="ref-wZKGHcO2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">606. </div><div class="csl-right-inline">Ding X, Yin K, Li Z, Lalla RV, Ballesteros E, Sfeir MM, Liu C. 2020. <a href="https://doi.org/10.1038/s41467-020-18575-6">Ultrasensitive and visual detection of SARS-CoV-2 using all-in-one dual CRISPR-Cas12a assay</a>. Nat Commun 11.</div>
</div>
<div id="ref-A8dUPgQd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">607. </div><div class="csl-right-inline">Guo L, Sun X, Wang X, Liang C, Jiang H, Gao Q, Dai M, Qu B, Fang S, Mao Y, Chen Y, Feng G, Gu Q, Wang RR, Zhou Q, Li W. 2020. <a href="https://doi.org/10.1038/s41421-020-0174-y">SARS-CoV-2 detection with CRISPR diagnostics</a>. Cell Discov 6.</div>
</div>
<div id="ref-jPdxTaCe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">608. </div><div class="csl-right-inline">Ramachandran A, Huyke DA, Sharma E, Sahoo MK, Huang C, Banaei N, Pinsky BA, Santiago JG. 2020. <a href="https://doi.org/10.1073/pnas.2010254117">Electric field-driven microfluidics for rapid CRISPR-based diagnostics and its application to detection of SARS-CoV-2</a>. Proc Natl Acad Sci USA 117:29518–29525.</div>
</div>
<div id="ref-16ItkGAe8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">609. </div><div class="csl-right-inline">Makarova KS, Koonin EV. 2015. <a href="https://doi.org/10.1007/978-1-4939-2687-9_4">Annotation and Classification of CRISPR-Cas Systems</a>, p. 47–75. <em>In</em> Methods in Molecular Biology. Springer New York.</div>
</div>
<div id="ref-RjmcDhmt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">610. </div><div class="csl-right-inline">Pyenson NC, Marraffini LA. 2017. <a href="https://doi.org/10.1016/j.mib.2017.08.003">Type III CRISPR-Cas systems: when DNA cleavage just isn’t enough</a>. Current Opinion in Microbiology 37:150–154.</div>
</div>
<div id="ref-1HTJW23NU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">611. </div><div class="csl-right-inline">Tamulaitis G, Venclovas Č, Siksnys V. 2017. <a href="https://doi.org/10.1016/j.tim.2016.09.012">Type III CRISPR-Cas Immunity: Major Differences Brushed Aside</a>. Trends in Microbiology 25:49–61.</div>
</div>
<div id="ref-QPNcH8m1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">612. </div><div class="csl-right-inline">Santiago-Frangos A, Hall LN, Nemudraia A, Nemudryi A, Krishna P, Wiegand T, Wilkinson RA, Snyder DT, Hedges JF, Cicha C, Lee HH, Graham A, Jutila MA, Taylor MP, Wiedenheft B. 2021. <a href="https://doi.org/10.1016/j.xcrm.2021.100319">Intrinsic signal amplification by type III CRISPR-Cas systems provides a sequence-specific SARS-CoV-2 diagnostic</a>. Cell Reports Medicine 2:100319.</div>
</div>
<div id="ref-ZwqOFOo8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">613. </div><div class="csl-right-inline">Fozouni P, Son S, Díaz de León Derby M, Knott GJ, Gray CN, D’Ambrosio MV, Zhao C, Switz NA, Kumar GR, Stephens SI, Boehm D, Tsou C-L, Shu J, Bhuiya A, Armstrong M, Harris AR, Chen P-Y, Osterloh JM, Meyer-Franke A, Joehnk B, Walcott K, Sil A, Langelier C, Pollard KS, Crawford ED, Puschnik AS, Phelps M, Kistler A, DeRisi JL, Doudna JA, Fletcher DA, Ott M. 2021. <a href="https://doi.org/10.1016/j.cell.2020.12.001">Amplification-free detection of SARS-CoV-2 with CRISPR-Cas13a and mobile phone microscopy</a>. Cell 184:323–333.e9.</div>
</div>
<div id="ref-KfuBLemx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">614. </div><div class="csl-right-inline">Brandsma E, Verhagen HJMP, van de Laar TJW, Claas ECJ, Cornelissen M, van den Akker E. 2020. <a href="https://doi.org/10.1093/infdis/jiaa641">Rapid, Sensitive, and Specific Severe Acute Respiratory Syndrome Coronavirus 2 Detection: A Multicenter Comparison Between Standard Quantitative Reverse-Transcriptase Polymerase Chain Reaction and CRISPR-Based DETECTR</a>. The Journal of Infectious Diseases 223:206–213.</div>
</div>
<div id="ref-15pFvXhZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">615. </div><div class="csl-right-inline">Azhar Mohd, Phutela R, Kumar M, Ansari AH, Rauthan R, Gulati S, Sharma N, Sinha D, Sharma S, Singh S, Acharya S, Paul D, Kathpalia P, Aich M, Sehgal P, Ranjan G, Bhoyar RC, Singhal K, Lad H, Patra PK, Makharia G, Chandak GR, Pesala B, Chakraborty D, Maiti S. 2020. <a href="https://doi.org/10.1101/2020.09.13.20193581">Rapid, accurate, nucleobase detection using FnCas9</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-Ki0q4jhw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">616. </div><div class="csl-right-inline">Drain PK. 2022. <a href="https://doi.org/10.1056/nejmcp2117115">Rapid Diagnostic Testing for SARS-CoV-2</a>. N Engl J Med 386:264–272.</div>
</div>
<div id="ref-XriI70m9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">617. </div><div class="csl-right-inline">Alghounaim M, Bastaki H, Bin Essa F, Motlagh H, Al-Sabah S. 2021. <a href="https://doi.org/10.3389/fmed.2021.797109">The Performance of Two Rapid Antigen Tests During Population-Level Screening for SARS-CoV-2 Infection</a>. Front Med 8.</div>
</div>
<div id="ref-nD6CEooX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">618. </div><div class="csl-right-inline">Posthuma-Trumpie GA, Korf J, van Amerongen A. 2008. <a href="https://doi.org/10.1007/s00216-008-2287-2">Lateral flow (immuno)assay: its strengths, weaknesses, opportunities and threats. A literature survey</a>. Anal Bioanal Chem 393:569–582.</div>
</div>
<div id="ref-tvdJOfV9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">619. </div><div class="csl-right-inline">Mistry DA, Wang JY, Moeser M-E, Starkey T, Lee LYW. 2021. <a href="https://doi.org/10.1186/s12879-021-06528-3">A systematic review of the sensitivity and specificity of lateral flow devices in the detection of SARS-CoV-2.</a> BMC Infect Dis 21:828.</div>
</div>
<div id="ref-cLQbewzh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">620. </div><div class="csl-right-inline">Government sets out next steps for living with COVID. GOVUK. <a href="https://www.gov.uk/government/news/government-sets-out-next-steps-for-living-with-covid">https://www.gov.uk/government/news/government-sets-out-next-steps-for-living-with-covid</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-hFI37GHb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">621. </div><div class="csl-right-inline">House TW. 2022. Fact Sheet: The Biden Administration to Begin Distributing At-Home, Rapid COVID-19 Tests to Americans for Free. The White House. <a href="https://www.whitehouse.gov/briefing-room/statements-releases/2022/01/14/fact-sheet-the-biden-administration-to-begin-distributing-at-home-rapid-covid-19-tests-to-americans-for-free/">https://www.whitehouse.gov/briefing-room/statements-releases/2022/01/14/fact-sheet-the-biden-administration-to-begin-distributing-at-home-rapid-covid-19-tests-to-americans-for-free/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1Bm6qAnYY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">622. </div><div class="csl-right-inline">Peeling RW, Olliaro PL, Boeras DI, Fongwen N. 2021. <a href="https://doi.org/10.1016/s1473-3099(21)00048-7">Scaling up COVID-19 rapid antigen tests: promises and challenges</a>. The Lancet Infectious Diseases 21:e290–e295.</div>
</div>
<div id="ref-ayaV1tKm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">623. </div><div class="csl-right-inline">Rai P, Kumar BK, Deekshit VK, Karunasagar I, Karunasagar I. 2021. <a href="https://doi.org/10.1007/s00253-020-11061-5">Detection technologies and recent developments in the diagnosis of COVID-19 infection</a>. Appl Microbiol Biotechnol 105:441–455.</div>
</div>
<div id="ref-15HaJFv2B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">624. </div><div class="csl-right-inline">Lequin RM. 2005. <a href="https://doi.org/10.1373/clinchem.2005.051532">Enzyme Immunoassay (EIA)/Enzyme-Linked Immunosorbent Assay (ELISA)</a>. Clinical Chemistry 51:2415–2418.</div>
</div>
<div id="ref-XVdGUsUr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">625. </div><div class="csl-right-inline">Schuurs AHWM, Van Weemen BK. 1980. <a href="https://doi.org/10.1080/01971528008055786">Enzyme-Immunoassay: A Powerful Analytical Tool</a>. Journal of Immunoassay 1:229–249.</div>
</div>
<div id="ref-lrHedci6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">626. </div><div class="csl-right-inline">Adams ER, Ainsworth M, Anand R, Andersson MI, Auckland K, Baillie JK, Barnes E, Beer S, Bell JI, Berry T, Bibi S, Carroll M, Chinnakannan SK, Clutterbuck E, Cornall RJ, Crook DW, de Silva T, Dejnirattisai W, Dingle KE, Dold C, Espinosa A, Eyre DW, Farmer H, Fernandez Mendoza M, Georgiou D, Hoosdally SJ, Hunter A, Jefferey K, Kelly DF, Klenerman P, Knight J, Knowles C, Kwok AJ, Leuschner U, Levin R, Liu C, López-Camacho C, Martinez J, Matthews PC, McGivern H, Mentzer AJ, Milton J, Mongkolsapaya J, Moore SC, Oliveira MS, Pereira F, Perez E, Peto T, Ploeg RJ, Pollard A, Prince T, Roberts DJ, Rudkin JK, Sanchez V, Screaton GR, Semple MG, Slon-Campos J, Skelly DT, Smith EN, Sobrinodiaz A, Staves J, Stuart DI, Supasa P, Surik T, Thraves H, Tsang P, Turtle L, Walker AS, Wang B, Washington C, Watkins N, Whitehouse J. 2020. <a href="https://doi.org/10.12688/wellcomeopenres.15927.1">Antibody testing for COVID-19: A report from the National COVID Scientific Advisory Panel</a>. Wellcome Open Res 5:139.</div>
</div>
<div id="ref-11QmOzidP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">627. </div><div class="csl-right-inline">Adnan N, Khandker SS, Haq A, Chaity MA, Khalek A, Nazim AQ, Kaitsuka T, Tomizawa K, Mie M, Kobatake E, Ahmed S, Ali nor AA, Khondoker MU, Haque M, Jamiruddin MohdR. 2021. <a href="https://doi.org/10.1080/14787210.2021.1976144">Detection of SARS-CoV-2 by antigen ELISA test is highly swayed by viral load and sample storage condition</a>. Expert Review of Anti-infective Therapy 20:473–481.</div>
</div>
<div id="ref-11vIuqmS2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">628. </div><div class="csl-right-inline">Winichakoon P, Chaiwarith R, Liwsrisakun C, Salee P, Goonna A, Limsukon A, Kaewpoowat Q. 2020. <a href="https://doi.org/10.1128/jcm.00297-20">Negative Nasopharyngeal and Oropharyngeal Swabs Do Not Rule Out COVID-19</a>. Journal of Clinical Microbiology 58:e00297–20.</div>
</div>
<div id="ref-Fdc7seLs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">629. </div><div class="csl-right-inline">Sheridan C. 2020. <a href="https://doi.org/10.1038/d41587-020-00002-2">Coronavirus and the race to distribute reliable diagnostics</a>. Nature Biotechnology 38:382–384.</div>
</div>
<div id="ref-1HtWSk427" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">630. </div><div class="csl-right-inline">CDC, CDC. 2022. Isolation. Centers for Disease Control and Prevention. <a href="https://www.cdc.gov/coronavirus/2019-ncov/your-health/isolation.html">https://www.cdc.gov/coronavirus/2019-ncov/your-health/isolation.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-MNxZTL29" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">631. </div><div class="csl-right-inline">Kumar P, Malik YS, Ganesh B, Rahangdale S, Saurabh S, Natesan S, Srivastava A, Sharun K, Yatoo MohdI, Tiwari R, Singh RK, Dhama K. 2020. <a href="https://doi.org/10.3389/fcimb.2020.576875">CRISPR-Cas System: An Approach With Potentials for COVID-19 Diagnosis and Therapeutics</a>. Front Cell Infect Microbiol 10.</div>
</div>
<div id="ref-NT5mrVGY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">632. </div><div class="csl-right-inline">Service R. 2020. The standard coronavirus test, if available, works well—but can new diagnostics help in this pandemic? Science https://doi.org/<a href="https://doi.org/10.1126/science.abb8400">10.1126/science.abb8400</a>.</div>
</div>
<div id="ref-AvQI3GSm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">633. </div><div class="csl-right-inline">Tang Y-W, Schmitz JE, Persing DH, Stratton CW. 2020. <a href="https://doi.org/10.1128/jcm.00512-20">Laboratory Diagnosis of COVID-19: Current Issues and Challenges</a>. Journal of Clinical Microbiology 58:e00512–20.</div>
</div>
<div id="ref-U8dFUyCj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">634. </div><div class="csl-right-inline">Sloss L. 2021. <a href="https://www.nytimes.com/2021/12/31/travel/covid-test-chaos.html">The Plane Is Boarding, Where Are Your Test Results?</a> The New York Times.</div>
</div>
<div id="ref-15MEHt7vJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">635. </div><div class="csl-right-inline">Kresge N. 2022. A Stark Contrast Between the U.S. and Europe on Tests. Bloomberg. <a href="https://www.bloomberg.com/news/newsletters/2022-01-07/a-stark-contrast-between-the-u-s-and-europe-on-tests">https://www.bloomberg.com/news/newsletters/2022-01-07/a-stark-contrast-between-the-u-s-and-europe-on-tests</a>.</div>
</div>
<div id="ref-fB995nF4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">636. </div><div class="csl-right-inline">Vidarsson G, Dekkers G, Rispens T. 2014. <a href="https://doi.org/10.3389/fimmu.2014.00520">IgG Subclasses and Allotypes: From Structure to Effector Functions</a>. Front Immunol 5.</div>
</div>
<div id="ref-5ejmbqQZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">637. </div><div class="csl-right-inline">2001. The distribution and functions of immunoglobulin isotypesImmunobiology: The Immune System in Health and Disease. Garland Science. <a href="https://www.ncbi.nlm.nih.gov/books/NBK27162">https://www.ncbi.nlm.nih.gov/books/NBK27162</a>.</div>
</div>
<div id="ref-pGXsPCCz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">638. </div><div class="csl-right-inline">MO H, ZENG G, REN X, LI H, KE C, TAN Y, CAI C, LAI K, CHEN R, CHAN-YEUNG M, ZHONG N. 2006. <a href="https://doi.org/10.1111/j.1440-1843.2006.00783.x">Longitudinal profile of antibodies against SARS-coronavirus in SARS patients and their clinical significance</a>. Respirology 11:49–53.</div>
</div>
<div id="ref-FFMfMDRN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">639. </div><div class="csl-right-inline">Du Z, Zhu F, Guo F, Yang B, Wang T. 2020. <a href="https://doi.org/10.1002/jmv.25820">Detection of antibodies against SARS‐CoV‐2 in patients with COVID‐19</a>. Journal of Medical Virology 92:1735–1738.</div>
</div>
<div id="ref-KxpufU9U" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">640. </div><div class="csl-right-inline">Espejo AP, Akgun Y, Al Mana AF, Tjendra Y, Millan NC, Gomez-Fernandez C, Cray C. 2020. <a href="https://doi.org/10.1093/ajcp/aqaa112">Review of Current Advances in Serologic Testing for COVID-19</a>. American Journal of Clinical Pathology 154:293–304.</div>
</div>
<div id="ref-rvAb0ZJy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">641. </div><div class="csl-right-inline">Amanat F, Stadlbauer D, Strohmeier S, Nguyen THO, Chromikova V, McMahon M, Jiang K, Arunkumar GA, Jurczyszak D, Polanco J, Bermudez-Gonzalez M, Kleiner G, Aydillo T, Miorin L, Fierer DS, Lugo LA, Kojic EM, Stoever J, Liu STH, Cunningham-Rundles C, Felgner PL, Moran T, García-Sastre A, Caplivski D, Cheng AC, Kedzierska K, Vapalahti O, Hepojoki JM, Simon V, Krammer F. 2020. <a href="https://doi.org/10.1038/s41591-020-0913-5">A serological assay to detect SARS-CoV-2 seroconversion in humans</a>. Nat Med 26:1033–1036.</div>
</div>
<div id="ref-pnHr92FQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">642. </div><div class="csl-right-inline">Mehdi F, Chattopadhyay S, Thiruvengadam R, Yadav S, Kumar M, Sinha SK, Goswami S, Kshetrapal P, Wadhwa N, Chandramouli Natchu U, Sopory S, Koundinya Desiraju B, Pandey AK, Das A, Verma N, Sharma N, Sharma P, Bhartia V, Gosain M, Lodha R, Lamminmäki U, Shrivastava T, Bhatnagar S, Batra G. 2021. <a href="https://doi.org/10.3389/fmicb.2020.618097">Development of a Fast SARS-CoV-2 IgG ELISA, Based on Receptor-Binding Domain, and Its Comparative Evaluation Using Temporally Segregated Samples From RT-PCR Positive Individuals</a>. Front Microbiol 11.</div>
</div>
<div id="ref-JEaoCUco" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">643. </div><div class="csl-right-inline">Villafañe L, Vaulet LG, Viere FM, Klepp LI, Forrellad MA, Bigi MM, Romano MI, Magistrelli G, Fermepin MR, Bigi F. 2022. <a href="https://doi.org/10.1016/j.jim.2021.113182">Development and evaluation of a low cost IgG ELISA test based in RBD protein for COVID-19</a>. Journal of Immunological Methods 500:113182.</div>
</div>
<div id="ref-IAZeyZZW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">644. </div><div class="csl-right-inline">Wang Y, Dzakah EE, Kang Y, Cai Y, Wu P, Tang B, Li R, He X. 2019. <a href="https://doi.org/10.2147/ijn.s200556">A sensitive and rapid chemiluminescence immunoassay for point-of-care testing (POCT) of copeptin in serum based on high-affinity monoclonal antibodies via cytokine-assisted immunization</a>. IJN Volume 14:4293–4307.</div>
</div>
<div id="ref-13kfmvz1C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">645. </div><div class="csl-right-inline">Cinquanta L, Fontana DE, Bizzaro N. 2017. <a href="https://doi.org/10.1007/s13317-017-0097-2">Chemiluminescent immunoassay technology: what does it change in autoantibody detection?</a> Autoimmun Highlights 8.</div>
</div>
<div id="ref-ciLSdA3x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">646. </div><div class="csl-right-inline">Cai X, Chen J, li Hu J-, Long Q, Deng H, Liu P, Fan K, Liao P, Liu B, Wu G, Chen Y, Li Z, Wang K, Zhang X, Tian W, Xiang J, Du H, Wang J, Hu Y, Tang N, Lin Y, Ren J, Huang L, Wei J, Gan C, Chen Y, Gao Q, Chen A, He C, Wang D-X, Hu P, Zhou F-C, Huang A, Wang D. 2020. <a href="https://doi.org/10.1093/infdis/jiaa243">A Peptide-Based Magnetic Chemiluminescence Enzyme Immunoassay for Serological Diagnosis of Coronavirus Disease 2019</a>. The Journal of Infectious Diseases 222:189–193.</div>
</div>
<div id="ref-sGrolLjn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">647. </div><div class="csl-right-inline">Coste AT, Jaton K, Papadimitriou-Olivgeris M, Greub G, Croxatto A. 2021. <a href="https://doi.org/10.1016/j.jcv.2020.104690">Comparison of SARS-CoV-2 serological tests with different antigen targets</a>. Journal of Clinical Virology 134:104690.</div>
</div>
<div id="ref-Lu9hVHtO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">648. </div><div class="csl-right-inline">Nicol T, Lefeuvre C, Serri O, Pivert A, Joubaud F, Dubée V, Kouatchet A, Ducancelle A, Lunel-Fabiani F, Le Guillou-Guillemette H. 2020. <a href="https://doi.org/10.1016/j.jcv.2020.104511">Assessment of SARS-CoV-2 serological tests for the diagnosis of COVID-19 through the evaluation of three immunoassays: Two automated immunoassays (Euroimmun and Abbott) and one rapid lateral flow immunoassay (NG Biotech)</a>. Journal of Clinical Virology 129:104511.</div>
</div>
<div id="ref-gwsGoZAk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">649. </div><div class="csl-right-inline">Charlton CL, Kanji JN, Johal K, Bailey A, Plitt SS, MacDonald C, Kunst A, Buss E, Burnes LE, Fonseca K, Berenger BM, Schnabl K, Hu J, Stokes W, Zelyas N, Tipples G. 2020. <a href="https://doi.org/10.1128/jcm.01361-20">Evaluation of Six Commercial Mid- to High-Volume Antibody and Six Point-of-Care Lateral Flow Assays for Detection of SARS-CoV-2 Antibodies</a>. J Clin Microbiol 58.</div>
</div>
<div id="ref-OT7f5HVt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">650. </div><div class="csl-right-inline">Infantino M, Grossi V, Lari B, Bambi R, Perri A, Manneschi M, Terenzi G, Liotti I, Ciotta G, Taddei C, Benucci M, Casprini P, Veneziani F, Fabbri S, Pompetti A, Manfredi M. 2020. <a href="https://doi.org/10.1002/jmv.25932">Diagnostic accuracy of an automated chemiluminescent immunoassay for anti‐SARS‐CoV‐2 IgM and IgG antibodies: an Italian experience</a>. J Med Virol 92:1671–1675.</div>
</div>
<div id="ref-YZJK3c9X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">651. </div><div class="csl-right-inline">Sekirov I, Barakauskas VE, Simons J, Cook D, Bates B, Burns L, Masud S, Charles M, McLennan M, Mak A, Chahil N, Vijh R, Hayden A, Goldfarb D, Levett PN, Krajden M, Morshed M. 2021. <a href="https://doi.org/10.1016/j.jcv.2021.104914">SARS-CoV-2 serology: Validation of high-throughput chemiluminescent immunoassay (CLIA) platforms and a field study in British Columbia</a>. Journal of Clinical Virology 142:104914.</div>
</div>
<div id="ref-n6sKSpIk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">652. </div><div class="csl-right-inline">Cellex. 2020. Cellex qSARS-CoV-2 IgG/IgM Rapid Test. <a href="https://www.fda.gov/media/136625/download">https://www.fda.gov/media/136625/download</a>.</div>
</div>
<div id="ref-vbwRvJPH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">653. </div><div class="csl-right-inline">Nam M, Seo JD, Moon H-W, Kim H, Hur M, Yun Y-M. 2021. <a href="https://doi.org/10.1128/spectrum.01202-21">Evaluation of Humoral Immune Response after SARS-CoV-2 Vaccination Using Two Binding Antibody Assays and a Neutralizing Antibody Assay</a>. Microbiol Spectr 9.</div>
</div>
<div id="ref-1AN4GuRw1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">654. </div><div class="csl-right-inline">Muruato AE, Fontes-Garfias CR, Ren P, Garcia-Blanco MA, Menachery VD, Xie X, Shi P-Y. 2020. <a href="https://doi.org/10.1038/s41467-020-17892-0">A high-throughput neutralizing antibody assay for COVID-19 diagnosis and vaccine evaluation</a>. Nat Commun 11.</div>
</div>
<div id="ref-X4iMjHsT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">655. </div><div class="csl-right-inline">Khoury DS, Cromer D, Reynaldi A, Schlub TE, Wheatley AK, Juno JA, Subbarao K, Kent SJ, Triccas JA, Davenport MP. 2021. <a href="https://doi.org/10.1038/s41591-021-01377-8">Neutralizing antibody levels are highly predictive of immune protection from symptomatic SARS-CoV-2 infection</a>. Nat Med 27:1205–1211.</div>
</div>
<div id="ref-NeoPSRp5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">656. </div><div class="csl-right-inline">Addetia A, Crawford KHD, Dingens A, Zhu H, Roychoudhury P, Huang M-L, Jerome KR, Bloom JD, Greninger AL. 2020. <a href="https://doi.org/10.1128/jcm.02107-20">Neutralizing Antibodies Correlate with Protection from SARS-CoV-2 in Humans during a Fishery Vessel Outbreak with a High Attack Rate</a>. J Clin Microbiol 58.</div>
</div>
<div id="ref-H82x3B90" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">657. </div><div class="csl-right-inline">Cromer D, Steain M, Reynaldi A, Schlub TE, Wheatley AK, Juno JA, Kent SJ, Triccas JA, Khoury DS, Davenport MP. 2022. <a href="https://doi.org/10.1016/s2666-5247(21)00267-6">Neutralising antibody titres as predictors of protection against SARS-CoV-2 variants and the impact of boosting: a meta-analysis</a>. The Lancet Microbe 3:e52–e61.</div>
</div>
<div id="ref-Xu7hrQyb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">658. </div><div class="csl-right-inline">Mravinacova S, Jönsson M, Christ W, Klingström J, Yousef J, Hellström C, Hedhammar M, Havervall S, Thålin C, Pin E, Tegel H, Nilsson P, Månberg A, Hober S. 2022. <a href="https://doi.org/10.1016/j.nbt.2021.10.002">A cell-free high throughput assay for assessment of SARS-CoV-2 neutralizing antibodies</a>. New Biotechnology 66:46–52.</div>
</div>
<div id="ref-11UpvND0K" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">659. </div><div class="csl-right-inline">Liu W, Fontanet A, Zhang P, Zhan L, Xin Z, Baril L, Tang F, Lv H, Cao W. 2006. <a href="https://doi.org/10.1086/500469">Two‐Year Prospective Study of the Humoral Immune Response of Patients with Severe Acute Respiratory Syndrome</a>. The Journal of Infectious Diseases 193:792–795.</div>
</div>
<div id="ref-FIOysZLl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">660. </div><div class="csl-right-inline">Callow KA, Parry HF, Sergeant M, Tyrrell DAJ. 2009. <a href="https://doi.org/10.1017/s0950268800048019">The time course of the immune response to experimental coronavirus infection of man</a>. Epidemiology and Infection 105:435–446.</div>
</div>
<div id="ref-4si096MH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">661. </div><div class="csl-right-inline">Choe PG, Kim K-H, Kang CK, Suh HJ, Kang E, Lee SY, Kim NJ, Yi J, Park WB, Oh M. 2021. <a href="https://doi.org/10.3201/eid2703.204543">Antibody Responses 8 Months after Asymptomatic or Mild SARS-CoV-2 Infection</a>. Emerging Infectious Diseases 27.</div>
</div>
<div id="ref-L5l2MrPj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">662. </div><div class="csl-right-inline">Dan JM, Mateus J, Kato Y, Hastie KM, Yu ED, Faliti CE, Grifoni A, Ramirez SI, Haupt S, Frazier A, Nakao C, Rayaprolu V, Rawlings SA, Peters B, Krammer F, Simon V, Saphire EO, Smith DM, Weiskopf D, Sette A, Crotty S. 2021. <a href="https://doi.org/10.1126/science.abf4063">Immunological memory to SARS-CoV-2 assessed for up to 8 months after infection</a>. Science eabf4063.</div>
</div>
<div id="ref-1HNwTvE3S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">663. </div><div class="csl-right-inline">Hartley GE, Edwards ESJ, Aui PM, Varese N, Stojanovic S, McMahon J, Peleg AY, Boo I, Drummer HE, Hogarth PM, O'Hehir RE, van Zelm MC. 2020. <a href="https://doi.org/10.1126/sciimmunol.abf8891">Rapid generation of durable B cell memory to SARS-CoV-2 spike and nucleocapsid proteins in COVID-19 and convalescence.</a> Sci Immunol 5.</div>
</div>
<div id="ref-H7Hdjjfx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">664. </div><div class="csl-right-inline">Sherina N, Piralla A, Du L, Wan H, Kumagai-Braesch M, Andréll J, Braesch-Andersen S, Cassaniti I, Percivalle E, Sarasini A, Bergami F, Di Martino R, Colaneri M, Vecchia M, Sambo M, Zuccaro V, Bruno R, Sachs M, Oggionni T, Meloni F, Abolhassani H, Bertoglio F, Schubert M, Byrne-Steele M, Han J, Hust M, Xue Y, Hammarström L, Baldanti F, Marcotte H, Pan-Hammarström Q. 2021. <a href="https://doi.org/10.1016/j.medj.2021.02.001">Persistence of SARS-CoV-2-specific B and T cell responses in convalescent COVID-19 patients 6–8 months after the infection</a>. Med 2:281–295.e4.</div>
</div>
<div id="ref-iP9vKRvv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">665. </div><div class="csl-right-inline">Gaebler C, Wang Z, Lorenzi JCC, Muecksch F, Finkin S, Tokuyama M, Cho A, Jankovic M, Schaefer-Babajew D, Oliveira TY, Cipolla M, Viant C, Barnes CO, Bram Y, Breton G, Hägglöf T, Mendoza P, Hurley A, Turroja M, Gordon K, Millard KG, Ramos V, Schmidt F, Weisblum Y, Jha D, Tankelevich M, Martinez-Delgado G, Yee J, Patel R, Dizon J, Unson-O’Brien C, Shimeliovich I, Robbiani DF, Zhao Z, Gazumyan A, Schwartz RE, Hatziioannou T, Bjorkman PJ, Mehandru S, Bieniasz PD, Caskey M, Nussenzweig MC. 2021. <a href="https://doi.org/10.1038/s41586-021-03207-w">Evolution of antibody immunity to SARS-CoV-2</a>. Nature 591:639–644.</div>
</div>
<div id="ref-19MYPpSCx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">666. </div><div class="csl-right-inline">Wajnberg A, Amanat F, Firpo A, Altman DR, Bailey MJ, Mansour M, McMahon M, Meade P, Mendu DR, Muellers K, Stadlbauer D, Stone K, Strohmeier S, Simon V, Aberg J, Reich DL, Krammer F, Cordon-Cardo C. 2020. <a href="https://doi.org/10.1126/science.abd7728">Robust neutralizing antibodies to SARS-CoV-2 infection persist for months</a>. Science 370:1227–1230.</div>
</div>
<div id="ref-m1wjdQSV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">667. </div><div class="csl-right-inline">Seow J, Graham C, Merrick B, Acors S, Pickering S, Steel KJA, Hemmings O, O’Byrne A, Kouphou N, Galao RP, Betancor G, Wilson HD, Signell AW, Winstone H, Kerridge C, Huettner I, Jimenez-Guardeño JM, Lista MJ, Temperton N, Snell LB, Bisnauthsing K, Moore A, Green A, Martinez L, Stokes B, Honey J, Izquierdo-Barras A, Arbane G, Patel A, Tan MKI, O’Connell L, O’Hara G, MacMahon E, Douthwaite S, Nebbia G, Batra R, Martinez-Nunez R, Shankar-Hari M, Edgeworth JD, Neil SJD, Malim MH, Doores KJ. 2020. <a href="https://doi.org/10.1038/s41564-020-00813-8">Longitudinal observation and decline of neutralizing antibody responses in the three months following SARS-CoV-2 infection in humans</a>. Nat Microbiol 5:1598–1607.</div>
</div>
<div id="ref-NVgdaNGd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">668. </div><div class="csl-right-inline">Wheatley AK, Juno JA, Wang JJ, Selva KJ, Reynaldi A, Tan H-X, Lee WS, Wragg KM, Kelly HG, Esterbauer R, Davis SK, Kent HE, Mordant FL, Schlub TE, Gordon DL, Khoury DS, Subbarao K, Cromer D, Gordon TP, Chung AW, Davenport MP, Kent SJ. 2021. <a href="https://doi.org/10.1038/s41467-021-21444-5">Evolution of immune responses to SARS-CoV-2 in mild-moderate COVID-19</a>. Nat Commun 12.</div>
</div>
<div id="ref-1CZlIiD0p" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">669. </div><div class="csl-right-inline">Garcia-Beltran WF, Lam EC, Astudillo MG, Yang D, Miller TE, Feldman J, Hauser BM, Caradonna TM, Clayton KL, Nitido AD, Murali MR, Alter G, Charles RC, Dighe A, Branda JA, Lennerz JK, Lingwood D, Schmidt AG, Iafrate AJ, Balazs AB. 2021. <a href="https://doi.org/10.1016/j.cell.2020.12.015">COVID-19-neutralizing antibodies predict disease severity and survival</a>. Cell 184:476–488.e11.</div>
</div>
<div id="ref-3LFrwvth" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">670. </div><div class="csl-right-inline">Dowell AC, Butler MS, Jinks E, Tut G, Lancaster T, Sylla P, Begum J, Bruton R, Pearce H, Verma K, Logan N, Tyson G, Spalkova E, Margielewska-Davies S, Taylor GS, Syrimi E, Baawuah F, Beckmann J, Okike IO, Ahmad S, Garstang J, Brent AJ, Brent B, Ireland G, Aiano F, Amin-Chowdhury Z, Jones S, Borrow R, Linley E, Wright J, Azad R, Waiblinger D, Davis C, Thomson EC, Palmarini M, Willett BJ, Barclay WS, Poh J, Amirthalingam G, Brown KE, Ramsay ME, Zuo J, Moss P, Ladhani S. 2021. <a href="https://doi.org/10.1038/s41590-021-01089-8">Children develop robust and sustained cross-reactive spike-specific immune responses to SARS-CoV-2 infection</a>. Nat Immunol 23:40–49.</div>
</div>
<div id="ref-1AAMeFyRr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">671. </div><div class="csl-right-inline">Gil-Manso S, Miguens Blanco I, Motyka B, Halpin A, López-Esteban R, Pérez-Fernández VA, Carbonell D, López-Fernández LA, West L, Correa-Rocha R, Pion M. 2021. <a href="https://doi.org/10.1080/21505594.2021.2019959">ABO blood group is involved in the quality of the specific immune response anti-SARS-CoV-2</a>. Virulence 13:30–45.</div>
</div>
<div id="ref-JcpiBiQt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">672. </div><div class="csl-right-inline">Kaneko N, Kuo H-H, Boucau J, Farmer JR, Allard-Chamard H, Mahajan VS, Piechocka-Trocha A, Lefteri K, Osborn M, Bals J, Bartsch YC, Bonheur N, Caradonna TM, Chevalier J, Chowdhury F, Diefenbach TJ, Einkauf K, Fallon J, Feldman J, Finn KK, Garcia-Broncano P, Hartana CA, Hauser BM, Jiang C, Kaplonek P, Karpell M, Koscher EC, Lian X, Liu H, Liu J, Ly NL, Michell AR, Rassadkina Y, Seiger K, Sessa L, Shin S, Singh N, Sun W, Sun X, Ticheli HJ, Waring MT, Zhu AL, Alter G, Li JZ, Lingwood D, Schmidt AG, Lichterfeld M, Walker BD, Yu XG, Padera RF, Pillai S. 2020. <a href="https://doi.org/10.1016/j.cell.2020.08.025">Loss of Bcl-6-Expressing T Follicular Helper Cells and Germinal Centers in COVID-19</a>. Cell 183:143–157.e13.</div>
</div>
<div id="ref-AKY95Yy3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">673. </div><div class="csl-right-inline">Zuo J, Dowell A, Pearce H, Verma K, Long H, Begum J, Aiano F, Amin-Chowdhury Z, Hallis B, Stapley L, Borrow R, Linley E, Ahmad S, Parker B, Horsley A, Amirthalingam G, Brown K, Ramsay M, Ladhani S, Moss P. 2020. Robust SARS-CoV-2-specific T-cell immunity is maintained at 6 months following primary infection. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.11.01.362319">10.1101/2020.11.01.362319</a>.</div>
</div>
<div id="ref-19p3NXKI5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">674. </div><div class="csl-right-inline">Breton G, Mendoza P, Hagglof T, Oliveira TY, Schaefer-Babajew D, Gaebler C, Turroja M, Hurley A, Caskey M, Nussenzweig MC. 2020. Persistent Cellular Immunity to SARS-CoV-2 Infection. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.12.08.416636">10.1101/2020.12.08.416636</a>.</div>
</div>
<div id="ref-CHRiQNCx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">675. </div><div class="csl-right-inline">Grifoni A, Weiskopf D, Ramirez SI, Mateus J, Dan JM, Moderbacher CR, Rawlings SA, Sutherland A, Premkumar L, Jadi RS, Marrama D, de Silva AM, Frazier A, Carlin AF, Greenbaum JA, Peters B, Krammer F, Smith DM, Crotty S, Sette A. 2020. <a href="https://doi.org/10.1016/j.cell.2020.05.015">Targets of T Cell Responses to SARS-CoV-2 Coronavirus in Humans with COVID-19 Disease and Unexposed Individuals</a>. Cell 181:1489–1501.e15.</div>
</div>
<div id="ref-10ICz5Jou" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">676. </div><div class="csl-right-inline">Sheridan C. 2021. <a href="https://doi.org/10.1038/s41587-021-00920-9">COVID-19 testing turns to T cells</a>. Nat Biotechnol 39:533–534.</div>
</div>
<div id="ref-uKR3FVp8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">677. </div><div class="csl-right-inline">Commissioner O of the. 2021. Coronavirus (COVID-19) Update: FDA Authorizes Adaptive Biotechnologies T-Detect COVID Test. FDA. <a href="https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-adaptive-biotechnologies-t-detect-covid-test">https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-adaptive-biotechnologies-t-detect-covid-test</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-NXf7l0z3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">678. </div><div class="csl-right-inline">Mina MJ, Parker R, Larremore DB. 2020. <a href="https://doi.org/10.1056/nejmp2025631">Rethinking Covid-19 Test Sensitivity — A Strategy for Containment</a>. N Engl J Med 383:e120.</div>
</div>
<div id="ref-Y35qBYLJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">679. </div><div class="csl-right-inline">Peeling RW, Olliaro PL. 2020. <a href="https://doi.org/10.1016/s2213-2600(20)30313-1">The time to do serosurveys for COVID-19 is now</a>. The Lancet Respiratory Medicine 8:836–838.</div>
</div>
<div id="ref-7zusLohQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">680. </div><div class="csl-right-inline">Schuler CF, Gherasim C, O’Shea K, Manthei DM, Chen J, Giacherio D, Troost JP, Baldwin JL, Baker JR. 2021. <a href="https://doi.org/10.1371/journal.pone.0248729">Accurate point-of-care serology tests for COVID-19</a>. PLoS ONE 16:e0248729.</div>
</div>
<div id="ref-7bzRqtiu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">681. </div><div class="csl-right-inline">Chen X, Chen Z, Azman AS, Sun R, Lu W, Zheng N, Zhou J, Wu Q, Deng X, Zhao Z, Chen X, Ge S, Yang J, Leung DT, Yu H. 2021. <a href="https://doi.org/10.1093/cid/ciab646">Neutralizing Antibodies Against Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Variants Induced by Natural Infection or Vaccination: A Systematic Review and Pooled Analysis</a>. Clinical Infectious Diseases 74:734–742.</div>
</div>
<div id="ref-16uL34FEP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">682. </div><div class="csl-right-inline">Murhekar MV, Clapham H. 2021. <a href="https://doi.org/10.1016/s2214-109x(21)00057-7">COVID-19 serosurveys for public health decision making</a>. The Lancet Global Health 9:e559–e560.</div>
</div>
<div id="ref-A3QBAHLv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">683. </div><div class="csl-right-inline">Ghaffari A, Meurant R, Ardakani A. 2020. <a href="https://doi.org/10.3390/diagnostics10070453">COVID-19 Serological Tests: How Well Do They Actually Perform?</a> Diagnostics 10:453.</div>
</div>
<div id="ref-1Gw1giZBH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">684. </div><div class="csl-right-inline">Bond K, Williams E, Howden BP, Williamson DA. 2020. <a href="https://doi.org/10.5694/mja2.50766">Serological tests for COVID-19</a>. Medical Journal of Australia 213:397.</div>
</div>
<div id="ref-WgmwXO4P" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">685. </div><div class="csl-right-inline">Rodriguez-Moncayo R, Cedillo-Alcantar DF, Guevara-Pantoja PE, Chavez-Pineda OG, Hernandez-Ortiz JA, Amador-Hernandez JU, Rojas-Velasco G, Sanchez-Muñoz F, Manzur-Sandoval D, Patino-Lopez LD, May-Arrioja DA, Posadas-Sanchez R, Vargas-Alarcon G, Garcia-Cordero JL. 2021. <a href="https://doi.org/10.1039/d0lc01068e">A high-throughput multiplexed microfluidic device for COVID-19 serology assays</a>. Lab Chip 21:93–104.</div>
</div>
<div id="ref-EUrJt06r" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">686. </div><div class="csl-right-inline">Huang AT, Garcia-Carreras B, Hitchings MDT, Yang B, Katzelnick L, Rattigan SM, Borgert B, Moreno C, Solomon BD, Rodriguez-Barraquer I, Lessler J, Salje H, Burke DS, Wesolowski A, Cummings DAT. 2020. A systematic review of antibody mediated immunity to coronaviruses: antibody kinetics, correlates of protection, and association of antibody responses with severity of disease. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.04.14.20065771">10.1101/2020.04.14.20065771</a>.</div>
</div>
<div id="ref-P9k7RTw6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">687. </div><div class="csl-right-inline">Spellberg B, Nielsen TB, Casadevall A. 2021. <a href="https://doi.org/10.1001/jamainternmed.2020.7986">Antibodies, Immunity, and COVID-19</a>. JAMA Intern Med 181:460.</div>
</div>
<div id="ref-TDWfc9A9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">688. </div><div class="csl-right-inline">Goldberg Y, Mandel M, Bar-On YM, Bodenheimer O, Freedman L, Ash N, Alroy-Preis S, Huppert A, Milo R. 2021. <a href="https://doi.org/10.1101/2021.12.04.21267114">Protection and waning of natural and hybrid COVID-19 immunity</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-91dto3z3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">689. </div><div class="csl-right-inline">Tostmann A, Bradley J, Bousema T, Yiek W-K, Holwerda M, Bleeker-Rovers C, ten Oever J, Meijer C, Rahamat-Langendoen J, Hopman J, van der Geest-Blankert N, Wertheim H. 2020. <a href="https://doi.org/10.2807/1560-7917.es.2020.25.16.2000508">Strong associations and moderate predictive value of early symptoms for SARS-CoV-2 test positivity among healthcare workers, the Netherlands, March 2020</a>. Eurosurveillance 25.</div>
</div>
<div id="ref-xn686bWT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">690. </div><div class="csl-right-inline">Ren X, Liu Y, Chen H, Liu W, Guo Z, Zhang Y, Chen C, Zhou J, Xiao Q, Jiang G, Shan H. 2020. <a href="https://doi.org/10.1101/2020.02.25.20027755">Application and optimization of RT-PCR in diagnosis of SARS-CoV-2 infection</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-itCGVb5b" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">691. </div><div class="csl-right-inline">Ai T, Yang Z, Hou H, Zhan C, Chen C, Lv W, Tao Q, Sun Z, Xia L. 2020. <a href="https://doi.org/10.1148/radiol.2020200642">Correlation of Chest CT and RT-PCR Testing for Coronavirus Disease 2019 (COVID-19) in China: A Report of 1014 Cases</a>. Radiology 296:E32–E40.</div>
</div>
<div id="ref-tkuUFNIX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">692. </div><div class="csl-right-inline">Bai HX, Hsieh B, Xiong Z, Halsey K, Choi JW, Tran TML, Pan I, Shi L-B, Wang D-C, Mei J, Jiang X-L, Zeng Q-H, Egglin TK, Hu P-F, Agarwal S, Xie F-F, Li S, Healey T, Atalay MK, Liao W-H. 2020. <a href="https://doi.org/10.1148/radiol.2020200823">Performance of Radiologists in Differentiating COVID-19 from Non-COVID-19 Viral Pneumonia at Chest CT</a>. Radiology 296:E46–E54.</div>
</div>
<div id="ref-nN9zfzPe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">693. </div><div class="csl-right-inline">Apostolopoulos ID, Mpesiana TA. 2020. <a href="https://doi.org/10.1007/s13246-020-00865-4">Covid-19: automatic detection from X-ray images utilizing transfer learning with convolutional neural networks</a>. Physical and Engineering Sciences in Medicine 43:635–640.</div>
</div>
<div id="ref-6kbMsiPz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">694. </div><div class="csl-right-inline">Roberts M, Driggs D, Thorpe M, Gilbey J, Yeung M, Ursprung S, Aviles-Rivero AI, Etmann C, McCague C, Beer L, Weir-McCall JR, Teng Z, Gkrania-Klotsas E, Ruggiero A, Korhonen A, Jefferson E, Ako E, Langs G, Gozaliasl G, Yang G, Prosch H, Preller J, Stanczuk J, Tang J, Hofmanninger J, Babar J, Sánchez LE, Thillai M, Gonzalez PM, Teare P, Zhu X, Patel M, Cafolla C, Azadbakht H, Jacob J, Lowe J, Zhang K, Bradley K, Wassin M, Holzer M, Ji K, Ortet MD, Ai T, Walton N, Lio P, Stranks S, Shadbahr T, Lin W, Zha Y, Niu Z, Rudd JHF, Sala E, Schönlieb C-B. 2021. <a href="https://doi.org/10.1038/s42256-021-00307-0">Common pitfalls and recommendations for using machine learning to detect and prognosticate for COVID-19 using chest radiographs and CT scans</a>. Nat Mach Intell 3:199–217.</div>
</div>
<div id="ref-jsXHJtkB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">695. </div><div class="csl-right-inline">Ibrahim NK. 2020. <a href="https://doi.org/10.1016/j.jiph.2020.07.019">Epidemiologic surveillance for controlling Covid-19 pandemic: types, challenges and implications</a>. Journal of Infection and Public Health 13:1630–1638.</div>
</div>
<div id="ref-EH5exCzT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">696. </div><div class="csl-right-inline">Farkas K, Hillary LS, Malham SK, McDonald JE, Jones DL. 2020. <a href="https://doi.org/10.1016/j.coesh.2020.06.001">Wastewater and public health: the potential of wastewater surveillance for monitoring COVID-19</a>. Current Opinion in Environmental Science &amp; Health 17:14–20.</div>
</div>
<div id="ref-OB8BJiwj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">697. </div><div class="csl-right-inline">CDC. 2020. Healthcare Workers. Centers for Disease Control and Prevention. <a href="https://www.cdc.gov/coronavirus/2019-ncov/hcp/testing-overview.html">https://www.cdc.gov/coronavirus/2019-ncov/hcp/testing-overview.html</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-IldxlHwA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">698. </div><div class="csl-right-inline">2020. NY Forward: a guide to reopening New York &amp; building back better. <a href="https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf">https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf</a>.</div>
</div>
<div id="ref-kpLjx1Kr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">699. </div><div class="csl-right-inline">Kyosei Y, Yamura S, Namba M, Yoshimura T, Watabe S, Ito E. 2021. <a href="https://doi.org/10.2142/biophysico.bppb-v18.004">Antigen tests for COVID-19</a>. BIOPHYSICS 18:28–39.</div>
</div>
<div id="ref-s2XX4fx4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">700. </div><div class="csl-right-inline">Waters A. 2022. <a href="https://doi.org/10.1136/bmj.o383">Covid-19: Show us evidence for lifting restrictions, doctors tell Johnson</a>. BMJ o383.</div>
</div>
<div id="ref-W79HEsxE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">701. </div><div class="csl-right-inline">Misra M, Joshi H, Sarwal R, Rao KD. 2022. <a href="https://doi.org/10.1186/s12889-022-12845-2">Exit strategies from lockdowns due to COVID-19: a scoping review</a>. BMC Public Health 22.</div>
</div>
<div id="ref-qZT0UMJi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">702. </div><div class="csl-right-inline">Jefferson T, Spencer EA, Brassey J, Heneghan C. 2020. <a href="https://doi.org/10.1093/cid/ciaa1764">Viral Cultures for Coronavirus Disease 2019 Infectivity Assessment: A Systematic Review</a>. Clinical Infectious Diseases 73:e3884–e3899.</div>
</div>
<div id="ref-VgiCmD2R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">703. </div><div class="csl-right-inline">Cevik M, Tate M, Lloyd O, Maraolo AE, Schafers J, Ho A. 2021. <a href="https://doi.org/10.1016/s2666-5247(20)30172-5">SARS-CoV-2, SARS-CoV, and MERS-CoV viral load dynamics, duration of viral shedding, and infectiousness: a systematic review and meta-analysis</a>. The Lancet Microbe 2:e13–e22.</div>
</div>
<div id="ref-8oW626i5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">704. </div><div class="csl-right-inline">Agarwal V, Venkatakrishnan AJ, Puranik A, Kirkup C, Lopez-Marquez A, Challener DW, Theel ES, O’Horo JC, Binnicker MJ, Kremers WK, Faubion WA Jr., Badley AD, Williams AW, Gores GJ, Halamka JD, Morice WG II, Soundararajan V. 2020. <a href="https://doi.org/10.1038/s41420-020-00375-y">Long-term SARS-CoV-2 RNA shedding and its temporal association to IgG seropositivity</a>. Cell Death Discov 6.</div>
</div>
<div id="ref-FMUORrT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">705. </div><div class="csl-right-inline">Stankiewicz Karita HC, Dong TQ, Johnston C, Neuzil KM, Paasche-Orlow MK, Kissinger PJ, Bershteyn A, Thorpe LE, Deming M, Kottkamp A, Laufer M, Landovitz RJ, Luk A, Hoffman R, Roychoudhury P, Magaret CA, Greninger AL, Huang M-L, Jerome KR, Wener M, Celum C, Chu HY, Baeten JM, Wald A, Barnabas RV, Brown ER. 2022. <a href="https://doi.org/10.1001/jamanetworkopen.2021.42796">Trajectory of Viral RNA Load Among Persons With Incident SARS-CoV-2 G614 Infection (Wuhan Strain) in Association With COVID-19 Symptom Onset and Severity</a>. JAMA Netw Open 5:e2142796.</div>
</div>
<div id="ref-Jwhgcpu8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">706. </div><div class="csl-right-inline">Armstrong S. 2020. <a href="https://doi.org/10.1136/bmj.m4941">Covid-19: Tests on students are highly inaccurate, early findings show</a>. BMJ m4941.</div>
</div>
<div id="ref-5KUr4fYD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">707. </div><div class="csl-right-inline">Kmietowicz Z. 2021. <a href="https://doi.org/10.1136/bmj.n81">Covid-19: Controversial rapid test policy divides doctors and scientists</a>. BMJ n81.</div>
</div>
<div id="ref-DBDX1KLX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">708. </div><div class="csl-right-inline">Deerain J, Druce J, Tran T, Batty M, Yoga Y, Fennell M, Dwyer DE, Kok J, Williamson DA. 2022. <a href="https://doi.org/10.1128/jcm.02479-21">Assessment of the Analytical Sensitivity of 10 Lateral Flow Devices against the SARS-CoV-2 Omicron Variant</a>. J Clin Microbiol 60.</div>
</div>
<div id="ref-UwLT34Vv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">709. </div><div class="csl-right-inline">Mina MJ, Peto TE, García-Fiñana M, Semple MG, Buchan IE. 2021. <a href="https://doi.org/10.1016/s0140-6736(21)00425-6">Clarifying the evidence on SARS-CoV-2 antigen rapid tests in public health responses to COVID-19</a>. The Lancet 397:1425–1427.</div>
</div>
<div id="ref-Pk9AqhFn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">710. </div><div class="csl-right-inline">Larremore DB, Wilder B, Lester E, Shehata S, Burke JM, Hay JA, Tambe M, Mina MJ, Parker R. 2021. <a href="https://doi.org/10.1126/sciadv.abd5393">Test sensitivity is secondary to frequency and turnaround time for COVID-19 screening</a>. Sci Adv 7.</div>
</div>
<div id="ref-u1jzzcCB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">711. </div><div class="csl-right-inline">Sun LH, Achenbach J. 2020. <a href="https://www.washingtonpost.com/health/2020/06/25/coronavirus-cases-10-times-larger/">CDC chief says coronavirus cases may be 10 times higher than reported</a>. Washington Post.</div>
</div>
<div id="ref-HoWB48LM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">712. </div><div class="csl-right-inline">Liverpool coronavirus (COVID-19) community testing pilot: full evaluation report summary. GOVUK. <a href="https://www.gov.uk/government/publications/liverpool-coronavirus-covid-19-community-testing-pilot-full-evaluation-report-summary/liverpool-coronavirus-covid-19-community-testing-pilot-full-evaluation-report-summary">https://www.gov.uk/government/publications/liverpool-coronavirus-covid-19-community-testing-pilot-full-evaluation-report-summary/liverpool-coronavirus-covid-19-community-testing-pilot-full-evaluation-report-summary</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-6cEtIAuH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">713. </div><div class="csl-right-inline">Iacobucci G. 2021. <a href="https://doi.org/10.1136/bmj.n902">Covid-19: Government rolls out twice weekly rapid testing to all in England</a>. BMJ n902.</div>
</div>
<div id="ref-12KT1j4NG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">714. </div><div class="csl-right-inline">Jae Moon M, Suzuki K, Park TI, Sakuwa K. 2021. <a href="https://doi.org/10.1177/0020852321997552">A comparative study of COVID-19 responses in South Korea and Japan: political nexus triad and policy responses</a>. International Review of Administrative Sciences 87:651–671.</div>
</div>
<div id="ref-14fDqEnne" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">715. </div><div class="csl-right-inline">Bennett M. 2021. <a href="https://doi.org/10.1016/j.worlddev.2020.105208">All things equal? Heterogeneity in policy effectiveness against COVID-19 spread in chile</a>. World Development 137:105208.</div>
</div>
<div id="ref-yfpylewZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">716. </div><div class="csl-right-inline">2022. UK ending Covid testing ‘very worrying’ as WHO chief warns pandemic ‘isn’t over’. The Independent. <a href="https://www.independent.co.uk/news/health/who-covid-testing-anil-soni-b2032884.html">https://www.independent.co.uk/news/health/who-covid-testing-anil-soni-b2032884.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-5ag9qde4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">717. </div><div class="csl-right-inline">Clarke J, Beaney T, Majeed A. 2022. <a href="https://doi.org/10.1136/bmj.o562">UK scales back routine covid-19 surveillance</a>. BMJ o562.</div>
</div>
<div id="ref-NIpc4LRf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">718. </div><div class="csl-right-inline">2022. Local groups continue push for COVID testing and vaccinations as larger state-run sites plan to close. KUSAcom. <a href="https://www.9news.com/article/news/health/coronavirus/local-groups-continue-push-for-covid-testing-and-vaccinations/73-bbcd8384-d96a-425e-aaeb-16ac9f36e581">https://www.9news.com/article/news/health/coronavirus/local-groups-continue-push-for-covid-testing-and-vaccinations/73-bbcd8384-d96a-425e-aaeb-16ac9f36e581</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-p0e1mRYk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">719. </div><div class="csl-right-inline">Utah will stop daily COVID case counts, close test sites in wind-down, Gov. Cox announces. The Salt Lake Tribune. <a href="https://www.sltrib.com/news/2022/02/18/utah-will-stop-daily/">https://www.sltrib.com/news/2022/02/18/utah-will-stop-daily/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1G2DPuXl7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">720. </div><div class="csl-right-inline">Killingley B, Mann AJ, Kalinova M, Boyers A, Goonawardane N, Zhou J, Lindsell K, Hare SS, Brown J, Frise R, Smith E, Hopkins C, Noulin N, Löndt B, Wilkinson T, Harden S, McShane H, Baillet M, Gilbert A, Jacobs M, Charman C, Mande P, Nguyen-Van-Tam JS, Semple MG, Read RC, Ferguson NM, Openshaw PJ, Rapeport G, Barclay WS, Catchpole AP, Chiu C. 2022. <a href="https://doi.org/10.1038/s41591-022-01780-9">Safety, tolerability and viral kinetics during SARS-CoV-2 human challenge in young adults</a>. Nat Med 28:1031–1041.</div>
</div>
<div id="ref-1BuS1KbQB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">721. </div><div class="csl-right-inline">Lordan R, Prior S, Hennessy E, Naik A, Ghosh S, Paschos GK, Skarke C, Barekat K, Hollingsworth T, Juska S, Mazaleuskaya LL, Teegarden S, Glascock AL, Anderson S, Meng H, Tang S-Y, Weljie A, Bottalico L, Ricciotti E, Cherfane P, Mrcela A, Grant G, Poole K, Mayer N, Waring M, Adang L, Becker J, Fries S, FitzGerald GA, Grosser T. 2021. <a href="https://doi.org/10.3389/fpubh.2021.751451">Considerations for the Safe Operation of Schools During the Coronavirus Pandemic</a>. Front Public Health 9.</div>
</div>
<div id="ref-yoOScnNH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">722. </div><div class="csl-right-inline">Theuring S, Thielecke M, van Loon W, Hommes F, Hülso C, von der Haar A, Körner J, Schmidt M, Böhringer F, Mall MA, Rosen A, von Kalle C, Kirchberger V, Kurth T, Seybold J, Mockenhaupt FP. 2021. <a href="https://doi.org/10.2807/1560-7917.es.2021.26.34.2100184">SARS-CoV-2 infection and transmission in school settings during the second COVID-19 wave: a cross-sectional study, Berlin, Germany, November 2020</a>. Eurosurveillance 26.</div>
</div>
<div id="ref-r6gl31QZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">723. </div><div class="csl-right-inline">Miller E. 2022. End to quarantines, contact tracing among changes in new Oregon guidance for schools, starting March 12. OPB. <a href="https://www.opb.org/article/2022/03/02/oregon-schools-guidance-mask-mandate-end/">https://www.opb.org/article/2022/03/02/oregon-schools-guidance-mask-mandate-end/</a>.</div>
</div>
<div id="ref-GN68N2pq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">724. </div><div class="csl-right-inline">2022. Palm Beach County public schools to stop COVID-19 contact tracing. WPTV News Channel 5 West Palm. <a href="https://www.wptv.com/news/education/palm-beach-county-public-schools-to-stop-covid-19-contact-tracing">https://www.wptv.com/news/education/palm-beach-county-public-schools-to-stop-covid-19-contact-tracing</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1CGvdSkU0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">725. </div><div class="csl-right-inline">Hassan A. 2022. <a href="https://www.nytimes.com/live/2022/03/02/world/covid-19-tests-cases-vaccine">Coronavirus Pandemic: Covid News: C.D.C. Drops Contact Tracing Recommendation</a>. The New York Times.</div>
</div>
<div id="ref-1G4q8eaJV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">726. </div><div class="csl-right-inline">Children and COVID-19: State-Level Data Report. American Academy of Pediatrics. <a href="http://www.aap.org/en/pages/2019-novel-coronavirus-covid-19-infections/children-and-covid-19-state-level-data-report/">http://www.aap.org/en/pages/2019-novel-coronavirus-covid-19-infections/children-and-covid-19-state-level-data-report/</a>. Retrieved 11 April 2022.</div>
</div>
<div id="ref-TSx8mpKC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">727. </div><div class="csl-right-inline">Schreiber M. 2022. <a href="https://www.theguardian.com/world/2022/mar/11/us-child-covid-deaths-omicron-surge">A fifth of all US child Covid deaths occurred during Omicron surge</a>. The Guardian.</div>
</div>
<div id="ref-WbYvi82" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">728. </div><div class="csl-right-inline">Unwin HJT, Hillis S, Cluver L, Flaxman S, Goldman PS, Butchart A, Bachman G, Rawlings L, Donnelly CA, Ratmann O, Green P, Nelson CA, Blenkinsop A, Bhatt S, Desmond C, Villaveces A, Sherr L. 2022. <a href="https://doi.org/10.1016/s2352-4642(22)00005-0">Global, regional, and national minimum estimates of children affected by COVID-19-associated orphanhood and caregiver death, by age and family circumstance up to Oct 31, 2021: an updated modelling study</a>. The Lancet Child &amp;amp; Adolescent Health 6:249–259.</div>
</div>
<div id="ref-NrqbGGrB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">729. </div><div class="csl-right-inline">Reed T, Waites W, Manheim D, de Walque D, Vallini C, Gatti R, Hallett TB. 2021. <a href="https://openknowledge.worldbank.org/handle/10986/35150">Five Ways that COVID-19 Diagnostics Can Save Lives: Prioritizing Uses of Tests to Maximize Cost-Effectiveness</a>. Brief. World Bank, Washington, DC.</div>
</div>
<div id="ref-bOegE5mX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">730. </div><div class="csl-right-inline">Hua J, Wang G, Huang M, Hua S, Yang S. 2020. <a href="https://doi.org/10.3390/ijerph17113973">A Visual Approach for the SARS (Severe Acute Respiratory Syndrome) Outbreak Data Analysis</a>. IJERPH 17:3973.</div>
</div>
<div id="ref-MrwDDw9R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">731. </div><div class="csl-right-inline">Center for Systems Science and Engineering at Johns Hopkins University. COVID-19 Data Repository. GitHub. <a href="https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series">https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series</a>. Retrieved 19 June 2020.</div>
</div>
<div id="ref-U1nCE851" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">732. </div><div class="csl-right-inline">Dong E, Du H, Gardner L. 2020. <a href="https://doi.org/10.1016/s1473-3099(20)30120-1">An interactive web-based dashboard to track COVID-19 in real time</a>. The Lancet Infectious Diseases 20:533–534.</div>
</div>
<div id="ref-8gZjmHY2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">733. </div><div class="csl-right-inline">Severe Acute Respiratory Syndrome (SARS). <a href="https://www.who.int/health-topics/severe-acute-respiratory-syndrome">https://www.who.int/health-topics/severe-acute-respiratory-syndrome</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1FrCknYoc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">734. </div><div class="csl-right-inline">kp D. 2021. sars-2003-outbreak-data-with-web-scrapping-munging-and-cleaning-code. <a href="https://github.com/imdevskp/sars-2003-outbreak-data-webscraping-code">https://github.com/imdevskp/sars-2003-outbreak-data-webscraping-code</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-ncMFeSeO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">735. </div><div class="csl-right-inline">Tang JW, Marr LC, Li Y, Dancer SJ. 2021. <a href="https://doi.org/10.1136/bmj.n913">Covid-19 has redefined airborne transmission</a>. BMJ n913.</div>
</div>
<div id="ref-7Ee6Sz9l" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">736. </div><div class="csl-right-inline">Zaki AM, van Boheemen S, Bestebroer TM, Osterhaus ADME, Fouchier RAM. 2012. <a href="https://doi.org/10.1056/nejmoa1211721">Isolation of a Novel Coronavirus from a Man with Pneumonia in Saudi Arabia</a>. New England Journal of Medicine 367:1814–1820.</div>
</div>
<div id="ref-6IJxPIQd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">737. </div><div class="csl-right-inline">Pushpakom S, Iorio F, Eyers PA, Escott KJ, Hopper S, Wells A, Doig A, Guilliams T, Latimer J, McNamee C, Norris A, Sanseau P, Cavalla D, Pirmohamed M. 2018. <a href="https://doi.org/10.1038/nrd.2018.168">Drug repurposing: progress, challenges and recommendations</a>. Nat Rev Drug Discov 18:41–58.</div>
</div>
<div id="ref-1BBiGYn1K" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">738. </div><div class="csl-right-inline">Mohs RC, Greig NH. 2017. <a href="https://doi.org/10.1016/j.trci.2017.10.005">Drug discovery and development: Role of basic biological research</a>. Alzheimer's &amp;amp; Dementia: Translational Research &amp;amp; Clinical Interventions 3:651–657.</div>
</div>
<div id="ref-SSbnPnzT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">739. </div><div class="csl-right-inline">DeVito N, Inglesby P. 2020. Evidence-Based Medicine Data Lab COVID-19 TrialsTracker. GitHub. <a href="https://github.com/ebmdatalab/covid_trials_tracker-covid">https://github.com/ebmdatalab/covid_trials_tracker-covid</a>.</div>
</div>
<div id="ref-3bpuMk1r" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">740. </div><div class="csl-right-inline">Siemieniuk RA, Bartoszko JJ, Zeraatkar D, Kum E, Qasim A, Díaz Martinez JP, Izcovich A, Rochwerg B, Lamontagne F, Han MA, Agarwal A, Agoritsas T, Azab M, Bravo G, Chu DK, Couban R, Cusano E, Devji T, Escamilla Z, Foroutan F, Gao Y, Ge L, Ghadimi M, Heels-Ansdell D, Honarmand K, Hou L, Ibrahim S, Khamis A, Lam B, Mansilla C, Loeb M, Miroshnychenko A, Marcucci M, McLeod SL, Motaghi S, Murthy S, Mustafa RA, Pardo-Hernandez H, Rada G, Rizwan Y, Saadat P, Switzer C, Thabane L, Tomlinson G, Vandvik PO, Vernooij RW, Viteri-García A, Wang Y, Yao L, Zhao Y, Guyatt GH, Brignardello-Petersen R. 2020. <a href="https://doi.org/10.1136/bmj.m2980">Drug treatments for covid-19: living systematic review and network meta-analysis</a>. BMJ m2980.</div>
</div>
<div id="ref-xnPm1mx8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">741. </div><div class="csl-right-inline">Elezkurtaj S, Greuel S, Ihlow J, Michaelis E, Bischoff P, Kunze CA, Sinn BV, Gerhold M, Hauptmann K, Ingold-Heppner B, Miller F, Herbst H, Corman VM, Martin H, Radbruch H, Heppner FL, Horst D. 2020. Causes of Death and Comorbidities in Patients with COVID-19. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.06.15.20131540">10.1101/2020.06.15.20131540</a>.</div>
</div>
<div id="ref-16mSlRgM3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">742. </div><div class="csl-right-inline">Zhang B, Zhou X, Qiu Y, Song Y, Feng F, Feng J, Song Q, Jia Q, Wang J. 2020. <a href="https://doi.org/10.1371/journal.pone.0235458">Clinical characteristics of 82 cases of death from COVID-19</a>. PLOS ONE 15:e0235458.</div>
</div>
<div id="ref-u87rwxD8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">743. </div><div class="csl-right-inline">Shi Y, Wang Y, Shao C, Huang J, Gan J, Huang X, Bucci E, Piacentini M, Ippolito G, Melino G. 2020. <a href="https://doi.org/10.1038/s41418-020-0530-3">COVID-19 infection: the perspectives on immune responses</a>. Cell Death Differ 27:1451–1454.</div>
</div>
<div id="ref-nCVMwr28" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">744. </div><div class="csl-right-inline">Fajgenbaum DC, June CH. 2020. <a href="https://doi.org/10.1056/nejmra2026131">Cytokine Storm</a>. New England Journal of Medicine 383:2255–2273.</div>
</div>
<div id="ref-gmWrMKlb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">745. </div><div class="csl-right-inline">Nicholls JM, Poon LL, Lee KC, Ng WF, Lai ST, Leung CY, Chu CM, Hui PK, Mak KL, Lim W, Yan KW, Chan KH, Tsang NC, Guan Y, Yuen KY, Malik Peiris J. 2003. <a href="https://doi.org/10.1016/s0140-6736(03)13413-7">Lung pathology of fatal severe acute respiratory syndrome</a>. The Lancet 361:1773–1778.</div>
</div>
<div id="ref-7pMLoPq9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">746. </div><div class="csl-right-inline">Gomersall CD, Kargel MJ, Lapinsky SE. 2004. <a href="https://doi.org/10.1186/cc2452">Pro/con clinical debate: Steroids are a key component in the treatment of SARS</a>. Critical Care 8:105.</div>
</div>
<div id="ref-RR3eH1DH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">747. </div><div class="csl-right-inline">Prager R, Pratte MT, Unni RR, Bala S, Ng Fat Hing N, Wu K, McGrath TA, Thomas A, Thoma B, Kyeremanteng K. 2021. Content Analysis and Characterization of Medical Tweets During the Early Covid-19 Pandemic. Cureus https://doi.org/<a href="https://doi.org/10.7759/cureus.13594">10.7759/cureus.13594</a>.</div>
</div>
<div id="ref-CcEjS89m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">748. </div><div class="csl-right-inline">Parkway NPSM, Durham S200. 2020. Small Molecules vs Biologics | Drug Development Differences. PK / PD and Clinical Pharmacology Consultants. <a href="https://www.nuventra.com/resources/blog/small-molecules-versus-biologics/">https://www.nuventra.com/resources/blog/small-molecules-versus-biologics/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-ZtfLQJhh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">749. </div><div class="csl-right-inline">Drews J. 2000. <a href="https://doi.org/10.1126/science.287.5460.1960">Drug Discovery: A Historical Perspective</a>. Science 287:1960–1964.</div>
</div>
<div id="ref-1EZ5IFkua" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">750. </div><div class="csl-right-inline">Thompson AE, Ranard BL, Wei Y, Jelic S. 2020. <a href="https://doi.org/10.1001/jamainternmed.2020.3030">Prone Positioning in Awake, Nonintubated Patients With COVID-19 Hypoxemic Respiratory Failure</a>. JAMA Intern Med 180:1537.</div>
</div>
<div id="ref-jJCnee9O" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">751. </div><div class="csl-right-inline">Stockman LJ, Bellamy R, Garner P. 2006. <a href="https://doi.org/10.1371/journal.pmed.0030343">SARS: Systematic Review of Treatment Effects</a>. PLoS Med 3:e343.</div>
</div>
<div id="ref-M1ZtcbRp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">752. </div><div class="csl-right-inline">Fujii T, Iwamoto A, Nakamura T, Iwamoto A. 2004. <a href="https://doi.org/10.1007/s10156-003-0296-9">Current concepts in SARS treatment</a>. Journal of Infection and Chemotherapy 10:1–7.</div>
</div>
<div id="ref-Xh7pYr41" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">753. </div><div class="csl-right-inline">Stern A, Skalsky K, Avni T, Carrara E, Leibovici L, Paul M. 2017. <a href="https://doi.org/10.1002/14651858.cd007720.pub3">Corticosteroids for pneumonia</a>. Cochrane Database of Systematic Reviews 2017.</div>
</div>
<div id="ref-ueLoJJY7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">754. </div><div class="csl-right-inline">Chen Y, Li K, Pu H, Wu T. 2011. Corticosteroids for pneumonia. Cochrane Database of Systematic Reviews https://doi.org/<a href="https://doi.org/10.1002/14651858.cd007720.pub2">10.1002/14651858.cd007720.pub2</a>.</div>
</div>
<div id="ref-11fgsEIDI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">755. </div><div class="csl-right-inline">Sibila O, Agusti C, Torres A. 2008. <a href="https://doi.org/10.1183/09031936.00154107">Corticosteroids in severe pneumonia</a>. European Respiratory Journal 32:259–264.</div>
</div>
<div id="ref-17fxIsLBV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">756. </div><div class="csl-right-inline">Mikami K, Suzuki M, Kitagawa H, Kawakami M, Hirota N, Yamaguchi H, Narumoto O, Kichikawa Y, Kawai M, Tashimo H, Arai H, Horiuchi T, Sakamoto Y. 2007. <a href="https://doi.org/10.1007/s00408-007-9020-3">Efficacy of Corticosteroids in the Treatment of Community-Acquired Pneumonia Requiring Hospitalization</a>. Lung 185:249–255.</div>
</div>
<div id="ref-yl2THbEA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">757. </div><div class="csl-right-inline">Nie W, Zhang Y, Cheng J, Xiu Q. 2012. <a href="https://doi.org/10.1371/journal.pone.0047926">Corticosteroids in the Treatment of Community-Acquired Pneumonia in Adults: A Meta-Analysis</a>. PLoS ONE 7:e47926.</div>
</div>
<div id="ref-KGBLpge1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">758. </div><div class="csl-right-inline">Fernández-Serrano S, Dorca J, Garcia-Vidal C, Fernández-Sabé N, Carratalà J, Fernández-Agüera A, Corominas M, Padrones S, Gudiol F, Manresa F. 2011. <a href="https://doi.org/10.1186/cc10103">Effect of corticosteroids on the clinical course of community-acquired pneumonia: a randomized controlled trial</a>. Critical Care 15:R96.</div>
</div>
<div id="ref-1DUXbHUEW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">759. </div><div class="csl-right-inline">Villar J, Ferrando C, Martínez D, Ambrós A, Muñoz T, Soler JA, Aguilar G, Alba F, González-Higueras E, Conesa LA, Martín-Rodríguez C, Díaz-Domínguez FJ, Serna-Grande P, Rivas R, Ferreres J, Belda J, Capilla L, Tallet A, Añón JM, Fernández RL, González-Martín JM, Aguilar G, Alba F, Álvarez J, Ambrós A, Añón JM, Asensio MJ, Belda J, Blanco J, Blasco M, Cachafeiro L, del Campo R, Capilla L, Carbonell JA, Carbonell N, Cariñena A, Carriedo D, Chico M, Conesa LA, Corpas R, Cuervo J, Díaz-Domínguez FJ, Domínguez-Antelo C, Fernández L, Fernández RL, Ferrando C, Ferreres J, Gamboa E, González-Higueras E, González-Luengo RI, González-Martín JM, Martínez D, Martín-Rodríguez C, Muñoz T, Ortiz Díaz-Miguel R, Pérez-González R, Prieto AM, Prieto I, Rivas R, Rojas-Viguera L, Romera MA, Sánchez-Ballesteros J, Segura JM, Serna-Grande P, Serrano A, Solano R, Soler JA, Soro M, Tallet A, Villar J. 2020. <a href="https://doi.org/10.1016/s2213-2600(19)30417-5">Dexamethasone treatment for the acute respiratory distress syndrome: a multicentre, randomised controlled trial</a>. The Lancet Respiratory Medicine 8:267–276.</div>
</div>
<div id="ref-WhuiEHlo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">760. </div><div class="csl-right-inline">Reddy K, O'Kane C, McAuley D. 2020. <a href="https://doi.org/10.1016/s2213-2600(20)30048-5">Corticosteroids in acute respiratory distress syndrome: a step forward, but more evidence is needed</a>. The Lancet Respiratory Medicine 8:220–222.</div>
</div>
<div id="ref-DWCwruB0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">761. </div><div class="csl-right-inline">Calfee CS, Matthay MA. 2007. <a href="https://doi.org/10.1378/chest.06-1743">Nonventilatory Treatments for Acute Lung Injury and ARDS</a>. Chest 131:913–920.</div>
</div>
<div id="ref-vx02q3mZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">762. </div><div class="csl-right-inline">Meduri GU, Marik PE, Pastores SM, Annane D. 2007. <a href="https://doi.org/10.1378/chest.07-0714">Corticosteroids in ARDS</a>. Chest 132:1093–1094.</div>
</div>
<div id="ref-V6tUK1F0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">763. </div><div class="csl-right-inline">2006. <a href="https://doi.org/10.1056/nejmoa051693">Efficacy and Safety of Corticosteroids for Persistent Acute Respiratory Distress Syndrome</a>. N Engl J Med 354:1671–1684.</div>
</div>
<div id="ref-4f82Cr8A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">764. </div><div class="csl-right-inline">Peter JV, John P, Graham PL, Moran JL, George IA, Bersten A. 2008. <a href="https://doi.org/10.1136/bmj.39537.939039.be">Corticosteroids in the prevention and treatment of acute respiratory distress syndrome (ARDS) in adults: meta-analysis</a>. BMJ 336:1006–1009.</div>
</div>
<div id="ref-6qEFbCHC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">765. </div><div class="csl-right-inline">Yu WC. 2004. <a href="https://doi.org/10.1136/thx.2003.017665">Antiviral agents and corticosteroids in the treatment of severe acute respiratory syndrome (SARS)</a>. Thorax 59:643–645.</div>
</div>
<div id="ref-15K1BCAlx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">766. </div><div class="csl-right-inline">Yin-Chun Yam L, Chun-Wing Lau A, Yuk-Lin Lai F, Shung E, Chan J, Wong V. 2007. <a href="https://doi.org/10.1016/j.jinf.2006.01.005">Corticosteroid treatment of severe acute respiratory syndrome in Hong Kong</a>. Journal of Infection 54:28–39.</div>
</div>
<div id="ref-Rmu8ZB9w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">767. </div><div class="csl-right-inline">Li H, Chen C, Hu F, Wang J, Zhao Q, Gale RP, Liang Y. 2020. <a href="https://doi.org/10.1038/s41375-020-0848-3">Impact of corticosteroid therapy on outcomes of persons with SARS-CoV-2, SARS-CoV, or MERS-CoV infection: a systematic review and meta-analysis</a>. Leukemia 34:1503–1511.</div>
</div>
<div id="ref-coPH1YK8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">768. </div><div class="csl-right-inline">Wenzel RP, Edmond MB. 2003. <a href="https://doi.org/10.1056/nejmp030072">Managing SARS amidst Uncertainty</a>. N Engl J Med 348:1947–1948.</div>
</div>
<div id="ref-q0TqfpEr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">769. </div><div class="csl-right-inline">Khan MOF, Lee HJ. 2008. <a href="https://doi.org/10.1021/cr068203e">Synthesis and Pharmacology of Anti-Inflammatory Steroidal Antedrugs</a>. Chemical Reviews 108:5131–5145.</div>
</div>
<div id="ref-2M8wLShq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">770. </div><div class="csl-right-inline">Drug vignettes: Dexamethasone. The Centre for Evidence-Based Medicine. <a href="https://www.cebm.net/covid-19/dexamethasone/">https://www.cebm.net/covid-19/dexamethasone/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-1ETGa7JyL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">771. </div><div class="csl-right-inline">Zabirowicz ES, Gan TJ. 2019. <a href="https://doi.org/10.1016/b978-0-323-48110-6.00034-x">Pharmacology of Postoperative Nausea and Vomiting</a>, p. 671–692. <em>In</em>. Elsevier BV.</div>
</div>
<div id="ref-19dEKfvUE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">772. </div><div class="csl-right-inline">Zhou W, Liu Y, Tian D, Wang C, Wang S, Cheng J, Hu M, Fang M, Gao Y. 2020. <a href="https://doi.org/10.1038/s41392-020-0127-9">Potential benefits of precise corticosteroids therapy for severe 2019-nCoV pneumonia</a>. Signal Transduction and Targeted Therapy 5:18.</div>
</div>
<div id="ref-KQ7uhAaa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">773. </div><div class="csl-right-inline">Arth GE, Johnston DBR, Fried J, Spooncer WW, Hoff DR, Sarett LH. 2002. <a href="https://doi.org/10.1021/ja01545a061">16-METHYLATED STEROIDS. I. 16α-METHYLATED ANALOGS OF CORTISONE, A NEW GROUP OF ANTI-INFLAMMATORY STEROIDS</a>. Journal of the American Chemical Society 80:3160–3161.</div>
</div>
<div id="ref-bSNVCQbh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">774. </div><div class="csl-right-inline">Cohen A. 1960. <a href="https://doi.org/10.1001/jama.1960.03030070009002">Treatment of Rheumatoid Arthritis with Dexamethasone</a>. JAMA 174:831.</div>
</div>
<div id="ref-17Dq3dNwL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">775. </div><div class="csl-right-inline">2007. Dexamethasone. DailyMed. <a href="https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=537b424a-3e07-4c81-978c-1ad99014032a">https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=537b424a-3e07-4c81-978c-1ad99014032a</a>.</div>
</div>
<div id="ref-nbiwprc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">776. </div><div class="csl-right-inline">Orlicka K, Barnes E, Culver EL. 2013. <a href="https://doi.org/10.1177/2040622313485275">Prevention of infection caused by immunosuppressive drugs in gastroenterology</a>. Therapeutic Advances in Chronic Disease 4:167–185.</div>
</div>
<div id="ref-Ji2fchyS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">777. </div><div class="csl-right-inline">Russell CD, Millar JE, Baillie JK. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30317-2">Clinical evidence does not support corticosteroid treatment for 2019-nCoV lung injury</a>. The Lancet 395:473–475.</div>
</div>
<div id="ref-xndHds7d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">778. </div><div class="csl-right-inline">Shang L, Zhao J, Hu Y, Du R, Cao B. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30361-5">On the use of corticosteroids for 2019-nCoV pneumonia</a>. The Lancet 395:683–684.</div>
</div>
<div id="ref-suHq6Qnm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">779. </div><div class="csl-right-inline">Ritchie AI, Singanayagam A. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30691-7">Immunosuppression for hyperinflammation in COVID-19: a double-edged sword?</a> The Lancet 395:1111.</div>
</div>
<div id="ref-pzQoUwz3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">780. </div><div class="csl-right-inline">Horby P, Lim WS, Emberson J, Mafham M, Bell J, Linsell L, Staplin N, Brightling C, Ustianowski A, Elmahi E, Prudon B, Green C, Felton T, Chadwick D, Rege K, Fegan C, Chappell LC, Faust SN, Jaki T, Jeffery K, Montgomery A, Rowan K, Juszczak E, Baillie JK, Haynes R, Landray MJ, RECOVERY Collaborative Group. 2020. Effect of Dexamethasone in Hospitalized Patients with COVID-19 – Preliminary Report. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.06.22.20137273">10.1101/2020.06.22.20137273</a>.</div>
</div>
<div id="ref-H3C2PalE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">781. </div><div class="csl-right-inline">The RECOVERY Collaborative Group. 2020. <a href="https://doi.org/10.1056/nejmoa2021436">Dexamethasone in Hospitalized Patients with Covid-19 — Preliminary Report</a>. New England Journal of Medicine NEJMoa2021436.</div>
</div>
<div id="ref-wNXTNffA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">782. </div><div class="csl-right-inline">Pasin L, Navalesi P, Zangrillo A, Kuzovlev A, Likhvantsev V, Hajjar LA, Fresilli S, Lacerda MVG, Landoni G. 2021. <a href="https://doi.org/10.1053/j.jvca.2020.11.057">Corticosteroids for Patients With Coronavirus Disease 2019 (COVID-19) With Different Disease Severity: A Meta-Analysis of Randomized Clinical Trials</a>. Journal of Cardiothoracic and Vascular Anesthesia 35:578–584.</div>
</div>
<div id="ref-H70g2GF9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">783. </div><div class="csl-right-inline">Lee DW, Gardner R, Porter DL, Louis CU, Ahmed N, Jensen M, Grupp SA, Mackall CL. 2014. <a href="https://doi.org/10.1182/blood-2014-05-552729">Current concepts in the diagnosis and management of cytokine release syndrome</a>. Blood 124:188–195.</div>
</div>
<div id="ref-QgKZfGgK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">784. </div><div class="csl-right-inline">Prescott HC, Rice TW. 2020. <a href="https://doi.org/10.1001/jama.2020.16747">Corticosteroids in COVID-19 ARDS</a>. JAMA 324:1292.</div>
</div>
<div id="ref-1EjFnlgU8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">785. </div><div class="csl-right-inline">Noreen S, Maqbool I, Madni A. 2021. <a href="https://doi.org/10.1016/j.ejphar.2021.173854">Dexamethasone: Therapeutic potential, risks, and future projection during COVID-19 pandemic</a>. European Journal of Pharmacology 894:173854.</div>
</div>
<div id="ref-e3ZiDOSQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">786. </div><div class="csl-right-inline">Mahase E. 2020. <a href="https://doi.org/10.1136/bmj.m2512">Covid-19: Demand for dexamethasone surges as RECOVERY trial publishes preprint</a>. BMJ m2512.</div>
</div>
<div id="ref-14gthSuhm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">787. </div><div class="csl-right-inline">Dimmock NJ, Easton AJ, Leppard KN. 2007. Introduction to modern virology6th ed. Blackwell Pub, Malden, MA.</div>
</div>
<div id="ref-zeTnEn7e" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">788. </div><div class="csl-right-inline">CORONA Data Viewer. Castleman Disease Collaborative Network. <a href="https://cdcn.org/corona-data-viewer/">https://cdcn.org/corona-data-viewer/</a>. Retrieved 12 August 2021.</div>
</div>
<div id="ref-tn6j6IZ0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">789. </div><div class="csl-right-inline">2015. CoronavirusesMethods in Molecular Biology. Springer New York. <a href="https://doi.org/ggqfqx">https://doi.org/ggqfqx</a>.</div>
</div>
<div id="ref-Sh1FwWps" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">790. </div><div class="csl-right-inline">Lung J, Lin Y, Yang Y, Chou Y, Shu L, Cheng Y, Liu HT, Wu C. 2020. <a href="https://doi.org/10.1002/jmv.25761">The potential chemical structure of anti‐SARS‐CoV‐2 RNA‐dependent RNA polymerase</a>. Journal of Medical Virology 92:693–697.</div>
</div>
<div id="ref-6Qlsbuqr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">791. </div><div class="csl-right-inline">Totura AL, Bavari S. 2019. <a href="https://doi.org/10.1080/17460441.2019.1581171">Broad-spectrum coronavirus antiviral drug discovery</a>. Expert Opinion on Drug Discovery 14:397–412.</div>
</div>
<div id="ref-kO7WRULi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">792. </div><div class="csl-right-inline">Tong S, Su Y, Yu Y, Wu C, Chen J, Wang S, Jiang J. 2020. <a href="https://doi.org/10.1016/j.ijantimicag.2020.106114">Ribavirin therapy for severe COVID-19: a retrospective cohort study</a>. International Journal of Antimicrobial Agents 56:106114.</div>
</div>
<div id="ref-MPJdSF1p" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">793. </div><div class="csl-right-inline">Seley-Radtke KL, Yates MK. 2018. <a href="https://doi.org/10.1016/j.antiviral.2018.04.004">The evolution of nucleoside analogue antivirals: A review for chemists and non-chemists. Part 1: Early structural modifications to the nucleoside scaffold</a>. Antiviral Research 154:66–86.</div>
</div>
<div id="ref-ukZRKllk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">794. </div><div class="csl-right-inline">Jin Z, Smith LK, Rajwanshi VK, Kim B, Deval J. 2013. <a href="https://doi.org/10.1371/journal.pone.0068347">The Ambiguous Base-Pairing and High Substrate Efficiency of T-705 (Favipiravir) Ribofuranosyl 5′-Triphosphate towards Influenza A Virus Polymerase</a>. PLoS ONE 8:e68347.</div>
</div>
<div id="ref-IkAU62Pd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">795. </div><div class="csl-right-inline">2020. Favipiravir. DrugBank. <a href="https://www.drugbank.ca/drugs/DB12466">https://www.drugbank.ca/drugs/DB12466</a>.</div>
</div>
<div id="ref-EgoJKtW4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">796. </div><div class="csl-right-inline">Furuta Y, Takahashi K, Fukuda Y, Kuno M, Kamiyama T, Kozaki K, Nomura N, Egawa H, Minami S, Watanabe Y, Narita H, Shiraki K. 2002. <a href="https://doi.org/10.1128/aac.46.4.977-981.2002">In Vitro and In Vivo Activities of Anti-Influenza Virus Compound T-705</a>. Antimicrobial Agents and Chemotherapy 46:977–981.</div>
</div>
<div id="ref-rG5RCRdJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">797. </div><div class="csl-right-inline">Sidwell RW, Barnard DL, Day CW, Smee DF, Bailey KW, Wong M-H, Morrey JD, Furuta Y. 2007. <a href="https://doi.org/10.1128/aac.01051-06">Efficacy of Orally Administered T-705 on Lethal Avian Influenza A (H5N1) Virus Infections in Mice</a>. Antimicrobial Agents and Chemotherapy 51:845–851.</div>
</div>
<div id="ref-DpdKJbki" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">798. </div><div class="csl-right-inline">Furuta Y, Takahashi K, Kuno-Maekawa M, Sangawa H, Uehara S, Kozaki K, Nomura N, Egawa H, Shiraki K. 2005. <a href="https://doi.org/10.1128/aac.49.3.981-986.2005">Mechanism of Action of T-705 against Influenza Virus</a>. Antimicrobial Agents and Chemotherapy 49:981–986.</div>
</div>
<div id="ref-9j7NVPJS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">799. </div><div class="csl-right-inline">Julander JG, Shafer K, Smee DF, Morrey JD, Furuta Y. 2009. <a href="https://doi.org/10.1128/aac.01074-08">Activity of T-705 in a Hamster Model of Yellow Fever Virus Infection in Comparison with That of a Chemically Related Compound, T-1106</a>. Antimicrobial Agents and Chemotherapy 53:202–209.</div>
</div>
<div id="ref-IYEoCGHy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">800. </div><div class="csl-right-inline">Gowen BB, Wong M-H, Jung K-H, Sanders AB, Mendenhall M, Bailey KW, Furuta Y, Sidwell RW. 2007. <a href="https://doi.org/10.1128/aac.00356-07">In Vitro and In Vivo Activities of T-705 against Arenavirus and Bunyavirus Infections</a>. Antimicrobial Agents and Chemotherapy 51:3168–3176.</div>
</div>
<div id="ref-7d1TnpE3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">801. </div><div class="csl-right-inline">Rocha-Pereira J, Jochmans D, Dallmeier K, Leyssen P, Nascimento MSJ, Neyts J. 2012. <a href="https://doi.org/10.1016/j.bbrc.2012.07.034">Favipiravir (T-705) inhibits in vitro norovirus replication</a>. Biochemical and Biophysical Research Communications 424:777–780.</div>
</div>
<div id="ref-15kQ4QYbx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">802. </div><div class="csl-right-inline">Mendenhall M, Russell A, Juelich T, Messina EL, Smee DF, Freiberg AN, Holbrook MR, Furuta Y, de la Torre J-C, Nunberg JH, Gowen BB. 2011. <a href="https://doi.org/10.1128/aac.01219-10">T-705 (Favipiravir) Inhibition of Arenavirus Replication in Cell Culture</a>. Antimicrobial Agents and Chemotherapy 55:782–787.</div>
</div>
<div id="ref-14p3btkRH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">803. </div><div class="csl-right-inline">FURUTA Y, KOMENO T, NAKAMURA T. 2017. <a href="https://doi.org/10.2183/pjab.93.027">Favipiravir (T-705), a broad spectrum inhibitor of viral RNA polymerase</a>. Proceedings of the Japan Academy, Series B 93:449–463.</div>
</div>
<div id="ref-YaxKcqq8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">804. </div><div class="csl-right-inline">Gordon CJ, Tchesnokov EP, Feng JY, Porter DP, Götte M. 2020. <a href="https://doi.org/10.1074/jbc.ac120.013056">The antiviral compound remdesivir potently inhibits RNA-dependent RNA polymerase from Middle East respiratory syndrome coronavirus</a>. Journal of Biological Chemistry 295:4773–4779.</div>
</div>
<div id="ref-lIko9Imd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">805. </div><div class="csl-right-inline">Agostini ML, Andres EL, Sims AC, Graham RL, Sheahan TP, Lu X, Smith EC, Case JB, Feng JY, Jordan R, Ray AS, Cihlar T, Siegel D, Mackman RL, Clarke MO, Baric RS, Denison MR. 2018. <a href="https://doi.org/10.1128/mbio.00221-18">Coronavirus Susceptibility to the Antiviral Remdesivir (GS-5734) Is Mediated by the Viral Polymerase and the Proofreading Exoribonuclease</a>. mBio 9:e00221–18.</div>
</div>
<div id="ref-IhdcvbzC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">806. </div><div class="csl-right-inline">Wang M, Cao R, Zhang L, Yang X, Liu J, Xu M, Shi Z, Hu Z, Zhong W, Xiao G. 2020. <a href="https://doi.org/10.1038/s41422-020-0282-0">Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</a>. Cell Research 30:269–271.</div>
</div>
<div id="ref-DtpdBrgt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">807. </div><div class="csl-right-inline">Beigel JH, Tomashek KM, Dodd LE, Mehta AK, Zingman BS, Kalil AC, Hohmann E, Chu HY, Luetkemeyer A, Kline S, Lopez de Castilla D, Finberg RW, Dierberg K, Tapson V, Hsieh L, Patterson TF, Paredes R, Sweeney DA, Short WR, Touloumi G, Lye DC, Ohmagari N, Oh M, Ruiz-Palacios GM, Benfield T, Fätkenheuer G, Kortepeter MG, Atmar RL, Creech CB, Lundgren J, Babiker AG, Pett S, Neaton JD, Burgess TH, Bonnett T, Green M, Makowski M, Osinusi A, Nayak S, Lane HC. 2020. <a href="https://doi.org/10.1056/nejmoa2007764">Remdesivir for the Treatment of Covid-19 — Final Report</a>. New England Journal of Medicine 383:1813–1826.</div>
</div>
<div id="ref-1ATTPbCCO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">808. </div><div class="csl-right-inline">National Institute of Allergy and Infectious Diseases (NIAID). 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04280705">A Multicenter, Adaptive, Randomized Blinded Controlled Trial of the Safety and Efficacy of Investigational Therapeutics for the Treatment of COVID-19 in Hospitalized Adults</a>. NCT04280705. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-yTCAmOyt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">809. </div><div class="csl-right-inline">Gilead Sciences. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04292899">A Phase 3 Randomized Study to Evaluate the Safety and Antiviral Activity of Remdesivir (GS-5734™) in Participants With Severe COVID-19</a>. NCT04292899. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-16zLDIOyk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">810. </div><div class="csl-right-inline">Grein J, Ohmagari N, Shin D, Diaz G, Asperges E, Castagna A, Feldt T, Green G, Green ML, Lescure F-X, Nicastri E, Oda R, Yo K, Quiros-Roldan E, Studemeister A, Redinski J, Ahmed S, Bernett J, Chelliah D, Chen D, Chihara S, Cohen SH, Cunningham J, D’Arminio Monforte A, Ismail S, Kato H, Lapadula G, L’Her E, Maeno T, Majumder S, Massari M, Mora-Rillo M, Mutoh Y, Nguyen D, Verweij E, Zoufaly A, Osinusi AO, DeZure A, Zhao Y, Zhong L, Chokkalingam A, Elboudwarej E, Telep L, Timbs L, Henne I, Sellers S, Cao H, Tan SK, Winterbourne L, Desai P, Mera R, Gaggar A, Myers RP, Brainard DM, Childs R, Flanigan T. 2020. <a href="https://doi.org/10.1056/nejmoa2007016">Compassionate Use of Remdesivir for Patients with Severe Covid-19</a>. New England Journal of Medicine 382:2327–2336.</div>
</div>
<div id="ref-jShm9uhy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">811. </div><div class="csl-right-inline">WHO Solidarity Trial Consortium. 2020. <a href="https://doi.org/10.1056/nejmoa2023184">Repurposed Antiviral Drugs for Covid-19 — Interim WHO Solidarity Trial Results</a>. New England Journal of Medicine NEJMoa2023184.</div>
</div>
<div id="ref-1CEGhPPDd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">812. </div><div class="csl-right-inline">2020. PLAQUENIL - hydroxychloroquine sulfate tablet. DailyMed. <a href="https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=34496b43-05a2-45fb-a769-52b12e099341">https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=34496b43-05a2-45fb-a769-52b12e099341</a>.</div>
</div>
<div id="ref-16IRfGFKb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">813. </div><div class="csl-right-inline">Kalia S, Dutz JP. 2007. <a href="https://doi.org/10.1111/j.1529-8019.2007.00131.x">New concepts in antimalarial use and mode of action in dermatology</a>. Dermatologic Therapy 20:160–174.</div>
</div>
<div id="ref-x6nInyCf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">814. </div><div class="csl-right-inline">Vincent MJ, Bergeron E, Benjannet S, Erickson BR, Rollin PE, Ksiazek TG, Seidah NG, Nichol ST. 2005. <a href="https://doi.org/10.1186/1743-422x-2-69">Chloroquine is a potent inhibitor of SARS coronavirus infection and spread</a>. Virology Journal 2:69.</div>
</div>
<div id="ref-1GBsnkeLK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">815. </div><div class="csl-right-inline">Yao X, Ye F, Zhang M, Cui C, Huang B, Niu P, Liu X, Zhao L, Dong E, Song C, Zhan S, Lu R, Li H, Tan W, Liu D. 2020. <a href="https://doi.org/10.1093/cid/ciaa237">In Vitro Antiviral Activity and Projection of Optimized Dosing Design of Hydroxychloroquine for the Treatment of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)</a>. Clinical Infectious Diseases 71:732–739.</div>
</div>
<div id="ref-ovHLMvCi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">816. </div><div class="csl-right-inline">Gautret P, Lagier J-C, Parola P, Hoang VT, Meddeb L, Mailhe M, Doudier B, Courjon J, Giordanengo V, Vieira VE, Tissot Dupont H, Honoré S, Colson P, Chabrière E, La Scola B, Rolain J-M, Brouqui P, Raoult D. 2020. <a href="https://doi.org/10.1016/j.ijantimicag.2020.105949">Hydroxychloroquine and azithromycin as a treatment of COVID-19: results of an open-label non-randomized clinical trial</a>. International Journal of Antimicrobial Agents 56:105949.</div>
</div>
<div id="ref-10J3hBiMo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">817. </div><div class="csl-right-inline">Voss A. 2020. Official Statement from International Society of Antimicrobial Chemotherapy. <a href="https://www.isac.world/news-and-publications/official-isac-statement">https://www.isac.world/news-and-publications/official-isac-statement</a>.</div>
</div>
<div id="ref-KwFZyfFx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">818. </div><div class="csl-right-inline">The RECOVERY Collaborative Group. 2020. <a href="https://doi.org/10.1056/nejmoa2022926">Effect of Hydroxychloroquine in Hospitalized Patients with Covid-19</a>. New England Journal of Medicine 383:2030–2040.</div>
</div>
<div id="ref-ToVSTZlB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">819. </div><div class="csl-right-inline">No clinical benefit from use of hydroxychloroquine in hospitalised patients with COVID-19 — RECOVERY Trial. <a href="https://www.recoverytrial.net/news/statement-from-the-chief-investigators-of-the-randomised-evaluation-of-covid-19-therapy-recovery-trial-on-hydroxychloroquine-5-june-2020-no-clinical-benefit-from-use-of-hydroxychloroquine-in-hospitalised-patients-with-covid-19">https://www.recoverytrial.net/news/statement-from-the-chief-investigators-of-the-randomised-evaluation-of-covid-19-therapy-recovery-trial-on-hydroxychloroquine-5-june-2020-no-clinical-benefit-from-use-of-hydroxychloroquine-in-hospitalised-patients-with-covid-19</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-iUFEavbp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">820. </div><div class="csl-right-inline">Õmura S, Crump A. 2004. <a href="https://doi.org/10.1038/nrmicro1048">The life and times of ivermectin — a success story</a>. Nat Rev Microbiol 2:984–989.</div>
</div>
<div id="ref-Z3mIi5lw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">821. </div><div class="csl-right-inline">Burg RW, Miller BM, Baker EE, Birnbaum J, Currie SA, Hartman R, Kong Y-L, Monaghan RL, Olson G, Putter I, Tunac JB, Wallick H, Stapley EO, Oiwa R, Ōmura S. 1979. <a href="https://doi.org/10.1128/aac.15.3.361">Avermectins, New Family of Potent Anthelmintic Agents: Producing Organism and Fermentation</a>. Antimicrob Agents Chemother 15:361–367.</div>
</div>
<div id="ref-JqaUFO6M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">822. </div><div class="csl-right-inline">CRUMP A, OMURA S. 2011. <a href="https://doi.org/10.2183/pjab.87.13">Ivermectin, 'Wonder drug' from Japan: the human use perspective</a>. Proceedings of the Japan Academy Ser B: Physical and Biological Sciences 87:13–28.</div>
</div>
<div id="ref-h0d0XvmB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">823. </div><div class="csl-right-inline">Crump A. 2017. <a href="https://doi.org/10.1038/ja.2017.11">Ivermectin: enigmatic multifaceted ‘wonder’ drug continues to surprise and exceed expectations</a>. J Antibiot 70:495–505.</div>
</div>
<div id="ref-LTczTlEH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">824. </div><div class="csl-right-inline">Egerton JR, Ostlind DA, Blair LS, Eary CH, Suhayda D, Cifelli S, Riek RF, Campbell WC. 1979. <a href="https://doi.org/10.1128/aac.15.3.372">Avermectins, New Family of Potent Anthelmintic Agents: Efficacy of the B1a Component</a>. Antimicrobial Agents and Chemotherapy 15:372–378.</div>
</div>
<div id="ref-tVRUg0uE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">825. </div><div class="csl-right-inline">Heidary F, Gharebaghi R. 2020. <a href="https://doi.org/10.1038/s41429-020-0336-z">Ivermectin: a systematic review from antiviral effects to COVID-19 complementary regimen</a>. J Antibiot 73:593–602.</div>
</div>
<div id="ref-1DkK0T25n" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">826. </div><div class="csl-right-inline">Sharun K, Dhama K, Patel SK, Pathak M, Tiwari R, Singh BR, Sah R, Bonilla-Aldana DK, Rodriguez-Morales AJ, Leblebicioglu H. 2020. <a href="https://doi.org/10.1186/s12941-020-00368-w">Ivermectin, a new candidate therapeutic against SARS-CoV-2/COVID-19</a>. Ann Clin Microbiol Antimicrob 19.</div>
</div>
<div id="ref-8if8wVXj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">827. </div><div class="csl-right-inline">Caly L, Druce JD, Catton MG, Jans DA, Wagstaff KM. 2020. <a href="https://doi.org/10.1016/j.antiviral.2020.104787">The FDA-approved drug ivermectin inhibits the replication of SARS-CoV-2 in vitro</a>. Antiviral Research 178:104787.</div>
</div>
<div id="ref-1Bfy4jI64" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">828. </div><div class="csl-right-inline">Chaccour C, Hammann F, Ramón-García S, Rabinovich NR. 2020. <a href="https://doi.org/10.4269/ajtmh.20-0271">Ivermectin and COVID-19: Keeping Rigor in Times of Urgency</a>. The American Journal of Tropical Medicine and Hygiene 102:1156–1157.</div>
</div>
<div id="ref-1HpjZVMRq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">829. </div><div class="csl-right-inline">Schmith VD, Zhou J(Jessie), Lohmer LRL. 2020. <a href="https://doi.org/10.1002/cpt.1889">The Approved Dose of Ivermectin Alone is not the Ideal Dose for the Treatment of COVID‐19</a>. Clin Pharmacol Ther 108:762–765.</div>
</div>
<div id="ref-CqfRlfo5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">830. </div><div class="csl-right-inline">Momekov G, Momekova D. 2020. <a href="https://doi.org/10.1080/13102818.2020.1775118">Ivermectin as a potential COVID-19 treatment from the pharmacokinetic point of view: antiviral levels are not likely attainable with known dosing regimens</a>. Biotechnology &amp;amp; Biotechnological Equipment 34:469–474.</div>
</div>
<div id="ref-L4pkrMXn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">831. </div><div class="csl-right-inline">Ménez C, Sutra J-F, Prichard R, Lespine A. 2012. <a href="https://doi.org/10.1371/journal.pntd.0001883">Relative Neurotoxicity of Ivermectin and Moxidectin in Mdr1ab (−/−) Mice and Effects on Mammalian GABA(A) Channel Activity</a>. PLoS Negl Trop Dis 6:e1883.</div>
</div>
<div id="ref-2vqz8GpA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">832. </div><div class="csl-right-inline">Rajter JC, Sherman MS, Fatteh N, Vogel F, Sacks J, Rajter J-J. 2021. <a href="https://doi.org/10.1016/j.chest.2020.10.009">Use of Ivermectin Is Associated With Lower Mortality in Hospitalized Patients With Coronavirus Disease 2019</a>. Chest 159:85–92.</div>
</div>
<div id="ref-17pJ7jpyv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">833. </div><div class="csl-right-inline">Camprubí D, Almuedo-Riera A, Martí-Soler H, Soriano A, Hurtado JC, Subirà C, Grau-Pujol B, Krolewiecki A, Muñoz J. 2020. <a href="https://doi.org/10.1371/journal.pone.0242184">Lack of efficacy of standard doses of ivermectin in severe COVID-19 patients</a>. PLoS ONE 15:e0242184.</div>
</div>
<div id="ref-uqtvrCJU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">834. </div><div class="csl-right-inline">Gheibi N, Shakhsi Niaee M, Namdar P, Allami A, Zolghadr L, Javadi A, Karampour A, Varnaseri M, Bijani B, Cheraghi F, Naderi Y, Amini F, Karamyan M, YadYad M, Jamshidian R. 2021. <a href="https://doi.org/10.4103/1995-7645.318304">Ivermectin as an adjunct treatment for hospitalized adult COVID-19 patients: A randomized multi-center clinical trial</a>. Asian Pac J Trop Med 14:266.</div>
</div>
<div id="ref-OxEDSXRt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">835. </div><div class="csl-right-inline">Babalola OE, Bode CO, Ajayi AA, Alakaloko FM, Akase IE, Otrofanowei E, Salu OB, Adeyemo WL, Ademuyiwa AO, Omilabu S. 2021. <a href="https://doi.org/10.1093/qjmed/hcab035">Ivermectin shows clinical benefits in mild to moderate COVID19: a randomized controlled double-blind, dose-response study in Lagos</a>. QJM: An International Journal of Medicine 114:780–788.</div>
</div>
<div id="ref-EOk46HI2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">836. </div><div class="csl-right-inline">Pott-Junior H, Paoliello MMB, Miguel A de QC, da Cunha AF, de Melo Freire CC, Neves FF, da Silva de Av´o LR, Roscani MG, dos Santos SDS, Chach´a SGF. 2021. <a href="https://doi.org/10.1016/j.toxrep.2021.03.003">RETRACTED: Use of ivermectin in the treatment of Covid-19: A pilot trial</a>. Toxicology Reports 8:505–510.</div>
</div>
<div id="ref-1BWas3u6Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">837. </div><div class="csl-right-inline">Ahmed S, Karim MM, Ross AG, Hossain MS, Clemens JD, Sumiya MK, Phru CS, Rahman M, Zaman K, Somani J, Yasmin R, Hasnat MA, Kabir A, Aziz AB, Khan WA. 2021. <a href="https://doi.org/10.1016/j.ijid.2020.11.191">A five-day course of ivermectin for the treatment of COVID-19 may reduce the duration of illness</a>. International Journal of Infectious Diseases 103:214–216.</div>
</div>
<div id="ref-2BejjOhs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">838. </div><div class="csl-right-inline">Shahbaznejad L, Davoudi A, Eslami G, Markowitz JS, Navaeifar MR, Hosseinzadeh F, Movahedi FS, Rezai MS. 2021. <a href="https://doi.org/10.1016/j.clinthera.2021.04.007">Effects of Ivermectin in Patients With COVID-19: A Multicenter, Double-blind, Randomized, Controlled Clinical Trial</a>. Clinical Therapeutics 43:1007–1019.</div>
</div>
<div id="ref-TSzFDROU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">839. </div><div class="csl-right-inline">Chaccour C, Casellas A, Blanco-Di Matteo A, Pineda I, Fernandez-Montero A, Ruiz-Castillo P, Richardson M-A, Rodríguez-Mateos M, Jordán-Iborra C, Brew J, Carmona-Torre F, Giráldez M, Laso E, Gabaldón-Figueira JC, Dobaño C, Moncunill G, Yuste JR, Del Pozo JL, Rabinovich NR, Schöning V, Hammann F, Reina G, Sadaba B, Fernández-Alonso M. 2021. <a href="https://doi.org/10.1016/j.eclinm.2020.100720">The effect of early treatment with ivermectin on viral load, symptoms and humoral response in patients with non-severe COVID-19: A pilot, double-blind, placebo-controlled, randomized clinical trial</a>. EClinicalMedicine 32:100720.</div>
</div>
<div id="ref-Ve3f25OQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">840. </div><div class="csl-right-inline">Mohan A, Tiwari P, Suri T, Mittal S, Patel A, Jain A, T. V, Das UK, Bopanna TK, Pandey R, Shelke S, Singh AR, Bhatnagar S, Masih S, Mahajan S, Dwivedi T, Sahoo B, Pandit A, Bhopale S, Vig S, Gupta R, Madan K, Hadda V, Gupta N, Garg R, Meena VP, Guleria R. 2021. <a href="https://doi.org/10.21203/rs.3.rs-191648/v1">Ivermectin in mild and moderate COVID-19 (RIVET-COV): a randomized, placebo-controlled trial</a>. Research Square Platform LLC.</div>
</div>
<div id="ref-17ZhndzAi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">841. </div><div class="csl-right-inline">Ravikirti, Roy R, Pattadar C, Raj R, Agarwal N, Biswas B, Manjhi PK, Rai DK, Shyama, Kumar A, Sarfaraz A. 2021. <a href="https://doi.org/10.18433/jpps32105">Evaluation of Ivermectin as a Potential Treatment for Mild to Moderate COVID-19: A Double-Blind Randomized Placebo Controlled Trial in Eastern India</a>. J Pharm Pharm Sci 24:343–350.</div>
</div>
<div id="ref-1Bo9wyMhs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">842. </div><div class="csl-right-inline">Elgazzar A, Eltaweel A, Youssef SA, Hany B, Hafez M, Moussa H. 2020. <a href="https://doi.org/10.21203/rs.3.rs-100956/v3">Efficacy and Safety of Ivermectin for Treatment and prophylaxis of COVID-19 Pandemic</a>. Research Square Platform LLC.</div>
</div>
<div id="ref-tG2qcJCU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">843. </div><div class="csl-right-inline">Lawrence J. 2021. Why Was a Major Study on Ivermectin for COVID-19 Just Retracted? <a href="https://grftr.news/why-was-a-major-study-on-ivermectin-for-covid-19-just-retracted">https://grftr.news/why-was-a-major-study-on-ivermectin-for-covid-19-just-retracted</a>.</div>
</div>
<div id="ref-NZZPnKnn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">844. </div><div class="csl-right-inline">Brown N. 2021. Nick Brown's blog: Some problems in the dataset of a large study of Ivermectin for the treatment of Covid-19. Nick Brown's blog. <a href="https://steamtraen.blogspot.com/2021/07/Some-problems-with-the-data-from-a-Covid-study.html">https://steamtraen.blogspot.com/2021/07/Some-problems-with-the-data-from-a-Covid-study.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-MbzcgLLh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">845. </div><div class="csl-right-inline">2021. <a href="https://doi.org/10.21203/rs.3.rs-100956/v4">Efficacy and Safety of Ivermectin for Treatment and prophylaxis of COVID-19 Pandemic</a>. Research Square Platform LLC.</div>
</div>
<div id="ref-PcA4ZwSb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">846. </div><div class="csl-right-inline">López-Medina E, López P, Hurtado IC, Dávalos DM, Ramirez O, Martínez E, Díazgranados JA, Oñate JM, Chavarriaga H, Herrera S, Parra B, Libreros G, Jaramillo R, Avendaño AC, Toro DF, Torres M, Lesmes MC, Rios CA, Caicedo I. 2021. <a href="https://doi.org/10.1001/jama.2021.3071">Effect of Ivermectin on Time to Resolution of Symptoms Among Adults With Mild COVID-19</a>. JAMA 325:1426.</div>
</div>
<div id="ref-b8Hd08ep" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">847. </div><div class="csl-right-inline">Roman YM, Burela PA, Pasupuleti V, Piscoya A, Vidal JE, Hernandez AV. 2021. <a href="https://doi.org/10.1101/2021.05.21.21257595">Ivermectin for the treatment of COVID-19: A systematic review and meta-analysis of randomized controlled trials</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-17Cooa2LC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">848. </div><div class="csl-right-inline">Castañeda-Sabogal A, Chambergo-Michilot D, Toro-Huamanchumo CJ, Silva-Rengifo C, Gonzales-Zamora J, Barboza JJ. 2021. <a href="https://doi.org/10.1101/2021.01.26.21250420">Outcomes of Ivermectin in the treatment of COVID-19: a systematic review and meta-analysis</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-pLcPOKgu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">849. </div><div class="csl-right-inline">Hill A, Garratt A, Levi J, Falconer J, Ellis L, McCann K, Pilkington V, Qavi A, Wang J, Wentzel H. 2021. <a href="https://doi.org/10.1093/ofid/ofab394">RETRACTED: Expression of Concern: “Meta-analysis of Randomized Trials of Ivermectin to Treat SARS-CoV-2 Infection”</a>. Open Forum Infectious Diseases 8.</div>
</div>
<div id="ref-xhdEdLXT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">850. </div><div class="csl-right-inline">Zein AFMZ, Sulistiyana CS, Raffaelo WM, Pranata R. 2021. <a href="https://doi.org/10.1016/j.dsx.2021.102186">Ivermectin and mortality in patients with COVID-19: A systematic review, meta-analysis, and meta-regression of randomized controlled trials</a>. Diabetes &amp;amp; Metabolic Syndrome: Clinical Research &amp;amp; Reviews 15:102186.</div>
</div>
<div id="ref-1FR7dFWA6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">851. </div><div class="csl-right-inline">Bryant A, Lawrie TA, Dowswell T, Fordham EJ, Mitchell S, Hill SR, Tham TC. 2021. <a href="https://doi.org/10.1097/mjt.0000000000001402">Ivermectin for Prevention and Treatment of COVID-19 Infection</a>. American Journal of Therapeutics Publish Ahead of Print.</div>
</div>
<div id="ref-77jd6q1O" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">852. </div><div class="csl-right-inline">Hariyanto TI, Halim DA, Rosalind J, Gunawan C, Kurniawan A. 2021. <a href="https://doi.org/10.1002/rmv.2265">Ivermectin and outcomes from Covid‐19 pneumonia: A systematic review and meta‐analysis of randomized clinical trial studies</a>. Reviews in Medical Virology 32.</div>
</div>
<div id="ref-w1um3tWE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">853. </div><div class="csl-right-inline">Bryant A, Lawrie TA, Dowswell T, Fordham E, Scott M MD, Hill SR, Tham TC. 2021. <a href="https://doi.org/10.31219/osf.io/k37ft">Ivermectin for prevention and treatment of COVID-19 infection: a systematic review and meta-analysis</a>. Center for Open Science.</div>
</div>
<div id="ref-wgyQ7l6C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">854. </div><div class="csl-right-inline">Kory P, Meduri GU, Varon J, Iglesias J, Marik PE. 2021. <a href="https://doi.org/10.1097/mjt.0000000000001377">Review of the Emerging Evidence Demonstrating the Efficacy of Ivermectin in the Prophylaxis and Treatment of COVID-19</a>. American Journal of Therapeutics 28:e299–e318.</div>
</div>
<div id="ref-BGQbf0zT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">855. </div><div class="csl-right-inline">Kow CS, Merchant HA, Mustafa ZU, Hasan SS. 2021. <a href="https://doi.org/10.1007/s43440-021-00245-z">The association between the use of ivermectin and mortality in patients with COVID-19: a meta-analysis</a>. Pharmacol Rep 73:1473–1479.</div>
</div>
<div id="ref-1BnKCCFKX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">856. </div><div class="csl-right-inline">Karale S, Bansal V, Makadia J, Tayyeb M, Khan H, Ghanta SS, Singh R, Tekin A, Bhurwal A, Mutneja H, Mehra I, Kashyap R. 2021. <a href="https://doi.org/10.1101/2021.04.30.21256415">An Updated Systematic Review and Meta-Analysis of Mortality, Need for ICU admission, Use of Mechanical Ventilation, Adverse effects and other Clinical Outcomes of Ivermectin Treatment in COVID-19 Patients</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-WBeU4g4R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">857. </div><div class="csl-right-inline">M-K G. 2021. Does Ivermectin Work for Covid-19? <a href="https://gidmk.medium.com/does-ivermectin-work-for-covid-19-1166126c364a">https://gidmk.medium.com/does-ivermectin-work-for-covid-19-1166126c364a</a>.</div>
</div>
<div id="ref-3hlhg35F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">858. </div><div class="csl-right-inline">Hill A, Garratt A, Levi J, Falconer J, Ellis L, McCann K, Pilkington V, Qavi A, Wang J, Wentzel H. 2021. <a href="https://doi.org/10.1093/ofid/ofab358">Retracted: Meta-analysis of Randomized Trials of Ivermectin to Treat SARS-CoV-2 Infection</a>. Open Forum Infectious Diseases 8.</div>
</div>
<div id="ref-ZUL8cUnf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">859. </div><div class="csl-right-inline">Medicine C for V. 2022. <a href="https://www.fda.gov/animal-veterinary/product-safety-information/faq-covid-19-and-ivermectin-intended-animals">FAQ: COVID-19 and Ivermectin Intended for Animals</a>. FDA.</div>
</div>
<div id="ref-N2gVvs0r" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">860. </div><div class="csl-right-inline">Cardresearch. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04727424">A Multicenter, Prospective, Adaptive, Double-blind, Randomized, Placebo-controlled Study to Evaluate the Effect of Interferon Lambda 1A, Fluvoxamina + Budesonida, Fluoxetina + Budesonida in Mild COVID-19 and High Risk of Complications</a>. NCT04727424. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-3N6m5mBF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">861. </div><div class="csl-right-inline"><a href="https://www.principletrial.org/news/ivermectin-to-be-investigated-as-a-possible-treatment-for-covid-19-in-oxford2019s-principle-trial">Ivermectin to be investigated in adults aged 18+ as a possible treatment for COVID-19 in the PRINCIPLE trial — PRINCIPLE Trial</a>.</div>
</div>
<div id="ref-ultwDbEG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">862. </div><div class="csl-right-inline">Reis G, Moreira Silva EA dos S, Medeiros Silva DC, Thabane L, Singh G, Park JJH, Forrest JI, Harari O, Quirino dos Santos CV, Guimarães de Almeida APF, Figueiredo Neto AD de, Savassi LCM, Milagres AC, Teixeira MM, Simplicio MIC, Ribeiro LB, Oliveira R, Mills EJ. 2021. <a href="https://doi.org/10.1001/jamanetworkopen.2021.6468">Effect of Early Treatment With Hydroxychloroquine or Lopinavir and Ritonavir on Risk of Hospitalization Among Patients With COVID-19</a>. JAMA Netw Open 4:e216468.</div>
</div>
<div id="ref-v5LPdCYe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">863. </div><div class="csl-right-inline">terren.green@duke.edu. 2021. August 6, 2021: Early Treatment of COVID-19 with Repurposed Therapies: The TOGETHER Adaptive Platform Trial (Edward Mills, PhD, FRCP). Rethinking Clinical Trials. <a href="https://rethinkingclinicaltrials.org/news/august-6-2021-early-treatment-of-covid-19-with-repurposed-therapies-the-together-adaptive-platform-trial-edward-mills-phd-frcp/">https://rethinkingclinicaltrials.org/news/august-6-2021-early-treatment-of-covid-19-with-repurposed-therapies-the-together-adaptive-platform-trial-edward-mills-phd-frcp/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-CIeWPFHG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">864. </div><div class="csl-right-inline">Lisinopril - Drug Usage Statistics. ClinCalc DrugStats Database. <a href="https://clincalc.com/DrugStats/Drugs/Lisinopril">https://clincalc.com/DrugStats/Drugs/Lisinopril</a>. Retrieved 30 July 2020.</div>
</div>
<div id="ref-pbfxOvnK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">865. </div><div class="csl-right-inline">Byrd JB, Chertow GM, Bhalla V. 2019. <a href="https://doi.org/10.1056/nejmp1901657">Hypertension Hot Potato — Anatomy of the Angiotensin-Receptor Blocker Recalls</a>. New England Journal of Medicine 380:1589–1591.</div>
</div>
<div id="ref-1BVslkmfk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">866. </div><div class="csl-right-inline">Bian B, Kelton CML, Guo JJ, Wigle PR. 2010. <a href="https://doi.org/10.18553/jmcp.2010.16.9.671">ACE Inhibitor and ARB Utilization and Expenditures in the Medicaid Fee-For-Service Program from 1991 to 2008</a>. JMCP 16:671–679.</div>
</div>
<div id="ref-161Tg5PUW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">867. </div><div class="csl-right-inline">Turner AJ, Hiscox JA, Hooper NM. 2004. <a href="https://doi.org/10.1016/j.tips.2004.04.001">ACE2: from vasopeptidase to SARS virus receptor</a>. Trends in Pharmacological Sciences 25:291–294.</div>
</div>
<div id="ref-ZMzupz3m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">868. </div><div class="csl-right-inline">Huentelman MJ, Zubcevic J, Hernández Prada JA, Xiao X, Dimitrov DS, Raizada MK, Ostrov DA. 2004. <a href="https://doi.org/10.1161/01.hyp.0000146120.29648.36">Structure-Based Discovery of a Novel Angiotensin-Converting Enzyme 2 Inhibitor</a>. Hypertension 44:903–906.</div>
</div>
<div id="ref-9rA6s6Qp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">869. </div><div class="csl-right-inline">Dimitrov DS. 2003. <a href="https://doi.org/10.1016/s0092-8674(03)00976-0">The Secret Life of ACE2 as a Receptor for the SARS Virus</a>. Cell 115:652–653.</div>
</div>
<div id="ref-17xTGqKp2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">870. </div><div class="csl-right-inline">Kreutz R, Algharably EAE-H, Azizi M, Dobrowolski P, Guzik T, Januszewicz A, Persu A, Prejbisz A, Riemer TG, Wang J-G, Burnier M. 2020. <a href="https://doi.org/10.1093/cvr/cvaa097">Hypertension, the renin–angiotensin system, and the risk of lower respiratory tract infections and lung injury: implications for COVID-19</a>. Cardiovascular Research 116:1688–1699.</div>
</div>
<div id="ref-ZhhV4ztO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">871. </div><div class="csl-right-inline">Walters TE, Kalman JM, Patel SK, Mearns M, Velkoska E, Burrell LM. 2016. <a href="https://doi.org/10.1093/europace/euw246">Angiotensin converting enzyme 2 activity and human atrial fibrillation: increased plasma angiotensin converting enzyme 2 activity is associated with atrial fibrillation and more advanced left atrial structural remodelling</a>. Europace euw246.</div>
</div>
<div id="ref-10q7vcaWw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">872. </div><div class="csl-right-inline">Mehra MR, Desai SS, Kuy S, Henry TD, Patel AN. 2020. <a href="https://doi.org/10.1056/nejmoa2007621">Cardiovascular Disease, Drug Therapy, and Mortality in Covid-19</a>. New England Journal of Medicine 382:e102.</div>
</div>
<div id="ref-3GIXDhwz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">873. </div><div class="csl-right-inline">Mehra MR, Desai SS, Kuy S, Henry TD, Patel AN. 2020. <a href="https://doi.org/10.1056/nejmc2021225">Retraction: Cardiovascular Disease, Drug Therapy, and Mortality in Covid-19. N Engl J Med. DOI: 10.1056/NEJMoa2007621.</a> N Engl J Med 382:2582–2582.</div>
</div>
<div id="ref-IRuDYZRB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">874. </div><div class="csl-right-inline">Cohen JB, Hanff TC, William P, Sweitzer N, Rosado-Santander NR, Medina C, Rodriguez-Mori JE, Renna N, Chang TI, Corrales-Medina V, Andrade-Villanueva JF, Barbagelata A, Cristodulo-Cortez R, Díaz-Cucho OA, Spaak J, Alfonso CE, Valdivia-Vega R, Villavicencio-Carranza M, Ayala-García RJ, Castro-Callirgos CA, González-Hernández LA, Bernales-Salas EF, Coacalla-Guerra JC, Salinas-Herrera CD, Nicolosi L, Basconcel M, Byrd JB, Sharkoski T, Bendezú-Huasasquiche LE, Chittams J, Edmonston DL, Vasquez CR, Chirinos JA. 2021. <a href="https://doi.org/10.1016/s2213-2600(20)30558-0">Continuation versus discontinuation of renin–angiotensin system inhibitors in patients admitted to hospital with COVID-19: a prospective, randomised, open-label trial</a>. The Lancet Respiratory Medicine 9:275–284.</div>
</div>
<div id="ref-1DqZPaCtf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">875. </div><div class="csl-right-inline">Lopes RD, Macedo AVS, de Barros E Silva PGM, Moll-Bernardes RJ, dos Santos TM, Mazza L, Feldman A, D’Andréa Saba Arruda G, de Albuquerque DC, Camiletti AS, de Sousa AS, de Paula TC, Giusti KGD, Domiciano RAM, Noya-Rabelo MM, Hamilton AM, Loures VA, Dionísio RM, Furquim TAB, De Luca FA, dos Santos Sousa ÍB, Bandeira BS, Zukowski CN, de Oliveira RGG, Ribeiro NB, de Moraes JL, Petriz JLF, Pimentel AM, Miranda JS, de Jesus Abufaiad BE, Gibson CM, Granger CB, Alexander JH, de Souza OF. 2021. <a href="https://doi.org/10.1001/jama.2020.25864">Effect of Discontinuing vs Continuing Angiotensin-Converting Enzyme Inhibitors and Angiotensin II Receptor Blockers on Days Alive and Out of the Hospital in Patients Admitted With COVID-19</a>. JAMA 325:254.</div>
</div>
<div id="ref-l9fFL5hb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">876. </div><div class="csl-right-inline">2020. Frequently Asked Questions on the Revocation of the Emergency Use Authorization for Hydroxychloroquine Sulfate and Chloroquine Phosphate. US Food and Drug Administration. <a href="https://www.fda.gov/media/138946/download">https://www.fda.gov/media/138946/download</a>.</div>
</div>
<div id="ref-hTnpKZb6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">877. </div><div class="csl-right-inline">2020. COVID-19: chloroquine and hydroxychloroquine only to be used in clinical trials or emergency use programmes. <a href="https://www.ema.europa.eu/en/news/covid-19-chloroquine-hydroxychloroquine-only-be-used-clinical-trials-emergency-use-programmes">https://www.ema.europa.eu/en/news/covid-19-chloroquine-hydroxychloroquine-only-be-used-clinical-trials-emergency-use-programmes</a>.</div>
</div>
<div id="ref-1ALDN7rfe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">878. </div><div class="csl-right-inline">Shire SJ. 2009. <a href="https://doi.org/10.1016/j.copbio.2009.10.006">Formulation and manufacturability of biologics</a>. Current Opinion in Biotechnology 20:708–714.</div>
</div>
<div id="ref-OgYDNJt3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">879. </div><div class="csl-right-inline">Baumann A. 2006. <a href="https://doi.org/10.2174/138920006774832604">Early Development of Therapeutic Biologics - Pharmacokinetics</a>. Current Drug Metabolism 7:15–21.</div>
</div>
<div id="ref-1AKi0FYUB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">880. </div><div class="csl-right-inline">Lu R-M, Hwang Y-C, Liu I-J, Lee C-C, Tsai H-Z, Li H-J, Wu H-C. 2020. <a href="https://doi.org/10.1186/s12929-019-0592-z">Development of therapeutic antibodies for the treatment of diseases</a>. Journal of Biomedical Science 27:1.</div>
</div>
<div id="ref-yumBaJ6U" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">881. </div><div class="csl-right-inline">Corti D, Lanzavecchia A. 2013. <a href="https://doi.org/10.1146/annurev-immunol-032712-095916">Broadly Neutralizing Antiviral Antibodies</a>. Annual Review of Immunology 31:705–742.</div>
</div>
<div id="ref-dUnB3gD6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">882. </div><div class="csl-right-inline">Iacob SA, Iacob DG. 2017. <a href="https://doi.org/10.3389/fmicb.2017.02323">Ibalizumab Targeting CD4 Receptors, An Emerging Molecule in HIV Therapy</a>. Frontiers in Microbiology 8:2323.</div>
</div>
<div id="ref-cwbHiM1o" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">883. </div><div class="csl-right-inline">Resch B. 2017. <a href="https://doi.org/10.1080/21645515.2017.1337614">Product review on the monoclonal antibody palivizumab for prevention of respiratory syncytial virus infection</a>. Human Vaccines &amp; Immunotherapeutics 13:2138–2149.</div>
</div>
<div id="ref-f50jNbA8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">884. </div><div class="csl-right-inline">Houser KV, Gretebeck L, Ying T, Wang Y, Vogel L, Lamirande EW, Bock KW, Moore IN, Dimitrov DS, Subbarao K. 2016. <a href="https://doi.org/10.1093/infdis/jiw080">Prophylaxis With a Middle East Respiratory Syndrome Coronavirus (MERS-CoV)–Specific Human Monoclonal Antibody Protects Rabbits From MERS-CoV Infection</a>. J Infect Dis 213:1557–1561.</div>
</div>
<div id="ref-KVTUgJqr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">885. </div><div class="csl-right-inline">van Doremalen N, Falzarano D, Ying T, de Wit E, Bushmaker T, Feldmann F, Okumura A, Wang Y, Scott DP, Hanley PW, Feldmann H, Dimitrov DS, Munster VJ. 2017. <a href="https://doi.org/10.1016/j.antiviral.2017.03.025">Efficacy of antibody-based therapies against Middle East respiratory syndrome coronavirus (MERS-CoV) in common marmosets</a>. Antiviral Research 143:30–37.</div>
</div>
<div id="ref-NDRyNd61" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">886. </div><div class="csl-right-inline">Tanaka T, Narazaki M, Kishimoto T. 2014. <a href="https://doi.org/10.1101/cshperspect.a016295">IL-6 in Inflammation, Immunity, and Disease</a>. Cold Spring Harbor Perspectives in Biology 6:a016295–a016295.</div>
</div>
<div id="ref-1636MNwvb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">887. </div><div class="csl-right-inline">2020. ACTEMRA - tocilizumab. DailyMed. <a href="https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=2e5365ff-cb2a-4b16-b2c7-e35c6bf2de13">https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=2e5365ff-cb2a-4b16-b2c7-e35c6bf2de13</a>.</div>
</div>
<div id="ref-ayc7zYqi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">888. </div><div class="csl-right-inline">Guaraldi G, Meschiari M, Cozzi-Lepri A, Milic J, Tonelli R, Menozzi M, Franceschini E, Cuomo G, Orlando G, Borghi V, Santoro A, Di Gaetano M, Puzzolante C, Carli F, Bedini A, Corradi L, Fantini R, Castaniere I, Tabbì L, Girardis M, Tedeschi S, Giannella M, Bartoletti M, Pascale R, Dolci G, Brugioni L, Pietrangelo A, Cossarizza A, Pea F, Clini E, Salvarani C, Massari M, Viale PL, Mussini C. 2020. <a href="https://doi.org/10.1016/s2665-9913(20)30173-9">Tocilizumab in patients with severe COVID-19: a retrospective cohort study</a>. The Lancet Rheumatology 2:e474–e484.</div>
</div>
<div id="ref-Wt9vgKLY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">889. </div><div class="csl-right-inline">Price CC, Altice FL, Shyr Y, Koff A, Pischel L, Goshua G, Azar MM, Mcmanus D, Chen S-C, Gleeson SE, Britto CJ, Azmy V, Kaman K, Gaston DC, Davis M, Burrello T, Harris Z, Villanueva MS, Aoun-Barakat L, Kang I, Seropian S, Chupp G, Bucala R, Kaminski N, Lee AI, LoRusso PM, Topal JE, Dela Cruz C, Malinis M. 2020. <a href="https://doi.org/10.1016/j.chest.2020.06.006">Tocilizumab Treatment for Cytokine Release Syndrome in Hospitalized Patients With Coronavirus Disease 2019</a>. Chest 158:1397–1408.</div>
</div>
<div id="ref-luPHOS0m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">890. </div><div class="csl-right-inline">Capra R, De Rossi N, Mattioli F, Romanelli G, Scarpazza C, Sormani MP, Cossi S. 2020. <a href="https://doi.org/10.1016/j.ejim.2020.05.009">Impact of low dose tocilizumab on mortality rate in patients with COVID-19 related pneumonia</a>. European Journal of Internal Medicine 76:31–35.</div>
</div>
<div id="ref-17hgtBpcV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">891. </div><div class="csl-right-inline">Klopfenstein T, Zayet S, Lohse A, Balblanc J-C, Badie J, Royer P-Y, Toko L, Mezher C, Kadiane-Oussou NJ, Bossert M, Bozgan A-M, Charpentier A, Roux M-F, Contreras R, Mazurier I, Dussert P, Gendrin V, Conrozier T. 2020. <a href="https://doi.org/10.1016/j.medmal.2020.05.001">Tocilizumab therapy reduced intensive care unit admissions and/or mortality in COVID-19 patients</a>. Médecine et Maladies Infectieuses 50:397–400.</div>
</div>
<div id="ref-WEqfULN1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">892. </div><div class="csl-right-inline">Rojas-Marte G, Khalid M, Mukhtar O, Hashmi AT, Waheed MA, Ehrlich S, Aslam A, Siddiqui S, Agarwal C, Malyshev Y, Henriquez-Felipe C, Sharma D, Sharma S, Chukwuka N, Rodriguez DC, Alliu S, Le J, Shani J. 2020. <a href="https://doi.org/10.1093/qjmed/hcaa206">Outcomes in patients with severe COVID-19 disease treated with tocilizumab: a case–controlled study</a>. QJM: An International Journal of Medicine 113:546–550.</div>
</div>
<div id="ref-bj2feMy4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">893. </div><div class="csl-right-inline">Xu X, Han M, Li T, Sun W, Wang D, Fu B, Zhou Y, Zheng X, Yang Y, Li X, Zhang X, Pan A, Wei H. 2020. <a href="https://doi.org/10.1073/pnas.2005615117">Effective treatment of severe COVID-19 patients with tocilizumab</a>. Proceedings of the National Academy of Sciences 117:10970–10975.</div>
</div>
<div id="ref-9wnr4sOS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">894. </div><div class="csl-right-inline">, Horby PW, Pessoa-Amorim G, Peto L, Brightling CE, Sarkar R, Thomas K, Jeebun V, Ashish A, Tully R, Chadwick D, Sharafat M, Stewart R, Rudran B, Baillie JK, Buch MH, Chappell LC, Day JN, Furst SN, Jaki T, Jeffery K, Juszczak E, Lim WS, Montgomery A, Mumford A, Rowan K, Thwaites G, Mafham M, Haynes R, Landray MJ. 2021. <a href="https://doi.org/10.1101/2021.02.11.21249258">Tocilizumab in patients admitted to hospital with COVID-19 (RECOVERY): preliminary results of a randomised, controlled, open-label, platform trial</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-3qsYQAdF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">895. </div><div class="csl-right-inline">Rosas IO, Bräu N, Waters M, Go RC, Hunter BD, Bhagani S, Skiest D, Aziz MS, Cooper N, Douglas IS, Savic S, Youngstein T, Del Sorbo L, Cubillo Gracian A, De La Zerda DJ, Ustianowski A, Bao M, Dimonaco S, Graham E, Matharu B, Spotswood H, Tsai L, Malhotra A. 2021. <a href="https://doi.org/10.1056/nejmoa2028700">Tocilizumab in Hospitalized Patients with Severe Covid-19 Pneumonia</a>. N Engl J Med 384:1503–1516.</div>
</div>
<div id="ref-11XXNLZ90" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">896. </div><div class="csl-right-inline">Salama C, Han J, Yau L, Reiss WG, Kramer B, Neidhart JD, Criner GJ, Kaplan-Lewis E, Baden R, Pandit L, Cameron ML, Garcia-Diaz J, Chávez V, Mekebeb-Reuter M, Lima de Menezes F, Shah R, González-Lara MF, Assman B, Freedman J, Mohan SV. 2021. <a href="https://doi.org/10.1056/nejmoa2030340">Tocilizumab in Patients Hospitalized with Covid-19 Pneumonia</a>. N Engl J Med 384:20–30.</div>
</div>
<div id="ref-TBjYbLjT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">897. </div><div class="csl-right-inline">Hoffmann-La Roche. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04409262">A Phase III, Randomized, Double-Blind, Multicenter Study to Evaluate the Efficacy and Safety of Remdesivir Plus Tocilizumab Compared With Remdesivir Plus Placebo in Hospitalized Patients With Severe COVID-19 Pneumonia</a>. NCT04409262. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-SytvPCdR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">898. </div><div class="csl-right-inline">Hoffmann-La Roche. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04320615">A Randomized, Double-Blind, Placebo-Controlled, Multicenter Study to Evaluate the Safety and Efficacy of Tocilizumab in Patients With Severe COVID-19 Pneumonia</a>. NCT04320615. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-eUSZ6Jb5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">899. </div><div class="csl-right-inline">Genentech, Inc. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04372186">A Randomized, Double-Blind, Placebo-Controlled, Multicenter Study to Evaluate the Efficacy and Safety of Tocilizumab in Hospitalized Patients With COVID-19 Pneumonia</a>. NCT04372186. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-sW1V7yT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">900. </div><div class="csl-right-inline">2021. Genentech tocilizumab Letter of Authority. US Food and Drug Administration. <a href="https://www.fda.gov/media/150319/download">https://www.fda.gov/media/150319/download</a>.</div>
</div>
<div id="ref-5GxCGzrT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">901. </div><div class="csl-right-inline">Synairgen Research Ltd. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04385095">A Randomised Double-blind Placebo-controlled Trial to Determine the Safety and Efficacy of Inhaled SNG001 (IFN-β1a for Nebulisation) for the Treatment of Patients With Confirmed SARS-CoV-2 Infection</a>. NCT04385095. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-bD1bgfSb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">902. </div><div class="csl-right-inline">2020. Synairgen announces positive results from trial of SNG001 in hospitalised COVID-19 patients. Synairgen plc press release. <a href="http://synairgen.web01.hosting.bdci.co.uk/umbraco/Surface/Download/GetFile?cid=1130026e-0983-4338-b648-4ac7928b9a37">http://synairgen.web01.hosting.bdci.co.uk/umbraco/Surface/Download/GetFile?cid=1130026e-0983-4338-b648-4ac7928b9a37</a>.</div>
</div>
<div id="ref-qFDbLzcP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">903. </div><div class="csl-right-inline">Monk PD, Marsden RJ, Tear VJ, Brookes J, Batten TN, Mankowski M, Gabbay FJ, Davies DE, Holgate ST, Ho L-P, Clark T, Djukanovic R, Wilkinson TMA, Crooks MG, Dosanjh DP, Siddiqui S, Rahman NM, Smith JA, Horsley A, Harrison TW, Saralaya D, McGarvey L, Watson A, Foster E, Fleet A, Singh D, Hemmings S, Aitken S, Dudley S, Beegan R, Thompson A, Rodrigues PM. 2021. <a href="https://doi.org/10.1016/s2213-2600(20)30511-7">Safety and efficacy of inhaled nebulised interferon beta-1a (SNG001) for treatment of SARS-CoV-2 infection: a randomised, double-blind, placebo-controlled, phase 2 trial</a>. The Lancet Respiratory Medicine 9:196–206.</div>
</div>
<div id="ref-f52YwpY3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">904. </div><div class="csl-right-inline">Chen L, Xiong J, Bao L, Shi Y. 2020. <a href="https://doi.org/10.1016/s1473-3099(20)30141-9">Convalescent plasma as a potential therapy for COVID-19</a>. The Lancet Infectious Diseases 20:398–400.</div>
</div>
<div id="ref-JPUZjTnq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">905. </div><div class="csl-right-inline">Roback JD, Guarner J. 2020. <a href="https://doi.org/10.1001/jama.2020.4940">Convalescent Plasma to Treat COVID-19</a>. JAMA 323:1561.</div>
</div>
<div id="ref-kAKKkMAW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">906. </div><div class="csl-right-inline">Rajendran K, Krishnasamy N, Rangarajan J, Rathinam J, Natarajan M, Ramachandran A. 2020. <a href="https://doi.org/10.1002/jmv.25961">Convalescent plasma transfusion for the treatment of COVID‐19: Systematic review</a>. J Med Virol 92:1475–1483.</div>
</div>
<div id="ref-EuYDqR7q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">907. </div><div class="csl-right-inline">Janiaud P, Axfors C, Schmitt AM, Gloy V, Ebrahimi F, Hepprich M, Smith ER, Haber NA, Khanna N, Moher D, Goodman SN, Ioannidis JPA, Hemkens LG. 2021. <a href="https://doi.org/10.1001/jama.2021.2747">Association of Convalescent Plasma Treatment With Clinical Outcomes in Patients With COVID-19</a>. JAMA 325:1185.</div>
</div>
<div id="ref-11J6lIInT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">908. </div><div class="csl-right-inline">Joyner MJ, Carter RE, Senefeld JW, Klassen SA, Mills JR, Johnson PW, Theel ES, Wiggins CC, Bruno KA, Klompas AM, Lesser ER, Kunze KL, Sexton MA, Diaz Soto JC, Baker SE, Shepherd JRA, van Helmond N, Verdun NC, Marks P, van Buskirk CM, Winters JL, Stubbs JR, Rea RF, Hodge DO, Herasevich V, Whelan ER, Clayburn AJ, Larson KF, Ripoll JG, Andersen KJ, Buras MR, Vogt MNP, Dennis JJ, Regimbal RJ, Bauer PR, Blair JE, Paneth NS, Fairweather D, Wright RS, Casadevall A. 2021. <a href="https://doi.org/10.1056/nejmoa2031893">Convalescent Plasma Antibody Levels and the Risk of Death from Covid-19</a>. New England Journal of Medicine NEJMoa2031893.</div>
</div>
<div id="ref-124Fu0rXD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">909. </div><div class="csl-right-inline">, Horby PW, Estcourt L, Peto L, Emberson JR, Staplin N, Spata E, Pessoa-Amorim G, Campbell M, Roddick A, Brunskill NE, George T, Zehnder D, Tiberi S, Aung NN, Uriel A, Widdrington J, Koshy G, Brown T, Scott S, Baillie JK, Buch MH, Chappell LC, Day JN, Faust SN, Jaki T, Jeffery K, Juszczak E, Lim WS, Montgomery A, Mumford A, Rowan K, Thwaites G, Mafham M, Roberts D, Haynes R, Landray MJ. 2021. <a href="https://doi.org/10.1101/2021.03.09.21252736">Convalescent plasma in patients admitted to hospital with COVID-19 (RECOVERY): a randomised, controlled, open-label, platform trial</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-3gp7t98X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">910. </div><div class="csl-right-inline">Hsueh P-R, Huang L-M, Chen P-J, Kao C-L, Yang P-C. 2004. <a href="https://doi.org/10.1111/j.1469-0691.2004.01009.x">Chronological evolution of IgM, IgA, IgG and neutralisation antibodies after infection with SARS-associated coronavirus</a>. Clinical Microbiology and Infection 10:1062–1066.</div>
</div>
<div id="ref-KULxo48U" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">911. </div><div class="csl-right-inline">Yuchun N, Guangwen W, Xuanling S, Hong Z, Yan Q, Zhongping H, Wei W, Gewei L, Xiaolei Y, Liying D, Lili R, Jianwei W, Xiong H, Taisheng L, Hongkui D, Mingxiao D. 2004. <a href="https://doi.org/10.1086/423286">Neutralizing Antibodies in Patients with Severe Acute Respiratory Syndrome-Associated Coronavirus Infection</a>. The Journal of Infectious Diseases 190:1119–1126.</div>
</div>
<div id="ref-MTcTiS7i" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">912. </div><div class="csl-right-inline">Prabakaran P, Zhu Z, Xiao X, Biragyn A, Dimitrov AS, Broder CC, Dimitrov DS. 2009. <a href="https://doi.org/10.1517/14712590902763755">Potent human monoclonal antibodies against SARS CoV, Nipah and Hendra viruses</a>. Expert Opinion on Biological Therapy 9:355–368.</div>
</div>
<div id="ref-41WAtZmB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">913. </div><div class="csl-right-inline">Sun C, Chen L, Yang J, Luo C, Zhang Y, Li J, Yang J, Zhang J, Xie L. 2020. SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.16.951723">10.1101/2020.02.16.951723</a>.</div>
</div>
<div id="ref-19R52ruRw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">914. </div><div class="csl-right-inline">Renn A, Fu Y, Hu X, Hall MD, Simeonov A. 2020. <a href="https://doi.org/10.1016/j.tips.2020.07.004">Fruitful Neutralizing Antibody Pipeline Brings Hope To Defeat SARS-Cov-2</a>. Trends in Pharmacological Sciences 41:815–829.</div>
</div>
<div id="ref-YigWg7xu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">915. </div><div class="csl-right-inline">Pinto D, Park Y-J, Beltramello M, Walls AC, Tortorici MA, Bianchi S, Jaconi S, Culap K, Zatta F, De Marco A, Peter A, Guarino B, Spreafico R, Cameroni E, Case JB, Chen RE, Havenar-Daughton C, Snell G, Telenti A, Virgin HW, Lanzavecchia A, Diamond MS, Fink K, Veesler D, Corti D. 2020. <a href="https://doi.org/10.1038/s41586-020-2349-y">Cross-neutralization of SARS-CoV-2 by a human monoclonal SARS-CoV antibody</a>. Nature 583:290–295.</div>
</div>
<div id="ref-XcuzhxrJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">916. </div><div class="csl-right-inline">Wang C, Li W, Drabek D, Okba NMA, van Haperen R, Osterhaus ADME, van Kuppeveld FJM, Haagmans BL, Grosveld F, Bosch B-J. 2020. A human monoclonal antibody blocking SARS-CoV-2 infection. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.11.987958">10.1101/2020.03.11.987958</a>.</div>
</div>
<div id="ref-hkzKw8NQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">917. </div><div class="csl-right-inline">Deb P, Molla MdMA, Saif-Ur-Rahman KM. 2021. <a href="https://doi.org/10.1016/j.bsheal.2021.02.001">An update to monoclonal antibody as therapeutic option against COVID-19</a>. Biosafety and Health 3:87–91.</div>
</div>
<div id="ref-f44jpdvR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">918. </div><div class="csl-right-inline">Jones BE, Brown-Augsburger PL, Corbett KS, Westendorf K, Davies J, Cujec TP, Wiethoff CM, Blackbourne JL, Heinz BA, Foster D, Higgs RE, Balasubramaniam D, Wang L, Bidshahri R, Kraft L, Hwang Y, Žentelis S, Jepson KR, Goya R, Smith MA, Collins DW, Hinshaw SJ, Tycho SA, Pellacani D, Xiang P, Muthuraman K, Sobhanifar S, Piper MH, Triana FJ, Hendle J, Pustilnik A, Adams AC, Berens SJ, Baric RS, Martinez DR, Cross RW, Geisbert TW, Borisevich V, Abiona O, Belli HM, de Vries M, Mohamed A, Dittmann M, Samanovic M, Mulligan MJ, Goldsmith JA, Hsieh C-L, Johnson NV, Wrapp D, McLellan JS, Barnhart BC, Graham BS, Mascola JR, Hansen CL, Falconer E. 2020. <a href="https://doi.org/10.1101/2020.09.30.318972">LY-CoV555, a rapidly isolated potent neutralizing antibody, provides protection in a non-human primate model of SARS-CoV-2 infection</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-4IdGvvyC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">919. </div><div class="csl-right-inline">Eli Lilly and Company. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04411628">A Randomized, Placebo-Controlled, Double-Blind, Sponsor Unblinded, Single Ascending Dose, Phase 1 First in Human Study to Evaluate the Safety, Tolerability, Pharmacokinetics and Pharmacodynamics of Intravenous LY3819253 in Participants Hospitalized for COVID-19</a>. NCT04411628. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-coZGelVr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">920. </div><div class="csl-right-inline">Eli Lilly and Company. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04441931">A Phase 1, Randomized, Placebo-Controlled Study to Evaluate the Tolerability, Safety, Pharmacokinetics, and Immunogenicity of LY3832479 Given as a Single Intravenous Dose in Healthy Participants</a>. NCT04441931. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-Gwct8aFG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">921. </div><div class="csl-right-inline">Gottlieb RL, Nirula A, Chen P, Boscia J, Heller B, Morris J, Huhn G, Cardona J, Mocherla B, Stosor V, Shawa I, Kumar P, Adams AC, Van Naarden J, Custer KL, Durante M, Oakley G, Schade AE, Holzer TR, Ebert PJ, Higgs RE, Kallewaard NL, Sabo J, Patel DR, Klekotka P, Shen L, Skovronsky DM. 2021. <a href="https://doi.org/10.1001/jama.2021.0202">Effect of Bamlanivimab as Monotherapy or in Combination With Etesevimab on Viral Load in Patients With Mild to Moderate COVID-19</a>. JAMA 325:632.</div>
</div>
<div id="ref-f6m2SeLi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">922. </div><div class="csl-right-inline">Eli Lilly and Company. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04427501">A Randomized, Double-blind, Placebo-Controlled, Phase 2/3 Study to Evaluate the Efficacy and Safety of LY3819253 and LY3832479 in Participants With Mild to Moderate COVID-19 Illness</a>. NCT04427501. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-Cblh7jlS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">923. </div><div class="csl-right-inline">Chen P, Nirula A, Heller B, Gottlieb RL, Boscia J, Morris J, Huhn G, Cardona J, Mocherla B, Stosor V, Shawa I, Adams AC, Van Naarden J, Custer KL, Shen L, Durante M, Oakley G, Schade AE, Sabo J, Patel DR, Klekotka P, Skovronsky DM. 2021. <a href="https://doi.org/10.1056/nejmoa2029849">SARS-CoV-2 Neutralizing Antibody LY-CoV555 in Outpatients with Covid-19</a>. N Engl J Med 384:229–237.</div>
</div>
<div id="ref-eNQP4PU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">924. </div><div class="csl-right-inline">2021. Bamlanivimab and Etesevimab EUA Letter of Authorization. US Food and Drug Administration. <a href="https://www.fda.gov/media/145801/download">https://www.fda.gov/media/145801/download</a>.</div>
</div>
<div id="ref-fom0jw7s" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">925. </div><div class="csl-right-inline">Hansen J, Baum A, Pascal KE, Russo V, Giordano S, Wloga E, Fulton BO, Yan Y, Koon K, Patel K, Chung KM, Hermann A, Ullman E, Cruz J, Rafique A, Huang T, Fairhurst J, Libertiny C, Malbec M, Lee W, Welsh R, Farr G, Pennington S, Deshpande D, Cheng J, Watty A, Bouffard P, Babb R, Levenkova N, Chen C, Zhang B, Romero Hernandez A, Saotome K, Zhou Y, Franklin M, Sivapalasingam S, Lye DC, Weston S, Logue J, Haupt R, Frieman M, Chen G, Olson W, Murphy AJ, Stahl N, Yancopoulos GD, Kyratsous CA. 2020. <a href="https://doi.org/10.1126/science.abd0827">Studies in humanized mice and convalescent humans yield a SARS-CoV-2 antibody cocktail</a>. Science 369:1010–1014.</div>
</div>
<div id="ref-Tn0PkRlx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">926. </div><div class="csl-right-inline">Regeneron Pharmaceuticals. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04426695">A Master Protocol Assessing the Safety, Tolerability, and Efficacy of Anti-Spike (S) SARS-CoV-2 Monoclonal Antibodies for the Treatment of Hospitalized Patients With COVID-19</a>. NCT04426695. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-1stdSBxx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">927. </div><div class="csl-right-inline">Regeneron Pharmaceuticals. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04425629">A Master Protocol Assessing the Safety, Tolerability, and Efficacy of Anti-Spike (S) SARS-CoV-2 Monoclonal Antibodies for the Treatment of Ambulatory Patients With COVID-19</a>. NCT04425629. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-Mc2i06U" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">928. </div><div class="csl-right-inline">Weinreich DM, Sivapalasingam S, Norton T, Ali S, Gao H, Bhore R, Musser BJ, Soo Y, Rofail D, Im J, Perry C, Pan C, Hosain R, Mahmood A, Davis JD, Turner KC, Hooper AT, Hamilton JD, Baum A, Kyratsous CA, Kim Y, Cook A, Kampman W, Kohli A, Sachdeva Y, Graber X, Kowal B, DiCioccio T, Stahl N, Lipsich L, Braunstein N, Herman G, Yancopoulos GD. 2021. <a href="https://doi.org/10.1056/nejmoa2035002">REGN-COV2, a Neutralizing Antibody Cocktail, in Outpatients with Covid-19</a>. N Engl J Med 384:238–251.</div>
</div>
<div id="ref-HGaNhhM7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">929. </div><div class="csl-right-inline">Commissioner O of the. 2020. Coronavirus (COVID-19) Update: FDA Authorizes Monoclonal Antibodies for Treatment of COVID-19. FDA. <a href="https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-monoclonal-antibodies-treatment-covid-19">https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-monoclonal-antibodies-treatment-covid-19</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-BlMyODD1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">930. </div><div class="csl-right-inline">Traggiai E, Becker S, Subbarao K, Kolesnikova L, Uematsu Y, Gismondo MR, Murphy BR, Rappuoli R, Lanzavecchia A. 2004. <a href="https://doi.org/10.1038/nm1080">An efficient method to make human monoclonal antibodies from memory B cells: potent neutralization of SARS coronavirus</a>. Nat Med 10:871–875.</div>
</div>
<div id="ref-Cuypqy2I" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">931. </div><div class="csl-right-inline">Dolgin E. 2021. <a href="https://doi.org/10.1038/s41587-021-00980-x">‘Super-antibodies’ could curb COVID-19 and help avert future pandemics</a>. Nat Biotechnol 39:783–785.</div>
</div>
<div id="ref-YPcraeO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">932. </div><div class="csl-right-inline">Walker LM, Burton DR. 2018. <a href="https://doi.org/10.1038/nri.2017.148">Passive immunotherapy of viral infections: 'super-antibodies' enter the fray</a>. Nat Rev Immunol 18:297–308.</div>
</div>
<div id="ref-1DiKhFgDx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">933. </div><div class="csl-right-inline">Vir Biotechnology, Inc. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04545060">A Phase II/III Randomized, Multi-center, Double-blind, Placebo-controlled Study to Assess the Safety and Efficacy of Monoclonal Antibody VIR-7831 for the Early Treatment of Coronavirus Disease 2019 (COVID-19) in Non-hospitalized Patients</a>. NCT04545060. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-6EgCVUr1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">934. </div><div class="csl-right-inline">Gupta A, Gonzalez-Rojas Y, Juarez E, Casal MC, Moya J, Falci DR, Sarkis E, Solis J, Zheng H, Scott N, Cathcart AL, Hebner CM, Sager J, Mogalian E, Tipple C, Peppercorn A, Alexander E, Pang PS, Free A, Brinson C, Aldinger M, Shapiro AE. 2021. <a href="https://doi.org/10.1101/2021.05.27.21257096">Early Covid-19 Treatment With SARS-CoV-2 Neutralizing Antibody Sotrovimab</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-1FtcInMeU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">935. </div><div class="csl-right-inline">Liu Z, VanBlargan LA, Bloyet L-M, Rothlauf PW, Chen RE, Stumpf S, Zhao H, Errico JM, Theel ES, Liebeskind MJ, Alford B, Buchser WJ, Ellebedy AH, Fremont DH, Diamond MS, Whelan SPJ. 2021. <a href="https://doi.org/10.1016/j.chom.2021.01.014">Identification of SARS-CoV-2 spike mutations that attenuate monoclonal and serum antibody neutralization</a>. Cell Host &amp;amp; Microbe 29:477–488.e4.</div>
</div>
<div id="ref-g9k7N6ig" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">936. </div><div class="csl-right-inline">Diamond M, Chen R, Xie X, Case J, Zhang X, VanBlargan L, Liu Y, Liu J, Errico J, Winkler E, Suryadevara N, Tahan S, Turner J, Kim W, Schmitz A, Thapa M, Wang D, Boon A, Pinto D, Presti R, O’Halloran J, Kim A, Deepak P, Fremont D, Corti D, Virgin H, Crowe J, Droit L, Ellebedy A, Shi P-Y, Gilchuk P. 2021. <a href="https://doi.org/10.21203/rs.3.rs-228079/v1">SARS-CoV-2 variants show resistance to neutralization by many monoclonal and serum-derived polyclonal antibodies</a>. Research Square Platform LLC.</div>
</div>
<div id="ref-ZDM0wbnx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">937. </div><div class="csl-right-inline">Graham C, Seow J, Huettner I, Khan H, Kouphou N, Acors S, Winstone H, Pickering S, Pedro Galao R, Jose Lista M, Jimenez-Guardeno JM, Laing AG, Wu Y, Joseph M, Muir L, Ng WM, Duyvesteyn HME, Zhao Y, Bowden TA, Shankar-Hari M, Rosa A, Cherepanov P, McCoy LE, Hayday AC, Neil SJD, Malim MH, Doores KJ. 2021. <a href="https://doi.org/10.1101/2021.02.03.429355">Impact of the B.1.1.7 variant on neutralizing monoclonal antibodies recognizing diverse epitopes on SARS–CoV–2 Spike</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-K18v3qZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">938. </div><div class="csl-right-inline">Wang P, Nair MS, Liu L, Iketani S, Luo Y, Guo Y, Wang M, Yu J, Zhang B, Kwong PD, Graham BS, Mascola JR, Chang JY, Yin MT, Sobieszczyk M, Kyratsous CA, Shapiro L, Sheng Z, Huang Y, Ho DD. 2021. <a href="https://doi.org/10.1101/2021.01.25.428137">Antibody Resistance of SARS-CoV-2 Variants B.1.351 and B.1.1.7</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-16pnbuotc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">939. </div><div class="csl-right-inline">2021. Pause in the Distribution of bamlanivimab/etesevimab. US Department of Health and Human Services. <a href="https://www.phe.gov/emergency/events/COVID19/investigation-MCM/Bamlanivimab-etesevimab/Pages/bamlanivimab-etesevimab-distribution-pause.aspx">https://www.phe.gov/emergency/events/COVID19/investigation-MCM/Bamlanivimab-etesevimab/Pages/bamlanivimab-etesevimab-distribution-pause.aspx</a>.</div>
</div>
<div id="ref-tnF7yB6H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">940. </div><div class="csl-right-inline">Sun Z-YJ, Oh KJ, Kim M, Yu J, Brusic V, Song L, Qiao Z, Wang J, Wagner G, Reinherz EL. 2008. <a href="https://doi.org/10.1016/j.immuni.2007.11.018">HIV-1 Broadly Neutralizing Antibody Extracts Its Epitope from a Kinked gp41 Ectodomain Region on the Viral Membrane</a>. Immunity 28:52–63.</div>
</div>
<div id="ref-1Dj5HnmXS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">941. </div><div class="csl-right-inline">Ekiert DC, Bhabha G, Elsliger M-A, Friesen RHE, Jongeneelen M, Throsby M, Goudsmit J, Wilson IA. 2009. <a href="https://doi.org/10.1126/science.1171491">Antibody Recognition of a Highly Conserved Influenza Virus Epitope</a>. Science 324:246–251.</div>
</div>
<div id="ref-NtlqU5Jy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">942. </div><div class="csl-right-inline">He Y, Li J, Du L, Yan X, Hu G, Zhou Y, Jiang S. 2006. <a href="https://doi.org/10.1016/j.vaccine.2006.04.054">Identification and characterization of novel neutralizing epitopes in the receptor-binding domain of SARS-CoV spike protein: Revealing the critical antigenic determinants in inactivated SARS-CoV vaccine</a>. Vaccine 24:5498–5508.</div>
</div>
<div id="ref-5sCeztks" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">943. </div><div class="csl-right-inline">Rockx B, Donaldson E, Frieman M, Sheahan T, Corti D, Lanzavecchia A, Baric Ralph S. 2010. <a href="https://doi.org/10.1086/651022">Escape from Human Monoclonal Antibody Neutralization Affects In Vitro and In Vivo Fitness of Severe Acute Respiratory Syndrome Coronavirus</a>. J INFECT DIS 201:946–955.</div>
</div>
<div id="ref-n1L18DDN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">944. </div><div class="csl-right-inline">Stanford Coronavirus Antiviral &amp; Resistance Database (CoVDB). <a href="https://covdb.stanford.edu/page/susceptibility-data">https://covdb.stanford.edu/page/susceptibility-data</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-B1wzHRpt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">945. </div><div class="csl-right-inline">Rappazzo CG, Tse LV, Kaku CI, Wrapp D, Sakharkar M, Huang D, Deveau LM, Yockachonis TJ, Herbert AS, Battles MB, O’Brien CM, Brown ME, Geoghegan JC, Belk J, Peng L, Yang L, Hou Y, Scobey TD, Burton DR, Nemazee D, Dye JM, Voss JE, Gunn BM, McLellan JS, Baric RS, Gralinski LE, Walker LM. 2021. <a href="https://doi.org/10.1126/science.abf4830">Broad and potent activity against SARS-like viruses by an engineered human monoclonal antibody</a>. Science 371:823–829.</div>
</div>
<div id="ref-12nqESooA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">946. </div><div class="csl-right-inline">Parvathaneni V, Kulkarni NS, Muth A, Gupta V. 2019. <a href="https://doi.org/10.1016/j.drudis.2019.06.014">Drug repurposing: a promising tool to accelerate the drug discovery process</a>. Drug Discovery Today 24:2076–2085.</div>
</div>
<div id="ref-DOd7lUdu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">947. </div><div class="csl-right-inline">Zheng W, Sun W, Simeonov A. 2017. <a href="https://doi.org/10.1111/bph.13895">Drug repurposing screens and synergistic drug-combinations for infectious diseases</a>. British Journal of Pharmacology 175:181–191.</div>
</div>
<div id="ref-SotNtXcx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">948. </div><div class="csl-right-inline">Muratov EN, Amaro R, Andrade CH, Brown N, Ekins S, Fourches D, Isayev O, Kozakov D, Medina-Franco JL, Merz KM, Oprea TI, Poroikov V, Schneider G, Todd MH, Varnek A, Winkler DA, Zakharov AV, Cherkasov A, Tropsha A. 2021. <a href="https://doi.org/10.1039/d0cs01065k">A critical overview of computational approaches employed for COVID-19 drug discovery</a>. Chem Soc Rev 50:9121–9151.</div>
</div>
<div id="ref-Jc3MrFiI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">949. </div><div class="csl-right-inline">Law GL, Tisoncik-Go J, Korth MJ, Katze MG. 2013. <a href="https://doi.org/10.1016/j.coi.2013.08.004">Drug repurposing: a better approach for infectious disease drug discovery?</a> Current Opinion in Immunology 25:588–592.</div>
</div>
<div id="ref-94R5R9oq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">950. </div><div class="csl-right-inline">Pereira DA, Williams JA. 2007. <a href="https://doi.org/10.1038/sj.bjp.0707373">Origin and evolution of high throughput screening</a>. British Journal of Pharmacology 152:53–61.</div>
</div>
<div id="ref-140r2Y6Ti" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">951. </div><div class="csl-right-inline">Vincent F, Loria P, Pregel M, Stanton R, Kitching L, Nocka K, Doyonnas R, Steppan C, Gilbert A, Schroeter T, Peakman M-C. 2015. <a href="https://doi.org/10.1126/scitranslmed.aab1201">Developing predictive assays: The phenotypic screening “rule of 3”</a>. Sci Transl Med 7.</div>
</div>
<div id="ref-WhCrTFgg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">952. </div><div class="csl-right-inline">Swinney DC. 2013. <a href="https://doi.org/10.1038/clpt.2012.236">Phenotypic vs. Target-Based Drug Discovery for First-in-Class Medicines</a>. Clin Pharmacol Ther 93:299–301.</div>
</div>
<div id="ref-qSXkcV36" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">953. </div><div class="csl-right-inline">Moffat JG, Rudolph J, Bailey D. 2014. <a href="https://doi.org/10.1038/nrd4366">Phenotypic screening in cancer drug discovery — past, present and future</a>. Nat Rev Drug Discov 13:588–602.</div>
</div>
<div id="ref-eT5KkS1V" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">954. </div><div class="csl-right-inline">Diversity Libraries. ChemBridge. <a href="https://chembridge.com/diversity-and-pre-plated-libraries/diversity-libraries/">https://chembridge.com/diversity-and-pre-plated-libraries/diversity-libraries/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-LSRYjaxD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">955. </div><div class="csl-right-inline">Wagner Bridget K, Schreiber Stuart L. 2016. <a href="https://doi.org/10.1016/j.chembiol.2015.11.008">The Power of Sophisticated Phenotypic Screening and Modern Mechanism-of-Action Methods</a>. Cell Chemical Biology 23:3–9.</div>
</div>
<div id="ref-1BgpXRivf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">956. </div><div class="csl-right-inline">Mercorelli B, Palù G, Loregian A. 2018. <a href="https://doi.org/10.1016/j.tim.2018.04.004">Drug Repurposing for Viral Infectious Diseases: How Far Are We?</a> Trends in Microbiology 26:865–876.</div>
</div>
<div id="ref-V2NNL2k2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">957. </div><div class="csl-right-inline">Challa AP, Lavieri RR, Lewis JT, Zaleski NM, Shirey-Rice JK, Harris PA, Aronoff DM, Pulley JM. 2019. <a href="https://doi.org/10.1089/adt.2019.950">Systematically Prioritizing Candidates in Genome-Based Drug Repurposing</a>. ASSAY and Drug Development Technologies 17:352–363.</div>
</div>
<div id="ref-Ubugg9KB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">958. </div><div class="csl-right-inline">Edwards A. 2020. <a href="https://doi.org/10.1021/acs.jcim.0c00861">What Are the Odds of Finding a COVID-19 Drug from a Lab Repurposing Screen?</a> J Chem Inf Model 60:5727–5729.</div>
</div>
<div id="ref-26dpptLw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">959. </div><div class="csl-right-inline">Kido H, Okumura Y, Yamada H, Quang Le T, Yano M. 2007. <a href="https://doi.org/10.2174/138161207780162971">Proteases Essential for Human Influenza Virus Entry into Cells and Their Inhibitors as Potential Therapeutic Agents</a>. Current Pharmaceutical Design 13:405–414.</div>
</div>
<div id="ref-f7csoyEu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">960. </div><div class="csl-right-inline">Zhou Y, Vedantham P, Lu K, Agudelo J, Carrion R, Nunneley JW, Barnard D, Pöhlmann S, McKerrow JH, Renslo AR, Simmons G. 2015. <a href="https://doi.org/10.1016/j.antiviral.2015.01.011">Protease inhibitors targeting coronavirus and filovirus entry</a>. Antiviral Research 116:76–84.</div>
</div>
<div id="ref-4Sja6dIz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">961. </div><div class="csl-right-inline">Jin Z, Du X, Xu Y, Deng Y, Liu M, Zhao Y, Zhang B, Li X, Zhang L, Peng C, Duan Y, Yu J, Wang L, Yang K, Liu F, Jiang R, Yang X, You T, Liu X, Yang X, Bai F, Liu H, Liu X, Guddat LW, Xu W, Xiao G, Qin C, Shi Z, Jiang H, Rao Z, Yang H. 2020. <a href="https://doi.org/10.1038/s41586-020-2223-y">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</a>. Nature 582:289–293.</div>
</div>
<div id="ref-15JYG0AIU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">962. </div><div class="csl-right-inline">Yang H, Xie W, Xue X, Yang K, Ma J, Liang W, Zhao Q, Zhou Z, Pei D, Ziebuhr J, Hilgenfeld R, Yuen KY, Wong L, Gao G, Chen S, Chen Z, Ma D, Bartlam M, Rao Z. 2005. <a href="https://doi.org/10.1371/journal.pbio.0030324">Design of Wide-Spectrum Inhibitors Targeting Coronavirus Main Proteases</a>. PLoS Biology 3:e324.</div>
</div>
<div id="ref-XfOTPQPJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">963. </div><div class="csl-right-inline">Ren Z, Yan L, Zhang N, Guo Y, Yang C, Lou Z, Rao Z. 2013. <a href="https://doi.org/10.1007/s13238-013-2841-3">The newly emerged SARS-Like coronavirus HCoV-EMC also has an “Achilles’ heel”: current effective inhibitor targeting a 3C-like protease</a>. Protein &amp; Cell 4:248–250.</div>
</div>
<div id="ref-16Gt8FHKW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">964. </div><div class="csl-right-inline">Wang F, Chen C, Tan W, Yang K, Yang H. 2016. <a href="https://doi.org/10.1038/srep22677">Structure of Main Protease from Human Coronavirus NL63: Insights for Wide Spectrum Anti-Coronavirus Drug Design</a>. Sci Rep 6.</div>
</div>
<div id="ref-T5r2uPRf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">965. </div><div class="csl-right-inline">Xue X, Yu H, Yang H, Xue F, Wu Z, Shen W, Li J, Zhou Z, Ding Y, Zhao Q, Zhang XC, Liao M, Bartlam M, Rao Z. 2008. <a href="https://doi.org/10.1128/jvi.02114-07">Structures of Two Coronavirus Main Proteases: Implications for Substrate Binding and Antiviral Drug Design</a>. J Virol 82:2515–2527.</div>
</div>
<div id="ref-17EErh01r" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">966. </div><div class="csl-right-inline">Azad GK, Tomar RS. 2014. <a href="https://doi.org/10.1007/s11033-014-3417-x">Ebselen, a promising antioxidant drug: mechanisms of action and targets of biological pathways</a>. Molecular Biology Reports 41:4865–4879.</div>
</div>
<div id="ref-3qlLwpK7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">967. </div><div class="csl-right-inline">Menéndez CA, Byléhn F, Perez-Lemus GR, Alvarado W, de Pablo JJ. 2020. <a href="https://doi.org/10.1126/sciadv.abd0345">Molecular characterization of ebselen binding activity to SARS-CoV-2 main protease</a>. Sci Adv 6.</div>
</div>
<div id="ref-vXFGEybh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">968. </div><div class="csl-right-inline">Chen Z, Jiang Z, Chen N, Shi Q, Tong L, Kong F, Cheng X, Chen H, Wang C, Tang B. 2018. <a href="https://doi.org/10.1039/c8cc04258f">Target discovery of ebselen with a biotinylated probe</a>. Chemical Communications 54:9506–9509.</div>
</div>
<div id="ref-G5cKNGMC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">969. </div><div class="csl-right-inline">FDA Clears SPI’s Ebselen for Phase II COVID-19 Trials. Contract Pharma. <a href="https://www.contractpharma.com/contents/view_breaking-news/2020-08-31/fda-clears-spis-ebselen-for-phase-ii-covid-19-trials/">https://www.contractpharma.com/contents/view_breaking-news/2020-08-31/fda-clears-spis-ebselen-for-phase-ii-covid-19-trials/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-cUTgHTRj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">970. </div><div class="csl-right-inline">Sound Pharmaceuticals, Incorporated. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04484025">A Phase 2, Randomized, Double-Blind, Placebo-Controlled, Dose Escalation Study to Evaluate the Safety and Efficacy of SPI-1005 in Moderate COVID-19 Patients</a>. NCT04484025. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-K13XqU1J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">971. </div><div class="csl-right-inline">Sound Pharmaceuticals, Incorporated. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04483973">A Phase 2, Randomized, Double-Blind, Placebo-Controlled, Dose Escalation Study to Evaluate the Safety and Efficacy of SPI-1005 in Severe COVID-19 Patients</a>. NCT04483973. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-27QvV2tx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">972. </div><div class="csl-right-inline">Pfizer. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04535167">A PHASE 1B, 2-PART, DOUBLE-BLIND, PLACEBO-CONTROLLED, SPONSOR-OPEN STUDY, TO EVALUATE THE SAFETY, TOLERABILITY AND PHARMACOKINETICS OF SINGLE ASCENDING (24-HOUR, PART 1) AND MULTIPLE ASCENDING (120-HOUR, PART 2) INTRAVENOUS INFUSIONS OF PF-07304814 IN HOSPITALIZED PARTICIPANTS WITH COVID-19</a>. NCT04535167. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-1EzX93BEG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">973. </div><div class="csl-right-inline">Pfizer. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04960202">AN INTERVENTIONAL EFFICACY AND SAFETY, PHASE 2/3, DOUBLE-BLIND, 2-ARM STUDY TO INVESTIGATE ORALLY ADMINISTERED PF-07321332/RITONAVIR COMPARED WITH PLACEBO IN NONHOSPITALIZED SYMPTOMATIC ADULT PARTICIPANTS WITH COVID-19 WHO ARE AT INCREASED RISK OF PROGRESSING TO SEVERE ILLNESS</a>. NCT04960202. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-1CFhvljOs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">974. </div><div class="csl-right-inline">Qian T, Zhu S, Hoshida Y. 2019. <a href="https://doi.org/10.1080/23808993.2019.1617632">Use of big data in drug development for precision medicine: an update</a>. Expert Review of Precision Medicine and Drug Development 4:189–200.</div>
</div>
<div id="ref-fYBqSPGl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">975. </div><div class="csl-right-inline">Kuleshov MV, Stein DJ, Clarke DJB, Kropiwnicki E, Jagodnik KM, Bartal A, Evangelista JE, Hom J, Cheng M, Bailey A, Zhou A, Ferguson LB, Lachmann A, Ma'ayan A. 2020. <a href="https://doi.org/10.1016/j.patter.2020.100090">The COVID-19 Drug and Gene Set Library</a>. Patterns 1:100090.</div>
</div>
<div id="ref-YRqgrrXw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">976. </div><div class="csl-right-inline">Sadegh S, Matschinske J, Blumenthal DB, Galindez G, Kacprowski T, List M, Nasirigerdeh R, Oubounyt M, Pichlmair A, Rose TD, Salgado-Albarrán M, Späth J, Stukalov A, Wenke NK, Yuan K, Pauling JK, Baumbach J. 2020. <a href="https://doi.org/10.1038/s41467-020-17189-2">Exploring the SARS-CoV-2 virus-host-drug interactome for drug repurposing</a>. Nat Commun 11.</div>
</div>
<div id="ref-1HnR6AuMx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">977. </div><div class="csl-right-inline">Zhou Y, Wang F, Tang J, Nussinov R, Cheng F. 2020. <a href="https://doi.org/10.1016/s2589-7500(20)30192-8">Artificial intelligence in COVID-19 drug repurposing</a>. The Lancet Digital Health 2:e667–e676.</div>
</div>
<div id="ref-KQ3m1fh0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">978. </div><div class="csl-right-inline">Maurice T, Su T-P. 2009. <a href="https://doi.org/10.1016/j.pharmthera.2009.07.001">The pharmacology of sigma-1 receptors</a>. Pharmacology &amp;amp; Therapeutics 124:195–206.</div>
</div>
<div id="ref-GZ4E3pwn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">979. </div><div class="csl-right-inline">Gordon DE, Hiatt J, Bouhaddou M, Rezelj VV, Ulferts S, Braberg H, Jureka AS, Obernier K, Guo JZ, Batra J, others. 2020. <a href="https://doi.org/10.1126/science.abe9403">Comparative host-coronavirus protein interaction networks reveal pan-viral disease mechanisms</a>. Science 370:eabe9403.</div>
</div>
<div id="ref-sLjk8RiX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">980. </div><div class="csl-right-inline">Vela JM. 2020. <a href="https://doi.org/10.3389/fphar.2020.582310">Repurposing Sigma-1 Receptor Ligands for COVID-19 Therapy?</a> Front Pharmacol 11.</div>
</div>
<div id="ref-8KoefI79" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">981. </div><div class="csl-right-inline">Hayashi T, Su T-. 2005. <a href="https://doi.org/10.2174/157015905774322516">The Sigma Receptor: Evolution of the Concept in Neuropsychopharmacology</a>. CN 3:267–280.</div>
</div>
<div id="ref-j2enN2JO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">982. </div><div class="csl-right-inline">Tummino TA, Rezelj VV, Fischer B, Fischer A, O’Meara MJ, Monel B, Vallet T, White KM, Zhang Z, Alon A, Schadt H, O’Donnell HR, Lyu J, Rosales R, McGovern BL, Rathnasinghe R, Jangra S, Schotsaert M, Galarneau J-R, Krogan NJ, Urban L, Shokat KM, Kruse AC, García-Sastre A, Schwartz O, Moretti F, Vignuzzi M, Pognan F, Shoichet BK. 2021. <a href="https://doi.org/10.1126/science.abi4708">Drug-induced phospholipidosis confounds drug repurposing for SARS-CoV-2</a>. Science 373:541–547.</div>
</div>
<div id="ref-M4yLbwsC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">983. </div><div class="csl-right-inline">Breiden B, Sandhoff K. 2019. <a href="https://doi.org/10.1515/hsz-2019-0270">Emerging mechanisms of drug-induced phospholipidosis</a>. Biological Chemistry 401:31–46.</div>
</div>
<div id="ref-1BV8xlc5v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">984. </div><div class="csl-right-inline">Obeidat M, Isaacson AL, Chen SJ, Ivanovic M, Holanda D. 2020. <a href="https://doi.org/10.1080/01913123.2020.1850966">Zebra-like bodies in COVID-19: is phospholipidosis evidence of hydroxychloroquine induced acute kidney injury?</a> Ultrastructural Pathology 44:519–523.</div>
</div>
<div id="ref-15kk2xRrj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">985. </div><div class="csl-right-inline">Dittmar M, Lee JS, Whig K, Segrist E, Li M, Kamalia B, Castellana L, Ayyanathan K, Cardenas-Diaz FL, Morrisey EE, Truitt R, Yang W, Jurado K, Samby K, Ramage H, Schultz DC, Cherry S. 2021. <a href="https://doi.org/10.1016/j.celrep.2021.108959">Drug repurposing screens reveal cell-type-specific entry pathways and FDA-approved drugs active against SARS-Cov-2</a>. Cell Reports 35:108959.</div>
</div>
<div id="ref-ugC5dX0E" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">986. </div><div class="csl-right-inline">Edwards A, Hartung IV. 2021. <a href="https://doi.org/10.1126/science.abj9488">No shortcuts to SARS-CoV-2 antivirals</a>. Science 373:488–489.</div>
</div>
<div id="ref-ZZ2cNql8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">987. </div><div class="csl-right-inline">Hashimoto K. 2021. <a href="https://doi.org/10.1007/s00406-020-01231-x">Repurposing of CNS drugs to treat COVID-19 infection: targeting the sigma-1 receptor</a>. Eur Arch Psychiatry Clin Neurosci 271:249–258.</div>
</div>
<div id="ref-jy4vKMhj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">988. </div><div class="csl-right-inline">Lenze EJ, Mattar C, Zorumski CF, Stevens A, Schweiger J, Nicol GE, Miller JP, Yang L, Yingling M, Avidan MS, Reiersen AM. 2020. <a href="https://doi.org/10.1001/jama.2020.22760">Fluvoxamine vs Placebo and Clinical Deterioration in Outpatients With Symptomatic COVID-19</a>. JAMA 324:2292.</div>
</div>
<div id="ref-s4UuQP07" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">989. </div><div class="csl-right-inline">Seftel D, Boulware DR. 2021. <a href="https://doi.org/10.1093/ofid/ofab050">Prospective Cohort of Fluvoxamine for Early Treatment of Coronavirus Disease 19</a>. Open Forum Infectious Diseases 8.</div>
</div>
<div id="ref-jMLLdjmN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">990. </div><div class="csl-right-inline">Rosen DA, Seki SM, Fernández-Castañeda A, Beiter RM, Eccles JD, Woodfolk JA, Gaultier A. 2019. <a href="https://doi.org/10.1126/scitranslmed.aau5266">Modulation of the sigma-1 receptor–IRE1 pathway is beneficial in preclinical models of inflammation and sepsis</a>. Sci Transl Med 11.</div>
</div>
<div id="ref-18xfYamot" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">991. </div><div class="csl-right-inline">Sukhatme VP, Reiersen AM, Vayttaden SJ, Sukhatme VV. 2021. <a href="https://doi.org/10.3389/fphar.2021.652688">Fluvoxamine: A Review of Its Mechanism of Action and Its Role in COVID-19</a>. Front Pharmacol 12.</div>
</div>
<div id="ref-dUvvJvv6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">992. </div><div class="csl-right-inline">Lowe D. 2021. Too Many Papers. In the Pipeline. <a href="https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers">https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers</a>.</div>
</div>
<div id="ref-yWY8AHZe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">993. </div><div class="csl-right-inline">Prinz F, Schlange T, Asadullah K. 2011. <a href="https://doi.org/10.1038/nrd3439-c1">Believe it or not: how much can we rely on published data on potential drug targets?</a> Nat Rev Drug Discov 10:712–712.</div>
</div>
<div id="ref-rlm4VbAG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">994. </div><div class="csl-right-inline">Swinney DC, Anthony J. 2011. <a href="https://doi.org/10.1038/nrd3480">How were new medicines discovered?</a> Nat Rev Drug Discov 10:507–519.</div>
</div>
<div id="ref-RfR64JTd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">995. </div><div class="csl-right-inline">Kupferschmidt K. 2020. <a href="https://doi.org/10.1126/science.368.6496.1166">Big studies dim hopes for hydroxychloroquine</a>. Science 368:1166–1167.</div>
</div>
<div id="ref-WwRJN5u3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">996. </div><div class="csl-right-inline">Bugin K, Woodcock J. 2021. <a href="https://doi.org/10.1038/d41573-021-00037-3">Trends in COVID-19 therapeutic clinical trials</a>. Nat Rev Drug Discov 20:254–255.</div>
</div>
<div id="ref-yyBcr7R9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">997. </div><div class="csl-right-inline">Dron L, Dillman A, Zoratti MJ, Haggstrom J, Mills EJ, Park JJH. 2021. <a href="https://doi.org/10.2196/26718">Clinical Trial Data Sharing for COVID-19–Related Research</a>. J Med Internet Res 23:e26718.</div>
</div>
<div id="ref-J9N2GoTR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">998. </div><div class="csl-right-inline">Lee Z, Rayner CR, Forrest JI, Nachega JB, Senchaudhuri E, Mills EJ. 2021. <a href="https://doi.org/10.4269/ajtmh.20-1320">The Rise and Fall of Hydroxychloroquine for the Treatment and Prevention of COVID-19</a>. The American Journal of Tropical Medicine and Hygiene 104:35–38.</div>
</div>
<div id="ref-XB0dP3L6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">999. </div><div class="csl-right-inline">Park JJH, Dron L, Mills EJ. 2021. <a href="https://doi.org/10.1016/j.cct.2021.106438">Moving forward in clinical research with master protocols</a>. Contemporary Clinical Trials 106:106438.</div>
</div>
<div id="ref-AJLaaguT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1000. </div><div class="csl-right-inline">2020. Main protease structure and XChem fragment screen. Diamond. <a href="https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem.html">https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem.html</a>.</div>
</div>
<div id="ref-jMzODpwS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1001. </div><div class="csl-right-inline">Barnes MA, Carson MJ, Nair MG. 2015. <a href="https://doi.org/10.1016/j.cyto.2015.01.008">Non-traditional cytokines: How catecholamines and adipokines influence macrophages in immunity, metabolism and the central nervous system</a>. Cytokine 72:210–219.</div>
</div>
<div id="ref-Riev3em1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1002. </div><div class="csl-right-inline">ELENKOV IJ, CHROUSOS GP. 2002. <a href="https://doi.org/10.1111/j.1749-6632.2002.tb04229.x">Stress Hormones, Proinflammatory and Antiinflammatory Cytokines, and Autoimmunity</a>. Annals of the New York Academy of Sciences 966:290–303.</div>
</div>
<div id="ref-JZyVNRCN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1003. </div><div class="csl-right-inline">García A, Martí O, Vallès A, Dal-Zotto S, Armario A. 2000. <a href="https://doi.org/10.1159/000054578">Recovery of the Hypothalamic-Pituitary-Adrenal Response to Stress</a>. Neuroendocrinology 72:114–125.</div>
</div>
<div id="ref-SiXrQUgV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1004. </div><div class="csl-right-inline">Elenkov IJ, Papanicolaou DA, Wilder RL, Chrousos GP. 1996. <a href="https://www.ncbi.nlm.nih.gov/pubmed/8902882">Modulatory effects of glucocorticoids and catecholamines on human interleukin-12 and interleukin-10 production: clinical implications.</a> Proc Assoc Am Physicians 108:374–81.</div>
</div>
<div id="ref-EgvIETTq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1005. </div><div class="csl-right-inline">Theoharides TC. 2020. <a href="https://doi.org/10.23812/20-editorial_1-5">Dexamethasone for COVID-19? Not so fast.</a> JOURNAL OF BIOLOGICAL REGULATORS AND HOMEOSTATIC AGENTS 34.</div>
</div>
<div id="ref-TytQ90Bl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1006. </div><div class="csl-right-inline">Matthay MA, Thompson BT. 2020. <a href="https://doi.org/10.1016/s2213-2600(20)30503-8">Dexamethasone in hospitalised patients with COVID-19: addressing uncertainties</a>. The Lancet Respiratory Medicine 8:1170–1172.</div>
</div>
<div id="ref-1E4Qx2UqB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1007. </div><div class="csl-right-inline">Brotherton H, Usuf E, Nadjm B, Forrest K, Bojang K, Samateh AL, Bittaye M, Roberts CA, d'Alessandro U, Roca A. 2020. <a href="https://doi.org/10.1016/s2214-109x(20)30318-1">Dexamethasone for COVID-19: data needed from randomised clinical trials in Africa</a>. The Lancet Global Health 8:e1125–e1126.</div>
</div>
<div id="ref-13Oz7yRPx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1008. </div><div class="csl-right-inline">Cai Q, Yang M, Liu D, Chen J, Shu D, Xia J, Liao X, Gu Y, Cai Q, Yang Y, Shen C, Li X, Peng L, Huang D, Zhang J, Zhang S, Wang F, Liu J, Chen L, Chen S, Wang Z, Zhang Z, Cao R, Zhong W, Liu Y, Liu L. 2020. <a href="https://doi.org/10.1016/j.eng.2020.03.007">Experimental Treatment with Favipiravir for COVID-19: An Open-Label Control Study</a>. Engineering 6:1192–1198.</div>
</div>
<div id="ref-fTWlYlvp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1009. </div><div class="csl-right-inline">Horby PW, Mafham M, Bell JL, Linsell L, Staplin N, Emberson J, Palfreeman A, Raw J, Elmahi E, Prudon B, Green C, Carley S, Chadwick D, Davies M, Wise MP, Baillie JK, Chappell LC, Faust SN, Jaki T, Jefferey K, Lim WS, Montgomery A, Rowan K, Juszczak E, Haynes R, Landray MJ. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)32013-4">Lopinavir–ritonavir in patients admitted to hospital with COVID-19 (RECOVERY): a randomised, controlled, open-label, platform trial</a>. The Lancet 396:1345–1352.</div>
</div>
<div id="ref-xM0IhxIr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1010. </div><div class="csl-right-inline">Harrington DP, Baden LR, Hogan JW. 2020. <a href="https://doi.org/10.1056/nejme2034294">A Large, Simple Trial Leading to Complex Questions</a>. New England Journal of Medicine NEJMe2034294.</div>
</div>
<div id="ref-hfAF6aDr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1011. </div><div class="csl-right-inline">2020. Retracted coronavirus (COVID-19) papers. Retraction Watch. <a href="https://retractionwatch.com/retracted-coronavirus-covid-19-papers/">https://retractionwatch.com/retracted-coronavirus-covid-19-papers/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-APU0cUZz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1012. </div><div class="csl-right-inline">Lou Y, Liu L, Yao H, Hu X, Su J, Xu K, Luo R, Yang X, He L, Lu X, Zhao Q, Liang T, Qiu Y. 2021. <a href="https://doi.org/10.1016/j.ejps.2020.105631">Clinical Outcomes and Plasma Concentrations of Baloxavir Marboxil and Favipiravir in COVID-19 Patients: An Exploratory Randomized, Controlled Trial</a>. European Journal of Pharmaceutical Sciences 157:105631.</div>
</div>
<div id="ref-15rGJX7Hl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1013. </div><div class="csl-right-inline">Dabbous HM, Abd-Elsalam S, El-Sayed MH, Sherief AF, Ebeid FFS, El Ghafar MSA, Soliman S, Elbahnasawy M, Badawi R, Tageldin MA. 2021. Efficacy of favipiravir in COVID-19 treatment: a multi-center randomized study. Archives of Virology https://doi.org/<a href="https://doi.org/10.1007/s00705-021-04956-9">10.1007/s00705-021-04956-9</a>.</div>
</div>
<div id="ref-32n9UEkQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1014. </div><div class="csl-right-inline">Ivashchenko AA, Dmitriev KA, Vostokova NV, Azarova VN, Blinow AA, Egorova AN, Gordeev IG, Ilin AP, Karapetian RN, Kravchenko DV, Lomakin NV, Merkulova EA, Papazova NA, Pavlikova EP, Savchuk NP, Simakina EN, Sitdekov TA, Smolyarchuk EA, Tikhomolova EG, Yakubova EV, Ivachtchenko AV. 2020. <a href="https://doi.org/10.1093/cid/ciaa1176">AVIFAVIR for Treatment of Patients With Moderate Coronavirus Disease 2019 (COVID-19): Interim Results of a Phase II/III Multicenter Randomized Clinical Trial</a>. Clinical Infectious Diseases ciaa1176.</div>
</div>
<div id="ref-qwlYY6jn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1015. </div><div class="csl-right-inline">Pilkington V, Pepperrell T, Hill A. 2020. <a href="https://doi.org/10.1016/s2055-6640(20)30016-9">A review of the safety of favipiravir – a potential treatment in the COVID-19 pandemic?</a> Journal of Virus Eradication 6:45–51.</div>
</div>
<div id="ref-gnfviuGp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1016. </div><div class="csl-right-inline">Mulangu S, Dodd LE, Davey RT, Tshiani Mbaya O, Proschan M, Mukadi D, Lusakibanza Manzo M, Nzolo D, Tshomba Oloma A, Ibanda A, Ali R, Coulibaly S, Levine AC, Grais R, Diaz J, Lane HC, Muyembe-Tamfum J-J, the PALM Writing Group. 2019. <a href="https://doi.org/10.1056/nejmoa1910993">A Randomized, Controlled Trial of Ebola Virus Disease Therapeutics</a>. New England Journal of Medicine 381:2293–2303.</div>
</div>
<div id="ref-1H0NPTMw9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1017. </div><div class="csl-right-inline">Sheahan TP, Sims AC, Graham RL, Menachery VD, Gralinski LE, Case JB, Leist SR, Pyrc K, Feng JY, Trantcheva I, Bannister R, Park Y, Babusis D, Clarke MO, Mackman RL, Spahn JE, Palmiotti CA, Siegel D, Ray AS, Cihlar T, Jordan R, Denison MR, Baric RS. 2017. <a href="https://doi.org/10.1126/scitranslmed.aal3653">Broad-spectrum antiviral GS-5734 inhibits both epidemic and zoonotic coronaviruses</a>. Science Translational Medicine 9:eaal3653.</div>
</div>
<div id="ref-D4HwvRMd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1018. </div><div class="csl-right-inline">Cohen J. 2020. Did an experimental drug help a U.S. coronavirus patient? Science https://doi.org/<a href="https://doi.org/10.1126/science.abb7243">10.1126/science.abb7243</a>.</div>
</div>
<div id="ref-zoLViMNg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1019. </div><div class="csl-right-inline">Kujawski SA, Wong KK, Collins JP, Epstein L, Killerby ME, Midgley CM, Abedi GR, Ahmed NS, Almendares O, Alvarez FN, Anderson KN, Balter S, Barry V, Bartlett K, Beer K, Ben-Aderet MA, Benowitz I, Biggs H, Binder AM, Black SR, Bonin B, Brown CM, Bruce H, Bryant-Genevier J, Budd A, Buell D, Bystritsky R, Cates J, Charles EM, Chatham-Stephens K, Chea N, Chiou H, Christiansen D, Chu V, Cody S, Cohen M, Conners E, Curns A, Dasari V, Dawson P, DeSalvo T, Diaz G, Donahue M, Donovan S, Duca LM, Erickson K, Esona MD, Evans S, Falk J, Feldstein LR, Fenstersheib M, Fischer M, Fisher R, Foo C, Fricchione MJ, Friedman O, Fry AM, Galang RR, Garcia MM, Gerber SI, Gerrard G, Ghinai I, Gounder P, Grein J, Grigg C, Gunzenhauser JD, Gutkin GI, Haddix M, Hall AJ, Han G, Harcourt J, Harriman K, Haupt T, Haynes A, Holshue M, Hoover C, Hunter JC, Jacobs MW, Jarashow C, Jhung MA, Joshi K, Kamali T, Kamili S, Kim L, Kim M, King J, Kirking HL, Kita-Yarbro A, Klos R, Kobayashi M, Kocharian A, Komatsu KK, Koppaka R, Layden JE, Li Y, Lindquist S, Lindstrom S, Link-Gelles R, Lively J, Livingston M, Lo K, Lo J, Lu X, Lynch B, Madoff L, Malapati L, Marks G, Marlow M, Mathisen GE, McClung N, McGovern O, McPherson TD, Mehta M, Meier A, Mello L, Moon S, Morgan M, Moro RN, Murray J, Murthy R, Novosad S, Oliver SE, O'Shea J, Pacilli M, Paden CR, Pallansch MA, Patel M, Patel S, Pedraza I, Pillai SK, Pindyck T, Pray I, Queen K, Quick N, Reese H, Rha B, Rhodes H, Robinson S, Robinson P, Rolfes M, Routh J, Rubin R, Rudman SL, Sakthivel SK, Scott S, Shepherd C, Shetty V, Smith EA, Smith S, Stierman B, Stoecker W, Sunenshine R, Sy-Santos R, Tamin A, Tao Y, Terashita D, Thornburg NJ, Tong S, Traub E, Tural A, Uehara A, Uyeki TM, Vahey G, Verani JR, Villarino E, Wallace M, Wang L, Watson JT, Westercamp M, Whitaker B, Wilkerson S, Woodruff RC, Wortham JM, Wu T, Xie A, Yousaf A, Zahn M, Zhang J, The COVID-19 Investigation Team. 2020. First 12 patients with coronavirus disease 2019 (COVID-19) in the United States. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.09.20032896">10.1101/2020.03.09.20032896</a>.</div>
</div>
<div id="ref-gleq16qf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1020. </div><div class="csl-right-inline">Holshue ML, DeBolt C, Lindquist S, Lofy KH, Wiesman J, Bruce H, Spitters C, Ericson K, Wilkerson S, Tural A, Diaz G, Cohn A, Fox L, Patel A, Gerber SI, Kim L, Tong S, Lu X, Lindstrom S, Pallansch MA, Weldon WC, Biggs HM, Uyeki TM, Pillai SK. 2020. <a href="https://doi.org/10.1056/nejmoa2001191">First Case of 2019 Novel Coronavirus in the United States</a>. New England Journal of Medicine 382:929–936.</div>
</div>
<div id="ref-w5y1LWUR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1021. </div><div class="csl-right-inline">Goldman JD, Lye DCB, Hui DS, Marks KM, Bruno R, Montejano R, Spinner CD, Galli M, Ahn M-Y, Nahass RG, Chen Y-S, SenGupta D, Hyland RH, Osinusi AO, Cao H, Blair C, Wei X, Gaggar A, Brainard DM, Towner WJ, Muñoz J, Mullane KM, Marty FM, Tashima KT, Diaz G, Subramanian A. 2020. <a href="https://doi.org/10.1056/nejmoa2015301">Remdesivir for 5 or 10 Days in Patients with Severe Covid-19</a>. New England Journal of Medicine 383:1827–1837.</div>
</div>
<div id="ref-1BYoe0y2H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1022. </div><div class="csl-right-inline">Hinton DM. 2020. Remdesivir EUA Letter of Authorization. <a href="https://www.fda.gov/media/137564/download">https://www.fda.gov/media/137564/download</a>.</div>
</div>
<div id="ref-oso9r21L" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1023. </div><div class="csl-right-inline">Gilead Sciences. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04292730">A Phase 3 Randomized Study to Evaluate the Safety and Antiviral Activity of Remdesivir (GS-5734™) in Participants With Moderate COVID-19 Compared to Standard of Care Treatment</a>. NCT04292730. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-1q4PmqTm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1024. </div><div class="csl-right-inline">2020. Multi-centre, adaptive, randomized trial of the safety and efficacy of treatments of COVID-19 in hospitalized adults. EU Clinical Trials Register. <a href="https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR">https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR</a>.</div>
</div>
<div id="ref-rUTsIlZs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1025. </div><div class="csl-right-inline">A Trial of Remdesivir in Adults With Mild and Moderate COVID-19 - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04252664">https://clinicaltrials.gov/ct2/show/NCT04252664</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-JSoOOX98" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1026. </div><div class="csl-right-inline">Cao B. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04257656">A Phase 3 Randomized, Double-blind, Placebo-controlled, Multicenter Study to Evaluate the Efficacy and Safety of Remdesivir in Hospitalized Adult Patients With Severe COVID-19.</a> NCT04257656. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-1DTFVDWTn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1027. </div><div class="csl-right-inline">Commissioner O of the. 2020. FDA Approves First Treatment for COVID-19. FDA. <a href="https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19">https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-zQT9JLP1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1028. </div><div class="csl-right-inline">Gilead Sciences Statement on the Solidarity Trial. <a href="https://www.gilead.com/news-and-press/company-statements/gilead-sciences-statement-on-the-solidarity-trial">https://www.gilead.com/news-and-press/company-statements/gilead-sciences-statement-on-the-solidarity-trial</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-vJVD0Wle" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1029. </div><div class="csl-right-inline">Galiuto L, Patrono C. 2020. <a href="https://doi.org/10.1093/eurheartj/ehaa934">Conflicting results on the efficacy of remdesivir in hospitalized Covid-19 patients: comment on the Adaptive Covid-19 Treatment Trial</a>. European Heart Journal 41:4387–4388.</div>
</div>
<div id="ref-pQkqi9ni" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1030. </div><div class="csl-right-inline">Cohen J, Kupferschmidt K. 2020. The ‘very, very bad look’ of remdesivir, the first FDA-approved COVID-19 drug. Science | AAAS. <a href="https://www.sciencemag.org/news/2020/10/very-very-bad-look-remdesivir-first-fda-approved-covid-19-drug">https://www.sciencemag.org/news/2020/10/very-very-bad-look-remdesivir-first-fda-approved-covid-19-drug</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-6NLZTSS0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1031. </div><div class="csl-right-inline">Spinner CD, Gottlieb RL, Criner GJ, Arribas López JR, Cattelan AM, Soriano Viladomiu A, Ogbuagu O, Malhotra P, Mullane KM, Castagna A, Chai LYA, Roestenberg M, Tsang OTY, Bernasconi E, Le Turnier P, Chang S-C, SenGupta D, Hyland RH, Osinusi AO, Cao H, Blair C, Wang H, Gaggar A, Brainard DM, McPhail MJ, Bhagani S, Ahn MY, Sanyal AJ, Huhn G, Marty FM, for the GS-US-540-5774 Investigators. 2020. <a href="https://doi.org/10.1001/jama.2020.16349">Effect of Remdesivir vs Standard Care on Clinical Status at 11 Days in Patients With Moderate COVID-19</a>. JAMA 324:1048.</div>
</div>
<div id="ref-pDl6Q4UP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1032. </div><div class="csl-right-inline">Kalil AC, Patterson TF, Mehta AK, Tomashek KM, Wolfe CR, Ghazaryan V, Marconi VC, Ruiz-Palacios GM, Hsieh L, Kline S, Tapson V, Iovine NM, Jain MK, Sweeney DA, El Sahly HM, Branche AR, Regalado Pineda J, Lye DC, Sandkovsky U, Luetkemeyer AF, Cohen SH, Finberg RW, Jackson PEH, Taiwo B, Paules CI, Arguinchona H, Erdmann N, Ahuja N, Frank M, Oh M, Kim E-S, Tan SY, Mularski RA, Nielsen H, Ponce PO, Taylor BS, Larson L, Rouphael NG, Saklawi Y, Cantos VD, Ko ER, Engemann JJ, Amin AN, Watanabe M, Billings J, Elie M-C, Davey RT, Burgess TH, Ferreira J, Green M, Makowski M, Cardoso A, de Bono S, Bonnett T, Proschan M, Deye GA, Dempsey W, Nayak SU, Dodd LE, Beigel JH. 2020. <a href="https://doi.org/10.1056/nejmoa2031994">Baricitinib plus Remdesivir for Hospitalized Adults with Covid-19</a>. New England Journal of Medicine NEJMoa2031994.</div>
</div>
<div id="ref-1HiXNTn3b" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1033. </div><div class="csl-right-inline">Hinton DM. 2020. Letter of Authorization: EUA for baricitinib (Olumiant), in combination with remdesivir (Veklury), for the treatment of suspected or laboratory confirmed coronavirus disease 2019 (COVID-19). Food and Drug Administration. <a href="https://www.fda.gov/media/143822/download">https://www.fda.gov/media/143822/download</a>.</div>
</div>
<div id="ref-ggYRK4FU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1034. </div><div class="csl-right-inline">Ashfaq UA, Javed T, Rehman S, Nawaz Z, Riazuddin S. 2011. <a href="https://doi.org/10.1186/1743-422x-8-163">Lysosomotropic agents as HCV entry inhibitors</a>. Virology Journal 8:163.</div>
</div>
<div id="ref-nEaG534R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1035. </div><div class="csl-right-inline">Müller-Calleja N, Manukyan D, Ruf W, Lackner K. 2016. <a href="https://doi.org/10.1182/blood.v128.22.5023.5023">Mechanism of Action of Hydroxychloroquine in the Antiphospholipid Syndrome</a>. Blood 128:5023–5023.</div>
</div>
<div id="ref-rL8owM5Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1036. </div><div class="csl-right-inline">Erkan D, Aguiar CL, Andrade D, Cohen H, Cuadrado MJ, Danowski A, Levy RA, Ortel TL, Rahman A, Salmon JE, Tektonidou MG, Willis R, Lockshin MD. 2014. <a href="https://doi.org/10.1016/j.autrev.2014.01.053">14th International Congress on Antiphospholipid Antibodies Task Force Report on Antiphospholipid Syndrome Treatment Trends</a>. Autoimmunity Reviews 13:685–696.</div>
</div>
<div id="ref-134vCyU3E" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1037. </div><div class="csl-right-inline">Wang T-F, Lim W. 2016. <a href="https://doi.org/10.1182/asheducation-2016.1.714">What is the role of hydroxychloroquine in reducing thrombotic risk in patients with antiphospholipid antibodies?</a> Hematology 2016:714–716.</div>
</div>
<div id="ref-WFGcbZhe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1038. </div><div class="csl-right-inline">Zhou D, Dai S-M, Tong Q. 2020. <a href="https://doi.org/10.1093/jac/dkaa114">COVID-19: a recommendation to examine the effect of hydroxychloroquine in preventing infection and progression</a>. Journal of Antimicrobial Chemotherapy 75:1667–1670.</div>
</div>
<div id="ref-hFGjQQN8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1039. </div><div class="csl-right-inline">Sperber K, Louie M, Kraus T, Proner J, Sapira E, Lin S, Stecher V, Mayer L. 1995. <a href="https://doi.org/10.1016/0149-2918(95)80039-5">Hydroxychloroquine treatment of patients with human immunodeficiency virus type 1</a>. Clinical Therapeutics 17:622–636.</div>
</div>
<div id="ref-rQNzKxRd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1040. </div><div class="csl-right-inline">Helal GK, Gad MA, Abd-Ellah MF, Eid MS. 2016. <a href="https://doi.org/10.1002/jmv.24575">Hydroxychloroquine augments early virological response to pegylated interferon plus ribavirin in genotype-4 chronic hepatitis C patients</a>. Journal of Medical Virology 88:2170–2178.</div>
</div>
<div id="ref-36WPxcLu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1041. </div><div class="csl-right-inline">Coster WJ. 2013. <a href="https://doi.org/10.5014/ajot.2013.006015">Making the Best Match: Selecting Outcome Measures for Clinical Trials and Outcome Studies</a>. The American Journal of Occupational Therapy 67:162–170.</div>
</div>
<div id="ref-18KmvBOFr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1042. </div><div class="csl-right-inline">Molina JM, Delaugerre C, Le Goff J, Mela-Lima B, Ponscarme D, Goldwirt L, de Castro N. 2020. <a href="https://doi.org/10.1016/j.medmal.2020.03.006">No evidence of rapid antiviral clearance or clinical benefit with the combination of hydroxychloroquine and azithromycin in patients with severe COVID-19 infection</a>. Médecine et Maladies Infectieuses 50:384.</div>
</div>
<div id="ref-wzHb7Mj5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1043. </div><div class="csl-right-inline">Chen Z, Hu J, Zhang Z, Jiang S, Han S, Yan D, Zhuang R, Hu B, Zhang Z. 2020. Efficacy of hydroxychloroquine in patients with COVID-19: results of a randomized clinical trial. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.22.20040758">10.1101/2020.03.22.20040758</a>.</div>
</div>
<div id="ref-65ggO2DM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1044. </div><div class="csl-right-inline">2020. Therapeutic effect of hydroxychloroquine on novel coronavirus pneumonia (COVID-19). Chinese Clinical Trial Registry. <a href="http://www.chictr.org.cn/showprojen.aspx?proj=48880">http://www.chictr.org.cn/showprojen.aspx?proj=48880</a>.</div>
</div>
<div id="ref-o5gwQA7W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1045. </div><div class="csl-right-inline">CHEN Jun, LIU Danping, LIU Li, LIU Ping, XU Qingnian, XIA Lu, LING Yun, HUANG Dan, SONG Shuli, ZHANG Dandan, QIAN Zhiping, LI Tao, SHEN Yinzhong, LU Hongzhou. 2020. <a href="https://doi.org/10.3785/j.issn.1008-9292.2020.03.03">A pilot study of hydroxychloroquine in treatment of patients with common coronavirus disease-19 (COVID-19)</a>. Journal of Zhejiang University (Medical Sciences) 49.</div>
</div>
<div id="ref-aiItPxSY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1046. </div><div class="csl-right-inline">Gao J, Tian Z, Yang X. 2020. <a href="https://doi.org/10.5582/bst.2020.01047">Breakthrough: Chloroquine phosphate has shown apparent efficacy in treatment of COVID-19 associated pneumonia in clinical studies</a>. BioScience Trends 14:72–73.</div>
</div>
<div id="ref-1BdQKtXnY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1047. </div><div class="csl-right-inline">Yang N, Shen H-M. 2020. <a href="https://doi.org/10.7150/ijbs.45498">Targeting the Endocytic Pathway and Autophagy Process as a Novel Therapeutic Strategy in COVID-19</a>. International Journal of Biological Sciences 16:1724–1731.</div>
</div>
<div id="ref-z8XnLYWM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1048. </div><div class="csl-right-inline">Zheng J. 2020. <a href="https://doi.org/10.7150/ijbs.45053">SARS-CoV-2: an Emerging Coronavirus that Causes a Global Threat</a>. International Journal of Biological Sciences 16:1678–1685.</div>
</div>
<div id="ref-1BApBNHvd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1049. </div><div class="csl-right-inline">Mehra MR, Desai SS, Ruschitzka F, Patel AN. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)31180-6">RETRACTED: Hydroxychloroquine or chloroquine with or without a macrolide for treatment of COVID-19: a multinational registry analysis</a>. The Lancet S0140673620311806.</div>
</div>
<div id="ref-11FsxWalS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1050. </div><div class="csl-right-inline">Mehra MR, Ruschitzka F, Patel AN. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)31324-6">Retraction—Hydroxychloroquine or chloroquine with or without a macrolide for treatment of COVID-19: a multinational registry analysis</a>. The Lancet 395:1820.</div>
</div>
<div id="ref-dY9kXLAS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1051. </div><div class="csl-right-inline">O’Laughlin JP, Mehta PH, Wong BC. 2016. <a href="https://doi.org/10.1155/2016/4626279">Life Threatening Severe QTc Prolongation in Patient with Systemic Lupus Erythematosus due to Hydroxychloroquine</a>. Case Reports in Cardiology 2016:1–4.</div>
</div>
<div id="ref-wy3S7Cpa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1052. </div><div class="csl-right-inline">Roden DM. 2008. <a href="https://doi.org/10.1016/j.hrthm.2008.05.008">Keep the QT interval: It is a reliable predictor of ventricular arrhythmias</a>. Heart Rhythm 5:1213–1215.</div>
</div>
<div id="ref-OQ0b31BA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1053. </div><div class="csl-right-inline">C.E.Lane J, Weaver J, Kostka K, Duarte-Salles T, Abrahao MTF, Alghoul H, Alser O, Alshammari TM, Biedermann P, Burn E, Casajust P, Conover M, Culhane AC, Davydov A, DuVall SL, Dymshyts D, Fernandez-Bertolin S, Fišter K, Hardin J, Hester L, Hripcsak G, Kent S, Khosla S, Kolovos S, Lambert CG, van der Lei J, Londhe AA, Lynch KE, Makadia R, Margulis AV, Matheny ME, Mehta P, Morales DR, Morgan-Stewart H, Mosseveld M, Newby D, Nyberg F, Ostropolets A, Park RW, Prats-Uribe A, Rao GA, Reich C, Reps J, Rijnbeek P, Kumaran Sathappan SM, Schuemie M, Seager S, Sena A, Shoaibi A, Spotnitz M, Suchard MA, Swerdel J, Torre CO, Vizcaya D, Wen H, de Wilde M, You SC, Zhang L, Zhuk O, Ryan P, Prieto-Alhambra D. 2020. Safety of hydroxychloroquine, alone and in combination with azithromycin, in light of rapid wide-spread use for COVID-19: a multinational, network cohort and self-controlled case series study. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.04.08.20054551">10.1101/2020.04.08.20054551</a>.</div>
</div>
<div id="ref-1CgZR55GZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1054. </div><div class="csl-right-inline">Silva Borba MG, Almeida Val FF, Sampaio VS, Araújo Alexandre MA, Melo GC, Brito M, Gomes Mourão MP, Brito-Sousa JD, Baía-da-Silva D, Farias Guerra MV, Abrahão Hajjar L, Pinto RC, Silva Balieiro AA, Naveca FG, Simão Xavier M, Salomão A, Siqueira AM, Schwarzbolt A, Rosa Croda JH, Nogueira ML, Sierra Romero GA, Bassat Q, Fontes CJ, Cláudio Albuquerque B, Daniel-Ribeiro CT, Monteiro WM, Guimarães Lacerda MV, CloroCovid-19 Team. 2020. Chloroquine diphosphate in two different dosages as adjunctive therapy of hospitalized patients with severe respiratory syndrome in the context of coronavirus (SARS-CoV-2) infection: Preliminary safety results of a randomized, double-blinded, phase IIb clinical trial (CloroCovid-19 Study). Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.04.07.20056424">10.1101/2020.04.07.20056424</a>.</div>
</div>
<div id="ref-1EcscrT32" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1055. </div><div class="csl-right-inline">Howard J, Cohen E, Kounang N, Nyberg P. 2020. Heart risk concerns mount around use of chloroquine and hydroxychloroquine for Covid-19 treatment. CNN. <a href="https://www.cnn.com/2020/04/13/health/chloroquine-risks-coronavirus-treatment-trials-study/index.html">https://www.cnn.com/2020/04/13/health/chloroquine-risks-coronavirus-treatment-trials-study/index.html</a>.</div>
</div>
<div id="ref-AMQ5402v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1056. </div><div class="csl-right-inline">2020. WHO Director-General's opening remarks at the media briefing on COVID-19. World Health Organization. <a href="https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---25-may-2020">https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---25-may-2020</a>.</div>
</div>
<div id="ref-RXr9BKKT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1057. </div><div class="csl-right-inline">Tang W, Cao Z, Han M, Wang Z, Chen J, Sun W, Wu Y, Xiao W, Liu S, Chen E, Chen W, Wang X, Yang J, Lin J, Zhao Q, Yan Y, Xie Z, Li D, Yang Y, Liu L, Qu J, Ning G, Shi G, Xie Q. 2020. Hydroxychloroquine in patients mainly with mild to moderate COVID–19: an open–label, randomized, controlled trial. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.04.10.20060558">10.1101/2020.04.10.20060558</a>.</div>
</div>
<div id="ref-MENbRfOl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1058. </div><div class="csl-right-inline">Magagnoli J, Narendran S, Pereira F, Cummings T, Hardin JW, Sutton SS, Ambati J. 2020. Outcomes of hydroxychloroquine usage in United States veterans hospitalized with Covid-19. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.04.16.20065920">10.1101/2020.04.16.20065920</a>.</div>
</div>
<div id="ref-V8fkkVZg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1059. </div><div class="csl-right-inline">Mitjà O, Corbacho-Monné M, Ubals M, Tebé C, Peñafiel J, Tobias A, Ballana E, Alemany A, Riera-Martí N, Pérez CA, Suñer C, Laporte P, Admella P, Mitjà J, Clua M, Bertran L, Sarquella M, Gavilán S, Ara J, Argimon JM, Casabona J, Cuatrecasas G, Cañadas P, Elizalde-Torrent A, Fabregat R, Farré M, Forcada A, Flores-Mateo G, Muntada E, Nadal N, Narejos S, Nieto A, Prat N, Puig J, Quiñones C, Reyes-Ureña J, Ramírez-Viaplana F, Ruiz L, Riveira-Muñoz E, Sierra A, Velasco C, Vivanco-Hidalgo RM, Sentís A, G-Beiras C, Clotet B, Vall-Mayans M. 2020. <a href="https://doi.org/10.1093/cid/ciaa1009">Hydroxychloroquine for Early Treatment of Adults With Mild Coronavirus Disease 2019: A Randomized, Controlled Trial</a>. Clinical Infectious Diseases ciaa1009.</div>
</div>
<div id="ref-NiVdIN7Z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1060. </div><div class="csl-right-inline">Boulware DR, Pullen MF, Bangdiwala AS, Pastick KA, Lofgren SM, Okafor EC, Skipper CP, Nascene AA, Nicol MR, Abassi M, Engen NW, Cheng MP, LaBar D, Lother SA, MacKenzie LJ, Drobot G, Marten N, Zarychanski R, Kelly LE, Schwartz IS, McDonald EG, Rajasingham R, Lee TC, Hullsiek KH. 2020. <a href="https://doi.org/10.1056/nejmoa2016638">A Randomized Trial of Hydroxychloroquine as Postexposure Prophylaxis for Covid-19</a>. New England Journal of Medicine 383:517–525.</div>
</div>
<div id="ref-oaqFxV4W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1061. </div><div class="csl-right-inline">Abella BS, Jolkovsky EL, Biney BT, Uspal JE, Hyman MC, Frank I, Hensley SE, Gill S, Vogl DT, Maillard I, Babushok DV, Huang AC, Nasta SD, Walsh JC, Wiletyo EP, Gimotty PA, Milone MC, Amaravadi RK, Prevention and Treatment of COVID-19 With Hydroxychloroquine (PATCH) Investigators. 2021. <a href="https://doi.org/10.1001/jamainternmed.2020.6319">Efficacy and Safety of Hydroxychloroquine vs Placebo for Pre-exposure SARS-CoV-2 Prophylaxis Among Health Care Workers</a>. JAMA Internal Medicine 181:195.</div>
</div>
<div id="ref-j3rGUH1f" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1062. </div><div class="csl-right-inline">Schrezenmeier E, Dörner T. 2020. <a href="https://doi.org/10.1038/s41584-020-0372-x">Mechanisms of action of hydroxychloroquine and chloroquine: implications for rheumatology</a>. Nature Reviews Rheumatology 16:155–166.</div>
</div>
<div id="ref-H0oEjZR8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1063. </div><div class="csl-right-inline">Elimination or Prolongation of ACE Inhibitors and ARB in Coronavirus Disease 2019 - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04338009">https://clinicaltrials.gov/ct2/show/NCT04338009</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-kuUcA1G9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1064. </div><div class="csl-right-inline">Stopping ACE-inhibitors in COVID-19 - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04353596">https://clinicaltrials.gov/ct2/show/NCT04353596</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-3ccJAyrG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1065. </div><div class="csl-right-inline">Losartan for Patients With COVID-19 Not Requiring Hospitalization - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04311177">https://clinicaltrials.gov/ct2/show/NCT04311177</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-NCb9Wq2z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1066. </div><div class="csl-right-inline">Losartan for Patients With COVID-19 Requiring Hospitalization - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04312009">https://clinicaltrials.gov/ct2/show/NCT04312009</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-5fkIarR7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1067. </div><div class="csl-right-inline">McEvoy PJW. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04330300">The CORONAvirus Disease 2019 Angiotensin Converting Enzyme Inhibitor/Angiotensin Receptor Blocker InvestigatiON (CORONACION) Randomized Clinical Trial</a>. NCT04330300. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-c7V9rAFR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1068. </div><div class="csl-right-inline">Ramipril for the Treatment of COVID-19 - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04366050">https://clinicaltrials.gov/ct2/show/NCT04366050</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-JP1lU6DV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1069. </div><div class="csl-right-inline">D'Or Institute for Research and Education. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04364893">Suspension of Angiotensin Receptor Blockers and Angiotensin-converting Enzyme Inhibitors and Adverse Outcomes in Hospitalized Patients With Coronavirus Infection (COVID-19). A Randomized Trial</a>. NCT04364893. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-19quOF2kK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1070. </div><div class="csl-right-inline">Cohen JB, Hanff TC, South AM, Sparks MA, Hiremath S, Bress AP, Byrd JB, Chirinos JA. 2020. <a href="https://doi.org/10.1161/circresaha.120.317205">Response by Cohen et al to Letter Regarding Article, “Association of Inpatient Use of Angiotensin-Converting Enzyme Inhibitors and Angiotensin II Receptor Blockers With Mortality Among Patients With Hypertension Hospitalized With COVID-19”</a>. Circulation Research 126.</div>
</div>
<div id="ref-k5xEN3Ke" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1071. </div><div class="csl-right-inline">Sparks MA, South A, Welling P, Luther JM, Cohen J, Byrd JB, Burrell LM, Batlle D, Tomlinson L, Bhalla V, Rheault MN, Soler MJ, Swaminathan S, Hiremath S. 2020. <a href="https://doi.org/10.2215/cjn.03530320">Sound Science before Quick Judgement Regarding RAS Blockade in COVID-19</a>. Clinical Journal of the American Society of Nephrology 15:714–716.</div>
</div>
<div id="ref-1Hrsy8Hiq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1072. </div><div class="csl-right-inline">Sparks M, Hiremath S. The Coronavirus Conundrum: ACE2 and Hypertension Edition. NephJC. <a href="http://www.nephjc.com/news/covidace2">http://www.nephjc.com/news/covidace2</a>. Retrieved 30 July 2020.</div>
</div>
<div id="ref-VXDsRwHC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1073. </div><div class="csl-right-inline">Luo Y, Zheng SG. 2016. <a href="https://doi.org/10.3389/fimmu.2016.00604">Hall of Fame among Pro-inflammatory Cytokines: Interleukin-6 Gene and Its Transcriptional Regulation Mechanisms</a>. Frontiers in Immunology 7.</div>
</div>
<div id="ref-14xZS2Gic" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1074. </div><div class="csl-right-inline">Rose-John S. 2012. <a href="https://doi.org/10.7150/ijbs.4989">IL-6 Trans-Signaling via the Soluble IL-6 Receptor: Importance for the Pro-Inflammatory Activities of IL-6</a>. International Journal of Biological Sciences 8:1237–1247.</div>
</div>
<div id="ref-LIqxTdN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1075. </div><div class="csl-right-inline">Scheller J, Rose-John S. 2006. <a href="https://doi.org/10.1007/s00430-006-0019-9">Interleukin-6 and its receptor: from bench to bedside</a>. Medical Microbiology and Immunology 195:173–183.</div>
</div>
<div id="ref-1BxkPZYnj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1076. </div><div class="csl-right-inline">Garbers C, Hermanns HM, Schaper F, Müller-Newen G, Grötzinger J, Rose-John S, Scheller J. 2012. <a href="https://doi.org/10.1016/j.cytogfr.2012.04.001">Plasticity and cross-talk of Interleukin 6-type cytokines</a>. Cytokine &amp; Growth Factor Reviews 23:85–97.</div>
</div>
<div id="ref-1EEloBjzn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1077. </div><div class="csl-right-inline">Rose-John S, Heinrich PC. 1994. <a href="https://doi.org/10.1042/bj3000281">Soluble receptors for cytokines and growth factors: generation and biological function</a>. Biochemical Journal 300:281–290.</div>
</div>
<div id="ref-1H8o8TFtM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1078. </div><div class="csl-right-inline">Tanaka T, Narazaki M, Masuda K, Kishimoto T. 2013. <a href="https://doi.org/10.2492/inflammregen.33.054">Interleukin-6; pathogenesis and treatment of autoimmune inflammatory diseases</a>. Inflammation and Regeneration 33:054–065.</div>
</div>
<div id="ref-3yV2bL9S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1079. </div><div class="csl-right-inline">Watad A, Bragazzi NL, Bridgewood C, Mansour M, Mahroum N, Riccò M, Nasr A, Hussein A, Gendelman O, Shoenfeld Y, Lindar M, Amital H, Wu J, McGonagle D. 2020. Systematic Review and Meta-Analysis of Case-Control Studies from 7,000 COVID-19 Pneumonia Patients Suggests a Beneficial Impact of Tocilizumab with Benefit Most Evident in Non-Corticosteroid Exposed Subjects. SSRN Electronic Journal https://doi.org/<a href="https://doi.org/10.2139/ssrn.3642653">10.2139/ssrn.3642653</a>.</div>
</div>
<div id="ref-WmK4YTdx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1080. </div><div class="csl-right-inline">Kaye AG, Siegel R. 2020. <a href="https://doi.org/10.7717/peerj.10322">The efficacy of IL-6 inhibitor Tocilizumab in reducing severe COVID-19 mortality: a systematic review</a>. PeerJ 8:e10322.</div>
</div>
<div id="ref-h4reluu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1081. </div><div class="csl-right-inline">Cortegiani A, Ippolito M, Greco M, Granone V, Protti A, Gregoretti C, Giarratano A, Einav S, Cecconi M. 2021. <a href="https://doi.org/10.1016/j.pulmoe.2020.07.003">Rationale and evidence on the use of tocilizumab in COVID-19: a systematic review</a>. Pulmonology 27:52–66.</div>
</div>
<div id="ref-CRriJXDk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1082. </div><div class="csl-right-inline">Jones G, Panova E. 2018. <a href="https://doi.org/10.1177/1759720x18798462">New insights and long-term safety of tocilizumab in rheumatoid arthritis</a>. Therapeutic Advances in Musculoskeletal Disease 10:195–199.</div>
</div>
<div id="ref-6xLhs9st" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1083. </div><div class="csl-right-inline">Saito J, Yakuwa N, Kaneko K, Takai C, Goto M, Nakajima K, Yamatani A, Murashima A. 2019. <a href="https://doi.org/10.1093/rheumatology/kez100">Tocilizumab during pregnancy and lactation: drug levels in maternal serum, cord blood, breast milk and infant serum</a>. Rheumatology 58:1505–1507.</div>
</div>
<div id="ref-IJzmtJOr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1084. </div><div class="csl-right-inline">Chen L-F, Mo Y-Q, Jing J, Ma J-D, Zheng D-H, Dai L. 2017. <a href="https://doi.org/10.1111/1756-185x.13010">Short-course tocilizumab increases risk of hepatitis B virus reactivation in patients with rheumatoid arthritis: a prospective clinical observation</a>. International Journal of Rheumatic Diseases 20:859–869.</div>
</div>
<div id="ref-SiOKIbl3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1085. </div><div class="csl-right-inline">Fu B, Xu X, Wei H. 2020. <a href="https://doi.org/10.1186/s12967-020-02339-3">Why tocilizumab could be an effective treatment for severe COVID-19?</a> Journal of Translational Medicine 18:164.</div>
</div>
<div id="ref-6Agll71m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1086. </div><div class="csl-right-inline">Campbell L, Chen C, Bhagat SS, Parker RA, Ostor AJK. 2010. <a href="https://doi.org/10.1093/rheumatology/keq343">Risk of adverse events including serious infections in rheumatoid arthritis patients treated with tocilizumab: a systematic literature review and meta-analysis of randomized controlled trials</a>. Rheumatology 50:552–562.</div>
</div>
<div id="ref-zRRSvckT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1087. </div><div class="csl-right-inline">Pawar A, Desai RJ, Solomon DH, Santiago Ortiz AJ, Gale S, Bao M, Sarsour K, Schneeweiss S, Kim SC. 2019. <a href="https://doi.org/10.1136/annrheumdis-2018-214367">Risk of serious infections in tocilizumab versus other biologic drugs in patients with rheumatoid arthritis: a multidatabase cohort study</a>. Annals of the Rheumatic Diseases 78:456–464.</div>
</div>
<div id="ref-TgV8gb30" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1088. </div><div class="csl-right-inline">Lang VR, Englbrecht M, Rech J, Nüsslein H, Manger K, Schuch F, Tony H-P, Fleck M, Manger B, Schett G, Zwerina J. 2012. <a href="https://doi.org/10.1093/rheumatology/ker223">Risk of infections in rheumatoid arthritis patients treated with tocilizumab</a>. Rheumatology 51:852–857.</div>
</div>
<div id="ref-p0tzgJvA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1089. </div><div class="csl-right-inline">Radbel J, Narayanan N, Bhatt PJ. 2020. <a href="https://doi.org/10.1016/j.chest.2020.04.024">Use of Tocilizumab for COVID-19-Induced Cytokine Release Syndrome</a>. Chest 158:e15–e19.</div>
</div>
<div id="ref-epRQ9yZu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1090. </div><div class="csl-right-inline">Tzotzos SJ, Fischer B, Fischer H, Zeitlinger M. 2020. <a href="https://doi.org/10.1186/s13054-020-03240-7">Incidence of ARDS and outcomes in hospitalized patients with COVID-19: a global literature survey</a>. Crit Care 24.</div>
</div>
<div id="ref-ZHqWXFcN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1091. </div><div class="csl-right-inline">Kaye A, Siegel R. 2020. The Efficacy of IL-6 Inhibitor Tocilizumab in Reducing Severe COVID-19 Mortality: A Systematic Review. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.07.10.20150938">10.1101/2020.07.10.20150938</a>.</div>
</div>
<div id="ref-hvOwSJvK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1092. </div><div class="csl-right-inline">Hassoun A, Thottacherry ED, Muklewicz J, Aziz Q, Edwards J. 2020. <a href="https://doi.org/10.1016/j.jcv.2020.104443">Utilizing tocilizumab for the treatment of cytokine release syndrome in COVID-19</a>. Journal of Clinical Virology 128:104443.</div>
</div>
<div id="ref-LtFtUo2P" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1093. </div><div class="csl-right-inline">Spiegel M, Pichlmair A, Mühlberger E, Haller O, Weber F. 2004. <a href="https://doi.org/10.1016/j.jcv.2003.11.013">The antiviral effect of interferon-beta against SARS-Coronavirus is not mediated by MxA protein</a>. Journal of Clinical Virology 30:211–213.</div>
</div>
<div id="ref-Nfu9kiae" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1094. </div><div class="csl-right-inline">DeDiego ML, Nieto-Torres JL, Jimenez-Guardeño JM, Regla-Nava JA, Castaño-Rodriguez C, Fernandez-Delgado R, Usera F, Enjuanes L. 2014. <a href="https://doi.org/10.1016/j.virusres.2014.07.024">Coronavirus virulence genes with main focus on SARS-CoV envelope gene</a>. Virus Research 194:124–137.</div>
</div>
<div id="ref-7k9Wp9or" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1095. </div><div class="csl-right-inline">2020. Synairgen to start trial of SNG001 in COVID-19 imminently. Synairgen plc press release. <a href="http://synairgen.web01.hosting.bdci.co.uk/umbraco/Surface/Download/GetFile?cid=23c9b12c-508b-48c3-9081-36605c5a9ccd">http://synairgen.web01.hosting.bdci.co.uk/umbraco/Surface/Download/GetFile?cid=23c9b12c-508b-48c3-9081-36605c5a9ccd</a>.</div>
</div>
<div id="ref-11hbhD5Rk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1096. </div><div class="csl-right-inline">Peiffer-Smadja N, Yazdanpanah Y. 2021. <a href="https://doi.org/10.1016/s2213-2600(20)30523-3">Nebulised interferon beta-1a for patients with COVID-19</a>. The Lancet Respiratory Medicine 9:122–123.</div>
</div>
<div id="ref-B2urVDo5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1097. </div><div class="csl-right-inline">Ranieri VM, Pettilä V, Karvonen MK, Jalkanen J, Nightingale P, Brealey D, Mancebo J, Ferrer R, Mercat A, Patroniti N, Quintel M, Vincent J-L, Okkonen M, Meziani F, Bellani G, MacCallum N, Creteur J, Kluge S, Artigas-Raventos A, Maksimow M, Piippo I, Elima K, Jalkanen S, Jalkanen M, Bellingan G. 2020. <a href="https://doi.org/10.1001/jama.2019.22525">Effect of Intravenous Interferon β-1a on Death and Days Free From Mechanical Ventilation Among Patients With Moderate to Severe Acute Respiratory Distress Syndrome</a>. JAMA 323:725.</div>
</div>
<div id="ref-xMRJgRES" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1098. </div><div class="csl-right-inline">Davoudi-Monfared E, Rahmani H, Khalili H, Hajiabdolbaghi M, Salehi M, Abbasian L, Kazemzadeh H, Yekaninejad MS. 2020. <a href="https://doi.org/10.1128/aac.01061-20">A Randomized Clinical Trial of the Efficacy and Safety of Interferon β-1a in Treatment of Severe COVID-19</a>. Antimicrob Agents Chemother 64.</div>
</div>
<div id="ref-17so9kzfj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1099. </div><div class="csl-right-inline">National Institute of Allergy and Infectious Diseases (NIAID). 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04492475">A Multicenter, Adaptive, Randomized Blinded Controlled Trial of the Safety and Efficacy of Investigational Therapeutics for the Treatment of COVID-19 in Hospitalized Adults (ACTT-3)</a>. NCT04492475. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-7fZ9eyMk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1100. </div><div class="csl-right-inline">2015. Tocilizumab (Actemra): Adult Patients with Moderately to Severely Active Rheumatoid Arthritis. Canadian Agency for Drugs and Technologies in Health, Ottawa (ON). <a href="https://www.ncbi.nlm.nih.gov/books/NBK349513/table/T43/">https://www.ncbi.nlm.nih.gov/books/NBK349513/table/T43/</a>.</div>
</div>
<div id="ref-trWLoLFz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1101. </div><div class="csl-right-inline">Hankin C, Feldman S, Szczotka A, Stinger R, Fish L, Hankin D. 2005. <a href="https://escholarship.umassmed.edu/meyers_pp/385">A Cost Comparison of Treatments of Moderate to Severe Psoriasis</a>. Drug Benefit Trends 17:200–214.</div>
</div>
<div id="ref-LDQlolY3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1102. </div><div class="csl-right-inline">Tobinick E. 2008. <a href="https://doi.org/10.1185/030079903125002757">TNF-α inhibition for potential therapeutic modulation of SARS coronavirus infection</a>. Current Medical Research and Opinion 20:39–40.</div>
</div>
<div id="ref-4DAIEbQF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1103. </div><div class="csl-right-inline">2020. Sanofi and Regeneron begin global KevzaraÂ® (sarilumab) clinical trial program in patients with severe COVID-19. Sanofi. <a href="http://www.news.sanofi.us/2020-03-16-Sanofi-and-Regeneron-begin-global-Kevzara-R-sarilumab-clinical-trial-program-in-patients-with-severe-COVID-19">http://www.news.sanofi.us/2020-03-16-Sanofi-and-Regeneron-begin-global-Kevzara-R-sarilumab-clinical-trial-program-in-patients-with-severe-COVID-19</a>.</div>
</div>
<div id="ref-amy3rDsj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1104. </div><div class="csl-right-inline">Sarilumab COVID-19 - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04327388">https://clinicaltrials.gov/ct2/show/NCT04327388</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-WF7ymA4m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1105. </div><div class="csl-right-inline">Stebbing J, Phelan A, Griffin I, Tucker C, Oechsle O, Smith D, Richardson P. 2020. <a href="https://doi.org/10.1016/s1473-3099(20)30132-8">COVID-19: combining antiviral and anti-inflammatory treatments</a>. The Lancet Infectious Diseases 20:400–402.</div>
</div>
<div id="ref-uCns3aFw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1106. </div><div class="csl-right-inline">Richardson P, Griffin I, Tucker C, Smith D, Oechsle O, Phelan A, Rawling M, Savory E, Stebbing J. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30304-4">Baricitinib as potential treatment for 2019-nCoV acute respiratory disease</a>. The Lancet 395:e30–e31.</div>
</div>
<div id="ref-jKRGxazA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1107. </div><div class="csl-right-inline">Lilly Begins Clinical Testing of Therapies for COVID-19 | Eli Lilly and Company. <a href="https://investor.lilly.com/news-releases/news-release-details/lilly-begins-clinical-testing-therapies-covid-19">https://investor.lilly.com/news-releases/news-release-details/lilly-begins-clinical-testing-therapies-covid-19</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-JCMhG5r9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1108. </div><div class="csl-right-inline">Cantini F. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04320277">Baricitinib Combined With Antiviral Therapy in Symptomatic Patients Infected by COVID-19: an Open-label, Pilot Study</a>. NCT04320277. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-Cd5uMaAr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1109. </div><div class="csl-right-inline">Biot C, Daher W, Chavain N, Fandeur T, Khalife J, Dive D, De Clercq E. 2006. <a href="https://doi.org/10.1021/jm0601856">Design and Synthesis of Hydroxyferroquine Derivatives with Antimalarial and Antiviral Activities</a>. Journal of Medicinal Chemistry 49:2845–2849.</div>
</div>
<div id="ref-TZleuwHX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1110. </div><div class="csl-right-inline">Sheahan TP, Sims AC, Zhou S, Graham RL, Pruijssers AJ, Agostini ML, Leist SR, Schäfer A, Dinnon KH, Stevens LJ, Chappell JD, Lu X, Hughes TM, George AS, Hill CS, Montgomery SA, Brown AJ, Bluemling GR, Natchus MG, Saindane M, Kolykhalov AA, Painter G, Harcourt J, Tamin A, Thornburg NJ, Swanstrom R, Denison MR, Baric RS. 2020. <a href="https://doi.org/10.1126/scitranslmed.abb5883">An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 in human airway epithelial cell cultures and multiple coronaviruses in mice</a>. Science Translational Medicine 12:eabb5883.</div>
</div>
<div id="ref-1FTnvyZlf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1111. </div><div class="csl-right-inline">Pelegrin M, Naranjo-Gomez M, Piechaczyk M. 2015. <a href="https://doi.org/10.1016/j.tim.2015.07.005">Antiviral Monoclonal Antibodies: Can They Be More Than Simple Neutralizing Agents?</a> Trends in Microbiology 23:653–665.</div>
</div>
<div id="ref-Ull2rQ5L" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1112. </div><div class="csl-right-inline">Zhao J, Wohlford-Lenane C, Zhao J, Fleming E, Lane TE, McCray PB, Perlman S. 2012. <a href="https://doi.org/10.1128/jvi.01410-12">Intranasal Treatment with Poly(I{middle dot}C) Protects Aged Mice from Lethal Respiratory Virus Infections</a>. Journal of Virology 86:11416–11424.</div>
</div>
<div id="ref-YY3x3bBV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1113. </div><div class="csl-right-inline">Plotkin S. 2014. <a href="https://doi.org/10.1073/pnas.1400472111">History of vaccination</a>. Proc Natl Acad Sci USA 111:12283–12287.</div>
</div>
<div id="ref-dqpEe5Lz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1114. </div><div class="csl-right-inline">Krammer F. 2020. <a href="https://doi.org/10.1038/s41586-020-2798-3">SARS-CoV-2 vaccines in development</a>. Nature 586:516–527.</div>
</div>
<div id="ref-sRy6js2o" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1115. </div><div class="csl-right-inline">Mathieu E, Ritchie H, Ortiz-Ospina E, Roser M, Hasell J, Appel C, Giattino C, Rodés-Guirao L. 2021. <a href="https://doi.org/10.1038/s41562-021-01122-8">A global database of COVID-19 vaccinations</a>. Nat Hum Behav 5:947–953.</div>
</div>
<div id="ref-plfPrQP7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1116. </div><div class="csl-right-inline">Corey L, Mascola JR, Fauci AS, Collins FS. 2020. <a href="https://doi.org/10.1126/science.abc5312">A strategic approach to COVID-19 vaccine R&amp;D</a>. Science 368:948–950.</div>
</div>
<div id="ref-DhPGhZPn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1117. </div><div class="csl-right-inline">Vaccine efficacy, effectiveness and protection. <a href="https://www.who.int/news-room/feature-stories/detail/vaccine-efficacy-effectiveness-and-protection">https://www.who.int/news-room/feature-stories/detail/vaccine-efficacy-effectiveness-and-protection</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-wiGjCZC8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1118. </div><div class="csl-right-inline">Jackson LA, Anderson EJ, Rouphael NG, Roberts PC, Makhene M, Coler RN, McCullough MP, Chappell JD, Denison MR, Stevens LJ, Pruijssers AJ, McDermott A, Flach B, Doria-Rose NA, Corbett KS, Morabito KM, O’Dell S, Schmidt SD, Swanson PA, Padilla M, Mascola JR, Neuzil KM, Bennett H, Sun W, Peters E, Makowski M, Albert J, Cross K, Buchanan W, Pikaart-Tautges R, Ledgerwood JE, Graham BS, Beigel JH. 2020. <a href="https://doi.org/10.1056/nejmoa2022483">An mRNA Vaccine against SARS-CoV-2 — Preliminary Report</a>. New England Journal of Medicine 383:1920–1931.</div>
</div>
<div id="ref-Q10m9bJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1119. </div><div class="csl-right-inline">Stewart AJ, Devlin PM. 2006. <a href="https://doi.org/10.1016/j.jinf.2005.07.021">The history of the smallpox vaccine</a>. Journal of Infection 52:329–334.</div>
</div>
<div id="ref-1Clt2Bek3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1120. </div><div class="csl-right-inline">Leung AKC. 2011. <a href="https://doi.org/10.1007/978-1-4419-1339-5_2">“Variolation” and Vaccination in Late Imperial China, Ca 1570–1911</a>, p. 5–12. <em>In</em> History of Vaccine Development. Springer New York.</div>
</div>
<div id="ref-ZUHALvLg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1121. </div><div class="csl-right-inline">Stern AM, Markel H. 2005. <a href="https://doi.org/10.1377/hlthaff.24.3.611">The History Of Vaccines And Immunization: Familiar Patterns, New Challenges</a>. Health Affairs 24:611–621.</div>
</div>
<div id="ref-1DFnwhtrq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1122. </div><div class="csl-right-inline">Esparza J, Schrick L, Damaso CR, Nitsche A. 2017. <a href="https://doi.org/10.1016/j.vaccine.2017.11.003">Equination (inoculation of horsepox): An early alternative to vaccination (inoculation of cowpox) and the potential role of horsepox virus in the origin of the smallpox vaccine</a>. Vaccine 35:7222–7230.</div>
</div>
<div id="ref-kC2tx3JC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1123. </div><div class="csl-right-inline">Minor PD. 2015. <a href="https://doi.org/10.1016/j.virol.2015.03.032">Live attenuated vaccines: Historical successes and current challenges</a>. Virology 479-480:379–392.</div>
</div>
<div id="ref-K0Ltu31S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1124. </div><div class="csl-right-inline">Zhang C, Maruggi G, Shan H, Li J. 2019. <a href="https://doi.org/10.3389/fimmu.2019.00594">Advances in mRNA Vaccines for Infectious Diseases</a>. Frontiers in Immunology 10:594.</div>
</div>
<div id="ref-U9ZIZWkB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1125. </div><div class="csl-right-inline">Cui Z. 2005. <a href="https://doi.org/10.1016/s0065-2660(05)54011-2">DNA Vaccine</a>, p. 257–289. <em>In</em> Non-Viral Vectors for Gene Therapy, Second Edition: Part 2. Elsevier.</div>
</div>
<div id="ref-iX8wXLPW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1126. </div><div class="csl-right-inline">Flint J, Skalka AM, Rall GF, Racaniello VR. 2015. Principles of Virology, Volume I: Molecular Biology. American Society of Microbiology. <a href="https://doi.org/gmqjck">https://doi.org/gmqjck</a>.</div>
</div>
<div id="ref-4RuaSyLg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1127. </div><div class="csl-right-inline">Sanchez-Felipe L, Vercruysse T, Sharma S, Ma J, Lemmens V, Van Looveren D, Arkalagud Javarappa MP, Boudewijns R, Malengier-Devlies B, Liesenborghs L, Kaptein SJF, De Keyzer C, Bervoets L, Debaveye S, Rasulova M, Seldeslachts L, Li L-H, Jansen S, Yakass MB, Verstrepen BE, Böszörményi KP, Kiemenyi-Kayere G, van Driel N, Quaye O, Zhang X, ter Horst S, Mishra N, Deboutte W, Matthijnssens J, Coelmont L, Vandermeulen C, Heylen E, Vergote V, Schols D, Wang Z, Bogers W, Kuiken T, Verschoor E, Cawthorne C, Van Laere K, Opdenakker G, Vande Velde G, Weynand B, Teuwen DE, Matthys P, Neyts J, Jan Thibaut H, Dallmeier K. 2020. <a href="https://doi.org/10.1038/s41586-020-3035-9">A single-dose live-attenuated YF17D-vectored SARS-CoV-2 vaccine candidate</a>. Nature 590:320–325.</div>
</div>
<div id="ref-wZ2tXSUH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1128. </div><div class="csl-right-inline">Siegrist C-A. 2018. <a href="https://doi.org/10.1016/b978-0-323-35761-6.00002-x">Vaccine Immunology</a>, p. 16–34.e7. <em>In</em> Plotkin's Vaccines. Elsevier.</div>
</div>
<div id="ref-7RHpaAHu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1129. </div><div class="csl-right-inline">Zhao J, Zhao S, Ou J, Zhang J, Lan W, Guan W, Wu X, Yan Y, Zhao W, Wu J, Chodosh J, Zhang Q. 2020. <a href="https://doi.org/10.3389/fimmu.2020.602256">COVID-19: Coronavirus Vaccine Development Updates</a>. Front Immunol 11.</div>
</div>
<div id="ref-zLL2yOJK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1130. </div><div class="csl-right-inline">Policy (OIDP) O of ID and H. 2021. Vaccine Types. HHSgov. Text. <a href="https://www.hhs.gov/immunization/basics/types/index.html">https://www.hhs.gov/immunization/basics/types/index.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-nwyfEEPl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1131. </div><div class="csl-right-inline">Marshall D. Griffith University researchers on the road to COVID-19 vaccine. <a href="https://news.griffith.edu.au/2020/04/23/griffith-university-researchers-on-the-road-to-covid-19-vaccine/">https://news.griffith.edu.au/2020/04/23/griffith-university-researchers-on-the-road-to-covid-19-vaccine/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-iCUWeMfX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1132. </div><div class="csl-right-inline">COVID-19 Treatment and Vaccine Tracker. Milken Institute. <a href="https://covid-19tracker.milkeninstitute.org/">https://covid-19tracker.milkeninstitute.org/</a>. Retrieved 5 August 2022.</div>
</div>
<div id="ref-xMxJTYge" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1133. </div><div class="csl-right-inline">Wang Y, Yang C, Song Y, Coleman JR, Stawowczyk M, Tafrova J, Tasker S, Boltz D, Baker R, Garcia L, Seale O, Kushnir A, Wimmer E, Mueller S. 2021. <a href="https://doi.org/10.1073/pnas.2102775118">Scalable live-attenuated SARS-CoV-2 vaccine candidate demonstrates preclinical safety and efficacy</a>. Proc Natl Acad Sci USA 118.</div>
</div>
<div id="ref-Gnk3ge6D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1134. </div><div class="csl-right-inline">Coleman JR, Papamichail D, Skiena S, Futcher B, Wimmer E, Mueller S. 2008. <a href="https://doi.org/10.1126/science.1155761">Virus Attenuation by Genome-Scale Changes in Codon Pair Bias</a>. Science 320:1784–1787.</div>
</div>
<div id="ref-bPMpOwp8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1135. </div><div class="csl-right-inline">Codagenix, Inc. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04619628">First-in-human, Randomised, Double-blind, Placebo-controlled, Dose-escalation Study in Healthy Young Adults Evaluating the Safety and Immunogenicity of COVI-VAC, a Live Attenuated Vaccine Candidate for Prevention of COVID-19</a>. NCT04619628. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-2VWtlcwJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1136. </div><div class="csl-right-inline">Kushnir A, Mueller S, Tasker S, Robert Coleman J. 2021. <a href="https://doi.org/10.1093/ofid/ofab466.775">577. COVI-VAC™, a Live Attenuated COVID-19 Vaccine, Provides Single Dose Protection Against Heterologous Challenge with SARS-CoV-2 Beta (B.1.351) in the Syrian Golden Hamster Model</a>. Open Forum Infect Dis 8:S390.</div>
</div>
<div id="ref-14GSJSuHC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1137. </div><div class="csl-right-inline">Gao S, Song S, Zhang L. 2019. <a href="https://doi.org/10.3389/fmicb.2019.02881">Recent Progress in Vaccine Development Against Chikungunya Virus</a>. Front Microbiol 10.</div>
</div>
<div id="ref-8ZMg94iW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1138. </div><div class="csl-right-inline">Meissa Vaccines, Inc. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04798001">Phase 1, Open-Label, Dose-Escalation Study to Evaluate Tolerability, Safety, and Immunogenicity of an Intranasal Live Attenuated Respiratory Syncytial Virus Vaccine Expressing Spike Protein of SARS-CoV-2 in Healthy Adults Ages 18 - 69 Years</a>. NCT04798001. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-yTsmmAYC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1139. </div><div class="csl-right-inline">Rando HM, Greene CS, Robson MP, Boca SM, Wellhausen N, Lordan R, Brueffer C, Ray S, McGowan LD, Gitter A, Dattoli AA, Velazquez R, Barton JP, Field JM, Ramsundar B, MacLean AL, Lee AJ, Medicine II of the IS of, Hu F, Jadavji NM, Sell E, Wang J, Rafizadeh DN, Skelly AN, Guebila MB, Kolla L, Manheim D, Ghosh S, Byrd JB, Park Y, Bansal V, Capone S, Dziak JJ, Sun Y, Qi Y, Shinholster L, Lukan T, Knyazev S, Perrin D, Mangul S, Das S, Szeto GL, Lubiana T, Mai D, Consortium CR, Goel RR. 2021. <a href="https://greenelab.github.io/covid19-review/">SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics</a>. Manubot.</div>
</div>
<div id="ref-1CJtdlM6d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1140. </div><div class="csl-right-inline">O’Neill LAJ, Netea MG. 2020. <a href="https://doi.org/10.1038/s41577-020-0337-y">BCG-induced trained immunity: can it offer protection against COVID-19?</a> Nat Rev Immunol 20:335–337.</div>
</div>
<div id="ref-eHrzWQ6D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1141. </div><div class="csl-right-inline">Moorlag SJCFM, Taks E, ten Doesschate T, van der Vaart TW, Janssen AB, Müller L, Ostermann P, Dijkstra H, Lemmers H, Simonetti E, Mazur M, Schaal H, ter Heine R, van de Veerdonk FL, Bleeker-Rovers CP, van Crevel R, ten Oever J, de Jonge MI, Bonten MJ, van Werkhoven CH, Netea MG. 2022. <a href="https://doi.org/10.1093/cid/ciac182">Efficacy of BCG Vaccination Against Respiratory Tract Infections in Older Adults During the Coronavirus Disease 2019 Pandemic</a>. Clinical Infectious Diseases 75:e938–e946.</div>
</div>
<div id="ref-9m3rP633" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1142. </div><div class="csl-right-inline">BCG Vaccination to Protect Healthcare Workers Against COVID-19 - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04327206">https://clinicaltrials.gov/ct2/show/NCT04327206</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-xdqxBruc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1143. </div><div class="csl-right-inline">BCG Vaccine for Health Care Workers as Defense Against COVID 19 - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04348370">https://clinicaltrials.gov/ct2/show/NCT04348370</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-bgKUtUIL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1144. </div><div class="csl-right-inline">Robert-Guroff M. 2007. <a href="https://doi.org/10.1016/j.copbio.2007.10.010">Replicating and non-replicating viral vectors for vaccine development</a>. Current Opinion in Biotechnology 18:546–556.</div>
</div>
<div id="ref-z1FVz33g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1145. </div><div class="csl-right-inline">Sharma S, Vercruysse T, Sanchez-Felipe L, Kerstens W, Rasulova M, Abdelnabi R, Foo CS, Lemmens V, Van Looveren D, Maes P, Baele G, Weynand B, Lemey P, Neyts J, Thibaut HJ, Dallmeier K. 2021. <a href="https://doi.org/10.1101/2021.11.12.468374">Updated vaccine protects from infection with SARS-CoV-2 variants, prevents transmission and is immunogenic against Omicron in hamsters</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-PwjPrwXa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1146. </div><div class="csl-right-inline">Barrett PN, Terpening SJ, Snow D, Cobb RR, Kistner O. 2017. <a href="https://doi.org/10.1080/14760584.2017.1357471">Vero cell technology for rapid development of inactivated whole virus vaccines for emerging viral diseases</a>. Expert Review of Vaccines 16:883–894.</div>
</div>
<div id="ref-7Knbo28h" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1147. </div><div class="csl-right-inline">Bachmann MF, Jennings GT. 2010. <a href="https://doi.org/10.1038/nri2868">Vaccine delivery: a matter of size, geometry, kinetics and molecular patterns</a>. Nat Rev Immunol 10:787–796.</div>
</div>
<div id="ref-10peSXMZx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1148. </div><div class="csl-right-inline">Roberts A, Lamirande EW, Vogel L, Jackson JP, Paddock CD, Guarner J, Zaki SR, Sheahan T, Baric R, Subbarao K. 2008. <a href="https://doi.org/10.1016/j.virusres.2007.03.025">Animal models and vaccines for SARS-CoV infection</a>. Virus Research 133:20–32.</div>
</div>
<div id="ref-iAa7uWOm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1149. </div><div class="csl-right-inline">Scherle PA, Gerhard W. 1986. <a href="https://doi.org/10.1084/jem.164.4.1114">Functional analysis of influenza-specific helper T cell clones in vivo. T cells specific for internal viral proteins provide cognate help for B cell responses to hemagglutinin.</a> Journal of Experimental Medicine 164:1114–1128.</div>
</div>
<div id="ref-xQSh0vVt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1150. </div><div class="csl-right-inline">Brüssow H. 2021. <a href="https://doi.org/10.1111/1751-7915.13818">COVID‐19: vaccine’s progress</a>. Microb Biotechnol 14:1246–1257.</div>
</div>
<div id="ref-ynMHxaPL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1151. </div><div class="csl-right-inline">Reuters. 2020. <a href="https://www.reuters.com/business/healthcare-pharmaceuticals/china-gives-its-first-covid-19-vaccine-approval-sinopharm-2020-12-31/">China gives its first COVID-19 vaccine approval to Sinopharm</a>. Reuters.</div>
</div>
<div id="ref-gwspw48F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1152. </div><div class="csl-right-inline">Wee S-L, Simões M. 2020. <a href="https://www.nytimes.com/2020/07/16/business/china-vaccine-coronavirus.html">In Coronavirus Vaccine Race, China Strays From the Official Paths</a>. The New York Times.</div>
</div>
<div id="ref-l28xUYm7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1153. </div><div class="csl-right-inline">Ball P. 2020. <a href="https://doi.org/10.1038/d41586-020-03626-1">The lightning-fast quest for COVID vaccines — and what it means for other diseases</a>. Nature 589:16–18.</div>
</div>
<div id="ref-1DymXCWa0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1154. </div><div class="csl-right-inline">Tsunetsugu-Yokota Y, Ohnishi K, Takemori T. 2006. <a href="https://doi.org/10.1002/rmv.492">Severe acute respiratory syndrome (SARS) coronavirus: application of monoclonal antibodies and development of an effective vaccine</a>. Rev Med Virol 16:117–131.</div>
</div>
<div id="ref-4Hh1wpwV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1155. </div><div class="csl-right-inline">Takasuka N. 2004. <a href="https://doi.org/10.1093/intimm/dxh143">A subcutaneously injected UV-inactivated SARS coronavirus vaccine elicits systemic humoral immunity in mice</a>. International Immunology 16:1423–1430.</div>
</div>
<div id="ref-4AwyoMvQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1156. </div><div class="csl-right-inline">Su S, Du L, Jiang S. 2020. <a href="https://doi.org/10.1038/s41579-020-00462-y">Learning from the past: development of safe and effective COVID-19 vaccines</a>. Nat Rev Microbiol 19:211–219.</div>
</div>
<div id="ref-ZXAfLbxM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1157. </div><div class="csl-right-inline">Wang Q, Zhang L, Kuwahara K, Li L, Liu Z, Li T, Zhu H, Liu J, Xu Y, Xie J, Morioka H, Sakaguchi N, Qin C, Liu G. 2016. <a href="https://doi.org/10.1021/acsinfecdis.6b00006">Immunodominant SARS Coronavirus Epitopes in Humans Elicited both Enhancing and Neutralizing Effects on Infection in Non-human Primates</a>. ACS Infect Dis 2:361–376.</div>
</div>
<div id="ref-ihrfEtMq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1158. </div><div class="csl-right-inline">Agrawal AS, Tao X, Algaissi A, Garron T, Narayanan K, Peng B-H, Couch RB, Tseng C-TK. 2016. <a href="https://doi.org/10.1080/21645515.2016.1177688">Immunization with inactivated Middle East Respiratory Syndrome coronavirus vaccine leads to lung immunopathology on challenge with live virus</a>. Human Vaccines &amp; Immunotherapeutics 12:2351–2356.</div>
</div>
<div id="ref-8qw9OBKX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1159. </div><div class="csl-right-inline">Li K, Li Z, Wohlford-Lenane C, Meyerholz DK, Channappanavar R, An D, Perlman S, McCray PB Jr., He B. 2020. <a href="https://doi.org/10.1128/mbio.00554-20">Single-Dose, Intranasal Immunization with Recombinant Parainfluenza Virus 5 Expressing Middle East Respiratory Syndrome Coronavirus (MERS-CoV) Spike Protein Protects Mice from Fatal MERS-CoV Infection</a>. mBio 11.</div>
</div>
<div id="ref-jswAyWIs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1160. </div><div class="csl-right-inline">Types of Vaccines – COVID19 Vaccine Tracker. <a href="https://covid19.trackvaccines.org/types-of-vaccines/">https://covid19.trackvaccines.org/types-of-vaccines/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-iGEyNO42" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1161. </div><div class="csl-right-inline">Jordahl K, Van Den Bossche J, Fleischmann M, McBride J, Wasserman J, Badaracco AG, Gerard J, Snow AD, Tratner J, Perry M, Farmer C, Hjelle GA, Cochran M, Gillies S, Culbertson L, Bartos M, Ward B, Caria G, Taves M, Eubank N, Sangarshanan, Flavin J, Richards M, Rey S, Maxalbert, Bilogur A, Ren C, Arribas-Bel D, Mesejo-León D, Wasser L. 2021. geopandas/geopandas: v0.10.2 (v0.10.2). Zenodo. <a href="https://doi.org/gqkzpv">https://doi.org/gqkzpv</a>.</div>
</div>
<div id="ref-Ozya5HP5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1162. </div><div class="csl-right-inline">Wu Z, Hu Y, Xu M, Chen Z, Yang W, Jiang Z, Li M, Jin H, Cui G, Chen P, Wang L, Zhao G, Ding Y, Zhao Y, Yin W. 2021. <a href="https://doi.org/10.1016/s1473-3099(20)30987-7">Safety, tolerability, and immunogenicity of an inactivated SARS-CoV-2 vaccine (CoronaVac) in healthy adults aged 60 years and older: a randomised, double-blind, placebo-controlled, phase 1/2 clinical trial</a>. The Lancet Infectious Diseases 21:803–812.</div>
</div>
<div id="ref-14fILrRWg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1163. </div><div class="csl-right-inline">Gao Q, Bao L, Mao H, Wang L, Xu K, Yang M, Li Y, Zhu L, Wang N, Lv Z, Gao H, Ge X, Kan B, Hu Y, Liu J, Cai F, Jiang D, Yin Y, Qin C, Li J, Gong X, Lou X, Shi W, Wu D, Zhang H, Zhu L, Deng W, Li Y, Lu J, Li C, Wang X, Yin W, Zhang Y, Qin C. 2020. <a href="https://doi.org/10.1126/science.abc1932">Development of an inactivated vaccine candidate for SARS-CoV-2</a>. Science 369:77–81.</div>
</div>
<div id="ref-N1txjPtt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1164. </div><div class="csl-right-inline">Zhang Y, Zeng G, Pan H, Li C, Hu Y, Chu K, Han W, Chen Z, Tang R, Yin W, Chen X, Hu Y, Liu X, Jiang C, Li J, Yang M, Song Y, Wang X, Gao Q, Zhu F. 2021. <a href="https://doi.org/10.1016/s1473-3099(20)30843-4">Safety, tolerability, and immunogenicity of an inactivated SARS-CoV-2 vaccine in healthy adults aged 18–59 years: a randomised, double-blind, placebo-controlled, phase 1/2 clinical trial</a>. The Lancet Infectious Diseases 21:181–192.</div>
</div>
<div id="ref-1A5wiKQAW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1165. </div><div class="csl-right-inline">Background document on the inactivated vaccine Sinovac-CoronaVac against COVID-19. <a href="https://www.who.int/publications-detail-redirect/WHO-2019-nCoV-vaccines-SAGE_recommendation-Sinovac-CoronaVac-background-2021.1">https://www.who.int/publications-detail-redirect/WHO-2019-nCoV-vaccines-SAGE_recommendation-Sinovac-CoronaVac-background-2021.1</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1FF7JOwSH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1166. </div><div class="csl-right-inline">Tanriover MD, Doğanay HL, Akova M, Güner HR, Azap A, Akhan S, Köse Ş, Erdinç FŞ, Akalın EH, Tabak ÖF, Pullukçu H, Batum Ö, Şimşek Yavuz S, Turhan Ö, Yıldırmak MT, Köksal İ, Taşova Y, Korten V, Yılmaz G, Çelen MK, Altın S, Çelik İ, Bayındır Y, Karaoğlan İ, Yılmaz A, Özkul A, Gür H, Unal S, Kayaaslan B, Hasanoğlu İ, Dalkıran A, Aydos Ö, Çınar G, Akdemir-Kalkan İ, İnkaya AÇ, Aydin M, Çakir H, Yıldız J, Kocabıyık Ö, Arslan S, Nallı B, Demir Ö, Singil S, Ataman-Hatipoğlu Ç, Tuncer-Ertem G, Kınıklı S, Önal U, Mete B, Dalgan G, Taşbakan M, Yamazhan T, Kömürcüoğlu B, Yalnız E, Benli A, Keskin-Sarıtaş Ç, Ertosun MG, Özkan Ö, Emre S, Arıca S, Kuşçu F, Candevir A, Ertürk-Şengel B, Ayvaz F, Aksoy F, Mermutluoğlu Ç, Demir Y, Günlüoğlu G, Tural-Önür S, Kılıç-Toker A, Eren E, Otlu B, Mete AÖ, Koçak K, Ateş H, Koca-Kalkan İ, Aksu K. 2021. <a href="https://doi.org/10.1016/s0140-6736(21)01429-x">Efficacy and safety of an inactivated whole-virion SARS-CoV-2 vaccine (CoronaVac): interim results of a double-blind, randomised, placebo-controlled, phase 3 trial in Turkey</a>. The Lancet 398:213–222.</div>
</div>
<div id="ref-UERG6dAd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1167. </div><div class="csl-right-inline">Bueno SM, Abarca K, González PA, Gálvez NM, Soto JA, Duarte LF, Schultz BM, Pacheco GA, González LA, Vázquez Y, Ríos M, Melo-González F, Rivera-Pérez D, Iturriaga C, Urzúa M, Dominguez A, Andrade CA, Berrios RV, Canedo-Marroquín G, Covián C, Moreno-Tapia D, Saavedra F, Vallejos OP, Donato P, Espinoza P, Fuentes D, González M, Guzmán P, Muñoz-Venturelli P, Pérez CM, Potin M, Rojas A, Fasce R, Fernández J, Mora J, Ramírez E, Gaete-Argel A, Oyarzún-Arrau A, Valiente-Echeverría F, Soto-Rifo R, Weiskopf D, Sette A, Zeng G, Meng W, González-Aramundiz JV, Kalergis AM. 2021. <a href="https://doi.org/10.1101/2021.03.31.21254494">Interim report: Safety and immunogenicity of an inactivated vaccine against SARS-CoV-2 in healthy chilean adults in a phase 3 clinical trial</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-18mREgqUz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1168. </div><div class="csl-right-inline">2020. <a href="https://www.reuters.com/article/us-health-coronavirus-sinovac-brazil-idUSKBN28X2CR">Brazil institute says CoronaVac efficacy above 50%, but delays full results</a>. Reuters.</div>
</div>
<div id="ref-V7MXd4X4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1169. </div><div class="csl-right-inline">Zimmer C, Londoño E. 2020. <a href="https://www.nytimes.com/2020/12/25/health/turkey-brazil-sinovac-coronavirus-vaccine.html">Turkey and Brazil Say Chinese Vaccine Effective, With Sparse Supporting Data</a>. The New York Times.</div>
</div>
<div id="ref-q2wQJULu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1170. </div><div class="csl-right-inline">Jara A, Undurraga EA, González C, Paredes F, Fontecilla T, Jara G, Pizarro A, Acevedo J, Leo K, Leon F, Sans C, Leighton P, Suárez P, García-Escorza H, Araos R. 2021. <a href="https://doi.org/10.1056/nejmoa2107715">Effectiveness of an Inactivated SARS-CoV-2 Vaccine in Chile</a>. N Engl J Med 385:875–884.</div>
</div>
<div id="ref-wByD9WaX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1171. </div><div class="csl-right-inline">Zimmer C, Corum J, Wee S-L, Kristoffersen M. 2020. <a href="https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html">Coronavirus Vaccine Tracker</a>. The New York Times.</div>
</div>
<div id="ref-miMRIMwa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1172. </div><div class="csl-right-inline">Kyriakidis NC, López-Cortés A, González EV, Grimaldos AB, Prado EO. 2021. <a href="https://doi.org/10.1038/s41541-021-00292-w">SARS-CoV-2 vaccines strategies: a comprehensive review of phase 3 candidates</a>. npj Vaccines 6.</div>
</div>
<div id="ref-VlnLw2HV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1173. </div><div class="csl-right-inline">Wang H, Zhang Y, Huang B, Deng W, Quan Y, Wang W, Xu W, Zhao Y, Li N, Zhang J, Liang H, Bao L, Xu Y, Ding L, Zhou W, Gao H, Liu J, Niu P, Zhao L, Zhen W, Fu H, Yu S, Zhang Z, Xu G, Li C, Lou Z, Xu M, Qin C, Wu G, Gao GF, Tan W, Yang X. 2020. <a href="https://doi.org/10.1016/j.cell.2020.06.008">Development of an Inactivated Vaccine Candidate, BBIBP-CorV, with Potent Protection against SARS-CoV-2</a>. Cell 182:713–721.e9.</div>
</div>
<div id="ref-T3MYavsH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1174. </div><div class="csl-right-inline">Xia S, Duan K, Zhang Y, Zhao D, Zhang H, Xie Z, Li X, Peng C, Zhang Y, Zhang W, Yang Y, Chen W, Gao X, You W, Wang X, Wang Z, Shi Z, Wang Y, Yang X, Zhang L, Huang L, Wang Q, Lu J, Yang Y, Guo J, Zhou W, Wan X, Wu C, Wang W, Huang S, Du J, Meng Z, Pan A, Yuan Z, Shen S, Guo W, Yang X. 2020. <a href="https://doi.org/10.1001/jama.2020.15543">Effect of an Inactivated Vaccine Against SARS-CoV-2 on Safety and Immunogenicity Outcomes</a>. JAMA 324:951.</div>
</div>
<div id="ref-fPGgVKYL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1175. </div><div class="csl-right-inline">Xia S, Zhang Y, Wang Y, Wang H, Yang Y, Gao GF, Tan W, Wu G, Xu M, Lou Z, Huang W, Xu W, Huang B, Wang H, Wang W, Zhang W, Li N, Xie Z, Ding L, You W, Zhao Y, Yang X, Liu Y, Wang Q, Huang L, Yang Y, Xu G, Luo B, Wang W, Liu P, Guo W, Yang X. 2021. <a href="https://doi.org/10.1016/s1473-3099(20)30831-8">Safety and immunogenicity of an inactivated SARS-CoV-2 vaccine, BBIBP-CorV: a randomised, double-blind, placebo-controlled, phase 1/2 trial</a>. The Lancet Infectious Diseases 21:39–51.</div>
</div>
<div id="ref-yN5KOfvE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1176. </div><div class="csl-right-inline">Al Kaabi N, Zhang Y, Xia S, Yang Y, Al Qahtani MM, Abdulrazzaq N, Al Nusair M, Hassany M, Jawad JS, Abdalla J, Hussein SE, Al Mazrouei SK, Al Karam M, Li X, Yang X, Wang W, Lai B, Chen W, Huang S, Wang Q, Yang T, Liu Y, Ma R, Hussain ZM, Khan T, Saifuddin Fasihuddin M, You W, Xie Z, Zhao Y, Jiang Z, Zhao G, Zhang Y, Mahmoud S, ElTantawy I, Xiao P, Koshy A, Zaher WA, Wang H, Duan K, Pan A, Yang X. 2021. Effect of 2 Inactivated SARS-CoV-2 Vaccines on Symptomatic COVID-19 Infection in Adults. JAMA https://doi.org/<a href="https://doi.org/10.1001/jama.2021.8565">10.1001/jama.2021.8565</a>.</div>
</div>
<div id="ref-w7gO6yGn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1177. </div><div class="csl-right-inline">2020. <a href="https://www.reuters.com/article/health-coronavirus-china-vaccine-int-idUSKBN2940CA">Sinopharm's COVID-19 vaccine 79% effective, seeks approval in China</a>. Reuters.</div>
</div>
<div id="ref-bcGxW9fA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1178. </div><div class="csl-right-inline">Mohandas S, Yadav PD, Shete A, Abraham P, Mohan K, Sapkal G, Mote C, Nyayanit D, Gupta N, Srini VK, Kadam M, Kumar A, Jain R, Majumdar T, Deshpande G, Patil S, Sarkale P, Patil D, Ella R, Prasad SD, Sharma S, Ella KM, Panda S, Bhargava B. 2020. <a href="https://doi.org/10.21203/rs.3.rs-76768/v1">Immunogenicity and protective efficacy of BBV152: a whole virion inactivated SARS CoV-2 vaccine in the Syrian hamster model</a>. Research Square Platform LLC.</div>
</div>
<div id="ref-GgdjKrYi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1179. </div><div class="csl-right-inline">Yadav PD, Ella R, Kumar S, Patil DR, Mohandas S, Shete AM, Vadrevu KM, Bhati G, Sapkal G, Kaushal H, Patil S, Jain R, Deshpande G, Gupta N, Agarwal K, Gokhale M, Mathapati B, Metkari S, Mote C, Nyayanit D, Patil DY, Sai Prasad BS, Suryawanshi A, Kadam M, Kumar A, Daigude S, Gopale S, Majumdar T, Mali D, Sarkale P, Baradkar S, Gawande P, Joshi Y, Fulari S, Dighe H, Sharma S, Gunjikar R, Kumar A, Kalele K, Srinivas VK, Gangakhedkar RR, Ella KM, Abraham P, Panda S, Bhargava B. 2021. <a href="https://doi.org/10.1038/s41467-021-21639-w">Immunogenicity and protective efficacy of inactivated SARS-CoV-2 vaccine candidate, BBV152 in rhesus macaques</a>. Nat Commun 12.</div>
</div>
<div id="ref-CGuGeB7m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1180. </div><div class="csl-right-inline">Ella R, Vadrevu KM, Jogdand H, Prasad S, Reddy S, Sarangi V, Ganneru B, Sapkal G, Yadav P, Abraham P, Panda S, Gupta N, Reddy P, Verma S, Kumar Rai S, Singh C, Redkar SV, Gillurkar CS, Kushwaha JS, Mohapatra S, Rao V, Guleria R, Ella K, Bhargava B. 2021. <a href="https://doi.org/10.1016/s1473-3099(20)30942-7">Safety and immunogenicity of an inactivated SARS-CoV-2 vaccine, BBV152: a double-blind, randomised, phase 1 trial</a>. The Lancet Infectious Diseases 21:637–646.</div>
</div>
<div id="ref-GxQSMH5l" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1181. </div><div class="csl-right-inline">Ella R, Reddy S, Jogdand H, Sarangi V, Ganneru B, Prasad S, Das D, Raju D, Praturi U, Sapkal G, Yadav P, Reddy P, Verma S, Singh C, Redkar SV, Gillurkar CS, Kushwaha JS, Mohapatra S, Bhate A, Rai S, Panda S, Abraham P, Gupta N, Ella K, Bhargava B, Vadrevu KM. 2021. <a href="https://doi.org/10.1016/s1473-3099(21)00070-0">Safety and immunogenicity of an inactivated SARS-CoV-2 vaccine, BBV152: interim results from a double-blind, randomised, multicentre, phase 2 trial, and 3-month follow-up of a double-blind, randomised phase 1 trial</a>. The Lancet Infectious Diseases 21:950–961.</div>
</div>
<div id="ref-n7BupOQ6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1182. </div><div class="csl-right-inline">Ella R, Reddy S, Blackwelder W, Potdar V, Yadav P, Sarangi V, Aileni VK, Kanungo S, Rai S, Reddy P, Verma S, Singh C, Redkar S, Mohapatra S, Pandey A, Ranganadin P, Gumashta R, Multani M, Mohammad S, Bhatt P, Kumari L, Sapkal G, Gupta N, Abraham P, Panda S, Prasad S, Bhargava B, Ella K, Vadrevu KM. 2021. <a href="https://doi.org/10.1101/2021.06.30.21259439">Efficacy, safety, and lot to lot immunogenicity of an inactivated SARS-CoV-2 vaccine (BBV152): a, double-blind, randomised, controlled phase 3 trial</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-19tYVbg8H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1183. </div><div class="csl-right-inline">Bharat Biotech: Covaxin – COVID19 Vaccine Tracker. <a href="https://covid19.trackvaccines.org/vaccines/9/">https://covid19.trackvaccines.org/vaccines/9/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-aq22z8M7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1184. </div><div class="csl-right-inline">Corum J, Zimmer C. 2020. <a href="https://www.nytimes.com/interactive/2020/health/sinovac-covid-19-vaccine.html">How the Sinovac Vaccine Works</a>. The New York Times.</div>
</div>
<div id="ref-d09adg1G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1185. </div><div class="csl-right-inline">Zraick K. 2020. <a href="https://www.nytimes.com/live/2020/12/16/world/covid-19-coronavirus">Officials Stress That the Pandemic ‘Is Not Over Yet’ as U.S. Vaccinations Begin</a>. The New York Times.</div>
</div>
<div id="ref-nxEJTLGU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1186. </div><div class="csl-right-inline">Universidad Peruana Cayetano Heredia. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04612972">Ensayo Clínico de Fase III, Aleatorio, Doble Ciego y Controlado Con Placebo Paralelo, Para Evaluar la Seguridad y la Eficacia Protectora de la Vacuna Inactivada Contra el SARS-CoV-2 en la Población Sana de 18 años o más, en Perú</a>. NCT04612972. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-1Fybz4oKI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1187. </div><div class="csl-right-inline">Moir NL. 2020. <a href="https://doi.org/10.1163/25895893-bja10009">To Boldly Remember Where We Have Already Been</a>. J Appl Hist 2:17–35.</div>
</div>
<div id="ref-19hWVUQQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1188. </div><div class="csl-right-inline">2022. Suspension of supply of COVID-19 vaccine (COVAXIN®). WHO - Prequalification of Medical Products (IVDs, Medicines, Vaccines and Immunization Devices, Vector Control). <a href="https://extranet.who.int/pqweb/vaccines/suspension-supply-covid-19-vaccine-covaxin">https://extranet.who.int/pqweb/vaccines/suspension-supply-covid-19-vaccine-covaxin</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-14KuNOjDi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1189. </div><div class="csl-right-inline">Tunjungputri RN, Tetrasiwi EN, Veronica M, Pandelaki J, Ibrahim F, Nelwan EJ. 2021. <a href="https://doi.org/10.1155/2021/9673453">Vaccine-Associated Disease Enhancement (VADE): Considerations in Postvaccination COVID-19</a>. Case Reports in Medicine 2021:1–5.</div>
</div>
<div id="ref-s2O5iyCV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1190. </div><div class="csl-right-inline">Wang G-L, Wang Z-Y, Duan L-J, Meng Q-C, Jiang M-D, Cao J, Yao L, Zhu K-L, Cao W-C, Ma M-J. 2021. <a href="https://doi.org/10.1056/nejmc2103022">Susceptibility of Circulating SARS-CoV-2 Variants to Neutralization</a>. N Engl J Med 384:2354–2356.</div>
</div>
<div id="ref-lr7INjf6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1191. </div><div class="csl-right-inline">Yadav PD, Sahay RR, Sapkal G, Nyayanit D, Shete AM, Deshpande G, Patil DY, Gupta N, Kumar S, Abraham P, Panda S, Bhargava B. 2021. <a href="https://doi.org/10.1101/2021.07.30.454511">Comparable neutralization of SARS-CoV-2 Delta AY.1 and Delta in individuals sera vaccinated with BBV152</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-UYE3NvU4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1192. </div><div class="csl-right-inline">Yadav PD, Sapkal GN, Ella R, Sahay RR, Nyayanit DA, Patil DY, Deshpande G, Shete AM, Gupta N, Mohan VK, Abraham P, Panda S, Bhargava B. 2021. <a href="https://doi.org/10.1093/jtm/taab104">Neutralization of Beta and Delta variant with sera of COVID-19 recovered cases and vaccinees of inactivated COVID-19 vaccine BBV152/Covaxin</a>. Journal of Travel Medicine 28.</div>
</div>
<div id="ref-IvOEe7bV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1193. </div><div class="csl-right-inline">Sapkal GN, Yadav PD, Ella R, Deshpande GR, Sahay RR, Gupta N, Vadrevu KM, Abraham P, Panda S, Bhargava B. 2021. <a href="https://doi.org/10.1093/jtm/taab051">Inactivated COVID-19 vaccine BBV152/COVAXIN effectively neutralizes recently emerged B.1.1.7 variant of SARS-CoV-2</a>. Journal of Travel Medicine 28.</div>
</div>
<div id="ref-1DD2pssEJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1194. </div><div class="csl-right-inline">Ella R, Reddy S, Blackwelder W, Potdar V, Yadav P, Sarangi V, Aileni VK, Kanungo S, Rai S, Reddy P, Verma S, Singh C, Redkar S, Mohapatra S, Pandey A, Ranganadin P, Gumashta R, Multani M, Mohammad S, Bhatt P, Kumari L, Sapkal G, Gupta N, Abraham P, Panda S, Prasad S, Bhargava B, Ella K, Vadrevu KM, Aggarwal P, Aglawe V, Ali A, Anand N, Awad N, Bafna V, Balasubramaniyam G, Bandkar A, Basha P, Bharge V, Bhate A, Bhate S, Bhavani V, Bhosale R, Chalapathy D, Chaubal C, Chaudhary D, Chavan A, Desai P, Dhodi D, Dutta S, Garg R, Garg K, George M, Goyal P, Guleria R, Gupta S, Jain M, Jain MK, Jindal S, Kalra M, Kant S, Khosla P, Kulkarni P, Kumar P, Kumar Y, Majumdar A, Meshram P, Mishra V, Mohanty S, Nair J, Pandey S, Panigrahi SK, Patil B, Patil V, Rahate P, Raj V, Ramanand S, Rami K, Ramraj B, Rane S, Rao EV, Rao N, Raphael R, Reddy G, Redkar V, Redkar S, Sachdeva A, Saha J, Sahoo J, Sampath P, Savith A, Shah M, Shanmugam L, Sharma R, Sharma P, Sharma D, Singh A, Singh J, Singh P, Sivaprakasam S, Subramaniam S, Sudheer D, Tandon S, Tariq M, Tripathi V, Vable M, Verma R, Waghmare S. 2021. <a href="https://doi.org/10.1016/s0140-6736(21)02000-6">Efficacy, safety, and lot-to-lot immunogenicity of an inactivated SARS-CoV-2 vaccine (BBV152): interim results of a randomised, double-blind, controlled, phase 3 trial</a>. The Lancet 398:2173–2184.</div>
</div>
<div id="ref-Pir4RpLb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1195. </div><div class="csl-right-inline">Yadav PD, Sahay RR, Sapkal G, Nyayanit D, Shete AM, Deshpande G, Patil DY, Gupta N, Kumar S, Abraham P, Panda S, Bhargava B. 2021. <a href="https://doi.org/10.1093/jtm/taab154">Comparable neutralization of SARS-CoV-2 Delta AY.1 and Delta with individuals sera vaccinated with BBV152</a>. Journal of Travel Medicine 28.</div>
</div>
<div id="ref-r5HwkOx8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1196. </div><div class="csl-right-inline">Edara V-V, Patel M, Suthar MS. 2022. <a href="https://doi.org/10.1101/2022.01.24.22269189">Covaxin (BBV152) Vaccine Neutralizes SARS-CoV-2 Delta and Omicron variants</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-CqXCzqeS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1197. </div><div class="csl-right-inline">Yadav PD, Sapkal GN, Sahay RR, Patil DY, Deshpande GR, Jain R, Nyayanit DA, Shete AM, Suryawanshi A, Nandapurkar A, Gurav YK, Abraham P. 2022. <a href="https://doi.org/10.1016/j.jinf.2022.03.016">Elevated neutralization of Omicron with sera of COVID-19 recovered and breakthrough cases vaccinated with Covaxin than two dose naïve vaccinees</a>. Journal of Infection 84:834–872.</div>
</div>
<div id="ref-w9KwrmQT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1198. </div><div class="csl-right-inline">Huang B, Dai L, Wang H, Hu Z, Yang X, Tan W, Gao GF. 2021. <a href="https://doi.org/10.1101/2021.02.01.429069">Neutralization of SARS-CoV-2 VOC 501Y.V2 by human antisera elicited by both inactivated BBIBP-CorV and recombinant dimeric RBD ZF2001 vaccines</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-GZ5Sf8Yd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1199. </div><div class="csl-right-inline">Jeewandara C, Aberathna IS, Pushpakumara PD, Kamaladasa A, Guruge D, Jayathilaka D, Gunasekara B, Tanussiya S, Kuruppu H, Ranasinghe T, Dayarathne S, Dissanayake O, Gamalath N, Ekanayake D, Jayamali M, Wijesinghe A, Dissanayake M, Madusanka D, Jayadas TT, Mudunkotuwa A, Somathilake G, Harvie M, Nimasha T, Danasekara S, Wijayamuni R, Schimanski L, Tan TK, Dong T, Townsend A, Ogg GS, Malavige GN. 2021. <a href="https://doi.org/10.1101/2021.07.15.21260621">Antibody and T cell responses to Sinopharm/BBIBP-CorV in naïve and previously infected individuals in Sri Lanka</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-fs7G9HyV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1200. </div><div class="csl-right-inline">Wee S-L. 2021. <a href="https://www.nytimes.com/2021/06/22/business/economy/china-vaccines-covid-outbreak.html">They Relied on Chinese Vaccines. Now They’re Battling Outbreaks.</a> The New York Times.</div>
</div>
<div id="ref-1GcPxd9Bn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1201. </div><div class="csl-right-inline">Sinovac Research and Development Co., Ltd. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04962308">An Open-label,Phase Ⅳ Clinical Trial to Evaluate the Immunogenicity and Safety of the Inactivated SARS-CoV-2 Vaccine (Vero Cell) in Healthy Population Aged From 18 to 59 Years.</a> NCT04962308. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-s1TGwKbT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1202. </div><div class="csl-right-inline">2021. Covaxin booster dose: What is it? What does govt say about this? Hindustan Times. <a href="https://www.hindustantimes.com/india-news/covaxin-booster-dose-what-is-it-what-does-govt-say-about-this-101625644184446.html">https://www.hindustantimes.com/india-news/covaxin-booster-dose-what-is-it-what-does-govt-say-about-this-101625644184446.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-AcxNwvVQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1203. </div><div class="csl-right-inline">Li M, Yang J, Wang L, Wu Q, Wu Z, Zheng W, Wang L, Lu W, Deng X, Peng C, Han B, Zhao Y, Yu H, Yin W. 2021. <a href="https://doi.org/10.1101/2021.08.03.21261544">A booster dose is immunogenic and will be needed for older adults who have completed two doses vaccination with CoronaVac: a randomised, double-blind, placebo-controlled, phase 1/2 clinical trial</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-1BPnaMPs4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1204. </div><div class="csl-right-inline">Pan H, Wu Q, Zeng G, Yang J, Jiang D, Deng X, Chu K, Zheng W, Zhu F, Yu H, Yin W. 2021. <a href="https://doi.org/10.1101/2021.07.23.21261026">Immunogenicity and safety of a third dose, and immune persistence of CoronaVac vaccine in healthy adults aged 18-59 years: interim results from a double-blind, randomized, placebo-controlled phase 2 clinical trial</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-QHtyW0Jz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1205. </div><div class="csl-right-inline">Liu Y, Zeng Q, Deng C, Li M, Li L, Liu D, Liu M, Ruan X, Mei J, Mo R, Zhou Q, Liu M, Peng S, Wang J, Zhang H, Xiao H. 2021. <a href="https://doi.org/10.1101/2021.09.12.21263373">Robust induction of B cell and T cell responses by a third dose of inactivated SARS-CoV-2 vaccine</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-9oJ3sbrk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1206. </div><div class="csl-right-inline">China approves first mixed-vaccine trial as Delta spreads. <a href="https://medicalxpress.com/news/2021-08-china-mixed-vaccine-trial-delta.html">https://medicalxpress.com/news/2021-08-china-mixed-vaccine-trial-delta.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-uPt61a0E" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1207. </div><div class="csl-right-inline">Araos R. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04992182">Reactogenicidad, Seguridad e Inmunogenicidad de Dosis de Refuerzo de Vacunas Contra SARS-CoV-2 en Chile (Estudio REFUERZO)</a>. NCT04992182. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-hcnFsRig" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1208. </div><div class="csl-right-inline">Li J, Hou L, Guo X, Jin P, Wu S, Zhu J, Pan H, Wang X, Song Z, Wan J, Cui L, Li J, Chen Y, Wang X, Jin L, Liu J, Shi F, Xu X, Zhu T, Chen W, Zhu F. 2022. <a href="https://doi.org/10.1038/s41591-021-01677-z">Heterologous AD5-nCOV plus CoronaVac versus homologous CoronaVac vaccination: a randomized phase 4 trial</a>. Nat Med 28:401–409.</div>
</div>
<div id="ref-1Cd41ucny" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1209. </div><div class="csl-right-inline">Pérez-Then E, Lucas C, Monteiro VS, Miric M, Brache V, Cochon L, Vogels CBF, Malik AA, De la Cruz E, Jorge A, De los Santos M, Leon P, Breban MI, Billig K, Yildirim I, Pearson C, Downing R, Gagnon E, Muyombwe A, Razeq J, Campbell M, Ko AI, Omer SB, Grubaugh ND, Vermund SH, Iwasaki A. 2022. <a href="https://doi.org/10.1038/s41591-022-01705-6">Neutralizing antibodies against the SARS-CoV-2 Delta and Omicron variants following heterologous CoronaVac plus BNT162b2 booster vaccination</a>. Nat Med 28:481–485.</div>
</div>
<div id="ref-wN3MgkJc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1210. </div><div class="csl-right-inline">The Valneva VLA2001 COVID-19 vaccine: What you need to know. <a href="https://www.who.int/news-room/feature-stories/detail/the-valneva-vla2001-covid-19-vaccine--what-you-need-to-know">https://www.who.int/news-room/feature-stories/detail/the-valneva-vla2001-covid-19-vaccine--what-you-need-to-know</a>. Retrieved 1 December 2022.</div>
</div>
<div id="ref-Sis3GVtq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1211. </div><div class="csl-right-inline">The Bharat Biotech BBV152 COVAXIN vaccine against COVID-19: What you need to know. <a href="https://www.who.int/news-room/feature-stories/detail/the-bharat-biotech-bbv152-covaxin-vaccine-against-covid-19-what-you-need-to-know">https://www.who.int/news-room/feature-stories/detail/the-bharat-biotech-bbv152-covaxin-vaccine-against-covid-19-what-you-need-to-know</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-huSLgEcW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1212. </div><div class="csl-right-inline">Is the Sinopharm BIBP (Covilo) COVID-19 vaccine safe and effective? | COVID-19 Info Vaccines. <a href="https://www.covid19infovaccines.com/en-posts/is-the-sinopharm-bibp-covilo-covid-19-vaccine-safe-and-effective">https://www.covid19infovaccines.com/en-posts/is-the-sinopharm-bibp-covilo-covid-19-vaccine-safe-and-effective</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-QTUuPhrc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1213. </div><div class="csl-right-inline">COVID-19 Vaccine (Vero Cell), Inactivated (Sinopharm). World Health Organization COVID-19 Vaccine Explainer. <a href="https://cdn.who.int/media/docs/default-source/immunization/covid-19/16-june-22080-sinopharm-vaccine-explainer-update.pdf">https://cdn.who.int/media/docs/default-source/immunization/covid-19/16-june-22080-sinopharm-vaccine-explainer-update.pdf</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-nc4lLcIg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1214. </div><div class="csl-right-inline">The Sinovac-CoronaVac COVID-19 vaccine: What you need to know. <a href="https://www.who.int/news-room/feature-stories/detail/the-sinovac-covid-19-vaccine-what-you-need-to-know">https://www.who.int/news-room/feature-stories/detail/the-sinovac-covid-19-vaccine-what-you-need-to-know</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-QWRc7Ob" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1215. </div><div class="csl-right-inline">Xiao C, Su J, Zhang C, Huang B, Mao L, Ren Z, Bai W, Li H, Lei G, Zheng J, Chen G, Liang X, Qiu C. 2022. <a href="https://doi.org/10.3390/v14092016">Effectiveness of Booster Doses of the SARS-CoV-2 Inactivated Vaccine KCONVAC against the Mutant Strains</a>. Viruses 14:2016.</div>
</div>
<div id="ref-DuPE0SrS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1216. </div><div class="csl-right-inline">Sattwika PD. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT05172193">A Phase II Non-Randomized Open Labelled Clinical Trial to Evaluate the Safety &amp; Immunogenicity of SARS-COV-2 Vaccine (Vero Cell) Inactivated as A Booster Dose</a>. NCT05172193. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-keNQQxsq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1217. </div><div class="csl-right-inline">China National Biotec Group Company Limited. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04510207">Multicenter, Randomized, Double Blind, Parallel Placebo Controlled, Phase III Clinical Trial to Evaluate the Protective Efficacy, Safety and Immunogenicity of Inactivated SARS-CoV-2 Vaccines (Vero Cell) in Healthy Population Aged 18 Years Old and Above</a>. NCT04510207. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-20kWCMZS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1218. </div><div class="csl-right-inline">Omma A, Batirel A, Aydin M, Yilmaz Karadag F, Erden A, Kucuksahin O, Armagan B, Güven SC, Karakas O, Gokdemir S, Altunal LN, Buber AA, Gemcioglu E, Zengin O, Inan O, Sahiner ES, Korukluoglu G, Sezer Z, Ozdarendeli A, Kara A, Ates I. 2022. Safety and immunogenicity of inactive vaccines as booster doses for COVID-19 in Türkiye: A randomized trial. Human Vaccines &amp;amp; Immunotherapeutics https://doi.org/<a href="https://doi.org/10.1080/21645515.2022.2122503">10.1080/21645515.2022.2122503</a>.</div>
</div>
<div id="ref-VYzX6ru8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1219. </div><div class="csl-right-inline">Ahi M, Hamidi Farahani R, Basiri P, Karimi Rahjerdi A, Sheidaei A, Gohari K, Rahimi Z, Gholami F, Moradi M, Ghafoori Naeeni F, Saffar KN, Ghasemi S, Barati B, Moradi S, Monazah A, Pouranvari F, Forooghizadeh M. 2022. <a href="https://doi.org/10.3390/vaccines10111800">Comparison of the Safety and Immunogenicity of FAKHRAVAC and BBIBP-CorV Vaccines when Administrated as Booster Dose: A Parallel Two Arms, Randomized, Double Blind Clinical Trial</a>. Vaccines 10:1800.</div>
</div>
<div id="ref-dggZoRQD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1220. </div><div class="csl-right-inline">Rappuoli R, Pizza M, Del Giudice G, De Gregorio E. 2014. <a href="https://doi.org/10.1073/pnas.1402981111">Vaccines, new opportunities for a new society</a>. Proc Natl Acad Sci USA 111:12288–12293.</div>
</div>
<div id="ref-99C1xJ2E" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1221. </div><div class="csl-right-inline">Donaldson B, Al-Barwani F, Young V, Scullion S, Ward V, Young S. 2014. <a href="https://doi.org/10.1007/978-1-4939-1417-3_9">Virus-Like Particles, a Versatile Subunit Vaccine Platform</a>, p. 159–180. <em>In</em> Advances in Delivery Science and Technology. Springer New York.</div>
</div>
<div id="ref-lH2HMMZi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1222. </div><div class="csl-right-inline">Noad R, Roy P. 2003. <a href="https://doi.org/10.1016/s0966-842x(03)00208-7">Virus-like particles as immunogens</a>. Trends in Microbiology 11:438–444.</div>
</div>
<div id="ref-1Bxg7Wj6w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1223. </div><div class="csl-right-inline">Nooraei S, Bahrulolum H, Hoseini ZS, Katalani C, Hajizade A, Easton AJ, Ahmadian G. 2021. <a href="https://doi.org/10.1186/s12951-021-00806-7">Virus-like particles: preparation, immunogenicity and their roles as nanovaccines and drug nanocarriers</a>. J Nanobiotechnol 19.</div>
</div>
<div id="ref-1FfwyYaj7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1224. </div><div class="csl-right-inline">Pollet J, Chen W-H, Strych U. 2021. <a href="https://doi.org/10.1016/j.addr.2021.01.001">Recombinant protein vaccines, a proven approach against coronavirus pandemics</a>. Advanced Drug Delivery Reviews 170:71–82.</div>
</div>
<div id="ref-12eGVhH5I" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1225. </div><div class="csl-right-inline">1995. Vaccine DesignPharmaceutical Biotechnology. Springer US. <a href="https://doi.org/gh3zp9">https://doi.org/gh3zp9</a>.</div>
</div>
<div id="ref-124bnGvPp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1226. </div><div class="csl-right-inline">Li W, Joshi M, Singhania S, Ramsey K, Murthy A. 2014. <a href="https://doi.org/10.3390/vaccines2030515">Peptide Vaccine: Progress and Challenges</a>. Vaccines 2:515–536.</div>
</div>
<div id="ref-mv42t1HV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1227. </div><div class="csl-right-inline">Garçon N, Wettendorff M, Van Mechelen M. 2011. <a href="https://doi.org/10.1517/14712598.2011.573624">Role of AS04 in human papillomavirus vaccine: mode of action and clinical profile</a>. Expert Opinion on Biological Therapy 11:667–677.</div>
</div>
<div id="ref-rioTBsLc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1228. </div><div class="csl-right-inline">Shi S, Zhu H, Xia X, Liang Z, Ma X, Sun B. 2019. <a href="https://doi.org/10.1016/j.vaccine.2019.04.055">Vaccine adjuvants: Understanding the structure and mechanism of adjuvanticity</a>. Vaccine 37:3167–3178.</div>
</div>
<div id="ref-vrNgFknF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1229. </div><div class="csl-right-inline">Stils HF. 2005. <a href="https://doi.org/10.1093/ilar.46.3.280">Adjuvants and Antibody Production: Dispelling the Myths Associated with Freund's Complete and Other Adjuvants</a>. ILAR Journal 46:280–293.</div>
</div>
<div id="ref-cLAQnckq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1230. </div><div class="csl-right-inline">Zakhartchouk AN, Sharon C, Satkunarajah M, Auperin T, Viswanathan S, Mutwiri G, Petric M, See RH, Brunham RC, Finlay BB, Cameron C, Kelvin DJ, Cochrane A, Rini JM, Babiuk LA. 2007. <a href="https://doi.org/10.1016/j.vaccine.2006.06.084">Immunogenicity of a receptor-binding domain of SARS coronavirus spike protein in mice: Implications for a subunit vaccine</a>. Vaccine 25:136–143.</div>
</div>
<div id="ref-1EirBATaN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1231. </div><div class="csl-right-inline">Du L, Zhao G, He Y, Guo Y, Zheng B-J, Jiang S, Zhou Y. 2007. <a href="https://doi.org/10.1016/j.vaccine.2006.10.031">Receptor-binding domain of SARS-CoV spike protein induces long-term protective immunity in an animal model</a>. Vaccine 25:2832–2838.</div>
</div>
<div id="ref-1AOG59epD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1232. </div><div class="csl-right-inline">Du L, Zhao G, Chan CC, Li L, He Y, Zhou Y, Zheng B-J, Jiang S. 2010. <a href="https://doi.org/10.1089/vim.2009.0090">A 219-mer CHO-Expressing Receptor-Binding Domain of SARS-CoV S Protein Induces Potent Immune Responses and Protective Immunity</a>. Viral Immunology 23:211–219.</div>
</div>
<div id="ref-Ow2ICHez" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1233. </div><div class="csl-right-inline">He Y, Li J, Heck S, Lustigman S, Jiang S. 2006. <a href="https://doi.org/10.1128/jvi.00083-06">Antigenic and Immunogenic Characterization of Recombinant Baculovirus-Expressed Severe Acute Respiratory Syndrome Coronavirus Spike Protein: Implication for Vaccine Design</a>. J Virol 80:5757–5767.</div>
</div>
<div id="ref-tzZeWNPV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1234. </div><div class="csl-right-inline">Li J, Ulitzky L, Silberstein E, Taylor DR, Viscidi R. 2013. <a href="https://doi.org/10.1089/vim.2012.0076">Immunogenicity and Protection Efficacy of Monomeric and Trimeric Recombinant SARS Coronavirus Spike Protein Subunit Vaccine Candidates</a>. Viral Immunology 26:126–132.</div>
</div>
<div id="ref-DsfTQFmb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1235. </div><div class="csl-right-inline">Du L, Zhao G, Li L, He Y, Zhou Y, Zheng B-J, Jiang S. 2009. <a href="https://doi.org/10.1016/j.bbrc.2009.05.003">Antigenicity and immunogenicity of SARS-CoV S protein receptor-binding domain stably expressed in CHO cells</a>. Biochemical and Biophysical Research Communications 384:486–490.</div>
</div>
<div id="ref-IYjNaaqv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1236. </div><div class="csl-right-inline">Guo Y, Sun S, Wang K, Zhang S, Zhu W, Chen Z. 2005. <a href="https://doi.org/10.1089/dna.2005.24.510">Elicitation of Immunity in Mice After Immunization with the S2 Subunit of the Severe Acute Respiratory Syndrome Coronavirus</a>. DNA and Cell Biology 24:510–515.</div>
</div>
<div id="ref-HvO79P9u" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1237. </div><div class="csl-right-inline">He Y, Zhou Y, Siddiqui P, Niu J, Jiang S. 2005. <a href="https://doi.org/10.1128/jcm.43.8.3718-3726.2005">Identification of Immunodominant Epitopes on the Membrane Protein of the Severe Acute Respiratory Syndrome-Associated Coronavirus</a>. J Clin Microbiol 43:3718–3726.</div>
</div>
<div id="ref-VUcwpJKL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1238. </div><div class="csl-right-inline">Zheng N, Xia R, Yang C, Yin B, Li Y, Duan C, Liang L, Guo H, Xie Q. 2009. <a href="https://doi.org/10.1016/j.vaccine.2009.05.073">Boosted expression of the SARS-CoV nucleocapsid protein in tobacco and its immunogenicity in mice</a>. Vaccine 27:5001–5007.</div>
</div>
<div id="ref-7hbgOaiE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1239. </div><div class="csl-right-inline">LIU S, LENG C, LIEN S, CHI H, HUANG C, LIN C, LIAN W, CHEN C, HSIEH S, CHONG P. 2006. <a href="https://doi.org/10.1016/j.vaccine.2006.01.058">Immunological characterizations of the nucleocapsid protein based SARS vaccine candidates</a>. Vaccine 24:3100–3108.</div>
</div>
<div id="ref-9Zv0eLa9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1240. </div><div class="csl-right-inline">Wang N, Shang J, Jiang S, Du L. 2020. <a href="https://doi.org/10.3389/fmicb.2020.00298">Subunit Vaccines Against Emerging Pathogenic Human Coronaviruses</a>. Front Microbiol 11.</div>
</div>
<div id="ref-8723Jsa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1241. </div><div class="csl-right-inline">Chen W-H, Du L, Chag SM, Ma C, Tricoche N, Tao X, Seid CA, Hudspeth EM, Lustigman S, Tseng C-TK, Bottazzi ME, Hotez PJ, Zhan B, Jiang S. 2013. <a href="https://doi.org/10.4161/hv.27464">Yeast-expressed recombinant protein of the receptor-binding domain in SARS-CoV spike protein with deglycosylated forms as a SARS vaccine candidate</a>. Human Vaccines &amp; Immunotherapeutics 10:648–658.</div>
</div>
<div id="ref-Mki0DaYb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1242. </div><div class="csl-right-inline">Lan J, Yao Y, Deng Y, Chen H, Lu G, Wang W, Bao L, Deng W, Wei Q, Gao GF, Qin C, Tan W. 2015. <a href="https://doi.org/10.1016/j.ebiom.2015.08.031">Recombinant Receptor Binding Domain Protein Induces Partial Protective Immunity in Rhesus Macaques Against Middle East Respiratory Syndrome Coronavirus Challenge</a>. EBioMedicine 2:1438–1446.</div>
</div>
<div id="ref-aApaHV1w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1243. </div><div class="csl-right-inline">Lan J, Deng Y, Chen H, Lu G, Wang W, Guo X, Lu Z, Gao GF, Tan W. 2014. <a href="https://doi.org/10.1371/journal.pone.0112602">Tailoring Subunit Vaccine Immunity with Adjuvant Combinations and Delivery Routes Using the Middle East Respiratory Coronavirus (MERS-CoV) Receptor-Binding Domain as an Antigen</a>. PLoS ONE 9:e112602.</div>
</div>
<div id="ref-deUFGhNI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1244. </div><div class="csl-right-inline">Nyon MP, Du L, Tseng C-TK, Seid CA, Pollet J, Naceanceno KS, Agrawal A, Algaissi A, Peng B-H, Tai W, Jiang S, Bottazzi ME, Strych U, Hotez PJ. 2018. <a href="https://doi.org/10.1016/j.vaccine.2018.02.065">Engineering a stable CHO cell line for the expression of a MERS-coronavirus vaccine antigen</a>. Vaccine 36:1853–1862.</div>
</div>
<div id="ref-sfM5QV3m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1245. </div><div class="csl-right-inline">Tai W, Zhao G, Sun S, Guo Y, Wang Y, Tao X, Tseng C-TK, Li F, Jiang S, Du L, Zhou Y. 2016. <a href="https://doi.org/10.1016/j.virol.2016.10.005">A recombinant receptor-binding domain of MERS-CoV in trimeric form protects human dipeptidyl peptidase 4 (hDPP4) transgenic mice from MERS-CoV infection</a>. Virology 499:375–382.</div>
</div>
<div id="ref-oghHqZDt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1246. </div><div class="csl-right-inline">Pallesen J, Wang N, Corbett KS, Wrapp D, Kirchdoerfer RN, Turner HL, Cottrell CA, Becker MM, Wang L, Shi W, Kong W-P, Andres EL, Kettenbach AN, Denison MR, Chappell JD, Graham BS, Ward AB, McLellan JS. 2017. <a href="https://doi.org/10.1073/pnas.1707304114">Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen</a>. Proc Natl Acad Sci USA 114.</div>
</div>
<div id="ref-11Zz9H0Dl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1247. </div><div class="csl-right-inline">Yang Y, Deng Y, Wen B, Wang H, Meng X, Lan J, Gao GF, Tan W. 2014. <a href="https://doi.org/10.1089/vim.2014.0080">The Amino Acids 736–761 of the MERS-CoV Spike Protein Induce Neutralizing Antibodies: Implications for the Development of Vaccines and Antiviral Agents</a>. Viral Immunology 27:543–550.</div>
</div>
<div id="ref-ZXsnsfvb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1248. </div><div class="csl-right-inline">Jiaming L, Yanfeng Y, Yao D, Yawei H, Linlin B, Baoying H, Jinghua Y, Gao GF, Chuan Q, Wenjie T. 2017. <a href="https://doi.org/10.1016/j.vaccine.2016.11.064">The recombinant N-terminal domain of spike proteins is a potential vaccine against Middle East respiratory syndrome coronavirus (MERS-CoV) infection</a>. Vaccine 35:10–18.</div>
</div>
<div id="ref-GurQD2dO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1249. </div><div class="csl-right-inline">Wang L, Shi W, Joyce MG, Modjarrad K, Zhang Y, Leung K, Lees CR, Zhou T, Yassine HM, Kanekiyo M, Yang Z, Chen X, Becker MM, Freeman M, Vogel L, Johnson JC, Olinger G, Todd JP, Bagci U, Solomon J, Mollura DJ, Hensley L, Jahrling P, Denison MR, Rao SS, Subbarao K, Kwong PD, Mascola JR, Kong W-P, Graham BS. 2015. <a href="https://doi.org/10.1038/ncomms8712">Evaluation of candidate vaccine approaches for MERS-CoV</a>. Nat Commun 6.</div>
</div>
<div id="ref-T7W7hnB9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1250. </div><div class="csl-right-inline">Wang Q, Wong G, Lu G, Yan J, Gao GF. 2016. <a href="https://doi.org/10.1016/j.antiviral.2016.07.015">MERS-CoV spike protein: Targets for vaccines and therapeutics</a>. Antiviral Research 133:165–177.</div>
</div>
<div id="ref-oqty7gXw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1251. </div><div class="csl-right-inline">Hashemzadeh A, Avan A, Ferns GA, Khazaei M. 2020. <a href="https://doi.org/10.1016/j.vaccine.2020.07.003">Vaccines based on virus-like nano-particles for use against Middle East Respiratory Syndrome (MERS) coronavirus</a>. Vaccine 38:5742–5746.</div>
</div>
<div id="ref-eHe78HXD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1252. </div><div class="csl-right-inline">Coleman CM, Liu YV, Mu H, Taylor JK, Massare M, Flyer DC, Glenn GM, Smith GE, Frieman MB. 2014. <a href="https://doi.org/10.1016/j.vaccine.2014.04.016">Purified coronavirus spike protein nanoparticles induce coronavirus neutralizing antibodies in mice</a>. Vaccine 32:3169–3174.</div>
</div>
<div id="ref-cnYnzav2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1253. </div><div class="csl-right-inline">Wang C, Zheng X, Gai W, Zhao Y, Wang H, Wang H, Feng N, Chi H, Qiu B, Li N, Wang T, Gao Y, Yang S, Xia X. 2016. <a href="https://doi.org/10.18632/oncotarget.8475">MERS-CoV virus-like particles produced in insect cells induce specific humoural and cellular imminity in rhesus macaques</a>. Oncotarget 8:12686–12694.</div>
</div>
<div id="ref-G87TcArN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1254. </div><div class="csl-right-inline">Lan J, Deng Y, Song J, Huang B, Wang W, Tan W. 2018. <a href="https://doi.org/10.1007/s12250-018-0064-8">Significant Spike-Specific IgG and Neutralizing Antibodies in Mice Induced by a Novel Chimeric Virus-Like Particle Vaccine Candidate for Middle East Respiratory Syndrome Coronavirus</a>. Virol Sin 33:453–455.</div>
</div>
<div id="ref-jLJEygoA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1255. </div><div class="csl-right-inline">Li Y-D, Chi W-Y, Su J-H, Ferrall L, Hung C-F, Wu T-C. 2020. <a href="https://doi.org/10.1186/s12929-020-00695-2">Coronavirus vaccine development: from SARS and MERS to COVID-19</a>. J Biomed Sci 27.</div>
</div>
<div id="ref-155fGivMy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1256. </div><div class="csl-right-inline">Liljeqvist S, Ståhl S. 1999. <a href="https://doi.org/10.1016/s0168-1656(99)00107-8">Production of recombinant subunit vaccines: protein immunogens, live delivery systems and nucleic acid vaccines</a>. Journal of Biotechnology 73:1–33.</div>
</div>
<div id="ref-CQog2bB7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1257. </div><div class="csl-right-inline">Ausiello CM, Cassone A. 2014. <a href="https://doi.org/10.1128/mbio.01339-14">Acellular Pertussis Vaccines and Pertussis Resurgence: Revise or Replace?</a> mBio 5.</div>
</div>
<div id="ref-1CYqHUt3n" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1258. </div><div class="csl-right-inline">Cherry JD. 2013. <a href="https://doi.org/10.1371/journal.ppat.1003418">Pertussis: Challenges Today and for the Future</a>. PLoS Pathog 9:e1003418.</div>
</div>
<div id="ref-FVWwwB0d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1259. </div><div class="csl-right-inline">McLemore MR. 2006. <a href="https://doi.org/10.1188/06.cjon.559-560">Gardasil®: Introducing the New Human Papillomavirus Vaccine</a>. Clinical Journal of Oncology Nursing 10:559–560.</div>
</div>
<div id="ref-1FQlt5Lqz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1260. </div><div class="csl-right-inline">Zhang N, Zheng B-J, Lu L, Zhou Y, Jiang S, Du L. 2015. <a href="https://doi.org/10.1016/j.micinf.2014.12.006">Advancements in the development of subunit influenza vaccines</a>. Microbes and Infection 17:123–134.</div>
</div>
<div id="ref-8dSIiLCt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1261. </div><div class="csl-right-inline">Vaccines – COVID19 Vaccine Tracker. <a href="https://covid19.trackvaccines.org/vaccines/">https://covid19.trackvaccines.org/vaccines/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-aAYBP21H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1262. </div><div class="csl-right-inline">Wang M, Jiang S, Wang Y. 2016. <a href="https://doi.org/10.1080/21655979.2016.1191707">Recent advances in the production of recombinant subunit vaccines in Pichia pastoris</a>. Bioengineered 7:155–165.</div>
</div>
<div id="ref-NdbXBY1h" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1263. </div><div class="csl-right-inline">COVID19 Vaccine Tracker. <a href="https://covid19.trackvaccines.org/">https://covid19.trackvaccines.org/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-Qk33ZrIC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1264. </div><div class="csl-right-inline">Tian J-H, Patel N, Haupt R, Zhou H, Weston S, Hammond H, Logue J, Portnoff AD, Norton J, Guebre-Xabier M, Zhou B, Jacobson K, Maciejewski S, Khatoon R, Wisniewska M, Moffitt W, Kluepfel-Stahl S, Ekechukwu B, Papin J, Boddapati S, Jason Wong C, Piedra PA, Frieman MB, Massare MJ, Fries L, Bengtsson KL, Stertman L, Ellingsworth L, Glenn G, Smith G. 2021. <a href="https://doi.org/10.1038/s41467-020-20653-8">SARS-CoV-2 spike glycoprotein vaccine candidate NVX-CoV2373 immunogenicity in baboons and protection in mice</a>. Nat Commun 12.</div>
</div>
<div id="ref-RQR2sOmx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1265. </div><div class="csl-right-inline">Yee CM, Zak AJ, Hill BD, Wen F. 2018. <a href="https://doi.org/10.1021/acs.iecr.8b00985">The Coming Age of Insect Cells for Manufacturing and Development of Protein Therapeutics</a>. Ind Eng Chem Res 57:10061–10070.</div>
</div>
<div id="ref-nHjzMY5L" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1266. </div><div class="csl-right-inline">Riley TP, Chou H-T, Hu R, Bzymek KP, Correia AR, Partin AC, Li D, Gong D, Wang Z, Yu X, Manzanillo P, Garces F. 2021. <a href="https://doi.org/10.3389/fimmu.2021.660198">Enhancing the Prefusion Conformational Stability of SARS-CoV-2 Spike Protein Through Structure-Guided Design</a>. Front Immunol 12.</div>
</div>
<div id="ref-dMLXxGAI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1267. </div><div class="csl-right-inline">Keech C, Albert G, Cho I, Robertson A, Reed P, Neal S, Plested JS, Zhu M, Cloney-Clark S, Zhou H, Smith G, Patel N, Frieman MB, Haupt RE, Logue J, McGrath M, Weston S, Piedra PA, Desai C, Callahan K, Lewis M, Price-Abbott P, Formica N, Shinde V, Fries L, Lickliter JD, Griffin P, Wilkinson B, Glenn GM. 2020. <a href="https://doi.org/10.1056/nejmoa2026920">Phase 1–2 Trial of a SARS-CoV-2 Recombinant Spike Protein Nanoparticle Vaccine</a>. N Engl J Med 383:2320–2332.</div>
</div>
<div id="ref-1D0f8OrG8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1268. </div><div class="csl-right-inline">Heath PT, Galiza EP, Baxter DN, Boffito M, Browne D, Burns F, Chadwick DR, Clark R, Cosgrove C, Galloway J, Goodman AL, Heer A, Higham A, Iyengar S, Jamal A, Jeanes C, Kalra PA, Kyriakidou C, McAuley DF, Meyrick A, Minassian AM, Minton J, Moore P, Munsoor I, Nicholls H, Osanlou O, Packham J, Pretswell CH, San Francisco Ramos A, Saralaya D, Sheridan RP, Smith R, Soiza RL, Swift PA, Thomson EC, Turner J, Viljoen ME, Albert G, Cho I, Dubovsky F, Glenn G, Rivers J, Robertson A, Smith K, Toback S. 2021. <a href="https://doi.org/10.1056/nejmoa2107659">Safety and Efficacy of NVX-CoV2373 Covid-19 Vaccine</a>. N Engl J Med 385:1172–1183.</div>
</div>
<div id="ref-IQOyFnfl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1269. </div><div class="csl-right-inline">Dunkle LM, Kotloff KL, Gay CL, Áñez G, Adelglass JM, Barrat Hernández AQ, Harper WL, Duncanson DM, McArthur MA, Florescu DF, McClelland RS, Garcia-Fragoso V, Riesenberg RA, Musante DB, Fried DL, Safirstein BE, McKenzie M, Jeanfreau RJ, Kingsley JK, Henderson JA, Lane DC, Ruíz-Palacios GM, Corey L, Neuzil KM, Coombs RW, Greninger AL, Hutter J, Ake JA, Smith K, Woo W, Cho I, Glenn GM, Dubovsky F. 2022. <a href="https://doi.org/10.1056/nejmoa2116185">Efficacy and Safety of NVX-CoV2373 in Adults in the United States and Mexico</a>. N Engl J Med 386:531–543.</div>
</div>
<div id="ref-oc5SBo0q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1270. </div><div class="csl-right-inline">Dunkle LM, Kotloff KL, Gay CL, Áñez G, Adelglass JM, Barrat Hernández AQ, Harper WL, Duncanson DM, McArthur MA, Florescu DF, McClelland RS, Garcia-Fragoso V, Riesenberg RA, Musante DB, Fried DL, Safirstein BE, McKenzie M, Jeanfreau RJ, Kingsley JK, Henderson JA, Lane DC, Ruíz-Palacios GM, Corey L, Neuzil KM, Coombs RW, Greninger AL, Hutter J, Ake JA, Smith K, Woo W, Cho I, Glenn GM, Dubovsky F. 2021. <a href="https://doi.org/10.1101/2021.10.05.21264567">Efficacy and Safety of NVX-CoV2373 in Adults in the United States and Mexico</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-q3wIcP4d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1271. </div><div class="csl-right-inline">Novavax COVID-19 vaccine Nuvaxovid approved by MHRA. GOVUK. <a href="https://www.gov.uk/government/news/novavax-covid-19-vaccine-nuvaxovid-approved-by-mhra">https://www.gov.uk/government/news/novavax-covid-19-vaccine-nuvaxovid-approved-by-mhra</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-rPtsd03" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1272. </div><div class="csl-right-inline">2021. EMA recommends Nuvaxovid for authorisation in the EU. <a href="https://www.ema.europa.eu/en/news/ema-recommends-nuvaxovid-authorisation-eu">https://www.ema.europa.eu/en/news/ema-recommends-nuvaxovid-authorisation-eu</a>.</div>
</div>
<div id="ref-x9xXbzuY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1273. </div><div class="csl-right-inline">Commissioner O of the. 2022. Coronavirus (COVID-19) Update: FDA Authorizes Emergency Use of Novavax COVID-19 Vaccine, Adjuvanted. FDA. <a href="https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-emergency-use-novavax-covid-19-vaccine-adjuvanted">https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-emergency-use-novavax-covid-19-vaccine-adjuvanted</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-pxEbK2Vl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1274. </div><div class="csl-right-inline">Duong D, Vogel L. 2022. <a href="https://doi.org/10.1503/cmaj.1095992">Why is WHO pushing back on a Health Canada–approved Medicago SARS-CoV-2 vaccine?</a> CMAJ 194:E504–E505.</div>
</div>
<div id="ref-18Xmc8WMR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1275. </div><div class="csl-right-inline">Covifenz®. Medicago. <a href="https://medicago.com/en/our-products/our-vaccines/covifenz-covid-19-vlp-vaccine/">https://medicago.com/en/our-products/our-vaccines/covifenz-covid-19-vlp-vaccine/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-WqqE64cZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1276. </div><div class="csl-right-inline">Ward BJ, Makarkov A, Séguin A, Pillet S, Trépanier S, Dhaliwall J, Libman MD, Vesikari T, Landry N. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)32014-6">Efficacy, immunogenicity, and safety of a plant-derived, quadrivalent, virus-like particle influenza vaccine in adults (18–64 years) and older adults (≥65 years): two multicentre, randomised phase 3 trials</a>. The Lancet 396:1491–1503.</div>
</div>
<div id="ref-18mcJS5j4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1277. </div><div class="csl-right-inline">Ward BJ, Gobeil P, Séguin A, Atkins J, Boulay I, Charbonneau P-Y, Couture M, D’Aoust M-A, Dhaliwall J, Finkle C, Hager K, Mahmood A, Makarkov A, Cheng MP, Pillet S, Schimke P, St-Martin S, Trépanier S, Landry N. 2021. <a href="https://doi.org/10.1038/s41591-021-01370-1">Phase 1 randomized trial of a plant-derived virus-like particle vaccine for COVID-19</a>. Nat Med 27:1071–1078.</div>
</div>
<div id="ref-KqsZ2xpA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1278. </div><div class="csl-right-inline">Garçon N, Vaughn DW, Didierlaurent AM. 2012. <a href="https://doi.org/10.1586/erv.11.192">Development and evaluation of AS03, an Adjuvant System containing α-tocopherol and squalene in an oil-in-water emulsion</a>. Expert Review of Vaccines 11:349–366.</div>
</div>
<div id="ref-hZr9yVvu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1279. </div><div class="csl-right-inline">Hager KJ, Pérez Marc G, Gobeil P, Diaz RS, Heizer G, Llapur C, Makarkov AI, Vasconcellos E, Pillet S, Riera F, Saxena P, Geller Wolff P, Bhutada K, Wallace G, Aazami H, Jones CE, Polack FP, Ferrara L, Atkins J, Boulay I, Dhaliwall J, Charland N, Couture MMJ, Jiang-Wright J, Landry N, Lapointe S, Lorin A, Mahmood A, Moulton LH, Pahmer E, Parent J, Séguin A, Tran L, Breuer T, Ceregido M-A, Koutsoukos M, Roman F, Namba J, D’Aoust M-A, Trepanier S, Kimura Y, Ward BJ. 2022. <a href="https://doi.org/10.1056/nejmoa2201300">Efficacy and Safety of a Recombinant Plant-Based Adjuvanted Covid-19 Vaccine</a>. N Engl J Med 386:2084–2096.</div>
</div>
<div id="ref-1JmpfIPn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1280. </div><div class="csl-right-inline">Medicago and GSK announce the approval by Health Canada of COVIFENZ®, an adjuvanted plant-based COVID-19 vaccine | GSK. <a href="https://www.gsk.com/en-gb/media/press-releases/medicago-and-gsk-announce-the-approval-by-health-canada-of-covifenz/">https://www.gsk.com/en-gb/media/press-releases/medicago-and-gsk-announce-the-approval-by-health-canada-of-covifenz/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-kbvZWBoy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1281. </div><div class="csl-right-inline">D'Aoust M-A, Lavoie P-O, Couture MM-J, Trépanier S, Guay J-M, Dargis M, Mongrand S, Landry N, Ward BJ, Vézina L-P. 2008. <a href="https://doi.org/10.1111/j.1467-7652.2008.00384.x">Influenza virus-like particles produced by transient expression in Nicotiana benthamiana induce a protective immune response against a lethal viral challenge in mice</a>. Plant Biotechnology Journal 6:930–940.</div>
</div>
<div id="ref-aEJzXfl6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1282. </div><div class="csl-right-inline">Dyer O. 2022. <a href="https://doi.org/10.1136/bmj.o811">Covid-19: WHO set to reject Canadian plant based vaccine because of links with tobacco industry</a>. BMJ o811.</div>
</div>
<div id="ref-11WYMWwb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1283. </div><div class="csl-right-inline">Chen Q, Lai H. 2013. <a href="https://doi.org/10.4161/hv.22218">Plant-derived virus-like particles as vaccines</a>. Human Vaccines &amp;amp; Immunotherapeutics 9:26–49.</div>
</div>
<div id="ref-14K1ANV1T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1284. </div><div class="csl-right-inline">Shinde V, Bhikha S, Hoosain Z, Archary M, Bhorat Q, Fairlie L, Lalloo U, Masilela MSL, Moodley D, Hanley S, Fouche L, Louw C, Tameris M, Singh N, Goga A, Dheda K, Grobbelaar C, Kruger G, Carrim-Ganey N, Baillie V, de Oliveira T, Lombard Koen A, Lombaard JJ, Mngqibisa R, Bhorat AE, Benadé G, Lalloo N, Pitsi A, Vollgraaff P-L, Luabeya A, Esmail A, Petrick FG, Oommen-Jose A, Foulkes S, Ahmed K, Thombrayil A, Fries L, Cloney-Clark S, Zhu M, Bennett C, Albert G, Faust E, Plested JS, Robertson A, Neal S, Cho I, Glenn GM, Dubovsky F, Madhi SA. 2021. <a href="https://doi.org/10.1056/nejmoa2103055">Efficacy of NVX-CoV2373 Covid-19 Vaccine against the B.1.351 Variant</a>. N Engl J Med 384:1899–1909.</div>
</div>
<div id="ref-S28ZsPCI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1285. </div><div class="csl-right-inline">Mallory R, Formica N, Pfeiffer S, Wilkinson B, Marcheschi A, Albert G, McFall H, Robinson M, Plested JS, Zhu M, Cloney-Clark S, Zhou B, Chau G, Robertson A, Maciejewski S, Smith G, Patel N, Glenn GM, Dubovsky F. 2021. <a href="https://doi.org/10.1101/2021.12.23.21267374">Immunogenicity and Safety Following a Homologous Booster Dose of a SARS-CoV-2 recombinant spike protein vaccine (NVX-CoV2373): A Phase 2 Randomized Placebo-Controlled Trial</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-RG0vzlcE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1286. </div><div class="csl-right-inline">Mills MC, Salisbury D. 2021. <a href="https://doi.org/10.1016/j.eclinm.2020.100674">The challenges of distributing COVID-19 vaccinations</a>. EClinicalMedicine 31:100674.</div>
</div>
<div id="ref-19CWe6pdS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1287. </div><div class="csl-right-inline">Emanuel EJ, Persad G, Kern A, Buchanan A, Fabre C, Halliday D, Heath J, Herzog L, Leland RJ, Lemango ET, Luna F, McCoy MS, Norheim OF, Ottersen T, Schaefer GO, Tan K-C, Wellman CH, Wolff J, Richardson HS. 2020. <a href="https://doi.org/10.1126/science.abe2803">An ethical framework for global vaccine allocation</a>. Science 369:1309–1312.</div>
</div>
<div id="ref-d0kUYq5Z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1288. </div><div class="csl-right-inline">Matrajt L, Eaton J, Leung T, Brown ER. 2021. <a href="https://doi.org/10.1126/sciadv.abf1374">Vaccine optimization for COVID-19: Who to vaccinate first?</a> Sci Adv 7.</div>
</div>
<div id="ref-dfl5iCJI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1289. </div><div class="csl-right-inline">Holder J. 2021. <a href="https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html">Tracking Coronavirus Vaccinations Around the World</a>. The New York Times.</div>
</div>
<div id="ref-cWMPXfju" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1290. </div><div class="csl-right-inline">Mathieu E, Ritchie H, Rodés-Guirao L, Appel C, Giattino C, Hasell J, Macdonald B, Dattani S, Beltekian D, Ortiz-Ospina E, Roser M. 2020. <a href="https://ourworldindata.org/covid-vaccinations">Coronavirus Pandemic (COVID-19)</a>. Our World in Data.</div>
</div>
<div id="ref-kL8PlRJu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1291. </div><div class="csl-right-inline">Goodman PS. 2020. <a href="https://www.nytimes.com/2020/12/25/business/coronavirus-vaccines-global-economy.html">One Vaccine Side Effect: Global Economic Inequality</a>. The New York Times.</div>
</div>
<div id="ref-3Gq7ETv7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1292. </div><div class="csl-right-inline">ECCLESTON‐TURNER M, UPTON H. 2021. <a href="https://doi.org/10.1111/1468-0009.12503">International Collaboration to Ensure Equitable Access to Vaccines for COVID‐19: The ACT‐Accelerator and the COVAX Facility</a>. The Milbank Quarterly 99:426–449.</div>
</div>
<div id="ref-KzHIbPMY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1293. </div><div class="csl-right-inline">Fair allocation mechanism for COVID-19 vaccines through the COVAX Facility. <a href="https://www.who.int/publications/m/item/fair-allocation-mechanism-for-covid-19-vaccines-through-the-covax-facility">https://www.who.int/publications/m/item/fair-allocation-mechanism-for-covid-19-vaccines-through-the-covax-facility</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1H0PiQpLz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1294. </div><div class="csl-right-inline">Kupferschmidt K. 2020. <a href="https://doi.org/10.1126/science.369.6503.489">Global plan seeks to promote vaccine equity, spread risks</a>. Science 369:489–490.</div>
</div>
<div id="ref-k2clrWSf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1295. </div><div class="csl-right-inline">Usher AD. 2021. <a href="https://doi.org/10.1016/s0140-6736(21)01367-2">A beautiful idea: how COVAX has fallen short</a>. The Lancet 397:2322–2325.</div>
</div>
<div id="ref-L5wrzuPV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1296. </div><div class="csl-right-inline">Kibirige H. 2022. plotnine. <a href="https://github.com/has2k1/plotnine">https://github.com/has2k1/plotnine</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-48MbU9B0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1297. </div><div class="csl-right-inline">Smoothed conditional means — geom_smooth. <a href="https://ggplot2.tidyverse.org/reference/geom_smooth.html">https://ggplot2.tidyverse.org/reference/geom_smooth.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-FCTCkZoZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1298. </div><div class="csl-right-inline">2022. COVID-19 Dataset by Our World in Data. Our World in Data. <a href="https://github.com/owid/covid-19-data/blob/9e0b8314da28f155a88bbaf7617390c01c1c26da/public/data/vaccinations/locations.csv">https://github.com/owid/covid-19-data/blob/9e0b8314da28f155a88bbaf7617390c01c1c26da/public/data/vaccinations/locations.csv</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1AvwH3T5y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1299. </div><div class="csl-right-inline">Twohey M, Collins K, Thomas K. 2020. <a href="https://www.nytimes.com/2020/12/15/us/coronavirus-vaccine-doses-reserved.html">With First Dibs on Vaccines, Rich Countries Have ‘Cleared the Shelves’</a>. The New York Times.</div>
</div>
<div id="ref-JM2MJEBG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1300. </div><div class="csl-right-inline">Obinna DN. 2021. <a href="https://doi.org/10.1002/hpm.3341">Solidarity across borders: A pragmatic need for global COVID‐19 vaccine equity</a>. Health Planning &amp;amp; Management 37:21–29.</div>
</div>
<div id="ref-1EnpYQzIq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1301. </div><div class="csl-right-inline">2021. <a href="https://www.bbc.com/news/56100076">Covid-19 vaccinations: African nations miss WHO target</a>. BBC News.</div>
</div>
<div id="ref-12QaZb4si" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1302. </div><div class="csl-right-inline">Herzog LM, Norheim OF, Emanuel EJ, McCoy MS. 2021. <a href="https://doi.org/10.1136/bmj.m4853">Covax must go beyond proportional allocation of covid vaccines to ensure fair and equitable access</a>. BMJ m4853.</div>
</div>
<div id="ref-7dkwQDUf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1303. </div><div class="csl-right-inline">COVAX. <a href="https://www.who.int/initiatives/act-accelerator/covax">https://www.who.int/initiatives/act-accelerator/covax</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-JNpdOLGo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1304. </div><div class="csl-right-inline">COVAX vaccine roll-out. <a href="https://www.gavi.org/covax-vaccine-roll-out">https://www.gavi.org/covax-vaccine-roll-out</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-sr5oRBgc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1305. </div><div class="csl-right-inline">2021. <a href="https://doi.org/10.1126/science.abh4476">Countries now scrambling for COVID-19 vaccines may soon have surpluses to donate</a>. AAAS Articles DO Group. American Association for the Advancement of Science (AAAS).</div>
</div>
<div id="ref-S8WhufUV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1306. </div><div class="csl-right-inline">Bubar KM, Reinholt K, Kissler SM, Lipsitch M, Cobey S, Grad YH, Larremore DB. 2021. <a href="https://doi.org/10.1126/science.abe6959">Model-informed COVID-19 vaccine prioritization strategies by age and serostatus</a>. Science 371:916–921.</div>
</div>
<div id="ref-dLbKv1xi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1307. </div><div class="csl-right-inline">Grauer J, Löwen H, Liebchen B. 2020. <a href="https://doi.org/10.1038/s41598-020-78447-3">Strategic spatiotemporal vaccine distribution increases the survival rate in an infectious disease like Covid-19</a>. Sci Rep 10.</div>
</div>
<div id="ref-jbpdQdOw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1308. </div><div class="csl-right-inline">Williams J, Degeling C, McVernon J, Dawson A. 2021. <a href="https://doi.org/10.1016/j.vaccine.2020.12.059">How should we conduct pandemic vaccination?</a> Vaccine 39:994–999.</div>
</div>
<div id="ref-z5c17nGB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1309. </div><div class="csl-right-inline">Jecker NS, Wightman AG, Diekema DS. 2021. <a href="https://doi.org/10.1136/medethics-2020-107036">Vaccine ethics: an ethical framework for global distribution of COVID-19 vaccines</a>. J Med Ethics medethics-2020-107036.</div>
</div>
<div id="ref-s2zZd6pb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1310. </div><div class="csl-right-inline">Tran TN-A, Wikle N, Albert J, Inam H, Strong E, Brinda K, Leighow SM, Yang F, Hossain S, Pritchard JR, Chan P, Hanage WP, Hanks EM, Boni MF. 2021. <a href="https://doi.org/10.1101/2021.01.12.21249694">Optimal SARS-CoV-2 vaccine allocation using real-time seroprevalence estimates in Rhode Island and Massachusetts</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-sEyIoYCS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1311. </div><div class="csl-right-inline">Mathieu E, Ritchie H, Rodés-Guirao L, Appel C, Giattino C, Hasell J, Macdonald B, Dattani S, Beltekian D, Ortiz-Ospina E, Roser M. 2020. <a href="https://ourworldindata.org/covid-vaccination-policy">Coronavirus Pandemic (COVID-19)</a>. Our World in Data.</div>
</div>
<div id="ref-T5hm2GMj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1312. </div><div class="csl-right-inline">Ferran M. CORBEVAX, a new patent-free COVID-19 vaccine, could be a pandemic game changer globally. The Conversation. <a href="http://theconversation.com/corbevax-a-new-patent-free-covid-19-vaccine-could-be-a-pandemic-game-changer-globally-174672">http://theconversation.com/corbevax-a-new-patent-free-covid-19-vaccine-could-be-a-pandemic-game-changer-globally-174672</a>. Retrieved 20 December 2022.</div>
</div>
<div id="ref-RueOviyL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1313. </div><div class="csl-right-inline">Biological E Limited: Corbevax – COVID19 Vaccine Tracker. <a href="https://covid19.trackvaccines.org/vaccines/54/">https://covid19.trackvaccines.org/vaccines/54/</a>. Retrieved 20 December 2022.</div>
</div>
<div id="ref-ZCOfVmLC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1314. </div><div class="csl-right-inline">James ER. 2021. <a href="https://doi.org/10.1093/inthealth/ihab010">Disrupting vaccine logistics</a>. International Health 13:211–214.</div>
</div>
<div id="ref-CfthGlKw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1315. </div><div class="csl-right-inline">AboulFotouh K, Cui Z, Williams RO III. 2021. <a href="https://doi.org/10.1208/s12249-021-01974-3">Next-Generation COVID-19 Vaccines Should Take Efficiency of Distribution into Consideration</a>. AAPS PharmSciTech 22.</div>
</div>
<div id="ref-QpYrg3Eh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1316. </div><div class="csl-right-inline">Bloomberg - Are you a robot? <a href="https://www.bloomberg.com/tosv2.html?vid=&amp;uuid=36df698d-809f-11ed-b884-5962526b6553&amp;url=L25ld3MvYXJ0aWNsZXMvMjAyMS0wMS0wMy9wZml6ZXItdG8tc3VwcGx5LWFmcmljYW4taGVhbHRoLWNhcmUtd29ya2Vycy13aXRoLXZhY2NpbmVz">https://www.bloomberg.com/tosv2.html?vid=&amp;uuid=36df698d-809f-11ed-b884-5962526b6553&amp;url=L25ld3MvYXJ0aWNsZXMvMjAyMS0wMS0wMy9wZml6ZXItdG8tc3VwcGx5LWFmcmljYW4taGVhbHRoLWNhcmUtd29ya2Vycy13aXRoLXZhY2NpbmVz</a>. Retrieved 20 December 2022.</div>
</div>
<div id="ref-LwA7POre" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1317. </div><div class="csl-right-inline">COVAX: ensuring global equitable access to COVID-19 vaccines. <a href="https://www.unicef.org/supply/covax-ensuring-global-equitable-access-covid-19-vaccines">https://www.unicef.org/supply/covax-ensuring-global-equitable-access-covid-19-vaccines</a>. Retrieved 20 December 2022.</div>
</div>
<div id="ref-h68ER2wh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1318. </div><div class="csl-right-inline">Bajaj SS, Maki L, Stanford FC. 2022. <a href="https://doi.org/10.1016/s0140-6736(22)00328-2">Vaccine apartheid: global cooperation and equity</a>. The Lancet 399:1452–1453.</div>
</div>
<div id="ref-sjDDQBRj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1319. </div><div class="csl-right-inline">Sparke M, Levy O. 2022. <a href="https://doi.org/10.1093/cid/ciac361">Competing Responses to Global Inequalities in Access to COVID Vaccines: Vaccine Diplomacy and Vaccine Charity Versus Vaccine Liberty</a>. Clinical Infectious Diseases 75:S86–S92.</div>
</div>
<div id="ref-vxL9u0id" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1320. </div><div class="csl-right-inline">Grenfell RFQ, Oyeyemi OT. 2022. <a href="https://doi.org/10.1080/20477724.2022.2091862">Access to COVID-19 vaccines and testing in Africa: the importance of COVAX - Nigeria as a case study</a>. Pathogens and Global Health 1–15.</div>
</div>
<div id="ref-Pjo8Db92" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1321. </div><div class="csl-right-inline">Taylor A. 2022. <a href="https://www.washingtonpost.com/world/2021/12/10/covax-doses-delivered/">Covax promised 2 billion vaccine doses to help the world’s neediest in 2021. It won’t deliver even half that.</a> Washington Post.</div>
</div>
<div id="ref-1683Ay0Ma" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1322. </div><div class="csl-right-inline">Ten health issues WHO will tackle this year. <a href="https://www.who.int/news-room/spotlight/ten-threats-to-global-health-in-2019">https://www.who.int/news-room/spotlight/ten-threats-to-global-health-in-2019</a>. Retrieved 20 December 2022.</div>
</div>
<div id="ref-vgBdaJfl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1323. </div><div class="csl-right-inline">Lopes L, 2021. 2021. KFF COVID-19 Vaccine Monitor: January 2021. KFF. <a href="https://www.kff.org/coronavirus-covid-19/report/kff-covid-19-vaccine-monitor-january-2021/">https://www.kff.org/coronavirus-covid-19/report/kff-covid-19-vaccine-monitor-january-2021/</a>. Retrieved 20 December 2022.</div>
</div>
<div id="ref-9WzhBzsN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1324. </div><div class="csl-right-inline">Rosenbaum L. 2021. <a href="https://doi.org/10.1056/nejmms2101220">Escaping Catch-22 — Overcoming Covid Vaccine Hesitancy</a>. N Engl J Med 384:1367–1371.</div>
</div>
<div id="ref-1GU6vsFSK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1325. </div><div class="csl-right-inline">Unroe KT, Evans R, Weaver L, Rusyniak D, Blackburn J. 2021. <a href="https://doi.org/10.1111/jgs.17022">Willingness of Long‐Term Care Staff to Receive a
&lt;scp&gt;COVID&lt;/scp&gt;
‐19 Vaccine: A Single State Survey</a>. J Am Geriatr Soc 69:593–599.</div>
</div>
<div id="ref-3Bv9rDjF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1326. </div><div class="csl-right-inline">Ruiz JB, Bell RA. 2022. <a href="https://doi.org/10.1177/00333549221114346">Parental COVID-19 Vaccine Hesitancy in the United States</a>. Public Health Rep 137:1162–1169.</div>
</div>
<div id="ref-aMyvkKF6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1327. </div><div class="csl-right-inline">Cénat JM, Noorishad P, Moshirian Farahi SMM, Darius WP, Mesbahi El Aouame A, Onesi O, Broussard C, Furyk SE, Yaya S, Caulley L, Chomienne M, Etowa J, Labelle PR. 2022. Prevalence and factors related to COVID‐19 vaccine hesitancy and unwillingness in Canada: A systematic review and meta‐analysis. Journal of Medical Virology https://doi.org/<a href="https://doi.org/10.1002/jmv.28156">10.1002/jmv.28156</a>.</div>
</div>
<div id="ref-Qc6ugUZB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1328. </div><div class="csl-right-inline">Steinert JI, Sternberg H, Prince H, Fasolo B, Galizzi MM, Büthe T, Veltri GA. 2022. <a href="https://doi.org/10.1126/sciadv.abm9825">COVID-19 vaccine hesitancy in eight European countries: Prevalence, determinants, and heterogeneity</a>. Sci Adv 8.</div>
</div>
<div id="ref-13Qw1QyDZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1329. </div><div class="csl-right-inline">Ackah BBB, Woo M, Stallwood L, Fazal ZA, Okpani A, Ukah UV, Adu PA. 2022. <a href="https://doi.org/10.1186/s41256-022-00255-1">COVID-19 vaccine hesitancy in Africa: a scoping review</a>. glob health res policy 7.</div>
</div>
<div id="ref-ADXpa112" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1330. </div><div class="csl-right-inline">Harapan H, Anwar S, Yufika A, Sharun K, Gachabayov M, Fahriani M, Husnah M, Raad R, Abdalla RY, Adam RY, Khiri NME, Ismaeil MIH, Ismail AY, Kacem W, Dahman NBH, Teyeb Z, Aloui K, Hafsi M, Ferjani M, Deeb DA, Emad D, Abbas KS, Monib FA, Sami FS, Subramaniam R, Panchawagh S, Anandu S, Haque MA, Ferreto LED, Briones MFC, Morales RBI, Díaz SAL, Aburto JTO, Rojas JET, Balogun EO, Enitan SS, Yomi AR, Durosinmi A, Ezigbo ED, Adejumo EN, Babadi E, Kakemam E, Malik NI, Ullah I, Rosiello DF, Emran TB, Wendt GW, Arab-Zozani M, Wagner AL, Mudatsir M. 2021. <a href="https://doi.org/10.1080/20477724.2021.2011580">Vaccine hesitancy among communities in ten countries in Asia, Africa, and South America during the COVID-19 pandemic</a>. Pathogens and Global Health 116:236–243.</div>
</div>
<div id="ref-qLCe8Fn4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1331. </div><div class="csl-right-inline">Dereje N, Tesfaye A, Tamene B, Alemeshet D, Abe H, Tesfa N, Gedion S, Biruk T, Lakew Y. 2022. <a href="https://doi.org/10.1136/bmjopen-2021-052432">COVID-19 vaccine hesitancy in Addis Ababa, Ethiopia: a mixed-method study</a>. BMJ Open 12:e052432.</div>
</div>
<div id="ref-rY8TOiPE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1332. </div><div class="csl-right-inline">Rathje S, He JK, Roozenbeek J, Van Bavel JJ, van der Linden S. 2022. <a href="https://doi.org/10.1093/pnasnexus/pgac207">Social media behavior is associated with vaccine hesitancy</a>. PNAS Nexus 1.</div>
</div>
<div id="ref-IQRxQrGA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1333. </div><div class="csl-right-inline">Pertwee E, Simas C, Larson HJ. 2022. <a href="https://doi.org/10.1038/s41591-022-01728-z">An epidemic of uncertainty: rumors, conspiracy theories and vaccine hesitancy</a>. Nat Med 28:456–459.</div>
</div>
<div id="ref-14zu298e4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1334. </div><div class="csl-right-inline">Ng JWJ, Vaithilingam S, Nair M, Hwang L-A, Musa KI. 2022. <a href="https://doi.org/10.1371/journal.pone.0268926">Key predictors of COVID-19 vaccine hesitancy in Malaysia: An integrated framework</a>. PLoS ONE 17:e0268926.</div>
</div>
<div id="ref-udYZFdZJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1335. </div><div class="csl-right-inline">Schernhammer E, Weitzer J, Laubichler MD, Birmann BM, Bertau M, Zenk L, Caniglia G, Jäger CC, Steiner G. 2021. <a href="https://doi.org/10.1093/pubmed/fdab122">Correlates of COVID-19 vaccine hesitancy in Austria: trust and the government</a>. Journal of Public Health 44:e106–e116.</div>
</div>
<div id="ref-hCKsU5CZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1336. </div><div class="csl-right-inline">Head KJ, Kasting ML, Sturm LA, Hartsock JA, Zimet GD. 2020. <a href="https://doi.org/10.1177/1075547020960463">A National Survey Assessing SARS-CoV-2 Vaccination Intentions: Implications for Future Public Health Communication Efforts</a>. Science Communication 42:698–723.</div>
</div>
<div id="ref-XWoyDldi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1337. </div><div class="csl-right-inline">Salmon D, Opel DJ, Dudley MZ, Brewer J, Breiman R. 2021. <a href="https://doi.org/10.1377/hlthaff.2020.02254">Reflections On Governance, Communication, And Equity: Challenges And Opportunities In COVID-19 Vaccination</a>. Health Affairs 40:419–425.</div>
</div>
<div id="ref-s6yd91AQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1338. </div><div class="csl-right-inline">Anand P, Stahel VP. 2021. <a href="https://doi.org/10.1186/s13037-021-00291-9">The safety of Covid-19 mRNA vaccines: a review</a>. Patient Saf Surg 15.</div>
</div>
<div id="ref-vIrtP9Np" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1339. </div><div class="csl-right-inline">Salmon DA, Dudley MZ, Glanz JM, Omer SB. 2015. <a href="https://doi.org/10.1016/j.vaccine.2015.09.035">Vaccine hesitancy</a>. Vaccine 33:D66–D71.</div>
</div>
<div id="ref-qSSc0Ow0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1340. </div><div class="csl-right-inline">Jacobson RM, St. Sauver JL, Finney Rutten LJ. 2015. <a href="https://doi.org/10.1016/j.mayocp.2015.09.006">Vaccine Hesitancy</a>. Mayo Clinic Proceedings 90:1562–1568.</div>
</div>
<div id="ref-1ADXp4JiF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1341. </div><div class="csl-right-inline">What sets apart Novavax option from other COVID-19 vaccines. American Medical Association. <a href="https://www.ama-assn.org/delivering-care/public-health/what-sets-apart-novavax-option-other-covid-19-vaccines">https://www.ama-assn.org/delivering-care/public-health/what-sets-apart-novavax-option-other-covid-19-vaccines</a>. Retrieved 20 December 2022.</div>
</div>
<div id="ref-1HJpEx8e7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1342. </div><div class="csl-right-inline">RUPP DMBAR. Novavax COVID vaccine is for those hesitant to get vaccinated. The Daily News. <a href="https://www.galvnews.com/health/free/article_8ddebef4-1db0-579a-b0d2-852f2031d881.html">https://www.galvnews.com/health/free/article_8ddebef4-1db0-579a-b0d2-852f2031d881.html</a>. Retrieved 20 December 2022.</div>
</div>
<div id="ref-13O5dayXS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1343. </div><div class="csl-right-inline">Rubin R. 2022. <a href="https://doi.org/10.1001/jama.2022.13661">Despite Its Fan Base, Newly Authorized “Traditional” Novavax COVID-19 Vaccine Is Having Trouble Gaining a Foothold in the US</a>. JAMA 328:1026.</div>
</div>
<div id="ref-MMu5x58p" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1344. </div><div class="csl-right-inline">Constantino AK. Novavax's new Covid vaccine is perfect for people scared of mRNA tech—but it won't win over the unvaccinated. CNBC. <a href="https://www.cnbc.com/2022/07/15/why-new-novavax-covid-vaccine-wont-win-over-unvaccinated-americans.html">https://www.cnbc.com/2022/07/15/why-new-novavax-covid-vaccine-wont-win-over-unvaccinated-americans.html</a>. Retrieved 20 December 2022.</div>
</div>
<div id="ref-H4USOXie" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1345. </div><div class="csl-right-inline">Jiang S, He Y, Liu S. 2005. <a href="https://doi.org/10.3201/eid1107.050219">SARS Vaccine Development</a>. Emerg Infect Dis 11:1016–1020.</div>
</div>
<div id="ref-AOGjkjCq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1346. </div><div class="csl-right-inline">Graham RL, Donaldson EF, Baric RS. 2013. <a href="https://doi.org/10.1038/nrmicro3143">A decade after SARS: strategies for controlling emerging coronaviruses</a>. Nat Rev Microbiol 11:836–848.</div>
</div>
<div id="ref-HyYY2agc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1347. </div><div class="csl-right-inline">Lurie N, Saville M, Hatchett R, Halton J. 2020. <a href="https://doi.org/10.1056/nejmp2005630">Developing Covid-19 Vaccines at Pandemic Speed</a>. New England Journal of Medicine 382:1969–1973.</div>
</div>
<div id="ref-138O0v19T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1348. </div><div class="csl-right-inline">Maslow JN. 2017. <a href="https://doi.org/10.1080/21645515.2017.1358325">Vaccines for emerging infectious diseases: Lessons from MERS coronavirus and Zika virus</a>. Human Vaccines &amp; Immunotherapeutics 13:2918–2930.</div>
</div>
<div id="ref-RGPoDfHS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1349. </div><div class="csl-right-inline">Sharma O, Sultan AA, Ding H, Triggle CR. 2020. <a href="https://doi.org/10.3389/fimmu.2020.585354">A Review of the Progress and Challenges of Developing a Vaccine for COVID-19</a>. Front Immunol 11.</div>
</div>
<div id="ref-1GUVvcQjL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1350. </div><div class="csl-right-inline">Li Y-P, Liang Z-L, Gao Q, Huang L-R, Mao Q-Y, Wen S-Q, Liu Y, Yin W-D, Li R-C, Wang J-Z. 2012. <a href="https://doi.org/10.1016/j.vaccine.2012.03.010">Safety and immunogenicity of a novel human Enterovirus 71 (EV71) vaccine: A randomized, placebo-controlled, double-blind, Phase I clinical trial</a>. Vaccine 30:3295–3303.</div>
</div>
<div id="ref-KewHbkLZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1351. </div><div class="csl-right-inline">Palacios R, Patiño EG, de Oliveira Piorelli R, Conde MTRP, Batista AP, Zeng G, Xin Q, Kallas EG, Flores J, Ockenhouse CF, Gast C. 2020. <a href="https://doi.org/10.1186/s13063-020-04775-4">Double-Blind, Randomized, Placebo-Controlled Phase III Clinical Trial to Evaluate the Efficacy and Safety of treating Healthcare Professionals with the Adsorbed COVID-19 (Inactivated) Vaccine Manufactured by Sinovac – PROFISCOV: A structured summary of a study protocol for a randomised controlled trial</a>. Trials 21.</div>
</div>
<div id="ref-F1lYwPSY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1352. </div><div class="csl-right-inline">Palacios R, Batista AP, Albuquerque CSN, Patiño EG, Santos J do P, Tilli Reis Pessoa Conde M, Piorelli R de O, Pereira Júnior LC, Raboni SM, Ramos F, Sierra Romero GA, Leal FE, Camargo LFA, Aoki FH, Coelho EB, Oliveira DS, Fontes CJF, Pileggi GCS, Oliveira ALL de, Siqueira AM de, Oliveira DBL de, Botosso VF, Zeng G, Xin Q, Teixeira MM, Nogueira ML, Kallas EG. 2021. Efficacy and Safety of a COVID-19 Inactivated Vaccine in Healthcare Professionals in Brazil: The PROFISCOV Study. SSRN Journal https://doi.org/<a href="https://doi.org/10.2139/ssrn.3822780">10.2139/ssrn.3822780</a>.</div>
</div>
<div id="ref-19z4gdhDS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1353. </div><div class="csl-right-inline">Sinovac: CoronaVac – COVID19 Vaccine Tracker. <a href="https://covid19.trackvaccines.org/vaccines/7/">https://covid19.trackvaccines.org/vaccines/7/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-8vzglCry" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1354. </div><div class="csl-right-inline">Medeiros-Ribeiro AC, Aikawa NE, Saad CGS, Yuki EFN, Pedrosa T, Fusco SRG, Rojo PT, Pereira RMR, Shinjo SK, Andrade DCO, Sampaio-Barros PD, Ribeiro CT, Deveza GBH, Martins VAO, Silva CA, Lopes MH, Duarte AJS, Antonangelo L, Sabino EC, Kallas EG, Pasoto SG, Bonfa E. 2021. <a href="https://doi.org/10.1038/s41591-021-01469-5">Immunogenicity and safety of the CoronaVac inactivated vaccine in patients with autoimmune rheumatic diseases: a phase 4 trial</a>. Nat Med 27:1744–1751.</div>
</div>
<div id="ref-6EYqf6s7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1355. </div><div class="csl-right-inline">Han B, Song Y, Li C, Yang W, Ma Q, Jiang Z, Li M, Lian X, Jiao W, Wang L, Shu Q, Wu Z, Zhao Y, Li Q, Gao Q. 2021. <a href="https://doi.org/10.1016/s1473-3099(21)00319-4">Safety, tolerability, and immunogenicity of an inactivated SARS-CoV-2 vaccine (CoronaVac) in healthy children and adolescents: a double-blind, randomised, controlled, phase 1/2 clinical trial</a>. The Lancet Infectious Diseases 21:1645–1653.</div>
</div>
<div id="ref-Ks3L7qHG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1356. </div><div class="csl-right-inline">2021. Bharat Biotech Announces Phase 3 Results of COVAXIN. Bharat Biotech. <a href="https://www.bharatbiotech.com/images/press/covaxin-phase3-efficacy-results.pdf">https://www.bharatbiotech.com/images/press/covaxin-phase3-efficacy-results.pdf</a>.</div>
</div>
<div id="ref-DaDKZXdu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1357. </div><div class="csl-right-inline">2021. Ocugen's COVID-19 Vaccine Co-Development Partner, Bharat Biotech shares Phase 3 Interim Results of COVAXIN, Demonstrates Efficacy of 81%. Biogenetech. <a href="https://www.biogenetech.co.th/wp-content/uploads/2021/03/5-Ocugen.pdf">https://www.biogenetech.co.th/wp-content/uploads/2021/03/5-Ocugen.pdf</a>.</div>
</div>
<div id="ref-13yEnvOyP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1358. </div><div class="csl-right-inline">Zimbabwe authorizes use of India's first indigenous COVID-19 vaccine - Xinhua | English.news.cn. <a href="http://www.xinhuanet.com/english/2021-03/04/c_139783893.htm">http://www.xinhuanet.com/english/2021-03/04/c_139783893.htm</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-UU0pj3ii" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1359. </div><div class="csl-right-inline">Das S. 2021. Booster dose: Bharat Biotech's nasal vaccine may be used with Covaxin. <a href="https://www.business-standard.com/article/current-affairs/booster-dose-bharat-biotech-s-nasal-vaccine-may-be-used-with-covaxin-121092500034_1.html">https://www.business-standard.com/article/current-affairs/booster-dose-bharat-biotech-s-nasal-vaccine-may-be-used-with-covaxin-121092500034_1.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-3Q7kvwO3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1360. </div><div class="csl-right-inline">2021. <a href="https://timesofindia.indiatimes.com/india/bharat-biotech-to-submit-covaxin-kids-trials-data-to-dcgi-in-weeks-time/articleshow/86392325.cms">Covaxin kids trial over, Bharat Biotech to submit data to DCGI next week</a>. The Times of India.</div>
</div>
<div id="ref-AeulXXFm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1361. </div><div class="csl-right-inline">WHO emergency approval for Covaxin delayed till October 5. India Today. <a href="https://www.indiatoday.in/coronavirus-covid-19-outbreak/video/who-emergency-approval-for-covaxin-delayed-till-october-1856178-2021-09-23">https://www.indiatoday.in/coronavirus-covid-19-outbreak/video/who-emergency-approval-for-covaxin-delayed-till-october-1856178-2021-09-23</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1F52Wz7mx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1362. </div><div class="csl-right-inline">Magnusson SE, Altenburg AF, Bengtsson KL, Bosman F, de Vries RD, Rimmelzwaan GF, Stertman L. 2018. <a href="https://doi.org/10.1007/s12026-018-8991-x">Matrix-M™ adjuvant enhances immunogenicity of both protein- and modified vaccinia virus Ankara-based influenza vaccines in mice</a>. Immunol Res 66:224–233.</div>
</div>
<div id="ref-scwOT7dw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1363. </div><div class="csl-right-inline">Magnusson SE, Reimer JM, Karlsson KH, Lilja L, Bengtsson KL, Stertman L. 2013. <a href="https://doi.org/10.1016/j.vaccine.2013.01.039">Immune enhancing properties of the novel Matrix-M™ adjuvant leads to potentiated immune responses to an influenza vaccine in mice</a>. Vaccine 31:1725–1733.</div>
</div>
<div id="ref-aD5iMC0Q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1364. </div><div class="csl-right-inline">Madhun AS, Haaheim LR, Nilsen MV, Cox RJ. 2009. <a href="https://doi.org/10.1016/j.vaccine.2009.09.044">Intramuscular Matrix-M-adjuvanted virosomal H5N1 vaccine induces high frequencies of multifunctional Th1 CD4+ cells and strong antibody responses in mice</a>. Vaccine 27:7367–7376.</div>
</div>
<div id="ref-twmXSpc9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1365. </div><div class="csl-right-inline">Pedersen G, Major D, Roseby S, Wood J, Madhun AS, Cox RJ. 2011. <a href="https://doi.org/10.1111/j.1750-2659.2011.00256.x">Matrix-M adjuvanted virosomal H5N1 vaccine confers protection against lethal viral challenge in a murine model</a>. Influenza and Other Respiratory Viruses 5:426–437.</div>
</div>
<div id="ref-jVCL0201" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1366. </div><div class="csl-right-inline">Cox RJ, Pedersen G, Madhun AS, Svindland S, Sævik M, Breakwell L, Hoschler K, Willemsen M, Campitelli L, Nøstbakken JK, Weverling GJ, Klap J, McCullough KC, Zambon M, Kompier R, Sjursen H. 2011. <a href="https://doi.org/10.1016/j.vaccine.2011.08.042">Evaluation of a virosomal H5N1 vaccine formulated with Matrix M™ adjuvant in a phase I clinical trial</a>. Vaccine 29:8049–8059.</div>
</div>
<div id="ref-Nq0cimEs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1367. </div><div class="csl-right-inline">Novavax. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04368988">A 2-Part, Phase 1/2, Randomized, Observer-Blinded Study To Evaluate The Safety And Immunogenicity Of A SARS-CoV-2 Recombinant Spike Protein Nanoparticle Vaccine (SARS-CoV-2 rS) With Or Without MATRIX-M™ Adjuvant In Healthy Subjects</a>. NCT04368988. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-ynvQwwo9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1368. </div><div class="csl-right-inline">Formica N, Mallory R, Albert G, Robinson M, Plested JS, Cho I, Robertson A, Dubovsky F, Glenn GM. 2021. <a href="https://doi.org/10.1371/journal.pmed.1003769">Different dose regimens of a SARS-CoV-2 recombinant spike protein vaccine (NVX-CoV2373) in younger and older adults: A phase 2 randomized placebo-controlled trial</a>. PLoS Med 18:e1003769.</div>
</div>
<div id="ref-1EWB3hKG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1369. </div><div class="csl-right-inline">Novavax COVID-19 Vaccine Demonstrates 89.3% Efficacy in UK Phase 3 Trial. Novavax Investor Relations. <a href="https://ir.novavax.com/2021-01-28-Novavax-COVID-19-Vaccine-Demonstrates-89-3-Efficacy-in-UK-Phase-3-Trial">https://ir.novavax.com/2021-01-28-Novavax-COVID-19-Vaccine-Demonstrates-89-3-Efficacy-in-UK-Phase-3-Trial</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-16iOtJ1vE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1370. </div><div class="csl-right-inline">Novavax COVID-19 Vaccine Demonstrates 90% Overall Efficacy and 100% Protection Against Moderate and Severe Disease in PREVENT-19 Phase 3 Trial. Novavax Investor Relations. <a href="https://ir.novavax.com/2021-06-14-Novavax-COVID-19-Vaccine-Demonstrates-90-Overall-Efficacy-and-100-Protection-Against-Moderate-and-Severe-Disease-in-PREVENT-19-Phase-3-Trial">https://ir.novavax.com/2021-06-14-Novavax-COVID-19-Vaccine-Demonstrates-90-Overall-Efficacy-and-100-Protection-Against-Moderate-and-Severe-Disease-in-PREVENT-19-Phase-3-Trial</a>. Retrieved 1 December 2022.</div>
</div>
<div id="ref-IUekaKY0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1371. </div><div class="csl-right-inline">Toback S, Galiza E, Cosgrove C, Galloway J, Goodman AL, Swift PA, Rajaram S, Graves-Jones A, Edelman J, Burns F, Minassian AM, Cho I, Kumar L, Plested JS, Rivers EJ, Robertson A, Dubovsky F, Glenn G, Heath PT. 2021. <a href="https://doi.org/10.1101/2021.06.09.21258556">Safety, Immunogenicity, and Efficacy of a COVID-19 Vaccine (NVX-CoV2373) Co-administered With Seasonal Influenza Vaccines</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-rclKBvtk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1372. </div><div class="csl-right-inline">Novavax Initiates Phase 1/2 Clinical Trial of Combination Vaccine for COVID-19 and Seasonal Influenza. Novavax Investor Relations. <a href="https://ir.novavax.com/2021-09-08-Novavax-Initiates-Phase-1-2-Clinical-Trial-of-Combination-Vaccine-for-COVID-19-and-Seasonal-Influenza">https://ir.novavax.com/2021-09-08-Novavax-Initiates-Phase-1-2-Clinical-Trial-of-Combination-Vaccine-for-COVID-19-and-Seasonal-Influenza</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-bOwPRh6q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1373. </div><div class="csl-right-inline">Massare MJ, Patel N, Zhou B, Maciejewski S, Flores R, Guebre-Xabier M, Tian J-H, Portnoff AD, Fries L, Shinde V, Ellingsworth L, Glenn G, Smith G. 2021. <a href="https://doi.org/10.1101/2021.05.05.442782">Combination Respiratory Vaccine Containing Recombinant SARS-CoV-2 Spike and Quadrivalent Seasonal Influenza Hemagglutinin Nanoparticles with Matrix-M Adjuvant</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-yoFGwkUO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1374. </div><div class="csl-right-inline">Raj VS, Mou H, Smits SL, Dekkers DHW, Müller MA, Dijkman R, Muth D, Demmers JAA, Zaki A, Fouchier RAM, Thiel V, Drosten C, Rottier PJM, Osterhaus ADME, Bosch BJ, Haagmans BL. 2013. <a href="https://doi.org/10.1038/nature12005">Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC</a>. Nature 495:251–254.</div>
</div>
<div id="ref-KZFnwRIN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1375. </div><div class="csl-right-inline">Lu G, Hu Y, Wang Q, Qi J, Gao F, Li Y, Zhang Y, Zhang W, Yuan Y, Bao J, Zhang B, Shi Y, Yan J, Gao GF. 2013. <a href="https://doi.org/10.1038/nature12328">Molecular basis of binding between novel human coronavirus MERS-CoV and its receptor CD26</a>. Nature 500:227–231.</div>
</div>
<div id="ref-YZcHzPcs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1376. </div><div class="csl-right-inline">Mou H, Raj VS, van Kuppeveld FJM, Rottier PJM, Haagmans BL, Bosch BJ. 2013. <a href="https://doi.org/10.1128/jvi.01277-13">The Receptor Binding Domain of the New Middle East Respiratory Syndrome Coronavirus Maps to a 231-Residue Region in the Spike Protein That Efficiently Elicits Neutralizing Antibodies</a>. J Virol 87:9379–9383.</div>
</div>
<div id="ref-GFWQw4mi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1377. </div><div class="csl-right-inline">Du L, Kou Z, Ma C, Tao X, Wang L, Zhao G, Chen Y, Yu F, Tseng C-TK, Zhou Y, Jiang S. 2013. <a href="https://doi.org/10.1371/journal.pone.0081587">A Truncated Receptor-Binding Domain of MERS-CoV Spike Protein Potently Inhibits MERS-CoV Infection and Induces Strong Neutralizing Antibody Responses: Implication for Developing Therapeutics and Vaccines</a>. PLoS ONE 8:e81587.</div>
</div>
<div id="ref-SF030Zqn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1378. </div><div class="csl-right-inline">Zhang N, Channappanavar R, Ma C, Wang L, Tang J, Garron T, Tao X, Tasneem S, Lu L, Tseng C-TK, Zhou Y, Perlman S, Jiang S, Du L. 2015. <a href="https://doi.org/10.1038/cmi.2015.03">Identification of an ideal adjuvant for receptor-binding domain-based subunit vaccines against Middle East respiratory syndrome coronavirus</a>. Cell Mol Immunol 13:180–190.</div>
</div>
<div id="ref-103fS7Kz2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1379. </div><div class="csl-right-inline">Netea MG, Domínguez-Andrés J, Barreiro LB, Chavakis T, Divangahi M, Fuchs E, Joosten LAB, van der Meer JWM, Mhlanga MM, Mulder WJM, Riksen NP, Schlitzer A, Schultze JL, Stabell Benn C, Sun JC, Xavier RJ, Latz E. 2020. <a href="https://doi.org/10.1038/s41577-020-0285-6">Defining trained immunity and its role in health and disease</a>. Nature Reviews Immunology 20:375–388.</div>
</div>
<div id="ref-Vu1VILWK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1380. </div><div class="csl-right-inline">Netea MG, Giamarellos-Bourboulis EJ, Domínguez-Andrés J, Curtis N, van Crevel R, van de Veerdonk FL, Bonten M. 2020. <a href="https://doi.org/10.1016/j.cell.2020.04.042">Trained Immunity: a Tool for Reducing Susceptibility to and the Severity of SARS-CoV-2 Infection</a>. Cell 181:969–977.</div>
</div>
<div id="ref-y9IYdfM3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1381. </div><div class="csl-right-inline">Reducing Health Care Workers Absenteeism in Covid-19 Pandemic Through BCG Vaccine - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04328441">https://clinicaltrials.gov/ct2/show/NCT04328441</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-962rELVS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1382. </div><div class="csl-right-inline">Khattab A. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04350931">Application of BCG Vaccine for Immune-prophylaxis Among Egyptian Healthcare Workers During the Pandemic of COVID-19</a>. NCT04350931. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-EuwTWcPi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1383. </div><div class="csl-right-inline">Universidad de Antioquia. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04362124">Performance Evaluation of BCG Vaccination in Healthcare Personnel to Reduce the Severity of SARS-COV-2 Infection in Medellín, Colombia, 2020</a>. NCT04362124. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-dQtUeruv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1384. </div><div class="csl-right-inline">COVID-19: BCG As Therapeutic Vaccine, Transmission Limitation, and Immunoglobulin Enhancement - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04369794">https://clinicaltrials.gov/ct2/show/NCT04369794</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-DjXsPR8O" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1385. </div><div class="csl-right-inline">Using BCG Vaccine to Protect Health Care Workers in the COVID-19 Pandemic - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04373291">https://clinicaltrials.gov/ct2/show/NCT04373291</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-10OE6y3Pv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1386. </div><div class="csl-right-inline">TASK Applied Science. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04379336">Reducing Morbidity and Mortality in Health Care Workers Exposed to SARS-CoV-2 by Enhancing Non-specific Immune Responses Through Bacillus Calmette-Guérin Vaccination, a Randomized Controlled Trial</a>. NCT04379336. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-86OjIybR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1387. </div><div class="csl-right-inline">Assistance Publique - Hôpitaux de Paris. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04384549">Randomized Controlled Trial Evaluating the Efficacy of Vaccination With Bacillus Calmette and Guérin (BCG) in the Prevention of COVID-19 Via the Strengthening of Innate Immunity in Health Care Workers</a>. NCT04384549. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-ITO15LIz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1388. </div><div class="csl-right-inline">Study to Assess VPM1002 in Reducing Healthcare Professionals' Absenteeism in COVID-19 Pandemic - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04387409">https://clinicaltrials.gov/ct2/show/NCT04387409</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-VkZGZxLn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1389. </div><div class="csl-right-inline">Hellenic Institute for the Study of Sepsis. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04414267">A Randomized Clinical Trial for Enhanced Trained Immune Responses Through Bacillus Calmette-Guérin Vaccination to Prevent Infections by COVID-19: The ACTIVATE II Trial</a>. NCT04414267. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-13JVjMfQI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1390. </div><div class="csl-right-inline">Radboud University. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04417335">Reducing Hospital Admission of Elderly in SARS-CoV-2 Pandemic Via the Induction of Trained Immunity by Bacillus Calmette-Guérin Vaccination, a Randomized Controlled Trial</a>. NCT04417335. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-nk2MVsld" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1391. </div><div class="csl-right-inline">Study to Assess VPM1002 in Reducing Hospital Admissions and/or Severe Respiratory Infectious Diseases in Elderly in COVID-19 Pandemic - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04435379">https://clinicaltrials.gov/ct2/show/NCT04435379</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-1E2t9tr8h" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1392. </div><div class="csl-right-inline">Efficacy and Safety of VPM1002 in Reducing SARS-CoV-2 (COVID-19) Infection Rate and Severity - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04439045">https://clinicaltrials.gov/ct2/show/NCT04439045</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-i5k18bpX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1393. </div><div class="csl-right-inline">Giamarellos-Bourboulis EJ, Netea MG, Rovina N, Akinosoglou K, Antoniadou A, Antonakos N, Damoraki G, Gkavogianni T, Adami M-E, Katsaounou P, Ntaganou M, Kyriakopoulou M, Dimopoulos G, Koutsodimitropoulos I, Velissaris D, Koufargyris P, Karageorgos A, Katrini K, Lekakis V, Lupse M, Kotsaki A, Renieris G, Theodoulou D, Panou V, Koukaki E, Koulouris N, Gogos C, Koutsoukou A. 2020. <a href="https://doi.org/10.1016/j.chom.2020.04.009">Complex Immune Dysregulation in COVID-19 Patients with Severe Respiratory Failure</a>. Cell Host &amp; Microbe 27:992–1000.e3.</div>
</div>
<div id="ref-1228YjPRv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1394. </div><div class="csl-right-inline">Park A, Iwasaki A. 2020. <a href="https://doi.org/10.1016/j.chom.2020.05.008">Type I and Type III Interferons – Induction, Signaling, Evasion, and Application to Combat COVID-19</a>. Cell Host &amp; Microbe 27:870–878.</div>
</div>
<div id="ref-gsNWcXHn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1395. </div><div class="csl-right-inline">2021. <a href="https://www.bbc.com/news/world-asia-india-55748124">Covaxin: India approves two Covid vaccines for children under 12</a>. BBC News.</div>
</div>
<div id="ref-I9Ha5JL3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1396. </div><div class="csl-right-inline">Service TN. DCGI grants EUA to Corbevax for those aged 5-12, Covaxin for 6-12 age group. Tribuneindia News Service. <a href="https://www.tribuneindia.com/news/nation/covaxin-cleared-for-6-12-age-group-by-drugs-regulator-389581">https://www.tribuneindia.com/news/nation/covaxin-cleared-for-6-12-age-group-by-drugs-regulator-389581</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-e8pnj0O3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1397. </div><div class="csl-right-inline">2020. <a href="https://www.reuters.com/article/health-coronavirus-novavax-idUSKBN2661PI">Novavax aims for 2 billion COVID-19 vaccine doses with expanded India deal</a>. Reuters.</div>
</div>
<div id="ref-P9mD7Gc9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1398. </div><div class="csl-right-inline">iNCOVACC - Intranasal Vaccine For Covid-19 | Bharat Biotech. <a href="https://www.bharatbiotech.com/intranasal-vaccine.html">https://www.bharatbiotech.com/intranasal-vaccine.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-gdTtuj5e" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1399. </div><div class="csl-right-inline">Wee S-L. 2021. <a href="https://www.nytimes.com/2021/01/25/business/china-covid-19-vaccine-backlash.html">China Wanted to Show Off Its Vaccines. It’s Backfiring.</a> The New York Times.</div>
</div>
<div id="ref-1Bgnim0gX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1400. </div><div class="csl-right-inline">Sharfstein JM, Goodman JL, Borio L. 2021. <a href="https://doi.org/10.1001/jama.2021.1961">The US Regulatory System and COVID-19 Vaccines</a>. JAMA 325:1153.</div>
</div>
<div id="ref-V2Gz2O9G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1401. </div><div class="csl-right-inline">Reuters. 2022. <a href="https://www.reuters.com/business/healthcare-pharmaceuticals/chinese-omicron-specific-mrna-covid-vaccine-candidate-be-trialed-uae-2022-04-30/">Chinese Omicron-specific mRNA COVID vaccine candidate to be trialed in UAE</a>. Reuters.</div>
</div>
<div id="ref-SGWVc4nD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1402. </div><div class="csl-right-inline">2021. Philippines receives COVID-19 vaccine after delays. AP NEWS. <a href="https://apnews.com/article/cabinets-philippines-coronavirus-pandemic-asia-east-asia-e6f826359395951b2c805dbd90f34205">https://apnews.com/article/cabinets-philippines-coronavirus-pandemic-asia-east-asia-e6f826359395951b2c805dbd90f34205</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-12zVLzkpB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1403. </div><div class="csl-right-inline">Dubai CD in T and JM in. China’s Covid-19 Vaccine Makers Struggle to Meet Demand. WSJ. <a href="https://www.wsj.com/articles/chinas-covid-19-vaccine-makers-struggle-to-meet-demand-11612958560">https://www.wsj.com/articles/chinas-covid-19-vaccine-makers-struggle-to-meet-demand-11612958560</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-vTrIB9zS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1404. </div><div class="csl-right-inline">Cohen J. 2014. <a href="https://doi.org/10.1126/science.345.6203.1441">Ebola vaccine: Little and late</a>. Science 345:1441–1442.</div>
</div>
<div id="ref-8uuVgxzA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1405. </div><div class="csl-right-inline">Coller B-AG, Blue J, Das R, Dubey S, Finelli L, Gupta S, Helmond F, Grant-Klein RJ, Liu K, Simon J, Troth S, VanRheenen S, Waterbury J, Wivel A, Wolf J, Heppner DG, Kemp T, Nichols R, Monath TP. 2017. <a href="https://doi.org/10.1016/j.vaccine.2017.05.097">Clinical development of a recombinant Ebola vaccine in the midst of an unprecedented epidemic</a>. Vaccine 35:4465–4469.</div>
</div>
<div id="ref-fQvzeptv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1406. </div><div class="csl-right-inline">Our Story. Moderna. <a href="https://www.modernatx.com/en-US/about-us/our-story?slug=about-us%2Four-story">https://www.modernatx.com/en-US/about-us/our-story?slug=about-us%2Four-story</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1GA95MF2m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1407. </div><div class="csl-right-inline">Thanh Le T, Andreadakis Z, Kumar A, Gómez Román R, Tollefsen S, Saville M, Mayhew S. 2020. <a href="https://doi.org/10.1038/d41573-020-00073-5">The COVID-19 vaccine development landscape</a>. Nat Rev Drug Discov 19:305–306.</div>
</div>
<div id="ref-Vnbw9o3T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1408. </div><div class="csl-right-inline">Voysey M, Clemens SAC, Madhi SA, Weckx LY, Folegatti PM, Aley PK, Angus B, Baillie VL, Barnabas SL, Bhorat QE, Bibi S, Briner C, Cicconi P, Collins AM, Colin-Jones R, Cutland CL, Darton TC, Dheda K, Duncan CJA, Emary KRW, Ewer KJ, Fairlie L, Faust SN, Feng S, Ferreira DM, Finn A, Goodman AL, Green CM, Green CA, Heath PT, Hill C, Hill H, Hirsch I, Hodgson SHC, Izu A, Jackson S, Jenkin D, Joe CCD, Kerridge S, Koen A, Kwatra G, Lazarus R, Lawrie AM, Lelliott A, Libri V, Lillie PJ, Mallory R, Mendes AVA, Milan EP, Minassian AM, McGregor A, Morrison H, Mujadidi YF, Nana A, O’Reilly PJ, Padayachee SD, Pittella A, Plested E, Pollock KM, Ramasamy MN, Rhead S, Schwarzbold AV, Singh N, Smith A, Song R, Snape MD, Sprinz E, Sutherland RK, Tarrant R, Thomson EC, Török ME, Toshner M, Turner DPJ, Vekemans J, Villafana TL, Watson MEE, Williams CJ, Douglas AD, Hill AVS, Lambe T, Gilbert SC, Pollard AJ, Aban M, Abayomi F, Abeyskera K, Aboagye J, Adam M, Adams K, Adamson J, Adelaja YA, Adewetan G, Adlou S, Ahmed K, Akhalwaya Y, Akhalwaya S, Alcock A, Ali A, Allen ER, Allen L, Almeida TCDSC, Alves MPS, Amorim F, Andritsou F, Anslow R, Appleby M, Arbe-Barnes EH, Ariaans MP, Arns B, Arruda L, Azi P, Azi L, Babbage G, Bailey C, Baker KF, Baker M, Baker N, Baker P, Baldwin L, Baleanu I, Bandeira D, Bara A, Barbosa MAS, Barker D, Barlow GD, Barnes E, Barr AS, Barrett JR, Barrett J, Bates L, Batten A, Beadon K, Beales E, Beckley R, Belij-Rammerstorfer S, Bell J, Bellamy D, Bellei N, Belton S, Berg A, Bermejo L, Berrie E, Berry L, Berzenyi D, Beveridge A, Bewley KR, Bexhell H, Bhikha S, Bhorat AE, Bhorat ZE, Bijker E, Birch G, Birch S, Bird A, Bird O, Bisnauthsing K, Bittaye M, Blackstone K, Blackwell L, Bletchly H, Blundell CL, Blundell SR, Bodalia P, Boettger BC, Bolam E, Boland E, Bormans D, Borthwick N, Bowring F, Boyd A, Bradley P, Brenner T, Brown P, Brown C, Brown-O'Sullivan C, Bruce S, Brunt E, Buchan R, Budd W, Bulbulia YA, Bull M, Burbage J, Burhan H, Burn A, Buttigieg KR, Byard N, Cabera Puig I, Calderon G, Calvert A, Camara S, Cao M, Cappuccini F, Cardoso JR, Carr M, Carroll MW, Carson-Stevens A, Carvalho Y de M, Carvalho JAM, Casey HR, Cashen P, Castro T, Castro LC, Cathie K, Cavey A, Cerbino-Neto J, Chadwick J, Chapman D, Charlton S, Chelysheva I, Chester O, Chita S, Cho J-S, Cifuentes L, Clark E, Clark M, Clarke A, Clutterbuck EA, Collins SLK, Conlon CP, Connarty S, Coombes N, Cooper C, Cooper R, Cornelissen L, Corrah T, Cosgrove C, Cox T, Crocker WEM, Crosbie S, Cullen L, Cullen D, Cunha DRMF, Cunningham C, Cuthbertson FC, Da Guarda SNF, da Silva LP, Damratoski BE, Danos Z, Dantas MTDC, Darroch P, Datoo MS, Datta C, Davids M, Davies SL, Davies H, Davis E, Davis J, Davis J, De Nobrega MMD, De Oliveira Kalid LM, Dearlove D, Demissie T, Desai A, Di Marco S, Di Maso C, Dinelli MIS, Dinesh T, Docksey C, Dold C, Dong T, Donnellan FR, Dos Santos T, dos Santos TG, Dos Santos EP, Douglas N, Downing C, Drake J, Drake-Brockman R, Driver K, Drury R, Dunachie SJ, Durham BS, Dutra L, Easom NJW, van Eck S, Edwards M, Edwards NJ, El Muhanna OM, Elias SC, Elmore M, English M, Esmail A, Essack YM, Farmer E, Farooq M, Farrar M, Farrugia L, Faulkner B, Fedosyuk S, Felle S, Feng S, Ferreira Da Silva C, Field S, Fisher R, Flaxman A, Fletcher J, Fofie H, Fok H, Ford KJ, Fowler J, Fraiman PHA, Francis E, Franco MM, Frater J, Freire MSM, Fry SH, Fudge S, Furze J, Fuskova M, Galian-Rubio P, Galiza E, Garlant H, Gavrila M, Geddes A, Gibbons KA, Gilbride C, Gill H, Glynn S, Godwin K, Gokani K, Goldoni UC, Goncalves M, Gonzalez IGS, Goodwin J, Goondiwala A, Gordon-Quayle K, Gorini G, Grab J, Gracie L, Greenland M, Greenwood N, Greffrath J, Groenewald MM, Grossi L, Gupta G, Hackett M, Hallis B, Hamaluba M, Hamilton E, Hamlyn J, Hammersley D, Hanrath AT, Hanumunthadu B, Harris SA, Harris C, Harris T, Harrison TD, Harrison D, Hart TC, Hartnell B, Hassan S, Haughney J, Hawkins S, Hay J, Head I, Henry J, Hermosin Herrera M, Hettle DB, Hill J, Hodges G, Horne E, Hou MM, Houlihan C, Howe E, Howell N, Humphreys J, Humphries HE, Hurley K, Huson C, Hyder-Wright A, Hyams C, Ikram S, Ishwarbhai A, Ivan M, Iveson P, Iyer V, Jackson F, De Jager J, Jaumdally S, Jeffers H, Jesudason N, Jones B, Jones K, Jones E, Jones C, Jorge MR, Jose A, Joshi A, Júnior EAMS, Kadziola J, Kailath R, Kana F, Karampatsas K, Kasanyinga M, Keen J, Kelly EJ, Kelly DM, Kelly D, Kelly S, Kerr D, Kfouri R de Á, Khan L, Khozoee B, Kidd S, Killen A, Kinch J, Kinch P, King LDW, King TB, Kingham L, Klenerman P, Knapper F, Knight JC, Knott D, Koleva S, Lang M, Lang G, Larkworthy CW, Larwood JPJ, Law R, Lazarus EM, Leach A, Lees EA, Lemm N-M, Lessa A, Leung S, Li Y, Lias AM, Liatsikos K, Linder A, Lipworth S, Liu S, Liu X, Lloyd A, Lloyd S, Loew L, Lopez Ramon R, Lora L, Lowthorpe V, Luz K, MacDonald JC, MacGregor G, Madhavan M, Mainwaring DO, Makambwa E, Makinson R, Malahleha M, Malamatsho R, Mallett G, Mansatta K, Maoko T, Mapetla K, Marchevsky NG, Marinou S, Marlow E, Marques GN, Marriott P, Marshall RP, Marshall JL, Martins FJ, Masenya M, Masilela M, Masters SK, Mathew M, Matlebjane H, Matshidiso K, Mazur O, Mazzella A, McCaughan H, McEwan J, McGlashan J, McInroy L, McIntyre Z, McLenaghan D, McRobert N, McSwiggan S, Megson C, Mehdipour S, Meijs W, Mendonça RNÁ, Mentzer AJ, Mirtorabi N, Mitton C, Mnyakeni S, Moghaddas F, Molapo K, Moloi M, Moore M, Moraes-Pinto MI, Moran M, Morey E, Morgans R, Morris S, Morris S, Morris HC, Morselli F, Morshead G, Morter R, Mottal L, Moultrie A, Moya N, Mpelembue M, Msomi S, Mugodi Y, Mukhopadhyay E, Muller J, Munro A, Munro C, Murphy S, Mweu P, Myasaki CH, Naik G, Naker K, Nastouli E, Nazir A, Ndlovu B, Neffa F, Njenga C, Noal H, Noé A, Novaes G, Nugent FL, Nunes G, O'Brien K, O'Connor D, Odam M, Oelofse S, Oguti B, Olchawski V, Oldfield NJ, Oliveira MG, Oliveira C, Oosthuizen A, O'Reilly P, Osborne P, Owen DRJ, Owen L, Owens D, Owino N, Pacurar M, Paiva BVB, Palhares EMF, Palmer S, Parkinson S, Parracho HMRT, Parsons K, Patel D, Patel B, Patel F, Patel K, Patrick-Smith M, Payne RO, Peng Y, Penn EJ, Pennington A, Peralta Alvarez MP, Perring J, Perry N, Perumal R, Petkar S, Philip T, Phillips DJ, Phillips J, Phohu MK, Pickup L, Pieterse S, Piper J, Pipini D, Plank M, Du Plessis J, Pollard S, Pooley J, Pooran A, Poulton I, Powers C, Presa FB, Price DA, Price V, Primeira M, Proud PC, Provstgaard-Morys S, Pueschel S, Pulido D, Quaid S, Rabara R, Radford A, Radia K, Rajapaska D, Rajeswaran T, Ramos ASF, Ramos Lopez F, Rampling T, Rand J, Ratcliffe H, Rawlinson T, Rea D, Rees B, Reiné J, Resuello-Dauti M, Reyes Pabon E, Ribiero CM, Ricamara M, Richter A, Ritchie N, Ritchie AJ, Robbins AJ, Roberts H, Robinson RE, Robinson H, Rocchetti TT, Rocha BP, Roche S, Rollier C, Rose L, Ross Russell AL, Rossouw L, Royal S, Rudiansyah I, Ruiz S, Saich S, Sala C, Sale J, Salman AM, Salvador N, Salvador S, Sampaio M, Samson AD, Sanchez-Gonzalez A, Sanders H, Sanders K, Santos E, Santos Guerra MFS, Satti I, Saunders JE, Saunders C, Sayed A, Schim van der Loeff I, Schmid AB, Schofield E, Screaton G, Seddiqi S, Segireddy RR, Senger R, Serrano S, Shah R, Shaik I, Sharpe HE, Sharrocks K, Shaw R, Shea A, Shepherd A, Shepherd JG, Shiham F, Sidhom E, Silk SE, da Silva Moraes AC, Silva-Junior G, Silva-Reyes L, Silveira AD, Silveira MBV, Sinha J, Skelly DT, Smith DC, Smith N, Smith HE, Smith DJ, Smith CC, Soares A, Soares T, Solórzano C, Sorio GL, Sorley K, Sosa-Rodriguez T, Souza CMCDL, Souza BSDF, Souza AR, Spencer AJ, Spina F, Spoors L, Stafford L, Stamford I, Starinskij I, Stein R, Steven J, Stockdale L, Stockwell LV, Strickland LH, Stuart AC, Sturdy A, Sutton N, Szigeti A, Tahiri-Alaoui A, Tanner R, Taoushanis C, Tarr AW, Taylor K, Taylor U, Taylor IJ, Taylor J, te Water Naude R, Themistocleous Y, Themistocleous A, Thomas M, Thomas K, Thomas TM, Thombrayil A, Thompson F, Thompson A, Thompson K, Thompson A, Thomson J, Thornton-Jones V, Tighe PJ, Tinoco LA, Tiongson G, Tladinyane B, Tomasicchio M, Tomic A, Tonks S, Towner J, Tran N, Tree J, Trillana G, Trinham C, Trivett R, Truby A, Tsheko BL, Turabi A, Turner R, Turner C, Ulaszewska M, Underwood BR, Varughese R, Verbart D, Verheul M, Vichos I, Vieira T, Waddington CS, Walker L, Wallis E, Wand M, Warbick D, Wardell T, Warimwe G, Warren SC, Watkins B, Watson E, Webb S, Webb-Bridges A, Webster A, Welch J, Wells J, West A, White C, White R, Williams P, Williams RL, Winslow R, Woodyer M, Worth AT, Wright D, Wroblewska M, Yao A, Zimmer R, Zizi D, Zuidewind P. 2021. <a href="https://doi.org/10.1016/s0140-6736(20)32661-1">Safety and efficacy of the ChAdOx1 nCoV-19 vaccine (AZD1222) against SARS-CoV-2: an interim analysis of four randomised controlled trials in Brazil, South Africa, and the UK</a>. The Lancet 397:99–111.</div>
</div>
<div id="ref-13wCBLnnu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1409. </div><div class="csl-right-inline">Duan L, Zheng Q, Zhang H, Niu Y, Lou Y, Wang H. 2020. <a href="https://doi.org/10.3389/fimmu.2020.576622">The SARS-CoV-2 Spike Glycoprotein Biosynthesis, Structure, Function, and Antigenicity: Implications for the Design of Spike-Based Vaccine Immunogens</a>. Front Immunol 11.</div>
</div>
<div id="ref-14FBejgLM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1410. </div><div class="csl-right-inline">BioRender. <a href="https://biorender.com/">https://biorender.com/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-EAzPxBbg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1411. </div><div class="csl-right-inline">Human Coronavirus Structure. <a href="https://app.biorender.com/biorender-templates/figures/all/t-5f21e90283765600b08fbe9d-human-coronavirus-structure">https://app.biorender.com/biorender-templates/figures/all/t-5f21e90283765600b08fbe9d-human-coronavirus-structure</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-Jzj97hJh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1412. </div><div class="csl-right-inline">New Images of Novel Coronavirus SARS-CoV-2 Now Available | NIH: National Institute of Allergy and Infectious Diseases. <a href="https://www.niaid.nih.gov/news-events/novel-coronavirus-sarscov2-images">https://www.niaid.nih.gov/news-events/novel-coronavirus-sarscov2-images</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-MCZBJ5sF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1413. </div><div class="csl-right-inline">Rhee JH. 2014. <a href="https://doi.org/10.7774/cevr.2014.3.1.1">Towards Vaccine 3.0: new era opened in vaccine research and industry</a>. Clin Exp Vaccine Res 3:1.</div>
</div>
<div id="ref-fw8IwtHq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1414. </div><div class="csl-right-inline">Seib KL, Zhao X, Rappuoli R. 2012. <a href="https://doi.org/10.1111/j.1469-0691.2012.03939.x">Developing vaccines in the era of genomics: a decade of reverse vaccinology</a>. Clinical Microbiology and Infection 18:109–116.</div>
</div>
<div id="ref-kqerKJKY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1415. </div><div class="csl-right-inline">Liu. 2019. <a href="https://doi.org/10.3390/vaccines7020037">A Comparison of Plasmid DNA and mRNA as Vaccine Technologies</a>. Vaccines 7:37.</div>
</div>
<div id="ref-pWMIo6pD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1416. </div><div class="csl-right-inline">Ellis RW. 1994. <a href="https://doi.org/10.1001/jama.1994.03510360055036">New Vaccine Technologies</a>. JAMA 271:929.</div>
</div>
<div id="ref-uPszIvSj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1417. </div><div class="csl-right-inline">Liu MA. 2003. <a href="https://doi.org/10.1046/j.1365-2796.2003.01140.x">DNA vaccines: a review</a>. J Intern Med 253:402–410.</div>
</div>
<div id="ref-BsrTDzJ2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1418. </div><div class="csl-right-inline">Kutzler MA, Weiner DB. 2008. <a href="https://doi.org/10.1038/nrg2432">DNA vaccines: ready for prime time?</a> Nat Rev Genet 9:776–788.</div>
</div>
<div id="ref-R7Xdh5nH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1419. </div><div class="csl-right-inline">Sternberg A, Naujokat C. 2020. <a href="https://doi.org/10.1016/j.lfs.2020.118056">Structural features of coronavirus SARS-CoV-2 spike protein: Targets for vaccination</a>. Life Sciences 257:118056.</div>
</div>
<div id="ref-3uddYea8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1420. </div><div class="csl-right-inline">Kirchdoerfer RN, Cottrell CA, Wang N, Pallesen J, Yassine HM, Turner HL, Corbett KS, Graham BS, McLellan JS, Ward AB. 2016. <a href="https://doi.org/10.1038/nature17200">Pre-fusion structure of a human coronavirus spike protein</a>. Nature 531:118–121.</div>
</div>
<div id="ref-hIc3bKWe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1421. </div><div class="csl-right-inline">Liu C, Mendonça L, Yang Y, Gao Y, Shen C, Liu J, Ni T, Ju B, Liu C, Tang X, Wei J, Ma X, Zhu Y, Liu W, Xu S, Liu Y, Yuan J, Wu J, Liu Z, Zhang Z, Liu L, Wang P, Zhang P. 2020. <a href="https://doi.org/10.1016/j.str.2020.10.001">The Architecture of Inactivated SARS-CoV-2 with Postfusion Spikes Revealed by Cryo-EM and Cryo-ET</a>. Structure 28:1218–1224.e4.</div>
</div>
<div id="ref-zK0rFpz1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1422. </div><div class="csl-right-inline">Ke Z, Oton J, Qu K, Cortese M, Zila V, McKeane L, Nakane T, Zivanov J, Neufeldt CJ, Cerikan B, Lu JM, Peukes J, Xiong X, Kräusslich H-G, Scheres SHW, Bartenschlager R, Briggs JAG. 2020. <a href="https://doi.org/10.1038/s41586-020-2665-2">Structures and distributions of SARS-CoV-2 spike proteins on intact virions</a>. Nature 588:498–502.</div>
</div>
<div id="ref-lvq9hGmj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1423. </div><div class="csl-right-inline">Hsieh C-L, Goldsmith JA, Schaub JM, DiVenere AM, Kuo H-C, Javanmardi K, Le KC, Wrapp D, Lee AG, Liu Y, Chou C-W, Byrne PO, Hjorth CK, Johnson NV, Ludes-Meyers J, Nguyen AW, Park J, Wang N, Amengor D, Lavinder JJ, Ippolito GC, Maynard JA, Finkelstein IJ, McLellan JS. 2020. <a href="https://doi.org/10.1126/science.abd0826">Structure-based design of prefusion-stabilized SARS-CoV-2 spikes</a>. Science 369:1501–1505.</div>
</div>
<div id="ref-10UC562ga" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1424. </div><div class="csl-right-inline">Bos R, Rutten L, van der Lubbe JEM, Bakkers MJG, Hardenberg G, Wegmann F, Zuijdgeest D, de Wilde AH, Koornneef A, Verwilligen A, van Manen D, Kwaks T, Vogels R, Dalebout TJ, Myeni SK, Kikkert M, Snijder EJ, Li Z, Barouch DH, Vellinga J, Langedijk JPM, Zahn RC, Custers J, Schuitemaker H. 2020. <a href="https://doi.org/10.1038/s41541-020-00243-x">Ad26 vector-based COVID-19 vaccine encoding a prefusion-stabilized SARS-CoV-2 Spike immunogen induces potent humoral and cellular immune responses</a>. npj Vaccines 5.</div>
</div>
<div id="ref-13bVbfc5h" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1425. </div><div class="csl-right-inline">Marrack P, McKee AS, Munks MW. 2009. <a href="https://doi.org/10.1038/nri2510">Towards an understanding of the adjuvant action of aluminium</a>. Nature Reviews Immunology 9:287–293.</div>
</div>
<div id="ref-122h6fIxE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1426. </div><div class="csl-right-inline">Hayashi T, Momota M, Kuroda E, Kusakabe T, Kobari S, Makisaka K, Ohno Y, Suzuki Y, Nakagawa F, Lee MSJ, Coban C, Onodera R, Higashi T, Motoyama K, Ishii KJ, Arima H. 2018. <a href="https://doi.org/10.3389/fimmu.2018.02619">DAMP-Inducing Adjuvant and PAMP Adjuvants Parallelly Enhance Protective Type-2 and Type-1 Immune Responses to Influenza Split Vaccination</a>. Frontiers in Immunology 9:2619.</div>
</div>
<div id="ref-uO0uqhxc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1427. </div><div class="csl-right-inline">Wang Z-B, Xu J. 2020. <a href="https://doi.org/10.3390/vaccines8010128">Better Adjuvants for Better Vaccines: Progress in Adjuvant Delivery Systems, Modifications, and Adjuvant–Antigen Codelivery</a>. Vaccines 8:128.</div>
</div>
<div id="ref-12jFcMeQY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1428. </div><div class="csl-right-inline">Hobernik D, Bros M. 2018. <a href="https://doi.org/10.3390/ijms19113605">DNA Vaccines—How Far From Clinical Use?</a> IJMS 19:3605.</div>
</div>
<div id="ref-fwumPoq1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1429. </div><div class="csl-right-inline">Ghaffarifar F. 2018. <a href="https://doi.org/10.1358/dot.2018.54.5.2807864">Plasmid DNA vaccines: where are we now?</a> Drugs Today 54:315.</div>
</div>
<div id="ref-Wjtx0VXu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1430. </div><div class="csl-right-inline">Glenting J, Wessels S. 2005. <a href="https://doi.org/10.1186/1475-2859-4-26">Ensuring safety of DNA vaccines</a>. Microb Cell Fact 4.</div>
</div>
<div id="ref-5fcD0JWR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1431. </div><div class="csl-right-inline">Williams J. 2013. <a href="https://doi.org/10.3390/vaccines1030225">Vector Design for Improved DNA Vaccine Efficacy, Safety and Production</a>. Vaccines 1:225–249.</div>
</div>
<div id="ref-yriARFOF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1432. </div><div class="csl-right-inline">Lim M, Badruddoza AZM, Firdous J, Azad M, Mannan A, Al-Hilal TA, Cho C-S, Islam MA. 2020. <a href="https://doi.org/10.3390/pharmaceutics12010030">Engineered Nanodelivery Systems to Improve DNA Vaccine Technologies</a>. Pharmaceutics 12:30.</div>
</div>
<div id="ref-XnrBoKVk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1433. </div><div class="csl-right-inline">DNA vaccines. World Health Organization. <a href="https://www.who.int/biologicals/areas/vaccines/dna/en">https://www.who.int/biologicals/areas/vaccines/dna/en</a>. Retrieved 5 August 2022.</div>
</div>
<div id="ref-NcVQyEao" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1434. </div><div class="csl-right-inline">Lapuente D, Stab V, Storcksdieck genannt Bonsmann M, Maaske A, Köster M, Xiao H, Ehrhardt C, Tenbusch M. 2020. <a href="https://doi.org/10.1371/journal.pone.0231138">Innate signalling molecules as genetic adjuvants do not alter the efficacy of a DNA-based influenza A vaccine</a>. PLoS ONE 15:e0231138.</div>
</div>
<div id="ref-3EKs730C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1435. </div><div class="csl-right-inline">Liu MA, Ulmer JB. 2005. <a href="https://doi.org/10.1016/s0065-2660(05)55002-8">Human Clinical Trials of Plasmid DNA Vaccines</a>, p. 25–40. <em>In</em> Advances in Genetics. Elsevier.</div>
</div>
<div id="ref-jPpzjaYO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1436. </div><div class="csl-right-inline">Roy MJ, Wu MS, Barr LJ, Fuller JT, Tussey LG, Speller S, Culp J, Burkholder JK, Swain WF, Dixon RM, Widera G, Vessey R, King A, Ogg G, Gallimore A, Haynes JR, Heydenburg Fuller D. 2000. <a href="https://doi.org/10.1016/s0264-410x(00)00302-9">Induction of antigen-specific CD8+ T cells, T helper cells, and protective levels of antibody in humans by particle-mediated administration of a hepatitis B virus DNA vaccine</a>. Vaccine 19:764–778.</div>
</div>
<div id="ref-11NePN3S6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1437. </div><div class="csl-right-inline">Weiner DB, Nabel GJ. 2018. <a href="https://doi.org/10.1016/b978-0-323-35761-6.00067-5">Development of Gene-Based Vectors for Immunization</a>, p. 1305–1319.e8. <em>In</em> Plotkin's Vaccines. Elsevier.</div>
</div>
<div id="ref-fgs4epPY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1438. </div><div class="csl-right-inline">A. Gómez L, A. Oñate A. 2019. <a href="https://doi.org/10.5772/intechopen.76754">Plasmid-Based DNA Vaccines</a>Plasmid. IntechOpen.</div>
</div>
<div id="ref-Wn7dAdt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1439. </div><div class="csl-right-inline">Eusébio D, Neves AR, Costa D, Biswas S, Alves G, Cui Z, Sousa Â. 2021. <a href="https://doi.org/10.1016/j.drudis.2021.06.008">Methods to improve the immunogenicity of plasmid DNA vaccines</a>. Drug Discovery Today 26:2575–2592.</div>
</div>
<div id="ref-LBoZ0TI7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1440. </div><div class="csl-right-inline">Garver K, LaPatra S, Kurath G. 2005. <a href="https://doi.org/10.3354/dao064013">Efficacy of an infectious hematopoietic necrosis (IHN) virus DNA vaccine in Chinook Oncorhynchus tshawytscha and sockeye O. nerka salmon</a>. Dis Aquat Org 64:13–22.</div>
</div>
<div id="ref-23N627SI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1441. </div><div class="csl-right-inline">Thacker EL, Holtkamp DJ, Khan AS, Brown PA, Draghia-Akli R. 2006. <a href="https://doi.org/10.2527/2006.843733x">Plasmid-mediated growth hormone-releasing hormone efficacy in reducing disease associated with Mycoplasma hyopneumoniae and porcine reproductive and respiratory syndrome virus infection1</a>. Journal of Animal Science 84:733–742.</div>
</div>
<div id="ref-du9lwp7h" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1442. </div><div class="csl-right-inline">Davidson AH, Traub-Dargatz JL, Rodeheaver RM, Ostlund EN, Pedersen DD, Moorhead RG, Stricklin JB, Dewell RD, Roach SD, Long RE, Albers SJ, Callan RJ, Salman MD. 2005. <a href="https://doi.org/10.2460/javma.2005.226.240">Immunologic responses to West Nile virus in vaccinated and clinically affected horses</a>. javma 226:240–245.</div>
</div>
<div id="ref-1AKXh1YvF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1443. </div><div class="csl-right-inline">Langellotti CA, Gammella M, Soria I, Bellusci C, Quattrocchi V, Vermeulen M, Mongini C, Zamorano PI. 2021. <a href="https://doi.org/10.1089/vim.2020.0082">An Improved DNA Vaccine Against Bovine Herpesvirus-1 Using CD40L and a Chemical Adjuvant Induces Specific Cytotoxicity in Mice</a>. Viral Immunology 34:68–78.</div>
</div>
<div id="ref-12LJUhdVJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1444. </div><div class="csl-right-inline">Collins C, Lorenzen N, Collet B. 2019. <a href="https://doi.org/10.1016/j.fsi.2018.07.012">DNA vaccination for finfish aquaculture</a>. Fish &amp;amp; Shellfish Immunology 85:106–125.</div>
</div>
<div id="ref-AfMvzFuk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1445. </div><div class="csl-right-inline">Chakraborty C, Agoramoorthy G. 2020. <a href="https://doi.org/10.1016/j.vaccine.2020.10.056">India’s cost-effective COVID-19 vaccine development initiatives</a>. Vaccine 38:7883–7884.</div>
</div>
<div id="ref-eIn1Qf3N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1446. </div><div class="csl-right-inline">Mallapaty S. 2021. <a href="https://doi.org/10.1038/d41586-021-02385-x">India’s DNA COVID vaccine is a world first – more are coming</a>. Nature 597:161–162.</div>
</div>
<div id="ref-11WPX816v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1447. </div><div class="csl-right-inline">Abbasi J. 2021. <a href="https://doi.org/10.1001/jama.2021.16625">India’s New COVID-19 DNA Vaccine for Adolescents and Adults Is a First</a>. JAMA 326:1365.</div>
</div>
<div id="ref-xuzLfS0y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1448. </div><div class="csl-right-inline">Safety, Tolerability and Immunogenicity of INO-4800 for COVID-19 in Healthy Volunteers - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04336410">https://clinicaltrials.gov/ct2/show/NCT04336410</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-1Hsm2J1sc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1449. </div><div class="csl-right-inline">Diehl MC, Lee JC, Daniels SE, Tebas P, Khan AS, Giffear M, Sardesai NY, Bagarazzi ML. 2013. <a href="https://doi.org/10.4161/hv.24702">Tolerability of intramuscular and intradermal delivery by CELLECTRA® adaptive constant current electroporation device in healthy volunteers</a>. Human Vaccines &amp; Immunotherapeutics 9:2246–2252.</div>
</div>
<div id="ref-H6tWVs5R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1450. </div><div class="csl-right-inline">Sardesai NY, Weiner DB. 2011. <a href="https://doi.org/10.1016/j.coi.2011.03.008">Electroporation delivery of DNA vaccines: prospects for success</a>. Current Opinion in Immunology 23:421–429.</div>
</div>
<div id="ref-4xraQp8j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1451. </div><div class="csl-right-inline">Tebas P, Yang S, Boyer JD, Reuschel EL, Patel A, Christensen-Quick A, Andrade VM, Morrow MP, Kraynyak K, Agnes J, Purwar M, Sylvester A, Pawlicki J, Gillespie E, Maricic I, Zaidi FI, Kim KY, Dia Y, Frase D, Pezzoli P, Schultheis K, Smith TRF, Ramos SJ, McMullan T, Buttigieg K, Carroll MW, Ervin J, Diehl MC, Blackwood E, Mammen MP, Lee J, Dallas MJ, Brown AS, Shea JE, Kim JJ, Weiner DB, Broderick KE, Humeau LM. 2021. <a href="https://doi.org/10.1016/j.eclinm.2020.100689">Safety and immunogenicity of INO-4800 DNA vaccine against SARS-CoV-2: A preliminary report of an open-label, Phase 1 clinical trial</a>. EClinicalMedicine 31:100689.</div>
</div>
<div id="ref-OYnqjMlC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1452. </div><div class="csl-right-inline">Mammen MP Jr., Tebas P, Agnes J, Giffear M, Kraynyak KA, Blackwood E, Amante D, Reuschel EL, Purwar M, Christensen-Quick A, Liu N, Andrade VM, Carter J, Garufi G, Diehl MC, Sylvester A, Morrow MP, Pezzoli P, Kulkarni AJ, Zaidi FI, Frase D, Liaw K, Badie H, Simon KO, Smith TRF, Ramos S, Spitz R, Juba RJ, Lee J, Dallas M, Brown AS, Shea JE, Kim JJ, Weiner DB, Broderick KE, Boyer JD, Humeau LM. 2021. <a href="https://doi.org/10.1101/2021.05.07.21256652">Safety and immunogenicity of INO-4800 DNA vaccine against SARS-CoV-2: a preliminary report of a randomized, blinded, placebo-controlled, Phase 2 clinical trial in adults at high risk of viral exposure</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-pxEE3VEQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1453. </div><div class="csl-right-inline">INOVIO Receives U.S. FDA Authorization to Proceed with INNOVATE Phase 3 Segment for its COVID-19 Vaccine Candidate, INO-4800, in the U.S. <a href="https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Receives-U.S.-FDA-Authorization-to-Proceed-with-INNOVATE-Phase-3-Segment-for-its-COVID-19-Vaccine-Candidate-INO-4800-in-the-U.S/default.aspx">https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Receives-U.S.-FDA-Authorization-to-Proceed-with-INNOVATE-Phase-3-Segment-for-its-COVID-19-Vaccine-Candidate-INO-4800-in-the-U.S/default.aspx</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-R79Wr1hU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1454. </div><div class="csl-right-inline">INOVIO Further Expands INNOVATE Phase 3 Trial for COVID-19 DNA Vaccine Candidate INO-4800 With Regulatory Authorization from India. <a href="https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Further-Expands-INNOVATE-Phase-3-Trial-for-COVID-19-DNA-Vaccine-Candidate-INO-4800-With-Regulatory-Authorization-from-India/default.aspx">https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Further-Expands-INNOVATE-Phase-3-Trial-for-COVID-19-DNA-Vaccine-Candidate-INO-4800-With-Regulatory-Authorization-from-India/default.aspx</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-yraL6YQa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1455. </div><div class="csl-right-inline">INOVIO Expands INNOVATE Phase 3 for INO-4800, its DNA Vaccine Candidate for COVID-19, to include Colombia following Regulatory Authorization. <a href="https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Expands-INNOVATE-Phase-3-for-INO-4800-its-DNA-Vaccine-Candidate-for-COVID-19-to-include-Colombia-following-Regulatory-Authorization/default.aspx">https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Expands-INNOVATE-Phase-3-for-INO-4800-its-DNA-Vaccine-Candidate-for-COVID-19-to-include-Colombia-following-Regulatory-Authorization/default.aspx</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-Wk6spoae" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1456. </div><div class="csl-right-inline">INOVIO Receives Regulatory Authorization to Conduct Phase 3 Efficacy Trial of its COVID-19 DNA Vaccine Candidate, INO-4800, in Mexico. <a href="https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Receives-Regulatory-Authorization-to-Conduct-Phase-3-Efficacy-Trial-of-its-COVID-19-DNA-Vaccine-Candidate-INO-4800-in-Mexico/default.aspx">https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Receives-Regulatory-Authorization-to-Conduct-Phase-3-Efficacy-Trial-of-its-COVID-19-DNA-Vaccine-Candidate-INO-4800-in-Mexico/default.aspx</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1CCmltPec" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1457. </div><div class="csl-right-inline">Momin T, Kansagra K, Patel H, Sharma S, Sharma B, Patel J, Mittal R, Sanmukhani J, Maithal K, Dey A, Chandra H, Rajanathan CT, Pericherla HP, Kumar P, Narkhede A, Parmar D. 2021. <a href="https://doi.org/10.1016/j.eclinm.2021.101020">Safety and Immunogenicity of a DNA SARS-CoV-2 vaccine (ZyCoV-D): Results of an open-label, non-randomized phase I part of phase I/II clinical study by intradermal route in healthy subjects in India</a>. EClinicalMedicine 38:101020.</div>
</div>
<div id="ref-3xlXzOoW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1458. </div><div class="csl-right-inline">Khobragade A, Bhate S, Ramaiah V, Deshpande S, Giri K, Phophle H, Supe P, Godara I, Revanna R, Nagarkar R, Sanmukhani J, Dey A, Rajanathan TMC, Kansagra K, Koradia P. 2022. <a href="https://doi.org/10.1016/s0140-6736(22)00151-9">Efficacy, safety, and immunogenicity of the DNA SARS-CoV-2 vaccine (ZyCoV-D): the interim efficacy results of a phase 3, randomised, double-blind, placebo-controlled study in India</a>. The Lancet 399:1313–1321.</div>
</div>
<div id="ref-15gaai8Xh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1459. </div><div class="csl-right-inline">Zydus Cadila: ZyCoV-D – COVID19 Vaccine Tracker. <a href="https://covid19.trackvaccines.org/vaccines/29/">https://covid19.trackvaccines.org/vaccines/29/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-BTB3S3sT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1460. </div><div class="csl-right-inline">2021. Covishield, Covaxin, ZyCoV-D Makers to Assess Efficacy of Vaccine as They Await Data on Omicron. News18. <a href="https://www.news18.com/news/india/covishield-covaxin-zycov-d-makers-to-assess-efficacy-of-their-vaccine-as-they-await-data-on-omicron-4504295.html">https://www.news18.com/news/india/covishield-covaxin-zycov-d-makers-to-assess-efficacy-of-their-vaccine-as-they-await-data-on-omicron-4504295.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-mbBuH8XY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1461. </div><div class="csl-right-inline">Andrade VM, Christensen-Quick A, Agnes J, Tur J, Reed C, Kalia R, Marrero I, Elwood D, Schultheis K, Purwar M, Reuschel E, McMullan T, Pezzoli P, Kraynyak K, Sylvester A, Mammen MP, Tebas P, Joseph Kim J, Weiner DB, Smith TRF, Ramos SJ, Humeau LM, Boyer JD, Broderick KE. 2021. <a href="https://doi.org/10.1038/s41541-021-00384-7">INO-4800 DNA vaccine induces neutralizing antibodies and T cell activity against global SARS-CoV-2 variants</a>. npj Vaccines 6.</div>
</div>
<div id="ref-CSlbNoGU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1462. </div><div class="csl-right-inline">Kraynyak KA, Blackwood E, Agnes J, Tebas P, Giffear M, Amante D, Reuschel EL, Purwar M, Christensen-Quick A, Liu N, Andrade VM, Diehl MC, Wani S, Lupicka M, Sylvester A, Morrow MP, Pezzoli P, McMullan T, Kulkarni AJ, Zaidi FI, Frase D, Liaw K, Smith TRF, Ramos SJ, Ervin J, Adams M, Lee J, Dallas M, Shah Brown A, Shea JE, Kim JJ, Weiner DB, Broderick KE, Humeau LM, Boyer JD, Mammen MP Jr. 2022. <a href="https://doi.org/10.1093/infdis/jiac016">SARS-CoV-2 DNA Vaccine INO-4800 Induces Durable Immune Responses Capable of Being Boosted in a Phase 1 Open-Label Trial</a>. The Journal of Infectious Diseases 225:1923–1932.</div>
</div>
<div id="ref-ysgD4Dcf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1463. </div><div class="csl-right-inline">Inc IP. INOVIO Announces Strategy to Address Omicron (B.1.1.529) and Future SARS-CoV-2 Variants. <a href="https://www.prnewswire.com/news-releases/inovio-announces-strategy-to-address-omicron-b1-1-529-and-future-sars-cov-2-variants-301433776.html">https://www.prnewswire.com/news-releases/inovio-announces-strategy-to-address-omicron-b1-1-529-and-future-sars-cov-2-variants-301433776.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-Aynz3sBj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1464. </div><div class="csl-right-inline">Walters JN, Schouest B, Patel A, Reuschel EL, Schultheis K, Parzych E, Maricic I, Gary EN, Purwar M, Andrade VM, Doan A, Elwood D, Eblimit Z, Nguyen B, Frase D, Zaidi FI, Kulkarni A, Generotti A, Joseph Kim J, Humeau LM, Ramos SJ, Smith TRF, Weiner DB, Broderick KE. 2022. <a href="https://doi.org/10.1016/j.vaccine.2022.03.060">Prime-boost vaccination regimens with INO-4800 and INO-4802 augment and broaden immune responses against SARS-CoV-2 in nonhuman primates</a>. Vaccine 40:2960–2969.</div>
</div>
<div id="ref-12zreC1Tk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1465. </div><div class="csl-right-inline">Reed CC, Schultheis K, Andrade VM, Kalia R, Tur J, Schouest B, Elwood D, Walters JN, Maricic I, Doan A, Vazquez M, Eblimit Z, Pezzoli P, Amante D, Porto M, Narvaez B, Lok M, Spence B, Bradette H, Horn H, Yang M, Fader J, Ferrer R, Weiner DB, Kar S, Kim JJ, Humeau LM, Ramos SJ, Smith TRF, Broderick KE. 2021. <a href="https://doi.org/10.1101/2021.05.11.443592">Design, immunogenicity and efficacy of a Pan-SARS-CoV-2 synthetic DNA vaccine</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-1Ff2BDzkT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1466. </div><div class="csl-right-inline">Lauer KB, Borrow R, Blanchard TJ. 2017. <a href="https://doi.org/10.1128/cvi.00298-16">Multivalent and Multipathogen Viral Vector Vaccines</a>. Clin Vaccine Immunol 24.</div>
</div>
<div id="ref-1FpZkxdl4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1467. </div><div class="csl-right-inline">Ewer KJ, Lambe T, Rollier CS, Spencer AJ, Hill AV, Dorrell L. 2016. <a href="https://doi.org/10.1016/j.coi.2016.05.014">Viral vectors as vaccine platforms: from immunogenicity to impact</a>. Current Opinion in Immunology 41:47–54.</div>
</div>
<div id="ref-XRmk1S6R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1468. </div><div class="csl-right-inline">Antrobus RD, Coughlan L, Berthoud TK, Dicks MD, Hill AV, Lambe T, Gilbert SC. 2014. <a href="https://doi.org/10.1038/mt.2013.284">Clinical Assessment of a Novel Recombinant Simian Adenovirus ChAdOx1 as a Vectored Vaccine Expressing Conserved Influenza A Antigens</a>. Molecular Therapy 22:668–674.</div>
</div>
<div id="ref-Jkm7jfS8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1469. </div><div class="csl-right-inline">Al-Kassmy J, Pedersen J, Kobinger G. 2020. <a href="https://doi.org/10.3390/v12080861">Vaccine Candidates against Coronavirus Infections. Where Does COVID-19 Stand?</a> Viruses 12:861.</div>
</div>
<div id="ref-8bpbvIro" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1470. </div><div class="csl-right-inline">Pastoret P-P, Vanderplasschen A. 2003. <a href="https://doi.org/10.1016/s0147-9571(03)00019-5">Poxviruses as vaccine vectors</a>. Comparative Immunology, Microbiology and Infectious Diseases 26:343–355.</div>
</div>
<div id="ref-1AZfAQ5py" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1471. </div><div class="csl-right-inline">García-Arriaza J, Esteban M. 2014. <a href="https://doi.org/10.4161/hv.28974">Enhancing poxvirus vectors vaccine immunogenicity</a>. Human Vaccines &amp;amp; Immunotherapeutics 10:2235–2244.</div>
</div>
<div id="ref-zX5UKhti" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1472. </div><div class="csl-right-inline">Lasaro MO, Ertl HC. 2009. <a href="https://doi.org/10.1038/mt.2009.130">New Insights on Adenovirus as Vaccine Vectors</a>. Molecular Therapy 17:1333–1339.</div>
</div>
<div id="ref-SNwg8Qkf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1473. </div><div class="csl-right-inline">Roberts A, Buonocore L, Price R, Forman J, Rose JK. 1999. <a href="https://doi.org/10.1128/jvi.73.5.3723-3732.1999">Attenuated vesicular stomatitis viruses as vaccine vectors.</a> J Virol 73:3723–32.</div>
</div>
<div id="ref-lvi4DH2g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1474. </div><div class="csl-right-inline">Lichty BD, Power AT, Stojdl DF, Bell JC. 2004. <a href="https://doi.org/10.1016/j.molmed.2004.03.003">Vesicular stomatitis virus: re-inventing the bullet</a>. Trends in Molecular Medicine 10:210–216.</div>
</div>
<div id="ref-YRgRziXN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1475. </div><div class="csl-right-inline">Rollier CS, Reyes-Sandoval A, Cottingham MG, Ewer K, Hill AV. 2011. <a href="https://doi.org/10.1016/j.coi.2011.03.006">Viral vectors as vaccine platforms: deployment in sight</a>. Current Opinion in Immunology 23:377–382.</div>
</div>
<div id="ref-tbs2wD7F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1476. </div><div class="csl-right-inline">Nayak S, Herzog RW. 2009. <a href="https://doi.org/10.1038/gt.2009.148">Progress and prospects: immune responses to viral vectors</a>. Gene Ther 17:295–304.</div>
</div>
<div id="ref-IUplTKEg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1477. </div><div class="csl-right-inline">Ura T, Okuda K, Shimada M. 2014. <a href="https://doi.org/10.3390/vaccines2030624">Developments in Viral Vector-Based Vaccines</a>. Vaccines 2:624–641.</div>
</div>
<div id="ref-BJRcCUIU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1478. </div><div class="csl-right-inline">Moffatt S, Hays J, HogenEsch H, Mittal SK. 2000. <a href="https://doi.org/10.1006/viro.2000.0350">Circumvention of Vector-Specific Neutralizing Antibody Response by Alternating Use of Human and Non-Human Adenoviruses: Implications in Gene Therapy</a>. Virology 272:159–167.</div>
</div>
<div id="ref-gDDhHhRd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1479. </div><div class="csl-right-inline">Fausther-Bovendo H, Kobinger GP. 2014. <a href="https://doi.org/10.4161/hv.29594">Pre-existing immunity against Ad vectors</a>. Human Vaccines &amp;amp; Immunotherapeutics 10:2875–2884.</div>
</div>
<div id="ref-MvKb0qJC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1480. </div><div class="csl-right-inline">Vrba SM, Kirk NM, Brisse ME, Liang Y, Ly H. 2020. <a href="https://doi.org/10.3390/vaccines8040680">Development and Applications of Viral Vectored Vaccines to Combat Zoonotic and Emerging Public Health Threats</a>. Vaccines 8:680.</div>
</div>
<div id="ref-9g5tmszW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1481. </div><div class="csl-right-inline">Ollmann Saphire E. 2020. <a href="https://doi.org/10.1016/j.cell.2020.03.011">A Vaccine against Ebola Virus</a>. Cell 181:6.</div>
</div>
<div id="ref-OZJWUaDW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1482. </div><div class="csl-right-inline">Bliss CM, Drammeh A, Bowyer G, Sanou GS, Jagne YJ, Ouedraogo O, Edwards NJ, Tarama C, Ouedraogo N, Ouedraogo M, Njie-Jobe J, Diarra A, Afolabi MO, Tiono AB, Yaro JB, Adetifa UJ, Hodgson SH, Anagnostou NA, Roberts R, Duncan CJA, Cortese R, Viebig NK, Leroy O, Lawrie AM, Flanagan KL, Kampmann B, Imoukhuede EB, Sirima SB, Bojang K, Hill AVS, Nébié I, Ewer KJ. 2017. <a href="https://doi.org/10.1016/j.ymthe.2016.11.003">Viral Vector Malaria Vaccines Induce High-Level T Cell and Antibody Responses in West African Children and Infants</a>. Molecular Therapy 25:547–559.</div>
</div>
<div id="ref-3tkGuMXx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1483. </div><div class="csl-right-inline">Li S, Locke E, Bruder J, Clarke D, Doolan DL, Havenga MJE, Hill AVS, Liljestrom P, Monath TP, Naim HY, Ockenhouse C, Tang DC, Van Kampen KR, Viret J-F, Zavala F, Dubovsky F. 2007. <a href="https://doi.org/10.1016/j.vaccine.2006.07.035">Viral vectors for malaria vaccine development</a>. Vaccine 25:2567–2574.</div>
</div>
<div id="ref-AgZwwt5u" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1484. </div><div class="csl-right-inline">Ledgerwood JE, DeZure AD, Stanley DA, Coates EE, Novik L, Enama ME, Berkowitz NM, Hu Z, Joshi G, Ploquin A, Sitar S, Gordon IJ, Plummer SA, Holman LA, Hendel CS, Yamshchikov G, Roman F, Nicosia A, Colloca S, Cortese R, Bailer RT, Schwartz RM, Roederer M, Mascola JR, Koup RA, Sullivan NJ, Graham BS. 2017. <a href="https://doi.org/10.1056/nejmoa1410863">Chimpanzee Adenovirus Vector Ebola Vaccine</a>. N Engl J Med 376:928–938.</div>
</div>
<div id="ref-9BEMTYn8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1485. </div><div class="csl-right-inline">Geisbert TW, Feldmann H. 2011. <a href="https://doi.org/10.1093/infdis/jir349">Recombinant Vesicular Stomatitis Virus–Based Vaccines Against Ebola and Marburg Virus Infections</a>. The Journal of Infectious Diseases 204:S1075–S1081.</div>
</div>
<div id="ref-PbGQOOI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1486. </div><div class="csl-right-inline">Marzi A, Feldmann H. 2014. <a href="https://doi.org/10.1586/14760584.2014.885841">Ebola virus vaccines: an overview of current approaches</a>. Expert Review of Vaccines 13:521–531.</div>
</div>
<div id="ref-1C8hgfvDF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1487. </div><div class="csl-right-inline">Parks CL, Picker LJ, King CR. 2013. <a href="https://doi.org/10.1097/coh.0b013e328363d389">Development of replication-competent viral vectors for HIV vaccine delivery</a>. Current Opinion in HIV and AIDS 8:402–411.</div>
</div>
<div id="ref-SAIfGNkZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1488. </div><div class="csl-right-inline">Trivedi S, Jackson RJ, Ranasinghe C. 2014. <a href="https://doi.org/10.1016/j.virol.2014.09.004">Different HIV pox viral vector-based vaccines and adjuvants can induce unique antigen presenting cells that modulate CD8 T cell avidity</a>. Virology 468-470:479–489.</div>
</div>
<div id="ref-umEOWDY5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1489. </div><div class="csl-right-inline">See RH, Zakhartchouk AN, Petric M, Lawrence DJ, Mok CPY, Hogan RJ, Rowe T, Zitzow LA, Karunakaran KP, Hitt MM, Graham FL, Prevec L, Mahony JB, Sharon C, Auperin TC, Rini JM, Tingle AJ, Scheifele DW, Skowronski DM, Patrick DM, Voss TG, Babiuk LA, Gauldie J, Roper RL, Brunham RC, Finlay BB. 2006. <a href="https://doi.org/10.1099/vir.0.81579-0">Comparative evaluation of two severe acute respiratory syndrome (SARS) vaccine candidates in mice challenged with SARS coronavirus</a>. Journal of General Virology 87:641–650.</div>
</div>
<div id="ref-DGTFML2b" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1490. </div><div class="csl-right-inline">See RH, Petric M, Lawrence DJ, Mok CPY, Rowe T, Zitzow LA, Karunakaran KP, Voss TG, Brunham RC, Gauldie J, Finlay BB, Roper RL. 2008. <a href="https://doi.org/10.1099/vir.0.2008/001891-0">Severe acute respiratory syndrome vaccine efficacy in ferrets: whole killed virus and adenovirus-vectored vaccines</a>. Journal of General Virology 89:2136–2146.</div>
</div>
<div id="ref-UCI0TCHy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1491. </div><div class="csl-right-inline">Yeung HT. 2018. Update with the development of Ebola vaccines and implications of emerging evidence to inform future policy recommendations. World Health Organization SAGE meeting background. <a href="https://www.who.int/immunization/sage/meetings/2018/october/2_Ebola_SAGE2018Oct_BgDoc_20180919.pdf">https://www.who.int/immunization/sage/meetings/2018/october/2_Ebola_SAGE2018Oct_BgDoc_20180919.pdf</a>.</div>
</div>
<div id="ref-P94sxWp4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1492. </div><div class="csl-right-inline">Alharbi NK, Padron-Regalado E, Thompson CP, Kupke A, Wells D, Sloan MA, Grehan K, Temperton N, Lambe T, Warimwe G, Becker S, Hill AVS, Gilbert SC. 2017. <a href="https://doi.org/10.1016/j.vaccine.2017.05.032">ChAdOx1 and MVA based vaccine candidates against MERS-CoV elicit neutralising antibodies and cellular immune responses in mice</a>. Vaccine 35:3780–3788.</div>
</div>
<div id="ref-3NtMBDMM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1493. </div><div class="csl-right-inline">van Doremalen N, Haddock E, Feldmann F, Meade-White K, Bushmaker T, Fischer RJ, Okumura A, Hanley PW, Saturday G, Edwards NJ, Clark MHA, Lambe T, Gilbert SC, Munster VJ. 2020. <a href="https://doi.org/10.1126/sciadv.aba8399">A single dose of ChAdOx1 MERS provides protective immunity in rhesus macaques</a>. Sci Adv 6.</div>
</div>
<div id="ref-ERfSJf5B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1494. </div><div class="csl-right-inline">Folegatti PM, Bittaye M, Flaxman A, Lopez FR, Bellamy D, Kupke A, Mair C, Makinson R, Sheridan J, Rohde C, Halwe S, Jeong Y, Park Y-S, Kim J-O, Song M, Boyd A, Tran N, Silman D, Poulton I, Datoo M, Marshall J, Themistocleous Y, Lawrie A, Roberts R, Berrie E, Becker S, Lambe T, Hill A, Ewer K, Gilbert S. 2020. <a href="https://doi.org/10.1016/s1473-3099(20)30160-2">Safety and immunogenicity of a candidate Middle East respiratory syndrome coronavirus viral-vectored vaccine: a dose-escalation, open-label, non-randomised, uncontrolled, phase 1 trial</a>. The Lancet Infectious Diseases 20:816–826.</div>
</div>
<div id="ref-1037p4Gvs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1495. </div><div class="csl-right-inline">van Doremalen N, Lambe T, Spencer A, Belij-Rammerstorfer S, Purushotham JN, Port JR, Avanzato VA, Bushmaker T, Flaxman A, Ulaszewska M, Feldmann F, Allen ER, Sharpe H, Schulz J, Holbrook M, Okumura A, Meade-White K, Pérez-Pérez L, Edwards NJ, Wright D, Bissett C, Gilbride C, Williamson BN, Rosenke R, Long D, Ishwarbhai A, Kailath R, Rose L, Morris S, Powers C, Lovaglio J, Hanley PW, Scott D, Saturday G, de Wit E, Gilbert SC, Munster VJ. 2020. <a href="https://doi.org/10.1038/s41586-020-2608-y">ChAdOx1 nCoV-19 vaccine prevents SARS-CoV-2 pneumonia in rhesus macaques</a>. Nature 586:578–582.</div>
</div>
<div id="ref-2bBVSpM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1496. </div><div class="csl-right-inline">Folegatti PM, Ewer KJ, Aley PK, Angus B, Becker S, Belij-Rammerstorfer S, Bellamy D, Bibi S, Bittaye M, Clutterbuck EA, Dold C, Faust SN, Finn A, Flaxman AL, Hallis B, Heath P, Jenkin D, Lazarus R, Makinson R, Minassian AM, Pollock KM, Ramasamy M, Robinson H, Snape M, Tarrant R, Voysey M, Green C, Douglas AD, Hill AVS, Lambe T, Gilbert SC, Pollard AJ, Aboagye J, Adams K, Ali A, Allen E, Allison JL, Anslow R, Arbe-Barnes EH, Babbage G, Baillie K, Baker M, Baker N, Baker P, Baleanu I, Ballaminut J, Barnes E, Barrett J, Bates L, Batten A, Beadon K, Beckley R, Berrie E, Berry L, Beveridge A, Bewley KR, Bijker EM, Bingham T, Blackwell L, Blundell CL, Bolam E, Boland E, Borthwick N, Bower T, Boyd A, Brenner T, Bright PD, Brown-O'Sullivan C, Brunt E, Burbage J, Burge S, Buttigieg KR, Byard N, Cabera Puig I, Calvert A, Camara S, Cao M, Cappuccini F, Carr M, Carroll MW, Carter V, Cathie K, Challis RJ, Charlton S, Chelysheva I, Cho J-S, Cicconi P, Cifuentes L, Clark H, Clark E, Cole T, Colin-Jones R, Conlon CP, Cook A, Coombes NS, Cooper R, Cosgrove CA, Coy K, Crocker WEM, Cunningham CJ, Damratoski BE, Dando L, Datoo MS, Davies H, De Graaf H, Demissie T, Di Maso C, Dietrich I, Dong T, Donnellan FR, Douglas N, Downing C, Drake J, Drake-Brockman R, Drury RE, Dunachie SJ, Edwards NJ, Edwards FDL, Edwards CJ, Elias SC, Elmore MJ, Emary KRW, English MR, Fagerbrink S, Felle S, Feng S, Field S, Fixmer C, Fletcher C, Ford KJ, Fowler J, Fox P, Francis E, Frater J, Furze J, Fuskova M, Galiza E, Gbesemete D, Gilbride C, Godwin K, Gorini G, Goulston L, Grabau C, Gracie L, Gray Z, Guthrie LB, Hackett M, Halwe S, Hamilton E, Hamlyn J, Hanumunthadu B, Harding I, Harris SA, Harris A, Harrison D, Harrison C, Hart TC, Haskell L, Hawkins S, Head I, Henry JA, Hill J, Hodgson SHC, Hou MM, Howe E, Howell N, Hutlin C, Ikram S, Isitt C, Iveson P, Jackson S, Jackson F, James SW, Jenkins M, Jones E, Jones K, Jones CE, Jones B, Kailath R, Karampatsas K, Keen J, Kelly S, Kelly D, Kerr D, Kerridge S, Khan L, Khan U, Killen A, Kinch J, King TB, King L, King J, Kingham-Page L, Klenerman P, Knapper F, Knight JC, Knott D, Koleva S, Kupke A, Larkworthy CW, Larwood JPJ, Laskey A, Lawrie AM, Lee A, Ngan Lee KY, Lees EA, Legge H, Lelliott A, Lemm N-M, Lias AM, Linder A, Lipworth S, Liu X, Liu S, Lopez Ramon R, Lwin M, Mabesa F, Madhavan M, Mallett G, Mansatta K, Marcal I, Marinou S, Marlow E, Marshall JL, Martin J, McEwan J, McInroy L, Meddaugh G, Mentzer AJ, Mirtorabi N, Moore M, Moran E, Morey E, Morgan V, Morris SJ, Morrison H, Morshead G, Morter R, Mujadidi YF, Muller J, Munera-Huertas T, Munro C, Munro A, Murphy S, Munster VJ, Mweu P, Noé A, Nugent FL, Nuthall E, O'Brien K, O'Connor D, Oguti B, Oliver JL, Oliveira C, O'Reilly PJ, Osborn M, Osborne P, Owen C, Owens D, Owino N, Pacurar M, Parker K, Parracho H, Patrick-Smith M, Payne V, Pearce J, Peng Y, Peralta Alvarez MP, Perring J, Pfafferott K, Pipini D, Plested E, Pluess-Hall H, Pollock K, Poulton I, Presland L, Provstgaard-Morys S, Pulido D, Radia K, Ramos Lopez F, Rand J, Ratcliffe H, Rawlinson T, Rhead S, Riddell A, Ritchie AJ, Roberts H, Robson J, Roche S, Rohde C, Rollier CS, Romani R, Rudiansyah I, Saich S, Sajjad S, Salvador S, Sanchez Riera L, Sanders H, Sanders K, Sapaun S, Sayce C, Schofield E, Screaton G, Selby B, Semple C, Sharpe HR, Shaik I, Shea A, Shelton H, Silk S, Silva-Reyes L, Skelly DT, Smee H, Smith CC, Smith DJ, Song R, Spencer AJ, Stafford E, Steele A, Stefanova E, Stockdale L, Szigeti A, Tahiri-Alaoui A, Tait M, Talbot H, Tanner R, Taylor IJ, Taylor V, Te Water Naude R, Thakur N, Themistocleous Y, Themistocleous A, Thomas M, Thomas TM, Thompson A, Thomson-Hill S, Tomlins J, Tonks S, Towner J, Tran N, Tree JA, Truby A, Turkentine K, Turner C, Turner N, Turner S, Tuthill T, Ulaszewska M, Varughese R, Van Doremalen N, Veighey K, Verheul MK, Vichos I, Vitale E, Walker L, Watson MEE, Welham B, Wheat J, White C, White R, Worth AT, Wright D, Wright S, Yao XL, Yau Y. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)31604-4">Safety and immunogenicity of the ChAdOx1 nCoV-19 vaccine against SARS-CoV-2: a preliminary report of a phase 1/2, single-blind, randomised controlled trial</a>. The Lancet 396:467–478.</div>
</div>
<div id="ref-sRAZYY9C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1497. </div><div class="csl-right-inline">Jones I, Roy P. 2021. <a href="https://doi.org/10.1016/s0140-6736(21)00191-4">Sputnik V COVID-19 vaccine candidate appears safe and effective</a>. The Lancet 397:642–643.</div>
</div>
<div id="ref-8jwp261S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1498. </div><div class="csl-right-inline">Barouch DH, Kik SV, Weverling GJ, Dilan R, King SL, Maxfield LF, Clark S, Ng’ang’a D, Brandariz KL, Abbink P, Sinangil F, de Bruyn G, Gray GE, Roux S, Bekker L-G, Dilraj A, Kibuuka H, Robb ML, Michael NL, Anzala O, Amornkul PN, Gilmour J, Hural J, Buchbinder SP, Seaman MS, Dolin R, Baden LR, Carville A, Mansfield KG, Pau MG, Goudsmit J. 2011. <a href="https://doi.org/10.1016/j.vaccine.2011.05.025">International seroepidemiology of adenovirus serotypes 5, 26, 35, and 48 in pediatric and adult populations</a>. Vaccine 29:5203–5209.</div>
</div>
<div id="ref-D3Px25HN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1499. </div><div class="csl-right-inline">Office USGA. Operation Warp Speed: Accelerated COVID-19 Vaccine Development Status and Efforts to Address Manufacturing Challenges. <a href="https://www.gao.gov/products/gao-21-319">https://www.gao.gov/products/gao-21-319</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-57BTbcko" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1500. </div><div class="csl-right-inline">Johnson &amp; Johnson Announces a Lead Vaccine Candidate for COVID-19; Landmark New Partnership with U.S. Department of Health &amp; Human Services; and Commitment to Supply One Billion Vaccines Worldwide for Emergency Pandemic Use | Johnson &amp; Johnson. Content Lab US. <a href="https://www.jnj.com/johnson-johnson-announces-a-lead-vaccine-candidate-for-covid-19-landmark-new-partnership-with-u-s-department-of-health-human-services-and-commitment-to-supply-one-billion-vaccines-worldwide-for-emergency-pandemic-use">https://www.jnj.com/johnson-johnson-announces-a-lead-vaccine-candidate-for-covid-19-landmark-new-partnership-with-u-s-department-of-health-human-services-and-commitment-to-supply-one-billion-vaccines-worldwide-for-emergency-pandemic-use</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-pWf2T8J8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1501. </div><div class="csl-right-inline">Sadoff J, Le Gars M, Shukarev G, Heerwegh D, Truyers C, de Groot AM, Stoop J, Tete S, Van Damme W, Leroux-Roels I, Berghmans P-J, Kimmel M, Van Damme P, de Hoon J, Smith W, Stephenson KE, De Rosa SC, Cohen KW, McElrath MJ, Cormier E, Scheper G, Barouch DH, Hendriks J, Struyf F, Douoguih M, Van Hoof J, Schuitemaker H. 2021. <a href="https://doi.org/10.1056/nejmoa2034201">Interim Results of a Phase 1–2a Trial of Ad26.COV2.S Covid-19 Vaccine</a>. N Engl J Med 384:1824–1835.</div>
</div>
<div id="ref-gOOBv1MD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1502. </div><div class="csl-right-inline">Mercado NB, Zahn R, Wegmann F, Loos C, Chandrashekar A, Yu J, Liu J, Peter L, McMahan K, Tostanoski LH, He X, Martinez DR, Rutten L, Bos R, van Manen D, Vellinga J, Custers J, Langedijk JP, Kwaks T, Bakkers MJG, Zuijdgeest D, Rosendahl Huber SK, Atyeo C, Fischinger S, Burke JS, Feldman J, Hauser BM, Caradonna TM, Bondzie EA, Dagotto G, Gebre MS, Hoffman E, Jacob-Dolan C, Kirilova M, Li Z, Lin Z, Mahrokhian SH, Maxfield LF, Nampanya F, Nityanandam R, Nkolola JP, Patel S, Ventura JD, Verrington K, Wan H, Pessaint L, Van Ry A, Blade K, Strasbaugh A, Cabus M, Brown R, Cook A, Zouantchangadou S, Teow E, Andersen H, Lewis MG, Cai Y, Chen B, Schmidt AG, Reeves RK, Baric RS, Lauffenburger DA, Alter G, Stoffels P, Mammen M, Van Hoof J, Schuitemaker H, Barouch DH. 2020. <a href="https://doi.org/10.1038/s41586-020-2607-z">Single-shot Ad26 vaccine protects against SARS-CoV-2 in rhesus macaques</a>. Nature 586:583–588.</div>
</div>
<div id="ref-HmMIiIv2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1503. </div><div class="csl-right-inline">Tostanoski LH, Wegmann F, Martinot AJ, Loos C, McMahan K, Mercado NB, Yu J, Chan CN, Bondoc S, Starke CE, Nekorchuk M, Busman-Sahay K, Piedra-Mora C, Wrijil LM, Ducat S, Custers J, Atyeo C, Fischinger S, Burke JS, Feldman J, Hauser BM, Caradonna TM, Bondzie EA, Dagotto G, Gebre MS, Jacob-Dolan C, Lin Z, Mahrokhian SH, Nampanya F, Nityanandam R, Pessaint L, Porto M, Ali V, Benetiene D, Tevi K, Andersen H, Lewis MG, Schmidt AG, Lauffenburger DA, Alter G, Estes JD, Schuitemaker H, Zahn R, Barouch DH. 2020. <a href="https://doi.org/10.1038/s41591-020-1070-6">Ad26 vaccine protects against SARS-CoV-2 severe clinical disease in hamsters</a>. Nat Med 26:1694–1700.</div>
</div>
<div id="ref-EpOXYGt4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1504. </div><div class="csl-right-inline">Solforosi L, Kuipers H, Huber SKR, van der Lubbe JEM, Dekking L, Czapska-Casey DN, Gil AI, Baert MRM, Drijver J, Vaneman J, van Huizen E, Choi Y, Vreugdenhil J, Kroos S, de Wilde AH, Kourkouta E, Custers J, Dalebout TJ, Myeni SK, Kikkert M, Snijder EJ, Barouch DH, Böszörményi KP, Stammes MA, Kondova I, Verschoor EJ, Verstrepen BE, Koopman G, Mooij P, Bogers WMJM, van Heerden M, Muchene L, Tolboom JTBM, Roozendaal R, Schuitemaker H, Wegmann F, Zahn RC. 2020. <a href="https://doi.org/10.1101/2020.11.17.368258">Immunogenicity and protective efficacy of one- and two-dose regimens of the Ad26.COV2.S COVID-19 vaccine candidate in adult and aged rhesus macaques</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-HGVDPMLm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1505. </div><div class="csl-right-inline">Roozendaal R, Solforosi L, Stieh D, Serroyen J, Straetemans R, Wegmann F, Rosendahl Huber SK, van der Lubbe JEM, Hendriks J, le Gars M, Dekking L, Czapska-Casey DN, Guimera N, Janssen S, Tete S, Chandrashekar A, Mercado N, Yu J, Koudstaal W, Sadoff J, Barouch DH, Schuitemaker H, Zahn R. 2021. <a href="https://doi.org/10.1101/2021.01.30.428921">SARS-CoV-2 binding and neutralizing antibody levels after vaccination with Ad26.COV2.S predict durable protection in rhesus macaques</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-69GoEX0X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1506. </div><div class="csl-right-inline">Sadoff J, Gray G, Vandebosch A, Cárdenas V, Shukarev G, Grinsztejn B, Goepfert PA, Truyers C, Van Dromme I, Spiessens B, Vingerhoets J, Custers J, Scheper G, Robb ML, Treanor J, Ryser MF, Barouch DH, Swann E, Marovich MA, Neuzil KM, Corey L, Stoddard J, Hardt K, Ruiz-Guiñazú J, Le Gars M, Schuitemaker H, Van Hoof J, Struyf F, Douoguih M. 2022. <a href="https://doi.org/10.1056/nejmoa2117608">Final Analysis of Efficacy and Safety of Single-Dose Ad26.COV2.S</a>. N Engl J Med 386:847–860.</div>
</div>
<div id="ref-1A7PjhDDR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1507. </div><div class="csl-right-inline">AstraZeneca’s COVID-19 vaccine authorised for emergency supply in the UK. <a href="https://www.astrazeneca.com/media-centre/press-releases/2020/astrazenecas-covid-19-vaccine-authorised-in-uk.html">https://www.astrazeneca.com/media-centre/press-releases/2020/astrazenecas-covid-19-vaccine-authorised-in-uk.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-X5LkVfY6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1508. </div><div class="csl-right-inline">The Brussels Times. <a href="https://www.brusselstimes.com/news-contents/world/149039/1-5-million-people-have-received-sputnik-v-vaccine-russia-says-russian-direct-investment-fund-mikhail-murashko">https://www.brusselstimes.com/news-contents/world/149039/1-5-million-people-have-received-sputnik-v-vaccine-russia-says-russian-direct-investment-fund-mikhail-murashko</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-16LczMwFO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1509. </div><div class="csl-right-inline">Daventry M. 2021. Hungary becomes first EU country to deploy Russia's COVID-19 vaccine. euronews. <a href="https://www.euronews.com/2021/02/12/hungary-to-begin-using-russia-s-sputnik-v-vaccine-today">https://www.euronews.com/2021/02/12/hungary-to-begin-using-russia-s-sputnik-v-vaccine-today</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-Z0V7NK7Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1510. </div><div class="csl-right-inline">2021. San Marino buys Russia's Sputnik V after EU vaccine delivery delays. euronews. <a href="https://www.euronews.com/2021/02/24/san-marino-buys-russia-s-sputnik-v-after-eu-vaccine-delivery-delays">https://www.euronews.com/2021/02/24/san-marino-buys-russia-s-sputnik-v-after-eu-vaccine-delivery-delays</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-16GYKbrOq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1511. </div><div class="csl-right-inline">AFP. 2020. Belarus Starts Coronavirus Vaccination With Sputnik V. The Moscow Times. <a href="https://www.themoscowtimes.com/2020/12/29/belarus-starts-coronavirus-vaccination-with-sputnik-v-a72512">https://www.themoscowtimes.com/2020/12/29/belarus-starts-coronavirus-vaccination-with-sputnik-v-a72512</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-gLAIyAHm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1512. </div><div class="csl-right-inline">Logunov DY, Dolzhikova IV, Shcheblyakov DV, Tukhvatulin AI, Zubkova OV, Dzharullaeva AS, Kovyrshina AV, Lubenets NL, Grousova DM, Erokhova AS, Botikov AG, Izhaeva FM, Popova O, Ozharovskaya TA, Esmagambetov IB, Favorskaya IA, Zrelkin DI, Voronina DV, Shcherbinin DN, Semikhin AS, Simakova YV, Tokarskaya EA, Egorova DA, Shmarov MM, Nikitenko NA, Gushchin VA, Smolyarchuk EA, Zyryanov SK, Borisevich SV, Naroditsky BS, Gintsburg AL. 2021. <a href="https://doi.org/10.1016/s0140-6736(21)00234-8">Safety and efficacy of an rAd26 and rAd5 vector-based heterologous prime-boost COVID-19 vaccine: an interim analysis of a randomised controlled phase 3 trial in Russia</a>. The Lancet 397:671–681.</div>
</div>
<div id="ref-GOZYHZz0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1513. </div><div class="csl-right-inline">Sadoff J, Gray G, Vandebosch A, Cárdenas V, Shukarev G, Grinsztejn B, Goepfert PA, Truyers C, Fennema H, Spiessens B, Offergeld K, Scheper G, Taylor KL, Robb ML, Treanor J, Barouch DH, Stoddard J, Ryser MF, Marovich MA, Neuzil KM, Corey L, Cauwenberghs N, Tanner T, Hardt K, Ruiz-Guiñazú J, Le Gars M, Schuitemaker H, Van Hoof J, Struyf F, Douoguih M. 2021. <a href="https://doi.org/10.1056/nejmoa2101544">Safety and Efficacy of Single-Dose Ad26.COV2.S Vaccine against Covid-19</a>. N Engl J Med 384:2187–2201.</div>
</div>
<div id="ref-iWMHpTBJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1514. </div><div class="csl-right-inline">2021. Janssen Investigational COVID-19 Vaccine: Interim Analysis of Phase 3 Clinical Data Released. National Institutes of Health (NIH). <a href="https://www.nih.gov/news-events/news-releases/janssen-investigational-covid-19-vaccine-interim-analysis-phase-3-clinical-data-released">https://www.nih.gov/news-events/news-releases/janssen-investigational-covid-19-vaccine-interim-analysis-phase-3-clinical-data-released</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1FcpboRMm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1515. </div><div class="csl-right-inline">2021. Johnson &amp; Johnson Announces Single-Shot Janssen COVID-19 Vaccine Candidate Met Primary Endpoints in Interim Analysis of its Phase 3 ENSEMBLE Trial. Janssen. <a href="https://www.janssen.com/emea/sites/www_janssen_com_emea/files/johnson_johnson_announces_single-shot_janssen_covid-19_vaccine_candidate_met_primary_endpoints_in_interim_analysis_of_its_phase_3_ensemble_trial.pdf">https://www.janssen.com/emea/sites/www_janssen_com_emea/files/johnson_johnson_announces_single-shot_janssen_covid-19_vaccine_candidate_met_primary_endpoints_in_interim_analysis_of_its_phase_3_ensemble_trial.pdf</a>.</div>
</div>
<div id="ref-3KMxmQhV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1516. </div><div class="csl-right-inline">Burki TK. 2020. <a href="https://doi.org/10.1016/s2213-2600(20)30402-1">The Russian vaccine for COVID-19</a>. The Lancet Respiratory Medicine 8:e85–e86.</div>
</div>
<div id="ref-JSzDvnk6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1517. </div><div class="csl-right-inline">Cohen J. 2020. Russia’s claim of a successful COVID-19 vaccine doesn’t pass the ‘smell test,’ critics say. Science https://doi.org/<a href="https://doi.org/10.1126/science.abf6791">10.1126/science.abf6791</a>.</div>
</div>
<div id="ref-Yzz3rwqk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1518. </div><div class="csl-right-inline">Callaway E. 2020. Russia announces positive COVID-vaccine results from controversial trial. Nature https://doi.org/<a href="https://doi.org/10.1038/d41586-020-03209-0">10.1038/d41586-020-03209-0</a>.</div>
</div>
<div id="ref-15DiM98Ae" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1519. </div><div class="csl-right-inline">Thorp HH. 2020. <a href="https://doi.org/10.1126/science.abe3147">A dangerous rush for vaccines</a>. Science 369:885–885.</div>
</div>
<div id="ref-x4aIj5Fr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1520. </div><div class="csl-right-inline">Jr BL. Scientists worry whether Russia's 'Sputnik V' coronavirus vaccine is safe and effective. CNBC. <a href="https://www.cnbc.com/2020/08/11/scientists-worry-whether-russias-sputnik-v-coronavirus-vaccine-is-safe-and-effective.html">https://www.cnbc.com/2020/08/11/scientists-worry-whether-russias-sputnik-v-coronavirus-vaccine-is-safe-and-effective.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-PNZEiId1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1521. </div><div class="csl-right-inline">Logunov DY, Dolzhikova IV, Zubkova OV, Tukhvatulin AI, Shcheblyakov DV, Dzharullaeva AS, Grousova DM, Erokhova AS, Kovyrshina AV, Botikov AG, Izhaeva FM, Popova O, Ozharovskaya TA, Esmagambetov IB, Favorskaya IA, Zrelkin DI, Voronina DV, Shcherbinin DN, Semikhin AS, Simakova YV, Tokarskaya EA, Lubenets NL, Egorova DA, Shmarov MM, Nikitenko NA, Morozova LF, Smolyarchuk EA, Kryukov EV, Babira VF, Borisevich SV, Naroditsky BS, Gintsburg AL. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)31866-3">Safety and immunogenicity of an rAd26 and rAd5 vector-based heterologous prime-boost COVID-19 vaccine in two formulations: two open, non-randomised phase 1/2 studies from Russia</a>. The Lancet 396:887–897.</div>
</div>
<div id="ref-hSWPgHOv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1522. </div><div class="csl-right-inline">Clinical Trials. <a href="https://sputnikvaccine.com/about-vaccine/clinical-trials/">https://sputnikvaccine.com/about-vaccine/clinical-trials/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-vwhmuwto" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1523. </div><div class="csl-right-inline">Phillips N, Cyranoski D, Mallapaty S. 2020. A leading coronavirus vaccine trial is on hold: scientists react. Nature https://doi.org/<a href="https://doi.org/10.1038/d41586-020-02594-w">10.1038/d41586-020-02594-w</a>.</div>
</div>
<div id="ref-Fz6kXAHy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1524. </div><div class="csl-right-inline">Cyranoski D, Mallapaty S. 2020. <a href="https://doi.org/10.1038/d41586-020-02633-6">Scientists relieved as coronavirus vaccine trial restarts — but question lack of transparency</a>. Nature 585:331–332.</div>
</div>
<div id="ref-Kt4zFpsF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1525. </div><div class="csl-right-inline">Robbins R, LaFraniere S, Weiland N, Kirkpatrick DD, Mueller B. 2020. <a href="https://www.nytimes.com/2020/12/08/business/covid-vaccine-oxford-astrazeneca.html">Blunders Eroded U.S. Confidence in Early Vaccine Front-Runner</a>. The New York Times.</div>
</div>
<div id="ref-4mDUvRId" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1526. </div><div class="csl-right-inline">2020. <a href="https://www.bbc.com/news/health-55086927">Oxford/AstraZeneca Covid vaccine 'dose error' explained</a>. BBC News.</div>
</div>
<div id="ref-SUGCFKUo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1527. </div><div class="csl-right-inline">Sanchez S, Palacio N, Dangi T, Ciucci T, Penaloza-MacMaster P. 2021. <a href="https://doi.org/10.1126/sciimmunol.abi8635">Fractionating a COVID-19 Ad5-vectored vaccine improves virus-specific immunity</a>. Sci Immunol 6.</div>
</div>
<div id="ref-5gqQwOi6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1528. </div><div class="csl-right-inline">Wolf ME, Luz B, Niehaus L, Bhogal P, Bäzner H, Henkes H. 2021. <a href="https://doi.org/10.3390/jcm10081599">Thrombocytopenia and Intracranial Venous Sinus Thrombosis after “COVID-19 Vaccine AstraZeneca” Exposure</a>. JCM 10:1599.</div>
</div>
<div id="ref-Q4qW6ARY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1529. </div><div class="csl-right-inline">Wise J. 2021. <a href="https://doi.org/10.1136/bmj.n699">Covid-19: European countries suspend use of Oxford-AstraZeneca vaccine after reports of blood clots</a>. BMJ n699.</div>
</div>
<div id="ref-xSBGseR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1530. </div><div class="csl-right-inline">Mahase E. 2021. <a href="https://doi.org/10.1136/bmj.n774">Covid-19: AstraZeneca vaccine is not linked to increased risk of blood clots, finds European Medicine Agency</a>. BMJ n774.</div>
</div>
<div id="ref-TtPVfMOL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1531. </div><div class="csl-right-inline">Mahase E. 2021. <a href="https://doi.org/10.1136/bmj.n970">Covid-19: US suspends Johnson and Johnson vaccine rollout over blood clots</a>. BMJ n970.</div>
</div>
<div id="ref-1F1ma0vMT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1532. </div><div class="csl-right-inline">Tanne JH. 2021. <a href="https://doi.org/10.1136/bmj.n1079">Covid-19: US authorises Johnson and Johnson vaccine again, ending pause in rollout</a>. BMJ n1079.</div>
</div>
<div id="ref-iRvYqPq6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1533. </div><div class="csl-right-inline">Mahase E. 2021. <a href="https://doi.org/10.1136/bmj.n1046">Covid-19: Unusual blood clots are “very rare side effect” of Janssen vaccine, says EMA</a>. BMJ n1046.</div>
</div>
<div id="ref-MnBdD0Mr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1534. </div><div class="csl-right-inline">Oliver SE, Wallace M, See I, Mbaeyi S, Godfrey M, Hadler SC, Jatlaoui TC, Twentyman E, Hughes MM, Rao AK, Fiore A, Su JR, Broder KR, Shimabukuro T, Lale A, Shay DK, Markowitz LE, Wharton M, Bell BP, Brooks O, McNally V, Lee GM, Talbot HK, Daley MF. 2022. <a href="https://doi.org/10.15585/mmwr.mm7103a4">Use of the Janssen (Johnson &amp; Johnson) COVID-19 Vaccine: Updated Interim Recommendations from the Advisory Committee on Immunization Practices — United States, December 2021</a>. MMWR Morb Mortal Wkly Rep 71:90–95.</div>
</div>
<div id="ref-2uAR3HBq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1535. </div><div class="csl-right-inline">Shay DK, Gee J, Su JR, Myers TR, Marquez P, Liu R, Zhang B, Licata C, Clark TA, Shimabukuro TT. 2021. <a href="https://doi.org/10.15585/mmwr.mm7018e2">Safety Monitoring of the Janssen (Johnson &amp; Johnson) COVID-19 Vaccine — United States, March–April 2021</a>. MMWR Morb Mortal Wkly Rep 70:680–684.</div>
</div>
<div id="ref-EqBIFx5T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1536. </div><div class="csl-right-inline">Rosenblum HG, Hadler SC, Moulia D, Shimabukuro TT, Su JR, Tepper NK, Ess KC, Woo EJ, Mba-Jonas A, Alimchandani M, Nair N, Klein NP, Hanson KE, Markowitz LE, Wharton M, McNally VV, Romero JR, Talbot HK, Lee GM, Daley MF, Mbaeyi SA, Oliver SE. 2021. <a href="https://doi.org/10.15585/mmwr.mm7032e4">Use of COVID-19 Vaccines After Reports of Adverse Events Among Adult Recipients of Janssen (Johnson &amp; Johnson) and mRNA COVID-19 Vaccines (Pfizer-BioNTech and Moderna): Update from the Advisory Committee on Immunization Practices — United States, July 2021</a>. MMWR Morb Mortal Wkly Rep 70:1094–1099.</div>
</div>
<div id="ref-tTpW7jvQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1537. </div><div class="csl-right-inline">Pottegård A, Lund LC, Karlstad Ø, Dahl J, Andersen M, Hallas J, Lidegaard Ø, Tapia G, Gulseth HL, Ruiz PL-D, Watle SV, Mikkelsen AP, Pedersen L, Sørensen HT, Thomsen RW, Hviid A. 2021. <a href="https://doi.org/10.1136/bmj.n1114">Arterial events, venous thromboembolism, thrombocytopenia, and bleeding after vaccination with Oxford-AstraZeneca ChAdOx1-S in Denmark and Norway: population based cohort study</a>. BMJ n1114.</div>
</div>
<div id="ref-rWaWz2Yg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1538. </div><div class="csl-right-inline">Chan B, Odutayo A, Juni P, Stall NM, Bobos P, Brown AD, Grill A, Ivers N, Maltsev A, McGeer A, Miller KJ, Niel U, Razak F, Sander B, Sholzberg M, Slutsky AS, Morris AM, Pai M. 2021. <a href="https://doi.org/10.47326/ocsat.2021.02.28.1.0">Risk of Vaccine-Induced Thrombotic Thrombocytopenia (VITT) following the AstraZeneca/COVISHIELD Adenovirus Vector COVID-19 Vaccines</a>. Ontario COVID-19 Science Advisory Table.</div>
</div>
<div id="ref-z5vyJuGl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1539. </div><div class="csl-right-inline">Baker AT, Boyd RJ, Sarkar D, Teijeira-Crespo A, Chan CK, Bates E, Waraich K, Vant J, Wilson E, Truong CD, Lipka-Lloyd M, Fromme P, Vermaas J, Williams D, Machiesky L, Heurich M, Nagalo BM, Coughlan L, Umlauf S, Chiu P-L, Rizkallah PJ, Cohen TS, Parker AL, Singharoy A, Borad MJ. 2021. <a href="https://doi.org/10.1126/sciadv.abl8213">ChAdOx1 interacts with CAR and PF4 with implications for thrombosis with thrombocytopenia syndrome</a>. Sci Adv 7.</div>
</div>
<div id="ref-XwlVcLVP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1540. </div><div class="csl-right-inline">Schultz NH, Sørvoll IH, Michelsen AE, Munthe LA, Lund-Johansen F, Ahlen MT, Wiedmann M, Aamodt A-H, Skattør TH, Tjønnfjord GE, Holme PA. 2021. <a href="https://doi.org/10.1056/nejmoa2104882">Thrombosis and Thrombocytopenia after ChAdOx1 nCoV-19 Vaccination</a>. N Engl J Med 384:2124–2130.</div>
</div>
<div id="ref-KUnUe1UK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1541. </div><div class="csl-right-inline">2021. EMA recommends COVID-19 Vaccine AstraZeneca for authorisation in the EU. <a href="https://www.ema.europa.eu/en/news/ema-recommends-covid-19-vaccine-astrazeneca-authorisation-eu">https://www.ema.europa.eu/en/news/ema-recommends-covid-19-vaccine-astrazeneca-authorisation-eu</a>.</div>
</div>
<div id="ref-fcm6ElC0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1542. </div><div class="csl-right-inline">Miesbach W, Makris M. 2020. <a href="https://doi.org/10.1177/1076029620938149">COVID-19: Coagulopathy, Risk of Thrombosis, and the Rationale for Anticoagulation</a>. Clin Appl Thromb Hemost 26:107602962093814.</div>
</div>
<div id="ref-D7ou3S22" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1543. </div><div class="csl-right-inline">Verbeke R, Lentacker I, De Smedt SC, Dewitte H. 2019. <a href="https://doi.org/10.1016/j.nantod.2019.100766">Three decades of messenger RNA vaccine development</a>. Nano Today 28:100766.</div>
</div>
<div id="ref-2YZ70C2y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1544. </div><div class="csl-right-inline">Schlake T, Thess A, Fotin-Mleczek M, Kallen K-J. 2012. <a href="https://doi.org/10.4161/rna.22269">Developing mRNA-vaccine technologies</a>. RNA Biology 9:1319–1330.</div>
</div>
<div id="ref-HCImhzy8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1545. </div><div class="csl-right-inline">Martinon F, Krishnan S, Lenzen G, Magné R, Gomard E, Guillet J-G, Lévy J-P, Meulien P. 1993. <a href="https://doi.org/10.1002/eji.1830230749">Induction of virus-specific cytotoxic T lymphocytesin vivo by liposome-entrapped mRNA</a>. European Journal of Immunology 23:1719–1722.</div>
</div>
<div id="ref-zNKWlCwE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1546. </div><div class="csl-right-inline">Reichmuth AM, Oberli MA, Jaklenec A, Langer R, Blankschtein D. 2016. <a href="https://doi.org/10.4155/tde-2016-0006">mRNA vaccine delivery using lipid nanoparticles</a>. Therapeutic Delivery 7:319–334.</div>
</div>
<div id="ref-pRoqjur8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1547. </div><div class="csl-right-inline">Iavarone C, O’hagan DT, Yu D, Delahaye NF, Ulmer JB. 2017. <a href="https://doi.org/10.1080/14760584.2017.1355245">Mechanism of action of mRNA-based vaccines</a>. Expert Review of Vaccines 16:871–881.</div>
</div>
<div id="ref-1EM5nGaYd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1548. </div><div class="csl-right-inline">RNA vaccines: an introduction. PHG Foundation. <a href="https://www.phgfoundation.org/briefing/rna-vaccines">https://www.phgfoundation.org/briefing/rna-vaccines</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-3LMMW7F0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1549. </div><div class="csl-right-inline">Crotty S. 2014. <a href="https://doi.org/10.1016/j.immuni.2014.10.004">T Follicular Helper Cell Differentiation, Function, and Roles in Disease</a>. Immunity 41:529–542.</div>
</div>
<div id="ref-ENBWnhAh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1550. </div><div class="csl-right-inline">Pardi N, Hogan MJ, Porter FW, Weissman D. 2018. <a href="https://doi.org/10.1038/nrd.2017.243">mRNA vaccines — a new era in vaccinology</a>. Nature Reviews Drug Discovery 17:261–279.</div>
</div>
<div id="ref-vh2AH9sg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1551. </div><div class="csl-right-inline">Stuart LM. 2021. <a href="https://doi.org/10.1056/nejmcibr2111445">In Gratitude for mRNA Vaccines</a>. N Engl J Med 385:1436–1438.</div>
</div>
<div id="ref-xUUwXDNv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1552. </div><div class="csl-right-inline">Pardi N, Hogan MJ, Weissman D. 2020. <a href="https://doi.org/10.1016/j.coi.2020.01.008">Recent advances in mRNA vaccine technology</a>. Current Opinion in Immunology 65:14–20.</div>
</div>
<div id="ref-wYZ6qJMu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1553. </div><div class="csl-right-inline">Amanat F, Krammer F. 2020. <a href="https://doi.org/10.1016/j.immuni.2020.03.007">SARS-CoV-2 Vaccines: Status Report</a>. Immunity 52:583–589.</div>
</div>
<div id="ref-17lluDFcc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1554. </div><div class="csl-right-inline">Fonteilles-Drabek S, Reddy D, Wells TNC. 2017. <a href="https://doi.org/10.1038/nrd.2017.24">Managing intellectual property to develop medicines for the world's poorest</a>. Nat Rev Drug Discov 16:223–224.</div>
</div>
<div id="ref-3EUiWZdN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1555. </div><div class="csl-right-inline">Study in Healthy Adults to Evaluate Gene Activation After Vaccination With GlaxoSmithKline (GSK) Biologicals' Candidate Tuberculosis (TB) Vaccine GSK 692342 - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT01669096">https://clinicaltrials.gov/ct2/show/NCT01669096</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-6wZy2mn8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1556. </div><div class="csl-right-inline">Pardi N, Parkhouse K, Kirkpatrick E, McMahon M, Zost SJ, Mui BL, Tam YK, Karikó K, Barbosa CJ, Madden TD, Hope MJ, Krammer F, Hensley SE, Weissman D. 2018. <a href="https://doi.org/10.1038/s41467-018-05482-0">Nucleoside-modified mRNA immunization elicits influenza virus hemagglutinin stalk-specific antibodies</a>. Nature Communications 9:3361.</div>
</div>
<div id="ref-Djz8x39x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1557. </div><div class="csl-right-inline">Veiga N, Goldsmith M, Granot Y, Rosenblum D, Dammes N, Kedmi R, Ramishetti S, Peer D. 2018. <a href="https://doi.org/10.1038/s41467-018-06936-1">Cell specific delivery of modified mRNA expressing therapeutic proteins to leukocytes</a>. Nature Communications 9:4493.</div>
</div>
<div id="ref-5x25saIz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1558. </div><div class="csl-right-inline">Vabret N, Britton GJ, Gruber C, Hegde S, Kim J, Kuksin M, Levantovsky R, Malle L, Moreira A, Park MD, Pia L, Risson E, Saffern M, Salomé B, Esai Selvan M, Spindler MP, Tan J, van der Heide V, Gregory JK, Alexandropoulos K, Bhardwaj N, Brown BD, Greenbaum B, Gümüş ZH, Homann D, Horowitz A, Kamphorst AO, Curotto de Lafaille MA, Mehandru S, Merad M, Samstein RM, Agrawal M, Aleynick M, Belabed M, Brown M, Casanova-Acebes M, Catalan J, Centa M, Charap A, Chan A, Chen ST, Chung J, Bozkus CC, Cody E, Cossarini F, Dalla E, Fernandez N, Grout J, Ruan DF, Hamon P, Humblin E, Jha D, Kodysh J, Leader A, Lin M, Lindblad K, Lozano-Ojalvo D, Lubitz G, Magen A, Mahmood Z, Martinez-Delgado G, Mateus-Tique J, Meritt E, Moon C, Noel J, O’Donnell T, Ota M, Plitt T, Pothula V, Redes J, Reyes Torres I, Roberto M, Sanchez-Paulete AR, Shang J, Schanoski AS, Suprun M, Tran M, Vaninov N, Wilk CM, Aguirre-Ghiso J, Bogunovic D, Cho J, Faith J, Grasset E, Heeger P, Kenigsberg E, Krammer F, Laserson U. 2020. <a href="https://doi.org/10.1016/j.immuni.2020.05.002">Immunology of COVID-19: Current State of the Science</a>. Immunity 52:910–941.</div>
</div>
<div id="ref-1CsCQi9wT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1559. </div><div class="csl-right-inline">Chien KR, Zangi L, Lui KO. 2014. <a href="https://doi.org/10.1101/cshperspect.a014035">Synthetic Chemically Modified mRNA (modRNA): Toward a New Technology Platform for Cardiovascular Biology and Medicine</a>. Cold Spring Harbor Perspectives in Medicine 5:a014035–a014035.</div>
</div>
<div id="ref-10VyxCgQU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1560. </div><div class="csl-right-inline">Pfizer and BioNTech Announce Early Positive Data from an Ongoing Phase 1/2 study of mRNA-based Vaccine Candidate Against SARS-CoV-2 | Pfizer. <a href="https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-early-positive-data-ongoing-0">https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-early-positive-data-ongoing-0</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-Biu1CQeQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1561. </div><div class="csl-right-inline">National Institute of Allergy and Infectious Diseases (NIAID). 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04283461">Phase I, Open-Label, Dose-Ranging Study of the Safety and Immunogenicity of 2019-nCoV Vaccine (mRNA-1273) in Healthy Adults</a>. NCT04283461. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-suRY1e0N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1562. </div><div class="csl-right-inline">Funk CD, Laferrière C, Ardakani A. 2020. <a href="https://doi.org/10.3389/fphar.2020.00937">A Snapshot of the Global Race for Vaccines Targeting SARS-CoV-2 and the COVID-19 Pandemic</a>. Frontiers in Pharmacology 11:937.</div>
</div>
<div id="ref-1F8hc8XkK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1563. </div><div class="csl-right-inline">Ledford H. 2021. <a href="https://doi.org/10.1038/d41586-021-03535-x">What the Moderna–NIH COVID vaccine patent fight means for research</a>. Nature 600:200–201.</div>
</div>
<div id="ref-BrC2lh5x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1564. </div><div class="csl-right-inline">Diamond D. 2021. <a href="https://www.washingtonpost.com/health/2021/12/17/moderna-vaccine-patent-dispute-nih/">Moderna halts patent fight over coronavirus vaccine with federal government</a>. Washington Post.</div>
</div>
<div id="ref-ZdLFrJY9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1565. </div><div class="csl-right-inline">Goodman AS Brenda. 2022. Moderna files patent infringement lawsuits against Pfizer and BioNTech over mRNA Covid-19 vaccines. CNN. <a href="https://www.cnn.com/2022/08/26/health/moderna-pfizer-mrna-patent-lawsuit/index.html">https://www.cnn.com/2022/08/26/health/moderna-pfizer-mrna-patent-lawsuit/index.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-ZYxoabEm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1566. </div><div class="csl-right-inline">Baden LR, El Sahly HM, Essink B, Kotloff K, Frey S, Novak R, Diemert D, Spector SA, Rouphael N, Creech CB, McGettigan J, Khetan S, Segall N, Solis J, Brosz A, Fierro C, Schwartz H, Neuzil K, Corey L, Gilbert P, Janes H, Follmann D, Marovich M, Mascola J, Polakowski L, Ledgerwood J, Graham BS, Bennett H, Pajon R, Knightly C, Leav B, Deng W, Zhou H, Han S, Ivarsson M, Miller J, Zaks T. 2020. <a href="https://doi.org/10.1056/nejmoa2035389">Efficacy and Safety of the mRNA-1273 SARS-CoV-2 Vaccine</a>. New England Journal of Medicine NEJMoa2035389.</div>
</div>
<div id="ref-cAaN4Te0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1567. </div><div class="csl-right-inline">Commissioner O of the. 2020. FDA Takes Key Action in Fight Against COVID-19 By Issuing Emergency Use Authorization for First COVID-19 Vaccine. FDA. <a href="https://www.fda.gov/news-events/press-announcements/fda-takes-key-action-fight-against-covid-19-issuing-emergency-use-authorization-first-covid-19">https://www.fda.gov/news-events/press-announcements/fda-takes-key-action-fight-against-covid-19-issuing-emergency-use-authorization-first-covid-19</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-13Ou1UUAd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1568. </div><div class="csl-right-inline">Oliver SE. 2021. <a href="https://doi.org/10.15585/mmwr.mm695152e1">The Advisory Committee on Immunization Practices’ Interim Recommendation for Use of Moderna COVID-19 Vaccine — United States, December 2020</a>. MMWR Morbidity and Mortality Weekly Report 69.</div>
</div>
<div id="ref-3Uhu8LzO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1569. </div><div class="csl-right-inline">Puranik A, Lenehan PJ, Silvert E, Niesen MJM, Corchado-Garcia J, O’Horo JC, Virk A, Swift MD, Halamka J, Badley AD, Venkatakrishnan AJ, Soundararajan V. 2021. <a href="https://doi.org/10.1101/2021.08.06.21261707">Comparison of two highly-effective mRNA vaccines for COVID-19 during periods of Alpha and Delta variant prevalence</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-JszheMvx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1570. </div><div class="csl-right-inline">Tseng HF, Ackerson BK, Luo Y, Sy LS, Talarico CA, Tian Y, Bruxvoort KJ, Tubert JE, Florea A, Ku JH, Lee GS, Choi SK, Takhar HS, Aragones M, Qian L. 2022. <a href="https://doi.org/10.1038/s41591-022-01753-y">Effectiveness of mRNA-1273 against SARS-CoV-2 Omicron and Delta variants</a>. Nat Med 28:1063–1071.</div>
</div>
<div id="ref-S6RHdOTJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1571. </div><div class="csl-right-inline">Collie S, Champion J, Moultrie H, Bekker L-G, Gray G. 2022. <a href="https://doi.org/10.1056/nejmc2119270">Effectiveness of BNT162b2 Vaccine against Omicron Variant in South Africa</a>. N Engl J Med 386:494–496.</div>
</div>
<div id="ref-JEucQ5jm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1572. </div><div class="csl-right-inline">Andrews N, Stowe J, Kirsebom F, Toffa S, Rickeard T, Gallagher E, Gower C, Kall M, Groves N, O’Connell A-M, Simons D, Blomquist PB, Zaidi A, Nash S, Iwani Binti Abdul Aziz N, Thelwall S, Dabrera G, Myers R, Amirthalingam G, Gharbia S, Barrett JC, Elson R, Ladhani SN, Ferguson N, Zambon M, Campbell CNJ, Brown K, Hopkins S, Chand M, Ramsay M, Lopez Bernal J. 2022. <a href="https://doi.org/10.1056/nejmoa2119451">Covid-19 Vaccine Effectiveness against the Omicron (B.1.1.529) Variant</a>. N Engl J Med 386:1532–1546.</div>
</div>
<div id="ref-19qv58Mv3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1573. </div><div class="csl-right-inline">Coronavirus in the U.S.: Latest Map and Case Count. The New York Times. <a href="https://www.nytimes.com/interactive/2021/us/covid-cases.html">https://www.nytimes.com/interactive/2021/us/covid-cases.html</a>. Retrieved 11 March 2022.</div>
</div>
<div id="ref-TkFSco2t" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1574. </div><div class="csl-right-inline">Tan S. Analysis | Four charts that analyze how omicron’s wave compares to previous coronavirus peaks. Washington Post. <a href="https://www.washingtonpost.com/health/interactive/2022/omicron-comparison-cases-deaths-hospitalizations/">https://www.washingtonpost.com/health/interactive/2022/omicron-comparison-cases-deaths-hospitalizations/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-16SmBKzGd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1575. </div><div class="csl-right-inline">Abu Mouch S, Roguin A, Hellou E, Ishai A, Shoshan U, Mahamid L, Zoabi M, Aisman M, Goldschmid N, Berar Yanay N. 2021. <a href="https://doi.org/10.1016/j.vaccine.2021.05.087">Myocarditis following COVID-19 mRNA vaccination</a>. Vaccine 39:3790–3793.</div>
</div>
<div id="ref-SN0p27mm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1576. </div><div class="csl-right-inline">Kim HW, Jenista ER, Wendell DC, Azevedo CF, Campbell MJ, Darty SN, Parker MA, Kim RJ. 2021. <a href="https://doi.org/10.1001/jamacardio.2021.2828">Patients With Acute Myocarditis Following mRNA COVID-19 Vaccination</a>. JAMA Cardiol 6:1196.</div>
</div>
<div id="ref-hEVGb0Ne" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1577. </div><div class="csl-right-inline">Mevorach D, Anis E, Cedar N, Bromberg M, Haas EJ, Nadir E, Olsha-Castell S, Arad D, Hasin T, Levi N, Asleh R, Amir O, Meir K, Cohen D, Dichtiar R, Novick D, Hershkovitz Y, Dagan R, Leitersdorf I, Ben-Ami R, Miskin I, Saliba W, Muhsen K, Levi Y, Green MS, Keinan-Boker L, Alroy-Preis S. 2021. <a href="https://doi.org/10.1056/nejmoa2109730">Myocarditis after BNT162b2 mRNA Vaccine against Covid-19 in Israel</a>. N Engl J Med 385:2140–2149.</div>
</div>
<div id="ref-13El9IHNw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1578. </div><div class="csl-right-inline">Goddard K, Hanson KE, Lewis N, Weintraub E, Fireman B, Klein NP. 2022. Incidence of Myocarditis/Pericarditis Following mRNA COVID-19 Vaccination Among Children and Younger Adults in the United States. Ann Intern Med https://doi.org/<a href="https://doi.org/10.7326/m22-2274">10.7326/m22-2274</a>.</div>
</div>
<div id="ref-bC5oAO24" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1579. </div><div class="csl-right-inline">Matta A, Kunadharaju R, Osman M, Jesme C, McMiller Z, Johnson EM, Matta D, Kallamadi R, Bande D. 2021. Clinical Presentation and Outcomes of Myocarditis Post mRNA Vaccination: A Meta-Analysis and Systematic Review. Cureus https://doi.org/<a href="https://doi.org/10.7759/cureus.19240">10.7759/cureus.19240</a>.</div>
</div>
<div id="ref-TOyAW98u" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1580. </div><div class="csl-right-inline">Commissioner O of the. 2022. Coronavirus (COVID-19) Update: FDA Recommends Inclusion of Omicron BA.4/5 Component for COVID-19 Vaccine Booster Doses. FDA. <a href="https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-recommends-inclusion-omicron-ba45-component-covid-19-vaccine-booster">https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-recommends-inclusion-omicron-ba45-component-covid-19-vaccine-booster</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-wQgkg2M4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1581. </div><div class="csl-right-inline">2022. FDA Panel Gives Thumbs Up to Omicron-Containing COVID Boosters. <a href="https://www.medpagetoday.com/infectiousdisease/covid19vaccine/99493">https://www.medpagetoday.com/infectiousdisease/covid19vaccine/99493</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1FMDHQCgs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1582. </div><div class="csl-right-inline">. <a href="https://www.fda.gov/media/159597/download">https://www.fda.gov/media/159597/download</a>.</div>
</div>
<div id="ref-Slgwtax5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1583. </div><div class="csl-right-inline">Erman M. 2022. <a href="https://www.reuters.com/business/healthcare-pharmaceuticals/fda-will-not-require-clinical-trial-data-authorize-redesigned-covid-boosters-2022-06-30/">U.S. FDA to use existing Omicron booster data to review shots targeting new subvariants -official</a>. Reuters.</div>
</div>
<div id="ref-1F0F5pWsm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1584. </div><div class="csl-right-inline">Tseng HF, Ackerson BK, Bruxvoort KJ, Sy LS, Tubert JE, Lee GS, Ku JH, Florea A, Luo Y, Qiu S, Choi SK, Takhar HS, Aragones M, Paila YD, Chavers S, Talarico CA, Qian L. 2022. <a href="https://doi.org/10.1101/2022.09.30.22280573">Effectiveness of mRNA-1273 against infection and COVID-19 hospitalization with SARS-CoV-2 Omicron subvariants: BA.1, BA.2, BA.2.12.1, BA.4, and BA.5</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-Cs2RaaCI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1585. </div><div class="csl-right-inline">Hardt K, Vandebosch A, Sadoff J, Gars ML, Truyers C, Lowson D, Van Dromme I, Vingerhoets J, Kamphuis T, Scheper G, Ruiz-Guiñazú J, Faust SN, Spinner CD, Schuitemaker H, Van Hoof J, Douoguih M, Struyf F. 2022. <a href="https://doi.org/10.1101/2022.01.28.22270043">Efficacy and Safety of a Booster Regimen of Ad26.COV2.S Vaccine against Covid-19</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-WtUhPusE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1586. </div><div class="csl-right-inline">Dolzhikova I, Iliukhina A, Kovyrshina A, Kuzina A, Gushchin V, Siniavin A, Pochtovyi A, Shidlovskaya E, Kuznetsova N, Megeryan M, Dzharullaeva A, Erokhova A, Izhaeva F, Grousova D, Botikov A, Shcheblyakov D, Tukhvatulin A, Zubkova O, Logunov D, Gintsburg A. 2021. <a href="https://doi.org/10.1101/2021.12.17.21267976">Sputnik Light booster after Sputnik V vaccination induces robust neutralizing antibody response to B.1.1.529 (Omicron) SARS-CoV-2 variant</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-17s40tqgW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1587. </div><div class="csl-right-inline">Flaxman A, Marchevsky NG, Jenkin D, Aboagye J, Aley PK, Angus B, Belij-Rammerstorfer S, Bibi S, Bittaye M, Cappuccini F, Cicconi P, Clutterbuck EA, Davies S, Dejnirattisai W, Dold C, Ewer KJ, Folegatti PM, Fowler J, Hill AVS, Kerridge S, Minassian AM, Mongkolsapaya J, Mujadidi YF, Plested E, Ramasamy MN, Robinson H, Sanders H, Sheehan E, Smith H, Snape MD, Song R, Woods D, Screaton G, Gilbert SC, Voysey M, Pollard AJ, Lambe T, Adlou S, Aley R, Ali A, Anslow R, Baker M, Baker P, Barrett JR, Bates L, Beadon K, Beckley R, Bell J, Bellamy D, Beveridge A, Bissett C, Blackwell L, Bletchly H, Boyd A, Bridges-Webb A, Brown C, Byard N, Camara S, Cifuentes Gutierrez L, Collins AM, Cooper R, Crocker WEM, Darton TC, Davies H, Davies J, Demissie T, Di Maso C, Dinesh T, Donnellan FR, Douglas AD, Drake-Brockman R, Duncan CJA, Elias SC, Emary KRW, Ghulam Farooq M, Faust SN, Felle S, Ferreira D, Ferreira Da Silva C, Finn A, Ford KJ, Francis E, Furze J, Fuskova M, Galiza E, Gibertoni Cruz A, Godfrey L, Goodman AL, Green C, Green CA, Greenwood N, Harrison D, Hart TC, Hawkins S, Heath PT, Hill H, Hillson K, Horsington B, Hou MM, Howe E, Howell N, Joe C, Jones E, Kasanyinga M, Keen J, Kelly S, Kerr D, Khan L, Khozoee B, Kinch J, Kinch P, Koleva S, Kwok J, Larkworthy CW, Lawrie AM, Lazarus R, Lees EA, Li G, Libri V, Lillie PJ, Linder A, Long F, Lopez Ramon R, Mabbett R, Makinson R, Marinou S, Marlow E, Marshall JL, Mazur O, McEwan J, McGregor AC, Mokaya J, Morey E, Morshead G, Morter R, Muller J, Mweu P, Noristani R, Owino N, Polo Peralta Alvarez M, Platt A, Pollock KM, Poulton I, Provstgaard-Morys S, Pulido-Gomez D, Rajan M, Ramos Lopez F, Ritchie A, Roberts H, Rollier C, Rudiansyah I, Sanders K, Saunders JE, Seddiqi S, Sharpe HR, Shaw R, Silva-Reyes L, Singh N, Smith DJ, Smith CC, Smith A, Spencer AJ, Stuart ASV, Sutherland R, Szigeti A, Tang K, Thomas M, Thomas TM, Thompson A, Thomson EC, Török EM, Toshner M, Tran N, Trivett R, Turnbull I, Turner C, Turner DPJ, Ulaszewska M, Vichos I, Walker L, Watson ME, Whelan C, White R, Williams SJ, Williams CJA, Wright D, Yao A. 2021. <a href="https://doi.org/10.1016/s0140-6736(21)01699-8">Reactogenicity and immunogenicity after a late second dose or a third dose of ChAdOx1 nCoV-19 in the UK: a substudy of two randomised controlled trials (COV001 and COV002)</a>. The Lancet 398:981–990.</div>
</div>
<div id="ref-Jv71MaZb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1588. </div><div class="csl-right-inline">Regev-Yochay G, Gonen T, Gilboa M, Mandelboim M, Indenbaum V, Amit S, Meltzer L, Asraf K, Cohen C, Fluss R, Biber A, Nemet I, Kliker L, Joseph G, Doolman R, Mendelson E, Freedman LS, Harats D, Kreiss Y, Lustig Y. 2022. <a href="https://doi.org/10.1101/2022.02.15.22270948">4th Dose COVID mRNA Vaccines' Immunogenicity &amp; Efficacy Against Omicron VOC</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-L0dD93f8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1589. </div><div class="csl-right-inline">Chiu N-C, Chi H, Tu Y-K, Huang Y-N, Tai Y-L, Weng S-L, Chang L, Huang DT-N, Huang F-Y, Lin C-Y. 2021. <a href="https://doi.org/10.1080/14760584.2021.1971522">To mix or not to mix? A rapid systematic review of heterologous prime–boost covid-19 vaccination</a>. Expert Review of Vaccines 20:1211–1220.</div>
</div>
<div id="ref-ybtjun8H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1590. </div><div class="csl-right-inline">Atmar RL, Lyke KE, Deming ME, Jackson LA, Branche AR, El Sahly HM, Rostad CA, Martin JM, Johnston C, Rupp RE, Mulligan MJ, Brady RC, Frenck RW Jr., Bäcker M, Kottkamp AC, Babu TM, Rajakumar K, Edupuganti S, Dobrzynski D, Coler RN, Posavad CM, Archer JI, Crandon S, Nayak SU, Szydlo D, Zemanek JA, Dominguez Islas CP, Brown ER, Suthar MS, McElrath MJ, McDermott AB, O’Connell SE, Montefiori DC, Eaton A, Neuzil KM, Stephens DS, Roberts PC, Beigel JH. 2022. <a href="https://doi.org/10.1056/nejmoa2116414">Homologous and Heterologous Covid-19 Booster Vaccinations</a>. N Engl J Med 386:1046–1057.</div>
</div>
<div id="ref-H6vRjWQg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1591. </div><div class="csl-right-inline">Jara A, Undurraga EA, Zubizarreta JR, González C, Pizarro A, Acevedo J, Leo K, Paredes F, Bralic T, Vergara V, Mosso M, Leon F, Parot I, Leighton P, Suárez P, Rios JC, García-Escorza H, Araos R. 2022. <a href="https://doi.org/10.1016/s2214-109x(22)00112-7">Effectiveness of homologous and heterologous booster doses for an inactivated SARS-CoV-2 vaccine: a large-scale prospective cohort study</a>. The Lancet Global Health 10:e798–e806.</div>
</div>
<div id="ref-15rMr9nh7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1592. </div><div class="csl-right-inline">Sapkota B, Saud B, Shrestha R, Al-Fahad D, Sah R, Shrestha S, Rodriguez-Morales AJ. 2021. Heterologous prime–boost strategies for COVID-19 vaccines. Journal of Travel Medicine https://doi.org/<a href="https://doi.org/10.1093/jtm/taab191">10.1093/jtm/taab191</a>.</div>
</div>
<div id="ref-cNs4cL36" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1593. </div><div class="csl-right-inline">Assawakosri S, Kanokudom S, Suntronwong N, Auphimai C, Nilyanimit P, Vichaiwattana P, Thongmee T, Duangchinda T, Chantima W, Pakchotanon P, Srimuan D, Thatsanatorn T, Klinfueng S, Yorsaeng R, Sudhinaraset N, Wanlapakorn N, Mongkolsapaya J, Honsawek S, Poovorawan Y. 2022. <a href="https://doi.org/10.1093/infdis/jiac092">Neutralizing Activities Against the Omicron Variant After a Heterologous Booster in Healthy Adults Receiving Two Doses of CoronaVac Vaccination</a>. The Journal of Infectious Diseases 226:1372–1381.</div>
</div>
<div id="ref-s1xmA2K6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1594. </div><div class="csl-right-inline">Accorsi EK, Britton A, Shang N, Fleming-Dutra KE, Link-Gelles R, Smith ZR, Derado G, Miller J, Schrag SJ, Verani JR. 2022. <a href="https://doi.org/10.1056/nejmc2203165">Effectiveness of Homologous and Heterologous Covid-19 Boosters against Omicron</a>. N Engl J Med 386:2433–2435.</div>
</div>
<div id="ref-exu6jkTJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1595. </div><div class="csl-right-inline">Munro APS, Janani L, Cornelius V, Aley PK, Babbage G, Baxter D, Bula M, Cathie K, Chatterjee K, Dodd K, Enever Y, Gokani K, Goodman AL, Green CA, Harndahl L, Haughney J, Hicks A, van der Klaauw AA, Kwok J, Lambe T, Libri V, Llewelyn MJ, McGregor AC, Minassian AM, Moore P, Mughal M, Mujadidi YF, Murira J, Osanlou O, Osanlou R, Owens DR, Pacurar M, Palfreeman A, Pan D, Rampling T, Regan K, Saich S, Salkeld J, Saralaya D, Sharma S, Sheridan R, Sturdy A, Thomson EC, Todd S, Twelves C, Read RC, Charlton S, Hallis B, Ramsay M, Andrews N, Nguyen-Van-Tam JS, Snape MD, Liu X, Faust SN, Riordan A, Ustianowski A, Rogers CA, Hughes S, Longshaw L, Stockport J, Hughes R, Grundy L, Tudor Jones L, Guha A, Snashall E, Eadsforth T, Reeder S, Storton K, Munusamy M, Tandy B, Egbo A, Cox S, Ahmed NN, Shenoy A, Bousfield R, Wixted D, Gutteridge H, Mansfield B, Herbert C, Holliday K, Calderwood J, Barker D, Brandon J, Tulloch H, Colquhoun S, Thorp H, Radford H, Evans J, Baker H, Thorpe J, Batham S, Hailstone J, Phillips R, Kumar D, Westwell F, Makia F, Hopkins N, Barcella L, Mpelembue M, dabagh M, lang M, khan F, Adebambo O, Chita S, Corrah T, Whittington A, John L, Roche S, Wagstaff L, Farrier A, Bisnauthsing K, Serafimova T, Nanino E, Cooney E, Wilson-Goldsmith J, Nguyen H, Mazzella A, Jackson B, Aslam S, Bawa T, Broadhead S, Farooqi S, Piper J, Weighell R, Pickup L, Shamtally D, Domingo J, Kourampa E, Hale C, Gibney J, Stackpoole M, Rashid-Gardner Z, Lyon R, McDonnell C, Cole C, Stewart A, McMillan G, Savage M, Beckett H, Moorbey C, Desai A, Brown C, Naker K, Qureshi E, Trinham C, Sabine C, Moore S, Hurdover S, Justice E, Smith D, Plested E, Ferreira Da Silva C, White R, Robinson H, Cifuentes L, Morshead G, Drake-Brockman R, Kinch P, Kasanyinga M, Clutterbuck EA, Bibi S, Stuart AS, Shaw RH, Singh M, Champaneri T, Irwin M, Khan M, Kownacka A, Nabunjo M, Osuji C, Hladkiwskyj J, Galvin D, Patel G, Mouland J, Longhurst B, Moon M, Giddins B, Pereira Dias Alves C, Richmond L, Minnis C, Baryschpolec S, Elliott S, Fox L, Graham V, Baker N, Godwin K, Buttigieg K, Knight C, Brown P, Lall P, Shaik I, Chiplin E, Brunt E, Leung S, Allen L, Thomas S, Fraser S, Choi B, Gouriet J, Freedman A, Perkins J, Gowland A, Macdonald J, Seenan JP, Starinskij I, Seaton A, Peters E, Singh S, Gardside B, Bonnaud A, Davies C, Gordon E, Keenan S, Hall J, Wilkins S, Tasker S, James R, Seath I, Littlewood K, Newman J, Boubriak I, Suggitt D, Haydock H, Bennett S, Woodyatt W, Hughes K, Bell J, Coughlan T, van Welsenes D, Kamal M, Cooper C, Tunstall S, Ronan N, Cutts R, Dare T, Yim YTN, Whittley S, Ricamara M, Hamal S, Adams K, Baker H, Driver K, Turner N, Rawlins T, Roy S, Merida-Morillas M, Sakagami Y, Andrews A, Goncalves cordeiro L, Stokes M, Ambihapathy W, Spencer J, Parungao N, Berry L, Cullinane J, Presland L, Ross-Russell A, Warren S, Baker J, Oliver A, Buadi A, Lee K, Haskell L, Romani R, Bentley I, Whitbred T, Fowler S, Gavin J, Magee A, Watson T, Nightingale K, Marius P, Summerton E, Locke E, Honey T, Lingwood A, de la Haye A, Elliott RS, Underwood K, King M, Davies-Dear S, Horsfall E, Chalwin O, Burton H, Edwards CJ, Welham B, Garrahy S, Hall F, Ladikou E, Mullan D, Hansen D, Campbell M, Dos Santos F, Habash-Bailey H, Lakeman N, Branney D, Vamplew L, Hogan A, Frankham J, Wiselka M, Vail D, Wenn V, Renals V, Ellis K, Lewis-Taylor J, Magan J, Hardy A, Appleby K. 2021. <a href="https://doi.org/10.1016/s0140-6736(21)02717-3">Safety and immunogenicity of seven COVID-19 vaccines as a third dose (booster) following two doses of ChAdOx1 nCov-19 or BNT162b2 in the UK (COV-BOOST): a blinded, multicentre, randomised, controlled, phase 2 trial</a>. The Lancet 398:2258–2276.</div>
</div>
<div id="ref-EzS3LTYM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1596. </div><div class="csl-right-inline">2021. Overview of EU/EEA country recommendations on COVID-19 vaccination with Vaxzevria, and a scoping review of evidence to guide decision-making. European Centre for Disease Prevention and Control. <a href="https://www.ecdc.europa.eu/en/publications-data/overview-eueea-country-recommendations-covid-19-vaccination-vaxzevria-and-scoping">https://www.ecdc.europa.eu/en/publications-data/overview-eueea-country-recommendations-covid-19-vaccination-vaxzevria-and-scoping</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-kFl0x9VR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1597. </div><div class="csl-right-inline">Duarte-Salles T, Prieto-Alhambra D. 2021. <a href="https://doi.org/10.1016/s0140-6736(21)01442-2">Heterologous vaccine regimens against COVID-19</a>. The Lancet 398:94–95.</div>
</div>
<div id="ref-OWsXXUTT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1598. </div><div class="csl-right-inline">Moderna Announces Omicron-Containing Bivalent Booster Candidate mRNA-1273.214 Demonstrates Superior Antibody Response Against Omicron. <a href="https://investors.modernatx.com/news/news-details/2022/Moderna-Announces-Omicron-Containing-Bivalent-Booster-Candidate-mRNA-1273.214-Demonstrates-Superior-Antibody-Response-Against-Omicron/default.aspx">https://investors.modernatx.com/news/news-details/2022/Moderna-Announces-Omicron-Containing-Bivalent-Booster-Candidate-mRNA-1273.214-Demonstrates-Superior-Antibody-Response-Against-Omicron/default.aspx</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-aPBb6pOb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1599. </div><div class="csl-right-inline">Choi A, Koch M, Wu K, Chu L, Ma L, Hill A, Nunna N, Huang W, Oestreicher J, Colpitts T, Bennett H, Legault H, Paila Y, Nestorova B, Ding B, Montefiori D, Pajon R, Miller JM, Leav B, Carfi A, McPhee R, Edwards DK. 2021. <a href="https://doi.org/10.1038/s41591-021-01527-y">Safety and immunogenicity of SARS-CoV-2 variant mRNA vaccine boosters in healthy adults: an interim analysis</a>. Nat Med 27:2025–2031.</div>
</div>
<div id="ref-198OSKmOq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1600. </div><div class="csl-right-inline">Pajon R, Doria-Rose NA, Shen X, Schmidt SD, O’Dell S, McDanal C, Feng W, Tong J, Eaton A, Maglinao M, Tang H, Manning KE, Edara V-V, Lai L, Ellis M, Moore KM, Floyd K, Foster SL, Posavad CM, Atmar RL, Lyke KE, Zhou T, Wang L, Zhang Y, Gaudinski MR, Black WP, Gordon I, Guech M, Ledgerwood JE, Misasi JN, Widge A, Sullivan NJ, Roberts PC, Beigel JH, Korber B, Baden LR, El Sahly H, Chalkias S, Zhou H, Feng J, Girard B, Das R, Aunins A, Edwards DK, Suthar MS, Mascola JR, Montefiori DC. 2022. <a href="https://doi.org/10.1056/nejmc2119912">SARS-CoV-2 Omicron Variant Neutralization after mRNA-1273 Booster Vaccination</a>. N Engl J Med 386:1088–1091.</div>
</div>
<div id="ref-1D9Fxz7bM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1601. </div><div class="csl-right-inline">Pfizer and BioNTech Announce Omicron-Adapted COVID-19 Vaccine Candidates Demonstrate High Immune Response Against Omicron | Pfizer. <a href="https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-omicron-adapted-covid-19">https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-omicron-adapted-covid-19</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-jU9YFYvB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1602. </div><div class="csl-right-inline">Moxon R, Reche PA, Rappuoli R. 2019. <a href="https://doi.org/10.3389/fimmu.2019.02776">Editorial: Reverse Vaccinology</a>. Front Immunol 10.</div>
</div>
<div id="ref-u0dESADU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1603. </div><div class="csl-right-inline">Kudchodkar SB, Choi H, Reuschel EL, Esquivel R, Jin-Ah Kwon J, Jeong M, Maslow JN, Reed CC, White S, Kim JJ, Kobinger GP, Tebas P, Weiner DB, Muthumani K. 2018. <a href="https://doi.org/10.1016/j.micinf.2018.03.001">Rapid response to an emerging infectious disease – Lessons learned from development of a synthetic DNA vaccine targeting Zika virus</a>. Microbes and Infection 20:676–684.</div>
</div>
<div id="ref-wPl93ATP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1604. </div><div class="csl-right-inline">Olena A. 2020. Newer Vaccine Technologies Deployed to Develop COVID-19 Shot. The Scientist Magazine. <a href="https://www.the-scientist.com/news-opinion/newer-vaccine-technologies-deployed-to-develop-covid-19-shot-67152">https://www.the-scientist.com/news-opinion/newer-vaccine-technologies-deployed-to-develop-covid-19-shot-67152</a>.</div>
</div>
<div id="ref-1B4h40dm5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1605. </div><div class="csl-right-inline">Pulliam JRC, van Schalkwyk C, Govender N, Gottberg A von, Cohen C, Groome MJ, Dushoff J, Mlisana K, Moultrie H. 2021. <a href="https://doi.org/10.1101/2021.11.11.21266068">Increased risk of SARS-CoV-2 reinfection associated with emergence of Omicron in South Africa</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-yqFoGUHl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1606. </div><div class="csl-right-inline">2022-07-15 12:20 | Archive of CDC Covid Pages. <a href="https://public4.pagefreezer.com/browse/CDC%20Covid%20Pages/15-07-2022T12:20/https://www.cdc.gov/coronavirus/2019-ncov/science/science-briefs/scientific-brief-omicron-variant.html">https://public4.pagefreezer.com/browse/CDC%20Covid%20Pages/15-07-2022T12:20/https://www.cdc.gov/coronavirus/2019-ncov/science/science-briefs/scientific-brief-omicron-variant.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-x5yLFKk8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1607. </div><div class="csl-right-inline">Planas D, Bruel T, Grzelak L, Guivel-Benhassine F, Staropoli I, Porrot F, Planchais C, Buchrieser J, Rajah MM, Bishop E, Albert M, Donati F, Prot M, Behillil S, Enouf V, Maquart M, Smati-Lafarge M, Varon E, Schortgen F, Yahyaoui L, Gonzalez M, De Sèze J, Péré H, Veyer D, Sève A, Simon-Lorière E, Fafi-Kremer S, Stefic K, Mouquet H, Hocqueloux L, van der Werf S, Prazuck T, Schwartz O. 2021. <a href="https://doi.org/10.1038/s41591-021-01318-5">Sensitivity of infectious SARS-CoV-2 B.1.1.7 and B.1.351 variants to neutralizing antibodies</a>. Nat Med 27:917–924.</div>
</div>
<div id="ref-19dwMfMGe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1608. </div><div class="csl-right-inline">Wang P, Nair MS, Liu L, Iketani S, Luo Y, Guo Y, Wang M, Yu J, Zhang B, Kwong PD, Graham BS, Mascola JR, Chang JY, Yin MT, Sobieszczyk M, Kyratsous CA, Shapiro L, Sheng Z, Huang Y, Ho DD. 2021. <a href="https://doi.org/10.1038/s41586-021-03398-2">Antibody resistance of SARS-CoV-2 variants B.1.351 and B.1.1.7</a>. Nature 593:130–135.</div>
</div>
<div id="ref-63wnlBQD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1609. </div><div class="csl-right-inline">Edara VV, Hudson WH, Xie X, Ahmed R, Suthar MS. 2021. <a href="https://doi.org/10.1001/jama.2021.4388">Neutralizing Antibodies Against SARS-CoV-2 Variants After Infection and Vaccination</a>. JAMA 325:1896.</div>
</div>
<div id="ref-k7L0WGEM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1610. </div><div class="csl-right-inline">Classification of Omicron (B.1.1.529): SARS-CoV-2 Variant of Concern. <a href="https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529)-sars-cov-2-variant-of-concern">https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529)-sars-cov-2-variant-of-concern</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-lexoTbIa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1611. </div><div class="csl-right-inline">Kuhlmann C, Mayer CK, Claassen M, Maponga TG, Sutherland AD, Suliman T, Shaw M, Preiser W. 2021. Breakthrough Infections with SARS-CoV-2 Omicron Variant Despite Booster Dose of mRNA Vaccine. SSRN Journal https://doi.org/<a href="https://doi.org/10.2139/ssrn.3981711">10.2139/ssrn.3981711</a>.</div>
</div>
<div id="ref-YTceJugW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1612. </div><div class="csl-right-inline">Self WH, Tenforde MW, Rhoads JP, Gaglani M, Ginde AA, Douin DJ, Olson SM, Talbot HK, Casey JD, Mohr NM, Zepeski A, McNeal T, Ghamande S, Gibbs KW, Files DC, Hager DN, Shehu A, Prekker ME, Erickson HL, Gong MN, Mohamed A, Henning DJ, Steingrub JS, Peltan ID, Brown SM, Martin ET, Monto AS, Khan A, Hough CL, Busse LW, ten Lohuis CC, Duggal A, Wilson JG, Gordon AJ, Qadir N, Chang SY, Mallow C, Rivas C, Babcock HM, Kwon JH, Exline MC, Halasa N, Chappell JD, Lauring AS, Grijalva CG, Rice TW, Jones ID, Stubblefield WB, Baughman A, Womack KN, Lindsell CJ, Hart KW, Zhu Y, Mills L, Lester SN, Stumpf MM, Naioti EA, Kobayashi M, Verani JR, Thornburg NJ, Patel MM, Calhoun N, Murthy K, Herrick J, McKillop A, Hoffman E, Zayed M, Smith M, Seattle N, Ettlinger J, Priest E, Thomas J, Arroliga A, Beeram M, Kindle R, Kozikowski L-A, De Souza L, Ouellette S, Thornton-Thompson S, Mehkri O, Ashok K, Gole S, King A, Poynter B, Stanley N, Hendrickson A, Maruggi E, Scharber T, Jorgensen J, Bowers R, King J, Aston V, Armbruster B, Rothman RE, Nair R, Chen J-TT, Karow S, Robart E, Maldonado PN, Khan M, So P, Levitt J, Perez C, Visweswaran A, Roque J, Rivera A, Angeles L, Frankel T, Angeles L, Goff J, Huynh D, Howell M, Friedel J, Tozier M, Driver C, Carricato M, Foster A, Nassar P, Stout L, Sibenaller Z, Walter A, Mares J, Olson L, Clinansmith B, Rivas C, Gershengorn H, McSpadden E, Truscon R, Kaniclides A, Thomas L, Bielak R, Valvano WD, Fong R, Fitzsimmons WJ, Blair C, Valesano AL, Gilbert J, Crider CD, Steinbock KA, Paulson TC, Anderson LA, Kampe C, Johnson J, McHenry R, Blair M, Conway D, LaRose M, Landreth L, Hicks M, Parks L, Bongu J, McDonald D, Cass C, Seiler S, Park D, Hink T, Wallace M, Burnham C-A, Arter OG. 2021. <a href="https://doi.org/10.15585/mmwr.mm7038e1">Comparative Effectiveness of Moderna, Pfizer-BioNTech, and Janssen (Johnson &amp;amp; Johnson) Vaccines in Preventing COVID-19 Hospitalizations Among Adults Without Immunocompromising Conditions — United States, March–August 2021</a>. MMWR Morb Mortal Wkly Rep 70:1337–1343.</div>
</div>
<div id="ref-17V4Lh5uy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1613. </div><div class="csl-right-inline">Mueller B, Robbins R. 2021. <a href="https://www.nytimes.com/2021/08/02/world/europe/covax-covid-vaccine-problems-africa.html">Where a Vast Global Vaccination Program Went Wrong</a>. The New York Times.</div>
</div>
<div id="ref-4eOROyon" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1614. </div><div class="csl-right-inline">Sheridan C. 2021. <a href="https://doi.org/10.1038/d41587-021-00001-x">Innovators target vaccines for variants and shortages in global South</a>. Nat Biotechnol 39:393–396.</div>
</div>
<div id="ref-hOkTKQ6z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1615. </div><div class="csl-right-inline">Nohynek H, Wilder-Smith A. 2022. <a href="https://doi.org/10.1056/nejme2204695">Does the World Still Need New Covid-19 Vaccines?</a> N Engl J Med 386:2140–2142.</div>
</div>
<div id="ref-eV6UplSu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1616. </div><div class="csl-right-inline">Bernal JL, Andrews N, Gower C, Stowe J, Robertson C, Tessier E, Simmons R, Cottrell S, Roberts R, O’Doherty M, Brown K, Cameron C, Stockton D, McMenamin J, Ramsay M. 2021. <a href="https://doi.org/10.1101/2021.03.01.21252652">Early effectiveness of COVID-19 vaccination with BNT162b2 mRNA vaccine and ChAdOx1 adenovirus vector vaccine on symptomatic disease, hospitalisations and mortality in older adults in England</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-SxiGicKs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1617. </div><div class="csl-right-inline">Balakrishnan VS. 2020. <a href="https://doi.org/10.1016/s1473-3099(20)30709-x">The arrival of Sputnik V</a>. The Lancet Infectious Diseases 20:1128.</div>
</div>
<div id="ref-3KBugyZN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1618. </div><div class="csl-right-inline">Tumban E. 2020. <a href="https://doi.org/10.3390/v13010054">Lead SARS-CoV-2 Candidate Vaccines: Expectations from Phase III Trials and Recommendations Post-Vaccine Approval</a>. Viruses 13:54.</div>
</div>
<div id="ref-wCTVieA3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1619. </div><div class="csl-right-inline">Mahase E. 2021. <a href="https://doi.org/10.1136/bmj.n309">Covid-19: Russian vaccine efficacy is 91.6%, show phase III trial results</a>. BMJ n309.</div>
</div>
<div id="ref-XC4joIv9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1620. </div><div class="csl-right-inline">2021. Russia cuts size of COVID-19 vaccine study, stops enrollment. AP NEWS. <a href="https://apnews.com/article/coronavirus-pandemic-russia-163068f551e080939b982389b576320a">https://apnews.com/article/coronavirus-pandemic-russia-163068f551e080939b982389b576320a</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-vrzaW9Gb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1621. </div><div class="csl-right-inline">About Sputnik V. <a href="https://sputnikvaccine.com/about-vaccine/">https://sputnikvaccine.com/about-vaccine/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-uOMlbWFc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1622. </div><div class="csl-right-inline">Pagotto V, Ferloni A, Soriano MM, Díaz M, Golde MB, González MI, Asprea V, Staneloni I, Vidal G, Silveira M, Zingoni P, Aliperti V, Michelangelo H, Figar S. 2021. <a href="https://doi.org/10.1101/2021.02.03.21251071">ACTIVE SURVEILLANCE OF THE SPUTNIK V VACCINE IN HEALTH WORKERS</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-jv875POj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1623. </div><div class="csl-right-inline">2021. <a href="https://www.reuters.com/article/health-coronavirus-india-vaccine-idUSL4N2JM2XA">UPDATE 1-Russia's Sputnik V vaccine found safe in India mid-stage trial -Dr.Reddy's</a>. Reuters.</div>
</div>
<div id="ref-QzHvKB8C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1624. </div><div class="csl-right-inline">Fund (RDIF) TRDI. RDIF, The Gamaleya National Center, AstraZeneca and R-Pharm sign an agreement to cooperate on COVID-19 vaccine development. <a href="https://www.prnewswire.com/ae/news-releases/rdif-the-gamaleya-national-center-astrazeneca-and-r-pharm-sign-an-agreement-to-cooperate-on-covid-19-vaccine-development-301196874.html">https://www.prnewswire.com/ae/news-releases/rdif-the-gamaleya-national-center-astrazeneca-and-r-pharm-sign-an-agreement-to-cooperate-on-covid-19-vaccine-development-301196874.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1G2ROkAsZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1625. </div><div class="csl-right-inline">R-Pharm. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04686773">Open-label, Non-randomized, Non-comparative, Phase II Study in Adult Subjects to Assess Safety and Immunogenicity of Combination of AZD1222, a Non-replicating ChAdOx1 Vector Vaccine, and rAd26-S, a Recombinant Adenovirus Type 26 Component of Gam-COVID-Vac Vaccine, for COVID-19 Prevention</a>. NCT04686773. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-AfkC38Sh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1626. </div><div class="csl-right-inline">The first registered COVID-19 vaccine. <a href="https://sputnikvaccine.com/">https://sputnikvaccine.com/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-10uLoe1rR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1627. </div><div class="csl-right-inline">Johnson &amp; Johnson Initiates Pivotal Global Phase 3 Clinical Trial of Janssen’s COVID-19 Vaccine Candidate | Johnson &amp; Johnson. Content Lab US. <a href="https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate">https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-lVoienSE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1628. </div><div class="csl-right-inline">He X, Chandrashekar A, Zahn R, Wegmann F, Yu J, Mercado NB, McMahan K, Martinot AJ, Piedra-Mora C, Beecy S, Ducat S, Chamanza R, Huber SR, van der Fits L, Borducchi EN, Lifton M, Liu J, Nampanya F, Patel S, Peter L, Tostanoski LH, Pessaint L, Van Ry A, Finneyfrock B, Velasco J, Teow E, Brown R, Cook A, Andersen H, Lewis MG, Schuitemaker H, Barouch DH. 2021. <a href="https://doi.org/10.1101/2021.01.27.428380">Low-Dose Ad26.COV2.S Protection Against SARS-CoV-2 Challenge in Rhesus Macaques</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-14g52GtO3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1629. </div><div class="csl-right-inline">Sadoff J, Gars ML, Shukarev G, Heerwegh D, Truyers C, de Groot AM, Stoop J, Tete S, Van Damme W, Leroux-Roels I, Berghmans P-J, Kimmel M, Van Damme P, de Hoon J, Smith W, Stephenson KE, Barouch DH, De Rosa SC, Cohen KW, McElrath MJ, Cormier E, Scheper G, Hendriks J, Struyf F, Douoguih M, Van Hoof J, Schuitemaker H. 2020. <a href="https://doi.org/10.1101/2020.09.23.20199604">Safety and immunogenicity of the Ad26.COV2.S COVID-19 vaccine candidate: interim results of a phase 1/2a, double-blind, randomized, placebo-controlled trial</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-1CBMCD5I2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1630. </div><div class="csl-right-inline">Stephenson KE, Le Gars M, Sadoff J, de Groot AM, Heerwegh D, Truyers C, Atyeo C, Loos C, Chandrashekar A, McMahan K, Tostanoski LH, Yu J, Gebre MS, Jacob-Dolan C, Li Z, Patel S, Peter L, Liu J, Borducchi EN, Nkolola JP, Souza M, Tan CS, Zash R, Julg B, Nathavitharana RR, Shapiro RL, Azim AA, Alonso CD, Jaegle K, Ansel JL, Kanjilal DG, Guiney CJ, Bradshaw C, Tyler A, Makoni T, Yanosick KE, Seaman MS, Lauffenburger DA, Alter G, Struyf F, Douoguih M, Van Hoof J, Schuitemaker H, Barouch DH. 2021. <a href="https://doi.org/10.1001/jama.2021.3645">Immunogenicity of the Ad26.COV2.S Vaccine for COVID-19</a>. JAMA 325:1535.</div>
</div>
<div id="ref-1GRYnvF01" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1631. </div><div class="csl-right-inline">McMahan K, Yu J, Mercado NB, Loos C, Tostanoski LH, Chandrashekar A, Liu J, Peter L, Atyeo C, Zhu A, Bondzie EA, Dagotto G, Gebre MS, Jacob-Dolan C, Li Z, Nampanya F, Patel S, Pessaint L, Van Ry A, Blade K, Yalley-Ogunro J, Cabus M, Brown R, Cook A, Teow E, Andersen H, Lewis MG, Lauffenburger DA, Alter G, Barouch DH. 2020. <a href="https://doi.org/10.1038/s41586-020-03041-6">Correlates of protection against SARS-CoV-2 in rhesus macaques</a>. Nature 590:630–634.</div>
</div>
<div id="ref-7n6WEkK8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1632. </div><div class="csl-right-inline">Janssen Vaccines &amp; Prevention B.V. 2022. <a href="https://clinicaltrials.gov/ct2/show/NCT04505722">A Randomized, Double-blind, Placebo-controlled Phase 3 Study to Assess the Efficacy and Safety of Ad26.COV2.S for the Prevention of SARS-CoV-2-mediated COVID-19 in Adults Aged 18 Years and Older</a>. NCT04505722. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-5BTmfe4Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1633. </div><div class="csl-right-inline">Johnson &amp; Johnson Prepares to Resume Phase 3 ENSEMBLE Trial of its Janssen COVID-19 Vaccine Candidate in the U.S. | Johnson &amp; Johnson. Content Lab US. <a href="https://www.jnj.com/our-company/johnson-johnson-prepares-to-resume-phase-3-ensemble-trial-of-its-janssen-covid-19-vaccine-candidate-in-the-us">https://www.jnj.com/our-company/johnson-johnson-prepares-to-resume-phase-3-ensemble-trial-of-its-janssen-covid-19-vaccine-candidate-in-the-us</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-sx7F1ktj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1634. </div><div class="csl-right-inline">Johnson &amp; Johnson Initiates Second Global Phase 3 Clinical Trial of its Janssen COVID-19 Vaccine Candidate | Johnson &amp; Johnson. Content Lab US. <a href="https://www.jnj.com/johnson-johnson-initiates-second-global-phase-3-clinical-trial-of-its-janssen-covid-19-vaccine-candidate">https://www.jnj.com/johnson-johnson-initiates-second-global-phase-3-clinical-trial-of-its-janssen-covid-19-vaccine-candidate</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-Ude2jR1F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1635. </div><div class="csl-right-inline">Commissioner O of the. 2022. Coronavirus (COVID-19) Update: FDA Authorizes Moderna and Pfizer-BioNTech COVID-19 Vaccines for Children Down to 6 Months of Age. FDA. <a href="https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-moderna-and-pfizer-biontech-covid-19-vaccines-children">https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-moderna-and-pfizer-biontech-covid-19-vaccines-children</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-IMmoqofb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1636. </div><div class="csl-right-inline">Moderna’s COVID-19 Vaccine Candidate Meets its Primary Efficacy Endpoint in the First Interim Analysis of the Phase 3 COVE Study | Moderna, Inc. <a href="https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy/">https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-QEdBZMLe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1637. </div><div class="csl-right-inline">2021. Vaccines and Related Biological Products Advisory Committee December 17, 2020 Meeting Announcement - 12/17/2020 - 12/17/2020. FDA. <a href="https://www.fda.gov/advisory-committees/advisory-committee-calendar/vaccines-and-related-biological-products-advisory-committee-december-17-2020-meeting-announcement">https://www.fda.gov/advisory-committees/advisory-committee-calendar/vaccines-and-related-biological-products-advisory-committee-december-17-2020-meeting-announcement</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-16kGLLlT8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1638. </div><div class="csl-right-inline">FDA/CBER. 2020.
Vaccines and Related Biological Products Advisory Committee December 17, 2020 Meeting Briefing Document - FDA
. <a href="https://www.fda.gov/media/144434/download">https://www.fda.gov/media/144434/download</a>.</div>
</div>
<div id="ref-vkczroFC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1639. </div><div class="csl-right-inline">Moderna Has Completed Case Accrual for First Planned Interim Analysis of its mRNA Vaccine Against COVID-19 (mRNA-1273) | Moderna, Inc. <a href="https://investors.modernatx.com/news-releases/news-release-details/moderna-has-completed-case-accrual-first-planned-interim/">https://investors.modernatx.com/news-releases/news-release-details/moderna-has-completed-case-accrual-first-planned-interim/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-oJWLuU0h" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1640. </div><div class="csl-right-inline">CBER. 2018.
Guidance for Industry: Toxicity Grading Scale for Healthy Adult and Adolescent Volunteers Enrolled in Preventive Vaccine Clinical Trials
. <a href="https://www.fda.gov/media/73679/download">https://www.fda.gov/media/73679/download</a>.</div>
</div>
<div id="ref-HwoGQ6DD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1641. </div><div class="csl-right-inline">Health Canada Authorizes Moderna COVID-19 Vaccine in Canada | Moderna, Inc. <a href="https://investors.modernatx.com/news-releases/news-release-details/health-canada-authorizes-moderna-covid-19-vaccine-canada/">https://investors.modernatx.com/news-releases/news-release-details/health-canada-authorizes-moderna-covid-19-vaccine-canada/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-4hVgIXyi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1642. </div><div class="csl-right-inline">GLANVILLE D. 2021. EMA recommends COVID-19 Vaccine Moderna for authorisation in the EU. European Medicines Agency. Text. <a href="https://www.ema.europa.eu/en/news/ema-recommends-covid-19-vaccine-moderna-authorisation-eu">https://www.ema.europa.eu/en/news/ema-recommends-covid-19-vaccine-moderna-authorisation-eu</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-16hlR7Xgi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1643. </div><div class="csl-right-inline">Pfizer and BioNTech Announce Vaccine Candidate Against COVID-19 Achieved Success in First Interim Analysis from Phase 3 Study | Pfizer. <a href="https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-vaccine-candidate-against">https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-vaccine-candidate-against</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-dlqAfQ7t" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1644. </div><div class="csl-right-inline">Mulligan MJ, Lyke KE, Kitchin N, Absalon J, Gurtman A, Lockhart S, Neuzil K, Raabe V, Bailey R, Swanson KA, Li P, Koury K, Kalina W, Cooper D, Fontes-Garfias C, Shi P-Y, Türeci Ö, Tompkins KR, Walsh EE, Frenck R, Falsey AR, Dormitzer PR, Gruber WC, Şahin U, Jansen KU. 2020. <a href="https://doi.org/10.1038/s41586-020-2639-4">Phase I/II study of COVID-19 RNA vaccine BNT162b1 in adults</a>. Nature 586:589–593.</div>
</div>
<div id="ref-T3BkYtz2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1645. </div><div class="csl-right-inline">Sahin U, Muik A, Derhovanessian E, Vogler I, Kranz LM, Vormehr M, Baum A, Pascal K, Quandt J, Maurus D, Brachtendorf S, Lörks V, Sikorski J, Hilker R, Becker D, Eller A-K, Grützner J, Boesler C, Rosenbaum C, Kühnle M-C, Luxemburger U, Kemmer-Brück A, Langer D, Bexon M, Bolte S, Karikó K, Palanche T, Fischer B, Schultz A, Shi P-Y, Fontes-Garfias C, Perez JL, Swanson KA, Loschko J, Scully IL, Cutler M, Kalina W, Kyratsous CA, Cooper D, Dormitzer PR, Jansen KU, Türeci Ö. 2020. <a href="https://doi.org/10.1038/s41586-020-2814-7">COVID-19 vaccine BNT162b1 elicits human antibody and TH1 T cell responses</a>. Nature 586:594–599.</div>
</div>
<div id="ref-EMWkcH5x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1646. </div><div class="csl-right-inline">Coronavirus COVID-19 Vaccine Update: Latest Developments | Pfizer. <a href="https://www.pfizer.com/science/coronavirus/vaccine">https://www.pfizer.com/science/coronavirus/vaccine</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-Ufs4s7hG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1647. </div><div class="csl-right-inline">Pfizer and BioNTech Conclude Phase 3 Study of COVID-19 Vaccine Candidate, Meeting All Primary Efficacy Endpoints | Pfizer. <a href="https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-conclude-phase-3-study-covid-19-vaccine">https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-conclude-phase-3-study-covid-19-vaccine</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-31rsgfCB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1648. </div><div class="csl-right-inline">Mahase E. 2020. <a href="https://doi.org/10.1136/bmj.m4714">Covid-19: UK approves Pfizer and BioNTech vaccine with rollout due to start next week</a>. BMJ m4714.</div>
</div>
<div id="ref-fhw7l8VO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1649. </div><div class="csl-right-inline">2020. <a href="https://www.bbc.com/news/uk-55227325">Covid-19 vaccine: First person receives Pfizer jab in UK</a>. BBC News.</div>
</div>
<div id="ref-lwxl64fn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1650. </div><div class="csl-right-inline">Commissioner O of the. 2021. FDA Approves First COVID-19 Vaccine. FDA. <a href="https://www.fda.gov/news-events/press-announcements/fda-approves-first-covid-19-vaccine">https://www.fda.gov/news-events/press-announcements/fda-approves-first-covid-19-vaccine</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-gZ33CJWT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1651. </div><div class="csl-right-inline">Aggarwal A, Stella AO, Walker G, Akerman A, Milogiannakis V, Brilot F, Amatayakul-Chantler S, Roth N, Coppola G, Schofield P, Jackson J, Henry JY, Mazigi O, Langley D, Lu Y, Forster C, McAllery S, Mathivanan V, Fichter C, Hoppe AC, Munier ML, Jack H-M, Cromer D, Darley D, Matthews G, Christ D, Khoury D, Davenport M, Rawlinson W, Kelleher AD, Turville S. 2021. <a href="https://doi.org/10.1101/2021.12.14.21267772">SARS-CoV-2 Omicron: evasion of potent humoral responses and resistance to clinical immunotherapeutics relative to viral variants of concern</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-j172syOP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1652. </div><div class="csl-right-inline">Khoury DS, Steain M, Triccas JA, Sigal A, Davenport MP, Cromer D. 2021. <a href="https://doi.org/10.1101/2021.12.13.21267748">A meta-analysis of Early Results to predict Vaccine efficacy against Omicron</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-vJlYzFrS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1653. </div><div class="csl-right-inline">Ferdinands JM, Rao S, Dixon BE, Mitchell PK, DeSilva MB, Irving SA, Lewis N, Natarajan K, Stenehjem E, Grannis SJ, Han J, McEvoy C, Ong TC, Naleway AL, Reese SE, Embi PJ, Dascomb K, Klein NP, Griggs EP, Konatham D, Kharbanda AB, Yang D-H, Fadel WF, Grisel N, Goddard K, Patel P, Liao I-C, Birch R, Valvi NR, Reynolds S, Arndorfer J, Zerbo O, Dickerson M, Murthy K, Williams J, Bozio CH, Blanton L, Verani JR, Schrag SJ, Dalton AF, Wondimu MH, Link-Gelles R, Azziz-Baumgartner E, Barron MA, Gaglani M, Thompson MG, Fireman B. 2022. <a href="https://doi.org/10.15585/mmwr.mm7107e2">Waning 2-Dose and 3-Dose Effectiveness of mRNA Vaccines Against COVID-19–Associated Emergency Department and Urgent Care Encounters and Hospitalizations Among Adults During Periods of Delta and Omicron Variant Predominance — VISION Network, 10 States, August 2021–January 2022</a>. MMWR Morb Mortal Wkly Rep 71:255–263.</div>
</div>
<div id="ref-sNDCRMQ5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1654. </div><div class="csl-right-inline">Zeng C, Evans JP, Chakravarthy K, Qu P, Reisinger S, Song N-J, Rubinstein MP, Shields PG, Li Z, Liu S-L. 2022. <a href="https://doi.org/10.1016/j.ccell.2021.12.014">COVID-19 mRNA booster vaccines elicit strong protection against SARS-CoV-2 Omicron variant in patients with cancer</a>. Cancer Cell 40:117–119.</div>
</div>
<div id="ref-J069om3D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1655. </div><div class="csl-right-inline">Gilbert PB, Montefiori DC, McDermott AB, Fong Y, Benkeser D, Deng W, Zhou H, Houchens CR, Martins K, Jayashankar L, Castellino F, Flach B, Lin BC, O’Connell S, McDanal C, Eaton A, Sarzotti-Kelsoe M, Lu Y, Yu C, Borate B, van der Laan LWP, Hejazi NS, Huynh C, Miller J, El Sahly HM, Baden LR, Baron M, De La Cruz L, Gay C, Kalams S, Kelley CF, Andrasik MP, Kublin JG, Corey L, Neuzil KM, Carpp LN, Pajon R, Follmann D, Donis RO, Koup RA. 2022. <a href="https://doi.org/10.1126/science.abm3425">Immune correlates analysis of the mRNA-1273 COVID-19 vaccine efficacy clinical trial</a>. Science 375:43–50.</div>
</div>
<div id="ref-1AtNPSzpd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1656. </div><div class="csl-right-inline">Goldblatt D, Fiore-Gartland A, Johnson M, Hunt A, Bengt C, Zavadska D, Snipe HD, Brown JS, Workman L, Zar HJ, Montefiori D, Shen X, Dull P, Plotkin S, Siber G, Ambrosino D. 2022. <a href="https://doi.org/10.1016/j.vaccine.2021.12.006">Towards a population-based threshold of protection for COVID-19 vaccines</a>. Vaccine 40:306–315.</div>
</div>
<div id="ref-QYbPf88B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1657. </div><div class="csl-right-inline">Earle KA, Ambrosino DM, Fiore-Gartland A, Goldblatt D, Gilbert PB, Siber GR, Dull P, Plotkin SA. 2021. <a href="https://doi.org/10.1016/j.vaccine.2021.05.063">Evidence for antibody as a protective correlate for COVID-19 vaccines</a>. Vaccine 39:4423–4428.</div>
</div>
<div id="ref-1HCZbWd9m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1658. </div><div class="csl-right-inline">Castillo-Olivares J, Wells DA, Ferrari M, Chan A, Smith P, Nadesalingam A, Paloniemi M, Carnell G, Ohlendorf L, Cantoni D, Mayora-Neto M, Palmer P, Tonks P, Temperton N, Wagner R, Neckermann P, Peterhoff D, Doffinger R, Kempster S, Otter A, Semper A, Brooks T, Page M, Albecka A, James LC, Briggs J, Schwaeble W, Baxendale H, Heeney J. 2021. <a href="https://doi.org/10.1101/2021.05.21.21257572">Towards Internationally standardised humoral Immune Correlates of Protection from SARS-CoV-2 infection and COVID-19 disease</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-4WhXhBth" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1659. </div><div class="csl-right-inline">Legros V, Denolly S, Vogrig M, Boson B, Siret E, Rigaill J, Pillet S, Grattard F, Gonzalo S, Verhoeven P, Allatif O, Berthelot P, Pélissier C, Thiery G, Botelho-Nevers E, Millet G, Morel J, Paul S, Walzer T, Cosset F-L, Bourlet T, Pozzetto B. 2021. <a href="https://doi.org/10.1038/s41423-020-00588-2">A longitudinal study of SARS-CoV-2-infected patients reveals a high correlation between neutralizing antibodies and COVID-19 severity</a>. Cell Mol Immunol 18:318–327.</div>
</div>
<div id="ref-1Mq7vYgc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1660. </div><div class="csl-right-inline">2022. Vaccine development and approval in Canada. <a href="https://www.canada.ca/en/health-canada/services/drugs-health-products/covid19-industry/drugs-vaccines-treatments/vaccines/development-approval-infographic.html">https://www.canada.ca/en/health-canada/services/drugs-health-products/covid19-industry/drugs-vaccines-treatments/vaccines/development-approval-infographic.html</a>.</div>
</div>
<div id="ref-vZF52A03" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1661. </div><div class="csl-right-inline">Vaccines and treatments for COVID-19: Safety after authorization. Government of Canada. <a href="https://www.canada.ca/en/public-health/services/diseases/2019-novel-coronavirus-infection/prevention-risks/covid-19-vaccine-treatment/safety-after-authorization.html">https://www.canada.ca/en/public-health/services/diseases/2019-novel-coronavirus-infection/prevention-risks/covid-19-vaccine-treatment/safety-after-authorization.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-ZkB13SYG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1662. </div><div class="csl-right-inline">. <a href="https://www.canada.ca/en/health-canada/services/drugs-health-products/covid19-industry/drugs-vaccines-treatments/authorization/applications.html">https://www.canada.ca/en/health-canada/services/drugs-health-products/covid19-industry/drugs-vaccines-treatments/authorization/applications.html</a>.</div>
</div>
<div id="ref-fQM1moSe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1663. </div><div class="csl-right-inline">News · JPT·C. An earlier end date for vaccination campaign is 'possible', Trudeau says | CBC News. CBC. <a href="https://www.cbc.ca/news/politics/trudeau-possible-vaccination-campaign-ends-sooner-1.5934994">https://www.cbc.ca/news/politics/trudeau-possible-vaccination-campaign-ends-sooner-1.5934994</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-18BMz232x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1664. </div><div class="csl-right-inline">Dooling K, McClung N, Chamberland M, Marin, M, Wallace M, Bell BP, Lee GM, Talbot HK, Romero JR, Oliver SE. 2020. <a href="https://doi.org/10.15585/mmwr.mm6949e1">The Advisory Committee on Immunization Practices’ Interim Recommendation for Allocating Initial Supplies of COVID-19 Vaccine — United States, 2020</a>. MMWR Morb Mortal Wkly Rep 69:1857–1859.</div>
</div>
<div id="ref-17wU8KTSP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1665. </div><div class="csl-right-inline">Oliver Sara E, Gargano Julia W, Marin M, Wallace M, Curran KG, Chamberland M, McClung N, Campos-Outcalt D, Morgan Rebecca L, Mbaeyi S, Romero José R, Talbot H Keipp, Lee Grace M, Bell Beth P, Dooling K. 2020. <a href="https://doi.org/10.15585/mmwr.mm6950e2">The Advisory Committee on Immunization Practices’ Interim Recommendation for Use of Pfizer-BioNTech COVID-19 Vaccine — United States, December 2020</a>. MMWR Morb Mortal Wkly Rep 69:1922–1924.</div>
</div>
<div id="ref-123cVqUNO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1666. </div><div class="csl-right-inline">News ABC. US administers 1st doses of Pfizer coronavirus vaccine. ABC News. <a href="https://abcnews.go.com/US/us-administer-1st-doses-pfizer-coronavirus-vaccine/story?id=74703018">https://abcnews.go.com/US/us-administer-1st-doses-pfizer-coronavirus-vaccine/story?id=74703018</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-7yAHeCqZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1667. </div><div class="csl-right-inline">Oliver Sara E, Gargano Julia W, Marin M, Wallace M, Curran KG, Chamberland M, McClung N, Campos-Outcalt D, Morgan Rebecca L, Mbaeyi S, Romero José R, Talbot H Keipp, Lee Grace M, Bell Beth P, Dooling K. 2021. <a href="https://doi.org/10.15585/mmwr.mm695152e1">The Advisory Committee on Immunization Practices’ Interim Recommendation for Use of Moderna COVID-19 Vaccine — United States, December 2020</a>. MMWR Morb Mortal Wkly Rep 69:1653–1656.</div>
</div>
<div id="ref-Y3jvGtR9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1668. </div><div class="csl-right-inline">Dooling K, Marin M, Wallace M, McClung N, Chamberland M, Lee GM, Talbot HK, Romero JR, Bell BP, Oliver SE. 2021. <a href="https://doi.org/10.15585/mmwr.mm695152e2">The Advisory Committee on Immunization Practices’ Updated Interim Recommendation for Allocation of COVID-19 Vaccine — United States, December 2020</a>. MMWR Morb Mortal Wkly Rep 69:1657–1660.</div>
</div>
<div id="ref-f5yIh2Xp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1669. </div><div class="csl-right-inline">Almasy MH Holly Yan,Steve. 2020. The Moderna vaccine is now in some Americans' arms as Covid-19 cases in the US pass 18 million. CNN. <a href="https://www.cnn.com/2020/12/21/health/us-coronavirus-monday/index.html">https://www.cnn.com/2020/12/21/health/us-coronavirus-monday/index.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-BG7N9ETs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1670. </div><div class="csl-right-inline">Commissioner O of the. 2022. <a href="https://www.fda.gov/emergency-preparedness-and-response/coronavirus-disease-2019-covid-19/janssen-covid-19-vaccine">Janssen COVID-19 Vaccine</a>. FDA.</div>
</div>
<div id="ref-yNaiGtW1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1671. </div><div class="csl-right-inline">Oliver SE, Gargano JW, Scobie H, Wallace M, Hadler SC, Leung J, Blain AE, McClung N, Campos-Outcalt D, Morgan RL, Mbaeyi S, MacNeil J, Romero JR, Talbot HK, Lee GM, Bell BP, Dooling K. 2021. <a href="https://doi.org/10.15585/mmwr.mm7009e4">The Advisory Committee on Immunization Practices’ Interim Recommendation for Use of Janssen COVID-19 Vaccine — United States, February 2021</a>. MMWR Morb Mortal Wkly Rep 70:329–332.</div>
</div>
<div id="ref-Vd1wOy6d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1672. </div><div class="csl-right-inline">WHO adds Janssen vaccine to list of safe and effective emergency tools against COVID-19. <a href="https://www.who.int/news/item/12-03-2021-who-adds-janssen-vaccine-to-list-of-safe-and-effective-emergency-tools-against-covid-19">https://www.who.int/news/item/12-03-2021-who-adds-janssen-vaccine-to-list-of-safe-and-effective-emergency-tools-against-covid-19</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-1CcsUnCiw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1673. </div><div class="csl-right-inline">Affairs (ASPA) AS for P. 2020. COVID-19 Vaccines. HHSgov. Text. <a href="https://www.hhs.gov/coronavirus/covid-19-vaccines/index.html">https://www.hhs.gov/coronavirus/covid-19-vaccines/index.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-iiTciwn1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1674. </div><div class="csl-right-inline">CDC. 2020. COVID Data Tracker. Centers for Disease Control and Prevention. <a href="https://covid.cdc.gov/covid-data-tracker">https://covid.cdc.gov/covid-data-tracker</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-ZkZ6ToLh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1675. </div><div class="csl-right-inline">Harwood KL Jeff Zeleny,John. 2021. Biden now says US will have enough vaccine for every adult by the end of May | CNN Politics. CNN. <a href="https://www.cnn.com/2021/03/02/politics/biden-merck-johnson--johnson-vaccine/index.html">https://www.cnn.com/2021/03/02/politics/biden-merck-johnson--johnson-vaccine/index.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-13bndHWdk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1676. </div><div class="csl-right-inline">2021. <a href="https://www.bbc.com/news/world-us-canada-55721437">Covid-19: Was US vaccine rollout a 'dismal failure' under Trump?</a> BBC News.</div>
</div>
<div id="ref-XOLE6iJT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1677. </div><div class="csl-right-inline">Camacho AE, Glicksman RL. 2021. <a href="https://papers.ssrn.com/abstract=3770368">Structured to Fail: Lessons from the Trump Administration's Faulty Pandemic Planning and Response</a>. 3770368. SSRN Scholarly Paper. Social Science Research Network, Rochester, NY.</div>
</div>
<div id="ref-I0vakLIc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1678. </div><div class="csl-right-inline">Arthur R. 2021. South Africa starts administering Janssen COVID-19 vaccine to health workers. BioPharma-Reporter. <a href="https://www.biopharma-reporter.com/Article/2021/02/18/South-Africa-starts-administering-Janssen-COVID-19-vaccine-to-health-workers">https://www.biopharma-reporter.com/Article/2021/02/18/South-Africa-starts-administering-Janssen-COVID-19-vaccine-to-health-workers</a>.</div>
</div>
<div id="ref-hYospmts" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1679. </div><div class="csl-right-inline">2021. EMA receives application for conditional marketing authorisation of COVID-19 Vaccine Janssen. <a href="https://www.ema.europa.eu/en/news/ema-receives-application-conditional-marketing-authorisation-covid-19-vaccine-janssen">https://www.ema.europa.eu/en/news/ema-receives-application-conditional-marketing-authorisation-covid-19-vaccine-janssen</a>.</div>
</div>
<div id="ref-hHW8U8rE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1680. </div><div class="csl-right-inline">Rowland C, McGinley L. 2021. <a href="https://www.washingtonpost.com/health/2021/03/02/merck-johnson-and-johnson-covid-vaccine-partnership/">Merck will help make Johnson &amp; Johnson coronavirus vaccine as rivals team up to help Biden accelerate shots</a>. Washington Post.</div>
</div>
<div id="ref-133HGZMEL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1681. </div><div class="csl-right-inline">Ledford H, Cyranoski D, Van Noorden R. 2020. <a href="https://doi.org/10.1038/d41586-020-03441-8">The UK has approved a COVID vaccine — here’s what scientists now want to know</a>. Nature 588:205–206.</div>
</div>
<div id="ref-ddNKvUqH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1682. </div><div class="csl-right-inline">2020. EMA recommends first COVID-19 vaccine for authorisation in the EU. <a href="https://www.ema.europa.eu/en/news/ema-recommends-first-covid-19-vaccine-authorisation-eu">https://www.ema.europa.eu/en/news/ema-recommends-first-covid-19-vaccine-authorisation-eu</a>.</div>
</div>
<div id="ref-1HIxYDTsj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1683. </div><div class="csl-right-inline">Regulatory approval of COVID-19 Vaccine AstraZeneca. GOVUK. <a href="https://www.gov.uk/government/publications/regulatory-approval-of-covid-19-vaccine-astrazeneca">https://www.gov.uk/government/publications/regulatory-approval-of-covid-19-vaccine-astrazeneca</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-q1Ui5Fm8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1684. </div><div class="csl-right-inline">2021. <a href="https://www.bbc.com/news/uk-55525542">Covid: Brian Pinker, 82, first to get Oxford-AstraZeneca vaccine</a>. BBC News.</div>
</div>
<div id="ref-1GGF2oHr5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1685. </div><div class="csl-right-inline">Spikevax (previously COVID-19 Vaccine Moderna). Europeans Medicine Agency. <a href="https://www.ema.europa.eu/en/medicines/human/EPAR/spikevax">https://www.ema.europa.eu/en/medicines/human/EPAR/spikevax</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-k9X9pXJe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1686. </div><div class="csl-right-inline">Regulatory approval of Spikevax (formerly COVID-19 Vaccine Moderna). GOVUK. <a href="https://www.gov.uk/government/publications/regulatory-approval-of-covid-19-vaccine-moderna">https://www.gov.uk/government/publications/regulatory-approval-of-covid-19-vaccine-moderna</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-uvQMgFXB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1687. </div><div class="csl-right-inline">Arkhipov I, Kravchenko S. 2020. Putin Orders Start of Mass Covid-19 Shots Hours After U.K. News. Bloomberg. <a href="https://www.bloomberg.com/news/articles/2020-12-02/within-hours-of-u-k-putin-orders-start-of-mass-covid-19-shots">https://www.bloomberg.com/news/articles/2020-12-02/within-hours-of-u-k-putin-orders-start-of-mass-covid-19-shots</a>.</div>
</div>
<div id="ref-Vwv7l7Hd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1688. </div><div class="csl-right-inline">Rodgers J. Facing Record Covid-19 Case Rise, Russia Rolls Out Sputnik V Vaccine. Forbes. <a href="https://www.forbes.com/sites/jamesrodgerseurope/2020/12/05/facing-record-covid-19-case-rise-russia-rolls-out-sputnik-v-vaccine/">https://www.forbes.com/sites/jamesrodgerseurope/2020/12/05/facing-record-covid-19-case-rise-russia-rolls-out-sputnik-v-vaccine/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-125VEHWS7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1689. </div><div class="csl-right-inline">Ellyatt H. Russia's coronavirus vaccine is alluring for Eastern Europe, creating a headache for the EU. CNBC. <a href="https://www.cnbc.com/2021/03/02/russias-sputnik-vaccine-is-luring-eastern-europe-worrying-the-eu.html">https://www.cnbc.com/2021/03/02/russias-sputnik-vaccine-is-luring-eastern-europe-worrying-the-eu.html</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-tsHhIaZS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1690. </div><div class="csl-right-inline">2021. Clarification on Sputnik V vaccine in the EU approval process. <a href="https://www.ema.europa.eu/en/news/clarification-sputnik-v-vaccine-eu-approval-process">https://www.ema.europa.eu/en/news/clarification-sputnik-v-vaccine-eu-approval-process</a>.</div>
</div>
<div id="ref-ID8IywJM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1691. </div><div class="csl-right-inline">Countries are lining up for Russia's once-scorned Sputnik vaccine after strong efficacy results. Fortune. <a href="https://fortune.com/2021/02/08/international-sputnik-russia-demand/">https://fortune.com/2021/02/08/international-sputnik-russia-demand/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-37uTx4hs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1692. </div><div class="csl-right-inline">RDIF announces delivery of the first batch of Sputnik V vaccine to Venezuela for clinical trials. <a href="https://sputnikvaccine.com/newsroom/pressreleases/rdif-announces-delivery-of-the-first-batch-of-sputnik-v-vaccine-to-venezuela-for-clinical-trials/">https://sputnikvaccine.com/newsroom/pressreleases/rdif-announces-delivery-of-the-first-batch-of-sputnik-v-vaccine-to-venezuela-for-clinical-trials/</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-FAQXPsyc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1693. </div><div class="csl-right-inline">Heath R. Unable to get U.S. vaccines, world turns to Russia and China. POLITICO. <a href="https://www.politico.com/news/2021/02/25/global-vaccine-public-relations-war-471665">https://www.politico.com/news/2021/02/25/global-vaccine-public-relations-war-471665</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-zCPl6A82" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1694. </div><div class="csl-right-inline">2021. Germany moves to bring Russian vaccine into EU orbit. France 24. <a href="https://www.france24.com/en/live-news/20210203-germany-moves-to-bring-russian-vaccine-into-eu-orbit">https://www.france24.com/en/live-news/20210203-germany-moves-to-bring-russian-vaccine-into-eu-orbit</a>. Retrieved 5 December 2022.</div>
</div>
<div id="ref-hpScvlYg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1695. </div><div class="csl-right-inline">2021. <a href="https://www.reuters.com/article/us-health-coronavirus-russia-vaccine-idUSKBN2AK07H">Russia approves its third COVID-19 vaccine, CoviVac</a>. Reuters.</div>
</div>
<div id="ref-Lyd8s4Mk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1696. </div><div class="csl-right-inline">Zabetakis I, Lordan R, Norton C, Tsoupras A. 2020. <a href="https://doi.org/10.3390/nu12051466">COVID-19: The Inflammation Link and the Role of Nutrition in Potential Mitigation</a>. Nutrients 12:1466.</div>
</div>
<div id="ref-UFrQw1gC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1697. </div><div class="csl-right-inline">James PT, Ali Z, Armitage AE, Bonell A, Cerami C, Drakesmith H, Jobe M, Jones KS, Liew Z, Moore SE, Morales-Berstein F, Nabwera HM, Nadjm B, Pasricha S-R, Scheelbeek P, Silver MJ, Teh MR, Prentice AM. 2020. Could nutrition modulate COVID-19 susceptibility and severity of disease? A systematic review. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.10.19.20214395">10.1101/2020.10.19.20214395</a>.</div>
</div>
<div id="ref-Hd5V5Usd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1698. </div><div class="csl-right-inline">Silverio R, Gonçalves DC, Andrade MF, Seelaender M. 2020. <a href="https://doi.org/10.1093/advances/nmaa125">Coronavirus Disease 2019 (COVID-19) and Nutritional Status: The Missing Link?</a> Advances in Nutrition nmaa125.</div>
</div>
<div id="ref-1FAOfFx5A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1699. </div><div class="csl-right-inline">Im JH, Je YS, Baek J, Chung M-H, Kwon HY, Lee J-S. 2020. <a href="https://doi.org/10.1016/j.ijid.2020.08.018">Nutritional status of patients with COVID-19</a>. International Journal of Infectious Diseases 100:390–393.</div>
</div>
<div id="ref-5tHMedSi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1700. </div><div class="csl-right-inline">Calder P, Carr A, Gombart A, Eggersdorfer M. 2020. <a href="https://doi.org/10.3390/nu12041181">Optimal Nutritional Status for a Well-Functioning Immune System Is an Important Factor to Protect against Viral Infections</a>. Nutrients 12:1181.</div>
</div>
<div id="ref-A225e05z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1701. </div><div class="csl-right-inline">Peak dietary supplement sales leveling off during COVID-19 pandemic, but growth still remains strong over last year, market researchers report during webcast. Nutritional Outlook. <a href="https://www.nutritionaloutlook.com/view/peak-dietary-supplement-sales-leveling-during-covid-19-pandemic-growth-still-remains-strong">https://www.nutritionaloutlook.com/view/peak-dietary-supplement-sales-leveling-during-covid-19-pandemic-growth-still-remains-strong</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-LMkJA6Ke" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1702. </div><div class="csl-right-inline">Zhao A, Li Z, Ke Y, Huo S, Ma Y, Zhang Y, Zhang J, Ren Z. 2020. <a href="https://doi.org/10.3390/nu12061699">Dietary Diversity among Chinese Residents during the COVID-19 Outbreak and Its Associated Factors</a>. Nutrients 12:1699.</div>
</div>
<div id="ref-puD7Ydak" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1703. </div><div class="csl-right-inline">nutraingredients-asia.com. Lockdown impact: Grocery stores bolstered NZ supplements sales as pharmacies slumped. nutraingredients-asiacom. <a href="https://www.nutraingredients-asia.com/Article/2020/07/06/Lockdown-impact-Grocery-stores-bolstered-NZ-supplements-sales-as-pharmacies-slumped">https://www.nutraingredients-asia.com/Article/2020/07/06/Lockdown-impact-Grocery-stores-bolstered-NZ-supplements-sales-as-pharmacies-slumped</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-1FjpTy3Dd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1704. </div><div class="csl-right-inline">COVID-19 temporarily bolsters European interest in supplements. nutritioninsightcom/. <a href="https://ni.cnsmedia.com/a/EHHJsDOG2oc=">https://ni.cnsmedia.com/a/EHHJsDOG2oc=</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-5NJabJny" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1705. </div><div class="csl-right-inline">nutraingredients.com. India’s immune health surge: Nation leads APAC in number of new product launches – new data. nutraingredientscom. <a href="https://www.nutraingredients.com/Article/2020/07/21/India-s-immune-health-surge-Nation-leads-APAC-in-number-of-new-product-launches-new-data">https://www.nutraingredients.com/Article/2020/07/21/India-s-immune-health-surge-Nation-leads-APAC-in-number-of-new-product-launches-new-data</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-10JmgRgZG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1706. </div><div class="csl-right-inline">Ayseli YI, Aytekin N, Buyukkayhan D, Aslan I, Ayseli MT. 2020. <a href="https://doi.org/10.1016/j.tifs.2020.09.001">Food policy, nutrition and nutraceuticals in the prevention and management of COVID-19: Advice for healthcare professionals</a>. Trends in Food Science &amp; Technology 105:186–199.</div>
</div>
<div id="ref-18NItmm1M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1707. </div><div class="csl-right-inline">5 Food and Beverage Trends in Europe During COVID-19. <a href="https://kerry.com/insights/kerrydigest/2020/5-food-and-beverage-trends-in-europe-during-covid-19">https://kerry.com/insights/kerrydigest/2020/5-food-and-beverage-trends-in-europe-during-covid-19</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-xLPeszeO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1708. </div><div class="csl-right-inline">McClements DJ, Decker EA, Park Y, Weiss J. 2009. <a href="https://doi.org/10.1080/10408390902841529">Structural Design Principles for Delivery of Bioactive Components in Nutraceuticals and Functional Foods</a>. Critical Reviews in Food Science and Nutrition 49:577–606.</div>
</div>
<div id="ref-2Us0gNMr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1709. </div><div class="csl-right-inline">Moss JWE, Ramji DP. 2016. <a href="https://doi.org/10.1038/nrcardio.2016.103">Nutraceutical therapies for atherosclerosis</a>. Nature Reviews Cardiology 13:513–532.</div>
</div>
<div id="ref-X0jEogv4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1710. </div><div class="csl-right-inline">Kalra EK. 2015. <a href="https://doi.org/10.1208/ps050325">Nutraceutical-definition and introduction</a>. AAPS PharmSci 5:27–28.</div>
</div>
<div id="ref-10Tnnl6nF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1711. </div><div class="csl-right-inline">Dietary Supplement Health and Education Act of 1994. National Institutes of Health Office of Dietary Supplements. <a href="https://ods.od.nih.gov/About/DSHEA_Wording.aspx">https://ods.od.nih.gov/About/DSHEA_Wording.aspx</a>. Retrieved 25 October 1994.</div>
</div>
<div id="ref-16eEQf0HM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1712. </div><div class="csl-right-inline">Commissioner O of the. 2018. Food and Drug Administration Modernization Act (FDAMA) of 1997. FDA. <a href="https://www.fda.gov/regulatory-information/selected-amendments-fdc-act/food-and-drug-administration-modernization-act-fdama-1997">https://www.fda.gov/regulatory-information/selected-amendments-fdc-act/food-and-drug-administration-modernization-act-fdama-1997</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-5TmtJWVQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1713. </div><div class="csl-right-inline">Santini A, Novellino E. 2018. <a href="https://doi.org/10.1080/17512433.2018.1464911">Nutraceuticals - shedding light on the grey area between pharmaceuticals and food</a>. Expert Review of Clinical Pharmacology 11:545–547.</div>
</div>
<div id="ref-CWdgoTUe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1714. </div><div class="csl-right-inline">. <a href="https://eur-lex.europa.eu/legal-content/EN/ALL">https://eur-lex.europa.eu/legal-content/EN/ALL</a>.</div>
</div>
<div id="ref-G7TnkK6t" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1715. </div><div class="csl-right-inline">. <a href="https://eur-lex.europa.eu/legal-content/EN/TXT">https://eur-lex.europa.eu/legal-content/EN/TXT</a>.</div>
</div>
<div id="ref-dzOveIQY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1716. </div><div class="csl-right-inline">EU Register of nutrition and health claims made on foods (v.3.5). <a href="https://ec.europa.eu/food/safety/labelling_nutrition/claims/register/public/">https://ec.europa.eu/food/safety/labelling_nutrition/claims/register/public/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-MhvDTmeu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1717. </div><div class="csl-right-inline">Santini A, Cammarata SM, Capone G, Ianaro A, Tenore GC, Pani L, Novellino E. 2018. <a href="https://doi.org/10.1111/bcp.13496">Nutraceuticals: opening the debate for a regulatory framework</a>. British Journal of Clinical Pharmacology 84:659–672.</div>
</div>
<div id="ref-LBf0kxOm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1718. </div><div class="csl-right-inline">Bröring S, Khedkar S, Ciliberti S. 2016. <a href="https://doi.org/10.1080/09637486.2016.1212816">Reviewing the Nutrition and Health Claims Regulation (EC) No. 1924/2006: What do we know about its challenges and potential impact on innovation?</a> International Journal of Food Sciences and Nutrition 68:1–9.</div>
</div>
<div id="ref-jovk0AV9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1719. </div><div class="csl-right-inline">Dwyer J, Coates P, Smith M. 2018. <a href="https://doi.org/10.3390/nu10010041">Dietary Supplements: Regulatory Challenges and Research Resources</a>. Nutrients 10:41.</div>
</div>
<div id="ref-l2tEuVtb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1720. </div><div class="csl-right-inline">Research C for DE and. 2020. Noetic Nutraceuticals - 607572 - 05/15/2020. Center for Drug Evaluation and Research. <a href="https://www.fda.gov/inspections-compliance-enforcement-and-criminal-investigations/warning-letters/noetic-nutraceuticals-607572-05152020">https://www.fda.gov/inspections-compliance-enforcement-and-criminal-investigations/warning-letters/noetic-nutraceuticals-607572-05152020</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-Qo0C93P6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1721. </div><div class="csl-right-inline">Regulations.gov. <a href="https://beta.regulations.gov/document/FDA-2020-S-0023-0068">https://beta.regulations.gov/document/FDA-2020-S-0023-0068</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-133nfCM4l" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1722. </div><div class="csl-right-inline">Research C for DE and. 2020. Spartan Enterprises Inc. dba Watershed Wellness Center - 610876 - 10/30/2020. Center for Drug Evaluation and Research. <a href="https://www.fda.gov/inspections-compliance-enforcement-and-criminal-investigations/warning-letters/spartan-enterprises-inc-dba-watershed-wellness-center-610876-10302020">https://www.fda.gov/inspections-compliance-enforcement-and-criminal-investigations/warning-letters/spartan-enterprises-inc-dba-watershed-wellness-center-610876-10302020</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-7B3shLSx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1723. </div><div class="csl-right-inline">2020. FTC Sues California Marketer of $23,000 COVID-19 “Treatment” Plan. Federal Trade Commission. <a href="https://www.ftc.gov/news-events/press-releases/2020/07/ftc-sues-california-marketer-23000-covid-19-treatment-plan">https://www.ftc.gov/news-events/press-releases/2020/07/ftc-sues-california-marketer-23000-covid-19-treatment-plan</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-Ssmsdskt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1724. </div><div class="csl-right-inline">. <a href="https://cen.acs.org/biological-chemistry/natural-products/oleandrin-compound-touted-possible-COVID/98/web/2020/08">https://cen.acs.org/biological-chemistry/natural-products/oleandrin-compound-touted-possible-COVID/98/web/2020/08</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-GT7yDBjv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1725. </div><div class="csl-right-inline">Zumla A, Hui DS, Azhar EI, Memish ZA, Maeurer M. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30305-6">Reducing mortality from 2019-nCoV: host-directed therapies should be an option</a>. The Lancet 395:e35–e36.</div>
</div>
<div id="ref-EEa0SZCF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1726. </div><div class="csl-right-inline">Infusino F, Marazzato M, Mancone M, Fedele F, Mastroianni CM, Severino P, Ceccarelli G, Santinelli L, Cavarretta E, Marullo AGM, Miraldi F, Carnevale R, Nocella C, Biondi-Zoccai G, Pagnini C, Schiavon S, Pugliese F, Frati G, d’Ettorre G. 2020. <a href="https://doi.org/10.3390/nu12061718">Diet Supplementation, Probiotics, and Nutraceuticals in SARS-CoV-2 Infection: A Scoping Review</a>. Nutrients 12:1718.</div>
</div>
<div id="ref-2hMFPQJR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1727. </div><div class="csl-right-inline">Zhang L, Liu Y. 2020. <a href="https://doi.org/10.1002/jmv.25707">Potential interventions for novel coronavirus in China: A systematic review</a>. Journal of Medical Virology 92:479–490.</div>
</div>
<div id="ref-eyBAUUCl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1728. </div><div class="csl-right-inline">McCarty MF, DiNicolantonio JJ. 2020. <a href="https://doi.org/10.1016/j.pcad.2020.02.007">Nutraceuticals have potential for boosting the type 1 interferon response to RNA viruses including influenza and coronavirus</a>. Progress in Cardiovascular Diseases 63:383–385.</div>
</div>
<div id="ref-Ufvsmc3N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1729. </div><div class="csl-right-inline">Lordan R, Redfern S, Tsoupras A, Zabetakis I. 2020. <a href="https://doi.org/10.1039/c9fo01742a">Inflammation and cardiovascular disease: are marine phospholipids the answer?</a> Food &amp; Function 11:2861–2885.</div>
</div>
<div id="ref-Gzqj7Okm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1730. </div><div class="csl-right-inline">Szabó Z, Marosvölgyi T, Szabó É, Bai P, Figler M, Verzár Z. 2020. <a href="https://doi.org/10.3389/fphys.2020.00752">The Potential Beneficial Effect of EPA and DHA Supplementation Managing Cytokine Storm in Coronavirus Disease</a>. Frontiers in Physiology 11:752.</div>
</div>
<div id="ref-Txn1mlRl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1731. </div><div class="csl-right-inline">Saha S, Murray P. 2018. <a href="https://doi.org/10.3390/fermentation4020046">Exploitation of Microalgae Species for Nutraceutical Purposes: Cultivation Aspects</a>. Fermentation 4:46.</div>
</div>
<div id="ref-S1A3WazG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1732. </div><div class="csl-right-inline">Ratha SK, Renuka N, Rawat I, Bux F. 2021. <a href="https://doi.org/10.1016/j.nut.2020.111089">Prospective options of algae-derived nutraceuticals as supplements to combat COVID-19 and human coronavirus diseases</a>. Nutrition 83:111089.</div>
</div>
<div id="ref-4xymCvKH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1733. </div><div class="csl-right-inline">Schmidt EB, Møller JM, Svaneborg N, Dyerberg J. 2012. <a href="https://doi.org/10.1007/bf03257413">Safety Aspects of Fish Oils</a>. Drug Investigation 7:215–220.</div>
</div>
<div id="ref-17CHOdNbe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1734. </div><div class="csl-right-inline">Update on Seafood Consumption During Pregnancy. <a href="https://www.acog.org/en/Clinical/Clinical Guidance/Practice Advisory/Articles/2017/01/Update on Seafood Consumption During Pregnancy">https://www.acog.org/en/Clinical/Clinical Guidance/Practice Advisory/Articles/2017/01/Update on Seafood Consumption During Pregnancy</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-g4dU6DMd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1735. </div><div class="csl-right-inline">Greenberg JA, Bell SJ, Ausdal WV. 2008. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2621042">Omega-3 Fatty Acid supplementation during pregnancy</a>. Rev Obstet Gynecol 1:162–9.</div>
</div>
<div id="ref-JNyhLAfe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1736. </div><div class="csl-right-inline">Calder PC. 2013. <a href="https://doi.org/10.1111/j.1365-2125.2012.04374.x">Omega-3 polyunsaturated fatty acids and inflammatory processes: nutrition or pharmacology?</a> British Journal of Clinical Pharmacology 75:645–662.</div>
</div>
<div id="ref-cvX4fl0q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1737. </div><div class="csl-right-inline">Calder PC. 2015. <a href="https://doi.org/10.1016/j.bbalip.2014.08.010">Marine omega-3 fatty acids and inflammatory processes: Effects, mechanisms and clinical relevance</a>. Biochimica et Biophysica Acta (BBA) - Molecular and Cell Biology of Lipids 1851:469–484.</div>
</div>
<div id="ref-165UR0Ipw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1738. </div><div class="csl-right-inline">Whelan J, Gowdy KM, Shaikh SR. 2016. <a href="https://doi.org/10.1016/j.ejphar.2015.03.100">N-3 polyunsaturated fatty acids modulate B cell activity in pre-clinical models: Implications for the immune response to infections</a>. European Journal of Pharmacology 785:10–17.</div>
</div>
<div id="ref-TTfh7mb9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1739. </div><div class="csl-right-inline">Asher A, Tintle NL, Myers M, Lockshon L, Bacareza H, Harris WS. 2021. <a href="https://doi.org/10.1016/j.plefa.2021.102250">Blood omega-3 fatty acids and death from COVID-19: A pilot study</a>. Prostaglandins, Leukotrienes and Essential Fatty Acids 166:102250.</div>
</div>
<div id="ref-FtxL59jJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1740. </div><div class="csl-right-inline">Tian T, Zhao Y, Huang Q, Li J. 2016. <a href="https://doi.org/10.1007/s11745-016-4129-x">n-3 Polyunsaturated Fatty Acids Improve Inflammation via Inhibiting Sphingosine Kinase 1 in a Rat Model of Parenteral Nutrition and CLP-Induced Sepsis</a>. Lipids 51:271–278.</div>
</div>
<div id="ref-12vyTsM9W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1741. </div><div class="csl-right-inline">Das UN. 2019. <a href="https://doi.org/10.1016/j.nut.2019.02.016">Polyunsaturated fatty acids and sepsis</a>. Nutrition 65:39–43.</div>
</div>
<div id="ref-LmdFoxGc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1742. </div><div class="csl-right-inline">Sabater J, Masclans J, Sacanell J, Chacon P, Sabin P, Planas M. 2011. <a href="https://doi.org/10.1186/1743-7075-8-22">Effects of an omega-3 fatty acid-enriched lipid emulsion on eicosanoid synthesis in acute respiratory distress syndrome (ARDS): A prospective, randomized, double-blind, parallel group study</a>. Nutrition &amp; Metabolism 8:22.</div>
</div>
<div id="ref-q9FzffQx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1743. </div><div class="csl-right-inline">Dushianthan A, Cusack R, Burgess VA, Grocott MP, Calder P. 2020. <a href="https://doi.org/10.4187/respcare.06965">Immunonutrition for Adults With ARDS: Results From a Cochrane Systematic Review and Meta-Analysis</a>. Respiratory Care 65:99–110.</div>
</div>
<div id="ref-EwajTsK5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1744. </div><div class="csl-right-inline">Chen H, Wang S, Zhao Y, Luo Y, Tong H, Su L. 2018. <a href="https://doi.org/10.1186/s12937-018-0356-8">Correlation analysis of omega-3 fatty acids and mortality of sepsis and sepsis-induced ARDS in adults: data from previous randomized controlled trials</a>. Nutrition Journal 17:57.</div>
</div>
<div id="ref-BIVORG2X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1745. </div><div class="csl-right-inline">Buckley Christopher D, Gilroy Derek W, Serhan Charles N. 2014. <a href="https://doi.org/10.1016/j.immuni.2014.02.009">Proresolving Lipid Mediators and Mechanisms in the Resolution of Acute Inflammation</a>. Immunity 40:315–327.</div>
</div>
<div id="ref-uPOTvvjR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1746. </div><div class="csl-right-inline">Basil MC, Levy BD. 2015. <a href="https://doi.org/10.1038/nri.2015.4">Specialized pro-resolving mediators: endogenous regulators of infection and inflammation</a>. Nature Reviews Immunology 16:51–67.</div>
</div>
<div id="ref-sxgmhJ68" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1747. </div><div class="csl-right-inline">Sandhaus S, Swick AG. 2020. Specialized mediators in infection and lung injury. BioFactors https://doi.org/<a href="https://doi.org/10.1002/biof.1691">10.1002/biof.1691</a>.</div>
</div>
<div id="ref-OSEnHt6H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1748. </div><div class="csl-right-inline">Serhan CN. 2014. <a href="https://doi.org/10.1038/nature13479">Pro-resolving lipid mediators are leads for resolution physiology</a>. Nature 510:92–101.</div>
</div>
<div id="ref-MK0GQ55y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1749. </div><div class="csl-right-inline">Ramon S, Baker SF, Sahler JM, Kim N, Feldsott EA, Serhan CN, Martínez-Sobrido L, Topham DJ, Phipps RP. 2014. <a href="https://doi.org/10.4049/jimmunol.1302795">The Specialized Proresolving Mediator 17-HDHA Enhances the Antibody-Mediated Immune Response against Influenza Virus: A New Class of Adjuvant?</a> The Journal of Immunology 193:6031–6040.</div>
</div>
<div id="ref-fIo3ESPM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1750. </div><div class="csl-right-inline">Morita M, Kuba K, Ichikawa A, Nakayama M, Katahira J, Iwamoto R, Watanebe T, Sakabe S, Daidoji T, Nakamura S, Kadowaki A, Ohto T, Nakanishi H, Taguchi R, Nakaya T, Murakami M, Yoneda Y, Arai H, Kawaoka Y, Penninger Josef M, Arita M, Imai Y. 2013. <a href="https://doi.org/10.1016/j.cell.2013.02.027">The Lipid Mediator Protectin D1 Inhibits Influenza Virus Replication and Improves Severe Influenza</a>. Cell 153:112–125.</div>
</div>
<div id="ref-14LjKv9ZH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1751. </div><div class="csl-right-inline">Panigrahy D, Gilligan MM, Huang S, Gartung A, Cortés-Puch I, Sime PJ, Phipps RP, Serhan CN, Hammock BD. 2020. <a href="https://doi.org/10.1007/s10555-020-09889-4">Inflammation resolution: a dual-pronged approach to averting cytokine storms in COVID-19?</a> Cancer and Metastasis Reviews 39:337–340.</div>
</div>
<div id="ref-PRLlghQS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1752. </div><div class="csl-right-inline">Regidor P-A, Santos FG, Rizo JM, Egea FM. 2020. <a href="https://doi.org/10.1016/j.mehy.2020.110340">Pro resolving inflammatory effects of the lipid mediators of omega 3 fatty acids and its implication in SARS COVID-19</a>. Medical Hypotheses 145:110340.</div>
</div>
<div id="ref-oLunKkIG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1753. </div><div class="csl-right-inline">Pal A, Gowdy KM, Oestreich KJ, Beck M, Shaikh SR. 2020. <a href="https://doi.org/10.3389/fimmu.2020.01997">Obesity-Driven Deficiencies of Specialized Pro-resolving Mediators May Drive Adverse Outcomes During SARS-CoV-2 Infection</a>. Frontiers in Immunology 11:1997.</div>
</div>
<div id="ref-B8toe6Mr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1754. </div><div class="csl-right-inline">Schwerbrock NMJ, Karlsson EA, Shi Q, Sheridan PA, Beck MA. 2009. <a href="https://doi.org/10.3945/jn.109.108027">Fish Oil-Fed Mice Have Impaired Resistance to Influenza Infection</a>. The Journal of Nutrition 139:1588–1594.</div>
</div>
<div id="ref-3Xht24k3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1755. </div><div class="csl-right-inline">Husson M-O, Ley D, Portal C, Gottrand M, Hueso T, Desseyn J-L, Gottrand F. 2016. <a href="https://doi.org/10.1016/j.jinf.2016.10.001">Modulation of host defence against bacterial and viral infections by omega-3 polyunsaturated fatty acids</a>. Journal of Infection 73:523–535.</div>
</div>
<div id="ref-Ge00xeR3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1756. </div><div class="csl-right-inline">Skarke C, Alamuddin N, Lawson JA, Li X, Ferguson JF, Reilly MP, FitzGerald GA. 2015. <a href="https://doi.org/10.1194/jlr.m060392">Bioactive products formed in humans from fish oils</a>. Journal of Lipid Research 56:1808–1820.</div>
</div>
<div id="ref-n93SvQ2p" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1757. </div><div class="csl-right-inline">Bäck M. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04647604">Resolving Inflammatory Storm in COVID-19 Patients by Omega-3 Polyunsaturated Fatty Acids - A Single-blind, Randomized, Placebo-controlled Feasibility Study</a>. NCT04647604. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-pIfWCVkv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1758. </div><div class="csl-right-inline">Arnardottir H, Pawelzik S-C, Öhlund Wistbacka U, Artiach G, Hofmann R, Reinholdsson I, Braunschweig F, Tornvall P, Religa D, Bäck M. 2021. <a href="https://doi.org/10.3389/fphys.2020.624657">Stimulating the Resolution of Inflammation Through Omega-3 Polyunsaturated Fatty Acids in COVID-19: Rationale for the COVID-Omega-F Trial</a>. Frontiers in Physiology 11:624657.</div>
</div>
<div id="ref-17jegDTvF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1759. </div><div class="csl-right-inline">Connors JM, Levy JH. 2020. <a href="https://doi.org/10.1182/blood.2020006000">COVID-19 and its implications for thrombosis and anticoagulation</a>. Blood 135:2033–2040.</div>
</div>
<div id="ref-SxPCbOB1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1760. </div><div class="csl-right-inline">Bikdeli B, Madhavan MV, Jimenez D, Chuich T, Dreyfus I, Driggin E, Nigoghossian CD, Ageno W, Madjid M, Guo Y, Tang LV, Hu Y, Giri J, Cushman M, Quéré I, Dimakakos EP, Gibson CM, Lippi G, Favaloro EJ, Fareed J, Caprini JA, Tafur AJ, Burton JR, Francese DP, Wang EY, Falanga A, McLintock C, Hunt BJ, Spyropoulos AC, Barnes GD, Eikelboom JW, Weinberg I, Schulman S, Carrier M, Piazza G, Beckman JA, Steg PG, Stone GW, Rosenkranz S, Goldhaber SZ, Parikh SA, Monreal M, Krumholz HM, Konstantinides SV, Weitz JI, Lip GYH. 2020. <a href="https://doi.org/10.1016/j.jacc.2020.04.031">COVID-19 and Thrombotic or Thromboembolic Disease: Implications for Prevention, Antithrombotic Therapy, and Follow-Up</a>. Journal of the American College of Cardiology 75:2950–2973.</div>
</div>
<div id="ref-rWVL91jq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1761. </div><div class="csl-right-inline">Tsoupras A, Lordan R, Zabetakis I. 2020. <a href="https://doi.org/10.3389/fnut.2020.583080">Thrombosis and COVID-19: The Potential Role of Nutrition</a>. Frontiers in Nutrition 7:583080.</div>
</div>
<div id="ref-umzPtQG7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1762. </div><div class="csl-right-inline">Adili R, Hawley M, Holinstat M. 2018. <a href="https://doi.org/10.1016/j.prostaglandins.2018.09.005">Regulation of platelet function and thrombosis by omega-3 and omega-6 polyunsaturated fatty acids</a>. Prostaglandins &amp; Other Lipid Mediators 139:10–18.</div>
</div>
<div id="ref-V9MDMfpj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1763. </div><div class="csl-right-inline">Lordan R, Tsoupras A, Zabetakis I. 2020. <a href="https://doi.org/10.1016/j.blre.2020.100694">Platelet activation and prothrombotic mediators at the nexus of inflammation and atherosclerosis: Potential role of antiplatelet agents</a>. Blood Reviews 100694.</div>
</div>
<div id="ref-QKHdIx2w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1764. </div><div class="csl-right-inline">An Investigation on the Effects of Icosapent Ethyl (VascepaTM) on Inflammatory Biomarkers in Individuals With COVID-19 - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04412018">https://clinicaltrials.gov/ct2/show/NCT04412018</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-12Q4bY0vF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1765. </div><div class="csl-right-inline">Bhatt DL, Steg PG, Miller M, Brinton EA, Jacobson TA, Ketchum SB, Doyle RT, Juliano RA, Jiao L, Granowitz C, Tardif J-C, Ballantyne CM. 2019. <a href="https://doi.org/10.1056/nejmoa1812792">Cardiovascular Risk Reduction with Icosapent Ethyl for Hypertriglyceridemia</a>. New England Journal of Medicine 380:11–22.</div>
</div>
<div id="ref-8lPjdnE7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1766. </div><div class="csl-right-inline">S.L.A. Pharma AG. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04335032">A Randomised, Double-blind, Placebo Controlled Study of Eicosapentaenoic Acid (EPA-FFA) Gastro-resistant Capsules to Treat Hospitalised Subjects With Confirmed SARS-CoV-2</a>. NCT04335032. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-1AGVAbQuf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1767. </div><div class="csl-right-inline">FACN MA M. D. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04323228">Anti-inflammatory/Antioxidant Oral Nutrition Supplementation on the Cytokine Storm and Progression of COVID-19: A Randomized Controlled Trial</a>. NCT04323228. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-XHtzuR5c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1768. </div><div class="csl-right-inline">Messina G, Polito R, Monda V, Cipolloni L, Di Nunno N, Di Mizio G, Murabito P, Carotenuto M, Messina A, Pisanelli D, Valenzano A, Cibelli G, Scarinci A, Monda M, Sessa F. 2020. <a href="https://doi.org/10.3390/ijms21093104">Functional Role of Dietary Intervention to Improve the Outcome of COVID-19: A Hypothesis of Work</a>. International Journal of Molecular Sciences 21:3104.</div>
</div>
<div id="ref-VMI1lOFR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1769. </div><div class="csl-right-inline">Maares M, Haase H. 2016. <a href="https://doi.org/10.1016/j.abb.2016.03.022">Zinc and immunity: An essential interrelation</a>. Archives of Biochemistry and Biophysics 611:58–65.</div>
</div>
<div id="ref-Qif5U0Ig" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1770. </div><div class="csl-right-inline">von Bülow V, Dubben S, Engelhardt G, Hebel S, Plümäkers B, Heine H, Rink L, Haase H. 2007. <a href="https://doi.org/10.4049/jimmunol.179.6.4180">Zinc-Dependent Suppression of TNF-α Production Is Mediated by Protein Kinase A-Induced Inhibition of Raf-1, IκB Kinase β, and NF-κB</a>. The Journal of Immunology 179:4180–4186.</div>
</div>
<div id="ref-1CEzU1G34" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1771. </div><div class="csl-right-inline">Prasad AS, Bao B, Beck FWJ, Sarkar FH. 2001. <a href="https://doi.org/10.1067/mlc.2001.118108">Zinc activates NF-κB in HUT-78 cells</a>. Journal of Laboratory and Clinical Medicine 138:250–256.</div>
</div>
<div id="ref-7ElRN12v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1772. </div><div class="csl-right-inline">Vivier E, Raulet DH, Moretta A, Caligiuri MA, Zitvogel L, Lanier LL, Yokoyama WM, Ugolini S. 2011. <a href="https://doi.org/10.1126/science.1198687">Innate or Adaptive Immunity? The Example of Natural Killer Cells</a>. Science 331:44–49.</div>
</div>
<div id="ref-kxuSwrCI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1773. </div><div class="csl-right-inline">Prasad AS, Beck FW, Bao B, Fitzgerald JT, Snell DC, Steinberg JD, Cardozo LJ. 2007. <a href="https://doi.org/10.1093/ajcn/85.3.837">Zinc supplementation decreases incidence of infections in the elderly: effect of zinc on generation of cytokines and oxidative stress</a>. The American Journal of Clinical Nutrition 85:837–844.</div>
</div>
<div id="ref-IAOnO0Gj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1774. </div><div class="csl-right-inline">Read SA, Obeid S, Ahlenstiel C, Ahlenstiel G. 2019. <a href="https://doi.org/10.1093/advances/nmz013">The Role of Zinc in Antiviral Immunity</a>. Advances in Nutrition 10:696–710.</div>
</div>
<div id="ref-sgr0mlJK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1775. </div><div class="csl-right-inline">Hulisz D. 2004. <a href="https://doi.org/10.1331/1544-3191.44.5.594.hulisz">Efficacy of Zinc Against Common Cold Viruses: An Overview</a>. Journal of the American Pharmacists Association 44:594–603.</div>
</div>
<div id="ref-lPexOhy6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1776. </div><div class="csl-right-inline">Hemilä H. 2011. <a href="https://doi.org/10.2174/1874306401105010051">Zinc Lozenges May Shorten the Duration of Colds: A Systematic Review</a>. The Open Respiratory Medicine Journal 5:51–58.</div>
</div>
<div id="ref-1BU6jG2lv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1777. </div><div class="csl-right-inline">Jothimani D, Kailasam E, Danielraj S, Nallathambi B, Ramachandran H, Sekar P, Manoharan S, Ramani V, Narasimhan G, Kaliamoorthy I, Rela M. 2020. <a href="https://doi.org/10.1016/j.ijid.2020.09.014">COVID-19: Poor outcomes in patients with zinc deficiency</a>. International Journal of Infectious Diseases 100:343–349.</div>
</div>
<div id="ref-6aOUJdHa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1778. </div><div class="csl-right-inline">te Velthuis AJW, van den Worm SHE, Sims AC, Baric RS, Snijder EJ, van Hemert MJ. 2010. <a href="https://doi.org/10.1371/journal.ppat.1001176">Zn2+ Inhibits Coronavirus and Arterivirus RNA Polymerase Activity In Vitro and Zinc Ionophores Block the Replication of These Viruses in Cell Culture</a>. PLoS Pathogens 6:e1001176.</div>
</div>
<div id="ref-NB1udFHF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1779. </div><div class="csl-right-inline">Báez-Santos YM, St. John SE, Mesecar AD. 2015. <a href="https://doi.org/10.1016/j.antiviral.2014.12.015">The SARS-coronavirus papain-like protease: Structure, function and inhibition by designed antiviral compounds</a>. Antiviral Research 115:21–38.</div>
</div>
<div id="ref-XJrj3MiZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1780. </div><div class="csl-right-inline">MD AT. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04370782">A Randomized Study Evaluating the Safety and Efficacy of Hydroxychloroquine and Zinc in Combination With Either Azithromycin or Doxycycline for the Treatment of COVID-19 in the Outpatient Setting</a>. NCT04370782. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-1EqFW3G4X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1781. </div><div class="csl-right-inline">A Study of Hydroxychloroquine and Zinc in the Prevention of COVID-19 Infection in Military Healthcare Workers - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04377646">https://clinicaltrials.gov/ct2/show/NCT04377646</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-Hc9mjVB2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1782. </div><div class="csl-right-inline">National Institute of Integrative Medicine, Australia. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04395768">Therapies to Prevent Progression of COVID-19, Including Hydroxychloroquine, Azithromycin, Zinc, Vitamin D, Vitamin B12 With or Without Vitamin C, a Multi-centre, International, Randomized Trial: The International ALLIANCE Study</a>. NCT04395768. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-T0aXNTIp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1783. </div><div class="csl-right-inline">Early Intervention in COVID-19: Favipiravir Verses Standard Care - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04373733">https://clinicaltrials.gov/ct2/show/NCT04373733</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-SmqHOsyB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1784. </div><div class="csl-right-inline">Cavalcanti AB, Zampieri FG, Rosa RG, Azevedo LCP, Veiga VC, Avezum A, Damiani LP, Marcadenti A, Kawano-Dourado L, Lisboa T, Junqueira DLM, de Barros e Silva PGM, Tramujas L, Abreu-Silva EO, Laranjeira LN, Soares AT, Echenique LS, Pereira AJ, Freitas FGR, Gebara OCE, Dantas VCS, Furtado RHM, Milan EP, Golin NA, Cardoso FF, Maia IS, Hoffmann Filho CR, Kormann APM, Amazonas RB, Bocchi de Oliveira MF, Serpa-Neto A, Falavigna M, Lopes RD, Machado FR, Berwanger O. 2020. <a href="https://doi.org/10.1056/nejmoa2019014">Hydroxychloroquine with or without Azithromycin in Mild-to-Moderate Covid-19</a>. New England Journal of Medicine 383:2041–2052.</div>
</div>
<div id="ref-T94OBz6q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1785. </div><div class="csl-right-inline">Lewis K, Chaudhuri D, Alshamsi F, Carayannopoulos L, Dearness K, Chagla Z, Alhazzani W, for the GUIDE Group. 2021. <a href="https://doi.org/10.1371/journal.pone.0244778">The efficacy and safety of hydroxychloroquine for COVID-19 prophylaxis: A systematic review and meta-analysis of randomized trials</a>. PLOS ONE 16:e0244778.</div>
</div>
<div id="ref-1F4wtzwpV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1786. </div><div class="csl-right-inline">Fiolet T, Guihur A, Rebeaud ME, Mulot M, Peiffer-Smadja N, Mahamat-Saleh Y. 2021. <a href="https://doi.org/10.1016/j.cmi.2020.08.022">Effect of hydroxychloroquine with or without azithromycin on the mortality of coronavirus disease 2019 (COVID-19) patients: a systematic review and meta-analysis</a>. Clinical Microbiology and Infection 27:19–27.</div>
</div>
<div id="ref-Xi9zDqwZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1787. </div><div class="csl-right-inline">Carlucci PM, Ahuja T, Petrilli C, Rajagopalan H, Jones S, Rahimian J. 2020. <a href="https://doi.org/10.1099/jmm.0.001250">Zinc sulfate in combination with a zinc ionophore may improve outcomes in hospitalized COVID-19 patients</a>. Journal of Medical Microbiology 69:1228–1234.</div>
</div>
<div id="ref-110pLAjkl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1788. </div><div class="csl-right-inline">Yao JS, Paguio JA, Dee EC, Tan HC, Moulick A, Milazzo C, Jurado J, Della Penna N, Celi LA. 2021. <a href="https://doi.org/10.1016/j.chest.2020.06.082">The Minimal Effect of Zinc on the Survival of Hospitalized Patients With COVID-19</a>. Chest 159:108–111.</div>
</div>
<div id="ref-1GGbb9IZt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1789. </div><div class="csl-right-inline">Desai M. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04342728">Coronavirus Disease 2019- Using Ascorbic Acid and Zinc Supplementation (COVIDAtoZ) Research Study A Randomized, Open Label Single Center Study</a>. NCT04342728. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-3RCeQRRc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1790. </div><div class="csl-right-inline">Narayanan N, Nair DT. 2020. Vitamin B12 May Inhibit RNA-Dependent-RNA Polymerase Activity of nsp12 from the COVID-19 Virus. Preprints https://doi.org/<a href="https://doi.org/10.20944/preprints202003.0347.v1">10.20944/preprints202003.0347.v1</a>.</div>
</div>
<div id="ref-190FMJu3y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1791. </div><div class="csl-right-inline">Cerullo G, Negro M, Parimbelli M, Pecoraro M, Perna S, Liguori G, Rondanelli M, Cena H, D’Antona G. 2020. <a href="https://doi.org/10.3389/fimmu.2020.574029">The Long History of Vitamin C: From Prevention of the Common Cold to Potential Aid in the Treatment of COVID-19</a>. Frontiers in Immunology 11:574029.</div>
</div>
<div id="ref-EbAGaMxL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1792. </div><div class="csl-right-inline">Carr AC, Rowe S. 2020. <a href="https://doi.org/10.3390/nu12113286">The Emerging Role of Vitamin C in the Prevention and Treatment of COVID-19</a>. Nutrients 12:3286.</div>
</div>
<div id="ref-9czQt5cR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1793. </div><div class="csl-right-inline">Chen Y, Luo G, Yuan J, Wang Y, Yang X, Wang X, Li G, Liu Z, Zhong N. 2014. <a href="https://doi.org/10.1155/2014/426740">Vitamin C Mitigates Oxidative Stress and Tumor Necrosis Factor-Alpha in Severe Community-Acquired Pneumonia and LPS-Induced Macrophages</a>. Mediators of Inflammation 2014:1–11.</div>
</div>
<div id="ref-RjFxOURM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1794. </div><div class="csl-right-inline">Hagel AF, Layritz CM, Hagel WH, Hagel H-J, Hagel E, Dauth W, Kressel J, Regnet T, Rosenberg A, Neurath MF, Molderings GJ, Raithel M. 2013. <a href="https://doi.org/10.1007/s00210-013-0880-1">Intravenous infusion of ascorbic acid decreases serum histamine concentrations in patients with allergic and non-allergic diseases</a>. Naunyn-Schmiedeberg's Archives of Pharmacology 386:789–793.</div>
</div>
<div id="ref-13M3VAfY9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1795. </div><div class="csl-right-inline">Carr A, Maggini S. 2017. <a href="https://doi.org/10.3390/nu9111211">Vitamin C and Immune Function</a>. Nutrients 9:1211.</div>
</div>
<div id="ref-uQFCDDIy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1796. </div><div class="csl-right-inline">Hume R, Weyers E. 2016. <a href="https://doi.org/10.1177/003693307301800102">Changes in Leucocyte Ascorbic Acid during the Common Cold</a>. Scottish Medical Journal 18:3–7.</div>
</div>
<div id="ref-10nBT1HbX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1797. </div><div class="csl-right-inline">Wilson CWM. 1975. <a href="https://doi.org/10.1111/j.1749-6632.1975.tb29312.x">ASCORBIC ACID FUNCTION AND METABOLISM DURING COLDS</a>. Annals of the New York Academy of Sciences 258:529–539.</div>
</div>
<div id="ref-1Bj8VMlSk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1798. </div><div class="csl-right-inline">Davies JEW, Hughes RE, Jones E, Reed SE, Craig JW, Tyrrell DAJ. 1979. <a href="https://doi.org/10.1016/0006-2944(79)90058-9">Metabolism of ascorbic acid (vitamin C) in subjects infected with common cold viruses</a>. Biochemical Medicine 21:78–85.</div>
</div>
<div id="ref-aEEZ8Pq9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1799. </div><div class="csl-right-inline">Hemilä H. 2017. <a href="https://doi.org/10.3390/nu9040339">Vitamin C and Infections</a>. Nutrients 9:339.</div>
</div>
<div id="ref-gZRPoTM9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1800. </div><div class="csl-right-inline">Hemilä H. 2007. <a href="https://doi.org/10.1079/bjn19920004">Vitamin C and the common cold</a>. British Journal of Nutrition 67:3–16.</div>
</div>
<div id="ref-DPhRTQw0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1801. </div><div class="csl-right-inline">Hemilä H, Chalker E. 2019. <a href="https://doi.org/10.3390/nu11040708">Vitamin C Can Shorten the Length of Stay in the ICU: A Meta-Analysis</a>. Nutrients 11:708.</div>
</div>
<div id="ref-WEWGGYKl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1802. </div><div class="csl-right-inline">Arvinte C, Singh M, Marik PE. 2020. <a href="https://doi.org/10.1016/j.medidd.2020.100064">Serum Levels of Vitamin C and Vitamin D in a Cohort of Critically Ill COVID-19 Patients of a North American Community Hospital Intensive Care Unit in May 2020: A Pilot Study</a>. Medicine in Drug Discovery 8:100064.</div>
</div>
<div id="ref-Zrm5dyM1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1803. </div><div class="csl-right-inline">Chiscano-Camón L, Ruiz-Rodriguez JC, Ruiz-Sanmartin A, Roca O, Ferrer R. 2020. <a href="https://doi.org/10.1186/s13054-020-03249-y">Vitamin C levels in patients with SARS-CoV-2-associated acute respiratory distress syndrome</a>. Critical Care 24:522.</div>
</div>
<div id="ref-19taZjxIT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1804. </div><div class="csl-right-inline">José RJ, Williams A, Manuel A, Brown JS, Chambers RC. 2020. <a href="https://doi.org/10.1183/16000617.0240-2020">Targeting coagulation activation in severe COVID-19 pneumonia: lessons from bacterial pneumonia and sepsis</a>. European Respiratory Review 29:200240.</div>
</div>
<div id="ref-WKBgtN6W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1805. </div><div class="csl-right-inline">Tyml K. 2017. <a href="https://doi.org/10.3390/antiox6030049">Vitamin C and Microvascular Dysfunction in Systemic Inflammation</a>. Antioxidants 6:49.</div>
</div>
<div id="ref-d9353XOW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1806. </div><div class="csl-right-inline">Hiedra R, Lo KB, Elbashabsheh M, Gul F, Wright RM, Albano J, Azmaiparashvili Z, Patarroyo Aponte G. 2020. <a href="https://doi.org/10.1080/14787210.2020.1794819">The use of IV vitamin C for patients with COVID-19: a case series</a>. Expert Review of Anti-infective Therapy 18:1259–1261.</div>
</div>
<div id="ref-Pb5PCH7y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1807. </div><div class="csl-right-inline">Hemilä H, Chalker E. 2013. Vitamin C for preventing and treating the common cold. Cochrane Database of Systematic Reviews https://doi.org/<a href="https://doi.org/10.1002/14651858.cd000980.pub4">10.1002/14651858.cd000980.pub4</a>.</div>
</div>
<div id="ref-JARvpqRw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1808. </div><div class="csl-right-inline">HEMILÄ H. 1997. <a href="https://doi.org/10.1097/00006454-199709000-00003">Vitamin C intake and susceptibility to pneumonia</a>. The Pediatric Infectious Disease Journal 16:836–837.</div>
</div>
<div id="ref-Rt5Aik4p" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1809. </div><div class="csl-right-inline">Fowler AA, Truwit JD, Hite RD, Morris PE, DeWilde C, Priday A, Fisher B, Thacker LR, Natarajan R, Brophy DF, Sculthorpe R, Nanchal R, Syed A, Sturgill J, Martin GS, Sevransky J, Kashiouris M, Hamman S, Egan KF, Hastings A, Spencer W, Tench S, Mehkri O, Bindas J, Duggal A, Graf J, Zellner S, Yanny L, McPolin C, Hollrith T, Kramer D, Ojielo C, Damm T, Cassity E, Wieliczko A, Halquist M. 2019. <a href="https://doi.org/10.1001/jama.2019.11825">Effect of Vitamin C Infusion on Organ Failure and Biomarkers of Inflammation and Vascular Injury in Patients With Sepsis and Severe Acute Respiratory Failure</a>. JAMA 322:1261.</div>
</div>
<div id="ref-nwa7WdNM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1810. </div><div class="csl-right-inline">Liu F, Zhu Y, Zhang J, Li Y, Peng Z. 2020. <a href="https://doi.org/10.1136/bmjopen-2020-039519">Intravenous high-dose vitamin C for the treatment of severe COVID-19: study protocol for a multicentre randomised controlled trial</a>. BMJ Open 10:e039519.</div>
</div>
<div id="ref-ZX7LDDp0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1811. </div><div class="csl-right-inline">Zhang J, Rao X, Li Y, Zhu Y, Liu F, Guo G, Luo G, Meng Z, Backer DD, Xiang H, Peng Z-Y. 2020. Pilot Trial of High-dose vitamin C in critically ill COVID-19 patients. Research Square https://doi.org/<a href="https://doi.org/10.21203/rs.3.rs-52778/v2">10.21203/rs.3.rs-52778/v2</a>.</div>
</div>
<div id="ref-RM8v7Y7H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1812. </div><div class="csl-right-inline">Zhang J, Rao X, Li Y, Zhu Y, Liu F, Guo G, Luo G, Meng Z, Backer DD, Xiang H, Peng Z-Y. 2020. High-dose vitamin C infusion for the treatment of critically ill COVID-19. Research Square https://doi.org/<a href="https://doi.org/10.21203/rs.3.rs-52778/v1">10.21203/rs.3.rs-52778/v1</a>.</div>
</div>
<div id="ref-hhf96MB1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1813. </div><div class="csl-right-inline">Panel on Dietary Antioxidants and Related Compounds, Subcommittee on Upper Reference Levels of Nutrients, Subcommittee on Interpretation and Uses of Dietary Reference Intakes, Standing Committee on the Scientific Evaluation of Dietary Reference Intakes, Food and Nutrition Board, Institute of Medicine. 2000. Dietary Reference Intakes for Vitamin C, Vitamin E, Selenium, and Carotenoids. The National Academies Press. <a href="https://doi.org/ghtvqx">https://doi.org/ghtvqx</a>.</div>
</div>
<div id="ref-3Nyo3kmR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1814. </div><div class="csl-right-inline">Gois P, Ferreira D, Olenski S, Seguro A. 2017. <a href="https://doi.org/10.3390/nu9070651">Vitamin D and Infectious Diseases: Simple Bystander or Contributing Factor?</a> Nutrients 9:651.</div>
</div>
<div id="ref-SKP7JCfL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1815. </div><div class="csl-right-inline">Gruber–Bzura BM. 2018. <a href="https://doi.org/10.3390/ijms19082419">Vitamin D and Influenza—Prevention or Therapy?</a> International Journal of Molecular Sciences 19:2419.</div>
</div>
<div id="ref-15xVUSQRH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1816. </div><div class="csl-right-inline">Charoenngam N, Holick MF. 2020. <a href="https://doi.org/10.3390/nu12072097">Immunologic Effects of Vitamin D on Human Health and Disease</a>. Nutrients 12:2097.</div>
</div>
<div id="ref-15ESVyKi0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1817. </div><div class="csl-right-inline">Hughes DA, Norton R. 2009. <a href="https://doi.org/10.1111/j.1365-2249.2009.04001.x">Vitamin D and respiratory health</a>. Clinical &amp; Experimental Immunology 158:20–25.</div>
</div>
<div id="ref-npPvtBLw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1818. </div><div class="csl-right-inline">Vanherwegen A-S, Gysemans C, Mathieu C. 2017. <a href="https://doi.org/10.1016/j.ecl.2017.07.010">Regulation of Immune Function by Vitamin D and Its Use in Diseases of Immunity</a>. Endocrinology and Metabolism Clinics of North America 46:1061–1094.</div>
</div>
<div id="ref-n3v9Thlz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1819. </div><div class="csl-right-inline">Aranow C. 2015. <a href="https://doi.org/10.2310/jim.0b013e31821b8755">Vitamin D and the Immune System</a>. Journal of Investigative Medicine 59:881–886.</div>
</div>
<div id="ref-iWLsflrG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1820. </div><div class="csl-right-inline">Jolliffe DA, Griffiths CJ, Martineau AR. 2013. <a href="https://doi.org/10.1016/j.jsbmb.2012.11.017">Vitamin D in the prevention of acute respiratory infection: Systematic review of clinical studies</a>. The Journal of Steroid Biochemistry and Molecular Biology 136:321–329.</div>
</div>
<div id="ref-nwEQBjV4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1821. </div><div class="csl-right-inline">Baeke F, Takiishi T, Korf H, Gysemans C, Mathieu C. 2010. <a href="https://doi.org/10.1016/j.coph.2010.04.001">Vitamin D: modulator of the immune system</a>. Current Opinion in Pharmacology 10:482–496.</div>
</div>
<div id="ref-9ac0GYir" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1822. </div><div class="csl-right-inline">Grant WB, Lahore H, McDonnell SL, Baggerly CA, French CB, Aliano JL, Bhattoa HP. 2020. <a href="https://doi.org/10.3390/nu12040988">Evidence that Vitamin D Supplementation Could Reduce Risk of Influenza and COVID-19 Infections and Deaths</a>. Nutrients 12:988.</div>
</div>
<div id="ref-LpqYmd4C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1823. </div><div class="csl-right-inline">Rhodes JM, Subramanian S, Laird E, Griffin G, Kenny RA. 2020. <a href="https://doi.org/10.1111/joim.13149">Perspective: Vitamin D deficiency and COVID‐19 severity – plausibly linked by latitude, ethnicity, impacts on cytokines, ACE2 and thrombosis</a>. Journal of Internal Medicine 289:97–115.</div>
</div>
<div id="ref-1EEHgnhPC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1824. </div><div class="csl-right-inline">Whittemore PB. 2020. <a href="https://doi.org/10.1016/j.ajic.2020.06.193">COVID-19 fatalities, latitude, sunlight, and vitamin D</a>. American Journal of Infection Control 48:1042–1044.</div>
</div>
<div id="ref-CS8krfan" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1825. </div><div class="csl-right-inline">Rhodes JM, Subramanian S, Laird E, Kenny RA. 2020. <a href="https://doi.org/10.1111/apt.15777">Editorial: low population mortality from COVID-19 in countries south of latitude 35 degrees North supports vitamin D as a factor determining severity</a>. Alimentary Pharmacology &amp; Therapeutics 51:1434–1437.</div>
</div>
<div id="ref-1Gy7Svl72" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1826. </div><div class="csl-right-inline">D’Avolio A, Avataneo V, Manca A, Cusato J, De Nicolò A, Lucchini R, Keller F, Cantù M. 2020. <a href="https://doi.org/10.3390/nu12051359">25-Hydroxyvitamin D Concentrations Are Lower in Patients with Positive PCR for SARS-CoV-2</a>. Nutrients 12:1359.</div>
</div>
<div id="ref-GMAEaWD1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1827. </div><div class="csl-right-inline">De Smet D, De Smet K, Herroelen P, Gryspeerdt S, Martens GA. 2020. Vitamin D deficiency as risk factor for severe COVID-19: a convergence of two pandemics. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.05.01.20079376">10.1101/2020.05.01.20079376</a>.</div>
</div>
<div id="ref-GM9W6HPG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1828. </div><div class="csl-right-inline">Maghbooli Z, Sahraian MA, Ebrahimi M, Pazoki M, Kafan S, Tabriz HM, Hadadi A, Montazeri M, Nasiri M, Shirvani A, Holick MF. 2020. <a href="https://doi.org/10.1371/journal.pone.0239799">Vitamin D sufficiency, a serum 25-hydroxyvitamin D at least 30 ng/mL reduced risk for adverse clinical outcomes in patients with COVID-19 infection</a>. PLOS ONE 15:e0239799.</div>
</div>
<div id="ref-kN3eqnZW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1829. </div><div class="csl-right-inline">Ali N. 2020. <a href="https://doi.org/10.1016/j.jiph.2020.06.021">Role of vitamin D in preventing of COVID-19 infection, progression and severity</a>. Journal of Infection and Public Health 13:1373–1380.</div>
</div>
<div id="ref-5MvUc8IL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1830. </div><div class="csl-right-inline">Merzon E, Tworowski D, Gorohovski A, Vinker S, Golan Cohen A, Green I, Frenkel‐Morgenstern M. 2020. <a href="https://doi.org/10.1111/febs.15495">Low plasma 25(OH) vitamin D level is associated with increased risk of COVID‐19 infection: an Israeli population‐based study</a>. The FEBS Journal 287:3693–3702.</div>
</div>
<div id="ref-18O6t6Jo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1831. </div><div class="csl-right-inline">Meltzer DO, Best TJ, Zhang H, Vokes T, Arora V, Solway J. 2020. <a href="https://doi.org/10.1001/jamanetworkopen.2020.19722">Association of Vitamin D Status and Other Clinical Characteristics With COVID-19 Test Results</a>. JAMA Network Open 3:e2019722.</div>
</div>
<div id="ref-8ZxZKpYS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1832. </div><div class="csl-right-inline">Hernández JL, Nan D, Fernandez-Ayala M, García-Unzueta M, Hernández-Hernández MA, López-Hoyos M, Muñoz-Cacho P, Olmos JM, Gutiérrez-Cuadra M, Ruiz-Cubillán JJ, Crespo J, Martínez-Taboada VM. 2020. <a href="https://doi.org/10.1210/clinem/dgaa733">Vitamin D Status in Hospitalized Patients with SARS-CoV-2 Infection</a>. The Journal of Clinical Endocrinology &amp; Metabolism dgaa733.</div>
</div>
<div id="ref-QAEcHBzN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1833. </div><div class="csl-right-inline">Jain A, Chaurasia R, Sengar NS, Singh M, Mahor S, Narain S. 2020. <a href="https://doi.org/10.1038/s41598-020-77093-z">Analysis of vitamin D level among asymptomatic and critically ill COVID-19 patients and its correlation with inflammatory markers</a>. Scientific Reports 10:20191.</div>
</div>
<div id="ref-EhakYR7D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1834. </div><div class="csl-right-inline">Vassiliou AG, Jahaj E, Pratikaki M, Orfanos SE, Dimopoulou I, Kotanidou A. 2020. <a href="https://doi.org/10.3390/nu12123773">Low 25-Hydroxyvitamin D Levels on Admission to the Intensive Care Unit May Predispose COVID-19 Pneumonia Patients to a Higher 28-Day Mortality Risk: A Pilot Study on a Greek ICU Cohort</a>. Nutrients 12:3773.</div>
</div>
<div id="ref-xv6TOhJg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1835. </div><div class="csl-right-inline">Carpagnano GE, Di Lecce V, Quaranta VN, Zito A, Buonamico E, Capozza E, Palumbo A, Di Gioia G, Valerio VN, Resta O. 2020. Vitamin D deficiency as a predictor of poor prognosis in patients with acute respiratory failure due to COVID-19. Journal of Endocrinological Investigation https://doi.org/<a href="https://doi.org/10.1007/s40618-020-01370-x">10.1007/s40618-020-01370-x</a>.</div>
</div>
<div id="ref-ZheSCcim" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1836. </div><div class="csl-right-inline">Radujkovic A, Hippchen T, Tiwari-Heckler S, Dreher S, Boxberger M, Merle U. 2020. <a href="https://doi.org/10.3390/nu12092757">Vitamin D Deficiency and Outcome of COVID-19 Patients</a>. Nutrients 12:2757.</div>
</div>
<div id="ref-pNuUm1NE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1837. </div><div class="csl-right-inline">Pizzini A, Aichner M, Sahanic S, Böhm A, Egger A, Hoermann G, Kurz K, Widmann G, Bellmann-Weiler R, Weiss G, Tancevski I, Sonnweber T, Löffler-Ragg J. 2020. <a href="https://doi.org/10.3390/nu12092775">Impact of Vitamin D Deficiency on COVID-19—A Prospective Analysis from the CovILD Registry</a>. Nutrients 12:2775.</div>
</div>
<div id="ref-5aqg2qxi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1838. </div><div class="csl-right-inline">Ye K, Tang F, Liao X, Shaw BA, Deng M, Huang G, Qin Z, Peng X, Xiao H, Chen C, Liu X, Ning L, Wang B, Tang N, Li M, Xu F, Lin S, Yang J. 2020. <a href="https://doi.org/10.1080/07315724.2020.1826005">Does Serum Vitamin D Level Affect COVID-19 Infection and Its Severity?-A Case-Control Study</a>. Journal of the American College of Nutrition 1–8.</div>
</div>
<div id="ref-10kobTskt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1839. </div><div class="csl-right-inline">Padhi S, Suvankar S, Panda VK, Pati A, Panda AK. 2020. <a href="https://doi.org/10.1016/j.intimp.2020.107001">Lower levels of vitamin D are associated with SARS-CoV-2 infection and mortality in the Indian population: An observational study</a>. International Immunopharmacology 88:107001.</div>
</div>
<div id="ref-Tcc0pGgI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1840. </div><div class="csl-right-inline">Luo X, Liao Q, Shen Y, Li H, Cheng L. 2021. <a href="https://doi.org/10.1093/jn/nxaa332">Vitamin D Deficiency Is Associated with COVID-19 Incidence and Disease Severity in Chinese People</a>. The Journal of Nutrition 151:98–103.</div>
</div>
<div id="ref-FZ8yXmxQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1841. </div><div class="csl-right-inline">Hastie CE, Mackay DF, Ho F, Celis-Morales CA, Katikireddi SV, Niedzwiedz CL, Jani BD, Welsh P, Mair FS, Gray SR, O’Donnell CA, Gill JMR, Sattar N, Pell JP. 2020. <a href="https://doi.org/10.1016/j.dsx.2020.04.050">Vitamin D concentrations and COVID-19 infection in UK Biobank</a>. Diabetes &amp; Metabolic Syndrome: Clinical Research &amp; Reviews 14:561–565.</div>
</div>
<div id="ref-7goCwNjQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1842. </div><div class="csl-right-inline">Hastie CE, Pell JP, Sattar N. 2020. Vitamin D and COVID-19 infection and mortality in UK Biobank. European Journal of Nutrition https://doi.org/<a href="https://doi.org/10.1007/s00394-020-02372-4">10.1007/s00394-020-02372-4</a>.</div>
</div>
<div id="ref-EtCW6lkq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1843. </div><div class="csl-right-inline">Panagiotou G, Tee SA, Ihsan Y, Athar W, Marchitelli G, Kelly D, Boot CS, Stock N, Macfarlane J, Martineau AR, Burns G, Quinton R. 2020. <a href="https://doi.org/10.1111/cen.14276">Low serum 25‐hydroxyvitamin D (25[OH]D) levels in patients hospitalized with COVID‐19 are associated with greater disease severity</a>. Clinical Endocrinology 93:508–511.</div>
</div>
<div id="ref-ClClnlJb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1844. </div><div class="csl-right-inline">Grant WB, McDonnell SL. 2020. <a href="https://doi.org/10.1016/j.dsx.2020.05.046">Letter in response to the article: Vitamin D concentrations and COVID-19 infection in UK biobank (Hastie et al.)</a>. Diabetes &amp; Metabolic Syndrome: Clinical Research &amp; Reviews 14:893–894.</div>
</div>
<div id="ref-1F2cyuTqD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1845. </div><div class="csl-right-inline">Martín Giménez VM, Inserra F, Ferder L, García J, Manucha W. 2020. Vitamin D deficiency in African Americans is associated with a high risk of severe disease and mortality by SARS-CoV-2. Journal of Human Hypertension https://doi.org/<a href="https://doi.org/10.1038/s41371-020-00398-z">10.1038/s41371-020-00398-z</a>.</div>
</div>
<div id="ref-F4hC9qly" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1846. </div><div class="csl-right-inline">Daneshkhah A, Agrawal V, Eshein A, Subramanian H, Roy HK, Backman V. 2020. <a href="https://doi.org/10.1007/s40520-020-01677-y">Evidence for possible association of vitamin D status with cytokine storm and unregulated inflammation in COVID-19 patients</a>. Aging Clinical and Experimental Research 32:2141–2158.</div>
</div>
<div id="ref-kurM6Av7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1847. </div><div class="csl-right-inline">Rastogi A, Bhansali A, Khare N, Suri V, Yaddanapudi N, Sachdeva N, Puri GD, Malhotra P. 2020. <a href="https://doi.org/10.1136/postgradmedj-2020-139065">Short term, high-dose vitamin D supplementation for COVID-19 disease: a randomised, placebo-controlled, study (SHADE study)</a>. Postgraduate Medical Journal postgradmedj-2020-139065.</div>
</div>
<div id="ref-rYXmRFCk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1848. </div><div class="csl-right-inline">Entrenas Castillo M, Entrenas Costa LM, Vaquero Barrios JM, Alcalá Díaz JF, López Miranda J, Bouillon R, Quesada Gomez JM. 2020. <a href="https://doi.org/10.1016/j.jsbmb.2020.105751">“Effect of calcifediol treatment and best available therapy versus best available therapy on intensive care unit admission and mortality among patients hospitalized for COVID-19: A pilot randomized clinical study”</a>. The Journal of Steroid Biochemistry and Molecular Biology 203:105751.</div>
</div>
<div id="ref-OAzFPABw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1849. </div><div class="csl-right-inline">COVID-19 rapid evidence summary: vitamin D for COVID-19 | Advice | NICE. <a href="https://www.nice.org.uk/advice/es28">https://www.nice.org.uk/advice/es28</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-1FHsBdDVU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1850. </div><div class="csl-right-inline">Jungreis I, Kellis M. 2020. Mathematical analysis of Córdoba calcifediol trial suggests strong role for Vitamin D in reducing ICU admissions of hospitalized COVID-19 patients. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.11.08.20222638">10.1101/2020.11.08.20222638</a>.</div>
</div>
<div id="ref-UlHOCdfk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1851. </div><div class="csl-right-inline">Ling SF, Broad E, Murphy R, Pappachan JM, Pardesi-Newton S, Kong M-F, Jude EB. 2020. <a href="https://doi.org/10.3390/nu12123799">High-Dose Cholecalciferol Booster Therapy is Associated with a Reduced Risk of Mortality in Patients with COVID-19: A Cross-Sectional Multi-Centre Observational Study</a>. Nutrients 12:3799.</div>
</div>
<div id="ref-1GjnSzUKx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1852. </div><div class="csl-right-inline">Murai IH, Fernandes AL, Sales LP, Pinto AJ, Goessler KF, Duran CSC, Silva CBR, Franco AS, Macedo MB, Dalmolin HHH, Baggio J, Balbi GGM, Reis BZ, Antonangelo L, Caparbo VF, Gualano B, Pereira RMR. 2020. Effect of Vitamin D &lt;sub&gt;3&lt;/sub&gt; Supplementation vs Placebo on Hospital Length of Stay in Patients with Severe COVID-19: A Multicenter, Double-blind, Randomized Controlled Trial. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.11.16.20232397">10.1101/2020.11.16.20232397</a>.</div>
</div>
<div id="ref-162t7zdGv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1853. </div><div class="csl-right-inline">Garzón MC. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04334005">Effect of Vitamin D Administration on Prevention and Treatment of Mild Forms of Suspected Covid-19</a>. NCT04334005. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-2K0jV5Bn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1854. </div><div class="csl-right-inline">Montano-Loza A. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04385940">Improving Vitamin D Status in the Management of COVID-19</a>. NCT04385940. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-rBiojRHk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1855. </div><div class="csl-right-inline">Cholecalciferol to Improve the Outcomes of COVID-19 Patients - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04411446">https://clinicaltrials.gov/ct2/show/NCT04411446</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-6kLoX5qT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1856. </div><div class="csl-right-inline">COvid-19 and Vitamin D Supplementation: a Multicenter Randomized Controlled Trial of High Dose Versus Standard Dose Vitamin D3 in High-risk COVID-19 Patients (CoVitTrial) - Full Text View - ClinicalTrials.gov. <a href="https://clinicaltrials.gov/ct2/show/NCT04344041">https://clinicaltrials.gov/ct2/show/NCT04344041</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-32C2erIe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1857. </div><div class="csl-right-inline">Louisiana State University Health Sciences Center in New Orleans. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04363840">The LEAD COVID-19 Trial: Low-risk, Early Aspirin and Vitamin D to Reduce COVID-19 Hospitalizations</a>. NCT04363840. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-tLVliODQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1858. </div><div class="csl-right-inline">MD MM. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04400890">Randomized Double-Blind Placebo-Controlled Proof-of-Concept Trial of a Plant Polyphenol for the Outpatient Treatment of Mild Coronavirus Disease (COVID-19)</a>. NCT04400890. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-Btt5H6f1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1859. </div><div class="csl-right-inline">Lips P, Cashman KD, Lamberg-Allardt C, Bischoff-Ferrari HA, Obermayer-Pietsch B, Bianchi ML, Stepan J, El-Hajj Fuleihan G, Bouillon R. 2019. <a href="https://doi.org/10.1530/eje-18-0736">Current vitamin D status in European and Middle East countries and strategies to prevent vitamin D deficiency: a position statement of the European Calcified Tissue Society</a>. European Journal of Endocrinology 180:P23–P54.</div>
</div>
<div id="ref-llQ4v7QI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1860. </div><div class="csl-right-inline">Communiqué de l’Académie nationale de Médecine : Vitamine D et Covid-19 – Académie nationale de médecine | Une institution dans son temps. <a href="https://www.academie-medecine.fr/communique-de-lacademie-nationale-de-medecine-vitamine-d-et-covid-19/">https://www.academie-medecine.fr/communique-de-lacademie-nationale-de-medecine-vitamine-d-et-covid-19/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-M4v2N83p" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1861. </div><div class="csl-right-inline">Iacobucci G. 2020. <a href="https://doi.org/10.1136/bmj.m1820">Covid-19: NHS bosses told to assess risk to ethnic minority staff who may be at greater risk</a>. BMJ m1820.</div>
</div>
<div id="ref-yoAwg53u" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1862. </div><div class="csl-right-inline">Torjesen I. 2020. <a href="https://doi.org/10.1136/bmj.m2475">Covid-19: Public health agencies review whether vitamin D supplements could reduce risk</a>. BMJ m2475.</div>
</div>
<div id="ref-StktHVeV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1863. </div><div class="csl-right-inline">Kohlmeier M. 2020. <a href="https://doi.org/10.1136/bmjnph-2020-000096">Avoidance of vitamin D deficiency to slow the COVID-19 pandemic</a>. BMJ Nutrition, Prevention &amp; Health 3:67–73.</div>
</div>
<div id="ref-4DE4ljR9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1864. </div><div class="csl-right-inline">National Institute for Health and Care Excellence (NICE). COVID-19 rapid guideline: vitamin D. <a href="https://www.nice.org.uk/guidance/ng187/resources/covid19-rapid-guideline-vitamin-d-pdf-66142026720709">https://www.nice.org.uk/guidance/ng187/resources/covid19-rapid-guideline-vitamin-d-pdf-66142026720709</a>.</div>
</div>
<div id="ref-lhYw2CB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1865. </div><div class="csl-right-inline">Parva NR, Tadepalli S, Singh P, Qian A, Joshi R, Kandala H, Nookala VK, Cheriyath P. 2018. Prevalence of Vitamin D Deficiency and Associated Risk Factors in the US Population (2011-2012). Cureus https://doi.org/<a href="https://doi.org/10.7759/cureus.2741">10.7759/cureus.2741</a>.</div>
</div>
<div id="ref-KNwYH6Y4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1866. </div><div class="csl-right-inline">Vitamin D. COVID-19 Treatment Guidelines. <a href="https://www.covid19treatmentguidelines.nih.gov/adjunctive-therapy/vitamin-d/">https://www.covid19treatmentguidelines.nih.gov/adjunctive-therapy/vitamin-d/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-19yy9SHf1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1867. </div><div class="csl-right-inline">Hamulka J, Jeruszka-Bielak M, Górnicka M, Drywień ME, Zielinska-Pukos MA. 2020. <a href="https://doi.org/10.3390/nu13010054">Dietary Supplements during COVID-19 Outbreak. Results of Google Trends Analysis Supported by PLifeCOVID-19 Online Studies</a>. Nutrients 13:54.</div>
</div>
<div id="ref-xkewCiC9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1868. </div><div class="csl-right-inline">2021. Court Orders Georgia Defendants to Stop Selling Vitamin D Products as Treatments for Covid-19 and Other Diseases. <a href="https://www.justice.gov/opa/pr/court-orders-georgia-defendants-stop-selling-vitamin-d-products-treatments-covid-19-and-other">https://www.justice.gov/opa/pr/court-orders-georgia-defendants-stop-selling-vitamin-d-products-treatments-covid-19-and-other</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-1FyHVZ9yx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1869. </div><div class="csl-right-inline">Hill C, Guarner F, Reid G, Gibson GR, Merenstein DJ, Pot B, Morelli L, Canani RB, Flint HJ, Salminen S, Calder PC, Sanders ME. 2014. <a href="https://doi.org/10.1038/nrgastro.2014.66">The International Scientific Association for Probiotics and Prebiotics consensus statement on the scope and appropriate use of the term probiotic</a>. Nature Reviews Gastroenterology &amp; Hepatology 11:506–514.</div>
</div>
<div id="ref-pvZU55k9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1870. </div><div class="csl-right-inline">Kang E-J, Kim SY, Hwang I-H, Ji Y-J. 2013. <a href="https://doi.org/10.4082/kjfm.2013.34.1.2">The Effect of Probiotics on Prevention of Common Cold: A Meta-Analysis of Randomized Controlled Trial Studies</a>. Korean Journal of Family Medicine 34:2.</div>
</div>
<div id="ref-HKeroTXj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1871. </div><div class="csl-right-inline">Kanauchi O, Andoh A, AbuBakar S, Yamamoto N. 2018. <a href="https://doi.org/10.2174/1381612824666180116163411">Probiotics and Paraprobiotics in Viral Infection: Clinical Application and Effects on the Innate and Acquired Immune Systems</a>. Current Pharmaceutical Design 24:710–717.</div>
</div>
<div id="ref-10e8cHDs8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1872. </div><div class="csl-right-inline">Baud D, Dimopoulou Agri V, Gibson GR, Reid G, Giannoni E. 2020. <a href="https://doi.org/10.3389/fpubh.2020.00186">Using Probiotics to Flatten the Curve of Coronavirus Disease COVID-2019 Pandemic</a>. Frontiers in Public Health 8:186.</div>
</div>
<div id="ref-5fU7dexB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1873. </div><div class="csl-right-inline">O’Toole PW, Marchesi JR, Hill C. 2017. <a href="https://doi.org/10.1038/nmicrobiol.2017.57">Next-generation probiotics: the spectrum from probiotics to live biotherapeutics</a>. Nature Microbiology 2:17057.</div>
</div>
<div id="ref-1GCrGCp5P" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1874. </div><div class="csl-right-inline">Plaza-Diaz J, Ruiz-Ojeda FJ, Gil-Campos M, Gil A. 2019. <a href="https://doi.org/10.1093/advances/nmy063">Mechanisms of Action of Probiotics</a>. Advances in Nutrition 10:S49–S66.</div>
</div>
<div id="ref-B2GeSSQo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1875. </div><div class="csl-right-inline">Halloran K, Underwood MA. 2019. <a href="https://doi.org/10.1016/j.earlhumdev.2019.05.010">Probiotic mechanisms of action</a>. Early Human Development 135:58–65.</div>
</div>
<div id="ref-nmXF8vkS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1876. </div><div class="csl-right-inline">Bermudez-Brito M, Plaza-Díaz J, Muñoz-Quezada S, Gómez-Llorente C, Gil A. 2012. <a href="https://doi.org/10.1159/000342079">Probiotic Mechanisms of Action</a>. Annals of Nutrition and Metabolism 61:160–174.</div>
</div>
<div id="ref-sX7GyWpV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1877. </div><div class="csl-right-inline">BOTIC T, KLINGBERG T, WEINGARTL H, CENCIC A. 2007. <a href="https://doi.org/10.1016/j.ijfoodmicro.2006.10.044">A novel eukaryotic cell culture model to study antiviral activity of potential probiotic bacteria</a>. International Journal of Food Microbiology 115:227–234.</div>
</div>
<div id="ref-IkuChxSp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1878. </div><div class="csl-right-inline">Waki N, Yajima N, Suganuma H, Buddle BM, Luo D, Heiser A, Zheng T. 2014. <a href="https://doi.org/10.1111/lam.12160">Oral administration of &lt;i&gt;Lactobacillus brevis&lt;/i&gt; KB290 to mice alleviates clinical symptoms following influenza virus infection</a>. Letters in Applied Microbiology 58:87–93.</div>
</div>
<div id="ref-8gQnAy2v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1879. </div><div class="csl-right-inline">Mastromarino P, Cacciotti F, Masci A, Mosca L. 2011. <a href="https://doi.org/10.1016/j.anaerobe.2011.04.022">Antiviral activity of Lactobacillus brevis towards herpes simplex virus type 2: Role of cell wall associated components</a>. Anaerobe 17:334–336.</div>
</div>
<div id="ref-zGORHhOM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1880. </div><div class="csl-right-inline">Percopo CM, Ma M, Brenner TA, Krumholz JO, Break TJ, Laky K, Rosenberg HF. 2019. <a href="https://doi.org/10.4049/jimmunol.1800927">Critical Adverse Impact of IL-6 in Acute Pneumovirus Infection</a>. The Journal of Immunology 202:871–882.</div>
</div>
<div id="ref-g0qfbA7l" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1881. </div><div class="csl-right-inline">Biliavska, Pankivska, Povnitsa, Zagorodnya. 2019. <a href="https://doi.org/10.3390/medicina55090519">Antiviral Activity of Exopolysaccharides Produced by Lactic Acid Bacteria of the Genera Pediococcus, Leuconostoc and Lactobacillus against Human Adenovirus Type 5</a>. Medicina 55:519.</div>
</div>
<div id="ref-DNbZomOZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1882. </div><div class="csl-right-inline">Eguchi K, Fujitani N, Nakagawa H, Miyazaki T. 2019. <a href="https://doi.org/10.1038/s41598-019-39602-7">Prevention of respiratory syncytial virus infection with probiotic lactic acid bacterium Lactobacillus gasseri SBT2055</a>. Scientific Reports 9:4812.</div>
</div>
<div id="ref-p9eKMfO2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1883. </div><div class="csl-right-inline">Turner RB, Woodfolk JA, Borish L, Steinke JW, Patrie JT, Muehling LM, Lahtinen S, Lehtinen MJ. 2017. <a href="https://doi.org/10.3920/bm2016.0160">Effect of probiotic on innate inflammatory response and viral shedding in experimental rhinovirus infection – a randomised controlled trial</a>. Beneficial Microbes 8:207–215.</div>
</div>
<div id="ref-x46F06Fr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1884. </div><div class="csl-right-inline">Zelaya H, Tsukida K, Chiba E, Marranzino G, Alvarez S, Kitazawa H, Agüero G, Villena J. 2014. <a href="https://doi.org/10.1016/j.intimp.2013.12.020">Immunobiotic lactobacilli reduce viral-associated pulmonary damage through the modulation of inflammation–coagulation interactions</a>. International Immunopharmacology 19:161–173.</div>
</div>
<div id="ref-uU3GXiaf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1885. </div><div class="csl-right-inline">Tonetti FR, Islam MdA, Vizoso-Pinto MG, Takahashi H, Kitazawa H, Villena J. 2020. <a href="https://doi.org/10.1016/j.intimp.2019.106115">Nasal priming with immunobiotic lactobacilli improves the adaptive immune response against influenza virus</a>. International Immunopharmacology 78:106115.</div>
</div>
<div id="ref-Sp05opqA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1886. </div><div class="csl-right-inline">Olaimat AN, Aolymat I, Al-Holy M, Ayyash M, Abu Ghoush M, Al-Nabulsi AA, Osaili T, Apostolopoulos V, Liu S-Q, Shah NP. 2020. <a href="https://doi.org/10.1038/s41538-020-00078-9">The potential application of probiotics and prebiotics for the prevention and treatment of COVID-19</a>. npj Science of Food 4:17.</div>
</div>
<div id="ref-13AWoYvqB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1887. </div><div class="csl-right-inline">Keely S, Talley NJ, Hansbro PM. 2011. <a href="https://doi.org/10.1038/mi.2011.55">Pulmonary-intestinal cross-talk in mucosal inflammatory disease</a>. Mucosal Immunology 5:7–18.</div>
</div>
<div id="ref-Smqu1EYN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1888. </div><div class="csl-right-inline">Dumas A, Bernard L, Poquet Y, Lugo-Villarino G, Neyrolles O. 2018. <a href="https://doi.org/10.1111/cmi.12966">The role of the lung microbiota and the gut-lung axis in respiratory infectious diseases</a>. Cellular Microbiology 20:e12966.</div>
</div>
<div id="ref-d27mQ4kO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1889. </div><div class="csl-right-inline">Dhar D, Mohanty A. 2020. <a href="https://doi.org/10.1016/j.virusres.2020.198018">Gut microbiota and Covid-19- possible link and implications</a>. Virus Research 285:198018.</div>
</div>
<div id="ref-HOd4nJPq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1890. </div><div class="csl-right-inline">Bao L, Zhang C, Dong J, Zhao L, Li Y, Sun J. 2020. <a href="https://doi.org/10.3389/fmicb.2020.01840">Oral Microbiome and SARS-CoV-2: Beware of Lung Co-infection</a>. Frontiers in Microbiology 11:1840.</div>
</div>
<div id="ref-OwqHmSrX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1891. </div><div class="csl-right-inline">Khatiwada S, Subedi A. 2020. <a href="https://doi.org/10.1016/j.humic.2020.100073">Lung microbiome and coronavirus disease 2019 (COVID-19): Possible link and implications</a>. Human Microbiome Journal 17:100073.</div>
</div>
<div id="ref-1MIGrWIP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1892. </div><div class="csl-right-inline">Lehtoranta L, Pitkäranta A, Korpela R. 2014. <a href="https://doi.org/10.1007/s10096-014-2086-y">Probiotics in respiratory virus infections</a>. European Journal of Clinical Microbiology &amp; Infectious Diseases 33:1289–1302.</div>
</div>
<div id="ref-58eXvYEV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1893. </div><div class="csl-right-inline">Hao Q, Dong BR, Wu T. 2015. Probiotics for preventing acute upper respiratory tract infections. Cochrane Database of Systematic Reviews https://doi.org/<a href="https://doi.org/10.1002/14651858.cd006895.pub3">10.1002/14651858.cd006895.pub3</a>.</div>
</div>
<div id="ref-kjavvCzp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1894. </div><div class="csl-right-inline">Vouloumanou EK, Makris GC, Karageorgopoulos DE, Falagas ME. 2009. <a href="https://doi.org/10.1016/j.ijantimicag.2008.11.005">Probiotics for the prevention of respiratory tract infections: a systematic review</a>. International Journal of Antimicrobial Agents 34:197.e1–197.e10.</div>
</div>
<div id="ref-dJIGDt4k" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1895. </div><div class="csl-right-inline">King S, Glanville J, Sanders ME, Fitzgerald A, Varley D. 2014. <a href="https://doi.org/10.1017/s0007114514000075">Effectiveness of probiotics on the duration of illness in healthy children and adults who develop common acute respiratory infectious conditions: a systematic review and meta-analysis</a>. British Journal of Nutrition 112:41–54.</div>
</div>
<div id="ref-AJav6c61" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1896. </div><div class="csl-right-inline">Zeng J, Wang C-T, Zhang F-S, Qi F, Wang S-F, Ma S, Wu T-J, Tian H, Tian Z-T, Zhang S-L, Qu Y, Liu L-Y, Li Y-Z, Cui S, Zhao H-L, Du Q-S, Ma Z, Li C-H, Li Y, Si M, Chu Y-F, Meng M, Ren H-S, Zhang J-C, Jiang J-J, Ding M, Wang Y-P. 2016. <a href="https://doi.org/10.1007/s00134-016-4303-x">Effect of probiotics on the incidence of ventilator-associated pneumonia in critically ill patients: a randomized controlled multicenter trial</a>. Intensive Care Medicine 42:1018–1028.</div>
</div>
<div id="ref-1E2aDJp4T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1897. </div><div class="csl-right-inline">Morrow LE, Kollef MH, Casale TB. 2010. <a href="https://doi.org/10.1164/rccm.200912-1853oc">Probiotic Prophylaxis of Ventilator-associated Pneumonia</a>. American Journal of Respiratory and Critical Care Medicine 182:1058–1064.</div>
</div>
<div id="ref-zccFgajh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1898. </div><div class="csl-right-inline">Shimizu K, Yamada T, Ogura H, Mohri T, Kiguchi T, Fujimi S, Asahara T, Yamada T, Ojima M, Ikeda M, Shimazu T. 2018. <a href="https://doi.org/10.1186/s13054-018-2167-x">Synbiotics modulate gut microbiota and reduce enteritis and ventilator-associated pneumonia in patients with sepsis: a randomized controlled trial</a>. Critical Care 22:239.</div>
</div>
<div id="ref-1DI06gFPc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1899. </div><div class="csl-right-inline">Su M, Jia Y, Li Y, Zhou D, Jia J. 2020. <a href="https://doi.org/10.4187/respcare.07097">Probiotics for the Prevention of Ventilator-Associated Pneumonia: A Meta-Analysis of Randomized Controlled Trials</a>. Respiratory Care 65:673–685.</div>
</div>
<div id="ref-164mVBFZx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1900. </div><div class="csl-right-inline">Helvécio Cardoso Corrêa Póvoa, Chianca GC, Iorio NLPP. 2020. <a href="https://doi.org/10.6084/m9.figshare.12340496">COVID-19: An Alert to Ventilator-Associated Bacterial Pneumonia</a>. Adis Journals.</div>
</div>
<div id="ref-w2VFo0h1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1901. </div><div class="csl-right-inline">François B, Laterre P-F, Luyt C-E, Chastre J. 2020. <a href="https://doi.org/10.1186/s13054-020-03013-2">The challenge of ventilator-associated pneumonia diagnosis in COVID-19 patients</a>. Critical Care 24:289.</div>
</div>
<div id="ref-gcyQ73v7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1902. </div><div class="csl-right-inline">Perceval C, Szajewska H, Indrio F, Weizman Z, Vandenplas Y. 2019. <a href="https://doi.org/10.1016/s2352-4642(19)30182-8">Prophylactic use of probiotics for gastrointestinal disorders in children</a>. The Lancet Child &amp; Adolescent Health 3:655–662.</div>
</div>
<div id="ref-67ihxDiV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1903. </div><div class="csl-right-inline">Zhou Z, Zhao N, Shu Y, Han S, Chen B, Shu X. 2020. <a href="https://doi.org/10.1053/j.gastro.2020.03.020">Effect of Gastrointestinal Symptoms in Patients With COVID-19</a>. Gastroenterology 158:2294–2297.</div>
</div>
<div id="ref-8Hvgbl3n" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1904. </div><div class="csl-right-inline">Zhang H, Kang Z, Gong H, Xu D, Wang J, Li Z, Cui X, Xiao J, Meng T, Zhou W, Liu J, Xu H. 2020. The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.01.30.927806">10.1101/2020.01.30.927806</a>.</div>
</div>
<div id="ref-Nhxoefru" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1905. </div><div class="csl-right-inline">Gui M, Song W, Zhou H, Xu J, Chen S, Xiang Y, Wang X. 2016. <a href="https://doi.org/10.1038/cr.2016.152">Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding</a>. Cell Research 27:119–129.</div>
</div>
<div id="ref-r9HNMqTS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1906. </div><div class="csl-right-inline">Yeo C, Kaushal S, Yeo D. 2020. <a href="https://doi.org/10.1016/s2468-1253(20)30048-0">Enteric involvement of coronaviruses: is faecal–oral transmission of SARS-CoV-2 possible?</a> The Lancet Gastroenterology &amp; Hepatology 5:335–337.</div>
</div>
<div id="ref-eOcc1BUx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1907. </div><div class="csl-right-inline">Xu Y, Li X, Zhu B, Liang H, Fang C, Gong Y, Guo Q, Sun X, Zhao D, Shen J, Zhang H, Liu H, Xia H, Tang J, Zhang K, Gong S. 2020. <a href="https://doi.org/10.1038/s41591-020-0817-4">Characteristics of pediatric SARS-CoV-2 infection and potential evidence for persistent fecal viral shedding</a>. Nature Medicine 26:502–505.</div>
</div>
<div id="ref-VwStn9Ps" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1908. </div><div class="csl-right-inline">Gonzalez-Ochoa G, Flores-Mendoza LK, Icedo-Garcia R, Gomez-Flores R, Tamez-Guerra P. 2017. <a href="https://doi.org/10.1007/s00203-017-1400-3">Modulation of rotavirus severe gastroenteritis by the combination of probiotics and prebiotics</a>. Archives of Microbiology 199:953–961.</div>
</div>
<div id="ref-UtAKolza" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1909. </div><div class="csl-right-inline">Freedman SB, Williamson-Urquhart S, Farion KJ, Gouin S, Willan AR, Poonai N, Hurley K, Sherman PM, Finkelstein Y, Lee BE, Pang X-L, Chui L, Schnadower D, Xie J, Gorelick M, Schuh S. 2018. <a href="https://doi.org/10.1056/nejmoa1802597">Multicenter Trial of a Combination Probiotic for Children with Gastroenteritis</a>. New England Journal of Medicine 379:2015–2026.</div>
</div>
<div id="ref-1AqF1ESKq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1910. </div><div class="csl-right-inline">Medical University of Graz. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04420676">Synbiotic Therapy of Gastrointestinal Symptoms During Covid-19 Infection: A Randomized, Double-blind, Placebo Controlled, Telemedicine Study (SynCov Study)</a>. NCT04420676. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-9UYsmETk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1911. </div><div class="csl-right-inline">Biosearch S.A. 2020. <a href="https://clinicaltrials.gov/ct2/show/NCT04366180">Multicentric Study to Assess the Effect of Consumption of Lactobacillus Coryniformis K8 on Healthcare Personnel Exposed to COVID-19</a>. NCT04366180. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-15pDlYtom" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1912. </div><div class="csl-right-inline">Bioithas SL. 2021. <a href="https://clinicaltrials.gov/ct2/show/NCT04390477">The Intestinal Microbiota as a Therapeutic Target in Hospitalized Patients With COVID-19 Infection</a>. NCT04390477. Clinical trial registration. clinicaltrials.gov.</div>
</div>
<div id="ref-m3SexO9F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1913. </div><div class="csl-right-inline">Reid G. 2016. <a href="https://doi.org/10.1016/j.bpg.2015.12.001">Probiotics: definition, scope and mechanisms of action</a>. Best Practice &amp; Research Clinical Gastroenterology 30:17–25.</div>
</div>
<div id="ref-5FTOk7ol" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1914. </div><div class="csl-right-inline">Rijkers GT, de Vos WM, Brummer R-J, Morelli L, Corthier G, Marteau P. 2011. <a href="https://doi.org/10.1017/s000711451100287x">Health benefits and health claims of probiotics: bridging science and marketing</a>. British Journal of Nutrition 106:1291–1296.</div>
</div>
<div id="ref-kv94VRLF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1915. </div><div class="csl-right-inline">Mak JWY, Chan FKL, Ng SC. 2020. <a href="https://doi.org/10.1016/s2468-1253(20)30122-9">Probiotics and COVID-19: one size does not fit all</a>. The Lancet Gastroenterology &amp; Hepatology 5:644–645.</div>
</div>
<div id="ref-3Hg8dHMg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1916. </div><div class="csl-right-inline">Bloomberg - Are you a robot? <a href="https://www.bloomberg.com/tosv2.html?vid=&amp;uuid=b91b9b90-6a34-11eb-a07d-15fd64b6d7f0&amp;url=L3ByZXNzLXJlbGVhc2VzLzIwMjAtMDgtMDMvcHJvYmlvdGljcy1tYXJrZXQtd29ydGgtNzYtNy1iaWxsaW9uLWJ5LTIwMjctZXhjbHVzaXZlLXJlcG9ydC1jb3ZlcmluZy1wcmUtYW5kLXBvc3QtY292aWQtMTktbWFya2V0LWFuYWx5c2lzLWJ5LW1ldGljdWxvdXM=">https://www.bloomberg.com/tosv2.html?vid=&amp;uuid=b91b9b90-6a34-11eb-a07d-15fd64b6d7f0&amp;url=L3ByZXNzLXJlbGVhc2VzLzIwMjAtMDgtMDMvcHJvYmlvdGljcy1tYXJrZXQtd29ydGgtNzYtNy1iaWxsaW9uLWJ5LTIwMjctZXhjbHVzaXZlLXJlcG9ydC1jb3ZlcmluZy1wcmUtYW5kLXBvc3QtY292aWQtMTktbWFya2V0LWFuYWx5c2lzLWJ5LW1ldGljdWxvdXM=</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-sWzpGBil" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1917. </div><div class="csl-right-inline">Pereira M, Dantas Damascena A, Galvão Azevedo LM, de Almeida Oliveira T, da Mota Santana J. 2020. <a href="https://doi.org/10.1080/10408398.2020.1841090">Vitamin D deficiency aggravates COVID-19: systematic review and meta-analysis</a>. Critical Reviews in Food Science and Nutrition 1–9.</div>
</div>
<div id="ref-1HHbyV6d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1918. </div><div class="csl-right-inline">Galland L. 2010. <a href="https://doi.org/10.1177/0884533610385703">Diet and Inflammation</a>. Nutrition in Clinical Practice 25:634–640.</div>
</div>
<div id="ref-ViPBMxtf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1919. </div><div class="csl-right-inline">Kain V, Van Der Pol W, Mariappan N, Ahmad A, Eipers P, Gibson DL, Gladine C, Vigor C, Durand T, Morrow C, Halade GV. 2019. <a href="https://doi.org/10.1096/fj.201802477r">Obesogenic diet in aging mice disrupts gut microbe composition and alters neutrophi:lymphocyte ratio, leading to inflamed milieu in acute heart failure</a>. The FASEB Journal 33:6456–6469.</div>
</div>
<div id="ref-wtYA1GbG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1920. </div><div class="csl-right-inline">Colloidal Silver. NCCIH. <a href="https://www.nccih.nih.gov/health/colloidal-silver">https://www.nccih.nih.gov/health/colloidal-silver</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-TDa7cEUV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1921. </div><div class="csl-right-inline">Zhou Y, Hou Y, Shen J, Huang Y, Martin W, Cheng F. 2020. <a href="https://doi.org/10.1038/s41421-020-0153-3">Network-based drug repurposing for novel coronavirus 2019-nCoV/SARS-CoV-2</a>. Cell Discovery 6:14.</div>
</div>
<div id="ref-GJAYfMR0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1922. </div><div class="csl-right-inline">Wongchitrat P, Shukla M, Sharma R, Govitrapong P, Reiter RJ. 2021. <a href="https://doi.org/10.3390/antiox10010047">Role of Melatonin on Virus-Induced Neuropathogenesis—A Concomitant Therapeutic Strategy to Understand SARS-CoV-2 Infection</a>. Antioxidants 10:47.</div>
</div>
<div id="ref-2CXoItPz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1923. </div><div class="csl-right-inline">McCarty MF, Iloki Assanga SB, Lewis Luján L, O’Keefe JH, DiNicolantonio JJ. 2020. <a href="https://doi.org/10.3390/nu13010047">Nutraceutical Strategies for Suppressing NLRP3 Inflammasome Activation: Pertinence to the Management of COVID-19 and Beyond</a>. Nutrients 13:47.</div>
</div>
<div id="ref-MNWnrcz6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1924. </div><div class="csl-right-inline">Cohen J. 2020. Update: Here’s what is known about Trump’s COVID-19 treatment. Science https://doi.org/<a href="https://doi.org/10.1126/science.abf0974">10.1126/science.abf0974</a>.</div>
</div>
<div id="ref-RJfF7BYs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1925. </div><div class="csl-right-inline">Louca P, Murray B, Klaser K, Graham MS, Mazidi M, Leeming ER, Thompson E, Bowyer R, Drew DA, Nguyen LH, Merino J, Gomez M, Mompeo O, Costeira R, Sudre CH, Gibson R, Steves CJ, Wolf J, Franks PW, Ourselin S, Chan AT, Berry SE, Valdes AM, Calder PC, Spector TD, Menni C. 2020. Dietary supplements during the COVID-19 pandemic: insights from 1.4M users of the COVID Symptom Study app - a longitudinal app-based community survey. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.11.27.20239087">10.1101/2020.11.27.20239087</a>.</div>
</div>
<div id="ref-pQxCF4GH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1926. </div><div class="csl-right-inline">Barazzoni R, Bischoff SC, Breda J, Wickramasinghe K, Krznaric Z, Nitzan D, Pirlich M, Singer P. 2020. <a href="https://doi.org/10.1016/j.clnu.2020.03.022">ESPEN expert statements and practical guidance for nutritional management of individuals with SARS-CoV-2 infection</a>. Clinical Nutrition 39:1631–1638.</div>
</div>
<div id="ref-d6c8mjAk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1927. </div><div class="csl-right-inline">Haraj NE, El Aziz S, Chadli A, Dafir A, Mjabber A, Aissaoui O, Barrou L, El Kettani El Hamidi C, Nsiri A, AL Harrar R, Ezzouine H, Charra B, Abdallaoui MS, El Kebbaj N, Kamal N, Bennouna GM, El Filali KM, Ramdani B, El Mdaghri N, Gharbi MB, Afif MH. 2021. <a href="https://doi.org/10.1016/j.clnesp.2020.09.214">Nutritional status assessment in patients with Covid-19 after discharge from the intensive care unit</a>. Clinical Nutrition ESPEN 41:423–428.</div>
</div>
<div id="ref-1HL90Mz7X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1928. </div><div class="csl-right-inline">Berger MM. 2020. <a href="https://doi.org/10.1002/jpen.1954">Nutrition Status Affects COVID‐19 Patient Outcomes</a>. Journal of Parenteral and Enteral Nutrition 44:1166–1167.</div>
</div>
<div id="ref-179QIwE1l" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1929. </div><div class="csl-right-inline">Zhao X, Li Y, Ge Y, Shi Y, Lv P, Zhang J, Fu G, Zhou Y, Jiang K, Lin N, Bai T, Jin R, Wu Y, Yang X, Li X. 2020. <a href="https://doi.org/10.1002/jpen.1953">Evaluation of Nutrition Risk and Its Association With Mortality Risk in Severely and Critically Ill COVID‐19 Patients</a>. Journal of Parenteral and Enteral Nutrition 45:32–42.</div>
</div>
<div id="ref-ARM0HdR3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1930. </div><div class="csl-right-inline">Ahmed M, Advani S, Moreira A, Zoretic S, Martinez J, Chorath K, Acosta S, Naqvi R, Burmeister-Morton F, Burmeister F, Tarriela A, Petershack M, Evans M, Hoang A, Rajasekaran K, Ahuja S, Moreira A. 2020. <a href="https://doi.org/10.1016/j.eclinm.2020.100527">Multisystem inflammatory syndrome in children: A systematic review</a>. EClinicalMedicine 26:100527.</div>
</div>
<div id="ref-1DmkRczqw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1931. </div><div class="csl-right-inline">Brugliera L, Spina A, Castellazzi P, Cimino P, Arcuri P, Negro A, Houdayer E, Alemanno F, Giordani A, Mortini P, Iannaccone S. 2020. <a href="https://doi.org/10.1038/s41430-020-0664-x">Nutritional management of COVID-19 patients in a rehabilitation unit</a>. European Journal of Clinical Nutrition 74:860–863.</div>
</div>
<div id="ref-lMhFSIiF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1932. </div><div class="csl-right-inline">Domínguez Díaz L, Fernández-Ruiz V, Cámara M. 2019. <a href="https://doi.org/10.1080/10408398.2019.1592107">The frontier between nutrition and pharma: The international regulatory framework of functional foods, food supplements and nutraceuticals</a>. Critical Reviews in Food Science and Nutrition 60:1738–1746.</div>
</div>
<div id="ref-fez8aZAu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1933. </div><div class="csl-right-inline">Commissioner O of the. 2020. Coronavirus Update: FDA and FTC Warn Seven Companies Selling Fraudulent Products that Claim to Treat or Prevent COVID-19. FDA. <a href="https://www.fda.gov/news-events/press-announcements/coronavirus-update-fda-and-ftc-warn-seven-companies-selling-fraudulent-products-claim-treat-or">https://www.fda.gov/news-events/press-announcements/coronavirus-update-fda-and-ftc-warn-seven-companies-selling-fraudulent-products-claim-treat-or</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-2Xu90WtD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1934. </div><div class="csl-right-inline">CDC. 2021. COVID-19 and Your Health. Centers for Disease Control and Prevention. <a href="https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/prevention.html">https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/prevention.html</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-EhbZIMo0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1935. </div><div class="csl-right-inline">Park SW, Sun K, Viboud C, Grenfell BT, Dushoff J. 2020. Potential roles of social distancing in mitigating the spread of coronavirus disease 2019 (COVID-19) in South Korea. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.27.20045815">10.1101/2020.03.27.20045815</a>.</div>
</div>
<div id="ref-12Met0WvX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1936. </div><div class="csl-right-inline">Matrajt L, Leung T. 2020. <a href="https://doi.org/10.3201/eid2608.201093">Evaluating the Effectiveness of Social Distancing Interventions to Delay or Flatten the Epidemic Curve of Coronavirus Disease</a>. Emerging Infectious Diseases 26:1740–1748.</div>
</div>
<div id="ref-hrOv770z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1937. </div><div class="csl-right-inline">Yehia BR, Winegar A, Fogel R, Fakih M, Ottenbacher A, Jesser C, Bufalino A, Huang R-H, Cacchione J. 2020. <a href="https://doi.org/10.1001/jamanetworkopen.2020.18039">Association of Race With Mortality Among Patients Hospitalized With Coronavirus Disease 2019 (COVID-19) at 92 US Hospitals</a>. JAMA Network Open 3:e2018039.</div>
</div>
<div id="ref-11xfoqpEl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1938. </div><div class="csl-right-inline">Wu Z, McGoogan JM. 2020. <a href="https://doi.org/10.1001/jama.2020.2648">Characteristics of and Important Lessons From the Coronavirus Disease 2019 (COVID-19) Outbreak in China</a>. JAMA 323:1239.</div>
</div>
<div id="ref-qlJ8hCUs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1939. </div><div class="csl-right-inline">Grasselli G, Pesenti A, Cecconi M. 2020. <a href="https://doi.org/10.1001/jama.2020.4031">Critical Care Utilization for the COVID-19 Outbreak in Lombardy, Italy</a>. JAMA 323:1545.</div>
</div>
<div id="ref-xZBEFXiG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1940. </div><div class="csl-right-inline">Garg S, Kim L, Whitaker M, O’Halloran A, Cummings C, Holstein R, Prill M, Chai SJ, Kirley PD, Alden NB, Kawasaki B, Yousey-Hindes K, Niccolai L, Anderson EJ, Openo KP, Weigel A, Monroe ML, Ryan P, Henderson J, Kim S, Como-Sabetti K, Lynfield R, Sosin D, Torres S, Muse A, Bennett NM, Billing L, Sutton M, West N, Schaffner W, Talbot HK, Aquino C, George A, Budd A, Brammer L, Langley G, Hall AJ, Fry A. 2020. <a href="https://doi.org/10.15585/mmwr.mm6915e3">Hospitalization Rates and Characteristics of Patients Hospitalized with Laboratory-Confirmed Coronavirus Disease 2019 — COVID-NET, 14 States, March 1–30, 2020</a>. MMWR Morbidity and Mortality Weekly Report 69:458–464.</div>
</div>
<div id="ref-aZlVFn2R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1941. </div><div class="csl-right-inline">CDC. 2020. COVID-19 and Your Health. Centers for Disease Control and Prevention. <a href="https://www.cdc.gov/coronavirus/2019-ncov/need-extra-precautions/older-adults.html">https://www.cdc.gov/coronavirus/2019-ncov/need-extra-precautions/older-adults.html</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-1D6lVXWbJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1942. </div><div class="csl-right-inline">Azar KMJ, Shen Z, Romanelli RJ, Lockhart SH, Smits K, Robinson S, Brown S, Pressman AR. 2020. <a href="https://doi.org/10.1377/hlthaff.2020.00598">Disparities In Outcomes Among COVID-19 Patients In A Large Health Care System In California</a>. Health Affairs 39:1253–1262.</div>
</div>
<div id="ref-Zk5CfYeW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1943. </div><div class="csl-right-inline">Killerby ME, Link-Gelles R, Haight SC, Schrodt CA, England L, Gomes DJ, Shamout M, Pettrone K, O'Laughlin K, Kimball A, Blau EF, Burnett E, Ladva CN, Szablewski CM, Tobin-D’Angelo M, Oosmanally N, Drenzek C, Murphy DJ, Blum JM, Hollberg J, Lefkove B, Brown FW, Shimabukuro T, Midgley CM, Tate JE, Browning SD, Bruce BB, da Silva J, Gold JAW, Jackson BR, Bamrah Morris S, Natarajan P, Neblett Fanfair R, Patel PR, Rogers-Brown J, Rossow J, Wong KK, CDC COVID-19 Response Clinical Team, CDC COVID-19 Response Clinical Team. 2020. <a href="https://doi.org/10.15585/mmwr.mm6925e1">Characteristics Associated with Hospitalization Among Patients with COVID-19 — Metropolitan Atlanta, Georgia, March–April 2020</a>. MMWR Morbidity and Mortality Weekly Report 69:790–794.</div>
</div>
<div id="ref-BQ4pZGxJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1944. </div><div class="csl-right-inline">Dowd JB, Andriano L, Brazel DM, Rotondi V, Block P, Ding X, Liu Y, Mills MC. 2020. <a href="https://doi.org/10.1073/pnas.2004911117">Demographic science aids in understanding the spread and fatality rates of COVID-19</a>. Proceedings of the National Academy of Sciences 117:9696–9698.</div>
</div>
<div id="ref-1DnayDKc4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1945. </div><div class="csl-right-inline">Shahid Z, Kalayanamitra R, McClafferty B, Kepko D, Ramgobin D, Patel R, Aggarwal CS, Vunnam R, Sahu N, Bhatt D, Jones K, Golamari R, Jain R. 2020. <a href="https://doi.org/10.1111/jgs.16472">‐19 and Older Adults: What We Know</a>. Journal of the American Geriatrics Society 68:926–929.</div>
</div>
<div id="ref-gObtnWSD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1946. </div><div class="csl-right-inline">Docherty AB, Harrison EM, Green CA, Hardwick HE, Pius R, Norman L, Holden KA, Read JM, Dondelinger F, Carson G, Merson L, Lee J, Plotkin D, Sigfrid L, Halpin S, Jackson C, Gamble C, Horby PW, Nguyen-Van-Tam JS, Ho A, Russell CD, Dunning J, Openshaw PJ, Baillie JK, Semple MG. 2020. <a href="https://doi.org/10.1136/bmj.m1985">Features of 20 133 UK patients in hospital with covid-19 using the ISARIC WHO Clinical Characterisation Protocol: prospective observational cohort study</a>. BMJ m1985.</div>
</div>
<div id="ref-DqBeNrR5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1947. </div><div class="csl-right-inline">Gebhard C, Regitz-Zagrosek V, Neuhauser HK, Morgan R, Klein SL. 2020. <a href="https://doi.org/10.1186/s13293-020-00304-9">Impact of sex and gender on COVID-19 outcomes in Europe</a>. Biology of Sex Differences 11:29.</div>
</div>
<div id="ref-ConBOYBB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1948. </div><div class="csl-right-inline">The Sex, Gender and COVID-19 Project | Global Health 50/50. <a href="https://globalhealth5050.org/the-sex-gender-and-covid-19-project/">https://globalhealth5050.org/the-sex-gender-and-covid-19-project/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-fH3qecwy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1949. </div><div class="csl-right-inline">Klein SL, Dhakal S, Ursin RL, Deshpande S, Sandberg K, Mauvais-Jarvis F. 2020. <a href="https://doi.org/10.1371/journal.ppat.1008570">Biological sex impacts COVID-19 outcomes</a>. PLOS Pathogens 16:e1008570.</div>
</div>
<div id="ref-RcYBFjm1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1950. </div><div class="csl-right-inline">Meng Y, Wu P, Lu W, Liu K, Ma K, Huang L, Cai J, Zhang H, Qin Y, Sun H, Ding W, Gui L, Wu P. 2020. <a href="https://doi.org/10.1371/journal.ppat.1008520">Sex-specific clinical characteristics and prognosis of coronavirus disease-19 infection in Wuhan, China: A retrospective study of 168 severe patients</a>. PLOS Pathogens 16:e1008520.</div>
</div>
<div id="ref-OJvjTsWz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1951. </div><div class="csl-right-inline">Liu J, Ji H, Zheng W, Wu X, Zhu JJ, Arnold AP, Sandberg K. 2010. <a href="https://doi.org/10.1186/2042-6410-1-6">Sex differences in renal angiotensin converting enzyme 2 (ACE2) activity are 17β-oestradiol-dependent and sex chromosome-independent</a>. Biology of Sex Differences 1:6.</div>
</div>
<div id="ref-ujWsdPdH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1952. </div><div class="csl-right-inline">Fallon A, Dukelow T, Kennelly SP, O’Neill D. 2020. <a href="https://doi.org/10.1093/qjmed/hcaa136">COVID-19 in nursing homes</a>. QJM: An International Journal of Medicine 113:391–392.</div>
</div>
<div id="ref-agpMLmxt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1953. </div><div class="csl-right-inline">Herman JL, O’Neill K. 2020. <a href="https://escholarship.org/uc/item/55t297mc">Vulnerabilities to COVID-19 Among Transgender Adults in the U.S.</a></div>
</div>
<div id="ref-9NfXSHjH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1954. </div><div class="csl-right-inline">Zhu L, She Z-G, Cheng X, Qin J-J, Zhang X-J, Cai J, Lei F, Wang H, Xie J, Wang W, Li H, Zhang P, Song X, Chen X, Xiang M, Zhang C, Bai L, Xiang D, Chen M-M, Liu Y, Yan Y, Liu M, Mao W, Zou J, Liu L, Chen G, Luo P, Xiao B, Zhang C, Zhang Z, Lu Z, Wang J, Lu H, Xia X, Wang D, Liao X, Peng G, Ye P, Yang J, Yuan Y, Huang X, Guo J, Zhang B-H, Li H. 2020. <a href="https://doi.org/10.1016/j.cmet.2020.04.021">Association of Blood Glucose Control and Outcomes in Patients with COVID-19 and Pre-existing Type 2 Diabetes</a>. Cell Metabolism 31:1068–1077.e3.</div>
</div>
<div id="ref-13BpwZaaq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1955. </div><div class="csl-right-inline">Wu Z, Tang Y, Cheng Q. 2020. Diabetes increases the mortality of patients with COVID-19: a meta-analysis. Acta Diabetologica https://doi.org/<a href="https://doi.org/10.1007/s00592-020-01546-0">10.1007/s00592-020-01546-0</a>.</div>
</div>
<div id="ref-UkCdtLwF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1956. </div><div class="csl-right-inline">Li J, Wang X, Chen J, Zuo X, Zhang H, Deng A. 2020. COVID-19 infection may cause ketosis and ketoacidosis. Diabetes, Obesity and Metabolism https://doi.org/<a href="https://doi.org/10.1111/dom.14057">10.1111/dom.14057</a>.</div>
</div>
<div id="ref-OT5GEOej" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1957. </div><div class="csl-right-inline">Muniyappa R, Gubbi S. 2020. <a href="https://doi.org/10.1152/ajpendo.00124.2020">COVID-19 pandemic, coronaviruses, and diabetes mellitus</a>. American Journal of Physiology-Endocrinology and Metabolism 318:E736–E741.</div>
</div>
<div id="ref-ZLmNMDrD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1958. </div><div class="csl-right-inline">Palaiodimos L, Kokkinidis DG, Li W, Karamanis D, Ognibene J, Arora S, Southern WN, Mantzoros CS. 2020. <a href="https://doi.org/10.1016/j.metabol.2020.154262">Severe obesity, increasing age and male sex are independently associated with worse in-hospital outcomes, and higher in-hospital mortality, in a cohort of patients with COVID-19 in the Bronx, New York</a>. Metabolism 108:154262.</div>
</div>
<div id="ref-2Ahpn6R2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1959. </div><div class="csl-right-inline">Docherty A, Harrison E, Green C, Hardwick H, Pius R, Norman L, Holden K, Read J, Dondelinger F, Carson G, Merson L, Lee J, Plotkin D, Sigfrid L, Halpin S, Jackson C, Gamble C, Horby P, Nguyen-Van-Tam J, Dunning J, Openshaw P, Baillie J, Semple M, ISARIC4C Investigators. 2020. Features of 16,749 hospitalised UK patients with COVID-19 using the ISARIC WHO Clinical Characterisation Protocol. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.04.23.20076042">10.1101/2020.04.23.20076042</a>.</div>
</div>
<div id="ref-1AmBg80Un" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1960. </div><div class="csl-right-inline">Lockhart SM, O’Rahilly S. 2020. <a href="https://doi.org/10.1016/j.medj.2020.06.005">When Two Pandemics Meet: Why Is Obesity Associated with Increased COVID-19 Mortality?</a> Med 1:33–42.</div>
</div>
<div id="ref-1GakM8xhV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1961. </div><div class="csl-right-inline">Nepomuceno MR, Acosta E, Alburez-Gutierrez D, Aburto JM, Gagnon A, Turra CM. 2020. <a href="https://doi.org/10.1073/pnas.2008760117">Besides population age structure, health and other demographic factors can contribute to understanding the COVID-19 burden</a>. Proceedings of the National Academy of Sciences 117:13881–13883.</div>
</div>
<div id="ref-H6dfv6Wv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1962. </div><div class="csl-right-inline">. <a href="https://www.who.int/publications/i/item/report-of-the-who-china-joint-mission-on-coronavirus-disease-2019">https://www.who.int/publications/i/item/report-of-the-who-china-joint-mission-on-coronavirus-disease-2019</a>.</div>
</div>
<div id="ref-TuzwDCPt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1963. </div><div class="csl-right-inline">Vincent J-L, Moreno R, Takala J, Willatts S, De Mendonça A, Bruining H, Reinhart CK, Suter PM, Thijs LG. 1996. <a href="https://doi.org/10.1007/bf01709751">The SOFA (Sepsis-related Organ Failure Assessment) score to describe organ dysfunction/failure</a>. Intensive Care Medicine 22:707–710.</div>
</div>
<div id="ref-a0h975P6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1964. </div><div class="csl-right-inline">Singer M, Deutschman CS, Seymour CW, Shankar-Hari M, Annane D, Bauer M, Bellomo R, Bernard GR, Chiche J-D, Coopersmith CM, Hotchkiss RS, Levy MM, Marshall JC, Martin GS, Opal SM, Rubenfeld GD, van der Poll T, Vincent J-L, Angus DC. 2016. <a href="https://doi.org/10.1001/jama.2016.0287">The Third International Consensus Definitions for Sepsis and Septic Shock (Sepsis-3)</a>. JAMA 315:801.</div>
</div>
<div id="ref-jKKxa9Yy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1965. </div><div class="csl-right-inline">Yancy CW. 2020. <a href="https://doi.org/10.1001/jama.2020.6548">COVID-19 and African Americans</a>. JAMA 323:1891.</div>
</div>
<div id="ref-yK5bczRb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1966. </div><div class="csl-right-inline">Webb Hooper M, Nápoles AM, Pérez-Stable EJ. 2020. <a href="https://doi.org/10.1001/jama.2020.8598">COVID-19 and Racial/Ethnic Disparities</a>. JAMA 323:2466.</div>
</div>
<div id="ref-u4MMV8aJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1967. </div><div class="csl-right-inline">Dyer O. 2020. <a href="https://doi.org/10.1136/bmj.m1483">Covid-19: Black people and other minorities are hardest hit in US</a>. BMJ m1483.</div>
</div>
<div id="ref-JAo2QxY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1968. </div><div class="csl-right-inline">Kakol M, Upson D, Sood A. 2020. <a href="https://doi.org/10.1111/jrh.12451">Susceptibility of Southwestern American Indian Tribes to Coronavirus Disease 2019 (COVID‐19)</a>. The Journal of Rural Health 37:197–199.</div>
</div>
<div id="ref-12WZObWRf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1969. </div><div class="csl-right-inline">Jr RAO, Gebeloff R, Lai KKR, Wright W, Smith M. 2020. <a href="https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html">The Fullest Look Yet at the Racial Inequity of Coronavirus</a>. The New York Times.</div>
</div>
<div id="ref-Pp8GWnQm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1970. </div><div class="csl-right-inline">Wang ML, Behrman P, Dulin A, Baskin ML, Buscemi J, Alcaraz KI, Goldstein CM, Carson TL, Shen M, Fitzgibbon M. 2020. <a href="https://doi.org/10.1093/tbm/ibaa055">Addressing inequities in COVID-19 morbidity and mortality: research and policy recommendations</a>. Translational Behavioral Medicine 10:516–519.</div>
</div>
<div id="ref-J9T8eR5H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1971. </div><div class="csl-right-inline">Parkhurst NAD, Huyser KR, Horse AJY. 2020. <a href="https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753">Historical Environmental Racism, Structural Inequalities, and Dik’os Ntsaaígíí-19 (COVID-19) on Navajo Nation</a>. Journal of Indigenous Social Development 9:127–140.</div>
</div>
<div id="ref-ODm1CaEq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1972. </div><div class="csl-right-inline">Ferrante L, Fearnside PM. 2020. <a href="https://doi.org/10.1126/science.abc0073">Protect Indigenous peoples from COVID-19</a>. Science 368:251.1–251.</div>
</div>
<div id="ref-17gnxmOnf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1973. </div><div class="csl-right-inline">Williamson EJ, Walker AJ, Bhaskaran K, Bacon S, Bates C, Morton CE, Curtis HJ, Mehrkar A, Evans D, Inglesby P, Cockburn J, McDonald HI, MacKenna B, Tomlinson L, Douglas IJ, Rentsch CT, Mathur R, Wong AYS, Grieve R, Harrison D, Forbes H, Schultze A, Croker R, Parry J, Hester F, Harper S, Perera R, Evans SJW, Smeeth L, Goldacre B. 2020. <a href="https://doi.org/10.1038/s41586-020-2521-4">Factors associated with COVID-19-related death using OpenSAFELY</a>. Nature 584:430–436.</div>
</div>
<div id="ref-pwbrxYv8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1974. </div><div class="csl-right-inline">Tishkoff SA, Kidd KK. 2004. <a href="https://doi.org/10.1038/ng1438">Implications of biogeography of human populations for 'race' and medicine</a>. Nature Genetics 36:S21–S27.</div>
</div>
<div id="ref-CMAPEmTp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1975. </div><div class="csl-right-inline">Campbell MC, Tishkoff SA. 2008. <a href="https://doi.org/10.1146/annurev.genom.9.081307.164258">African Genetic Diversity: Implications for Human Demographic History, Modern Human Origins, and Complex Disease Mapping</a>. Annual Review of Genomics and Human Genetics 9:403–433.</div>
</div>
<div id="ref-q4yhes98" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1976. </div><div class="csl-right-inline">Yudell M, Roberts D, DeSalle R, Tishkoff S, 70 signatories. 2020. <a href="https://doi.org/10.1126/science.abd4842">NIH must confront the use of race in science</a>. Science 369:1313–1314.</div>
</div>
<div id="ref-nn67UTjV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1977. </div><div class="csl-right-inline">Kuo C-L, Pilling LC, Atkins JL, Masoli JAH, Delgado J, Kuchel GA, Melzer D. 2020. <a href="https://doi.org/10.1093/gerona/glaa131">APOE e4 Genotype Predicts Severe COVID-19 in the UK Biobank Community Cohort</a>. The Journals of Gerontology: Series A 75:2231–2232.</div>
</div>
<div id="ref-I2mIU0wn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1978. </div><div class="csl-right-inline">Wei J, Alfajaro MM, Hanna RE, DeWeirdt PC, Strine MS, Lu-Culligan WJ, Zhang S-M, Graziano VR, Schmitz CO, Chen JS, Mankowski MC, Filler RB, Gasque V, de Miguel F, Chen H, Oguntuyo K, Abriola L, Surovtseva YV, Orchard RC, Lee B, Lindenbach B, Politi K, van Dijk D, Simon MD, Yan Q, Doench JG, Wilen CB. 2020. Genome-wide CRISPR screen reveals host genes that regulate SARS-CoV-2 infection. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.06.16.155101">10.1101/2020.06.16.155101</a>.</div>
</div>
<div id="ref-KbDdcSgo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1979. </div><div class="csl-right-inline">Hou Y, Zhao J, Martin W, Kallianpur A, Chung MK, Jehi L, Sharifi N, Erzurum S, Eng C, Cheng F. 2020. <a href="https://doi.org/10.1186/s12916-020-01673-z">New insights into genetic susceptibility of COVID-19: an ACE2 and TMPRSS2 polymorphism analysis</a>. BMC Medicine 18:216.</div>
</div>
<div id="ref-4SqaP0Ub" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1980. </div><div class="csl-right-inline">Uyoga S, M.O. Adetifa I, Karanja HK, Nyagwange J, Tuju J, Wanjiku P, Aman R, Mwangangi M, Amoth P, Kasera K, Ng’ang’a W, Rombo C, Yegon C, Kithi K, Odhiambo E, Rotich T, Orgut I, Kihara S, Otiende M, Bottomley C, N. Mupe Z, W. Kagucia E, E Gallagher K, Etyang A, Voller S, N. Gitonga J, Mugo D, N. Agoti C, Otieno E, Ndwiga L, Lambe T, Wright D, Barasa E, Tsofa B, Bejon P, I. Ochola-Oyier L, Agweyu A, Scott JAG, Warimwe GM. 2020. Seroprevalence of anti-SARS-CoV-2 IgG antibodies in Kenyan blood donors. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.07.27.20162693">10.1101/2020.07.27.20162693</a>.</div>
</div>
<div id="ref-eWIOKa0u" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1981. </div><div class="csl-right-inline">Chibwana MG, Jere KC, Kamn’gona R, Mandolo J, Katunga-Phiri V, Tembo D, Mitole N, Musasa S, Sichone S, Lakudzala A, Sibale L, Matambo P, Kadwala I, Byrne RL, Mbewe A, Morton B, Phiri C, Mallewa J, Mwandumba HC, Adams ER, Gordon SB, Jambo KC. 2020. High SARS-CoV-2 seroprevalence in Health Care Workers but relatively low numbers of deaths in urban Malawi. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.07.30.20164970">10.1101/2020.07.30.20164970</a>.</div>
</div>
<div id="ref-p4AYZss0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1982. </div><div class="csl-right-inline">Nordling L. 2020. <a href="https://doi.org/10.1126/science.369.6505.756">Africa's pandemic puzzle: why so few cases and deaths?</a> Science 369:756–757.</div>
</div>
<div id="ref-xFKmPg6l" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1983. </div><div class="csl-right-inline">Are some ethnic groups more vulnerable to COVID-19 than others? <a href="https://www.ifs.org.uk/inequality/chapter/are-some-ethnic-groups-more-vulnerable-to-covid-19-than-others/">https://www.ifs.org.uk/inequality/chapter/are-some-ethnic-groups-more-vulnerable-to-covid-19-than-others/</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-bDNDEYNn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1984. </div><div class="csl-right-inline">Dasgupta N, Funk MJ, Lazard A, White BE, Marshall SW. 2020. Quantifying the social distancing privilege gap: a longitudinal study of smartphone movement. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.05.03.20084624">10.1101/2020.05.03.20084624</a>.</div>
</div>
<div id="ref-RdnghmVU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1985. </div><div class="csl-right-inline">Fraiberger SP, Astudillo P, Candeago L, Chunet A, Jones NKW, Khan MF, Lepri B, Gracia NL, Lucchini L, Massaro E, Montfort A. 2020. <a href="https://arxiv.org/abs/2006.15195">Uncovering socioeconomic gaps in mobility reduction during the COVID-19 pandemic using location data</a>. 2006.15195arXiv. arXiv.</div>
</div>
<div id="ref-9OksxLmM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1986. </div><div class="csl-right-inline">Chang S, Pierson E, Koh PW, Gerardin J, Redbird B, Grusky D, Leskovec J. 2020. <a href="https://doi.org/10.1038/s41586-020-2923-3">Mobility network models of COVID-19 explain inequities and inform reopening</a>. Nature 589:82–87.</div>
</div>
<div id="ref-15qfRPnUG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1987. </div><div class="csl-right-inline">Brown HJRF. 2020. A Basic Demographic Profile of Workers in Frontline Industries. <a href="https://mronline.org/wp-content/uploads/2020/06/2020-04-Frontline-Workers.pdf">https://mronline.org/wp-content/uploads/2020/06/2020-04-Frontline-Workers.pdf</a>.</div>
</div>
<div id="ref-HihYbn6k" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1988. </div><div class="csl-right-inline">Hawkins D. 2020. <a href="https://doi.org/10.1002/ajim.23145">Differential occupational risk for COVID‐19 and other infection exposure according to race and ethnicity</a>. American Journal of Industrial Medicine 63:817–820.</div>
</div>
<div id="ref-F4laSOXM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1989. </div><div class="csl-right-inline">Baker MG, Peckham TK, Seixas NS. 2020. <a href="https://doi.org/10.1371/journal.pone.0232452">Estimating the burden of United States workers exposed to infection or disease: A key factor in containing risk of COVID-19 infection</a>. PLOS ONE 15:e0232452.</div>
</div>
<div id="ref-hAYohR8m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1990. </div><div class="csl-right-inline">Windsor-Shellard B, Kaur J. 2020. Coronavirus (COVID-19) related deaths by occupation, England and Wales: deaths registered up to and including 20 April 2020. <a href="https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/bulletins/coronaviruscovid19relateddeathsbyoccupationenglandandwales/deathsregistereduptoandincluding20april2020">https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/bulletins/coronaviruscovid19relateddeathsbyoccupationenglandandwales/deathsregistereduptoandincluding20april2020</a>.</div>
</div>
<div id="ref-DOyiuaup" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1991. </div><div class="csl-right-inline">Office for National Statistics. 2020. Which occupations have the highest potential exposure to the coronavirus (COVID-19)? <a href="https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/articles/whichoccupationshavethehighestpotentialexposuretothecoronaviruscovid19/2020-05-11">https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/articles/whichoccupationshavethehighestpotentialexposuretothecoronaviruscovid19/2020-05-11</a>.</div>
</div>
<div id="ref-13qkVV6eN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1992. </div><div class="csl-right-inline">Public Health England. 2020. Disparities in the risk and outcomes from COVID-19. <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/892085/disparities_review.pdf">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/892085/disparities_review.pdf</a>.</div>
</div>
<div id="ref-4zjepZ8T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1993. </div><div class="csl-right-inline">Cook T, Kursumovic E, Lennane2020-04-22T12:42:00+01:00 S. Exclusive: deaths of NHS staff from covid-19 analysed. Health Service Journal. <a href="https://www.hsj.co.uk/exclusive-deaths-of-nhs-staff-from-covid-19-analysed/7027471.article">https://www.hsj.co.uk/exclusive-deaths-of-nhs-staff-from-covid-19-analysed/7027471.article</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-2gxJPENM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1994. </div><div class="csl-right-inline">. <a href="https://act.nationalnursesunited.org/page">https://act.nationalnursesunited.org/page</a>.</div>
</div>
<div id="ref-Moc3BREQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1995. </div><div class="csl-right-inline">McLaren J. 2020. <a href="https://www.nber.org/papers/w27407">Racial Disparity in COVID-19 Deaths: Seeking Economic Roots with Census data.</a> w27407. National Bureau of Economic Research.</div>
</div>
<div id="ref-aM2XPE3J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1996. </div><div class="csl-right-inline">Barnett ML, Hu L, Martin T, Grabowski DC. 2020. <a href="https://doi.org/10.1001/jama.2020.11642">Mortality, Admissions, and Patient Census at SNFs in 3 US Cities During the COVID-19 Pandemic</a>. JAMA 324:507.</div>
</div>
<div id="ref-BeedQx3I" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1997. </div><div class="csl-right-inline">Hawks L, Woolhandler S, McCormick D. 2020. <a href="https://doi.org/10.1001/jamainternmed.2020.1856">COVID-19 in Prisons and Jails in the United States</a>. JAMA Internal Medicine 180:1041.</div>
</div>
<div id="ref-2uUcva6Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1998. </div><div class="csl-right-inline">Saloner B, Parish K, Ward JA, DiLaura G, Dolovich S. 2020. <a href="https://doi.org/10.1001/jama.2020.12528">COVID-19 Cases and Deaths in Federal and State Prisons</a>. JAMA 324:602.</div>
</div>
<div id="ref-zuipD7ZP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1999. </div><div class="csl-right-inline">nlizanna. 2018.
State Rates of Incarceration Race &amp;amp; Ethnicity_updated2
. <a href="https://www.issuelab.org/resources/695/695.pdf">https://www.issuelab.org/resources/695/695.pdf</a>.</div>
</div>
<div id="ref-OzhmHOxJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2000. </div><div class="csl-right-inline">Keene JR, Batson CD. 2010. <a href="https://doi.org/10.1111/j.1751-9020.2010.00306.x">Under One Roof: A Review of Research on Intergenerational Coresidence and Multigenerational Households in the United States</a>. Sociology Compass 4:642–657.</div>
</div>
<div id="ref-13wfXDXqr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2001. </div><div class="csl-right-inline">Evans GW, Eckenrode J, Marcynyszyn LA. 2010. <a href="https://doi.org/10.1037/12057-014">Chaos and the macrosetting: The role of poverty and socioeconomic status.</a>, p. 225–238. <em>In</em>. American Psychological Association (APA).</div>
</div>
<div id="ref-1Axz3a1Hj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2002. </div><div class="csl-right-inline">Curtis MA, Geller AB. 2010. Housing insecurity among urban fathers. Columbia University https://doi.org/<a href="https://doi.org/10.7916/d8wh2w9t">10.7916/d8wh2w9t</a>.</div>
</div>
<div id="ref-64YOO3yt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2003. </div><div class="csl-right-inline">Desmond M, Gershenson C. 2016. <a href="https://doi.org/10.1093/socpro/spv025">Housing and Employment Insecurity among the Working Poor</a>. Social Problems 63:46–67.</div>
</div>
<div id="ref-sRZCzj6j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2004. </div><div class="csl-right-inline">Khaodhiar L, McCowen KC, Blackburn GL. 1999. <a href="https://doi.org/10.1016/s1098-3597(99)90002-9">Obesity and its comorbid conditions</a>. Clinical Cornerstone 2:17–31.</div>
</div>
<div id="ref-JizR1ocj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2005. </div><div class="csl-right-inline">Mauvais-Jarvis F. 2020. <a href="https://doi.org/10.2337/dbi19-0023">Aging, Male Sex, Obesity, and Metabolic Inflammation Create the Perfect Storm for COVID-19</a>. Diabetes 69:1857–1863.</div>
</div>
<div id="ref-nYLB9EMw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2006. </div><div class="csl-right-inline">Mendenhall E, Kohrt BA, Norris SA, Ndetei D, Prabhakaran D. 2017. <a href="https://doi.org/10.1016/s0140-6736(17)30402-6">Non-communicable disease syndemics: poverty, depression, and diabetes among low-income populations</a>. The Lancet 389:951–963.</div>
</div>
<div id="ref-h1e9QTlK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2007. </div><div class="csl-right-inline">Żukiewicz-Sobczak W, Wróblewska P, Zwoliński J, Chmielewska-Badora J, Adamczuk P, Krasowska E, Zagórski J, Oniszczuk A, Piątek J, Silny W. 2014. <a href="https://doi.org/10.5604/12321966.1120608">Obesity and poverty paradox in developed countries</a>. Annals of Agricultural and Environmental Medicine 21:590–594.</div>
</div>
<div id="ref-94eOAGnA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2008. </div><div class="csl-right-inline">Ashby NJS. 2020. <a href="https://doi.org/10.1002/oby.22940">Impact of the COVID‐19 Pandemic on Unhealthy Eating in Populations with Obesity</a>. Obesity 28:1802–1805.</div>
</div>
<div id="ref-5G5UozXd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2009. </div><div class="csl-right-inline">Myers CA, Broyles ST. 2020. <a href="https://doi.org/10.1002/oby.22993">Fast Food Patronage and Obesity Prevalence During the COVID‐19 Pandemic: An Alternative Explanation</a>. Obesity 28:1796–1797.</div>
</div>
<div id="ref-17yfPiYAS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2010. </div><div class="csl-right-inline">Huizar MI, Arena R, Laddu DR. 2020. <a href="https://doi.org/10.1016/j.pcad.2020.07.002">The global food syndemic: The impact of food insecurity, Malnutrition and obesity on the healthspan amid the COVID-19 pandemic</a>. Progress in Cardiovascular Diseases S0033062020301390.</div>
</div>
<div id="ref-Hu7PyNNp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2011. </div><div class="csl-right-inline">Chrousos GP. 2000. <a href="https://doi.org/10.1067/mai.2000.110163">Stress, chronic inflammation, and emotional and physical well-being: Concurrent effects and chronic sequelae</a>. Journal of Allergy and Clinical Immunology 106:S275–S291.</div>
</div>
<div id="ref-P2wUkjrR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2012. </div><div class="csl-right-inline">Miller GE, Cohen S, Ritchey AK. 2002. <a href="https://doi.org/10.1037/0278-6133.21.6.531">Chronic psychological stress and the regulation of pro-inflammatory cytokines: A glucocorticoid-resistance model.</a> Health Psychology 21:531–541.</div>
</div>
<div id="ref-109EtOYzw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2013. </div><div class="csl-right-inline">Gouin J-P, Glaser R, Malarkey WB, Beversdorf D, Kiecolt-Glaser J. 2012. <a href="https://doi.org/10.1037/a0025536">Chronic stress, daily stressors, and circulating inflammatory markers.</a> Health Psychology 31:264–268.</div>
</div>
<div id="ref-11oGPv27i" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2014. </div><div class="csl-right-inline">Miller GE, Blackwell E. 2016. <a href="https://doi.org/10.1111/j.1467-8721.2006.00450.x">Turning Up the Heat</a>. Current Directions in Psychological Science 15:269–272.</div>
</div>
<div id="ref-KZSJklnl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2015. </div><div class="csl-right-inline">Sapolsky R. 2005. <a href="https://doi.org/10.1038/scientificamerican1205-92">Sick of Poverty</a>. Scientific American 293:92–99.</div>
</div>
<div id="ref-5BMDcXz3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2016. </div><div class="csl-right-inline">Wu X, Nethery RC, Sabath MB, Braun D, Dominici F. 2020. Exposure to air pollution and COVID-19 mortality in the United States: A nationwide cross-sectional study. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.04.05.20054502">10.1101/2020.04.05.20054502</a>.</div>
</div>
<div id="ref-vImXUPNS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2017. </div><div class="csl-right-inline">Burnett R, Chen H, Szyszkowicz M, Fann N, Hubbell B, Pope CA, Apte JS, Brauer M, Cohen A, Weichenthal S, Coggins J, Di Q, Brunekreef B, Frostad J, Lim SS, Kan H, Walker KD, Thurston GD, Hayes RB, Lim CC, Turner MC, Jerrett M, Krewski D, Gapstur SM, Diver WR, Ostro B, Goldberg D, Crouse DL, Martin RV, Peters P, Pinault L, Tjepkema M, van Donkelaar A, Villeneuve PJ, Miller AB, Yin P, Zhou M, Wang L, Janssen NAH, Marra M, Atkinson RW, Tsang H, Quoc Thach T, Cannon JB, Allen RT, Hart JE, Laden F, Cesaroni G, Forastiere F, Weinmayr G, Jaensch A, Nagel G, Concin H, Spadaro JV. 2018. <a href="https://doi.org/10.1073/pnas.1803222115">Global estimates of mortality associated with long-term exposure to outdoor fine particulate matter</a>. Proceedings of the National Academy of Sciences 115:9592–9597.</div>
</div>
<div id="ref-6ACdDJOV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2018. </div><div class="csl-right-inline">Kravitz-Wirtz N, Teixeira S, Hajat A, Woo B, Crowder K, Takeuchi D. 2018. <a href="https://doi.org/10.3390/ijerph15061114">Early-Life Air Pollution Exposure, Neighborhood Poverty, and Childhood Asthma in the United States, 1990–2014</a>. International Journal of Environmental Research and Public Health 15:1114.</div>
</div>
<div id="ref-11Ift2eIl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2019. </div><div class="csl-right-inline">Olvera Alvarez HA, Kubzansky LD, Campen MJ, Slavich GM. 2018. <a href="https://doi.org/10.1016/j.neubiorev.2018.06.002">Early life stress, air pollution, inflammation, and disease: An integrative review and immunologic model of social-environmental adversity and lifespan health</a>. Neuroscience &amp; Biobehavioral Reviews 92:226–242.</div>
</div>
<div id="ref-aHM6WZzd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2020. </div><div class="csl-right-inline">Belanger MJ, Hill MA, Angelidi AM, Dalamaga M, Sowers JR, Mantzoros CS. 2020. <a href="https://doi.org/10.1056/nejmp2021264">Covid-19 and Disparities in Nutrition and Obesity</a>. New England Journal of Medicine 383:e69.</div>
</div>
<div id="ref-11Ldf3Ch3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2021. </div><div class="csl-right-inline">Gravlee CC. 2020. <a href="https://doi.org/10.1002/ajhb.23482">Systemic racism, chronic health inequities, and ‐19: A syndemic in the making?</a> American Journal of Human Biology 32.</div>
</div>
<div id="ref-XtKjlYFF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2022. </div><div class="csl-right-inline">Lopez L III, Hart LH III, Katz MH. 2021. <a href="https://doi.org/10.1001/jama.2020.26443">Racial and Ethnic Health Disparities Related to COVID-19</a>. JAMA 325:719.</div>
</div>
<div id="ref-cSA2h9hO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2023. </div><div class="csl-right-inline">Chavez S, Long B, Koyfman A, Liang SY. 2020. <a href="https://doi.org/10.1016/j.ajem.2020.03.036">Coronavirus Disease (COVID-19): A primer for emergency physicians</a>. The American Journal of Emergency Medicine S0735675720301789.</div>
</div>
<div id="ref-1Intrlax" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2024. </div><div class="csl-right-inline">Insurers May Only Pay For Coronavirus Tests When They're 'Medically Necessary'. NPRorg. <a href="https://www.npr.org/sections/health-shots/2020/06/19/880543755/insurers-may-only-pay-for-coronavirus-tests-when-theyre-medically-necessary">https://www.npr.org/sections/health-shots/2020/06/19/880543755/insurers-may-only-pay-for-coronavirus-tests-when-theyre-medically-necessary</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-jc9vc7ws" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2025. </div><div class="csl-right-inline">Private Health Insurance Coverage in the COVID-19 Public Health Emergency | Commonwealth Fund. <a href="https://www.commonwealthfund.org/blog/2020/private-health-insurance-coverage-covid-19-public-health-emergency">https://www.commonwealthfund.org/blog/2020/private-health-insurance-coverage-covid-19-public-health-emergency</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-13MGsHBtw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2026. </div><div class="csl-right-inline">Shah M, Sachdeva M, Dodiuk-Gad RP. 2020. <a href="https://doi.org/10.1016/j.jaad.2020.04.046">COVID-19 and racial disparities</a>. Journal of the American Academy of Dermatology 83:e35.</div>
</div>
<div id="ref-P8VNkYTE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2027. </div><div class="csl-right-inline">FAQs for COVID-19 Claims Reimbursement to Health Care Providers and Facilities for Testing, Treatment and Vaccine Administration. Official web site of the US Health Resources &amp; Services Administration. Text. <a href="https://www.hrsa.gov/coviduninsuredclaim/frequently-asked-questions">https://www.hrsa.gov/coviduninsuredclaim/frequently-asked-questions</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-tLVYjfMk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2028. </div><div class="csl-right-inline">Ji Y, Ma Z, Peppelenbosch MP, Pan Q. 2020. <a href="https://doi.org/10.1016/s2214-109x(20)30068-1">Potential association between COVID-19 mortality and health-care resource availability</a>. The Lancet Global Health 8:e480.</div>
</div>
<div id="ref-cOQK0zNa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2029. </div><div class="csl-right-inline">Wang Z, Tang K. 2020. <a href="https://doi.org/10.1038/s41591-020-0823-6">Combating COVID-19: health equity matters</a>. Nature Medicine 26:458–458.</div>
</div>
<div id="ref-gOiB5HDd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2030. </div><div class="csl-right-inline">chen0307. 2020.
lockup black
. <a href="https://www.census.gov/content/dam/Census/library/publications/2019/demo/p60-267.pdf">https://www.census.gov/content/dam/Census/library/publications/2019/demo/p60-267.pdf</a>.</div>
</div>
<div id="ref-15Yaf3fc8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2031. </div><div class="csl-right-inline">Raluca D, Veronica C, Oana MC, Alexandra E. 2020. <a href="https://doi.org/10.19044/esj.2020.v16n21p24">An Ethical Dilemma in SARS-Cov-2 Pandemic : Who Gets the Ventilator?</a> European Scientific Journal ESJ 16.</div>
</div>
<div id="ref-1As9kWVvA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2032. </div><div class="csl-right-inline">Raffensperger J, Brauner M, Briggs R. 2020. Planning Hospital Needs for Ventilators and Respiratory Therapists in the COVID-19 Crisis. Rand Corporation. <a href="https://doi.org/ghfprp">https://doi.org/ghfprp</a>.</div>
</div>
<div id="ref-FaGAJ7AI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2033. </div><div class="csl-right-inline">Pathak PA, Sönmez T, Ünver MU, Yenmez MB. 2021. <a href="https://arxiv.org/abs/2008.00374">Fair Allocation of Vaccines, Ventilators and Antiviral Treatments: Leaving No Ethical Value Behind in Health Care Rationing</a>. 2008.00374arXiv. arXiv.</div>
</div>
<div id="ref-B3awlFvs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2034. </div><div class="csl-right-inline">Chu Q, Correa R, Henry TL, McGregor KA, Stoklosa H, Robinson L, Jha S, Annamalai A, Hsu BS, Gupta R, Patton DU, Moreno-Walton LA, Butts C, Chai C, Kuy S. 2020. <a href="https://doi.org/10.1016/j.surg.2020.04.044">Reallocating ventilators during the coronavirus disease 2019 pandemic: Is it ethical?</a> Surgery 168:388–391.</div>
</div>
<div id="ref-2i1Y7SYn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2035. </div><div class="csl-right-inline">Dudzinski DM, Hoisington BY, Brown CE. 2020. <a href="https://doi.org/10.1080/15265161.2020.1764137">Ethics Lessons From Seattle’s Early Experience With COVID-19</a>. The American Journal of Bioethics 20:67–74.</div>
</div>
<div id="ref-GTL02UT6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2036. </div><div class="csl-right-inline">Farrell TW, Francis L, Brown T, Ferrante LE, Widera E, Rhodes R, Rosen T, Hwang U, Witt LJ, Thothala N, Liu SW, Vitale CA, Braun UK, Stephens C, Saliba D. 2020. <a href="https://doi.org/10.1111/jgs.16539">Rationing Limited Healthcare Resources in the COVID‐19 Era and Beyond: Ethical Considerations Regarding Older Adults</a>. Journal of the American Geriatrics Society 68:1143–1149.</div>
</div>
<div id="ref-KQfJ4llw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2037. </div><div class="csl-right-inline">Haward MF, Moore GP, Lantos J, Janvier A. 2020. <a href="https://doi.org/10.1111/apa.15334">Paediatric ethical issues during the COVID‐19 pandemic are not just about ventilator triage</a>. Acta Paediatrica 109:1519–1521.</div>
</div>
<div id="ref-K6yc6elC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2038. </div><div class="csl-right-inline">McGuire AL, Aulisio MP, Davis FD, Erwin C, Harter TD, Jagsi R, Klitzman R, Macauley R, Racine E, Wolf SM, Wynia M, Wolpe PR, The COVID-19 Task Force of the Association of Bioethics Program Directors (ABPD). 2020. <a href="https://doi.org/10.1080/15265161.2020.1764138">Ethical Challenges Arising in the COVID-19 Pandemic: An Overview from the Association of Bioethics Program Directors (ABPD) Task Force</a>. The American Journal of Bioethics 20:15–27.</div>
</div>
<div id="ref-1EaLRFk21" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2039. </div><div class="csl-right-inline">Sabatello M, Burke TB, McDonald KE, Appelbaum PS. 2020. <a href="https://doi.org/10.2105/ajph.2020.305837">Disability, Ethics, and Health Care in the COVID-19 Pandemic</a>. American Journal of Public Health 110:1523–1527.</div>
</div>
<div id="ref-cPsuWT7U" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2040. </div><div class="csl-right-inline">Wicclair M. 2020. <a href="https://doi.org/10.1080/15265161.2020.1777347">Allocating Ventilators During the COVID-19 Pandemic and Conscientious Objection</a>. The American Journal of Bioethics 20:204–207.</div>
</div>
<div id="ref-1D1FX4zQZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2041. </div><div class="csl-right-inline">Williams JC, Anderson N, Mathis M, Sanford E, Eugene J, Isom J. 2020. <a href="https://doi.org/10.1016/j.jnma.2020.05.010">Colorblind Algorithms: Racism in the Era of COVID-19</a>. Journal of the National Medical Association 112:550–552.</div>
</div>
<div id="ref-LrqhFVIz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2042. </div><div class="csl-right-inline">Egede LE, Walker RJ. 2020. <a href="https://doi.org/10.1056/nejmp2023616">Structural Racism, Social Risk Factors, and Covid-19 — A Dangerous Convergence for Black Americans</a>. New England Journal of Medicine 383:e77.</div>
</div>
<div id="ref-PFHCN5ep" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2043. </div><div class="csl-right-inline">Weber E, Bliton MJ. 2020. <a href="https://doi.org/10.1080/15265161.2020.1795538">Allocating Remdesivir Under Scarcity: Social Justice or More Systemic Racism</a>. The American Journal of Bioethics 20:31–33.</div>
</div>
<div id="ref-1ExRX1x4C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2044. </div><div class="csl-right-inline">Ballantyne A, Rogers WA, Entwistle V, Towns C. 2020. <a href="https://doi.org/10.1136/medethics-2020-106460">Revisiting the equity debate in COVID-19: ICU is no panacea</a>. Journal of Medical Ethics 46:641–645.</div>
</div>
<div id="ref-ztTunmqF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2045. </div><div class="csl-right-inline">Fink S. 2020. <a href="https://doi.org/10.1080/15265161.2020.1788663">Ethical Dilemmas in Covid-19 Medical Care: Is a Problematic Triage Protocol Better or Worse than No Protocol at All?</a> The American Journal of Bioethics 20:1–5.</div>
</div>
<div id="ref-18et5lcBc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2046. </div><div class="csl-right-inline">Lim S, DeBruin DA, Leider JP, Sederstrom N, Lynfield R, Baker JV, Kline S, Kesler S, Rizza S, Wu J, Sharp RR, Wolf SM. 2020. <a href="https://doi.org/10.1016/j.mayocp.2020.06.016">Developing an Ethics Framework for Allocating Remdesivir in the COVID-19 Pandemic</a>. Mayo Clinic Proceedings 95:1946–1954.</div>
</div>
<div id="ref-11gslRJJU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2047. </div><div class="csl-right-inline">Webb J, Shah LD, Lynch HF. 2020. <a href="https://doi.org/10.1080/15265161.2020.1795529">Ethically Allocating COVID-19 Drugs Via Pre-approval Access and Emergency Use Authorization</a>. The American Journal of Bioethics 20:4–17.</div>
</div>
<div id="ref-ugEOYgfx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2048. </div><div class="csl-right-inline">2011. <a href="https://www.cdc.gov/about/advisory/pdf/VentDocument_Release.pdf">https://www.cdc.gov/about/advisory/pdf/VentDocument_Release.pdf</a>.</div>
</div>
<div id="ref-1A5E2TkBS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2049. </div><div class="csl-right-inline">Jefferson AA. 2020. <a href="https://doi.org/10.1080/15265161.2020.1795541">Adopting an Anti-Racist Model of COVID-19 Drug Allocation and Prioritization</a>. The American Journal of Bioethics 20:33–36.</div>
</div>
<div id="ref-TFKtmAIg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2050. </div><div class="csl-right-inline">Coleman CH. 2020. <a href="https://doi.org/10.1002/eahr.500055">Equitably Sharing the Benefits and Burdens of Research: Covid‐19 Raises the Stakes</a>. Ethics &amp; Human Research 42:38–40.</div>
</div>
<div id="ref-MYW3TeXJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2051. </div><div class="csl-right-inline">Yamey G, Schäferhoff M, Hatchett R, Pate M, Zhao F, McDade KK. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30763-7">Ensuring global access to COVID-19 vaccines</a>. The Lancet 395:1405–1406.</div>
</div>
<div id="ref-rh28K8GB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2052. </div><div class="csl-right-inline">Bollyky TJ, Gostin LO, Hamburg MA. 2020. <a href="https://doi.org/10.1001/jama.2020.6641">The Equitable Distribution of COVID-19 Therapeutics and Vaccines</a>. JAMA 323:2462.</div>
</div>
<div id="ref-RZiEc6kh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2053. </div><div class="csl-right-inline">Baquet CR, Commiskey P, Daniel Mullins C, Mishra SI. 2006. <a href="https://doi.org/10.1016/j.cdp.2005.12.001">Recruitment and participation in clinical trials: Socio-demographic, rural/urban, and health care access predictors</a>. Cancer Detection and Prevention 30:24–33.</div>
</div>
<div id="ref-BLKtWt4w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2054. </div><div class="csl-right-inline">Makoni M. 2020. <a href="https://doi.org/10.1016/s2213-2600(20)30401-x">COVID-19 vaccine trials in Africa</a>. The Lancet Respiratory Medicine 8:e79–e80.</div>
</div>
<div id="ref-FasLN7DR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2055. </div><div class="csl-right-inline">Langford AT, Resnicow K, Dimond EP, Denicoff AM, Germain DSt, McCaskill-Stevens W, Enos RA, Carrigan A, Wilkinson K, Go RS. 2014. <a href="https://doi.org/10.1002/cncr.28483">Racial/ethnic differences in clinical trial enrollment, refusal rates, ineligibility, and reasons for decline among patients at sites in the National Cancer Institute's Community Cancer Centers Program</a>. Cancer 120:877–884.</div>
</div>
<div id="ref-14jpPBgHd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2056. </div><div class="csl-right-inline">Murthy VH, Krumholz HM, Gross CP. 2004. <a href="https://doi.org/10.1001/jama.291.22.2720">Participation in Cancer Clinical Trials</a>. JAMA 291:2720.</div>
</div>
<div id="ref-GNMGw3Ff" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2057. </div><div class="csl-right-inline">Stewart JH, Bertoni AG, Staten JL, Levine EA, Gross CP. 2007. <a href="https://doi.org/10.1245/s10434-007-9500-y">Participation in Surgical Oncology Clinical Trials: Gender-, Race/Ethnicity-, and Age-based Disparities</a>. Annals of Surgical Oncology 14:3328–3334.</div>
</div>
<div id="ref-ZdYcndGw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2058. </div><div class="csl-right-inline">Geller SE, Koch A, Pellettieri B, Carnes M. 2011. <a href="https://doi.org/10.1089/jwh.2010.2469">Inclusion, Analysis, and Reporting of Sex and Race/Ethnicity in Clinical Trials: Have We Made Progress?</a> Journal of Women's Health 20:315–320.</div>
</div>
<div id="ref-QNqutuqe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2059. </div><div class="csl-right-inline">Falasinnu T, Chaichian Y, Bass MB, Simard JF. 2018. <a href="https://doi.org/10.1007/s11926-018-0728-2">The Representation of Gender and Race/Ethnic Groups in Randomized Clinical Trials of Individuals with Systemic Lupus Erythematosus</a>. Current Rheumatology Reports 20:20.</div>
</div>
<div id="ref-2DrMhVAo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2060. </div><div class="csl-right-inline">Chastain DB, Osae SP, Henao-Martínez AF, Franco-Paredes C, Chastain JS, Young HN. 2020. <a href="https://doi.org/10.1056/nejmp2021971">Racial Disproportionality in Covid Clinical Trials</a>. New England Journal of Medicine 383:e59.</div>
</div>
<div id="ref-M7sLngTJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2061. </div><div class="csl-right-inline">Johnson JL, Bottorff JL, Browne AJ, Grewal S, Hilton BA, Clarke H. 2004. <a href="https://doi.org/10.1207/s15327027hc1602_7">Othering and Being Othered in the Context of Health Care Services</a>. Health Communication 16:255–271.</div>
</div>
<div id="ref-sDygnf7I" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2062. </div><div class="csl-right-inline">Maina IW, Belton TD, Ginzberg S, Singh A, Johnson TJ. 2018. <a href="https://doi.org/10.1016/j.socscimed.2017.05.009">A decade of studying implicit racial/ethnic bias in healthcare providers using the implicit association test</a>. Social Science &amp; Medicine 199:219–229.</div>
</div>
<div id="ref-upbPnFPI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2063. </div><div class="csl-right-inline">Dehon E, Weiss N, Jones J, Faulconer W, Hinton E, Sterling S. 2017. <a href="https://doi.org/10.1111/acem.13214">A Systematic Review of the Impact of Physician Implicit Racial Bias on Clinical Decision Making</a>. Academic Emergency Medicine 24:895–904.</div>
</div>
<div id="ref-AaFVjNFc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2064. </div><div class="csl-right-inline">Penner LA, Dovidio JF, West TV, Gaertner SL, Albrecht TL, Dailey RK, Markova T. 2010. <a href="https://doi.org/10.1016/j.jesp.2009.11.004">Aversive racism and medical interactions with Black patients: A field study</a>. Journal of Experimental Social Psychology 46:436–440.</div>
</div>
<div id="ref-smVwFlNn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2065. </div><div class="csl-right-inline">Johnson TJ. 2020. <a href="https://doi.org/10.1542/peds.2020-003657">Intersection of Bias, Structural Racism, and Social Determinants With Health Care Inequities</a>. Pediatrics 146:e2020003657.</div>
</div>
<div id="ref-Vp7KObiA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2066. </div><div class="csl-right-inline">Feldman WB, Hey SP, Kesselheim AS. 2018. <a href="https://doi.org/10.1377/hlthaff.2018.0501">A Systematic Review Of The Food And Drug Administration’s ‘Exception From Informed Consent’ Pathway</a>. Health Affairs 37:1605–1614.</div>
</div>
<div id="ref-JNha7DEl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2067. </div><div class="csl-right-inline">Schmidt TA. 2003. <a href="https://doi.org/10.1067/mem.2003.17">The legacy of the tuskegee syphilis experiments for emergency exception from informed consent</a>. Annals of Emergency Medicine 41:79–81.</div>
</div>
<div id="ref-i53Z5Ye8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2068. </div><div class="csl-right-inline">Al-Arshani S. CDC officials are considering a plan to distribute COVID-19 vaccines to the most vulnerable first — including people of color. Business Insider. <a href="https://www.businessinsider.com/cdc-official-considering-giving-covid-19-vaccine-most-vulnerable-first-2020-10">https://www.businessinsider.com/cdc-official-considering-giving-covid-19-vaccine-most-vulnerable-first-2020-10</a>. Retrieved 8 February 2021.</div>
</div>
<div id="ref-GP85Y1Jo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2069. </div><div class="csl-right-inline">Federico MJ, Covar RA, Brown EE, Leung DYM, Spahn JD. 2005. <a href="https://doi.org/10.1378/chest.127.2.571">Racial Differences in T-Lymphocyte Response to Glucocorticoids</a>. Chest 127:571–578.</div>
</div>
<div id="ref-gkEMgdku" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2070. </div><div class="csl-right-inline">Charmandari E, Tsigos C, Chrousos G. 2005. <a href="https://doi.org/10.1146/annurev.physiol.67.040403.120816">ENDOCRINOLOGY OF THE STRESS RESPONSE</a>. Annual Review of Physiology 67:259–284.</div>
</div>
<div id="ref-BlEZu684" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2071. </div><div class="csl-right-inline">Cohen S, Janicki-Deverts D, Doyle WJ, Miller GE, Frank E, Rabin BS, Turner RB. 2012. <a href="https://doi.org/10.1073/pnas.1118355109">Chronic stress, glucocorticoid receptor resistance, inflammation, and disease risk</a>. Proceedings of the National Academy of Sciences 109:5995–5999.</div>
</div>
<div id="ref-7ub6VM4Z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2072. </div><div class="csl-right-inline">Vlasschaert C, Topf JM, Hiremath S. 2020. <a href="https://doi.org/10.1053/j.ackd.2020.08.003">Proliferation of Papers and Preprints During the Coronavirus Disease 2019 Pandemic: Progress or Problems With Peer Review?</a> Advances in Chronic Kidney Disease 27:418–426.</div>
</div>
<div id="ref-nnfOazAC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2073. </div><div class="csl-right-inline">Zarocostas J. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30461-x">How to fight an infodemic</a>. The Lancet 395:676.</div>
</div>
<div id="ref-CiOwklc6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2074. </div><div class="csl-right-inline">Wang LL, Lo K, Chandrasekhar Y, Reas R, Yang J, Burdick D, Eide D, Funk K, Katsis Y, Kinney R, Li Y, Liu Z, Merrill W, Mooney P, Murdick D, Rishi D, Sheehan J, Shen Z, Stilson B, Wade A, Wang K, Wang NXR, Wilhelm C, Xie B, Raymond D, Weld DS, Etzioni O, Kohlmeier S. 2020. <a href="https://arxiv.org/abs/2004.10706">CORD-19: The COVID-19 Open Research Dataset</a>. 2004.10706arXiv. arXiv.</div>
</div>
<div id="ref-Ybg667S0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2075. </div><div class="csl-right-inline">Lever J, Altman RB. 2021. <a href="https://doi.org/10.1073/pnas.2100766118">Analyzing the vast coronavirus literature with CoronaCentral</a>. Proc Natl Acad Sci USA 118.</div>
</div>
<div id="ref-DfSr1Ohc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2076. </div><div class="csl-right-inline">Eysenbach G. 2000. <a href="https://doi.org/10.1016/s0952-7915(00)00127-8">The impact of preprint servers and electronic publishing on biomedical research</a>. Current Opinion in Immunology 12:499–503.</div>
</div>
<div id="ref-ZUk10707" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2077. </div><div class="csl-right-inline">Yeo-Teh NSL, Tang BL. 2020. <a href="https://doi.org/10.1080/08989621.2020.1782203">An alarming retraction rate for scientific publications on Coronavirus Disease 2019 (COVID-19)</a>. Accountability in Research 28:47–53.</div>
</div>
<div id="ref-caxpZEmy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2078. </div><div class="csl-right-inline">Abritis A, Marcus A, Oransky I. 2020. <a href="https://doi.org/10.1080/08989621.2020.1793675">An “alarming” and “exceptionally high” rate of COVID-19 retractions?</a> Accountability in Research 28:58–59.</div>
</div>
<div id="ref-1Dez1ZOc5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2079. </div><div class="csl-right-inline">Agoramoorthy G, Hsu MJ, Shieh P. 2020. <a href="https://doi.org/10.1111/bioe.12772">Queries on the COVID‐19 quick publishing ethics</a>. Bioethics 34:633–634.</div>
</div>
<div id="ref-JdWiPJCL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2080. </div><div class="csl-right-inline">Boodman C, Lee S, Bullard J. 2020. <a href="https://doi.org/10.1080/10872981.2020.1770562">Idle medical students review emerging COVID-19 research</a>. Medical Education Online 25.</div>
</div>
<div id="ref-1CBWvhTdy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2081. </div><div class="csl-right-inline">Brainard J. 2020. Scientists are drowning in COVID-19 papers. Can new tools keep them afloat? Science https://doi.org/<a href="https://doi.org/10.1126/science.abc7839">10.1126/science.abc7839</a>.</div>
</div>
<div id="ref-YZ4cHNuH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2082. </div><div class="csl-right-inline">Vabret N, Samstein R, Fernandez N, Merad M, The Sinai Immunology Review Project, Trainees, Faculty. 2020. <a href="https://doi.org/10.1038/s41577-020-0319-0">Advancing scientific knowledge in times of pandemics</a>. Nature Reviews Immunology 20:338–338.</div>
</div>
<div id="ref-I2EsJmfs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2083. </div><div class="csl-right-inline">Sun J, He W-T, Wang L, Lai A, Ji X, Zhai X, Li G, Suchard MA, Tian J, Zhou J, Veit M, Su S. 2020. <a href="https://doi.org/10.1016/j.molmed.2020.02.008">COVID-19: Epidemiology, Evolution, and Cross-Disciplinary Perspectives</a>. Trends in Molecular Medicine 26:483–495.</div>
</div>
<div id="ref-evtsR3C5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2084. </div><div class="csl-right-inline">Weissleder R, Lee H, Ko J, Pittet MJ. 2020. <a href="https://doi.org/10.1126/scitranslmed.abc1931">COVID-19 diagnostics in context</a>. Science Translational Medicine 12:eabc1931.</div>
</div>
<div id="ref-18eCxyLhx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2085. </div><div class="csl-right-inline">Sanders JM, Monogue ML, Jodlowski TZ, Cutrell JB. 2020. Pharmacologic Treatments for Coronavirus Disease 2019 (COVID-19). JAMA https://doi.org/<a href="https://doi.org/10.1001/jama.2020.6019">10.1001/jama.2020.6019</a>.</div>
</div>
<div id="ref-SAE5ME3N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2086. </div><div class="csl-right-inline">Carvalho T. 2020. <a href="https://doi.org/10.1038/d41591-020-00026-w">COVID-19 Research in Brief: December, 2019 to June, 2020</a>. Nature Medicine 26:1152–1153.</div>
</div>
<div id="ref-xOs5ctsW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2087. </div><div class="csl-right-inline">Wiersinga WJ, Rhodes A, Cheng AC, Peacock SJ, Prescott HC. 2020. <a href="https://doi.org/10.1001/jama.2020.12839">Pathophysiology, Transmission, Diagnosis, and Treatment of Coronavirus Disease 2019 (COVID-19)</a>. JAMA 324:782.</div>
</div>
<div id="ref-YuJbg3zO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2088. </div><div class="csl-right-inline">Himmelstein DS, Rubinetti V, Slochower DR, Hu D, Malladi VS, Greene CS, Gitter A. 2019. <a href="https://doi.org/10.1371/journal.pcbi.1007128">Open collaborative writing with Manubot</a>. PLOS Computational Biology 15:e1007128.</div>
</div>
<div id="ref-PoDz2q0A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2089. </div><div class="csl-right-inline">Tennant JP, Bielczyk N, Tzovaras BG, Masuzzo P, Steiner T. 2020. <a href="https://doi.org/10.5334/kula.63">Introducing Massively Open Online Papers (MOOPs)</a>. KULA 4:1.</div>
</div>
<div id="ref-1HZeeO4Cs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2090. </div><div class="csl-right-inline">Gog JR. 2020. <a href="https://doi.org/10.1038/s42254-020-0175-7">How you can help with COVID-19 modelling</a>. Nature Reviews Physics 2:274–275.</div>
</div>
<div id="ref-1HmO21gZN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2091. </div><div class="csl-right-inline">Bell J, LaToza TD, Baldmitsi F, Stavrou A. 2017. <a href="https://doi.org/10.1109/se4science.2017.11">Advancing Open Science with Version Control and Blockchains</a>2017 IEEE/ACM 12th International Workshop on Software Engineering for Science (SE4Science). IEEE.</div>
</div>
<div id="ref-OO1DuZd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2092. </div><div class="csl-right-inline">Vuorre M, Curley JP. 2018. <a href="https://doi.org/10.1177/2515245918754826">Curating Research Assets: A Tutorial on the Git Version Control System</a>. Advances in Methods and Practices in Psychological Science 1:219–236.</div>
</div>
<div id="ref-4ny1onB0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2093. </div><div class="csl-right-inline">Ram K. 2013. <a href="https://doi.org/10.1186/1751-0473-8-7">Git can facilitate greater reproducibility and increased transparency in science</a>. Source Code Biol Med 8.</div>
</div>
<div id="ref-17OQtAY4l" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2094. </div><div class="csl-right-inline">Corcoran M. 2006. <a href="https://doi.org/10.5014/ajot.60.4.367">Using the MAARIE Framework To Read the Research Literature</a>. American Journal of Occupational Therapy 60:367–368.</div>
</div>
<div id="ref-k1GJLUxP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2095. </div><div class="csl-right-inline">Riegelman RK. 2013. Studying a study &amp; testing a test: reading evidence-based health research6th ed. Wolters Kluwer/Lippincott Williams &amp; Wilkins Heath, Philadelphia.</div>
</div>
<div id="ref-1026Gxdsi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2096. </div><div class="csl-right-inline">Hunter JD. 2007. <a href="https://doi.org/10.1109/mcse.2007.55">Matplotlib: A 2D Graphics Environment</a>. Computing in Science &amp; Engineering 9:90–95.</div>
</div>
<div id="ref-14UJbLWf4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2097. </div><div class="csl-right-inline">Nicholson JM, Mordaunt M, Lopez P, Uppala A, Rosati D, Rodrigues NP, Grabitz P, Rife SC. 2021. <a href="https://doi.org/10.1101/2021.03.15.435418">scite: a smart citation index that displays the context of citations and classifies their intent using deep learning</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-AE0QcVgJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2098. </div><div class="csl-right-inline">Perkel JM. 2020. <a href="https://doi.org/10.1038/d41586-020-00916-6">Synchronized editing: the future of collaborative writing</a>. Nature 580:154–155.</div>
</div>
<div id="ref-ZHvhFakW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2099. </div><div class="csl-right-inline">Roser M, Ritchie H, Ortiz-Ospina E, Hasell J. 2020. <a href="https://ourworldindata.org/coronavirus">Coronavirus pandemic (COVID-19)</a>. Our World in Data.</div>
</div>
<div id="ref-pESBLU4c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2100. </div><div class="csl-right-inline">Nicholson DN, Rubinetti V, Hu D, Thielk M, Hunter LE, Greene CS. 2021. <a href="https://doi.org/10.1101/2021.03.04.433874">Linguistic Analysis of the bioRxiv Preprint Landscape</a>. Cold Spring Harbor Laboratory.</div>
</div>
<div id="ref-1HLSeqnY8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2101. </div><div class="csl-right-inline">Tian X, Li C, Huang A, Xia S, Lu S, Shi Z, Lu L, Jiang S, Yang Z, Wu Y, Ying T. 2020. Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.01.28.923011">10.1101/2020.01.28.923011</a>.</div>
</div>
<div id="ref-lOQPScZt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2102. </div><div class="csl-right-inline">He X, Zhang L, Ran Q, Wang J, Xiong A, Wu D, Chen F, Li G. 2020. Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.03.20020206">10.1101/2020.02.03.20020206</a>.</div>
</div>
<div id="ref-rL9IRhVy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2103. </div><div class="csl-right-inline">Liang W, Feng Z, Rao S, Xiao C, Lin Z-X, Zhang Q, Wei Q. 2020. Diarrhea may be underestimated: a missing link in 2019 novel coronavirus. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.03.20020289">10.1101/2020.02.03.20020289</a>.</div>
</div>
<div id="ref-BHQvMYPG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2104. </div><div class="csl-right-inline">Chai X, Hu L, Zhang Y, Han W, Lu Z, Ke A, Zhou J, Shi G, Fang N, Fan J, Cai J, Fan J, Lan F. 2020. Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.03.931766">10.1101/2020.02.03.931766</a>.</div>
</div>
<div id="ref-OuDMtCRl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2105. </div><div class="csl-right-inline">Zhao B, Ni C, Gao R, Wang Y, Yang L, Wei J, Lv T, Liang J, Zhang Q, Xu W, Xie Y, Wang X, Yuan Z, Liang J, Zhang R, Lin X. 2020. Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.16.990317">10.1101/2020.03.16.990317</a>.</div>
</div>
<div id="ref-ViWCb8MJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2106. </div><div class="csl-right-inline">Wang J, Zhao S, Liu M, Zhao Z, Xu Y, Wang P, Lin M, Xu Y, Huang B, Zuo X, Chen Z, Bai F, Cui J, Lew AM, Zhao J, Zhang Y, Luo H-B, Zhang Y. 2020. ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.05.20020545">10.1101/2020.02.05.20020545</a>.</div>
</div>
<div id="ref-MgLBA5m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2107. </div><div class="csl-right-inline">Bao L, Deng W, Huang B, Gao H, Liu J, Ren L, Wei Q, Yu P, Xu Y, Qi F, Qu Y, Li F, Lv Q, Wang W, Xue J, Gong S, Liu M, Wang G, Wang S, Song Z, Zhao L, Liu P, Zhao L, Ye F, Wang H, Zhou W, Zhu N, Zhen W, Yu H, Zhang X, Guo L, Chen L, Wang C, Wang Y, Wang X, Xiao Y, Sun Q, Liu H, Zhu F, Ma C, Yan L, Yang M, Han J, Xu W, Tan W, Peng X, Jin Q, Wu G, Qin C. 2020. The Pathogenicity of SARS-CoV-2 in hACE2 Transgenic Mice. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.07.939389">10.1101/2020.02.07.939389</a>.</div>
</div>
<div id="ref-1Acv5UMSt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2108. </div><div class="csl-right-inline">Li Z, Wu M, Yao J, Guo J, Liao X, Song S, Li J, Duan G, Zhou Y, Wu X, Zhou Z, Wang T, Hu M, Chen X, Fu Y, Lei C, Dong H, Xu C, Hu Y, Han M, Zhou Y, Jia H, Chen X, Yan J. 2020. Caution on Kidney Dysfunctions of COVID-19 Patients. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.08.20021212">10.1101/2020.02.08.20021212</a>.</div>
</div>
<div id="ref-DMaiZBE1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2109. </div><div class="csl-right-inline">Chu KH, Tsang WK, Tang CS, Lam MF, Lai FM, To KF, Fung KS, Tang HL, Yan WW, Chan HWH, Lai TST, Tong KL, Lai KN. 2005. <a href="https://doi.org/10.1111/j.1523-1755.2005.67130.x">Acute renal impairment in coronavirus-associated severe acute respiratory syndrome</a>. Kidney International 67:698–705.</div>
</div>
<div id="ref-16Xj27QYj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2110. </div><div class="csl-right-inline">Lin W, Hu L, Zhang Y, Ooi JD, Meng T, Jin P, Ding X, Peng L, Song L, Xiao Z, Ao X, Xiao X, Zhou Q, Xiao P, Fan J, Zhong Y. 2020. Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.08.939892">10.1101/2020.02.08.939892</a>.</div>
</div>
<div id="ref-QhE5mVCc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2111. </div><div class="csl-right-inline">Zhu J, Kim J, Xiao X, Wang Y, Luo D, Jiang S, Chen R, Xu L, Zhang H, Moise L, Gutierrez AH, De Groot AS, Xiao G, Schoggins JW, Zhan X, Wang T, Xie Y. 2020. The immune vulnerability landscape of the 2019 Novel Coronavirus, SARS-CoV-2. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.08.939553">10.1101/2020.02.08.939553</a>.</div>
</div>
<div id="ref-iObQU2el" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2112. </div><div class="csl-right-inline">Arabi YM, Arifi AA, Balkhy HH, Najm H, Aldawood AS, Ghabashi A, Hawa H, Alothman A, Khaldi A, Al Raiy B. 2014. <a href="https://doi.org/10.7326/m13-2486">Clinical Course and Outcomes of Critically Ill Patients With Middle East Respiratory Syndrome Coronavirus Infection</a>. Annals of Internal Medicine 160:389–397.</div>
</div>
<div id="ref-gK5e7Yhl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2113. </div><div class="csl-right-inline">Liu J, Liu Y, Xiang P, Pu L, Xiong H, Li C, Zhang M, Tan J, Xu Y, Song R, Song M, Wang L, Zhang W, Han B, Yang L, Wang X, Zhou G, Zhang T, Li B, Wang Y, Chen Z, Wang X. 2020. Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.10.20021584">10.1101/2020.02.10.20021584</a>.</div>
</div>
<div id="ref-6Y81C4P3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2114. </div><div class="csl-right-inline">Qin C, Zhou L, Hu Z, Zhang S, Yang S, Tao Y, Xie C, Ma K, Shang K, Wang W, Tian D-S. 2020. <a href="https://doi.org/10.1093/cid/ciaa248">Dysregulation of Immune Response in Patients With Coronavirus 2019 (COVID-19) in Wuhan, China</a>. Clinical Infectious Diseases 71:762–768.</div>
</div>
<div id="ref-P4assQHj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2115. </div><div class="csl-right-inline">Wan S, Yi Q, Fan S, Lv J, Zhang X, Guo L, Lang C, Xiao Q, Xiao K, Yi Z, Qiang M, Xiang J, Zhang B, Chen Y. 2020. Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP). Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.10.20021832">10.1101/2020.02.10.20021832</a>.</div>
</div>
<div id="ref-zskgSnle" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2116. </div><div class="csl-right-inline">Liu J, Li S, Liu J, Liang B, Wang X, Li W, Wang H, Tong Q, Yi J, Zhao L, Xiong L, Guo C, Tian J, Luo J, Yao J, Pang R, Shen H, Peng C, Liu T, Zhang Q, Wu J, Xu L, Lu S, Wang B, Weng Z, Han C, Zhu H, Zhou R, Zhou H, Chen X, Ye P, Zhu B, He S, He Y, Jie S, Wei P, Zhang J, Lu Y, Wang W, Zhang L, Li L, Zhou F, Wang J, Dittmer U, Lu M, Hu Y, Yang D, Zheng X. 2020. Longitudinal Characteristics of Lymphocyte Responses and Cytokine Profiles in the Peripheral Blood of SARS-CoV-2 Infected Patients. SSRN Electronic Journal https://doi.org/<a href="https://doi.org/10.2139/ssrn.3539682">10.2139/ssrn.3539682</a>.</div>
</div>
<div id="ref-TXlNzmXd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2117. </div><div class="csl-right-inline">Li J, Li S, Cai Y, Liu Q, Li X, Zeng Z, Chu Y, Zhu F, Zeng F. 2020. Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.11.20022053">10.1101/2020.02.11.20022053</a>.</div>
</div>
<div id="ref-15K5Ovntj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2118. </div><div class="csl-right-inline">Fan C, Li K, Ding Y, Lu W, Wang J. 2020. ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.12.20022418">10.1101/2020.02.12.20022418</a>.</div>
</div>
<div id="ref-Bgo7Cmj3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2119. </div><div class="csl-right-inline">Zhou Y, Fu B, Zheng X, Wang D, Zhao C, qi Y, Sun R, Tian Z, Xu X, Wei H. 2020. Aberrant pathogenic GM-CSF &lt;sup&gt;+&lt;/sup&gt; T cells and inflammatory CD14 &lt;sup&gt;+&lt;/sup&gt; CD16 &lt;sup&gt;+&lt;/sup&gt; monocytes in severe pulmonary syndrome patients of a new coronavirus. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.12.945576">10.1101/2020.02.12.945576</a>.</div>
</div>
<div id="ref-4Z1pEioX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2120. </div><div class="csl-right-inline">Qian K, Deng Y, Tai Y-H, Peng J, Peng H, Jiang L-H. 2020. Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia: A Systemic Review and Meta-analysis. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.14.20021535">10.1101/2020.02.14.20021535</a>.</div>
</div>
<div id="ref-zTGzuiTI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2121. </div><div class="csl-right-inline">Liu J, Li S, Liu J, Liang B, Wang X, Wang H, Li W, Tong Q, Yi J, Zhao L, Xiong L, Guo C, Tian J, Luo J, Yao J, Pang R, Shen H, Peng C, Liu T, Zhang Q, Wu J, Xu L, Lu S, Wang B, Weng Z, Han C, Zhu H, Zhou R, Zhou H, Chen X, Ye P, Zhu B, He S, He Y, Jie S, Wei P, Zhang J, Lu Y, Wang W, Zhang L, Li L, Zhou F, Wang J, Dittmer U, Lu M, Hu Y, Yang D, Zheng X. 2020. Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.16.20023671">10.1101/2020.02.16.20023671</a>.</div>
</div>
<div id="ref-1EIEx8Qe4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2122. </div><div class="csl-right-inline">Chen G, Wu D, Guo W, Cao Y, Huang D, Wang H, Wang T, Zhang X, Chen H, Yu H, Zhang X, Zhang M, Wu S, Song J, Chen T, Han M, Li S, Luo X, Zhao J, Ning Q. 2020. Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.16.20023903">10.1101/2020.02.16.20023903</a>.</div>
</div>
<div id="ref-NPxs7bbq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2123. </div><div class="csl-right-inline">Chen Y, Wei Q, Li R, Gao H, Zhu H, Deng W, Bao L, Tong W, Cong Z, Jiang H, Qin C. 2020. Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.17.951939">10.1101/2020.02.17.951939</a>.</div>
</div>
<div id="ref-f50ahNxF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2124. </div><div class="csl-right-inline">Diao B, Wang C, Tan Y, Chen X, Liu Y, Ning L, Chen L, Li M, Liu Y, Wang G, Yuan Z, Feng Z, Wu Y, Chen Y. 2020. Reduction and Functional Exhaustion of T Cells in Patients with Coronavirus Disease 2019 (COVID-19). Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.18.20024364">10.1101/2020.02.18.20024364</a>.</div>
</div>
<div id="ref-EwNi3WJq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2125. </div><div class="csl-right-inline">Li X, Wang L, Yan S, Yang F, Xiang L, Zhu J, Shen B, Gong Z. 2020. Clinical characteristics of 25 death cases with COVID-19: a retrospective review of medical records in a single medical center, Wuhan, China. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.19.20025239">10.1101/2020.02.19.20025239</a>.</div>
</div>
<div id="ref-KG5Lg65z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2126. </div><div class="csl-right-inline">Wang L, Li X, Chen H, Yan S, Li Y, Li D, Gong Z. 2020. SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.19.20025288">10.1101/2020.02.19.20025288</a>.</div>
</div>
<div id="ref-VRoblZFJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2127. </div><div class="csl-right-inline">Wang D, Hu B, Hu C, Zhu F, Liu X, Zhang J, Wang B, Xiang H, Cheng Z, Xiong Y, Zhao Y, Li Y, Wang X, Peng Z. 2020. <a href="https://doi.org/10.1001/jama.2020.1585">Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus–Infected Pneumonia in Wuhan, China</a>. JAMA 323:1061.</div>
</div>
<div id="ref-bgrzawWp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2128. </div><div class="csl-right-inline">Guan W, Ni Z, Hu Y, Liang W, Ou C, He J, Liu L, Shan H, Lei C, Hui DSC, Du B, Li L, Zeng G, Yuen K-Y, Chen R, Tang C, Wang T, Chen P, Xiang J, Li S, Wang J, Liang Z, Peng Y, Wei L, Liu Y, Hu Y, Peng P, Wang J, Liu J, Chen Z, Li G, Zheng Z, Qiu S, Luo J, Ye C, Zhu S, Zhong N. 2020. Clinical characteristics of 2019 novel coronavirus infection in China. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.06.20020974">10.1101/2020.02.06.20020974</a>.</div>
</div>
<div id="ref-hdSafq1o" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2129. </div><div class="csl-right-inline">Fast E, Altman RB, Chen B. 2020. Potential T-cell and B-cell Epitopes of 2019-nCoV. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.19.955484">10.1101/2020.02.19.955484</a>.</div>
</div>
<div id="ref-8xnxgpag" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2130. </div><div class="csl-right-inline">Walls AC, Park Y-J, Tortorici MA, Wall A, McGuire AT, Veesler D. 2020. Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.19.956581">10.1101/2020.02.19.956581</a>.</div>
</div>
<div id="ref-QdimT8mN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2131. </div><div class="csl-right-inline">Thevarajan I, Nguyen TH, Koutsakos M, Druce J, Caly L, van de Sandt CE, Jia X, Nicholson S, Catton M, Cowie B, Tong SY, Lewin SR, Kedzierska K. 2020. Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.20.20025841">10.1101/2020.02.20.20025841</a>.</div>
</div>
<div id="ref-tVkKuIJt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2132. </div><div class="csl-right-inline">Liao M, Liu Y, Yuan J, Wen Y, Xu G, Zhao J, Chen L, Li J, Wang X, Wang F, Liu L, Zhang S, Zhang Z. 2020. The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.23.20026690">10.1101/2020.02.23.20026690</a>.</div>
</div>
<div id="ref-87CJN2CG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2133. </div><div class="csl-right-inline">Scharenberg M, Vangeti S, Kekäläinen E, Bergman P, Al-Ameri M, Johansson N, Sondén K, Falck-Jones S, Färnert A, Ljunggren H-G, Michaëlsson J, Smed-Sörensen A, Marquardt N. 2019. <a href="https://doi.org/10.3389/fimmu.2019.01116">Influenza A Virus Infection Induces Hyperresponsiveness in Human Lung Tissue-Resident and Peripheral Blood NK Cells</a>. Frontiers in Immunology 10:1116.</div>
</div>
<div id="ref-yJW1RVQb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2134. </div><div class="csl-right-inline">Amanat F, Stadlbauer D, Strohmeier S, Nguyen THO, Chromikova V, McMahon M, Jiang K, Arunkumar GA, Jurczyszak D, Polanco J, Bermudez-Gonzalez M, Kleiner G, Aydillo T, Miorin L, Fierer D, Lugo LA, Kojic EM, Stoever J, Liu STH, Cunningham-Rundles C, Felgner PL, Moran T, Garcia-Sastre A, Caplivski D, Cheng A, Kedzierska K, Vapalahti O, Hepojoki JM, Simon V, Krammer F. 2020. A serological assay to detect SARS-CoV-2 seroconversion in humans. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.17.20037713">10.1101/2020.03.17.20037713</a>.</div>
</div>
<div id="ref-vasHgjGH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2135. </div><div class="csl-right-inline">Huang C, Wang Y, Li X, Ren L, Zhao J, Hu Y, Zhang L, Fan G, Xu J, Gu X, Cheng Z, Yu T, Xia J, Wei Y, Wu W, Xie X, Yin W, Li H, Liu M, Xiao Y, Gao H, Guo L, Xie J, Wang G, Jiang R, Gao Z, Jin Q, Wang J, Cao B. 2020. <a href="https://doi.org/10.1016/s0140-6736(20)30183-5">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</a>. The Lancet 395:497–506.</div>
</div>
<div id="ref-9GEyqic3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2136. </div><div class="csl-right-inline">Allard B, Panariti A, Martin JG. 2018. <a href="https://doi.org/10.3389/fimmu.2018.01777">Alveolar Macrophages in the Resolution of Inflammation, Tissue Repair, and Tolerance to Infection</a>. Frontiers in Immunology 9:1777.</div>
</div>
<div id="ref-WBN5f37b" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2137. </div><div class="csl-right-inline">Huang S, Zhu B, Cheon IS, Goplen NP, Jiang L, Zhang R, Peebles RS, Mack M, Kaplan MH, Limper AH, Sun J. 2019. <a href="https://doi.org/10.1128/jvi.00030-19">PPAR-γ in Macrophages Limits Pulmonary Inflammation and Promotes Host Recovery following Respiratory Viral Infection</a>. Journal of Virology 93:e00030–19.</div>
</div>
<div id="ref-FOVDaThy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2138. </div><div class="csl-right-inline">Pan Y, Ye G, Zeng X, Liu G, Zeng X, Jiang X, Zhao J, Chen L, Guo S, Deng Q, Hong X, Yang Y, Li Y, Wang X. 2020. Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia? Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.25.20024711">10.1101/2020.02.25.20024711</a>.</div>
</div>
<div id="ref-1Du40IOOH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2139. </div><div class="csl-right-inline">Gong J, Dong H, Xia Q, Huang Z, Wang D, Zhao Y, Liu W, Tu S, Zhang M, Wang Q, Lu F. 2020. Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.25.20025643">10.1101/2020.02.25.20025643</a>.</div>
</div>
<div id="ref-1DTLMIDoM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2140. </div><div class="csl-right-inline">Herst C, Burkholz S, Sidney J, Sette A, Harris P, Massey S, Brasel T, Cunha-Neto E, Rosa D, Chao W, Carback R, Hodge T, Wang L, Ciotlos S, Lloyd P, Rubsamen R. 2020. An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.25.963546">10.1101/2020.02.25.963546</a>.</div>
</div>
<div id="ref-9spK49tt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2141. </div><div class="csl-right-inline">Li L, Sun T, He Y, Li W, Fan Y, Zhang J. 2020. Epitope-based peptide vaccine design and target site characterization against novel coronavirus disease caused by SARS-CoV-2. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.25.965434">10.1101/2020.02.25.965434</a>.</div>
</div>
<div id="ref-qg1bmQng" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2142. </div><div class="csl-right-inline">Wenjun W, Xiaoqing L, Sipei W, Puyi L, Liyan H, Yimin L, Linling C, Sibei C, Lingbo N, Yongping L, Jianxing H. 2020. The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.26.20026989">10.1101/2020.02.26.20026989</a>.</div>
</div>
<div id="ref-YODgfrtM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2143. </div><div class="csl-right-inline">Huang Y, Yang R, Xu Y, Gong P. 2020. Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.27.20029009">10.1101/2020.02.27.20029009</a>.</div>
</div>
<div id="ref-5CGbn9Jj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2144. </div><div class="csl-right-inline">Li L, Li S, Xu M, Yu P, Zheng S, Duan Z, Liu J, Chen Y, Li J. 2020. Risk factors related to hepatic injury in patients with corona virus disease 2019. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.28.20028514">10.1101/2020.02.28.20028514</a>.</div>
</div>
<div id="ref-T0BDaXkw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2145. </div><div class="csl-right-inline">Chen X, Zhao B, Qu Y, Chen Y, Xiong J, Feng Y, Men D, Huang Q, Liu Y, Yang B, Ding J, Li F. 2020. Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.29.20029520">10.1101/2020.02.29.20029520</a>.</div>
</div>
<div id="ref-l3CzJ5ni" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2146. </div><div class="csl-right-inline">Chen W, Ware LB. 2015. <a href="https://doi.org/10.1186/s40169-015-0065-2">Prognostic factors in the acute respiratory distress syndrome</a>. Clinical and Translational Medicine 4.</div>
</div>
<div id="ref-uIwTzZkJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2147. </div><div class="csl-right-inline">Tan L, Wang Q, Zhang D, Ding J, Huang Q, Tang Y-Q, Wang Q, Miao H. 2020. Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.01.20029074">10.1101/2020.03.01.20029074</a>.</div>
</div>
<div id="ref-1GMxHWnxN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2148. </div><div class="csl-right-inline">Liu T, Zhang J, Yang Y, Ma H, Li Z, Zhang J, Cheng J, Zhang X, Zhao Y, Xia Z, Zhang L, Wu G, Yi J. 2020. The potential role of IL-6 in monitoring severe case of coronavirus disease 2019. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.01.20029769">10.1101/2020.03.01.20029769</a>.</div>
</div>
<div id="ref-1B0vBwv1q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2149. </div><div class="csl-right-inline">Zhao Z, Xie J, Yin M, Yang Y, He H, Jin T, Li W, Zhu X, Xu J, Zhao C, Li L, Li Y, Mengist HM, Zahid A, Yao Z, Ding C, Qi Y, Gao Y, Ma X. 2020. Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.01.20029785">10.1101/2020.03.01.20029785</a>.</div>
</div>
<div id="ref-RyXNw87v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2150. </div><div class="csl-right-inline">Yang Y, Shen C, Li J, Yuan J, Yang M, Wang F, Li G, Li Y, Xing L, Peng L, Wei J, Cao M, Zheng H, Wu W, Zou R, Li D, Xu Z, Wang H, Zhang M, Zhang Z, Liu L, Liu Y. 2020. Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.02.20029975">10.1101/2020.03.02.20029975</a>.</div>
</div>
<div id="ref-5HXuCa5t" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2151. </div><div class="csl-right-inline">Zhao J, Yuan Q, Wang H, Liu W, Liao X, Su Y, Wang X, Yuan J, Li T, Li J, Qian S, Hong C, Wang F, Liu Y, Wang Z, He Q, Li Z, He B, Zhang T, Ge S, Liu L, Zhang J, Xia N, Zhang Z. 2020. Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.02.20030189">10.1101/2020.03.02.20030189</a>.</div>
</div>
<div id="ref-uCyoHtV3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2152. </div><div class="csl-right-inline">Chen X, Ling J, Mo P, Zhang Y, Jiang Q, Ma Z, Cao Q, Hu W, Zou S, Chen L, Yao L, Luo M, Chen T, Deng L, Liang K, Song S, Yang R, Zheng R, Gao S, Gui X, Ke H, Hou W, Lundkvist Å, Xiong Y. 2020. Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.03.20030437">10.1101/2020.03.03.20030437</a>.</div>
</div>
<div id="ref-L6kwccjn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2153. </div><div class="csl-right-inline">Olnes MJ, Kotliarov Y, Biancotto A, Cheung F, Chen J, Shi R, Zhou H, Wang E, Tsang JS, Nussenblatt R, The CHI Consortium. 2016. <a href="https://doi.org/10.1038/srep23002">Effects of Systemically Administered Hydrocortisone on the Human Immunome</a>. Scientific Reports 6:23002.</div>
</div>
<div id="ref-K9G9QT8y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2154. </div><div class="csl-right-inline">Lippi G, Plebani M. 2020. <a href="https://doi.org/10.1016/j.cca.2020.03.004">Procalcitonin in patients with severe coronavirus disease 2019 (COVID-19): A meta-analysis</a>. Clinica Chimica Acta 505:190–191.</div>
</div>
<div id="ref-USOnp7c0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2155. </div><div class="csl-right-inline">Xu Y, Xu Z, Liu X, Cai L, Zheng H, Huang Y, Zhou L, Huang L, Lin Y, Deng L, Li J, Chen S, Liu D, Lin Z, Zhou L, He W, Liu X, Li Y. 2020. Clinical findings in critically ill patients infected with SARS-CoV-2 in Guangdong Province, China: a multi-center, retrospective, observational study. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.03.20030668">10.1101/2020.03.03.20030668</a>.</div>
</div>
<div id="ref-14OLQIYK0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2156. </div><div class="csl-right-inline">Feng Y, Qiu M, Liu L, Zou S, Li Y, Luo K, Guo Q, Han N, Sun Y, Wang K, Zhuang X, Zhang S, Chen S, Mo F. 2020. Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus (SARS-CoV-2). Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.03.962332">10.1101/2020.03.03.962332</a>.</div>
</div>
<div id="ref-12ilBRq6I" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2157. </div><div class="csl-right-inline">Cao M, Zhang D, Wang Y, Lu Y, Zhu X, Li Y, Xue H, Lin Y, Zhang M, Sun Y, Yang Z, Shi J, Wang Y, Zhou C, Dong Y, Peng L, Liu P, Dudek SM, Xiao Z, Lu H. 2020. Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.04.20030395">10.1101/2020.03.04.20030395</a>.</div>
</div>
<div id="ref-IPJURDxi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2158. </div><div class="csl-right-inline">Zhang J, Liu J, Li N, Liu Y, Ye R, Qin X, Zheng R. 2020. Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.04.20030916">10.1101/2020.03.04.20030916</a>.</div>
</div>
<div id="ref-H0nvocJz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2159. </div><div class="csl-right-inline">Diao B, Wang C, Wang R, Feng Z, Tan Y, Wang H, Wang C, Liu L, Liu Y, Liu Y, Wang G, Yuan Z, Ren L, Wu Y, Chen Y. 2020. Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.04.20031120">10.1101/2020.03.04.20031120</a>.</div>
</div>
<div id="ref-KHNnjP7M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2160. </div><div class="csl-right-inline">Song C-Y, Xu J, He J-Q, Lu Y-Q. 2020. COVID-19 early warning score: a multi-parameter screening tool to identify highly suspected patients. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.05.20031906">10.1101/2020.03.05.20031906</a>.</div>
</div>
<div id="ref-19xwRoatu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2161. </div><div class="csl-right-inline">Pfaender S, Mar KB, Michailidis E, Kratzel A, Hirt D, V’kovski P, Fan W, Ebert N, Stalder H, Kleine-Weber H, Hoffmann M, Hoffmann HH, Saeed M, Dijkman R, Steinmann E, Wight-Carter M, Hanners NW, Pöhlmann S, Gallagher T, Todt D, Zimmer G, Rice CM, Schoggins JW, Thiel V. 2020. LY6E impairs coronavirus fusion and confers immune control of viral disease. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.05.979260">10.1101/2020.03.05.979260</a>.</div>
</div>
<div id="ref-574aSa8C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2162. </div><div class="csl-right-inline">Liu L, Liu W, Zheng Y, Jiang X, Kou G, Ding J, Wang Q, Huang Q, Ding Y, Ni W, Wu W, Tang S, Tan L, Hu Z, Xu W, Zhang Y, Zhang B, Tang Z, Zhang X, Li H, Rao Z, Jiang H, Ren X, Wang S, Zheng S. 2020. A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.06.20031856">10.1101/2020.03.06.20031856</a>.</div>
</div>
<div id="ref-NT3T9a8H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2163. </div><div class="csl-right-inline">Zheng Z, Monteil VM, Maurer-Stroh S, Yew CW, Leong C, Mohd-Ismail NK, Arularasu SC, Chow VTK, Pin RLT, Mirazimi A, Hong W, Tan Y-J. 2020. Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.06.980037">10.1101/2020.03.06.980037</a>.</div>
</div>
<div id="ref-1GEbBtBgh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2164. </div><div class="csl-right-inline">Zeng Q, Li Y, Huang G, Wu W, Dong S, Xu Y. 2020. Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.08.20031229">10.1101/2020.03.08.20031229</a>.</div>
</div>
<div id="ref-WuHcOc9T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2165. </div><div class="csl-right-inline">Fan H, Zhang L, Huang B, Zhu M, Zhou Y, Zhang H, Tao X, Cheng S, Yu W, Zhu L, Chen J. 2020. Retrospective Analysis of Clinical Features in 101 Death Cases with COVID-19. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.09.20033068">10.1101/2020.03.09.20033068</a>.</div>
</div>
<div id="ref-JwL9G6bd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2166. </div><div class="csl-right-inline">Matsuyama S, Kawase M, Nao N, Shirato K, Ujike M, Kamitani W, Shimojima M, Fukushi S. 2020. The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.11.987016">10.1101/2020.03.11.987016</a>.</div>
</div>
<div id="ref-1CwCTXPqj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2167. </div><div class="csl-right-inline">Zhang B, Zhou X, Zhu C, Feng F, Qiu Y, Feng J, Jia Q, Song Q, Zhu B, Wang J. 2020. Immune phenotyping based on neutrophil-to-lymphocyte ratio and IgG predicts disease severity and outcome for patients with COVID-19. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.12.20035048">10.1101/2020.03.12.20035048</a>.</div>
</div>
<div id="ref-3XqFhi8l" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2168. </div><div class="csl-right-inline">Bao L, Deng W, Gao H, Xiao C, Liu J, Xue J, Lv Q, Liu J, Yu P, Xu Y, Qi F, Qu Y, Li F, Xiang Z, Yu H, Gong S, Liu M, Wang G, Wang S, Song Z, Liu Y, Zhao W, Han Y, Zhao L, Liu X, Wei Q, Qin C. 2020. Lack of Reinfection in Rhesus Macaques Infected with SARS-CoV-2. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.13.990226">10.1101/2020.03.13.990226</a>.</div>
</div>
<div id="ref-6gQxRkWM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2169. </div><div class="csl-right-inline">Yuan M, Wu NC, Zhu X, Lee C-CD, So RTY, Lv H, Mok CKP, Wilson IA. 2020. A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.13.991570">10.1101/2020.03.13.991570</a>.</div>
</div>
<div id="ref-UYD5T2mp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2170. </div><div class="csl-right-inline">Dong L, Zhou J, Niu C, Wang Q, Pan Y, Sheng S, Wang X, Zhang Y, Yang J, Liu M, Zhao Y, Zhang X, Zhu T, Peng T, Xie J, Gao Y, Wang D, Zhao Y, Dai X, Fang X. 2020. Highly accurate and sensitive diagnostic detection of SARS-CoV-2 by digital PCR. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.14.20036129">10.1101/2020.03.14.20036129</a>.</div>
</div>
<div id="ref-14y8UvAbH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2171. </div><div class="csl-right-inline">Wang K, Chen W, Zhou Y-S, Lian J-Q, Zhang Z, Du P, Gong L, Zhang Y, Cui H-Y, Geng J-J, Wang B, Sun X-X, Wang C-F, Yang X, Lin P, Deng Y-Q, Wei D, Yang X-M, Zhu Y-M, Zhang K, Zheng Z-H, Miao J-L, Guo T, Shi Y, Zhang J, Fu L, Wang Q-Y, Bian H, Zhu P, Chen Z-N. 2020. SARS-CoV-2 invades host cells via a novel route: CD147-spike protein. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.14.988345">10.1101/2020.03.14.988345</a>.</div>
</div>
<div id="ref-BcOf65BE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2172. </div><div class="csl-right-inline">Kosugi T, Maeda K, Sato W, Maruyama S, Kadomatsu K. 2015. <a href="https://doi.org/10.1093/ndt/gfu302">CD147 (EMMPRIN/Basigin) in kidney diseases: from an inflammation and immune system viewpoint</a>. Nephrology Dialysis Transplantation 30:1097–1103.</div>
</div>
<div id="ref-34nQJalJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2173. </div><div class="csl-right-inline">Su H, Yang Y. 2018. <a href="https://doi.org/10.1016/j.yexmp.2018.05.001">The roles of CyPA and CD147 in cardiac remodelling</a>. Experimental and Molecular Pathology 104:222–226.</div>
</div>
<div id="ref-KcThTnsm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2174. </div><div class="csl-right-inline">Weidle UH, Scheuer W, Eggle D, Klostermann S, Stockinger H. <a href="https://www.ncbi.nlm.nih.gov/pubmed/20551248">Cancer-related issues of CD147.</a> Cancer Genomics Proteomics 7:157–69.</div>
</div>
<div id="ref-18l4l0nLX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2175. </div><div class="csl-right-inline">Huang L, Shi Y, Gong B, Jiang L, Liu X, Yang J, Tang J, You C, Jiang Q, Long B, Zeng T, Luo M, Zeng F, Zeng F, Wang S, Yang X, Yang Z. 2020. Blood single cell immune profiling reveals the interferon-MAPK pathway mediated adaptive immune response for COVID-19. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.15.20033472">10.1101/2020.03.15.20033472</a>.</div>
</div>
<div id="ref-dM1rOHmH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2176. </div><div class="csl-right-inline">Lv H, Wu NC, Tsang OT-Y, Yuan M, Perera RAPM, Leung WS, So RTY, Chun Chan JM, Yip GK, Hong Chik TS, Wang Y, Chung Choi CY, Lin Y, Ng WW, Zhao J, Poon LLM, Malik Peiris JS, Wilson IA, Mok CKP. 2020. Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infections. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.15.993097">10.1101/2020.03.15.993097</a>.</div>
</div>
<div id="ref-UENoLkhz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2177. </div><div class="csl-right-inline">Duan K, Liu B, Li C, Zhang H, Yu T, Qu J, Zhou M, Chen L, Meng S, Hu Y, Peng C, Yuan M, Huang J, Wang Z, Yu J, Gao X, Wang D, Yu X, Li L, Zhang J, Wu X, Li B, Xu Y, Chen W, Peng Y, Hu Y, Lin L, Liu X, Huang S, Zhou Z, Zhang L, Wang Y, Zhang Z, Deng K, Xia Z, Gong Q, Zhang W, Zheng X, Liu Y, Yang H, Zhou D, Yu D, Hou J, Shi Z, Saijuan C, Chen Z, Zhang X, Yang X. 2020. The feasibility of convalescent plasma therapy in severe COVID- 19 patients: a pilot study. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.16.20036145">10.1101/2020.03.16.20036145</a>.</div>
</div>
<div id="ref-Xbkby2Jr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2178. </div><div class="csl-right-inline">Gautret P, Lagier J-C, Parola P, Hoang VT, Meddeb L, Mailhe M, Doudier B, Courjon J, Giordanengo V, Vieira VE, Dupont HT, Honoré S, Colson P, Chabrière E, Scola BL, Rolain J-M, Brouqui P, Raoult D. 2020. Hydroxychloroquine and azithromycin as a treatment of COVID-19: results of an open-label non-randomized clinical trial. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.16.20037135">10.1101/2020.03.16.20037135</a>.</div>
</div>
<div id="ref-BIlqPh6M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2179. </div><div class="csl-right-inline">Thomé R, Lopes SCP, Costa FTM, Verinaud L. 2013. <a href="https://doi.org/10.1016/j.imlet.2013.07.004">Chloroquine: Modes of action of an undervalued drug</a>. Immunology Letters 153:50–57.</div>
</div>
<div id="ref-Q9tXjG1R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2180. </div><div class="csl-right-inline">Lo B, Zhang K, Lu W, Zheng L, Zhang Q, Kanellopoulou C, Zhang Y, Liu Z, Fritz JM, Marsh R, Husami A, Kissell D, Nortman S, Chaturvedi V, Haines H, Young LR, Mo J, Filipovich AH, Bleesing JJ, Mustillo P, Stephens M, Rueda CM, Chougnet CA, Hoebe K, McElwee J, Hughes JD, Karakoc-Aydiner E, Matthews HF, Price S, Su HC, Rao VK, Lenardo MJ, Jordan MB. 2015. <a href="https://doi.org/10.1126/science.aaa1663">Patients with LRBA deficiency show CTLA4 loss and immune dysregulation responsive to abatacept therapy</a>. Science 349:436–440.</div>
</div>
<div id="ref-3VPi509z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2181. </div><div class="csl-right-inline">Procko E. 2020. The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.16.994236">10.1101/2020.03.16.994236</a>.</div>
</div>
<div id="ref-1EcOLnVXk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2182. </div><div class="csl-right-inline">Rockx B, Kuiken T, Herfst S, Bestebroer T, Lamers MM, de Meulder D, van Amerongen G, van den Brand J, Okba NMA, Schipper D, van Run P, Leijten L, Verschoor E, Verstrepen B, Langermans J, Drosten C, van Vlissingen MF, Fouchier R, de Swart R, Koopmans M, Haagmans BL. 2020. Comparative Pathogenesis Of COVID-19, MERS And SARS In A Non-Human Primate Model. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.17.995639">10.1101/2020.03.17.995639</a>.</div>
</div>
<div id="ref-NzFgXYvQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2183. </div><div class="csl-right-inline">McCray PB, Pewe L, Wohlford-Lenane C, Hickey M, Manzel L, Shi L, Netland J, Jia HP, Halabi C, Sigmund CD, Meyerholz DK, Kirby P, Look DC, Perlman S. 2007. <a href="https://doi.org/10.1128/jvi.02012-06">Lethal Infection of K18-hACE2 Mice Infected with Severe Acute Respiratory Syndrome Coronavirus</a>. Journal of Virology 81:813–821.</div>
</div>
<div id="ref-OvFueDLZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2184. </div><div class="csl-right-inline">Aguilar JB, Faust JS, Westafer LM, Gutierrez JB. 2020. Modeling the Impact of Asymptomatic Carriers on COVID-19 Transmission Dynamics During Lockdown. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.18.20037994">10.1101/2020.03.18.20037994</a>.</div>
</div>
<div id="ref-Y9g1sQ1Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2185. </div><div class="csl-right-inline">Long Q, Deng H, Chen J, Hu J, Liu B, Liao P, Lin Y, Yu L, Mo Z, Xu Y, Gong F, Wu G, Zhang X, Chen Y, Li Z, Wang K, Zhang X, Tian W, Niu C, Yang Q, Xiang J, Du H, Liu H, Lang C, Luo X-H, Wu S, Cui X, Zhou Z, Wang J, Xue C, Li X, Wang L, Tang X, Zhang Y, Qiu J, Liu X, Li J, Zhang D, Zhang F, Cai X, Wang D, Hu Y, Ren J, Tang N, Liu P, Li Q, Huang A. 2020. Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.18.20038018">10.1101/2020.03.18.20038018</a>.</div>
</div>
<div id="ref-p1c9Nilh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2186. </div><div class="csl-right-inline">Hu X, An T, Situ B, Hu Y, Ou Z, Li Q, He X, Zhang Y, Tian P, Sun D, Rui Y, Wang Q, Ding D, Zheng L. 2020. Heat inactivation of serum interferes with the immunoanalysis of antibodies to SARS-CoV-2. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.12.20034231">10.1101/2020.03.12.20034231</a>.</div>
</div>
<div id="ref-1BLNvgEao" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2187. </div><div class="csl-right-inline">Okba NMA, Müller MA, Li W, Wang C, GeurtsvanKessel CH, Corman VM, Lamers MM, Sikkema RS, de Bruin E, Chandler FD, Yazdanpanah Y, Le Hingrat Q, Descamps D, Houhou-Fidouh N, Reusken CBEM, Bosch B-J, Drosten C, Koopmans MPG, Haagmans BL. 2020. SARS-CoV-2 specific antibody responses in COVID-19 patients. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.18.20038059">10.1101/2020.03.18.20038059</a>.</div>
</div>
<div id="ref-K41YFM2N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2188. </div><div class="csl-right-inline">Belhadi D, Peiffer-Smadja N, Lescure F-X, Yazdanpanah Y, Mentré F, Laouénan C. 2020. A brief review of antiviral drugs evaluated in registered clinical trials for COVID-19. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.18.20038190">10.1101/2020.03.18.20038190</a>.</div>
</div>
<div id="ref-eTsOvBVf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2189. </div><div class="csl-right-inline">Leung JM, Yang CX, Tam A, Shaipanich T, Hackett T-L, Singhera GK, Dorscheid DR, Sin DD. 2020. ACE-2 Expression in the Small Airway Epithelia of Smokers and COPD Patients: Implications for COVID-19. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.18.20038455">10.1101/2020.03.18.20038455</a>.</div>
</div>
<div id="ref-18KZGwJ0J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2190. </div><div class="csl-right-inline">Xu Y. 2020. Dynamic profile of severe or critical COVID-19 cases. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.18.20038513">10.1101/2020.03.18.20038513</a>.</div>
</div>
<div id="ref-1BJXDZ04c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2191. </div><div class="csl-right-inline">Fu H, Xu H, Zhang N, Xu H, Li Z, Chen H, Xu R, Sun R, Wen L, Xie L, Liu H, Zhang K, Fu C, Hou K, Yang Z, Yang M, Guo Y. 2020. Association between Clinical, Laboratory and CT Characteristics and RT-PCR Results in the Follow-up of COVID-19 patients. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.19.20038315">10.1101/2020.03.19.20038315</a>.</div>
</div>
<div id="ref-iMCz5Twj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2192. </div><div class="csl-right-inline">Sheahan TP, Sims AC, Zhou S, Graham RL, Hill CS, Leist SR, Schäfer A, Dinnon KH, Montgomery SA, Agostini ML, Pruijssers AJ, Chapell JD, Brown AJ, Bluemling GR, Natchus MG, Saindane M, Kolykhalov AA, Painter G, Harcourt J, Tamin A, Thornburg NJ, Swanstrom R, Denison MR, Baric RS. 2020. An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 and multiple endemic, epidemic and bat coronavirus. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.19.997890">10.1101/2020.03.19.997890</a>.</div>
</div>
<div id="ref-ZDP9vSpg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2193. </div><div class="csl-right-inline">Jeon S, Ko M, Lee J, Choi I, Byun SY, Park S, Shum D, Kim S. 2020. Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.20.999730">10.1101/2020.03.20.999730</a>.</div>
</div>
<div id="ref-LYFzvlzE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2194. </div><div class="csl-right-inline">Munster VJ, Feldmann F, Williamson BN, van Doremalen N, Pérez-Pérez L, Schulz J, Meade-White K, Okumura A, Callison J, Brumbaugh B, Avanzato VA, Rosenke R, Hanley PW, Saturday G, Scott D, Fischer ER, de Wit E. 2020. Respiratory disease and virus shedding in rhesus macaques inoculated with SARS-CoV-2. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.21.001628">10.1101/2020.03.21.001628</a>.</div>
</div>
<div id="ref-unVgVIfU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2195. </div><div class="csl-right-inline">Deng W, Bao L, Gao H, Xiang Z, Qu Y, Song Z, Gong S, Liu J, Liu J, Yu P, Qi F, Xu Y, Li F, Xiao C, Lv Q, Xue J, Wei Q, Liu M, Wang G, Wang S, Yu H, Liu X, Zhao W, Han Y, Qin C. 2020. Ocular conjunctival inoculation of SARS-CoV-2 can cause mild COVID-19 in Rhesus macaques. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.13.990036">10.1101/2020.03.13.990036</a>.</div>
</div>
<div id="ref-wjtVy2v0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2196. </div><div class="csl-right-inline">Pinto BGG, Oliveira AER, Singh Y, Jimenez L, Gonçalves ANA, Ogava RLT, Creighton R, Schatzmann Peron JP, Nakaya HI. 2020. ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.21.20040261">10.1101/2020.03.21.20040261</a>.</div>
</div>
<div id="ref-DFhY5nfb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2197. </div><div class="csl-right-inline">Bian H, Zheng Z-H, Wei D, Zhang Z, Kang W-Z, Hao C-Q, Dong K, Kang W, Xia J-L, Miao J-L, Xie R-H, Wang B, Sun X-X, Yang X-M, Lin P, Geng J-J, Wang K, Cui H-Y, Zhang K, Chen X-C, Tang H, Du H, Yao N, Liu S-S, Liu L-N, Zhang Z, Gao Z-W, Nan G, Wang Q-Y, Lian J-Q, Chen Z-N, Zhu P. 2020. Meplazumab treats COVID-19 pneumonia: an open-labelled, concurrent controlled add-on clinical trial. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.21.20040691">10.1101/2020.03.21.20040691</a>.</div>
</div>
<div id="ref-yP8TXdq2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2198. </div><div class="csl-right-inline">Pushkarsky T, Zybarth G, Dubrovsky L, Yurchenko V, Tang H, Guo H, Toole B, Sherry B, Bukrinsky M. 2001. <a href="https://doi.org/10.1073/pnas.111583198">CD147 facilitates HIV-1 infection by interacting with virus-associated cyclophilin A</a>. Proceedings of the National Academy of Sciences 98:6360–6365.</div>
</div>
<div id="ref-X8l1n4zY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2199. </div><div class="csl-right-inline">Watanabe A, Yoneda M, Ikeda F, Terao-Muto Y, Sato H, Kai C. 2010. <a href="https://doi.org/10.1128/jvi.02168-09">CD147/EMMPRIN Acts as a Functional Entry Receptor for Measles Virus on Epithelial Cells</a>. Journal of Virology 84:4183–4193.</div>
</div>
<div id="ref-YcyNaIVC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2200. </div><div class="csl-right-inline">Crosnier C, Bustamante LY, Bartholdson SJ, Bei AK, Theron M, Uchikawa M, Mboup S, Ndir O, Kwiatkowski DP, Duraisingh MT, Rayner JC, Wright GJ. 2011. <a href="https://doi.org/10.1038/nature10606">Basigin is a receptor essential for erythrocyte invasion by Plasmodium falciparum</a>. Nature 480:534–537.</div>
</div>
<div id="ref-m9yLJ5Da" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2201. </div><div class="csl-right-inline">Chen Z, Mi L, Xu J, Yu J, Wang X, Jiang J, Xing J, Shang P, Qian A, Li Y, Shaw Peter X, Wang J, Duan S, Ding J, Fan C, Zhang Y, Yang Y, Yu X, Feng Q, Li B, Yao X, Zhang Z, Li L, Xue X, Zhu P. 2005. <a href="https://doi.org/10.1086/427811">Function of HAb18G/CD147 in Invasion of Host Cells by Severe Acute Respiratory Syndrome Coronavirus</a>. The Journal of Infectious Diseases 191:755–760.</div>
</div>
<div id="ref-J0ibHE3l" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2202. </div><div class="csl-right-inline">Yee C, Main NM, Terry A, Stevanovski I, Maczurek A, Morgan AJ, Calabro S, Potter AJ, Iemma TL, Bowen DG, Ahlenstiel G, Warner FJ, McCaughan GW, McLennan SV, Shackel NA. 2019. <a href="https://doi.org/10.1371/journal.pone.0215557">CD147 mediates intrahepatic leukocyte aggregation and determines the extent of liver injury</a>. PLOS ONE 14:e0215557.</div>
</div>
<div id="ref-bCYYaFKM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2203. </div><div class="csl-right-inline">Davidson AD, Williamson MK, Lewis S, Shoemark D, Carroll MW, Heesom K, Zambon M, Ellis J, Lewis PA, Hiscox JA, Matthews DA. 2020. Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.22.002204">10.1101/2020.03.22.002204</a>.</div>
</div>
<div id="ref-16tHVjVzu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2204. </div><div class="csl-right-inline">Sun X, Tse LV, Ferguson AD, Whittaker GR. 2010. <a href="https://doi.org/10.1128/jvi.00797-10">Modifications to the Hemagglutinin Cleavage Site Control the Virulence of a Neurotropic H1N1 Influenza Virus</a>. Journal of Virology 84:8683–8690.</div>
</div>
<div id="ref-hRVfFbS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2205. </div><div class="csl-right-inline">Kim D, Lee J-Y, Yang J-S, Kim JW, Kim VN, Chang H. 2020. The architecture of SARS-CoV-2 transcriptome. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.12.988865">10.1101/2020.03.12.988865</a>.</div>
</div>
<div id="ref-phJM8g2Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2206. </div><div class="csl-right-inline">Gordon DE, Jang GM, Bouhaddou M, Xu J, Obernier K, O’Meara MJ, Guo JZ, Swaney DL, Tummino TA, Hüttenhain R, Kaake RM, Richards AL, Tutuncuoglu B, Foussard H, Batra J, Haas K, Modak M, Kim M, Haas P, Polacco BJ, Braberg H, Fabius JM, Eckhardt M, Soucheray M, Bennett MJ, Cakir M, McGregor MJ, Li Q, Naing ZZC, Zhou Y, Peng S, Kirby IT, Melnyk JE, Chorba JS, Lou K, Dai SA, Shen W, Shi Y, Zhang Z, Barrio-Hernandez I, Memon D, Hernandez-Armenta C, Mathy CJP, Perica T, Pilla KB, Ganesan SJ, Saltzberg DJ, Ramachandran R, Liu X, Rosenthal SB, Calviello L, Venkataramanan S, Lin Y, Wankowicz SA, Bohn M, Trenker R, Young JM, Cavero D, Hiatt J, Roth T, Rathore U, Subramanian A, Noack J, Hubert M, Roesch F, Vallet T, Meyer B, White KM, Miorin L, Agard D, Emerman M, Ruggero D, García-Sastre A, Jura N, Zastrow M von, Taunton J, Schwartz O, Vignuzzi M, d’Enfert C, Mukherjee S, Jacobson M, Malik HS, Fujimori DG, Ideker T, Craik CS, Floor S, Fraser JS, Gross J, Sali A, Kortemme T, Beltrao P, Shokat K, Shoichet BK, Krogan NJ. 2020. A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug-Repurposing. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.22.002386">10.1101/2020.03.22.002386</a>.</div>
</div>
<div id="ref-RTIk8DTo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2207. </div><div class="csl-right-inline">Chen H, Zhang Z, Wang L, Huang Z, Gong F, Li X, Chen Y, Wu JJ. 2020. First Clinical Study Using HCV Protease Inhibitor Danoprevir to Treat Naïve and Experienced COVID-19 Patients. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.22.20034041">10.1101/2020.03.22.20034041</a>.</div>
</div>
<div id="ref-YoCPtq6P" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2208. </div><div class="csl-right-inline">Seiwert SD, Andrews SW, Jiang Y, Serebryany V, Tan H, Kossen K, Rajagopalan PTR, Misialek S, Stevens SK, Stoycheva A, Hong J, Lim SR, Qin X, Rieger R, Condroski KR, Zhang H, Do MG, Lemieux C, Hingorani GP, Hartley DP, Josey JA, Pan L, Beigelman L, Blatt LM. 2008. <a href="https://doi.org/10.1128/aac.00699-08">Preclinical Characteristics of the Hepatitis C Virus NS3/4A Protease Inhibitor ITMN-191 (R7227)</a>. Antimicrobial Agents and Chemotherapy 52:4432–4441.</div>
</div>
<div id="ref-10GnhBctk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2209. </div><div class="csl-right-inline">Xu X, Feng B, Guan Y, Zheng S, Sheng J, Yang X, Ma Y, Huang Y, Kang Y, Wen X, Li J, Tan Y, He Q, Xie Q, Wang M, An P, Gong G, Liu H, Ning Q, Hua R, Ning B, Xie W, Zhang J, Huang W, Yang Y, Lin M, Zhao Y, Yu Y, Jia J, Yang D, Chen L, Ye Y, Nan Y, Gong Z, Zhang Q, Hu P, Wang F, Li Y, Li D, Jia Z, Hou J, Chen C, Wu JJ, Wei L. 2019. <a href="https://doi.org/10.14218/jcth.2019.00033">Efficacy and Safety of All-oral, 12-week Ravidasvir Plus Ritonavir-boosted Danoprevir and Ribavirin in Treatment-naïve Noncirrhotic HCV Genotype 1 Patients: Results from a Phase 2/3 Clinical Trial in China</a>. Journal of Clinical and Translational Hepatology 7:1–8.</div>
</div>
<div id="ref-7zBwK1wn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2210. </div><div class="csl-right-inline">Nguyen DD, Gao K, Chen J, Wang R, Wei G-W. 2020. Potentially highly potent drugs for 2019-nCoV. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.02.05.936013">10.1101/2020.02.05.936013</a>.</div>
</div>
<div id="ref-lnuNSHEW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2211. </div><div class="csl-right-inline">Lou B, Li T-D, Zheng S-F, Su Y-Y, Li Z-Y, Liu W, Yu F, Ge S-X, Zou Q-D, Yuan Q, Lin S, Hong C-M, Yao X-Y, Zhang X-J, Wu D-H, Zhou G-L, Hou W-H, Li T-T, Zhang Y-L, Zhang S-Y, Fan J, Zhang J, Xia N-S, Chen Y. 2020. Serology characteristics of SARS-CoV-2 infection since the exposure and post symptoms onset. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.23.20041707">10.1101/2020.03.23.20041707</a>.</div>
</div>
<div id="ref-aNSdARwe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2212. </div><div class="csl-right-inline">Blanco-Melo D, Nilsson-Payant BE, Liu W-C, Møller R, Panis M, Sachs D, Albrecht RA, tenOever BR. 2020. SARS-CoV-2 launches a unique transcriptional signature from in vitro, ex vivo, and in vivo systems. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.24.004655">10.1101/2020.03.24.004655</a>.</div>
</div>
<div id="ref-EWZmuLvu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2213. </div><div class="csl-right-inline">Zhou Y, Yang Z, Guo Y, Geng S, Gao S, Ye S, Hu Y, Wang Y. 2020. A New Predictor of Disease Severity in Patients with COVID-19 in Wuhan, China. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.24.20042119">10.1101/2020.03.24.20042119</a>.</div>
</div>
<div id="ref-95sQSGzx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2214. </div><div class="csl-right-inline">Nie S, Zhao X, Zhao K, Zhang Z, Zhang Z, Zhang Z. 2020. Metabolic disturbances and inflammatory dysfunction predict severity of coronavirus disease 2019 (COVID-19): a retrospective study. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.24.20042283">10.1101/2020.03.24.20042283</a>.</div>
</div>
<div id="ref-btVuoX6V" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2215. </div><div class="csl-right-inline">Tan W, Lu Y, Zhang J, Wang J, Dan Y, Tan Z, He X, Qian C, Sun Q, Hu Q, Liu H, Ye S, Xiang X, Zhou Y, Zhang W, Guo Y, Wang X-H, He W, Wan X, Sun F, Wei Q, Chen C, Pan G, Xia J, Mao Q, Chen Y, Deng G. 2020. Viral Kinetics and Antibody Responses in Patients with COVID-19. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.24.20042382">10.1101/2020.03.24.20042382</a>.</div>
</div>
<div id="ref-LIf5SSUs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2216. </div><div class="csl-right-inline">Jiang H, Li Y, Zhang H, Wang W, Men D, Yang X, Qi H, Zhou J, Tao S. 2020. Global profiling of SARS-CoV-2 specific IgG/ IgM responses of convalescents using a proteome microarray. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.20.20039495">10.1101/2020.03.20.20039495</a>.</div>
</div>
<div id="ref-tC5vJmwj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2217. </div><div class="csl-right-inline">Liu L, Wei Q, Lin Q, Fang J, Wang H, Kwok H, Tang H, Nishiura K, Peng J, Tan Z, Wu T, Cheung K-W, Chan K-H, Alvarez X, Qin C, Lackner A, Perlman S, Yuen K-Y, Chen Z. 2019. <a href="https://doi.org/10.1172/jci.insight.123158">Anti–spike IgG causes severe acute lung injury by skewing macrophage responses during acute SARS-CoV infection</a>. JCI Insight 4:e123158.</div>
</div>
<div id="ref-1FZgghazg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2218. </div><div class="csl-right-inline">Zhang D, Guo R, Lei L, Liu H, Wang Y, Wang Y, Qian H, Dai T, Zhang T, Lai Y, Wang J, Liu Z, Chen T, He A, O’Dwyer M, Hu J. 2020. COVID-19 infection induces readily detectable morphological and inflammation-related phenotypic changes in peripheral blood monocytes, the severity of which correlate with patient outcome. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.24.20042655">10.1101/2020.03.24.20042655</a>.</div>
</div>
<div id="ref-15gdkEnfn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2219. </div><div class="csl-right-inline">Miller A, Reandelar MJ, Fasciglione K, Roumenova V, Li Y, Otazu GH. 2020. Correlation between universal BCG vaccination policy and reduced morbidity and mortality for COVID-19: an epidemiological study. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.24.20042937">10.1101/2020.03.24.20042937</a>.</div>
</div>
<div id="ref-16FOT89K5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2220. </div><div class="csl-right-inline">Moorlag SJCFM, Arts RJW, van Crevel R, Netea MG. 2019. <a href="https://doi.org/10.1016/j.cmi.2019.04.020">Non-specific effects of BCG vaccine on viral infections</a>. Clinical Microbiology and Infection 25:1473–1478.</div>
</div>
<div id="ref-MCuIRGFE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2221. </div><div class="csl-right-inline">BCG vaccination to reduce the impact of COVID-19 in healthcare workers (The BRACE Trial). Murdoch Children's Research Institute. <a href="https://www.mcri.edu.au/BRACE">https://www.mcri.edu.au/BRACE</a>. Retrieved 31 July 2020.</div>
</div>
<div id="ref-12oDb7aki" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2222. </div><div class="csl-right-inline">Brann DH, Tsukahara T, Weinreb C, Lipovsek M, Van den Berge K, Gong B, Chance R, Macaulay IC, Chou H, Fletcher R, Das D, Street K, de Bezieux HR, Choi Y-G, Risso D, Dudoit S, Purdom E, Mill JS, Hachem RA, Matsunami H, Logan DW, Goldstein BJ, Grubb MS, Ngai J, Datta SR. 2020. Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.25.009084">10.1101/2020.03.25.009084</a>.</div>
</div>
<div id="ref-t4vofknR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2223. </div><div class="csl-right-inline">Smith JC, Sausville EL, Girish V, Yuan ML, John KM, Sheltzer JM. 2020. Cigarette smoke exposure and inflammatory signaling increase the expression of the SARS-CoV-2 receptor ACE2 in the respiratory tract. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.28.013672">10.1101/2020.03.28.013672</a>.</div>
</div>
<div id="ref-k1E3DuFN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2224. </div><div class="csl-right-inline">Liu R, Liu X, Han H, Shereen MA, Niu Z, Li D, Liu F, Wu K, Luo Z, Zhu C. 2020. The comparative superiority of IgM-IgG antibody test to real-time reverse transcriptase PCR detection for SARS-CoV-2 infection diagnosis. Cold Spring Harbor Laboratory https://doi.org/<a href="https://doi.org/10.1101/2020.03.28.20045765">10.1101/2020.03.28.20045765</a>.</div>
</div>
</div>
<!-- Define citation tags below -->
<!-- Individual sections that have been published as preprints or journal manuscripts -->
<h1 class="page_break_before" data-number="14" id="appendix-a"><span class="header-section-number">14</span> Appendix A</h1>
<p>This appendix contains reviews produced by the Immunology Institute of the Icahn School of Medicine</p>
<!-- Appendix source: https://github.com/ismms-himc/covid-19_sinai_reviews/tree/05d387bf10b829dad6bedb4bf313686750b78cc9 -->
<h2 data-number="14.1" id="potent-binding-of-2019-novel-coronavirus-spike-protein-by-a-sars-coronavirus-specific-human-monoclonal-antibody"><span class="header-section-number">14.1</span> Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</h2>
<p>Tian et al. <em>Emerg Microbes Infect 2020</em> <span class="citation" data-cites="1HLSeqnY8">(<a href="#ref-1HLSeqnY8" role="doc-biblioref">2101</a>)</span></p>
<h3 data-number="14.1.1" id="keywords-1"><span class="header-section-number">14.1.1</span> Keywords</h3>
<ul>
<li>Monoclonal antibody</li>
<li>Cross-reactivity</li>
<li>receptor binding domain</li>
</ul>
<h3 data-number="14.1.2" id="summary"><span class="header-section-number">14.1.2</span> Summary</h3>
<p>Considering the relatively high identity of the receptor binding domain
(RBD) of the spike proteins from 2019-nCoV and SARS-CoV (73%), this
study aims to assess the cross-reactivity of several anti-SARS-CoV
monoclonal antibodies with 2019-nCoV. The results showed that the
SARS-CoV-specific antibody CR3022 can potently bind 2019-nCoV RBD.</p>
<h3 data-number="14.1.3" id="main-findings"><span class="header-section-number">14.1.3</span> Main Findings</h3>
<p>The structure of the 2019-nCoV spike RBD and its conformation in complex
with the receptor angiotensin-converting enzyme (ACE2) was modeled <em>in
silico</em> and compared with the SARS-CoV RBD structure. The models
predicted very similar RBD-ACE2 interactions for both viruses. The
binding capacity of representative SARS-CoV-RBD specific monoclonal
antibodies (m396, CR3014, and CR3022) to recombinant 2019-nCoV RBD was
then investigated by ELISA and their binding kinetics studied using
biolayer interferometry. The analysis showed that only CR3022 was able
to bind 2019-nCoV RBD with high affinity (KD of 6.3 nM), however it did
not interfere with ACE2 binding. Antibodies m396 and CR3014, which
target the ACE2 binding site of SARS-CoV failed to bind 2019-nCoV spike
protein.</p>
<h3 data-number="14.1.4" id="limitations"><span class="header-section-number">14.1.4</span> Limitations</h3>
<p>The 2019-nCoV RBD largely differ from the SARS-CoV at the C-terminus
residues, which drastically impact the cross-reactivity of antibodies
described for other B beta-coronaviruses, including SARS-CoV. This study
claims that CR3022 antibody could be a potential candidate for therapy.
However, none of the antibodies assayed in this work showed
cross-reactivity with the ACE2 binding site of 2019-nCoV, essential for
the replication of this virus. Furthermore, neutralization assays with
2019-nCoV virus or pseudovirus were not performed. Although the use of
neutralizing antibodies is an interesting approach, these results
suggest that it is critical the development of novel monoclonal
antibodies able to specifically bind 2019-nCoV spike protein.</p>
<h3 data-number="14.1.5" id="credit"><span class="header-section-number">14.1.5</span> Credit</h3>
<p><em>Review by D.L.O as part of a project by students, postdocs and faculty
at the Immunology Institute of the Icahn School of Medicine, Mount
Sinai.</em></p>
<h2 data-number="14.2" id="integrative-bioinformatics-analysis-provides-insight-into-the-molecular-mechanisms-of-2019-ncov"><span class="header-section-number">14.2</span> Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV</h2>
<p>He et al. <em>medRxiv</em> <span class="citation" data-cites="lOQPScZt">(<a href="#ref-lOQPScZt" role="doc-biblioref">2102</a>)</span></p>
<h3 data-number="14.2.1" id="keywords-2"><span class="header-section-number">14.2.1</span> Keywords</h3>
<ul>
<li>ACE2</li>
<li>lungs</li>
<li>smoking</li>
<li>COPD</li>
<li>asthma</li>
<li>SARS-Cov</li>
<li>IL-1</li>
<li>IL-10</li>
<li>IL-6</li>
<li>IL-8</li>
</ul>
<h3 data-number="14.2.2" id="main-findings-1"><span class="header-section-number">14.2.2</span> Main Findings</h3>
<p>The authors used bioinformatics tools to identify features
of ACE2 expression in the lungs of different patent groups: healthy,
smokers, patients with chronic airway disease (i.e., COPD) or asthma.
They used gene expression data publicly available from GEO that included
lung tissues, bronchoalveolar lavage, bronchial epithelial cells, small
airway epithelial cells, or SARS-Cov infected cells.</p>
<p>The authors describe no significant differences in ACE2 expression in
lung tissues of Healthy, COPD, and Asthma groups (p=0.85); or in BAL of
Healthy and COPD (p=0.48); or in epithelial brushings of Healthy and
Mild/Moderate/Severe Asthma (p=0.99). ACE2 was higher in the small
airway epithelium of long-term smokers vs non-smokers (p&lt;0.001).
Consistently, there was a trend of higher ACE2 expression in the
bronchial airway epithelial cells 24h post-acute smoking exposure
(p=0.073). Increasing ACE2 expression at 24h and 48h compared to 12h
post SARS-Cov infection (p=0.026; n=3 at each time point) was also
detected.</p>
<p>15 lung samples’ data from healthy participants were separated into high
and low ACE2 expression groups. “High” ACE2 expression was associated
with the following GO pathways: innate and adaptive immune responses, B
cell mediated immunity, cytokine secretion, and IL-1, IL-10, IL-6, IL-8
cytokines. The authors speculate that a high basal ACE2 expression will
increase susceptibility to SARS-CoV infection.</p>
<p>In 3 samples SARS-Cov infection was associated with IL-1, IL-10 and IL-6
cytokine production (GO pathways) at 24h. And later, at 48h, with T-cell
activation and T-cell cytokine production. It is unclear whether those
changes were statistically significant.</p>
<p>The authors describe a time course quantification of immune infiltrates
in epithelial cells infected with SARS-Cov infection. They state that in
healthy donors ACE2 expression did not correlate with the immune cell
infiltration. However, in SARS-Cov samples, at 48h they found that ACE2
correlated with neutrophils, NK-, Th17-, Th2-, Th1- cells, and DCs.
Again, while authors claim significance, the corresponding correlation
coefficients and p-values are not presented in the text or figures. In
addition, the source of the data for this analysis is not clear.</p>
<p>Using network analysis, proteins SRC, FN1, MAPK3, LYN, MBP, NLRC4, NLRP1
and PRKCD were found to be central (Hub proteins) in the regulating
network of cytokine secretion after coronavirus infection. Authors
conclude this indicates that these molecules were critically important
in ACE2-induced inflammatory response. Additionally, authors speculate
that the increased expression of ACE2 affected RPS3 and SRC, which were
the two hub genes involved in viral replication and inflammatory
response.</p>
<h3 data-number="14.2.3" id="limitations-1"><span class="header-section-number">14.2.3</span> Limitations</h3>
<p>The methods section is very limited and does not
describe any of the statistical analyses; and description of the
construction of the regulatory protein networks is also limited. For the
findings in Figures 2 authors claim significance, which is not supported
by p-values or coefficients. For the sample selection, would be useful
if sample sizes and some of the patients’ demographics (e.g. age) were
described.</p>
<p>For the analysis of high vs low ACE2 expression in healthy subjects, it
is not clear what was the cut off for ‘high’ expression and how it was
determined. Additionally, further laboratory studies are warranted to
confirm that high ACE2 gene expression would have high correlation with
the amount of ACE2 protein on cell surface. For the GO pathway analysis
significance was set at p&lt;0.05, but not adjusted for multiple
comparisons.</p>
<p>There were no samples with SARS-CoV-2 infection. While SARS-Cov and
SARS-CoV-2 both use ACE2 to enter the host cells, the analysis only
included data on SARS-Cov and any conclusions about SARS-CoV2 are
limited.</p>
<p>Upon checking GSE accession numbers of the datasets references, two
might not be cited correctly: GSE37758 (“A spergillus niger: Control
(fructose) vs. steam-exploded sugarcane induction (SEB)”” was used in
this paper as “lung tissue” data) and GSE14700 (“Steroid Pretreatment
of Organ Donors to Prevent Postischemic Renal Allograft Failure: A
Randomized, Controlled Trial” – was used as SARS-Cov infection data).</p>
<h3 data-number="14.2.4" id="credit-1"><span class="header-section-number">14.2.4</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.3" id="diarrhea-may-be-underestimated-a-missing-link-in-2019-novel-coronavirus"><span class="header-section-number">14.3</span> Diarrhea may be underestimated: a missing link in 2019 novel coronavirus</h2>
<p>Liang et al. <em>medRxiv</em> <span class="citation" data-cites="rL9IRhVy">(<a href="#ref-rL9IRhVy" role="doc-biblioref">2103</a>)</span></p>
<h3 data-number="14.3.1" id="keywords-3"><span class="header-section-number">14.3.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2</li>
<li>diarrhea</li>
<li>ACE2</li>
<li>scRNA-seq</li>
</ul>
<h3 data-number="14.3.2" id="main-findings-2"><span class="header-section-number">14.3.2</span> Main Findings</h3>
<p>This study examined the incidence of diarrhea in
patients infected with SARS-CoV-2 across three recently published
cohorts and found that there are statistically significant differences
by Fisher’s exact test. They report that this could be due to subjective
diagnosis criterion for diarrhea or from patients first seeking medical
care from gastroenterologist. In order to minimize nosocomial infections
arising from unsuspected patients with diarrhea and gain comprehensive
understanding of transmission routes for this viral pathogen, they
compared the transcriptional levels of ACE2 of various human tissues
from NCBI public database as well as in small intestine tissue from
CD57BL/6 mice using single cell sequencing. They show that ACE2
expression is not only increased in the human small intestine, but
demonstrate a particular increase in mice enterocytes positioned on the
surface of the intestinal lining exposed to viral pathogens. Given that
ACE2 is the viral receptor for SARS-CoV-2 and also reported to regulate
diarrhea, their data suggests the small intestine as a potential
transmission route and diarrhea as a potentially underestimated symptom
in COVID19 patients that must be carefully monitored. Interestingly,
however, they show that ACE2 expression level is not elevated in human
lung tissue.</p>
<h3 data-number="14.3.3" id="limitations-2"><span class="header-section-number">14.3.3</span> Limitations</h3>
<p>Although this study demonstrates a
statistical difference in the incidence of diarrhea across three
separate COVID19 patient cohorts, their conclusions are limited by a
small sample size. Specifically, the p-value computed by Fisher’s exact
test is based on a single patient cohort of only six cases of which 33%
are reported to have diarrhea, while the remaining two larger cohorts
with 41 and 99 cases report 3% and 2% diarrhea incidence, respectively.
Despite showing significance, they would need to acquire larger sample
sizes and cohorts to minimize random variability and draw meaningful
conclusions. Furthermore, they do not address why ACE2 expression level
is not elevated in human lung tissue despite it being a major
established route of transmission for SARS-CoV-2. It could be helpful to
validate this result by looking at ACE2 expression in mouse lung tissue.
Finally, although this study is descriptive and shows elevated ACE2
expression in small intestinal epithelial cells, it does not establish a
mechanistic link to SARS-CoV-2 infection of the host. Overall, their
claim that infected patients exhibiting diarrhea pose an increased risk
to hospital staff needs to be further substantiated.</p>
<h3 data-number="14.3.4" id="significance"><span class="header-section-number">14.3.4</span> Significance</h3>
<p>This study provides a possible transmission route and a
potentially underappreciated clinical symptom for SARS-CoV-2 for better
clinical management and control of COVID19.</p>
<h3 data-number="14.3.5" id="credit-2"><span class="header-section-number">14.3.5</span> Credit</h3>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.4" id="specific-ace2-expression-in-cholangiocytes-may-cause-liver-damage-after-2019-ncov-infection"><span class="header-section-number">14.4</span> Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection</h2>
<p>Chai et al. <em>bioRxiv</em> <span class="citation" data-cites="BHQvMYPG">(<a href="#ref-BHQvMYPG" role="doc-biblioref">2104</a>)</span></p>
<h3 data-number="14.4.1" id="keywords-4"><span class="header-section-number">14.4.1</span> Keywords</h3>
<ul>
<li>ACE2</li>
<li>Cholangiocytes</li>
<li>COVID-associated Liver Damage</li>
</ul>
<h3 data-number="14.4.2" id="summary-1"><span class="header-section-number">14.4.2</span> Summary</h3>
<p>Using both publicly available scRNA-seq dataset of liver
samples from colorectal patients and scRNA-sequencing of four liver
samples from healthy volunteers, the authors show that ACE2 is
significantly enriched in the majority of cholangiocytes (59.7 %) but
not in hepatocytes (2.6%).</p>
<h3 data-number="14.4.3" id="main-findings-3"><span class="header-section-number">14.4.3</span> Main Findings</h3>
<p>Using bioinformatics approaches of
RNASeq analysis, this study reveals that ACE2 dominates in
cholangiocytes and is present at very low levels in hepatocytes.</p>
<h3 data-number="14.4.4" id="limitations-3"><span class="header-section-number">14.4.4</span> Limitations</h3>
<p>The
study does not provide mechanistic insights into how SARS-CoV-2 can
infect and replicate in cholangiocytes and the types of intrinsic
anti-viral responses induced by cholangiocytes when infected. In
addition, because the study relies on the assumption that SARS-CoV-2
infects cells only through ACE2, it cannot discount the possibility that
the virus can infect hepatocytes through mechanisms other than
ACE2-mediated entry. Furthermore, because the scRNA-seq analysis were
performed on healthy liver samples, one cannot draw any definitive
conclusions about gene expression states (including ACE2 expression in
liver cell types) in system-wide inflammatory contexts.</p>
<h3 data-number="14.4.5" id="significance-1"><span class="header-section-number">14.4.5</span> Significance</h3>
<p>This article with other studies on liver
damage in COVID patients suggests that liver damage observed in COVID
patients is more due to inflammatory cytokines than direct infection of
the liver. Even if cholangiocytes are infectable by SARS-CoV-2 (which
was demonstrated by human liver ductal organoid study
(<span class="citation" data-cites="OuDMtCRl">(<a href="#ref-OuDMtCRl" role="doc-biblioref">2105</a>)</span>),
published clinical data show no significant increase in bile duct injury
related indexes (i.e. alkaline phosphatase, gamma-glutamyl
transpeptidase and total bilirubin). In sum, it underscores the
importance of future studies characterizing cellular responses of
extra-pulmonary organs in the context of COVID or at least in viral lung
infections..</p>
<h3 data-number="14.4.6" id="credit-3"><span class="header-section-number">14.4.6</span> Credit</h3>
<p><em>Summary generated by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.5" id="ace2-expression-by-colonic-epithelial-cells-is-associated-with-viral-infection-immunity-and-energy-metabolism"><span class="header-section-number">14.5</span> ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism</h2>
<p>Wang et al. <em>medRxiv</em>. <span class="citation" data-cites="ViWCb8MJ">(<a href="#ref-ViWCb8MJ" role="doc-biblioref">2106</a>)</span></p>
<h3 data-number="14.5.1" id="keywords-5"><span class="header-section-number">14.5.1</span> Keywords</h3>
<ul>
<li>single cell RNA seq</li>
<li>ACE2 expression</li>
<li>human colonic biopsy</li>
</ul>
<h3 data-number="14.5.2" id="main-findings-4"><span class="header-section-number">14.5.2</span> Main Findings</h3>
<p>Colonic enterocytes primarily express ACE2. Cellular pathways associated with ACE2 expression include innate immune signaling, HLA up regulation, energy metabolism and apoptotic signaling.</p>
<h3 data-number="14.5.3" id="limitations-4"><span class="header-section-number">14.5.3</span> Limitations</h3>
<p>This is a study of colonic biopsies taken from 17 children with and without IBD and analyzed using scRNAseq to look at ACE2 expression and identify gene families correlated with ACE2 expression. The authors find ACE2 expression to be primarily in colonocytes. It is not clear why both healthy and IBD patients were combined for the analysis. Biopsies were all of children so extrapolation to adults is limited. The majority of genes found to be negatively correlated with ACE2 expression include immunoglobulin genes (IGs). IG expression will almost certainly be low in colonocytes irrespective of ACE2 expression.</p>
<h3 data-number="14.5.4" id="significance-2"><span class="header-section-number">14.5.4</span> Significance</h3>
<p>This study performs a retrospective analysis of ACE2 expression using an RNAseq dataset from intestinal biopsies of children with and without IBD. The implications for the CoV-19 epidemic are modest, but do provide support that ACE2 expression is specific to colonocytes in the intestines. The ontological pathway analysis provides some limited insights into gene expression associated with ACE2.</p>
<h3 data-number="14.5.5" id="credit-4"><span class="header-section-number">14.5.5</span> Credit</h3>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.6" id="the-pathogenicity-of-2019-novel-coronavirus-in-hace2-transgenic-mice"><span class="header-section-number">14.6</span> The Pathogenicity of 2019 Novel Coronavirus in hACE2 Transgenic Mice</h2>
<p>Bao et al. <em>bioRxiv</em> <span class="citation" data-cites="MgLBA5m">(<a href="#ref-MgLBA5m" role="doc-biblioref">2107</a>)</span></p>
<h3 data-number="14.6.1" id="keywords-6"><span class="header-section-number">14.6.1</span> Keywords</h3>
<ul>
<li>Covid-19 mouse model</li>
<li>hACE2 mice</li>
<li>2019-nCoV model</li>
<li>ACE2</li>
<li>2019-nCoV </li>
</ul>
<h3 data-number="14.6.2" id="main-findings-5"><span class="header-section-number">14.6.2</span> Main Findings</h3>
<p>Using a transgenic human Angiotensin-converting enzyme 2
(hACE2) mouse that has previously been shown susceptible to infection by
SARS-CoV, Bao et al. create a model of pandemic 2019-nCoV strain
coronavirus. The model includes interstitial hyperplasia in lung tissue,
moderate inflammation in bronchioles and blood vessels, and histology
consistent with viral pneumonia at 3 days post infection. Wildtype did
not experience these symptoms. In addition, viral antigen and hACE2
receptor were found to co-localize the lung by immunofluorescence 3-10
days post infection only in the hACE2 infected mice.</p>
<h3 data-number="14.6.3" id="limitations-5"><span class="header-section-number">14.6.3</span> Limitations</h3>
<p>The characterization of the infection remains incomplete,
as well as lacking characterization of the immune response other than
the presence of a single antiviral antibody. Though they claim to
fulfill Koch’s postulates, they only isolate the virus and re-infect
Vero cells, rather than naive mice.</p>
<h3 data-number="14.6.4" id="significance-3"><span class="header-section-number">14.6.4</span> Significance</h3>
<p>This paper establishes a murine model for 2019-nCoV
infection with symptoms consistent with viral pneumonia. Though not
fully characterized, this model allows in vivo analysis of viral entry
and pathology that is important for the development of vaccines and
antiviral therapeutics.</p>
<h3 data-number="14.6.5" id="credit-5"><span class="header-section-number">14.6.5</span> Credit</h3>
<p><em>Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty
at the Immunology Institute of the Icahn School of Medicine, Mount
Sinai.</em></p>
<h2 data-number="14.7" id="caution-on-kidney-dysfunctions-of-2019-ncov-patients"><span class="header-section-number">14.7</span> Caution on Kidney Dysfunctions of 2019-nCoV Patients</h2>
<p>Li et al. <em>medRxiv</em>. <span class="citation" data-cites="1Acv5UMSt">(<a href="#ref-1Acv5UMSt" role="doc-biblioref">2108</a>)</span></p>
<h3 data-number="14.7.1" id="keywords-7"><span class="header-section-number">14.7.1</span> Keywords</h3>
<p>CoVID-19, 2019-nCoV, SARS-CoV-2, kidney, clinical, creatinine,
proteinuria, albuminuria, CT</p>
<h3 data-number="14.7.2" id="main-findings-6"><span class="header-section-number">14.7.2</span> Main Findings</h3>
<ul>
<li><p>Retrospective study of 59 patients assayed key function indicators
of the kidney–including urine protein, blood urea nitrogen (BUN),
plasma creatinine (Cre), and renal CT scan data.</p></li>
<li><p>Found that 34% of patients developed massive albuminuria on the
first day of admission, and 63% developed proteinuria during their
stay in hospital; and 19% of patients had high plasma creatinine,
especially the terminal cases.</p></li>
<li><p>CT analyses of 27 patients showed all patients to have abnormal
kidney damage; indicate that inflammation and edema of the renal
parenchyma very common.</p></li>
</ul>
<h3 data-number="14.7.3" id="limitations-6"><span class="header-section-number">14.7.3</span> Limitations</h3>
<ul>
<li><p>No analysis of immunity-dependent damage and cytokines in
blood/plasma/urine. Will be worth correlating disease progression
with cytokine production, immune activity and kidney function.</p></li>
<li><p>Extrapolating to earlier SARS-CoV studies provides the only
rationale for viral-damage in kidney and resultant pathologic immune
response <em>(understandable for this clinical study)</em>.</p></li>
</ul>
<h3 data-number="14.7.4" id="significance-4"><span class="header-section-number">14.7.4</span> Significance</h3>
<ul>
<li><p>Multiple lines of evidence along this study’s finding point to the
idea that renal impairment/injury is a key risk factor in 2019-nCoV
patients similar to what has been reported for SARS-CoV<span class="citation" data-cites="DMaiZBE1">(<a href="#ref-DMaiZBE1" role="doc-biblioref">2109</a>)</span>; this
may be one of the major causes of virally-induced damage and
contribute to multiorgan failure.</p></li>
<li><p>ACE2 expression in kidney proximal tubule epithelia and bladder
epithelia <span class="citation" data-cites="16Xj27QYj">(<a href="#ref-16Xj27QYj" role="doc-biblioref">2110</a>)</span> support
these clinical findings.</p></li>
<li><p>Study argues for closely monitoring kidney function, and applying
potential interventions including continuous renal replacement
therapies (CRRT) for protecting kidney functions as early as
possible, particularly for those with rising plasma creatinine.</p></li>
</ul>
<h3 data-number="14.7.5" id="credit-6"><span class="header-section-number">14.7.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.8" id="profiling-the-immune-vulnerability-landscape-of-the-2019-novel-coronavirus"><span class="header-section-number">14.8</span> Profiling the immune vulnerability landscape of the 2019 Novel Coronavirus</h2>
<p>Zhu et al. <em>bioRxiv</em> <span class="citation" data-cites="QhE5mVCc">(<a href="#ref-QhE5mVCc" role="doc-biblioref">2111</a>)</span></p>
<h3 data-number="14.8.1" id="keywords-8"><span class="header-section-number">14.8.1</span> Keywords</h3>
<ul>
<li>epitope prediction</li>
<li>vaccine development.</li>
</ul>
<h3 data-number="14.8.2" id="main-findings-7"><span class="header-section-number">14.8.2</span> Main Findings</h3>
<p>This study harnesses bioinformatic profiling to predict the potential of
COV2 viral proteins to be presented on MHC I and II and to form linear
B-cell epitopes. These estimates suggest a T-cell antigenic profile
distinct from SARS-CoV or MERS-CoV, identify focused regions of the
virus with a high density of predicted epitopes, and provide preliminary
evidence for adaptive immune pressure in the genetic evolution of the
virus.</p>
<h3 data-number="14.8.3" id="limitations-7"><span class="header-section-number">14.8.3</span> Limitations</h3>
<p>While the study performs a comprehensive analysis of potential epitopes
within the virus genome, the analysis relies solely on bioinformatic
prediction to examine MHC binding affinity and B-cell epitope potential
and does not capture the immunogenicity or recognition of
these epitopes. Future experimental validation in data from patients
infected with SARS-CoV-2 will be important to validate and refine these
findings. Thus some of the potential conclusions stated, including viral
evolution toward lower immunogenicity or a dominant role for CD4+
T-cells rather than CD8+ T-cells in viral clearance, require further
valiadtion.</p>
<h3 data-number="14.8.4" id="significance-5"><span class="header-section-number">14.8.4</span> Significance</h3>
<p>These findings may help direct peptide vaccine design toward relevant
epitopes and provide intriguing evidence of viral evolution in response
to immune pressure.</p>
<h3 data-number="14.8.5" id="credit-7"><span class="header-section-number">14.8.5</span> Credit</h3>
<p><em>Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.9" id="single-cell-analysis-of-ace2-expression-in-human-kidneys-and-bladders-reveals-a-potential-route-of-2019-ncov-infection"><span class="header-section-number">14.9</span> Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection</h2>
<p>Lin et al. <em>bioRxiv</em> <span class="citation" data-cites="16Xj27QYj">(<a href="#ref-16Xj27QYj" role="doc-biblioref">2110</a>)</span></p>
<h3 data-number="14.9.1" id="keywords-9"><span class="header-section-number">14.9.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>ACE2</li>
<li>scRNAseq</li>
<li>kidney</li>
<li>bladder</li>
<li>public dataset</li>
</ul>
<h3 data-number="14.9.2" id="main-findings-8"><span class="header-section-number">14.9.2</span> Main Findings</h3>
<ul>
<li><p>To investigate the possible cause of kidney damage in 2019-nCoV
patients, authors used published kidney and bladder cell atlas data
(GSE131685, GSE108097; 3 healthy donors each) as well as an
unpublished kidney single-cell RNA-Seq data (in-house from 2
transplant donors) to evaluate ACE2 gene expressions in all cell
types of healthy kidneys and bladders.</p></li>
<li><p>They find enriched expression of ACE2 transcript in all subtypes of
proximal tubule cells of kidney, with 5%-15% of both straight and
convoluted proximal tubule cells expressing ACE2.</p></li>
<li><p>They also find detectable levels of ACE2 in bladder epithelial
cells, noting expression from around 1.5% of cells in the outer
layer umbrella cells of the bladder epithelium and decreasing in the
basal cells.</p></li>
<li><p>Importantly endothelial or immune cells in kidney/bladder do not
express ACE2.</p></li>
</ul>
<h3 data-number="14.9.3" id="limitations-8"><span class="header-section-number">14.9.3</span> Limitations</h3>
<ul>
<li><p>This study primarily characterizes ACE2 expression (amongst other
genes) from a small healthy-donor dataset, and will benefit from
supporting data in (expired) patient samples to show functional
viral damage. ACE2 transcript does not necessarily translate to
viral permissiveness in kidney/bladder epithelia or cytokine
release.</p></li>
<li><p>This study focuses on only healthy tissue; it will be useful to
analyze kidney/bladder epithelial ACE2 expression under inflammatory
conditions or in patients with underlying kidney conditions.</p></li>
<li><p>Given what is known about protease TMPRSS2 expression during
SARS-CoV-2 infection, ACE2+TMPRSS2+ double-positive cell
identification would be useful in these datasets.</p></li>
</ul>
<h3 data-number="14.9.4" id="significance-6"><span class="header-section-number">14.9.4</span> Significance</h3>
<ul>
<li><p>ACE2 protein is spatially restricted to brush border of proximal
tubules and in bladder umbrella cells <span class="citation" data-cites="1FE1YGsaV">(<a href="#ref-1FE1YGsaV" role="doc-biblioref">67</a>)</span>, such cells in direct
contact with viral particles are likely to be highly sensitive to
viral-induced damage.</p></li>
<li><p>SARS-CoV and MERS-CoV have been shown to be detected in urine of
patients and associate with higher mortality <span class="citation" data-cites="DMaiZBE1 iObQU2el">(<a href="#ref-DMaiZBE1" role="doc-biblioref">2109</a>, <a href="#ref-iObQU2el" role="doc-biblioref">2112</a>)</span>, thus worth
understanding kidney damage and resultant immune response in
SARS-CoV-2 as well.</p></li>
<li><p>This study argues for a potential mode of viral infectivity and
resultant inflammatory responses in these tissue in addition to
reported infectivity in the lung and digestive system, which is
supported by clinical data showing acute and early kidney
complications in 2019-nCoV patients <span class="citation" data-cites="1Acv5UMSt">(<a href="#ref-1Acv5UMSt" role="doc-biblioref">2108</a>)</span>.</p></li>
<li><p>Clinically, thus very important to track urinary CoVID-19 shedding
as well as study acute kidney injury-related co-morbidities.</p></li>
</ul>
<h3 data-number="14.9.5" id="credit-8"><span class="header-section-number">14.9.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.10" id="neutrophil-to-lymphocyte-ratio-predicts-severe-illness-patients-with-2019-novel-coronavirus-in-the-early-stage"><span class="header-section-number">14.10</span> Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage</h2>
<p>Liu et al. <em>medRxiv</em> <span class="citation" data-cites="gK5e7Yhl">(<a href="#ref-gK5e7Yhl" role="doc-biblioref">2113</a>)</span></p>
<h3 data-number="14.10.1" id="keywords-10"><span class="header-section-number">14.10.1</span> Keywords</h3>
<ul>
<li>severe disease</li>
<li>pneumonia</li>
<li>lymphocytes</li>
<li>neutrophils</li>
</ul>
<h3 data-number="14.10.2" id="main-findings-9"><span class="header-section-number">14.10.2</span> Main Findings</h3>
<p>This study aimed to find prognostic biomarkers of COVID-19
pneumonia severity. Sixty-one (61) patients with COVID-19 treated in
January at a hospital in Beijing, China were included. On average,
patients were seen within 5 days from illness onset. Samples were
collected on admission; and then patients were monitored for the
development of severe illness with a median follow-up of 10 days].</p>
<p>Patients were grouped as “mild” (N=44) or “moderate/severe” (N=17)
according to symptoms on admission and compared for different
clinical/laboratory features. “Moderate/severe” patients were
significantly older (median of 56 years old, compared to 41 years old).
Whereas comorbidies rates were largely similar between the groups,
except for hypertension, which was more frequent in the severe group (p=
0.056). ‘Severe’ patients had higher counts of neutrophils, and serum
glucose levels; but lower lymphocyte counts, sodium and serum chlorine
levels. The ratio of neutrophils to lymphocytes (NLR) was also higher
for the ‘severe’ group. ‘Severe’ patients had a higher rate of bacterial
infections (and antibiotic treatment) and received more intensive
respiratory support and treatment.</p>
<p>26 clinical/laboratory variables were used to select NLR and age as the
best predictors of the severe disease. Predictive cutoffs for a severe
illness as NLR ≥ 3.13 or age ≥ 50 years.</p>
<h3 data-number="14.10.3" id="limitations-9"><span class="header-section-number">14.10.3</span> Limitations</h3>
<p>Identification of early biomarkers is important for
making clinical decisions, but large sample size and validation cohorts
are necessary to confirm findings. It is worth noting that patients
classified as “mild” showed pneumonia by imaging and fever, and in
accordance with current classifications this would be consistent with
“moderate” cases. Hence it would be more appropriate to refer to the
groups as “moderate” vs “severe/critical”. Furthermore, there are
several limitations that could impact the interpretation of the results:
e.g. classification of patients was based on symptoms presented on
admission and not based on disease progression, small sample size,
especially the number of ‘severe’ cases (with no deaths among these
patients). Given the small sample size, the proposed NLR and age cut
offs might not hold for a slightly different set of patients. For
example, in a study of &gt;400 patients, ‘non-severe’ and ‘severe’ NLR
were 3.2 and 5.5, respectively <span class="citation" data-cites="6Y81C4P3">(<a href="#ref-6Y81C4P3" role="doc-biblioref">2114</a>)</span>.</p>
<h3 data-number="14.10.4" id="credit-9"><span class="header-section-number">14.10.4</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.11" id="characteristics-of-lymphocyte-subsets-and-cytokines-in-peripheral-blood-of-123-hospitalized-patients-with-2019-novel-coronavirus-pneumonia-ncp"><span class="header-section-number">14.11</span> Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)</h2>
<p>Wan et. al. <em>medRxiv</em> <span class="citation" data-cites="P4assQHj">(<a href="#ref-P4assQHj" role="doc-biblioref">2115</a>)</span></p>
<h3 data-number="14.11.1" id="keywords-11"><span class="header-section-number">14.11.1</span> Keywords</h3>
<ul>
<li>Cytokines</li>
<li>lymphocyte subsets</li>
<li>CD8 + T</li>
<li>B cells</li>
<li>NK cells,</li>
<li>PBMCs</li>
<li>IL-6</li>
<li>IL-10</li>
</ul>
<h3 data-number="14.11.2" id="main-findings-10"><span class="header-section-number">14.11.2</span> Main Findings</h3>
<p>The authors analyzed lymphocyte subsets and cytokines of 102 patients
with mild disease and 21 with severe disease. CD8+T cells and CD4+T
cells were significantly reduced in both cohort. particularly in severe
patients. The cytokines IL6 and IL10 were significantly elevated in
severe patients as compared to mild. No significant differences were
observed in frequency of B cells and NK cells.</p>
<p>The authors argue that the measurement of T cell frequencies and
cytokine levels of IL6 and IL10 can be used to predict progression of
disease from Mild to severe Cov-2 infection.</p>
<h3 data-number="14.11.3" id="limitations-10"><span class="header-section-number">14.11.3</span> Limitations</h3>
<p>The study demonstrates in a limited cohort
similar associations to several other reported studies. The authors
didn’t compare the changes in lymphocyte and cytokine with healthy
individual (Covid-19 Negative) rather used an internal standard value.
The recently preprint in LANCET shows The degree of lymphopenia and a
pro-inflammatory cytokine storm is higher in severe COVID-19 patients
than in mild cases, and is associated with the disease severity <span class="citation" data-cites="zskgSnle">(<a href="#ref-zskgSnle" role="doc-biblioref">2116</a>)</span>.</p>
<h3 data-number="14.11.4" id="significance-7"><span class="header-section-number">14.11.4</span> Significance</h3>
<p>This translational data identifies key cytokines and lymphopenia
associated with disease severity although mechanism and key cellular
players are still unknown. Higher level IL-6 production in severe
patient suggests potential role of Tocilizumab (anti-IL6R) biologic
although clinical trial will be necessary.</p>
<h3 data-number="14.11.5" id="credit-10"><span class="header-section-number">14.11.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.12" id="epidemiological-and-clinical-characteristics-of-17-hospitalized-patients-with-2019-novel-coronavirus-infections-outside-wuhan-china"><span class="header-section-number">14.12</span> Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China</h2>
<p>Li et al. <em>medRxiv</em> <span class="citation" data-cites="TXlNzmXd">(<a href="#ref-TXlNzmXd" role="doc-biblioref">2117</a>)</span></p>
<h3 data-number="14.12.1" id="keywords-12"><span class="header-section-number">14.12.1</span> Keywords</h3>
<ul>
<li>epidemiology</li>
<li>clinical characteristics</li>
</ul>
<h3 data-number="14.12.2" id="major-findings"><span class="header-section-number">14.12.2</span> Major Findings</h3>
<p>These authors looked at 17 hospitalized patients with COVID-19
confirmed by RT-PCR in Dazhou, Sichuan. Patients were admitted between
January 22 and February 10 and the final data were collected on February
11. Of the 17 patients, 12 remained hospitalized while 5 were discharged
after meeting national standards. The authors observed no differences
based on the sex of the patients but found that the discharged patients
were younger in age (p = 0.026) and had higher lymphocyte counts (p =
0.005) and monocyte counts (p = 0.019) upon admission.</p>
<h3 data-number="14.12.3" id="limitations-11"><span class="header-section-number">14.12.3</span> Limitations</h3>
<p>This study is limited in the sample size of the study and the last
data collection point was only one day after some of the patients were
admitted.</p>
<h3 data-number="14.12.4" id="significance-8"><span class="header-section-number">14.12.4</span> Significance</h3>
<p>These findings have been somewhat supported by subsequent studies
that show that older age and an immunocompromised state are more likely
to result in a more severe clinical course with COVID-19. However, other
studies have been published that report on larger numbers of cases.</p>
<h3 data-number="14.12.5" id="credit-11"><span class="header-section-number">14.12.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.13" id="ace2-expression-in-kidney-and-testis-may-cause-kidney-and-testis-damage-after-2019-ncov-infection"><span class="header-section-number">14.13</span> ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection</h2>
<p><span class="citation" data-cites="15K5Ovntj">(<a href="#ref-15K5Ovntj" role="doc-biblioref">2118</a>)</span></p>
<h3 data-number="14.13.1" id="keywords-13"><span class="header-section-number">14.13.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>kidney</li>
<li>testis</li>
<li>ACE2</li>
<li>scRNAseq</li>
</ul>
<h3 data-number="14.13.2" id="main-findings-11"><span class="header-section-number">14.13.2</span> Main Findings</h3>
<ul>
<li><p>Study used online datasets (scRNAseq GSE131685, scRNAseq GSE107585,
Human Protein Atlas, GTEx portal, CCLE) to analyze ACE2 expression
in different human organs.</p></li>
<li><p>Study re-analyzed three clinical datasets (n=6, n=99, and n=41) to
show 3~10% of 2019-nCoV patients present with abnormal renal
function.</p></li>
<li><p>results indicate ACE2 highly expressed in renal tubular cells,
Leydig cells and seminiferous ductal cells of testis.</p></li>
</ul>
<h3 data-number="14.13.3" id="limitations-12"><span class="header-section-number">14.13.3</span> Limitations</h3>
<ul>
<li><p>Very preliminary transcript/protein dataset analysis in healthy
cohorts; does not necessarily translate to actual viral tropism and
permissiveness.</p></li>
<li><p>Clinically, would be important to determine with larger longitudinal
dataset if SARS-CoV-2 infection changes sperm quality or testicular
inflammation.</p></li>
<li><p>Similarly, would be important to determine if simultaneous HBV or
syphilis infection and orchitis impacts SARS-CoV-2 severity.</p></li>
<li><p>Examination and follow-up of renal function and viral orchitis/sperm
quality of CoVID-19 patients not done in this preliminary study.</p></li>
</ul>
<h3 data-number="14.13.4" id="significance-9"><span class="header-section-number">14.13.4</span> Significance</h3>
<ul>
<li><p>Kidney ACE2 result supports other concurrent sequencing studies
<span class="citation" data-cites="16Xj27QYj">(<a href="#ref-16Xj27QYj" role="doc-biblioref">2110</a>)</span> and clinical reports
of abnormal renal function or even kidney damage in patients
infected with 2019-nCoV
<span class="citation" data-cites="1Acv5UMSt">(<a href="#ref-1Acv5UMSt" role="doc-biblioref">2108</a>)</span>.</p></li>
<li><p>High ACE2 expression in testis suggests potential tropism of the
virus to testicular tissues and indicates potential risks for male
fertility. Viral orchitis reported for SARS-CoV previously [1],
but no clear evidence so far of infertility in SARS, MERS or
CoVID-19 patients.</p></li>
</ul>
<h3 data-number="14.13.5" id="credit-12"><span class="header-section-number">14.13.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.14" id="aberrant-pathogenic-gm-csf-t-cells-and-inflammatory-cd14cd16-monocytes-in-severe-pulmonary-syndrome-patients-of-a-new-coronavirus"><span class="header-section-number">14.14</span> Aberrant pathogenic GM-CSF+ T cells and inflammatory CD14+CD16+ monocytes in severe pulmonary syndrome patients of a new coronavirus</h2>
<p><span class="citation" data-cites="Bgo7Cmj3">(<a href="#ref-Bgo7Cmj3" role="doc-biblioref">2119</a>)</span></p>
<h3 data-number="14.14.1" id="keywords-14"><span class="header-section-number">14.14.1</span> Keywords</h3>
<ul>
<li>immunopathology</li>
<li>Th1</li>
<li>inflammatory monocytes</li>
<li>GM-CSF</li>
<li>IFN-γ</li>
<li>IL-6</li>
</ul>
<h3 data-number="14.14.2" id="main-findings-12"><span class="header-section-number">14.14.2</span> Main Findings</h3>
<p>The authors of this study sought to characterize the immune
mechanism causing severe pulmonary disease and mortality in 2019-nCoV
(COVID-19) patients. Peripheral blood was collected from hospitalized
ICU (n=12) and non-ICU (n=21) patients with confirmed 2019-nCoV and from
healthy controls (n=10) in The First Affiliated Hospital of University
of Science and Technology China (Hefei, Anhui). Immune analysis was
conducted by flow cytometry. 2019-nCoV patients had decreased
lymphocyte, monocyte, and CD4 T cell counts compared to healthy
controls. ICU patients had fewer lymphocytes than non-ICU patients. CD4
T cells of 2019-nCoV patients expressed higher levels of activation
markers (OX40, CD69, CD38, CD44) and exhaustion markers (PD-1 and Tim3)
than those of healthy controls. CD4 cells of ICU patients expressed
significantly higher levels of OX40, PD-1, and Tim3 than those of
non-ICU patients. 2019-nCoV patients had higher percentages of CD4 T
cells co-expressing GM-CSF and IL-6 compared to healthy controls, while
ICU patients had a markedly higher percentage of GM-CSF+ IFN-γ+ CD4 T
cells than non-ICU patients. The CD4 T cells of nCoV patients and
healthy controls showed no differences in TNF-α secretion.</p>
<p>The CD8 T cells of 2019-nCoV patients also showed higher expression of
activation markers CD69, CD38, and CD44, as well as exhaustion markers
PD-1 and Tim3, compared to healthy controls. CD8 T cells of ICU patients
expressed higher levels of GM-CSF than those of non-ICU patients and
healthy controls. No IL-6 or TNF-α was found in the CD8 T cells of any
group. There were no differences in numbers of NK cells or B cells in
2019-nCoV patients and healthy controls, nor was there any GM-CSF or
IL-6 secretion from these cells in either group.</p>
<p>Percentages of CD14+ CD16+ GM-CSF+ and CD14+ CD16+ IL-6+ inflammatory
monocytes were significantly increased in nCoV patients compared to
healthy controls; in particular, patients in the ICU had greater
percentages of CD14+ CD16+ IL-6+ monocytes than non-ICU patients. The
authors suggest that in 2019-nCoV patients, pathogenic Th1 cells produce
GM-CSF, recruiting CD14+ CD16+ inflammatory monocytes that secrete high
levels of IL-6. These may enter pulmonary circulation and damage lung
tissue while initiating the cytokine storm that causes mortality in
severe cases. This is consistent with the cytokine storm seen in similar
coronaviruses, as IL-6, IFN-γ, and GM-CSF are key inflammatory mediators
seen in patients with SARS-CoV-1 and MERS-CoV.</p>
<h3 data-number="14.14.3" id="limitations-13"><span class="header-section-number">14.14.3</span> Limitations</h3>
<p>Though the results of this study open
questions for further investigation, this is an early study on a small
cohort of patients, and as such there are a number of limitations. The
study included only 12 ICU patients and 21 non-ICU patients, and ideally
would be repeated with a much larger patient cohort. Though the authors
make claims about differences in lymphocyte and monocyte counts between
patients and healthy controls, they did not report baseline laboratory
findings for the control group. Additionally, severity of disease was
classified based on whether or not patients were in the ICU. It would be
interesting to contextualize the authors’ immunological findings with
more specific metrics of disease severity or time course. Noting
mortality, time from disease onset, pre-existing conditions, or severity
of lung pathology in post-mortem tissue samples would paint a fuller
picture of how to assess risk level and the relationship between
severity of disease and immunopathology. Another limitation is the
selection of cytokines and immune markers for analysis, as the selection
criteria were based on the cell subsets and cytokine storm typically
seen in SARS-CoV-1 and MERS-CoV patients. Unbiased cytokine screens and
immune profiling may reveal novel therapeutic targets that were not
included in this study.</p>
<h3 data-number="14.14.4" id="significance-10"><span class="header-section-number">14.14.4</span> Significance</h3>
<p>This study identifies potential therapeutic targets
that could prevent acute respiratory disease syndrome (ARDS) and
mortality in patients most severely affected by COVID-19. The authors
propose testing monoclonal antibodies against IL6-R or GM-CSF to block
recruitment of inflammatory monocytes and the subsequent cytokine storm
in these patients.</p>
<h3 data-number="14.14.5" id="credit-13"><span class="header-section-number">14.14.5</span> Credit</h3>
<p><em>Review by Gabrielle Lubitz as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.15" id="clinical-characteristics-of-2019-novel-infected-coronavirus-pneumoniaa-systemic-review-and-meta-analysis"><span class="header-section-number">14.15</span> Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis</h2>
<p>Qian et al. <em>medRxiv</em>.
<span class="citation" data-cites="4Z1pEioX">(<a href="#ref-4Z1pEioX" role="doc-biblioref">2120</a>)</span></p>
<h3 data-number="14.15.1" id="keywords-15"><span class="header-section-number">14.15.1</span> Keywords</h3>
<ul>
<li>White Blood Cells</li>
<li>Lymphocytes</li>
<li>Neutrophils</li>
</ul>
<h3 data-number="14.15.2" id="main-findings-13"><span class="header-section-number">14.15.2</span> Main Findings</h3>
<p>The authors performed a meta analysis of literature on clinical,
laboratory and radiologic characteristics of patients presenting with
pneumonia related to SARSCoV2 infection, published up to Feb 6 2020.
They found that symptoms that were mostly consistent among studies were
sore throat, headache, diarrhea and rhinorrhea. Fever, cough, malaise
and muscle pain were highly variable across studies. Leukopenia (mostly
lymphocytopenia) and increased white blood cells were highly variable
across studies. They identified three most common patterns seen on CT
scan, but there was high variability across studies. Consistently across
the studies examined, the authors found that about 75% of patients need
supplemental oxygen therapy, about 23% mechanical ventilation and about
5% extracorporeal membrane oxygenation (ECMO). The authors calculated a
staggering pooled mortality incidence of 78% for these patients.</p>
<h3 data-number="14.15.3" id="limitations-14"><span class="header-section-number">14.15.3</span> Limitations</h3>
<p>The authors mention that the total number of studies included in this
meta analysis is nine, however they also mentioned that only three
studies reported individual patient data. It is overall unclear how many
patients in total were included in their analysis. This is mostly
relevant as they reported an incredibly high mortality (78%) and mention
an absolute number of deaths of 26 cases overall. It is not clear from
their report how the mortality rate was calculated.</p>
<p>The data is based on reports from China and mostly from the Wuhan area,
which somewhat limits the overall generalizability and applicability of
these results.</p>
<h3 data-number="14.15.4" id="significance-11"><span class="header-section-number">14.15.4</span> Significance</h3>
<p>This meta analysis offers some important data for clinicians to refer to
when dealing with patients with COVID-19 and specifically with
pneumonia. It is very helpful to set expectations about the course of
the disease.</p>
<h3 data-number="14.15.5" id="credit-14"><span class="header-section-number">14.15.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.16" id="longitudinal-characteristics-of-lymphocyte-responses-and-cytokine-profiles-in-the-peripheral-blood-of-sars-cov-2-infected-patients"><span class="header-section-number">14.16</span> Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients</h2>
<p>Liu et al. <em>medRxiv</em> <span class="citation" data-cites="zTGzuiTI">(<a href="#ref-zTGzuiTI" role="doc-biblioref">2121</a>)</span></p>
<h3 data-number="14.16.1" id="keywords-16"><span class="header-section-number">14.16.1</span> Keywords</h3>
<ul>
<li>Lymphopenia</li>
<li>Neutrophil to CD8 T cell ratio (N8R)</li>
<li>inflammatory cytokines</li>
</ul>
<h3 data-number="14.16.2" id="main-findings-14"><span class="header-section-number">14.16.2</span> Main Findings</h3>
<p>Liu et al. enrolled a cohort of 40 patients from Wuhan
including 27 mild cases and 13 severe cases of COVID-19. They performed
a 16-day kinetic analysis of peripheral blood from time of disease
onset. Patients in the severe group were older (medium age of 59.7,
compared to 48.7 in mild group) and more likely to have hypertension as
a co-morbidity. Lymphopenia was observed in 44.4% of the mild patients
and 84.6% of the severe patients. Lymphopenia was due to low T cell
count, specially CD8 T cells. Severe patients showed higher neutrophil
counts and an increase of cytokines in the serum (IL2, IL6, IL10 and
IFNγ). The authors measured several other clinical laboratory parameters
were also higher in severe cases compared to mild, but concluded that
neutrophil to CD8 T cell ratio (N8R) as the best prognostic factor to
identify the severe cases compared to other receiver operating
characteristic (ROC).</p>
<h3 data-number="14.16.3" id="limitations-15"><span class="header-section-number">14.16.3</span> Limitations</h3>
<p>This was a small cohort (N=40), and two of the patients
initially included in the severe group (N=13) passed away and were
excluded from the analysis due to lack of longitudinal data. However, it
would be most important to be able to identify patients with severe
disease with higher odds of dying. It seems that the different time
points analyzed relate to hospital admission, which the authors describe
as disease onset. The time between first symptoms and first data points
is not described. It would have been important to analyze how the
different measured parameters change according to health condition, and
not just time (but that would require a larger cohort). The predictive
value of N8R compared to the more commonly used NLR needs to be assessed
in other independent and larger cohorts. Lastly, it is important to note
that pneumonia was detected in patients included in the “mild” group,
but according to the Chinese Clinical Guidance for COVID-19 Pneumonia
Diagnosis and Treatment (7th edition) this group should be considered
“moderate”.</p>
<h3 data-number="14.16.4" id="significance-12"><span class="header-section-number">14.16.4</span> Significance</h3>
<p>Lymphopenia and cytokine storm have been described to be
detrimental in many other infections including SARS-CoV1 and MERS-CoV.
However, it was necessary to confirm that this dramatic immune response
was also observed in the SARS-CoV2 infected patients. These results and
further validation of the N8R ratio as a predictor of disease severity
will contribute for the management of COVID19 patients and potential
development of therapies.</p>
<h3 data-number="14.16.5" id="credit-15"><span class="header-section-number">14.16.5</span> Credit</h3>
<p><em>Review by Pauline Hamon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.17" id="clinical-and-immunologic-features-in-severe-and-moderate-forms-of-coronavirus-disease-2019"><span class="header-section-number">14.17</span> Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019</h2>
<p>Chen et al. <em>medRxiv</em> <span class="citation" data-cites="1EIEx8Qe4">(<a href="#ref-1EIEx8Qe4" role="doc-biblioref">2122</a>)</span></p>
<h3 data-number="14.17.1" id="keywords-17"><span class="header-section-number">14.17.1</span> Keywords</h3>
<ul>
<li>severe disease</li>
<li>lymphocytes</li>
<li>cytokines</li>
<li>IFNy</li>
<li>CD4 Tcells</li>
<li>HLA-DR CD8</li>
<li>Tcells</li>
</ul>
<h3 data-number="14.17.2" id="main-findings-15"><span class="header-section-number">14.17.2</span> Main Findings</h3>
<p>This study retrospectively evaluated clinical, laboratory,
hematological, biochemical and immunologic data from 21 subjects
admitted to the hospital in Wuhan, China (late December/January) with
confirmed SARS-CoV-2 infection. The aim of the study was to compare
‘severe’ (n=11, ~64 years old) and ‘moderate’ (n=10, ~51 years old)
COVID-19 cases. Disease severity was defined by patients’ blood oxygen
level and respiratory output. They were classified as ‘severe’ if SpO2
93% or respiratory rates 30 per min.</p>
<p>In terms of the clinical laboratory measures, ‘severe’ patients had
higher CRP and ferritin, alanine and aspartate aminotransferases, and
lactate dehydrogenase but lower albumin concentrations.</p>
<p>The authors then compared plasma cytokine levels (ELISA) and immune cell
populations (PBMCs, Flow Cytometry). ‘Severe’ cases had higher levels of
IL-2R, IL-10, TNFa, and IL-6 (marginally significant). For the immune
cell counts, ‘severe’ group had higher neutrophils, HLA-DR+ CD8 T cells
and total B cells; and lower total lymphocytes, CD4 and CD8 T cells
(except for HLA-DR+), CD45RA Tregs, and IFNy-expressing CD4 T cells. No
significant differences were observed for IL-8, counts of NK cells,
CD45+RO Tregs, IFNy-expressing CD8 T and NK cells.</p>
<h3 data-number="14.17.3" id="limitations-16"><span class="header-section-number">14.17.3</span> Limitations</h3>
<p>Several potential limitations should be noted: 1) Blood samples were
collected 2 days post hospital admission and no data on viral loads were
available; 2) Most patients were administered medications (e.g.
corticosteroids), which could have affected lymphocyte counts.
Medications are briefly mentioned in the text of the manuscript; authors
should include medications as part of Table 1. 3) ‘Severe’ cases were
significantly older and 4/11 ‘severe’ patients died within 20 days.
Authors should consider a sensitivity analysis of biomarkers with the
adjustment for patients’ age.</p>
<h3 data-number="14.17.4" id="significance-13"><span class="header-section-number">14.17.4</span> Significance</h3>
<p>Although the sample size was small, this paper presented a broad range
of clinical, biochemical, and immunologic data on patients with
COVID-19. One of the main findings is that SARS-CoV-2 may affect T
lymphocytes, primarily CD4+ T cells, resulting in decreased IFNy
production. Potentially, diminished T lymphocytes and elevated cytokines
can serve as biomarkers of severity of COVID-19.</p>
<h3 data-number="14.17.5" id="credit-16"><span class="header-section-number">14.17.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.18" id="sars-cov-2-and-sars-cov-spike-rbd-structure-and-receptor-binding-comparison-and-potential-implications-on-neutralizing-antibody-and-vaccine-development"><span class="header-section-number">14.18</span> SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development</h2>
<p>Sun et al. <em>bioRxiv</em> <span class="citation" data-cites="41WAtZmB">(<a href="#ref-41WAtZmB" role="doc-biblioref">913</a>)</span></p>
<h3 data-number="14.18.1" id="keywords-18"><span class="header-section-number">14.18.1</span> Keywords</h3>
<ul>
<li>SARS-CoV</li>
<li>SARS-CoV-2</li>
<li>ACE2</li>
<li>Spike (S) protein</li>
<li>receptor binding domain (RBD)</li>
<li>receptor binding motif (RBM)</li>
<li>neutralizing antibody</li>
</ul>
<h3 data-number="14.18.2" id="main-findings-16"><span class="header-section-number">14.18.2</span> Main Findings</h3>
<p>This study compared the structure of SARS-CoV and SARS-CoV-2 Spike (S)
protein receptor binding domain (RBD) and interactions with ACE2 using
computational modeling, and interrogated cross-reactivity and
cross-neutralization of SARS-CoV-2 by antibodies against SARS-CoV. While
SARS-CoV and SARS-CoV-2 have over 70 % sequence homology and share the
same human receptor ACE2, the receptor binding motif (RBM) is only 50%
homologous.</p>
<p>Computational prediction of the SARS-CoV-2 and ACE2 interactions based
on the previous crystal structure data of SARS-CoV, and measurement of
binding affinities against human ACE2 using recombinant SARS-CoV and
SARS-CoV-2 S1 peptides, demonstrated similar binding of the two S1
peptides to ACE2, explaining the similar transmissibility of SARS-CoV
and SARS-CoV-2 and consistent with previous data (Wall et al Cell
2020).</p>
<p>The neutralization activity of SARS-CoV-specific rabbit polyclonal
antibodies were about two-order of magnitude less efficient to
neutralize SARS-CoV-2 than SARS-CoV, and four potently neutralizing
monoclonal antibodies against SARS-CoV had poor binding and neutralizing
activity against SARS-CoV-2. In contrast, 3 poor SARS-CoV-binding
monoclonal antibodies show some efficiency to bind and neutralize
SARS-CoV-2. The results suggest that that antibodies to more conserved
regions outside the RBM motif might possess better cross-protective
neutralizing activities between two strains.</p>
<h3 data-number="14.18.3" id="limitations-17"><span class="header-section-number">14.18.3</span> Limitations</h3>
<p>It would have been helpful to show the epitopes recognized by the
monoclonal antibodies tested on both SARS-CoV, SARS-CoV-2 to be able to
make predictions for induction of broadly neutralizing antibodies. The
data on monoclonal antibody competition with ACE2 for binding to
SARS-CoV RBD should have also included binding on SARS-CoV2, especially
for the three monoclonal antibodies that showed neutralization activity
for SARS-CoV2. Because of the less homology in RBM sequences between
viruses, it still may be possible that these antibodies would recognize
the ACE2 RBD in SARS-CoV-2.</p>
<h3 data-number="14.18.4" id="significance-14"><span class="header-section-number">14.18.4</span> Significance</h3>
<p>It is noteworthy that immunization to mice and rabbit with SARS-CoV S1
or RBD protein could induce monoclonal antibodies to cross-bind and
cross-neutralize SARS-CoV-2 even if they are not ACE2-blocking. If these
types of antibodies could be found in human survivors or in the
asymptomatic populations as well, it might suggest that exposure to
previous Coronavirus strains could have induced cross-neutralizing
antibodies and resulted in the protection from severe symptoms in some
cases of SARS-CoV2.</p>
<h3 data-number="14.18.5" id="credit-17"><span class="header-section-number">14.18.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.19" id="protection-of-rhesus-macaque-from-sars-coronavirus-challenge-by-recombinant-adenovirus-vaccine"><span class="header-section-number">14.19</span> Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine</h2>
<p>Chen et al. <em>bioRxiv</em> <span class="citation" data-cites="NPxs7bbq">(<a href="#ref-NPxs7bbq" role="doc-biblioref">2123</a>)</span></p>
<h3 data-number="14.19.1" id="keywords-19"><span class="header-section-number">14.19.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-1</li>
<li>rhesus macaque</li>
<li>recombinant adenovirus vaccine</li>
</ul>
<h3 data-number="14.19.2" id="main-findings-17"><span class="header-section-number">14.19.2</span> Main Findings</h3>
<p>Rhesus macaques were immunized intramuscularly twice (week 0 and
week 4) with SV8000 carrying the information to express a S1-orf8
fusion protein and the N protein from the BJ01 strain of SARS-CoV-1.
By week 8, immunized animals had signs of immunological protection
(IgG and neutralization titers) against SARS-CoV-1 and were
protected against challenge with the PUMC-1 strain, with fewer
detectable symptoms of respiratory distress, lower viral load,
shorter periods of viral persistence, and less pathology in the
lungs compared to non-immunized animals.</p>
<h3 data-number="14.19.3" id="limitations-18"><span class="header-section-number">14.19.3</span> Limitations</h3>
<p>The authors should write clearer descriptions of the methods used in
this article. They do not describe how the IgG titers or
neutralization titers were determined. There are some issues with
the presentation of data, for example, in Figure 1a, y-axis should
not be Vmax; forming cells and 1d would benefit from showing error
bars. Furthermore, although I inferred that the animals were
challenged at week 8, the authors did not explicitly detail when the
animals were challenged. The authors should explain the design of
their vaccine, including the choice of antigens and vector. The
authors also do not include a description of the ethical use of
animals in their study.</p>
<h3 data-number="14.19.4" id="significance-15"><span class="header-section-number">14.19.4</span> Significance</h3>
<p>The authors describe a vaccine for SARS-CoV-1 with no discussion of
possible implications for the current SARS-CoV-2 pandemic. Could a
similar vaccine be designed to protect against SARS-CoV-2 and would
the concerns regarding emerging viral mutations that the authors
describe as a limitation for SARS-CoV-1 also be true in the context
of SARS-CoV-2?</p>
<h3 data-number="14.19.5" id="credit-18"><span class="header-section-number">14.19.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.20" id="reduction-and-functional-exhaustion-of-t-cells-in-patients-with-coronavirus-disease-2019-covid-19"><span class="header-section-number">14.20</span> Reduction and Functional Exhaustion of T cells in Patients with Coronavirus Disease 2019 (COVID-19)</h2>
<p><span class="citation" data-cites="f50ahNxF">(<a href="#ref-f50ahNxF" role="doc-biblioref">2124</a>)</span></p>
<h3 data-number="14.20.1" id="keywords-20"><span class="header-section-number">14.20.1</span> Keywords</h3>
<ul>
<li>T cell exhaustion</li>
<li>T cell lymphopenia</li>
<li>IL-6</li>
<li>IL-10</li>
<li>TNF-α</li>
</ul>
<h3 data-number="14.20.2" id="main-findings-18"><span class="header-section-number">14.20.2</span> Main Findings</h3>
<p>Based on a retrospective study of 522 COVID patients and 40
healthy controls from two hospitals in Wuhan, China, authors show both
age-dependent and clinical severity-dependent decrease in T cell numbers
with elderly patients and patients who are in ICU-care showing the most
dramatic decrease in T cell counts. Cytokine profiling of COVID patients
reveal that TNF-α, IL-6 and IL-10 are increased in infected patients
with patients in the ICU showing the highest levels. Interestingly,
these three cytokine levels were inversely correlated with T cell counts
and such inverse relationship was preserved throughout the disease
progression. Surface staining of exhaustion markers (PD-1 and Tim-3) and
flow cytometry of stained peripheral blood of 14 patients and 3 healthy
volunteers demonstrate that T cells of COVID patients have increased
expression of PD-1 with patients in ICU having the highest number of
CD8<sup>+</sup>PD-1<sup>+</sup> cells than their counterparts in non-ICU groups.</p>
<h3 data-number="14.20.3" id="limitations-19"><span class="header-section-number">14.20.3</span> Limitations</h3>
<p>Compared to the number of patients, number of control (n=
40) is small and is not controlled for age. Additional data linking
inflammatory cytokines and the quality of the adaptive response
including humoral and antigen specific T cell response is much needed. T
cell exhaustion study relies on marker-dependent labeling of T cell
functionality of a very limited sample size (n=17)—a
functional/mechanistic study of these T cells from PBMCs would have
bolstered their claims.</p>
<h3 data-number="14.20.4" id="significance-16"><span class="header-section-number">14.20.4</span> Significance</h3>
<p>Limited but contains interesting
implications. It is already known in literature that in the context of
acute respiratory viral infections CD8 T cells exhibit exhaustion-like
phenotypes which further underscores the importance of mechanistic
studies that can elucidate how COVID infection leads to lymphopenia and
T cell exhaustion-like phenotype.</p>
<p>However, as authors have noted, the data does point to an interesting
question: How these inflammatory cytokines (TNF-α, IL-6 and IL-10)
correlate with or affect effective viral immunity and what types of
cells produce these cytokines? Answering that question will help us
refine our targets for immune-modulatory therapies especially in
patients suffering from cytokine storms.</p>
<h3 data-number="14.20.5" id="credit-19"><span class="header-section-number">14.20.5</span> Credit</h3>
<p><em>This review by Chang Moon was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.21" id="clinical-characteristics-of-25-death-cases-infected-with-covid-19-pneumonia-a-retrospective-review-of-medical-records-in-a-single-medical-center-wuhan-china"><span class="header-section-number">14.21</span> Clinical Characteristics of 25 death cases infected with COVID-19 pneumonia: a retrospective review of medical records in a single medical center, Wuhan, China</h2>
<p><span class="citation" data-cites="EwNi3WJq">(<a href="#ref-EwNi3WJq" role="doc-biblioref">2125</a>)</span></p>
<h3 data-number="14.21.1" id="keywords-21"><span class="header-section-number">14.21.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>pneumonia</li>
<li>hypertension</li>
<li>diabetes</li>
<li>biomarker</li>
<li>neutrophilia</li>
<li>lymphopenia</li>
</ul>
<h3 data-number="14.21.2" id="main-findings-19"><span class="header-section-number">14.21.2</span> Main Findings</h3>
<p>Most common chronic conditions among 25 patients that died from
COVID-19 related respiratory failure were hypertension (64%) and
diabetes (40%). Disease progression was marked by progressive organ
failure, starting first with lung dysfunction, then heart (e.g.
increased cTnI and pro-BNP), followed by kidney (e.g. increased BUN,
Cr), and liver (e.g. ALT, AST). 72% of patients had neutrophilia and 88%
also had lymphopenia. General markers of inflammation were also
increased (e.g. PCT, D-Dimer, CRP, LDH, and SAA).</p>
<h3 data-number="14.21.3" id="limitations-20"><span class="header-section-number">14.21.3</span> Limitations</h3>
<p>The limitations of this study include small sample size and
lack of measurements for some tests for several patients. This study
would also have been stronger with comparison of the same measurements
to patients suffering from less severe disease to further validate and
correlate proposed biomarkers with disease severity.</p>
<h3 data-number="14.21.4" id="significance-17"><span class="header-section-number">14.21.4</span> Significance</h3>
<p>This study identifies chronic conditions (i.e. hypertension
and diabetes) that strongly correlates with disease severity. In
addition to general markers of inflammation, the authors also identify
concomitant neutrophilia and lymphopenia among their cohort of patients.
This is a potentially interesting immunological finding because we would
typically expect increased lymphocytes during a viral infection.
Neutrophilia may also be contributing to cytokine storm. In addition,
PCT was elevated in 90.5% of patients, suggesting a role for sepsis or
secondary bacterial infection in COVID-19 related respiratory failure.</p>
<h3 data-number="14.21.5" id="credit-20"><span class="header-section-number">14.21.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.22" id="sars-cov-2-infection-does-not-significantly-cause-acute-renal-injury-an-analysis-of-116-hospitalized-patients-with-covid-19-in-a-single-hospital-wuhan-china"><span class="header-section-number">14.22</span> SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China</h2>
<p><span class="citation" data-cites="KG5Lg65z">(<a href="#ref-KG5Lg65z" role="doc-biblioref">2126</a>)</span></p>
<h3 data-number="14.22.1" id="keywords-22"><span class="header-section-number">14.22.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>kidney</li>
<li>clinical</li>
<li>longitudinal</li>
</ul>
<h3 data-number="14.22.2" id="main-findings-20"><span class="header-section-number">14.22.2</span> Main Findings</h3>
<ul>
<li><p>Clinical data from 116 hospitalized CoVID-19 patients analyzed over
4 weeks for correlation with renal injury. Comorbidities included
chronic renal failure (CRF) in 5 patients (4.3%).</p></li>
<li><p>10.8% of patients with no prior kidney disease showed elevations in
blood urea or creatinine, and 7.2% of patients with no prior kidney
disease showed albuminuria.</p></li>
<li><p>Patients with pre-existing CRF underwent continuous renal
replacement therapy (CRRT) alongside CoVID-19 treatment. Renal
functions remained stable in these patients.</p></li>
<li><p>All 5 patients with CRF survived CoVID-19 therapy without
progression to ARDS or worsening of CRF.</p></li>
</ul>
<h3 data-number="14.22.3" id="limitations-21"><span class="header-section-number">14.22.3</span> Limitations</h3>
<ul>
<li><p>Renal injury biomarkers in patients with incipient kidney
abnormalities not tabulated separately, making overall data hard to
interpret. It will be critical to separately examine kidney function
(BUN, urine creatinine and eGFR) in patients that developed any
kidney abnormalities (7.2~10.8% of cohort).</p></li>
<li><p>No information on type of CoVID-19 therapy used across cohort; will
be useful to correlate how treatment modality influences kidney
function (and other parameters).</p></li>
<li><p>Invokes previous clinical-correlation studies that indicate low
instances of kidney damage<span class="citation" data-cites="VRoblZFJ bgrzawWp">(<a href="#ref-VRoblZFJ" role="doc-biblioref">2127</a>, <a href="#ref-bgrzawWp" role="doc-biblioref">2128</a>)</span>, but those studies did not track
longitudinal urine samples for acute renal injury markers and viral
shedding.</p></li>
<li><p>CRRT in patients with CRF is standard therapy irrespective of
CoVID-19 status; it will be important to compare clinical parameters
of these patients (n=5) with virus-naïve CRF patients (none in this
study) to make any meaningful conclusions.</p></li>
</ul>
<h3 data-number="14.22.4" id="significance-18"><span class="header-section-number">14.22.4</span> Significance</h3>
<ul>
<li><p>This study argues that renal impairment is uncommon in CoVID-19 and
not associated with high mortaility, in stark contrast with a
concurrent study <span class="citation" data-cites="1Acv5UMSt">(<a href="#ref-1Acv5UMSt" role="doc-biblioref">2108</a>)</span>.
If supported by further studies, this argues kidney impairment is
secondary to cytokine storm/inflammation-induced organ failure, and
not due to direct viral replication.</p></li>
<li><p>Will be important to comprehensively characterize large-datasets of
CoVID-19 patients to conclude if kidney function actively disrupted
due to viral infection.</p></li>
</ul>
<h3 data-number="14.22.5" id="credit-21"><span class="header-section-number">14.22.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.23" id="potential-t-cell-and-b-cell-epitopes-of-2019-ncov"><span class="header-section-number">14.23</span> Potential T-cell and B-Cell Epitopes of 2019-nCoV</h2>
<p><span class="citation" data-cites="hdSafq1o">(<a href="#ref-hdSafq1o" role="doc-biblioref">2129</a>)</span></p>
<h3 data-number="14.23.1" id="keywords-23"><span class="header-section-number">14.23.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>vaccine</li>
<li>epitopes</li>
<li>spike protein</li>
<li>MHC-I</li>
<li>MHC-II</li>
<li>neutralizing antibodies</li>
<li>ACE2</li>
</ul>
<h3 data-number="14.23.2" id="main-findings-21"><span class="header-section-number">14.23.2</span> Main Findings</h3>
<p>The authors use 2 neural network algorithms, NetMHCpan4 and
MARIA, to identify regions within the COVID-19 genome that are
presentable by HLA. They identify 405 viral epitopes that are
presentable on MHC-I and MHC-II and validate using known epitopes from
SARS-CoV. To determine whether immune surveillance drives viral
mutations to evade MHC presentation, the authors analyzed 68 viral
genomes from 4 continents. They identified 93 point mutations that
occurred preferentially in regions predicted to be presented by MHC-I
(p=0.02) suggesting viral evolution to evade CD8 T-cell mediated
killing. 2 nonsense mutations were also identified that resulted in loss
of presentation of an associated antigen (FGDSVEEVL) predicted to be
good antigen for presentation across multiple HLA alleles.</p>
<p>To identify potential sites of neutralizing antibody binding, the
authors used homology modeling to the SARS-CoV’s spike protein (S
protein) to determine the putative structure of the CoV2 spike protein.
They used Discotope2 to identify antibody binding sites on the protein
surface in both the down and up conformations of the S protein. The
authors validate this approach by first identifying antibody binding
site in SARS-CoV S protein. In both the down and up conformation of the
CoV2 S protein, the authors identified a potential antibody binding site
on the S protein receptor binding domain (RBD) of the ACE2 receptor
(residues 440-460, 494-506). While RBDs in both SARS-CoV and CoV2 spike
proteins may be important for antibody binding, the authors note that
SARS-CoV has larger attack surfaces than CoV2. These results were later
validated on published crystal structures of the CoV2 S protein RBD and
human ACE2. Furthermore, analysis of 68 viral genomes did not identify
any mutations in this potential antibody binding site in CoV2.</p>
<p>Finally, the authors compile a list of potential peptide vaccine
candidates across the viral genome that can be presented by multiple HLA
alleles. Several of the peptides showed homology to SARS-CoV T-cell and
B-cell epitopes.</p>
<h3 data-number="14.23.3" id="limitations-22"><span class="header-section-number">14.23.3</span> Limitations</h3>
<p>While the authors used computational methods of validation,
primarily through multiple comparisons to published SARS-CoV structures
and epitopes, future work should include experimental validation of
putative T-cell and B-cell epitopes.</p>
<h3 data-number="14.23.4" id="significance-19"><span class="header-section-number">14.23.4</span> Significance</h3>
<p>The authors identified potential T-cell and B-cell epitopes
that may be good candidates for peptide based vaccines against CoV2.
They also made interesting observations in comparing SARS-CoV and CoV2
potential antibody binding sites, noting that SARS-CoV had larger attack
surfaces for potential neutralizing antibody binding. One of the
highlights of this paper was the authors’ mutation analysis of 68 viral
genomes from 4 continents. This analysis not only validated their
computational method for identifying T-cell epitopes, but showed that
immune surveillance likely drives viral mutation in MHC-I binding
peptides. The smaller attack surface may point to potential mechanisms
of immune evasion by CoV2. However, absence of mutations in the RBD of
CoV2 and the small number of mutations in peptides presentable to T
cells suggests that vaccines against multiple epitopes could still
elicit robust immunity against CoV2.</p>
<h3 data-number="14.23.5" id="credit-22"><span class="header-section-number">14.23.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.24" id="structure-function-and-antigenicity-of-the-sarscov-2-spike-glycoprotein"><span class="header-section-number">14.24</span> Structure, Function, and Antigenicity of the SARSCoV-2 Spike Glycoprotein</h2>
<p>Walls et al. <em>bioRxiv</em>. <span class="citation" data-cites="8xnxgpag">(<a href="#ref-8xnxgpag" role="doc-biblioref">2130</a>)</span> now <span class="citation" data-cites="qcVbT0w4">(<a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>)</span></p>
<h3 data-number="14.24.1" id="keywords-24"><span class="header-section-number">14.24.1</span> Keywords</h3>
<ul>
<li>binding affinity</li>
<li>antigenicity</li>
<li>neutralizing antibody</li>
</ul>
<h3 data-number="14.24.2" id="main-findings-22"><span class="header-section-number">14.24.2</span> Main Findings</h3>
<p>The authors highlight a human angiotensin-converting enzyme 2
(hACE2), as a potential receptor used by the current Severe Acute
respiratory syndrome coronavirus-2 (SARS-CoV-2) as a host factor that
allows the virus target human cells. This virus-host interaction
facilitates the infection of human cells with a high affinity comparable
with SARS-CoV. The authors propose this mechanism as a probable
explanation of the efficient transmission of SARS-CoV-2 between humans.
Besides, Walls and colleagues described SARS-CoV-2 S glycoprotein S by
Cryo-EM along with neutralizing polyclonal response against SAR-CoV-2 S
from mice immunized with SAR-CoV and blocking SAR-CoV-2 S-mediated entry
into VeroE6 infected cells.**</p>
<h3 data-number="14.24.3" id="limitations-23"><span class="header-section-number">14.24.3</span> Limitations</h3>
<p>The SARS-CoV-2 depends on the cell factors ACE2 and TMPRSS2, this
last, according to a recent manuscript by Markus Hoffman et al., <em>Cell</em>,
2020. The authors used green monkey (VeroE6) and hamster (BHK) cell
lines in the experiments to drive its conclusions to humans; however, it
is well known the caucasian colon adenocarcinoma human cell line
(CaCo-2), highly express the hACE2 receptor as the TMPRSS2 protease as
well. In humans, ACE2 protein is highly expressed in the
gastrointestinal tract, which again, makes the CaCo-2 cell line suitable
for the following SARS-CoV-2 studies.</p>
<h3 data-number="14.24.4" id="significance-20"><span class="header-section-number">14.24.4</span> Significance</h3>
<p>The results propose a functional receptor used by SARS-CoV-2 to
infect humans worldwide and defining two distinct conformations of spike
(S) glycoprotein by cryogenic electron microscopy (Cryo-EM). This study
might help establish a precedent for initial drug design and treatment
of the current global human coronavirus epidemic.</p>
<h3 data-number="14.24.5" id="credit-23"><span class="header-section-number">14.24.5</span> Credit</h3>
<p><em>Review by postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.25" id="breadth-of-concomitant-immune-responses-underpinning-viral-clearance-and-patient-recovery-in-a-non-severe-case-of-covid-19"><span class="header-section-number">14.25</span> Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19</h2>
<p>Thevarajan et al. <em>medRxiv</em> <span class="citation" data-cites="QdimT8mN">(<a href="#ref-QdimT8mN" role="doc-biblioref">2131</a>)</span></p>
<h3 data-number="14.25.1" id="keywords-25"><span class="header-section-number">14.25.1</span> Keywords</h3>
<ul>
<li>IgG</li>
<li>IgM</li>
<li>TfH cells</li>
<li>NK cells</li>
<li>SNP</li>
</ul>
<h3 data-number="14.25.2" id="main-findings-23"><span class="header-section-number">14.25.2</span> Main Findings</h3>
<p>The authors characterized the immune response in peripheral blood of a
47-year old COVID-19 patient.</p>
<p>SARS-CoV2 was detected in nasopharyngeal swab, sputum and faeces
samples, but not in urine, rectal swab, whole blood or throat swab. 7
days after symptom onset, the nasopharyngeal swab test turned negative,
at day 10 the radiography infiltrates were cleared and at day 13 the
patient became asymptomatic.</p>
<p>Immunofluorescence staining shows from day 7 the presence of
<strong>COVID-19-binding IgG and IgM</strong> antibodies in plasma, that increase
until day 20.</p>
<p>Flow cytometry on whole blood reveals a plasmablast peak at day 8, a
gradual increase in T follicular helper cells, stable HLA-DR<sup>+</sup> NK
frequencies and decreased monocyte frequencies compared to healthy
counterparts. The expression of CD38 and HLA-DR peaked on T cells at D9
and was associated with higher production of cytotoxic mediators by
CD8<sup>+</sup> T cells.</p>
<p>IL-6 and IL-8 were undetectable in plasma.</p>
<p>The authors further highlight the presence of the <strong>IFITM3
SNP-rs12252-C/C variant</strong> in this patient, which is associated with
higher susceptibility to influenza virus.</p>
<h3 data-number="14.25.3" id="limitations-24"><span class="header-section-number">14.25.3</span> Limitations</h3>
<p>These results need to be confirmed in additional patients.</p>
<p>COVID-19 patients have increased infiltration of macrophages in their
lungs <span class="citation" data-cites="tVkKuIJt">(<a href="#ref-tVkKuIJt" role="doc-biblioref">2132</a>)</span>. Monitoring monocyte proportions in blood earlier in the
disease might help to evaluate their eventual migration to the lungs.</p>
<p>The stable concentration of HLA-DR<sup>+</sup> NK cells in blood from day 7 is
not sufficient to rule out NK cell activation upon SARS-CoV2 infection.
In response to influenza A virus, NK cells express higher levels of
activation markers CD69 and CD38, proliferate better and display higher
cytotoxicity <span class="citation" data-cites="87CJN2CG">(<a href="#ref-87CJN2CG" role="doc-biblioref">2133</a>)</span>. Assessing these parameters in COVID-19 patients is
required to better understand NK cell role in clearing this infection.</p>
<p>Neutralization potential of the COVID-19-binding IgG and IgM antibodies
should be assessed in future studies.</p>
<p>This patient was able to clear the virus, while presenting a SNP
associated with severe outcome following influenza infection. The
association between this SNP and outcome upon SARS-CoV2 infection should
be further investigated.</p>
<h3 data-number="14.25.4" id="significance-21"><span class="header-section-number">14.25.4</span> Significance</h3>
<p>This study is among the first to describe the appearance of
COVID-19-binding IgG and IgM antibodies upon infection. The emergence of
new serological assays might contribute to monitor more precisely the
seroconversion kinetics of COVID-19 patients <span class="citation" data-cites="yJW1RVQb">(<a href="#ref-yJW1RVQb" role="doc-biblioref">2134</a>)</span>. Further association
studies between IFITM3 SNP-rs12252-C/C variant and clinical data might
help to refine the COVID-19 outcome prediction tools.</p>
<h3 data-number="14.25.5" id="credit-24"><span class="header-section-number">14.25.5</span> Credit</h3>
<p><em>Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.26" id="the-landscape-of-lung-bronchoalveolar-immune-cells-in-covid-19-revealed-by-single-cell-rna-sequencing"><span class="header-section-number">14.26</span> The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing</h2>
<p>Liao et al. <em>medRxiv</em> <span class="citation" data-cites="tVkKuIJt">(<a href="#ref-tVkKuIJt" role="doc-biblioref">2132</a>)</span></p>
<h3 data-number="14.26.1" id="keywords-26"><span class="header-section-number">14.26.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>Broncho-alveolar lavage</li>
<li>macrophages</li>
<li>NK cells</li>
<li>T cells</li>
<li>cytokine storm</li>
<li>scRNAseq</li>
</ul>
<h3 data-number="14.26.2" id="main-findings-24"><span class="header-section-number">14.26.2</span> Main Findings</h3>
<p>The authors performed single-cell RNA sequencing (scRNAseq) on
bronchoalveolar lavage fluid (BAL) from 6 COVID-19 patients (n=3 mild
cases, n=3 severe cases). Data was compared to previously generated
scRNAseq data from healthy donor lung tissue (n=8).</p>
<p>Clustering analysis of the 6 patients revealed distinct immune cell
organization between mild and severe disease. Specifically, they found
that transcriptional clusters annotated as tissue resident alveolar
macrophages were strongly reduced while monocytes-derived FCN1<sup>+</sup>SPP1<sup>+</sup>
inflammatory macrophages dominated the BAL of patients with severe
COVID19 diseases. They show that inflammatory macrophages upregulated
interferon-signaling genes, monocytes recruiting chemokines including
CCL2, CCL3, CCL4 as well as IL-6, TNF, IL-8 and profibrotic cytokine
TGF-β, while alveolar macrophages expressed lipid metabolism genes, such
as PPARG.</p>
<p>The lymphoid compartment was overall enriched in lungs from patients.
Clonally expanded CD8 T cells were enriched in mild cases suggesting
that CD8 T cells contribute to viral clearance as in Flu infection,
whereas proliferating T cells were enriched in severe cases.</p>
<p>SARS-CoV-2 viral transcripts were detected in severe patients, but
considered here as ambient contaminations.</p>
<h3 data-number="14.26.3" id="limitations-25"><span class="header-section-number">14.26.3</span> Limitations</h3>
<p>These results are based on samples from 6 patients and should therefore
be confirmed in the future in additional patients. Longitudinal
monitoring of BAL during disease progression or resolution would have
been most useful.</p>
<p>The mechanisms underlying the skewing of the macrophage compartment in
patients towards inflammatory macrophages should be investigated in
future studies.</p>
<p>Deeper characterization of the lymphoid subsets is required. The
composition of the “proliferating” cluster and how these cells differ
from conventional T cell clusters should be assessed. NK and CD8 T cell
transcriptomic profile, in particular the expression of cytotoxic
mediator and immune checkpoint transcripts, should be compared between
healthy and diseased lesions.</p>
<h3 data-number="14.26.4" id="significance-22"><span class="header-section-number">14.26.4</span> Significance</h3>
<p>COVID-19 induces a robust inflammatory cytokine storm in patients that
contributes to severe lung tissue damage and ARDS <span class="citation" data-cites="vasHgjGH">(<a href="#ref-vasHgjGH" role="doc-biblioref">2135</a>)</span>. Accumulation of
monocyte-derived inflammatory macrophages at the expense of Alveolar
macrophages known to play an anti-inflammatory role following
respiratory viral infection, in part through the PPARγ pathway <span class="citation" data-cites="9GEyqic3 WBN5f37b">(<a href="#ref-9GEyqic3" role="doc-biblioref">2136</a>, <a href="#ref-WBN5f37b" role="doc-biblioref">2137</a>)</span> are
likely contributing to lung tissue injuries. These data suggest that
reduction of monocyte accumulation in the lung tissues could help
modulate COVID-19-induced inflammation. Further analysis of lymphoid
subsets is required to understand the contribution of adaptive immunity
to disease outcome.</p>
<h3 data-number="14.26.5" id="credit-25"><span class="header-section-number">14.26.5</span> Credit</h3>
<p><em>Review by Bérengère Salomé and Assaf Magen as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.27" id="can-routine-laboratory-tests-discriminate-2019-novel-coronavirus-infected-pneumonia-from-other-community-acquired-pneumonia"><span class="header-section-number">14.27</span> Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia?</h2>
<p>Pan et al. <em>medRxiv</em> <span class="citation" data-cites="FOVDaThy">(<a href="#ref-FOVDaThy" role="doc-biblioref">2138</a>)</span></p>
<h3 data-number="14.27.1" id="keywords-27"><span class="header-section-number">14.27.1</span> Keywords</h3>
<ul>
<li>Routine laboratory testing</li>
</ul>
<h3 data-number="14.27.2" id="main-findings-25"><span class="header-section-number">14.27.2</span> Main Findings</h3>
<p>In an attempt to use standard laboratory testing for the discrimination
between “Novel Coronavirus Infected Pneumonia” (NCIP) and a usual
community acquired pneumonia (CAP), the authors compared laboratory
testing results of 84 NCIP patients with those of a historical group of
316 CAP patients from 2018 naturally COVID-19 negative. The authors
describe significantly lower white blood- as well as red blood- and
platelet counts in NCIP patients. When analyzing differential blood
counts, lower absolute counts were measured in all subsets of NCIP
patients. With regard to clinical chemistry parameters, they found
increased AST and bilirubin in NCIP patients as compared to CAP
patients.</p>
<h3 data-number="14.27.3" id="limitations-26"><span class="header-section-number">14.27.3</span> Limitations</h3>
<p>The authors claim to describe a simple method to rapidly assess a
pre-test probability for NCIP. However, the study has substantial
weakpoints. The deviation in clinical laboratory values in NCIP patients
described here can usually be observed in severely ill patients. The
authors do not comment on how severely ill the patients tested here were
in comparison to the historical control. Thus, the conclusion that the
tests discriminate between CAP and NCIP lacks justification.</p>
<h3 data-number="14.27.4" id="significance-23"><span class="header-section-number">14.27.4</span> Significance</h3>
<p>The article strives to compare initial laboratory testing results in
patients with COVID-19 pneumonia as compared to patients with a usual
community acquired pneumonia. The implications of this study for the
current clinical situation seem restricted due to a lack in clinical
information and the use of a control group that might not be
appropriate.</p>
<h3 data-number="14.27.5" id="credit-26"><span class="header-section-number">14.27.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.28" id="correlation-analysis-between-disease-severity-and-inflammation-related-parameters-in-patients-with-covid-19-pneumonia"><span class="header-section-number">14.28</span> Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia</h2>
<p><span class="citation" data-cites="1Du40IOOH">(<a href="#ref-1Du40IOOH" role="doc-biblioref">2139</a>)</span></p>
<h3 data-number="14.28.1" id="keywords-28"><span class="header-section-number">14.28.1</span> Keywords</h3>
<ul>
<li>cytokine</li>
<li>COVID-19 pneumonia</li>
<li>severity</li>
<li>disease progression</li>
</ul>
<h3 data-number="14.28.2" id="main-findings-26"><span class="header-section-number">14.28.2</span> Main Findings</h3>
<p>This study is a cross-sectional analysis of 100 patients with
COVID-19 pneumonia, divided into mild (n = 34), severe (n = 34), and
critical (n = 32) disease status based on clinical definitions.</p>
<p>The criteria used to define disease severity are as follows:</p>
<ol type="1">
<li><p><em>Severe</em> – any of the following: respiratory distress or respiratory
rate ≥ 30 respirations/minute; oxygen saturation ≤ 93% at rest;
oxygen partial pressure (PaO2)/oxygen concentration (FiO2) in
arterial blood ≤ 300mmHg, progression of disease on imaging
to &gt;50% lung involvement in the short term.</p></li>
<li><p><em>Critical</em> – any of the following: respiratory failure that requires
mechanical ventilation; shock; other organ failure that requires
treatment in the ICU.</p></li>
<li><p>Patients with pneumonia who test positive for COVID-19 who do not
have the symptoms delineated above are considered <em>mild</em>.</p></li>
</ol>
<p>Peripheral blood inflammatory markers were correlated to disease status.
Disease severity was significantly associated with levels of IL-2R,
IL-6, IL-8, IL-10, TNF-α, CRP, ferroprotein, and procalcitonin. Total
WBC count, lymphocyte count, neutrophil count, and eosinophil count were
also significantly correlated with disease status. Since this is a
retrospective, cross-sectional study of clinical laboratory values,
these data may be extrapolated for clinical decision making, but without
studies of underlying cellular causes of these changes this study does
not contribute to a deeper understanding of SARS-CoV-2 interactions with
the immune system.</p>
<p>It is also notable that the mean age of patients in the mild group was
significantly different from the mean ages of patients designated as
severe or critical (p &lt; 0.001). The mean patient age was not
significantly different between the severe and critical groups. However,
IL-6, IL-8, procalcitonin (Table 2), CRP, ferroprotein (Figure 3A, 3B),
WBC count, and neutrophil count (Figure 4A, 4B) were all significantly
elevated in the critical group compared to severe. These data suggest
underlying differences in COVID-19 progression that is unrelated to age.</p>
<h3 data-number="14.28.3" id="significance-24"><span class="header-section-number">14.28.3</span> Significance</h3>
<p>Given the inflammatory profile outlined in this study,
patients who have mild or severe COVID-19 pneumonia, who <em>also</em> have any
elevations in the inflammatory biomarkers listed above, should be
closely monitored for potential progression to critical status.</p>
<h3 data-number="14.28.4" id="credit-27"><span class="header-section-number">14.28.4</span> Credit</h3>
<p><em>This review by JJF was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.29" id="an-effective-ctl-peptide-vaccine-for-ebola-zaire-based-on-survivors-cd8-targeting-of-a-particular-nucleocapsid-protein-epitope-with-potential-implications-for-covid-19-vaccine-design"><span class="header-section-number">14.29</span> An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors’ CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design</h2>
<p>Herst et al. <em>bioRxiv</em> <span class="citation" data-cites="1DTLMIDoM">(<a href="#ref-1DTLMIDoM" role="doc-biblioref">2140</a>)</span></p>
<h3 data-number="14.29.1" id="keywords-29"><span class="header-section-number">14.29.1</span> Keywords</h3>
<ul>
<li>Peptide vaccine</li>
<li>Ebolavirus</li>
<li>nucleocapsid</li>
<li>epitope</li>
<li>vaccine design</li>
<li>microsphere</li>
</ul>
<h3 data-number="14.29.2" id="main-findings-27"><span class="header-section-number">14.29.2</span> Main Findings</h3>
<p>Vaccination of mice with a single dose of a 9-amino-acid peptide NP44-52
located in a conserved region of ebolavirus (EBOV) nucleocapsid protein
(NP) confers CD8+ T-cell-mediated immunity against mouse adapted EBOV
(maEBOV). Bioinformatic analyses predict multiple conserved CD8+ T cell
epitopes in the SARS-CoV-2 NP, suggesting that a similar approach may be
feasible for vaccine design against SARS-CoV-2.</p>
<p>The authors focus on a site within a 20-peptide region of EBOV NP which
was commonly targeted by CD8+ T cells in a group of EBOV survivors
carrying the HLA-A*30:01:01 allele. To justify the testing of specific
vaccine epitopes in a mouse challenge setting, the authors cite known
examples of human pathogen-derived peptide antigens that are also
recognized by C57BL/6 mice, as well as existing data surrounding known
mouse immunogenicity of peptides related to this EBOV NP region. Testing
3 distinct 9mer peptides over an 11 amino-acid window and comparing to
vaccination with the 11mer with a T-cell reactivity readout demonstrated
that optimizing peptide length and position for immunogenicity may be
crucial, likely due to suboptimal peptide processing and MHC-class-I
loading.</p>
<p>Vaccines for maEBOV challenge studies were constructed by packaging
NP44-52 in d,l poly(lactic-co-glycolic) acid microspheres. CpG was also
packaged within the microspheres, while Monophosphoryl Lipid A (a TLR4
ligand) was added to the injectate solution. A second peptide consisting
of a predicted MHC-II epitope from the EBOV VG19 protein was added using
a separate population of microspheres, and the formulation was injected
by intraperitoneal administration. The vaccine was protective against a
range of maEBOV doses up to at least 10,000 PFU. Survival was
anticorrelated with levels of IL6, MCP-1 (CCL2), IL9, and GM-CSF, which
recapitulated trends seen in human EBOV infection.</p>
<p>While HLA-A*30:01:01 is only present in a minority of humans, the
authors state that MHC binding algorithms predict NP44-52 to be a strong
binder of a set of more common HLA-A*02 alleles. The authors predict
that a peptide vaccine based on the proposed formulation could elicit
responses in up to 50% of people in Sudan or 30% of people in North
America.</p>
<p>SARS-CoV-2 NP, meanwhile, has conserved regions which may provide
peptide-vaccine candidates. Scanning the SARS-CoV-2 NP sequence for
HLA-binding 9mers identified 53 peptides with predicted binding affinity
&lt; 500nM, including peptides that are predicted to bind to HLA-class-I
alleles of 97% of humans, 7 of which have previously been tested
<em>in-vitro</em>.</p>
<p>The results support previously appreciated correlations between certain
cytokines and disease severity, specifically IL6 which relates to
multiple trial therapies. Prediction of HLA-class-I binding of
SARS-CoV-2 NP peptides suggests the plausibility of a peptide vaccine
targeting conserved regions of SARS-CoV-2 NP although further validation
in previously infected patient samples will be essential.</p>
<h3 data-number="14.29.3" id="credit-28"><span class="header-section-number">14.29.3</span> Credit</h3>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.30" id="epitope-based-peptide-vaccines-predicted-against-novel-coronavirus-disease-caused-by-sars-cov-2"><span class="header-section-number">14.30</span> Epitope-based peptide vaccines predicted against novel coronavirus disease caused by SARS-CoV-2</h2>
<p>Li et al. <em>bioRxiv</em>. <span class="citation" data-cites="9spK49tt">(<a href="#ref-9spK49tt" role="doc-biblioref">2141</a>)</span></p>
<h3 data-number="14.30.1" id="keywords-30"><span class="header-section-number">14.30.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2</li>
<li>immune-informatics</li>
<li>vaccine design</li>
<li>T cell epitope</li>
<li>B cell epitope</li>
</ul>
<h3 data-number="14.30.2" id="main-findings-28"><span class="header-section-number">14.30.2</span> Main Findings</h3>
<p>This study employs a series of bioinformatic
pipelines to identify T and B cell epitopes on spike (S) protein of
SARS-CoV-2 and assess their properties for vaccine potential. To
identify B cell epitopes, they assessed structural accessibility,
hydrophilicity, and beta-turn and flexibility which are all factors that
promote their targeting by antibodies. To identify T cell epitopes, they
filtered for peptides with high antigenicity score and capacity to bind
3 or more MHC alleles. Using the protein digest server, they also
demonstrated that their identified T and B cell epitopes are stable,
having multiple non-digesting enzymes per epitope. Epitopes were also
determined to be non-allergenic and non-toxin as assessed by Allergen FP
1.0 and ToxinPred, respectively. For T cell epitopes, they assessed the
strength of epitope-HLA interaction via PepSite. Overall, they predict
four B cell and eleven T cell epitopes (two MHC I and nine MHC II
binding) to pass stringent computational thresholds as candidates for
vaccine development. Furthermore, they performed sequence alignment
between all identified SARS-CoV-2 S protein mutations and predicted
epitopes, and showed that the epitopes are conserved across 134 isolates
from 38 locations worldwide. However, they report that these conserved
epitopes may soon become obsolete given the known mutation rate of
related SARS-CoV is estimated to be 4x10-4/site/year, underscoring the
urgency of anti-viral vaccine development.</p>
<h3 data-number="14.30.3" id="limitations-27"><span class="header-section-number">14.30.3</span> Limitations</h3>
<p>While spike (S) protein may have a
critical role in viral entry into host cells and their epitope
prediction criterion were comprehensive, this study did not examine
other candidate SARS-CoV-2 proteins. This point is particularly
important given that a single epitope may not be sufficient to induce
robust immune memory, and recent approaches involve multi-epitope
vaccine design. Furthermore, their study only included a direct
implementation of various published methods, but did not validate
individual bioinformatic tools with controls to demonstrate robustness.
Finally, it is critical that these predicted epitopes are experimentally
validated before any conclusions can be drawn about their potential as
vaccine candidates or their clinical efficacy.</p>
<h3 data-number="14.30.4" id="significance-25"><span class="header-section-number">14.30.4</span> Significance</h3>
<p>This study provides a computational framework to
rapidly identify epitopes that may serve as potential vaccine candidates
for treating SARS-CoV-2.</p>
<h3 data-number="14.30.5" id="credit-29"><span class="header-section-number">14.30.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.31" id="the-definition-and-risks-of-cytokine-release-syndrome-like-in-11-covid-19-infected-pneumonia-critically-ill-patients-disease-characteristics-and-retrospective-analysis"><span class="header-section-number">14.31</span> The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis</h2>
<p>Wang Jr. et al. <em>medRxiv</em>.
<span class="citation" data-cites="qg1bmQng">(<a href="#ref-qg1bmQng" role="doc-biblioref">2142</a>)</span></p>
<h3 data-number="14.31.1" id="keywords-31"><span class="header-section-number">14.31.1</span> Keywords</h3>
<ul>
<li>Cytokine release syndrome (CRS)</li>
<li>biomarkers</li>
<li>ARDS</li>
<li>IL-6</li>
<li>lymphopenia</li>
</ul>
<h3 data-number="14.31.2" id="main-findings-29"><span class="header-section-number">14.31.2</span> Main Findings</h3>
<p>This study describes the occurrence of a cytokine release syndrome-like
(CRSL) toxicity in ICU patients with COVID-19 pneumonia. The median time
from first symptom to acute respiratory distress syndrome (ARDS) was 10
days. All patients had decreased CD3, CD4 and CD8 cells, and a
significant increase of serum IL-6. Furthermore, 91% had decreased NK
cells. The changes in IL-6 levels preceded those in CD4 and CD8 cell
counts. All of these parameters correlated with the area of pulmonary
inflammation in CT scan images. Mechanical ventilation increased the
numbers of CD4 and CD8 cells, while decreasing the levels of IL-6, and
improving the immunological parameters.</p>
<h3 data-number="14.31.3" id="limitations-28"><span class="header-section-number">14.31.3</span> Limitations</h3>
<p>The number of patients included in this retrospective single center
study is small (n=11), and the follow-up period very short (25 days).
Eight of the eleven patients were described as having CRSL, and were
treated by intubation (7) or ECMO (2). Nine patients were still in the
intensive care unit at the time of publication of this article, so their
disease outcome is unknown.</p>
<h3 data-number="14.31.4" id="significance-26"><span class="header-section-number">14.31.4</span> Significance</h3>
<p>The authors define a cytokine release syndrome-like toxicity in patients
with COVID-19 with clinical radiological and immunological criteria: 1)
decrease of circulating CD4, CD8 and NK cells; 2) substantial increase
of IL-6 in peripheral blood; 3) continuous fever; 4) organ and tissue
damage. This event seems to occur very often in critically ill patients
with COVID-19 pneumonia. Interestingly, the increase of IL-6 in the
peripheral blood preceded other laboratory alterations, thus, IL-6 might
be an early biomarker for the severity of COVID-19 pneumonia. The
manuscript will require considerable editing for organization and
clarity.</p>
<h3 data-number="14.31.5" id="credit-30"><span class="header-section-number">14.31.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.32" id="clinical-characteristics-of-36-non-survivors-with-covid-19-in-wuhan-china"><span class="header-section-number">14.32</span> Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China</h2>
<p>Huang et al. <em>medRxiv</em>. <span class="citation" data-cites="YODgfrtM">(<a href="#ref-YODgfrtM" role="doc-biblioref">2143</a>)</span></p>
<h3 data-number="14.32.1" id="keywords-32"><span class="header-section-number">14.32.1</span> Keywords</h3>
<ul>
<li>Clinical Characteristics</li>
<li>Non Survivors</li>
<li>retrospective study</li>
</ul>
<h3 data-number="14.32.2" id="main-findings-30"><span class="header-section-number">14.32.2</span> Main Findings</h3>
<p>This is a simple study reporting clinical
characteristics of patients who did not survive COVID-19. All patients
(mean age=69.22 years) had acute respiratory distress syndrome (ARDS)
and their median time from onset to ARDS was 11 days. The median time
from onset to death was 17 days. Most patients were older male (70%
male) with co-morbidities and only 11 % were smokers. 75% patients
showed bilateral pneumonia. Many patients had chronic diseases,
including hypertension (58.33%). cardiovascular disease (22.22%) and
diabetes (19.44%). Typical clinical feature measured in these patients
includes lymphopenia and elevated markers of inflammation.</p>
<h3 data-number="14.32.3" id="limitations-29"><span class="header-section-number">14.32.3</span> Limitations</h3>
<p>As noted by the authors, the conclusions of this study are
very limited because this is single-centered study focusing on a small
cohort of patients who did not survive. Many clinical parameters
observed by the authors (such* as increase levels of serum CRP, PCT,
IL-6) have also been described in other COVID19 patients who survived
the infection</p>
<h3 data-number="14.32.4" id="significance-27"><span class="header-section-number">14.32.4</span> Significance</h3>
<p>This study is essentially descriptive and may be useful for
clinical teams monitoring COVID19 patients.</p>
<h3 data-number="14.32.5" id="credit-31"><span class="header-section-number">14.32.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.33" id="risk-factors-related-to-hepatic-injury-in-patients-with-corona-virus-disease-2019"><span class="header-section-number">14.33</span> Risk Factors Related to Hepatic Injury in Patients with Corona Virus Disease 2019</h2>
<p><span class="citation" data-cites="5CGbn9Jj">(<a href="#ref-5CGbn9Jj" role="doc-biblioref">2144</a>)</span></p>
<h3 data-number="14.33.1" id="keywords-33"><span class="header-section-number">14.33.1</span> Keywords</h3>
<ul>
<li>COVID-related Hepatic Injury</li>
</ul>
<h3 data-number="14.33.2" id="main-findings-31"><span class="header-section-number">14.33.2</span> Main Findings</h3>
<p>Based on a retrospective study of 85 hospitalized COVID
patients in a Beijing hospital, authors showed that patients with
elevated ALT levels (n = 33) were characterized by significantly higher
levels of lactic acid and CRP as well as lymphopenia and hypoalbuminemia
compared to their counterparts with normal ALT levels. Proportion of
severe and critical patients in the ALT elevation group was
significantly higher than that of normal ALT group. Multivariate
logistic regression performed on clinical factors related to ALT
elevation showed that CRP <span class="math inline">\(\geq\)</span> 20mg/L and low lymphocyte count
(&lt;1.1*10^9 cells/L) were independently related to ALT elevation—a
finding that led the authors to suggest cytokine storm as a major
mechanism of liver damage.</p>
<h3 data-number="14.33.3" id="limitations-30"><span class="header-section-number">14.33.3</span> Limitations</h3>
<p>The article’s most attractive claim that liver damage seen in COVID
patients is caused by cytokine storm (rather than direct infection of
the liver) hinges solely on their multivariate regression analysis.
Without further mechanistic studies a) demonstrating how high levels of
inflammatory cytokines can induce liver damage and b) contrasting types
of liver damage incurred by direct infection of the liver vs.
system-wide elevation of inflammatory cytokines, their claim remains
thin. It is also worth noting that six of their elevated ALT group
(n=33) had a history of liver disease (i.e. HBV infection, alcoholic
liver disease, fatty liver) which can confound their effort to pin down
the cause of hepatic injury to COVID.</p>
<h3 data-number="14.33.4" id="significance-28"><span class="header-section-number">14.33.4</span> Significance</h3>
<p>Limited. This article confirms a rich body
of literature describing liver damage and lymphopenia in COVID patients.</p>
<h3 data-number="14.33.5" id="credit-32"><span class="header-section-number">14.33.5</span> Credit</h3>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.34" id="detectable-serum-sars-cov-2-viral-load-rnaaemia-is-closely-associated-with-drastically-elevated-interleukin-6-il-6-level-in-critically-ill-covid-19-patients"><span class="header-section-number">14.34</span> Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients</h2>
<p><span class="citation" data-cites="T0BDaXkw">(<a href="#ref-T0BDaXkw" role="doc-biblioref">2145</a>)</span></p>
<h3 data-number="14.34.1" id="keywords-34"><span class="header-section-number">14.34.1</span> Keywords</h3>
<ul>
<li>ARDS</li>
<li>interleukin-6 (IL-6)</li>
<li>procalcitonin (PCT)</li>
<li>pro-inflammatory cytokines</li>
<li>SARS-CoV-2 RNAaemia</li>
</ul>
<h3 data-number="14.34.2" id="main-findings-32"><span class="header-section-number">14.34.2</span> Main Findings</h3>
<p>48 adult patients diagnosed with Covid19 according to Chinese guidelines
for Covid19 diagnosis and treatment version 6 were included in this
study. Patients were further sub-divided into three groups based on
clinical symptoms and disease severity: (1) mild, positive Covid19 qPCR
with no or mild clinical symptoms (fever; respiratory; radiological
abnormalities); (2) severe, at least one of the following: shortness of
breath/respiratory rate &gt;30/min, oxygen saturation SaO<sub>2</sub>&lt;93%,
Horowitz index paO2/FiO<sub>2</sub> &lt; 300 mmHg (indicating moderate pulmonary
damage); and (3) critically ill, at least one additional complicating
factor: respiratory failure with need for mechanical ventilation;
systemic shock; multi-organ failure and transfer to ICU. Serum samples
and throat-swaps were collected from all 48 patients enrolled.
SARS-CoV-2 RNA was assessed by qPCR with positive results being defined
as Ct values &lt; 40, and serum interleukin-6 (IL-6) was quantified
using a commercially available detection kit. Briefly, patient
characteristics in this study confirm previous reports suggesting that
higher age and comorbidities are significant risk factors of clinical
severity. Of note, 5 out of 48 of patients (10.41%), all in the
critically ill category, were found to have detectable serum SARS-CoV-2
RNA levels, so-called RNAaemia. Moreover, serum IL-6 levels in these
patients were found to be substantially higher and this correlated with
the presence of detectable SARS-CoV-2 RNA levels. The authors
hypothesize that viral RNA might be released from acutely damages
tissues in moribund patients during the course of Covid19 and that
RNaemia along with IL-6 could potentially be used as a prognostic
marker.</p>
<h3 data-number="14.34.3" id="limitations-31"><span class="header-section-number">14.34.3</span> Limitations</h3>
<p>While this group’s report generally confirms some of the major findings
of a more extensive study, published in early February 2020, <span class="citation" data-cites="vasHgjGH">(<a href="#ref-vasHgjGH" role="doc-biblioref">2135</a>)</span>,
there are limitations that should be taken into account. First, the
number of patients enrolled is relatively small; second, interpretation
of these data would benefit from inclusion of information about study
specifics as well as providing relevant data on the clinical course of
these patients other than the fact that some were admitted to ICU (i.e.
demographics on how many patients needed respiratory support, dialysis,
APACHE Ii/III or other standard ICU scores as robust prognostic markers
for mortality etc). It also remains unclear at which time point the
serum samples were taken, i.e. whether at admission, when the diagnosis
was made or during the course of the hospital stay (and potentially
after onset of therapy, which could have affected both IL-6 and RNA
levels). The methods section lacks important information on the qPCR
protocol employed, including primers and cycling conditions used. From a
technical point of view, Ct values &gt;35 seem somewhat non-specific
(although Ct &lt;40 was defined as the CDC cutoff as well) indicating
that serum RNA levels are probably very low, therefore stressing the
need for highly specific primers and high qPCR efficiency. In addition,
the statistical tests used (t-tests, according to the methods section)
do not seem appropriate as the organ-specific data such as BUN and
troponin T values seem to be not normally distributed across groups (n=
5 RNAaemia+ vs. n= 43 RNAaemia-). Given the range of standard deviations
and the differences in patient sample size, it is difficult to believe
that these data are statistically significantly different.</p>
<h3 data-number="14.34.4" id="significance-29"><span class="header-section-number">14.34.4</span> Significance</h3>
<p>This study is very rudimentary and lacks a lot of relevant clinical
details. However, it corroborates some previously published observations
regarding RNAemia and IL-6 by another group. Generally, regarding future
studies, it would be important to address the question of IL-6 and other
inflammatory cytokine dynamics in relation to Covid19 disease kinetics
(high levels of IL-6, IL-8 and plasma leukotriene were shown to have
prognostic value at the onset of ARDS ; serum IL-2 and IL-15 have been
associated with mortality; reviewed by Chen W &amp; Ware L, Clin Transl Med.
2015 <span class="citation" data-cites="l3CzJ5ni">(<a href="#ref-l3CzJ5ni" role="doc-biblioref">2146</a>)</span>).</p>
<h3 data-number="14.34.5" id="credit-33"><span class="header-section-number">14.34.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.35" id="lymphopenia-predicts-disease-severity-of-covid-19-a-descriptive-and-predictive-study"><span class="header-section-number">14.35</span> Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study</h2>
<p><span class="citation" data-cites="uIwTzZkJ">(<a href="#ref-uIwTzZkJ" role="doc-biblioref">2147</a>)</span></p>
<h3 data-number="14.35.1" id="keywords-35"><span class="header-section-number">14.35.1</span> Keywords</h3>
<ul>
<li>Lymphopenia</li>
</ul>
<h3 data-number="14.35.2" id="main-findings-33"><span class="header-section-number">14.35.2</span> Main Findings</h3>
<p>Based on a retrospective study of 162 COVID patients from a
local hospital in Wuhan, China, the authors show an inverse correlation
between lymphocyte % (LYM%) of patients and their disease severity. The
authors have also tracked LYM% of 70 cases (15 deaths; 15 severe; 40
moderate) throughout the disease progression with fatal cases showing no
recovery of lymphocytes ( &lt;5%) even after 17-19 days post-onset. The
temporal data of LYM % in COVID patients was used to construct a
Time-Lymphocyte% model which is used to categorize and predict patients’
disease severity and progression. The model was validated using 92
hospitalized cases and kappa statistic test was used to assess agreement
between predicted disease severity and the assigned clinical severity (k
= 0.49).</p>
<h3 data-number="14.35.3" id="limitations-32"><span class="header-section-number">14.35.3</span> Limitations</h3>
<p>Time-Lymphocyte % Model (TLM) that authors have proposed as
a predictive model for clinical severity is very simple in its
construction and derives from correlative data of 162 patients. In order
for the model to be of use, it needs validation using a far more robust
data set and possibly a mechanistic study on how COVID leads to
lymphopenia in the first place. In addition, it should be noted that no
statistical test assessing significance of LYM % values between disease
severities was performed.</p>
<h3 data-number="14.35.4" id="significance-30"><span class="header-section-number">14.35.4</span> Significance</h3>
<p>This article is of limited significance as
it simply reports similar descriptions of COVID patients made in
previous literature that severe cases are characterized by lymphopenia.</p>
<h3 data-number="14.35.5" id="credit-34"><span class="header-section-number">14.35.5</span> Credit</h3>
<p><em>Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.36" id="the-potential-role-of-il-6-in-monitoring-severe-case-of-coronavirus-disease-2019"><span class="header-section-number">14.36</span> The potential role of IL-6 in monitoring severe case of coronavirus disease 2019</h2>
<p>Liu et al. <em>medRxiv</em>. <span class="citation" data-cites="1GMxHWnxN">(<a href="#ref-1GMxHWnxN" role="doc-biblioref">2148</a>)</span></p>
<h3 data-number="14.36.1" id="keywords-36"><span class="header-section-number">14.36.1</span> Keywords</h3>
<ul>
<li>Cytokine Release Syndrome</li>
<li>lymphocytopenia</li>
<li>IL-6</li>
<li>CRP</li>
<li>COVID19</li>
<li>pneumonia</li>
</ul>
<h3 data-number="14.36.2" id="main-findings-34"><span class="header-section-number">14.36.2</span> Main Findings</h3>
<p>Study on blood biomarkers on 80 COVID19 patients (69 severe
and 11 non-severe). Patients with severe symptoms at admission
(baseline) showed obvious lymphocytopenia and significantly increased
interleukin-6 (IL-6) and CRP, which was positively correlated with
symptoms severity. IL-6 at baseline positively correlates with CRP, LDH,
ferritin and D-Dimer abundance in blood.</p>
<p>Longitudinal analysis of 30 patients (before and after treatment) showed
significant reduction of IL-6 in remission cases.</p>
<h3 data-number="14.36.3" id="limitations-33"><span class="header-section-number">14.36.3</span> Limitations</h3>
<p>Limited sample size at baseline, especially for the
non-severe leads to question on representativeness. The longitudinal
study method is not described in detail and suffers from
non-standardized treatment. Limited panel of pro-inflammatory cytokine
was analyzed. Patients with severe disease show a wide range of altered
blood composition and biomarkers of inflammation, as well as differences
in disease course (53.6% were cured, about 10% developed acute
respiratory distress syndrome). The authors comment on associations
between IL-6 levels and outcomes, but these were not statistically
significant (maybe due to the number of patients, non-standardized
treatments, etc.) and data is not shown. Prognostic biomarkers could
have been better explored. Study lacks multivariate analysis.</p>
<h3 data-number="14.36.4" id="significance-31"><span class="header-section-number">14.36.4</span> Significance</h3>
<p>IL-6 could be used as a pharmacodynamic
marker of disease severity. Cytokine Release Syndrome (CRS) is a
well-known side effect for CAR-T cancer therapy and there are several
effective drugs to manage CRS. Drugs used to manage CRS could be tested
to treat the most severe cases of COVID19.</p>
<h3 data-number="14.36.5" id="credit-35"><span class="header-section-number">14.36.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.37" id="clinical-and-laboratory-profiles-of-75-hospitalized-patients-with-novel-coronavirus-disease-2019-in-hefei-china"><span class="header-section-number">14.37</span> Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China</h2>
<p>Zhao et al. <em>medRxiv</em>. <span class="citation" data-cites="1B0vBwv1q">(<a href="#ref-1B0vBwv1q" role="doc-biblioref">2149</a>)</span></p>
<h3 data-number="14.37.1" id="keywords-37"><span class="header-section-number">14.37.1</span> Keywords</h3>
<ul>
<li>Routine laboratory testing</li>
</ul>
<h3 data-number="14.37.2" id="main-findings-35"><span class="header-section-number">14.37.2</span> Main Findings</h3>
<p>The authors of this study provide a comprehensive analysis of clinical
laboratory assessments in 75 patients (median age 47 year old)
hospitalized for Corona virus infection in China measuring differential
blood counts including T-cell subsets (CD4, CD8), coagulation function,
basic blood chemistry, of infection-related biomarkers including CRP,
Procalcitonin (PCT) (Precursor of calcitonin that increases during
bacterial infection or tissue injury), IL-6 and erythrocyte
sedimentation rate as well as clinical parameters. Among the most common
hematological changes they found increased neutrophils, reduced CD4 and
CD8 lymphocytes, increased LDH, CRP and PCT</p>
<p>When looking at patients with elevated IL-6, the authors describe
significantly reduced CD4 and CD8 lymphocyte counts and elevated CRP and
PCT levels were significantly increased in infected patients suggesting
that increased IL-6 may correlate well with disease severity in COVID-19
infections</p>
<h3 data-number="14.37.3" id="limitations-34"><span class="header-section-number">14.37.3</span> Limitations</h3>
<p>The authors performed an early assessment of clinical standard
parameters in patients infected with COVID-19. Overall, the number of
cases (75) is rather low and the snapshot approach does not inform about
dynamics and thus potential relevance in the assessment of treatment
options in this group of patients.</p>
<h3 data-number="14.37.4" id="significance-32"><span class="header-section-number">14.37.4</span> Significance</h3>
<p>The article summarizes provides a good summary of some of the common
changes in immune cells inflammatory cytokines in patients with a
COVID-19 infection and. Understanding how these changes can help predict
severity of disease and guide therapy including IL-6 cytokine receptor
blockade using Tocilizumab or Sarilumab will be important to explore.</p>
<h3 data-number="14.37.5" id="credit-36"><span class="header-section-number">14.37.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.38" id="exuberant-elevation-of-ip-10-mcp-3-and-il-1ra-during-sars-cov-2-infection-is-associated-with-disease-severity-and-fatal-outcome"><span class="header-section-number">14.38</span> Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome</h2>
<p>Yang et al. <em>medRxiv</em> <span class="citation" data-cites="RyXNw87v">(<a href="#ref-RyXNw87v" role="doc-biblioref">2150</a>)</span></p>
<h3 data-number="14.38.1" id="keywords-38"><span class="header-section-number">14.38.1</span> Keywords</h3>
<ul>
<li>cytokine</li>
<li>IP-10</li>
<li>MCP-3</li>
<li>IL-1Ra</li>
<li>lymphocyte</li>
<li>neutrophil</li>
<li>stratification</li>
<li>disease severity</li>
<li>viral load</li>
<li>lung function</li>
<li>complications</li>
<li>clinical data</li>
</ul>
<h3 data-number="14.38.2" id="summary-2"><span class="header-section-number">14.38.2</span> Summary</h3>
<p>Plasma cytokine analysis (48 cytokines) was
performed on COVID-19 patient plasma samples, who were sub-stratified as
severe (N=34), moderate (N=19), and compared to healthy controls (N=8).
Patients were monitored for up to 24 days after illness onset: viral
load (qRT-PCR), cytokine (multiplex on subset of patients), lab tests,
and epidemiological/clinical characteristics of patients were reported.</p>
<h3 data-number="14.38.3" id="main-findings-36"><span class="header-section-number">14.38.3</span> Main Findings</h3>
<ul>
<li><p>Many elevated cytokines with COVID-19 onset compared to healthy
controls (IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10, IL-18, HGF, MCP-3, MIG,
M-CSF, G-CSF, MIG-1a, and IP-10).</p></li>
<li><p>IP-10, IL-1Ra, and MCP-3 (esp. together) were associated with
disease severity and fatal outcome.</p></li>
<li><p>IP-10 was correlated to patient viral load (r=0.3006, p=0.0075).</p></li>
<li><p>IP-10, IL-1Ra, and MCP-3 were correlated to loss of lung function
(PaO<sub>2</sub>/FaO<sub>2</sub> (arterial/atmospheric O2) and Murray Score (lung
injury) with MCP-3 being the most correlated (r=0.4104 p&lt;0.0001
and r=0.5107 p&lt;0.0001 respectively).</p></li>
<li><p>Viral load (Lower Ct Value from qRT-PCR) was associated with
upregulated IP-10 only (not IL-1Ra or MCP-3) and was mildly
correlated with decreased lung function: PaO<sub>2</sub>/FaO<sub>2</sub>
(arterial/atmospheric O2) and Murray Score (lung injury).</p></li>
<li><p>Lymphopenia (decreased CD4 and CD8 T cells) and increased neutrophil
correlated w/ severe patients.</p></li>
<li><p>Complications were associated with COVID severity (ARDS, hepatic
insufficiency, renal insufficiency).</p></li>
</ul>
<h3 data-number="14.38.4" id="limitations-35"><span class="header-section-number">14.38.4</span> Limitations</h3>
<p>Collection time of clinical data and lab results
not reported directly (likely 4 days (2,6) after illness onset), making
it very difficult to determine if cytokines were predictive of patient
outcome or reflective of patient compensatory immune response (likely
the latter). Small N for cytokine analysis (N=2 fatal and N=5
severe/critical, and N=7 moderate or discharged). Viral treatment
strategy not clearly outlined.</p>
<h3 data-number="14.38.5" id="expanded-results"><span class="header-section-number">14.38.5</span> Expanded Results</h3>
<p><em>NOTE</em>: Moderate COVID-19 was classified by fever, respiratory
manifestations, and radiological findings consistent with pneumonia
while severe patients had one or more of the following: 1) respiratory
distraction, resting O2 saturation, or 3) arterial PaO2/FiO2 <em>&lt;</em> 300
mmHg.</p>
<p>Cytokine Results (Human Cytokine Screening Panel, Bio-Rad):</p>
<ul>
<li><p><strong>Significant elevation of cytokines observed in COVID patients
compared to healthy controls: IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10,
IL-18, HGF, MCP-3, MIG, M-CSF, G-CSF, MIG-1a, and IP-10.</strong></p></li>
<li><p>Severity was correlated <strong>with increase in measured IP-10, MCP-3,
and IL-Ra</strong> as measure by area under the curve analysis during
sample timecourse (2-24 days after illness onset).</p></li>
<li><p>IL-1Ra incr. significant 0-7 days after onset, MCP-3 signif.
upregulated throughout observation timecourse, and IP-10 increased
and upregulated throughout (trending downwards over time).</p></li>
<li><p><strong>The three cytokines together (IP-10, IL-1Ra, and MCP-3 AUC) served
as the best predictors of disease deterioration and fatal outcome.</strong></p></li>
<li><p>No significant differences between moderate/severe observed between
groups in IL-2Ra, IL-6, IL-10, IL-18, CTACK, G-CSF, HGF, M-CSF,
MIP-1a, MIG, and IFNy at any timepoints.</p></li>
<li><p><strong>Viral load (Lower Ct Value from qRT-PCR) was associated with
upregulated IP-10 only (not IL-1Ra or MCP-3) and was highly
correlated with decreased lung function: PaO<sub>2</sub>/FaO<sub>2</sub>
(arterial/atmospheric O2) and Murray Score (lung injury).</strong></p></li>
<li><p><strong>Antibodies against these cytokines (esp. anti-IP-10) may serve as
a potential treatment for amelioration of COVID-19 (and associated
ARDS).</strong></p></li>
</ul>
<p>Lab results:</p>
<ul>
<li><p><strong>Decreased lymphocytes (%) in all patients – lymphopenia corr. w/
severe patients</strong></p>
<ul>
<li><strong>Decreased CD4 and CD8 T cells</strong> – no monocyte or
eosino/basophil % measured</li>
</ul></li>
<li><p><strong>Increased neutrophils (%)</strong></p></li>
<li><p>Increased BUN (mmol/L) – other kidney markers, liver markers, and
LDH were not significantly different between groups and were not
compared to healthy controls.</p></li>
</ul>
<p>Clinical features (between moderate vs. severe patient groups):</p>
<ul>
<li><p><strong>Complications were associated with severity (ARDS, hepatic
insufficiency, renal insufficiency).</strong></p></li>
<li><p>Coexisting conditions between groups were not significantly
different (chronic heart/lung/renal/liver disease, diabetes, or
cancer) and patient time courses (onset to admission and onset to
viral tx) also not significantly different – 4 days (2, 6) on
average for admission and 4 (3,7) for antiviral.</p></li>
<li><p>Increased corticosteroids and mechanical/ invasive mechanical
ventilation in severe patients.</p></li>
<li><p>Increased median age in severe group (Median (Range = 63.5 (42-74)
vs. 51 (22-78)) and patients <em>&gt;</em> 60 yrs had higher ratio of
severe patients as compared patients 16-59 yrs.</p></li>
<li><p>Higher incidence of fever is severe patients (91.2 vs. 68.4%),
myalgia (57.7 vs. 48.1%), and chill (17.6% vs. 0%).</p></li>
<li><p>No differences in cough, headache, nausea/vomiting, or diarrhea.</p></li>
</ul>
<h3 data-number="14.38.6" id="significance-33"><span class="header-section-number">14.38.6</span> Significance</h3>
<p>Outline of pathological time course (implicating innate
immunity esp.) and identification key cytokines associated with disease
severity and prognosis (+ comorbidities). Anti-IP-10 as a possible
therapeutic intervention (ex: Eldelumab).</p>
<h3 data-number="14.38.7" id="credit-37"><span class="header-section-number">14.38.7</span> Credit</h3>
<p><em>Review by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.39" id="antibody-responses-to-sars-cov-2-in-patients-of-novel-coronavirus-disease-2019"><span class="header-section-number">14.39</span> Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019</h2>
<p>Zhao Jr. et al. <em>medRxiv</em>. <span class="citation" data-cites="5HXuCa5t">(<a href="#ref-5HXuCa5t" role="doc-biblioref">2151</a>)</span></p>
<h3 data-number="14.39.1" id="keywords-39"><span class="header-section-number">14.39.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2 IgG</li>
<li>seroconversion rate</li>
<li>total Ab</li>
<li>Ig</li>
<li>IgM</li>
</ul>
<h3 data-number="14.39.2" id="main-findings-37"><span class="header-section-number">14.39.2</span> Main Findings</h3>
<p>This study examined antibody responses in
the blood of COVID-19 patients during the early SARS CoV2 outbreak in
China. Total 535 plasma samples were collected from 173 patients (51.4%
female) and were tested for seroconversion rate using ELISA. Authors
also compared the sensitivity of RNA and antibody tests over the course
of the disease . The key findings are:</p>
<ul>
<li><p>Among 173 patients, the seroconversion rates for total antibody
(Ab), IgM and IgG were 93.1% (161/173), 82.7% (143/173) and 64.7%
(112/173), respectively.</p></li>
<li><p>The seroconversion sequentially appeared for Ab, IgM and then IgG,
with a median time of 11, 12 and 14 days, respectively. Overall, the
seroconversion of Ab was significantly quicker than that of IgM (p =
0.012) and IgG (p &lt; 0.001). Comparisons of seroconversion rates
between critical and non-critical patients did not reveal any
significant differences.</p></li>
<li><p>RNA tests had higher sensitivity in early phase and within 7 days of
disease onset than antibody assays (66.7% Vs 38.3% respectively).</p></li>
<li><p>The sensitivity of the Ab assays was higher 8 days after disease
onset, reached 90% at day 13 and 100% at later time points (15-39
days). In contrast, RNA was only detectable in 45.5% of samples at
days 15-39.</p></li>
<li><p>In patients with undetectable RNA in nasal samples collected during
day 1-3, day 4-7, day 8-14 and day 15-39 since disease onset, 28.6%
(2/7), 53.6% (15/28), 98.2% (56/57) and 100% (30/30) had detectable
total Ab titers respectively Combining RNA and antibody tests
significantly raised the sensitivity for detecting COVID-19 patients
in different stages of the disease (p &lt; 0.001).</p></li>
<li><p>There was a strong positive correlation between clinical severity
and antibody titer 2-weeks after illness onset.</p></li>
<li><p>Dynamic profiling of viral RNA and antibodies in representative
COVID-19 patients (n=9) since onset of disease revealed that
antibodies may not be sufficient to clear the virus. It should be
noted that increases in of antibody titers were not always
accompanied by RNA clearance.</p></li>
</ul>
<h3 data-number="14.39.3" id="limitations-36"><span class="header-section-number">14.39.3</span> Limitations</h3>
<p>Because different types of ELISA assays were used for
determining antibody concentrations at different time points after
disease onset, sequential seroconversion of total Ab, IgM and IgG may
not represent actual temporal differences but rather differences in the
affinities of the assays used. Also, due to the lack of blood samples
collected from patients in the later stage of illness, how long the
antibodies could last remain unknown. For investigative dynamics of
antibodies, more samples were required.</p>
<h3 data-number="14.39.4" id="significance-34"><span class="header-section-number">14.39.4</span> Significance</h3>
<p>Total and IgG antibody titers could be used to understand
the epidemiology of SARS CoV-2 infection and to assist in determining
the level of humoral immune response in patients.</p>
<p>The findings provide strong clinical evidence for routine serological
and RNA testing in the diagnosis and clinical management of COVID-19
patients. The understanding of antibody responses and their half-life
during and after SARS CoV2 infection is important and warrants further
investigations.</p>
<h3 data-number="14.39.5" id="credit-38"><span class="header-section-number">14.39.5</span> Credit</h3>
<p><em>This review was undertaken by Zafar Mahmood and edited by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.40" id="restoration-of-leukomonocyte-counts-is-associated-with-viral-clearance-in-covid-19-hospitalized-patients"><span class="header-section-number">14.40</span> Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients</h2>
<p>Chen et al. <em>medRxiv</em> <span class="citation" data-cites="uCyoHtV3">(<a href="#ref-uCyoHtV3" role="doc-biblioref">2152</a>)</span></p>
<h3 data-number="14.40.1" id="keywords-40"><span class="header-section-number">14.40.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>T cell</li>
<li>B cell</li>
<li>NK cell</li>
<li>IL-6</li>
<li>pro-calcitonin</li>
<li>cytokine storm</li>
</ul>
<h3 data-number="14.40.2" id="main-findings-38"><span class="header-section-number">14.40.2</span> Main Findings</h3>
<p>The authors collected data on 25 COVID-19 patients (n=11 men, n=14
women) using standard laboratory tests and flow cytometry. All patients
were treated with antibiotics. Twenty-four of the 25 patients were also
treated with anti-viral Umefinovir and 14 of the patients were treated
with corticosteroids. 14 patients became negative for the virus after
8-14 days of treatment. The same treatment course was extended to 15-23
days for patients who were still positive for the virus at day 14.</p>
<p>The authors found a negative association between age and resolution of
infection. Patients with hypertension, diabetes, malignancy or chronic
liver disease were all unable to clear the virus at day 14, though not
statistically significant.</p>
<p>Elevated procalcitonin and a trend for increased IL-6 were also found in
peripheral blood prior to the treatment.</p>
<p>A trend for lower NK cell, T cell and B cell counts in patients was also
reported. B cell, CD4 and CD8 T cell counts were only increased upon
treatment in patients who cleared the virus. NK cell frequencies
remained unchanged after treatment in all the patients.</p>
<h3 data-number="14.40.3" id="limitations-37"><span class="header-section-number">14.40.3</span> Limitations</h3>
<p>73% of the patients who remained positive for SARS-CoV2 after the 1<sup>st</sup>
treatment, and 43% of all patients who cleared the virus were treated
with corticosteroids. Corticosteroids have strong effects on the immune
compartment in blood <span class="citation" data-cites="L6kwccjn">(<a href="#ref-L6kwccjn" role="doc-biblioref">2153</a>)</span>. The authors should have accounted for
corticosteroid treatment when considering changes in T, NK and B cell
frequencies.</p>
<p>Assessing if IL-6 concentrations were back to baseline levels following
treatment would have provided insights into the COVID-19 cytokine storm
biology. Patients with higher baseline levels of IL-6 have been reported
to have lower CD8 and CD4 T cell frequencies <span class="citation" data-cites="1B0vBwv1q">(<a href="#ref-1B0vBwv1q" role="doc-biblioref">2149</a>)</span>. Correlating IL-6 with
cell counts before and after treatment would thus have also been of
interest. The report of the laboratory measures in table 2 is incomplete
and should include the frequencies of patients with increased/decreased
levels for each parameter.</p>
<p>Correction is needed for the 1<sup>st</sup> paragraph of the discussion as data
does not support NK cell restoration upon treatment in patients who
cleared the virus. NK cells remain unchanged after the 1<sup>st</sup> treatment
course and only seem to increase in 2 out of 6 donors after the 2<sup>nd</sup>
treatment course in those patients.</p>
<h3 data-number="14.40.4" id="significance-35"><span class="header-section-number">14.40.4</span> Significance</h3>
<p>Previous reports suggest an association between disease severity and
elevated IL-6 or pro-calcitonin concentrations in COVID-19
patients <span class="citation" data-cites="T0BDaXkw K9G9QT8y">(<a href="#ref-T0BDaXkw" role="doc-biblioref">2145</a>, <a href="#ref-K9G9QT8y" role="doc-biblioref">2154</a>)</span>. IL-6 receptor blockade is also being administered to
patients enrolled in clinical trials (NCT04317092). This report thus
contributes to highlight elevated concentrations of these analytes in
COVID-19 patients. Mechanisms underlying the association between viral
clearance and restoration of the T cell and B cell frequencies suggests
viral-driven immune dysregulation, which needs to be investigated in
further studies.</p>
<h3 data-number="14.40.5" id="credit-39"><span class="header-section-number">14.40.5</span> Credit</h3>
<p><em>Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.41" id="clinical-findings-in-critically-ill-patients-infected-with-sars-cov-2-in-guangdong-province-china-a-multi-center-retrospective-observational-study"><span class="header-section-number">14.41</span> Clinical findings in critically ill patients infected with SARS-CoV-2 in Guangdong Province, China: a multi-center, retrospective, observational study </h2>
<p>Xu et al. <em>medRxiv</em>. <span class="citation" data-cites="USOnp7c0">(<a href="#ref-USOnp7c0" role="doc-biblioref">2155</a>)</span></p>
<h3 data-number="14.41.1" id="keywords-41"><span class="header-section-number">14.41.1</span> Keywords</h3>
<ul>
<li>clinical outcomes</li>
<li>prognosis</li>
<li>critically ill patients</li>
<li>ICU</li>
<li>lymphopenia</li>
<li>LDH</li>
</ul>
<h3 data-number="14.41.2" id="main-findings-39"><span class="header-section-number">14.41.2</span> Main Findings:</h3>
<p>This work analyses laboratory and clinical data from 45 patients treated
in the in ICU in a single province in China. Overall, 44% of the
patients were intubated within 3 days of ICU admission with only 1
death.</p>
<p>Lymphopenia was noted in 91% of patient with an inverse correlation with
LDH.</p>
<p>Lymphocyte levels are negatively correlated with Sequential Organ
Failure Assessment (SOFA) score (clinical score, the higher the more
critical state), LDH levels are positively correlated to SOFA score.
Overall, older patients (&gt;60yo), with high SOFA score, high LDH
levels and low lymphocytes levels at ICU admission are at higher risk of
intubation.</p>
<p>Of note, convalescent plasma was administered to 6 patients but due to
limited sample size no conclusion can be made.</p>
<h3 data-number="14.41.3" id="limitations-38"><span class="header-section-number">14.41.3</span> Limitations</h3>
<p>While the study offers important insights into disease course and
clinical lab correlates of outcome, the cohort is relatively small and
is likely skewed towards a less-severe population compared to other ICU
reports given the outcomes observed. Analysis of laboratory values and
predictors of outcomes in larger cohorts will be important to make
triage and treatment decisions. As with many retrospective analyses,
pre-infection data is limited and thus it is not possible to understand
whether lymphopenia was secondary to underlying comorbidities or
infection.</p>
<p>Well-designed studies are necessary to evaluate the effect of
convalescent plasma administration.</p>
<h3 data-number="14.41.4" id="significance-36"><span class="header-section-number">14.41.4</span> Significance</h3>
<p>This clinical data enables the identification of at-risk
patients and gives guidance for research for treatment options. Indeed,
further work is needed to better understand the causes of the
lymphopenia and its correlation with outcome.</p>
<h3 data-number="14.41.5" id="credit-40"><span class="header-section-number">14.41.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.42" id="immune-multi-epitope-vaccine-design-using-an-immunoinformatics-approach-for-2019-novel-coronavirus-in-china-sars-cov-2"><span class="header-section-number">14.42</span> Immune Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus in China (SARS-CoV-2)</h2>
<p><span class="citation" data-cites="14OLQIYK0">(<a href="#ref-14OLQIYK0" role="doc-biblioref">2156</a>)</span></p>
<h3 data-number="14.42.1" id="keywords-42"><span class="header-section-number">14.42.1</span> Keywords</h3>
<ul>
<li>Vaccine</li>
<li>in silico</li>
<li>B cell epitopes</li>
<li>T cell epitopes</li>
</ul>
<h3 data-number="14.42.2" id="main-findings-40"><span class="header-section-number">14.42.2</span> Main Findings</h3>
<p>Using in silico bioinformatic tools, this study identified putative
antigenic B-cell epitopes and HLA restricted T-cell epitopes from the
spike, envelope and membrane proteins of SARS-CoV-2, based on the genome
sequence available on the NCBI database. T cell epitopes were selected
based on predicted affinity for the more common HLA-I alleles in the
Chinese population. Subsequently, the authors designed vaccine peptides
by bridging selected B-cell epitopes and adjacent T-cell epitopes.
Vaccine peptides containing only T-cell epitopes were also generated.</p>
<p>From 61 predicted B-cell epitopes, only 19 were exposed on the surface
of the virion and had a high antigenicity score. A total of 499 T-cell
epitopes were predicted. Based on the 19 B-cell epitopes and their 121
adjacent T-cell epitopes, 17 candidate vaccine peptides were designed.
Additionally, another 102 vaccine peptides containing T-cell epitopes
only were generated. Based on the epitope counts and HLA score, 13 of
those were selected. Thus, a total of 30 peptide vaccine candidates were
designed.</p>
<h3 data-number="14.42.3" id="limitations-39"><span class="header-section-number">14.42.3</span> Limitations</h3>
<p>While this study provides candidates for the development of vaccines
against SARS-CoV-2, in vitro and in vivo trials are required to validate
the immunogenicity of the selected B and T cell epitopes. This could be
done using serum and cells from CoV-2-exposed individuals, and in
preclinical studies. The implication of this study for the current
epidemic are thus limited. Nevertheless, further research on this field
is greatly needed.</p>
<h3 data-number="14.42.4" id="credit-41"><span class="header-section-number">14.42.4</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.43" id="clinical-features-of-patients-infected-with-the-2019-novel-coronavirus-covid-19-in-shanghai-china"><span class="header-section-number">14.43</span> Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China</h2>
<p>Cao et al. <em>medRxiv</em> <span class="citation" data-cites="12ilBRq6I">(<a href="#ref-12ilBRq6I" role="doc-biblioref">2157</a>)</span></p>
<h3 data-number="14.43.1" id="keywords-43"><span class="header-section-number">14.43.1</span> Keywords</h3>
<ul>
<li>Disease severity</li>
<li>clinical features</li>
<li>laboratory abnormalities</li>
</ul>
<h3 data-number="14.43.2" id="main-findings-41"><span class="header-section-number">14.43.2</span> Main Findings</h3>
<p>This single-center cohort study analyzes the clinical and
laboratory features of 198 patients with confirmed COVID-19 infection in
Shanghai, China and correlated these parameters with clinical disease
severity, including subsequent intensive care unit (ICU) admission. 19
cases (9.5%) required ICU admission after developing respiratory failure
or organ dysfunction. Age, male sex, underlying cardiovascular disease,
and high symptom severity (high fever, dyspnea) were all significantly
correlated with ICU admission. Additionally, ICU admission was more
common in patients who presented with lymphopenia and elevated
neutrophil counts, among other laboratory abnormalities. Flow cytometric
analysis revealed that patients admitted to the ICU had significantly
reduced circulating CD3+ T cell, CD4+ T cell, CD8+ T cell, and CD45+
leukocyte populations compared to the cohort of patients not requiring
ICU admission.</p>
<h3 data-number="14.43.3" id="limitations-40"><span class="header-section-number">14.43.3</span> Limitations</h3>
<p>The limitations of this study include the relatively
small sample size and lack of longitudinal testing. The authors also did
not assess whether respiratory comorbidity – such as asthma or chronic
obstructive lung disease – in addition to immunosuppression affected ICU
admission likelihood.</p>
<h3 data-number="14.43.4" id="significance-37"><span class="header-section-number">14.43.4</span> Significance</h3>
<p>COVID-19 has already sickened thousands across the globe,
though the severity of these infections is markedly diverse, ranging
from mild symptoms to respiratory failure requiring maximal
intervention. Understanding what clinical, laboratory, and immunologic
factors predict the clinical course of COVID-19 infection permits
frontline providers to distribute limited medical resources more
effectively.</p>
<h3 data-number="14.43.5" id="credit-42"><span class="header-section-number">14.43.5</span> Credit</h3>
<p><em>Review by Andrew Charap as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine at
Mount Sinai.</em></p>
<h2 data-number="14.44" id="serological-detection-of-2019-ncov-respond-to-the-epidemic-a-useful-complement-to-nucleic-acid-testing"><span class="header-section-number">14.44</span> Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing</h2>
<p>Zhang et al. <em>medRxiv</em>. <span class="citation" data-cites="IPJURDxi">(<a href="#ref-IPJURDxi" role="doc-biblioref">2158</a>)</span></p>
<h3 data-number="14.44.1" id="keywords-44"><span class="header-section-number">14.44.1</span> Keywords</h3>
<ul>
<li>Immunoassay</li>
<li>serum IgM and IgG</li>
<li>specific antibodies</li>
</ul>
<h3 data-number="14.44.2" id="main-finding"><span class="header-section-number">14.44.2</span> Main Finding</h3>
<p>This study showed that both anti-2019-nCov IgM and IgG were detected by
automated chemiluminescent immunoassay in the patients who had been
already confirmed as positive by nucleic acid detection, while single
positivity of IgM or IgG were detected in a very few cases in the other
population including 225 non-COVID-19 cases. In addition to the increase
of anti-2019-nCov IgM 7-12 days after morbidity, the increase of IgG was
detected in three patients with COVID-19 within a very short of time
(0-1 day).</p>
<h3 data-number="14.44.3" id="limitations-41"><span class="header-section-number">14.44.3</span> Limitations</h3>
<p>The limitation of this study is only
3 confirmed COVID-19 cases were included, so that the relationship
between anti-2019-nCov antibodies and disease progression might not be
clearly defined. Another limitation is that they did not show the course
of 2019-nCov specific antibodies in the cases with positive for COVID-19
but without clinical symptoms.</p>
<h3 data-number="14.44.4" id="significance-38"><span class="header-section-number">14.44.4</span> Significance</h3>
<p>The detection of anti-2019-nCov antibodies can be an alternative method
to diagnose and treat COVID-19 more comprehensively by distinguish non
COVID-19 patients. It may be helpful to understand the course of
individual cases with COVID-19 to predict the prognosis if more cases
will be evaluated.</p>
<h3 data-number="14.44.5" id="credit-43"><span class="header-section-number">14.44.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.45" id="human-kidney-is-a-target-for-novel-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-infection"><span class="header-section-number">14.45</span> Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection</h2>
<p><span class="citation" data-cites="H0nvocJz">(<a href="#ref-H0nvocJz" role="doc-biblioref">2159</a>)</span></p>
<h3 data-number="14.45.1" id="keywords-45"><span class="header-section-number">14.45.1</span> Keywords</h3>
<ul>
<li>Kidney/Renal Failure</li>
<li>Macrophage Infiltration</li>
<li>Complement Activation</li>
</ul>
<h3 data-number="14.45.2" id="main-finding-1"><span class="header-section-number">14.45.2</span> Main Finding</h3>
<p>Analyzing the eGFR (effective glomerular flow rate) of 85 Covid-19
patients and characterizing tissue damage and viral presence in
post-mortem kidney samples from 6 Covid-19 patients, the authors
conclude that significant damage occurs to the kidney, following
Covid-19 infection. This is in contrast to the SARS infection from the
2003 outbreak. They determine this damage to be more prevalent in
patients older than 60 years old, as determined by analysis of eGFR. H&amp;E
and IHC analysis in 6 Covid-19 patients revealed that damage was in the
tubules, not the glomeruli of the kidneys and suggested that macrophage
accumulation and C5b-9 deposition are key to this process. </p>
<h3 data-number="14.45.3" id="limitations-42"><span class="header-section-number">14.45.3</span> Limitations</h3>
<p>Severe limitations include that the H&amp;E and IHC samples were performed
on post-mortem samples of unknown age, thus we cannot assess how/if age
correlates with kidney damage, upon Covid-19 infection. Additionally,
eGFR was the only <em>in-vivo</em> measurement. Blood urea nitrogen and
proteinuria are amongst other measurements that could have been obtained
from patient records. An immune panel of the blood was not performed to
assess immune system activation. Additionally, patients are only from
one hospital. </p>
<h3 data-number="14.45.4" id="significance-39"><span class="header-section-number">14.45.4</span> Significance</h3>
<p>This report makes clear that kidney damage is prevalent
in Covid-19 patients and should be accounted for. </p>
<h3 data-number="14.45.5" id="credit-44"><span class="header-section-number">14.45.5</span> Credit</h3>
<p><em>Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine at Mount Sinai.</em></p>
<h2 data-number="14.46" id="covid-19-early-warning-score-a-multi-parameter-screening-tool-to-identify-highly-suspected-patients"><span class="header-section-number">14.46</span> COVID-19 early warning score: a multi-parameter screening tool to identify highly suspected patients</h2>
<p>Song et al. <em>medRxiv</em>. <span class="citation" data-cites="KHNnjP7M">(<a href="#ref-KHNnjP7M" role="doc-biblioref">2160</a>)</span></p>
<h3 data-number="14.46.1" id="keywords-46"><span class="header-section-number">14.46.1</span> Keywords</h3>
<ul>
<li>retrospective</li>
<li>electronic health records</li>
<li>blood counts</li>
<li>diagnostic</li>
<li>prognostic</li>
<li>modeling</li>
</ul>
<h3 data-number="14.46.2" id="main-findings-42"><span class="header-section-number">14.46.2</span> Main Findings</h3>
<p>The aim of this study was to identify diagnostic or prognostic criteria
which could identify patients with COVID-19 and predict patients who
would go on to develop severe respiratory disease. The authors use EMR
data from individuals taking a COVID-19 test at Zhejiang hospital, China
in late January/Early February. A large number of clinical parameters
were different between individuals with COVID-19 and also between
‘severe’ and ‘non-severe’ infections and the authors combine these into
a multivariate linear model to derive a weighted score, presumably
intended for clinical use.</p>
<h3 data-number="14.46.3" id="limitations-43"><span class="header-section-number">14.46.3</span> Limitations</h3>
<p>Unfortunately, the paper is lacking a lot of crucial information, making
it impossible to determine the importance or relevance of the findings.
Most importantly, the timings of the clinical measurements are not
described relative to the disease course, so it is unclear if the
differences between ‘severe’ and ‘non-severe’ infections are occurring
before progression to severe disease (which would make them useful
prognostic markers), or after (which would not).</p>
<h3 data-number="14.46.4" id="significance-40"><span class="header-section-number">14.46.4</span> Significance</h3>
<p>This paper is one of many retrospective studies coming from hospitals in
China studying individuals with COVID-19. Because of the sparse
description of the study design, this paper offers little new
information. However, studies like this could be very valuable and we
would strongly encourage the authors to revise this manuscript to
include more information about the timeline of clinical measurements in
relation to disease onset and more details of patient outcomes.</p>
<h3 data-number="14.46.5" id="credit-45"><span class="header-section-number">14.46.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.47" id="ly6e-impairs-coronavirus-fusion-and-confers-immune-control-of-viral-disease"><span class="header-section-number">14.47</span> LY6E impairs coronavirus fusion and confers immune control of viral disease</h2>
<p>Pfaender et al. <em>bioRxiv</em>. <span class="citation" data-cites="19xwRoatu">(<a href="#ref-19xwRoatu" role="doc-biblioref">2161</a>)</span></p>
<h3 data-number="14.47.1" id="keywords-47"><span class="header-section-number">14.47.1</span> Keywords</h3>
<ul>
<li>interferon-stimulated genes</li>
<li>antiviral interferons</li>
<li>human coronaviruses (CoV)</li>
<li>murine hepatitis virus (MHV)</li>
</ul>
<h3 data-number="14.47.2" id="main-findings-43"><span class="header-section-number">14.47.2</span> Main Findings</h3>
<p>Screening a cDNA library of &gt;350 human interferon-stimulated genes
for antiviral activity against endemic human coronavirus HCoV-229E
(associated with the common cold), Pfaender S &amp; Mar K <em>et al.</em> identify
lymphocyte antigen 6 complex, locus E (Ly6E) as an inhibitor of cellular
infection of Huh7 cells, a human hepatoma cell line susceptible to
HCoV-229E and other coronaviruses. In a series of consecutive <em>in vitro</em>
experiments including both stable Ly6E overexpression and
CRISPR-Cas9-mediated knockout the authors further demonstrate that Ly6E
reduces cellular infection by various other coronaviruses including
human SARS-CoV and SARS-CoV-2 as well as murine CoV mouse hepatitis
virus (MHV). Their experiments suggest that this effect is dependent on
Ly6E inhibition of CoV strain-specific spike protein-mediated membrane
fusion required for viral cell entry.</p>
<p>To address the function of Ly6E <em>in vivo</em>, hematopoietic stem
cell-specific Ly6E knock-out mice were generated by breeding Ly6E<sup>fl/fl</sup>
mice (referred to as functional wild-type mice) with transgenic
<em>Vav-iCre</em> mice (offspring referred to as Ly6E HSC ko mice); wild-type
and Ly6E HSC ko mice of both sexes were infected intraperitoneally with
varying doses of the natural murine coronavirus MHV, generally causing a
wide range of diseases in mice including hepatitis, enteritis and
encephalomyelitis. Briefly, compared to wild-type controls, mice lacking
hematopoietic cell-expressed Ly6E were found to present with a more
severe disease phenotype as based on serum ALT levels (prognostic of
liver damage), liver histopathology, and viral titers in the spleen.
Moreover, bulk RNAseq analysis of infected liver and spleen tissues
indicated changes in gene expression pathways related to tissue damage
and antiviral immune responses as well as a reduction of genes
associated with type I IFN response and inflammation. Finally, the
authors report substantial differences in the numbers of hepatic and
splenic APC subsets between wild-type and knockout mice following MHV
infection and show that Ly6E-deficient B cells and to a lesser extent
also DCs are particularly susceptible to MHV infection <em>in vitro</em>.</p>
<h3 data-number="14.47.3" id="limitations-44"><span class="header-section-number">14.47.3</span> Limitations</h3>
<p>Experiments and data in this study are presented in an overall logical
and coherent fashion; however, some observations and the conclusions
drawn are problematic and should be further addressed &amp; discussed by the
authors. Methodological &amp; formal limitations include relatively low
replicate numbers as well as missing technical replicates for some <em>in
vitro</em> experiments (<em>cf</em>. Fig. legend 1; Fig. legend 2e); the omission
of “outliers” in Fig. legend 2 without an apparent rationale as to why
this approach was chosen; the lack of detection of actual Ly6E protein
levels in Ly6E HSC ko or wild-type mice; and most importantly, missing
information on RNAseq data collection &amp; analysis in the method section
and throughout the paper. A more relevant concern though is that the
interpretation of the experimental data presented and the language used
tend to overrate and at times overgeneralize findings: for example,
while the authors demonstrate statistically significant, Ly6E-mediated
reduction of coronavirus titers in stable cells lines <em>in vitro</em>, it
remains unclear whether a viral titer reduction by one log decade would
be of actual biological relevance in face of high viral titers <em>in
vivo</em>. After high-dose intraperitoneal MHV infection <em>in vivo</em>, early
viral titers in Ly6E HSC knockout vs. wt mice only showed an elevation
in the spleen (~1.5 log decades) but not liver of the ko mice (other
tissue not evaluated), and while ko mice presented with only modestly
increased liver pathology, both male and female ko mice exhibited
significantly higher mortality. Thus, the manuscript tile statement that
“Ly6E … confers immune control of viral disease” is supported by only
limited <em>in vivo</em> data, and gain-of-function experiments (eg. Ly6E
overexpression) were not performed. Of additional note here, tissue
tropism and virulence differ greatly among various MHV strains and
isolates whereas dose, route of infection, age, genetic background and
sex of the mice used may additionally affect disease outcome and
phenotype (<em>cf</em>. Taguchi F &amp; Hirai-Yuki A,
<a href="https://doi.org/10.3389/fmicb.2012.00068" class="uri">https://doi.org/10.3389/fmicb.2012.00068</a>; Kanolkhar A et al,
<a href="https://jvi.asm.org/content/%2083/18/9258">https://jvi.asm.org/content/
83/18/9258</a>). Observations
attributed to hematopoietic stem cell-specific Ly6E deletion could
therefore be influenced by the different genetic backgrounds of floxed
and cre mice used, and although it appears that littermates wt and ko
littermates were used in the experiments, the potentially decisive
impact of strain differences should at least have been discussed. Along
these lines, it should also be taken into account that the majority of
human coronaviruses cause respiratory symptoms, which follow a different
clinical course engaging other primary cellular mediators than the
hepatotropic murine MHV disease studied here. It therefore remains
highly speculative how the findings reported in this study will
translate to human disease and it would therefore be important to test
other routes of MHV infection and doses that have been described to
produce a more comparable phenotype to human coronavirus disease (<em>cf.</em>
Kanolkhar A et al, <a href="https://jvi.asm.org/content/%2083/18/9258">https://jvi.asm.org/content/
83/18/9258</a>). Another
important shortcoming of this study is the lack of any information on
functional deficits or changes in Ly6E-deficient immune cells and how
this might relate to the phenotype observed. Overall, the <em>in vitro</em>
experiments are more convincing than the <em>in vivo</em> studies which appear
somewhat limited.</p>
<h3 data-number="14.47.4" id="significance-41"><span class="header-section-number">14.47.4</span> Significance</h3>
<p>Despite some shortcomings, the experiments performed in this study
suggest a novel and somewhat unexpected role of Ly6E in the protection
against coronaviruses across species. These findings are of relevance
and should be further explored in ongoing research on potential
coronavirus therapies. Yet an important caveat pertains to the authors’
suggestion that “therapeutic mimicking of Ly6E action” may constitute a
first line of defense against novel coronaviruses since their own prior
work demonstrated that Ly6E can enhance rather than curtail infection
with influenza A and other viruses.</p>
<h3 data-number="14.47.5" id="credit-46"><span class="header-section-number">14.47.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.48" id="a-preliminary-study-on-serological-assay-for-severe-acute-respiratory-syndrome-coronavirus-2-sars-cov-2-in-238-admitted-hospital-patients"><span class="header-section-number">14.48</span> A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients</h2>
<p>Liu et al. <em>medRxiv</em>. <span class="citation" data-cites="574aSa8C">(<a href="#ref-574aSa8C" role="doc-biblioref">2162</a>)</span></p>
<h3 data-number="14.48.1" id="keywords-48"><span class="header-section-number">14.48.1</span> Keywords</h3>
<ul>
<li>diagnosis</li>
<li>serological assay</li>
<li>ELISA</li>
<li>RT-PCR</li>
</ul>
<h3 data-number="14.48.2" id="main-findings-44"><span class="header-section-number">14.48.2</span> Main Findings</h3>
<p>While RT-PCR is being used currently to routinely diagnose infection
with SARS-CoV-2, there are significant limitations to the use of a
nucleic acid test that lead to a high false-negative rate. This
article describes ELISAs that can measure IgM and IgG antibodies
against the N protein of SARS-CoV-2 to test samples from 238
patients (153 positive by RT-PCR and 85 negative by RT-PCR) at
different times after symptom onset. The positivity rate of the IgM
and/or IgG ELISAs was greater than that of the RT-PCR (81.5%
compared to 64.3%) with similar positive rates in the confirmed and
suspected cases (83% and 78.8%, respectively), suggesting that many
of the suspected but RT-PCR-negative cases were also infected. The
authors also found that the ELISAs have higher positive rates later
after symptom onset while RT-PCR is more effective as a diagnostic
test early during the infection.</p>
<h3 data-number="14.48.3" id="limitations-45"><span class="header-section-number">14.48.3</span> Limitations</h3>
<p>I cannot identify any limitations to this study.</p>
<h3 data-number="14.48.4" id="significance-42"><span class="header-section-number">14.48.4</span> Significance</h3>
<p>The authors make a strong case for using a combination of ELISA and
RT-PCR for diagnosis of infection with SARS-CoV-2, especially
considering the dynamics of positivity rates of RT-PCR and ELISA.
Fewer false-negative diagnoses would improve infection control and
patient management.</p>
<h3 data-number="14.48.5" id="credit-47"><span class="header-section-number">14.48.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.49" id="monoclonal-antibodies-for-the-s2-subunit-of-spike-of-sars-cov-cross-react-with-the-newly-emerged-sars-cov-2"><span class="header-section-number">14.49</span> Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2</h2>
<p><span class="citation" data-cites="NT3T9a8H">(<a href="#ref-NT3T9a8H" role="doc-biblioref">2163</a>)</span></p>
<h3 data-number="14.49.1" id="keywords-49"><span class="header-section-number">14.49.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>Spike protein</li>
<li>Cross- reactive antibodies</li>
</ul>
<h3 data-number="14.49.2" id="main-findings-45"><span class="header-section-number">14.49.2</span> Main Findings</h3>
<p>Whole genome sequencing-based comparisons of the 2003
Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and the 2019
SARS-CoV-2 revealed conserved receptor binding domain (RBD) and host
cell receptor, angiotensin-converting enzyme 2 (ACE2). In line with
this, the authors tested cross-reactivity of murine monoclonal
antibodies (mAbs) previously generated against the SARS-CoV spike (S)
glycoprotein involved in viral entry. One of the screened mAb, 1A9, was
able to bind and cross-neutralize multiple strains of SARS-CoV, as well
as, detect the S protein in SARS-CoV-2-infected cells. mAb 1A9 was
generated using an immunogenic fragment in the S2 subunit of SARS-CoV
and binds through a novel epitope within the S2 subunit at amino acids
1111-1130. It is important to note that CD8+ T lymphocyte epitopes
overlap with these residues, suggesting that S2 subunit could be
involved in inducing both, humoral and cell-mediated immunity.</p>
<h3 data-number="14.49.3" id="limitations-46"><span class="header-section-number">14.49.3</span> Limitations</h3>
<p>The authors used previously generated mouse mAbs
against the S protein in SARS-CoV expressed in mammalian cell line.
Future experimental validation using COVID-19 patient samples is needed
to validate these findings. In addition, the results of these studies
are predominantly based on in vitro experiments and so, evaluating the
effects of the mAb 1A9 in an animal model infected with this virus will
help us better understand the host immune responses in COVID-19 and
potential therapeutic vaccines.</p>
<h3 data-number="14.49.4" id="significance-43"><span class="header-section-number">14.49.4</span> Significance</h3>
<p>This study identified mAbs that recognize the new
coronavirus, SARS-Cov-2. These cross-reactive mAbs will help in
developing diagnostic assays for COVID-19.</p>
<h3 data-number="14.49.5" id="credit-48"><span class="header-section-number">14.49.5</span> Credit</h3>
<p><em>This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.50" id="mortality-of-covid-19-is-associated-with-cellular-immune-function-compared-to-immune-function-in-chinese-han-population"><span class="header-section-number">14.50</span> Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population</h2>
<p>Zeng et al. <em>medRxiv</em>. <span class="citation" data-cites="1GEbBtBgh">(<a href="#ref-1GEbBtBgh" role="doc-biblioref">2164</a>)</span></p>
<h3 data-number="14.50.1" id="keywords-50"><span class="header-section-number">14.50.1</span> Keywords</h3>
<ul>
<li>WBC</li>
<li>peripheral blood</li>
<li>CD4</li>
<li>CD8 T cells</li>
</ul>
<h3 data-number="14.50.2" id="main-findings-46"><span class="header-section-number">14.50.2</span> Main Findings</h3>
<p>Retrospective study of the clinical characteristics of 752
patients infected with COVID-19 at Chinese PLA General Hospital, Peking
Union Medical College Hospital, and affiliated hospitals at Shanghai
University of medicine &amp; Health Sciences. This study is the first one
that compares PB from healthy controls from the same regions in Shanghai
and Beijing, and infected COVID-19 patients to standardize a reference
range of WBCs of people at high risk.</p>
<h3 data-number="14.50.3" id="limitations-47"><span class="header-section-number">14.50.3</span> Limitations</h3>
<p>Lower levels of leukocyte counts -B cells, CD4 and CD8 T
cells- correlated with mortality (WBCs are significantly lower in severe
or critical UCI patients vs mild ones). Based on 14,117 normal controls
in Chinese Han population (ranging in age from 18-86) it is recommended
that reference ranges of people at high risk of COVID-19 infection are
CD3+ lymphocytes below 900 cells/mm3, CD4+ lymphocytes below 500
cells/mm3, and CD8+ lymphocytes below 300 cells/mm3. Importantly, this
study also reported that the levels of D-dimer, C-reactive protein and
IL-6 were elevated in COVID-19 pts., indicating clot formation, severe
inflammation and cytokine storm.</p>
<h3 data-number="14.50.4" id="significance-44"><span class="header-section-number">14.50.4</span> Significance</h3>
<p>This study sets a threshold to identify patients at risk
by analyzing their levels of leukocytes, which is an easy and fast
approach to stratify individuals that require hospitalization. Although
the study is limited (only counts of WBC are analyzed and not its
profile) the data is solid and statistically robust to correlate levels
of lymphopenia with mortality.</p>
<h3 data-number="14.50.5" id="credit-49"><span class="header-section-number">14.50.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.51" id="retrospective-analysis-of-clinical-features-in-101-death-cases-with-covid-19"><span class="header-section-number">14.51</span> Retrospective Analysis of Clinical Features in 101 Death Cases with COVID-19</h2>
<p>Chen et al. <em>medRxiv</em>. <span class="citation" data-cites="WuHcOc9T">(<a href="#ref-WuHcOc9T" role="doc-biblioref">2165</a>)</span></p>
<h3 data-number="14.51.1" id="keywords-51"><span class="header-section-number">14.51.1</span> Keywords</h3>
<ul>
<li>death biomarkers</li>
<li>cardiac damage</li>
<li>Troponin</li>
<li>Blood type</li>
<li>respiratory failure</li>
<li>hypertension</li>
</ul>
<h3 data-number="14.51.2" id="main-findings-47"><span class="header-section-number">14.51.2</span> Main Findings</h3>
<p>This is a retrospective study involving 101 death cases with COVID-19 in
Wuhan Jinyintan Hospital. The aim was to describe clinical,
epidemiological and laboratory features of fatal cases in order to
identify the possible primary mortality causes related to COVID-19.</p>
<p>Among 101 death cases, 56.44% were confirmed by RT-PCR and 43.6% by
clinical diagnostics. Males dominated the number of deaths and the
average age was 65.46 years. All patients died of respiratory failure
and multiple organs failure, except one (acute coronary syndrome). The
predominant comorbidities were hypertension (42.57%) and diabetes
(22.77%). 25.74% of the patients presented more than two underlying
diseases. 82% of patients presented myocardial enzymes abnormalities at
admission and further increase in myocardial damage indicators with
disease progression: patients with elevated Troponin I progressed faster
to death. Alterations in coagulation were also detected. Indicators of
liver and kidney damage increased 48 hours before death. The authors
studied the deceased patients’ blood type and presented the following
results: type A (44.44%), type B (29.29%), type AB (8.08%) and type O
(18.19%), which is inconsistent with the distribution in Han population
in Wuhan.</p>
<p>Clinical analysis showed that the most common symptom was fever (91.9%),
followed by cough and dyspnea. The medium time from onset of symptoms to
acute respiratory distress syndrome (ARDS) development was 12 days.
Unlike SARS, only 2 patients with COVID-19 had diarrhea. 98% presented
abnormal lung imaging at admission and most had double-lung
abnormalities. Related to the laboratorial findings some inflammatory
indicators gradually increased during the disease progression, such as
IL-6 secretion in the circulation, procalcitonin (PCT) and C-reactive
protein (CRP), while platelets numbers decreased. The authors also
reported an initial lymphopenia that was followed by an increase in the
lymphocytes numbers. Neutrophil count increased with disease
progression.</p>
<p>The patients received different treatments such as antiviral drugs
(60.40%), glucocorticoids, thymosin and immunoglobulins. All patients
received antibiotic treatment and some received antifungal drugs. All
patients received oxygen therapy (invasive or non-invasive ones).</p>
<h3 data-number="14.51.3" id="limitations-48"><span class="header-section-number">14.51.3</span> Limitations</h3>
<p>This study involves just fatal patients, lacking comparisons with other
groups of patients e.g. patients that recovered from COVID-19. The
authors didn’t discuss the different approaches used for treatments and
how these may affect the several parameters measured. The possible
relationship between the increase of inflammatory indicators and
morbidities of COVID-19 are not discussed.</p>
<h3 data-number="14.51.4" id="significance-45"><span class="header-section-number">14.51.4</span> Significance</h3>
<p>This study has the largest cohort of fatal cases reported so far. The
authors show that COVID-19 causes fatal respiratory distress syndrome
and multiple organ failure. This study highlights prevalent myocardial
damage and indicates that cardiac function of COVID-19 patients should
be carefully monitored. The data suggest that Troponin I should be
further investigated as an early indicator of patients with high risk of
accelerated health deterioration. Secondary bacterial and fungal
infections were frequent in critically ill patients and these need to be
carefully monitored in severe COVID-19 patients. Differences in blood
type distribution were observed, suggesting that type A is detrimental
while type O is protective – but further studies are needed to confirm
these findings and elucidate if blood type influences infection or
disease severity. Several inflammatory indicators (neutrophils, PCT, CRP
and IL-6, D-dimer) increased according to disease severity and should be
assessed as biomarkers and to better understand the biology of
progression to severe disease.</p>
<h3 data-number="14.51.5" id="credit-50"><span class="header-section-number">14.51.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.52" id="relationship-between-the-abo-blood-group-and-the-covid-19-susceptibility"><span class="header-section-number">14.52</span> Relationship between the ABO Blood Group and the COVID-19 Susceptibility</h2>
<p>Zhao et al. <em>medRxiv</em>. <span class="citation" data-cites="1DbV444Pj">(<a href="#ref-1DbV444Pj" role="doc-biblioref">452</a>)</span></p>
<h3 data-number="14.52.1" id="keywords-52"><span class="header-section-number">14.52.1</span> Keywords</h3>
<ul>
<li>ABO blood group</li>
<li>COVID-19 susceptibility</li>
</ul>
<h3 data-number="14.52.2" id="main-findings-48"><span class="header-section-number">14.52.2</span> Main Findings</h3>
<p>These authors compared the ABO blood group of 2,173 patients with
RT-PCR-confirmed COVID-19 from hospitals in Wuhan and Shenzhen with the
ABO blood group distribution in unaffected people in the same cities
from previous studies (2015 and 2010 for Wuhan and Shenzhen,
respectively). They found that people with blood group A are
statistically over-represented in the number of those infected and who
succumb to death while those with blood group O are statistically
underrepresented with no influence of age or sex.</p>
<h3 data-number="14.52.3" id="limitations-49"><span class="header-section-number">14.52.3</span> Limitations</h3>
<p>This study compares patients with COVID-19 to the general population
but relies on data published 5 and 10 years ago for the control. The
mechanisms that the authors propose may underlie the differences they
observed require further study.</p>
<h3 data-number="14.52.4" id="significance-46"><span class="header-section-number">14.52.4</span> Significance</h3>
<p>Risk stratification based on blood group may be beneficial for
patients and also healthcare workers in infection control. Additionally,
investigating the mechanism behind these findings could lead to better
developing prophylactic and therapeutic targets for COVID-19.</p>
<h3 data-number="14.52.5" id="credit-51"><span class="header-section-number">14.52.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.53" id="the-inhaled-corticosteroid-ciclesonide-blocks-coronavirus-rna-replication-by-targeting-viral-nsp15"><span class="header-section-number">14.53</span> The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15</h2>
<p>Matsuyama et al. <em>bioRxiv</em> <span class="citation" data-cites="JwL9G6bd">(<a href="#ref-JwL9G6bd" role="doc-biblioref">2166</a>)</span></p>
<h3 data-number="14.53.1" id="keywords-53"><span class="header-section-number">14.53.1</span> Keywords</h3>
<ul>
<li>Corticosteroids</li>
<li>ciclesonide</li>
<li>mometasone</li>
<li>NSP15</li>
<li>MERS-CoV</li>
</ul>
<h3 data-number="14.53.2" id="main-findings-49"><span class="header-section-number">14.53.2</span> Main Findings</h3>
<p>This study reconsiders the use of inhaled corticosteroids in the
treatment of pneumonia by coronavirus. Corticosteroids were associated
with increased mortality for SARS in 2003 and for MERS in 2013, probably
due to that fact that systemic corticosteroids suppress the innate
immune system, resulting in increased viral replication. However, some
steroid compounds might block coronavirus replication. The authors
screened steroids from a chemical library and assessed the viral growth
suppression and drug cytotoxicity. Ciclesonide demonstrated low
cytotoxicity and potent suppression of MERS-CoV viral growth. The
commonly used systemic steroids cortisone, prednisolone and
dexamethasone did not suppress viral growth, nor did the commonly used
inhaled steroid fluticasone. To identify the drug target of virus
replication, the authors conducted 11 consecutive MERS-CoV passages in
the presence of ciclesonide or mometasone, and they could generate a
mutant virus that developed resistance to ciclesonide, but not to
mometasone. Afterwards, they performed next-generation sequencing and
identified an amino acid substitution in nonstructural protein 15
(NSP15) as the predicted mechanism for viral resistance to ciclesonide.
The authors were able to successfully generate a recombinant virus
carrying that amino acid substitution, which overcome the antiviral
effect of ciclesonide, suggesting that ciclosenide interacts with NSP15.
The mutant virus was inhibited by mometasone, suggesting that the
antiviral target of mometasone is different from that of ciclesonide.
Lastly, the effects of ciclesonide and mometason on suppressing the
replication of SARS-CoV-2 were evaluated. Both compounds were found to
suppress viral replication with a similar efficacy to lopinavir.</p>
<h3 data-number="14.53.3" id="limitations-50"><span class="header-section-number">14.53.3</span> Limitations</h3>
<p>Most of the experiments, including the identification of the mutation in
NSP15 were conducted with MERS-CoV. This is not the closest related
virus to SARS-CoV-2, as that would be SARS-CoV. Thus, to repeat the
initial experiments with SARS-CoV, or preferably SARS-CoV-2, is
essential. The manuscript should address this and, therefore, it will
require considerable editing for organization and clarity. Also, in
terms of cell immunogenic epitopes, while SARS-CoV-2 spike protein
contains several predicted B and T cell immunogenic epitopes that are
shared with other coronaviruses, some studies have shown critical
differences between MERS-CoV, SARS-CoV and SARS-CoV-2. A main criticism
is that the authors only used VeroE6/TMPRSS2 cells to gauge the direct
cytotoxic effects of viral replication. To evaluate this in other cell
lines, including human airway epithelial cells, is crucial, as the
infectivity of coronavirus strains greatly varies in different cell
lines,</p>
<h3 data-number="14.53.4" id="significance-47"><span class="header-section-number">14.53.4</span> Significance</h3>
<p>Nevertheless, these findings encourage evaluating ciclesonide and
mometasone as better options for patients with COVID-19 in need of
inhaled steroids, especially as an alternative to other corticosteroids
that have been shown to increase viral replication in vitro. This should
be evaluated in future clinical studies.</p>
<h3 data-number="14.53.5" id="credit-52"><span class="header-section-number">14.53.5</span> Credit</h3>
<p><em>This review was undertaken by Alvaro Moreira, MD as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.54" id="a-human-monoclonal-antibody-blocking-sars-cov-2-infection"><span class="header-section-number">14.54</span> A human monoclonal antibody blocking SARS-CoV-2 infection **</h2>
<p>Wang et al. <em>bioRxiv</em>. <span class="citation" data-cites="XcuzhxrJ">(<a href="#ref-XcuzhxrJ" role="doc-biblioref">916</a>)</span></p>
<h3 data-number="14.54.1" id="keywords-54"><span class="header-section-number">14.54.1</span> Keywords</h3>
<ul>
<li>Monoclonal antibodies</li>
<li>SARS-CoV2</li>
<li>cross-neutralization</li>
<li>potential treatment</li>
<li>spike receptor</li>
</ul>
<h3 data-number="14.54.2" id="main-findings-50"><span class="header-section-number">14.54.2</span> Main Findings</h3>
<p>The authors reported a human monoclonal
antibody that neutralizes SARS-CoV-2 and SARS-Cov which belong to same
family of corona viruses. For identifying mAbs, supernatants of a
collection of 51 hybridomas raised against the spike protein of SARS-CoV
(SARS-S) were screened by ELISA for cross-reactivity against the spike
protein of SARS-CoN2 (SARS2-S). Hybridomas were derived from immunized
transgenic H2L2 mice (chimeric for fully human VH-VL and rat constant
region). Four SARS-S hybridomas displayed cross-reactivity with SARS2-S,
one of which (47D11) exhibited cross-neutralizing activity for SARS-S
and SARS2-S pseudotyped VSV infection. A recombinant, fully human IgG1
isotype antibody was generated and used for further characterization.</p>
<p>The humanized 47D11 antibody inhibited infection of VeroE6 cells with
SARS-CoV and SARS-CoV-2 with IC50 values of 0.19 and 0.57 μg/ml
respectively. 47D11 mAb bound a conserved epitope on the spike receptor
binding domain (RBD) explaining its ability to cross-neutralize SARS-CoV
and SARS-CoV-2. 47D11 was shown to target the S1B RBD of SARS-S and
SARS2-S with similar affinities. Interestingly, binding of 47D11 to
SARS-S1B and SARS2-S1B did not interfere with S1B binding to ACE2
receptor-expressing cells assayed by flow cytometry.</p>
<h3 data-number="14.54.3" id="limitations-51"><span class="header-section-number">14.54.3</span> Limitations</h3>
<p>These results show that the human 47D11 antibody
neutralizes SARS-CoV and SARS-Cov2 infectivity via an as yet unknown
mechanism that is different from receptor binding interference.
Alternative mechanisms were proposed but these as yet remain to be
tested in the context of SARS-CoV2. From a therapeutic standpoint and in
the absence of in vivo data, it is unclear whether the 47D11 ab can
alter the course of infection in an infected host through virus
clearance or protect an uninfected host that is exposed to the virus.
There is a precedent for the latter possibility as it relates to
SARS-CoV that was cited by the authors and could turn out to be true for
SARS-CoV2.</p>
<h3 data-number="14.54.4" id="significance-48"><span class="header-section-number">14.54.4</span> Significance</h3>
<p>This study enabled the identification of novel neutralizing
antibody against COV-that could potentially be used as first line of
treatment in the near future to reduce the viral load and adverse
effects in infected patients. In addition, neutralizing antibodies such
as 47D11 represent promising reagents for developing
antigen-antibody-based detection test kits and assays.</p>
<h3 data-number="14.54.5" id="credit-53"><span class="header-section-number">14.54.5</span> Credit</h3>
<p><em>This review was edited by K. Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Heat inactivation of serum interferes with the immunoanalysis of
antibodies to SARS-CoV-2</strong></p>
<p>Heat inactivation, immunochromatography, diagnosis, serum antibodies,
IgM, IgG</p>
<p><em>Summary</em></p>
<p>The use of heat inactivation to neutralize pathogens in serum samples
collected from suspected COVID-19 patients reduces the sensitivity of a
fluorescent immunochromatographic assay to detect anti-SARS-CoV-2 IgM
and IgG.</p>
<p><em>Major findings</em></p>
<p>Coronaviruses can be killed by heat inactivation, and this is an
important safety precaution in laboratory manipulation of clinical
samples. However, the effect of this step on downstream
SARS-CoV-2-specific serum antibody assays has not been examined. The
authors tested the effect of heat inactivation (56 deg C for 30 minutes)
versus no heat inactivation on a fluorescence immunochromatography
assay. Heat inactivation reduced all IgM measurements by an average of
54% and most IgG measurements (22/36 samples, average reduction of 50%),
consistent with the lower thermal stability of IgM than that of IgG.
Heat inactivation caused a subset of IgM but not IgG readings to fall
below a specified positivity threshold.</p>
<p><em>Limitations</em></p>
<p>Limitations included the use of only one type of assay for testing heat
inactivated vs non-inactivated sera, and the use of the same baseline
for heat inactivated and non-inactivated sera. The results indicate that
heat inactivation affects the quantification of SARS-CoV-2-antibody
response, specially IgM, but still allows to distinguish positive
specific IgG. Therefore, the effect of heat inactivation should be
studied when designing assays that quantitatively associate
immunoglobulin levels (especially IgM) to immune state.</p>
<p><em>Review by Andrew M. Leader as part of a project by students, postdocs
and faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</em></p>
<h2 data-number="14.55" id="immune-phenotyping-based-on-neutrophil-to-lymphocyte-ratio-and-igg-predicts-disease-severity-and-outcome-for-patients-with-covid-19"><span class="header-section-number">14.55</span> Immune phenotyping based on neutrophil-to-lymphocyte ratio and IgG predicts disease severity and outcome for patients with COVID-19</h2>
<p>Zhang et al. <em>medRxiv</em> <span class="citation" data-cites="1CwCTXPqj">(<a href="#ref-1CwCTXPqj" role="doc-biblioref">2167</a>)</span></p>
<h3 data-number="14.55.1" id="keywords-55"><span class="header-section-number">14.55.1</span> Keywords</h3>
<ul>
<li>Biomarkers</li>
<li>cytokines</li>
<li>IgG</li>
<li>immune cells</li>
</ul>
<h3 data-number="14.55.2" id="main-findings-51"><span class="header-section-number">14.55.2</span> Main Findings</h3>
<p>In a cohort of 222 patients, anti-SARS-CoV-2 IgM and IgG levels were
analyzed during acute and convalescent phases (up to day 35) and
correlated to the diseases’ severity. The same was done with
neutrophil-to-lymphocyte ratio. High IgG levels and high
neutrophil-to-lymphocyte ratio in convalescence were both independently
associated to the severity of the disease. The simultaneous occurrence
of both of these laboratory findings correlated even stronger to the
diseases’ severity.</p>
<p>Severe cases with high neutrophil-to-lymphocyte ratios had clearly
higher levels of IL-6. The authors propose that a robust IgG response
leads to immune-mediated tissue damage, thus explaining the worse
outcome in patients with overexuberant antibody response.</p>
<h3 data-number="14.55.3" id="limitations-52"><span class="header-section-number">14.55.3</span> Limitations</h3>
<p>A main criticism is that the criteria for stratifying patients in severe
vs. non-severe are not described. The only reference related to this is
the difference between the percentage of patients who needed mechanical
ventilation, which was greater in patients with both high IgG levels and
high neutrophil-to-lymphocyte ratio. No patient with both low IgG levels
and low neutrophil-to-lymphocyte ratio was treated with mechanical
ventilation.</p>
<p>The proposed correlation of severity with IL-2 and IL-10 levels is not
very strong.</p>
<p>Furthermore, although mostly ignored in the paper’s discussion, one of
the most interesting findings is that an early increase in
anti-SARS-CoV-2 IgM levels also seems to correlate with severe disease.
However, as only median values are shown for antibody kinetics curves,
the extent of variation in acute phase cannot be assessed.</p>
<h3 data-number="14.55.4" id="significance-49"><span class="header-section-number">14.55.4</span> Significance</h3>
<p>Anti-SARS-CoV-2 IgG levels and with neutrophil-to-lymphocyte ratio
predict severity of COVID-19 independently of each other. An additive
predictive value of both variables is noticeable. Importantly, an
early-on increase in anti-SARS-CoV-2 IgM levels also seem to predict
outcome.</p>
<h3 data-number="14.55.5" id="credit-54"><span class="header-section-number">14.55.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.56" id="reinfection-could-not-occur-in-sars-2-cov-2-infected-rhesus-macaques"><span class="header-section-number">14.56</span> Reinfection could not occur in SARS-2 CoV-2 infected rhesus macaques</h2>
<p>Bao et al. <em>bioRxiv</em> <span class="citation" data-cites="3XqFhi8l">(<a href="#ref-3XqFhi8l" role="doc-biblioref">2168</a>)</span></p>
<h3 data-number="14.56.1" id="keywords-56"><span class="header-section-number">14.56.1</span> Keywords</h3>
<ul>
<li>SARS-CoV-2</li>
<li>viral load</li>
<li>reinfection</li>
<li>relapse</li>
<li>non-human primate model</li>
</ul>
<h3 data-number="14.56.2" id="main-findings-52"><span class="header-section-number">14.56.2</span> Main Findings</h3>
<p>This study addresses the issue or acquired immunity after a primary
COVID-19 infection in rhesus monkeys. Four Chinese rhesus macaques were
intratracheally infected with SARS-CoV-2 and two out of the four were
re-infected at 28 days post initial infection (dpi) with the same viral
dose after confirming the recovery by the absence of clinical symptoms,
radiological abnormalities and viral detection (2 negative RT-PCR
tests). While the initial infection led the viral loads in nasal and
pharyngeal swabs that reach approximately 6.5 log<sub>10</sub> RNA copies/ml at 3
dpi in all four monkeys, viral loads in the swabs tested negative after
reinfection in the two reinfected monkeys. In addition, the necropsies
from a monkey (M1) at 7 days after primary infection, and another monkey
(M3) at 5 days post reinfection, revealed the histopathological damages
and viral replication in the examined tissues from M1, while no viral
replication as well as no histological damages were detected in the
tissues from M3. Furthermore, sera from three monkeys at 21 and 28 dpi
exhibited neutralizing activity against SARS-CoV-2 in vitro, suggesting
the production of protective neutralizing antibodies in these monkeys.
Overall, this study indicates that primary infection with SARS-CoV-2 may
protect from subsequent exposure to the same virus.</p>
<h3 data-number="14.56.3" id="limitations-53"><span class="header-section-number">14.56.3</span> Limitations</h3>
<p>In human, virus has been detected by nasopharyngeal swabs until 9 to 15
days after the onset of symptoms. In the infected monkeys in this study,
virus were detected from day 1 after the infection, declining to
undetectable level by day 15 post infection. It may suggest that there
is a faster viral clearance mechanism in monkeys, therefore the
conclusions of reinfection protection for humans need to be carefully
considered. In addition, only two monkeys were re-infected in this study
and the clinical signs of these monkeys were not similar: M3 did not
show weight loss and M4 showed relatively higher fever on the day of
infection and the day of re-challenge.</p>
<h3 data-number="14.56.4" id="significance-50"><span class="header-section-number">14.56.4</span> Significance</h3>
<p>This study showed clear viral clearance and no indications of relapse or
viremia after a secondary infection with SARS-CoV-2 in a Chinese rhesus
macaque model. These results support the idea that patients with full
recovery (two negative RT-PCR results) may also be protected from
secondary SARS-CoV-2 infection. Recovered patients may be able to
reintegrate to normal public life and provide protective serum perhaps
even if having had a mild infection. The results are also encouraging
for successful vaccine development against SARS-CoV-2.</p>
<h3 data-number="14.56.5" id="credit-55"><span class="header-section-number">14.56.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.57" id="a-highly-conserved-cryptic-epitope-in-the-receptor-binding-domains-of-sars-cov-2-and-sars-cov"><span class="header-section-number">14.57</span> A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV</h2>
<p><span class="citation" data-cites="6gQxRkWM">(<a href="#ref-6gQxRkWM" role="doc-biblioref">2169</a>)</span></p>
<h3 data-number="14.57.1" id="keywords-57"><span class="header-section-number">14.57.1</span> Keywords</h3>
<ul>
<li>neutralizing antibody</li>
<li>cross-reactivity</li>
</ul>
<h3 data-number="14.57.2" id="main-findings-53"><span class="header-section-number">14.57.2</span> Main Findings</h3>
<p>Given the sequence similarity of the surface spike glycoprotein (S) of
SARS-CoV-2 and SAR-CoV, Yuan et al. (2020) propose that neutralizing
antibodies isolated from convalescent SARS-CoV patients may offer
insight into cross-reactive antibodies targeting SARS-CoV-2. In
particular, they find that the receptor-binding domain (RBD) of
SARS-CoV-2 S protein shares 86% sequence similarity with the RBD of
SARS-CoV S protein that binds to the CR3022 neutralizing antibody.
CR3022 also displays increased affinity for the “up” conformation of the
SARS-CoV-2 S protein compared to the “down” conformation as it does for
the SARS-CoV S protein. Therefore, the authors propose that this
cross-reactive antibody may confer some degree of protection <em>in vivo</em>
even if it fails to neutralize <em>in vitro</em>.</p>
<h3 data-number="14.57.3" id="limitations-54"><span class="header-section-number">14.57.3</span> Limitations</h3>
<p>Although the authors offer a logical rationale for
identifying cross-reactive neutralizing antibodies derived from
SARS-CoV, their study using only CR3022 failed to demonstrate whether
this approach will be successful. After all, CR3022 failed to neutralize
<em>in vitro</em> despite the binding affinity to a similar epitope on
SARS-CoV-2. They would benefit from testing more candidates and using an
<em>in vivo</em> model to demonstrate their claim that protection may be
possible in the absence neutralization if combinations are used <em>in
vivo.</em></p>
<h3 data-number="14.57.4" id="significance-51"><span class="header-section-number">14.57.4</span> Significance</h3>
<p>The ability to make use of previously characterized
neutralizing antibodies for conserved epitopes can expedite drug design
and treatment options.</p>
<h3 data-number="14.57.5" id="credit-56"><span class="header-section-number">14.57.5</span> Credit</h3>
<p><em>This review was undertaken by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.58" id="highly-accurate-and-sensitive-diagnostic-detection-of-sars-cov-2-by-digital-pcr"><span class="header-section-number">14.58</span> Highly accurate and sensitive diagnostic detection of SARS-CoV-2 by digital PCR</h2>
<p>Dong et al. <em>medRxiv</em> <span class="citation" data-cites="UYD5T2mp">(<a href="#ref-UYD5T2mp" role="doc-biblioref">2170</a>)</span></p>
<h3 data-number="14.58.1" id="keywords-58"><span class="header-section-number">14.58.1</span> Keywords</h3>
<ul>
<li>Diagnosis</li>
<li>digital PCR</li>
</ul>
<h3 data-number="14.58.2" id="main-findings-54"><span class="header-section-number">14.58.2</span> Main Findings</h3>
<p>The authors present a digital PCR (dPCR) diagnostic test for
SARS-CoV-2 infection. In 103 individuals that were confirmed in a
follow-up to be infected, the standard qPCR test had a positivity
rate of 28.2% while the dPCR test detected 87.4% of the infections
by detecting an additional 61 positive cases. The authors also
tested samples from close contacts (early in infection stage) and
convalescing individuals (late in infection stage) and were able to
detect SARS-CoV-2 nucleic acid in many more samples using dPCR
compared to qPCR.</p>
<h3 data-number="14.58.3" id="limitations-55"><span class="header-section-number">14.58.3</span> Limitations</h3>
<p>I did not detect limitations.</p>
<h3 data-number="14.58.4" id="significance-52"><span class="header-section-number">14.58.4</span> Significance</h3>
<p>The authors make a strong case for the need for a highly sensitive
and accurate confirmatory method for diagnosing COVID-19 during this
outbreak and present a potential addition to the diagnostic arsenal.
They propose a dPCR test that they present has a dramatically lower
false negative rate than the standard RT-qPCR tests and can be
especially beneficial in people with low viral load, whether they
are in the earlier or later stages of infection.</p>
<h3 data-number="14.58.5" id="credit-57"><span class="header-section-number">14.58.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.59" id="sars-cov-2-invades-host-cells-via-a-novel-route-cd147-spike-protein"><span class="header-section-number">14.59</span> SARS-CoV-2 invades host cells via a novel route: CD147-spike protein</h2>
<p>Wang et al. <em>bioRxiv</em> <span class="citation" data-cites="14y8UvAbH">(<a href="#ref-14y8UvAbH" role="doc-biblioref">2171</a>)</span></p>
<h3 data-number="14.59.1" id="keywords-59"><span class="header-section-number">14.59.1</span> Keywords</h3>
<ul>
<li>spike protein</li>
<li>viral entry</li>
<li>CD147</li>
<li>SARS-CoV-2</li>
</ul>
<h3 data-number="14.59.2" id="main-findings-55"><span class="header-section-number">14.59.2</span> Main Findings</h3>
<p>The authors propose a novel mechanism of SARS-CoV-2 viral entry through
the interaction of the viral spike protein (SP) and the immunoglobulin
superfamily protein CD147 (also known as Basigin). Using an in-house
developed humanized antibody against CD147 (maplazumab), they show that
blocking CD147 decreases viral replication in Vero E6 cells. Using
surface plasmon resonance (SPR), ELISA, and Co-IP assays, they show that
the spike protein of SARS-CoV-2 directly interacts with CD147. Lastly,
they utilize immune-election microscopy to show spike protein and CD147
localize to viral inclusion bodies of Vero E6 cells.</p>
<h3 data-number="14.59.3" id="limitations-56"><span class="header-section-number">14.59.3</span> Limitations</h3>
<p>The authors claim that an anti-CD147 antibody (Meplazumab) inhibits
SARS-CoV-2 replication by testing cell growth and viral load in cells
infected with SARS-CoV-2, however there are key pieces of this
experiment that are missing. First, the authors fail to use a
non-specific antibody control. Second, the authors claim that viral
replication is inhibited, and that they test this by qPCR, however this
data is <strong>not shown.</strong> To further prove specificity, the authors should
introduce CD147 to non-susceptible cells and show that they become
permissive.</p>
<p>The authors claim that there is a direct interaction between CD147 and
SP through SPR, ELISA, and Co-IP, and this data seems generally
convincing. The electron microscopy provides further correlative
evidence that SARS-CoV-2 may interact with CD147 as they are both found
in the same viral inclusion body. A quantification of this data would
make the findings more robust.</p>
<p>Finally, the data in this paper lacks replicates, error bars, and
statistics to show that the data are reproducible and statistically
significant.</p>
<h3 data-number="14.59.4" id="significance-53"><span class="header-section-number">14.59.4</span> Significance</h3>
<p>It has been shown in various studies that SARS-CoV-2 binds to the cell
surface protein ACE2 for cell entry, yet ACE2 is highly expressed in
heart, kidney, and intestinal cells, raising the concern that blocking
ACE2 would result in harmful side effects <span class="citation" data-cites="BcOf65BE">(<a href="#ref-BcOf65BE" role="doc-biblioref">2172</a>)</span> CD147 on the other hand
is highly expressed in various tumor types, inflamed tissues, and
pathogen infected cells, suggesting that the inhibition of CD147 would
not result in major side effects <span class="citation" data-cites="34nQJalJ KcThTnsm">(<a href="#ref-34nQJalJ" role="doc-biblioref">2173</a>, <a href="#ref-KcThTnsm" role="doc-biblioref">2174</a>)</span> The research in this paper has
resulted in an ongoing clinical trail in China to test the safety and
efficacy of anti-CD147 Meplazumab to treat COVID-19. (ClinicalTrails.gov
identifier NCT04275245).</p>
<h3 data-number="14.59.5" id="credit-58"><span class="header-section-number">14.59.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.60" id="blood-single-cell-immune-profiling-reveals-that-interferon-mapk-pathway-mediated-adaptive-immune-response-for-covid-19"><span class="header-section-number">14.60</span> Blood single cell immune profiling reveals that interferon-MAPK pathway mediated adaptive immune response for COVID-19</h2>
<p>Huang et al. <em>medRxiv</em> <span class="citation" data-cites="18l4l0nLX">(<a href="#ref-18l4l0nLX" role="doc-biblioref">2175</a>)</span></p>
<h3 data-number="14.60.1" id="keywords-60"><span class="header-section-number">14.60.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>PBMC</li>
<li>single cell</li>
<li>MAPK</li>
</ul>
<h3 data-number="14.60.2" id="main-findings-56"><span class="header-section-number">14.60.2</span> Main Findings</h3>
<p>The authors performed single-cell RNA sequencing (scRNAseq) of
peripheral blood mononuclear cells isolated from whole blood samples of
COVID-19 patients (n=10). Data was compared to scRNAseq of samples
collected from patients with influenza A (n=1), acute pharyngitis (n=1),
and cerebral infarction (n=1), as well as, three healthy controls.
COVID-19 patients were categorized into those with moderate (n=6),
severe (n=1), critical (n=1), and cured (n=2) disease. Analysis across
all COVID-19 disease levels revealed 56 different cellular subtypes,
among 17 immune cell types; comparisons between each category to the
normal controls revealed <strong>increased proportions of CD1c<sup>+</sup> dendritic
cells, CD8<sup>+</sup> CTLs, and plasmacytoid dendritic cells and a decrease in
proportions of B cells and CD4<sup>+</sup> T cells</strong>.</p>
<p><strong>TCR sequencing revealed that greater clonality is associated with
milder COVID-19 disease</strong>; BCR sequencing revealed that COVID-19
patients have circulating antibodies against known viral antigens,
including EBV, HIV, influenza A, and other RNA viruses. This may suggest
that the immune response to SARS-CoV-2 infection elicits production of
antibodies against known RNA viruses.</p>
<p>Excluding enriched pathways shared by COVID-19 patients and patients
with other conditions (influenza A, acute pharyngitis, and cerebral
infarction), the authors identified the <strong>interferon-MAPK signaling
pathway as a major response to SARS-CoV-2 infection</strong>. The authors
performed quantitative real-time reverse transcriptase polymerase chain
reaction (RT-PCR) for interferon-MAPK signaling genes: <em>IRF27, BST2,</em>
and <em>FOS</em>. These samples were collected from a separate cohort of
COVID-19 patients (critical, n=3; severe, n=3; moderate, n=19; mild,
n=3; and cured, n=10; and healthy controls, n=5). Notably, consistent
with the original scRNAseq data, <em>FOS</em> showed up-regulation in COVID-19
patients and down-regulation in cured patients. <strong>The authors propose
that <em>FOS</em> may be a candidate marker gene for curative COVID-19
disease</strong>.</p>
<h3 data-number="14.60.3" id="limitations-57"><span class="header-section-number">14.60.3</span> Limitations</h3>
<p>The sample size of this study is limited. To further delineate
differences in the immune profile of peripheral blood of COVID-19
patients, a greater sample size is needed, and longitudinal samples are
needed, as well. A better understanding of the immunological
interactions in cured patients, for example, would require a profile
before and after improvement.</p>
<p>Moreover, the conclusions drawn from this scRNAseq study point to
potential autoimmunity and immune deficiency to distinguish different
severities of COVID-19 disease. However, this requires an expanded
number of samples and a more robust organization of specific immune cell
subtypes that can be compared across different patients. Importantly,
this criterion is likely needed to ensure greater specificity in
identifying markers for COVID-19 infection and subsequent immune
response.</p>
<h3 data-number="14.60.4" id="significance-54"><span class="header-section-number">14.60.4</span> Significance</h3>
<p>At the single-cell level, COVID-19 disease has been characterized in the
lung, but a greater understanding of systemic immunological responses is
furthered in this study. Type I interferon is an important signaling
molecule for the anti-viral response. The identification of the
interferon-MAPK signaling pathway and the differential expression of
MAPK regulators between patients of differing COVID-19 severity and
compared to cured patients may underscore the importance of either
immune deficiency or autoimmunity in COVID-19 disease.</p>
<h3 data-number="14.60.5" id="credit-59"><span class="header-section-number">14.60.5</span> Credit</h3>
<p><em>This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.61" id="cross-reactive-antibody-response-between-sars-cov-2-and-sars-cov-infection."><span class="header-section-number">14.61</span> Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infection.</h2>
<p>Lv et al. <em>bioRxiv</em> <span class="citation" data-cites="dM1rOHmH">(<a href="#ref-dM1rOHmH" role="doc-biblioref">2176</a>)</span></p>
<h3 data-number="14.61.1" id="keywords-61"><span class="header-section-number">14.61.1</span> Keywords</h3>
<p>SARS-CoV-2, SARS-CoV, spike protein, RBD, cross-reactivity,
cross-neutralization, antibody, human patients, mouse</p>
<h3 data-number="14.61.2" id="main-findings-57"><span class="header-section-number">14.61.2</span> Main Findings</h3>
<p>The authors explore the antigenic differences between SARS-CoV-2 and
SARS-CoV by analyzing plasma samples from SARS-CoV-2 (n = 15) and
SARS-CoV (n = 7) patients. Cross-reactivity in antibody binding to the
spike protein between SARS-CoV-2 and SARS-CoV was found to be common,
mostly targeting non-RBD regions in plasma from SARS-CoV-2 patients.
Only one SARS-CoV-2 plasma sample was able to cross-neutralize SARS-CoV,
with low neutralization activity. No cross-neutralization response was
detected in plasma from SARS-CoV patients.</p>
<p>To further investigate the cross-reactivity of antibody responses to
SARS-CoV-2 and SARS-CoV, the authors analyzed the antibody response of
plasma collected from mice infected or immunized with SARS-CoV-2 or
SARS-CoV (n = 5 or 6 per group). Plasma from mice immunized with
SARS-CoV-2 displayed cross-reactive responses to SARS-CoV S ectodomain
and, to a lesser extent, SARS-CoV RBD. Similarly, plasma from mice
immunized with SARS-CoV displayed cross-reactive responses to SARS-CoV-2
S ectodomain. Cross-neutralization activity was not detected in any of
the mouse plasma samples.</p>
<h3 data-number="14.61.3" id="limitations-58"><span class="header-section-number">14.61.3</span> Limitations</h3>
<p>The size of each patient cohort is insufficient to accurately determine
the frequency of cross-reactivity and cross-neutralization in the
current SARS-CoV-2 pandemic. Recruitment of additional patients from a
larger range of geographical regions and time points would also enable
exploration into the effect of the genetic diversity and evolution of
the SARS-CoV-2 virus on cross-reactivity. This work would also benefit
from the mapping of specific epitopes for each sample. Future studies
may determine whether the non-neutralizing antibody responses can confer
<em>in vitro</em> protection or lead to antibody-dependent disease enhancement.</p>
<h3 data-number="14.61.4" id="significance-55"><span class="header-section-number">14.61.4</span> Significance</h3>
<p>The cross-reactive antibody responses to S protein in the majority of
SARS-CoV-2 patients is an important consideration for development of
serological assays and vaccine development during the current outbreak.
The limited extent of cross-neutralization demonstrated in this study
indicates that vaccinating to cross-reactive conserved epitopes may have
limited efficacy, presenting a key concern for the development of a more
universal coronavirus vaccine to address the global health risk of novel
coronavirus outbreaks.</p>
<h3 data-number="14.61.5" id="credit-60"><span class="header-section-number">14.61.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.62" id="the-feasibility-of-convalescent-plasma-therapy-in-severe-covid-19-patients-a-pilot-study"><span class="header-section-number">14.62</span> The feasibility of convalescent plasma therapy in severe COVID-19 patients: a pilot study</h2>
<p>Duan et al. <em>medRxiv</em> <span class="citation" data-cites="UENoLkhz">(<a href="#ref-UENoLkhz" role="doc-biblioref">2177</a>)</span></p>
<h3 data-number="14.62.1" id="keywords-62"><span class="header-section-number">14.62.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS-CoV-2</li>
<li>convalescent plasma</li>
<li>treatment outcome</li>
<li>pilot</li>
<li>therapy</li>
<li>transfusion</li>
</ul>
<h3 data-number="14.62.2" id="main-findings-58"><span class="header-section-number">14.62.2</span> Main Findings</h3>
<p>This is the first report to date of convalescent plasma therapy as a
therapeutic against COVID-19 disease. This is a feasibility pilot study.
The authors report the administration and clinical benefit of 200 mL of
convalescent plasma (CP) (1:640 titer) derived from recently cured
donors (CP selected among 40 donors based on high neutralizing titer and
ABO compatibility) to 10 severe COVID-19 patients with confirmed
viremia. The primary endpoint was the safety of CP transfusion. The
secondary endpoint were clinical signs of improvement based on symptoms
and laboratory parameters.</p>
<p>The authors reported use of methylene blue photochemistry to inactivate
any potential residual virus in the plasma samples, without compromising
neutralizing antibodies, and no virus was detected before transfusion.</p>
<p>The authors report the following:</p>
<ul>
<li><p>No adverse events were observed in all patients, except 1 patient
who exhibited transient facial red spotting.</p></li>
<li><p>All patients showed significant improvement in or complete
disappearance of clinical symptoms, including fever, cough,
shortness of breath, and chest pain after 3 days of CP therapy.</p></li>
<li><p>Reduction of pulmonary lesions revealed by chest CT.</p></li>
<li><p>Elevation of lymphocyte counts in patients with lymphocytopenia.</p></li>
<li><p>Increase in SaO<sub>2</sub> in all patients, indicative of recuperating lung
function.</p></li>
<li><p>Resolution of SARS-CoV-2 viremia in 7 patients and increase in
neutralizing antibody titers in 5 patients. Persistence of
neutralizing antibody levels in 4 patients.</p></li>
</ul>
<h3 data-number="14.62.3" id="limitations-59"><span class="header-section-number">14.62.3</span> Limitations</h3>
<p>It is important to note that most recipients had high neutralization
titers of antibodies before plasma transfusion and even without
transfusion it would be expected to see an increase in neutralizing
antibodies over time. In addition to the small sample set number (n=10),
there are additional limitations to this pilot study:</p>
<ol type="1">
<li><p>All patients received concurrent therapy, in addition to the CP
transfusion. Therefore, it is unclear whether a combinatorial or
synergistic effect between these standards of care and CP
transfusion contributed to the clearance of viremia and improvement
of symptoms in these COVID-19 patients.</p></li>
<li><p>The kinetics of viral clearance was not investigated, with respect
to the administration of CP transfusion. So, the definitive impact
of CP transfusion on immune dynamics and subsequent viral load is
not well defined.</p></li>
<li><p>Comparison with a small historical control group is not ideal.</p></li>
</ol>
<h3 data-number="14.62.4" id="significance-56"><span class="header-section-number">14.62.4</span> Significance</h3>
<p>For the first time, a pilot study provides promising results involving
the use of convalescent plasma from cured COVID-19 patients to treat
others with more severe disease. The authors report that the
administration of a single, high-dose of neutralizing antibodies is
safe. In addition, there were encouraging results with regards to the
reduction of viral load and improvement of clinical outcomes. It is,
therefore, necessary to expand this type of study with more
participants, in order to determine optimal dose and treatment kinetics.
It is important to note that CP has been studied to treat H1N1
influenza, SARS-CoV-1, and MERS-CoV, although it has not been proven to
be effective in treating these infections.</p>
<h3 data-number="14.62.5" id="credit-61"><span class="header-section-number">14.62.5</span> Credit</h3>
<p><em>Review by Matthew D. Park and revised by Alice O. Kamphorst and Maria A. Curotto de Lafaille as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.63" id="hydroxychloroquine-and-azithromycin-as-a-treatment-of-covid-19-results-of-an-open-label-non-randomized-clinical-trial"><span class="header-section-number">14.63</span> Hydroxychloroquine and azithromycin as a treatment of COVID-19: results of an open label non-randomized clinical trial</h2>
<p><span class="citation" data-cites="Xbkby2Jr">(<a href="#ref-Xbkby2Jr" role="doc-biblioref">2178</a>)</span></p>
<h3 data-number="14.63.1" id="keywords-63"><span class="header-section-number">14.63.1</span> Keywords</h3>
<ul>
<li>hydroxychloroquine</li>
<li>clearance</li>
<li>viral load</li>
<li>clinical trial</li>
</ul>
<h3 data-number="14.63.2" id="main-findings-59"><span class="header-section-number">14.63.2</span> Main Findings</h3>
<p>This study was a single-arm, open label clinical trial with
600 mg hydroxychloroquine (HCQ) in the treatment arm (n = 20). Patients
who refused participation or patients from another center not treated
with HCQ were included as negative controls (n = 16). Among the patients
in the treatment arm, 6 received concomitant azithromycin to prevent
superimposed bacterial infection. The primary endpoint was respiratory
viral loads on day 6 post enrollment, measured by nasopharyngeal swab
followed by real-time reverse transcription-PCR.</p>
<p>HCQ alone was able to significantly reduce viral loads by day 6 (n =
8/14, 57.1% complete clearance, p &lt; 0.001); azithromycin appears to
be synergistic with HCQ, as 6/6 patients receiving combined treatment
had complete viral clearance (p &lt; 0.001).</p>
<h3 data-number="14.63.3" id="limitations-60"><span class="header-section-number">14.63.3</span> Limitations</h3>
<p>Despite what is outlined above, this study has a number
of limitations that must be considered. First, there were originally n =
26 patients in the treatment arm, with 6 lost to follow up for the
following reasons: 3 transferred to ICU, 1 discharge, 1
self-discontinued treatment d/t side effects, and 1 patient expired.
Total length of clinical follow up was 14 days, but the data beyond day
6 post-inclusion are not shown.</p>
<p>Strikingly, in supplementary table 1, results of the real-time RT-PCR
are listed for the control and treatment arms from D0 – D6. However, the
data are not reported in a standard way, with a mix of broadly positive
or negative result delineation with Ct (cycle threshold) values, the
standard output of real time PCR. It is impossible to compare what is
defined as a positive value between the patients in the control and
treatment arms without a standardized threshold for a positive test.
Further, the starting viral loads reported at D0 in the groups receiving
HCQ or HCQ + azithromycin were significantly different (ct of 25.3 vs
26.8 respectively), which could explain in part the differences observed
in the response to treatment between 2 groups. Finally, patients in the
control arm from outside the primary medical center in this study
(Marseille) did not actually have samples tested by PCR daily. Instead,
positive test results from every other day were extrapolated to mean
positive results on the day before and after testing as well (Table 2,
footnote <sup>a</sup>).</p>
<p>Taken together, the results of this study suggest that HCQ represents a
promising treatment avenue for COVID-19 patients. However, the limited
size of the trial, and the way in which the results were reported does
not allow for other medical centers to extrapolate a positive or
negative result in the treatment of their own patients with HCQ +/-
azithromycin. Further larger randomized clinical trials will be required
to ascertain the efficacy of HCQ +/- azithromycin in the treatment of
COVID-19.</p>
<h3 data-number="14.63.4" id="significance-57"><span class="header-section-number">14.63.4</span> Significance</h3>
<p>Chloroquine is thought to inhibit viral infection,
including SARS-Cov-2, by increasing pH within endosomes and lysosomes,
altering the biochemical conditions required for viral fusion <span class="citation" data-cites="IhdcvbzC BIlqPh6M">(<a href="#ref-IhdcvbzC" role="doc-biblioref">806</a>, <a href="#ref-BIlqPh6M" role="doc-biblioref">2179</a>)</span>.
However, chloroquine also has immuno-modulatory effects that I think may
play a role. Chloroquine has been shown to increase CTLA-4 expression at
the cell surface by decreasing its degradation in the endo-lysosome
pathway; AP-1 traffics the cytoplasmic tail of CTLA-4 to lysosomes, but
in conditions of increased pH, the protein machinery required for
degradation is less functional <span class="citation" data-cites="Q9tXjG1R">(<a href="#ref-Q9tXjG1R" role="doc-biblioref">2180</a>)</span>. As such, more CTLA-4 remains in
endosomes and is trafficked back to the cell surface. It is possible
that this may also contribute to patient recovery via reduction of
cytokine storm, in addition to the direct anti-viral effects of HCQ.</p>
<h3 data-number="14.63.5" id="credit-62"><span class="header-section-number">14.63.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.64" id="recapitulation-of-sars-cov-2-infection-and-cholangiocyte-damage-with-human-liver-organoids"><span class="header-section-number">14.64</span> Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids</h2>
<p><span class="citation" data-cites="OuDMtCRl">(<a href="#ref-OuDMtCRl" role="doc-biblioref">2105</a>)</span></p>
<h3 data-number="14.64.1" id="keywords-64"><span class="header-section-number">14.64.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>liver</li>
<li>organoids</li>
<li>Cholangiocyte</li>
</ul>
<h3 data-number="14.64.2" id="main-findings-60"><span class="header-section-number">14.64.2</span> Main Findings</h3>
<ul>
<li><p>Used human liver ductal organoids to determine ACE2+ cholangiocytes
in healthy liver (2.92% of all cells) are infectable and support
SARS-CoV-2 viral replication.</p></li>
<li><p>Plaque-purified SARS-CoV-2 viral infection disrupted organoid
barrier and bile transporting functions of cholangiocytes through
dysregulation of genes involved in tight junction formation (CLDN1)
and bile acid transportation (ASBT and CFTR).</p></li>
</ul>
<h3 data-number="14.64.3" id="limitations-61"><span class="header-section-number">14.64.3</span> Limitations:</h3>
<ul>
<li><p>Unclear if liver damage observed in patients due to direct
cholangiocyte infection or due to secondary immune/cytokine effects.
This study argues for direct damage as it lacks immune contexture;
but further studies needed with autopsy samples or organoid-immune
cell co-culture to conclude strongly.</p></li>
<li><p>Would be important to measure cholangiocyte-intrinsic anti-viral
response and alarmins secreted upon infection, and furthermore study
tropism of various immune cells to conditioned media from organoids
infected with SARS-CoV-2.</p></li>
<li><p>Does not address how cirrhotic liver or
alcohol/smoking/obesity-associated liver organoids respond to
SARS-CoV-2 infectivity and replication, worth pursuing to
experimentally address clinical data indicating co-morbidities.</p></li>
</ul>
<h3 data-number="14.64.4" id="significance-58"><span class="header-section-number">14.64.4</span> Significance</h3>
<ul>
<li><p>Useful model to rapidly study drug activity against SARS-CoV-2
infection in liver, while monitoring baseline liver damage.</p></li>
<li><p>Liver abnormality observed in &gt;50% of CoVID-19 patients; the
results from this study could explain the bile acid accumulation and
consequent liver damage observed.</p></li>
</ul>
<h3 data-number="14.64.5" id="credit-63"><span class="header-section-number">14.64.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.65" id="the-sequence-of-human-ace2-is-suboptimal-for-binding-the-s-spike-protein-of-sars-coronavirus-2"><span class="header-section-number">14.65</span> The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2</h2>
<p><span class="citation" data-cites="3VPi509z">(<a href="#ref-3VPi509z" role="doc-biblioref">2181</a>)</span></p>
<h3 data-number="14.65.1" id="keywords-65"><span class="header-section-number">14.65.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>Spike protein S</li>
<li>ACE2</li>
</ul>
<h3 data-number="14.65.2" id="main-findings-61"><span class="header-section-number">14.65.2</span> Main Findings</h3>
<p>Severe Acute Respiratory Syndrome Coronavirus 2
(SARS-CoV-2) infects cells through S spike glycoprotein binding
angiotensin-converting enzyme (ACE2) on host cells. S protein can bind
both membrane-bound ACE2 and soluble ACE2 (sACE2), which can serve as a
decoy that neutralizes infection. Recombinant sACE2 is now being tested
in clinical trials for COVID-19. To determine if a therapeutic sACE2
with higher affinity for S protein could be designed, authors generated
a library containing every amino acid substitution possible at the 117
sites spanning the binding interface with S protein. The ACE2 library
was expressed in human Expi293F cells and cells were incubated with
medium containing the receptor binding domain (RBD) of SARS-CoV-2 fused
to GFP. Cells with high or low affinity mutant ACE2 receptor compared to
affinity of wild type ACE2 for the RBD were FACS sorted and transcripts
from these sorted populations were deep sequenced. Deep mutagenesis
identified numerous mutations in ACE2 that enhance RBD binding. This
work serves to identify putative high affinity ACE2 therapeutics for the
treatment of CoV-2.</p>
<h3 data-number="14.65.3" id="limitations-62"><span class="header-section-number">14.65.3</span> Limitations</h3>
<p>The authors generated a large library of mutated
ACE2, expressed them in human Expi293F cells, and performed deep
mutagenesis to identify enhanced binders for the RBD of SARS-CoV-2 S
protein. While these data serve as a useful resource, the ability of the
high affinity ACE2 mutants identified to serve as therapeutics needs
further validation in terms of conformational stability when purified as
well as efficacy/safety both <em>in vitro</em> and <em>in vivo</em>. Additionally,
authors mentioned fusing the therapeutic ACE2 to Fc receptors to elicit
beneficial host immune responses, which would need further design and
validation.</p>
<h3 data-number="14.65.4" id="significance-59"><span class="header-section-number">14.65.4</span> Significance</h3>
<p>This study identified structural ACE2 mutants that
have potential to serve as therapeutics in the treatment of SARS-CoV-2
upon further testing and validation.</p>
<h3 data-number="14.65.5" id="credit-64"><span class="header-section-number">14.65.5</span> Credit</h3>
<p><em>This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong><em>Title</em></strong>:  A serological assay to detect SARS-Cov-2 seroconversion in
humans</p>
<p><strong><em>Immunology keywords</em></strong>:  specific serological assay - ELISA -
seroconversion - antibody titers</p>
<p><em>Note: the authors of this review work in the same institution as the
authors of the study</em></p>
<p><strong><em>Main findings</em></strong>:  </p>
<p>Production of recombinant whole Spike (S) protein and the smaller
Receptor Binding Domain (RBD) based on the sequence of Wuhan-Hu-1
SARS-CoV-2 isolate. The S protein was modified to allow trimerization
and increase stability. The authors compared the antibody reactivity of
59 banked human serum samples (non-exposed) and 3 serum samples from
confirmed SARS-CoV-2 infected patients. All Covid-19 patient sera
reacted to the S protein and RBD domain compared to the control sera.</p>
<p>The authors also characterized the antibody isotypes from the Covid-19
patients, and observed stronger IgG3 response than IgG1. IgM and IgA
responses were also prevalent.</p>
<p><strong><em>Limitations of the study</em></strong>:  </p>
<p>The authors analyzed a total of 59 control human serum samples, and
samples from only three different patients to test for reactivity
against the RBD domain and full-length spike protein. It will be
important to follow up with a larger number of patient samples to
confirm the data obtained. Furthermore, it would be interesting to
assess people at different age groups and determine whether unexposed
control kids have a higher “background”.</p>
<p>Applications of the assay described in this study in diagnosis are
limited, since antibody response should start to be detectable only one
to two weeks after infection. Future studies will be required to assess
how long after infection this assay allow to detect anti-CoV2
antibodies. Finally, while likely, the association of seroconversion
with protective immunity against SARS-Cov-2 infection still needs to be
fully established.</p>
<p><strong><em>Relevance</em></strong>:</p>
<p>This study has strong implications in the research against SARS-Cov-2.
First, it is now possible to perform serosurveys and determine who has
been infected, allowing a more accurate estimate of infection prevalence
and death rate. Second, if it is confirmed that re-infection does not
happen (or is rare), this assay can be used as a tool to screen
healthcare workers and prioritize immune ones to work with infected
patients. Third, potential convalescent plasma donors can now be
screened to help treating currently infected patients. Of note, this
assay does not involve live virus handling. experimentally, this is an
advantage as the assay does not require the precautions required by
manipulation of live virus. Finally, the recombinant proteins described
in this study represent new tools that can be used for further
applications, including vaccine development.</p>
<h2 data-number="14.66" id="comparative-pathogenesis-of-covid-19-mers-and-sars-in-a-non-human-primate-model"><span class="header-section-number">14.66</span> COMPARATIVE PATHOGENESIS OF COVID-19, MERS AND SARS IN A NON-HUMAN PRIMATE MODEL</h2>
<p><span class="citation" data-cites="1EcOLnVXk">(<a href="#ref-1EcOLnVXk" role="doc-biblioref">2182</a>)</span></p>
<h3 data-number="14.66.1" id="keywords-66"><span class="header-section-number">14.66.1</span> Keywords</h3>
<ul>
<li>SARS-CoV2</li>
<li>cynomolgus macaque</li>
<li>SARS-CoV</li>
</ul>
<h3 data-number="14.66.2" id="main-findings-62"><span class="header-section-number">14.66.2</span> Main Findings</h3>
<p>This work assesses SARS-CoV-2 infection in young adult and aged
cynomolgus macaques. 4 macaques per age group were infected with
low-passage clinical sample of SARS-CoV-2 by intranasal and
intratracheal administration. Viral presence was assessed in nose,
throat and rectum through RT-PCR and viral culture. SARS-CoV-2
replication was confirmed in the respiratory track (including nasal
samples), and it was also detected in ileum. Viral nucleocapsid
detection by IHC showed infection of type I and II pneumocytes and
epithelia. Virus was found to peak between 2 and 4 days after
administration and reached higher levels in aged vs. young animals. The
early peak is consistent with data in patients and contrasts to SARS-CoV
replication. SARS-CoV-2 reached levels below detection between 8 and 21
days after inoculation and macaques established antibody immunity
against the virus by day 14. There were histopathological alteration in
lung, but no overt clinical signs. At day 4 post inoculation of
SARS-CoV-2, two of four animals presented foci of pulmonary
consolidation, with limited areas of alveolar edema and pneumonia, as
well as immune cell infiltration. In sum, cynomolgus macaques are
permissive to SARS- CoV-2 and develop lung pathology (less severe than
SARC-CoV, but more severe than MERS-CoV).</p>
<h3 data-number="14.66.3" id="limitations-63"><span class="header-section-number">14.66.3</span> Limitations</h3>
<p>Even though cynomolgus macaques were permissive to SARS-CoV-2
replication, it is unclear if the viral load reaches levels comparable
to humans and there wasn’t overt clinical pathology.</p>
<h3 data-number="14.66.4" id="significance-60"><span class="header-section-number">14.66.4</span> Significance</h3>
<p>The development of platforms in which to carry out relevant
experimentation on SARS-CoV-2 pathophysiology is of great urgency.
Cynomolgus macaques offer an environment in which viral replication can
happen, albeit in a limited way and without translating into clinically
relevant symptoms. Other groups are contributing to SARS-CoV2 literature
using this animal model <span class="citation" data-cites="3XqFhi8l">(<a href="#ref-3XqFhi8l" role="doc-biblioref">2168</a>)</span>, potentially showing protection against
reinfection in cured macaques. Therefore, this platform could be used to
examine SARS-CoV2 pathophysiology while studies in other animal models
are also underway <span class="citation" data-cites="NzFgXYvQ MgLBA5m">(<a href="#ref-MgLBA5m" role="doc-biblioref">2107</a>, <a href="#ref-NzFgXYvQ" role="doc-biblioref">2183</a>)</span>.</p>
<h3 data-number="14.66.5" id="credit-65"><span class="header-section-number">14.66.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.67" id="investigating-the-impact-of-asymptomatic-carriers-on-covid-19-transmission"><span class="header-section-number">14.67</span> Investigating the Impact of Asymptomatic Carriers on COVID-19 Transmission</h2>
<p><span class="citation" data-cites="OvFueDLZ">(<a href="#ref-OvFueDLZ" role="doc-biblioref">2184</a>)</span></p>
<h3 data-number="14.67.1" id="keywords-67"><span class="header-section-number">14.67.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>asymptomatic carriers</li>
<li>mathematical model</li>
<li>transmission</li>
</ul>
<h3 data-number="14.67.2" id="main-findings-63"><span class="header-section-number">14.67.2</span> Main Findings</h3>
<p>Multiple studies reported the same level of infectiousness
between symptomatic and asymptomatic carriers of SARS-CoV-2. Given that
asymptomatic and undocumented carriers escape public health surveillance
systems, a better mathematical model of transmission is needed to
determine a more accurate estimate of the basic reproductive number
(R<sub>0</sub>) of the virus to assess the contagiousness of virus. The authors
developed a SEYAR dynamical model for transmission of the new
coronavirus that takes into account asymptomatic and undocumented
carriers. The model was validated using data reported from thirteen
countries during the first three weeks of community transmission. While
current studies estimate R<sub>0</sub> to be around 3, this model indicates that
the value could range between 5.5 to 25.4.</p>
<h3 data-number="14.67.3" id="limitations-64"><span class="header-section-number">14.67.3</span> Limitations</h3>
<p>The SEYAR model realistically depicts
transmission of the virus only during the initial stages of the disease.
More data is necessary to better fit the model with current trends. In
addition, multiple factors (e.g. behavioral patterns, surveillance
capabilities, environmental and socioeconomic factors) affect
transmission of the virus and so, these factors must be taken into
consideration when estimating the R<sub>0</sub>.</p>
<h3 data-number="14.67.4" id="significance-61"><span class="header-section-number">14.67.4</span> Significance</h3>
<p>Public health authorities use the basic reproductive
number to determine the severity of disease. An accurate estimate of
R<sub>0</sub> will inform intervention strategies. This model can be applied to
different locations to assess the potential impact of COVID-19.</p>
<h3 data-number="14.67.5" id="credit-66"><span class="header-section-number">14.67.5</span> Credit</h3>
<p><em>This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.68" id="antibody-responses-to-sars-cov-2-in-covid-19-patients-the-perspective-application-of-serological-tests-in-clinical-practice"><span class="header-section-number">14.68</span> Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice</h2>
<p>Long et al. <em>medRxiv</em> <span class="citation" data-cites="Y9g1sQ1Y">(<a href="#ref-Y9g1sQ1Y" role="doc-biblioref">2185</a>)</span></p>
<h3 data-number="14.68.1" id="keywords-68"><span class="header-section-number">14.68.1</span> Keywords</h3>
<ul>
<li>Serum antibodies</li>
<li>IgM</li>
<li>IgG</li>
<li>immunoassay</li>
<li>diagnosis</li>
<li>seroconversion</li>
</ul>
<h3 data-number="14.68.2" id="main-findings-64"><span class="header-section-number">14.68.2</span> Main Findings</h3>
<p>This study investigated the profile of the acute antibody response
against SARS-CoV-2 and provided proposals for serologic tests in
clinical practice. Magnetic Chemiluminescence Enzyme Immunoassay was
used to evaluate IgM and IgG seroconversion in 285 hospital admitted
patients who tested positive for SARS-CoV-2 by RT-PCR and in 52 COVID-19
suspected patients that tested negative by RT-PCR. A follow up study
with 63 patients was performed to investigate longitudinal effects. In
addition, IgG and IgM titers were evaluated in a cohort of close
contacts (164 persons) of an infected couple.</p>
<p>The median day of seroconversion for both IgG and IgM was 13 days after
symptom onset. Patients varied in the order of IgM/ IgG seroconversion
and there was no apparent correlation of order with age, severity, or
hospitalization time. This led the authors to conclude that for
diagnosis IgM and IgG should be detected simultaneously at the early
phase of infection.</p>
<p>IgG titers, but not IgM titers were higher in severe patients compared
to non-severe patients after controlling for days post-symptom onset.
Importantly, 12% of COVID-19 patients (RT-PCR confirmed) did not meet
the WHO serological diagnosis criterion of either seroconversion or
<em>&gt;</em> 4-fold increase in IgG titer in sequential samples. This suggests
the current serological criteria may be too stringent for COVID-19
diagnosis.</p>
<p>Of note, 4 patients from a group of 52 suspects (negative RT-PCR test)
had anti-SARS-Cov-2 IgM and IgG. Similarly, 4.3% (7/162) of “close
contacts” who had negative RT-PCR tests were positive for IgG and/or
IgM. This highlights the usefulness of a serological assay to identify
asymptomatic infections and/or infections that are missed by RT-PCR.</p>
<h3 data-number="14.68.3" id="limitations-65"><span class="header-section-number">14.68.3</span> Limitations</h3>
<p>This group’s report generally confirms the findings of others that have
evaluated the acute antibody response to SARS-Cov-2. However, these data
would benefit from inclusion of data on whether the participants had a
documented history of viral infection. Moreover, serum samples that were
collected prior to SARS-Cov-2 outbreak from patients with other viral
infections would serve as a useful negative control for their assay.
Methodological limitations include that only one serum sample per case
was tested as well as the heat inactivation of serum samples prior to
testing. It has previously been reported that heat inactivation
interferes with the level of antibodies to SARS-Cov-2 and their protocol
may have resulted in diminished quantification of IgM, specifically <span class="citation" data-cites="p1c9Nilh">(<a href="#ref-p1c9Nilh" role="doc-biblioref">2186</a>)</span>.</p>
<h3 data-number="14.68.4" id="significance-62"><span class="header-section-number">14.68.4</span> Significance</h3>
<p>Understanding the features of the antibody responses against SARS-CoV is
useful in the development of a serological test for the diagnosis of
COVID-19. This paper addresses the need for additional screening methods
that can detect the presence of infection despite lower viral titers.
Detecting the production of antibodies, especially IgM, which are
produced rapidly after infection can be combined with PCR to enhance
detection sensitivity and accuracy and map the full spread of infection
in communities, Moreover, serologic assays would be useful to screen
health care workers in order to identify those with immunity to care for
patients with COVID19.</p>
<h3 data-number="14.68.5" id="credit-67"><span class="header-section-number">14.68.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.69" id="sars-cov-2-specific-antibody-responses-in-covid-19-patients"><span class="header-section-number">14.69</span> SARS-CoV-2 specific antibody responses in COVID-19 patients</h2>
<p><span class="citation" data-cites="1BLNvgEao">(<a href="#ref-1BLNvgEao" role="doc-biblioref">2187</a>)</span></p>
<h3 data-number="14.69.1" id="keywords-69"><span class="header-section-number">14.69.1</span> Keywords</h3>
<ul>
<li>immunoassay</li>
<li>antibody specificity</li>
<li>serology</li>
<li>cross-reactivity</li>
</ul>
<h3 data-number="14.69.2" id="main-findings-65"><span class="header-section-number">14.69.2</span> Main findings</h3>
<p>Antibodies specific to SARS-CoV-2 S protein, the S1 subunit and the RBD (receptor-binding domain) were detected in all SARS-CoV-2 patient sera by 13 to 21 days post onset of disease. Antibodies specific to SARS-CoV N protein (90% similarity to SARS-CoV-2) were able to neutralize SARS-CoV-2 by PRNT (plaque reduction neutralizing test). SARS-CoV-2 serum cross-reacted with SARS-CoV S and S1 proteins, and to a lower extent with MERS-CoV S protein, but not with the MERS-CoV S1 protein, consistent with an analysis of genetic similarity. No reactivity to SARS-CoV-2 antigens was observed in serum from patients with ubiquitous human CoV infections (common cold) or to non-CoV viral respiratory infections.</p>
<h3 data-number="14.69.3" id="limitations-66"><span class="header-section-number">14.69.3</span> Limitations</h3>
<p>Authors describe development of a serological ELISA based assay for the detection of neutralizing antibodies towards regions of the spike and nucleocapsid domains of the SARS-CoV-2 virus. Serum samples were obtained from PCR-confirmed COVID-19 patients. Negative control samples include a cohort of patients with confirmed recent exposure to non-CoV infections (i.e. adenovirus, bocavirus, enterovirus, influenza, RSV, CMV, EBV) as well as a cohort of patients with confirmed infections with ubiquitous human CoV infections known to cause the common cold. The study also included serum from patients with previous MERS-CoV and SARS-CoV zoonotic infections. This impressive patient cohort allowed the authors to determine the sensitivity and specificity of the development of their in-house ELISA assay. Of note, seroconversion was observed as early as 13 days following COVID-19 onset but the authors were not clear how disease onset was determined.</p>
<h3 data-number="14.69.4" id="significance-63"><span class="header-section-number">14.69.4</span> Significance</h3>
<p>Validated serological tests are urgently needed to map the full spread of SARS-CoV-2 in the population and to determine the kinetics of the antibody response to SARS-CoV-2. Furthermore, clinical trials are ongoing using plasma from patients who have recovered from SARS-CoV-2 as a therapeutic option. An assay such as the one described in this study could be used to screen for strong antibody responses in recovered patients. Furthermore, the assay could be used to screen health care workers for antibody responses to SARS-CoV-2 as personal protective equipment continues to dwindle. The challenge going forward will be to standardize and scale-up the various in-house ELISA’s being developed in independent laboratories across the world.</p>
<h2 data-number="14.70" id="a-brief-review-of-antiviral-drugs-evaluated-in-registered-clinical-trials-for-covid-19"><span class="header-section-number">14.70</span> A brief review of antiviral drugs evaluated in registered clinical trials for COVID-19</h2>
<p>Belhadi et al. <span class="citation" data-cites="K41YFM2N">(<a href="#ref-K41YFM2N" role="doc-biblioref">2188</a>)</span></p>
<h3 data-number="14.70.1" id="keywords-70"><span class="header-section-number">14.70.1</span> Keywords</h3>
<ul>
<li>Clinical trials</li>
<li>COVID-19</li>
<li>SARS CoV-2</li>
<li>2019-nCoV</li>
<li>SARS Cov-2</li>
<li>Hcov-19</li>
<li>novel coronal virus</li>
<li>new corona virus</li>
<li>antiviral drugs</li>
</ul>
<h3 data-number="14.70.2" id="main-findings-66"><span class="header-section-number">14.70.2</span> Main Findings</h3>
<p>Summary of clinical trials registered as
of March7, 2020 from U.S, Chinese, Korean, Iranian and European
registries. Out of the 353 studies identified, 115 were selected for
data extraction. 80% of the trials were randomized with parallel
assignment and the median number of planned inclusions was 63 (IRQ,
36-120). Most frequent therapies in the trials included; 1) antiviral
drugs [lopinavir/ritonavir (n-15); umifenovir (n=9); favipiravir (n=7);
redmesivir (n=5)]; 2) anti-malaria drugs [chloroquine (n-11);
hydroxychloroquine (n=7)}; immunosuppressant drugs [methylprednisolone
(n=5)]; and stem cell therapies (n=23). Medians of the total number of
planned inclusions per trial for these therapies were also included.
Stem cells and lopunavir/ritonavir were the most frequently evaluated
candidate therapies (23 and 15 trials respectively), whereas remdesivir
was only tested in 5 trials but these trials had the highest median
number of planned inclusions per trial (400, IQR 394-453). Most of the
agents used in the different trials were chosen based on preclinical
assessments of antiviral activity against SARS CoV and MERS Cov corona
viruses.</p>
<p>The primary outcomes of the studies were clinical (66%); virological
(23%); radiological (8%); or immunological (3%). The trials were
classified as those that included patients with severe disease only;
trials that included patients with moderate disease; and trials that
included patients with severe or moderate disease.</p>
<h3 data-number="14.70.3" id="limitations-67"><span class="header-section-number">14.70.3</span> Limitations</h3>
<p>The trials evaluated provided incomplete information:
23% of these were phase IV trials but the bulk of the trials (54%) did
not describe the phase of the study. Only 52% of the trials (n=60)
reported treatment dose and only 34% (n=39) reported the duration. A lot
of the trials included a small number of patients and the trials are
still ongoing, therefore no insight was provided on the outcome of the
trials.</p>
<h3 data-number="14.70.4" id="significance-64"><span class="header-section-number">14.70.4</span> Significance</h3>
<p>Nonetheless, this review serves as framework for
identifying COVID-19 related trials, which can be expanded upon as new
trials begin at an accelerated rate as the disease spreads around the
world.</p>
<h3 data-number="14.70.5" id="credit-68"><span class="header-section-number">14.70.5</span> Credit</h3>
<p><em>This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.71" id="ace-2-expression-in-the-small-airway-epithelia-of-smokers-and-copd-patients-implications-for-covid-19"><span class="header-section-number">14.71</span> ACE-2 Expression in the Small Airway Epithelia of Smokers and COPD Patients: Implications for COVID-19</h2>
<p>Leung et al. <em>medRxiv</em>. <span class="citation" data-cites="eTsOvBVf">(<a href="#ref-eTsOvBVf" role="doc-biblioref">2189</a>)</span></p>
<h3 data-number="14.71.1" id="keywords-71"><span class="header-section-number">14.71.1</span> Keywords</h3>
<ul>
<li>chronic obstructive pulmonary disease</li>
<li>COPD</li>
<li>smokingE-2</li>
<li>risk factors</li>
</ul>
<h3 data-number="14.71.2" id="main-findings-67"><span class="header-section-number">14.71.2</span> Main Findings</h3>
<p>In bronchial epithelial samples from 3 different cohorts of
individuals, ACE-2 gene expression was found to be significantly
increased in both COPD patients and smokers relative to healthy
controls. Across all test subjects, ACE-2 gene expression was also
highly correlated with decreased forced expiratory volume in 1 second
(FEV1), which may explain the increased COVID-19 disease severity in
COPD patients. Former smokers were also found to show decreased ACE2
expression relative to current smokers and had no significant difference
when compared to non-smokers.</p>
<h3 data-number="14.71.3" id="limitations-68"><span class="header-section-number">14.71.3</span> Limitations</h3>
<p>While the upregulation of ACE-2 is an interesting
hypothesis for COVID-19 disease severity in COPD patients, this study
leaves many more unanswered questions than it addresses. Further studies
are required to show whether the specific cell type isolated in these
studies is relevant to the pathophysiology of COVID-19. Furthermore,
there is no attempt to show whether that increased ACE-2 expression
contributes to greater disease severity. Does the increased ACE-2
expression lead to greater infectivity with SARS-CoV-2? There is no
mechanistic explanation for why ACE-2 levels are increased in COPD
patients. The authors could also have considered the impact of
co-morbidities and interventions such as corticosteroids or
bronchodilators on ACE-2 expression. Finally, given the extensive
sequencing performed, the authors could have conducted significantly
more in-depth analyses into gene signature differences.</p>
<h3 data-number="14.71.4" id="significance-65"><span class="header-section-number">14.71.4</span> Significance</h3>
<p>This study attempts to address an
important clinical finding that both smokers and COPD patients show
increased mortality from COVID-19. The novel finding that ACE-2
expression is induced in smokers and COPD patients suggests not only a
mechanism for the clinical observation, but also highlights the
potential benefit of smoking cessation in reducing the risk of severe
COVID-19 disease.</p>
<h3 data-number="14.71.5" id="credit-69"><span class="header-section-number">14.71.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.72" id="dynamic-profile-of-severe-or-critical-covid-19-cases"><span class="header-section-number">14.72</span> Dynamic profile of severe or critical COVID-19 cases</h2>
<p><span class="citation" data-cites="18KZGwJ0J">(<a href="#ref-18KZGwJ0J" role="doc-biblioref">2190</a>)</span></p>
<h3 data-number="14.72.1" id="keywords-72"><span class="header-section-number">14.72.1</span> Keywords</h3>
<ul>
<li>Coronavirus Disease 2019 (COVID-19)</li>
<li>SARS-CoV-2</li>
<li>progressive lymphopenia (PLD)</li>
<li>T-lymphocytes</li>
<li>clinical data</li>
<li>co-infection</li>
<li>influenza A</li>
</ul>
<h3 data-number="14.72.2" id="main-findings-68"><span class="header-section-number">14.72.2</span> Main Findings</h3>
<p>Authors evaluate clinical correlates of 10 patients (6 male
and 4 female) hospitalized for severe Acute Respiratory Syndrome
Coronavirus 2 (SARS-CoV-2). All patients required oxygen support and
received broad spectrum antibiotics and 6 patients received anti-viral
drugs. Additionally, 40% of patients were co-infected with influenza A.
All 10 patients developed lymphopenia, two of which developed
progressive lymphopenia (PLD) and died. Peripheral blood (PB)
lymphocytes were analyzed – low CD4 and CD8 counts were noted in most
patients, though CD4:CD8 ratio remained normal.</p>
<h3 data-number="14.72.3" id="limitations-69"><span class="header-section-number">14.72.3</span> Limitations</h3>
<p>The authors evaluated a small cohort of severe
SARS-CoV-2 cases and found an association between T cell lymphopenia and
adverse outcomes. However, this is an extremely small and diverse cohort
(40% of patients were co-infected with influenza A). These findings need
to be validated in a larger cohort. Additionally, the value of this data
would be greatly increased by adding individual data points for each
patient as well as by adding error bars to each of the figures.</p>
<h3 data-number="14.72.4" id="significance-66"><span class="header-section-number">14.72.4</span> Significance</h3>
<p>This study provides a collection of clinical data and
tracks evolution of T lymphocyte in 10 patients hospitalized for
SARS-CoV-2, of which 4 patients were co-infected with influenza A.</p>
<h3 data-number="14.72.5" id="credit-70"><span class="header-section-number">14.72.5</span> Credit</h3>
<p><em>This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.73" id="association-between-clinical-laboratory-and-ct-characteristics-and-rt-pcr-results-in-the-follow-up-of-covid-19-patients"><span class="header-section-number">14.73</span> Association between Clinical, Laboratory and CT Characteristics and RT-PCR Results in the Follow-up of COVID-19 patients</h2>
<p>Fu et al. <em>medRxiv</em>. <span class="citation" data-cites="1BJXDZ04c">(<a href="#ref-1BJXDZ04c" role="doc-biblioref">2191</a>)</span></p>
<h3 data-number="14.73.1" id="keywords-73"><span class="header-section-number">14.73.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>clinical</li>
<li>lymphocyte</li>
<li>CRP</li>
<li>LDH</li>
<li>HSST TNT</li>
<li>PCR test</li>
<li>readmission</li>
<li>CT</li>
<li>GGO</li>
<li>disease progression</li>
</ul>
<h3 data-number="14.73.2" id="study-description"><span class="header-section-number">14.73.2</span> Study description</h3>
<p>Data analyzed from 52 COVID-19 patients admitted
and then discharged with COVID-19. Clinical, laboratory, and
radiological data were longitudinally recorded with illness timecourse
(PCR + to PCR-) and 7 patients (13.5%) were readmitted with a follow up
positive test (PCR+) within two weeks of discharge.</p>
<h3 data-number="14.73.3" id="main-findings-69"><span class="header-section-number">14.73.3</span> Main Findings</h3>
<ul>
<li><p>At admission:</p>
<ul>
<li><p>The majority of patients had increased CRP at admission (63.5%).</p></li>
<li><p>LDH, and HSST TNT were significantly increased at admission.</p></li>
<li><p>Radiographic signs via chest CT showed increased involvement in
lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe
(37 cases, 71.2%).</p></li>
<li><p>GGO (90.4%), interlobular septal thickening (42.3%), vascular
enlargement (42.3%), and reticulation (11.5%) were most commonly
observed.</p></li>
</ul></li>
<li><p>After negative PCR test (discharge):</p>
<ul>
<li><p>CRP levels decreased lymphocyte counts (#/L) increased
significantly (CD3+, CD3+/8+ and CD3+/4+) after negative PCR.</p></li>
<li><p>Consolidation and mixed GGO observed in longitudinal CT imaging
w different extents of inflammatory exudation in lungs, with
overall tendency for improvement (except 2/7 patients that were
readmitted after discharge with re-positive test) after negative
PCR.</p></li>
</ul></li>
<li><p>Seven patients repeated positive RT-PCR test and were readmitted to
the hospital (9 to 17 day after initial discharge).</p>
<ul>
<li><p>Follow up CT necessary to monitor improvement during recovery
and patients with lesion progression should be given more
attention.</p></li>
<li><p>Dynamic CT in addition to negative test essential in clinical
diagnosis due to nasal swab PCR sampling bias (false-negatives).</p></li>
<li><p>Increase in CRP occurred in 2 readmitted patients (and decr. in
lymphocytes in one patient), but was not correlated with new
lesions or disease progression vs. improvement (very low N).</p></li>
<li><p>Patients readmitted attributed to false-negative PCR vs.
re-exposure.</p></li>
</ul></li>
</ul>
<h3 data-number="14.73.4" id="limitations-70"><span class="header-section-number">14.73.4</span> Limitations</h3>
<p>Patients sampled in this study were generally
younger (65.4% &lt; 50 yrs) and less critically ill/all discharged.
Small number of recovered patients (N=18). Time of follow up was
relatively short.. Limited clinical information available about patients
with re-positive test (except CRP and lymph tracking).</p>
<h3 data-number="14.73.5" id="extended-results"><span class="header-section-number">14.73.5</span> Extended Results</h3>
<p><em>NOTE</em>: Patients sampled in this study were generally younger (65.4%
&lt; 50 yrs) and less critically ill/all discharged. After two
consecutive negative PCR tests, patients were discharged.</p>
<p>Clinical Results at Admission</p>
<ul>
<li><p>Median interval disease onset to admission (5 days, IQR: 3-7)</p></li>
<li><p>Most common symptoms included fever, fatigue, dry cough, and
expectoration.</p></li>
<li><p>Fifteen patients had reduced lymphocyte counts (28.8%).</p></li>
<li><p>No change in WBC or Neutrophil counts.</p></li>
<li><p><strong>The majority of patients had increased CRP at admission (63.5%).</strong></p></li>
<li><p><strong>LDH, and HSST TNT were significantly increased at admission.</strong></p></li>
<li><p>Fibrinogen was trending high though not significant.</p></li>
<li><p>No major changes in liver function observed.</p></li>
<li><p><strong>Radiographic signs via chest CT showed increased involvement in
lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe (37
cases, 71.2%).</strong></p></li>
<li><p><strong>GGO (90.4%), interlobular septal thickening (42.3%), vascular
enlargement (42.3%)</strong>, and reticulation (11.5%) were most commonly
observed.</p></li>
</ul>
<p>Change in Clinical Results following Negative Test</p>
<ul>
<li><p><strong>CRP levels decreased after negative PCR.</strong></p></li>
<li><p><strong>Lymphocyte counts (#/L) increased significantly (CD3+, CD3+/8+
and CD3+/4+).</strong></p></li>
<li><p>No significant change to CD4/8 ratio.</p></li>
<li><p>LDH, HSST TNT, and Fibronegin remained high throughout, though range
observed decreased over time.</p></li>
<li><p><strong>Consolidation and mixed GGO observed in longitudinal CT imaging.</strong></p></li>
<li><p><strong>Patients showed different extents of inflammatory exudation in
lungs, with overall tendency for improvement (except 2/7 patients
that were readmitted after discharge with re-positive test).</strong></p></li>
</ul>
<p>Patients Readmitted with PCR+ test</p>
<ul>
<li><p><strong>Seven patients repeated positive RT-PCR test and were readmitted
to the hospital (9 to 17 day after initial discharge).</strong></p></li>
<li><p>Improvement during readmission in 4 patients and observation of
segmental progression CT in 2 patients (2/18 or 11.1% - re-positive
9 and 10 days post-discharge).</p></li>
<li><p>Two patients showed new GGO, while others improved greatly.</p></li>
<li><p><strong>Follow up CT necessary to monitor improvement during recovery and
patients with lesion progression should be given more attention.</strong></p></li>
<li><p><strong>Dynamic CT in addition to negative test essential in clinical
diagnosis due to nasal swab PCR sampling bias (false-negatives).</strong></p></li>
<li><p><strong>Increase in CRP occurred in 2 readmitted patients (and decr. in
lymphocytes in one patient), but was not correlated with new lesions
or disease progression vs. improvement (very low N).</strong></p></li>
</ul>
<h3 data-number="14.73.6" id="significance-67"><span class="header-section-number">14.73.6</span> Significance</h3>
<p>Study tracked key clinical features associated with
disease progression, recovery, and determinants of clinical
diagnosis/management of COVID-19 patients.</p>
<h3 data-number="14.73.7" id="credit-71"><span class="header-section-number">14.73.7</span> Credit</h3>
<p><em>This review was undertaken by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.74" id="an-orally-bioavailable-broad-spectrum-antiviral-inhibits-sars-cov-2-and-multiple-2-endemic-epidemic-and-bat-coronavirus"><span class="header-section-number">14.74</span> An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 and multiple 2 endemic, epidemic and bat coronavirus</h2>
<p>Sheahan et al. <em>bioRxiv</em>. <span class="citation" data-cites="iMCz5Twj">(<a href="#ref-iMCz5Twj" role="doc-biblioref">2192</a>)</span></p>
<h3 data-number="14.74.1" id="keywords-74"><span class="header-section-number">14.74.1</span> Keywords</h3>
<ul>
<li>Treatment</li>
<li>Antiviral</li>
<li>Broad spectrum antiviral</li>
<li>ribonucleoside analog β-D-N4 30 hydroxycytidine (NHC)</li>
<li>Remdesivir</li>
</ul>
<h3 data-number="14.74.2" id="main-findings-70"><span class="header-section-number">14.74.2</span> Main Findings</h3>
<p>β-D-N4 30 –hydroxycytidine (NHC, EIDD-1931) is an
orally bioavailable ribonucleoside with antiviral activity against
various RNA viruses including Ebola, Influenza and CoV. NHC activity
introduce<em>d</em><del>s</del> mutations in the viral (but not cellular) RNA in a
dose dependent <em>manner</em> that directly correlated with a decrease in
viral titers. Authors show that NHC inhibited multiple genetically
distinct Bat-CoV viruses in human primary epithelial cells <em>without
affecting</em> <em>cell viability even at high concentrations (100</em> <em>µM)</em>.
Prophylactic oral administration of NHC in C57BL/6 mice reduce lung
titers of SARS-CoV and prevented weight loss and hemorrhage. Therapeutic
administration of NHC in C57BL/6 mice 12 hours post infected with
SARS-CoV reduced acute lung injury, viral titer, and lung hemorrhage.
The degree of clinical benefit was dependent on the time of treatment
initiation post infection. The authors also demonstrate that NHC reduces
MERS-CoV infection titers, pathogenesis, and viral RNA in prophylactic
and therapeutic settings.</p>
<h3 data-number="14.74.3" id="limitations-71"><span class="header-section-number">14.74.3</span> Limitations</h3>
<p>Most of the experiments were conducted using MERS-CoV, and
SARS-CoV and a few experiments were conducted using other strains of CoV
as opposed to SARS-CoV-2. The authors note the core residues that make
up the RNA interaction sites (which constitutes the NHC interaction
sites) are highly conserved among CoV and because of this conservation
their understanding is that NHC can inhibit a broad-spectrum of CoV
including SARS-CoV-2.</p>
<p>The increased viral mutation rates associated with NHC activity may have
adverse effects if mutations cause the virus to become drug resistant,
more infectious or speed-up immune evasion. <em>In addition, the temporal
diminishing effectiveness of NHC on clinical outcome when NHC was used
therapeutically is concerning. However, the longer window (7-10 days)
for clinical disease onset in human patients from the time of infection
compared to that of mice (24-48 hours), may associate with increased NHC
effectiveness in the clinic.</em></p>
<h3 data-number="14.74.4" id="significance-68"><span class="header-section-number">14.74.4</span> Significance</h3>
<p>Prophylactic or therapeutic oral administration of NHC
reduces lung titers and prevents acute lung failure in C57B\6 mice
infected with CoV. Given its <em>broad-spectrum antiviral activity, NHC
could turn out to be a useful drug for treating current, emerging and
future corona virus outbreaks.</em>
 
#### Credit</p>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.75" id="identification-of-antiviral-drug-candidates-against-sars-cov-2-from-fda-approved-drugs"><span class="header-section-number">14.75</span> Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs</h2>
<p>Sangeun Jeon et al. <span class="citation" data-cites="ZDP9vSpg">(<a href="#ref-ZDP9vSpg" role="doc-biblioref">2193</a>)</span></p>
<h3 data-number="14.75.1" id="keywords-75"><span class="header-section-number">14.75.1</span> Keywords</h3>
<ul>
<li>COVID-19</li>
<li>SARS CoV-2</li>
<li>antiviral drugs</li>
<li>niclosamide</li>
<li>ciclesonide</li>
</ul>
<h3 data-number="14.75.2" id="main-findings-71"><span class="header-section-number">14.75.2</span> Main Findings</h3>
<p>A panel of ~3,000 FDA- and IND-approved
antiviral drugs were previously screened for inhibitory efficacy against
SARS CoV, a coronavirus related to the novel coronavirus SARS CoV-2
(79.5%) homology. 35 of these drugs along with another 15 (suggested by
infectious disease specialists) were tested in vitro for their ability
to inhibit SARS CoV-2 infectivity of Vero cells while preserving cell
viability. The infected cells were scored by immunofluorescence analysis
using an antibody against the N protein of SARS CoV-2. Chloroquine,
lopinavir and remdesivir were used as reference drugs.</p>
<p>Twenty four out of 50 drugs exhibited antiviral activity with IC50
values ranging from 0.1-10µM. Among these, two stood ou: 1) the-anti
helminthic drug niclosamide which exhibited potent antiviral activity
against SARS CoV-2 (IC50=0.28 µM). The broad-spectrum antiviral effect
of niclosamide against SARS and MERS-CoV have been previously documented
and recent evidence suggests that in may inhibit autophagy and reduce
MERS C0V replication. 2) Ciclesonide, a corticosteroid used to treat
asthma and allergic rhinitis, also exhibited antiviral efficacy but with
a lower IC50 (4.33µM) compared to niclosamide. The antiviral effects of
ciclesonide were directed against NSP15, a viral riboendonuclease which
is the molecular target of this drug.</p>
<h3 data-number="14.75.3" id="limitations-72"><span class="header-section-number">14.75.3</span> Limitations</h3>
<p>The drugs were tested against SARS CoV-2 infectivity in
vitro only, therefore preclinical studies in animals and clinical trials
in patients will be need for validation of these drugs as therapeutic
agents for COVID-19. In addition, niclosamide exhibits low adsorption
pharmatokinetically which could be alleviated with further development
of drug formulation to increase effective delivery of this drug to
target tissues. Nonetheless, niclosamide and ciclesonide represent
promising therapeutic agents against SARS CoV-2 given that other
compounds tested in the same study including favipiravir (currently used
in clinical trials) and atazanavir (predicted as the most potent
antiviral drug by AI-inference modeling) did not exhibit antiviral
activity in the current study.</p>
<h3 data-number="14.75.4" id="credit-72"><span class="header-section-number">14.75.4</span> Credit</h3>
<p><em>This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.76" id="respiratory-disease-and-virus-shedding-in-rhesus-macaques-inoculated-with-sars-cov-2"><span class="header-section-number">14.76</span> Respiratory disease and virus shedding in rhesus macaques inoculated with SARS-CoV-2</h2>
<p>Munster et al. <em>bioRxiv</em>. <span class="citation" data-cites="LYFzvlzE">(<a href="#ref-LYFzvlzE" role="doc-biblioref">2194</a>)</span></p>
<h3 data-number="14.76.1" id="keywords-76"><span class="header-section-number">14.76.1</span> Keywords</h3>
<p>animal model, pulmonary infiltrates, dynamic
of antibody response, cytokine</p>
<h3 data-number="14.76.2" id="main-findings-72"><span class="header-section-number">14.76.2</span> Main Findings</h3>
<p>Inoculation of 8 Resus macaques with SARS-Cov-2 , which
all showed clinical signs of infection (respiratory pattern, reduced
appetite, weight loss, elevated body temperature) resulting in moderate,
transient disease. Four animals were euthanized at 3dpi, the 4 others at
21 dpi. Study of viral loads in different organs showed that nose swab
and throat swabs were the most sensitive, with bronchio-alveolar lavage.
Interstitial pneumonia was visible in radiographies and at the
histological scale too. Clinically, the macaques had similar symptoms as
described in human patients with moderate disease.</p>
<p>Viral shedding was consistently detected in nose swabs and throat swabs
immediately after infection but less consistent thereafter which could
reflect virus administration route (intranasal, oral). Bronchoalveolar
lavages performed as a measure of virus replication in the lower
respiratory tract on animals maintained for 21 days, contained high
viral loads in 1 and 3dpi. The majority of the animals exhibited
pulmonary edema and mild to moderate interstitial pneumonia on terminal
bronchioles. In addition to the lung, viral RNA could also be detected
throughout the respiratory track where viral replication mainly
occurred.</p>
<p>Immunologic responses included leukocytosis, neutrophilia, monocytosis
and lymphopenia in the majority of the animals at 1dpi. Lymphocytes and
monocytes re-normalized at 2dpi. Neutrophils declined after 3dpi and
through 10dpi after which they started to recover. After infection,
serum analysis revealed significant increases in <strong>IL1ra, IL6, IL10,
IL15, MCP-1, MIP-1b, but quick normalization</strong> (3dpi). <strong>Antibody
response started around 7dpi, and the antibody titers stayed elevated
until 21dpi</strong> (day of animal euthanasia).</p>
<h3 data-number="14.76.3" id="limitations-73"><span class="header-section-number">14.76.3</span> Limitations</h3>
<p>The macaques were inoculated via a combination
of intratracheal, intranasal, ocular and oral routes, which might not
reproduce how humans get infected. Maybe this can lead to different
dynamics in the host immune response. Also, the authors noted that the
seroconversion was not directly followed by a decline in viral loads, as
observed in covid19 patients.</p>
<h3 data-number="14.76.4" id="significance-69"><span class="header-section-number">14.76.4</span> Significance</h3>
<p>This work confirms that rhesus
macaques can be a good model to study Covid-19, as it has been shown by
other groups <span class="citation" data-cites="3XqFhi8l unVgVIfU 1EcOLnVXk">(<a href="#ref-3XqFhi8l" role="doc-biblioref">2168</a>, <a href="#ref-1EcOLnVXk" role="doc-biblioref">2182</a>, <a href="#ref-unVgVIfU" role="doc-biblioref">2195</a>)</span>. While these experiments recapitulate
moderate COVID-19 in humans, the mode of inoculation via a combination
of intratracheal, intranasal, ocular and oral routes, might not
reproduce how humans get infected and may lead to different dynamics in
the host immune response. For example, the authors noted that the
seroconversion was not directly followed by a decline in viral loads, as
observed in COVID-19 patients. Therefore, it will be interesting to
follow their antibody titers longer and further assess the
possibility/effect of reinfection in these macaques. It is essential to
be able to understand the dynamic of the disease and associated immune
responses, and to work on vaccine development and antiviral drug
testing.</p>
<h3 data-number="14.76.5" id="credit-73"><span class="header-section-number">14.76.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.77" id="ace2-expression-is-increased-in-the-lungs-of-patients-with-comorbidities-associated-with-severe-covid-19"><span class="header-section-number">14.77</span> ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19</h2>
<p><span class="citation" data-cites="wjtVy2v0">(<a href="#ref-wjtVy2v0" role="doc-biblioref">2196</a>)</span></p>
<h3 data-number="14.77.1" id="keywords-77"><span class="header-section-number">14.77.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>lung</li>
<li>comorbidities</li>
<li>histone</li>
<li>epigenetics</li>
</ul>
<h3 data-number="14.77.2" id="main-findings-73"><span class="header-section-number">14.77.2</span> Main Findings</h3>
<ul>
<li><p>Transcriptomic analysis using systems-level meta-analysis and
network analysis of existing literature to determine ACE2 regulation
in patients who have frequent COVID-19 comorbidities [eg-
cardiovascular diseases, familial pulmonary hypertension, cancer].</p></li>
<li><p>Enrichment analyses indicated pathways associated with inflammation,
metabolism, macrophage autophagy, and ER stress.</p></li>
<li><p>ACE2 higher in adenocarcinoma compared to adjacent normal lung; ACE2
higher in COPD patients compared to normal.</p></li>
<li><p>Co-expression analysis identified genes important to viral entry
such as RAB1A, ADAM10, HMGBs, and TLR3 to be associated with ACE2 in
diseased lungs.</p></li>
<li><p>ACE2 expression could be potentially regulated by enzymes that
modify histones, including HAT1, HDAC2, and KDM5B.</p></li>
</ul>
<h3 data-number="14.77.3" id="limitations-74"><span class="header-section-number">14.77.3</span> Limitations:</h3>
<ul>
<li><p>Not actual CoVID-19 patients with co-morbidities, so interpretations
in this study need to be confirmed by analyzing upcoming
transcriptomics from CoVID-19 patients having co-morbidity metadata.</p></li>
<li><p>As mentioned by authors, study does not look at diabetes and
autoimmunity as risk factors in CoVID-19 patients due to lack of
data; would be useful to extend such analyses to those datasets when
available.</p></li>
<li><p>Co-expression analysis is perfunctory and needs
validation-experiments especially in CoVID-19 lung samples to mean
anything.</p></li>
<li><p>Epigenomic analyses are intriguing but incomplete, as existence of
histone marks does not necessarily mean occupancy. Would be
pertinent to check cell-line data (CCLE) or actual CoVID-19 patient
samples to confirm ACE2 epigenetic control.</p></li>
</ul>
<h3 data-number="14.77.4" id="significance-70"><span class="header-section-number">14.77.4</span> Significance</h3>
<ul>
<li><p>Study implies vulnerable populations have ACE2 upregulation that
could promote CoVID-19 severity. Shows important data-mining
strategy to find gene-networks associated with ACE2 upregulation in
co-morbid patients.</p></li>
<li><p>Several of the genes co-upregulated with ACE2 in diseased lung might
play an important role in CoVID-19 and can be preliminary targets
for therapeutics</p></li>
<li><p>If in silico findings hold true, epigenetic control of ACE2
expression could be a new target for CoVID-19 therapy with
strategies such as KDM5 demethylases.</p></li>
</ul>
<h3 data-number="14.77.5" id="credit-74"><span class="header-section-number">14.77.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.78" id="meplazumab-treats-covid-19-pneumonia-an-open-labelled-concurrent-controlled-add-on-clinical-trial"><span class="header-section-number">14.78</span> Meplazumab treats COVID-19 pneumonia: an open-labelled, concurrent controlled add-on clinical trial</h2>
<p>Bian et al. <em>medRxiv</em>. <span class="citation" data-cites="DFhY5nfb">(<a href="#ref-DFhY5nfb" role="doc-biblioref">2197</a>)</span></p>
<h3 data-number="14.78.1" id="keywords-78"><span class="header-section-number">14.78.1</span> Keywords</h3>
<ul>
<li>Meplazumab</li>
<li>CD147</li>
<li>humanized antibody</li>
<li>clinical trial</li>
</ul>
<h3 data-number="14.78.2" id="main-findings-74"><span class="header-section-number">14.78.2</span> Main Findings</h3>
<p>This work is based on previous work by the same group
that demonstrated that SARS-CoV-2can also enter host cells via CD147
(also called Basigin, part of the immunoglobulin superfamily, is
expressed by many cell types) consistent with their previous work with
SARS-CoV-1. <sup>1</sup> A prospective clinical trial was conducted with 17
patients receiving Meplazumab, a humanized anti-CD147 antibody, in
addition to all other treatments. 11 patients were included as a control
group (non-randomized).</p>
<p>They observed a faster overall improvement rate in the Meplazumab group
(e.g. at day 14 47% vs 17% improvement rate) compared to the control
patients. Also, virological clearance was more rapid with median of 3
days in the Meplazumab group vs 13 days in control group. In laboratory
values, a faster normalization of lymphocyte counts in the Meplazumab
group was observed, but no clear difference was observed for CRP levels.</p>
<h3 data-number="14.78.3" id="limitations-75"><span class="header-section-number">14.78.3</span> Limitations</h3>
<p>While the results from the study are encouraging, this
study was non-randomized, open-label and on a small number of patients,
all from the same hospital. It offers evidence to perform a larger scale
study. Selection bias as well as differences between treatment groups
(e.g. age 51yo vs 64yo) may have contributed to results. The authors
mention that there was no toxic effect to Meplazumab injection but more
patient and longer-term studies are necessary to assess this.</p>
<h3 data-number="14.78.4" id="significance-71"><span class="header-section-number">14.78.4</span> Significance</h3>
<p>These results seem promising as for now there are
limited treatments for Covid-19 patients, but a larger cohort of patient
is needed. CD147 has already been described to facilitate HIV <span class="citation" data-cites="yP8TXdq2">(<a href="#ref-yP8TXdq2" role="doc-biblioref">2198</a>)</span>,
measles virus <span class="citation" data-cites="X8l1n4zY">(<a href="#ref-X8l1n4zY" role="doc-biblioref">2199</a>)</span>, and malaria <span class="citation" data-cites="YcyNaIVC">(<a href="#ref-YcyNaIVC" role="doc-biblioref">2200</a>)</span> entry into host cells. This group was
the first to describe the CD147-spike route of SARS-Cov-2 entry in host
cells <span class="citation" data-cites="14y8UvAbH">(<a href="#ref-14y8UvAbH" role="doc-biblioref">2171</a>)</span> p147. Indeed, they had previously shown in 2005 that SARS-Cov
could enter host cells via this transmembrane protein <span class="citation" data-cites="m9yLJ5Da">(<a href="#ref-m9yLJ5Da" role="doc-biblioref">2201</a>)</span>. Further
biological understanding of how SARS-CoV-2 can enter host cells and how
this integrates with ACE2R route of entry is needed. Also, the specific
cellular targets of the anti-CD147 antibody need to be assessed, as this
protein can be expressed by many cell types and has been shown to
involved in leukocytes aggregation <span class="citation" data-cites="J0ibHE3l">(<a href="#ref-J0ibHE3l" role="doc-biblioref">2202</a>)</span>. Lastly, Meplazumab is not a
commercially-available drug and requires significant health resources to
generate and administer which might prevent rapid development and use.</p>
<h3 data-number="14.78.5" id="credit-75"><span class="header-section-number">14.78.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.79" id="potent-human-neutralizing-antibodies-elicited-1-by-sars-cov-2-infection"><span class="header-section-number">14.79</span> Potent human neutralizing antibodies elicited 1 by SARS-CoV-2 infection</h2>
<p>Ju et al. <em>bioRxiv</em>. <span class="citation" data-cites="GhJYjnft">(<a href="#ref-GhJYjnft" role="doc-biblioref">186</a>)</span></p>
<h3 data-number="14.79.1" id="keywords-79"><span class="header-section-number">14.79.1</span> Keywords</h3>
<ul>
<li>monoclonal antibodies</li>
<li>neutralization</li>
<li>antibody cross-reactivity</li>
<li>Receptor Binding Domain</li>
</ul>
<h3 data-number="14.79.2" id="main-findings-75"><span class="header-section-number">14.79.2</span> Main Findings</h3>
<p>In this study the authors report the affinity, cross reactivity (with
SARS-CoV and MERS-CoV virus) and viral neutralization capacity of 206
monoclonal antibodies engineered from isolated IgG memory B cells of
patients suffering from SARS-CoV-2 infection in Wuhan, China. All
patients but one recovered from disease. Interestingly, the patient that
did not recover had less SARS-CoV-2 specific B cells circulating
compared to other patients.</p>
<p>Plasma from all patients reacted to trimeric Spike proteins from
SARS-CoV-2, SARS-CoV and MERS-CoV but no HIV BG505 trimer. Furthermore,
plasma from patients recognized the receptor binding domain (RBD) from
SARS-CoV-2 but had little to no cross-reactivity against the RBD of
related viruses SARS-CoV and MERS-CoV, suggesting significant
differences between the RBDs of the different viruses. Negligible levels
of cross-neutralization using pseudoviruses bearing Spike proteins of
SARS-CoV-2, SARS-CoV or MERS-CoV, were observed, corroborating the ELISA
cross-reactivity assays on the RBDs.</p>
<p>SARS-CoV-2 RBD specific B cells constituted 0.005-0.065% of the total B
cell population and 0.023-0.329% of the memory subpopulation. SARS-CoV
specific IgG memory B cells were single cell sorted to sequence the
antibody genes that were subsequently expressed as recombinant IgG1
antibodies. From this library, 206 antibodies with different binding
capacities were obtained. No discernible patterns of VH usage were found
in the 206 antibodies suggesting immunologically distinct responses to
the infection. Nevertheless, most high-binding antibodies were derived
by clonal expansion. Further analyses in one of the patient derived
clones, showed that the antibodies from three different timepoints did
not group together in phylogenetic analysis, suggesting selection during
early infection.</p>
<p>Using surface plasmon resonance (SPR) 13 antibodies were found to have
10<sup>-8</sup> tp 10<sup>-9</sup> dissociation constants (Kd). Of the 13 antibodies, two
showed 98-99% blocking of SARS-CoV-2 RBD-ACE2 receptor binding in
competition assays. Thus, low Kd values alone did not predict ACE2
competing capacities. Consistent with competition assays the two
antibodies that show high ACE2 blocking (P2C-2F6 and P2C-1F11) were the
most capable of neutralizing pseudoviruses bearing SARS-CoV-2 spike
protein (IC<sub>50</sub> of 0.06 and 0.03 µg/mL, respectively). Finally, using
SPR the neutralizing antibodies were found to recognize both overlapping
and distinct epitopes of the RBD of SARS-CoV-2.</p>
<h3 data-number="14.79.3" id="limitations-76"><span class="header-section-number">14.79.3</span> Limitations</h3>
<ol type="1">
<li><p>Relatively low number of patients</p>
<ol type="a">
<li>No significant conclusion can be drawn about the possible
&gt; correlation between humoral response and disease severity</li>
</ol></li>
<li><p><em>In vitro</em> Cytopathic Effect Assay (CPE) for neutralization activity</p>
<ol type="a">
<li><p>Huh7 cells were used in neutralization assays with
&gt; pseudoviruses, and they may not entirely mimic what happens in
&gt; the upper respiratory tract</p></li>
<li><p>CPE assay is not quantitative</p></li>
</ol></li>
<li><p>Duplicated panel in Figure 4C reported (has been fixed in version 2)</p></li>
</ol>
<h3 data-number="14.79.4" id="significance-72"><span class="header-section-number">14.79.4</span> Significance</h3>
<p>This paper offers an explanation as to why
previously isolated antibodies against SARS-CoV do not effectively block
SARS-CoV-2. Also, it offers important insight into the development of
humoral responses at various time points during the first weeks of the
disease in small but clinically diverse group of patients. Furthermore,
it provides valuable information and well characterized antibody
candidates for the development of a recombinant antibody treatment for
SARS-CoV-2. Nevertheless, it also shows that plasmapheresis might have
variability in its effectiveness, depending on the donor’s antibody
repertoire at the time of donation.</p>
<h3 data-number="14.79.5" id="credit-76"><span class="header-section-number">14.79.5</span> Credit</h3>
<p><em>Review by Jovani Catalan-Dibene as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.80" id="characterisation-of-the-transcriptome-and-proteome-of-sars-cov-2-using-direct-rna-sequencing-and-tandem-mass-spectrometry-reveals-evidence-for-a-cell-passage-induced-in-frame-deletion-in-the-spike-glycoprotein-that-removes-the-furin-like-cleavage-site."><span class="header-section-number">14.80</span> Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site.</h2>
<p>Davidson et al. <span class="citation" data-cites="bCYYaFKM">(<a href="#ref-bCYYaFKM" role="doc-biblioref">2203</a>)</span></p>
<h3 data-number="14.80.1" id="keywords-80"><span class="header-section-number">14.80.1</span> Keywords</h3>
<ul>
<li>Transcription</li>
<li>RNA-seq</li>
<li>proteomics</li>
<li>mass spec</li>
<li>furin cleavage site</li>
<li>mutation</li>
<li>pathogenicity</li>
</ul>
<h3 data-number="14.80.2" id="main-findings-76"><span class="header-section-number">14.80.2</span> Main Findings</h3>
<p>The authors performed long read RNA sequencing using an Oxford Nanopore
MinION as well as tandem mass spec (MS) on Vero cells (a cell line
derived from kidney cells of the African green monkey that is deficient
in interferon) infected with SARS-CoV-2.</p>
<p>The authors found that passage of the virus in Vero cells gave rise to a
spontaneous 9 amino acid deletion (679-NSPRRARSV-687 to I) in the spike
(S) protein. The deleted sequence overlaps a predicted furin cleavage
site at the S1 / S2 domain boundary that is present in SARS-CoV-2 but
not SARS-CoV or the closely related bat coronavirus RaTG13, which are
cleaved at S1 / S2 by other proteases <span class="citation" data-cites="CqQVkaqj">(<a href="#ref-CqQVkaqj" role="doc-biblioref">16</a>)</span>. Furin cleavage sites at
similar positions in other viruses have been linked to increased
pathogenicity and greater cell tropism <span class="citation" data-cites="16tHVjVzu">(<a href="#ref-16tHVjVzu" role="doc-biblioref">2204</a>)</span>. Loss of this site in
SARS-CoV-2 has also already been shown to increase viral entry into Vero
but not BHK cells (which are also interferon deficient) <span class="citation" data-cites="qcVbT0w4">(<a href="#ref-qcVbT0w4" role="doc-biblioref">25</a>)</span>. The
authors therefore make an important contribution in demonstrating that
passage in Vero cells may lead to spontaneous loss of a key
pathogenicity-conferring element in SARS-CoV-2.</p>
<h3 data-number="14.80.3" id="limitations-77"><span class="header-section-number">14.80.3</span> Limitations</h3>
<p>As the authors note, a similar study posted earlier by Kim et al., which
also passaged SARS-CoV-2 in Vero cells, did not identify any loss in the
S protein furin cleavage site <span class="citation" data-cites="hRVfFbS">(<a href="#ref-hRVfFbS" role="doc-biblioref">2205</a>)</span>. It therefore remains to be
determined how likely it is that this mutation spontaneously arises. A
quantitative investigation using multiple experimental replicas to
understand the spontaneous viral mutation rate at this site and
elsewhere would be informative. Also, the mechanistic basis for the
higher viral fitness conferred by loss of the furin cleavage site in
Vero cells – but, evidently, not in vivo in humans, as this site is
maintained in all currently sequenced circulating isolates - remains to
be understood.</p>
<p>Due to the high base-call error rate of MinION sequencing, the authors’
bioinformatic pipeline required aligning transcripts to a reference to
correct sequencing artifacts. This presumably made it difficult or
impossible to identify other kinds of mutations, such as single
nucleotide substitutions, which may occur even more frequently than the
deletions identified in this work. Pairing long read sequencing with
higher-accuracy short-read sequencing may be one approach to overcome
this issue.</p>
<h3 data-number="14.80.4" id="significance-73"><span class="header-section-number">14.80.4</span> Significance</h3>
<p>As the authors suggest, animal studies using live virus challenge may
need to periodically verify the genomic integrity of the virus, or
potentially risk unknowingly using a likely less-pathogenic variant of
the virus.</p>
<p>More broadly, the results emphasize the complexity and plasticity of the
SARS-CoV-2 viral transcriptome and proteome. For example, the authors
found multiple versions of transcripts encoding the nucleocapsid (N)
protein, each with different small internal deletions, some of which
were verified for translation by MS. A number of peptides arising from
translation of unexpected rearrangements of transcripts were also
detected. Additionally, the authors identified phosphorylation of a
number of viral proteins (N, M, ORF 3a, nsp3, nsp9, nsp12 and S). For
any cases where these have functional consequences, targeting the
kinases responsible could be an avenue for drug development.
Understanding the functional consequences of the mutations, transcript
variations, and post translational modifications identified in this
study will be important future work.</p>
<h3 data-number="14.80.5" id="credit-77"><span class="header-section-number">14.80.5</span> Credit</h3>
<p><em>This review was undertaken by Tim O’Donnell, Maria Kuksin as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.81" id="a-sars-cov-2-human-protein-protein-interaction-map-reveals-drug-targets-and-potential-drug--repurposing"><span class="header-section-number">14.81</span> A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug- Repurposing</h2>
<p>Gordon et al. <em>bioRxiv</em> <span class="citation" data-cites="phJM8g2Y">(<a href="#ref-phJM8g2Y" role="doc-biblioref">2206</a>)</span></p>
<h3 data-number="14.81.1" id="keywords-81"><span class="header-section-number">14.81.1</span> Keywords</h3>
<ul>
<li>protein-protein interactions</li>
<li>mass spectrometry</li>
<li>drug targets</li>
</ul>
<h3 data-number="14.81.2" id="main-findings-77"><span class="header-section-number">14.81.2</span> Main Findings</h3>
<p>Gordon et al cloned, tagged and expressed 26 of the 29
SARS-CoV-2 proteins individually in HEK293T cells and used mass
spectrometry to identify protein-protein interactions. They identified
332 viral-host protein-protein interactions. Furthermore, they used
these interactions to identify 66 existing drugs known to target host
proteins or host pathways (eg SARS-CoV-2 N and Orf8 proteins interact
with proteins regulated by the mTOR pathway, so mTOR inhibitors
Silmitasertib and Rapamycin are possible drug candidates).</p>
<h3 data-number="14.81.3" id="limitations-78"><span class="header-section-number">14.81.3</span> Limitations</h3>
<p>The main limitation of the study stems from
the reductionist model: overexpression of plasmids encoding individual
viral proteins in HEK293T cells. This precludes any interactions between
the viral proteins, or the combined effects of multiple proteins on the
host, as they are expressed individually. Moreover, HEK293T cells come
from primary embryonic kidney and therefore might not reflect how
SARS-CoV-2 interacts with its primary target, the lung. However, the
authors found that the proteins found to interact with viral proteins in
their experiments are enriched in lung tissue compared to HEK293Ts.</p>
<h3 data-number="14.81.4" id="significance-74"><span class="header-section-number">14.81.4</span> Significance</h3>
<p>The authors provide a “SARS-CoV-2 interaction map,”
which may provide potential hypotheses as to how the virus interacts
with the host. Further, they identified existing drugs that could
disrupt these host-viral interactions and curb SARS-CoV-2 infection.
Although these interactions have not been validated, this paper acts as
a valuable resource.</p>
<h3 data-number="14.81.5" id="credit-78"><span class="header-section-number">14.81.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.82" id="first-clinical-study-using-hcv-protease-inhibitor-danoprevir-to-treat-naïve-and-experienced-covid-19-patients"><span class="header-section-number">14.82</span> First Clinical Study Using HCV Protease Inhibitor Danoprevir to Treat Naïve and Experienced COVID-19 Patients</h2>
<p>Chen et al. <em>medRxiv</em>.
<span class="citation" data-cites="RTIk8DTo">(<a href="#ref-RTIk8DTo" role="doc-biblioref">2207</a>)</span></p>
<h3 data-number="14.82.1" id="keywords-82"><span class="header-section-number">14.82.1</span> Keywords</h3>
<ul>
<li>Clinical study</li>
<li>HCV protease inhibitor</li>
<li>Danoprevir</li>
<li>Ritonavir</li>
<li>Covid19 treatment</li>
</ul>
<h3 data-number="14.82.2" id="main-findings-78"><span class="header-section-number">14.82.2</span> Main Findings</h3>
<p>The authors treated 11 Covid-19 patients with Danoprevir, a
commercialized HCV protease inhibitor <span class="citation" data-cites="YoCPtq6P">(<a href="#ref-YoCPtq6P" role="doc-biblioref">2208</a>)</span><sup>(p4)</sup>, boosted by ritonavir <span class="citation" data-cites="10GnhBctk">(<a href="#ref-10GnhBctk" role="doc-biblioref">2209</a>)</span>, a
CYP3A4 inhibitor (which enhances the plasma concentration and
bioavailabilty of Danoprevir). Two patients had never received
anti-viral therapy before (=naïve), whereas nine patients were on
Lopinavir/Ritonavir treatment before switching to Danoprevir/Ritonavir
(=experienced). The age ranged from 18 to 66yo.</p>
<p>Naïve patients that received Danoprevir/Ritonavir treatment had a
decreased hospitalization time. Patients treated with
Lopinavir/Ritonavir did not have a negative PCR test, while after
switching to Danoprevir/Ritonavir treatment, the first negative PCR test
occurred at a median of two days.</p>
<h3 data-number="14.82.3" id="limitations-79"><span class="header-section-number">14.82.3</span> Limitations</h3>
<p>The results of the study are very hard to interpret as there is no
control group not receiving Danoprevir/Ritonavir treatment. This was
especially true in naïve patients who seemed to have more mild symptoms
before the start of the study and were younger (18 and 44yo) compared to
the experienced patients (18 to 66yo). The possibility that the patients
would have recovered without Danoprevir/Ritonavir treatment cannot be
excluded.</p>
<h3 data-number="14.82.4" id="significance-75"><span class="header-section-number">14.82.4</span> Significance</h3>
<p>The authors of the study treated patients with Danoprevir, with the
rational to that this is an approved and well tolerated drug for HCV
patients <span class="citation" data-cites="10GnhBctk">(<a href="#ref-10GnhBctk" role="doc-biblioref">2209</a>)</span>, and that it could also target the protease from SARS-CoV-2
(essential for viral replication and transcription). Indeed, homology
modelling data indicated that HCV protease inhibitors have the highest
binding affinity to Sars-Cov2 protease among other approved drugs <span class="citation" data-cites="7zBwK1wn">(<a href="#ref-7zBwK1wn" role="doc-biblioref">2210</a>)</span>.</p>
<p>While this study shows that the combination of Danoprevir and Ritonavir
might be beneficial for Covid-19 patients, additional clinical trials
with more patients and with better methodology (randomization and
control group) are needed to make further conclusions.</p>
<h3 data-number="14.82.5" id="credit-79"><span class="header-section-number">14.82.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.83" id="efficacy-of-hydroxychloroquine-in-patients-with-covid-19-results-of-a-randomized-clinical-trial"><span class="header-section-number">14.83</span> Efficacy of hydroxychloroquine in patients with COVID-19: results of a randomized clinical trial</h2>
<p><span class="citation" data-cites="wzHb7Mj5">(<a href="#ref-wzHb7Mj5" role="doc-biblioref">1043</a>)</span></p>
<h3 data-number="14.83.1" id="keywords-83"><span class="header-section-number">14.83.1</span> Keywords</h3>
<ul>
<li>hydroxychloroquine</li>
</ul>
<h3 data-number="14.83.2" id="study-description-1"><span class="header-section-number">14.83.2</span> Study Description</h3>
<p>This is a randomized clinical trial of hydroxychloroquine (HCQ) efficacy
in the treatment of COVID-19. From February 4 – February 28, 2020 142
COVID-19 positive patients were admitted to Renmin Hospital of Wuhan
University. 62 patients met inclusion criteria and were enrolled in a
double blind, randomized control trial, with 31 patients in each arm.</p>
<p>Inclusion criteria:</p>
<ol type="1">
<li><p>Age ≥ 18 years</p></li>
<li><p>Positive diagnosis COVID-19 by detection of SARS-CoV-2 by RT-PCR</p></li>
<li><p>Diagnosis of pneumonia on chest CT</p></li>
<li><p>Mild respiratory illness, defined by SaO<sub>2</sub>/SPO<sub>2</sub> ratio &gt; 93% or
PaO<sub>2</sub>/FIO<sub>2</sub> ratio &gt; 300 mmHg in hospital room conditions (Note:
relevant clinical references described below.)</p>
<ol type="a">
<li><p>Hypoxia is defined as an SpO2 of 85-94%; severe hypoxia &lt;
85%.</p></li>
<li><p>The PaO<sub>2</sub>/FIO<sub>2</sub> (ratio of arterial oxygen tension to fraction
of inspired oxygen) is used to classify the severity of acute
respiratory distress syndrome (ARDS). Mild ARDS has a
PaO<sub>2</sub>/FIO<sub>2</sub> of 200-300 mmHg, moderate is 100-200, and severe
&lt; 100.</p></li>
</ol></li>
<li><p>Willing to receive a random assignment to any designated treatment
group; not participating in another study at the same time</p></li>
</ol>
<p>Exclusion criteria:</p>
<ol type="1">
<li><p>Severe or critical respiratory illness (not explicitly defined,
presumed to be respiratory function worse than outlined in inclusion
criteria); or participation in trial does not meet patient’s maximum
benefit or safe follow up criteria</p></li>
<li><p>Retinopathy or other retinal diseases</p></li>
<li><p>Conduction block or other arrhythmias</p></li>
<li><p>Severe liver disease, defined by Child-Pugh score ≥ C or AST &gt;
twice the upper limit</p></li>
<li><p>Pregnant or breastfeeding</p></li>
<li><p>Severe renal failure, defined by eGFR ≤ 30 mL/min/1.73m<sup>2</sup>, or on
dialysis</p></li>
<li><p>Potential transfer to another hospital within 72h of enrollment</p></li>
<li><p>Received any trial treatment for COVID-19 within 30 days before the
current study</p></li>
</ol>
<p>All patients received the standard of care: oxygen therapy, antiviral
agents, antibacterial agents, and immunoglobulin, with or without
corticosteroids. Patients in the HCQ treatment group received additional
oral HCQ 400 mg/day, given as 200 mg 2x/day. HCQ was administered from
days 1-5 of the trial. The primary endpoint was 5 days post enrollment
or a severe adverse reaction to HCQ. The primary outcome evaluated was
time to clinical recovery (TTCR), defined as return to normal body
temperature and cough cessation for &gt; 72h. Chest CT were imaged on
days 0 and 6 of the trial for both groups; body temperature and patient
reports of cough were collected 3x/day from day 0 – 6. The mean age and
sex distribution between the HCQ and control arms were comparable.</p>
<h3 data-number="14.83.3" id="main-findings-79"><span class="header-section-number">14.83.3</span> Main Findings</h3>
<p>There were 2 patients showing mild secondary effects of HCQ treatment.
More importantly, while 4 patients in the control group progressed to
severe disease, none progressed in the treatment group.</p>
<p>TTCR was significantly decreased in the HCQ treatment arm; recovery from
fever was shortened by one day (3.2 days control vs. 2.2 days HCQ, p =
0.0008); time to cessation of cough was similarly reduced (3.1 days
control vs. 2.0 days HCQ, p = 0.0016).</p>
<p>Overall, it appears that HCQ treatment of patients with mild COVID-19
has a modest effect on clinical recovery (symptom relief on average 1
day earlier) but may be more potent in reducing the progression from
mild to severe disease.</p>
<h3 data-number="14.83.4" id="limitations-80"><span class="header-section-number">14.83.4</span> Limitations</h3>
<p>This study is limited in its inclusion of only patients with mild
disease, and exclusion of those on any treatment other than the standard
of care. It would also have been important to include the laboratory
values of positive RT-PCR detection of SARS-CoV-2 to compare the
baseline and evolution of the patients’ viral load.</p>
<h3 data-number="14.83.5" id="limitations-81"><span class="header-section-number">14.83.5</span> Limitations</h3>
<p>Despite its limitations, the study design has good rigor as a double
blind RCT and consistent symptom checks on each day of the trail. Now
that the FDA has approved HCQ for treatment of COVID-19 in the USA, this
study supports the efficacy of HCQ use early in treatment of patients
showing mild symptoms, to improve time to clinical recovery, and
possibly reduce disease progression. However, most of the current
applications of HCQ have been in patients with severe disease and for
compassionate use, which are out of the scope of the findings presented
in this trial. Several additional clinical trials to examine
<a href="https://twitter.com/hashtag/hydroxychloroquine?src=hashtag_click">hydroxychloroquine</a>
are now undergoing; their results will be critical to further validate
these findings.</p>
<h3 data-number="14.83.6" id="credit-80"><span class="header-section-number">14.83.6</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<p><strong>Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens</strong></p>
<p><a href="https://doi.org/10.1101/2020.03.23.004176" class="uri">https://doi.org/10.1101/2020.03.23.004176</a></p>
<p><em>Immunology keywords</em>:</p>
<p>CoVID-19, 2019-nCoV, SARS-CoV-2, comparative, homology, peptide,
modeling, simulation, HLA-A, antigen</p>
<p><em>Summary of Findings</em>:</p>
<ul>
<li><p>The authors utilize homology modeling to identify peptides from the
SARS-CoV-2 proteome that potentially bind HLA-A*02:01.</p></li>
<li><p>They utilize high-resolution X-ray structures of peptide/MHC
complexes on Protein Data Bank, substitute homologous peptides with
SARS-CoV-2 peptides, and calculate MHC/SARS-CoV-2 peptide Rosetta
binding energy.</p></li>
<li><p>They select MHC/SARS-CoV-2 complex models with highest binding
energy for further study and publish models in an online database
(https://rosettamhc.chemistry.ucsc.edu).</p></li>
</ul>
<p><em>Limitations</em>:</p>
<ul>
<li><p>The authors only utilize computational methods and predicted
SARS-CoV-2 peptides must be validated experimentally for
immunogenicity and clinical response.</p></li>
<li><p>Due to computational burden and limited availability of high
resolution X-ray structures on PDB, authors only simulate 9-mer and
10-mer peptide binding to HLA-A*02:01.</p></li>
<li><p>Since the authors compare select existing X-ray structures as a
starting point, backbone conformations that deviate significantly
between test and template peptides are not captured. Furthermore,
Rosetta modeling protocols do not capture all possible structures
and binding energy scoring does not fully recapitulate fundamental
forces.<sup>1,2</sup></p></li>
</ul>
<p><em>Importance/Relevance:</em></p>
<ul>
<li><p>The authors identify and publish high-scoring SARS-CoV-2 peptides
that may direct a targeted, experimental validation approach toward
a COVID-19 vaccine.</p></li>
<li><p>The authors utilize Rosetta simulation to further filter results
from NetMHCpan 4.0, supporting machine learning prediction with
structural analysis.</p></li>
<li><p>The authors develop RosettaMHC, a computationally efficient method
of leveraging existing X-ray structures for identification of
immunogenic peptides.</p></li>
</ul>
<p><em>References:</em></p>
<ol type="1">
<li><p>Bender, B. J., Cisneros, A., 3rd, Duran, A. M., Finn, J. A., Fu, D.,
Lokits, A. D., . . . Moretti, R. (2016). Protocols for Molecular
Modeling with Rosetta3 and RosettaScripts. Biochemistry, 55(34),
4748-4763. doi:10.1021/acs.biochem.6b00444</p></li>
<li><p>Alford, R. F., Leaver-Fay, A., Jeliazkov, J. R., O’Meara, M. J.,
DiMaio, F. P., Park, H., . . . Gray, J. J. (2017). The Rosetta
All-Atom Energy Function for Macromolecular Modeling and Design. J
Chem Theory Comput, 13(6), 3031-3048. doi:10.1021/acs.jctc.7b00125</p></li>
</ol>
<p><em>Review by Jonathan Chung as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.</em></p>
<h2 data-number="14.84" id="serology-characteristics-of-sars-cov-2-infection-since-the-exposure-and-post-symptoms-onset"><span class="header-section-number">14.84</span> Serology characteristics of SARS-CoV-2 infection since the exposure and post symptoms onset</h2>
<p>Lou et al. <em>medRxiv</em>. <span class="citation" data-cites="lnuNSHEW">(<a href="#ref-lnuNSHEW" role="doc-biblioref">2211</a>)</span></p>
<h3 data-number="14.84.1" id="keywords-84"><span class="header-section-number">14.84.1</span> Keywords</h3>
<ul>
<li>Seroconversion rate</li>
<li>Total Antibody</li>
<li>Ab</li>
<li>IgG and IgM</li>
<li>antibody</li>
</ul>
<h3 data-number="14.84.2" id="main-findings-80"><span class="header-section-number">14.84.2</span> Main Findings</h3>
<p>Currently, the diagnosis of SARS-CoV-2 infection
entirely depends on the detection of viral RNA using polymerase chain
reaction (PCR) assays. False negative results are common, particularly
when the samples are collected from upper respiratory. Serological
detection may be useful as an additional testing strategy. In this study
the authors reported that a typical acute antibody response was induced
during the SARS-CoV-2 infection, which was discuss earlier<sup>1</sup>. The
seroconversion rate for Ab, IgM and IgG in COVID-19 patients was 98.8%
(79/80), 93.8% (75/80) and 93.8% (75/80), respectively. The first
detectible serology marker was total antibody followed by IgM and IgG,
with a median seroconversion time of 15, 18 and 20 days-post exposure
(d.p.e) or 9, 10- and 12-days post-onset (d.p.o). Seroconversion was
first detected at day 7d.p.e in 98.9% of the patients. Interestingly
they found that viral load declined as antibody levels increased. This
was in contrast to a previous study <span class="citation" data-cites="5HXuCa5t">(<a href="#ref-5HXuCa5t" role="doc-biblioref">2151</a>)</span>, showing that increased antibody
titers did not always correlate with RNA clearance (low number of
patient sample).</p>
<h3 data-number="14.84.3" id="limitations-82"><span class="header-section-number">14.84.3</span> Limitations</h3>
<p>Current knowledge of the antibody response to SAR-CoV-2
infection and its mechanism is not yet well elucidated. Similar to the
RNA test, the absence of antibody titers in the early stage of illness
could not exclude the possibility of infection. A diagnostic test, which
is the aim of the authors, would not be useful at the early time points
of infection but it could be used to screen asymptomatic patients or
patients with mild disease at later times after exposure.</p>
<h3 data-number="14.84.4" id="significance-76"><span class="header-section-number">14.84.4</span> Significance</h3>
<p>Understanding the antibody responses against SARS-CoV2 is
useful in the development of a serological test for the diagnosis of
COVID-19. This manuscript discussed acute antibody responses which can
be deducted in plasma for diagnostic as well as prognostic purposes.
Thus, patient-derived plasma with known antibody titers may be used
therapeutically for treating COVID-19 patients with severe illness.
<span id="OLE_LINK3" class="anchor"><span id="OLE_LINK4" class="anchor"></span></span></p>
<h3 data-number="14.84.5" id="credit-81"><span class="header-section-number">14.84.5</span> Credit</h3>
<p><em>This review was undertaken and edited by Konstantina A as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.85" id="sars-cov-2-launches-a-unique-transcriptional-signature-from-in-vitro-ex-vivo-and-in-vivo-systems"><span class="header-section-number">14.85</span> SARS-CoV-2 launches a unique transcriptional signature from in vitro, ex vivo, and in vivo systems</h2>
<p>Blanco-Melo et al. <em>bioRxiv</em>. <span class="citation" data-cites="aNSdARwe">(<a href="#ref-aNSdARwe" role="doc-biblioref">2212</a>)</span></p>
<h3 data-number="14.85.1" id="keywords-85"><span class="header-section-number">14.85.1</span> Keywords</h3>
<ul>
<li>host cellular response</li>
<li>host-pathogen interaction</li>
<li>type I interferon</li>
<li>type III interferon</li>
<li>inflammation</li>
<li>RNA-seq</li>
<li>comparative analysis</li>
</ul>
<h3 data-number="14.85.2" id="main-findings-81"><span class="header-section-number">14.85.2</span> Main Findings</h3>
<p>Given the high mortality rate of SARS-CoV-2 relative to
other respiratory viruses such seasonal IAV and RSV, there may be
underlying host-pathogen interactions specific to SARS-CoV-2 that
predispose to a worse clinical outcome. Using <em>in vivo</em>, <em>ex vivo</em>, and
<em>in vitro</em> systems, the authors profiled host cell transcriptional
responses to SARS-CoV-2 and to other common respiratory viruses
(seasonal IAV and RSV). SARS-CoV-2 infection <em>in vitro</em> led to an
induction of type I interferon response signaling and the upregulation
of cytokine/chemokines transcripts. In comparison with IAV and RSV
infection, SARS-CoV-2 <em>in vitro</em> appears to uniquely induce less type I
and type III interferon expression and higher levels of two cytokines
previously implicated in respiratory inflammation. Lastly, <em>in vivo</em>
data from ferrets showed a reduced induction of cytokines and chemokines
by SARS-CoV-2 infection relative to IAV infection.</p>
<h3 data-number="14.85.3" id="limitations-83"><span class="header-section-number">14.85.3</span> Limitations</h3>
<p>While these results are promising, there are several
key weaknesses of this paper. 1) As the authors point out, there is an
undetectable level of SARS-CoV-2 putative receptor (ACE2) and protease
(TMPRSS2) expression in the lung epithelial cell line used for the <em>in
vitro</em> studies. This raises the important question of whether viral
replication actually occurs in any of the models used, which may explain
the lack of interferon production observed <em>in vitro</em> in SARS-CoV-2
treated cells. Further studies characterizing viral titers across
timepoints are needed. 2) Furthermore, these studies only characterize
the host response at a single dose and timepoint per virus, and it is
unclear why these doses/timepoints were chosen. This leaves open the
possibility that the observed differences between viruses could be due
to differences in dose, timing, host response, or a combination of all
of these. 3) It is unclear whether ferrets are productively infected,
which cell types are infected, and the extent/timing of the clinical
course of infection. Moreover, the <em>in vitro</em> and <em>in vivo</em> data do not
strongly correlate and the reasons for this are unclear.</p>
<h3 data-number="14.85.4" id="significance-77"><span class="header-section-number">14.85.4</span> Significance</h3>
<p>This paper describes potentially unique
transcriptional signatures of host cells exposed to SARS-CoV-2. If
validated, these findings may help explain clinical outcomes and could
be targeted in future therapeutic interventions.</p>
<h3 data-number="14.85.5" id="potential-conflicts-of-interest-disclosure"><span class="header-section-number">14.85.5</span> Potential Conflicts of Interest Disclosure</h3>
<p>The reviewers are also researchers at the Icahn School
of Medicine at Mount Sinai.</p>
<h3 data-number="14.85.6" id="credit-82"><span class="header-section-number">14.85.6</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.86" id="a-new-predictor-of-disease-severity-in-patients-with-covid-19-in-wuhan-china"><span class="header-section-number">14.86</span> A New Predictor of Disease Severity in Patients with COVID-19 in Wuhan, China</h2>
<p>Zhou et al. <em>bioRxiv</em>. <span class="citation" data-cites="EWZmuLvu">(<a href="#ref-EWZmuLvu" role="doc-biblioref">2213</a>)</span></p>
<h3 data-number="14.86.1" id="keywords-86"><span class="header-section-number">14.86.1</span> Keywords</h3>
<ul>
<li>disease severity</li>
<li>clinical data</li>
<li>Neutrophils/lymphocytes ratio</li>
<li>CRP</li>
<li>D-dimer</li>
</ul>
<h3 data-number="14.86.2" id="main-findings-82"><span class="header-section-number">14.86.2</span> Main Findings</h3>
<p>377 hospitalized patients were divided into two groups: severe and
non-severe pneumonia. The laboratory results of their first day of
admission were retrospectively analyzed to identify predictors of
disease severity.</p>
<p>After adjusting for confounding factors from chronic comorbidities (such
as high blood pressure, type 2 diabetes, coronary heart disease, and
chronic obstructive pulmonary disease), the independent risk factors
identified for severe pneumonia were <strong>age</strong>, the <strong>ratio of
neutrophil/lymphocytes counts</strong>, <strong>CRP</strong> and <strong>D-dimer</strong> levels.</p>
<p>To further increase the specificity and sensibility of these markers,
they showed that their multiplication <strong>[(Neutrophil/lymphocyte count)
* CRP * D-dimer]</strong> was a better predictor of disease severity, with
higher sensitivity (95.7%) and specificity (63.3%), with a cutoff value
of 2.68.</p>
<h3 data-number="14.86.3" id="limitations-84"><span class="header-section-number">14.86.3</span> Limitations</h3>
<p>This study included 377 hospitalized patients. Among them, 45.6%
patients tested positive for SARS-Cov-2 nucleic acid test results, and
others were included in the study based on clinically diagnosis even if
the molecular diagnosis was negative. Thus, additional studies are
needed to verify this on a larger number of covid-19 certified patients
and the cutoff value might be adjusted. Also, all the patients that did
not have the clinical characteristics of severe pneumonia were included
in the non-severe pneumonia group, but usually patients are also divided
into moderate and mild disease.</p>
<p>Also, studying different subset of lymphocytes could lead to a more
specific predictor. Another study showed that the neutrophils to CD8+ T
cells ratio was a strong predictor of disease severity <span class="citation" data-cites="zTGzuiTI">(<a href="#ref-zTGzuiTI" role="doc-biblioref">2121</a>)</span>. Another more
precise study showed that the percentage of helper T cells and
regulatory T cells decrease but the percentage of naïve helper T cells
increases in severe cases <span class="citation" data-cites="6Y81C4P3">(<a href="#ref-6Y81C4P3" role="doc-biblioref">2114</a>)</span>. Taking these subpopulations into account
might make the predictor more powerful.</p>
<p>Other studies also noted an inverse correlation between disease severity
and LDH <span class="citation" data-cites="USOnp7c0">(<a href="#ref-USOnp7c0" role="doc-biblioref">2155</a>)</span> or IL6 <span class="citation" data-cites="1GEbBtBgh">(<a href="#ref-1GEbBtBgh" role="doc-biblioref">2164</a>)</span> levels, but the authors here do not discuss LDH nor
IL6 levels, although this could help to strengthen the predictor.</p>
<p>The study is based on the results obtained on the first day of
admission, studying the dynamic of the changes in patients might also be
interesting to better predict disease severity.</p>
<h3 data-number="14.86.4" id="significance-78"><span class="header-section-number">14.86.4</span> Significance</h3>
<p>This study confirms that the neutrophil to lymphocyte ratio can be a
predictor of disease severity as shown by many others <span class="citation" data-cites="6Y81C4P3 VRoblZFJ gK5e7Yhl">(<a href="#ref-gK5e7Yhl" role="doc-biblioref">2113</a>, <a href="#ref-6Y81C4P3" role="doc-biblioref">2114</a>, <a href="#ref-VRoblZFJ" role="doc-biblioref">2127</a>)</span>. The
novelty here is that they show that a combination with other markers can
enhance the specificity and sensibility of the predictor, although the
study could be improved by taking into account sub-populations of
lymphocytes and more biological factors from patients such as LDH and
IL6.</p>
<h3 data-number="14.86.5" id="credit-83"><span class="header-section-number">14.86.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.87" id="metabolic-disturbances-and-inflammatory-dysfunction-predict-severity-of-coronavirus-disease-2019-covid-19-a-retrospective-study"><span class="header-section-number">14.87</span> Metabolic disturbances and inflammatory dysfunction predict severity of coronavirus disease 2019 (COVID-19): a retrospective study</h2>
<p>Shuke Nie et al. <em>medRxiv</em>. <span class="citation" data-cites="95sQSGzx">(<a href="#ref-95sQSGzx" role="doc-biblioref">2214</a>)</span></p>
<h3 data-number="14.87.1" id="keywords-87"><span class="header-section-number">14.87.1</span> Keywords</h3>
<ul>
<li>metabolism</li>
<li>fasting blood glucose</li>
<li>serum total protein</li>
<li>albumin</li>
<li>blood lipid</li>
<li>HDL-C</li>
<li>APOA1</li>
<li>lymphocytopenia</li>
<li>IL-6</li>
<li>CRP</li>
<li>severity prediction of COVID19</li>
</ul>
<h3 data-number="14.87.2" id="main-findings-83"><span class="header-section-number">14.87.2</span> Main Findings</h3>
<p>Retrospective Study on 97 COVID-19 hospitalized patients
(25 severe and 72 non-severe) analyzing clinical and laboratory
parameter to predict transition from mild to severe disease based on
more accessible indicators (such as fasting blood glucose, serum protein
or blood lipid) than inflammatory indicators. In accordance with other
studies, age and hypertension were risk factors for disease severity,
and lymphopenia and increased IL-6 was observed in severe patients. The
authors show that fasting blood glucose (FBG) was altered and patients
with severe disease were often hyperglycemic. Data presented support
that hypoproteinaemia, hypoalbuminemia, and reduction in
high-densitylipoprotein (HDL-C) and ApoA1 were associated with disease
severity.</p>
<h3 data-number="14.87.3" id="limitations-85"><span class="header-section-number">14.87.3</span> Limitations</h3>
<p>In this study non-severe patients were divided in two groups based on
average course of the disease: mild group1 (14 days, n=28) and mild
group 2 (30 days, n=44). However mild patients with a longer disease
course did not show an intermediate phenotype (between mild patients
with shorter disease course and severe patients), hence it is unclear
whether this was a useful and how it impacted the analysis. Furthermore,
the non-exclusion of co-morbidity factors in the analysis may bias the
results (e.g. diabetic patients and glucose tests) It is not clear at
what point in time the laboratory parameters are sampled. In table 3, it
would have been interesting to explore a multivariate multiple
regression. The correlation lacks of positive control to assess the
specificity of the correlation to the disease vs. correlation in any
inflammatory case. The dynamic study assessing the predictability of the
laboratory parameter is limited to 2 patients. Hence there are several
associations with disease severity, but larger studies are necessary to
test the independent predictive value of these potential biomarkers.</p>
<h3 data-number="14.87.4" id="significance-79"><span class="header-section-number">14.87.4</span> Significance</h3>
<p>As hospital are getting overwhelmed a set of
easily accessible laboratory indicators (such as serum total protein)
would potentially provide a triage methodology between potentially
severe cases and mild ones. This paper also opens the question regarding
metabolic deregulation and COVID-19 severity.</p>
<h3 data-number="14.87.5" id="credit-84"><span class="header-section-number">14.87.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.88" id="viral-kinetics-and-antibody-responses-in-patients-with-covid-19"><span class="header-section-number">14.88</span> Viral Kinetics and Antibody Responses in Patients with COVID-19</h2>
<p><span class="citation" data-cites="btVuoX6V">(<a href="#ref-btVuoX6V" role="doc-biblioref">2215</a>)</span></p>
<h3 data-number="14.88.1" id="keywords-88"><span class="header-section-number">14.88.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>IgG</li>
<li>IgM</li>
<li>clinical</li>
<li>kinetics</li>
<li>antibodies</li>
</ul>
<h3 data-number="14.88.2" id="main-findings-84"><span class="header-section-number">14.88.2</span> Main Findings</h3>
<ul>
<li><p>Prospective cohort of 67 patients, clinical specimens taken and
follow-up conducted.</p></li>
<li><p>Viral shedding, serum IgM, IgG antibody against NP evaluated and
correlated to disease severity and clinical outcome</p></li>
<li><p>Viral RNA levels peaked at 1 week from febrile/cough symptom onset
in sputum, nasal swabs, and stool samples. Shedding ranged from
12-19 days (median ranges) and was longer in severe patients.</p></li>
<li><p>IgM and IgG titers stratified patients into three archetypes as
‘strong vs weak vs non-responders’. Strong responders (with higher
IgM/IgG titers) were significantly higher in severe patients.</p></li>
</ul>
<h3 data-number="14.88.3" id="limitations-86"><span class="header-section-number">14.88.3</span> Limitations</h3>
<p>Specific for immune monitoring.</p>
<ul>
<li><p>Not clear if stool RNA captured from live infection in
intestine/liver or from swallowed sputum. Transmission electron
microscopy (TEM) carried out on sputum samples as proof of concept,
but not stools. TEM unreasonable for actual clinical diagnosis.</p></li>
<li><p>Several patients had co-morbidities (such as pulmonary and liver
disease) that were not accounted for when tracking antibody
responses. Viral kinetics and IgM/IgG titers in subsets of patients
with underlying conditions/undergoing certain medication would be
informative.</p></li>
</ul>
<h3 data-number="14.88.4" id="significance-80"><span class="header-section-number">14.88.4</span> Significance</h3>
<ul>
<li><p>Three archetypes of antibody response to SARS-CoV-2 with different
disease progression and kinetics is useful to stratify patients, and
for future serological tests.</p></li>
<li><p>Strong spike-IgG levels often correlate with lymphopenia and
CoVID-19 disease severity <span class="citation" data-cites="LIf5SSUs">(<a href="#ref-LIf5SSUs" role="doc-biblioref">2216</a>)</span>, similar to macaque
studies in SARS <span class="citation" data-cites="tC5vJmwj">(<a href="#ref-tC5vJmwj" role="doc-biblioref">2217</a>)</span>. It would be critical to see if anti-NP or
anti-Spike IgG antibodies for SARS-CoV-2 also elicit similar
detrimental effects before clinical use.</p></li>
</ul>
<h3 data-number="14.88.5" id="credit-85"><span class="header-section-number">14.88.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.89" id="covid-19-infection-induces-readily-detectable-morphological-and-inflammation-related-phenotypic-changes-in-peripheral-blood-monocytes-the-severity-of-which-correlate-with-patient-outcome"><span class="header-section-number">14.89</span> COVID-19 infection induces readily detectable morphological and inflammation-related phenotypic changes in peripheral blood monocytes, the severity of which correlate with patient outcome</h2>
<p><span class="citation" data-cites="1FZgghazg">(<a href="#ref-1FZgghazg" role="doc-biblioref">2218</a>)</span></p>
<h3 data-number="14.89.1" id="keywords-89"><span class="header-section-number">14.89.1</span> Keywords</h3>
<ul>
<li>Monocytes</li>
<li>FSC-high</li>
<li>PBMC</li>
<li>ACE2</li>
<li>inflammatory cytokines</li>
</ul>
<h3 data-number="14.89.2" id="main-findings-85"><span class="header-section-number">14.89.2</span> Main Findings</h3>
<p>This study is based on flow cytometry immunophenotyping of
PBMCs from 28 patients diagnosed positive for SARS-Cov2 (COVID19). The
authors identify a population of abnormally large (FSC-hi) monocytes,
present in COVID19 patients, but absent in PBMCs of healthy volunteers
(n=16) or patients with different infections (AIDS, malaria, TB). This
FSC-hi monocytic population contains classical, intermediate and
non-classical (monocytes (based on CD14 and CD16 expression) that
produce inflammatory cytokines (IL-6, TNF and IL-10). The authors
suggest an association of FSC-hi monocytes with poor outcome and
correlate a high percentage of FSC-low monocytes, or higher ratio of
FSC-low/hi monocytes, with faster hospital discharge.</p>
<h3 data-number="14.89.3" id="limitations-87"><span class="header-section-number">14.89.3</span> Limitations</h3>
<p>While identification of the monocytic population based
on FSC is rather robust, the characterization of these cells remains
weak. A comprehensive comparison of FSC-hi monocytes with FSC-low
monocytes from patients and healthy controls would be of high value. It
is unclear if percentages in blood are among CD45+ cells. Furthermore,
it would have been important to include absolute numbers of different
monocytic populations (in table 1 there are not enough samples and it is
unclear what the authors show).</p>
<p>The authors show expression of the ACE2 receptor on the surface of the
monocytes, and highlight these cells as potential targets of SARS-Cov2.
However, appropriate controls are needed. CD16 has high affinity to
rabbit IgG and it is unclear whether the authors considered unspecific
binding of rabbit anti-ACE2 to Fc receptors. Gene expression of ACE-2 on
monocytes needs to be assessed. Furthermore, it would be important to
confirm infection of monocytes by presence of viral proteins or viral
particles by microscopy.</p>
<p>Considering the predictive role of FSC-hi monocytes on the development
of the disease and its severity, some data expected at this level are
neither present nor addressed. Although the cohort is small, it does
include 3 ICU patients. What about their ratio of FSC-low vs FSC-hi
monocytes in comparison to other patients? Was this apparent early in
the disease course? Does this population of FSC-hi monocytes differ
between ICU patients and others in terms of frequency, phenotype or
cytokine secretion?</p>
<p>In general, figures need to revised to make the data clear. For example,
in Fig. 5, according to the legend it seems that patients with FSC-high
monocytes are discharged faster from the hospital. However according to
description in the text, patients were grouped in high or low levels of
FSC-low monocytes.</p>
<h3 data-number="14.89.4" id="significance-81"><span class="header-section-number">14.89.4</span> Significance</h3>
<p>Despite the limitations of this study, the discovery of
a FSC-high monocyte population in COVID-19 patients is of great
interest. With similar implication, a the recent study by Zhou et al. <span class="citation" data-cites="Bgo7Cmj3">(<a href="#ref-Bgo7Cmj3" role="doc-biblioref">2119</a>)</span>
identified a connection between an inflammatory CD14+CD16+ monocyte
population and pulmonary immunopathology leading to deleterious clinical
manifestations and even acute mortality after SARS-CoV-2 infections.
Although the presence of these monocytes in the lungs has yet to be
demonstrated, such results support the importance of monocytes in the
critical inflammation observed in some COVID19 patients.</p>
<h3 data-number="14.89.5" id="credit-86"><span class="header-section-number">14.89.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.90" id="correlation-between-universal-bcg-vaccination-policy-and-reduced-morbidity-and-mortality-for-covid-19-an-epidemiological-study"><span class="header-section-number">14.90</span> Correlation between universal BCG vaccination policy and reduced morbidity and mortality for COVID-19: an epidemiological study</h2>
<p>Miller et al. <em>medRxiv</em>. <span class="citation" data-cites="15gdkEnfn">(<a href="#ref-15gdkEnfn" role="doc-biblioref">2219</a>)</span></p>
<h3 data-number="14.90.1" id="keywords-90"><span class="header-section-number">14.90.1</span> Keywords</h3>
<ul>
<li>BCG vaccine</li>
<li>epidemiology</li>
<li>vaccination policy</li>
</ul>
<h3 data-number="14.90.2" id="main-findings-86"><span class="header-section-number">14.90.2</span> Main Findings</h3>
<p>The authors compared middle and high income countries that never had a
universal BCG vaccination policy (Italy, Lebanon, Nederland, Belgium)
and countries with a current policy (low income countries were excluded
from the analysis as their number of cases and deaths might be
underreported for the moment). <strong>Countries that never implement BCG
vaccination have a higher mortality rate than countries which have a BCG
vaccination policy</strong> (16.38 deaths per million people vs 0.78). Next,
<strong>the authors show that an earlier start of vaccination correlates with
a lower number of deaths per million inhabitants</strong>. They interpret this
as the vaccine protecting a larger fraction of elderly people, which are
usually more affected by COVID-19. Moreover, higher number of COVID-19
<strong>cases</strong> were presented in countries that never implemented a universal
BCG vaccination policy.</p>
<h3 data-number="14.90.3" id="limitations-88"><span class="header-section-number">14.90.3</span> Limitations</h3>
<p>While this study aims to test an intriguing hypothesis unfortunately,
the data is not sufficient at this time to accurately make any
determinations. Several caveats must be noted including: not all
countries are in the same stage of the pandemic, the number of
cases/deaths is still changing very rapidly in a lot of countries and
thus the association may only reflect exposure to the virus. This
analysis would need to be re-evaluated when all the countries are passed
the pandemic and more accurate numbers are available. Additionally, very
few middle and high-income countries ever implemented universal BCG
vaccination, which can be a source of bias (5 countries, vs 55 that have
a BCG vaccine policy). Effective screening and social isolation policies
also varied considerable across the countries tested and may reflect
another important confounder. The authors could consider analyzing the
Case Fatality Rate (CFR, % of patients with COVID-19 that die), to more
correct for exposure although testing availability will still bias this
result. Variability in mortality within countries or cities with
variable vaccination and similar exposure could also be appropriate
although confounders will still be present.</p>
<h3 data-number="14.90.4" id="significance-82"><span class="header-section-number">14.90.4</span> Significance</h3>
<p>BCG vaccine is a live attenuated strain derived from <em>Mycobacterium
bovis</em> and used for a vaccine for tuberculosis (TB). This vaccine has
been proven to be efficient in preventing childhood meningitis TB, but
doesn’t prevent adult TB as efficiently. For this reason, several
countries are now only recommending this vaccine for at-risk population
only.</p>
<p><strong>This study shows that there is a correlation between BCG vaccination
policy and reduced mortality for Covid-19</strong>. Indeed, BCG vaccine has
been shown to protect against several viruses and enhance innate
immunity <span class="citation" data-cites="16FOT89K5">(<a href="#ref-16FOT89K5" role="doc-biblioref">2220</a>)</span>, which could explain why it could protect against SARS-CoV-2
infection, but the exact mechanism is still unknown. <strong>Moreover, the
efficiency of adult/older people vaccination and protection against
Covid-19 still needs to be assessed.</strong> Regarding this, Australian
researchers are starting a clinical trial of BCG vaccine for healthcare
workers <span class="citation" data-cites="MCuIRGFE">(<a href="#ref-MCuIRGFE" role="doc-biblioref">2221</a>)</span>, to assess if it can protect them against Covid-19.</p>
<h3 data-number="14.90.5" id="credit-87"><span class="header-section-number">14.90.5</span> Credit</h3>
<p><em>This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.91" id="non-neural-expression-of-sars-cov-2-entry-genes-in-the-olfactory-epithelium"><span class="header-section-number">14.91</span> Non-neural expression of SARS-CoV-2 entry genes in the olfactory epithelium</h2>
<p><span class="citation" data-cites="12oDb7aki">(<a href="#ref-12oDb7aki" role="doc-biblioref">2222</a>)</span></p>
<h3 data-number="14.91.1" id="keywords-91"><span class="header-section-number">14.91.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>anosmia</li>
<li>olfaction</li>
<li>scRNAseq</li>
</ul>
<h3 data-number="14.91.2" id="main-findings-87"><span class="header-section-number">14.91.2</span> Main Findings</h3>
<ul>
<li><p>Study analyzed bulk and scRNAseq data of olfactory cell types from
publicly-available mouse, nonhuman primate and human datasets.</p></li>
<li><p>show that <em>ACE2</em> and <em>TMPRSS2</em> (genes involved in SARS-CoV-2 entry)
are expressed in olfactory epithelial (OE) cells, basal stem cells
and respiratory epithelium (RE), but not sensory neurons.</p></li>
<li><p>Comparison of human RE and OE datasets (Deprez et al. 2019; Durante
et al. 2020) revealed that <em>ACE2</em> and <em>TMPRSS2</em> expression in OE
sustentacular cells was similar to expression in the remainder of
the non-nasal respiratory tract.</p></li>
</ul>
<h3 data-number="14.91.3" id="limitations-89"><span class="header-section-number">14.91.3</span> Limitations</h3>
<ul>
<li><p>Transcript data alone from healthy respiratory/olfactory cells is
not sufficient to confirm infectivity of nasal passage, or to
indicate damage to epithelia.</p></li>
<li><p>No mechanism defined for anosmia; it is not clear if epithelial
injury leads to reduced sensitivity or increased inflammation and
altered immune contexture drives neural/epithelial dysfunction. Will
be critical to test this in CoVID-19 patient samples or mouse
models.</p></li>
</ul>
<h3 data-number="14.91.4" id="significance-83"><span class="header-section-number">14.91.4</span> Significance</h3>
<ul>
<li><p>Study provides possible rationale for anosmia observed in several
CoVID-19 patients.</p></li>
<li><p>Raises possibility that nasal respiratory goblet, ciliated cells,
and olfactory epithelia may serve as a viral reservoir after initial
SARS-CoV-2 infection.</p></li>
<li><p>Human olfactory sensory neurons express several other molecules
important to CoV (not CoV-19) entry such as <em>FURIN, ST6GAL1,
ST3GAL4</em>; this suggests wider mechanism of neuronal infectivity in
other coronaviruses.</p></li>
</ul>
<h3 data-number="14.91.5" id="credit-88"><span class="header-section-number">14.91.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><strong>Title: </strong></p>
<p><strong>SARS-CoV-2 proteome microarray for mapping COVID-19 antibody
interactions at amino acid resolution</strong></p>
<p><strong>Immunology keywords:</strong> SARS-CoV-2, COVID-19, high throughput, peptide
microarray, antibody epitope screening</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study screened the viral protein epitopes recognized by antibodies
in the serum of 10 COVID-19 patients using a new SARS-CoV-2 proteome
peptide microarray. The peptide library was constructed with 966 linear
peptides, each 15 amino acids long with a 5 amino acid overlap, based on
the protein sequences encoded by the genome of the Wuhan-Hu-1 strain.</p>
<p>To investigate crossreactivity between SARS-CoV-1 and SARS-CoV-2, they
tested rabbit monoclonal and polyclonal antibodies against SARS-CoV-1
nucleocapsid (N) in the microarray. Antibodies against SARS-CoV-1 N
displayed binding to the SARS-CoV-2 nucleocapsid (N) peptides.
Polyclonal antibodies showed some crossreactivity to other epitopes from
membrane (M), spike (S), ORF1ab and ORF8. This suggests that previous
exposure to SARS-CoV-1 may induced antibodies recognizing both viruses.</p>
<p>Screening of IgM and IgG antibodies from 10 COVID-19 patients showed
that many antibodies targeted peptides on M, N, S, Orf1ab, Orf3a, Orf7a,
and Orf8 from SARS-CoV-2, while immunodominant epitopes with antibodies
in more than 80 % COVID-19 patients were present in N, S and Orf3. It is
shown that the receptor binding domain (RBD) resides on S protein and
RBD is important for SARS-CoV-2 to enter the host cells via ACE2. Among
six epitopes on S protein, structural analysis predicted that three
epitopes were located at the surface and three epitopes were located
inside of the protein. Furthermore, some IgM antibodies from 1 patient
and IgG antibodies from 2 patients bound to the same epitope (residue
456-460, FRKSN) which resided within the RBD, and structural analysis
determined that this epitope was located in the region of the RBD loop
that engages with ACE2.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>In addition to the limitations mentioned in the manuscript, it would
have been informative to do the analysis over the course of the disease.
The pattern of antibody recognition, especially on S protein, and the
course of antibodies of different isotypes recognizing the same peptide
might correlate to the clinical course in these patients. It would alos
have been informative to analyze the presence of cross-reactive
antibodies from pateints previously exposed to SARS-CoV-1.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>This study identified linear immunodominant epitopes on SARS-CoV-2,
Wuhan-Hu-1 strain. This is a valuable information to design vaccines
that will elicit desirable immune responses.</p>
<p><strong>The Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)
Directly Decimates Human Spleens and Lymph Nodes</strong></p>
<p>Review by Matthew D. Park</p>
<p>Revised by Miriam Merad</p>
<p>Keywords: COVID-19, SARS-CoV-2, spleen, lymph node, ACE2, macrophage</p>
<p><em>Main findings</em></p>
<p>It has been previously reported that COVID-19 patients exhibit severe
lymphocytopenia, but the mechanism through which this depletion occurs
has not been described. In order to characterize the cause and process
of lymphocyte depletion in COVID-19 patients, the authors performed
gross anatomical and <em>in situ</em> immune-histochemical analyses of spleens
and lymph nodes (hilar and subscapular) obtained from post-mortem
autopsies of 6 patients with confirmed positive viremia and 3 healthy
controls (deceased due to vehicle accidents).</p>
<p>Primary gross observations noted significant splenic and LN atrophy,
hemorrhaging, and necrosis with congestion of interstitial blood vessels
and large accumulation of mononuclear cells and massive lymphocyte
death. They found that CD68<sup>+</sup> CD169<sup>+</sup> cells in the spleens, hilar and
subscapular LN, and capillaries of these secondary lymphoid organs
expressed the ACE2 receptor and stain positive for the SARS-CoV-2
nucleoprotein (NP) antigen, while CD3<sup>+</sup> T cells and B220<sup>+</sup> B cells
lacked both the ACE2 receptor and SARS-CoV-2 NP antigen. ACE2<sup>+</sup> NP<sup>+</sup>
CD169<sup>+</sup> macrophages were positioned in the splenic marginal zone (MZ)
and in the marginal sinuses of LN, which suggests that these macrophages
were positioned to encounter invading pathogens first and may contribute
to virus dissemination.</p>
<p>Since SARS-CoV-2 does not directly infect lymphocytes, the authors
hypothesized that the NP<sup>+</sup> CD169<sup>+</sup> macrophages are responsible for
persistent activation of lymphocytes via Fas::FasL interactions that
would mediate activation-induced cell death (AICD). Indeed, the
expression of Fas was significantly higher in virus-infected tissue than
that of healthy controls, and TUNEL staining showed significant
lymphocytic apoptosis. Since pro-inflammatory cytokines like IL-6 and
TNF-α can also engage cellular apoptosis and necrosis, the authors
interrogated the cytokine expression of the secondary lymphoid organs
from COVID-19 patients; IL-6, not TNF-α, was elevated in virus-infected
splenic and lymph node tissues, compared to those of healthy controls,
and immunofluorescent staining showed that IL-6 is primarily produced by
the infected macrophages. <em>In vitro</em> infection of THP1 cells with
SARS-CoV-2 spike protein resulted in selectively increased <em>Il6</em>
expression, as opposed to <em>Il1b</em> and <em>Tnfa</em> transcription. Collectively,
the authors concluded that a combination of Fas up-regulation and IL-6
production by NP<sup>+</sup> CD169<sup>+</sup> macrophages induce AICD in lymphocytes in
secondary lymphoid organs, resulting in lymphocytopenia.</p>
<p>In summary, this study reports that CD169<sup>+</sup> macrophages in the splenic
MZ, subscapular LN, and the lining capillaries of the secondary lymphoid
tissues express ACE2 and are susceptible to SARS-CoV-2 infection. The
findings point to the potential role of these macrophages in viral
dissemination, immunopathology of these secondary lymphoid organs,
hyperinflammation and lymphopenia.</p>
<p><em>Limitations</em></p>
<p><em>Technical</em></p>
<p>A notable technical limitation is the small number of samples (n=6);
moreover, the analysis of these samples using multiplexed
immunohistochemistry and immunofluorescence do not necessarily provide
the depth of unbiased interrogation needed to better identify the cell
types involved.</p>
<p><em>Biological</em></p>
<p>The available literature and ongoing unpublished studies, including
single-cell experiments of spleen and LN from organ donors, do not
indicate that ACE2 is expressed by macrophages; however, it remains
possible that ACE2 expression may be triggered by type I IFN in COVID-19
patients. Importantly, the SARS-CoV-2 NP staining of the macrophages
does not necessarily reflect direct infection of these macrophages;
instead, positive staining only indicates that these macrophages carry
SARS-CoV-2 NP as antigen cargo, which may have been phagocytosed. Direct
viral culture of macrophages isolated from the secondary lymphoid organs
with SARS-CoV-2 is required to confirm the potential for direct
infection of macrophages by SARS-CoV-2. Additionally, it is important to
note that the low to negligible viremia reported in COVID-19 patients
to-date does not favor a dissemination route via the blood, as suggested
by this study, which would be necessary to explain the presence of
virally infected cells in the spleen.</p>
<p><em>Relevance</em></p>
<p>Excess inflammation in response to SARS-CoV-2 infection is characterized
by cytokine storm in many COVID-19 patients. The contribution of this
pathology to the overall fatality rate due to COVID-19, not even
necessarily directly due to SARS-CoV-2 infection, is significant. A
better understanding of the full effect and source of some of these
major cytokines, like IL-6, as well as the deficient immune responses,
like lymphocytopenia, is urgently needed. In this study, the authors
report severe tissue damage in spleens and lymph nodes of COVID-19
patients and identify the role that CD169<sup>+</sup> macrophages may play in the
hyperinflammation and lymphocytopenia that are both characteristic of
the disease. It may, therefore, be important to note the effects that
IL-6 inhibitors like Tocilizumab and Sarilumab may specifically have on
splenic and LN function. It is important to note that similar
observations of severe splenic and LN necrosis and inflammation in
patients infected with SARS-CoV-1 further support the potential
importance and relevance of this study.</p>
<h2 data-number="14.92" id="cigarette-smoke-triggers-the-expansion-of-a-subpopulation-of-respiratory-epithelial-cells-that-express-the-sars-cov-2-receptor-ace2"><span class="header-section-number">14.92</span> Cigarette smoke triggers the expansion of a subpopulation of respiratory epithelial cells that express the SARS-CoV-2 receptor ACE2</h2>
<p><span class="citation" data-cites="t4vofknR">(<a href="#ref-t4vofknR" role="doc-biblioref">2223</a>)</span></p>
<h3 data-number="14.92.1" id="keywords-92"><span class="header-section-number">14.92.1</span> Keywords</h3>
<ul>
<li>CoVID-19</li>
<li>2019-nCoV</li>
<li>SARS-CoV-2</li>
<li>respiration</li>
<li>cigarette</li>
<li>ACE2</li>
<li>lung</li>
</ul>
<h3 data-number="14.92.2" id="main-findings-88"><span class="header-section-number">14.92.2</span> Main Findings</h3>
<ul>
<li><p>Study uses scRNAseq, bulk seq data and air-liquid interface culture
experiments to show that cigarette smoke causes a dose-dependent
upregulation of ACE2 in mouse and human lungs (transplantation,
tumor resection, or IPF datasets).</p></li>
<li><p>ACE2 was not up-regulated in patients with asthma or
lung-sarcoidosis or in mouse models of cystic fibrosis or carcinogen
exposure.</p></li>
<li><p>Cathepsin B (alternate protease involved in viral entry) is
increased in smoke-exposed mouse or human lungs.</p></li>
<li><p>Smoke triggers a protective expansion of mucus-secreting MUC5AC+
goblet and SCGB1A1+ club cells; ACE2 presence in these cells is
increased upon smoke exposure.</p></li>
</ul>
<h3 data-number="14.92.3" id="limitations-90"><span class="header-section-number">14.92.3</span> Limitations:</h3>
<ul>
<li><p>Long-term smokers usually have several co-morbidities including
immune dysfunction, which can affect interpretation of CoV-2
susceptibility in these datasets. Ideally, analyses can control for
major co-morbidities across smokers and non-smokers (immune
suppression, cardiovascular disease and atherosclerosis).</p></li>
<li><p>Hyperplasia of ACE2+ goblet cells upon smoking needs to be separated
from ACE2 upregulation in existing goblet cells.</p></li>
<li><p>ACE2 expression increase alone does not confirm increased viral
entry into goblet cells; future studies with air-liquid interface
cultures testing CoV-2 infectivity in <em>ex vivo</em> epithelial cells
from human epithelial lines, ex vivo samples or hACE2 mice will be
very informative.</p></li>
</ul>
<h3 data-number="14.92.4" id="significance-84"><span class="header-section-number">14.92.4</span> Significance</h3>
<ul>
<li><p>This study may partially explain why smokers are more likely to
develop severe SARS-CoV-2 infections. Also, the reversibility of
ACE2 expression upon smoking cessation suggests that quitting
smoking could lessen CoV-2 susceptibility.</p></li>
<li><p>Absence of ACE2 upregulation in other lung inflammation pathologies
implies CoV-2 susceptibility might be smoking-specific, and not
fibrosis-specific.</p></li>
<li><p>Another preprint showed ACE2 expression increases in lung of
patients with CoV-2 co-morbidities such as hypertension <span class="citation" data-cites="wjtVy2v0">(<a href="#ref-wjtVy2v0" role="doc-biblioref">2196</a>)</span>; these studies
collectively paint a better picture of CoV-2 susceptibility before
actual experiments can be carried out.</p></li>
</ul>
<h3 data-number="14.92.5" id="credit-89"><span class="header-section-number">14.92.5</span> Credit</h3>
<p><em>Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.</em></p>
<h2 data-number="14.93" id="the-comparative-superiority-of-igm-igg-antibody-test-to-real-time-reverse-transcriptase-pcr-detection-for-sars-cov-2-infection-diagnosis"><span class="header-section-number">14.93</span> The comparative superiority of IgM-IgG antibody test to real-time reverse transcriptase PCR detection for SARS-CoV-2 infection diagnosis</h2>
<p>Liu et al. <em>medRxiv</em>.
<span class="citation" data-cites="k1E3DuFN">(<a href="#ref-k1E3DuFN" role="doc-biblioref">2224</a>)</span></p>
<h3 data-number="14.93.1" id="keywords-93"><span class="header-section-number">14.93.1</span> Keywords</h3>
<ul>
<li>IgM/IgG antibody test</li>
<li>nucleic acid test</li>
<li>SARS-CoV-2 detection</li>
</ul>
<h3 data-number="14.93.2" id="main-findings-89"><span class="header-section-number">14.93.2</span> Main Findings</h3>
<p>The study compares IgM and IgG antibody testing to RT-PCR detection of
SARS-CoV-2 infection. 133 patients diagnosed with SARS-CoV-2 in Renmin
Hospital (Wuhan University, China) were analyzed. The positive ratio was
78.95% (105/133) in IgM antibody test (SARS-CoV-2 antibody detection kit
from YHLO Biotech) and 68.42% (91/133) in RT-PCR (SARS-CoV-2 ORF1ab/N
qPCR detection kit). There were no differences in the sensitivity of
SARS-CO-V2 diagnosis in patients grouped according to disease severity.
For example, IgG responses were detected in 93.18% of moderate cases,
100% of severe cases and 97.3% of critical cases. In sum, positive
ratios were higher in antibody testing compared to RT-PCR detection,
demonstrating a higher detection sensitivity of IgM-IgG testing for
patients hospitalized with COVID-19 symptoms.</p>
<h3 data-number="14.93.3" id="limitations-91"><span class="header-section-number">14.93.3</span> Limitations</h3>
<p>This analysis only included one-time point of 133 hospitalized patients,
and the time from symptom onset was not described. There was no
discussion about specificity of the tests and no healthy controls were
included. It would be important to perform similar studies with more
patients, including younger age groups and patients with mild symptoms
as well as asymptomatic individuals. It is critical to determine how
early after infection/symptom onset antibodies can be detected and the
duration of this immune response.</p>
<h3 data-number="14.93.4" id="significance-85"><span class="header-section-number">14.93.4</span> Significance</h3>
<p>The IgM-IgG combined testing is important to improve clinical
sensitivity and diagnose COVID-19 patients. The combined antibody test
shows higher sensitivity than individual IgM and IgG tests or nucleic
acid-based methods, at least in patients hospitalized with symptoms.</p>
<h3 data-number="14.93.5" id="credit-90"><span class="header-section-number">14.93.5</span> Credit</h3>
<p><em>Review by Erica Dalla as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.</em></p>
<p><strong>Title:</strong> Lectin-like Intestinal Defensin Inhibits 2019-nCoV Spike
binding to ACE2</p>
<p><strong>Immunology keywords:</strong> defensins, spike protein, intestinal Paneth
cells, ACE2 binding</p>
<p><strong>Main Findings:</strong></p>
<p>Human ACE2 was previously identified as the host receptor for
SARS-CoV-2. Despite ACE2 being expressed in both lung alveolar
epithelial cells and small intestine enterocytes, respiratory problems
are the most common symptom after viral infection while intestinal
symptoms are much less frequent. Thus, the authors here investigate the
biology behind the observed protection of the intestinal epithelium from
SARS-CoV-2. Human defensin 5 (HD5), produced by Paneth cells in the
small intestine, was shown to interact with human ACE2, with a binding
affinity of 39.3 nM by biolayer interferometry (BLI). A blocking
experiment using different doses of HD5 coating ACE2 showed that HD5
lowered viral spike protein S1 binding to ACE2. Further, a molecular
dynamic simulation demonstrated a strong intermolecular interaction
between HD5 and the ACE2 ligand binding domain. To test HD5 inhibitory
effect on S1 binding to ACE2, human intestinal epithelium Caco-2 cells
were preincubated with HD5. Preincubation strongly reduced adherence of
S1 to surface of cells. HD5 was effective at a concentration as low as
10 µg/mL, comparable to the concentration found in the intestinal fluid.</p>
<p><strong>Limitations:</strong></p>
<p>The study focuses exclusively on intestinal cells. However, HD5 could
have been tested to block ACE2-S1 binding in human lung epithelial cells
as a potential treatment strategy. It would be useful to know whether
HD5 could also prevent viral entry in lung cells.</p>
<p><strong>Relevance:</strong></p>
<p>This work provides the first understanding of the different efficiency
of viral entry and infection among ACE2-expressing cells and tissues.
Specifically, the authors show that human defensin 5 produced in the
small intestine is able to block binding between S1 and ACE2 necessary
for viral entry into cells. The study provides a plausible explanation
on why few patients show intestinal symptoms and suggests that patients
with intestinal disease that decrease defensins’ production may be more
susceptible to SARS-CoV-2. It also indicates that HD5 could be used as a
molecule to be exogenously administered to patients to prevent viral
infection in lung epithelial cells.</p>
<p><strong>Title: </strong></p>
<p><strong>Susceptibility of ferrets, cats, dogs and different domestic animals
to SARS-coronavirus-2</strong></p>
<p><strong>Immunology keywords:</strong> SARS-CoV-2, ferret, cat, laboratory animal,
domestic animals</p>
<p><strong>The main finding of the article: </strong></p>
<p>This study evaluated the susceptibility of different model laboratory
animals (ferrets), as well as companion (cats and dogs), and domestic
animals (pigs, chickens and ducks) to SARS-CoV-2. They tested infection
with two SARS-CoV2 isolates, one from an environmental sample collected
in the Huanan Seafood Market in Wuhan (F13-E) and the other from a human
patient in Wuhan (CTan-H).</p>
<p><em>Ferrets</em> were inoculated with either of the two viruses by intranasal
route with 10<sup>5</sup> pfu, and the viral replication was evaluated. Two
ferrets from each group were euthanized on day 4 post infection (p.i.).
AT day 4 p.i., viral RNA and infectious viruses were detected only in
upper respiratory tract (nasal turbinate, upper palate, tonisls, but not
in the trachea, lungs or other tissues. Viral RNA and virus titer in the
remaining ferrets were monitored in nasal washes and rectal swabs on
days 2, 4, 6, 8 and 10 p.i. Viral RNA and infectious viruses were
detected in nasal washes until day 8 p.i. One ferret in each group
developed fever and loss of appetite on days 10 and 12 p.i., however,
viral RNA was practically undetactable. These two ferrets showed severe
lymphoplasmacytic perivasculitis and vasculitis in the lungs and lower
antibody titers compare to other 4 ferrets.</p>
<p><em>Cats</em>. Five <em>s</em>ubadult 8-month-old domestic cats were inoculated with
CTan-h virus and three uninfected cats were placed in a cage adjacent to
each of the infected cats to monitor respiratory droplet transmission.
Viral RNA was detected in the upper respiratory organs from all infected
cats and in one out of three exposed cats. All infected (inoculated and
exposed) cats developed elevated antibodies against SARS-CoV2. Viral
replication studies with juvenile cats (70-100 days) revealed massive
lesions in the nasal and tracheal mucosa epithelium and lungs of two
inoculated cats which died or were euthanized on day 3 p.i., and
infection in one out of three exposed cats. These results indicated
SARS-CoV2 could replicate in cats, that juvenile cats were more
susceptible that adults, and theat SARS-CoV2 could be transmit via
respiratory droplets between cats.</p>
<p><em>Dogs and others.</em> Five 3-month-old beagle dogs were inoculated and
housed with two uninoculated beagles in a room. Two virus inoculated
dogs seroconverted, but others including two contact dogs were all
seronegative for SARS-CoV2 and infectious virus was not detected in any
swabs collected. Viral RNA was not detected in swabs from pigs,
chickens, and ducks inoculated or contacted. These results indicated
that dogs, pigs, chickens, and ducks might have low or no susceptibility
to SARS-CoV2.</p>
<p><strong>Critical analysis of the study:</strong></p>
<p>This manuscript describes the viral replication and clinical symptoms of
SARS-CoV2 infection in ferrets, and the SARS-CoV2 infection and
transmission in cats. Clinical and pathological analysis was not
performed in cats, therefore the correlation of virus titer with
symptoms severity in the adult and juvenile cats could not be
determined.</p>
<p><strong>The importance and implications for the current epidemics:</strong></p>
<p>SARS-CoV-2 transmission to tigers, cats and dogs has been previously
reported. It should be noted that this manuscript did not evaluate the
transmission from cats to human. Nevertheless, it clearly showed higher
susceptibility of ferrets and domestic cats to SARS-CoV-2. This data
strongly indicates the need for surveillance of possible infection and
transmission of SARS-CoV-2 by domestic cats.</p>
<h2 data-number="14.94" id="virus-host-interactome-and-proteomic-survey-of-pmbcs-from-covid-19-patients-reveal-potential-virulence-factors-influencing-sars-cov-2-pathogenesis"><span class="header-section-number">14.94</span> Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis</h2>
<p>Li et al. <em>bioRxiv</em>.
<span class="citation" data-cites="NLoN4aYj">(<a href="#ref-NLoN4aYj" role="doc-biblioref">195</a>)</span></p>
<h3 data-number="14.94.1" id="keywords-94"><span class="header-section-number">14.94.1</span> Keywords</h3>
<ul>
<li>PBMC</li>
<li>virulence factors
– interaction network
– nsp9</li>
<li>nsp10
– NKRF</li>
</ul>
<h3 data-number="14.94.2" id="main-findings-90"><span class="header-section-number">14.94.2</span> Main findings</h3>
<p>The authors identified <strong>intra-viral protein-protein interactions</strong>
(PPI) with two different approaches: genome wide yeast-two hybrid (Y2H)
and co-immunoprecipitation (co-IP). A total of 58 distinct PPI were
characterized. A screen of <strong>viral-host PPI</strong> was also established by
overexpressing all the SARS-CoV-2 genes with a Flag epitope into HEK293
cells and purifying each protein complex. Interacting host proteins were
then identified by liquid chromatography and tandem mass spectrometry.
251 cellular proteins were identified, such as subunits of ATPase, 40S
ribosomal proteins, T complex proteins and proteasome related proteins,
for a total of 631 viral-host PPI. Several interactions suggesting
protein-mediated modulation of the immune response were identified,
highlighting the multiple ways SARS-CoV-2 might reprogram infected
cells.</p>
<p>Subsequently, the authors compared global proteome profiles of PBMCs
from healthy donors (n=6) with PBMC from COVID-19 patients with mild
(n=22) or severe (n=13) symptoms. 220 proteins were found to be
differentially expressed between <em>healthy donors and mild COVID-19
patients</em>, and a pathway analysis showed <strong>a general activation of the
innate immune response.</strong> 553 proteins were differentially expressed
between the PBMC of <em>mild and severe COVID-19 patients</em>, most of them
(95%) being downregulated in severe patients<em>.</em> Functional pathway
analysis indicated a defect of T cell activation and function in severe
COVID-19. There was also evidence suggesting reduced antibody secretion
by B cells. Together, these results suggest a <strong>functional decline of
adaptive immunity</strong>. A FACS analysis of PBMC from severe patients
indicated higher levels of IL6 and IL8 but not IL17 compared to mild
patients.</p>
<p>Finally, the authors focused on NKRF, an endogenous repressor of IL8/IL6
synthesis that was previously identified as interacting with SARS-Cov-2
nsp9,10,12,13 and 15. Individually expressed nsp9 and nsp10 (but not
nsp12, nsp13, nsp15) induced both IL6 and IL8 in lung epithelial A459
cells, indicating that nsp9 and nsp10 may be directly involved in the
induction of these pro-inflammatory cytokines. The authors finally argue
that nsp9 and nsp10 represent potential drug targets to prevent
over-production of IL6 and IL8 in infected cells, and reducing the
over-activation of neutrophils.</p>
<h3 data-number="14.94.3" id="limitations-92"><span class="header-section-number">14.94.3</span> Limitations</h3>
<p>First, the authors seem to have forgotten to include the extended data
in the manuscript, and their proteomic data does not seem to be publicly
available for the moment, which limits greatly our analysis of their
results.</p>
<p>While this work provides important data on host and viral PPI, only 19
interactions were identified by Y2H system but 52 with co-IP. The
authors do not comment about what could lead to such differences between
the two techniques and they don’t specify whether they detected the same
interactions using the two techniques.</p>
<p>Moreover, the PBMC protein quantification was performed comparing bulk
PBMC. Consequently, protein differences likely reflect differences in
cell populations rather than cell-intrinsic differences in protein
expression. While this analysis is still interesting, a similar
experiment performed on pre-sorted specific cell populations would allow
measuring proteome dynamics at a higher resolution.</p>
<p>Finally, the authors did not discussed their results in regards to
another SARS-CoV-2 interactome of host-viral PPI that had been published
previously<sup>1</sup>. This study reported 332 host-virus PPI, but no
interaction of viral proteins with NKRF was found. Some interactions
were found in both studies (eg. N and G3BP1, Orf6 and RAE1). However,
the time point used to lyse the cells were different (40h previously vs
72h here), which could explain some of the differences.</p>
<h3 data-number="14.94.4" id="relevance"><span class="header-section-number">14.94.4</span> Relevance</h3>
<p>The identification of many interactions between intra-viral and
host-virus PPI provides an overview of host protein and pathways that
are modulated by SARS-CoV-2, which can lead to the identification of
potential targets for drug development.</p>
<p>In the model proposed by the authors, nsp9 and nsp10 from SARS-Cov-2
induce an over-expression of IL6 and IL8 by lung epithelial cells, which
recruits neutrophils and could lead to an excess in lung infiltration.
Nsp9 has been shown to be essential for viral replication for
SARS-Cov-1<sup>2</sup>, and shares a 97% homology with nsp9 from SARS-Cov-2<sup>3</sup>.
Further, nsp9 crystal structure was recently solved<sup>3</sup>, which can help
to develop drug inhibitors if this protein is further confirmed as being
important for the virulence of SARS-Cov-2.</p>
<p>1. Gordon DE, Jang GM, Bouhaddou M, et al. A SARS-CoV-2-Human
Protein-Protein Interaction Map Reveals Drug Targets and Potential
Drug-Repurposing. <em>bioRxiv</em>. March 2020:2020.03.22.002386.
doi:10.1101/2020.03.22.002386</p>
<p>2. Miknis ZJ, Donaldson EF, Umland TC, Rimmer RA, Baric RS, Schultz LW.
Severe acute respiratory syndrome coronavirus nsp9 dimerization is
essential for efficient viral growth. <em>J Virol</em>. 2009;83(7):3007-3018.
doi:10.1128/JVI.01505-08</p>
<p>3. Littler DR, Gully BS, Colson RN, Rossjohn J. <em>Crystal Structure of
the SARS-CoV-2 Non-Structural Protein 9, Nsp9</em>. Molecular Biology; 2020.
doi:10.1101/2020.03.28.013920</p>
<p><strong><em>Title:</em></strong> Prediction and Evolution of B Cell Epitopes of Surface
Protein in SARS-CoV-2</p>
<p><strong><em>Keywords:</em></strong> SARS-CoV-2; Epitopes; Bioinformatics; Evolution</p>
<p><strong><em>Summary/Main findings: </em></strong></p>
<p>Lon et al. used a bioinformatic analysis of the published SARS-CoV-2
genomes in order to identify conserved linear and conformational B cell
epitopes found on the spike (S), envelope (E), and membrane (M)
proteins. The characterization of the surface proteins in this study
began with an assessment of the peptide sequences in order to identify
hydrophilicity indices and protein instability indices using the
Port-Param tool in ExPASy. All three surface proteins were calculated to
have an instability score under 40 indicating that they were stable.
Linear epitopes were identified on the basis of surface probability and
antigenicity, excluding regions of glycosylation. Using BepiPred 2.0
(with a cutoff value of 0.35) and ABCpred (with a cutoff value of 0.51),
4 linear B cell epitopes were predicted for the S protein, 1 epitope for
the E protein, and 1 epitope for the M protein. For structural analysis,
SARS-CoV assemblies published in the Protein Data Bank (PDB) acting as
scaffolds for the SARS-CoV-2 S and E amino acid sequences were used for
input into the SWISS-MODEL server in order to generate three-dimensional
structural models for the assessment of conformational epitopes. Using
Ellipro (cutoff value of 0.063) and SEPPA (cutoff value of 0.5), 1
conformational epitope was identified for the S protein and 1 epitope
was identified for the E protein, both of which are accessible on the
surface of the virus. Finally, the Consurf Server was used to assess the
conservation of these epitopes. All epitopes were conserved across the
published SARS-CoV-2 genomes and one epitope of the spike protein was
predicted to be the most stable across coronavirus phylogeny.</p>
<p><strong><em>Critical Analysis/Limitations:</em></strong></p>
<p>While this study provides a preliminary identification of potential
linear and conformational B cell epitopes, the translational value of
the epitopes described still needs extensive experimental validation to
ascertain whether these elicit a humoral immune response. The
conformational epitope analyses are also limited by the fact that they
are based off of predicted 3D structure from homology comparisons and
not direct crystal structures of the proteins themselves. Additionally,
since there was not a published M protein with a high homology to
SARS-CoV-2, no conformational epitopes were assessed for this protein.
Finally, while evolutionary conservation is an important consideration
in understanding the biology of the virus, conservation does not
necessarily imply that these sites neutralize the virus or aid in
non-neutralizing <em>in vivo</em> protection.</p>
<p><strong><em>Relevance/Implications:</em></strong></p>
<p>With further experimental validation that confirms that these epitopes
induce effective antibody responses to the virus, the epitopes described
can be used for the development of treatments and vaccines as well as
better characterize the viral structure to more deeply understand
pathogenesis.</p>
<h1 class="page_break_before" data-number="15" id="appendix-b"><span class="header-section-number">15</span> Appendix B</h1>
<p>Contributors were asked to complete this template to summarize and evaluate new papers related to diagnostics.</p>
<!--Hi there! Please use the template below as a guide for what information about this paper to include. It's ok to leave fields blank so that other contributors can fill them in later, or to add fields that you think are important.-->
<p>Title: Please edit the title to add the name of the paper after the colon</p>
<p>Please paste a link to the paper or a citation here:</p>
<p>Link:</p>
<p>What is the paper’s <a href="https://github.com/greenelab/covid19-review/blob/master/USAGE.md#citations">Manubot-style citation</a>?
<!--Leave the citation blank if you are unsure.--></p>
<p>Citation:</p>
<p>Please list some keywords (3-10) that help identify the relevance of this paper to COVID-19</p>
<ul>
<li>keyword 1 (replace me, copy and paste more than three if needed)</li>
<li>keyword 2 (replace me, copy and paste more than three if needed)</li>
<li>keyword 3 (replace me, copy and paste more than three if needed)</li>
</ul>
<p>Please note the publication / review status
<!--Replace one of the empty brackets below with [x].--></p>
<ul class="task-list">
<li><input type="checkbox" disabled="" />
Pre-print</li>
<li><input type="checkbox" disabled="" />
New Peer-Reviewed Paper</li>
<li><input type="checkbox" disabled="" />
Peer-Reviewed Paper Pre-2020</li>
</ul>
<p>Which areas of expertise are particularly relevant to the paper?
<!--Replace the empty brackets below with [x].--></p>
<ul class="task-list">
<li><input type="checkbox" disabled="" />
virology</li>
<li><input type="checkbox" disabled="" />
epidemiology</li>
<li><input type="checkbox" disabled="" />
biostatistics</li>
<li><input type="checkbox" disabled="" />
immunology</li>
<li><input type="checkbox" disabled="" />
pharmacology</li>
</ul>
<!-- The following questions are the things that we'll want to answer about each paper reporting a diagnostic. If you are ready to answer some of them now, please create the issue, then answer them in a comment. Otherwise, just create the issue so that a future reviewer can address them in the comments.-->
<p>Questions to answer about each paper:</p>
<p>Please provide 1-2 sentences introducing the study and its main findings</p>
<p>Study question(s) being investigated:</p>
<p>What type of testing scenario is being considered?</p>
<p>Is it a screening test (used for individuals with no symptoms), diagnostic test (used for individuals with symptoms), or definitive test (used for individuals who have had previous positive test results on diagnostic or screening tests)?</p>
<p>Study population:</p>
<p>What is the model system (e.g., human study, animal model, cell line study)?</p>
<p>What is the sample size?</p>
<p>What is the “pre-test” probability of disease in the study population (i.e., what is the anticipated prevalence of the disease?)</p>
<p>For human studies, the following are related to the pre-test probability:</p>
<p>What countries/regions are considered?</p>
<p>What is the age range, gender, other relevant characteristics?</p>
<p>What is the setting of the study (e.g., random sample of school children, retirement communities, etc.)?</p>
<p>What other specific inclusion-exclusion criteria are considered?</p>
<p>Reference test:</p>
<p>What reference test is considered as a “gold standard” comparator for the test under investigation?</p>
<p>Test assignment:</p>
<p>How are the new and reference tests assigned?</p>
<p>Examples of assignment could include: Recruited individuals have initially undergone neither the new nor the reference test; individuals tested as positive or negative by the reference test undergo the new test; individuals who have undertaken the new test are assessed by the standard test.</p>
<p>Are there any other relevant details about the study design?</p>
<p>Depending on how individuals are chosen, the test may be biasing towards more sick or less sick individuals or very clear-cut positive/negative cases.
Any factors that would influence this bias should be included here.</p>
<p>Test conduct:</p>
<p>How were tests performed?</p>
<p>Describe technical details of assays used, when measurements were taken and by whom, etc. for both the new and standard tests.</p>
<p>Test Assessment</p>
<p>Describe how individuals are classified as positive or negative, e.g. if a threshold is used.</p>
<p>Is there evidence that the test is precise/reproducible when repeated more than once?</p>
<p>Are measurements complete?</p>
<p>For example: Do some participants undergo just one test (the new or the reference test)?
Are there individuals with inconclusive results?</p>
<p>Results summary:</p>
<p>What are the estimated sensitivity, specificity, positive predictive value (PPV), and negative predicted value (NPV)?</p>
<p>Note that the PPV and NPV represent “post-test” probabilities of disease and are generally more meaningful than sensitivity and specificity.
Sometimes the post-test odds will be given instead.</p>
<p>What are the confidence bounds around these intervals?</p>
<p>Interpretation of results for study population:</p>
<p>How good is the test at ruling in or ruling out a disease based on the post-test probabilities?</p>
<p>Are there identified side affects of the test?</p>
<p>Is patient adherence to the test likely to be an issue?</p>
<p>Extrapolation of conclusions to other groups of individuals</p>
<p>How well is the test likely to work in populations with different pretest odds?</p>
<p>For example, if the prevalence is lower, then the PPV will also be lower, but the NPV will be higher.</p>
<p>How costly is the test?</p>
<p>How difficult is it to perform the test in different settings?</p>
<p>Could the test be combined with other existing tests?</p>
<p>Summary of reliability</p>
<p>1-2 sentences on concluding remarks, including summary of strengths, weaknesses, limitations.</p>
<p>Progress</p>
<p><em>Check off the components as they are completed. If the component is not applicable, check the box as well.</em></p>
<!--Check off the components that you have completed. It is fine if you have not completed all of them, we hope others will add on in the comments. Switch to "Preview" mode to click the box(es), or else replace the empty brackets below with [x].-->
<ul class="task-list">
<li><input type="checkbox" disabled="" />
1-2 sentences introducing the study and its main findings</li>
<li><input type="checkbox" disabled="" />
Describe testing scenario</li>
<li><input type="checkbox" disabled="" />
Describe model system</li>
<li><input type="checkbox" disabled="" />
Sample size</li>
<li><input type="checkbox" disabled="" />
Describe prevalnce of disease</li>
<li><input type="checkbox" disabled="" />
Describe countries/regions are considered</li>
<li><input type="checkbox" disabled="" />
Describe age range, gender, other relevant characteristics</li>
<li><input type="checkbox" disabled="" />
Describe setting of the study</li>
<li><input type="checkbox" disabled="" />
Describe other specific inclusion-exclusion criteria</li>
<li><input type="checkbox" disabled="" />
Describe “gold standard”</li>
<li><input type="checkbox" disabled="" />
Describe how the new and reference tests assigned</li>
<li><input type="checkbox" disabled="" />
Describe other relevant details about the study design</li>
<li><input type="checkbox" disabled="" />
Describe how the tests were performed</li>
<li><input type="checkbox" disabled="" />
Describe how individuals are classified as positive or negative</li>
<li><input type="checkbox" disabled="" />
Describe if test is precise/reproducible</li>
<li><input type="checkbox" disabled="" />
Describe whether measurements are complete</li>
<li><input type="checkbox" disabled="" />
What are the estimated sensitivity, specificity, positive predictive value (PPV), and negative predicted value (NPV)?</li>
<li><input type="checkbox" disabled="" />
What are the confidence bounds around these intervals?</li>
<li><input type="checkbox" disabled="" />
Describe post-test probabilities</li>
<li><input type="checkbox" disabled="" />
Describe side affects of the test</li>
<li><input type="checkbox" disabled="" />
Describe patient adherence</li>
<li><input type="checkbox" disabled="" />
Describe how it will extrapolate</li>
<li><input type="checkbox" disabled="" />
How costly is the test?</li>
<li><input type="checkbox" disabled="" />
How difficult is it to perform the test in different settings?</li>
<li><input type="checkbox" disabled="" />
Could the test be combined with other existing tests?</li>
<li><input type="checkbox" disabled="" />
Summary of reliability</li>
</ul>
<h1 class="page_break_before" data-number="16" id="appendix-c"><span class="header-section-number">16</span> Appendix C</h1>
<p>Contributors were asked to complete this template to summarize and evaluate new papers related to therapeutics.</p>
<!--Hi there! Please use the template below as a guide for what information about this paper to include. It's ok to leave fields blank so that other contributors can fill them in later, or to add fields that you think are important.-->
<p>Title: Please edit the title to add the name of the paper after the colon</p>
<p>Please paste a link to the paper or a citation here:</p>
<p>Link:</p>
<p>What is the paper’s <a href="https://github.com/greenelab/covid19-review/blob/master/USAGE.md#citations">Manubot-style citation</a>?
<!--Leave the citation blank if you are unsure.--></p>
<p>Citation:</p>
<p>Please list some keywords (3-10) that help identify the relevance of this paper to COVID-19</p>
<ul>
<li>keyword 1 (replace me, copy and paste more than three if needed)</li>
<li>keyword 2 (replace me, copy and paste more than three if needed)</li>
<li>keyword 3 (replace me, copy and paste more than three if needed)</li>
</ul>
<p>Please note the publication / review status
<!--Replace one of the empty brackets below with [x].--></p>
<ul class="task-list">
<li><input type="checkbox" disabled="" />
Pre-print</li>
<li><input type="checkbox" disabled="" />
New Peer-Reviewed Paper</li>
<li><input type="checkbox" disabled="" />
Peer-Reviewed Paper Pre-2020</li>
</ul>
<p>Which areas of expertise are particularly relevant to the paper?
<!--Replace the empty brackets below with [x].--></p>
<ul class="task-list">
<li><input type="checkbox" disabled="" />
virology</li>
<li><input type="checkbox" disabled="" />
epidemiology</li>
<li><input type="checkbox" disabled="" />
biostatistics</li>
<li><input type="checkbox" disabled="" />
immunology</li>
<li><input type="checkbox" disabled="" />
pharmacology</li>
</ul>
<!-- The following questions are the things that we'll want to answer about each paper reporting a diagnostic. If you are ready to answer some of them now, please create the issue, then answer them in a comment. Otherwise, just create the issue so that a future reviewer can address them in the comments.-->
<p>Questions to answer about each paper:</p>
<p>Please provide 1-2 sentences introducing the study and its main findings</p>
<p>Study question(s) being investigated:</p>
<p>How many/what drugs/combinations are being considered?</p>
<p>What are the main hypotheses being tested?</p>
<p>Study population:</p>
<p>What is the model system (e.g., human study, animal model, cell line study)?</p>
<p>What is the sample size? If multiple groups are considered, give sample size for each group (including controls).</p>
<ul>
<li>number treated with treatment A</li>
<li>number treated with treatment B</li>
</ul>
<p>For human studies:</p>
<p>What countries/regions are considered?</p>
<p>What is the age range, gender, other relevant characteristics?</p>
<p>What is the setting of the study (random sample of school children, inpatient, outpatient, etc)?</p>
<p>What other specific inclusion-exclusion criteria are considered?</p>
<p>For example, do the investigators exclude patients with diagnosed neoplasms or patients over/under a certain age?</p>
<p>Treatment assignment:</p>
<p>How are treatments assigned?</p>
<p>For example, is it an interventional or an observational study?</p>
<p>Is the study randomized?</p>
<p>A study can be interventional but not randomized (e.g., a phase I or II clinical trial is interventional but often not randomized).</p>
<p>Provide other relevant details about the design.</p>
<p>This includes possible treatment stratification (e.g., within litters for animal studies, within hospitals for human studies), possible confounding variables (e.g., having a large age range of individuals), possible risks of bias and how they are addressed (e.g., is there masking in a clinical trial? how are individuals chosen in an observational study?).</p>
<p>Outcome Assessment:</p>
<p>Describe the outcome that is assessed and whether it is appropriate.</p>
<p>For example: Is the outcome assessed by a clinician or is it self-reported?
Is the outcome based on viral load or a functional measurement (e.g., respiratory function, discharge from hospital)?
What method is used to measure the outcome?
How long after a treatment is the outcome measured?</p>
<p>Are outcome measurements complete?</p>
<p>For example, are there individuals lost to follow up?</p>
<p>Are outcome measurements subject to various kinds of bias?</p>
<p>For example, a lack of masking in randomized clinical trials.</p>
<p>Statistical Methods Assessment:</p>
<p>What methods are used for inference?</p>
<p>For example, logistic regression, nonparametric methods.</p>
<p>Are the methods appropriate for the study?</p>
<p>For example, are clustered data treated independently or are clusters adjusted for, such as different hospitals or litters?</p>
<p>Are adjustments made for possible confounders?</p>
<p>For example, adjustment for age, sex, or comorbidities.</p>
<p>Results Summary:</p>
<p>What is the estimated association?</p>
<p>For example, is it an estimated odds ratio, a median difference in detected cases, etc?</p>
<p>What measures of confidence or statistical significance are provided?</p>
<p>For example, confidence intervals, p-values, and/or Bayes factors.</p>
<p>Interpretation of results for study population:</p>
<p>Can we make a causal interpretation for the individuals in the study of drug -&gt; outcome, such as “taking drug A improves likelihood of survival twofold over taking drug B.”</p>
<p>For example, with a well-performed animal study or randomized trial it is often possible to infer causality.
If is an observational study, does it match up with some of the Bradford Hill criteria? https://www.edwardtufte.com/tufte/hill https://en.wikipedia.org/wiki/Bradford_Hill_criteria</p>
<p>Are there identified side effects or interactions with other drugs?</p>
<p>For example, is the treatment known to cause liver damage or to not be prescribed for individuals with certain comorbities?</p>
<p>Are there specific subgroups with different findings?</p>
<p>For example, do individuals with a specific baseline seem to do particularly well? Be particularly cautious with respect to multiple testing here.</p>
<p>Extrapolation of conclusions to other groups of individuals not specifically included in the study:</p>
<p>If the study is an animal study, which animal and how relevant is that model?</p>
<p>Is the model system appropriate? Is there evidence from past use that it’s highly-relevant to therapeutic design in this context?</p>
<p>If it is a human study, what characteristics of the study population may support/limit extrapolation?</p>
<ul>
<li>Can results extrapolate easily to other similar groups? (e.g., same country, similar age groups)</li>
<li>What would happen if conditions are extended in terms of dose or duration?</li>
<li>Can results be extrapolated to other populations or in very different settings? (e.g., different age group, primary care setting vs emergency department etc)</li>
</ul>
<p>Summary of reliability</p>
<p>1-2 sentences on concluding remarks, including summary of strengths, weaknesses, limitations.</p>
<p>Progress</p>
<p><em>Check off the components as they are completed. If the component is not applicable, check the box as well.</em></p>
<!--Check off the components that you have completed. It is fine if you have not completed all of them, we hope others will add on in the comments. Switch to "Preview" mode to click the box(es), or else replace the empty brackets below with [x].-->
<ul class="task-list">
<li><input type="checkbox" disabled="" />
1-2 sentences introducing the study and its main findings</li>
<li><input type="checkbox" disabled="" />
Describe How many/what drugs/combinations are being considered</li>
<li><input type="checkbox" disabled="" />
Describe the model system</li>
<li><input type="checkbox" disabled="" />
What is the sample size?</li>
<li><input type="checkbox" disabled="" />
What countries/regions are considered</li>
<li><input type="checkbox" disabled="" />
What is the age range, gender, other relevant characteristics</li>
<li><input type="checkbox" disabled="" />
Describe study setting</li>
<li><input type="checkbox" disabled="" />
Describe other specific inclusion-exclusion criteria</li>
<li><input type="checkbox" disabled="" />
Describe how treatments are assigned</li>
<li><input type="checkbox" disabled="" />
Describe randomization (or not) and other relavent details about the design</li>
<li><input type="checkbox" disabled="" />
Describe the outcome that is assessed and whether it is appropriate.</li>
<li><input type="checkbox" disabled="" />
Describe whether the outcome measurements are complete</li>
<li><input type="checkbox" disabled="" />
Are outcome measurements subject to various kinds of bias?</li>
<li><input type="checkbox" disabled="" />
Describe methods used for inference</li>
<li><input type="checkbox" disabled="" />
Describe whether the methods are appropriate for the study</li>
<li><input type="checkbox" disabled="" />
Are adjustments made for possible confounders?</li>
<li><input type="checkbox" disabled="" />
Describe the estimated association</li>
<li><input type="checkbox" disabled="" />
What measures of confidence or statistical significance are provided?</li>
<li><input type="checkbox" disabled="" />
Describe whether a causal interpretation can be made</li>
<li><input type="checkbox" disabled="" />
Are there identified side effects or interactions with other drugs?</li>
<li><input type="checkbox" disabled="" />
Are there specific subgroups with different findings?</li>
<li><input type="checkbox" disabled="" />
If the study is an animal study, which animal and how relevant is that model?</li>
<li><input type="checkbox" disabled="" />
If it is a human study, what characteristics of the study population may support/limit extrapolation?</li>
<li><input type="checkbox" disabled="" />
Summary of reliability</li>
</ul>
<h1 class="page_break_before" data-number="17" id="appendix-d"><span class="header-section-number">17</span> Appendix D</h1>
<p>Contributors were asked to complete this template to summarize and evaluate new papers related to topics besides therapeutics and diagnostics.</p>
<!--Hi there! Please use the template below as a guide for what information about this paper to include. It's ok to leave fields blank so that other contributors can fill them in later, or to add fields that you think are important.-->
<p>Title: Please edit the title to add the name of the paper after the colon.</p>
<p>General Information
Please paste a link to the paper or a citation here:</p>
<p>Link:</p>
<p>What is the paper’s <a href="https://github.com/greenelab/covid19-review/blob/master/USAGE.md#citations">Manubot-style citation</a>?
<!--Leave the citation blank if you are unsure.--></p>
<p>Citation:</p>
<p>Is this paper primarily relevant to Background or Pathogenesis?
<!--Replace the empty brackets below with [x].
If it's primarily relevant to Diagnostics or Therapeutics, please go back and use the associated template--></p>
<ul class="task-list">
<li><input type="checkbox" disabled="" />
Background</li>
<li><input type="checkbox" disabled="" />
Pathogenesis</li>
<li><input type="checkbox" disabled="" />
Methods</li>
</ul>
<p>Please list some keywords (3-10) that help identify the relevance of this paper to COVID-19</p>
<ul>
<li>keyword 1 (replace me, copy and paste more than three if needed)</li>
<li>keyword 2 (replace me, copy and paste more than three if needed)</li>
<li>keyword 3 (replace me, copy and paste more than three if needed)</li>
</ul>
<p>Please note the publication / review status
<!--Replace one of the empty brackets below with [x].--></p>
<ul class="task-list">
<li><input type="checkbox" disabled="" />
Pre-print</li>
<li><input type="checkbox" disabled="" />
New Peer-Reviewed Paper</li>
<li><input type="checkbox" disabled="" />
Peer-Reviewed Paper Pre-2020</li>
</ul>
<p>Which areas of expertise are particularly relevant to the paper?
<!--Replace the empty brackets below with [x].--></p>
<ul class="task-list">
<li><input type="checkbox" disabled="" />
virology</li>
<li><input type="checkbox" disabled="" />
epidemiology</li>
<li><input type="checkbox" disabled="" />
biostatistics</li>
<li><input type="checkbox" disabled="" />
immunology</li>
<li><input type="checkbox" disabled="" />
pharmacology</li>
<li><input type="checkbox" disabled="" />
other:</li>
</ul>
<!-- STOP
Please fill out as much of the above as you feel able, then:
   (1) Select all the text below and cut it onto your clipboard
   (2) Submit this issue using the green button below
   (3) If you would like to submit a summary of the paper, paste the cut text from your clipboard to a new comment below and fill in the summary information
-->
<p>Summary</p>
<p>Suggested questions to answer about each paper:
- What did they analyze?
- What methods did they use?
- Does this paper study COVID-19, SARS-CoV-2, or a related disease and/or virus?
- What is the main finding (or a few main takeaways)?
- What does this paper tell us about the background and/or diagnostics/therapeutics for COVID-19 / SARS-CoV-2?
- Do you have any concerns about methodology or the interpretation of these results beyond this analysis?</p>
<p>Any comments or notes?</p>
<section class="footnotes footnotes-end-of-document" role="doc-endnotes">
<hr />
<ol>
<li id="fn1" role="doc-endnote"><p><a href="https://asapbio.org/preprints-and-covid-19" class="uri">https://asapbio.org/preprints-and-covid-19</a> as well as <a href="https://retractionwatch.com/retracted-coronavirus-covid-19-papers" class="uri">https://retractionwatch.com/retracted-coronavirus-covid-19-papers</a><a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn2" role="doc-endnote"><p><a href="https://depts.washington.edu/pandemicalliance/covid-19-literature-report/latest-reports" class="uri">https://depts.washington.edu/pandemicalliance/covid-19-literature-report/latest-reports</a><a href="#fnref2" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn3" role="doc-endnote"><p><a href="https://outbreaksci.prereview.org" class="uri">https://outbreaksci.prereview.org</a><a href="#fnref3" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn4" role="doc-endnote"><p><a href="https://asapbio.org/preprints-and-covid-19" class="uri">https://asapbio.org/preprints-and-covid-19</a><a href="#fnref4" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn5" role="doc-endnote"><p><a href="https://disqus.com/by/sinaiimmunologyreviewproject" class="uri">https://disqus.com/by/sinaiimmunologyreviewproject</a><a href="#fnref5" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn6" role="doc-endnote"><p><a href="https://rapidreviewscovid19.mitpress.mit.edu" class="uri">https://rapidreviewscovid19.mitpress.mit.edu</a><a href="#fnref6" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn7" role="doc-endnote"><p><a href="https://greenelab.github.io/covid19-review" class="uri">https://greenelab.github.io/covid19-review</a><a href="#fnref7" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn8" role="doc-endnote"><p><a href="https://casrai.org/credit" class="uri">https://casrai.org/credit</a><a href="#fnref8" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn9" role="doc-endnote"><p><a href="https://www.gitter.im" class="uri">https://www.gitter.im</a><a href="#fnref9" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn10" role="doc-endnote"><p><a href="https://greenelab.github.io/covid19-review" class="uri">https://greenelab.github.io/covid19-review</a><a href="#fnref10" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn11" role="doc-endnote"><p><a href="https://greenelab.github.io/covid19-review/manuscript.pdf" class="uri">https://greenelab.github.io/covid19-review/manuscript.pdf</a><a href="#fnref11" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn12" role="doc-endnote"><p>Vaccines: <a href="https://github.com/owid/covid-19-data" class="uri">https://github.com/owid/covid-19-data</a>; Clinical Trials: <a href="https://github.com/ebmdatalab/covid_trials_tracker-covid" class="uri">https://github.com/ebmdatalab/covid_trials_tracker-covid</a>; Cases and Deaths: <a href="https://github.com/CSSEGISandData/COVID-19" class="uri">https://github.com/CSSEGISandData/COVID-19</a><a href="#fnref12" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn13" role="doc-endnote"><p><a href="https://github.com/greenelab/covid19-review/blob/master/.github/workflows/update-external-resources.yaml" class="uri">https://github.com/greenelab/covid19-review/blob/master/.github/workflows/update-external-resources.yaml</a><a href="#fnref13" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn14" role="doc-endnote"><p><a href="https://github.com/greenelab/covid19-review/tree/external-resources" class="uri">https://github.com/greenelab/covid19-review/tree/external-resources</a><a href="#fnref14" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn15" role="doc-endnote"><p><a href="https://github.com/greenelab/covid19-review/blob/external-resources/environment.yml" class="uri">https://github.com/greenelab/covid19-review/blob/external-resources/environment.yml</a><a href="#fnref15" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn16" role="doc-endnote"><p><a href="https://forums.zotero.org/discussion/74933/import-from-clinical-trials-registry" class="uri">https://forums.zotero.org/discussion/74933/import-from-clinical-trials-registry</a> and <a href="https://forums.zotero.org/discussion/77721/add-reference-from-clinical-trials-org" class="uri">https://forums.zotero.org/discussion/77721/add-reference-from-clinical-trials-org</a><a href="#fnref16" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn17" role="doc-endnote"><p><a href="https://www.zotero.org" class="uri">https://www.zotero.org</a> and <a href="https://github.com/zotero/translation-server" class="uri">https://github.com/zotero/translation-server</a><a href="#fnref17" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn18" role="doc-endnote"><p><a href="https://github.com/zotero/translators/pull/2153" class="uri">https://github.com/zotero/translators/pull/2153</a><a href="#fnref18" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn19" role="doc-endnote"><p><a href="https://identifiers.org" class="uri">https://identifiers.org</a><a href="#fnref19" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn20" role="doc-endnote"><p><a href="https://pandoc.org" class="uri">https://pandoc.org</a><a href="#fnref20" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn21" role="doc-endnote"><p><a href="http://aspell.net" class="uri">http://aspell.net</a><a href="#fnref21" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn22" role="doc-endnote"><p><a href="https://github.com/pandoc/lua-filters/tree/master/spellcheck" class="uri">https://github.com/pandoc/lua-filters/tree/master/spellcheck</a><a href="#fnref22" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn23" role="doc-endnote"><p><a href="https://twitter.com/j_perkel/status/1245454628235309057" class="uri">https://twitter.com/j_perkel/status/1245454628235309057</a><a href="#fnref23" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn24" role="doc-endnote"><p>CONTRIBUTING.md and INSTRUCTIONS.md within the
repository<a href="#fnref24" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn25" role="doc-endnote"><p><a href="https://github.com/ismms-himc/covid-19_sinai_reviews" class="uri">https://github.com/ismms-himc/covid-19_sinai_reviews</a><a href="#fnref25" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn26" role="doc-endnote"><p><a href="https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series" class="uri">https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series</a><a href="#fnref26" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn27" role="doc-endnote"><p><a href="https://github.com/owid/covid-19-data" class="uri">https://github.com/owid/covid-19-data</a><a href="#fnref27" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
<!-- 
  Scite Plugin (third party)

  Adds a Scite badge next to each citation in the references section.
  See https://scite.ai/.
-->

<script
  async
  type="application/javascript"
  src="https://cdn.scite.ai/badge/scite-badge-latest.min.js"
></script>

<script type="module">
  // start script
  function start() {
    // if printing, exit and don't run badges
    if (window.matchMedia("print").matches) return;

    // get citation elements
    const query = ".references div[id*='ref-']";
    const citations = document.querySelectorAll(query);
    for (const citation of citations) addBadge(citation);
  }

  // add badge in citation element
  function addBadge(citation) {
    // get citation text
    const text = citation.innerText;
    // find doi string, eg "DOI: xxxxx/xxxxx"
    const match = text.match(/DOI:\s+(\S*)/) || {};
    const doi = match[1];
    if (!doi) return;
    // wait until citation in view to add badge
    new IntersectionObserver(([{ isIntersecting }], observer) => {
      if (isIntersecting) {
        makeBadge(citation, doi);
        observer.disconnect();
      }
    }).observe(citation);
  }

  // make badge by adding data attribute
  function makeBadge(citation, doi) {
    // create cite badge and put after citation
    const badge = document.createElement("div");
    badge.classList.add("scite-badge");
    badge.setAttribute("data-doi", doi);
    badge.setAttribute("data-layout", "horizontal");
    badge.setAttribute("data-show-zero", "true");
    citation.append(badge);
    // modify preceding citation <p> element
    const p = citation.querySelector("p");
    if (p) p.style.marginBottom = "0";
    // have scite scan for new badges
    window.__SCITE.insertBadges();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  .scite-badge {
    text-indent: 0;
    margin-top: 10px;
  }

  @media print {
    .scite-badge {
      display: none;
    }
  }
</style>
</body>
</html>
